<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="9228eddc-d2bc-43f6-97c5-18a1853496a2" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;issue:issue:pii\:S0092867420X00043;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;article:article:pii\:S0092867421000015;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="3D Genome of macaque fetal brain reveals evolutionary innovations during primate corticogenesis">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)00001-5">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/8f927550-a945-4d92-a51f-58e11ee2ffca/fx1.jpg">
    


        <meta property="og:description" content="High-resolution map of developing rhesus macaque brain genome architecture during
corticogenesis reveals human-specific chromatin structure and regulatory changes that
impact dendrite development.">
    











        <meta name="twitter:title" content="3D Genome of macaque fetal brain reveals evolutionary innovations during primate corticogenesis">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/8f927550-a945-4d92-a51f-58e11ee2ffca/fx1.jpg">
    


        <meta name="twitter:description" content="High-resolution map of developing rhesus macaque brain genome architecture during
corticogenesis reveals human-specific chromatin structure and regulatory changes that
impact dendrite development.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)00001-5">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>3D Genome of macaque fetal brain reveals evolutionary innovations during primate corticogenesis: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="3D Genome of macaque fetal brain reveals evolutionary innovations during primate corticogenesis"> <meta name="citation_date" content="2021/02/04"> <meta name="citation_online_date" content="2021/01/27"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="3"> <meta name="citation_firstpage" content="723"> <meta name="citation_lastpage" content="740.e21"> <meta name="citation_pmid" content="33508230"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421000015"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421000015"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421000015"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.01.001"> <meta name="citation_pmid" content="33508230"> <meta name="citation_author" content="Xin Luo"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="National Resource Center for Non-Human Primates, Kunming Primate Research Center, and National Research Facility for Phenotypic and Genetic Analysis of Model Animals (Primate Facility), Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650107, China"><meta name="citation_author" content="Yuting Liu"><meta name="citation_author_institution" content="Center for Bioinformatics, School of Life Sciences, Center for Statistical Science, Peking University, Beijing 100871, China"><meta name="citation_author" content="Dachang Dang"><meta name="citation_author_institution" content="NCMIS, CEMS, RCSDS, Academy of Mathematics and Systems Science, Chinese Academy of Sciences, Beijing 100190, China"><meta name="citation_author" content="Ting Hu"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China"><meta name="citation_author" content="Yingping Hou"><meta name="citation_author_institution" content="Peking-Tsinghua Center for Life Sciences, Academy for Advanced Interdisciplinary Studies, Peking University, Beijing 100871, China"><meta name="citation_author" content="Xiaoyu Meng"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China"><meta name="citation_author" content="Fengyun Zhang"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China"><meta name="citation_author" content="Tingting Li"><meta name="citation_author_institution" content="Center for Bioinformatics, School of Life Sciences, Center for Statistical Science, Peking University, Beijing 100871, China"><meta name="citation_author" content="Can Wang"><meta name="citation_author_institution" content="NCMIS, CEMS, RCSDS, Academy of Mathematics and Systems Science, Chinese Academy of Sciences, Beijing 100190, China"><meta name="citation_author" content="Min Li"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author" content="Haixu Wu"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China"><meta name="citation_author" content="Qiushuo Shen"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author" content="Yan Hu"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author" content="Xuerui Zeng"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China"><meta name="citation_author" content="Xiechao He"><meta name="citation_author_institution" content="National Resource Center for Non-Human Primates, Kunming Primate Research Center, and National Research Facility for Phenotypic and Genetic Analysis of Model Animals (Primate Facility), Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650107, China"><meta name="citation_author" content="Lanzhen Yan"><meta name="citation_author_institution" content="National Resource Center for Non-Human Primates, Kunming Primate Research Center, and National Research Facility for Phenotypic and Genetic Analysis of Model Animals (Primate Facility), Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650107, China"><meta name="citation_author" content="Shihua Zhang"><meta name="citation_author_email" content="zsh@amss.ac.cn"><meta name="citation_author_orcid" content="0000-0003-0192-7118"><meta name="citation_author_institution" content="NCMIS, CEMS, RCSDS, Academy of Mathematics and Systems Science, Chinese Academy of Sciences, Beijing 100190, China"><meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author" content="Cheng Li"><meta name="citation_author_email" content="cheng_li@pku.edu.cn"><meta name="citation_author_institution" content="Center for Bioinformatics, School of Life Sciences, Center for Statistical Science, Peking University, Beijing 100871, China"><meta name="citation_author" content="Bing Su"><meta name="citation_author_email" content="sub@mail.kiz.ac.cn"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China"><meta name="citation_author_institution" content="National Resource Center for Non-Human Primates, Kunming Primate Research Center, and National Research Facility for Phenotypic and Genetic Analysis of Model Animals (Primate Facility), Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650107, China"><meta name="citation_keywords" content="brain; chromatin structure; TAD; loop; subplate; macaque; brain evolution; 3D genome; corticogenesis; EPHA7"><meta name="citation_reference" content="citation_title=Neuronal Ig/Caspr recognition promotes the formation of axoaxonic synapses in mouse spinal cord; citation_author=S. Ashrafi; citation_author=J.N. Betley; citation_author=J.D. Comer; citation_author=S. Brenner-Morton; citation_author=V. Bar; citation_author=Y. Shimoda; citation_author=K. Watanabe; citation_author=E. Peles; citation_author=T.M. Jessell; citation_author=J.A. Kaltschmidt; citation_journal_title=Neuron; citation_volume=81; citation_firstpage=120; citation_lastpage=129; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Hi-C 2.0: An optimized Hi-C procedure for high-resolution genome-wide mapping of chromosome conformation; citation_author=H. Belaghzal; citation_author=J. Dekker; citation_author=J.H. Gibcus; citation_journal_title=Methods; citation_volume=123; citation_firstpage=56; citation_lastpage=65; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=OligoMiner provides a rapid, flexible environment for the design of genome-scale oligonucleotide in&nbsp;situ hybridization probes; citation_author=B.J. Beliveau; citation_author=J.Y. Kishi; citation_author=G. Nir; citation_author=H.M. Sasaki; citation_author=S.K. Saka; citation_author=S.C. Nguyen; citation_author=C.T. Wu; citation_author=P. Yin; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=115; citation_firstpage=E2183; citation_lastpage=E2192; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Fast-evolving noncoding sequences in the human genome; citation_author=C.P. Bird; citation_author=B.E. Stranger; citation_author=M. Liu; citation_author=D.J. Thomas; citation_author=C.E. Ingle; citation_author=C. Beazley; citation_author=W. Miller; citation_author=M.E. Hurles; citation_author=E.T. Dermitzakis; citation_journal_title=Genome Biol.; citation_volume=8; citation_firstpage=R118; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Multiscale 3D Genome Rewiring during Mouse Neural Development; citation_author=B. Bonev; citation_author=N. Mendelson Cohen; citation_author=Q. Szabo; citation_author=L. Fritsch; citation_author=G.L. Papadopoulos; citation_author=Y. Lubling; citation_author=X. Xu; citation_author=X. Lv; citation_author=J.P. Hugnot; citation_author=A. Tanay; citation_author=G. Cavalli; citation_journal_title=Cell; citation_volume=171; citation_firstpage=557; citation_lastpage=572.e24; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A genome-wide screen for noncoding elements important in primate evolution; citation_author=E.C. Bush; citation_author=B.T. Lahn; citation_journal_title=BMC Evol. Biol.; citation_volume=8; citation_firstpage=17; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Fast-Evolving Human-Specific Neural Enhancers Are Associated with Aging-Related Diseases; citation_author=H. Chen; citation_author=C. Li; citation_author=Z. Zhou; citation_author=H. Liang; citation_journal_title=Cell Syst.; citation_volume=6; citation_firstpage=604; citation_lastpage=611.e4; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=EphA7 signaling guides cortical dendritic development and spine maturation; citation_author=M.A. Clifford; citation_author=W. Athar; citation_author=C.E. Leonard; citation_author=A. Russo; citation_author=P.J. Sampognaro; citation_author=M.-S. Van der Goes; citation_author=D.A. Burton; citation_author=X. Zhao; citation_author=R.R. Lalchandani; citation_author=M. Sahin; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=111; citation_firstpage=4994; citation_lastpage=4999; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The 3D folding of metazoan genomes correlates with the association of similar repetitive elements; citation_author=A. Cournac; citation_author=R. Koszul; citation_author=J. Mozziconacci; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=245; citation_lastpage=255; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Condensin-driven remodelling of X chromosome topology during dosage compensation; citation_author=E. Crane; citation_author=Q. Bian; citation_author=R.P. McCord; citation_author=B.R. Lajoie; citation_author=B.S. Wheeler; citation_author=E.J. Ralston; citation_author=S. Uzawa; citation_author=J. Dekker; citation_author=B.J. Meyer; citation_journal_title=Nature; citation_volume=523; citation_firstpage=240; citation_lastpage=244; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The Dynamic Landscape of Open Chromatin during Human Cortical Neurogenesis; citation_author=L. de la Torre-Ubieta; citation_author=J.L. Stein; citation_author=H. Won; citation_author=C.K. Opland; citation_author=D. Liang; citation_author=D. Lu; citation_author=D.H. Geschwind; citation_journal_title=Cell; citation_volume=172; citation_firstpage=289; citation_lastpage=304.e18; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Neuroligins and neurexins: linking cell adhesion, synapse formation and cognitive function; citation_author=C. Dean; citation_author=T. Dresbach; citation_journal_title=Trends Neurosci.; citation_volume=29; citation_firstpage=21; citation_lastpage=29; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=DAVID: Database for Annotation, Visualization, and Integrated Discovery; citation_author=G. Dennis; citation_author=B.T. Sherman; citation_author=D.A. Hosack; citation_author=J. Yang; citation_author=W. Gao; citation_author=H.C. Lane; citation_author=R.A. Lempicki; citation_journal_title=Genome Biol.; citation_volume=4; citation_firstpage=3; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=The evolution and population diversity of human-specific segmental duplications; citation_author=M.Y. Dennis; citation_author=L. Harshman; citation_author=B.J. Nelson; citation_author=O. Penn; citation_author=S. Cantsilieris; citation_author=J. Huddleston; citation_author=F. Antonacci; citation_author=K. Penewit; citation_author=L. Denman; citation_author=A. Raja; citation_author=et al.; citation_journal_title=Nat. Ecol. Evol.; citation_volume=1; citation_firstpage=69; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Topological domains in mammalian genomes identified by analysis of chromatin interactions; citation_author=J.R. Dixon; citation_author=S. Selvaraj; citation_author=F. Yue; citation_author=A. Kim; citation_author=Y. Li; citation_author=Y. Shen; citation_author=M. Hu; citation_author=J.S. Liu; citation_author=B. Ren; citation_journal_title=Nature; citation_volume=485; citation_firstpage=376; citation_lastpage=380; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Mutations in Human Accelerated Regions Disrupt Cognition and Social Behavior; citation_author=R.N. Doan; citation_author=B.I. Bae; citation_author=B. Cubelos; citation_author=C. Chang; citation_author=A.A. Hossain; citation_author=S. Al-Saad; citation_author=N.M. Mukaddes; citation_author=O. Oner; citation_author=M. Al-Saffar; citation_author=S. Balkhy; citation_author=Homozygosity Mapping Consortium for Autism; citation_author=et al.; citation_journal_title=Cell; citation_volume=167; citation_firstpage=341; citation_lastpage=354.e12; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A. Dobin; citation_author=C.A. Davis; citation_author=F. Schlesinger; citation_author=J. Drenkow; citation_author=C. Zaleski; citation_author=S. Jha; citation_author=P. Batut; citation_author=M. Chaisson; citation_author=T.R. Gingeras; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=15; citation_lastpage=21; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Juicer Provides a One-Click System for Analyzing Loop-Resolution Hi-C Experiments; citation_author=N.C. Durand; citation_author=M.S. Shamim; citation_author=I. Machol; citation_author=S.S. Rao; citation_author=M.H. Huntley; citation_author=E.S. Lander; citation_author=E.L. Aiden; citation_journal_title=Cell Syst.; citation_volume=3; citation_firstpage=95; citation_lastpage=98; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Large-scale imputation of epigenomic datasets for systematic annotation of diverse human tissues; citation_author=J. Ernst; citation_author=M. Kellis; citation_journal_title=Nat. Biotechnol.; citation_volume=33; citation_firstpage=364; citation_lastpage=376; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Chromatin features constrain structural variation across evolutionary timescales; citation_author=G. Fudenberg; citation_author=K.S. Pollard; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_firstpage=2175; citation_lastpage=2180; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Comprehensive identification and analysis of human accelerated regulatory DNA; citation_author=R.M. Gittelman; citation_author=E. Hun; citation_author=F. Ay; citation_author=J. Madeoy; citation_author=L. Pennacchio; citation_author=W.S. Noble; citation_author=R.D. Hawkins; citation_author=J.M. Akey; citation_journal_title=Genome Res.; citation_volume=25; citation_firstpage=1245; citation_lastpage=1255; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=FIMO: scanning for occurrences of a given motif; citation_author=C.E. Grant; citation_author=T.L. Bailey; citation_author=W.S. Noble; citation_journal_title=Bioinformatics; citation_volume=27; citation_firstpage=1017; citation_lastpage=1018; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=A simplified protocol for differentiation of electrophysiologically mature neuronal networks from human induced pluripotent stem cells; citation_author=N. Gunhanlar; citation_author=G. Shpak; citation_author=M. van der Kroeg; citation_author=L.A. Gouty-Colomer; citation_author=S.T. Munshi; citation_author=B. Lendemeijer; citation_author=M. Ghazvini; citation_author=C. Dupont; citation_author=W.J.G. Hoogendijk; citation_author=J. Gribnau; citation_author=et al.; citation_journal_title=Mol. Psychiatry; citation_volume=23; citation_firstpage=1336; citation_lastpage=1344; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Removing technical variability in RNA-seq data using conditional quantile normalization; citation_author=K.D. Hansen; citation_author=R.A. Irizarry; citation_author=Z. Wu; citation_journal_title=Biostatistics; citation_volume=13; citation_firstpage=204; citation_lastpage=216; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Nuclear factor one transcription factors: Divergent functions in developmental versus adult stem cell populations; citation_author=L. Harris; citation_author=L.A. Genovesi; citation_author=R.M. Gronostajski; citation_author=B.J. Wainwright; citation_author=M. Piper; citation_journal_title=Dev. Dyn.; citation_volume=244; citation_firstpage=227; citation_lastpage=238; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Refined DNase-seq protocol and data analysis reveals intrinsic bias in transcription factor footprint identification; citation_author=H.H. He; citation_author=C.A. Meyer; citation_author=S.S. Hu; citation_author=M.W. Chen; citation_author=C. Zang; citation_author=Y. Liu; citation_author=P.K. Rao; citation_author=T. Fei; citation_author=H. Xu; citation_author=H. Long; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=11; citation_firstpage=73; citation_lastpage=78; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Long-read assembly of the Chinese rhesus macaque genome and identification of ape-specific structural variants; citation_author=Y. He; citation_author=X. Luo; citation_author=B. Zhou; citation_author=T. Hu; citation_author=X. Meng; citation_author=P.A. Audano; citation_author=Z.N. Kronenberg; citation_author=E.E. Eichler; citation_author=J. Jin; citation_author=Y. Guo; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=4233; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Generation of mouse models of myeloid malignancy with combinatorial genetic lesions using CRISPR-Cas9 genome editing; citation_author=D. Heckl; citation_author=M.S. Kowalczyk; citation_author=D. Yudovich; citation_author=R. Belizaire; citation_author=R.V. Puram; citation_author=M.E. McConkey; citation_author=A. Thielke; citation_author=J.C. Aster; citation_author=A. Regev; citation_author=B.L. Ebert; citation_journal_title=Nat. Biotechnol.; citation_volume=32; citation_firstpage=941; citation_lastpage=946; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities; citation_author=S. Heinz; citation_author=C. Benner; citation_author=N. Spann; citation_author=E. Bertolino; citation_author=Y.C. Lin; citation_author=P. Laslo; citation_author=J.X. Cheng; citation_author=C. Murre; citation_author=H. Singh; citation_author=C.K. Glass; citation_journal_title=Mol. Cell; citation_volume=38; citation_firstpage=576; citation_lastpage=589; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Tbr1 regulates differentiation of the preplate and layer 6; citation_author=R.F. Hevner; citation_author=L. Shi; citation_author=N. Justice; citation_author=Y. Hsueh; citation_author=M. Sheng; citation_author=S. Smiga; citation_author=A. Bulfone; citation_author=A.M. Goffinet; citation_author=A.T. Campagnoni; citation_author=J.L. Rubenstein; citation_journal_title=Neuron; citation_volume=29; citation_firstpage=353; citation_lastpage=366; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Chromatin Decondensation by FOXP2 Promotes Human Neuron Maturation and Expression of Neurodevelopmental Disease Genes; citation_author=S.L. Hickey; citation_author=S. Berto; citation_author=G. Konopka; citation_journal_title=Cell Rep.; citation_volume=27; citation_firstpage=1699; citation_lastpage=1711; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The UCSC Genome Browser Database: update 2006; citation_author=A.S. Hinrichs; citation_author=D. Karolchik; citation_author=R. Baertsch; citation_author=G.P. Barber; citation_author=G. Bejerano; citation_author=H. Clawson; citation_author=M. Diekhans; citation_author=T.S. Furey; citation_author=R.A. Harte; citation_author=F. Hsu; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=34; citation_firstpage=D590; citation_lastpage=D598; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Development, evolution and pathology of neocortical subplate neurons; citation_author=A. Hoerder-Suabedissen; citation_author=Z. Molnár; citation_journal_title=Nat. Rev. Neurosci.; citation_volume=16; citation_firstpage=133; citation_lastpage=146; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Iterative correction of Hi-C data reveals hallmarks of chromosome organization; citation_author=M. Imakaev; citation_author=G. Fudenberg; citation_author=R.P. McCord; citation_author=N. Naumova; citation_author=A. Goloborodko; citation_author=B.R. Lajoie; citation_author=J. Dekker; citation_author=L.A. Mirny; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=999; citation_lastpage=1003; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The significance of the subplate for evolution and developmental plasticity of the human brain; citation_author=M. Judaš; citation_author=G. Sedmak; citation_author=I. Kostović; citation_journal_title=Front. Hum. Neurosci.; citation_volume=7; citation_firstpage=423; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The ChAHP Complex Counteracts Chromatin Looping at CTCF Sites that Emerged from SINE Expansions in Mouse; citation_author=L.J.T. Kaaij; citation_author=F. Mohn; citation_author=R.H. van der Weide; citation_author=E. de Wit; citation_author=M. Bühler; citation_journal_title=Cell; citation_volume=178; citation_firstpage=1437; citation_lastpage=1451.e14; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Role of subplate neurons in functional maturation of visual cortical columns; citation_author=P.O. Kanold; citation_author=P. Kara; citation_author=R.C. Reid; citation_author=C.J. Shatz; citation_journal_title=Science; citation_volume=301; citation_firstpage=521; citation_lastpage=525; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=The age of Alu subfamilies; citation_author=V. Kapitonov; citation_author=J. Jurka; citation_journal_title=J.&nbsp;Mol. Evol.; citation_volume=42; citation_firstpage=59; citation_lastpage=65; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=JASPAR 2018: update of the open-access database of transcription factor binding profiles and its web framework; citation_author=A. Khan; citation_author=O. Fornes; citation_author=A. Stigliani; citation_author=M. Gheorghe; citation_author=J.A. Castro-Mondragon; citation_author=R. van der Lee; citation_author=A. Bessy; citation_author=J. Chèneby; citation_author=S.R. Kulkarni; citation_author=G. Tan; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=D1; citation_firstpage=D260; citation_lastpage=D266; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation; citation_author=K.R. Kieffer-Kwon; citation_author=Z. Tang; citation_author=E. Mathe; citation_author=J. Qian; citation_author=M.H. Sung; citation_author=G. Li; citation_author=W. Resch; citation_author=S. Baek; citation_author=N. Pruett; citation_author=L. Grøntved; citation_author=et al.; citation_journal_title=Cell; citation_volume=155; citation_firstpage=1507; citation_lastpage=1520; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Evolution at two levels in humans and chimpanzees; citation_author=M.C. King; citation_author=A.C. Wilson; citation_journal_title=Science; citation_volume=188; citation_firstpage=107; citation_lastpage=116; citation_publication_date=1975; "><meta name="citation_reference" content="citation_title=New interpretation of laminar development of cerebral cortex synaptogenesis in different layers of neopallium in human fetus; citation_author=I. Kostovic; citation_author=M.E. Molliver; citation_journal_title=Anat. Rec.; citation_volume=178; citation_firstpage=395; citation_publication_date=1974; "><meta name="citation_reference" content="citation_title=Developmental history of the transient subplate zone in the visual and somatosensory cortex of the macaque monkey and human brain; citation_author=I. Kostovic; citation_author=P. Rakic; citation_journal_title=J.&nbsp;Comp. Neurol.; citation_volume=297; citation_firstpage=441; citation_lastpage=470; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=High-resolution comparative analysis of great ape genomes; citation_author=Z.N. Kronenberg; citation_author=I.T. Fiddes; citation_author=D. Gordon; citation_author=S. Murali; citation_author=S. Cantsilieris; citation_author=O.S. Meyerson; citation_author=J.G. Underwood; citation_author=B.J. Nelson; citation_author=M.J.P. Chaisson; citation_author=M.L. Dougherty; citation_author=et al.; citation_journal_title=Science; citation_volume=360; citation_firstpage=eaar6343; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Integrative analysis of 111 reference human epigenomes; citation_author=A. Kundaje; citation_author=W. Meuleman; citation_author=J. Ernst; citation_author=M. Bilenky; citation_author=A. Yen; citation_author=A. Heravi-Moussavi; citation_author=P. Kheradpour; citation_author=Z. Zhang; citation_author=J. Wang; citation_author=M.J. Ziller; citation_author=Roadmap Epigenomics Consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=518; citation_firstpage=317; citation_lastpage=330; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The SPCH1 region on human 7q31: genomic characterization of the critical interval and localization of translocations associated with speech and language disorder; citation_author=C.S. Lai; citation_author=S.E. Fisher; citation_author=J.A. Hurst; citation_author=E.R. Levy; citation_author=S. Hodgson; citation_author=M. Fox; citation_author=S. Jeremiah; citation_author=S. Povey; citation_author=D.C. Jamison; citation_author=E.D. Green; citation_author=et al.; citation_journal_title=Am. J. Hum. Genet.; citation_volume=67; citation_firstpage=357; citation_lastpage=368; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=A forkhead-domain gene is mutated in a severe speech and language disorder; citation_author=C.S. Lai; citation_author=S.E. Fisher; citation_author=J.A. Hurst; citation_author=F. Vargha-Khadem; citation_author=A.P. Monaco; citation_journal_title=Nature; citation_volume=413; citation_firstpage=519; citation_lastpage=523; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=WGCNA: an R package for weighted correlation network analysis; citation_author=P. Langfelder; citation_author=S. Horvath; citation_journal_title=BMC Bioinformatics; citation_volume=9; citation_firstpage=559; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B. Langmead; citation_author=S.L. Salzberg; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=357; citation_lastpage=359; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Software for computing and annotating genomic ranges; citation_author=M. Lawrence; citation_author=W. Huber; citation_author=H. Pagès; citation_author=P. Aboyoun; citation_author=M. Carlson; citation_author=R. Gentleman; citation_author=M.T. Morgan; citation_author=V.J. Carey; citation_journal_title=PLoS Comput. Biol.; citation_volume=9; citation_firstpage=e1003118; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Epigenetic maintenance of topological domains in the highly rearranged gibbon genome; citation_author=N.H. Lazar; citation_author=K.A. Nevonen; citation_author=B. O’Connell; citation_author=C. McCann; citation_author=R.J. O’Neill; citation_author=R.E. Green; citation_author=T.J. Meyer; citation_author=M. Okhovat; citation_author=L. Carbone; citation_journal_title=Genome Res.; citation_volume=28; citation_firstpage=983; citation_lastpage=997; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Deletion of 7q31.1 supports involvement of FOXP2 in language impairment: clinical report and review; citation_author=P.A. Lennon; citation_author=M.L. Cooper; citation_author=D.A. Peiffer; citation_author=K.L. Gunderson; citation_author=A. Patel; citation_author=S. Peters; citation_author=S.W. Cheung; citation_author=C.A. Bacino; citation_journal_title=Am. J. Med. Genet. A.; citation_volume=143A; citation_firstpage=791; citation_lastpage=798; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools; citation_author=H. Li; citation_author=B. Handsaker; citation_author=A. Wysoker; citation_author=T. Fennell; citation_author=J. Ruan; citation_author=N. Homer; citation_author=G. Marth; citation_author=G. Abecasis; citation_author=R. Durbin; citation_author=1000 Genome Project Data Processing Subgroup; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2078; citation_lastpage=2079; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The R package Rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads; citation_author=Y. Liao; citation_author=G.K. Smyth; citation_author=W. Shi; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_firstpage=e47; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Comprehensive mapping of long-range interactions reveals folding principles of the human genome; citation_author=E. Lieberman-Aiden; citation_author=N.L. van Berkum; citation_author=L. Williams; citation_author=M. Imakaev; citation_author=T. Ragoczy; citation_author=A. Telling; citation_author=I. Amit; citation_author=B.R. Lajoie; citation_author=P.J. Sabo; citation_author=M.O. Dorschner; citation_author=et al.; citation_journal_title=Science; citation_volume=326; citation_firstpage=289; citation_lastpage=293; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2; citation_author=M.I. Love; citation_author=W. Huber; citation_author=S. Anders; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=550; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Development and evolution of the human neocortex; citation_author=J.H. Lui; citation_author=D.V. Hansen; citation_author=A.R. Kriegstein; citation_journal_title=Cell; citation_volume=146; citation_firstpage=18; citation_lastpage=36; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Disruptions of topological chromatin domains cause pathogenic rewiring of gene-enhancer interactions; citation_author=D.G. Lupiáñez; citation_author=K. Kraft; citation_author=V. Heinrich; citation_author=P. Krawitz; citation_author=F. Brancati; citation_author=E. Klopocki; citation_author=D. Horn; citation_author=H. Kayserili; citation_author=J.M. Opitz; citation_author=R. Laxova; citation_author=et al.; citation_journal_title=Cell; citation_volume=161; citation_firstpage=1012; citation_lastpage=1025; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Reconstructing contiguous regions of an ancestral genome; citation_author=J. Ma; citation_author=L. Zhang; citation_author=B.B. Suh; citation_author=B.J. Raney; citation_author=R.C. Burhans; citation_author=W.J. Kent; citation_author=M. Blanchette; citation_author=D. Haussler; citation_author=W. Miller; citation_journal_title=Genome Res.; citation_volume=16; citation_firstpage=1557; citation_lastpage=1565; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Species-specific maturation profiles of human, chimpanzee and bonobo neural cells; citation_author=M.C. Marchetto; citation_author=B. Hrvoj-Mihic; citation_author=B.E. Kerman; citation_author=D.X. Yu; citation_author=K.C. Vadodaria; citation_author=S.B. Linker; citation_author=I. Narvaiza; citation_author=R. Santos; citation_author=A.M. Denli; citation_author=A.P. Mendes; citation_author=et al.; citation_journal_title=eLife; citation_volume=8; citation_firstpage=e37527; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Human-specific loss of regulatory DNA and the evolution of human-specific traits; citation_author=C.Y. McLean; citation_author=P.L. Reno; citation_author=A.A. Pollen; citation_author=A.I. Bassan; citation_author=T.D. Capellini; citation_author=C. Guenther; citation_author=V.B. Indjeian; citation_author=X. Lim; citation_author=D.B. Menke; citation_author=B.T. Schaar; citation_author=et al.; citation_journal_title=Nature; citation_volume=471; citation_firstpage=216; citation_lastpage=219; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Transcriptional landscape of the prenatal human brain; citation_author=J.A. Miller; citation_author=S.L. Ding; citation_author=S.M. Sunkin; citation_author=K.A. Smith; citation_author=L. Ng; citation_author=A. Szafer; citation_author=A. Ebbert; citation_author=Z.L. Riley; citation_author=J.J. Royall; citation_author=K. Aiona; citation_author=et al.; citation_journal_title=Nature; citation_volume=508; citation_firstpage=199; citation_lastpage=206; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Enhancer connectome in primary human cells identifies target genes of disease-associated DNA elements; citation_author=M.R. Mumbach; citation_author=A.T. Satpathy; citation_author=E.A. Boyle; citation_author=C. Dai; citation_author=B.G. Gowen; citation_author=S.W. Cho; citation_author=M.L. Nguyen; citation_author=A.J. Rubin; citation_author=J.M. Granja; citation_author=K.R. Kazane; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=1602; citation_lastpage=1612; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Canonical TGF-β Signaling Negatively Regulates Neuronal Morphogenesis through TGIF/Smad Complex-Mediated CRMP2 Suppression; citation_author=H. Nakashima; citation_author=K. Tsujimura; citation_author=K. Irie; citation_author=M. Ishizu; citation_author=M. Pan; citation_author=T. Kameda; citation_author=K. Nakashima; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=38; citation_firstpage=4791; citation_lastpage=4810; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Rapid and pervasive changes in genome-wide enhancer usage during mammalian development; citation_author=A.S. Nord; citation_author=M.J. Blow; citation_author=C. Attanasio; citation_author=J.A. Akiyama; citation_author=A. Holt; citation_author=R. Hosseini; citation_author=S. Phouanenavong; citation_author=I. Plajzer-Frick; citation_author=M. Shoukry; citation_author=V. Afzal; citation_author=et al.; citation_journal_title=Cell; citation_volume=155; citation_firstpage=1521; citation_lastpage=1531; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Enhancer hijacking activates GFI1 family oncogenes in medulloblastoma; citation_author=P.A. Northcott; citation_author=C. Lee; citation_author=T. Zichner; citation_author=A.M. Stütz; citation_author=S. Erkek; citation_author=D. Kawauchi; citation_author=D.J. Shih; citation_author=V. Hovestadt; citation_author=M. Zapatka; citation_author=D. Sturm; citation_author=et al.; citation_journal_title=Nature; citation_volume=511; citation_firstpage=428; citation_lastpage=434; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex; citation_author=T.J. Nowakowski; citation_author=A. Bhaduri; citation_author=A.A. Pollen; citation_author=B. Alvarado; citation_author=M.A. Mostajo-Radji; citation_author=E. Di Lullo; citation_author=M. Haeussler; citation_author=C. Sandoval-Espinosa; citation_author=S.J. Liu; citation_author=D. Velmeshev; citation_author=et al.; citation_journal_title=Science; citation_volume=358; citation_firstpage=1318; citation_lastpage=1323; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=hPSC Modeling Reveals that Fate Selection of Cortical Deep Projection Neurons Occurs in the Subplate; citation_author=M.Z. Ozair; citation_author=C. Kirst; citation_author=B.L. van den Berg; citation_author=A. Ruzo; citation_author=T. Rito; citation_author=A.H. Brivanlou; citation_journal_title=Cell Stem Cell; citation_volume=23; citation_firstpage=60; citation_lastpage=73.e6; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism; citation_author=N.N. Parikshak; citation_author=R. Luo; citation_author=A. Zhang; citation_author=H. Won; citation_author=J.K. Lowe; citation_author=V. Chandran; citation_author=S. Horvath; citation_author=D.H. Geschwind; citation_journal_title=Cell; citation_volume=155; citation_firstpage=1008; citation_lastpage=1021; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Oncogenic Notch Promotes Long-Range Regulatory Interactions within Hyperconnected 3D Cliques; citation_author=J. Petrovic; citation_author=Y. Zhou; citation_author=M. Fasolino; citation_author=N. Goldman; citation_author=G.W. Schwartz; citation_author=M.R. Mumbach; citation_author=S.C. Nguyen; citation_author=K.S. Rome; citation_author=Y. Sela; citation_author=Z. Zapataro; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=73; citation_firstpage=1174; citation_lastpage=1190.e12; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Static and Dynamic DNA Loops form AP-1-Bound Activation Hubs during Macrophage Development; citation_author=D.H. Phanstiel; citation_author=K. Van Bortle; citation_author=D. Spacek; citation_author=G.T. Hess; citation_author=M.S. Shamim; citation_author=I. Machol; citation_author=M.I. Love; citation_author=E.L. Aiden; citation_author=M.C. Bassik; citation_author=M.P. Snyder; citation_journal_title=Mol. Cell; citation_volume=67; citation_firstpage=1037; citation_lastpage=1048.e6; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A Single-Cell Transcriptomic Atlas of Human Neocortical Development during Mid-gestation; citation_author=D. Polioudakis; citation_author=L. de la Torre-Ubieta; citation_author=J. Langerman; citation_author=A.G. Elkins; citation_author=X. Shi; citation_author=J.L. Stein; citation_author=C.K. Vuong; citation_author=S. Nichterwitz; citation_author=M. Gevorgian; citation_author=C.K. Opland; citation_author=et al.; citation_journal_title=Neuron; citation_volume=103; citation_firstpage=785; citation_lastpage=801.e8; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=An RNA gene expressed during cortical development evolved rapidly in humans; citation_author=K.S. Pollard; citation_author=S.R. Salama; citation_author=N. Lambert; citation_author=M.-A. Lambot; citation_author=S. Coppens; citation_author=J.S. Pedersen; citation_author=S. Katzman; citation_author=B. King; citation_author=C. Onodera; citation_author=A. Siepel; citation_author=et al.; citation_journal_title=Nature; citation_volume=443; citation_firstpage=167; citation_lastpage=172; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=A candidate gene association study further corroborates involvement of contactin genes in autism; citation_author=M. Poot; citation_journal_title=Mol. Syndromol.; citation_volume=5; citation_firstpage=229; citation_lastpage=235; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Neuron-specific signatures in the chromosomal connectome associated with schizophrenia risk; citation_author=P. Rajarajan; citation_author=T. Borrman; citation_author=W. Liao; citation_author=N. Schrode; citation_author=E. Flaherty; citation_author=C. Casiño; citation_author=S. Powell; citation_author=C. Yashaswini; citation_author=E.A. LaMarca; citation_author=B. Kassim; citation_author=et al.; citation_journal_title=Science; citation_volume=362; citation_firstpage=eaat4311; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=deepTools: a flexible platform for exploring deep-sequencing data; citation_author=F. Ramírez; citation_author=F. Dündar; citation_author=S. Diehl; citation_author=B.A. Grüning; citation_author=T. Manke; citation_journal_title=Nucleic Acids Res.; citation_volume=42; citation_firstpage=W187-91; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping; citation_author=S.S. Rao; citation_author=M.H. Huntley; citation_author=N.C. Durand; citation_author=E.K. Stamenova; citation_author=I.D. Bochkov; citation_author=J.T. Robinson; citation_author=A.L. Sanborn; citation_author=I. Machol; citation_author=A.D. Omer; citation_author=E.S. Lander; citation_author=E.L. Aiden; citation_journal_title=Cell; citation_volume=159; citation_firstpage=1665; citation_lastpage=1680; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Evolutionary genomics. Evolutionary changes in promoter and enhancer activity during human corticogenesis; citation_author=S.K. Reilly; citation_author=J. Yin; citation_author=A.E. Ayoub; citation_author=D. Emera; citation_author=J. Leng; citation_author=J. Cotney; citation_author=R. Sarro; citation_author=P. Rakic; citation_author=J.P. Noonan; citation_journal_title=Science; citation_volume=347; citation_firstpage=1155; citation_lastpage=1159; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes; citation_author=A.L. Sanborn; citation_author=S.S. Rao; citation_author=S.C. Huang; citation_author=N.C. Durand; citation_author=M.H. Huntley; citation_author=A.I. Jewett; citation_author=I.D. Bochkov; citation_author=D. Chinnappan; citation_author=A. Cutkosky; citation_author=J. Li; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=E6456; citation_lastpage=E6465; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Fiji: an open-source platform for biological-image analysis; citation_author=J. Schindelin; citation_author=I. Arganda-Carreras; citation_author=E. Frise; citation_author=V. Kaynig; citation_author=M. Longair; citation_author=T. Pietzsch; citation_author=S. Preibisch; citation_author=C. Rueden; citation_author=S. Saalfeld; citation_author=B. Schmid; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=676; citation_lastpage=682; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome; citation_author=A.D. Schmitt; citation_author=M. Hu; citation_author=I. Jung; citation_author=Z. Xu; citation_author=Y. Qiu; citation_author=C.L. Tan; citation_author=Y. Li; citation_author=S. Lin; citation_author=Y. Lin; citation_author=C.L. Barr; citation_author=B. Ren; citation_journal_title=Cell Rep.; citation_volume=17; citation_firstpage=2042; citation_lastpage=2059; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=HiC-Pro: an optimized and flexible pipeline for Hi-C data processing; citation_author=N. Servant; citation_author=N. Varoquaux; citation_author=B.R. Lajoie; citation_author=E. Viara; citation_author=C.-J. Chen; citation_author=J.-P. Vert; citation_author=E. Heard; citation_author=J. Dekker; citation_author=E. Barillot; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=259; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Three-dimensional folding and functional organization principles of the Drosophila genome; citation_author=T. Sexton; citation_author=E. Yaffe; citation_author=E. Kenigsberg; citation_author=F. Bantignies; citation_author=B. Leblanc; citation_author=M. Hoichman; citation_author=H. Parrinello; citation_author=A. Tanay; citation_author=G. Cavalli; citation_journal_title=Cell; citation_volume=148; citation_firstpage=458; citation_lastpage=472; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Brain Plasticity and Human Evolution; citation_author=C.C. Sherwood; citation_author=A. Gómez-Robles; citation_journal_title=Annu. Rev. Anthropol.; citation_volume=46; citation_firstpage=399; citation_lastpage=419; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A transgenic monkey model for the study of human brain evolution; citation_author=L. Shi; citation_author=B. Su; citation_journal_title=Zool. Res.; citation_volume=40; citation_firstpage=236; citation_lastpage=238; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Transgenic rhesus monkeys carrying the human MCPH1 gene copies show human-like neoteny of brain development; citation_author=L. Shi; citation_author=X. Luo; citation_author=J. Jiang; citation_author=Y. Chen; citation_author=C. Liu; citation_author=T. Hu; citation_author=M. Li; citation_author=Q. Lin; citation_author=Y. Li; citation_author=J. Huang; citation_author=et al.; citation_journal_title=Natl. Sci. Rev.; citation_volume=6; citation_firstpage=480; citation_lastpage=493; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Mapping cis-regulatory chromatin contacts in neural cells links neuropsychiatric disorder risk variants to target genes; citation_author=M. Song; citation_author=X. Yang; citation_author=X. Ren; citation_author=L. Maliskova; citation_author=B. Li; citation_author=I.R. Jones; citation_author=C. Wang; citation_author=F. Jacob; citation_author=K. Wu; citation_author=M. Traglia; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=51; citation_firstpage=1252; citation_lastpage=1262; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Evolution of the Human Nervous System Function, Structure, and Development; citation_author=A.M.M. Sousa; citation_author=K.A. Meyer; citation_author=G. Santpere; citation_author=F.O. Gulden; citation_author=N. Sestan; citation_journal_title=Cell; citation_volume=170; citation_firstpage=226; citation_lastpage=247; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Structural variation in the 3D genome; citation_author=M. Spielmann; citation_author=D.G. Lupiáñez; citation_author=S. Mundlos; citation_journal_title=Nat. Rev. Genet.; citation_volume=19; citation_firstpage=453; citation_lastpage=467; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Transcription factors orchestrate dynamic interplay between genome topology and gene regulation during cell reprogramming; citation_author=R. Stadhouders; citation_author=E. Vidal; citation_author=F. Serra; citation_author=B. Di Stefano; citation_author=F. Le Dily; citation_author=J. Quilez; citation_author=A. Gomez; citation_author=S. Collombet; citation_author=C. Berenguer; citation_author=Y. Cuartero; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=238; citation_lastpage=249; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Transcription factors and 3D genome conformation in cell-fate decisions; citation_author=R. Stadhouders; citation_author=G.J. Filion; citation_author=T. Graf; citation_journal_title=Nature; citation_volume=569; citation_firstpage=345; citation_lastpage=354; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Understanding axon guidance: are we nearly there yet?; citation_author=E.T. Stoeckli; citation_journal_title=Development; citation_volume=145; citation_firstpage=dev151415; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Deficiency of neural recognition molecule NB-2 affects the development of glutamatergic auditory pathways from the ventral cochlear nucleus to the superior olivary complex in mouse; citation_author=M. Toyoshima; citation_author=K. Sakurai; citation_author=K. Shimazaki; citation_author=Y. Takeda; citation_author=Y. Shimoda; citation_author=K. Watanabe; citation_journal_title=Dev. Biol.; citation_volume=336; citation_firstpage=192; citation_lastpage=200; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Social Responsiveness Scale-aided analysis of the clinical impact of copy number variations in autism; citation_author=E. van Daalen; citation_author=C. Kemner; citation_author=N.E. Verbeek; citation_author=B. van der Zwaag; citation_author=T. Dijkhuizen; citation_author=P. Rump; citation_author=R. Houben; citation_author=R. van ’t Slot; citation_author=M.V. de Jonge; citation_author=W.G. Staal; citation_author=et al.; citation_journal_title=Neurogenetics; citation_volume=12; citation_firstpage=315; citation_lastpage=323; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=VISTA Enhancer Browser--a database of tissue-specific human enhancers; citation_author=A. Visel; citation_author=S. Minovitsky; citation_author=I. Dubchak; citation_author=L.A. Pennacchio; citation_journal_title=Nucleic Acids Res.; citation_volume=35; citation_firstpage=D88; citation_lastpage=D92; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=MIR retrotransposon sequences provide insulators to the human genome; citation_author=J. Wang; citation_author=C. Vicente-García; citation_author=D. Seruggia; citation_author=E. Moltó; citation_author=A. Fernandez-Miñán; citation_author=A. Neto; citation_author=E. Lee; citation_author=J.L. Gómez-Skarmeta; citation_author=L. Montoliu; citation_author=V.V. Lunyak; citation_author=I.K. Jordan; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=E4428; citation_lastpage=E4437; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Initial sequence of the chimpanzee genome and comparison with the human genome; citation_author=R.H. Waterson; citation_author=E.S. Lander; citation_author=R.K. Wilson; citation_author=Chimpanzee Sequencing and Analysis Consortium; citation_journal_title=Nature; citation_volume=437; citation_firstpage=69; citation_lastpage=87; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Neocortex expansion in development and evolution - from cell biology to single genes; citation_author=M. Wilsch-Bräuninger; citation_author=M. Florio; citation_author=W.B. Huttner; citation_journal_title=Curr. Opin. Neurobiol.; citation_volume=39; citation_firstpage=122; citation_lastpage=132; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Chromosome conformation elucidates regulatory relationships in developing human brain; citation_author=H. Won; citation_author=L. de la Torre-Ubieta; citation_author=J.L. Stein; citation_author=N.N. Parikshak; citation_author=J. Huang; citation_author=C.K. Opland; citation_author=M.J. Gandal; citation_author=G.J. Sutton; citation_author=F. Hormozdiari; citation_author=D. Lu; citation_author=et al.; citation_journal_title=Nature; citation_volume=538; citation_firstpage=523; citation_lastpage=527; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Human evolved regulatory elements modulate genes involved in cortical expansion and neurodevelopmental disease susceptibility; citation_author=H. Won; citation_author=J. Huang; citation_author=C.K. Opland; citation_author=C.L. Hartl; citation_author=D.H. Geschwind; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=2396; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Human-specific features of spatial gene expression and regulation in eight brain regions; citation_author=C. Xu; citation_author=Q. Li; citation_author=O. Efimova; citation_author=L. He; citation_author=S. Tatsumoto; citation_author=V. Stepanova; citation_author=T. Oishi; citation_author=T. Udono; citation_author=K. Yamaguchi; citation_author=S. Shigenobu; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=28; citation_firstpage=1097; citation_lastpage=1110; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Model-based analysis of ChIP-Seq (MACS); citation_author=Y. Zhang; citation_author=T. Liu; citation_author=C.A. Meyer; citation_author=J. Eeckhoute; citation_author=D.S. Johnson; citation_author=B.E. Bernstein; citation_author=C. Nusbaum; citation_author=R.M. Myers; citation_author=M. Brown; citation_author=W. Li; citation_author=X.S. Liu; citation_journal_title=Genome Biol.; citation_volume=9; citation_firstpage=R137; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells; citation_author=Y. Zhang; citation_author=T. Li; citation_author=S. Preissl; citation_author=M.L. Amaral; citation_author=J.D. Grinstein; citation_author=E.N. Farah; citation_author=E. Destici; citation_author=Y. Qiu; citation_author=R. Hu; citation_author=A.Y. Lee; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=51; citation_firstpage=1380; citation_lastpage=1388; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=XSAnno: a framework for building ortholog models in cross-species transcriptome comparisons; citation_author=Y. Zhu; citation_author=M. Li; citation_author=A.M.M. Sousa; citation_author=N. Sestan; citation_journal_title=BMC Genomics; citation_volume=15; citation_firstpage=343; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Spatiotemporal transcriptomic divergence across human and macaque brain development; citation_author=Y. Zhu; citation_author=A.M.M. Sousa; citation_author=T. Gao; citation_author=M. Skarica; citation_author=M. Li; citation_author=G. Santpere; citation_author=P. Esteller-Cucala; citation_author=D. Juan; citation_author=L. Ferrández-Peral; citation_author=F.O. Gulden; citation_author=et al.; citation_journal_title=Science; citation_volume=362; citation_firstpage=eaat8077; citation_publication_date=2018; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Elucidating the regulatory mechanisms of human brain evolution is essential to understanding human cognition and mental disorders. We generated multi-omics profiles and constructed a high-resolution map of 3D genome architecture of rhesus macaque during corticogenesis. By comparing the 3D genomes of human, macaque, and mouse brains, we identified many human-specific chromatin structure changes, including 499 topologically associating domains (TADs) and 1,266 chromatin loops. The human-specific loops are significantly enriched in enhancer-enhancer interactions, and the regulated genes show human-specific expression changes in the subplate, a transient zone of the developing brain critical for neural circuit formation and plasticity. Notably, many human-specific sequence changes are located in the human-specific TAD boundaries and loop anchors, which may generate new transcription factor binding sites and chromatin structures in human. Collectively, the presented data highlight the value of comparative 3D genome analyses in dissecting the regulatory mechanisms of brain development and evolution.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=6382&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)00001-5&amp;be=4363&amp;fe=6172&amp;dc=5634&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749589209830,%22n%22:0,%22f%22:3724,%22dn%22:3725,%22dne%22:3725,%22c%22:3725,%22s%22:3725,%22ce%22:3725,%22rq%22:3725,%22rp%22:4338,%22rpe%22:5068,%22dl%22:4344,%22di%22:5497,%22ds%22:5634,%22de%22:5860,%22dc%22:6172,%22l%22:6172,%22le%22:6275%7D,%22navigation%22:%7B%7D%7D&amp;fp=4836&amp;fcp=4836&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c49 {
  padding: 0.683em 0.361em 0 0;
}

mjx-c.mjx-c69 {
  padding: 0.657em 0.278em 0 0;
}

mjx-c.mjx-c6E {
  padding: 0.442em 0.556em 0 0;
}

mjx-c.mjx-c74 {
  padding: 0.615em 0.389em 0.011em 0;
}

mjx-c.mjx-c65 {
  padding: 0.448em 0.444em 0.011em 0;
}

mjx-c.mjx-c72 {
  padding: 0.442em 0.392em 0 0;
}

mjx-c.mjx-c54 {
  padding: 0.677em 0.722em 0 0;
}

mjx-c.mjx-c41 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c44 {
  padding: 0.683em 0.764em 0 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c61 {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c62 {
  padding: 0.694em 0.556em 0.011em 0;
}

mjx-c.mjx-c63 {
  padding: 0.448em 0.444em 0.011em 0;
}

mjx-c.mjx-c73 {
  padding: 0.448em 0.394em 0.011em 0;
}

mjx-c.mjx-c75 {
  padding: 0.442em 0.556em 0.011em 0;
}

mjx-c.mjx-c6C {
  padding: 0.694em 0.278em 0 0;
}

mjx-c.mjx-c6F {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c67 {
  padding: 0.453em 0.5em 0.206em 0;
}

mjx-c.mjx-c68 {
  padding: 0.694em 0.556em 0 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c1D450 {
  padding: 0.442em 0.433em 0.011em 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c3C {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c6D {
  padding: 0.442em 0.833em 0 0;
}

mjx-c.mjx-c78 {
  padding: 0.431em 0.528em 0 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c1D462 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c2C {
  padding: 0.106em 0.278em 0.193em 0;
}

mjx-c.mjx-cD7 {
  padding: 0.491em 0.778em 0 0;
}

mjx-c.mjx-c7C {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c1D457 {
  padding: 0.661em 0.412em 0.205em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900001-5" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00001-5"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)00001-5"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900001-5" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-800624" aria-expanded="false" aria-controls="login-links-800624" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-800624" id="login-links--800624" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900001-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900001-5" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900001-5" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-800624" data-opens="#links-dialog-800624" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-800624" aria-modal="true"><button data-closes="#links-dialog-800624" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900001-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900001-5" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900001-5" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.01.001">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)00001-5" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0004-3"><span>Volume 184</span>, <span>Issue 3</span></a></span><span class="meta-panel__pages"><span>p723-740.e21</span></span><span class="meta-panel__onlineDate">February 04, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=3&amp;pii=S0092-8674%2820%29X0004-3" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">3D Genome of macaque fetal brain reveals evolutionary innovations during primate corticogenesis</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Xin</span> <span property="familyName">Luo</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xin</span> <span property="familyName">Luo</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">National Resource Center for Non-Human Primates, Kunming Primate Research Center, and National Research Facility for Phenotypic and Genetic Analysis of Model Animals (Primate Facility), Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650107, China</span></div></div><div class="core-author-link"><a href="/authored-by/Luo/Xin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a role="doc-noteref">8</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Yuting</span> <span property="familyName">Liu</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yuting</span> <span property="familyName">Liu</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Bioinformatics, School of Life Sciences, Center for Statistical Science, Peking University, Beijing 100871, China</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/Yuting" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">8</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Dachang</span> <span property="familyName">Dang</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Dachang</span> <span property="familyName">Dang</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">NCMIS, CEMS, RCSDS, Academy of Mathematics and Systems Science, Chinese Academy of Sciences, Beijing 100190, China</span></div></div><div class="core-author-link"><a href="/authored-by/Dang/Dachang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">8</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Ting</span> <span property="familyName">Hu</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ting</span> <span property="familyName">Hu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China</span></div></div><div class="core-author-link"><a href="/authored-by/Hu/Ting" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Yingping</span> <span property="familyName">Hou</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yingping</span> <span property="familyName">Hou</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Peking-Tsinghua Center for Life Sciences, Academy for Advanced Interdisciplinary Studies, Peking University, Beijing 100871, China</span></div></div><div class="core-author-link"><a href="/authored-by/Hou/Yingping" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Xiaoyu</span> <span property="familyName">Meng</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xiaoyu</span> <span property="familyName">Meng</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China</span></div></div><div class="core-author-link"><a href="/authored-by/Meng/Xiaoyu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Fengyun</span> <span property="familyName">Zhang</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Fengyun</span> <span property="familyName">Zhang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Fengyun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Tingting</span> <span property="familyName">Li</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tingting</span> <span property="familyName">Li</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Bioinformatics, School of Life Sciences, Center for Statistical Science, Peking University, Beijing 100871, China</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Tingting" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Can</span> <span property="familyName">Wang</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Can</span> <span property="familyName">Wang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">NCMIS, CEMS, RCSDS, Academy of Mathematics and Systems Science, Chinese Academy of Sciences, Beijing 100190, China</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Can" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Min</span> <span property="familyName">Li</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Min</span> <span property="familyName">Li</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Min" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Haixu</span> <span property="familyName">Wu</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Haixu</span> <span property="familyName">Wu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China</span></div></div><div class="core-author-link"><a href="/authored-by/Wu/Haixu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Qiushuo</span> <span property="familyName">Shen</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Qiushuo</span> <span property="familyName">Shen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div></div><div class="core-author-link"><a href="/authored-by/Shen/Qiushuo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Yan</span> <span property="familyName">Hu</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yan</span> <span property="familyName">Hu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div></div><div class="core-author-link"><a href="/authored-by/Hu/Yan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Xuerui</span> <span property="familyName">Zeng</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xuerui</span> <span property="familyName">Zeng</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China</span></div></div><div class="core-author-link"><a href="/authored-by/Zeng/Xuerui" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Xiechao</span> <span property="familyName">He</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xiechao</span> <span property="familyName">He</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">National Resource Center for Non-Human Primates, Kunming Primate Research Center, and National Research Facility for Phenotypic and Genetic Analysis of Model Animals (Primate Facility), Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650107, China</span></div></div><div class="core-author-link"><a href="/authored-by/He/Xiechao" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Lanzhen</span> <span property="familyName">Yan</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lanzhen</span> <span property="familyName">Yan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">National Resource Center for Non-Human Primates, Kunming Primate Research Center, and National Research Facility for Phenotypic and Genetic Analysis of Model Animals (Primate Facility), Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650107, China</span></div></div><div class="core-author-link"><a href="/authored-by/Yan/Lanzhen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Shihua</span> <span property="familyName">Zhang</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shihua</span> <span property="familyName">Zhang</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0192-7118" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0192-7118</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:zsh@amss.ac.cn" property="email" title="Send email to Shihua Zhang">zsh@amss.ac.cn</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">NCMIS, CEMS, RCSDS, Academy of Mathematics and Systems Science, Chinese Academy of Sciences, Beijing 100190, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Shihua" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup> <a id="eadd0010" href="mailto:zsh@amss.ac.cn" property="email" title="Send email to Shihua Zhang">zsh@amss.ac.cn</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Cheng</span> <span property="familyName">Li</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Cheng</span> <span property="familyName">Li</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:cheng_li@pku.edu.cn" property="email" title="Send email to Cheng Li">cheng_li@pku.edu.cn</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Bioinformatics, School of Life Sciences, Center for Statistical Science, Peking University, Beijing 100871, China</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Cheng" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup> <a id="eadd0015" href="mailto:cheng_li@pku.edu.cn" property="email" title="Send email to Cheng Li">cheng_li@pku.edu.cn</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Bing</span> <span property="familyName">Su</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bing</span> <span property="familyName">Su</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:sub@mail.kiz.ac.cn" property="email" title="Send email to Bing Su">sub@mail.kiz.ac.cn</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">9</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span property="name">National Resource Center for Non-Human Primates, Kunming Primate Research Center, and National Research Facility for Phenotypic and Genetic Analysis of Model Animals (Primate Facility), Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650107, China</span></div></div><div class="core-author-link"><a href="/authored-by/Su/Bing" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a role="doc-noteref">9</a></sup> <a id="eadd0020" href="mailto:sub@mail.kiz.ac.cn" property="email" title="Send email to Bing Su">sub@mail.kiz.ac.cn</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Center for Bioinformatics, School of Life Sciences, Center for Statistical Science, Peking University, Beijing 100871, China</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">NCMIS, CEMS, RCSDS, Academy of Mathematics and Systems Science, Chinese Academy of Sciences, Beijing 100190, China</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Peking-Tsinghua Center for Life Sciences, Academy for Advanced Interdisciplinary Studies, Peking University, Beijing 100871, China</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, China</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">National Resource Center for Non-Human Primates, Kunming Primate Research Center, and National Research Facility for Phenotypic and Genetic Analysis of Model Animals (Primate Facility), Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650107, China</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Kunming College of Life Science, University of Chinese Academy of Sciences, Beijing 100101, China</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">9</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received November 25, 2019</span>; <span>Revised November 9, 2020</span>; <span>Accepted December 31, 2020</span>; <span>Published online January 27, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.01.001" property="sameAs" target="_blank">10.1016/j.cell.2021.01.001 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421000015" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.01.001" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0004-3"><img src="/cms/asset/c22743ff-7b25-4b26-aaca-7f1760300f13/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 3"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2900001-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (4)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900001-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.01.001">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2900001-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.01.001&amp;pii=S0092-8674%2821%2900001-5" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900001-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.01.001">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.01.001" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421000015&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421000015&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=3&amp;pii=S0092-8674%2820%29X0004-3" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)31748-7" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)31754-2" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical Abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental information (4)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Construction of a high-resolution Hi-C map of macaque fetal brain</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Cross-species 3D genome analyses uncover human-specific chromatin structures</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">The subplate lamina shows human-specific regulatory changes during corticogenesis</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">The human-specific loop regulating <i>EPHA7</i> affects neuron dendrite development</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Elucidating the regulatory mechanisms of human brain evolution is essential to understanding human cognition and mental disorders. We generated multi-omics profiles and constructed a high-resolution map of 3D genome architecture of rhesus macaque during corticogenesis. By comparing the 3D genomes of human, macaque, and mouse brains, we identified many human-specific chromatin structure changes, including 499 topologically associating domains (TADs) and 1,266 chromatin loops. The human-specific loops are significantly enriched in enhancer-enhancer interactions, and the regulated genes show human-specific expression changes in the subplate, a transient zone of the developing brain critical for neural circuit formation and plasticity. Notably, many human-specific sequence changes are located in the human-specific TAD boundaries and loop anchors, which may generate new transcription factor binding sites and chromatin structures in human. Collectively, the presented data highlight the value of comparative 3D genome analyses in dissecting the regulatory mechanisms of brain development and evolution.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/45d3557e-a366-4dd5-ab20-f0f514ed94d4/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.01.001/asset/8f927550-a945-4d92-a51f-58e11ee2ffca/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/45d3557e-a366-4dd5-ab20-f0f514ed94d4/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;brain&quot;" alt="Search on this keyword" rel="nofollow">brain</a></li><li><a href="/action/doSearch?AllField=&quot;chromatin+structure&quot;" alt="Search on this keyword" rel="nofollow">chromatin structure</a></li><li><a href="/action/doSearch?AllField=&quot;TAD&quot;" alt="Search on this keyword" rel="nofollow">TAD</a></li><li><a href="/action/doSearch?AllField=&quot;loop&quot;" alt="Search on this keyword" rel="nofollow">loop</a></li><li><a href="/action/doSearch?AllField=&quot;subplate&quot;" alt="Search on this keyword" rel="nofollow">subplate</a></li><li><a href="/action/doSearch?AllField=&quot;macaque&quot;" alt="Search on this keyword" rel="nofollow">macaque</a></li><li><a href="/action/doSearch?AllField=&quot;brain+evolution&quot;" alt="Search on this keyword" rel="nofollow">brain evolution</a></li><li><a href="/action/doSearch?AllField=&quot;3D+genome&quot;" alt="Search on this keyword" rel="nofollow">3D genome</a></li><li><a href="/action/doSearch?AllField=&quot;corticogenesis&quot;" alt="Search on this keyword" rel="nofollow">corticogenesis</a></li><li><a href="/action/doSearch?AllField=&quot;EPHA7&quot;" alt="Search on this keyword" rel="nofollow">EPHA7</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">The human brain exhibits unique organizing principles and developmental patterns when compared with nonhuman primates (NHPs) (<span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Sousa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sousa, A.M.M. ∙ Meyer, K.A. ∙ Santpere, G. ...</div><div><strong>Evolution of the Human Nervous System Function, Structure, and Development</strong></div><div><em>Cell.</em> 2017; <strong>170</strong>:226-247</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85023612934" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28708995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.06.036&amp;pmid=28708995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It was proposed that the developing human brain has an expanded proliferative zone, the outer subventricular zone (OSVZ), and more diverse neural progenitor cell types that account for human brain size expansion (<span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Lui et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Lui, J.H. ∙ Hansen, D.V. ∙ Kriegstein, A.R.</div><div><strong>Development and evolution of the human neocortex</strong></div><div><em>Cell.</em> 2011; <strong>146</strong>:18-36</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-79959924122" target="_blank" aria-label="Open Scopus in new tab">Scopus (677)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21729779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2011.06.030&amp;pmid=21729779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Wilsch-Bräuninger et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Wilsch-Bräuninger, M. ∙ Florio, M. ∙ Huttner, W.B.</div><div><strong>Neocortex expansion in development and evolution - from cell biology to single genes</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2016; <strong>39</strong>:122-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.conb.2016.05.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84973139075" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27258840/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.conb.2016.05.004&amp;pmid=27258840" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, the maturation of neurons appears to be prolonged, and neural networks are more complex in hominids compared to NHPs (<span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Marchetto et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Marchetto, M.C. ∙ Hrvoj-Mihic, B. ∙ Kerman, B.E. ...</div><div><strong>Species-specific maturation profiles of human, chimpanzee and bonobo neural cells</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e37527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.37527" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85061147585" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30730291/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.37527&amp;pmid=30730291" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Sherwood and Gómez-Robles, 2017</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Sherwood, C.C. ∙ Gómez-Robles, A.</div><div><strong>Brain Plasticity and Human Evolution</strong></div><div><em>Annu. Rev. Anthropol.</em> 2017; <strong>46</strong>:399-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-anthro-102215-100009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85032203822" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-anthro-102215-100009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example, our recent study showed that the transgenic monkeys carrying a human version of <i>MCPH1</i> (a key brain developmental gene) had delayed neuron differentiation and maturation and improved short-term memory, mimicking the brain developmental neoteny of humans (<span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Shi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Shi, L. ∙ Luo, X. ∙ Jiang, J. ...</div><div><strong>Transgenic rhesus monkeys carrying the human MCPH1 gene copies show human-like neoteny of brain development</strong></div><div><em>Natl. Sci. Rev.</em> 2019; <strong>6</strong>:480-493</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nsr/nwz043" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85068564152" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnsr%2Fnwz043" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Shi and Su, 2019</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Shi, L. ∙ Su, B.</div><div><strong>A transgenic monkey model for the study of human brain evolution</strong></div><div><em>Zool. Res.</em> 2019; <strong>40</strong>:236-238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.24272/j.issn.2095-8137.2019.031" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85065115577" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31011135/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.24272%2Fj.issn.2095-8137.2019.031&amp;pmid=31011135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally, the subplate neurons form one of the first functional cortical circuits during fetal brain development and may associate with the unique developmental patterns of human neural circuits (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Hoerder-Suabedissen and Molnár, 2015</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hoerder-Suabedissen, A. ∙ Molnár, Z.</div><div><strong>Development, evolution and pathology of neocortical subplate neurons</strong></div><div><em>Nat. Rev. Neurosci.</em> 2015; <strong>16</strong>:133-146</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn3915" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84923633876" target="_blank" aria-label="Open Scopus in new tab">Scopus (119)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25697157/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn3915&amp;pmid=25697157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Judaš et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Judaš, M. ∙ Sedmak, G. ∙ Kostović, I.</div><div><strong>The significance of the subplate for evolution and developmental plasticity of the human brain</strong></div><div><em>Front. Hum. Neurosci.</em> 2013; <strong>7</strong>:423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fnhum.2013.00423" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84880141282" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23935575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffnhum.2013.00423&amp;pmid=23935575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), although the underlying molecular mechanisms remain elusive.</div><div id="p0035" role="paragraph">Recent advances in comparative genomic analyses have provided increasing evidences that our evolution, since the human-chimpanzee divergence, has been driven mostly by gene regulation (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Bird et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bird, C.P. ∙ Stranger, B.E. ∙ Liu, M. ...</div><div><strong>Fast-evolving noncoding sequences in the human genome</strong></div><div><em>Genome Biol.</em> 2007; <strong>8</strong>:R118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2007-8-6-r118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-39049157173" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17578567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2007-8-6-r118&amp;pmid=17578567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Bush and Lahn, 2008</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bush, E.C. ∙ Lahn, B.T.</div><div><strong>A genome-wide screen for noncoding elements important in primate evolution</strong></div><div><em>BMC Evol. Biol.</em> 2008; <strong>8</strong>:17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-8-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-39049088628" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18215302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-8-17&amp;pmid=18215302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Gittelman et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Gittelman, R.M. ∙ Hun, E. ∙ Ay, F. ...</div><div><strong>Comprehensive identification and analysis of human accelerated regulatory DNA</strong></div><div><em>Genome Res.</em> 2015; <strong>25</strong>:1245-1255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.192591.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84937476478" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26104583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.192591.115&amp;pmid=26104583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">King and Wilson, 1975</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>King, M.C. ∙ Wilson, A.C.</div><div><strong>Evolution at two levels in humans and chimpanzees</strong></div><div><em>Science.</em> 1975; <strong>188</strong>:107-116</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1090005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1090005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1090005&amp;pmid=1090005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-1" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-1" aria-controls="sref73-1">Pollard et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref73-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Pollard, K.S. ∙ Salama, S.R. ∙ Lambert, N. ...</div><div><strong>An RNA gene expressed during cortical development evolved rapidly in humans</strong></div><div><em>Nature.</em> 2006; <strong>443</strong>:167-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33748675254" target="_blank" aria-label="Open Scopus in new tab">Scopus (638)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16915236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05113&amp;pmid=16915236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Reilly et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Reilly, S.K. ∙ Yin, J. ∙ Ayoub, A.E. ...</div><div><strong>Evolutionary genomics. Evolutionary changes in promoter and enhancer activity during human corticogenesis</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1155-1159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84924561748" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25745175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260943&amp;pmid=25745175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Xu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Xu, C. ∙ Li, Q. ∙ Efimova, O. ...</div><div><strong>Human-specific features of spatial gene expression and regulation in eight brain regions</strong></div><div><em>Genome Res.</em> 2018; <strong>28</strong>:1097-1110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.231357.117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85050863625" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29898898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.231357.117&amp;pmid=29898898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Several studies revealed that long-range chromatin interactions might regulate human-specific brain gene expression and associate with aging-related diseases, such as Alzheimer’s disease (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Chen et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Chen, H. ∙ Li, C. ∙ Zhou, Z. ...</div><div><strong>Fast-Evolving Human-Specific Neural Enhancers Are Associated with Aging-Related Diseases</strong></div><div><em>Cell Syst.</em> 2018; <strong>6</strong>:604-611.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2018.04.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2018.04.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85047107335" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29792826/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2018.04.002&amp;pmid=29792826" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">de la Torre-Ubieta et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>de la Torre-Ubieta, L. ∙ Stein, J.L. ∙ Won, H. ...</div><div><strong>The Dynamic Landscape of Open Chromatin during Human Cortical Neurogenesis</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:289-304.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.12.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.12.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85039917750" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29307494/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.014&amp;pmid=29307494" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref100-1" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-1" aria-controls="sref100-1">Won et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref100-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Won, H. ∙ de la Torre-Ubieta, L. ∙ Stein, J.L. ...</div><div><strong>Chromosome conformation elucidates regulatory relationships in developing human brain</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:523-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84993993658" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19847&amp;pmid=27760116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" aria-label="Reference 2019" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">2019</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Won, H. ∙ Huang, J. ∙ Opland, C.K. ...</div><div><strong>Human evolved regulatory elements modulate genes involved in cortical expansion and neurodevelopmental disease susceptibility</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2396</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10248-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85066601975" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160561/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10248-3&amp;pmid=31160561" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hence, it is feasible to hypothesize that the human-specific changes of chromatin structure may lead to gene regulation alterations and eventually contribute to the unique human brain phenotypes. To test this hypothesis, we need to generate brain 3D genome data of NHPs at key developmental stages in order to identify the human-specific gene regulatory changes. In theory, as the closest relative of human, chimpanzee would be the ideal NHP species to study (<span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Waterson et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Waterson, R.H. ∙ Lander, E.S. ∙ Wilson, R.K., Chimpanzee Sequencing and Analysis Consortium</div><div><strong>Initial sequence of the chimpanzee genome and comparison with the human genome</strong></div><div><em>Nature.</em> 2005; <strong>437</strong>:69-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature04072" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-24344500211" target="_blank" aria-label="Open Scopus in new tab">Scopus (1553)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16136131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature04072&amp;pmid=16136131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, it is extremely difficult to obtain chimpanzee fetal brain samples for 3D genome experiments. Alternatively, as the most widely used NHP species, macaque is phylogenetically close to human (∼7% sequence divergence), and it is relatively easy to obtain brain samples at different developmental stages, making it a suitable model for exploring human brain evolution at 3D genome level.</div><div id="p0040" role="paragraph">Here, we collected brain tissues of Chinese rhesus macaque (<i>Macaca mulatta</i>) at mid-gestation, and constructed a high-resolution 3D genome map, which provides valuable basis for further comparative 3D genomic analysis. Through multi-omics evolutionary analyses among human, rhesus macaque, and mouse, we identified many human-specific chromatin structure changes including topologically associating domains (TADs) and loops. We also studied the genomic features and functional enrichment of the human-specific TADs and loops to shed new light on the genetic basis of human brain development and evolution.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Generation of a high-resolution 3D genome map of rhesus macaque during corticogenesis</h3><div id="p0045" role="paragraph">We sampled three macaque fetal brains at mid-gestation (E84), the time of neurogenesis peak during development. We dissected cerebral cortex into two major zones: (1) cortex plate (CP), consisting primarily of post-mitotic neurons, and (2) germinal zone (GZ), consisting primarily of neural progenitors (<a id="crosref0320" href="#fig1">Figure&nbsp;1</a>A; <a id="crosref0325" href="#sec-4">STAR methods</a>). To reveal the regulatory landscape of the developing macaque cortex, we constructed high-resolution genome-wide chromatin interaction maps using Hi-C. We also mapped gene expression (RNA sequencing [RNA-seq]), DNase I hypersensitive sites (DNase-seq), and CTCF occupancy (CTCF chromatin immunoprecipitation sequencing [ChIP-seq]). We validated the accuracy of zone dissection by checking the expression of the known marker genes for neural progenitors (<i>SOX2</i>, <i>PAX6</i>, and <i>FOXG1</i>) and neurons (<i>FEZF2</i>, <i>SYP</i>, and <i>NEUROD1</i>) (<a id="crosref0330" href="#fig1">Figure&nbsp;1</a>B).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/62b05598-f378-4155-9620-5c62693cc416/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.01.001/asset/d88660e3-6ede-4266-a3a4-54df6f240263/main.assets/gr1.jpg" height="940" width="689" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/62b05598-f378-4155-9620-5c62693cc416/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">A high-resolution 3D genome map of the developing macaque cortex</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Schematic representation of the experimental design.</div><div id="fspara0020" role="paragraph">(B) Bar plots showing the expression levels (fragments per kilobase of transcript per million [FPKM]) of six marker genes for neural progenitors and neurons. Data are shown as mean ± SD.</div><div id="fspara0025" role="paragraph">(C) The Hi-C interaction matrices for GZ at 500 kb, 40 kb, and 5 kb resolution, respectively. The matrices were Knight-Ruiz (KR) matrix-balance normalized.</div><div id="fspara0030" role="paragraph">(D) The Hi-C map of CP (red) and GZ (blue) around the <i>SOX2</i> gene. The detected loops are indicated by the black boxes. The two GZ-specific interactions are indicated by the yellow arrows.</div><div id="fspara0035" role="paragraph">(E) Loops regulating <i>FOXP2</i> gene in macaque fetal brain. The anchors of the detected loops are highlighted in yellow.</div><div id="fspara0040" role="paragraph">(F) The images of Oligopaint 3D FISH in CP. Approximately 69% (49/71) cells show a shorter distance between the promoter and the enhancer than between the promoter and the control probes. DAPI are in blue.</div><div id="fspara0045" role="paragraph">(G) The similar percentages of cells in CP and GZ showing significantly shorter distances of the loops than the controls. p value is calculated by Fisher’s exact test.</div><div id="fspara0050" role="paragraph">See also <a id="crosref1845" href="#figs1">Figure&nbsp;S1</a> and <a id="crosref1850" href="#mmc1">Tables S1</a> and <a id="crosref1855" href="#mmc2">S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0050" role="paragraph">Using Hi-C 2.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Belaghzal et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Belaghzal, H. ∙ Dekker, J. ∙ Gibcus, J.H.</div><div><strong>Hi-C 2.0: An optimized Hi-C procedure for high-resolution genome-wide mapping of chromosome conformation</strong></div><div><em>Methods.</em> 2017; <strong>123</strong>:56-65</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ymeth.2017.04.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85018788636" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28435001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymeth.2017.04.004&amp;pmid=28435001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we sequenced ∼8 billion paired-end reads and generated nearly 5 billion unique mapped contacts in total (<a id="crosref0340" href="#mmc1">Table S1</a>). The biological and technical replicates are highly correlated (Pearson’s correlation R ≥ 0.95) (<a id="crosref0345" href="#figs1">Figures S1</a>A and S1B), and we therefore merged the data by zone type for downstream analysis. We validated contact quality of the Hi-C data by <i>cis</i>/<i>trans</i> interaction ratio and distance-dependent interaction frequency decay (<a id="crosref0350" href="#figs1">Figure&nbsp;S1</a>C; <a id="crosref0355" href="#mmc1">Table S1</a>). We reached an ultra-deep resolution of 1.5 kilobase pairs (kb) (<a id="crosref0360" href="#mmc1">Table S1</a>; <a id="crosref0365" href="#sec-4">STAR methods</a>), which enables us to explore macaque 3D genome structure at different scales, including compartments, TADs, and loops (<a id="crosref0370" href="#fig1">Figure&nbsp;1</a>C).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/2ce50164-ddae-4474-ab25-f2919f66a364/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.01.001/asset/11eca952-9ee5-4700-833b-b9377dd70760/main.assets/figs1.jpg" height="814" width="762" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/2ce50164-ddae-4474-ab25-f2919f66a364/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Basic characterization of fetal rhesus macaque 3D genome, related to <a id="crosref1905" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0355" role="paragraph">(A) Pairwise Pearson’s correlation between the Hi-C biological replicates (at 100 kb resolution using log2 normalized contacts separated by at least 100 kb and no more than 5Mb). (B) Bar plots showing pairwise Pearson’s correlation between Hi-C technological replicates within biological replicates. Data are shown as mean ± SD (C) Log-log contact frequency as a function of the genomic distance. The curves show the mean slope ± SD of the best-fit line between 100 kb and 2Mb. (D) Stacked bar plots showing the percentages of genome regions with a stable or switched compartment between CP and GZ. (E) Overlap of TADs between CP and GZ. (F) Overlap of loops between CP and GZ. (G) Examples of 3C validation of loops including <i>SOX2</i> and <i>FOXP2.</i> (H) Boxplots showing the log2 FPKM of genes with promoters located in the loops anchors of the CP-specific loops, the conserved loops and the GZ-specific loops. Zone-specific loops are identified using HiCCUPS Diff. (I) The Hi-C map at <i>BHLHE22 locus</i> in CP and GZ. Detected loops are indicated by black dashed boxes, and the CP-specific loop is indicated by yellow arrow. (J) Two putative enhancers contained within loop anchors, predicted by VISTA, which drive forebrain-specific and midbrain. (K) The image of Oligopaint 3D FISH for cells in GZ. About 76% (66 out of 86) cells showing distance of loop significantly shorter (by at least 0.1&nbsp;μm) than the control. (L) The distribution for distances of loop and distances of control in CP and GZ. The P values are calculated by Kolmogorov-Smirnov test. (M) The Hi-C map of CP (red) and GZ (blue) around the SOX2 gene. Two loops validated by 3D FISH are indicated by arrows. The CP/GZ shared loop is indicated by orange arrow, and the GZ-biased loop is indicated by green arrow. (N-O) Left and middle: The distribution of distances of the shared loop (N) or the GZ-biased loop (O) and distances of the corresponding controls in CP and GZ. The P values are calculated by Kolmogorov-Smirnov test. Right: The percentages of cells in CP and GZ with the distance of the shared loop (N) or the GZ-biased loop (O) significantly shorter than controls. P values are calculated by Fisher’s exact test. (P) Boxplots showing the distance of the GZ-biased loop and control in CP and GZ grouped by whether the distance of loop is significantly shorter than the distance of control. P values are calculated by t test.</div></div></div></div></figcaption></figure></div><div id="p0055" role="paragraph">Most of the A/B compartments are conserved between CP and GZ, and the ratio (3.6%–3.8%) of switched compartments is consistent with previous findings in human fetal brains (CP versus GZ: 3.78%) (<span class="dropBlock reference-citations"><a id="body-ref-sref100-2" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-2" aria-controls="sref100-2">Won et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref100-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Won, H. ∙ de la Torre-Ubieta, L. ∙ Stein, J.L. ...</div><div><strong>Chromosome conformation elucidates regulatory relationships in developing human brain</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:523-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84993993658" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19847&amp;pmid=27760116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0380" href="#figs1">Figure&nbsp;S1</a>D). We identified a total of 1,977 and 2,265 TADs for CP and GZ, respectively, among which 64.4% are shared between them (<a id="crosref0385" href="#figs1">Figure&nbsp;S1</a>E), and the numbers of the identified TADs in macaque is similar to the reported numbers of TADs in the human fetal brain (<span class="dropBlock reference-citations"><a id="body-ref-sref100-3" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-3" aria-controls="sref100-3">Won et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref100-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Won, H. ∙ de la Torre-Ubieta, L. ∙ Stein, J.L. ...</div><div><strong>Chromosome conformation elucidates regulatory relationships in developing human brain</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:523-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84993993658" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19847&amp;pmid=27760116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because the resolution of the published human fetal brain Hi-C data was only 10 kb (<span class="dropBlock reference-citations"><a id="body-ref-sref100-4" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-4" aria-controls="sref100-4">Won et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref100-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Won, H. ∙ de la Torre-Ubieta, L. ∙ Stein, J.L. ...</div><div><strong>Chromosome conformation elucidates regulatory relationships in developing human brain</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:523-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84993993658" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19847&amp;pmid=27760116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), for cross-species comparison, we called loops in macaque with the same resolution using the HICCUPS pipeline (<span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Rao, S.S. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0405" href="#sec-4">STAR methods</a>). In total, we identified 8,399 and 15,048 loops in CP and GZ, respectively (<a id="crosref0410" href="#figs1">Figure&nbsp;S1</a>F). The 3C experiments show that 86.6% (26/30) of the loops can be validated, suggesting our Hi-C results are reliable (<a id="crosref0415" href="#figs1">Figure&nbsp;S1</a>G; <a id="crosref0420" href="#mmc2">Table S2</a>). There are 8,179 (54.4%) loops shared between CP and GZ (<a id="crosref0425" href="#figs1">Figure&nbsp;S1</a>F), and the smaller number of loops (∼44% reduction) in CP relative to GZ is consistent with the reported reduction (∼40% to 50% decrease) of loops in human neurons relative to neural progenitor cells (<span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Rajarajan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Rajarajan, P. ∙ Borrman, T. ∙ Liao, W. ...</div><div><strong>Neuron-specific signatures in the chromosomal connectome associated with schizophrenia risk</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat4311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat4311" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85058384538" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat4311&amp;pmid=30545851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As expected, the zone-specific loops are associated with differential gene expression (<a id="crosref0435" href="#figs1">Figure&nbsp;S1</a>H). For example, we observed zone-specific chromatin interactions at <i>SOX2</i> (highly expressed in GZ) and <i>BHLHE22</i> (highly expressed in CP), respectively (<a id="crosref0440" href="#fig1">Figures 1</a>D and <a id="crosref0445" href="#figs1">S1</a>I).</div><div id="p0060" role="paragraph">To show that the identified loops using our ultra-deep contact maps are informative in detecting regulatory events in the genome, as a proof of principle, we checked interactions between two putative enhancers and the promoter of <i>FOXP2</i> (<a id="crosref0450" href="#fig1">Figure&nbsp;1</a>E), a well-known gene that plays an important role in human speech and language development (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Lai et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Lai, C.S. ∙ Fisher, S.E. ∙ Hurst, J.A. ...</div><div><strong>The SPCH1 region on human 7q31: genomic characterization of the critical interval and localization of translocations associated with speech and language disorder</strong></div><div><em>Am. J. Hum. Genet.</em> 2000; <strong>67</strong>:357-368</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1086%2F303011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1086%2F303011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0033865944" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10880297/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F303011&amp;pmid=10880297" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" aria-label="Reference 2001" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">2001</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Lai, C.S. ∙ Fisher, S.E. ∙ Hurst, J.A. ...</div><div><strong>A forkhead-domain gene is mutated in a severe speech and language disorder</strong></div><div><em>Nature.</em> 2001; <strong>413</strong>:519-523</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/35097076" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0035807360" target="_blank" aria-label="Open Scopus in new tab">Scopus (1273)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11586359/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F35097076&amp;pmid=11586359" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Lennon et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Lennon, P.A. ∙ Cooper, M.L. ∙ Peiffer, D.A. ...</div><div><strong>Deletion of 7q31.1 supports involvement of FOXP2 in language impairment: clinical report and review</strong></div><div><em>Am. J. Med. Genet. A.</em> 2007; <strong>143A</strong>:791-798</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajmg.a.31632" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-34147120769" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17330859/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajmg.a.31632&amp;pmid=17330859" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These two putative enhancers are also active in humans (hs218 and hs956) and can drive forebrain- and midbrain-specific expression in mouse embryos (<span class="dropBlock reference-citations"><a id="body-ref-sref96-1" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-1" aria-controls="sref96-1">Visel et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref96-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Visel, A. ∙ Minovitsky, S. ∙ Dubchak, I. ...</div><div><strong>VISTA Enhancer Browser--a database of tissue-specific human enhancers</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:D88-D92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkl822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33846112470" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17130149/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkl822&amp;pmid=17130149" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0475" href="#figs1">Figure&nbsp;S1</a>J). The loops between these two enhancers and the promoter of <i>FOXP2</i> were successfully identified by the Hi-C data (<a id="crosref0480" href="#fig1">Figure&nbsp;1</a>E).</div><div id="p0065" role="paragraph">To assess the penetrance of the detected loops around <i>FOXP2</i>, we performed 3D-DNA FISH using the brain tissue sections of fetal monkeys (<a id="crosref0485" href="#sec-4">STAR methods</a>), and the results are consistent with the Hi-C data that the loop regulating <i>FOXP2</i> are present in both CP (69.0%, 49/71) and GZ (76.7%, 66/86) at similar frequency (Fisher’s exact test, p&nbsp;= 0.284) (<a id="crosref0490" href="#fig1">Figures 1</a>F, 1G, <a id="crosref0495" href="#figs1">S1</a>K, and S1L; <a id="crosref0500" href="#mmc2">Table S2</a>). We saw similar penetrance ratios for another GZ-CP shared loop involving the <i>SOX2</i> gene (63.0%, 53/84 in CP and 57.1%, 48/84 in GZ) (Fisher’s exact test, p&nbsp;= 0.528) (<a id="crosref0505" href="#figs1">Figures S1</a>M and S1N). In contrast, although a GZ-biased loop from Hi-C has overall lower penetrance ratio compared to the GZ-CP shared loops, it has a higher penetrance ratio in GZ (11.4%, 9/79) than in CP (3.7%, 3/81) (Fisher’s exact test, p&nbsp;= 0.077) (<a id="crosref0510" href="#figs1">Figures S1</a>M, S1O, and S1P). These cases indicate that our Hi-C data can effectively capture regulatory events during corticogenesis.</div><div id="p0070" role="paragraph">We next performed cross-species Hi-C analyses at multiple scales, which allows us to characterize human-specific patterns in 3D genome organization.</div></section><section id="sec-2-2"><h3>Cross-species analyses uncover human-specific TADs</h3><div id="p0075" role="paragraph">TADs are fundamental regulatory chromatin structures and are largely conserved across tissues and species (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Dixon et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Dixon, J.R. ∙ Selvaraj, S. ∙ Yue, F. ...</div><div><strong>Topological domains in mammalian genomes identified by analysis of chromatin interactions</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:376-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84861095603" target="_blank" aria-label="Open Scopus in new tab">Scopus (3008)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11082&amp;pmid=22495300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Rao, S.S. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, how TAD changes during evolution influence gene regulation of brain development are not well defined. Here, we characterize the evolutionary dynamics of TADs by comparative brain Hi-C across human, macaque, and mouse (outgroup). The developmental stage (17–18 gestation weeks) of the published human Hi-C data (<span class="dropBlock reference-citations"><a id="body-ref-sref100-5" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-5" aria-controls="sref100-5">Won et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref100-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Won, H. ∙ de la Torre-Ubieta, L. ∙ Stein, J.L. ...</div><div><strong>Chromosome conformation elucidates regulatory relationships in developing human brain</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:523-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84993993658" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19847&amp;pmid=27760116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) is equivalent to that of rhesus macaque, and the mouse Hi-C data were from <i>in&nbsp;vivo</i> neural progenitors (corresponding to GZ) and neurons (corresponding to CP) (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Bonev et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bonev, B. ∙ Mendelson Cohen, N. ∙ Szabo, Q. ...</div><div><strong>Multiscale 3D Genome Rewiring during Mouse Neural Development</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:557-572.e24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85031779520" target="_blank" aria-label="Open Scopus in new tab">Scopus (361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29053968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.043&amp;pmid=29053968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0080" role="paragraph">We identified TADs and TAD boundaries using the directionality index (DI) and the insulation score methods (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Crane et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Crane, E. ∙ Bian, Q. ∙ McCord, R.P. ...</div><div><strong>Condensin-driven remodelling of X chromosome topology during dosage compensation</strong></div><div><em>Nature.</em> 2015; <strong>523</strong>:240-244</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14450" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84936945257" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26030525/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14450&amp;pmid=26030525" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Dixon et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Dixon, J.R. ∙ Selvaraj, S. ∙ Yue, F. ...</div><div><strong>Topological domains in mammalian genomes identified by analysis of chromatin interactions</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:376-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84861095603" target="_blank" aria-label="Open Scopus in new tab">Scopus (3008)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11082&amp;pmid=22495300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0545" href="#figs2">Figure&nbsp;S2</a>A; <a id="crosref0550" href="#sec-4">STAR methods</a>). We compared the features of TADs among human, macaque, and mouse. The number and size of TADs among species are similar in both CP and GZ (<a id="crosref0555" href="#figs2">Figure&nbsp;S2</a>B). We define the shared TADs among the three species as the conserved TADs, and those only gained or lost in human as the human-specific TADs (hsTADs) (<a id="crosref0560" href="#sec-4">STAR methods</a>). We found 499 hsTADs, including 410 human gained TADs (hgTADs) and 89 human lost TADs (hlTADs) (<a id="crosref0565" href="#figs2">Figure&nbsp;S2</a>C). The medium size of hgTADs is 640 kb, smaller than the conserved TADs (900 kb) (<a id="crosref0570" href="#figs2">Figure&nbsp;S2</a>C), consistent with the published data showing a large size of the conserved TADs by comparing human, gibbon, and other mammalian species (<span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Lazar et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Lazar, N.H. ∙ Nevonen, K.A. ∙ O’Connell, B. ...</div><div><strong>Epigenetic maintenance of topological domains in the highly rearranged gibbon genome</strong></div><div><em>Genome Res.</em> 2018; <strong>28</strong>:983-997</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.233874.117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85049247052" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29914971/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.233874.117&amp;pmid=29914971" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It was reported that the transcriptionally active domains in <i>Drosophila</i> tend to have smaller size than the inactive domains (<span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Sexton et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Sexton, T. ∙ Yaffe, E. ∙ Kenigsberg, E. ...</div><div><strong>Three-dimensional folding and functional organization principles of the Drosophila genome</strong></div><div><em>Cell.</em> 2012; <strong>148</strong>:458-472</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2012.01.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2012.01.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84856747483" target="_blank" aria-label="Open Scopus in new tab">Scopus (1046)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22265598/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2012.01.010&amp;pmid=22265598" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hence, hgTADs are likely active regulatory domains. By contrast, hlTADs occur less frequently than hgTADs in the human genome, likely due to a strong functional constraint on loosing previously functional TADs during evolution, and their size (1,320 kb) is larger than the conserved TADs (<a id="crosref0585" href="#figs2">Figure&nbsp;S2</a>C). We saw similar species-specific TAD size patterns in macaque and mouse (<a id="crosref0590" href="#figs2">Figure&nbsp;S2</a>C).</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/ed8096e9-1ee3-4987-936e-448908764cd8/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.01.001/asset/453b14d7-6b07-42bc-bdf3-38724984166f/main.assets/figs2.jpg" height="952" width="752" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/ed8096e9-1ee3-4987-936e-448908764cd8/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Characterization of TAD among species, related to <a id="crosref1910" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0365" role="paragraph">(A) Representative Hi-C maps of human (top), macaque (middle) and mouse (bottom), along with insulation score and domain regions detected by domain caller. Boundaries are indicated by green lines. (B) Bar plots showing the numbers (left) and sizes (right) of TADs detected per brain zone per species. (C) Bar plots showing the numbers (left) of and sizes (right) of species-conserved and species-specific TADs. (D) Violin plots showing insulation strength (Methods) comparisons between the species-conserved and species-specific TAD boundaries in human (left), macaque (middle) and mouse (right). P values are calculated by t&nbsp;test. <sup>∗∗∗</sup>: p&nbsp;&lt; 2.2&nbsp;× 10-16. (E) Enrichment of CTCF ChIP-seq signals in the Cons and HG TAD boundaries for H1NPC, iPSC-induced excitatory neuron and iPSC-induced lower motor neuron. (F) Representative Hi-C maps for the Cons (left) and HG (right) TAD boundaries, along with boundaries of different tissues and cell lines. CP: cortical plate; H1: hESC; NPC: neural progenitor cell; CO: dorsolateral prefrontal cortex; AD: adrenal. (G-H) Enrichment for various repeat element classes (G) or subclasses (H) at the HG TAD boundaries relative to the Cons TAD boundaries. (I) Stacked bar plots showing the fractions of differential expressed genes (DEG) with promoters located 20 kb around the Cons or HG TAD boundaries using CP or GZ RNA-seq profiles. The enrichment difference of DEG is evaluated by Fisher’s exact test. (J) Pie charts showing the percentage of the Cons and HG TAD boundaries associated with CTCF and/or genes. (K) GO enrichment analysis for genes with promoter located 20 kb around the Cons or HG TAD boundaries.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">We then compared the TAD boundaries among species (<a id="crosref0595" href="#sec-4">STAR methods</a>). The results show that ∼55% (n&nbsp;= 1,513) of the TADs boundaries are conserved across species, and ∼14% (n&nbsp;= 385) of the TAD boundaries are the hgTAD boundaries (<a id="crosref0600" href="#fig2">Figures 2</a>A and 2B; <a id="crosref0605" href="#mmc3">Table S3</a>).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/768e2b13-1f1a-4d41-8047-53807744e56c/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.01.001/asset/ad155ed4-b163-4ffe-bac4-ad428d347417/main.assets/gr2.jpg" height="842" width="724" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/768e2b13-1f1a-4d41-8047-53807744e56c/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Evolutionary dynamics of TADs</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0060" role="paragraph">(A) Overlap between the TAD boundaries of human, macaque, and mouse.</div><div id="fspara0065" role="paragraph">(B) Heatmap based on the insulation scores for the species-conserved and the species-specific TAD boundaries. Cons, species-conserved; HG, human-gained; RG, macaque-gained; MS, mouse-specific.</div><div id="fspara0070" role="paragraph">(C) The average insulation score profiles in 800 kb regions around the species-conserved and species-specific TAD boundaries.</div><div id="fspara0075" role="paragraph">(D) Pie charts showing the percentages of the TAD boundaries associated with the CTCF binding sites in human (left), macaque (middle), and mouse (right).</div><div id="fspara0080" role="paragraph">(E) Bar plots showing the conservation degrees of the Cons and the HG TAD boundaries among 21 cell lines and tissues. p value is calculated by the Kolmogorov-Smirnov test.</div><div id="fspara0085" role="paragraph">(F) Enrichment for various repeat element classes at the HG TAD boundaries relative to the Cons TAD boundaries.</div><div id="fspara0090" role="paragraph">(G) Boxplots showing the density of the Alu subfamilies at the Cons TAD boundaries, the HG TAD boundaries, and the randomly selected regions. p values are determined by two-sided t test on the normalized transposon counts per 40 kb.</div><div id="fspara0095" role="paragraph">(H) The Hi-C map around the <i>CNTN5</i> gene in human, macaque, and mouse. The Cons TAD boundary is highlighted with yellow bars, and the hgTAD boundary is highlighted with blue bar. The locations of the Alu Y elements in human are indicated by red lines.</div><div id="fspara0100" role="paragraph">See also <a id="crosref1860" href="#figs2">Figure&nbsp;S2</a> and <a id="crosref1865" href="#mmc3">Table S3</a>.</div></div></div></div></figcaption></figure></div><div id="p0090" role="paragraph">We further explored the features of the conserved and the hgTAD boundaries. The conserved TAD boundaries have stronger insulation than the hgTAD boundaries (<a id="crosref0610" href="#fig2">Figures 2</a>C and <a id="crosref0615" href="#figs2">S2</a>D). Although most of the conserved and the hgTADs boundaries are enriched with CTCF binding, the enrichment levels are lower for the hgTAD boundaries (63%) than the conserved TAD boundaries (77%), indicating that the hgTAD boundaries are less CTCF dependent than the conserved TAD boundaries (<a id="crosref0620" href="#fig2">Figure&nbsp;2</a>D). We also found the hgTAD boundaries showed weaker CTCF ChIP-seq signals than the conserved boundaries in neural cells (<span class="dropBlock reference-citations"><a id="body-ref-sref87-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-1" aria-controls="sref87-1">Song et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref87-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Song, M. ∙ Yang, X. ∙ Ren, X. ...</div><div><strong>Mapping cis-regulatory chromatin contacts in neural cells links neuropsychiatric disorder risk variants to target genes</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:1252-1262</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0472-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85070866778" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31367015/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0472-1&amp;pmid=31367015" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0630" href="#figs2">Figure&nbsp;S2</a>E). We finally examined the tissue and cell type conservation ratios (<span class="dropBlock reference-citations"><a id="body-ref-sref81-1" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-1" aria-controls="sref81-1">Schmitt et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref81-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Schmitt, A.D. ∙ Hu, M. ∙ Jung, I. ...</div><div><strong>A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome</strong></div><div><em>Cell Rep.</em> 2016; <strong>17</strong>:2042-2059</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85003815206" target="_blank" aria-label="Open Scopus in new tab">Scopus (272)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27851967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2016.10.061&amp;pmid=27851967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and found that the conserved TAD boundaries had higher conservation ratio across tissues and cell types, while the hgTAD boundaries were more tissue and cell-type-specific (<a id="crosref0640" href="#fig2">Figures 2</a>E and <a id="crosref0645" href="#figs2">S2</a>F). Collectively, through cross-species comparison, we were able to identify human-specific TADs, especially the hgTADs, which showed distinct features in view of size, CTCF binding strength, and tissue/cell specificity, an implication functional specialization of these newly evolved TADs during human brain evolution.</div></section><section id="sec-2-3"><h3>The hgTAD boundaries are enriched with evolutionarily young transposable elements</h3><div id="p0095" role="paragraph">Previously, it was reported that the mammalian interspersed repeats (MIRs), a conserved family of transposable elements (TEs), can serve as insulators within TAD boundaries in a CTCF-independent manner (<span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Wang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Wang, J. ∙ Vicente-García, C. ∙ Seruggia, D. ...</div><div><strong>MIR retrotransposon sequences provide insulators to the human genome</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4428-E4437</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507253112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26216945/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507253112&amp;pmid=26216945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We performed an enrichment analysis for TEs in the human genome and compared the patterns between the hgTAD boundaries and the conserved TAD boundaries. We found the conserved TAD boundaries were enriched for evolutionarily old TEs, including MIR, LINE-2 (L2), Charlie, and MaLR (<a id="crosref0655" href="#figs2">Figure&nbsp;S2</a>G). By contrast, the hgTAD boundaries are enriched in evolutionarily young TEs, and four TE families are slightly enriched, including Alu, LINE-1 (L1), human endogenous retrovirus subfamily 1, and K (ERV1 and ERVK) (<a id="crosref0660" href="#fig2">Figures 2</a>F and <a id="crosref0665" href="#figs2">S2</a>G). In particular, compared to the conserved boundaries, the hgTAD boundaries are significantly enriched in AluY, the youngest Alu elements in primates (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Kapitonov and Jurka, 1996</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Kapitonov, V. ∙ Jurka, J.</div><div><strong>The age of Alu subfamilies</strong></div><div><em>J.&nbsp;Mol. Evol.</em> 1996; <strong>42</strong>:59-65</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00163212" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0030045979" target="_blank" aria-label="Open Scopus in new tab">Scopus (239)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8576965/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00163212&amp;pmid=8576965" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while no significant enrichment was detected in the relatively old Alu elements (AluJ and AluS) (<a id="crosref0675" href="#fig2">Figure&nbsp;2</a>G), implying that the young Alu elements might be actively involved in forming the human-specific contact patterns (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Cournac et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Cournac, A. ∙ Koszul, R. ∙ Mozziconacci, J.</div><div><strong>The 3D folding of metazoan genomes correlates with the association of similar repetitive elements</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:245-255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1292" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84959206565" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26609133/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1292&amp;pmid=26609133" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similarly, in the L1 subfamilies, the evolutionarily young L1PA is significantly enriched for the hgTAD boundaries, but not the relatively old L1PB (<a id="crosref0685" href="#figs2">Figure&nbsp;S2</a>H).</div><div id="p0100" role="paragraph">We then examined the genes associated with the conserved and the hgTAD boundaries (<a id="crosref0690" href="#sec-4">STAR methods</a>). We found that genes associated with the hgTAD boundaries showed marginally significant expression difference across species compared with the conserved TAD boundaries (Fisher’s exact test, p&nbsp;= 0.06) (<a id="crosref0695" href="#figs2">Figure&nbsp;S2</a>I). The conserved TAD boundaries were more enriched in housekeeping genes, while the hgTAD boundaries were more enriched in brain-development-related genes (<a id="crosref0700" href="#figs2">Figures S2</a>J and S2K; <a id="crosref0705" href="#mmc3">Table S3</a>). One example is the hgTAD boundary around <i>CNTN5</i>, a gene involved in neuron circuit formation and autism spectrum disorders (ASDs) (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Ashrafi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Ashrafi, S. ∙ Betley, J.N. ∙ Comer, J.D. ...</div><div><strong>Neuronal Ig/Caspr recognition promotes the formation of axoaxonic synapses in mouse spinal cord</strong></div><div><em>Neuron.</em> 2014; <strong>81</strong>:120-129</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2013.10.060&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2013.10.060&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84891809781" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24411736/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2013.10.060&amp;pmid=24411736" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Poot, 2014</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Poot, M.</div><div><strong>A candidate gene association study further corroborates involvement of contactin genes in autism</strong></div><div><em>Mol. Syndromol.</em> 2014; <strong>5</strong>:229-235</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1159/000362891" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84906874671" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25337070/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1159%2F000362891&amp;pmid=25337070" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Toyoshima et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Toyoshima, M. ∙ Sakurai, K. ∙ Shimazaki, K. ...</div><div><strong>Deficiency of neural recognition molecule NB-2 affects the development of glutamatergic auditory pathways from the ventral cochlear nucleus to the superior olivary complex in mouse</strong></div><div><em>Dev. Biol.</em> 2009; <strong>336</strong>:192-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2009.09.043" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-70449529713" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19818338/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2009.09.043&amp;pmid=19818338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">van Daalen et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>van Daalen, E. ∙ Kemner, C. ∙ Verbeek, N.E. ...</div><div><strong>Social Responsiveness Scale-aided analysis of the clinical impact of copy number variations in autism</strong></div><div><em>Neurogenetics.</em> 2011; <strong>12</strong>:315-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10048-011-0297-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84855277684" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21837366/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10048-011-0297-2&amp;pmid=21837366" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This hgTAD boundary contains Alu Y elements and is correlated with an increased <i>CNTN5</i> expression in human than in macaque (<a id="crosref0730" href="#fig2">Figure&nbsp;2</a>H).</div><div id="p0105" role="paragraph">Collectively, we observed a significant enrichment of TEs, especially the evolutionarily young TEs (AluY and L1PA) at the hgTADs boundaries, which may contribute to the origin of the human-specific TADs in brain during evolution.</div></section><section id="sec-2-4"><h3>The human-specific chromatin loops are enriched in enhancer-enhancer interactions</h3><div id="p0110" role="paragraph">Compared to TADs, chromatin loops are more dynamic across developmental stages and may affect gene expression (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Bonev et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bonev, B. ∙ Mendelson Cohen, N. ∙ Szabo, Q. ...</div><div><strong>Multiscale 3D Genome Rewiring during Mouse Neural Development</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:557-572.e24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85031779520" target="_blank" aria-label="Open Scopus in new tab">Scopus (361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29053968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.043&amp;pmid=29053968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We identified loops in the three species at 10 kb resolution using the HICCUPS pipeline (<span class="dropBlock reference-citations"><a id="body-ref-sref77-3" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-3" aria-controls="sref77-3">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Rao, S.S. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0745" href="#figs3">Figure&nbsp;S3</a>A; <a id="crosref0750" href="#sec-4">STAR methods</a>). Aggregating peak analysis (APA) demonstrated the accuracy of loop calling (<a id="crosref0755" href="#figs3">Figure&nbsp;S3</a>B). This was further supported by the epigenetic features of loops (<a id="crosref0760" href="#figs3">Figure&nbsp;S3</a>C).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/7246f472-9470-4c11-99ff-00c78221201a/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.01.001/asset/086a18c8-0ad4-4a9f-ab71-52db4d2980b9/main.assets/figs3.jpg" height="957" width="689" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/7246f472-9470-4c11-99ff-00c78221201a/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Characterization of loop anchors among species, related to <a id="crosref1915" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0375" role="paragraph">(A) Representative Hi-C maps for human, macaque and mouse. The detected loops are indicated by dashed boxes. (B) APA plots of all detected loops per brain zone in human, macaque and mouse. (C) Bar plots showing the percentages of loops that connect promoter-promoter, promoter-enhancer and enhancer-enhancer in three species. P value was determined by Fisher’s exact test. <sup>∗</sup>: p&nbsp;&lt; 2.2&nbsp;× 10<sup>−16</sup>. (D) APA plots for the macaque-specific loops (RSLs) and the mouse-specific loops (bottom) in human, macaque and mouse. (E) Cumulative percentage plots comparing the size distributions of the species-conserved and species-specific loops in human (top), macaque (middle) and mouse (bottom). P values are calculated by Kolmogorov-Smirnov test. (F) Bar plots showing the percentage of the species-conserved and species-specific loops with CTCF binding sites in human (top), macaque (middle) and mouse (bottom). P values are calculated by Fisher’s exact test. (G-H) Bar plots showing the percentages of loops that connect promoter-promoter, promoter-enhancer and enhancer-enhancer in macaque (G) and mouse (H). P value was determined by Fisher’s exact test. (I) The Hi-C map around <i>AUTS2</i>. HSLs are indicated by red boxes, and other loops are indicated by dark boxes. The corresponding positions of HSLs are also indicated (by red boxes) in the Hi-C maps of macaque and mouse. (J) Two putative enhancers overlapped with loop anchors, predicted by VISTA, which can drive midbrain-specific and neural tube-specific expression in mouse embryos. (K) Boxplots showing the expression levels of <i>AUTS2</i> in human and macaque.</div></div></div></div></figcaption></figure></div><div id="p0115" role="paragraph">Next, we compared loops across species using the merged CP and GZ loop profiles so that the species-specific rather than the zone-specific changes can be detected. In total, we identified 1,904 conserved loops (ConsLs) among the three species and 1,266 human-specific loops (HSLs) (<a id="crosref0765" href="#fig3">Figures 3</a>A and 3B; <a id="crosref0770" href="#mmc4">Table S4</a>), and these loops were validated by APA (<a id="crosref0775" href="#fig3">Figures 3</a>A and <a id="crosref0780" href="#figs3">S3</a>D). In general, the lengths of HSLs and the macaque-specific loops are significantly smaller than that of ConsLs (Kolmogorov-Smirnov test, p&nbsp;&lt; 2.2&nbsp;× 10<sup>−16</sup>), while the pattern is opposite in mouse (<a id="crosref0785" href="#figs3">Figure&nbsp;S3</a>E). Similar to the result of the TAD boundaries, the species-specific loop anchors are less enriched in CTCF ChIP-seq signals than the ConsL anchors (<a id="crosref0790" href="#figs3">Figure&nbsp;S3</a>F).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/983d5d9b-929d-4acb-85f1-3eea948d882f/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.01.001/asset/d1923315-8247-47c1-a795-f34dcf1b4bda/main.assets/gr3.jpg" height="731" width="755" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/983d5d9b-929d-4acb-85f1-3eea948d882f/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">HSLs are enriched for enhancer-enhancer interactions</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0110" role="paragraph">(A) The APA plots of the conserved loops (ConsL, top) and the human-specific loops (HSL, bottom) among human (H), macaque (R) and mouse (M).</div><div id="fspara0115" role="paragraph">(B) Bar plots showing the numbers of the conserved and the species-specific loops.</div><div id="fspara0120" role="paragraph">(C and D) The percentages of ConsLs and HSLs with gene promoters (C) or enhancers (D) at the anchors. p value is determined by Fisher’s exact test.</div><div id="fspara0125" role="paragraph">(E) The percentages of ConsLs and HSLs that connect promoter-promoter, promoter-enhancer and enhancer-enhancer. p value is determined by Fisher’s exact test.</div><div id="fspara0130" role="paragraph">(F) Twenty randomly chosen interaction communities, showing the diversity of the inferred communities.</div><div id="fspara0135" role="paragraph">(G) Boxplots showing the co-expression correlation for genes from the same communities (S. Genes) and genes from different communities (D. Genes) using the RNA-seq profiles of CP (left) and GZ (right). p values are determined by two-sided t test.</div><div id="fspara0140" role="paragraph">(H) The interaction community around the <i>AUTS2</i> gene.</div><div id="fspara0145" role="paragraph">(I) Loops and enhancers predicated by VISTA, along with histone and CTCF ChIP-seq, ATAC-seq, and RNA-seq profiles around <i>AUTS2</i> in human and macaque. HSLs are indicated by red lines. The human-biased CTCF binding sites around loop anchors are highlighted with yellow bars.</div><div id="fspara0150" role="paragraph">See also <a id="crosref1870" href="#figs3">Figure&nbsp;S3</a> and <a id="crosref1875" href="#mmc4">Table S4</a>.</div></div></div></div></figcaption></figure></div><div id="p0120" role="paragraph">To explore the function of the different loop categories, we integrated them with human fetal brain H3K4me1 signals (Kundaje et&nbsp;al., 2015) which reflect enhancer potentials. Remarkably, HSLs are significantly enriched for enhancers compared to ConsLs (Fisher’s exact test, p&nbsp;&lt; 2.2&nbsp;× 10<sup>−16</sup>), while no such enrichment is detected for promoters (<a id="crosref0795" href="#fig3">Figures 3</a>C and 3D). Almost 80% of the HSL anchors contain at least one enhancer, compared to less than 60% for the ConsL anchors (<a id="crosref0800" href="#fig3">Figure&nbsp;3</a>D). In terms of loop types, the enhancer-promoter interactions account for only ∼27% of HSLs, and nearly 40% of HSLs are enhancer-enhancer interactions, which is a significant enrichment compared to ConsLs (Fisher’s exact test, p&nbsp;&lt; 2.2&nbsp;× 10<sup>−16</sup>) (<a id="crosref0805" href="#fig3">Figure&nbsp;3</a>E). The enrichment of enhancer-enhancer interactions is not seen in the macaque/mouse specific loops (<a id="crosref0810" href="#figs3">Figures S3</a>G and S3H).</div><div id="p0125" role="paragraph">Enhancer-enhancer interactions are implicated in gene control by forming enhancer cliques or spatially interaction hubs to regulate gene expression (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Mumbach et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Mumbach, M.R. ∙ Satpathy, A.T. ∙ Boyle, E.A. ...</div><div><strong>Enhancer connectome in primary human cells identifies target genes of disease-associated DNA elements</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1602-1612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85032453347" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28945252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3963&amp;pmid=28945252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Petrovic et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Petrovic, J. ∙ Zhou, Y. ∙ Fasolino, M. ...</div><div><strong>Oncogenic Notch Promotes Long-Range Regulatory Interactions within Hyperconnected 3D Cliques</strong></div><div><em>Mol. Cell.</em> 2019; <strong>73</strong>:1174-1190.e12</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2019.01.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2019.01.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85063029087" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30745086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.01.006&amp;pmid=30745086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Sanborn et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Sanborn, A.L. ∙ Rao, S.S. ∙ Huang, S.C. ...</div><div><strong>Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E6456-E6465</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1518552112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84948403758" target="_blank" aria-label="Open Scopus in new tab">Scopus (653)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26499245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1518552112&amp;pmid=26499245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To identify target genes regulated by enhancer-enhancer interactions, we generated an undirected graph using the Hi-C loops and detected communities using the fast greedy algorithm (<span class="dropBlock reference-citations"><a id="body-ref-sref71-1" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-1" aria-controls="sref71-1">Phanstiel et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref71-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Phanstiel, D.H. ∙ Van Bortle, K. ∙ Spacek, D. ...</div><div><strong>Static and Dynamic DNA Loops form AP-1-Bound Activation Hubs during Macrophage Development</strong></div><div><em>Mol. Cell.</em> 2017; <strong>67</strong>:1037-1048.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2017.08.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2017.08.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85030242070" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28890333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2017.08.006&amp;pmid=28890333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0835" href="#sec-4">STAR methods</a>). In many communities, multiple enhancers regulate one or several promoters (<a id="crosref0840" href="#fig3">Figure&nbsp;3</a>F). Genes in the same communities have higher co-expression correlation than those in different communities both in CP and GZ (<a id="crosref0845" href="#fig3">Figure&nbsp;3</a>G), suggesting that the detected communities may represent co-regulation units.</div><div id="p0130" role="paragraph">One remarkable case showing the regulatory feature of the human-specific interactions involves the <i>AUTS2</i> (autism susceptibility candidate 2) gene. Strikingly, there is a complex interaction hub regulating <i>AUTS2</i> in the human fetal brain. The hub contains 14 enhancers and more than half of the loops in the hub are human-specific (<a id="crosref0850" href="#fig3">Figures 3</a>H, 3I, and <a id="crosref0855" href="#figs3">S3</a>I). Two human enhancers (hs1425 and hs658) showed midbrain and neural-tube-specific expression in mouse embryos (<span class="dropBlock reference-citations"><a id="body-ref-sref96-2" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-2" aria-controls="sref96-2">Visel et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref96-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Visel, A. ∙ Minovitsky, S. ∙ Dubchak, I. ...</div><div><strong>VISTA Enhancer Browser--a database of tissue-specific human enhancers</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:D88-D92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkl822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33846112470" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17130149/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkl822&amp;pmid=17130149" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0865" href="#figs3">Figure&nbsp;S3</a>J). By contrast, there is only one loop detected in macaque and three in mouse within the orthologous region (<a id="crosref0870" href="#fig3">Figures 3</a>I and <a id="crosref0875" href="#figs3">S3</a>I). Consistently, in the CP zone, the expression of <i>AUTS2</i> is significantly higher in human than in macaque (<a id="crosref0880" href="#figs3">Figure&nbsp;S3</a>K). In addition, compared with macaque, human has more CTCF-binding sites in this region, some of which are close to the HSL anchors (<a id="crosref0885" href="#fig3">Figure&nbsp;3</a>I). The difference in CTCF-binding sites may be responsible for the chromatin structure and gene expression difference between human and macaque brains.</div></section><section id="sec-2-5"><h3>Genes associated with HSLs are preferentially expressed in the subplate lamina of the developing brain</h3><div id="p0135" role="paragraph">We defined 1,411 and 1,079 target genes for ConsLs and HSLs, respectively, by combining target genes of direct promoter-enhancer or promoter-promoter loops and target genes of enhancer-enhancer loops through contact communities (<a id="crosref0890" href="#mmc4">Table S4</a>; <a id="crosref0895" href="#sec-4">STAR methods</a>). We found that the HSL target genes tend to be more upregulated in human compared to macaque than the ConsL target genes and the target genes of the macaque-specific loops (<a id="crosref0900" href="#figs4">Figures S4</a>A and S4B), an indication of gene regulatory changes associated with HSLs.</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/bc1618e7-17e1-4ebd-afe7-2dd4e557dada/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.01.001/asset/20e8b942-d3fd-4db5-9e08-6148213bf8cd/main.assets/figs4.jpg" height="952" width="732" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/bc1618e7-17e1-4ebd-afe7-2dd4e557dada/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Functional enrichment of the species-specific and conserved loop regulated genes, related to <a id="crosref1920" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0385" role="paragraph">(A) Boxplots showing the expression level fold change between human and macaque of genes regulated by ConsL, HSL and RLS using the CP RNA-seq profile (left) or the GZ RNA-seq profile (right). (B) Volcano plots of genes regulated by ConsL, HSL and RSL using CP RNA-seq profile. (C) Enrichment of genes regulated by HRLs (human-macaque conserved loops) and RSLs relative to background within human fetal cortical laminae. P values are determined by two-sided t test. (D) The percentages of genes regulated by HSL, HRL and RSL overlap with markers of cell types from human fetal cortex. (E) Enrichment of target genes regulated by ConsL and HSL in modules from published data (<span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Miller et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Red line indicates the P value of 0.01. P values are calculated by Permutation test. (F) ME expression of HSL enriched modules averaged across brain and layer. White, low expression; red high expression. (G) GO enrichment analysis of target genes regulated by HSL (top) and ConsL (bottom). (H) REACTOME pathway analysis of target genes regulated by HSL (top) and ConsL (bottom).</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph">Gene regulation usually works in a tissue and cell-type-specific manner. We analyzed enrichment of target genes in different laminae using the microarray data from the laser-micro-dissected human fetal brain (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Miller et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Surprisingly, the HSL target genes are significantly enriched in inner cortical plate (CPi) and subplate (SP) compared to the ConsL target genes (<a id="crosref0910" href="#fig4">Figure&nbsp;4</a>A). For example, <i>NR4A2</i> and <i>TMEM178A</i>, the two SP preferentially expressed genes, are regulated by HSLs (<a id="crosref0915" href="#mmc4">Table S4</a>). We confirmed this pattern by checking the published human fetal brain single-cell RNA-seq data (<span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Nowakowski et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (220)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Interestingly, the HSL target genes are enriched in excitatory projection neurons from the cortical deep layer, and this enrichment is the most prominent in the excitatory neurons of SP (<a id="crosref0925" href="#fig4">Figure&nbsp;4</a>B). In contrast, the target genes of the macaque specific loops are not enriched in any lamina (<a id="crosref0930" href="#figs4">Figure&nbsp;S4</a>C). We also checked the target genes of the human-macaque shared loops (not present in mouse), and we saw a significant enrichment in ventricular zone (VZ) (<a id="crosref0935" href="#figs4">Figure&nbsp;S4</a>C), consistent with the view that the primate brains have an enlarged neural progenitor pool compared with rodents (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Lui et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Lui, J.H. ∙ Hansen, D.V. ∙ Kriegstein, A.R.</div><div><strong>Development and evolution of the human neocortex</strong></div><div><em>Cell.</em> 2011; <strong>146</strong>:18-36</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-79959924122" target="_blank" aria-label="Open Scopus in new tab">Scopus (677)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21729779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2011.06.030&amp;pmid=21729779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, the target genes of the human-macaque shared loops are slightly enriched in excitatory SP neurons, but statistically not significant (<a id="crosref0945" href="#figs4">Figure&nbsp;S4</a>D).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/22750d77-77d3-4794-9d3f-58ff9b837629/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.01.001/asset/5b69bb46-19e3-4aaa-b0b5-c88b05799874/main.assets/gr4.jpg" height="899" width="763" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/22750d77-77d3-4794-9d3f-58ff9b837629/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Target genes regulated by HSLs are enriched in the subplate lamina</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0160" role="paragraph">(A) Enrichment of genes regulated by ConsLs and HSLs in the human fetal cortical laminae (PCW15/16/21). p values are determined by two-sided t test.</div><div id="fspara0165" role="paragraph">(B) The percentages of genes regulated by ConsLs and HSLs overlapped with highly-expressed gene sets of cell types from the human fetal cortex. MEG-RG/PC, medial ganglionic eminence derived radial glia and progenitor cells; CTX-RG, cortical derived radial glia; OPC, oligodendrocyte progenitor cell; IN, interneuron; IPC/newborn-EN, intermediate progenitor cell/newborn excitatory neuron; EPN, excitatory projection neuron; ESP, excitatory subplate neuron. p values are determined by Fisher’s exact test.</div><div id="fspara0170" role="paragraph">(C) Module eigengene (ME) expression of module-16 averaged across brain and laminae. White, low expression; red, high expression (left). GO analysis of ME for module-16 (middle). Top gene-gene co-expression connections for module-16 (right). Genes regulated by HSLs are represented in blue, and the other genes are represented in yellow.</div><div id="fspara0175" role="paragraph">(D) Schematic view of the axon guidance pathway.</div><div id="fspara0180" role="paragraph">(E) Boxplot showing the expression level of <i>EPHA7</i> in human fetal cortical laminae. Data from PCW15/16/21 are used (<span class="dropBlock reference-citations"><a id="body-ref-sref62-3" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-3" aria-controls="sref62-3">Miller et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref62-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0185" role="paragraph">(F) Left: cell clusters from the published data (human fetal brain, GW17-18) (<span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Polioudakis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Polioudakis, D. ∙ de la Torre-Ubieta, L. ∙ Langerman, J. ...</div><div><strong>A Single-Cell Transcriptomic Atlas of Human Neocortical Development during Mid-gestation</strong></div><div><em>Neuron.</em> 2019; <strong>103</strong>:785-801.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85070182242" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31303374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2019.06.011&amp;pmid=31303374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) are presented in the dimension-reduced UMAP plot. Right: expression of <i>EPHA7</i> in the cell clusters.</div><div id="fspara0190" role="paragraph">(G) Boxplot showing expression difference of <i>EPHA7</i> between human and macaque.</div><div id="fspara0195" role="paragraph">(H) Representative Hi-C map at the <i>EPHA7</i> locus. The HSL regulating <i>EPHA7</i> is indicated by blue circle. The count track represents the interaction profile between loop anchors and their flanking regions.</div><div id="fspara0200" role="paragraph">See also <a id="crosref1890" href="#figs4">Figure&nbsp;S4</a>.</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">We then explored the function of the HSL target genes. First, we performed gene co-expression module enrichment analysis using the modules identified in the previous report (<span class="dropBlock reference-citations"><a id="body-ref-sref62-4" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-4" aria-controls="sref62-4">Miller et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref62-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found that the HSL target genes were significantly enriched in five modules (M6, M16, M26, M34, and M38) (<a id="crosref0955" href="#figs4">Figures S4</a>E and S4F). The most enriched module is M16, which is enriched in SP, and the function of M16 is related to synaptic transmission and synaptogenesis (<a id="crosref0960" href="#fig4">Figure&nbsp;4</a>C), consistent with the known function of SP (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Judaš et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Judaš, M. ∙ Sedmak, G. ∙ Kostović, I.</div><div><strong>The significance of the subplate for evolution and developmental plasticity of the human brain</strong></div><div><em>Front. Hum. Neurosci.</em> 2013; <strong>7</strong>:423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fnhum.2013.00423" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84880141282" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23935575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffnhum.2013.00423&amp;pmid=23935575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Of note, there are many synapse-related genes regulated by HSLs in M16, such as <i>NLGN1</i>, one of the synaptic cell-adhesion molecules that mediate synapse formation (<a id="crosref0970" href="#fig4">Figure&nbsp;4</a>C). Alterations of <i>NLGN1</i> have been implicated in autism and other cognitive diseases (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Dean and Dresbach, 2006</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Dean, C. ∙ Dresbach, T.</div><div><strong>Neuroligins and neurexins: linking cell adhesion, synapse formation and cognitive function</strong></div><div><em>Trends Neurosci.</em> 2006; <strong>29</strong>:21-29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.tins.2005.11.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.tins.2005.11.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-30344454380" target="_blank" aria-label="Open Scopus in new tab">Scopus (222)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16337696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tins.2005.11.003&amp;pmid=16337696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The other four enriched modules are involved in regulation of transcription (M6), long-term depression (M26), DNA replication (M34) and cell cycle (M38) (<a id="crosref0980" href="#figs4">Figure&nbsp;S4</a>F). These data suggest HSLs regulate genes in several functional modules, especially synapse development of the SP neurons.</div><div id="p0150" role="paragraph">Next, we performed Gene Ontology (GO) enrichment analysis, and the results show that the HSL target genes are related to biological processes such as axon guidance, neuron migration, and synapse assembly (<a id="crosref0985" href="#figs4">Figures S4</a>G and S4H; <a id="crosref0990" href="#mmc4">Table S4</a>). Axon guidance plays an important role in shaping neuron circuits (<span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">Stoeckli, 2018</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Stoeckli, E.T.</div><div><strong>Understanding axon guidance: are we nearly there yet?</strong></div><div><em>Development.</em> 2018; <strong>145</strong>:dev151415</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.151415" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85048007634" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29759980/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.151415&amp;pmid=29759980" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Many genes involved in the axon guidance pathway are regulated by HSLs, especially genes encoding key guidance molecules (ephrins and semaphorins) such as <i>EPHA7</i> and <i>SEMA3A</i> (<a id="crosref1000" href="#fig4">Figure&nbsp;4</a>D). Altogether, these results further support our hypothesis that the human-specific chromatin interactions play a key role in human cortex development, especially the human-specific changes in the SP lamina during early neural circuit development.</div><div id="p0155" role="paragraph">One promising HSL target gene is <i>EPHA7</i>. It is SP-preferentially expressed and upregulated in humans compared with macaque (<a id="crosref1005" href="#fig4">Figures 4</a>E–4G), and can influence cortical dendritic development and spine maturation (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Clifford et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Clifford, M.A. ∙ Athar, W. ∙ Leonard, C.E. ...</div><div><strong>EphA7 signaling guides cortical dendritic development and spine maturation</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:4994-4999</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1323793111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84897509280" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24707048/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1323793111&amp;pmid=24707048" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We identified a human-specific loop showing enhancer-promoter interaction in the human fetal brain (<a id="crosref1015" href="#fig4">Figure&nbsp;4</a>H). Concordantly, the enhancer activity in macaque is much weaker than that in human (<a id="crosref1020" href="#fig4">Figure&nbsp;4</a>H). Furthermore, the enhancer in HSL anchor is brain-specific compared with other human fetal tissues (<span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Kundaje et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Kundaje, A. ∙ Meuleman, W. ∙ Ernst, J. ..., Roadmap Epigenomics Consortium</div><div><strong>Integrative analysis of 111 reference human epigenomes</strong></div><div><em>Nature.</em> 2015; <strong>518</strong>:317-330</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14248" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84923362619" target="_blank" aria-label="Open Scopus in new tab">Scopus (2646)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25693563/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14248&amp;pmid=25693563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1030" href="#fig5">Figures 5</a>A and 5B). We used a luciferase assay to validate the enhancer activity among species, the results show that the enhancer activity in human is significantly higher than those of chimpanzee and macaque (<a id="crosref1035" href="#fig5">Figure&nbsp;5</a>C). We compared the enhancer sequences among species and found six evolutionarily conserved sites that only changed in the human lineage (<a id="crosref1040" href="#fig5">Figure&nbsp;5</a>D). These six human-specific single nucleotide changes (SNCs) formed new potential transcription factor (TF) binding sites in human (<a id="crosref1045" href="#fig5">Figure&nbsp;5</a>D). Therefore, we speculate that the new TF binding sites may induce human-specific chromatin interaction changes, leading to expression divergence between human and macaque (<a id="crosref1050" href="#fig4">Figure&nbsp;4</a>G).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/20650e04-a163-46ab-817b-ca69748fd6c6/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.01.001/asset/2eeaba82-4fae-4492-83de-03e9a73d2449/main.assets/gr5.jpg" height="532" width="762" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/20650e04-a163-46ab-817b-ca69748fd6c6/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">HSL-regulated <i>EPHA7</i> impacts early neural circuit development</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0210" role="paragraph">(A) Genome browser tracks of histone modification profiles, CTCF ChIP-seq and ATAC-seq profiles around the HSL regulating <i>EPHA7</i>. The putative distal enhancer and promoter of <i>EPHA7</i> are highlighted in yellow.</div><div id="fspara0215" role="paragraph">(B) The putative enhancer at the HSL distal anchor has brain-specific peaks in the H3K4me1 ChIP-seq profiles.</div><div id="fspara0220" role="paragraph">(C) <i>EPHA7</i> enhancer activity among primates measured by luciferase assay (t test, n&nbsp;= 6, mean ± SEM).</div><div id="fspara0225" role="paragraph">(D) Sequences and predicted motif binding sites in the putative enhancers of human and macaque. The human-specific sequence change is indicated by the red dashed box.</div><div id="fspara0230" role="paragraph">(E) Schematic of functional validation of the human-specific <i>EPHA7</i> enhancer (Chr6:93529923-93531028).</div><div id="fspara0235" role="paragraph">(F) Genomic PCR of the 1.1 kb region containing the <i>EPHA7</i> enhancer. The red arrows point to the expected knockout bands.</div><div id="fspara0240" role="paragraph">(G) Knockout of the <i>EPHA7</i> enhancer leads to a 50% reduction of <i>EPHA7</i> expression as measured by qPCR in neurons differentiated from hNPCs (p&nbsp;= 0.0017, t test, n&nbsp;= 3, mean ± SEM).</div><div id="fspara0245" role="paragraph">(H and I) The <i>EPHA7</i> enhancer knockout leads to a significant increase of dendrite length in neurons after 2&nbsp;weeks of differentiation from hNPCs. Neuron morphologies were traced using the Simple Neurite Tracer plugin in ImageJ (p&nbsp;= 1.2&nbsp;× 10<sup>−5</sup>, t test, mean ± SEM).</div></div></div></div></figcaption></figure></div><div id="p0160" role="paragraph">To explore the functional role of the enhancer regulating human <i>EPHA7</i>, we disturbed the enhancer sequence using the CRISPR-Cas9 method (<a id="crosref1055" href="#fig5">Figure&nbsp;5</a>E). Three deletions were introduced within the enhancer locus, resulting in a 50% reduction of <i>EPHA7</i> expression in neurons differentiated from human NPCs (<a id="crosref1060" href="#fig5">Figures 5</a>F and 5G). Next, we evaluated whether the disturbed enhancer affects neuron dendrite development. We observed significantly longer dendrite length in neurons with the disturbed enhancer compared with the controls (<a id="crosref1065" href="#fig5">Figures 5</a>H and 5I). These data suggest that the human-specific enhancer of the SP-preferentially expressed <i>EPHA7</i> gene plays an important role in early neural circuit development which may contribute to the unique traits of the human brain.</div></section><section id="sec-2-6"><h3>HSL anchors are enriched in motifs of transcription factors regulating SP neuron specialization</h3><div id="p0165" role="paragraph">TFs can bind to specific DNA motifs within gene regulatory elements and influence the chromatin 3D conformation to determine cell fate during development (<span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Stadhouders et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Stadhouders, R. ∙ Vidal, E. ∙ Serra, F. ...</div><div><strong>Transcription factors orchestrate dynamic interplay between genome topology and gene regulation during cell reprogramming</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:238-249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-017-0030-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85040677170" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29335546/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-017-0030-7&amp;pmid=29335546" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" aria-label="Reference 2019" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">2019</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Stadhouders, R. ∙ Filion, G.J. ∙ Graf, T.</div><div><strong>Transcription factors and 3D genome conformation in cell-fate decisions</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:345-354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1182-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85065766396" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31092938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1182-7&amp;pmid=31092938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To explore how TFs may interact with the human brain 3D genome and regulate SP cell fate decision, we performed motif binding site analysis in the open chromatin regions of the HSL and ConsL anchors using the ATAC-seq profile from CP (<a id="crosref1080" href="#sec-4">STAR methods</a>). As expected, the architectural proteins such as CTCF and CTCFL fall in the top of the TF list (<a id="crosref1085" href="#figs5">Figure&nbsp;S5</a>A). The percentage of binding in anchors and the significance of enrichment show distinctive changes for some of the motifs between HSL and ConsL anchors (<a id="crosref1090" href="#fig6">Figures 6</a>A and 6B). Architectural proteins (CTCF and CTCFL) are less enriched in the HSL anchors compared to the ConsL ones, consistent with our previous finding that HSLs are less CTCF dependent than ConsLs using CTCF ChIP-seq (<a id="crosref1095" href="#fig6">Figures 6</a>B and <a id="crosref1100" href="#figs3">S3</a>F). In contrast, the neural cell-type-related TFs are more enriched in HSLs compared to ConsLs. Notably, we found three NFI factors (NFIA, NFIC, and NFIX) enriched in the HSL anchors, which can repress self-renewal of neural stem cells and activate the expression of genes associated with neuronal and astrocyte differentiation (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Harris et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Harris, L. ∙ Genovesi, L.A. ∙ Gronostajski, R.M. ...</div><div><strong>Nuclear factor one transcription factors: Divergent functions in developmental versus adult stem cell populations</strong></div><div><em>Dev. Dyn.</em> 2015; <strong>244</strong>:227-238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24182" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84924046828" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25156673/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24182&amp;pmid=25156673" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We obtained similar results in motif binding analysis when using the ATAC-seq profile from GZ (<a id="crosref1110" href="#figs5">Figure&nbsp;S5</a>B) and different motif enrichment methods (<a id="crosref1115" href="#figs5">Figure&nbsp;S5</a>C).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/abe40e4c-50de-48e5-ba7b-61366502ecd7/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.01.001/asset/080c8f03-22f7-4ba6-8ea2-f4a51810c29e/main.assets/gr6.jpg" height="881" width="763" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/abe40e4c-50de-48e5-ba7b-61366502ecd7/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Identification of transcription factor motifs associated with HSLs</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0255" role="paragraph">(A) Scatterplot showing the enrichment changes for various TFs at HSLs relative to ConsLs.</div><div id="fspara0260" role="paragraph">(B) The representative motifs of TF enriched at ConsLs and HSLs.</div><div id="fspara0265" role="paragraph">(C) The SNC enrichment (Fisher’s exact test, -log10 FDR) for TFs with HSL-binding preference. 11 out of 17 TFs are significantly enriched of the human-specific SNCs (FDR &lt;0.001, indicated by the red dotted line).</div><div id="fspara0270" role="paragraph">(D) Examples of the human-specific SNCs-enriched TFs with HSL-binding preference. The human-specific SNCs-containing ratio is compared with the random regions per TFs. p value is calculated by Fisher’s exact test.</div><div id="fspara0275" role="paragraph">(E) Example of a HSL containing a human-specific SNC. The enhancer regulating <i>AUTS2</i> contains several human-specific SNCs that generate a new TGIF1 binding site in human.</div><div id="fspara0280" role="paragraph">(F) The sequence alignments among 6 primate species including human around the human-specific SNC in (E).</div><div id="fspara0285" role="paragraph">(G) The TF-target gene networks for TFs showing differential enrichment between HSLs and ConsLs. Genes regulated by HSLs are in blue, other genes are in yellow.</div><div id="fspara0290" role="paragraph">(H) The expression pattern of TFs in different human fetal brain laminae.</div><div id="fspara0295" role="paragraph">(I) The expression pattern of genes regulated by TBR1 in different human fetal brain laminae.</div><div id="fspara0300" role="paragraph">See also <a id="crosref1895" href="#figs5">Figure&nbsp;S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/a3f7dcac-c96d-4e68-a03a-e597b83ab782/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.01.001/asset/ea81b589-29cb-41d9-b885-ef84078ef47b/main.assets/figs5.jpg" height="588" width="670" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/a3f7dcac-c96d-4e68-a03a-e597b83ab782/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Transcription factor motifs enrichment of the HSL anchors, related to <a id="crosref1930" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0395" role="paragraph">(A-B) TFs motifs enriched in the open chromatin regions defined by CP (A) and GZ (B) ATAC-seq profiles in ConsL (yellow) and HSL (blue). The top 25 TFs are presented, and ranked by P value. (C) Scatterplot showing the comparison of –log10(P value) changes between ConsL and HSL using different motif prediction algorithm. (D) Definitions of the TF set with HSL-binding preference (indicated by red dots) and the TF set without binding preference (indicated by green dots) by the difference of motif enrichment P Value (-log10) between HSL and ConsL, using 10 (indicated by the red line) and ± 2 (indicated by green lines) as thresholds respectively. (E) Percentages of variation-enrichment TFs for three TF sets.</div></div></div></div></figcaption></figure></div><div id="p0170" role="paragraph">Next, we intend to know whether part of these HSL-preferred bindings is sequence driven (i.e., containing SNCs). We checked the enrichment in the human-specific SNCs for TFs showing HSL-binding preference (<a id="crosref1120" href="#figs5">Figure&nbsp;S5</a>D). Approximately 65% (11 out of 17) of these TFs with HSL-binding preference are significantly enriched in the human-specific SNCs (<a id="crosref1125" href="#fig6">Figure&nbsp;6</a>C) (false discovery rate [FDR] &lt;0.001), such as TGIF1, EOMES, and TBR1 (<a id="crosref1130" href="#fig6">Figure&nbsp;6</a>D). The human-fixed SNC enrichment ratio in the TF set with HSL-binding preference (65%) is much higher than all TFs (30.3%) and the TF set without binding preference (33.7%) (<a id="crosref1135" href="#figs5">Figure&nbsp;S5</a>E, Fisher’s exact test, p&nbsp;= 0.0056 and p&nbsp;= 0.016, respectively). One example of the human-specific SNC that generates new binding site is the enhancer regulating <i>AUTS2</i> (<a id="crosref1140" href="#fig3">Figure&nbsp;3</a>K) that contains one human-specific SNC resulting in a new TGIF1 binding site in human (<a id="crosref1145" href="#fig6">Figures 6</a>E and 6F). TGIF1 has an important role in regulating neuronal morphogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Nakashima et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Nakashima, H. ∙ Tsujimura, K. ∙ Irie, K. ...</div><div><strong>Canonical TGF-β Signaling Negatively Regulates Neuronal Morphogenesis through TGIF/Smad Complex-Mediated CRMP2 Suppression</strong></div><div><em>J.&nbsp;Neurosci.</em> 2018; <strong>38</strong>:4791-4810</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.2423-17.2018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85050908774" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29695415/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.2423-17.2018&amp;pmid=29695415" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0175" role="paragraph">We then asked how many HSL target genes are regulated by these TFs showing enrichment difference between ConsLs and HSLs. We define target genes of TFs by combining motif binding analysis of promoter and gene-TF co-expression (<a id="crosref1155" href="#sec-4">STAR methods</a>). We generated TF-target gene networks for the TFs with HSL-binding preference (<a id="crosref1160" href="#fig6">Figure&nbsp;6</a>G). In fact, some of the TFs are the HSL target genes themselves such as TBR1, NFIA, NFIX, and TCF4, which display different lamina-specificity in the expression data. <i>TBR1</i> and <i>NFIA</i> are SP-enriched, while <i>NEUROG1/2</i>, <i>EOMES</i>, and <i>TGIF1</i> are germinal layers enriched (<a id="crosref1165" href="#fig6">Figure&nbsp;6</a>H). Remarkably, TBR1 has the largest number of target genes and most of them show CPi/SP enrichment (<a id="crosref1170" href="#fig6">Figures 6</a>G and 6I). A former study has revealed that <i>TBR1</i> gene loss affects SP formation and thalamocortical connectivity (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Hevner et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Hevner, R.F. ∙ Shi, L. ∙ Justice, N. ...</div><div><strong>Tbr1 regulates differentiation of the preplate and layer 6</strong></div><div><em>Neuron.</em> 2001; <strong>29</strong>:353-366</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2FS0896-6273%2801%2900211-2&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2FS0896-6273%2801%2900211-2&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-17744398634" target="_blank" aria-label="Open Scopus in new tab">Scopus (612)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11239428/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2801%2900211-2&amp;pmid=11239428" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), suggesting that TBR1 might be one of the key TFs regulating SP neuron specialization.</div><div id="p0180" role="paragraph">Collectively, we show that TBR1 and NFIA are candidate TFs mediating expression of the HSL-regulated gene to determine the fate of human SP neurons. Our results also echo a recent study, in which they demonstrated that NFIA can act as FOXP2 coactivators to promote chromatin decondensation and regulate the maturation of early excitatory SP neurons (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Hickey et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Hickey, S.L. ∙ Berto, S. ∙ Konopka, G.</div><div><strong>Chromatin Decondensation by FOXP2 Promotes Human Neuron Maturation and Expression of Neurodevelopmental Disease Genes</strong></div><div><em>Cell Rep.</em> 2019; <strong>27</strong>:1699-1711</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2019.04.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2019.04.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85064887595" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31067457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.04.044&amp;pmid=31067457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-2-7"><h3>Genomic basis of the human-specific chromatin structure changes</h3><div id="p0185" role="paragraph">Former studies have shown that sequence changes in the genome may cause chromatin structural changes and pathogenic phenotypes (<span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Lupiáñez et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Lupiáñez, D.G. ∙ Kraft, K. ∙ Heinrich, V. ...</div><div><strong>Disruptions of topological chromatin domains cause pathogenic rewiring of gene-enhancer interactions</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1012-1025</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2015.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2015.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84930091577" target="_blank" aria-label="Open Scopus in new tab">Scopus (829)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25959774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.04.004&amp;pmid=25959774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Spielmann et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Spielmann, M. ∙ Lupiáñez, D.G. ∙ Mundlos, S.</div><div><strong>Structural variation in the 3D genome</strong></div><div><em>Nat. Rev. Genet.</em> 2018; <strong>19</strong>:453-467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41576-018-0007-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85045905339" target="_blank" aria-label="Open Scopus in new tab">Scopus (137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29692413/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41576-018-0007-0&amp;pmid=29692413" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, we proceed to explore the sequence basis of the detected human-specific chromatin changes. The results suggesting that the HSL anchors are less conserved than the ConsL anchors when compared with chimpanzee and macaque (<a id="crosref1195" href="#figs6">Figure&nbsp;S6</a>A), and the between-species sequence divergence may contribute to HSL formation. We also checked whether HARs (human accelerated regions) (<span class="dropBlock reference-citations"><a id="body-ref-sref73-2" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-2" aria-controls="sref73-2">Pollard et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref73-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Pollard, K.S. ∙ Salama, S.R. ∙ Lambert, N. ...</div><div><strong>An RNA gene expressed during cortical development evolved rapidly in humans</strong></div><div><em>Nature.</em> 2006; <strong>443</strong>:167-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33748675254" target="_blank" aria-label="Open Scopus in new tab">Scopus (638)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16915236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05113&amp;pmid=16915236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Doan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Doan, R.N. ∙ Bae, B.I. ∙ Cubelos, B. ..., Homozygosity Mapping Consortium for Autism</div><div><strong>Mutations in Human Accelerated Regions Disrupt Cognition and Social Behavior</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:341-354.e12</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2016.08.071&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2016.08.071&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84990861958" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27667684/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.071&amp;pmid=27667684" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) are enriched in HSLs. The results indicate that the HSL anchors are significantly enriched for HARs (compared to random regions, permutation test, p&nbsp;= 1.07&nbsp;× 10<sup>−38</sup>, and compared to ConsL anchors, Fisher’s exact test, p&nbsp;= 0.04) (<a id="crosref1210" href="#fig7">Figure&nbsp;7</a>A). In particular, we identified 45 HARs located within the HSL anchors including enhancers or promoters (<a id="crosref1215" href="#fig7">Figure&nbsp;7</a>B). In some cases, both anchors of a HSL contain HARs, and other cases show one HAR associated with multiple HSLs (<a id="crosref1220" href="#fig7">Figure&nbsp;7</a>B). Interestingly, we found one HAR (ANC584) located within an HSL anchor on human chromosome 12 (<a id="crosref1225" href="#fig7">Figure&nbsp;7</a>C). Both ChIP-seq (H3K4me1) and ATAC-seq data suggest ANC584 overlaps with an enhancer and show higher activity in human compared to macaque. The Hi-C data indicate that ANC584 may regulate the SP-preferentially expressed gene-<i>SOX5</i> (<a id="crosref1230" href="#fig7">Figure&nbsp;7</a>C), providing an example of how HARs may cause human-specific regulatory changes through spatial chromatin interaction in the SP laminae of the developing brain.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/d8dbc7e3-01f2-4d0a-ba4f-35a6be9958f3/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.01.001/asset/ae1b8300-df49-47ef-98d1-39f72b8e3fbf/main.assets/gr7.jpg" height="703" width="739" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/d8dbc7e3-01f2-4d0a-ba4f-35a6be9958f3/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">The genetic basis of human-specific chromatin structural changes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0310" role="paragraph">(A) Percentage of loop anchors containing at least one HAR for ConsL, HSL, and all CP/GZ loops. Histogram showing the density of percentage of random loop anchors containing at least one HAR. p values are calculated by permutation test.</div><div id="fspara0315" role="paragraph">(B) Selected chromatin interaction networks defined by the GZ Hi-C data of human. Loop anchors containing HARs are indicated by red, and the others are indicated by gray. P, promoter; E, enhancer; O, other.</div><div id="fspara0320" role="paragraph">(C) Enrichment matrix at 10 kb resolution around the HSL regulating <i>SOX5</i> gene. The anchors are highlighted in yellow.</div><div id="fspara0325" role="paragraph">(D) Bar plots showing the numbers of the human-specific TAD boundaries and HSL anchors overlapped with the human-specific SVs.</div><div id="fspara0330" role="paragraph">(E) Schematic representation of the 316&nbsp;bp human-specific insertion located in the <i>MAP4K3</i> gene.</div><div id="fspara0335" role="paragraph">(F) Left: contact matrix at 25 kb resolution in human around <i>TMEM178A</i>. The ConS boundaries are indicated by yellow bars, and the human-specific-lost boundary is indicated by red bars. Right: contact matrix of the corresponding regions in macaque and mouse. IS and DI represent the profiles of insulation score and directionality index, respectively.</div><div id="fspara0340" role="paragraph">(G) Boxplots showing the expression levels (FPKM) of <i>MAP4K3</i> and <i>TMEM178A</i> in human and macaque brain zones.</div><div id="fspara0345" role="paragraph">See also <a id="crosref1900" href="#figs6">Figure&nbsp;S6</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/fb0e10c8-2ae3-469e-ad66-2be60b5fa17b/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.01.001/asset/fb32724b-e950-4930-a719-ed641fc619c1/main.assets/figs6.jpg" height="633" width="678" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/fb0e10c8-2ae3-469e-ad66-2be60b5fa17b/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Human-specific chromatin structures are associated with sequence variations, related to <a id="crosref1935" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0405" role="paragraph">(A) The cumulative distribution function plot of Levenshtein distances among human, chimpanzee and macaque for the open chromatin regions (defined by ATAC-seq) of the ConsL and HSL anchors. P value was calculated by Kolmogorov-Smirnov test. (B) Volcano plots of genes within TADs around the human-gained or human-lost boundaries with SVs using the CP RNA-seq profile. Genes upregulated in human are represented by red, and genes downregulated in human are represented by blue. (C) GO enrichment analysis of genes within TADs around the human-gained or human-lost boundaries with SVs. (D) Insulation scores for 3 individuals per brain zone at 40 kb around the human-specific boundary. Boundaries detected by the insulation score method are indicated by dotted lines. (E) The yellow boxes indicate the conserved boundaries and the red box indicates the human-specific-lost boundary in the human fetal brain. Blue boxes indicate boundaries for 6 cell lines/tissues around the human-specific boundary. SX: spleen; SB: small bowel; H1: embryonic stem cell; NPC: neural progenitor cell; CO: prefrontal cortex; HC: hippocampus; Boundaries are from <span class="dropBlock reference-citations"><a id="body-ref-sref81-2" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-2" aria-controls="sref81-2">Schmitt et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref81-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Schmitt, A.D. ∙ Hu, M. ∙ Jung, I. ...</div><div><strong>A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome</strong></div><div><em>Cell Rep.</em> 2016; <strong>17</strong>:2042-2059</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85003815206" target="_blank" aria-label="Open Scopus in new tab">Scopus (272)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27851967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2016.10.061&amp;pmid=27851967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. (F) Hi-C maps along with the Cons/HS boundaries and the insulation scores of the orthologous regions around the human-specific lost boundary in chimpanzee and bonobo. Hi-C data from <span class="dropBlock reference-citations"><a id="body-ref-sref104-1" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-1" aria-controls="sref104-1">Zhang et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref104-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Zhang, Y. ∙ Li, T. ∙ Preissl, S. ...</div><div><strong>Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:1380-1388</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0479-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85071088740" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31427791/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0479-7&amp;pmid=31427791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. (G) Hi-C maps around the human-specific loop in human, macaque and mouse. The human-specific loop is indicated by black box.</div></div></div></div></figcaption></figure></div><div id="p0190" role="paragraph">Next, we investigated the role of human-specific structural variants (SVs) in the formation of the human specific chromatin structure. We obtained the human-specific SV profile (human versus great apes) from the published data (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Dennis et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Dennis, M.Y. ∙ Harshman, L. ∙ Nelson, B.J. ...</div><div><strong>The evolution and population diversity of human-specific segmental duplications</strong></div><div><em>Nat. Ecol. Evol.</em> 2017; <strong>1</strong>:69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-016-0069" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85029929169" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28580430/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-016-0069&amp;pmid=28580430" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Kronenberg et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kronenberg, Z.N. ∙ Fiddes, I.T. ∙ Gordon, D. ...</div><div><strong>High-resolution comparative analysis of great ape genomes</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar6343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar6343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85048568612" target="_blank" aria-label="Open Scopus in new tab">Scopus (103)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29880660/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar6343&amp;pmid=29880660" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">McLean et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>McLean, C.Y. ∙ Reno, P.L. ∙ Pollen, A.A. ...</div><div><strong>Human-specific loss of regulatory DNA and the evolution of human-specific traits</strong></div><div><em>Nature.</em> 2011; <strong>471</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09774" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-79952386187" target="_blank" aria-label="Open Scopus in new tab">Scopus (287)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21390129/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09774&amp;pmid=21390129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We checked the relationship between the human-specific SVs and HSL. We found 165 human-specific SVs overlapped with 161 HSL anchors (<a id="crosref1250" href="#fig7">Figure&nbsp;7</a>D). We investigated if the human-specific SVs cause changes at the TAD level. We found 60 human-specific SVs located in 48 hgTAD boundaries and 74 human-specific SVs located in 57 hlTAD boundaries (<a id="crosref1255" href="#fig7">Figure&nbsp;7</a>D). There are no significant differences of gene expression around the SV-overlapped hgTAD and hlTAD boundaries compared with the background, and no neural-related functions are enriched (<a id="crosref1260" href="#figs6">Figures S6</a>B and S6C). However, the ratio of upregulated gene in human compared with macaque is higher in the SV-overlapped hlTAD boundaries than the SV-overlapped hgTAD boundaries (<a id="crosref1265" href="#figs6">Figure&nbsp;S6</a>B). For example, a human-specific insertion (316&nbsp;bp) is overlapped with an hlTAD boundary near the <i>MAP4K3</i> gene (<a id="crosref1270" href="#fig7">Figures 7</a>E and 7F). Interestingly, this hlTAD boundary is tissue-specific. It is absent in human fetal brain (<span class="dropBlock reference-citations"><a id="body-ref-sref100-6" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-6" aria-controls="sref100-6">Won et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref100-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Won, H. ∙ de la Torre-Ubieta, L. ∙ Stein, J.L. ...</div><div><strong>Chromosome conformation elucidates regulatory relationships in developing human brain</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:523-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84993993658" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19847&amp;pmid=27760116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), adult hippocampus and prefrontal cortex, and neural progenitor cells (<span class="dropBlock reference-citations"><a id="body-ref-sref81-3" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-3" aria-controls="sref81-3">Schmitt et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref81-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Schmitt, A.D. ∙ Hu, M. ∙ Jung, I. ...</div><div><strong>A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome</strong></div><div><em>Cell Rep.</em> 2016; <strong>17</strong>:2042-2059</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85003815206" target="_blank" aria-label="Open Scopus in new tab">Scopus (272)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27851967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2016.10.061&amp;pmid=27851967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1285" href="#figs6">Figures S6</a>D and S6E). Nevertheless, it is present in human small bowel and spleen, and in chimpanzee and bonobo&nbsp;(<span class="dropBlock reference-citations"><a id="body-ref-sref104-2" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-2" aria-controls="sref104-2">Zhang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref104-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Zhang, Y. ∙ Li, T. ∙ Preissl, S. ...</div><div><strong>Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:1380-1388</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0479-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85071088740" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31427791/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0479-7&amp;pmid=31427791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1295" href="#figs6">Figures S6</a>E and S6F). More importantly, associating with this human-specific loss of TAD boundary, the <i>TMEM178A</i> gene (also a SP marker gene) gains&nbsp;an HSL that are absent in macaque and mouse (<a id="crosref1300" href="#fig6">Figures 6</a>F and <a id="crosref1305" href="#figs6">S6</a>G), which may explain the observed higher expression of <i>TMEM178A</i> in the human fetal brain compared to macaque through “enhancer hijacking” (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Northcott et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Northcott, P.A. ∙ Lee, C. ∙ Zichner, T. ...</div><div><strong>Enhancer hijacking activates GFI1 family oncogenes in medulloblastoma</strong></div><div><em>Nature.</em> 2014; <strong>511</strong>:428-434</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13379" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84904816744" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25043047/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13379&amp;pmid=25043047" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1315" href="#fig7">Figure&nbsp;7</a>G).</div><div id="p0195" role="paragraph">The presented data indicate the close involvement of human-specific genetic changes (HARs and SVs) in forming chromatin structural changes during human brain development, and we discovered cases showing SP-related changes in humans, suggesting SP is an important developmental lamina that has not been studied adequately.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0200" role="paragraph">We constructed an ultra-deep 3D genome map of macaque fetal brain and carried out comparative evolutionary analysis among human, rhesus macaque, and mouse at 3D genome level. We uncovered a prominent feature of HSLs (i.e., the enrichment of enhancer-enhancer interactions). Together with the enhancer-promoter interactions, they form regulatory communities. A former study found loops that appear during cell differentiation were more enriched in enhancer-enhancer interactions than static loops (<span class="dropBlock reference-citations"><a id="body-ref-sref71-2" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-2" aria-controls="sref71-2">Phanstiel et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref71-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Phanstiel, D.H. ∙ Van Bortle, K. ∙ Spacek, D. ...</div><div><strong>Static and Dynamic DNA Loops form AP-1-Bound Activation Hubs during Macrophage Development</strong></div><div><em>Mol. Cell.</em> 2017; <strong>67</strong>:1037-1048.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2017.08.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2017.08.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85030242070" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28890333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2017.08.006&amp;pmid=28890333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our data suggest that, not&nbsp;only cell-type-specific loops but also species-specific loops&nbsp;may directly mediate enhancer-enhancer interactions. Compared with promoters, enhancers are spatiotemporal specific and may change constantly during development (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Nord et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Nord, A.S. ∙ Blow, M.J. ∙ Attanasio, C. ...</div><div><strong>Rapid and pervasive changes in genome-wide enhancer usage during mammalian development</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1521-1531</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.11.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.11.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84894598129" target="_blank" aria-label="Open Scopus in new tab">Scopus (202)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24360275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.033&amp;pmid=24360275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The precise control of gene expression may require more sophisticated enhancer-enhancer interactions in a tissue or cell-type-specific manner during human brain development. Additionally, genes regulated by multiple enhancers may have an enlarged expression spectrum because turning on and off one enhancer may have limited effect on gene expression, while the expression change can be quite large when two or more enhancers are involved (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Kieffer-Kwon et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Kieffer-Kwon, K.R. ∙ Tang, Z. ∙ Mathe, E. ...</div><div><strong>Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1507-1520</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.11.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.11.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84894589713" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24360274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.039&amp;pmid=24360274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hence, this multi-enhancer regulation is likely an acquired mechanism for the fine-tuning of human brain development.</div><div id="p0205" role="paragraph">SP, the largest transient zone expanded in the human fetal brain, is approximately four times thicker than the cortical plate during mid-gestation (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Kostovic and Molliver, 1974</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Kostovic, I. ∙ Molliver, M.E.</div><div><strong>New interpretation of laminar development of cerebral cortex synaptogenesis in different layers of neopallium in human fetus</strong></div><div><em>Anat. Rec.</em> 1974; <strong>178</strong>:395</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.KostovicM.E.MolliverNew+interpretation+of+laminar+development+of+cerebral+cortex+synaptogenesis+in+different+layers+of+neopallium+in+human+fetusAnat.+Rec.1781974395" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Kostovic and Rakic, 1990</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kostovic, I. ∙ Rakic, P.</div><div><strong>Developmental history of the transient subplate zone in the visual and somatosensory cortex of the macaque monkey and human brain</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 1990; <strong>297</strong>:441-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.902970309" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0025294391" target="_blank" aria-label="Open Scopus in new tab">Scopus (675)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2398142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.902970309&amp;pmid=2398142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SP has an important role in establishment of thalamocortical connection and contributes to cortical lamination through migration to cortical deep layer (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Kanold et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Kanold, P.O. ∙ Kara, P. ∙ Reid, R.C. ...</div><div><strong>Role of subplate neurons in functional maturation of visual cortical columns</strong></div><div><em>Science.</em> 2003; <strong>301</strong>:521-525</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1084152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0041843767" target="_blank" aria-label="Open Scopus in new tab">Scopus (213)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12881571/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1084152&amp;pmid=12881571" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Ozair et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Ozair, M.Z. ∙ Kirst, C. ∙ van den Berg, B.L. ...</div><div><strong>hPSC Modeling Reveals that Fate Selection of Cortical Deep Projection Neurons Occurs in the Subplate</strong></div><div><em>Cell Stem Cell.</em> 2018; <strong>23</strong>:60-73.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.stem.2018.05.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.stem.2018.05.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85048567581" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29937203/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2018.05.024&amp;pmid=29937203" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A recent study reported that human accelerated conserved noncoding sequences regulate genes enriched in SP, which might be involved in human specific neurodevelopment (<span class="dropBlock reference-citations"><a id="body-ref-sref62-5" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-5" aria-controls="sref62-5">Miller et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref62-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We show that HSLs regulate genes mainly enriched in the SP lamina, implying that the SP neurons are important in neural circuit formation and plasticity during human brain development.</div><div id="p0210" role="paragraph">We investigated the role of HARs and human-specific SVs in forming human-specific chromatin structures. However, different from the former study showing that 42% HARs interact with the flanking gene promoters (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Doan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Doan, R.N. ∙ Bae, B.I. ∙ Cubelos, B. ..., Homozygosity Mapping Consortium for Autism</div><div><strong>Mutations in Human Accelerated Regions Disrupt Cognition and Social Behavior</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:341-354.e12</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2016.08.071&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2016.08.071&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84990861958" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27667684/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.071&amp;pmid=27667684" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we observe that HARs may occur within the enhancer-enhancer mediated HSL anchors, suggesting that HARs can work in different ways to regulate gene expression in the developing human brain. On the other hand, although the chromatin landscape constrains SVs within humans and across primate evolution (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Fudenberg and Pollard, 2019</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Fudenberg, G. ∙ Pollard, K.S.</div><div><strong>Chromatin features constrain structural variation across evolutionary timescales</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:2175-2180</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1808631116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85061144757" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30659153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1808631116&amp;pmid=30659153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we identified 134 human-specific SVs located in the human-specific TAD boundaries. Importantly, we found that the SVs overlapped with the hlTAD boundaries may contribute to the detected regulatory changes during SP development. To fully understand the contribution of the human-specific genetic changes (such as HARs and SVs), more omics and functional experiments are needed.</div><div id="p0215" role="paragraph">We show that four TE families are enriched in the hgTAD boundaries, especially the two young TE subfamilies (AluY and L1PA), which may contribute to hgTAD formation and influence gene expression during human brain development. Currently, how TEs regulate TAD boundary formation is still in question. Two possible mechanisms have been proposed: (1) the transcriptional activity of TEs can decide TAD boundary formation (<span class="dropBlock reference-citations"><a id="body-ref-sref104-3" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-3" aria-controls="sref104-3">Zhang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref104-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Zhang, Y. ∙ Li, T. ∙ Preissl, S. ...</div><div><strong>Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:1380-1388</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0479-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85071088740" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31427791/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0479-7&amp;pmid=31427791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); and (2) other proteins can compete with CTCF in binding TEs to form TAD boundaries (<span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Kaaij et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Kaaij, L.J.T. ∙ Mohn, F. ∙ van der Weide, R.H. ...</div><div><strong>The ChAHP Complex Counteracts Chromatin Looping at CTCF Sites that Emerged from SINE Expansions in Mouse</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1437-1451.e14</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2019.08.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2019.08.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85071315791" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31491387/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.08.007&amp;pmid=31491387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Kaaij et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Kaaij, L.J.T. ∙ Mohn, F. ∙ van der Weide, R.H. ...</div><div><strong>The ChAHP Complex Counteracts Chromatin Looping at CTCF Sites that Emerged from SINE Expansions in Mouse</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1437-1451.e14</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2019.08.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2019.08.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85071315791" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31491387/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.08.007&amp;pmid=31491387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> reported the ChAHP complex (CHD4, ADNP, HP1) can bind to SINE B2 transposable elements and compete with CTCF in forming TAD boundaries in mouse. We therefore speculate that those TEs enriched in the hgTAD boundaries may provide binding sites for neural-development-related TFs and promote formation of the hgTAD boundaries.</div><div id="p0220" role="paragraph">It should be noted that we used liftover to get the corresponding locations of TAD boundaries and loop anchors when performing cross-species analyses. Therefore, the non-syntenic regions were mostly excluded in this process, which may give an inflated picture of the overall conservation of 3D genome architecture. We find most human TADs and loops are located in the syntenic regions (<a id="crosref1385" href="#figs7">Figures S7</a>A and S7B), and the conservation levels of TADs and loops in the syntenic regions are slightly overestimated due to the low conservation of the non-syntenic regions (<a id="crosref1390" href="#figs7">Figures S7</a>C and S7D). The non-syntenic TADs and loops are associated with different types of genome structural changes during evolution such as translocations, inversions, and large gaps (<a id="crosref1395" href="#figs7">Figure&nbsp;S7</a>E). We presented examples to show how these non-syntenic regions affect the evolution of human chromatin structures (<a id="crosref1400" href="#figs7">Figures S7</a>F and S7G). More accurate genome landscape (such as a high-quality macaque reference genome) (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">He et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>He, Y. ∙ Luo, X. ∙ Zhou, B. ...</div><div><strong>Long-read assembly of the Chinese rhesus macaque genome and identification of ape-specific structural variants</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4233</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12174-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85072289655" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31530812/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12174-w&amp;pmid=31530812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) would be needed to better understand the relationship between non-syntenic regions and 3D genome architectures.</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/cd4ee25d-0dc9-4f76-b6c8-23783d6d75f4/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.01.001/asset/e622b673-55e5-4e2d-ab9d-172e013f6a3e/main.assets/figs7.jpg" height="795" width="762" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/cd4ee25d-0dc9-4f76-b6c8-23783d6d75f4/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Dynamics of 3D genome architectures within syntenic and non-syntenic regions, related to <a id="crosref1950" href="#sec-4">STAR methods</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0415" role="paragraph">(A) The number of human TADs and loops that fall in the syntenic regions and the non-syntenic regions. (B) The syntenic ratio of human TADs and loops compared with 1000 random permutations. P values are calculated by single sample t test. <sup>∗∗∗</sup>: p&nbsp;&lt; <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="10 Superscript negative 20" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="10 Superscript negative 20"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-children="2" data-semantic-content="1" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.666em;">2</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>. (C) The ratio of the conserved human TADs and the human-gained TADs in three different conditions: the overall results, limited to syntenic regions, and limited to non-syntenic regions. Overall versus Syntenic: Fisher’s exact test, p&nbsp;= 0.0018. (D) The ratio of the conserved human loops and the human-gained loops in three different conditions: the overall results, limited to syntenic regions, and limited to non-syntenic regions. Overall versus Syntenic: Fisher’s exact test, p&nbsp;= 0.064. (E) The proportions of different types of non-syntenic human TADs and loops according to the formation mechanisms of the non-syntenic regions. (F) A case of several non-syntenic human TADs falling in a large gap between human and macaque. This region separates two human-macaque syntenic blocks in the human genome and it is absent in the macaque genome, and it contains several human-gained TADs. (G) A case of non-syntenic human TAD on chromosome 8 resulting from the interchromosomal recombination between mouse chromosome 3 and chromosome 4.</div></div></div></div></figcaption></figure></div><div id="p0225" role="paragraph">In summary, we generated a high-resolution 3D genome profile of NHP during corticogenesis, which allowed us to reveal the human-specific 3D genome changes through cross-species analyses. Comparative 3D genome analyses are powerful approaches in deciphering the regulatory mechanisms of brain development and evolution.</div></section><section id="sec-4"><h2>STAR★methods</h2><section id="sec-4-1"><h3>KEY RESOURCES TABLE</h3><div id="p0235" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Rabbit anti-CTCF</td><td>Abclonal</td><td>Cat# A1133; RRID:<a id="interref295" href="https://antibodyregistry.org/AB_2753146" target="_blank">AB_2753146</a></td></tr><tr class="rowsep"><td>Chicken anti-GFP</td><td>Abcam</td><td>Cat# ab13970; RRID:<a id="interref290" href="https://antibodyregistry.org/AB_300798" target="_blank">AB_300798</a></td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td>pL-CRISPR.EFS.GFP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Heckl et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Heckl, D. ∙ Kowalczyk, M.S. ∙ Yudovich, D. ...</div><div><strong>Generation of mouse models of myeloid malignancy with combinatorial genetic lesions using CRISPR-Cas9 genome editing</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:941-946</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2951" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84911422892" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24952903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2951&amp;pmid=24952903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene #57818; RRID:<a id="intref0010" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_002037" target="_blank">SCR_002037</a></td></tr><tr><td>pL-CRISPR.EFS.tRFP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Heckl et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Heckl, D. ∙ Kowalczyk, M.S. ∙ Yudovich, D. ...</div><div><strong>Generation of mouse models of myeloid malignancy with combinatorial genetic lesions using CRISPR-Cas9 genome editing</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:941-946</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2951" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84911422892" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24952903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2951&amp;pmid=24952903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene #57819; RRID:<a id="intref0015" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_002037" target="_blank">SCR_002037</a></td></tr><tr><td>pL-CRISPR.EFS.GFP_EPHA7 enhancer-sgRNA1</td><td>This paper</td><td>N/A</td></tr><tr><td>pL-CRISPR.EFS.GFP_EPHA7 enhancer-sgRNA2</td><td>This paper</td><td>N/A</td></tr><tr><td>pL-CRISPR.EFS.tRFP_EPHA7 enhancer-sgRNA3</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td>pL-CRISPR.EFS. tRFP _EPHA7 enhancer-sgRNA4</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Triton X-100</td><td>Thermo Fisher</td><td>Cat# 85111</td></tr><tr><td>NP-40</td><td>Thermo Fisher</td><td>Cat# 85124</td></tr><tr><td>Dulbecco’s Modification of Eagle’s Medium (DMEM)</td><td>GIBCO</td><td>Cat# 12430054</td></tr><tr><td>Dulbecco’s Modified Eagle Medium/Nutrient Mixture F-12 (DMEM/F12)</td><td>GIBCO</td><td>Cat# 11320033</td></tr><tr><td>Poly-L-ornithine</td><td>Sigma</td><td>Cat# P4957</td></tr><tr><td>Laminin</td><td>GIBCO</td><td>Cat# 23017015</td></tr><tr><td>Neurobasal medium</td><td>GIBCO</td><td>Cat# 21103049</td></tr><tr><td>B27</td><td>GIBCO</td><td>Cat# 17504044</td></tr><tr><td>N2</td><td>GIBCO</td><td>Cat# A1370701</td></tr><tr><td>NEAA</td><td>Sigma</td><td>Cat# M7145</td></tr><tr><td>BDNF</td><td>GIBCO</td><td>Cat# PHC7074</td></tr><tr><td>GDNF</td><td>GIBCO</td><td>Cat# PHC7041</td></tr><tr><td>bFGF</td><td>GIBCO</td><td>Cat# 13256029</td></tr><tr><td>TrypLE Express Enzyme</td><td>GIBCO</td><td>Cat# 12604021</td></tr><tr><td>37% formaldehyde</td><td>Sigma-Aldrich</td><td>Cat# F1635</td></tr><tr><td>cOmplete protease inhibitors</td><td>Sigma-Aldrich</td><td>Cat# 11697498001</td></tr><tr><td>DpnII</td><td>NEB</td><td>Cat# R0543</td></tr><tr><td>biotin-14-dATP</td><td>Life Technologies</td><td>Cat# 19524-016</td></tr><tr><td>DNA polymerase I Kenow</td><td>NEB</td><td>Cat# M0210</td></tr><tr><td>T4 DNA ligase</td><td>NEB</td><td>Cat# M0202</td></tr><tr><td>T4 DNA polymerase</td><td>NEB</td><td>Cat# M0203</td></tr><tr><td>Streptavidin T1 beads</td><td>Life Technologies</td><td>Cat# 65602</td></tr><tr><td>DNase I recombinant, RNase-free</td><td>Roche</td><td>Cat# 4716728001</td></tr><tr><td>Protein A dynabeads</td><td>Thermo Fisher</td><td>Cat# 10002D</td></tr><tr><td>AMpure XPmagnetic beads</td><td>Beckman Coulter</td><td>Cat# A63881</td></tr><tr><td>2&nbsp;× SSC buffer</td><td>Thermo Fisher</td><td>Cat# AM9763</td></tr><tr><td>pepsin</td><td>Macklin</td><td>Cat# P6322-100 g</td></tr><tr><td>Formamide</td><td>Sigma</td><td>Cat# S6639-1L</td></tr><tr><td>ProLong Diamond Antifade Mountant with DAPI</td><td>Thermo Fisher</td><td>Cat# P36966</td></tr><tr><td>Lenti-X Concentrator</td><td>Takara</td><td>Cat# 631231</td></tr><tr><td>Dual Luferase Reporter System</td><td>Promega</td><td>Cat# E1960</td></tr><tr class="rowsep"><td>Lipofectamine 3000</td><td>Thermo Fisher</td><td>Cat# L3000015</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>miRNeasy Mini Kit</td><td>QIAGEN</td><td>Cat# 217004</td></tr><tr><td>MinElute Gel Extraction Kit</td><td>QIAGEN</td><td>Cat# 28606</td></tr><tr><td>VAHTS Total RNA-seq Library Prep Kit for Illumina</td><td>Vazyme</td><td>Cat# NR603-01</td></tr><tr><td>NEBNext Ultra II DNA Library Prep Kit for Illumina</td><td>NEB</td><td>Cat# E7645L</td></tr><tr class="rowsep"><td>Multiplex Oligos for Illumina</td><td>NEB</td><td>Cat# E7600</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Raw and analyzed data: RNA-seq</td><td>This paper</td><td>GSA: <a id="intref0020" href="gsa:/CRA001934" target="_blank">CRA001934</a>; GEO: <a id="interref305" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE163177" target="_blank">GSE163177</a></td></tr><tr><td>Raw and analyzed data: Hi-C</td><td>This paper</td><td>GSA: <a id="intref0025" href="gsa:/CRA001934" target="_blank">CRA001934</a>; GEO: <a id="interref310" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE163177" target="_blank">GSE163177</a></td></tr><tr><td>Raw and analyzed data: CTCF ChIP-seq</td><td>This paper</td><td>GSA: <a id="intref0030" href="gsa:/CRA001934" target="_blank">CRA001934</a>; GEO: <a id="interref315" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE163177" target="_blank">GSE163177</a></td></tr><tr><td>Raw and analyzed data: DNase-seq</td><td>This paper</td><td>GSA: <a id="intref0035" href="gsa:/CRA001934" target="_blank">CRA001934</a>; GEO: <a id="interref320" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE163177" target="_blank">GSE163177</a></td></tr><tr><td>Developing human cortex Hi-C data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref100-7" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-7" aria-controls="sref100-7">Won et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref100-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Won, H. ∙ de la Torre-Ubieta, L. ∙ Stein, J.L. ...</div><div><strong>Chromosome conformation elucidates regulatory relationships in developing human brain</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:523-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84993993658" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19847&amp;pmid=27760116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>https://<a id="intref0040" href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77565" target="_blank">www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77565</a></td></tr><tr><td>Developing human cortex ATAC-seq data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">de la Torre-Ubieta et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>de la Torre-Ubieta, L. ∙ Stein, J.L. ∙ Won, H. ...</div><div><strong>The Dynamic Landscape of Open Chromatin during Human Cortical Neurogenesis</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:289-304.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.12.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.12.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85039917750" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29307494/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.014&amp;pmid=29307494" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO and dbGaP: GSE95023 and phs001438</td></tr><tr><td>Mouse neural differentiation Hi-C data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref5-3" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-3" aria-controls="sref5-3">Bonev et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref5-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bonev, B. ∙ Mendelson Cohen, N. ∙ Szabo, Q. ...</div><div><strong>Multiscale 3D Genome Rewiring during Mouse Neural Development</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:557-572.e24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85031779520" target="_blank" aria-label="Open Scopus in new tab">Scopus (361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29053968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.043&amp;pmid=29053968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107</a></td></tr><tr><td>Mouse NPC and CN ChIP-seq data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref5-4" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-4" aria-controls="sref5-4">Bonev et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref5-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bonev, B. ∙ Mendelson Cohen, N. ∙ Szabo, Q. ...</div><div><strong>Multiscale 3D Genome Rewiring during Mouse Neural Development</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:557-572.e24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85031779520" target="_blank" aria-label="Open Scopus in new tab">Scopus (361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29053968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.043&amp;pmid=29053968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107</a></td></tr><tr><td>H1 Neuronal Progenitor cells CTCF ChIP-seq data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref55-1" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-1" aria-controls="sref55-1">Lieberman-Aiden et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref55-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Lieberman-Aiden, E. ∙ van Berkum, N.L. ∙ Williams, L. ...</div><div><strong>Comprehensive mapping of long-range interactions reveals folding principles of the human genome</strong></div><div><em>Science.</em> 2009; <strong>326</strong>:289-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1181369" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-70349873824" target="_blank" aria-label="Open Scopus in new tab">Scopus (3659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19815776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1181369&amp;pmid=19815776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0055" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1267209" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1267209</a></td></tr><tr><td>iPSC-induced excitatory neurons and iPSC-induced lower motor neurons CTCF CUT&amp;RUN data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref87-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-2" aria-controls="sref87-2">Song et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref87-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Song, M. ∙ Yang, X. ∙ Ren, X. ...</div><div><strong>Mapping cis-regulatory chromatin contacts in neural cells links neuropsychiatric disorder risk variants to target genes</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:1252-1262</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0472-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85070866778" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31367015/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0472-1&amp;pmid=31367015" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126211" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126211</a></td></tr><tr><td>Human fetal brain single cell RNA-seq data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Nowakowski et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (220)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0065" href="https://ars.els-cdn.com/content/image/1-s2.0-S2211124719305091-mmc6.xlsx" target="_blank">https://ars.els-cdn.com/content/image/1-s2.0-S2211124719305091-mmc6.xlsx</a></td></tr><tr><td>Human fetal brain single cell RNA-seq data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Polioudakis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Polioudakis, D. ∙ de la Torre-Ubieta, L. ∙ Langerman, J. ...</div><div><strong>A Single-Cell Transcriptomic Atlas of Human Neocortical Development during Mid-gestation</strong></div><div><em>Neuron.</em> 2019; <strong>103</strong>:785-801.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85070182242" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31303374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2019.06.011&amp;pmid=31303374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>dbGAP: phs001836; <a id="intref0070" href="http://solo.bmap.ucla.edu/shiny/webapp/" target="_blank">http://solo.bmap.ucla.edu/shiny/webapp/</a></td></tr><tr><td>Human fetal cortex laser-capture microdissection</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-6" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-6" aria-controls="sref62-6">Miller et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref62-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="http://brainspan.org/" target="_blank">http://brainspan.org/</a> RRID:<a id="intref0080" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_008083" target="_blank">SCR_008083</a></td></tr><tr><td>Roadmap Epigenomics Human H3K4me1 data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Ernst and Kellis, 2015</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Ernst, J. ∙ Kellis, M.</div><div><strong>Large-scale imputation of epigenomic datasets for systematic annotation of diverse human tissues</strong></div><div><em>Nat. Biotechnol.</em> 2015; <strong>33</strong>:364-376</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84926632357" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25690853/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3157&amp;pmid=25690853" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Kundaje et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Kundaje, A. ∙ Meuleman, W. ∙ Ernst, J. ..., Roadmap Epigenomics Consortium</div><div><strong>Integrative analysis of 111 reference human epigenomes</strong></div><div><em>Nature.</em> 2015; <strong>518</strong>:317-330</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14248" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84923362619" target="_blank" aria-label="Open Scopus in new tab">Scopus (2646)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25693563/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14248&amp;pmid=25693563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0085" href="http://www.roadmapepigenomics.org/" target="_blank">http://www.roadmapepigenomics.org/</a></td></tr><tr><td>Rhesus macaque H3K4me2 and H3K27ac data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Reilly et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Reilly, S.K. ∙ Yin, J. ∙ Ayoub, A.E. ...</div><div><strong>Evolutionary genomics. Evolutionary changes in promoter and enhancer activity during human corticogenesis</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1155-1159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84924561748" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25745175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260943&amp;pmid=25745175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0090" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63648" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63648</a> RRID: not found</td></tr><tr><td>VISTA Enhancer browser</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref96-3" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-3" aria-controls="sref96-3">Visel et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref96-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Visel, A. ∙ Minovitsky, S. ∙ Dubchak, I. ...</div><div><strong>VISTA Enhancer Browser--a database of tissue-specific human enhancers</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:D88-D92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkl822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33846112470" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17130149/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkl822&amp;pmid=17130149" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0095" href="https://enhancer.lbl.gov/" target="_blank">https://enhancer.lbl.gov/</a></td></tr><tr class="rowsep"><td>Human-specific single nucleotide changes data</td><td>Ensemble</td><td><a id="intref0100" href="http://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/compara/" target="_blank">http://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/compara/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: cell lines</b></td></tr><tr><td>iPSC induced human neural progenitor cell</td><td>Hopstem biotechnology</td><td>Cat# HopCell-NPC-100F</td></tr><tr class="rowsep"><td>SK-N-SH cell</td><td>ATCC</td><td>Cat# HTB-11</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: organisms/strains</b></td></tr><tr class="rowsep"><td>Rhesus macaques</td><td>Kunming Primate Research Center, Chinese Academy of Sciences</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>EPHA7 enhancer PCR Primers; See <a id="crosref1490" href="#sec-4">STAR methods</a></td><td>This paper</td><td>N/A</td></tr><tr><td>EPHA7 qPCR primers; See <a id="crosref1495" href="#sec-4">STAR methods</a></td><td>This paper</td><td>N/A</td></tr><tr><td>Oligopaint libraries; See <a id="crosref1500" href="#mmc2">Table S2</a></td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td>3C assay validation primers; See <a id="crosref1505" href="#mmc2">Table S2</a></td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>pL-CRISPR.EFS.GFP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-3" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-3" aria-controls="sref28-3">Heckl et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref28-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Heckl, D. ∙ Kowalczyk, M.S. ∙ Yudovich, D. ...</div><div><strong>Generation of mouse models of myeloid malignancy with combinatorial genetic lesions using CRISPR-Cas9 genome editing</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:941-946</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2951" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84911422892" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24952903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2951&amp;pmid=24952903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene #57818; RRID:<a id="intref0105" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_002037" target="_blank">SCR_002037</a></td></tr><tr><td>pL-CRISPR.EFS.tRFP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-4" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-4" aria-controls="sref28-4">Heckl et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref28-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Heckl, D. ∙ Kowalczyk, M.S. ∙ Yudovich, D. ...</div><div><strong>Generation of mouse models of myeloid malignancy with combinatorial genetic lesions using CRISPR-Cas9 genome editing</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:941-946</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2951" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84911422892" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24952903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2951&amp;pmid=24952903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene #57819; RRID:<a id="intref0110" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_002037" target="_blank">SCR_002037</a></td></tr><tr><td>pL-CRISPR.EFS.GFP_EPHA7 enhancer-sgRNA1</td><td>This paper</td><td>N/A</td></tr><tr><td>pL-CRISPR.EFS.GFP_EPHA7 enhancer-sgRNA2</td><td>This paper</td><td>N/A</td></tr><tr><td>pL-CRISPR.EFS.tRFP_EPHA7 enhancer-sgRNA3</td><td>This paper</td><td>N/A</td></tr><tr><td>pL-CRISPR.EFS. tRFP _EPHA7 enhancer-sgRNA4</td><td>This paper</td><td>N/A</td></tr><tr><td>pGL3-human EPHA7 enhancer_promoter vector</td><td>This paper</td><td>N/A</td></tr><tr><td>pGL3-chimpanzee EPHA7 enhancer_promoter vector</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td>pGL3-macaque EPHA7 enhancer_promoter vector</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Bowtie2 (v2.3.4)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Langmead and Salzberg, 2012</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (17046)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0115" href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td></tr><tr><td>PicardTools (v2.20.2)</td><td>N/A</td><td><a id="intref0120" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a></td></tr><tr><td>Samtools (v1.9)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Li et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The Sequence Alignment/Map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (21853)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0125" href="https://github.com/samtools/samtools" target="_blank">https://github.com/samtools/samtools</a></td></tr><tr><td>DeepTools (v3.2.1)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Ramírez et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Ramírez, F. ∙ Dündar, F. ∙ Diehl, S. ...</div><div><strong>deepTools: a flexible platform for exploring deep-sequencing data</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:W187-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900988020" target="_blank" aria-label="Open Scopus in new tab">Scopus (707)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24799436/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku365&amp;pmid=24799436" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0130" href="https://usegalaxy.eu/" target="_blank">https://usegalaxy.eu/</a></td></tr><tr><td>MACS2 (v2.1.2)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref103-1" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-1" aria-controls="sref103-1">Zhang et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref103-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (6096)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0135" href="https://github.com/taoliu/MACS" target="_blank">https://github.com/taoliu/MACS</a></td></tr><tr><td>STAR (v2.6.16) (<span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (10402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td><td>N/A</td></tr><tr><td><span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (10402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0140" href="https://github.com/alexdobin/STAR" target="_blank">https://github.com/alexdobin/STAR</a></td><td>N/A</td></tr><tr><td>Genomic Alignments package (v1.18.1)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Lawrence et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Lawrence, M. ∙ Huber, W. ∙ Pagès, H. ...</div><div><strong>Software for computing and annotating genomic ranges</strong></div><div><em>PLoS Comput. Biol.</em> 2013; <strong>9</strong>:e1003118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84883368195" target="_blank" aria-label="Open Scopus in new tab">Scopus (1155)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23950696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003118&amp;pmid=23950696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0145" href="http://bioconductor.org/packages/release/bioc/html/GenomicRanges.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/GenomicRanges.html</a></td></tr><tr><td>XSAnno</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref105-1" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-1" aria-controls="sref105-1">Zhu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref105-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Zhu, Y. ∙ Li, M. ∙ Sousa, A.M.M. ...</div><div><strong>XSAnno: a framework for building ortholog models in cross-species transcriptome comparisons</strong></div><div><em>BMC Genomics.</em> 2014; <strong>15</strong>:343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-15-343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900840147" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24884593/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-15-343&amp;pmid=24884593" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0150" href="https://hbatlas.org/xsanno/" target="_blank">https://hbatlas.org/xsanno/</a></td></tr><tr><td>Rsubread packsge (v1.32.4)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Liao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>The R package Rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:e47</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkz114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85066058200" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30783653/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkz114&amp;pmid=30783653" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0155" href="https://bioconductor.org/packages/release/bioc/html/Rsubread.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/Rsubread.html</a></td></tr><tr><td>DESeq2 (v1.22.2)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (16017)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0160" href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/DESeq2.html</a></td></tr><tr><td>HiC-Pro (v2.9.0)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref82-1" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-1" aria-controls="sref82-1">Servant et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref82-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Servant, N. ∙ Varoquaux, N. ∙ Lajoie, B.R. ...</div><div><strong>HiC-Pro: an optimized and flexible pipeline for Hi-C data processing</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0831-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84952861819" target="_blank" aria-label="Open Scopus in new tab">Scopus (432)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26619908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0831-x&amp;pmid=26619908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0165" href="https://github.com/nservant/HiC-Pro" target="_blank">https://github.com/nservant/HiC-Pro</a></td></tr><tr><td>juicer (v1.11.04)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Durand et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Durand, N.C. ∙ Shamim, M.S. ∙ Machol, I. ...</div><div><strong>Juicer Provides a One-Click System for Analyzing Loop-Resolution Hi-C Experiments</strong></div><div><em>Cell Syst.</em> 2016; <strong>3</strong>:95-98</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27467249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2016.07.002&amp;pmid=27467249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0170" href="http://aidenlab.org/juicer/" target="_blank">http://aidenlab.org/juicer/</a></td></tr><tr><td>HiCCUPS (v1.11.04)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref77-4" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-4" aria-controls="sref77-4">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Rao, S.S. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0175" href="https://github.com/aidenlab/juicer/wiki/CPU-HiCCUPS" target="_blank">https://github.com/aidenlab/juicer/wiki/CPU-HiCCUPS</a></td></tr><tr><td>DAVID</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref13-1" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-1" aria-controls="sref13-1">Dennis et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref13-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Dennis, Jr., G. ∙ Sherman, B.T. ∙ Hosack, D.A. ...</div><div><strong>DAVID: Database for Annotation, Visualization, and Integrated Discovery</strong></div><div><em>Genome Biol.</em> 2003; <strong>4</strong>:3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2003-4-5-p3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12734009/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2003-4-5-p3&amp;pmid=12734009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0180" href="https://david.ncifcrf.gov/" target="_blank">https://david.ncifcrf.gov/</a></td></tr><tr><td>JASPAR (v2018)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref39-1" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-1" aria-controls="sref39-1">Khan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref39-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Khan, A. ∙ Fornes, O. ∙ Stigliani, A. ...</div><div><strong>JASPAR 2018: update of the open-access database of transcription factor binding profiles and its web framework</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D260-D266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1126" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85040936337" target="_blank" aria-label="Open Scopus in new tab">Scopus (568)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29140473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1126&amp;pmid=29140473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0185" href="http://jaspar.genereg.net/" target="_blank">http://jaspar.genereg.net/</a></td></tr><tr><td>HOMER (v4.10)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref29-1" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-1" aria-controls="sref29-1">Heinz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref29-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (4333)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0190" href="http://homer.ucsd.edu/homer/" target="_blank">http://homer.ucsd.edu/homer/</a></td></tr><tr><td>FIMO (v5.0.2)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Grant et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Grant, C.E. ∙ Bailey, T.L. ∙ Noble, W.S.</div><div><strong>FIMO: scanning for occurrences of a given motif</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:1017-1018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr064" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-79953300078" target="_blank" aria-label="Open Scopus in new tab">Scopus (1382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21330290/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr064&amp;pmid=21330290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="interref0300" href="http://meme-suite.org/tools/fimo" target="_blank">http://meme-suite.org/tools/fimo</a></td></tr><tr><td>WGCNA (v1.68)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Langfelder and Horvath, 2008</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Langfelder, P. ∙ Horvath, S.</div><div><strong>WGCNA: an R package for weighted correlation network analysis</strong></div><div><em>BMC Bioinformatics.</em> 2008; <strong>9</strong>:559</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-9-559" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-60549111634" target="_blank" aria-label="Open Scopus in new tab">Scopus (5546)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19114008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-9-559&amp;pmid=19114008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0200" href="https://www.rdocumentation.org/packages/WGCNA/versions/1.68" target="_blank">https://www.rdocumentation.org/packages/WGCNA/versions/1.68</a></td></tr><tr><td>R (v3.0.6)</td><td>N/A</td><td><a id="intref0205" href="https://www.r-project.org/" target="_blank">https://www.r-project.org/</a></td></tr><tr><td>CQN package (v1.28.1)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Hansen et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Hansen, K.D. ∙ Irizarry, R.A. ∙ Wu, Z.</div><div><strong>Removing technical variability in RNA-seq data using conditional quantile normalization</strong></div><div><em>Biostatistics.</em> 2012; <strong>13</strong>:204-216</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/biostatistics/kxr054" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84858068675" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22285995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbiostatistics%2Fkxr054&amp;pmid=22285995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0210" href="http://bioconductor.org/packages/release/bioc/html/cqn.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/cqn.html</a></td></tr><tr><td>inferCars</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Ma et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Ma, J. ∙ Zhang, L. ∙ Suh, B.B. ...</div><div><strong>Reconstructing contiguous regions of an ancestral genome</strong></div><div><em>Genome Res.</em> 2006; <strong>16</strong>:1557-1565</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.5383506" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33750386912" target="_blank" aria-label="Open Scopus in new tab">Scopus (210)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16983148/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.5383506&amp;pmid=16983148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0215" href="http://www.bx.psu.edu/miller_lab/car/" target="_blank">http://www.bx.psu.edu/miller_lab/car/</a></td></tr><tr><td>imageJ</td><td>N/A</td><td><a id="intref0220" href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2900001-5" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0240" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Bing Su (<a id="intref0225" href="mailto:sub@mail.kiz.ac.cn">sub@mail.kiz.ac.cn</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0245" role="paragraph">This study did not generate new unique reagents.</div></section><section id="sec-4-2-3"><h4>Data and code availability</h4><div id="p0250" role="paragraph">The sequencing data generated in this study are available at the Gene Expression Omnibus (GEO, <a id="intref0230" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE163177" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE163177</a>; accession number: <a id="intref0235" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE163177" target="_blank">GSE163177</a>), and the Genome Sequence Achieve (GSA, <a id="intref0240" href="https://bigd.big.ac.cn/gsa/" target="_blank">https://bigd.big.ac.cn/gsa/</a>; accession number: <a id="intref0245" href="gsa:/CRA001934" target="_blank">CRA001934</a>).</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Rhesus macaque fetal brain samples</h4><div id="p0255" role="paragraph">Fetal brain of rhesus macaque was obtained from Kunming Primate Research Center, Kunming Institute of Zoology. All animal procedures were conducted following the international standards, and were approved in advance by the Institutional Animal Care and Use Committee of Kunming Institute of Zoology, Chinese Academy of Sciences (Approval No: SMKX-2017021).</div><div id="p0260" role="paragraph">We collected three fetal brains of rhesus macaque (<i>Macaca mulatta</i>) at mid-gestation (E84), and the fronto-parietal cortex was dissected. The brains were kept in ice-cold DMEM/F12 (GIBCO, 11320033) during dissection. Under a dissection microscope, the coronal sections were divided into two major zones: CP - consisting primarily of post-mitotic neurons, including the marginal zone (MZ), cortical plate (CP) and subplate (SP) and GZ - consisting primarily of neural progenitors, containing intermediate zone (IZ), outer subventricular zone (OSVZ), inner subventricular zone (ISVZ) and ventricular zone (VZ). The dissected tissues were further cut into small pieces and digested in TrypLE Express Enzyme (GIBCO, 12604021). The tissues were vortexed and kept at 37°C for 10&nbsp;min. The cell suspension was pipetted further to disperse into single cells and centrifuged at 300&nbsp;g for 5&nbsp;mins to obtain cell pellets. The digestion medium was carefully removed, the cells were washed twice with PBS. Finally, the cell pellets were re-suspended in 1ml DMEM medium (GIBCO, 12430054) and kept on ice. The cells were divided into two parts: one was crosslinked by formaldehyde for Hi-C, DNase-seq and ChIP-seq, and the other was immediately frozen for RNA-seq. Cells were fixed by final concentration of 1% formaldehyde (Sigma-Aldrich, F1635), mixed for 10&nbsp;mins at room temperature. Fixation was quenched using 2.5&nbsp;M glycine at room temperature for 5&nbsp;mins and immediately centrifuged at 800&nbsp;g for 10&nbsp;mins. Supernatant was removed and frozen in liquid nitrogen and stored at −80°C until further use.</div></section><section id="sec-4-3-2"><h4>Neural cell lines</h4><div id="p0265" role="paragraph">Human neural progenitor cells (NPCs) induced from human iPSC were expanded by plating on Poly-L-ornithine (Sigma, P4957) and Laminin (GIBCO, 23017015) coated plates with proliferation media [Neurobasal media (GIBCO, 21103049), 1% N2 supplement (GIBCO, A1370701), 2% B27 (GIBCO, 17504044), 20&nbsp;ng ml<sup>−1</sup> basic fibroblast growth factor (GIBCO, 13256029) and 1% penicillin/streptomycin] following the published protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Gunhanlar et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Gunhanlar, N. ∙ Shpak, G. ∙ van der Kroeg, M. ...</div><div><strong>A simplified protocol for differentiation of electrophysiologically mature neuronal networks from human induced pluripotent stem cells</strong></div><div><em>Mol. Psychiatry.</em> 2018; <strong>23</strong>:1336-1344</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/mp.2017.56" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85017511992" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28416807/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmp.2017.56&amp;pmid=28416807" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For differentiation, NPCs were cultured in the differentiation media composed of Neurobasal, N2, B27, NEAA (Sigma, M7145), 1% Glutmax (Sigma, 3505061), 10&nbsp;ng/ml BDNF (GIBCO, PHC7074) and 10&nbsp;ng/ml GDNF (GIBCO, PHC7041), 1&nbsp;μM Dibutyryl CAMP (Sigma, D0627), 200&nbsp;μM ascorbic acid (Sigma, A5960) and 1% penicillin/ streptomycin. Half of the media was replaced by fresh medium every 3&nbsp;days. All cells were incubated at 37°C and 5% CO<sub>2</sub>.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Hi-C library preparation</h4><div id="p0270" role="paragraph">Hi-C libraries were constructed based on the published Hi-C 2.0 protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Belaghzal et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Belaghzal, H. ∙ Dekker, J. ∙ Gibcus, J.H.</div><div><strong>Hi-C 2.0: An optimized Hi-C procedure for high-resolution genome-wide mapping of chromosome conformation</strong></div><div><em>Methods.</em> 2017; <strong>123</strong>:56-65</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ymeth.2017.04.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85018788636" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28435001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymeth.2017.04.004&amp;pmid=28435001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Five million fixed cells were re-suspended in lysis buffer (final concentration of 10&nbsp;mMTris, pH 8.0, 10&nbsp;mM NaCl, 0.2% Igepal CA-630 and 1&nbsp;× cOmplete protease inhibitors&nbsp;(Sigma-Aldrich, 11697498001) and incubated on ice for 10&nbsp;mins. Removal of lysis buffer was done by centrifugation at 2500&nbsp;g for 5&nbsp;mins at 4°C, followed by removal of supernatant. The pellet was re-suspended in 342&nbsp;μL 1&nbsp;× NEBuffer 3.1, and incubated with 38&nbsp;μL 10% SDS at 65°C for 10&nbsp;mins. Next, 43&nbsp;μL of 10% Triton X-100 was added to the Hi-C-tube to quench the SDS at 37°C for 15min. We then add 12&nbsp;μL 10&nbsp;× NEBuffer 3.1 and 400U DpnII (NEB, R0543), and mix gently to digest the chromatin overnight at 37°C on a rocking platform. On the next day, the DpnII restriction enzyme was inactivated at 65°C for 25&nbsp;mins. Then, biotin-14-dATP (Life Technologies, 19524-016), dCTP, dGTP, dTTP and DNA polymerase I Kenow were added (NEB, M0210), and incubated at 23°C for 4 h. The digested chromatin was diluted and re-ligated by T4 DNA ligase (NEB, M0202), incubated at 16°C for 4 h, and shaken manually for three times. The chromatin products were de-cross-linked overnight by adding 30&nbsp;μL proteinase K, and incubated at 65°C overnight. The DNAs were extracted and dissolved in 50ul 10&nbsp;mMTris, pH 8.0. Then T4 DNA polymerase (NEB, M0203) was used to remove biotin from un-ligation ends for 4&nbsp;hr at 20°C, and the enzymes were inactivated for 20&nbsp;mins at 75°C. The DNAs were sheared to a size of 200-300&nbsp;bp using Covaris M220. DNA size selection was conducted using AMpure XP magnetic beads (Beckman Coulter, A63881). Biotin pulldown was performed with Streptavidin T1 beads (Life Technologies, 65602). End repair, A adding, adaptor adding reaction, PCR amplification and DNA products size selection were performed sequentially. The libraries were sequenced by the Illumina PE150 paired-end sequencing platform.</div></section><section id="sec-4-4-2"><h4>DNase-seq library preparation</h4><div id="p0275" role="paragraph">DNase-seq processing followed the published protocol with slight modifications for our samples (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">He et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>He, H.H. ∙ Meyer, C.A. ∙ Hu, S.S. ...</div><div><strong>Refined DNase-seq protocol and data analysis reveals intrinsic bias in transcription factor footprint identification</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:73-78</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2762" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84894613651" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24317252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2762&amp;pmid=24317252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). One million fixed cells were re-suspended in 1&nbsp;mL buffer A+ (15&nbsp;mM Tris-HCl, pH 8.0, 15&nbsp;mM NaCl, 60&nbsp;mM KCl, 1&nbsp;mM EDTA, pH 8.0, 0.5&nbsp;mM EGTA, pH 8.0, 0.15mM Spermine, 0.5mM Spermidine, 1 x PIC 50&nbsp;× 200ul, 1mM PMSF, 0.5mM DTT, 0.2% NP40), digested on ice for 10&nbsp;minutes, and centrifuged at 2500&nbsp;rpm for 3&nbsp;mins. The supernatant was removed, and the pellet was re-suspended in 500ul 37°C pre-warmed 1x digestion buffer. 25U DNaseI (Roche, 4716728001) was added and inverted to mix, and incubated at 37°C for 5&nbsp;mins. 500ul stop buffer was added (50&nbsp;mM Tris-HCl, pH 8.0, 100&nbsp;mM NaCl, 0.10% SDS, 100&nbsp;mM EDTA, pH 8.0 with spermine, spermidine, and 2ul of RNase) to each reaction and mixed by inverting. The chromatin products were de-cross-linked overnight, and incubated at 65°C overnight. DNA extraction, end repair, A adding, adaptor adding reaction and PCR amplification were performed sequentially. The PCR products were run on a 2% agarose gel, the 180-500bp fragments were selected by gel cutting. The size selected DNAs were purified using QIAGEN Gel Purification Kit. The libraries were sequenced by the Illumina PE150 paired-end sequencing platform.</div></section><section id="sec-4-4-3"><h4>CTCF ChIP-seq library preparation</h4><div id="p0280" role="paragraph">Five million fixed cells were re-suspended in cold cell lysis buffer (10mM Tris pH 8, 10mM NaCl, 0.2% NP-40, 1xEDTA-free Protease Inhibitors). Cells were lysed for 10&nbsp;mins, washed once with cold PBS and re-suspended in 100ul per IP cold nuclei lysis buffer (50mM Tris pH8, 10mM EDTA, 1% SDS, 1xProtease Inhibitors). Nuclei were lysed for 15&nbsp;mins at 4°C and then sonicated for 20 cycles using Bioruptor (Diagenode). After sonication, 5x volumes of IP dilution buffer (20mM Tris pH8, 2mM EDTA, 150mM NaCl, 1% Triton X-100, 0.01% SDS, 1x protease inhibitors) was added, and chromatin was precleared using 25ul Protein A dynabeads (ThermoFisher, 10002D) / 1mL for 1hrs at 4°C with rotation. Beads were then washed once with 0.5% BSA in PBS, and added to the precleared chromatin and incubated overnight at 4°C with rotation. Beads were then washed once with cold IP wash buffer 1 (20mM Tris pH8, 2mM EDTA, 50mM NaCl, 1% Triton X-100, 0.1% SDS), twice with high salt wash buffer (20mM Tris pH8, 2mM EDTA, 500mM NaCl, 1% Triton X-100,0.1% SDS), once with cold IP wash buffer 2 (10mM Tris pH8,1mM EDTA,250mM LiCl,1% NP-40,1% sodium deoxycholate) and twice with cold TE buffer (1mM Tris pH8, 1mM EDTA). DNA extraction, end repair, A adding, adaptor adding reaction and PCR amplification were performed sequentially. The libraries were sequenced by the Illumina PE150 paired-end sequencing platform.</div></section><section id="sec-4-4-4"><h4>RNA extraction and library preparation</h4><div id="p0285" role="paragraph">RNA was extracted from GZ and CP cells using the QIAGEN miRNeasy Mini Kit (QIAGEN, 217004). Concentration and purity were evaluated with a NanoDrop. 2&nbsp;μg total RNA per sample was used as input material for the RNA sample preparation. Library preparation was completed using the VAHTS Total RNA-seq Library Prep Kit for Illumina (Vazyme, NR603-01) following the manufacturer’s recommendations. The libraries were sequenced by the Illumina PE150 paired-end sequencing platform.</div></section><section id="sec-4-4-5"><h4>3D DNA-FISH</h4><div id="p0290" role="paragraph">Oligopaint libraries were designed using the OligoMiner pipeline (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Beliveau et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Beliveau, B.J. ∙ Kishi, J.Y. ∙ Nir, G. ...</div><div><strong>OligoMiner provides a rapid, flexible environment for the design of genome-scale oligonucleotide in&nbsp;situ hybridization probes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E2183-E2192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1714530115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85042947754" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29463736/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1714530115&amp;pmid=29463736" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We run OligoMiner on the rheMac8 reference genome with the “balance” parameter sets (36-41 nucleotide homology length) using LDM and kmerFilter. Probes were designed to target a 70 Kb region at a density of about 5.17 probes per kilobase (Table S2).</div><div id="p0295" role="paragraph">Frozen fetal monkey brain was coronally sectioned at 8&nbsp;μm. The tissues were fixed using 4% PFA for 10min at room temperature, washed with PBS for three times, then permeabilized with 1% Triton X-100 (Sigma, 10789704001) in PBS for 15min at room temperature and treated with 100ug/ml RNaseA at 37°C for 45&nbsp;min. The slides were incubated in 0.5M HCl for 10&nbsp;min at room temperature, washed with 2&nbsp;× SSC (Thermo Fisher Scientific, AM9763), and treated with 0.01M HCl/0.0025% pepsin (Macklin, P6322-100 g) for 10min at 37°C. The pepsin was then inactivated by washing with 50mM Mgcl<sub>2</sub>/PBS for twice, and the slides were washed with 2&nbsp;× SSC for three times, then incubated in 50% of Formamide (Sigma, S6639-1L) with 2&nbsp;× SSC overnight at 4°C. In the next day, the cells were predenatured at 78°C for 10min and put into 70%, 90% and 100% cold ethanol for 2&nbsp;min sequentially and then air-dried. The 1st probes were denatured at 86°C for 5min and put quickly on ice. Then the probes were loaded on slide areas with cells, and the slides were incubated at 86°C for 3min and put quickly on ice, hybridized at 42°C overnight in a dark and humidified chamber. In the next day, the slides were washed with 2&nbsp;× SSC/0.1% Tri-tonX-100 for three times at 60°C. The 2st probes were added on the slides and incubated at room temperature for 1h. The slides were washed with 40% Formamide/2&nbsp;× SSC one time and 30% Formamide/2&nbsp;× SSC two times, mounted by ProLong Diamond Antifade Mountant with DAPI (ThermoFisher Scientific, P36966).</div></section><section id="sec-4-4-6"><h4>Functional validation of the human EPHA7 enhancer</h4><div id="p0300" role="paragraph">We designed four sgRNAs to disturb the human EPHA7 enhancer using the guide RNA design tool CHOPCHOP (<a id="intref0250" href="http://chopchop.cbu.uib.no/" target="_blank">http://chopchop.cbu.uib.no/</a>) (sgRNA1: CCCTAAATTCAGGAACTATT; sgRNA2: GGCTTGACTCAACTGACCAA; sgRNA3: TAGTTACTCTCTGCACAACC; sgRNA4: GCCAAATTCATGTCGTTTCC). sgRNAs were cloned into the pL-CRISPR.EFS.GFP (Addgene, 57818) or pL-CRISPR.EFS.tRFP (Addgene, 57819) plasmids (<span class="dropBlock reference-citations"><a id="body-ref-sref28-5" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-5" aria-controls="sref28-5">Heckl et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref28-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Heckl, D. ∙ Kowalczyk, M.S. ∙ Yudovich, D. ...</div><div><strong>Generation of mouse models of myeloid malignancy with combinatorial genetic lesions using CRISPR-Cas9 genome editing</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:941-946</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2951" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84911422892" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24952903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2951&amp;pmid=24952903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Lenti-virus were produced using 293T cells and concentrated with Lenti-X Concentrator reagent (Takara, 631231). Human NPCs were cultured and infected with the concentrated virus, and in the next day, the virus was removed and replaced with fresh medium. Two days later, the NPCs were differentiated by replacing differentiation medium and cultured for two weeks. The dual GFP/tRFP positive cells were isolated by FACS for genome DNA and RNA extraction. The sgRNAs efficiency were validated by PCR amplification of the human EPHA7 enhancer locus and the expression of <i>EPHA7</i> was quantified by qPCR. EPHA7-enhancer PCR Primers: forward- ACTAGGTTTCTTGGTGTTCATTCCC, reverse- TTGTACAGTCCCTCTATCACATGCA; qPCR primers: GAPDH forward- GTCTCCTCTGACTTCAACAGCG, GAPDH reverse: ACCACCCTGTTGCTG-TAGCCAA, EPHA7 forward- CTCGCAAGTGAGTGGAGTAATG, EPHA7 reverse- GTTGAGTAGGTCCGTTCCCTTTG.</div><div id="p0305" role="paragraph">The differentiated neurons were fixed with 4% PFA, permeabilized with 0.3% Triton X-100, blocked with 10% BSA, and incubated with primary antibodies GFP (Abcam, ab13970, 1:1000) at 4°C overnight, then incubated with secondary antibodies at 25°C for 1 h, counterstained with DAPI. Digital image acquisition was performed using LEICA TCS SP8 microscope. ImageJ was used to measure dendrite length. Each group was repeated with three replicates, and P values were calculated using t test.</div></section><section id="sec-4-4-7"><h4>Luciferase assay</h4><div id="p0310" role="paragraph">Human SK-N-SH cells were cultured with basic growth media (DMEM, 10%FBS, 2&nbsp;mM Glutamine and 1x Pen/Strep). The human EHAP7 enhancer and the orthologous chimpanzee and macaque sequences were cloned into the pGL3-promoter vector (Promega, E1761). 500&nbsp;ng pGL3-EPHA7-promoter or pGL3-promoter and 20ng pRL-TK (Promega) were transfected into SK-N-SH cells in 24-well plates using lipofectamine 3000. After 48 hours, cells were washed for three times using PBS, and luciferase activity was measured by Dual Luferase Reporter System (Promega). Each group was repeated with six technical replicates. Firefly luciferase activity was normalized to Renilla luciferase activity. P values were calculated using t test.</div></section><section id="sec-4-4-8"><h4>3C assay</h4><div id="p0315" role="paragraph">Five million cross-linked cells were re-suspended in lysis buffer and incubated on ice for 15&nbsp;mins. Removal of lysis buffer was done by centrifugation at 2500&nbsp;g for 5&nbsp;mins at 4°C, followed by removal of supernatant. The pellet was re-suspended in 342&nbsp;μL 1&nbsp;× NEBuffer 3.1, and incubated with 38&nbsp;μL 10% SDS at 65°C for 10&nbsp;mins. Next, 43&nbsp;μL of 10% Triton X-100 was added to the Hi-C-tube to quench the SDS at 37°C for 15min, then add 12&nbsp;μL 10&nbsp;× NEBuffer 3.1 and 400U DpnII (NEB, R0543), and mix gently to digest the chromatin overnight at 37°C on a rocking platform. In the next day, the DpnII re-striction enzyme was inactivated at 65°C for 25&nbsp;mins. Then, re-ligated by T4 DNA ligase (NEB, M0202), incubated at 16°C for 4 h, and shaken manually for three times. The chromatin products were de-cross-linked overnight by adding 30&nbsp;μL proteinase K, and incubated at 65°C overnight. Phenolic/chloroform was used to extract DNA. PCR and Sanger sequencing were carried out to confirm the long-range chromatin interaction. PCR primers are listed in Table S2.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><section id="sec-4-5-1"><h4>ChIP-Seq data analysis</h4><div id="p0320" role="paragraph">The Fastq reads were aligned to reference genome (hg19 for human, rheMac8 for macaque, mm10 for mouse) using bowtie2 (v2.3.4) (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Langmead and Salzberg, 2012</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (17046)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with settings “bowtie2 –very-sensitive.” Bam files were transferred to sam files using Samtools (v1.9) (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Li et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The Sequence Alignment/Map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (21853)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Duplicated alignments were removed using PicardTools (v2.20.2) (<a id="intref0255" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a>). To avoid double contigs for the paired-end alignments, only one mate count. Only uniquely mapped alignments (MAPQ &gt; 30) were kept. Averaged reads per million of mapped reads (RPM) in 10bp bins were calculated by deepTools (V3.2.1) (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Ramírez et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Ramírez, F. ∙ Dündar, F. ∙ Diehl, S. ...</div><div><strong>deepTools: a flexible platform for exploring deep-sequencing data</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:W187-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900988020" target="_blank" aria-label="Open Scopus in new tab">Scopus (707)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24799436/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku365&amp;pmid=24799436" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and transferred to coverage tracks. Technical replicates data were merged. Peaks were identified using MACS2 (V2.1.2) (<span class="dropBlock reference-citations"><a id="body-ref-sref103-2" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-2" aria-controls="sref103-2">Zhang et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref103-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (6096)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with settings “macs2 call Peak –B –p 1e-5 –nomodle –extsize 200.”</div></section><section id="sec-4-5-2"><h4>RNA-seq and differential expression analysis</h4><div id="p0325" role="paragraph">The intra-species RNA-Seq Fastq reads were aligned to Mmul_8.0.1 reference genome <span data-style="underline">(</span><a id="intref0260" href="ftp://ftp.ensembl.org/pub/release-96/fasta/macaca_mulatta/dna/" target="_blank">ftp://ftp.ensembl.org/pub/release-96/fasta/macaca_mulatta/dna/</a>) using STAR (V2.6.16) (<span class="dropBlock reference-citations"><a id="body-ref-sref17-3" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-3" aria-controls="sref17-3">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref17-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (10402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ENSEMBL gene annotation was used. Duplicated alignments were removed using PicardTools (v2.20.2) (<a id="intref0265" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a>). The number of alignments mapped to exons was counted using the summarizeOverlap function from the GenomicAlignments package (V1.18.1) (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Lawrence et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Lawrence, M. ∙ Huber, W. ∙ Pagès, H. ...</div><div><strong>Software for computing and annotating genomic ranges</strong></div><div><em>PLoS Comput. Biol.</em> 2013; <strong>9</strong>:e1003118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84883368195" target="_blank" aria-label="Open Scopus in new tab">Scopus (1155)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23950696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003118&amp;pmid=23950696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conditional quantile normalization was performed using CQN package (V1.28.1) (<span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Hansen et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Hansen, K.D. ∙ Irizarry, R.A. ∙ Wu, Z.</div><div><strong>Removing technical variability in RNA-seq data using conditional quantile normalization</strong></div><div><em>Biostatistics.</em> 2012; <strong>13</strong>:204-216</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/biostatistics/kxr054" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84858068675" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22285995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbiostatistics%2Fkxr054&amp;pmid=22285995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to correct read depth, exon length and GC content. Log2 normalized FPKM was used for quantification of expression levels. For inter-species RNA-Seq data comparison, we generated inter-species mRNA annotation in the way described in a previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref106" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106" aria-controls="sref106">Zhu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref106"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Zhu, Y. ∙ Sousa, A.M.M. ∙ Gao, T. ...</div><div><strong>Spatiotemporal transcriptomic divergence across human and macaque brain development</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat8077</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat8077" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85058394301" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545855/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat8077&amp;pmid=30545855" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to minimize the bias of gene annotation across species. We generated human-macaque orthologous annotation based on human Gencode v19. In brief, the XSAnno (<span class="dropBlock reference-citations"><a id="body-ref-sref105-2" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-2" aria-controls="sref105-2">Zhu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref105-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Zhu, Y. ∙ Li, M. ∙ Sousa, A.M.M. ...</div><div><strong>XSAnno: a framework for building ortholog models in cross-species transcriptome comparisons</strong></div><div><em>BMC Genomics.</em> 2014; <strong>15</strong>:343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-15-343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900840147" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24884593/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-15-343&amp;pmid=24884593" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) method first liftOver human genome annotation to macaque, then low quality alignments was filtered out. The settings were, “-minMatch” 0.913, inter-species PID 0.9, PL 0.95, intra-species PID 0.97, PL 0.95. The human and macaque RNA-Seq data were aligned to hg19 and Mmul_8.0.1 respectively using STAR (V2.6.16) and the orthologous mRNA annotation. The reads mapped to genes were counted using featureCounts function from Rsubread packsge (V1.32.4) (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Liao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>The R package Rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:e47</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkz114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85066058200" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30783653/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkz114&amp;pmid=30783653" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). FPKM was used for quantification of expression levels. Differential expression genes across species were identified using DESeq2 (V1.22.2) (<span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (16017)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-3"><h4>Hi-C data pre-process, matrix normalization and visualization</h4><div id="p0330" role="paragraph">Paired-end reads of the Hi-C libraries were pre-processed using HiC-Pro (V2.9.0) as described in the previous report (<span class="dropBlock reference-citations"><a id="body-ref-sref82-2" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-2" aria-controls="sref82-2">Servant et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref82-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Servant, N. ∙ Varoquaux, N. ∙ Lajoie, B.R. ...</div><div><strong>HiC-Pro: an optimized and flexible pipeline for Hi-C data processing</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0831-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84952861819" target="_blank" aria-label="Open Scopus in new tab">Scopus (432)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26619908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0831-x&amp;pmid=26619908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, reads pairs were aligned to the reference genomes (hg19 for human, rheMac8 for macaque, and mm10 for mouse) in two steps. The first step aligns reads using bowtie2 end-to-end algorithm. The second step detect the ligation sites and the 5′ fraction of the reads were aligned back to the genome. Aligned read pairs were assigned to DpnII restriction fragments. Invalid pairs like dangling end were filtered out. The genome was divided into specific size bins, and valid pairs were counted per bins. We used 500 kb, 100 kb, 40 kb bin sizes to generate raw and ICE (iterative correction and eigenvector decomposition) (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Imakaev et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Imakaev, M. ∙ Fudenberg, G. ∙ McCord, R.P. ...</div><div><strong>Iterative correction of Hi-C data reveals hallmarks of chromosome organization</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:999-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2148" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84866997011" target="_blank" aria-label="Open Scopus in new tab">Scopus (527)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22941365/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2148&amp;pmid=22941365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) normalized matrix. 500 kb matrix was used to identify compartments, 100 kb matrix was used to calculate Pearson correlation of replicates, and 40 kb matrix was used to define TAD boundaries. The valid pair files were transformed to .hic files which were ready for juicer (V1.11.04) (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Durand et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Durand, N.C. ∙ Shamim, M.S. ∙ Machol, I. ...</div><div><strong>Juicer Provides a One-Click System for Analyzing Loop-Resolution Hi-C Experiments</strong></div><div><em>Cell Syst.</em> 2016; <strong>3</strong>:95-98</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27467249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2016.07.002&amp;pmid=27467249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using pre command. We extracted KR (Knight-Ruiz) and VC_SQRT (square root of vanilla coverage) normalized matrix at 10 kb resolution. KR-normalized matrix was used to detect loops and to show Hi-C heatmaps. VC_SQRT-normalized matrix was used to show Hi-C heatmaps.</div></section><section id="sec-4-5-4"><h4>Definition and calculation of resolution</h4><div id="p0335" role="paragraph">we defined and calculated the Hi-C map resolution as described in <span class="dropBlock reference-citations"><a id="body-ref-sref77-5" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-5" aria-controls="sref77-5">Rao et&nbsp;al. (2014)</a><div class="dropBlock__holder" data-db-target-of="sref77-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Rao, S.S. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. The map resolution is defined as the smallest bin size where 80% of bins have at least 1,000 contact reads. The calculating scripts are from the published study</div><div id="p0340" role="paragraph">(<a id="intref0270" href="https://github.com/aidenlab/juicer/blob/master/misc/calculate_map_resolution.sh" target="_blank">https://github.com/aidenlab/juicer/blob/master/misc/calculate_map_resolution.sh</a>.).</div></section><section id="sec-4-5-5"><h4>Compartments analysis</h4><div id="p0345" role="paragraph">We assigned A/B compartments as described in the previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref55-2" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-2" aria-controls="sref55-2">Lieberman-Aiden et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref55-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Lieberman-Aiden, E. ∙ van Berkum, N.L. ∙ Williams, L. ...</div><div><strong>Comprehensive mapping of long-range interactions reveals folding principles of the human genome</strong></div><div><em>Science.</em> 2009; <strong>326</strong>:289-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1181369" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-70349873824" target="_blank" aria-label="Open Scopus in new tab">Scopus (3659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19815776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1181369&amp;pmid=19815776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, the expected contact frequency was calculated by the distance between the two loci. Pearson correlation matrix was calculated based on the observed/expected matrix. Principal component analysis was performed on the Pearson correlation matrix. The first principal component (PC1) was used to assign compartments. Eigenvalues with higher gene density was assigned as A compartment, otherwise assigned as B compartment.</div></section><section id="sec-4-5-6"><h4>TAD calling</h4><div id="p0350" role="paragraph">We defined TAD boundaries by two methods: insulation score analysis and domain caller (<span class="dropBlock reference-citations"><a id="body-ref-sref15-3" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-3" aria-controls="sref15-3">Dixon et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref15-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Dixon, J.R. ∙ Selvaraj, S. ∙ Yue, F. ...</div><div><strong>Topological domains in mammalian genomes identified by analysis of chromatin interactions</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:376-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84861095603" target="_blank" aria-label="Open Scopus in new tab">Scopus (3008)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11082&amp;pmid=22495300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Crane et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Crane, E. ∙ Bian, Q. ∙ McCord, R.P. ...</div><div><strong>Condensin-driven remodelling of X chromosome topology during dosage compensation</strong></div><div><em>Nature.</em> 2015; <strong>523</strong>:240-244</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14450" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84936945257" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26030525/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14450&amp;pmid=26030525" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For insulation score analysis, local contact insulation score was calculated as described in the previous report. We calculated insulation score using 40 kb bin size and 800 kb sliding window size (average size of TAD). The insulation score was normalized by the mean of all bins per chromosome. Valleys along the normalized insulation score indicate the reduced Hi-C contact. The valleys were detected by the finding horizontal zero point of delta vector. We filtered the horizontal zero points at peaks. Boundaries with distance less than 40 kb were merged into one. For domain caller, we calculated directionality index in 40 kb bin size using 2 Mb window size. Then we inferred the true hidden directionality bias by hidden Markov model (HMM). Domains were inferred from the HMM state calls. Regions between TADs and less than 400 kb were defined as boundaries.</div></section><section id="sec-4-5-7"><h4>Definition of insulation strength</h4><div id="p0355" role="paragraph">To test the insulation intensity of different types of boundaries, we defined insulation strength (see the illustration below). In brief, we calculated interactions within intra-TAD and inter-TAD 100 kb around a given boundary. Then we defined the ratio of (I<sub>intra-</sub>TAD – I<sub>inter-TAD</sub>) and (I<sub>intra-TAD</sub>&nbsp;+ I<sub>inter-TAD</sub>) as insulation strength. Interactions in the diagonal were filtered out.<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I Subscript inter negative TAD Baseline equals upper I Subscript a plus upper I Subscript b" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(20 (19 (7 0 (6 1 (4 2 3))) 18 (13 (11 10 8 9) 12)) 14 (17 15 16))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="20" data-semantic-children="19,17" data-semantic-content="14" data-semantic-owns="19 14 17" data-semantic-speech="upper I Subscript inter negative TAD Baseline equals upper I Subscript a plus upper I Subscript b"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="19" data-semantic-children="7,13" data-semantic-content="18" data-semantic-parent="20" data-semantic-owns="7 18 13"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="7" data-semantic-children="0,6" data-semantic-parent="19" data-semantic-owns="0 6"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c49">I</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.153em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="6" data-semantic-children="1,4" data-semantic-parent="7" data-semantic-collapsed="(6 (c 5) 1 4)" data-semantic-owns="1 4"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.657em;">n</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.657em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.657em;">e</mjx-c><mjx-c class="mjx-c72" style="padding-top: 0.657em;">r</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="4" data-semantic-children="3" data-semantic-content="2" data-semantic-parent="6" data-semantic-owns="2 3"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="unit" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c54" noic="true" style="padding-top: 0.716em;">T</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-c44" style="padding-top: 0.716em;">D</mjx-c></mjx-mtext></mjx-mrow></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="equality" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="19" data-semantic-owns="11 12"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="10,9" data-semantic-content="8" data-semantic-parent="13" data-semantic-owns="10 8 9"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="10" data-semantic-parent="11"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="8" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mtext space="4" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c49">I</mjx-c></mjx-mtext></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mtext size="s" data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c61">a</mjx-c></mjx-mtext></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="14" data-semantic-parent="20" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="20" data-semantic-owns="15 16"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c49">I</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.15em;"><mjx-mtext size="s" data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c62">b</mjx-c></mjx-mtext></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container></div></div></div><div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I Subscript inter negative TAD Baseline equals upper I Subscript c" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 (7 0 (6 1 (4 2 3))) 14 (13 (11 10 8 9) 12))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="15" data-semantic-children="7,13" data-semantic-content="14" data-semantic-owns="7 14 13" data-semantic-speech="upper I Subscript inter negative TAD Baseline equals upper I Subscript c"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="7" data-semantic-children="0,6" data-semantic-parent="15" data-semantic-owns="0 6"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c49">I</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.153em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="6" data-semantic-children="1,4" data-semantic-parent="7" data-semantic-collapsed="(6 (c 5) 1 4)" data-semantic-owns="1 4"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.657em;">n</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.657em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.657em;">e</mjx-c><mjx-c class="mjx-c72" style="padding-top: 0.657em;">r</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="4" data-semantic-children="3" data-semantic-content="2" data-semantic-parent="6" data-semantic-owns="2 3"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="unit" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c54" noic="true" style="padding-top: 0.716em;">T</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-c44" style="padding-top: 0.716em;">D</mjx-c></mjx-mtext></mjx-mrow></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="equality" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="15" data-semantic-owns="11 12"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="10,9" data-semantic-content="8" data-semantic-parent="13" data-semantic-owns="10 8 9"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="10" data-semantic-parent="11"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="8" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mtext space="4" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c49">I</mjx-c></mjx-mtext></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mtext size="s" data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c63">c</mjx-c></mjx-mtext></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container></div></div></div><div id="ufd3" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="insulation strength equals StartFraction upper I Subscript intra negative TAD Baseline negative upper I Subscript inter negative TAD Baseline Over upper I Subscript intra negative TAD Baseline plus upper I Subscript inter negative TAD Baseline EndFraction" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="(50 (46 0 2) (48 47 3 (44 (23 (11 4 (10 5 (8 6 7))) 22 (21 (14 12 13) (20 15 (18 16 17)))) (43 (31 24 (30 25 (28 26 27))) 42 (41 (34 32 33) (40 35 (38 36 37)))))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="50" data-semantic-children="46,48" data-semantic-collapsed="(50 (c 49) 46 48)" data-semantic-owns="46 48" data-semantic-speech="insulation strength equals StartFraction upper I Subscript intra negative TAD Baseline negative upper I Subscript inter negative TAD Baseline Over upper I Subscript intra negative TAD Baseline plus upper I Subscript inter negative TAD Baseline EndFraction"><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="46" data-semantic-children="0,2" data-semantic-parent="50" data-semantic-collapsed="(46 (c 45) 0 2)" data-semantic-owns="0 2"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="46"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.694em;">n</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.694em;">s</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.694em;">u</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.694em;">n</mjx-c></mjx-mtext><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="2" data-semantic-parent="46"><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.694em;">s</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.694em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.694em;">e</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.694em;">n</mjx-c><mjx-c class="mjx-c67" noic="true" style="padding-top: 0.694em;">g</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c68" style="padding-top: 0.694em;">h</mjx-c></mjx-mtext></mjx-mrow><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="48" data-semantic-children="47,44" data-semantic-content="3" data-semantic-parent="50" data-semantic-owns="47 3 44"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="47" data-semantic-parent="48"></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="48" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mfrac space="4" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="44" data-semantic-children="23,43" data-semantic-parent="48" data-semantic-owns="23 43"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="23" data-semantic-children="11,21" data-semantic-content="22" data-semantic-parent="44" data-semantic-owns="11 22 21"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="11" data-semantic-children="4,10" data-semantic-parent="23" data-semantic-owns="4 10"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="4" data-semantic-parent="11"><mjx-c class="mjx-c49">I</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.153em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="10" data-semantic-children="5,8" data-semantic-parent="11" data-semantic-collapsed="(10 (c 9) 5 8)" data-semantic-owns="5 8"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="5" data-semantic-parent="10"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.657em;">n</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.657em;">t</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.657em;">r</mjx-c><mjx-c class="mjx-c61" style="padding-top: 0.657em;">a</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="8" data-semantic-children="7" data-semantic-content="6" data-semantic-parent="10" data-semantic-owns="6 7"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="unit" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c54" noic="true" style="padding-top: 0.716em;">T</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-c44" style="padding-top: 0.716em;">D</mjx-c></mjx-mtext></mjx-mrow></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="negative" data-semantic-id="21" data-semantic-children="14,20" data-semantic-parent="23" data-semantic-owns="14 20"><mjx-mrow data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-children="13" data-semantic-content="12" data-semantic-parent="21" data-semantic-owns="12 13"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="12" data-semantic-parent="14" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c49">I</mjx-c></mjx-mtext></mjx-mrow><mjx-script style="vertical-align: -0.153em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="20" data-semantic-children="15,18" data-semantic-parent="21" data-semantic-collapsed="(20 (c 19) 15 18)" data-semantic-owns="15 18"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="15" data-semantic-parent="20"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.657em;">n</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.657em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.657em;">e</mjx-c><mjx-c class="mjx-c72" style="padding-top: 0.657em;">r</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="18" data-semantic-children="17" data-semantic-content="16" data-semantic-parent="20" data-semantic-owns="16 17"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="16" data-semantic-parent="18" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="unit" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="17" data-semantic-parent="18"><mjx-c class="mjx-c54" noic="true" style="padding-top: 0.716em;">T</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-c44" style="padding-top: 0.716em;">D</mjx-c></mjx-mtext></mjx-mrow></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="43" data-semantic-children="31,41" data-semantic-content="42" data-semantic-parent="44" data-semantic-owns="31 42 41"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="31" data-semantic-children="24,30" data-semantic-parent="43" data-semantic-owns="24 30"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="24" data-semantic-parent="31"><mjx-c class="mjx-c49">I</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.153em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="30" data-semantic-children="25,28" data-semantic-parent="31" data-semantic-collapsed="(30 (c 29) 25 28)" data-semantic-owns="25 28"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="25" data-semantic-parent="30"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.657em;">n</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.657em;">t</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.657em;">r</mjx-c><mjx-c class="mjx-c61" style="padding-top: 0.657em;">a</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="28" data-semantic-children="27" data-semantic-content="26" data-semantic-parent="30" data-semantic-owns="26 27"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="26" data-semantic-parent="28" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="unit" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c54" noic="true" style="padding-top: 0.716em;">T</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-c44" style="padding-top: 0.716em;">D</mjx-c></mjx-mtext></mjx-mrow></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="42" data-semantic-parent="43" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="positive" data-semantic-id="41" data-semantic-children="34,40" data-semantic-parent="43" data-semantic-owns="34 40"><mjx-mrow data-semantic-type="prefixop" data-semantic-role="positive" data-semantic-id="34" data-semantic-children="33" data-semantic-content="32" data-semantic-parent="41" data-semantic-owns="32 33"><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="32" data-semantic-parent="34" data-semantic-operator="prefixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="33" data-semantic-parent="34"><mjx-c class="mjx-c49">I</mjx-c></mjx-mtext></mjx-mrow><mjx-script style="vertical-align: -0.153em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="40" data-semantic-children="35,38" data-semantic-parent="41" data-semantic-collapsed="(40 (c 39) 35 38)" data-semantic-owns="35 38"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="35" data-semantic-parent="40"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.657em;">n</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.657em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.657em;">e</mjx-c><mjx-c class="mjx-c72" style="padding-top: 0.657em;">r</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="38" data-semantic-children="37" data-semantic-content="36" data-semantic-parent="40" data-semantic-owns="36 37"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="36" data-semantic-parent="38" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="unit" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="37" data-semantic-parent="38"><mjx-c class="mjx-c54" noic="true" style="padding-top: 0.716em;">T</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-c44" style="padding-top: 0.716em;">D</mjx-c></mjx-mtext></mjx-mrow></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div><div class="figure-wrap"><figure id="undfig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/22394018-867d-4761-9176-9ca0977be255/main.assets/fx2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx2" alt="Figure undfig2" src="/cms/10.1016/j.cell.2021.01.001/asset/67581f3b-000f-48ea-b87b-5e7d0ed7391e/main.assets/fx2.jpg" height="118" width="239" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/22394018-867d-4761-9176-9ca0977be255/main.assets/fx2_lrg.jpg" loading="lazy"></a><div><button class="figure__open__ctrl">Figure viewer</button></div></figure></div></div></section><section id="sec-4-5-8"><h4>Definition of conserved and species-specific boundaries and TADs</h4><div id="p0360" role="paragraph">To compare boundaries across species, we liftOver (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Hinrichs et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hinrichs, A.S. ∙ Karolchik, D. ∙ Baertsch, R. ...</div><div><strong>The UCSC Genome Browser Database: update 2006</strong></div><div><em>Nucleic Acids Res.</em> 2006; <strong>34</strong>:D590-D598</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkj144" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33644874573" target="_blank" aria-label="Open Scopus in new tab">Scopus (559)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16381938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkj144&amp;pmid=16381938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) macaque’s and mouse’s boundaries to the human reference genome (hg19). If distance of two boundaries was less than 80 kb (2 bins), these two boundaries are defined as conserved boundaries of two species (see the illustration below). The boundary conserved in three species was defined as species-conserved boundary. To select the most-likely distinct boundaries, we defined species-specific boundary as distance more than 200 kb (5 bins). TADs flanked by the conserved boundaries at both sides are defined as conserved TADs. TADs flanked by the species-specific boundaries at least one side are defined as species-specific TADs. For the data shown in figures, we merged boundaries called from different brain regions (CP and GZ) or cell types (CN and NPC) for each species.<div class="figure-wrap"><figure id="undfig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/f109f5f0-a786-4d8e-bacd-0beb3beb20b1/main.assets/fx3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx3" alt="Figure undfig3" src="/cms/10.1016/j.cell.2021.01.001/asset/de5aa33c-94e1-4072-b390-888043777d72/main.assets/fx3.jpg" height="166" width="488" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/f109f5f0-a786-4d8e-bacd-0beb3beb20b1/main.assets/fx3_lrg.jpg" loading="lazy"></a><div><button class="figure__open__ctrl">Figure viewer</button></div></figure></div></div></section><section id="sec-4-5-9"><h4>Overlap of TAD boundaries with repeat elements</h4><div id="p0365" role="paragraph">Repeat elements were downloaded from the UCSC Genome Browser (<a id="intref0275" href="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/rmsk.txt.gz" target="_blank">http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/rmsk.txt.gz</a>). We calculated the density of repeat elements overlapped with Cons or HG TAD boundaries (numbers of repeat element per boundaries). Then we used t test to quantify the difference of repeat elements density between Cons and HG TAD boundaries.</div></section><section id="sec-4-5-10"><h4>Loop calling</h4><div id="p0370" role="paragraph">We detected significantly enriched contact interactions using Hi-C computational Unbiased Peak Search (HiCCUPS) (V1.11.04) (<span class="dropBlock reference-citations"><a id="body-ref-sref18-3" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-3" aria-controls="sref18-3">Durand et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref18-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Durand, N.C. ∙ Shamim, M.S. ∙ Machol, I. ...</div><div><strong>Juicer Provides a One-Click System for Analyzing Loop-Resolution Hi-C Experiments</strong></div><div><em>Cell Syst.</em> 2016; <strong>3</strong>:95-98</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27467249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2016.07.002&amp;pmid=27467249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with default settings as described in the previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref77-6" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-6" aria-controls="sref77-6">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Rao, S.S. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, fold change values quantifying enrichment of contact frequency compared to local background were calculated. Then we tested if the number of the observed contacts is significantly enriched. P value was Benjamini-Hochberg adjusted to correct for multiple tests. Pixels with enrichment in all local neighborhoods (donut, horizontal, vertical and lower right) were kept. Then cluster of nearby enriched pixels were analyzed using a greedy algorithm. To generate the maximum counts of reads, samples were merged for each brain zones. We detected loops at 10 kb resolution.</div></section><section id="sec-4-5-11"><h4>Aggregated peak analysis</h4><div id="p0375" role="paragraph">To test the enrichment of the detected loops, we performed the aggregated peak analysis (APA) using juicer tools (V1.11.04) (<span class="dropBlock reference-citations"><a id="body-ref-sref18-4" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-4" aria-controls="sref18-4">Durand et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref18-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Durand, N.C. ∙ Shamim, M.S. ∙ Machol, I. ...</div><div><strong>Juicer Provides a One-Click System for Analyzing Loop-Resolution Hi-C Experiments</strong></div><div><em>Cell Syst.</em> 2016; <strong>3</strong>:95-98</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27467249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2016.07.002&amp;pmid=27467249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as described in the previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref77-7" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-7" aria-controls="sref77-7">Rao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Rao, S.S. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, submatrices around peaks were taken from KR-normalized matrix and then summed. The APA matrix was plotted as the heatmap. APA score was calculated as ratio of number of reads in the center bin to average number of reads in the lower-left corner. APA score greater than 1 represents enrichment peak. Some APA scores labeled in figures were normalized by dividing all the submatrices by the mean value of the submatrix.</div></section><section id="sec-4-5-12"><h4>Definition of conserved and species-specific loops</h4><div id="p0380" role="paragraph">Cross-species loop comparison was achieved by the following pipeline. For each species, loops obtained from different brain zones or cell types are combined to get a concatenate loop list. The coordinates for loop anchors on target genome were obtained from source genome using UCSC liftOver and only loops with both anchors that can map to a 5kb to 20kb region are retained. LiftOver and reciprocal liftOver have to be unique. Anchors that have more than 50% overlapped with source coordinates during reciprocal liftOver are retained. Then the mapped position of each loop anchor was aligned to the unit resolution interval with the highest degree of coincidence.</div><div id="p0385" role="paragraph">For each mapped loop, two kinds of judgements were performed: the distance judgement which determines whether the mapped loop is located near any loop of target species and the enrichment judgement which determines whether the mapped loop still show signal enrichment in the Hi-C contact map of target species. The distance judgment is achieved by comparing the two-dimensional Euclidean distance between loops with some preset thresholds:<div id="ufd4" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="italic d i s t a n c e less than max left parenthesis r a d i u s comma min left parenthesis r 1 times StartAbsoluteValue i minus j EndAbsoluteValue comma r 2 right parenthesis right parenthesis" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="(45 0 1 (44 2 43 (42 3 (40 (37 4 32 5 33 6 34 7 35 8 36 9) 10 (39 11 38 (31 12 (29 (28 (15 13 14) 16 (23 17 (21 18 19 20) 22)) 24 (27 25 26)) 30))) 41)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="45" data-semantic-children="0,44" data-semantic-content="1" data-semantic-owns="0 1 44" data-semantic-speech="italic d i s t a n c e less than max left parenthesis r a d i u s comma min left parenthesis r 1 times StartAbsoluteValue i minus j EndAbsoluteValue comma r 2 right parenthesis right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-id="0" data-semantic-parent="45"><mjx-c class="mjx-c1D451" noic="true" style="padding-top: 0.694em;">𝑑</mjx-c><mjx-c class="mjx-c1D456" noic="true" style="padding-top: 0.694em;">𝑖</mjx-c><mjx-c class="mjx-c1D460" noic="true" style="padding-top: 0.694em;">𝑠</mjx-c><mjx-c class="mjx-c1D461" noic="true" style="padding-top: 0.694em;">𝑡</mjx-c><mjx-c class="mjx-c1D44E" noic="true" style="padding-top: 0.694em;">𝑎</mjx-c><mjx-c class="mjx-c1D45B" noic="true" style="padding-top: 0.694em;">𝑛</mjx-c><mjx-c class="mjx-c1D450" noic="true" style="padding-top: 0.694em;">𝑐</mjx-c><mjx-c class="mjx-c1D452" style="padding-top: 0.694em;">𝑒</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="45" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="limit function" data-semantic-id="44" data-semantic-children="2,42" data-semantic-content="43,2" data-semantic-parent="45" data-semantic-owns="2 43 42"><mjx-mi data-semantic-type="function" data-semantic-role="limit function" data-semantic-font="normal" data-semantic-id="2" data-semantic-parent="44" data-semantic-operator="appl"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.448em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.448em;">a</mjx-c><mjx-c class="mjx-c78" style="padding-top: 0.448em;">x</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="42" data-semantic-children="40" data-semantic-content="3,41" data-semantic-parent="44" data-semantic-owns="3 40 41"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="42" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="40" data-semantic-children="37,10,39" data-semantic-content="10" data-semantic-parent="42" data-semantic-owns="37 10 39"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="37" data-semantic-children="4,5,6,7,8,9" data-semantic-content="32,33,34,35,36" data-semantic-parent="40" data-semantic-owns="4 32 5 33 6 34 7 35 8 36 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="37"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="32" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="37"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="37"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="34" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="37"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="35" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="37"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="36" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="37"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="10" data-semantic-parent="40" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="limit function" data-semantic-id="39" data-semantic-children="11,31" data-semantic-content="38,11" data-semantic-parent="40" data-semantic-owns="11 38 31"><mjx-mi data-semantic-type="function" data-semantic-role="limit function" data-semantic-font="normal" data-semantic-id="11" data-semantic-parent="39" data-semantic-operator="appl"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.657em;">m</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.657em;">n</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="31" data-semantic-children="29" data-semantic-content="12,30" data-semantic-parent="39" data-semantic-owns="12 29 30"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="12" data-semantic-parent="31" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="29" data-semantic-children="28,24,27" data-semantic-content="24" data-semantic-parent="31" data-semantic-owns="28 24 27"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="28" data-semantic-children="15,23" data-semantic-content="16" data-semantic-parent="29" data-semantic-owns="15 16 23"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="28" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="28" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="23" data-semantic-children="21" data-semantic-content="17,22" data-semantic-parent="28" data-semantic-owns="17 21 22"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="17" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="21" data-semantic-children="18,20" data-semantic-content="19" data-semantic-parent="23" data-semantic-owns="18 19 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="21"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="19" data-semantic-parent="21" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="22" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="24" data-semantic-parent="29" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="27" data-semantic-children="25,26" data-semantic-parent="29" data-semantic-owns="25 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="27"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="27"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="30" data-semantic-parent="31" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="41" data-semantic-parent="42" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div></div><div id="p0390" role="paragraph">Here, the radius which is the output of HiCCUPS represents the surrounding area of corresponding target species loop and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartAbsoluteValue i minus j EndAbsoluteValue" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 (4 1 2 3) 5)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="6" data-semantic-children="4" data-semantic-content="0,5" data-semantic-owns="0 4 5" data-semantic-speech="StartAbsoluteValue i minus j EndAbsoluteValue"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> represents the distance between two anchors of it, while <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r 1" tabindex="0" ctxtmenu_counter="6" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r 2" tabindex="0" ctxtmenu_counter="7" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are relative distance coefficient and absolute distance threshold, respectively, which are preset parameters, and 0.2 and 20kb used for identifying conserved loops while 0.2 and 50kb used for finding species-specific loops.</div><div id="p0395" role="paragraph">The enrichment judgement was performed to check whether the mapped loops showed signal enrichment over the local background. Matrices were normalized and the expected normalized contacts at each distance were calculated. For each mapped loop, we defined several local neighborhoods as described by <span class="dropBlock reference-citations"><a id="body-ref-sref77-8" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-8" aria-controls="sref77-8">Rao et&nbsp;al. (2014)</a><div class="dropBlock__holder" data-db-target-of="sref77-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Rao, S.S. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>: donut, horizontal, vertical, and lower left. For each neighborhood, ratio of sum of the observed counts to sum of expected counts were calculated and then multiplied by the expected signal of the mapped loop, and this value is the estimation of expected contacts for the corresponding neighborhood. Then the fold-changes of observed contacts of mapped loop over the expected contacts of four kinds of neighborhoods are calculated separately. We calculated these four kinds of fold-changes for each enriched-pixel of target species and selected 20% quantiles of them to be thresholds. For example, when the target species is mouse, a mapped loop with fold-change of observed versus expected (horizontal and vertical) &gt; 1.5, fold-change of observed versus expected (lower-left and donut) &gt; 1.65, fold-change of observed versus expected (lower-left or donut) &gt; 1.76 is considered as signal enriched. Otherwise, if all the fold-changes above are less than 1.3, the mapped loop is considered as no-enriched pixel. Combining the distance judgement and enrichment judgement, we can determine whether a mapped loop is species-conserved or species-specific in the following way: a mapped loop which is close enough to a loop of target species or has signal enrichment over local background will be considered as species-conserved; if it’s separated from all the loops of target species and determined as no-enriched pixel, it will be considered as species-specific. It should be noted that here we do not classify all the loops, but use different parameters when identifying conserved and species-specific loops to ensure that both loop sets are credible.<div class="figure-wrap"><figure id="undfig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.001/asset/dbd7339b-f38e-4f8d-b5c5-79cd50458ef1/main.assets/fx4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx4" alt="Figure undfig4" src="/cms/10.1016/j.cell.2021.01.001/asset/c6d0a589-7e6d-4a5d-a04f-53b1b84f3a24/main.assets/fx4.jpg" height="211" width="561" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/dbd7339b-f38e-4f8d-b5c5-79cd50458ef1/main.assets/fx4_lrg.jpg" loading="lazy"></a><div><button class="figure__open__ctrl">Figure viewer</button></div></figure></div></div></section><section id="sec-4-5-13"><h4>GO enrichment analysis</h4><div id="p0400" role="paragraph">GO enrichment was performed on genes with promoter overlapped boundaries (20kb around) or loop anchors. Promoters were defined as upstream 2 kb and downstream 2kb of transcription start site (TSS). Biological process, cellular component and molecular function gene ontology were performed using DAVID (<span class="dropBlock reference-citations"><a id="body-ref-sref13-2" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-2" aria-controls="sref13-2">Dennis et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref13-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Dennis, Jr., G. ∙ Sherman, B.T. ∙ Hosack, D.A. ...</div><div><strong>DAVID: Database for Annotation, Visualization, and Integrated Discovery</strong></div><div><em>Genome Biol.</em> 2003; <strong>4</strong>:3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2003-4-5-p3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12734009/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2003-4-5-p3&amp;pmid=12734009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-14"><h4>Enhancer definition</h4><div id="p0405" role="paragraph">We use chromatin epigenomic features to infer enhancer activity. For all of the analyses, enhancers of human were defined as regions with an H3K4me1 peak of human fetal brain (E081) from the Roadmap Epigenomics Consortium (<span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Ernst and Kellis, 2015</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Ernst, J. ∙ Kellis, M.</div><div><strong>Large-scale imputation of epigenomic datasets for systematic annotation of diverse human tissues</strong></div><div><em>Nat. Biotechnol.</em> 2015; <strong>33</strong>:364-376</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84926632357" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25690853/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3157&amp;pmid=25690853" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45-3" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-3" aria-controls="sref45-3">Kundaje et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref45-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Kundaje, A. ∙ Meuleman, W. ∙ Ernst, J. ..., Roadmap Epigenomics Consortium</div><div><strong>Integrative analysis of 111 reference human epigenomes</strong></div><div><em>Nature.</em> 2015; <strong>518</strong>:317-330</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14248" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84923362619" target="_blank" aria-label="Open Scopus in new tab">Scopus (2646)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25693563/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14248&amp;pmid=25693563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and H3K4me1 peaks overlapped a promoter were filtered out. Enhancers of macaque were defined as regions with an H3K4me2 or H3K27ac peak of 12 PCW macaque frontal cortex from <span class="dropBlock reference-citations"><a id="body-ref-sref78-3" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-3" aria-controls="sref78-3">Reilly et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref78-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Reilly, S.K. ∙ Yin, J. ∙ Ayoub, A.E. ...</div><div><strong>Evolutionary genomics. Evolutionary changes in promoter and enhancer activity during human corticogenesis</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1155-1159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84924561748" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25745175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260943&amp;pmid=25745175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, H3K4me2 peaks and H3K27ac peaks overlapped a promoter were filtered out. Enhancers of mouse were defined as regions with H3K4me1 peaks of mouse cortical neurons from <span class="dropBlock reference-citations"><a id="body-ref-sref5-5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-5" aria-controls="sref5-5">Bonev et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref5-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bonev, B. ∙ Mendelson Cohen, N. ∙ Szabo, Q. ...</div><div><strong>Multiscale 3D Genome Rewiring during Mouse Neural Development</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:557-572.e24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85031779520" target="_blank" aria-label="Open Scopus in new tab">Scopus (361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29053968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.043&amp;pmid=29053968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, and H3K4me1 peaks overlapped a promoter were filtered out.</div></section><section id="sec-4-5-15"><h4>Motif enrichment analysis</h4><div id="p0410" role="paragraph">To find enriched motif for transcription factor binding sites in ATAC-Seq peaks (for human) or DNase-I Hypersensitivity (DNase) peaks (for macaque and mouse) which overlapped with loop anchors, we obtained motifs from JASPAR v2018 (<span class="dropBlock reference-citations"><a id="body-ref-sref39-2" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-2" aria-controls="sref39-2">Khan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref39-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Khan, A. ∙ Fornes, O. ∙ Stigliani, A. ...</div><div><strong>JASPAR 2018: update of the open-access database of transcription factor binding profiles and its web framework</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D260-D266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1126" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85040936337" target="_blank" aria-label="Open Scopus in new tab">Scopus (568)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29140473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1126&amp;pmid=29140473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and known motifs from HOMER (V4.10) (<span class="dropBlock reference-citations"><a id="body-ref-sref29-2" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-2" aria-controls="sref29-2">Heinz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref29-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (4333)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sequences in the ATAC-Seq peaks or the DNase peaks were input for HOMER&nbsp;package motif finder algorithm findMotifGenome.pl. We used a random set of background regions chosen by HOMER. For the occurrence of motif, we used the FIMO (V5.0.2) (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Grant et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Grant, C.E. ∙ Bailey, T.L. ∙ Noble, W.S.</div><div><strong>FIMO: scanning for occurrences of a given motif</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:1017-1018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr064" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-79953300078" target="_blank" aria-label="Open Scopus in new tab">Scopus (1382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21330290/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr064&amp;pmid=21330290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with P value cutoff 10<sup>-4</sup>.</div></section><section id="sec-4-5-16"><h4>Community detection</h4><div id="p0415" role="paragraph">We detected community as described in the previous report (<span class="dropBlock reference-citations"><a id="body-ref-sref71-3" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-3" aria-controls="sref71-3">Phanstiel et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref71-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Phanstiel, D.H. ∙ Van Bortle, K. ∙ Spacek, D. ...</div><div><strong>Static and Dynamic DNA Loops form AP-1-Bound Activation Hubs during Macrophage Development</strong></div><div><em>Mol. Cell.</em> 2017; <strong>67</strong>:1037-1048.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2017.08.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2017.08.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85030242070" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28890333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2017.08.006&amp;pmid=28890333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A non-directed network was generated from Hi-C interactions using igraph package (V1.2.4). Loop anchors were defined as nodes, and loops as edges. Community was detected by fastgreedy.community function from igraph with default settings. Protein-protein interaction network (STRING) community was detected in the same way.</div></section><section id="sec-4-5-17"><h4>Lamina-specific and module-specific analysis</h4><div id="p0420" role="paragraph">We used mid-gestational human neocortical laminar gene expression data from the published study (<span class="dropBlock reference-citations"><a id="body-ref-sref62-7" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-7" aria-controls="sref62-7">Miller et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref62-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to do lamina-specific and module-specific analysis. Lamina specificity was quantified as described in the previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Parikshak et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Parikshak, N.N. ∙ Luo, R. ∙ Zhang, A. ...</div><div><strong>Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1008-1021</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84889561601" target="_blank" aria-label="Open Scopus in new tab">Scopus (523)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.031&amp;pmid=24267887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, specific expression of genes in a lamina was against to all other laminae, and t values of each gene per lamina were generated to quantify the expression specificity. Given a set of genes (such as all target genes regulated by HSLs), we quantified the skew of t value sets using whole genome genes as the background. We got z-scores and corresponding P values. We quantified module-specificity of a set of genes for 42 modules detected in the previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref62-8" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-8" aria-controls="sref62-8">Miller et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref62-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We assigned enrichment of a set of genes by permutation test. We randomly selected the same number as given gene sets 10,000 times. Then the numbers of the overlapped genes between randomly selected genes sets and genes in modules were used as background.</div></section><section id="sec-4-5-18"><h4>Human-specific SNC enrichment analysis</h4><div id="p0425" role="paragraph">We evaluated the enrichment of the human-specific single nucleotide changes (SNCs) (determined by inter-species whole genome alignments, downloaded from the <a id="intref0280" href="http://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/compara/" target="_blank">http://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/compara/</a>) at the TFs binding motifs. First, we defined the TF set with HSL-binding preference and the TF set without binding preference compared to the conserved loops (using -log10 P Value (H-S - Cons)&nbsp;= 10 and ± 2 as thresholds respectively). Then we calculated the SNC-containing ratio per TFs and compared with the randomly selected regions (TFs with fewer than 5 SNC-containing motif instances were filtered out). The enrichments were evaluated by Fisher’s exact test, and P values were adjusted using the Benjamini &amp; Hochberg method. About 65% (11 out of 17) of these TFs with HSL-binding preference are significantly enriched of the human-specific SNCs (FDR&nbsp;&lt; 0.001).</div></section><section id="sec-4-5-19"><h4>TF-target gene regulatory network</h4><div id="p0430" role="paragraph">For transcription factors showing differential enrichment between open chromatin regions of HSLs and ConsLs (NFIX, NFIA, NFIC, BHLHA15, TBR1, ATOH1, NR2E3, TCF4, OLIG2, NEUROG1, NEUROG2, EOMES and TGIF1), we predicted their putative target genes and generated TF-gene regulatory networks. First, we calculated Pearson correlation of TF-gene pairs using human fetal brain laminar gene expression data (<span class="dropBlock reference-citations"><a id="body-ref-sref62-9" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-9" aria-controls="sref62-9">Miller et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref62-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) per TF. We selected the genes with the top 1,000 most significant Pearson correlations. Then we filtered out genes of which promoter contains at least one binding site for the corresponding TF. We generated TF-gene regulatory networks using igraph package (V1.2.4). Nodes are TFs and putative target genes, and edges are Pearson correlations.</div></section><section id="sec-4-5-20"><h4>3D-DNA FISH imaging and analysis</h4><div id="p0435" role="paragraph">Images were acquired by a LEICA TCS SP8 confocal microscope with 63&nbsp;× objective and 2.5&nbsp;× Zoom factor. Images were obtained at 1,024&nbsp;× 1,024 pixels with a scan speed of 600&nbsp;Hz, frame average with 3, Z size with 0.33&nbsp;μm and 25 Z-slices were taken. All the images were acquired under the same parameters. Image analysis was performed using imageJ (<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (18835)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The centroid positions in xyz were calculated by “3D Object Counter” plug-in with manually curated threshold per channels. Euclidean distances between each channel pairs were calculated.</div></section><section id="sec-4-5-21"><h4>Definition of syntenic and non-syntenic 3D genome architectures</h4><div id="p0440" role="paragraph">We identified syntenic blocks between human and two other species (mouse and macaque) using Inferring Contiguous Ancestral Regions (inferCars) (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Ma et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Ma, J. ∙ Zhang, L. ∙ Suh, B.B. ...</div><div><strong>Reconstructing contiguous regions of an ancestral genome</strong></div><div><em>Genome Res.</em> 2006; <strong>16</strong>:1557-1565</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.5383506" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33750386912" target="_blank" aria-label="Open Scopus in new tab">Scopus (210)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16983148/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.5383506&amp;pmid=16983148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with the default settings. These blocks represent syntenic genomic regions across-species and the left part of the genome is considered as the non-syntenic regions. If the entire region of a TAD or loop is contained in a syntenic block, we consider it to be syntenic across species. For comparison, we define TADs and loops that span different syntenic blocks or fall in the junction areas of two adjacent syntenic blocks as non-syntenic 3D chromatin features. We also divide the non-syntenic human TADs and loops into 6 categories, including indel, translocation, inversion, large gap, interchromosomal recombination and other type according to the formation mechanisms of the related non-syntenic regions.</div></section><section id="sec-4-5-22"><h4>Statistical methods</h4><div id="p0445" role="paragraph">All of the statistical details can be found in the figure legend and <a id="crosref1840" href="#sec-4">STAR methods</a>.</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0450" role="paragraph">We thank Yan Guo for her technical assistance in this study. We thank Prof. Yujie Sun from Peking University for his help in FISH imaging experiment and data analysis. Part of the analysis was performed on the High Performance Computing Platform of the Center for Life Science of Peking University and State Key Laboratory of Scientific and Engineering Computing, Chinese Academy of Sciences. This study was supported by grants from the Strategic Priority Research Program (XDB13010000), National Natural Science Foundation of China (U2002207, 31730088, 31621062, and 91631306 to B.S.; 32000406 to X.L.; 31871266 and 71532001 to Y.L., T.L., and C.L.; and 11661141019, 2019YFA0709501 and 61621003 to S.Z.), National Key Research and Development Program of China (2016YFA0100103), National Ten Thousand Talent Program for Young Top-notch Talents, and the Ten Thousand Talents Project of Yunnan Province of China.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0455" role="paragraph">X.L. and B.S. designed the study. X.L., T.H., X.M., F.Z., M.L., H.W., Yingping Hou, Q.S., Yan Hu, and X.Z. performed the experiments. Y.L., D.D., X.L., T.L., and C.W. performed the data analysis. X.H. and L.Y. assisted in the reproductive technique. X.L., Y.L., D.D., C.L., S.Z., and B.S. wrote the paper.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0460" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (4)</h2><div id="p0470" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.01.001/attachment/340c8d17-bfdd-4c86-bc3c-cbced9ce9d0c/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (17.29 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Hi-C library stats of rhesus macaque fetal brain, related to Figure&nbsp;1</div><div id="espara0010" role="paragraph">Summary statistics for macaque fetal brain Hi-C data</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.01.001/attachment/814d397f-302d-4714-ada5-014e4fad35dd/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (225.95 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Primers of 3C and DNA-FISH, related to Figure&nbsp;1</div><div id="espara0015" role="paragraph">Lists of primers used for the 3C and FISH</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.01.001/attachment/6fa51f28-0444-4b3e-8175-6e5c808bede0/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (518.46 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. TAD boundary positions and genes around TADs, related to Figure&nbsp;2</div><div id="espara0020" role="paragraph">Genomic coordinates of the detected domain boundaries in three species, including the conserved and the species-specific boundaries, and the GO terms for genes around the conserved and the human-specific boundaries.</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.01.001/attachment/e9bb96b8-383c-4caf-b836-48f557e754b6/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (7.97 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Loop positions and loop target genes, related to Figure&nbsp;3</div><div id="espara0025" role="paragraph">Genomic coordinates of the detected loops in three species, including the conserved and the species-specific loops, and the GO terms for genes around the conserved and the human-specific loops.</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Ashrafi, S. ∙ Betley, J.N. ∙ Comer, J.D. ...</div><div><strong>Neuronal Ig/Caspr recognition promotes the formation of axoaxonic synapses in mouse spinal cord</strong></div><div><em>Neuron.</em> 2014; <strong>81</strong>:120-129</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2013.10.060&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2013.10.060&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84891809781" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24411736/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2013.10.060&amp;pmid=24411736" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Belaghzal, H. ∙ Dekker, J. ∙ Gibcus, J.H.</div><div><strong>Hi-C 2.0: An optimized Hi-C procedure for high-resolution genome-wide mapping of chromosome conformation</strong></div><div><em>Methods.</em> 2017; <strong>123</strong>:56-65</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ymeth.2017.04.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85018788636" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28435001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymeth.2017.04.004&amp;pmid=28435001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Beliveau, B.J. ∙ Kishi, J.Y. ∙ Nir, G. ...</div><div><strong>OligoMiner provides a rapid, flexible environment for the design of genome-scale oligonucleotide in&nbsp;situ hybridization probes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E2183-E2192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1714530115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85042947754" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29463736/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1714530115&amp;pmid=29463736" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Bird, C.P. ∙ Stranger, B.E. ∙ Liu, M. ...</div><div><strong>Fast-evolving noncoding sequences in the human genome</strong></div><div><em>Genome Biol.</em> 2007; <strong>8</strong>:R118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2007-8-6-r118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-39049157173" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17578567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2007-8-6-r118&amp;pmid=17578567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Bonev, B. ∙ Mendelson Cohen, N. ∙ Szabo, Q. ...</div><div><strong>Multiscale 3D Genome Rewiring during Mouse Neural Development</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:557-572.e24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.09.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85031779520" target="_blank" aria-label="Open Scopus in new tab">Scopus (361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29053968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.043&amp;pmid=29053968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Bush, E.C. ∙ Lahn, B.T.</div><div><strong>A genome-wide screen for noncoding elements important in primate evolution</strong></div><div><em>BMC Evol. Biol.</em> 2008; <strong>8</strong>:17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-8-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-39049088628" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18215302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-8-17&amp;pmid=18215302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Chen, H. ∙ Li, C. ∙ Zhou, Z. ...</div><div><strong>Fast-Evolving Human-Specific Neural Enhancers Are Associated with Aging-Related Diseases</strong></div><div><em>Cell Syst.</em> 2018; <strong>6</strong>:604-611.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2018.04.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2018.04.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85047107335" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29792826/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2018.04.002&amp;pmid=29792826" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Clifford, M.A. ∙ Athar, W. ∙ Leonard, C.E. ...</div><div><strong>EphA7 signaling guides cortical dendritic development and spine maturation</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:4994-4999</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1323793111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84897509280" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24707048/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1323793111&amp;pmid=24707048" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Cournac, A. ∙ Koszul, R. ∙ Mozziconacci, J.</div><div><strong>The 3D folding of metazoan genomes correlates with the association of similar repetitive elements</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:245-255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1292" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84959206565" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26609133/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1292&amp;pmid=26609133" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Crane, E. ∙ Bian, Q. ∙ McCord, R.P. ...</div><div><strong>Condensin-driven remodelling of X chromosome topology during dosage compensation</strong></div><div><em>Nature.</em> 2015; <strong>523</strong>:240-244</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14450" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84936945257" target="_blank" aria-label="Open Scopus in new tab">Scopus (292)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26030525/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14450&amp;pmid=26030525" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>de la Torre-Ubieta, L. ∙ Stein, J.L. ∙ Won, H. ...</div><div><strong>The Dynamic Landscape of Open Chromatin during Human Cortical Neurogenesis</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:289-304.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.12.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.12.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85039917750" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29307494/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.014&amp;pmid=29307494" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Dean, C. ∙ Dresbach, T.</div><div><strong>Neuroligins and neurexins: linking cell adhesion, synapse formation and cognitive function</strong></div><div><em>Trends Neurosci.</em> 2006; <strong>29</strong>:21-29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.tins.2005.11.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.tins.2005.11.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-30344454380" target="_blank" aria-label="Open Scopus in new tab">Scopus (222)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16337696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tins.2005.11.003&amp;pmid=16337696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13-1" title="View in article">13.</a></div><div>Dennis, Jr., G. ∙ Sherman, B.T. ∙ Hosack, D.A. ...</div><div><strong>DAVID: Database for Annotation, Visualization, and Integrated Discovery</strong></div><div><em>Genome Biol.</em> 2003; <strong>4</strong>:3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2003-4-5-p3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12734009/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2003-4-5-p3&amp;pmid=12734009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Dennis, M.Y. ∙ Harshman, L. ∙ Nelson, B.J. ...</div><div><strong>The evolution and population diversity of human-specific segmental duplications</strong></div><div><em>Nat. Ecol. Evol.</em> 2017; <strong>1</strong>:69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-016-0069" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85029929169" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28580430/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-016-0069&amp;pmid=28580430" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Dixon, J.R. ∙ Selvaraj, S. ∙ Yue, F. ...</div><div><strong>Topological domains in mammalian genomes identified by analysis of chromatin interactions</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:376-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84861095603" target="_blank" aria-label="Open Scopus in new tab">Scopus (3008)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11082&amp;pmid=22495300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Doan, R.N. ∙ Bae, B.I. ∙ Cubelos, B. ..., Homozygosity Mapping Consortium for Autism</div><div><strong>Mutations in Human Accelerated Regions Disrupt Cognition and Social Behavior</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:341-354.e12</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2016.08.071&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2016.08.071&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84990861958" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27667684/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.071&amp;pmid=27667684" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (10402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Durand, N.C. ∙ Shamim, M.S. ∙ Machol, I. ...</div><div><strong>Juicer Provides a One-Click System for Analyzing Loop-Resolution Hi-C Experiments</strong></div><div><em>Cell Syst.</em> 2016; <strong>3</strong>:95-98</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cels.2016.07.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27467249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2016.07.002&amp;pmid=27467249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">19.</a></div><div>Ernst, J. ∙ Kellis, M.</div><div><strong>Large-scale imputation of epigenomic datasets for systematic annotation of diverse human tissues</strong></div><div><em>Nat. Biotechnol.</em> 2015; <strong>33</strong>:364-376</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84926632357" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25690853/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3157&amp;pmid=25690853" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Fudenberg, G. ∙ Pollard, K.S.</div><div><strong>Chromatin features constrain structural variation across evolutionary timescales</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:2175-2180</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1808631116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85061144757" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30659153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1808631116&amp;pmid=30659153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Gittelman, R.M. ∙ Hun, E. ∙ Ay, F. ...</div><div><strong>Comprehensive identification and analysis of human accelerated regulatory DNA</strong></div><div><em>Genome Res.</em> 2015; <strong>25</strong>:1245-1255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.192591.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84937476478" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26104583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.192591.115&amp;pmid=26104583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Grant, C.E. ∙ Bailey, T.L. ∙ Noble, W.S.</div><div><strong>FIMO: scanning for occurrences of a given motif</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:1017-1018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr064" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-79953300078" target="_blank" aria-label="Open Scopus in new tab">Scopus (1382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21330290/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr064&amp;pmid=21330290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Gunhanlar, N. ∙ Shpak, G. ∙ van der Kroeg, M. ...</div><div><strong>A simplified protocol for differentiation of electrophysiologically mature neuronal networks from human induced pluripotent stem cells</strong></div><div><em>Mol. Psychiatry.</em> 2018; <strong>23</strong>:1336-1344</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/mp.2017.56" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85017511992" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28416807/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmp.2017.56&amp;pmid=28416807" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">24.</a></div><div>Hansen, K.D. ∙ Irizarry, R.A. ∙ Wu, Z.</div><div><strong>Removing technical variability in RNA-seq data using conditional quantile normalization</strong></div><div><em>Biostatistics.</em> 2012; <strong>13</strong>:204-216</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/biostatistics/kxr054" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84858068675" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22285995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbiostatistics%2Fkxr054&amp;pmid=22285995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Harris, L. ∙ Genovesi, L.A. ∙ Gronostajski, R.M. ...</div><div><strong>Nuclear factor one transcription factors: Divergent functions in developmental versus adult stem cell populations</strong></div><div><em>Dev. Dyn.</em> 2015; <strong>244</strong>:227-238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24182" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84924046828" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25156673/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24182&amp;pmid=25156673" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>He, H.H. ∙ Meyer, C.A. ∙ Hu, S.S. ...</div><div><strong>Refined DNase-seq protocol and data analysis reveals intrinsic bias in transcription factor footprint identification</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:73-78</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2762" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84894613651" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24317252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2762&amp;pmid=24317252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>He, Y. ∙ Luo, X. ∙ Zhou, B. ...</div><div><strong>Long-read assembly of the Chinese rhesus macaque genome and identification of ape-specific structural variants</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4233</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12174-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85072289655" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31530812/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12174-w&amp;pmid=31530812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">28.</a></div><div>Heckl, D. ∙ Kowalczyk, M.S. ∙ Yudovich, D. ...</div><div><strong>Generation of mouse models of myeloid malignancy with combinatorial genetic lesions using CRISPR-Cas9 genome editing</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:941-946</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2951" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84911422892" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24952903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2951&amp;pmid=24952903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29-1" title="View in article">29.</a></div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (4333)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Hevner, R.F. ∙ Shi, L. ∙ Justice, N. ...</div><div><strong>Tbr1 regulates differentiation of the preplate and layer 6</strong></div><div><em>Neuron.</em> 2001; <strong>29</strong>:353-366</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2FS0896-6273%2801%2900211-2&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2FS0896-6273%2801%2900211-2&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-17744398634" target="_blank" aria-label="Open Scopus in new tab">Scopus (612)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11239428/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2801%2900211-2&amp;pmid=11239428" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Hickey, S.L. ∙ Berto, S. ∙ Konopka, G.</div><div><strong>Chromatin Decondensation by FOXP2 Promotes Human Neuron Maturation and Expression of Neurodevelopmental Disease Genes</strong></div><div><em>Cell Rep.</em> 2019; <strong>27</strong>:1699-1711</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2019.04.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2019.04.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85064887595" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31067457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.04.044&amp;pmid=31067457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Hinrichs, A.S. ∙ Karolchik, D. ∙ Baertsch, R. ...</div><div><strong>The UCSC Genome Browser Database: update 2006</strong></div><div><em>Nucleic Acids Res.</em> 2006; <strong>34</strong>:D590-D598</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkj144" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33644874573" target="_blank" aria-label="Open Scopus in new tab">Scopus (559)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16381938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkj144&amp;pmid=16381938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Hoerder-Suabedissen, A. ∙ Molnár, Z.</div><div><strong>Development, evolution and pathology of neocortical subplate neurons</strong></div><div><em>Nat. Rev. Neurosci.</em> 2015; <strong>16</strong>:133-146</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn3915" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84923633876" target="_blank" aria-label="Open Scopus in new tab">Scopus (119)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25697157/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn3915&amp;pmid=25697157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">34.</a></div><div>Imakaev, M. ∙ Fudenberg, G. ∙ McCord, R.P. ...</div><div><strong>Iterative correction of Hi-C data reveals hallmarks of chromosome organization</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:999-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2148" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84866997011" target="_blank" aria-label="Open Scopus in new tab">Scopus (527)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22941365/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2148&amp;pmid=22941365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">35.</a></div><div>Judaš, M. ∙ Sedmak, G. ∙ Kostović, I.</div><div><strong>The significance of the subplate for evolution and developmental plasticity of the human brain</strong></div><div><em>Front. Hum. Neurosci.</em> 2013; <strong>7</strong>:423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fnhum.2013.00423" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84880141282" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23935575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffnhum.2013.00423&amp;pmid=23935575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">36.</a></div><div>Kaaij, L.J.T. ∙ Mohn, F. ∙ van der Weide, R.H. ...</div><div><strong>The ChAHP Complex Counteracts Chromatin Looping at CTCF Sites that Emerged from SINE Expansions in Mouse</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1437-1451.e14</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2019.08.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2019.08.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85071315791" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31491387/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.08.007&amp;pmid=31491387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Kanold, P.O. ∙ Kara, P. ∙ Reid, R.C. ...</div><div><strong>Role of subplate neurons in functional maturation of visual cortical columns</strong></div><div><em>Science.</em> 2003; <strong>301</strong>:521-525</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1084152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0041843767" target="_blank" aria-label="Open Scopus in new tab">Scopus (213)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12881571/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1084152&amp;pmid=12881571" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>Kapitonov, V. ∙ Jurka, J.</div><div><strong>The age of Alu subfamilies</strong></div><div><em>J.&nbsp;Mol. Evol.</em> 1996; <strong>42</strong>:59-65</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00163212" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0030045979" target="_blank" aria-label="Open Scopus in new tab">Scopus (239)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8576965/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00163212&amp;pmid=8576965" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39-1" title="View in article">39.</a></div><div>Khan, A. ∙ Fornes, O. ∙ Stigliani, A. ...</div><div><strong>JASPAR 2018: update of the open-access database of transcription factor binding profiles and its web framework</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D260-D266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1126" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85040936337" target="_blank" aria-label="Open Scopus in new tab">Scopus (568)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29140473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1126&amp;pmid=29140473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Kieffer-Kwon, K.R. ∙ Tang, Z. ∙ Mathe, E. ...</div><div><strong>Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1507-1520</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.11.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.11.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84894589713" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24360274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.039&amp;pmid=24360274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">41.</a></div><div>King, M.C. ∙ Wilson, A.C.</div><div><strong>Evolution at two levels in humans and chimpanzees</strong></div><div><em>Science.</em> 1975; <strong>188</strong>:107-116</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1090005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1090005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1090005&amp;pmid=1090005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Kostovic, I. ∙ Molliver, M.E.</div><div><strong>New interpretation of laminar development of cerebral cortex synaptogenesis in different layers of neopallium in human fetus</strong></div><div><em>Anat. Rec.</em> 1974; <strong>178</strong>:395</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.KostovicM.E.MolliverNew+interpretation+of+laminar+development+of+cerebral+cortex+synaptogenesis+in+different+layers+of+neopallium+in+human+fetusAnat.+Rec.1781974395" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Kostovic, I. ∙ Rakic, P.</div><div><strong>Developmental history of the transient subplate zone in the visual and somatosensory cortex of the macaque monkey and human brain</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 1990; <strong>297</strong>:441-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.902970309" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0025294391" target="_blank" aria-label="Open Scopus in new tab">Scopus (675)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2398142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.902970309&amp;pmid=2398142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>Kronenberg, Z.N. ∙ Fiddes, I.T. ∙ Gordon, D. ...</div><div><strong>High-resolution comparative analysis of great ape genomes</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar6343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar6343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85048568612" target="_blank" aria-label="Open Scopus in new tab">Scopus (103)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29880660/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar6343&amp;pmid=29880660" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">45.</a></div><div>Kundaje, A. ∙ Meuleman, W. ∙ Ernst, J. ..., Roadmap Epigenomics Consortium</div><div><strong>Integrative analysis of 111 reference human epigenomes</strong></div><div><em>Nature.</em> 2015; <strong>518</strong>:317-330</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14248" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84923362619" target="_blank" aria-label="Open Scopus in new tab">Scopus (2646)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25693563/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14248&amp;pmid=25693563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">46.</a></div><div>Lai, C.S. ∙ Fisher, S.E. ∙ Hurst, J.A. ...</div><div><strong>The SPCH1 region on human 7q31: genomic characterization of the critical interval and localization of translocations associated with speech and language disorder</strong></div><div><em>Am. J. Hum. Genet.</em> 2000; <strong>67</strong>:357-368</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1086%2F303011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1086%2F303011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0033865944" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10880297/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F303011&amp;pmid=10880297" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Lai, C.S. ∙ Fisher, S.E. ∙ Hurst, J.A. ...</div><div><strong>A forkhead-domain gene is mutated in a severe speech and language disorder</strong></div><div><em>Nature.</em> 2001; <strong>413</strong>:519-523</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/35097076" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-0035807360" target="_blank" aria-label="Open Scopus in new tab">Scopus (1273)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11586359/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F35097076&amp;pmid=11586359" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Langfelder, P. ∙ Horvath, S.</div><div><strong>WGCNA: an R package for weighted correlation network analysis</strong></div><div><em>BMC Bioinformatics.</em> 2008; <strong>9</strong>:559</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-9-559" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-60549111634" target="_blank" aria-label="Open Scopus in new tab">Scopus (5546)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19114008/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-9-559&amp;pmid=19114008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">49.</a></div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (17046)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">50.</a></div><div>Lawrence, M. ∙ Huber, W. ∙ Pagès, H. ...</div><div><strong>Software for computing and annotating genomic ranges</strong></div><div><em>PLoS Comput. Biol.</em> 2013; <strong>9</strong>:e1003118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84883368195" target="_blank" aria-label="Open Scopus in new tab">Scopus (1155)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23950696/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003118&amp;pmid=23950696" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">51.</a></div><div>Lazar, N.H. ∙ Nevonen, K.A. ∙ O’Connell, B. ...</div><div><strong>Epigenetic maintenance of topological domains in the highly rearranged gibbon genome</strong></div><div><em>Genome Res.</em> 2018; <strong>28</strong>:983-997</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.233874.117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85049247052" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29914971/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.233874.117&amp;pmid=29914971" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">52.</a></div><div>Lennon, P.A. ∙ Cooper, M.L. ∙ Peiffer, D.A. ...</div><div><strong>Deletion of 7q31.1 supports involvement of FOXP2 in language impairment: clinical report and review</strong></div><div><em>Am. J. Med. Genet. A.</em> 2007; <strong>143A</strong>:791-798</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajmg.a.31632" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-34147120769" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17330859/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajmg.a.31632&amp;pmid=17330859" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">53.</a></div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The Sequence Alignment/Map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (21853)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">54.</a></div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>The R package Rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:e47</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkz114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85066058200" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30783653/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkz114&amp;pmid=30783653" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55-1" title="View in article">55.</a></div><div>Lieberman-Aiden, E. ∙ van Berkum, N.L. ∙ Williams, L. ...</div><div><strong>Comprehensive mapping of long-range interactions reveals folding principles of the human genome</strong></div><div><em>Science.</em> 2009; <strong>326</strong>:289-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1181369" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-70349873824" target="_blank" aria-label="Open Scopus in new tab">Scopus (3659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19815776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1181369&amp;pmid=19815776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">56.</a></div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (16017)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">57.</a></div><div>Lui, J.H. ∙ Hansen, D.V. ∙ Kriegstein, A.R.</div><div><strong>Development and evolution of the human neocortex</strong></div><div><em>Cell.</em> 2011; <strong>146</strong>:18-36</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-79959924122" target="_blank" aria-label="Open Scopus in new tab">Scopus (677)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21729779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2011.06.030&amp;pmid=21729779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">58.</a></div><div>Lupiáñez, D.G. ∙ Kraft, K. ∙ Heinrich, V. ...</div><div><strong>Disruptions of topological chromatin domains cause pathogenic rewiring of gene-enhancer interactions</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1012-1025</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2015.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2015.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84930091577" target="_blank" aria-label="Open Scopus in new tab">Scopus (829)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25959774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.04.004&amp;pmid=25959774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">59.</a></div><div>Ma, J. ∙ Zhang, L. ∙ Suh, B.B. ...</div><div><strong>Reconstructing contiguous regions of an ancestral genome</strong></div><div><em>Genome Res.</em> 2006; <strong>16</strong>:1557-1565</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.5383506" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33750386912" target="_blank" aria-label="Open Scopus in new tab">Scopus (210)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16983148/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.5383506&amp;pmid=16983148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">60.</a></div><div>Marchetto, M.C. ∙ Hrvoj-Mihic, B. ∙ Kerman, B.E. ...</div><div><strong>Species-specific maturation profiles of human, chimpanzee and bonobo neural cells</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e37527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.37527" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85061147585" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30730291/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.37527&amp;pmid=30730291" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">61.</a></div><div>McLean, C.Y. ∙ Reno, P.L. ∙ Pollen, A.A. ...</div><div><strong>Human-specific loss of regulatory DNA and the evolution of human-specific traits</strong></div><div><em>Nature.</em> 2011; <strong>471</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09774" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-79952386187" target="_blank" aria-label="Open Scopus in new tab">Scopus (287)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21390129/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09774&amp;pmid=21390129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#figs4" title="View in article">62.</a></div><div>Miller, J.A. ∙ Ding, S.L. ∙ Sunkin, S.M. ...</div><div><strong>Transcriptional landscape of the prenatal human brain</strong></div><div><em>Nature.</em> 2014; <strong>508</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900303453" target="_blank" aria-label="Open Scopus in new tab">Scopus (550)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13185&amp;pmid=24695229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Mumbach, M.R. ∙ Satpathy, A.T. ∙ Boyle, E.A. ...</div><div><strong>Enhancer connectome in primary human cells identifies target genes of disease-associated DNA elements</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1602-1612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85032453347" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28945252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3963&amp;pmid=28945252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">64.</a></div><div>Nakashima, H. ∙ Tsujimura, K. ∙ Irie, K. ...</div><div><strong>Canonical TGF-β Signaling Negatively Regulates Neuronal Morphogenesis through TGIF/Smad Complex-Mediated CRMP2 Suppression</strong></div><div><em>J.&nbsp;Neurosci.</em> 2018; <strong>38</strong>:4791-4810</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.2423-17.2018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85050908774" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29695415/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.2423-17.2018&amp;pmid=29695415" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">65.</a></div><div>Nord, A.S. ∙ Blow, M.J. ∙ Attanasio, C. ...</div><div><strong>Rapid and pervasive changes in genome-wide enhancer usage during mammalian development</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1521-1531</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.11.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.11.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84894598129" target="_blank" aria-label="Open Scopus in new tab">Scopus (202)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24360275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.033&amp;pmid=24360275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">66.</a></div><div>Northcott, P.A. ∙ Lee, C. ∙ Zichner, T. ...</div><div><strong>Enhancer hijacking activates GFI1 family oncogenes in medulloblastoma</strong></div><div><em>Nature.</em> 2014; <strong>511</strong>:428-434</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13379" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84904816744" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25043047/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13379&amp;pmid=25043047" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">67.</a></div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (220)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">68.</a></div><div>Ozair, M.Z. ∙ Kirst, C. ∙ van den Berg, B.L. ...</div><div><strong>hPSC Modeling Reveals that Fate Selection of Cortical Deep Projection Neurons Occurs in the Subplate</strong></div><div><em>Cell Stem Cell.</em> 2018; <strong>23</strong>:60-73.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.stem.2018.05.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.stem.2018.05.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85048567581" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29937203/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2018.05.024&amp;pmid=29937203" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">69.</a></div><div>Parikshak, N.N. ∙ Luo, R. ∙ Zhang, A. ...</div><div><strong>Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1008-1021</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84889561601" target="_blank" aria-label="Open Scopus in new tab">Scopus (523)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.031&amp;pmid=24267887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">70.</a></div><div>Petrovic, J. ∙ Zhou, Y. ∙ Fasolino, M. ...</div><div><strong>Oncogenic Notch Promotes Long-Range Regulatory Interactions within Hyperconnected 3D Cliques</strong></div><div><em>Mol. Cell.</em> 2019; <strong>73</strong>:1174-1190.e12</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2019.01.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2019.01.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85063029087" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30745086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.01.006&amp;pmid=30745086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71-1" title="View in article">71.</a></div><div>Phanstiel, D.H. ∙ Van Bortle, K. ∙ Spacek, D. ...</div><div><strong>Static and Dynamic DNA Loops form AP-1-Bound Activation Hubs during Macrophage Development</strong></div><div><em>Mol. Cell.</em> 2017; <strong>67</strong>:1037-1048.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2017.08.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.molcel.2017.08.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85030242070" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28890333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2017.08.006&amp;pmid=28890333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#fig4" title="View in article">72.</a></div><div>Polioudakis, D. ∙ de la Torre-Ubieta, L. ∙ Langerman, J. ...</div><div><strong>A Single-Cell Transcriptomic Atlas of Human Neocortical Development during Mid-gestation</strong></div><div><em>Neuron.</em> 2019; <strong>103</strong>:785-801.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85070182242" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31303374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2019.06.011&amp;pmid=31303374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73-1" title="View in article">73.</a></div><div>Pollard, K.S. ∙ Salama, S.R. ∙ Lambert, N. ...</div><div><strong>An RNA gene expressed during cortical development evolved rapidly in humans</strong></div><div><em>Nature.</em> 2006; <strong>443</strong>:167-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33748675254" target="_blank" aria-label="Open Scopus in new tab">Scopus (638)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16915236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05113&amp;pmid=16915236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">74.</a></div><div>Poot, M.</div><div><strong>A candidate gene association study further corroborates involvement of contactin genes in autism</strong></div><div><em>Mol. Syndromol.</em> 2014; <strong>5</strong>:229-235</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1159/000362891" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84906874671" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25337070/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1159%2F000362891&amp;pmid=25337070" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">75.</a></div><div>Rajarajan, P. ∙ Borrman, T. ∙ Liao, W. ...</div><div><strong>Neuron-specific signatures in the chromosomal connectome associated with schizophrenia risk</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat4311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat4311" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85058384538" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat4311&amp;pmid=30545851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">76.</a></div><div>Ramírez, F. ∙ Dündar, F. ∙ Diehl, S. ...</div><div><strong>deepTools: a flexible platform for exploring deep-sequencing data</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:W187-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900988020" target="_blank" aria-label="Open Scopus in new tab">Scopus (707)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24799436/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku365&amp;pmid=24799436" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">77.</a></div><div>Rao, S.S. ∙ Huntley, M.H. ∙ Durand, N.C. ...</div><div><strong>A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:1665-1680</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2014.11.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84919949716" target="_blank" aria-label="Open Scopus in new tab">Scopus (2411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25497547/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.11.021&amp;pmid=25497547" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78-1" title="View in article">78.</a></div><div>Reilly, S.K. ∙ Yin, J. ∙ Ayoub, A.E. ...</div><div><strong>Evolutionary genomics. Evolutionary changes in promoter and enhancer activity during human corticogenesis</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1155-1159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84924561748" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25745175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260943&amp;pmid=25745175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">79.</a></div><div>Sanborn, A.L. ∙ Rao, S.S. ∙ Huang, S.C. ...</div><div><strong>Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E6456-E6465</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1518552112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84948403758" target="_blank" aria-label="Open Scopus in new tab">Scopus (653)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26499245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1518552112&amp;pmid=26499245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">80.</a></div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (18835)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81-1" title="View in article">81.</a></div><div>Schmitt, A.D. ∙ Hu, M. ∙ Jung, I. ...</div><div><strong>A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome</strong></div><div><em>Cell Rep.</em> 2016; <strong>17</strong>:2042-2059</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.celrep.2016.10.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85003815206" target="_blank" aria-label="Open Scopus in new tab">Scopus (272)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27851967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2016.10.061&amp;pmid=27851967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82-1" title="View in article">82.</a></div><div>Servant, N. ∙ Varoquaux, N. ∙ Lajoie, B.R. ...</div><div><strong>HiC-Pro: an optimized and flexible pipeline for Hi-C data processing</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:259</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0831-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84952861819" target="_blank" aria-label="Open Scopus in new tab">Scopus (432)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26619908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0831-x&amp;pmid=26619908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">83.</a></div><div>Sexton, T. ∙ Yaffe, E. ∙ Kenigsberg, E. ...</div><div><strong>Three-dimensional folding and functional organization principles of the Drosophila genome</strong></div><div><em>Cell.</em> 2012; <strong>148</strong>:458-472</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2012.01.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2012.01.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84856747483" target="_blank" aria-label="Open Scopus in new tab">Scopus (1046)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22265598/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2012.01.010&amp;pmid=22265598" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">84.</a></div><div>Sherwood, C.C. ∙ Gómez-Robles, A.</div><div><strong>Brain Plasticity and Human Evolution</strong></div><div><em>Annu. Rev. Anthropol.</em> 2017; <strong>46</strong>:399-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-anthro-102215-100009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85032203822" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-anthro-102215-100009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">85.</a></div><div>Shi, L. ∙ Su, B.</div><div><strong>A transgenic monkey model for the study of human brain evolution</strong></div><div><em>Zool. Res.</em> 2019; <strong>40</strong>:236-238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.24272/j.issn.2095-8137.2019.031" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85065115577" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31011135/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.24272%2Fj.issn.2095-8137.2019.031&amp;pmid=31011135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">86.</a></div><div>Shi, L. ∙ Luo, X. ∙ Jiang, J. ...</div><div><strong>Transgenic rhesus monkeys carrying the human MCPH1 gene copies show human-like neoteny of brain development</strong></div><div><em>Natl. Sci. Rev.</em> 2019; <strong>6</strong>:480-493</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nsr/nwz043" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85068564152" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnsr%2Fnwz043" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87-1" title="View in article">87.</a></div><div>Song, M. ∙ Yang, X. ∙ Ren, X. ...</div><div><strong>Mapping cis-regulatory chromatin contacts in neural cells links neuropsychiatric disorder risk variants to target genes</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:1252-1262</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0472-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85070866778" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31367015/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0472-1&amp;pmid=31367015" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">88.</a></div><div>Sousa, A.M.M. ∙ Meyer, K.A. ∙ Santpere, G. ...</div><div><strong>Evolution of the Human Nervous System Function, Structure, and Development</strong></div><div><em>Cell.</em> 2017; <strong>170</strong>:226-247</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85023612934" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28708995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.06.036&amp;pmid=28708995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">89.</a></div><div>Spielmann, M. ∙ Lupiáñez, D.G. ∙ Mundlos, S.</div><div><strong>Structural variation in the 3D genome</strong></div><div><em>Nat. Rev. Genet.</em> 2018; <strong>19</strong>:453-467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41576-018-0007-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85045905339" target="_blank" aria-label="Open Scopus in new tab">Scopus (137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29692413/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41576-018-0007-0&amp;pmid=29692413" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">90.</a></div><div>Stadhouders, R. ∙ Vidal, E. ∙ Serra, F. ...</div><div><strong>Transcription factors orchestrate dynamic interplay between genome topology and gene regulation during cell reprogramming</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:238-249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-017-0030-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85040677170" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29335546/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-017-0030-7&amp;pmid=29335546" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">91.</a></div><div>Stadhouders, R. ∙ Filion, G.J. ∙ Graf, T.</div><div><strong>Transcription factors and 3D genome conformation in cell-fate decisions</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:345-354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1182-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85065766396" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31092938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1182-7&amp;pmid=31092938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">92.</a></div><div>Stoeckli, E.T.</div><div><strong>Understanding axon guidance: are we nearly there yet?</strong></div><div><em>Development.</em> 2018; <strong>145</strong>:dev151415</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.151415" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85048007634" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29759980/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.151415&amp;pmid=29759980" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">93.</a></div><div>Toyoshima, M. ∙ Sakurai, K. ∙ Shimazaki, K. ...</div><div><strong>Deficiency of neural recognition molecule NB-2 affects the development of glutamatergic auditory pathways from the ventral cochlear nucleus to the superior olivary complex in mouse</strong></div><div><em>Dev. Biol.</em> 2009; <strong>336</strong>:192-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2009.09.043" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-70449529713" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19818338/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2009.09.043&amp;pmid=19818338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">94.</a></div><div>van Daalen, E. ∙ Kemner, C. ∙ Verbeek, N.E. ...</div><div><strong>Social Responsiveness Scale-aided analysis of the clinical impact of copy number variations in autism</strong></div><div><em>Neurogenetics.</em> 2011; <strong>12</strong>:315-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10048-011-0297-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84855277684" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21837366/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10048-011-0297-2&amp;pmid=21837366" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96-1" title="View in article">95.</a></div><div>Visel, A. ∙ Minovitsky, S. ∙ Dubchak, I. ...</div><div><strong>VISTA Enhancer Browser--a database of tissue-specific human enhancers</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:D88-D92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkl822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-33846112470" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17130149/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkl822&amp;pmid=17130149" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">96.</a></div><div>Wang, J. ∙ Vicente-García, C. ∙ Seruggia, D. ...</div><div><strong>MIR retrotransposon sequences provide insulators to the human genome</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4428-E4437</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507253112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26216945/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507253112&amp;pmid=26216945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">97.</a></div><div>Waterson, R.H. ∙ Lander, E.S. ∙ Wilson, R.K., Chimpanzee Sequencing and Analysis Consortium</div><div><strong>Initial sequence of the chimpanzee genome and comparison with the human genome</strong></div><div><em>Nature.</em> 2005; <strong>437</strong>:69-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature04072" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-24344500211" target="_blank" aria-label="Open Scopus in new tab">Scopus (1553)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16136131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature04072&amp;pmid=16136131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">98.</a></div><div>Wilsch-Bräuninger, M. ∙ Florio, M. ∙ Huttner, W.B.</div><div><strong>Neocortex expansion in development and evolution - from cell biology to single genes</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2016; <strong>39</strong>:122-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.conb.2016.05.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84973139075" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27258840/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.conb.2016.05.004&amp;pmid=27258840" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100-1" title="View in article">99.</a></div><div>Won, H. ∙ de la Torre-Ubieta, L. ∙ Stein, J.L. ...</div><div><strong>Chromosome conformation elucidates regulatory relationships in developing human brain</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:523-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84993993658" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19847&amp;pmid=27760116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">100.</a></div><div>Won, H. ∙ Huang, J. ∙ Opland, C.K. ...</div><div><strong>Human evolved regulatory elements modulate genes involved in cortical expansion and neurodevelopmental disease susceptibility</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2396</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10248-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85066601975" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31160561/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10248-3&amp;pmid=31160561" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">101.</a></div><div>Xu, C. ∙ Li, Q. ∙ Efimova, O. ...</div><div><strong>Human-specific features of spatial gene expression and regulation in eight brain regions</strong></div><div><em>Genome Res.</em> 2018; <strong>28</strong>:1097-1110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.231357.117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85050863625" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29898898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.231357.117&amp;pmid=29898898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-1" title="View in article">102.</a></div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (6096)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#figs6" title="View in article">103.</a></div><div>Zhang, Y. ∙ Li, T. ∙ Preissl, S. ...</div><div><strong>Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:1380-1388</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0479-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85071088740" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31427791/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0479-7&amp;pmid=31427791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105-1" title="View in article">104.</a></div><div>Zhu, Y. ∙ Li, M. ∙ Sousa, A.M.M. ...</div><div><strong>XSAnno: a framework for building ortholog models in cross-species transcriptome comparisons</strong></div><div><em>BMC Genomics.</em> 2014; <strong>15</strong>:343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-15-343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-84900840147" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24884593/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-15-343&amp;pmid=24884593" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106" title="View in article">105.</a></div><div>Zhu, Y. ∙ Sousa, A.M.M. ∙ Gao, T. ...</div><div><strong>Spatiotemporal transcriptomic divergence across human and macaque brain development</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat8077</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat8077" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.001&amp;key=2-s2.0-85058394301" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545855/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat8077&amp;pmid=30545855" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (17)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/d88660e3-6ede-4266-a3a4-54df6f240263/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/11eca952-9ee5-4700-833b-b9377dd70760/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/453b14d7-6b07-42bc-bdf3-38724984166f/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/ad155ed4-b163-4ffe-bac4-ad428d347417/main.assets/gr2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/086a18c8-0ad4-4a9f-ab71-52db4d2980b9/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/d1923315-8247-47c1-a795-f34dcf1b4bda/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/20e8b942-d3fd-4db5-9e08-6148213bf8cd/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/5b69bb46-19e3-4aaa-b0b5-c88b05799874/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/2eeaba82-4fae-4492-83de-03e9a73d2449/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/080c8f03-22f7-4ba6-8ea2-f4a51810c29e/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/ea81b589-29cb-41d9-b885-ef84078ef47b/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/ae1b8300-df49-47ef-98d1-39f72b8e3fbf/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/fb32724b-e950-4930-a719-ed641fc619c1/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/e622b673-55e5-4e2d-ab9d-172e013f6a3e/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#undfig2" aria-label="Go to "><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/67581f3b-000f-48ea-b87b-5e7d0ed7391e/main.assets/fx2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#undfig3" aria-label="Go to "><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/de5aa33c-94e1-4072-b390-888043777d72/main.assets/fx3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#undfig4" aria-label="Go to "><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/asset/c6d0a589-7e6d-4a5d-a04f-53b1b84f3a24/main.assets/fx4.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00043%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Barticle%3Aarticle%3Apii%5C%3AS0092867421000015%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">83</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">165</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">2</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">39</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2021.01.001&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (4)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/attachment/340c8d17-bfdd-4c86-bc3c-cbced9ce9d0c/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (17.29 KB)</span></a><div class="core-description">Table S1. Hi-C library stats of rhesus macaque fetal brain, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/attachment/814d397f-302d-4714-ada5-014e4fad35dd/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (225.95 KB)</span></a><div class="core-description">Table S2. Primers of 3C and DNA-FISH, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/attachment/6fa51f28-0444-4b3e-8175-6e5c808bede0/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (518.46 KB)</span></a><div class="core-description">Table S3. TAD boundary positions and genes around TADs, related to Figure&nbsp;2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.01.001/attachment/e9bb96b8-383c-4caf-b836-48f557e754b6/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (7.97 MB)</span></a><div class="core-description">Table S4. Loop positions and loop target genes, related to Figure&nbsp;3</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421000015" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-genomics/fulltext/S2666-979X(24)00332-X" target="_blank" title="Single-nucleus multi-omics analyses reveal cellular and molecular innovations in the anterior cingulate cortex during primate evolution">Single-nucleus multi-omics analyses reveal cellular and molecular innovations in the anterior cingulate cortex during primate evolution</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yuan et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Genomics</i></span><span id="item_date">December 3, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/developmental-cell/fulltext/S1534-5807(24)00605-1" target="_blank" title="Comparative single-cell multiome identifies evolutionary changes in neural progenitor cells during primate brain development">Comparative single-cell multiome identifies evolutionary changes in neural progenitor cells during primate brain development</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Developmental Cell</i></span><span id="item_date">October 30, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/genetics/fulltext/S0168-9525(24)00265-8" target="_blank" title="Advancing evolutionary medicine with complete primate genomes and advanced biotechnologies">Advancing evolutionary medicine with complete primate genomes and advanced biotechnologies</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ma et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Genetics</i></span><span id="item_date">December 2, 2024</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-stem-cell/fulltext/S1934-5909(24)00458-2" target="_blank" title="Molecular signature of primate astrocytes reveals pathways and regulatory changes contributing to human brain evolution">Molecular signature of primate astrocytes reveals pathways and regulatory changes contributing to human brain evolution</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ciuba et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Stem Cell</i></span><span id="item_date">February 4, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-genomics/fulltext/S2666-979X(24)00173-3" target="_blank" title="Evolutionary and biomedical implications of sex differences in the primate brain transcriptome">Evolutionary and biomedical implications of sex differences in the primate brain transcriptome</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>DeCasien et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Genomics</i></span><span id="item_date">June 27, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)12649-7" target="_blank" title="An evolutionary game analysis of incentive of industrial parks, government support and enterprise innovation willingness in China">An evolutionary game analysis of incentive of industrial parks, government support and enterprise innovation willingness in China</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhen et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 22, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)12012-9" target="_blank" title="Does financialization inhibit enterprise innovation? Analysis of innovation behavior of Chinese enterprises based on evolutionary game">Does financialization inhibit enterprise innovation? Analysis of innovation behavior of Chinese enterprises based on evolutionary game</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Tang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 9, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(24)01433-7" target="_blank" title="Genome dynamics across the evolutionary transition to endosymbiosis">Genome dynamics across the evolutionary transition to endosymbiosis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Siozios et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">November 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)04720-5" target="_blank" title="Evolutionary game study of crowdsourcing open innovation synergy mechanism">Evolutionary game study of crowdsourcing open innovation synergy mechanism</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jiang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">June 21, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)00683-7" target="_blank" title="Eco-evolutionary experience and behavioral innovation in interactions with non-native species">Eco-evolutionary experience and behavioral innovation in interactions with non-native species</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ruland et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">March 8, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/ajhg/fulltext/S0002-9297(24)00374-4" target="_blank" title="Genomes and epigenomes of matched normal and tumor breast tissue reveal diverse evolutionary trajectories and tumor-host interactions">Genomes and epigenomes of matched normal and tumor breast tissue reveal diverse evolutionary trajectories and tumor-host interactions</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhu et al.</li></ul></div><div class="linked-article__details"><span><i>The American Journal of Human Genetics</i></span><span id="item_date">November 3, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/plant-communications/fulltext/S2590-3462(25)00104-X" target="_blank" title="The genome of giant waterlily provides insights into the origin of angiosperms, leaf gigantism, and stamen function innovation">The genome of giant waterlily provides insights into the origin of angiosperms, leaf gigantism, and stamen function innovation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wen et al.</li></ul></div><div class="linked-article__details"><span><i>Plant Communications</i></span><span id="item_date">April 16, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/plant-communications/fulltext/S2590-3462(24)00652-7" target="_blank" title="Solanaceae pan-genomes reveal extensive fractionation and functional innovation of duplicated genes">Solanaceae pan-genomes reveal extensive fractionation and functional innovation of duplicated genes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Plant Communications</i></span><span id="item_date">December 23, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/developmental-cell/fulltext/S1534-5807(24)00632-4" target="_blank" title="Evolutionary innovations in germline biology of placental mammals identified by transcriptomics of first-wave spermatogenesis in opossum">Evolutionary innovations in germline biology of placental mammals identified by transcriptomics of first-wave spermatogenesis in opossum</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Marshall et al.</li></ul></div><div class="linked-article__details"><span><i>Developmental Cell</i></span><span id="item_date">November 12, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(24)00524-4" target="_blank" title="Camelus knoblochi genome reveals the complex evolutionary history of Old World camels">Camelus knoblochi genome reveals the complex evolutionary history of Old World camels</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yuan et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">May 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/plant-communications/fulltext/S2590-3462(25)00108-7" target="_blank" title="An integrative 3D-genome database for plants">An integrative 3D-genome database for plants</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yang et al.</li></ul></div><div class="linked-article__details"><span><i>Plant Communications</i></span><span id="item_date">April 29, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(22)01731-4" target="_blank" title="Principles of 3D chromosome folding and evolutionary genome reshuffling in mammals">Principles of 3D chromosome folding and evolutionary genome reshuffling in mammals</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Álvarez-González et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">December 20, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(23)01909-0" target="_blank" title="Live-bearing cockroach genome reveals convergent evolutionary mechanisms linked to viviparity in insects and beyond">Live-bearing cockroach genome reveals convergent evolutionary mechanisms linked to viviparity in insects and beyond</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Fouks et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">September 7, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)01800-5" target="_blank" title="Chromosome-scale genome assembly reveals insights into the evolution and ecology of the harmful algal bloom species Phaeocystis globosa Scherffel">Chromosome-scale genome assembly reveals insights into the evolution and ecology of the harmful algal bloom species Phaeocystis globosa Scherffel</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chen et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">July 24, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/genetics/fulltext/S0168-9525(25)00009-5" target="_blank" title="Transposon–host arms race: a saga of genome evolution">Transposon–host arms race: a saga of genome evolution</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Iwasaki et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Genetics</i></span><span id="item_date">February 19, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)00223-7" target="_blank" title="Evolutionary analysis of gene ages across TADs associates chromatin topology with whole-genome duplications">Evolutionary analysis of gene ages across TADs associates chromatin topology with whole-genome duplications</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>James et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">March 21, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/plant-communications/fulltext/S2590-3462(24)00527-3" target="_blank" title="Genome architecture of the allotetraploid wild grass Aegilops ventricosa reveals its evolutionary history and contributions to wheat improvement">Genome architecture of the allotetraploid wild grass Aegilops ventricosa reveals its evolutionary history and contributions to wheat improvement</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Plant Communications</i></span><span id="item_date">September 9, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)02481-2" target="_blank" title="Study on the evolution of collaborative innovation in marine economy considering the participation of financial institutions and two types of cooperation">Study on the evolution of collaborative innovation in marine economy considering the participation of financial institutions and two types of cooperation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ma et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">February 16, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)03102-5" target="_blank" title="Single-cell RNA sequencing reveals distinct transcriptomic profiles and evolutionary patterns in lung cancer brain metastasis">Single-cell RNA sequencing reveals distinct transcriptomic profiles and evolutionary patterns in lung cancer brain metastasis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">February 23, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-systems/fulltext/S2405-4712(24)00402-2" target="_blank" title="Subspecies phylogeny in the human gut revealed by co-evolutionary constraints across the bacterial kingdom">Subspecies phylogeny in the human gut revealed by co-evolutionary constraints across the bacterial kingdom</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Doran et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Systems</i></span><span id="item_date">January 17, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/parasitology/fulltext/S1471-4922(25)00102-3" target="_blank" title="Innovation through instability? Genome (dis)organisation in Trypanosoma cruzi">Innovation through instability? Genome (dis)organisation in Trypanosoma cruzi</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Dickson et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Parasitology</i></span><span id="item_date">May 20, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/plant-communications/fulltext/S2590-3462(24)00598-4" target="_blank" title="A telomere-to-telomere genome assembly of Salix cheilophila reveals its evolutionary signatures for environmental adaptation">A telomere-to-telomere genome assembly of Salix cheilophila reveals its evolutionary signatures for environmental adaptation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gao et al.</li></ul></div><div class="linked-article__details"><span><i>Plant Communications</i></span><span id="item_date">November 4, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/neuron/fulltext/S0896-6273(25)00313-7" target="_blank" title="Machine learning identification of enhancers in the rhesus macaque genome">Machine learning identification of enhancers in the rhesus macaque genome</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>He et al.</li></ul></div><div class="linked-article__details"><span><i>Neuron</i></span><span id="item_date">May 21, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(25)00417-9" target="_blank" title="Tracing the evolutionary history of the CCR5delta32 deletion via ancient and modern genomes">Tracing the evolutionary history of the CCR5delta32 deletion via ancient and modern genomes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ravn et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">May 5, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(24)00899-7" target="_blank" title="Bronze Age cheese reveals human-Lactobacillus interactions over evolutionary history">Bronze Age cheese reveals human-Lactobacillus interactions over evolutionary history</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">September 25, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-methods/fulltext/S2667-2375(24)00325-4" target="_blank" title="A core genome MLST scheme for Borrelia burgdorferi sensu lato improves insights into the evolutionary history of the species complex">A core genome MLST scheme for Borrelia burgdorferi sensu lato improves insights into the evolutionary history of the species complex</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hepner et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Methods</i></span><span id="item_date">December 18, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)09562-8" target="_blank" title="The evolution of the ambidextrous innovation synergy strategy of new entrants from the perspective of key core technology monopoly">The evolution of the ambidextrous innovation synergy strategy of new entrants from the perspective of key core technology monopoly</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yao et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">June 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(24)00121-1" target="_blank" title="Structurally divergent and recurrently mutated regions of primate genomes">Structurally divergent and recurrently mutated regions of primate genomes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Mao et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">February 29, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(24)01445-3" target="_blank" title="Hidden evolutionary constraints dictate the retention of coronavirus accessory genes">Hidden evolutionary constraints dictate the retention of coronavirus accessory genes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Goldstein et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">November 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/plant-communications/fulltext/S2590-3462(23)00275-4" target="_blank" title="Genomic evolution and insights into agronomic trait innovations of Sesamum species">Genomic evolution and insights into agronomic trait innovations of Sesamum species</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Miao et al.</li></ul></div><div class="linked-article__details"><span><i>Plant Communications</i></span><span id="item_date">October 4, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/neuron/fulltext/S0896-6273(25)00308-3" target="_blank" title="Multimodal analyses reveal genes driving electrophysiological maturation of neurons in the primate prefrontal cortex">Multimodal analyses reveal genes driving electrophysiological maturation of neurons in the primate prefrontal cortex</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gao et al.</li></ul></div><div class="linked-article__details"><span><i>Neuron</i></span><span id="item_date">May 20, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-genomics/fulltext/S2666-979X(24)00129-0" target="_blank" title="Lineage-specific splicing regulation of MAPT gene in the primate brain">Lineage-specific splicing regulation of MAPT gene in the primate brain</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Recinos et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Genomics</i></span><span id="item_date">May 20, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)00905-7" target="_blank" title="Kinetochore and ionomic adaptation to whole-genome duplication in Cochlearia shows evolutionary convergence in three autopolyploids">Kinetochore and ionomic adaptation to whole-genome duplication in Cochlearia shows evolutionary convergence in three autopolyploids</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Bray et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">August 7, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)17594-9" target="_blank" title="Analysis of fetal fraction in non-invasive prenatal testing with low-depth whole genome sequencing">Analysis of fetal fraction in non-invasive prenatal testing with low-depth whole genome sequencing</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Xie et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">December 28, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(23)00679-7" target="_blank" title="Single-cell spatial transcriptome reveals cell-type organization in the macaque cortex">Single-cell spatial transcriptome reveals cell-type organization in the macaque cortex</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chen et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">July 12, 2023</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.01.001","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">3D Genome of macaque fetal brain reveals evolutionary innovations during primate corticogenesis</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.01.001/asset/d88660e3-6ede-4266-a3a4-54df6f240263/main.assets/gr1.jpg" height="940" width="689" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/62b05598-f378-4155-9620-5c62693cc416/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.01.001/asset/11eca952-9ee5-4700-833b-b9377dd70760/main.assets/figs1.jpg" height="814" width="762" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/2ce50164-ddae-4474-ab25-f2919f66a364/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.01.001/asset/453b14d7-6b07-42bc-bdf3-38724984166f/main.assets/figs2.jpg" height="952" width="752" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/ed8096e9-1ee3-4987-936e-448908764cd8/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.01.001/asset/ad155ed4-b163-4ffe-bac4-ad428d347417/main.assets/gr2.jpg" height="842" width="724" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/768e2b13-1f1a-4d41-8047-53807744e56c/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.01.001/asset/086a18c8-0ad4-4a9f-ab71-52db4d2980b9/main.assets/figs3.jpg" height="957" width="689" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/7246f472-9470-4c11-99ff-00c78221201a/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.01.001/asset/d1923315-8247-47c1-a795-f34dcf1b4bda/main.assets/gr3.jpg" height="731" width="755" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/983d5d9b-929d-4acb-85f1-3eea948d882f/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.01.001/asset/20e8b942-d3fd-4db5-9e08-6148213bf8cd/main.assets/figs4.jpg" height="952" width="732" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/bc1618e7-17e1-4ebd-afe7-2dd4e557dada/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.01.001/asset/5b69bb46-19e3-4aaa-b0b5-c88b05799874/main.assets/gr4.jpg" height="899" width="763" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/22750d77-77d3-4794-9d3f-58ff9b837629/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.01.001/asset/2eeaba82-4fae-4492-83de-03e9a73d2449/main.assets/gr5.jpg" height="532" width="762" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/20650e04-a163-46ab-817b-ca69748fd6c6/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.01.001/asset/080c8f03-22f7-4ba6-8ea2-f4a51810c29e/main.assets/gr6.jpg" height="881" width="763" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/abe40e4c-50de-48e5-ba7b-61366502ecd7/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.01.001/asset/ea81b589-29cb-41d9-b885-ef84078ef47b/main.assets/figs5.jpg" height="588" width="670" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/a3f7dcac-c96d-4e68-a03a-e597b83ab782/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.01.001/asset/ae1b8300-df49-47ef-98d1-39f72b8e3fbf/main.assets/gr7.jpg" height="703" width="739" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/d8dbc7e3-01f2-4d0a-ba4f-35a6be9958f3/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.01.001/asset/fb32724b-e950-4930-a719-ed641fc619c1/main.assets/figs6.jpg" height="633" width="678" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/fb0e10c8-2ae3-469e-ad66-2be60b5fa17b/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.01.001/asset/e622b673-55e5-4e2d-ab9d-172e013f6a3e/main.assets/figs7.jpg" height="795" width="762" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/cd4ee25d-0dc9-4f76-b6c8-23783d6d75f4/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="fx2" alt="Figure undfig2" src="/cms/10.1016/j.cell.2021.01.001/asset/67581f3b-000f-48ea-b87b-5e7d0ed7391e/main.assets/fx2.jpg" height="118" width="239" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/22394018-867d-4761-9176-9ca0977be255/main.assets/fx2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="fx3" alt="Figure undfig3" src="/cms/10.1016/j.cell.2021.01.001/asset/de5aa33c-94e1-4072-b390-888043777d72/main.assets/fx3.jpg" height="166" width="488" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/f109f5f0-a786-4d8e-bacd-0beb3beb20b1/main.assets/fx3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="fx4" alt="Figure undfig4" src="/cms/10.1016/j.cell.2021.01.001/asset/c6d0a589-7e6d-4a5d-a04f-53b1b84f3a24/main.assets/fx4.jpg" height="211" width="561" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/dbd7339b-f38e-4f8d-b5c5-79cd50458ef1/main.assets/fx4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.01.001/asset/d88660e3-6ede-4266-a3a4-54df6f240263/main.assets/gr1.jpg" height="940" width="689" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/62b05598-f378-4155-9620-5c62693cc416/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.01.001/asset/11eca952-9ee5-4700-833b-b9377dd70760/main.assets/figs1.jpg" height="814" width="762" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/2ce50164-ddae-4474-ab25-f2919f66a364/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.01.001/asset/453b14d7-6b07-42bc-bdf3-38724984166f/main.assets/figs2.jpg" height="952" width="752" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/ed8096e9-1ee3-4987-936e-448908764cd8/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.01.001/asset/ad155ed4-b163-4ffe-bac4-ad428d347417/main.assets/gr2.jpg" height="842" width="724" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/768e2b13-1f1a-4d41-8047-53807744e56c/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.01.001/asset/086a18c8-0ad4-4a9f-ab71-52db4d2980b9/main.assets/figs3.jpg" height="957" width="689" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/7246f472-9470-4c11-99ff-00c78221201a/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.01.001/asset/d1923315-8247-47c1-a795-f34dcf1b4bda/main.assets/gr3.jpg" height="731" width="755" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/983d5d9b-929d-4acb-85f1-3eea948d882f/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.01.001/asset/20e8b942-d3fd-4db5-9e08-6148213bf8cd/main.assets/figs4.jpg" height="952" width="732" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/bc1618e7-17e1-4ebd-afe7-2dd4e557dada/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.01.001/asset/5b69bb46-19e3-4aaa-b0b5-c88b05799874/main.assets/gr4.jpg" height="899" width="763" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/22750d77-77d3-4794-9d3f-58ff9b837629/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.01.001/asset/2eeaba82-4fae-4492-83de-03e9a73d2449/main.assets/gr5.jpg" height="532" width="762" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/20650e04-a163-46ab-817b-ca69748fd6c6/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.01.001/asset/080c8f03-22f7-4ba6-8ea2-f4a51810c29e/main.assets/gr6.jpg" height="881" width="763" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/abe40e4c-50de-48e5-ba7b-61366502ecd7/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.01.001/asset/ea81b589-29cb-41d9-b885-ef84078ef47b/main.assets/figs5.jpg" height="588" width="670" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/a3f7dcac-c96d-4e68-a03a-e597b83ab782/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.01.001/asset/ae1b8300-df49-47ef-98d1-39f72b8e3fbf/main.assets/gr7.jpg" height="703" width="739" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/d8dbc7e3-01f2-4d0a-ba4f-35a6be9958f3/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.01.001/asset/fb32724b-e950-4930-a719-ed641fc619c1/main.assets/figs6.jpg" height="633" width="678" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/fb0e10c8-2ae3-469e-ad66-2be60b5fa17b/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.01.001/asset/e622b673-55e5-4e2d-ab9d-172e013f6a3e/main.assets/figs7.jpg" height="795" width="762" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/cd4ee25d-0dc9-4f76-b6c8-23783d6d75f4/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure</div><div class="thumbnail"><div class="thumbnail__image"><img id="fx2" alt="Figure undfig2" src="/cms/10.1016/j.cell.2021.01.001/asset/67581f3b-000f-48ea-b87b-5e7d0ed7391e/main.assets/fx2.jpg" height="118" width="239" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/22394018-867d-4761-9176-9ca0977be255/main.assets/fx2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure</div><div class="thumbnail"><div class="thumbnail__image"><img id="fx3" alt="Figure undfig3" src="/cms/10.1016/j.cell.2021.01.001/asset/de5aa33c-94e1-4072-b390-888043777d72/main.assets/fx3.jpg" height="166" width="488" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/f109f5f0-a786-4d8e-bacd-0beb3beb20b1/main.assets/fx3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure</div><div class="thumbnail"><div class="thumbnail__image"><img id="fx4" alt="Figure undfig4" src="/cms/10.1016/j.cell.2021.01.001/asset/c6d0a589-7e6d-4a5d-a04f-53b1b84f3a24/main.assets/fx4.jpg" height="211" width="561" data-viewer-src="/cms/10.1016/j.cell.2021.01.001/asset/dbd7339b-f38e-4f8d-b5c5-79cd50458ef1/main.assets/fx4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00001-5';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00001-5';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749589214689","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"184","issueNumber":"3","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421000015","coverDate":"202102","type":"XOCS-JOURNAL:SCOPE-FULL","title":"3D Genome of macaque fetal brain reveals evolutionary innovations during primate corticogenesis","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dbcc088f6d4ee9',t:'MTc0OTU4OTIxNC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dbcc088f6d4ee9&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.01.001&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>