<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="3c6b09a2-4411-45a3-ad3f-f301667f499f" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;issue:issue:pii\:S0092867420X00110;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;article:article:pii\:S0092867421004360;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="LanCLs add glutathione to dehydroamino acids generated at phosphorylated sites in the proteome">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)00436-0">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/227ad26b-7b3f-4e31-aadf-ce00b078ad33/fx1.jpg">
    


        <meta property="og:description" content="LanCL enzymes respond to sites of protein damage arising from conversion of phosphorylated
Ser and Thr residues into highly reactive non-canonical amino acids, catalyzing irreversible
capping of the residues with glutathione.">
    











        <meta name="twitter:title" content="LanCLs add glutathione to dehydroamino acids generated at phosphorylated sites in the proteome">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/227ad26b-7b3f-4e31-aadf-ce00b078ad33/fx1.jpg">
    


        <meta name="twitter:description" content="LanCL enzymes respond to sites of protein damage arising from conversion of phosphorylated
Ser and Thr residues into highly reactive non-canonical amino acids, catalyzing irreversible
capping of the residues with glutathione.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)00436-0">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>LanCLs add glutathione to dehydroamino acids generated at phosphorylated sites in the proteome: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="LanCLs add glutathione to dehydroamino acids generated at phosphorylated sites in the proteome"> <meta name="citation_date" content="2021/05/13"> <meta name="citation_online_date" content="2021/04/30"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="10"> <meta name="citation_firstpage" content="2680"> <meta name="citation_lastpage" content="2695.e26"> <meta name="citation_pmid" content="33932340"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421004360"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421004360"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421004360"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.04.001"> <meta name="citation_pmid" content="33932340"> <meta name="citation_author" content="Kuan-Yu Lai"><meta name="citation_author_institution" content="Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author" content="Sébastien R.G. Galan"><meta name="citation_author_institution" content="Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK"><meta name="citation_author" content="Yibo Zeng"><meta name="citation_author_institution" content="Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK"><meta name="citation_author_institution" content="UK Catalysis Hub, Research Complex at Harwell, Rutherford Appleton Laboratory, Harwell, Oxford OX11 0FA, UK"><meta name="citation_author_institution" content="The Rosalind Franklin Institute, Oxfordshire OX11 0FA, UK"><meta name="citation_author" content="Tianhui Hina Zhou"><meta name="citation_author_institution" content="Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author" content="Chang He"><meta name="citation_author_institution" content="Department of Chemistry and Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author" content="Ritu Raj"><meta name="citation_author_institution" content="Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK"><meta name="citation_author" content="Jitka Riedl"><meta name="citation_author_institution" content="Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK"><meta name="citation_author" content="Shi Liu"><meta name="citation_author_institution" content="Department of Chemistry and Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author" content="K. Phin Chooi"><meta name="citation_author_institution" content="Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK"><meta name="citation_author" content="Neha Garg"><meta name="citation_author_institution" content="Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author" content="Min Zeng"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author" content="Lyn H. Jones"><meta name="citation_author_institution" content="Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, MA 02115, USA"><meta name="citation_author" content="Graham J. Hutchings"><meta name="citation_author_institution" content="UK Catalysis Hub, Research Complex at Harwell, Rutherford Appleton Laboratory, Harwell, Oxford OX11 0FA, UK"><meta name="citation_author_institution" content="Cardiff Catalysis Institute, School of Chemistry, Cardiff University, Cardiff CF10 3AT, UK"><meta name="citation_author" content="Shabaz Mohammed"><meta name="citation_author_institution" content="Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK"><meta name="citation_author_institution" content="The Rosalind Franklin Institute, Oxfordshire OX11 0FA, UK"><meta name="citation_author" content="Satish K. Nair"><meta name="citation_author_institution" content="Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author" content="Jie Chen"><meta name="citation_author_email" content="jiechen@life.illinois.edu"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author" content="Benjamin G. Davis"><meta name="citation_author_email" content="ben.davis@chem.ox.ac.uk"><meta name="citation_author_institution" content="Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK"><meta name="citation_author_institution" content="The Rosalind Franklin Institute, Oxfordshire OX11 0FA, UK"><meta name="citation_author" content="Wilfred A. van der Donk"><meta name="citation_author_email" content="vddonk@illinois.edu"><meta name="citation_author_institution" content="Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author_institution" content="Department of Chemistry and Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_keywords" content="C-glutathionylation; dehydroalanine; dehydrobutyrine; LanCL; phosphoThr lyase; eliminylome; protein damage; MEK1; lanthionine"><meta name="citation_reference" content="citation_title=Identification of the sites in MAP kinase kinase-1 phosphorylated by p74raf-1; citation_author=D.R. Alessi; citation_author=Y. Saito; citation_author=D.G. Campbell; citation_author=P. Cohen; citation_author=G. Sithanandam; citation_author=U. Rapp; citation_author=A. Ashworth; citation_author=C.J. Marshall; citation_author=S. Cowley; citation_journal_title=EMBO J.; citation_volume=13; citation_firstpage=1610; citation_lastpage=1619; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=An injected bacterial effector targets chromatin access for transcription factor NF-kappaB to alter transcription of host genes involved in immune responses; citation_author=L. Arbibe; citation_author=D.W. Kim; citation_author=E. Batsche; citation_author=T. Pedron; citation_author=B. Mateescu; citation_author=C. Muchardt; citation_author=C. Parsot; citation_author=P.J. Sansonetti; citation_journal_title=Nat. Immunol.; citation_volume=8; citation_firstpage=47; citation_lastpage=56; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Characterization of p40/GPR69A as a peripheral membrane protein related to the lantibiotic synthetase component C; citation_author=H. Bauer; citation_author=H. Mayer; citation_author=A. Marchler-Bauer; citation_author=U. Salzer; citation_author=R. Prohaska; citation_journal_title=Biochem. Biophys. Res. Commun.; citation_volume=275; citation_firstpage=69; citation_lastpage=74; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=A high-throughput assay for phosphoprotein-specific phosphatase activity in cellular extracts; citation_author=A.K. Bose; citation_author=K.A. Janes; citation_journal_title=Mol. Cell. Proteomics; citation_volume=12; citation_firstpage=797; citation_lastpage=806; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Eliminylation: a post-translational modification catalyzed by phosphothreonine lyases; citation_author=D.F. Brennan; citation_author=D. Barford; citation_journal_title=Trends Biochem. Sci.; citation_volume=34; citation_firstpage=108; citation_lastpage=114; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Human glutathione transferase A4-4 crystal structures and mutagenesis reveal the basis of high catalytic efficiency with toxic lipid peroxidation products; citation_author=C.M. Bruns; citation_author=I. Hubatsch; citation_author=M. Ridderström; citation_author=B. Mannervik; citation_author=J.A. Tainer; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=288; citation_firstpage=427; citation_lastpage=439; citation_publication_date=1999; "><meta name="citation_reference" content="citation_author=J.M. Chalker; citation_author=S.B. Gunnoo; citation_author=O. Boutureira; citation_author=S.C. Gerstberger; citation_author=M. Fernndez-Gonzlez; citation_author=G.J.L. Bernardes; citation_author=L. Griffin; citation_author=H. Hailu; citation_author=C.J. Schofield; citation_author=B.G. Davis; citation_journal_title=Chem. Sci. (Camb.); citation_volume=2; citation_firstpage=1666; citation_lastpage=1676; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Selectivity within a family of bacterial phosphothreonine lyases; citation_author=K.A. Chambers; citation_author=N.S. Abularrage; citation_author=R.A. Scheck; citation_journal_title=Biochemistry; citation_volume=57; citation_firstpage=3790; citation_lastpage=3796; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Biosynthesis and mode of action of lantibiotics; citation_author=C. Chatterjee; citation_author=M. Paul; citation_author=L. Xie; citation_author=W.A. van der Donk; citation_journal_title=Chem. Rev.; citation_volume=105; citation_firstpage=633; citation_lastpage=684; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Irreversible inactivation of glutathione peroxidase 1 and reversible inactivation of peroxiredoxin II by H2O2 in red blood cells; citation_author=C.S. Cho; citation_author=S. Lee; citation_author=G.T. Lee; citation_author=H.A. Woo; citation_author=E.J. Choi; citation_author=S.G. Rhee; citation_journal_title=Antioxid. Redox Signal.; citation_volume=12; citation_firstpage=1235; citation_lastpage=1246; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Identification of lanthionine synthase C-like protein-1 as a prominent glutathione binding protein expressed in the mammalian central nervous system; citation_author=C.H. Chung; citation_author=B.T. Kurien; citation_author=P. Mehta; citation_author=M. Mhatre; citation_author=S. Mou; citation_author=Q.N. Pye; citation_author=C. Stewart; citation_author=M. West; citation_author=K.S. Williamson; citation_author=J. Post; citation_author=et al.; citation_journal_title=Biochemistry; citation_volume=46; citation_firstpage=3262; citation_lastpage=3269; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Reversible and irreversible protein glutathionylation: biological and clinical aspects; citation_author=A.J. Cooper; citation_author=J.T. Pinto; citation_author=P.S. Callery; citation_journal_title=Expert Opin. Drug Metab. Toxicol.; citation_volume=7; citation_firstpage=891; citation_lastpage=910; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification; citation_author=J. Cox; citation_author=M. Mann; citation_journal_title=Nat. Biotechnol.; citation_volume=26; citation_firstpage=1367; citation_lastpage=1372; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Andromeda: a peptide search engine integrated into the MaxQuant environment; citation_author=J. Cox; citation_author=N. Neuhauser; citation_author=A. Michalski; citation_author=R.A. Scheltema; citation_author=J.V. Olsen; citation_author=M. Mann; citation_journal_title=J.&nbsp;Proteome Res.; citation_volume=10; citation_firstpage=1794; citation_lastpage=1805; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Salt effect accelerates site-selective cysteine bioconjugation; citation_author=P. Dai; citation_author=C. Zhang; citation_author=M. Welborn; citation_author=J.J. Shepherd; citation_author=T. Zhu; citation_author=T. Van Voorhis; citation_author=B.L. Pentelute; citation_journal_title=ACS Cent. Sci.; citation_volume=2; citation_firstpage=637; citation_lastpage=646; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Protein S-glutathionylation: a regulatory device from bacteria to humans; citation_author=I. Dalle-Donne; citation_author=R. Rossi; citation_author=G. Colombo; citation_author=D. Giustarini; citation_author=A. Milzani; citation_journal_title=Trends Biochem. Sci.; citation_volume=34; citation_firstpage=85; citation_lastpage=96; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The enterococcal cytolysin synthetase has an unanticipated lipid kinase fold; citation_author=S.H. Dong; citation_author=W. Tang; citation_author=T. Lukk; citation_author=Y. Yu; citation_author=S.K. Nair; citation_author=W.A. van der Donk; citation_journal_title=eLife; citation_volume=4; citation_firstpage=e07607; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Bacteriophage targeting of gut bacterium attenuates alcoholic liver disease; citation_author=Y. Duan; citation_author=C. Llorente; citation_author=S. Lang; citation_author=K. Brandl; citation_author=H. Chu; citation_author=L. Jiang; citation_author=R.C. White; citation_author=T.H. Clarke; citation_author=K. Nguyen; citation_author=M. Torralba; citation_author=et al.; citation_journal_title=Nature; citation_volume=575; citation_firstpage=505; citation_lastpage=511; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A chromosomal region 7p11.2 transcript map: its development and application to the study of EGFR amplicons in glioblastoma; citation_author=G.D. Eley; citation_author=J.L. Reiter; citation_author=A. Pandita; citation_author=S. Park; citation_author=R.B. Jenkins; citation_author=N.J. Maihle; citation_author=C.D. James; citation_journal_title=Neuro-oncol.; citation_volume=4; citation_firstpage=86; citation_lastpage=94; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Crystal structures of MEK1 binary and ternary complexes with nucleotides and inhibitors; citation_author=T.O. Fischmann; citation_author=C.K. Smith; citation_author=T.W. Mayhood; citation_author=J.E. Myers; citation_author=P. Reichert; citation_author=A. Mannarino; citation_author=D. Carr; citation_author=H. Zhu; citation_author=J. Wong; citation_author=R.S. Yang; citation_author=et al.; citation_journal_title=Biochemistry; citation_volume=48; citation_firstpage=2661; citation_lastpage=2674; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=A promoter-level mammalian expression atlas; citation_author=A.R. Forrest; citation_author=H. Kawaji; citation_author=M. Rehli; citation_author=J.K. Baillie; citation_author=M.J. de Hoon; citation_author=V. Haberle; citation_author=T. Lassmann; citation_author=I.V. Kulakovskiy; citation_author=M. Lizio; citation_author=M. Itoh; citation_author=FANTOM Consortium and the RIKEN PMI and CLST (DGT); citation_author=et al.; citation_journal_title=Nature; citation_volume=507; citation_firstpage=462; citation_lastpage=470; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Chemistry, biochemistry, nutrition, and microbiology of lysinoalanine, lanthionine, and histidinoalanine in food and other proteins; citation_author=M. Friedman; citation_journal_title=J.&nbsp;Agric. Food Chem.; citation_volume=47; citation_firstpage=1295; citation_lastpage=1319; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Initial quantitative proteomic map of 28 mouse tissues using the SILAC mouse; citation_author=T. Geiger; citation_author=A. Velic; citation_author=B. Macek; citation_author=E. Lundberg; citation_author=C. Kampf; citation_author=N. Nagaraj; citation_author=M. Uhlen; citation_author=J. Cox; citation_author=M. Mann; citation_journal_title=Mol. Cell. Proteomics; citation_volume=12; citation_firstpage=1709; citation_lastpage=1722; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Negative regulation of MAPKK by phosphorylation of a conserved serine residue equivalent to Ser212 of MEK1; citation_author=K. Gopalbhai; citation_author=G. Jansen; citation_author=G. Beauregard; citation_author=M. Whiteway; citation_author=F. Dumas; citation_author=C. Wu; citation_author=S. Meloche; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=278; citation_firstpage=8118; citation_lastpage=8125; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Causes and consequences of cysteine S-glutathionylation; citation_author=C.L. Grek; citation_author=J. Zhang; citation_author=Y. Manevich; citation_author=D.M. Townsend; citation_author=K.D. Tew; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=288; citation_firstpage=26497; citation_lastpage=26504; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=LanCL proteins are not Involved in Lanthionine Synthesis in Mammals; citation_author=C. He; citation_author=M. Zeng; citation_author=D. Dutta; citation_author=T.H. Koh; citation_author=J. Chen; citation_author=W.A. van der Donk; citation_journal_title=Sci. Rep.; citation_volume=7; citation_firstpage=40980; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=PhosphoSitePlus, 2014: mutations, PTMs and recalibrations; citation_author=P.V. Hornbeck; citation_author=B. Zhang; citation_author=B. Murray; citation_author=J.M. Kornhauser; citation_author=V. Latham; citation_author=E. Skrzypek; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=D512; citation_lastpage=D520; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Developmental and activity-dependent expression of LanCL1 confers antioxidant activity required for neuronal survival; citation_author=C. Huang; citation_author=M. Chen; citation_author=D. Pang; citation_author=D. Bi; citation_author=Y. Zou; citation_author=X. Xia; citation_author=W. Yang; citation_author=L. Luo; citation_author=R. Deng; citation_author=H. Tan; citation_author=et al.; citation_journal_title=Dev. Cell; citation_volume=30; citation_firstpage=479; citation_lastpage=487; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Human glutathione transferase A4-4: an alpha class enzyme with high catalytic efficiency in the conjugation of 4-hydroxynonenal and other genotoxic products of lipid peroxidation; citation_author=I. Hubatsch; citation_author=M. Ridderström; citation_author=B. Mannervik; citation_journal_title=Biochem. J.; citation_volume=330; citation_firstpage=175; citation_lastpage=179; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=NACCESS; citation_author=S. Hubbard; citation_author=J. Thornton; citation_publication_date=1993; citation_publisher=University College London; "><meta name="citation_reference" content="citation_title=A tissue-specific atlas of mouse protein phosphorylation and expression; citation_author=E.L. Huttlin; citation_author=M.P. Jedrychowski; citation_author=J.E. Elias; citation_author=T. Goswami; citation_author=R. Rad; citation_author=S.A. Beausoleil; citation_author=J. Villén; citation_author=W. Haas; citation_author=M.E. Sowa; citation_author=S.P. Gygi; citation_journal_title=Cell; citation_volume=143; citation_firstpage=1174; citation_lastpage=1189; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The FANTOM5 collection, a data series underpinning mammalian transcriptome atlases in diverse cell types; citation_author=H. Kawaji; citation_author=T. Kasukawa; citation_author=A. Forrest; citation_author=P. Carninci; citation_author=Y. Hayashizaki; citation_journal_title=Sci. Data; citation_volume=4; citation_firstpage=170113; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Reconstitution of mitogen-activated protein kinase phosphorylation cascades in bacteria. Efficient synthesis of active protein kinases; citation_author=A. Khokhlatchev; citation_author=S. Xu; citation_author=J. English; citation_author=P. Wu; citation_author=E. Schaefer; citation_author=M.H. Cobb; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=272; citation_firstpage=11057; citation_lastpage=11062; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=A draft map of the human proteome; citation_author=M.S. Kim; citation_author=S.M. Pinto; citation_author=D. Getnet; citation_author=R.S. Nirujogi; citation_author=S.S. Manda; citation_author=R. Chaerkady; citation_author=A.K. Madugundu; citation_author=D.S. Kelkar; citation_author=R. Isserlin; citation_author=S. Jain; citation_author=et al.; citation_journal_title=Nature; citation_volume=509; citation_firstpage=575; citation_lastpage=581; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Chemical activation of MEK1--a redox trigger for evaluating the effects of phosphorylation; citation_author=T.L. Lamoureaux; citation_author=D.H. Lee; citation_journal_title=Chem. Commun. (Camb.); citation_volume=47; citation_firstpage=8623; citation_lastpage=8625; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The interpretation of protein structures: estimation of static accessibility; citation_author=B. Lee; citation_author=F.M. Richards; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=55; citation_firstpage=379; citation_lastpage=400; citation_publication_date=1971; "><meta name="citation_reference" content="citation_title=Identification of essential catalytic residues of the cyclase NisC involved in the biosynthesis of nisin; citation_author=B. Li; citation_author=W.A. van der Donk; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=282; citation_firstpage=21169; citation_lastpage=21175; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Structure and mechanism of the lantibiotic cyclase involved in nisin biosynthesis; citation_author=B. Li; citation_author=J.P. Yu; citation_author=J.S. Brunzelle; citation_author=G.N. Moll; citation_author=W.A. van der Donk; citation_author=S.K. Nair; citation_journal_title=Science; citation_volume=311; citation_firstpage=1464; citation_lastpage=1467; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=The phosphothreonine lyase activity of a bacterial type III effector family; citation_author=H. Li; citation_author=H. Xu; citation_author=Y. Zhou; citation_author=J. Zhang; citation_author=C. Long; citation_author=S. Li; citation_author=S. Chen; citation_author=J.M. Zhou; citation_author=F. Shao; citation_journal_title=Science; citation_volume=315; citation_firstpage=1000; citation_lastpage=1003; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Catalytic promiscuity in the biosynthesis of cyclic peptide secondary metabolites in planktonic marine cyanobacteria; citation_author=B. Li; citation_author=D. Sher; citation_author=L. Kelly; citation_author=Y. Shi; citation_author=K. Huang; citation_author=P.J. Knerr; citation_author=I. Joewono; citation_author=D. Rusch; citation_author=S.W. Chisholm; citation_author=W.A. van der Donk; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=107; citation_firstpage=10430; citation_lastpage=10435; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Glutathionylation of lens proteins through the formation of thioether bond; citation_author=M. Linetsky; citation_author=R.D. LeGrand; citation_journal_title=Mol. Cell. Biochem.; citation_volume=272; citation_firstpage=133; citation_lastpage=144; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Dehydroalanine crosslinks in human lens; citation_author=M. Linetsky; citation_author=J.M. Hill; citation_author=R.D. LeGrand; citation_author=F. Hu; citation_journal_title=Exp. Eye Res.; citation_volume=79; citation_firstpage=499; citation_lastpage=512; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Solid supported chemical syntheses of both components of the lantibiotic lacticin 3147; citation_author=W. Liu; citation_author=A.S.H. Chan; citation_author=H. Liu; citation_author=S.A. Cochrane; citation_author=J.C. Vederas; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=133; citation_firstpage=14216; citation_lastpage=14219; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Developing irreversible inhibitors of the protein kinase cysteinome; citation_author=Q. Liu; citation_author=Y. Sabnis; citation_author=Z. Zhao; citation_author=T. Zhang; citation_author=S.J. Buhrlage; citation_author=L.H. Jones; citation_author=N.S. Gray; citation_journal_title=Chem. Biol.; citation_volume=20; citation_firstpage=146; citation_lastpage=159; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Loss of selenium from selenoproteins: conversion of selenocysteine to dehydroalanine in&nbsp;vitro; citation_author=S. Ma; citation_author=R.M. Caprioli; citation_author=K.E. Hill; citation_author=R.F. Burk; citation_journal_title=J.&nbsp;Am. Soc. Mass Spectrom.; citation_volume=14; citation_firstpage=593; citation_lastpage=600; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Isolation, molecular characterization, and tissue-specific expression of a novel putative G protein-coupled receptor; citation_author=H. Mayer; citation_author=U. Salzer; citation_author=J. Breuss; citation_author=S. Ziegler; citation_author=A. Marchler-Bauer; citation_author=R. Prohaska; citation_journal_title=Biochim. Biophys. Acta; citation_volume=1395; citation_firstpage=301; citation_lastpage=308; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Characterization of rat LANCL1, a novel member of the lanthionine synthetase C-like protein family, highly expressed in testis and brain; citation_author=H. Mayer; citation_author=H. Bauer; citation_author=J. Breuss; citation_author=S. Ziegler; citation_author=R. Prohaska; citation_journal_title=Gene; citation_volume=269; citation_firstpage=73; citation_lastpage=80; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Molecular cloning, characterization, and tissue-specific expression of human LANCL2, a novel member of the LanC-like protein family; citation_author=H. Mayer; citation_author=M. Pongratz; citation_author=R. Prohaska; citation_journal_title=DNA Seq.; citation_volume=12; citation_firstpage=161; citation_lastpage=166; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Conversion of isoaspartyl peptides to normal peptides: implications for the cellular repair of damaged proteins; citation_author=P.N. McFadden; citation_author=S. Clarke; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=84; citation_firstpage=2595; citation_lastpage=2599; citation_publication_date=1987; "><meta name="citation_reference" content="citation_title=Optimized chemical proteomics assay for kinase inhibitor profiling; citation_author=G. Médard; citation_author=F. Pachl; citation_author=B. Ruprecht; citation_author=S. Klaeger; citation_author=S. Heinzlmeir; citation_author=D. Helm; citation_author=H. Qiao; citation_author=X. Ku; citation_author=M. Wilhelm; citation_author=T. Kuehne; citation_author=et al.; citation_journal_title=J.&nbsp;Proteome Res.; citation_volume=14; citation_firstpage=1574; citation_lastpage=1586; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Glutathione; citation_author=A. Meister; citation_author=M.E. Anderson; citation_journal_title=Annu. Rev. Biochem.; citation_volume=52; citation_firstpage=711; citation_lastpage=760; citation_publication_date=1983; "><meta name="citation_reference" content="citation_title=Phenotyping breast cancer cell lines EM-G3, HCC1937, MCF7 and MDA-MB-231 using 2-D electrophoresis and affinity chromatography for glutathione-binding proteins; citation_author=J. Mladkova; citation_author=M. Sanda; citation_author=E. Matouskova; citation_author=I. Selicharova; citation_journal_title=BMC Cancer; citation_volume=10; citation_firstpage=449; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Chemical generation and modification of peptides containing multiple dehydroalanines; citation_author=P.M. Morrison; citation_author=P.J. Foley; citation_author=S.L. Warriner; citation_author=M.E. Webb; citation_journal_title=Chem. Commun. (Camb.); citation_volume=51; citation_firstpage=13470; citation_lastpage=13473; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability; citation_author=F.H. Niesen; citation_author=H. Berglund; citation_author=M. Vedadi; citation_journal_title=Nat. Protoc.; citation_volume=2; citation_firstpage=2212; citation_lastpage=2221; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Proteomics: the move to mixtures; citation_author=J. Peng; citation_author=S.P. Gygi; citation_journal_title=J.&nbsp;Mass Spectrom.; citation_volume=36; citation_firstpage=1083; citation_lastpage=1091; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Mechanistic understanding of lanthipeptide biosynthetic enzymes; citation_author=L.M. Repka; citation_author=J.R. Chekan; citation_author=S.K. Nair; citation_author=W.A. van der Donk; citation_journal_title=Chem. Rev.; citation_volume=117; citation_firstpage=5457; citation_lastpage=5520; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Blot-based detection of dehydroalanine-containing glutathione peroxidase with the use of biotin-conjugated cysteamine; citation_author=S.G. Rhee; citation_author=C.S. Cho; citation_journal_title=Methods Enzymol.; citation_volume=474; citation_firstpage=23; citation_lastpage=34; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Regulation and properties of extracellular signal-regulated protein kinases 1 and 2 in&nbsp;vitro; citation_author=D.J. Robbins; citation_author=E. Zhen; citation_author=H. Owaki; citation_author=C.A. Vanderbilt; citation_author=D. Ebert; citation_author=T.D. Geppert; citation_author=M.H. Cobb; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=268; citation_firstpage=5097; citation_lastpage=5106; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Systems-level overview of host protein phosphorylation during Shigella flexneri infection revealed by phosphoproteomics; citation_author=C. Schmutz; citation_author=E. Ahrné; citation_author=C.A. Kasper; citation_author=T. Tschon; citation_author=I. Sorg; citation_author=R.F. Dreier; citation_author=A. Schmidt; citation_author=C. Arrieumerlou; citation_journal_title=Mol. Cell. Proteomics; citation_volume=12; citation_firstpage=2952; citation_lastpage=2968; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Structural characterization of four prochlorosins: a novel class of lantipeptides produced by planktonic marine cyanobacteria; citation_author=W. Tang; citation_author=W.A. van der Donk; citation_journal_title=Biochemistry; citation_volume=51; citation_firstpage=4271; citation_lastpage=4279; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The sequence of the enterococcal cytolysin imparts unusual lanthionine stereochemistry; citation_author=W. Tang; citation_author=W.A. van der Donk; citation_journal_title=Nat. Chem. Biol.; citation_volume=9; citation_firstpage=157; citation_lastpage=159; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=CylA is a sequence-specific protease involved in toxin biosynthesis; citation_author=W. Tang; citation_author=S.C. Bobeica; citation_author=L. Wang; citation_author=W.A. van der Donk; citation_journal_title=J.&nbsp;Ind. Microbiol. Biotechnol.; citation_volume=46; citation_firstpage=537; citation_lastpage=549; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=UniProt: a worldwide hub of protein knowledge; citation_author=The Uniprot consortium; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_firstpage=D506; citation_lastpage=D515; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A price to pay for relaxed substrate specificity: a comparative kinetic analysis of the class II lanthipeptide synthetases ProcM and HalM2; citation_author=C.J. Thibodeaux; citation_author=T. Ha; citation_author=W.A. van der Donk; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=136; citation_firstpage=17513; citation_lastpage=17529; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Expanding the diversity of chemical protein modification allows post-translational mimicry; citation_author=S.I. van Kasteren; citation_author=H.B. Kramer; citation_author=H.H. Jensen; citation_author=S.J. Campbell; citation_author=J. Kirkpatrick; citation_author=N.J. Oldham; citation_author=D.C. Anthony; citation_author=B.G. Davis; citation_journal_title=Nature; citation_volume=446; citation_firstpage=1105; citation_lastpage=1109; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Application of text-mining for updating protein post-translational modification annotation in UniProtKB; citation_author=A.L. Veuthey; citation_author=A. Bridge; citation_author=J. Gobeill; citation_author=P. Ruch; citation_author=J.R. McEntyre; citation_author=L. Bougueleret; citation_author=I. Xenarios; citation_journal_title=BMC Bioinformatics; citation_volume=14; citation_firstpage=104; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Dissecting therapeutic resistance to RAF inhibition in melanoma by tumor genomic profiling; citation_author=N. Wagle; citation_author=C. Emery; citation_author=M.F. Berger; citation_author=M.J. Davis; citation_author=A. Sawyer; citation_author=P. Pochanard; citation_author=S.M. Kehoe; citation_author=C.M. Johannessen; citation_author=L.E. Macconaill; citation_author=W.C. Hahn; citation_author=et al.; citation_journal_title=J.&nbsp;Clin. Oncol.; citation_volume=29; citation_firstpage=3085; citation_lastpage=3096; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Knockout of SOD1 promotes conversion of selenocysteine to dehydroalanine in murine hepatic GPX1 protein; citation_author=S.K. Wang; citation_author=J.D. Weaver; citation_author=S. Zhang; citation_author=X.G. Lei; citation_journal_title=Free Radic. Biol. Med.; citation_volume=51; citation_firstpage=197; citation_lastpage=204; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Human protein aging: modification and crosslinking through dehydroalanine and dehydrobutyrine intermediates; citation_author=Z. Wang; citation_author=B. Lyons; citation_author=R.J. Truscott; citation_author=K.L. Schey; citation_journal_title=Aging Cell; citation_volume=13; citation_firstpage=226; citation_lastpage=234; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Selective metal-site-guided arylation of proteins; citation_author=J. Willwacher; citation_author=R. Raj; citation_author=S. Mohammed; citation_author=B.G. Davis; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=138; citation_firstpage=8678; citation_lastpage=8681; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Human cytosolic glutathione transferases: structure, function, and drug discovery; citation_author=B. Wu; citation_author=D. Dong; citation_journal_title=Trends Pharmacol. Sci.; citation_volume=33; citation_firstpage=656; citation_lastpage=668; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The N-terminal ERK-binding site of MEK1 is required for efficient feedback phosphorylation by&nbsp;ERK2 in&nbsp;vitro and ERK activation in&nbsp;vivo; citation_author=B.e. Xu; citation_author=J.L. Wilsbacher; citation_author=T. Collisson; citation_author=M.H. Cobb; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=274; citation_firstpage=34029; citation_lastpage=34035; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Michael-type cyclizations in lantibiotic biosynthesis are reversible; citation_author=X. Yang; citation_author=W.A. van der Donk; citation_journal_title=ACS Chem. Biol.; citation_volume=10; citation_firstpage=1234; citation_lastpage=1238; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The expanding landscape of the thiol redox proteome; citation_author=J. Yang; citation_author=K.S. Carroll; citation_author=D.C. Liebler; citation_journal_title=Mol. Cell. Proteomics; citation_volume=15; citation_firstpage=1; citation_lastpage=11; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A lanthipeptide library used to identify a protein-protein interaction inhibitor; citation_author=X. Yang; citation_author=K.R. Lennard; citation_author=C. He; citation_author=M.C. Walker; citation_author=A.T. Ball; citation_author=C. Doigneaux; citation_author=A. Tavassoli; citation_author=W.A. van der Donk; citation_journal_title=Nat. Chem. Biol.; citation_volume=14; citation_firstpage=375; citation_lastpage=380; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Lanthionine synthetase C-like protein 2 (LanCL2) is a novel regulator of Akt; citation_author=M. Zeng; citation_author=W.A. van der Donk; citation_author=J. Chen; citation_journal_title=Mol. Biol. Cell; citation_volume=25; citation_firstpage=3954; citation_lastpage=3961; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Structure of human lanthionine synthetase C-like protein 1 and its interaction with Eps8 and glutathione; citation_author=W. Zhang; citation_author=L. Wang; citation_author=Y. Liu; citation_author=J. Xu; citation_author=G. Zhu; citation_author=H. Cang; citation_author=X. Li; citation_author=M. Bartlam; citation_author=K. Hensley; citation_author=G. Li; citation_author=et al.; citation_journal_title=Genes Dev.; citation_volume=23; citation_firstpage=1387; citation_lastpage=1392; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=S-sulfhydration of MEK1 leads to PARP-1 activation and DNA damage repair; citation_author=K. Zhao; citation_author=Y. Ju; citation_author=S. Li; citation_author=Z. Altaany; citation_author=R. Wang; citation_author=G. Yang; citation_journal_title=EMBO Rep.; citation_volume=15; citation_firstpage=792; citation_lastpage=800; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Activation of MEK family kinases requires&nbsp;phosphorylation of two conserved Ser/Thr residues; citation_author=C.F. Zheng; citation_author=K.L. Guan; citation_journal_title=EMBO J.; citation_volume=13; citation_firstpage=1123; citation_lastpage=1131; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=Structural insights into the enzymatic mechanism of the pathogenic MAPK phosphothreonine lyase; citation_author=Y. Zhu; citation_author=H. Li; citation_author=C. Long; citation_author=L. Hu; citation_author=H. Xu; citation_author=L. Liu; citation_author=S. Chen; citation_author=D.C. Wang; citation_author=F. Shao; citation_journal_title=Mol. Cell; citation_volume=28; citation_firstpage=899; citation_lastpage=913; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=OspF and OspC1 are Shigella flexneri type III secretion system effectors that are required for postinvasion aspects of virulence; citation_author=D.V. Zurawski; citation_author=C. Mitsuhata; citation_author=K.L. Mumy; citation_author=B.A. McCormick; citation_author=A.T. Maurelli; citation_journal_title=Infect. Immun.; citation_volume=74; citation_firstpage=5964; citation_lastpage=5976; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Shigella flexneri type III secretion system effectors OspB and OspF target the nucleus to downregulate the host inflammatory response via interactions with retinoblastoma protein; citation_author=D.V. Zurawski; citation_author=K.L. Mumy; citation_author=C.S. Faherty; citation_author=B.A. McCormick; citation_author=A.T. Maurelli; citation_journal_title=Mol. Microbiol.; citation_volume=71; citation_firstpage=350; citation_lastpage=368; citation_publication_date=2009; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Enzyme-mediated damage repair or mitigation, while common for nucleic acids, is rare for proteins. Examples of protein damage are elimination of phosphorylated Ser/Thr to dehydroalanine/dehydrobutyrine (Dha/Dhb) in pathogenesis and aging. Bacterial LanC enzymes use Dha/Dhb to form carbon-sulfur linkages in antimicrobial peptides, but the functions of eukaryotic LanC-like (LanCL) counterparts are unknown. We show that LanCLs catalyze the addition of glutathione to Dha/Dhb in proteins, driving irreversible <i>C</i>-glutathionylation. Chemo-enzymatic methods were developed to site-selectively incorporate Dha/Dhb at phospho-regulated sites in kinases. In human MAPK-MEK1, such &quot;elimination damage&quot; generated aberrantly activated kinases, which were deactivated by LanCL-mediated <i>C</i>-glutathionylation. Surveys of endogenous proteins bearing damage from elimination (the eliminylome) also suggest it is a source of electrophilic reactivity. LanCLs thus remove these reactive electrophiles and their potentially dysregulatory effects from the proteome. As knockout of LanCL in mice can result in premature death, repair of this kind of protein damage appears important physiologically.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7263&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)00436-0&amp;be=3571&amp;fe=7218&amp;dc=6917&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749589983266,%22n%22:0,%22f%22:3014,%22dn%22:3014,%22dne%22:3014,%22c%22:3014,%22ce%22:3014,%22rq%22:3015,%22rp%22:3520,%22rpe%22:4211,%22dl%22:3525,%22di%22:4497,%22ds%22:6916,%22de%22:6928,%22dc%22:7217,%22l%22:7217,%22le%22:7252%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900436-0" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00436-0"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)00436-0"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900436-0" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-702383" aria-expanded="false" aria-controls="login-links-702383" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-702383" id="login-links--702383" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900436-0" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900436-0" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900436-0" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-702383" data-opens="#links-dialog-702383" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-702383" aria-modal="true"><button data-closes="#links-dialog-702383" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900436-0" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900436-0" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900436-0" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.04.001">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)00436-0" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0011-0"><span>Volume 184</span>, <span>Issue 10</span></a></span><span class="meta-panel__pages"><span>p2680-2695.e26</span></span><span class="meta-panel__onlineDate">May 13, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=10&amp;pii=S0092-8674%2820%29X0011-0" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">LanCLs add glutathione to dehydroamino acids generated at phosphorylated sites in the proteome</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Kuan-Yu</span> <span property="familyName">Lai</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kuan-Yu</span> <span property="familyName">Lai</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn2"><div id="ntpara0015" role="paragraph">Present address: Cytiva, 2780 N. 200 W., North Logan, UT 84341, USA</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lai/Kuan-Yu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">10</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Sébastien R.G.</span> <span property="familyName">Galan</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sébastien R.G.</span> <span property="familyName">Galan</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">9</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn3"><div id="ntpara0020" role="paragraph">Present address: Evotec (UK), 114 Innovation Drive, Milton Park, Abingdon, Oxfordshire OX14 4RZ, UK</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Galan/S%C3%A9bastien+R+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">9</a>,<a role="doc-noteref">11</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Yibo</span> <span property="familyName">Zeng</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yibo</span> <span property="familyName">Zeng</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">9</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">UK Catalysis Hub, Research Complex at Harwell, Rutherford Appleton Laboratory, Harwell, Oxford OX11 0FA, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">The Rosalind Franklin Institute, Oxfordshire OX11 0FA, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Zeng/Yibo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a role="doc-noteref">9</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Tianhui Hina</span> <span property="familyName">Zhou</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tianhui Hina</span> <span property="familyName">Zhou</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zhou/Tianhui+Hina" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Chang</span> <span property="familyName">He</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Chang</span> <span property="familyName">He</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn4"><div id="ntpara0025" role="paragraph">Kumquat Biosciences, 10770 Wateridge Cir Unit 120, San Diego, CA 92121, USA</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/He/Chang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Ritu</span> <span property="familyName">Raj</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ritu</span> <span property="familyName">Raj</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Raj/Ritu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Jitka</span> <span property="familyName">Riedl</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jitka</span> <span property="familyName">Riedl</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn6"><div id="ntpara0035" role="paragraph">Present address: Spirogen, a member of the AstraZeneca group, QMB Innovation Centre, 42 New Road, London E1 2AX, UK</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Riedl/Jitka" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">13</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Shi</span> <span property="familyName">Liu</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shi</span> <span property="familyName">Liu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/Shi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">K. Phin</span> <span property="familyName">Chooi</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">K. Phin</span> <span property="familyName">Chooi</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn6"><div id="ntpara0035" role="paragraph">Present address: Spirogen, a member of the AstraZeneca group, QMB Innovation Centre, 42 New Road, London E1 2AX, UK</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Chooi/K+Phin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">13</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Neha</span> <span property="familyName">Garg</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Neha</span> <span property="familyName">Garg</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn7"><div id="ntpara0040" role="paragraph">Present address: School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, GA 30332-2000, USA</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Garg/Neha" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Min</span> <span property="familyName">Zeng</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Min</span> <span property="familyName">Zeng</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn8"><div id="ntpara0045" role="paragraph">Present address: Facebook, 1 Hacker Way, Menlo Park, CA 94025, USA</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zeng/Min" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Lyn H.</span> <span property="familyName">Jones</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lyn H.</span> <span property="familyName">Jones</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jones/Lyn+H" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Graham J.</span> <span property="familyName">Hutchings</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Graham J.</span> <span property="familyName">Hutchings</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">UK Catalysis Hub, Research Complex at Harwell, Rutherford Appleton Laboratory, Harwell, Oxford OX11 0FA, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">Cardiff Catalysis Institute, School of Chemistry, Cardiff University, Cardiff CF10 3AT, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Hutchings/Graham+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Shabaz</span> <span property="familyName">Mohammed</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shabaz</span> <span property="familyName">Mohammed</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">The Rosalind Franklin Institute, Oxfordshire OX11 0FA, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Mohammed/Shabaz" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Satish K.</span> <span property="familyName">Nair</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Satish K.</span> <span property="familyName">Nair</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nair/Satish+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Jie</span> <span property="familyName">Chen</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jie</span> <span property="familyName">Chen</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:jiechen@life.illinois.edu" property="email" title="Send email to Jie Chen">jiechen@life.illinois.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chen/Jie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup> <a id="eadd0010" href="mailto:jiechen@life.illinois.edu" property="email" title="Send email to Jie Chen">jiechen@life.illinois.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Benjamin G.</span> <span property="familyName">Davis</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Benjamin G.</span> <span property="familyName">Davis</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:ben.davis@chem.ox.ac.uk" property="email" title="Send email to Benjamin G. Davis">ben.davis@chem.ox.ac.uk</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">The Rosalind Franklin Institute, Oxfordshire OX11 0FA, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Davis/Benjamin+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup> <a id="eadd0015" href="mailto:ben.davis@chem.ox.ac.uk" property="email" title="Send email to Benjamin G. Davis">ben.davis@chem.ox.ac.uk</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Wilfred A.</span> <span property="familyName">van der Donk</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Wilfred A.</span> <span property="familyName">van der Donk</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:vddonk@illinois.edu" property="email" title="Send email to Wilfred A. van der Donk">vddonk@illinois.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">16</div><div id="fn9"><div id="ntpara0050" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/van+der+Donk/Wilfred+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">16</a></sup> <a id="eadd0020" href="mailto:vddonk@illinois.edu" property="email" title="Send email to Wilfred A. van der Donk">vddonk@illinois.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Chemistry Research Laboratory, Department of Chemistry, University of Oxford, Mansfield, Oxford OX1 3TA, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Chemistry and Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Dana-Farber Cancer Institute, 360 Longwood Avenue, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Cell and Developmental Biology, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">UK Catalysis Hub, Research Complex at Harwell, Rutherford Appleton Laboratory, Harwell, Oxford OX11 0FA, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Cardiff Catalysis Institute, School of Chemistry, Cardiff University, Cardiff CF10 3AT, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">The Rosalind Franklin Institute, Oxfordshire OX11 0FA, UK</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">9</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn2"><div id="ntpara0015" role="paragraph">Present address: Cytiva, 2780 N. 200 W., North Logan, UT 84341, USA</div></div></div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn3"><div id="ntpara0020" role="paragraph">Present address: Evotec (UK), 114 Innovation Drive, Milton Park, Abingdon, Oxfordshire OX14 4RZ, UK</div></div></div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn4"><div id="ntpara0025" role="paragraph">Kumquat Biosciences, 10770 Wateridge Cir Unit 120, San Diego, CA 92121, USA</div></div></div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn6"><div id="ntpara0035" role="paragraph">Present address: Spirogen, a member of the AstraZeneca group, QMB Innovation Centre, 42 New Road, London E1 2AX, UK</div></div></div><div role="doc-footnote" data-has="label"><div class="label">14</div><div id="fn7"><div id="ntpara0040" role="paragraph">Present address: School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, GA 30332-2000, USA</div></div></div><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn8"><div id="ntpara0045" role="paragraph">Present address: Facebook, 1 Hacker Way, Menlo Park, CA 94025, USA</div></div></div><div role="doc-footnote" data-has="label"><div class="label">16</div><div id="fn9"><div id="ntpara0050" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received April 30, 2020</span>; <span>Revised January 22, 2021</span>; <span>Accepted March 31, 2021</span>; <span>Published online April 30, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.04.001" property="sameAs" target="_blank">10.1016/j.cell.2021.04.001 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421004360" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 The Authors. Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution (CC BY 4.0)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.04.001" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0011-0"><img src="/cms/asset/45dddb07-239f-4018-9c64-7a5ec73e2cde/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 10"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2900436-0" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (5)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900436-0" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.04.001">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2900436-0" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.04.001&amp;pii=S0092-8674%2821%2900436-0" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900436-0" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.04.001">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.04.001" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421004360&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421004360&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=10&amp;pii=S0092-8674%2820%29X0011-0" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)00430-X" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)00379-2" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (5)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">⋅</div><div class="content"><div id="p0010" role="paragraph">pSer/pThr and Cys are converted to reactive dehydroamino acids during protein damage</div></div></div><div id="u0015" role="listitem"><div class="label">⋅</div><div class="content"><div id="p0015" role="paragraph">MEK1 with dehydroalanine in its activation loop is an active, deregulated kinase</div></div></div><div id="u0020" role="listitem"><div class="label">⋅</div><div class="content"><div id="p0020" role="paragraph">LanCL adds GSH to dehydroamino acids, removing electrophiles from the proteome</div></div></div><div id="u0025" role="listitem"><div class="label">⋅</div><div class="content"><div id="p0025" role="paragraph"><i>C-</i>glutathionylation of MEK1-dehydroalanine inactivates its kinase activity</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Enzyme-mediated damage repair or mitigation, while common for nucleic acids, is rare for proteins. Examples of protein damage are elimination of phosphorylated Ser/Thr to dehydroalanine/dehydrobutyrine (Dha/Dhb) in pathogenesis and aging. Bacterial LanC enzymes use Dha/Dhb to form carbon-sulfur linkages in antimicrobial peptides, but the functions of eukaryotic LanC-like (LanCL) counterparts are unknown. We show that LanCLs catalyze the addition of glutathione to Dha/Dhb in proteins, driving irreversible <i>C</i>-glutathionylation. Chemo-enzymatic methods were developed to site-selectively incorporate Dha/Dhb at phospho-regulated sites in kinases. In human MAPK-MEK1, such “elimination damage” generated aberrantly activated kinases, which were deactivated by LanCL-mediated <i>C</i>-glutathionylation. Surveys of endogenous proteins bearing damage from elimination (the eliminylome) also suggest it is a source of electrophilic reactivity. LanCLs thus remove these reactive electrophiles and their potentially dysregulatory effects from the proteome. As knockout of LanCL in mice can result in premature death, repair of this kind of protein damage appears important physiologically.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/7d9b57e4-d45d-437b-ab74-1e633c18203b/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.04.001/asset/227ad26b-7b3f-4e31-aadf-ce00b078ad33/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/7d9b57e4-d45d-437b-ab74-1e633c18203b/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;C-glutathionylation&quot;" alt="Search on this keyword" rel="nofollow">C-glutathionylation</a></li><li><a href="/action/doSearch?AllField=&quot;dehydroalanine&quot;" alt="Search on this keyword" rel="nofollow">dehydroalanine</a></li><li><a href="/action/doSearch?AllField=&quot;dehydrobutyrine&quot;" alt="Search on this keyword" rel="nofollow">dehydrobutyrine</a></li><li><a href="/action/doSearch?AllField=&quot;LanCL&quot;" alt="Search on this keyword" rel="nofollow">LanCL</a></li><li><a href="/action/doSearch?AllField=&quot;phosphoThr+lyase&quot;" alt="Search on this keyword" rel="nofollow">phosphoThr lyase</a></li><li><a href="/action/doSearch?AllField=&quot;eliminylome&quot;" alt="Search on this keyword" rel="nofollow">eliminylome</a></li><li><a href="/action/doSearch?AllField=&quot;protein+damage&quot;" alt="Search on this keyword" rel="nofollow">protein damage</a></li><li><a href="/action/doSearch?AllField=&quot;MEK1&quot;" alt="Search on this keyword" rel="nofollow">MEK1</a></li><li><a href="/action/doSearch?AllField=&quot;lanthionine&quot;" alt="Search on this keyword" rel="nofollow">lanthionine</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">Lanthionine-containing peptides or lanthipeptides are ribosomally synthesized and post-translationally modified peptides produced by bacteria (<span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Repka et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Repka, L.M. ∙ Chekan, J.R. ∙ Nair, S.K. ...</div><div><strong>Mechanistic understanding of lanthipeptide biosynthetic enzymes</strong></div><div><em>Chem. Rev.</em> 2017; <strong>117</strong>:5457-5520</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.chemrev.6b00591" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85017930087" target="_blank" aria-label="Open Scopus in new tab">Scopus (342)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28135077/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.chemrev.6b00591&amp;pmid=28135077" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Lanthionines are installed through dehydratase-mediated conversion of Ser/Thr residues in precursor peptides to dehydroamino acids. Subsequently, intramolecular addition of thiols of cysteines onto the dehydroamino acid residues is mediated by LanC enzymes or LanC-type domains in bifunctional LanM proteins (<span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Repka et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Repka, L.M. ∙ Chekan, J.R. ∙ Nair, S.K. ...</div><div><strong>Mechanistic understanding of lanthipeptide biosynthetic enzymes</strong></div><div><em>Chem. Rev.</em> 2017; <strong>117</strong>:5457-5520</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.chemrev.6b00591" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85017930087" target="_blank" aria-label="Open Scopus in new tab">Scopus (342)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28135077/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.chemrev.6b00591&amp;pmid=28135077" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mammalian genomes encode multiple LanC-like proteins (LanCLs) but lack genes encoding an obvious dehydratase ortholog. The structure of human LanCL1 resembles NisC (a bacterial LanC); both structures contain two-layered α helix barrels that bind Zn<sup>2+</sup> (<a id="crosref0210" href="#fig1">Figure&nbsp;1</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Li et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Li, B. ∙ Yu, J.P. ∙ Brunzelle, J.S. ...</div><div><strong>Structure and mechanism of the lantibiotic cyclase involved in nisin biosynthesis</strong></div><div><em>Science.</em> 2006; <strong>311</strong>:1464-1467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1121422" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33644854595" target="_blank" aria-label="Open Scopus in new tab">Scopus (263)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16527981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1121422&amp;pmid=16527981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Zhang et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zhang, W. ∙ Wang, L. ∙ Liu, Y. ...</div><div><strong>Structure of human lanthionine synthetase C-like protein 1 and its interaction with Eps8 and glutathione</strong></div><div><em>Genes Dev.</em> 2009; <strong>23</strong>:1387-1392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1789209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-67149147296" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19528316/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1789209&amp;pmid=19528316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). LanCL1 and 2 are ubiquitously expressed in various tissues (<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Bauer et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Bauer, H. ∙ Mayer, H. ∙ Marchler-Bauer, A. ...</div><div><strong>Characterization of p40/GPR69A as a peripheral membrane protein related to the lantibiotic synthetase component C</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 2000; <strong>275</strong>:69-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/bbrc.2000.3260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0034682901" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10944443/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fbbrc.2000.3260&amp;pmid=10944443" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Eley et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Eley, G.D. ∙ Reiter, J.L. ∙ Pandita, A. ...</div><div><strong>A chromosomal region 7p11.2 transcript map: its development and application to the study of EGFR amplicons in glioblastoma</strong></div><div><em>Neuro-oncol.</em> 2002; <strong>4</strong>:86-94</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/4.2.86" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11916499/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2F4.2.86&amp;pmid=11916499" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Mayer et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Mayer, H. ∙ Salzer, U. ∙ Breuss, J. ...</div><div><strong>Isolation, molecular characterization, and tissue-specific expression of a novel putative G protein-coupled receptor</strong></div><div><em>Biochim. Biophys. Acta.</em> 1998; <strong>1395</strong>:301-308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0167-4781(97)00178-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0032506996" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9512664/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0167-4781%2897%2900178-4&amp;pmid=9512664" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">2001b</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Mayer, H. ∙ Pongratz, M. ∙ Prohaska, R.</div><div><strong>Molecular cloning, characterization, and tissue-specific expression of human LANCL2, a novel member of the LanC-like protein family</strong></div><div><em>DNA Seq.</em> 2001; <strong>12</strong>:161-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/10425170109080770" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0035238674" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11762191/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F10425170109080770&amp;pmid=11762191" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with the highest expression levels in brain, heart, and testes, whereas the expression pattern of LanCL3 remains to be determined due to the lack of a specific antibody. Human LanCL1 and LanCL2 bind to GSH and LanCL1 possesses weak glutathione <i>S-</i>transferase activity in assays with a xenobiotic substrate (<span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Chung et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Chung, C.H. ∙ Kurien, B.T. ∙ Mehta, P. ...</div><div><strong>Identification of lanthionine synthase C-like protein-1 as a prominent glutathione binding protein expressed in the mammalian central nervous system</strong></div><div><em>Biochemistry.</em> 2007; <strong>46</strong>:3262-3269</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi061888s" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33947395638" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17305318/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi061888s&amp;pmid=17305318" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Huang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Huang, C. ∙ Chen, M. ∙ Pang, D. ...</div><div><strong>Developmental and activity-dependent expression of LanCL1 confers antioxidant activity required for neuronal survival</strong></div><div><em>Dev. Cell.</em> 2014; <strong>30</strong>:479-487</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.devcel.2014.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.devcel.2014.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84907304641" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25158856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2014.06.011&amp;pmid=25158856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Mladkova et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Mladkova, J. ∙ Sanda, M. ∙ Matouskova, E. ...</div><div><strong>Phenotyping breast cancer cell lines EM-G3, HCC1937, MCF7 and MDA-MB-231 using 2-D electrophoresis and affinity chromatography for glutathione-binding proteins</strong></div><div><em>BMC Cancer.</em> 2010; <strong>10</strong>:449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2407-10-449" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-77955803289" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20731849/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2407-10-449&amp;pmid=20731849" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Zhang et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zhang, W. ∙ Wang, L. ∙ Liu, Y. ...</div><div><strong>Structure of human lanthionine synthetase C-like protein 1 and its interaction with Eps8 and glutathione</strong></div><div><em>Genes Dev.</em> 2009; <strong>23</strong>:1387-1392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1789209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-67149147296" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19528316/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1789209&amp;pmid=19528316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, the endogenous function of LanCL proteins remains unknown.</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/2771d953-4355-43cd-9341-0ad94fbb26d5/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.04.001/asset/6be4ebdf-7882-4bda-9c1d-c1f188986d78/main.assets/gr1.jpg" height="866" width="750" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/2771d953-4355-43cd-9341-0ad94fbb26d5/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Structural and pull-down analyses of LanC-type domains and LanCL proteins reveal common architectures and affinity for kinases</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Crystal structures of human LanCLs (PDB: <a id="interref240" href="http://firstglance.jmol.org/fg.htm?mol3E6U" target="_blank">3E6U</a> and <a id="interref245" href="http://firstglance.jmol.org/fg.htm?mol6WQ1" target="_blank">6WQ1</a>), bacterial NisC (PDB: <a id="intref0220" href="http://firstglance.jmol.org/fg.htm?mol2G0D" target="_blank">2G0D</a>), and a LanC-domain in a LanM (CylM; PDB: <a id="intref0225" href="http://firstglance.jmol.org/fg.htm?mol5DZT" target="_blank">5DZT</a>).</div><div id="fspara0020" role="paragraph">(B) Partial sequence alignment of NisC, the cyclase domain of CylM, and human LanCLs. The residues proposed to protonate the enolate intermediate during Michael-type addition to dehydroamino acids are in red. Zinc ion binding residues are in yellow.</div><div id="fspara0025" role="paragraph">(C) GSH binding pocket is conserved in LanCL1 and LanCL2 but not LanC/LanMs. The <i>apo</i> structure of Δ<sub>1–18</sub>-LanCL2 (green sticks) was superimposed with the LanCL1-GSH co-crystal structure (blue sticks; PDB: <a id="intref0230" href="http://firstglance.jmol.org/fg.htm?mol3E73" target="_blank">3E73</a>). Arg22 is not observed in the electron density of LANCL2. Pink sticks show GSH.</div><div id="fspara0030" role="paragraph">(D) Crystal structure of CylM showing interaction of a LanC-type cyclase domain (red) with a kinase-like dehydratase domain (gold).</div><div id="fspara0035" role="paragraph">(E) His-pull-down assay with LanCL proteins. Recombinant His-tagged human LanCL1 and LanCL2 show affinity for several cellular kinases from HEK293 cell extracts.</div><div id="fspara0040" role="paragraph">For pull-down of endogenous LanCL2 with a subset of these kinases, see <a id="crosref1595" href="#figs1">Figure&nbsp;S1</a>.</div></div></div></div></figcaption></figure></div><div id="p0035" role="paragraph">Bacterial LanC catalyze the addition of thiols of Cys to dehydroamino acids. In mammals, dehydroamino acids are thought to be rare, but under certain conditions they are formed in proteins as a result of enzymatic or background non-specific chemical elimination of the phosphate group of phosphoserine/phosphothreonine (pSer/pThr) or of oxidized Cys residues (<span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Cooper et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cooper, A.J. ∙ Pinto, J.T. ∙ Callery, P.S.</div><div><strong>Reversible and irreversible protein glutathionylation: biological and clinical aspects</strong></div><div><em>Expert Opin. Drug Metab. Toxicol.</em> 2011; <strong>7</strong>:891-910</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1517/17425255.2011.577738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79959221560" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21557709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1517%2F17425255.2011.577738&amp;pmid=21557709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For instance, pathogens like <i>Salmonella</i> or <i>Shigella</i> contain effector proteins as virulence factors that eliminate the phosphate from pSer/pThr in the activation loop of kinases to generate Dha/Dhb (<span class="dropBlock reference-citations"><a id="body-ref-sref39-1" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-1" aria-controls="sref39-1">Li et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref39-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, H. ∙ Xu, H. ∙ Zhou, Y. ...</div><div><strong>The phosphothreonine lyase activity of a bacterial type III effector family</strong></div><div><em>Science.</em> 2007; <strong>315</strong>:1000-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1138960" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33847154642" target="_blank" aria-label="Open Scopus in new tab">Scopus (345)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17303758/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1138960&amp;pmid=17303758" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref80-1" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-1" aria-controls="sref80-1">Zhu et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref80-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Zhu, Y. ∙ Li, H. ∙ Long, C. ...</div><div><strong>Structural insights into the enzymatic mechanism of the pathogenic MAPK phosphothreonine lyase</strong></div><div><em>Mol. Cell.</em> 2007; <strong>28</strong>:899-913</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.molcel.2007.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.molcel.2007.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-36749011295" target="_blank" aria-label="Open Scopus in new tab">Scopus (109)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18060821/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2007.11.011&amp;pmid=18060821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition to this enzymatic process during infections, non-enzymatic elimination of phosphate is thought to lead to irreversible damage to proteins during aging since Dha/Dhb residues cannot be rephosphorylated (<span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Wang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Wang, Z. ∙ Lyons, B. ∙ Truscott, R.J. ...</div><div><strong>Human protein aging: modification and crosslinking through dehydroalanine and dehydrobutyrine intermediates</strong></div><div><em>Aging Cell.</em> 2014; <strong>13</strong>:226-234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/acel.12164" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84895822337" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24134651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Facel.12164&amp;pmid=24134651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because Dha and Dhb are reactive electrophilic residues, they can also lead to protein crosslinking during aging (<span class="dropBlock reference-citations"><a id="body-ref-sref42-1" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-1" aria-controls="sref42-1">Linetsky et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref42-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Linetsky, M. ∙ Hill, J.M. ∙ LeGrand, R.D. ...</div><div><strong>Dehydroalanine crosslinks in human lens</strong></div><div><em>Exp. Eye Res.</em> 2004; <strong>79</strong>:499-512</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.exer.2004.06.026" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-4544350320" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15381034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.exer.2004.06.026&amp;pmid=15381034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Collectively, we will call Dha/Dhb-containing proteins in the proteome the “eliminylome.”</div><div id="p0040" role="paragraph">Scattered clues to the enigmatic function of LanCLs exist. The crystal structure of a bacterial LanM unexpectedly showed structural homology of its dehydratase domain with mammalian kinases. In particular, the adjoining LanC-type cyclase domain in the LanM structure makes intramolecular interactions with the dehydratase domain (<span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Dong et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dong, S.H. ∙ Tang, W. ∙ Lukk, T. ...</div><div><strong>The enterococcal cytolysin synthetase has an unanticipated lipid kinase fold</strong></div><div><em>eLife.</em> 2015; <strong>4</strong>:e07607</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.07607" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84940538541" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.07607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). LanCL proteins have previously been shown to interact with kinases such as Akt and mammalian target of rapamycin (mTOR) (<span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Zeng et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Zeng, M. ∙ van der Donk, W.A. ∙ Chen, J.</div><div><strong>Lanthionine synthetase C-like protein 2 (LanCL2) is a novel regulator of Akt</strong></div><div><em>Mol. Biol. Cell.</em> 2014; <strong>25</strong>:3954-3961</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E14-01-0004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84913529605" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25273559/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E14-01-0004&amp;pmid=25273559" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Based on such inferred domain-domain interaction of kinase-type domains with LanC-type domains, we explored its fuller possible significance in mammalian systems.</div><div id="p0045" role="paragraph">Here, we report that LanCL proteins catalyze conjugate addition in an intermolecular manner. They activate the Cys thiol of GSH and catalyze <i>C</i>–<i>S</i> bond-forming <i>C</i>-glutathionylation of dehydroamino-acid-containing peptides and kinases in their activation loops. Unexpected partial activation of MEK1 by dehydroamino acids in the activation loop and then deactivation by such LanCL “scavenging” identifies a putative role for this reaction in the removal of potentially dysregulatory protein function within the eliminylome. Mice with all three LanCL knocked out (triple knockout [TKO]) mostly show no phenotype, but in response to a currently unknown stimulus ∼25%–30% of them display a striking phenotype of premature death by 6&nbsp;months of age.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Structure of LanCL2</h3><div id="p0050" role="paragraph">The NisC and LanCL1 crystal structures were previously reported (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Li et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Li, B. ∙ Yu, J.P. ∙ Brunzelle, J.S. ...</div><div><strong>Structure and mechanism of the lantibiotic cyclase involved in nisin biosynthesis</strong></div><div><em>Science.</em> 2006; <strong>311</strong>:1464-1467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1121422" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33644854595" target="_blank" aria-label="Open Scopus in new tab">Scopus (263)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16527981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1121422&amp;pmid=16527981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-3" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-3" aria-controls="sref77-3">Zhang et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref77-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zhang, W. ∙ Wang, L. ∙ Liu, Y. ...</div><div><strong>Structure of human lanthionine synthetase C-like protein 1 and its interaction with Eps8 and glutathione</strong></div><div><em>Genes Dev.</em> 2009; <strong>23</strong>:1387-1392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1789209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-67149147296" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19528316/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1789209&amp;pmid=19528316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this work, we solved the structure of Δ<sub>1–18</sub>-LanCL2 (PDB: <a id="intref0010" href="http://firstglance.jmol.org/fg.htm?mol6WQ1" target="_blank">6WQ1</a>), which proved more amenable to crystallization than the full-length protein. Its overall structure resembles that of NisC and LanCL1 and is made up of two layers of 14 α helices with a zinc ion binding site located on one side (<a id="crosref0310" href="#fig1">Figure&nbsp;1</a>A). The Zn<sup>2+</sup>-binding residues in LanCL2 are Cys321, Cys367, and His368, and corresponding residues are both conserved in NisC and essential for NisC activity (<a id="crosref0315" href="#fig1">Figure&nbsp;1</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Li and van der Donk, 2007</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, B. ∙ van der Donk, W.A.</div><div><strong>Identification of essential catalytic residues of the cyclase NisC involved in the biosynthesis of nisin</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:21169-21175</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-34547137156" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17513866/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M701802200&amp;pmid=17513866" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Bovine LanCL1 binds GSH (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Chung et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Chung, C.H. ∙ Kurien, B.T. ∙ Mehta, P. ...</div><div><strong>Identification of lanthionine synthase C-like protein-1 as a prominent glutathione binding protein expressed in the mammalian central nervous system</strong></div><div><em>Biochemistry.</em> 2007; <strong>46</strong>:3262-3269</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi061888s" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33947395638" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17305318/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi061888s&amp;pmid=17305318" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using five residues: Arg4, Lys317, Arg364, Asp367, and Glu374 (<span class="dropBlock reference-citations"><a id="body-ref-sref77-4" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-4" aria-controls="sref77-4">Zhang et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref77-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zhang, W. ∙ Wang, L. ∙ Liu, Y. ...</div><div><strong>Structure of human lanthionine synthetase C-like protein 1 and its interaction with Eps8 and glutathione</strong></div><div><em>Genes Dev.</em> 2009; <strong>23</strong>:1387-1392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1789209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-67149147296" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19528316/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1789209&amp;pmid=19528316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The corresponding residues in LanCL2 are Arg22, Lys362, Arg409, Asp412, and Glu419. Importantly, these residues are not conserved in bacterial LanC proteins or the LanC domains of LanM proteins but highly conserved in eukaryotic LanCLs suggestive of a common yet distinct and undiscovered function for the eukaryotic LanCLs compared to bacterial LanC/LanMs.</div></section><section id="sec-2-2"><h3>LanCL proteins interact with a range of kinases</h3><div id="p0055" role="paragraph">Beyond its binding of GSH, LanCL2 interacts with Akt and mTOR (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Zeng et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Zeng, M. ∙ van der Donk, W.A. ∙ Chen, J.</div><div><strong>Lanthionine synthetase C-like protein 2 (LanCL2) is a novel regulator of Akt</strong></div><div><em>Mol. Biol. Cell.</em> 2014; <strong>25</strong>:3954-3961</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E14-01-0004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84913529605" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25273559/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E14-01-0004&amp;pmid=25273559" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This finding was intriguing since in bacterial LanM proteins, the LanC-like domain is part of the same polypeptide as a kinase domain (<a id="crosref0340" href="#fig1">Figure&nbsp;1</a>D). We investigated here whether LanCLs also interact with other cellular kinases. We expressed C-terminal His-tagged recombinant human LanCL2 in bacteria and incubated the purified protein with HEK293 cell lysate followed by His pull-down. As expected, mTOR was detected in the pull-down by western blotting (<a id="crosref0345" href="#fig1">Figure&nbsp;1</a>E). The pull-down samples also contained several other protein and lipid kinases, including ATR (a PIK-related kinase like mTOR), p110α (catalytic subunit of a class I phosphatidylinositol 3-kinase), PIP4K2A (a phosphatidylinositol-5-phosphate 4-kinase), Vps34 (class III phosphatidylinositol 3-kinase), and the Ser/Thr kinases p70-S6K1, ERK1/2, and MEK1/2 (<a id="crosref0350" href="#fig1">Figure&nbsp;1</a>E; <a id="crosref0355" href="#figs1">Figure&nbsp;S1</a>A). On the other hand, glyceraldehyde-3-phosphate dehydrogenase and leucyl-tRNA synthetase were not present in the pull-down samples (<a id="crosref0360" href="#fig1">Figure&nbsp;1</a>E). These data suggested that LanCL2 might have a broad binding affinity toward kinases.</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/783b9af0-0b39-4304-b76c-3f2bec54405d/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.04.001/asset/2f423bcc-01b8-45c1-86e9-0e11f063c223/main.assets/figs1.jpg" height="898" width="637" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/783b9af0-0b39-4304-b76c-3f2bec54405d/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">His-pull-down assays with His<sub>6</sub>-LanCL1 and LanCL2-His<sub>6</sub>, generation of Dhb-peptides, and activity of LanCL mutants, related to <a id="crosref1640" href="#fig1">Figures 1</a> and <a id="crosref1645" href="#fig2">2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0175" role="paragraph">(<b>A</b>) His-pulldown assay for LanCL2-His<sub>6</sub> and His<sub>6</sub>-LanCL1. Only a small fraction of each kinase was pulled down by His<sub>6</sub>-LanCL1 and LanCL2-His<sub>6</sub>. (<b>B</b>) Endogenous LanCL1/2 were co-immunoprecipitated by anti-p110, anti-mTOR and anti-Vps34 antibodies in HEK293 cells. (<b>C</b>) MSMS for Dhb-ERK peptide and Dhb-Akt peptide. (<b>D-F</b>) Phosphate elimination by His<sub>6</sub>-SpvC and glutathionylation of Dhb-ERK (with pTyr) by LanCL2-His<sub>6</sub>. (<b>D</b>) Phosphate elimination by His<sub>6</sub>-SpvC using phospho-ERK peptide containing a pTyr residue as substrate. (<b>E</b>) MSMS for Dhb-ERK (with pTyr residue). (<b>F</b>) ESI-LC-MS analysis of LanCL2 activity using Dhb-ERK (with pTyr residue) as substrate. EIC, extracted ion chromatogram; ESI, electrospray ionization. (<b>G</b>) Generation of Dha in GSH peroxidase (GPX) by H<sub>2</sub>O<sub>2</sub> treatment, which was verified by addition of biotinylated cysteamine. No GSH adduct was observed upon incubation of the Dha-GPX with LanCL1/2 and GSH. (<b>H</b>) Left, partial conversion with γ-Glu-Cys (γ-EC) (1&nbsp;mM) and Dhb-ERK by LanCL2 but not its double mutant after 12 h. Right, near complete conversion with free Cys (1&nbsp;mM) and Dhb-ERK by LanCL2 but not its double mutant after 12 h. (<b>I</b>) Anti-GSH antibody can recognize non-canonical glutathionylation. (<b>J</b>) His<sub>6</sub>-phospho-ERK2 was eliminated by His<sub>6</sub>-OspF and the resulting His<sub>6</sub>-ERK2-Dhb185pTyr187 was glutathionylated by MBP-LanCL2 as demonstrated by western blot. (<b>K</b>) Tandem MS analysis of native and glutathionylated CylL<sub>S</sub>’’ and CylL<sub>L</sub>". (<b>L</b>) Non-canonical glutathionylation is catalyzed by endogenous LanCLs. Glutathionylation of His<sub>6</sub>-MEK1-Dha222 was observed by western blotting.</div></div></div></div></figcaption></figure></div><div id="p0060" role="paragraph">We also tested whether LanCL1 demonstrates binding affinity toward kinases. Using His<sub>6</sub>-LanCL1 pull-down assays, LanCL1 was shown to interact with mTOR, ATR, Akt, p110α, PIP4K2A, and MEK1/2 but not p70-S6K1 and ERK1/2 (<a id="crosref0365" href="#fig1">Figure&nbsp;1</a>E; <a id="crosref0370" href="#figs1">Figure&nbsp;S1</a>B). The seemingly general interaction of LanCLs with a range of kinases was surprising and somewhat difficult to explain. We also analyzed the pull-down samples by mass spectrometry. Several of the aforementioned kinases were again observed compared to control experiments, but also many non-kinases suggesting LanCLs interact directly or indirectly with a wide range of proteins (<a id="crosref0375" href="#mmc1">Table S1</a>).</div></section><section id="sec-2-3"><h3>LanCLs add GSH to dehydroamino acids in peptides corresponding to kinase activation loops</h3><div id="p0065" role="paragraph">The structural and interaction data suggested that LanCLs bind both GSH and broad protein motifs. Bacterial LanC enzymes catalyze the intramolecular addition of Cys thiols to Dha/Dhb (<span class="dropBlock reference-citations"><a id="body-ref-sref38-3" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-3" aria-controls="sref38-3">Li et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref38-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Li, B. ∙ Yu, J.P. ∙ Brunzelle, J.S. ...</div><div><strong>Structure and mechanism of the lantibiotic cyclase involved in nisin biosynthesis</strong></div><div><em>Science.</em> 2006; <strong>311</strong>:1464-1467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1121422" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33644854595" target="_blank" aria-label="Open Scopus in new tab">Scopus (263)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16527981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1121422&amp;pmid=16527981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). By extension of mechanistic logic, we investigated whether LanCLs could catalyze intermolecular GSH addition to eliminated (Dha- or Dhb-containing) peptide sequences corresponding to functional motifs (e.g., the activation loops of kinases). Indeed, the possibility that LanCL proteins might catalyze Michael-type additions to dehydroamino acids has been previously, presciently postulated but not experimentally tested (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Brennan and Barford, 2009</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Brennan, D.F. ∙ Barford, D.</div><div><strong>Eliminylation: a post-translational modification catalyzed by phosphothreonine lyases</strong></div><div><em>Trends Biochem. Sci.</em> 2009; <strong>34</strong>:108-114</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-62849120767" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19233656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tibs.2008.11.005&amp;pmid=19233656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0070" role="paragraph">The bacterial pathogen pThr lyases SpvC and OspF catalyze phosphate elimination from mitogen activated protein kinase (MAPK) activation loop peptides containing pThr to form the corresponding Dhb-containing peptides (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Chambers et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Chambers, K.A. ∙ Abularrage, N.S. ∙ Scheck, R.A.</div><div><strong>Selectivity within a family of bacterial phosphothreonine lyases</strong></div><div><em>Biochemistry.</em> 2018; <strong>57</strong>:3790-3796</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.biochem.8b00534" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85047643839" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29792689/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.biochem.8b00534&amp;pmid=29792689" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39-2" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-2" aria-controls="sref39-2">Li et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref39-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, H. ∙ Xu, H. ∙ Zhou, Y. ...</div><div><strong>The phosphothreonine lyase activity of a bacterial type III effector family</strong></div><div><em>Science.</em> 2007; <strong>315</strong>:1000-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1138960" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33847154642" target="_blank" aria-label="Open Scopus in new tab">Scopus (345)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17303758/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1138960&amp;pmid=17303758" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref80-2" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-2" aria-controls="sref80-2">Zhu et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref80-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Zhu, Y. ∙ Li, H. ∙ Long, C. ...</div><div><strong>Structural insights into the enzymatic mechanism of the pathogenic MAPK phosphothreonine lyase</strong></div><div><em>Mol. Cell.</em> 2007; <strong>28</strong>:899-913</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.molcel.2007.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.molcel.2007.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-36749011295" target="_blank" aria-label="Open Scopus in new tab">Scopus (109)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18060821/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2007.11.011&amp;pmid=18060821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We therefore used these lyases to mimic such pathogen-induced damage to prepare model, Dhb-containing peptides corresponding to the activation loops of ERK and Akt (<a id="crosref0405" href="#fig2">Figure&nbsp;2</a>A; <a id="crosref0410" href="#figs1">Figure&nbsp;S1</a>C). Following this generation of eliminative damage, we tested LanCL activity. The Dhb-containing ERK peptides (containing either Tyr or pTyr, <a id="crosref0415" href="#figs1">Figures S1</a>D–S1F) were reacted with GSH and recombinant LanCLs and analyzed by matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF MS). Strikingly, a glutathione adduct was observed when the Dhb-ERK peptides were treated with wild-type (WT) LanCL1/2 but not with mutant LanCL1/2 in which residues corresponding in sequence and structure to amino acids that are critical for NisC activity (see below) were mutated (<a id="crosref0420" href="#fig2">Figure&nbsp;2</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Li and van der Donk, 2007</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, B. ∙ van der Donk, W.A.</div><div><strong>Identification of essential catalytic residues of the cyclase NisC involved in the biosynthesis of nisin</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:21169-21175</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-34547137156" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17513866/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M701802200&amp;pmid=17513866" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This observation constitutes not only an experimentally demonstrated catalytic function for LanCLs but also implicates some of the key catalytic residues responsible (see following section).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/373da9f4-9ddc-454f-91dc-3c9ef8738584/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.04.001/asset/e0271357-c929-4b58-91f9-7795921c5009/main.assets/gr2.jpg" height="840" width="750" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/373da9f4-9ddc-454f-91dc-3c9ef8738584/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Eliminative damage in peptides catalyzed by the pathogen pThr lyases SpvC or OspF and addition of GSH to these eliminated peptides by WT LanCL1 and LanCL2 but not mutants</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0050" role="paragraph">(A) MALDI-TOF mass spectra of phospho-ERK peptide treated with His<sub>6</sub>-SpvC and phospho-Akt peptide treated with His<sub>6</sub>-OspF resulting in a loss of 98 Da. See <a id="crosref1600" href="#mmc2">Table S2</a> for calculated and observed masses and <a id="crosref1605" href="#figs1">Figure&nbsp;S1</a>C for fragmentation data.</div><div id="fspara0055" role="paragraph">(B) MALDI-TOF mass spectra of Dhb-containing ERK peptide treated with WT LanCL1- and 2 or mutants in the presence of GSH.</div><div id="fspara0060" role="paragraph">(C) MALDI-TOF mass spectra of Dhb-containing Akt peptide treated with WT LanCL1- and 2 or mutants in the presence of GSH.</div><div id="fspara0065" role="paragraph">(D) Active sites of LanCL1 bound to GSH (PDB: <a id="intref0235" href="http://firstglance.jmol.org/fg.htm?mol3E73" target="_blank">3E73</a>) and the <i>intra</i>-molecular C-S bond forming catalyst NisC (PDB: <a id="interref250" href="http://firstglance.jmol.org/fg.htm?mol2G02" target="_blank">2G02</a>).</div><div id="fspara0070" role="paragraph">(E and F) Loss of catalytic activity upon mutation reveals the importance of the His proposed to protonate the enolate during intramolecular nisin cyclase <i>C</i>–<i>S</i>-bond-forming activity as well as intermolecular LanCL-catalyzed <i>C-</i>glutathionylation. The zinc ion binding Cys residues are also essential for LanCL activity.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">Next, we evaluated the substrate tolerance of LanCL1/2-mediated catalysis. The Dhb-containing Akt peptide was incubated with WT LanCL1/2 and again a 307&nbsp;Da shift corresponding to GSH was observed when compared to negative controls using mutant LanCL1/2 (<a id="crosref0435" href="#fig2">Figure&nbsp;2</a>C). Together these data suggested LanCLs are capable of potentially broad “trapping” of eliminated peptide motifs. We also evaluated alternative nucleophiles. The dipeptide γ-glutamyl-Cys and free Cys were both substrates but less efficient than GSH with Dhb-ERK peptide, whereas the dipeptide GlyCys or the reactive Cys-containing protein thioredoxin were not accepted (<a id="crosref0440" href="#figs1">Figure&nbsp;S1</a>H).</div></section><section id="sec-2-4"><h3>LanCLs use residues conserved in LanC cyclases to catalyze intermolecular <i>C</i>–<i>S</i>-bond-forming <i>C</i>-glutathionylation</h3><div id="p0080" role="paragraph">His212 in NisC has been proposed to protonate the enolate intermediate generated upon Cys addition to Dha/Dhb (<span class="dropBlock reference-citations"><a id="body-ref-sref37-3" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-3" aria-controls="sref37-3">Li and van der Donk, 2007</a><div class="dropBlock__holder" data-db-target-of="sref37-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, B. ∙ van der Donk, W.A.</div><div><strong>Identification of essential catalytic residues of the cyclase NisC involved in the biosynthesis of nisin</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:21169-21175</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-34547137156" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17513866/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M701802200&amp;pmid=17513866" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Yang and van der Donk, 2015</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Yang, X. ∙ van der Donk, W.A.</div><div><strong>Michael-type cyclizations in lantibiotic biosynthesis are reversible</strong></div><div><em>ACS Chem. Biol.</em> 2015; <strong>10</strong>:1234-1238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acschembio.5b00007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84929583013" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25723375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facschembio.5b00007&amp;pmid=25723375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and this residue is conserved in LanCLs (His219 in LanCL1; His264 in LanCL2). Furthermore, the zinc binding residues (Cys284, Cys330, His331) in NisC are conserved in LanCL1 (Cys276, Cys322, His323) and LanCL2 (Cys321, Cys367, His368) (<a id="crosref0455" href="#fig2">Figure&nbsp;2</a>D) (<span class="dropBlock reference-citations"><a id="body-ref-sref37-4" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-4" aria-controls="sref37-4">Li and van der Donk, 2007</a><div class="dropBlock__holder" data-db-target-of="sref37-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, B. ∙ van der Donk, W.A.</div><div><strong>Identification of essential catalytic residues of the cyclase NisC involved in the biosynthesis of nisin</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:21169-21175</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-34547137156" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17513866/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M701802200&amp;pmid=17513866" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-5" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-5" aria-controls="sref77-5">Zhang et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref77-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zhang, W. ∙ Wang, L. ∙ Liu, Y. ...</div><div><strong>Structure of human lanthionine synthetase C-like protein 1 and its interaction with Eps8 and glutathione</strong></div><div><em>Genes Dev.</em> 2009; <strong>23</strong>:1387-1392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1789209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-67149147296" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19528316/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1789209&amp;pmid=19528316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The H212N and C330A mutants of NisC cannot produce correctly cyclized nisin (<span class="dropBlock reference-citations"><a id="body-ref-sref37-5" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-5" aria-controls="sref37-5">Li and van der Donk, 2007</a><div class="dropBlock__holder" data-db-target-of="sref37-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, B. ∙ van der Donk, W.A.</div><div><strong>Identification of essential catalytic residues of the cyclase NisC involved in the biosynthesis of nisin</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:21169-21175</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-34547137156" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17513866/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M701802200&amp;pmid=17513866" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The corresponding variants LanCL1-H219A and C322A and LanCL2-H264A and C321A/C367A were expressed in <i>E.&nbsp;coli</i> Rosetta 2 (DE3) with N-terminal or C-terminal hexahistidine tags and purified by affinity chromatography. None of the variants was able to catalyze GSH addition to Dhb-ERK and Dhb-Akt (<a id="crosref0475" href="#fig2">Figures 2</a>C, 2E, and 2F). These data suggest that the active site residues have similarly important functions in NisC and LanCL.</div></section><section id="sec-2-5"><h3>Kinases with dehydro amino acids in the activation loop show unexpected activation</h3><div id="p0085" role="paragraph">Activation by phosphorylation in kinases is attributed to conformational changes in activation loops (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Alessi et&nbsp;al., 1994</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alessi, D.R. ∙ Saito, Y. ∙ Campbell, D.G. ...</div><div><strong>Identification of the sites in MAP kinase kinase-1 phosphorylated by p74raf-1</strong></div><div><em>EMBO J.</em> 1994; <strong>13</strong>:1610-1619</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/j.1460-2075.1994.tb06424.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0028293931" target="_blank" aria-label="Open Scopus in new tab">Scopus (474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8157000/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fj.1460-2075.1994.tb06424.x&amp;pmid=8157000" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Gopalbhai et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Gopalbhai, K. ∙ Jansen, G. ∙ Beauregard, G. ...</div><div><strong>Negative regulation of MAPKK by phosphorylation of a conserved serine residue equivalent to Ser212 of MEK1</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2003; <strong>278</strong>:8118-8125</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M211870200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M211870200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0037424509" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12506122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M211870200&amp;pmid=12506122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Zheng and Guan, 1994</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Zheng, C.F. ∙ Guan, K.L.</div><div><strong>Activation of MEK family kinases requires&nbsp;phosphorylation of two conserved Ser/Thr residues</strong></div><div><em>EMBO J.</em> 1994; <strong>13</strong>:1123-1131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/j.1460-2075.1994.tb06361.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0028329630" target="_blank" aria-label="Open Scopus in new tab">Scopus (329)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8131746/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fj.1460-2075.1994.tb06361.x&amp;pmid=8131746" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Given the greater planarity (flattening) at Cα caused by elimination we wondered whether this too might act as a source of conformational alteration in kinases. While access to cleanly eliminated peptide motifs from the activation loop of representative kinases could be accomplished enzymatically (see above), we chose to also explore a scaleable chemical method for site-selectively and cleanly installing dehydroamino acids in the activation loops of full-length MAPKKs.</div><div id="p0090" role="paragraph">We have previously developed a three-step, one-pot method (involving <i>bis</i>-alkylation/elimination) that chemically converts free Cys residues to Dha using the reagent DBHDA (<a id="crosref0495" href="#fig3">Figure&nbsp;3</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref7-1" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-1" aria-controls="sref7-1">Chalker et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref7-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Chalker, J.M. ∙ Gunnoo, S.B. ∙ Boutureira, O. ...</div><div><em>Chem. Sci. (Camb.).</em> 2011; <strong>2</strong>:1666-1676</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c1sc00185j" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-81355138529" target="_blank" aria-label="Open Scopus in new tab">Scopus (284)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc1sc00185j" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We considered applying this strategy to a human kinase identified above as interacting with LanCLs, MEK1. However, MEK1 contains several existing, native Cys residues and, in preliminary experiments, removal of these native Cys created functional alterations and/or expression problems (<span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Wagle et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Wagle, N. ∙ Emery, C. ∙ Berger, M.F. ...</div><div><strong>Dissecting therapeutic resistance to RAF inhibition in melanoma by tumor genomic profiling</strong></div><div><em>J.&nbsp;Clin. Oncol.</em> 2011; <strong>29</strong>:3085-3096</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1200/JCO.2010.33.2312" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-80051625929" target="_blank" aria-label="Open Scopus in new tab">Scopus (830)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21383288/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1200%2FJCO.2010.33.2312&amp;pmid=21383288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Zhao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Zhao, K. ∙ Ju, Y. ∙ Li, S. ...</div><div><strong>S-sulfhydration of MEK1 leads to PARP-1 activation and DNA damage repair</strong></div><div><em>EMBO Rep.</em> 2014; <strong>15</strong>:792-800</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/embr.201338271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84899806842" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24778456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fembr.201338271&amp;pmid=24778456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); these observations therefore prevented direct application of prior methods that relied upon complete removal of native Cys residues.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/87e1c8d6-fb5d-4a3a-ad1a-dca8b837cac6/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.04.001/asset/924e0fa0-dd39-4441-8352-3c8a71e7f237/main.assets/gr3.jpg" height="770" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/87e1c8d6-fb5d-4a3a-ad1a-dca8b837cac6/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">A strategy for chemical generation of eliminated MEK1 proteins</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0080" role="paragraph">(A) Overall sequence for site-selective incorporation of Dha at regulatory Ser sites in kinases. Reaction of the most reactive Cys over less reactive Cys was used to allow chemical, regioselective incorporation of Dha at different sites. SDM, site-directed mutagenesis.</div><div id="fspara0085" role="paragraph">(B) MEK1, its six native free Cys (black, gray, red), and two activating Ser sites 218 (cyan) and 222 (blue); table: predicted side-chain accessibility of sites with/without “masking” nucleotide and Mg(II).</div><div id="fspara0090" role="paragraph">(C) Schematic of predicted structural and accessibility effects of masking and enhancing ligands; Cys207 is blocked by ATPγS; 218 and 222 are more accessible with Mg(II). See also <a id="crosref1610" href="#figs2">Figure&nbsp;S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph">We therefore considered a more challenging approach based on <i>regio-</i>selective chemical conversion of target Cys residues to Dha in the presence of other non-target native Cys residues. Current chemistries for targeting one free Cys residue in proteins among other Cys are rare; they require engineering (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Dai et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Dai, P. ∙ Zhang, C. ∙ Welborn, M. ...</div><div><strong>Salt effect accelerates site-selective cysteine bioconjugation</strong></div><div><em>ACS Cent. Sci.</em> 2016; <strong>2</strong>:637-646</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acscentsci.6b00180" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85019331701" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27725962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facscentsci.6b00180&amp;pmid=27725962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or exploitation of privileged arrangements of residues that were not applicable in MEK1 (<span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Willwacher et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Willwacher, J. ∙ Raj, R. ∙ Mohammed, S. ...</div><div><strong>Selective metal-site-guided arylation of proteins</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2016; <strong>138</strong>:8678-8681</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jacs.6b04043" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84979263737" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27336299/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjacs.6b04043&amp;pmid=27336299" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In principle, however, the different protein environments around each Cys residue could allow direct, chemical differentiation. In the three-step conversion of Cys to Dha (<a id="crosref0525" href="#fig3">Figure&nbsp;3</a>A, inset), the first irreversible alkylation step is rate- and hence regio-selectivity determining. We reasoned therefore that reagent tuning (without losing reactivity in steps 2 and 3) could allow the site-to-site control required for this chemical process.</div><div id="p0100" role="paragraph">Wild-type MEK1 contains six free, native Cys residues at positions 121, 142, 207, 277, 341, and 376. To discriminate potential reactivities of these residues, relative side-chain accessibility was estimated using a predictive computational approach (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Hubbard and Thornton, 1993</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Hubbard, S. ∙ Thornton, J.</div><div><strong>NACCESS</strong></div><div>University College London, 1993</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.HubbardJ.ThorntonNACCESS1993University+College+London" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) based on accessible surfaces determined from X-ray crystal structures (<a id="crosref0535" href="#fig3">Figure&nbsp;3</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Lee and Richards, 1971</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Lee, B. ∙ Richards, F.M.</div><div><strong>The interpretation of protein structures: estimation of static accessibility</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1971; <strong>55</strong>:379-400</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(71)90324-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0015222647" target="_blank" aria-label="Open Scopus in new tab">Scopus (5511)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5551392/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2871%2990324-X&amp;pmid=5551392" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A similar approach has proven successful previously for predicting chemical regioselectivity in other protein modifications (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">van Kasteren et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>van Kasteren, S.I. ∙ Kramer, H.B. ∙ Jensen, H.H. ...</div><div><strong>Expanding the diversity of chemical protein modification allows post-translational mimicry</strong></div><div><em>Nature.</em> 2007; <strong>446</strong>:1105-1109</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05757" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-34247622775" target="_blank" aria-label="Open Scopus in new tab">Scopus (275)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17460675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05757&amp;pmid=17460675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This analysis, when applied to the <i>apo</i>-MEK1 structure (<a id="crosref0550" href="#fig3">Figure&nbsp;3</a>B), predicted that a MEK1 mutant in which Cys was introduced at activation loop phosphorylation sites 218 and/or 222 (and removed from non-functional sites 277 and 376) would allow regioselective elimination. Importantly, when expressed, the MEK1-C277S/C376S mutant kinase displayed essentially identical enzymatic properties to WT MEK1 (<a id="crosref0555" href="#mmc2">Table S2</a>; <a id="crosref0560" href="#sec-4">STAR Methods</a>). The same predictive analysis of reactivity (based on analyses of binary (MEK1⋅ATPγS [PDB <a id="intref0015" href="http://firstglance.jmol.org/fg.htm?mol3W8Q" target="_blank">3W8Q</a>]) or ternary (MEK1⋅ATPγS⋅Mg [PDB <a id="intref0020" href="http://firstglance.jmol.org/fg.htm?mol3EQD" target="_blank">3EQD</a>] (<span class="dropBlock reference-citations"><a id="body-ref-sref20-1" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-1" aria-controls="sref20-1">Fischmann et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref20-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Fischmann, T.O. ∙ Smith, C.K. ∙ Mayhood, T.W. ...</div><div><strong>Crystal structures of MEK1 binary and ternary complexes with nucleotides and inhibitors</strong></div><div><em>Biochemistry.</em> 2009; <strong>48</strong>:2661-2674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi801898e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-65249090229" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19161339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi801898e&amp;pmid=19161339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and MEK1⋅ADP⋅Mg [PDB <a id="intref0025" href="http://firstglance.jmol.org/fg.htm?mol3EQI" target="_blank">3EQI</a>]) complexes) as well as prior studies (<span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Lamoureaux and Lee, 2011</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Lamoureaux, T.L. ∙ Lee, D.H.</div><div><strong>Chemical activation of MEK1--a redox trigger for evaluating the effects of phosphorylation</strong></div><div><em>Chem. Commun. (Camb.).</em> 2011; <strong>47</strong>:8623-8625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c1cc11745a" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79960474817" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21717004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc1cc11745a&amp;pmid=21717004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) also suggested the additional consideration of nucleotides (ATPγS, ATP, ADP) and metals (Mg(II)) as masking or enhancing ligands to usefully enhance the selectivity of this chemistry (<a id="crosref0575" href="#fig3">Figure&nbsp;3</a>C; <a id="crosref0580" href="#sec-4">STAR Methods</a>).</div><div id="p0105" role="paragraph">In this way, we designed and generated three mutant variants of MEK1 for testing with this regioselective elimination chemistry: MEK1-C218, MEK1-C222, and MEK1-C218C222 (bearing five, five, or six Cys residues, respectively; see <a id="crosref0585" href="#sec-4">STAR Methods</a> for full mutational patterns used to remove and introduce Cys). Next, we tuned a panel of systematically varied 1,4-<i>bis</i>-alkylating reagents derived from parent reagent DBHDA (<a id="crosref0590" href="#figs2">Figure&nbsp;S2</a>A). These varied in their reactivity, hydrophobicity, sterics, and charge while all retaining the ability to create 5-membered ring sulfonium intermediates that are critical to elimination of the side chain of the Cys residues introduced (<span class="dropBlock reference-citations"><a id="body-ref-sref7-2" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-2" aria-controls="sref7-2">Chalker et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref7-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Chalker, J.M. ∙ Gunnoo, S.B. ∙ Boutureira, O. ...</div><div><em>Chem. Sci. (Camb.).</em> 2011; <strong>2</strong>:1666-1676</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c1sc00185j" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-81355138529" target="_blank" aria-label="Open Scopus in new tab">Scopus (284)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc1sc00185j" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, consistent with our design predictions, use of these reagents in combination with masking ligands and metals (see <a id="crosref0600" href="#sec-4">STAR Methods</a>) allowed clean access (avoiding competing pathways, <a id="crosref0605" href="#fig4">Figure&nbsp;4</a>) to different MEK1 kinases containing complete elimination at different sites in their activation loops: MEK1-Dha222, MEK1-Dha218, and MEK1-Dha218Dha222 (<a id="crosref0610" href="#fig4">Figures 4</a>A–4C, respectively; <a id="crosref1745" href="#figs2">Figures S2</a>, <a id="crosref1750" href="#figs3">S3</a>, <a id="crosref1755" href="#figs4">S4</a>, and <a id="crosref1740" href="#figs5">S5</a>).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/69c4dabd-06c5-4027-b9aa-798498854716/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.04.001/asset/e3427577-2316-4f56-8334-db5e4d563c95/main.assets/gr4.jpg" height="750" width="497" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/69c4dabd-06c5-4027-b9aa-798498854716/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Reagent-controlled, regioselective, chemical elimination reactions of MEK1 allow single-site and double-site elimination</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0100" role="paragraph">Switching of reagents and conditions selectively avoids unwanted, competing chemical pathways (dotted) and yields clean chemical elimination (bold) to desired proteins: (A) MEK1-Dha222, (B) MEK1-Dha218, or (C) MEK1-Dha218Dha222. Intact protein LC-ESI-MS shown; see <a id="crosref1615" href="#mmc2">Table S2</a> for calculated and observed masses, <a id="crosref1620" href="#figs2">Figure&nbsp;S2</a> for structures of reagents, and <a id="crosref1760" href="#figs3">Figures S3</a>, <a id="crosref1765" href="#figs4">S4</a>, and <a id="crosref1770" href="#figs5">S5</a> for all data.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/926b8367-92b7-42ad-95b5-a801a6ed17aa/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.04.001/asset/3efc3874-bac3-4c0b-ac56-0f725fc187b3/main.assets/figs2.jpg" height="622" width="750" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/926b8367-92b7-42ad-95b5-a801a6ed17aa/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Scoping and controlling chemical elimination of MEK1, related to <a id="crosref1650" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0185" role="paragraph"><b>(A)</b> Panel of DBHDA derivatives used in this study. 1,4-Bis-alkylating/elimination agents ‘tuned’ by indicated properties (magenta). <b>(B)</b> Reagent Synthesis: i. SOCl<sub>2</sub>, reflux, 1.5 h then NBS,HBr/CCl<sub>4</sub>, reflux, 3 h; ii. NH<sub>4</sub>OH, 0°C, 1.5 h; iii. NaI, acetone, reflux, 4 h; iv. H<sub>2</sub>O/THF (1:1), 0°C, 2 h; v. SOCl<sub>2</sub>, reflux, 1.5 h then RNH<sub>2</sub> then trituration in H<sub>2</sub>O. <b>(C)</b> Differential scanning fluorimetry (<span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Niesen et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Niesen, F.H. ∙ Berglund, H. ∙ Vedadi, M.</div><div><strong>The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</strong></div><div><em>Nat. Protoc.</em> 2007; <strong>2</strong>:2212-2221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2007.321" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-37249005205" target="_blank" aria-label="Open Scopus in new tab">Scopus (1888)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17853878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2007.321&amp;pmid=17853878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) showed strong binding of putative masking/enhancing ligands to form MEK1⋅ATP(±Mg) and MEK1⋅ATPγS(±Mg<sup>2+</sup>) for model mutant substrate MEK1-C222. <b>(D)</b> MEK1 mutant proteins, in the absence of masking nucleotide / metal gave products of unwanted reaction at Cys207 with parent reagent DBHDA. MEK1-C218 (22&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, pH 8 buffer treated with DBHDA (4.4&nbsp;mM) at 37°C afforded overalkylated MEK1Dha218 (M<sub>calc.</sub>&nbsp;= 45862, M<sub>obs.</sub>&nbsp;= 45866) after 4 h; MEK1-C222 (22&nbsp;μM) in the same buffer treated with DBHDA (4.4&nbsp;mM) at 37°C afforded overalkylated MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45862, M<sub>obs.</sub>&nbsp;= 45867) after 4 h, MSMS of the tryptic peptide 206-227 confirmed the overalkylation at C207; MEK1-C218C222 (22&nbsp;μM) in the same buffer treated with DBHDA (4.4&nbsp;mM) at 37°C afforded overalkylated stapled MEK1-C218C222 (M<sub>calc.</sub>&nbsp;= 46052, M<sub>obs.</sub>&nbsp;= 46057) after 4 h. <b>(E)</b> Use of alternative alkylating agents did not prevent unwanted C207 reaction in MEK1-C222. MEK1-C222 (33&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, pH 8 buffer treated with DBDMet (6.6&nbsp;mM) at 37°C afforded a mixture 85% MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45642, M<sub>obs.</sub>&nbsp;= 45646) and 15% overalkylated MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45890, M<sub>obs.</sub>&nbsp;= 45891) after 3 h; MEK1-C222 (22&nbsp;μM) in the same buffer treated with DBDGla (4.4&nbsp;mM) at 37°C afforded overalkylated MEK1Dha222 (M<sub>calc.</sub>&nbsp;= 45982, M<sub>obs.</sub>&nbsp;= 45981) after 3 h; MEK1-C222 (33&nbsp;μM) in the same buffer treated with DBHDAc (6.6&nbsp;mM) at room temperature afforded substituted MEK1 (M<sub>calc.</sub>&nbsp;= 45979, M<sub>obs.</sub>&nbsp;= 45983) after 24 h. <b>(F)</b> Treatment of MEK1-C222 with DBHDA in the presence of ATP. MEK1-C222 (22&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ATP, pH 8 buffer treated with DBHDA (4.4&nbsp;mM) at 37°C afforded a mixture of 85% MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45642, M<sub>obs.</sub>&nbsp;= 45645) and 15% overalkylated MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45862, M<sub>obs.</sub>&nbsp;= 45866) after 6 h.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/9faf8f42-ee32-46a8-b1a6-aa5ad10cfdd8/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.04.001/asset/21a2ffbb-bcd0-4a0c-bc28-c0dc1c019f0a/main.assets/figs3.jpg" height="486" width="750" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/9faf8f42-ee32-46a8-b1a6-aa5ad10cfdd8/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Construction of MEK1-Dha222, related to <a id="crosref1660" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0195" role="paragraph"><b>(A)</b> Treatment of MEK1-C222 with DBHDA in the presence of ADP-Mg. MEK1-C222 (33&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with DBHDA (6.6&nbsp;mM) at 37°C afforded pure MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45642, M<sub>obs.</sub>&nbsp;= 45646); <b>(B)</b> LCMSMS of the tryptic/AspN peptides containing cysteine residues confirmed the formation of Dha exclusively at position 222. <b>(C)</b> MEK1-C222 (44&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with DBDGla (8.8&nbsp;mM) at 37°C afforded MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45642, M<sub>obs.</sub>&nbsp;= 45641) after 4 h; Dha position was confirmed by LC-MSMS of peptide 206-227; <b>(D)</b> MEK1-C222 (44&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with DIB (2000 equiv) at 37°C afforded MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45642, M<sub>obs.</sub>&nbsp;= 45645) after 20 h; Dha position was confirmed by LC-MSMS of peptide 207-227; <b>(E)</b> MEK1-C222 (44&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with MDBP (4.4&nbsp;mM) at 37°C afforded MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45642, M<sub>obs.</sub>&nbsp;= 45642) after 5 h, Dha position was confirmed by LC-MSMS of peptide 217-227.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/5b4b43f5-1dd3-48aa-9359-74692910d851/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.04.001/asset/756aa32a-c9fb-4084-b079-0b3a3e13c6b0/main.assets/figs4.jpg" height="618" width="750" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/5b4b43f5-1dd3-48aa-9359-74692910d851/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Construction of MEK1-Dha218, related to <a id="crosref1665" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0205" role="paragraph"><b>(A)</b> Treatment of MEK1-C218 with DBHDA in the presence of ADP-Mg. MEK1-C218 (22&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with DBHDA (4.4&nbsp;mM) at 37°C afforded a mixture of MEK1Dha218 (M<sub>calc.</sub>&nbsp;= 45642, M<sub>obs.</sub>&nbsp;= 45645) and 25% overalkylated MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45785, M<sub>obs.</sub>&nbsp;= 45785) after 15 h; <b>(B)</b> LC-MSMS of the tryptic peptide 206-227 confirmed the formation of Dha at position 218; <b>(C)</b> Longer reaction time lead to the formation of more adducts. <b>(D)</b> MEK1-C218 (44&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with DBDGla (4.4&nbsp;mM) at 37°C afforded exclusively MEK1-Dha218 (M<sub>calc.</sub>&nbsp;= 45642, M<sub>obs.</sub>&nbsp;= 45647) after 20 h, the position was confirmed by LC-MSMS analysis of peptide 206-227; <b>(E)</b> MEK1-C218 (44&nbsp;μM) in the same buffer treated with MDBP (4.4&nbsp;mM) at 37°C afforded exclusively MEK1-Dha218 (M<sub>calc.</sub>&nbsp;= 45642, M<sub>obs.</sub>&nbsp;= 45647) after 4 h, the position was confirmed by LC-MSMS analysis of peptide 206-227. <b>(F)</b> Treatment of Δ<sub>1-60</sub>MEK1-C218 with MDBP in the presence of ADP-Mg. Δ<sub>160</sub>MEK1-C218 (55&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with MDBP (5.5&nbsp;mM) at room temperature only afforded Δ<sub>160</sub>MEK1-Dha218 (M<sub>calc.</sub>&nbsp;= 36611, M<sub>obs.</sub>&nbsp;= 36616) after 14 h.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/93fa428f-64e3-4af1-968c-3310734dc4f1/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.04.001/asset/424a9c7f-1e31-4571-80fd-251f84b70ae0/main.assets/figs5.jpg" height="939" width="479" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/93fa428f-64e3-4af1-968c-3310734dc4f1/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Construction of MEK1-Dha218Dha222, related to <a id="crosref1670" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0215" role="paragraph"><b>(A)</b> Treatment of MEK1-C218C222 with DBHDA analogs in the presence of nucleotide-Mg. MEK1-C218C222 (22&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ATP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with DBHDA (4.4&nbsp;mM) at 37°C afforded a mixture 60% pMEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45738, M<sub>obs.</sub>&nbsp;= 45741), 20% MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45658, M<sub>obs.</sub>&nbsp;= 45661) and 20% phosphorylated stapled MEK1-C218C222 (M<sub>calc.</sub>&nbsp;= 45912, M<sub>obs.</sub>&nbsp;= 45914) after 5 h; <b>(B)</b> MEK1-C218C222 (22&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with DBHDA (4.4&nbsp;mM) at 37°C afforded a mixture 50% MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45658, M<sub>obs.</sub>&nbsp;= 45658) and 50% stapled MEK1C218C222 (M<sub>calc.</sub>&nbsp;= 45832, M<sub>obs.</sub>&nbsp;= 45831) after 5 h; <b>(C)</b> MEK1-C218C222 (44&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with DBHDA (8.8&nbsp;mM) at 37°C afforded a mixture 23% MEK1-Dha218Dha222 (M<sub>calc.</sub>&nbsp;= 45624, M<sub>obs.</sub>&nbsp;= 45627), 18% MEK1-Dha222 (M<sub>calc.</sub>&nbsp;= 45658, M<sub>obs.</sub>&nbsp;= 45668) and 59% stapled MEK1-C218C222 (M<sub>calc.</sub>&nbsp;= 45946, M<sub>obs.</sub>&nbsp;= 45951) after 14 h. <b>(D)</b> Treatment of MEK1-C218C222 with MDBP in the presence of ADP-Mg. MEK1-C218C222 (22&nbsp;μM) in 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 10&nbsp;mM ADP, 25&nbsp;mM MgCl<sub>2</sub>, pH 8 buffer treated with MDBP (4.4&nbsp;mM) at room temperature afforded exclusively MEK1-Dha218Dha222 (M<sub>calc.</sub>&nbsp;= 45624, M<sub>obs.</sub>&nbsp;= 45628) after 15 h; <b>(E)</b> LC-MSMS of the tryptic peptide 206-227 confirmed the formation of Dha at position 218 and 222. <b>(F)</b> Comparative circular dichroism measurement of MEK1-Cs and MEK1-Dhas.</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph">The enzymatic activities of these “eliminated” MEK1 variants on the native substrate protein ERK (both mono- (k<sub>f1</sub>) and di-phosphorylation (k<sub>f2</sub>)) were assessed directly by real-time monitoring using total protein ESI-MS (<a id="crosref0615" href="#figs6">Figure&nbsp;S6</a>). The ERK1-K71R mutant that cannot autophosphorylate (<span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Robbins et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Robbins, D.J. ∙ Zhen, E. ∙ Owaki, H. ...</div><div><strong>Regulation and properties of extracellular signal-regulated protein kinases 1 and 2 in&nbsp;vitro</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1993; <strong>268</strong>:5097-5106</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2FS0021-9258%2818%2953507-9&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2FS0021-9258%2818%2953507-9&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8444886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2953507-9&amp;pmid=8444886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was utilized to allow cleaner analysis of ERK by MS. Strikingly, all Dha-containing MEK1 variants showed <i>activated</i> phosphorylation kinetics over WT MEK1. Moreover, His<sub>6</sub>-MEK1-Dha218 and His<sub>6</sub>-MEK1-Dha218Dha222 displayed similar or even higher activity than “constitutively active” MEK1-Ser→Glu mutants (<a id="crosref0625" href="#mmc2">Table S2</a>; <a id="crosref0630" href="#sec-4">STAR Methods</a>). Notably, despite different behavior as a kinase, elimination in an alternative construct His<sub>6</sub>-Δ<sub>1–60</sub>-MEK1-Dha218 (which displays altered selectivity due to deletion of the ERK-docking domain [<span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Xu et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Xu, B.e. ∙ Wilsbacher, J.L. ∙ Collisson, T. ...</div><div><strong>The N-terminal ERK-binding site of MEK1 is required for efficient feedback phosphorylation by&nbsp;ERK2 in&nbsp;vitro and ERK activation in&nbsp;vivo</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1999; <strong>274</strong>:34029-34035</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.274.48.34029&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.274.48.34029&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0033607522" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10567369/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.274.48.34029&amp;pmid=10567369" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)) also caused significant activation, suggesting a general function of manipulation of the activation loop of MEK1 and not peculiar artifacts of one kinase structure. Notably, His<sub>6</sub>-MEK1-S218A and His<sub>6</sub>-MEK1-S222A variants of MEK1 are <i>inactive</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Alessi et&nbsp;al., 1994</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alessi, D.R. ∙ Saito, Y. ∙ Campbell, D.G. ...</div><div><strong>Identification of the sites in MAP kinase kinase-1 phosphorylated by p74raf-1</strong></div><div><em>EMBO J.</em> 1994; <strong>13</strong>:1610-1619</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/j.1460-2075.1994.tb06424.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0028293931" target="_blank" aria-label="Open Scopus in new tab">Scopus (474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8157000/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fj.1460-2075.1994.tb06424.x&amp;pmid=8157000" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), although they differ only from the activated His<sub>6</sub>-MEK1-Dha218 and His<sub>6</sub>-MEK1-Dha222 in unsaturation. This precisely implicates the presence of the sp<sup>2</sup> center at the α carbon of Dha218 and Dha222 in this activation, thereby highlighting the eliminylome as a putative source of unwanted dysregulation through activation. This reasoning is also consistent with the greater effect seen at the α-helical site 218 (cf<i>.</i> site 222, which is in a flexible loop). Any such conformational effects were too localized to be detected by global measures, such as circular dichroism (<a id="crosref0645" href="#figs5">Figure&nbsp;S5</a>F).</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/d647826b-5433-414c-9279-7efa6452a19c/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.04.001/asset/9e392dcf-38a7-44ec-935a-b9d714b3b3f3/main.assets/figs6.jpg" height="939" width="413" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/d647826b-5433-414c-9279-7efa6452a19c/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Kinase activity assay of MEK1, related to <a id="crosref1675" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0225" role="paragraph"><b>(A)</b> Sequential and distributive proposed mechanism of ERK1 phosphorylation by MEK1. <b>(B)</b> The phosphorylation of ERK1K71R by MEK1 construct was monitored by LCMS. The intensities of the deconvoluted peaks corresponding to ERK1 (M<sub>calc.</sub>&nbsp;= 44058), pERK1 (M<sub>calc.</sub>&nbsp;= 44138), and ppERK1 (M<sub>calc.</sub>&nbsp;= 44218) were recorded over time. The conversions were calculated from these intensities and plotted. <b>(C)</b> Differential equations define the phosphorylation constants k<sub>f1</sub> and k<sub>f2</sub>. These constants were calculated by fitting the analytical equations. <b>(D)</b> ESI-MS progress curves allowing time course analysis of ERK1 by MEK1 variants. ERK1-K71R was incubated with MEK1 variant in 50&nbsp;mM Tris, 30&nbsp;mM NaCl, 1&nbsp;mM ATP, 10&nbsp;mM MgCl<sub>2</sub>, 0.2&nbsp;mM DTT, pH 7.5 at 30°C.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-6"><h3>LanCLs add GSH to dehydroamino acids in representative kinases and deactivate dysregulated MEK1</h3><div id="p0115" role="paragraph">Next, we tested LanCL activity against full-length MEK1 proteins. First, we investigated the interaction between LanCL proteins and&nbsp;dehydroamino-acid-containing MEK1. MBP-tagged LanCL2 was expressed in bacterial cells and purified. The MBP-tagged LanCL2 was incubated with His<sub>6</sub>-MEK1-Dha218 and pulled down by cobalt resin. The pull-down assay shows that LanCL2 has a direct physical interaction with His<sub>6</sub>-MEK1-Dha218 (<a id="crosref0650" href="#fig5">Figure&nbsp;5</a>A).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/38d0103d-2825-49e0-8e5e-fc032d5d54d8/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.04.001/asset/34c36566-0275-45e6-995f-5431f549fd32/main.assets/gr5.jpg" height="750" width="750" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/38d0103d-2825-49e0-8e5e-fc032d5d54d8/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">LanCL proteins catalyze non-canonical <i>C-</i>glutathionylation of dehydroamino-acid-containing proteins</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0110" role="paragraph">(A) <i>In&nbsp;vitro</i> binding analysis of MBP-LanCL2 with His<sub>6</sub>-MEK1-Dha218 by His-pull-down assay.</div><div id="fspara0115" role="paragraph">(B) LanCL1 and LanCL2 (2&nbsp;μM) <i>C</i>-glutathionylate His<sub>6</sub>-MEK1-Dha218. The conversions of Dha-MEK1 (2&nbsp;μM) to GS-MEK1 after 40-min incubation with 1.0&nbsp;mM GSH at 25°C (Dha-MEK1: LanCL&nbsp;= 1:1) are shown as deconvoluted LC mass spectra.</div><div id="fspara0120" role="paragraph">(C) Time course of GSH (2.5&nbsp;mM) addition to MEK1-Dha218 (6&nbsp;μM) catalyzed by LanCL1/2 (1.2&nbsp;μM). The formation of GS-MEK1 was monitored according to the intensities of each product from deconvoluted spectra on LCMS.</div><div id="fspara0125" role="paragraph">(D) Pseudo-single substrate Michaelis-Menten plot in the presence of LanCL1 or LanCL2 (Dha-MEK1, 6&nbsp;μM; LanCL1/2, 1.2&nbsp;μM). The results of kinetic assays are shown as mean ± standard deviation of triplicate experiments. Initial reaction velocity at different GSH concentrations was calculated by linear regression fit to the Michaelis-Menten equation of the formation of GS-MEK1 over time using OriginPro9.7 for apparent kinetic parameters <i>K</i><sub>M</sub> and <i>k</i><sub>cat</sub></div><div id="fspara0130" role="paragraph">For spectra and data with Dhb-ERK2, see <a id="crosref1625" href="#figs7">Figure&nbsp;S7</a>; for kinase activity of Dha- and GS-MEK1, see <a id="crosref1630" href="#figs6">Figure&nbsp;S6</a>.</div></div></div></div></figcaption></figure></div><div id="p0120" role="paragraph">Second, we investigated <i>C-</i>glutathionylation activity. The Dha218-containing MEK1 was treated with WT LanCL1/2 and GSH, and the reactions were monitored by MS. A GSH adduct was observed, whereas no reactions occurred with LanCL mutants (<a id="crosref0655" href="#fig5">Figures&nbsp;5</a>B and 5C; <a id="optzdXYvW9zvk" href="#figs7">Figure S7</a>). Kinetic analyses using intact protein MS were performed in the presence of increasing amounts of GSH, and initial rates of forming glutathione conjugate were plotted against substrate concentration. The data demonstrated that <i>C</i>-glutathionylation of His<sub>6</sub>-MEK1-Dha218 was catalyzed by LanCL1 and LanCL2, with LanCL1 exhibiting a higher catalytic activity than LanCL2 (<a id="crosref0660" href="#fig5">Figures 5</a>C and 5D; <a id="crosref0665" href="#figs7">Figure S7</a>). Pseudo-single substrate plots of initial reaction velocity versus GSH concentration and non-linear Michaelis-Menten regression analyses (<a id="crosref0670" href="#fig5">Figure&nbsp;5</a>D) provided apparent <i>K</i><sub>M</sub> and <i>k</i><sub>cat</sub> values (<a id="crosref0675" href="#tbl1">Table 1</a>).</div><div class="figure-wrap"><figure id="tbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th>Substrate</th><th>Enzyme</th><th><i>K</i><sub>M</sub> (μM)</th><th><i>k</i><sub>cat</sub> (min<sup>–1</sup>)</th><th>V<sub>max</sub> (μM/min)</th></tr></thead><tbody><tr><td>MEK-Dha218<br></td><td>LanCL1<br></td><td>28.1 ± 6.1<a id="crosref1695" href="#tblfn1"><sup>a</sup></a><br></td><td>0.15 ± 0.01<a id="crosref1705" href="#tblfn1"><sup>a</sup></a><br></td><td>0.18 ± 0.01<a id="crosref1715" href="#tblfn1"><sup>a</sup></a><br></td></tr><tr><td>Dhb-ERK peptide</td><td>LanCL1</td><td>115 ± 21<a id="crosref1725" href="#tblfn2"><sup>b</sup></a></td><td>7.7 ± 0.5</td><td>3.9 ± 0.3</td></tr><tr><td>Dhb-ERK peptide</td><td>LanCL2</td><td>210 ± 75<a id="crosref1730" href="#tblfn2"><sup>b</sup></a></td><td>10.1 ± 1.9</td><td>5.0 ± 1.0</td></tr><tr><td>Dhb-CylLs″</td><td>LanCL2</td><td>250 ± 51<a id="crosref1735" href="#tblfn2"><sup>b</sup></a></td><td>11.8 ± 1.2</td><td>5.9 ± 0.6</td></tr></tbody></table></div><figcaption><div class="caption"><span class="heading">Table 1</span><div id="tspara0010" role="paragraph">Kinetic parameters of addition of GSH to various substrates catalyzed by LanCL1/2</div></div><div class="notes"><div role="doc-footnote" data-has="label"><div class="label">a</div><div id="tblfn1"><div id="ntpara0055" role="paragraph">Apparent <i>K</i><sub>m,GSH</sub> and <i>k</i><sub>cat</sub> values at fixed [MEK-Dha218] of 6.0&nbsp;μM.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">b</div><div id="tblfn2"><div id="ntpara0060" role="paragraph"><i>K</i><sub>m,peptides</sub> at 1 mM GSH. The error bars are derived from fits of the data from triplicate experiments by global least-square regression to solutions of the Michaelis Menten equation using OriginPro9.7 or PRISM 6 (GraphPad).</div></div></div></div><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=tbl1&amp;pii=S0092-8674%2821%2900436-0" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div><div id="p0125" role="paragraph">Third, to test other targets, we also investigated the addition of GSH to Dha-containing ERK (chemically generated in a similar manner to that used for eliminated MEK1; see <a id="crosref0680" href="#sec-4">STAR Methods</a>). GSH addition experiments were conducted using ERK1-Dha202. Compared with controls (background and mutant LanCL proteins), higher percentages of ERK1-GS202 adducts were formed in samples treated with WT LanCL1 or LanCL2 (<a id="crosref0685" href="#figs7">Figure&nbsp;S7</a>). Some background reactivity of ERK1-Dha202 was observed, and therefore we also generated the Dhb version of ERK2 (ERK2-Dhb185) by phosphorylating ERK2 using MEK-Dha218Dha222 and then using the pThr lyase OspF to eliminate the pThr (see <a id="crosref0690" href="#sec-4">STAR Methods</a>). Dhb-containing peptides and proteins showed negligible background reaction with GSH, but LanCL2 again catalyzed addition of GSH to ERK2-Dhb185 as demonstrated by MS and western blot analysis (<a id="crosrefs0015" href="#figs1">Figures S1</a>J and <a id="crosref0695" href="#figs7">S7</a>C). Interestingly, a doubly eliminated variant of ERK1 (ERK1-Dha82Dha202, created in the same manner as for ERK1-Dha202) was <i>C-</i>glutathionylated once by LanCL2 with essentially the same kinetics as ERK1-Dha202, suggesting that LanCL selectively addresses elimination at the natural phosphosite 202 but not when found at non-phosphorylated site 82. Taken together, our data demonstrate that LanCL proteins not only catalyze GSH addition to Dhb-containing peptides but also catalyze glutathionylation of Dha/Dhb-containing MEK1 and ERK1/2.</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/16fbeded-9ade-4158-8766-94ed33b4ae6d/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.04.001/asset/57992bb5-7bea-45dc-a878-a21d6850e2e0/main.assets/figs7.jpg" height="860" width="750" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/16fbeded-9ade-4158-8766-94ed33b4ae6d/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">LanCL proteins catalyze <i>C-</i>glutathionylation of MEK-Dha and ERK2-Dha, related to <a id="crosref1685" href="#fig5">Figure&nbsp;5</a> and <a id="crosref1690" href="#tbl1">Table 1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0235" role="paragraph">(<b>A</b>) End-point assays of LanCL1 and LanCL2 (6.0&nbsp;μM) with His<sub>6</sub>- MEK1-Dha218 as the substrate for glutathionylation. The conversions of MEK1-Dha218 (6&nbsp;μM) to MEK1-GS218 after 2-h incubation with 1.0&nbsp;mM GSH at 25°C were shown by deconvoluted spectra of LC-MS. (<b>B</b>) Wild-type, but not mutant, LanCL catalyze C-S glutathionylation of ERK1-Dha202. The Michael addition of 0.5&nbsp;mM GSH to Dha-containing ERK protein (6&nbsp;μM) was monitored by LC-MS after 3-h incubation with LanCL proteins (6.0&nbsp;μM) at 25°C. Top right inset: Time-dependent GSH addition to ERK1-Dha202 catalyzed by wild-type (LanCL1 or LanCL2) or mutant LanCL proteins (LanCL1-C322A or LanCL2-C321A/C367A). The conversion to ERK1-GS202 was quantified using intact mass LC-ESI-MS and is plotted here as the value over background chemical GSH addition (~0.13 M<sup>-1</sup> s<sup>-1</sup> under these conditions) in the absence of enzyme. (<b>C</b>) LanCL2 adds GSH to GST-3XFlag-ERK2-Dhb185pY187 generated using the pThr lyase OspF. (<b>D</b>) Kaplan-Meier survival curve of female WT (n&nbsp;= 33) and LanCL TKO (n&nbsp;= 27) FVB mice (p&nbsp;= 0.0052) and male WT (n&nbsp;= 25) and LanCL TKO (n&nbsp;= 20) FVB mice (p&nbsp;= 0.0332). Gehan-Breslow-Wilcoxon P test. (<b>E</b>) Survival curve of age-matched FVB WT mice (n&nbsp;= 24) and LanCL2&nbsp;−/− mice (n&nbsp;= 50).</div></div></div></div></figcaption></figure></div><div id="p0130" role="paragraph">The observed LanCL activity generates a carbon-sulfur thioether bond between GSH and the protein (<i>C-</i>glutathionylation) as opposed to the well-known (reversible) <i>S-</i>glutathionylation of Cys residues via disulfide linkages (<span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Cooper et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cooper, A.J. ∙ Pinto, J.T. ∙ Callery, P.S.</div><div><strong>Reversible and irreversible protein glutathionylation: biological and clinical aspects</strong></div><div><em>Expert Opin. Drug Metab. Toxicol.</em> 2011; <strong>7</strong>:891-910</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1517/17425255.2011.577738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79959221560" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21557709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1517%2F17425255.2011.577738&amp;pmid=21557709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Dalle-Donne et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Dalle-Donne, I. ∙ Rossi, R. ∙ Colombo, G. ...</div><div><strong>Protein S-glutathionylation: a regulatory device from bacteria to humans</strong></div><div><em>Trends Biochem. Sci.</em> 2009; <strong>34</strong>:85-96</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-59849129012" target="_blank" aria-label="Open Scopus in new tab">Scopus (545)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19135374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tibs.2008.11.002&amp;pmid=19135374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Grek et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Grek, C.L. ∙ Zhang, J. ∙ Manevich, Y. ...</div><div><strong>Causes and consequences of cysteine S-glutathionylation</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2013; <strong>288</strong>:26497-26504</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.R113.461368&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.R113.461368&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84884179149" target="_blank" aria-label="Open Scopus in new tab">Scopus (254)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23861399/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.R113.461368&amp;pmid=23861399" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-1" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-1" aria-controls="sref74-1">Yang et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref74-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Yang, J. ∙ Carroll, K.S. ∙ Liebler, D.C.</div><div><strong>The expanding landscape of the thiol redox proteome</strong></div><div><em>Mol. Cell. Proteomics.</em> 2016; <strong>15</strong>:1-11</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O115.056051&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O115.056051&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84955506189" target="_blank" aria-label="Open Scopus in new tab">Scopus (163)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26518762/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.O115.056051&amp;pmid=26518762" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As such, the non-canonical glutathionylation activity demonstrated here produces an <i>irreversible</i> modification of dehydroamino acids in proteins (<span class="dropBlock reference-citations"><a id="body-ref-sref12-3" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-3" aria-controls="sref12-3">Cooper et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref12-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cooper, A.J. ∙ Pinto, J.T. ∙ Callery, P.S.</div><div><strong>Reversible and irreversible protein glutathionylation: biological and clinical aspects</strong></div><div><em>Expert Opin. Drug Metab. Toxicol.</em> 2011; <strong>7</strong>:891-910</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1517/17425255.2011.577738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79959221560" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21557709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1517%2F17425255.2011.577738&amp;pmid=21557709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0135" role="paragraph">Notably, the GSH adducts of MEK1 showed reduced kinase activity compared to their Dha-containing counterparts (<a id="opth1aXzlQ0Gb" href="#figs6">Figure S6</a>; <a id="crosref0725" href="#mmc2">Table S2</a>) suggesting the possibility of not only an effective scavenging/trapping pathway to remove reactive electrophiles but also one that directly reduces kinase activity dysregulation caused by elimination (see above).</div></section><section id="sec-2-7"><h3>LanCL proteins have plastic but selective substrate specificity</h3><div id="p0140" role="paragraph">Based on the data with the ERK and Akt peptides and the MEK and ERK proteins, LanCLs appear to have considerable substrate tolerance with respect to the sequences in which the dehydroamino acids can be embedded. To further investigate the substrate scope, we assessed the reactivity of several bacterial peptides with LanCL2. We first focused on a prochlorosin peptide (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Li et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Li, B. ∙ Sher, D. ∙ Kelly, L. ...</div><div><strong>Catalytic promiscuity in the biosynthesis of cyclic peptide secondary metabolites in planktonic marine cyanobacteria</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2010; <strong>107</strong>:10430-10435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0913677107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-77953742146" target="_blank" aria-label="Open Scopus in new tab">Scopus (226)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20479271/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0913677107&amp;pmid=20479271" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (ProcA 2.8-mut, <a id="crosref0735" href="#fig6">Figure&nbsp;6</a>A) that contains a single Dha and reacted it with WT or mutant LanCL2 proteins. MALDI-TOF MS analysis demonstrated glutathionylation with WT LanCL2 protein but not with mutant LanCL2 (<a id="crosref0740" href="#fig6">Figure&nbsp;6</a>A). Next, we used the cytolysin peptides (CylL<sub>L</sub>″ and CylL<sub>S</sub>″) that are virulence factors in <i>Enterococcus faecalis</i> and that contain several dehydroamino acids and lanthionine rings (<a id="crosref0745" href="#fig6">Figure&nbsp;6</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Duan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Duan, Y. ∙ Llorente, C. ∙ Lang, S. ...</div><div><strong>Bacteriophage targeting of gut bacterium attenuates alcoholic liver disease</strong></div><div><em>Nature.</em> 2019; <strong>575</strong>:505-511</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1742-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85074974768" target="_blank" aria-label="Open Scopus in new tab">Scopus (512)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31723265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1742-x&amp;pmid=31723265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Tang and van der Donk, 2013</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Tang, W. ∙ van der Donk, W.A.</div><div><strong>The sequence of the enterococcal cytolysin imparts unusual lanthionine stereochemistry</strong></div><div><em>Nat. Chem. Biol.</em> 2013; <strong>9</strong>:157-159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nchembio.1162" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84874115767" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23314913/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnchembio.1162&amp;pmid=23314913" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CylL<sub>S</sub>″ contains two lanthionine rings and two Dhb residues. LanCL2 added one GSH, whereas no adduct was formed by a LanCL2 mutant (<a id="crosref0760" href="#fig6">Figure&nbsp;6</a>B). MS/MS analysis showed that the glutathionylation site on CylL<sub>S</sub>″ was Dhb7 that is outside of the lanthionine rings (<a id="crosref0765" href="#figs1">Figure&nbsp;S1</a>K). CylL<sub>L</sub>″ contains a Dha and Dhb located within rings and two Dhb residues that are outside of rings (<a id="crosref0770" href="#fig6">Figure&nbsp;6</a>C). LanCL2 catalyzed glutathionylation of the two Dhb residues located outside of lanthionine rings (Dhb10 and Dhb30; <a id="crosref0775" href="#fig6">Figure&nbsp;6</a>C; <a id="crosref0780" href="#figs1">Figure&nbsp;S1</a>K). Lanthionine rings might prevent LanCL2 from adding GSH to dehydroamino acid residues within rings through steric hindrance.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/cdad30fd-8167-43c7-8fc4-b67941a8088a/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.04.001/asset/499c1e6f-3a33-4d47-9a25-4b7fd882d70e/main.assets/gr6.jpg" height="939" width="650" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/cdad30fd-8167-43c7-8fc4-b67941a8088a/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">LanCL-catalyzed <i>C</i>-glutathionylation shows flexible substrate selectivity and high stereoselectivity and is the major <i>C</i>-glutathionylation activity in cells, and under certain circumstances the absence of LanCL proteins leads to lethality</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0140" role="paragraph">(A–C) LanCL2, but not an active site variant, adds GSH to (A) ProcA 2.8 mut, (B) CylL<sub>S</sub>″, and (C) CylL<sub>L</sub>″. Eliminated (Dha or Dhb) residues are highlighted in red; Cys residues involved in thioether linkages are in green.</div><div id="fspara0145" role="paragraph">(D) Comparative activity of LanCL2-His<sub>6</sub> and His<sub>6</sub>-GSTA4 with Dhb-containing ERK peptide.</div><div id="fspara0150" role="paragraph">(E) Non-canonical glutathionylation was observed with His<sub>6</sub>-MEK1-Dha222 when incubated with WT MEF cell extracts. No glutathionylation was observed when His<sub>6</sub>-MEK1-Dha222 was treated with cell extract of LanCL1–3 triple knockout (TKO) MEF cells.</div><div id="fspara0155" role="paragraph">(F) Structure of DL- and LL-MeLan.</div><div id="fspara0160" role="paragraph">(G) GC-MS analysis of the methyllanthionine in glutathionylated Dhb-ERK peptide. Deep blue: DL-MeLan synthetic standard; red: LL-MeLan synthetic standard; cyan: derivatized residue from GS-ERK peptide; co-injection of derivatized MeLan from the GS-ERK peptide with DL standard (orange) or LL standard (pink).</div><div id="fspara0165" role="paragraph">(H) Kaplan-Meier survival curve of WT (n&nbsp;= 58) and LanCL TKO (n&nbsp;= 47) FVB mice. Censored individual mice are shown as a tick mark (see <a id="crosref1635" href="#sec-4">STAR Methods</a>). Gehan-Breslow-Wilcoxon test showed a p value of 0.0004. Statistical significance was accepted at p&nbsp;&lt; 0.05.</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">We also investigated other dehydroamino-acid-containing proteins. Glutathione peroxidase (GPX) contains a selenocysteine in its active site that is important for catalysis but is prone to oxidative elimination generating Dha (<a id="optmx1jz93fPJ" href="#figs1">Figure S1</a>G) (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Cho et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Cho, C.S. ∙ Lee, S. ∙ Lee, G.T. ...</div><div><strong>Irreversible inactivation of glutathione peroxidase 1 and reversible inactivation of peroxiredoxin II by H<sub>2</sub>O<sub>2</sub> in red blood cells</strong></div><div><em>Antioxid. Redox Signal.</em> 2010; <strong>12</strong>:1235-1246</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ars.2009.2701" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-77951813369" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20070187/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Fars.2009.2701&amp;pmid=20070187" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Ma et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Ma, S. ∙ Caprioli, R.M. ∙ Hill, K.E. ...</div><div><strong>Loss of selenium from selenoproteins: conversion of selenocysteine to dehydroalanine in&nbsp;vitro</strong></div><div><em>J.&nbsp;Am. Soc. Mass Spectrom.</em> 2003; <strong>14</strong>:593-600</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S1044-0305(03)00141-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0037811713" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12781460/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1044-0305%2803%2900141-7&amp;pmid=12781460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Wang et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Wang, S.K. ∙ Weaver, J.D. ∙ Zhang, S. ...</div><div><strong>Knockout of SOD1 promotes conversion of selenocysteine to dehydroalanine in murine hepatic GPX1 protein</strong></div><div><em>Free Radic. Biol. Med.</em> 2011; <strong>51</strong>:197-204</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.freeradbiomed.2011.03.018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79957968064" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21420488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.freeradbiomed.2011.03.018&amp;pmid=21420488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). LanCL1/2 were unable to add GSH to the Dha-containing GPX, presumably because the Dha is in a protected active site pocket.</div><div id="p0150" role="paragraph">Mammalian systems contain multiple glutathione <i>S-</i>transferases (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Wu and Dong, 2012</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Wu, B. ∙ Dong, D.</div><div><strong>Human cytosolic glutathione transferases: structure, function, and drug discovery</strong></div><div><em>Trends Pharmacol. Sci.</em> 2012; <strong>33</strong>:656-668</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tips.2012.09.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tips.2012.09.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84869236062" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23121834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tips.2012.09.007&amp;pmid=23121834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The glutathione transferase A4 (GSTA4) has been demonstrated to catalyze GSH addition to another α,β-unsaturated compound, 4-hydroxynonenal (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Bruns et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bruns, C.M. ∙ Hubatsch, I. ∙ Ridderström, M. ...</div><div><strong>Human glutathione transferase A4-4 crystal structures and mutagenesis reveal the basis of high catalytic efficiency with toxic lipid peroxidation products</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1999; <strong>288</strong>:427-439</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jmbi.1999.2697" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0033531970" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10329152/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjmbi.1999.2697&amp;pmid=10329152" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Hubatsch et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Hubatsch, I. ∙ Ridderström, M. ∙ Mannervik, B.</div><div><strong>Human glutathione transferase A4-4: an alpha class enzyme with high catalytic efficiency in the conjugation of 4-hydroxynonenal and other genotoxic products of lipid peroxidation</strong></div><div><em>Biochem. J.</em> 1998; <strong>330</strong>:175-179</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1042/bj3300175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0032519517" target="_blank" aria-label="Open Scopus in new tab">Scopus (335)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9461507/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1042%2Fbj3300175&amp;pmid=9461507" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because GSTA4 catalyzes Michael-type addition, we tested its activity with Dhb-containing ERK peptide in parallel with LanCL2 and the reaction mixtures were monitored by MALDI-TOF MS. GSH addition was observed with LanCL2-His<sub>6</sub> treatment; however, no glutathionylation was observed with His<sub>6</sub>-GSTA4 (<a id="crosref0820" href="#fig6">Figure&nbsp;6</a>D). These data suggest that LanCL proteins might be specialized in catalyzing GSH addition to eliminated (Dha- and Dhb-containing) peptides or proteins.</div><div id="p0155" role="paragraph">To test whether LanCLs have substrate preference, we compared the kinetics of LanCL2 with Dhb-ERK peptide and CylL<sub>S</sub>″. Two-substrate steady-state kinetic analysis was performed with saturating GSH at a concentration similar to that found in cells (1&nbsp;mM) (<span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Meister and Anderson, 1983</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Meister, A. ∙ Anderson, M.E.</div><div><strong>Glutathione</strong></div><div><em>Annu. Rev. Biochem.</em> 1983; <strong>52</strong>:711-760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.bi.52.070183.003431" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0020972417" target="_blank" aria-label="Open Scopus in new tab">Scopus (6229)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6137189/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.bi.52.070183.003431&amp;pmid=6137189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The production of glutathionylated product was monitored and quantified by LC/MS. The <i>K</i><sub>M</sub> and <i>k</i><sub>cat</sub> values observed for the Dhb-ERK and CylL<sub>S</sub>″ substrates are similar (<a id="crosref0830" href="#tbl1">Table 1</a>) suggesting that LanCL2 does not have a clear sequence preference and Dha/Dhb might serve as the recognition feature. LanCL1 displayed similar kinetic parameters as LanCL2 with the Dhb-ERK peptide (<a id="crosref0835" href="#tbl1">Table 1</a>).</div></section><section id="sec-2-8"><h3>Other cellular proteins do not add glutathione to dehydroamino acids</h3><div id="p0160" role="paragraph">To test whether endogenous LanCL adds GSH to MEK1-Dha222, the latter protein was incubated with cell lysates from WT or LanCL1–3 TKO mouse embryonic fibroblast (MEF) cells (<span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">He et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>He, C. ∙ Zeng, M. ∙ Dutta, D. ...</div><div><strong>LanCL proteins are not Involved in Lanthionine Synthesis in Mammals</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:40980</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep40980" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85010403537" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28106097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep40980&amp;pmid=28106097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The primary MEF cells were isolated and cultured as described in the <a id="crosref0845" href="#sec-4">STAR Methods</a>, and expression of LanCL1 and LanCL2 in WT MEF cells but not TKO MEF cells was observed by western blotting (<a id="crosref0850" href="#fig6">Figure&nbsp;6</a>E). Equal numbers of WT and TKO MEF cells were collected, lysed, and incubated with His<sub>6</sub>-MEK1-Dha222. Cobalt-bound resin was used to enrich the His-tagged MEK1 protein, and the protein was glutathionylated in the presence of WT MEF cell extract but not in the presence of TKO MEF cell extract as detected by western blot with an anti-GSH antibody (<a id="crosref0855" href="#fig6">Figure&nbsp;6</a>E; <a id="crosref0860" href="#figs1">Figure S1</a>L). These data suggest that endogenous LanCL proteins are the major, if not only, enzymes that can catalyze glutathione addition to His<sub>6</sub>-MEK1-Dha222 in cells.</div></section><section id="sec-2-9"><h3>Glutathione addition by LanCL is stereoselective</h3><div id="p0165" role="paragraph">Methyllanthione (MeLan) formed by LanC enzymes typically has the DL-configuration (<a id="crosref0865" href="#fig6">Figure&nbsp;6</a>F) (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Chatterjee et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Chatterjee, C. ∙ Paul, M. ∙ Xie, L. ...</div><div><strong>Biosynthesis and mode of action of lantibiotics</strong></div><div><em>Chem. Rev.</em> 2005; <strong>105</strong>:633-684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/cr030105v" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-14844340728" target="_blank" aria-label="Open Scopus in new tab">Scopus (644)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15700960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fcr030105v&amp;pmid=15700960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), in which the α carbon from the former Thr residue possesses the D-configuration, whereas the α carbon on the Cys retains the L-configuration. In order to determine the stereochemistry of the GSH adduct generated by LanCL2, the glutathionylated ERK peptide was hydrolyzed and derivatized. The derivatized sample was analyzed by gas chromatography-mass spectrometry (GC/MS) with a chiral stationary phase. The MeLan residue in the glutathionylated peptide has the (2<i>S</i>, 3<i>S</i>, 6<i>R</i>) configuration as shown by coinjections with synthetic MeLan standards (<span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Liu et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Liu, W. ∙ Chan, A.S.H. ∙ Liu, H. ...</div><div><strong>Solid supported chemical syntheses of both components of the lantibiotic lacticin 3147</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2011; <strong>133</strong>:14216-14219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja206017p" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-80052580657" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21848315/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja206017p&amp;pmid=21848315" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Tang and van der Donk, 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Tang, W. ∙ van der Donk, W.A.</div><div><strong>Structural characterization of four prochlorosins: a novel class of lantipeptides produced by planktonic marine cyanobacteria</strong></div><div><em>Biochemistry.</em> 2012; <strong>51</strong>:4271-4279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi300255s" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84861624891" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22574919/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi300255s&amp;pmid=22574919" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0885" href="#fig6">Figure&nbsp;6</a>G). The data indicate that LanCL2 catalyzes stereoselective GSH addition to Dhb forming a D-stereocenter, which may contribute to the aforementioned deactivation of MEK1-Dha218 upon glutathionylation by creating non-natural stereochemistry.</div></section><section id="sec-2-10"><h3>Survey of the eliminylome from proteomics databases and MEF cells</h3><div id="p0170" role="paragraph">To examine the extent of the kinase-associated eliminylome, we conducted analyses of published proteomics data on human cell kinomes (<span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Médard et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Médard, G. ∙ Pachl, F. ∙ Ruprecht, B. ...</div><div><strong>Optimized chemical proteomics assay for kinase inhibitor profiling</strong></div><div><em>J.&nbsp;Proteome Res.</em> 2015; <strong>14</strong>:1574-1586</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/pr5012608" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84924371112" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25660469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fpr5012608&amp;pmid=25660469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0895" href="#mmc3">Table S3</a>). These analyses confirmed that Dha and Dhb are detected in many proteins. Among these are at least 30 protein kinases (MAPK1, MAPK3, MAPK8, MAPK9, MAPK10, MAPK14, MAP2K3/6, MAP3K19, MAP4K5, PRKAA1, CAMK2B, CAMK2D, CAMK2G, PRKX/Y, CDK5, PTK2, PTK2B, GSK3B, Src, Akt1/2, Ribosomal protein S6 kinase, Stk3, MRCKβ, N1, Nek9, TBK1, CSK, Fer, Fes, Lck, Lyn). About ∼50% of these Dha/Dhb sites map to corresponding Ser/Thr sites that have been observed to be phosphorylated (including MAPK1, MAPK14, MAPK3, MAP3K19, MAP2K3/6, CAMK2D, PTK2, Ribosomal protein S6 kinase, MRCKβ, N1, Nek9, TBK1, Lck, Lyn). Another smaller number of Dha and Dha adducts detected in the proteomic databases are at Cys sites suggesting that elimination also takes place at these residues (<a id="crosref0900" href="#mmc3">Table S3</a>).</div><div id="p0175" role="paragraph">We attempted to detect dehydroamino-acid-containing proteins by incubating lysates of TKO and WT MEF cells with GSH and LanCL1/2 and conducting western blot analysis with anti-GSH antibodies. No clear differences were observed. We attribute the absence of an abundance of dehydroamino-acid-containing proteins in the TKO MEF cells to the expected low rate and hence low occurrence of elimination as well as the relatively short time points (days) of these cell-based assays. Therefore, the eliminylome is not anticipated to be highly abundant and catalytic effects due to misregulation of enzymatic signaling activity such as shown for MEK1 may be more detrimental than stoichiometric effects through the formation of electrophiles in the proteome.</div></section><section id="sec-2-11"><h3>LanCL1/2/3 TKO mice are at risk of premature death</h3><div id="p0180" role="paragraph">We previously reported that TKO mice did not display any obvious phenotype (<span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">He et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>He, C. ∙ Zeng, M. ∙ Dutta, D. ...</div><div><strong>LanCL proteins are not Involved in Lanthionine Synthesis in Mammals</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:40980</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep40980" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85010403537" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28106097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep40980&amp;pmid=28106097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, at two different time periods separated by several years (see <a id="crosref0910" href="#sec-4">STAR Methods</a>), we observed a strikingly similar and prominent phenotype for LanCL KO mice. In both cases, roughly 25%–30% of KO mice died prematurely at ages between 2 and 6&nbsp;months, whereas age-matched WT mice that were co-housed with the KO mice did not display such a phenotype (<a id="crosref0915" href="#fig6">Figures 6</a>H, <a id="crosref0920" href="#figs7">S7</a>D, and S7E). The KO mice that did not die by 6&nbsp;months did not display any further phenotype. The TKO mice that died were submitted for histopathology analysis, but no common underlying abnormalities were detected that could explain the premature death (see <a id="crosref0925" href="#sec-4">STAR Methods</a>). In addition, heart and brain tissue of mice that died prematurely as well as age- and gender-matched WT and surviving TKO mice were incubated with LanCL1/2 and GSH and analyzed by western blot for glutathionylated proteins. No differences were observed for the three populations. Thus, the cause of death is not known and neither is the stimulus that causes the highly similar phenotype at two different times (2011 and 2020) but not in the intervening period. However, the premature death of a large number of LanCL KO mice (<a id="crosref0930" href="#fig6">Figures 6</a>H, <a id="crosref0935" href="#figs7">S7</a>D, and S7E) clearly demonstrates that under certain circumstances the absence of these proteins causes lethality.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0185" role="paragraph">LanCL proteins were first identified two decades ago (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Bauer et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Bauer, H. ∙ Mayer, H. ∙ Marchler-Bauer, A. ...</div><div><strong>Characterization of p40/GPR69A as a peripheral membrane protein related to the lantibiotic synthetase component C</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 2000; <strong>275</strong>:69-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/bbrc.2000.3260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0034682901" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10944443/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fbbrc.2000.3260&amp;pmid=10944443" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Mayer et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Mayer, H. ∙ Salzer, U. ∙ Breuss, J. ...</div><div><strong>Isolation, molecular characterization, and tissue-specific expression of a novel putative G protein-coupled receptor</strong></div><div><em>Biochim. Biophys. Acta.</em> 1998; <strong>1395</strong>:301-308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0167-4781(97)00178-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0032506996" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9512664/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0167-4781%2897%2900178-4&amp;pmid=9512664" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">2001b</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Mayer, H. ∙ Pongratz, M. ∙ Prohaska, R.</div><div><strong>Molecular cloning, characterization, and tissue-specific expression of human LANCL2, a novel member of the LanC-like protein family</strong></div><div><em>DNA Seq.</em> 2001; <strong>12</strong>:161-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/10425170109080770" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0035238674" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11762191/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F10425170109080770&amp;pmid=11762191" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but their activity remained unknown. The possibility that these proteins might be responsible for generation of lanthionine in the brain was recently ruled out (<span class="dropBlock reference-citations"><a id="body-ref-sref26-3" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-3" aria-controls="sref26-3">He et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref26-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>He, C. ∙ Zeng, M. ∙ Dutta, D. ...</div><div><strong>LanCL proteins are not Involved in Lanthionine Synthesis in Mammals</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:40980</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep40980" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85010403537" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28106097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep40980&amp;pmid=28106097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this study, we uncovered a non-canonical glutathionylation activity of LanCL proteins. Pathogen effector pThr lyases were utilized to generate dehydroamino-acid-containing peptides that mimic the activation loop of ERK and Akt kinases. With these peptides, LanCL proteins added GSH to the Dhb residues, which are much less electrophilic than Dha residues and do not show appreciable non-enzymatic background reactivity with GSH. Importantly, the glutathionylation activity results in a thioether-linked structure as opposed to the well known disulfide linked <i>S-</i>glutathionylation generated in response to oxidative stress (<span class="dropBlock reference-citations"><a id="body-ref-sref12-4" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-4" aria-controls="sref12-4">Cooper et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref12-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cooper, A.J. ∙ Pinto, J.T. ∙ Callery, P.S.</div><div><strong>Reversible and irreversible protein glutathionylation: biological and clinical aspects</strong></div><div><em>Expert Opin. Drug Metab. Toxicol.</em> 2011; <strong>7</strong>:891-910</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1517/17425255.2011.577738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79959221560" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21557709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1517%2F17425255.2011.577738&amp;pmid=21557709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Dalle-Donne et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Dalle-Donne, I. ∙ Rossi, R. ∙ Colombo, G. ...</div><div><strong>Protein S-glutathionylation: a regulatory device from bacteria to humans</strong></div><div><em>Trends Biochem. Sci.</em> 2009; <strong>34</strong>:85-96</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-59849129012" target="_blank" aria-label="Open Scopus in new tab">Scopus (545)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19135374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tibs.2008.11.002&amp;pmid=19135374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Grek et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Grek, C.L. ∙ Zhang, J. ∙ Manevich, Y. ...</div><div><strong>Causes and consequences of cysteine S-glutathionylation</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2013; <strong>288</strong>:26497-26504</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.R113.461368&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.R113.461368&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84884179149" target="_blank" aria-label="Open Scopus in new tab">Scopus (254)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23861399/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.R113.461368&amp;pmid=23861399" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-2" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-2" aria-controls="sref74-2">Yang et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref74-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Yang, J. ∙ Carroll, K.S. ∙ Liebler, D.C.</div><div><strong>The expanding landscape of the thiol redox proteome</strong></div><div><em>Mol. Cell. Proteomics.</em> 2016; <strong>15</strong>:1-11</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O115.056051&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O115.056051&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84955506189" target="_blank" aria-label="Open Scopus in new tab">Scopus (163)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26518762/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.O115.056051&amp;pmid=26518762" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0190" role="paragraph">Additionally, Dha-containing MEK and ERK were generated using a chemical mutagenesis strategy and glutathionylation activity was observed when treating Dha-MEK1 and Dha-ERK1 with LanCL1/2. These data potentially explain why LanCL proteins have affinities toward many different cellular kinases in mammalian systems (<a id="crosref0980" href="#fig1">Figure&nbsp;1</a>E), but unbiased proteomic analysis suggests LanCL1/2 interact with many other proteins that are not kinases. Consistent with a potentially wide array of substrates, LanCL proteins appear tolerant of the protein sequence in which the dehydroamino acids occur as GSH was added to both mammalian and bacterial substrates. Collectively, these findings potentially indicate that LanCL proteins might act on a variety of dehydroamino-acid-containing proteins, which may be formed by damage to phosphorylated proteins as well as proteins that undergo eliminative damage at Cys. Our data also show that His<sub>6</sub>-MEK1-Dha222 is glutathionylated by WT MEF cell extracts but not by triple LanCL knockout MEF cell extracts. These data imply that no other GSH activating proteins can catalyze non-canonical <i>C-</i>glutathionylation of Dha-MEK1 in these cells. Importantly, the glutathione-conjugated MEK1 proteins lost most of the activity present in Dha-containing MEK1.</div><div id="p0195" role="paragraph">Overall, our study provides insights into the long-standing puzzle of LanCL protein activities. Glutathione adducts have been observed previously in cells with long-lived proteins in tissues such as the human lens, cardiac myocytes, and neurons but were believed to be formed non-enzymatically (<span class="dropBlock reference-citations"><a id="body-ref-sref12-5" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-5" aria-controls="sref12-5">Cooper et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref12-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cooper, A.J. ∙ Pinto, J.T. ∙ Callery, P.S.</div><div><strong>Reversible and irreversible protein glutathionylation: biological and clinical aspects</strong></div><div><em>Expert Opin. Drug Metab. Toxicol.</em> 2011; <strong>7</strong>:891-910</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1517/17425255.2011.577738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79959221560" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21557709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1517%2F17425255.2011.577738&amp;pmid=21557709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Friedman, 1999</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Friedman, M.</div><div><strong>Chemistry, biochemistry, nutrition, and microbiology of lysinoalanine, lanthionine, and histidinoalanine in food and other proteins</strong></div><div><em>J.&nbsp;Agric. Food Chem.</em> 1999; <strong>47</strong>:1295-1319</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jf981000%2B" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0032965144" target="_blank" aria-label="Open Scopus in new tab">Scopus (311)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10563973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjf981000%252B&amp;pmid=10563973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42-2" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-2" aria-controls="sref42-2">Linetsky et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref42-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Linetsky, M. ∙ Hill, J.M. ∙ LeGrand, R.D. ...</div><div><strong>Dehydroalanine crosslinks in human lens</strong></div><div><em>Exp. Eye Res.</em> 2004; <strong>79</strong>:499-512</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.exer.2004.06.026" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-4544350320" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15381034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.exer.2004.06.026&amp;pmid=15381034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Linetsky and LeGrand, 2005</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Linetsky, M. ∙ LeGrand, R.D.</div><div><strong>Glutathionylation of lens proteins through the formation of thioether bond</strong></div><div><em>Mol. Cell. Biochem.</em> 2005; <strong>272</strong>:133-144</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s11010-005-6908-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-20944446301" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16010980/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs11010-005-6908-1&amp;pmid=16010980" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Wang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Wang, Z. ∙ Lyons, B. ∙ Truscott, R.J. ...</div><div><strong>Human protein aging: modification and crosslinking through dehydroalanine and dehydrobutyrine intermediates</strong></div><div><em>Aging Cell.</em> 2014; <strong>13</strong>:226-234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/acel.12164" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84895822337" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24134651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Facel.12164&amp;pmid=24134651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Importantly, these adducts are found at sites that are phosphorylated during normal physiology, and the dehydroamino acids are thought to be formed by non-enzymatic phosphate elimination (<span class="dropBlock reference-citations"><a id="body-ref-sref69-3" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-3" aria-controls="sref69-3">Wang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref69-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Wang, Z. ∙ Lyons, B. ∙ Truscott, R.J. ...</div><div><strong>Human protein aging: modification and crosslinking through dehydroalanine and dehydrobutyrine intermediates</strong></div><div><em>Aging Cell.</em> 2014; <strong>13</strong>:226-234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/acel.12164" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84895822337" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24134651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Facel.12164&amp;pmid=24134651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Intriguingly, LanCL levels in eyes, brain, cardiac myocytes, and neurons are elevated at the transcriptomic and proteomic level (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Forrest et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Forrest, A.R. ∙ Kawaji, H. ∙ Rehli, M. ..., FANTOM Consortium and the RIKEN PMI and CLST (DGT)</div><div><strong>A promoter-level mammalian expression atlas</strong></div><div><em>Nature.</em> 2014; <strong>507</strong>:462-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13182" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84897406127" target="_blank" aria-label="Open Scopus in new tab">Scopus (1401)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24670764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13182&amp;pmid=24670764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Geiger et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Geiger, T. ∙ Velic, A. ∙ Macek, B. ...</div><div><strong>Initial quantitative proteomic map of 28 mouse tissues using the SILAC mouse</strong></div><div><em>Mol. Cell. Proteomics.</em> 2013; <strong>12</strong>:1709-1722</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.M112.024919&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.M112.024919&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84878741907" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23436904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M112.024919&amp;pmid=23436904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Huttlin et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Huttlin, E.L. ∙ Jedrychowski, M.P. ∙ Elias, J.E. ...</div><div><strong>A tissue-specific atlas of mouse protein phosphorylation and expression</strong></div><div><em>Cell.</em> 2010; <strong>143</strong>:1174-1189</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.cell.2010.12.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.cell.2010.12.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-78650466243" target="_blank" aria-label="Open Scopus in new tab">Scopus (1348)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21183079/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2010.12.001&amp;pmid=21183079" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Kawaji et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Kawaji, H. ∙ Kasukawa, T. ∙ Forrest, A. ...</div><div><strong>The FANTOM5 collection, a data series underpinning mammalian transcriptome atlases in diverse cell types</strong></div><div><em>Sci. Data.</em> 2017; <strong>4</strong>:170113</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/sdata.2017.113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85028458606" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28850107/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsdata.2017.113&amp;pmid=28850107" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Kim et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Kim, M.S. ∙ Pinto, S.M. ∙ Getnet, D. ...</div><div><strong>A draft map of the human proteome</strong></div><div><em>Nature.</em> 2014; <strong>509</strong>:575-581</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13302" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84901599553" target="_blank" aria-label="Open Scopus in new tab">Scopus (1678)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24870542/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13302&amp;pmid=24870542" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Mayer et&nbsp;al., 2001a</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Mayer, H. ∙ Bauer, H. ∙ Breuss, J. ...</div><div><strong>Characterization of rat LANCL1, a novel member of the lanthionine synthetase C-like protein family, highly expressed in testis and brain</strong></div><div><em>Gene.</em> 2001; <strong>269</strong>:73-80</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0378-1119(01)00463-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0035897569" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11376939/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0378-1119%2801%2900463-2&amp;pmid=11376939" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref48-3" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-3" aria-controls="sref48-3">2001b</a><div class="dropBlock__holder" data-db-target-of="sref48-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Mayer, H. ∙ Pongratz, M. ∙ Prohaska, R.</div><div><strong>Molecular cloning, characterization, and tissue-specific expression of human LANCL2, a novel member of the LanC-like protein family</strong></div><div><em>DNA Seq.</em> 2001; <strong>12</strong>:161-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/10425170109080770" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0035238674" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11762191/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F10425170109080770&amp;pmid=11762191" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), providing a potential link between tissue distribution of the enzyme and substrate presence. Whereas non-enzymatic GSH addition to Dha has a detectable background rate (<a id="crosref1050" href="#fig5">Figure&nbsp;5</a>), non-enzymatic addition to Dhb is not detected under physiological conditions. Therefore, the GSH adducts observed in proteomic studies at positions that are known Thr phosphorylation sites (<span class="dropBlock reference-citations"><a id="body-ref-sref69-4" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-4" aria-controls="sref69-4">Wang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref69-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Wang, Z. ∙ Lyons, B. ∙ Truscott, R.J. ...</div><div><strong>Human protein aging: modification and crosslinking through dehydroalanine and dehydrobutyrine intermediates</strong></div><div><em>Aging Cell.</em> 2014; <strong>13</strong>:226-234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/acel.12164" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84895822337" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24134651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Facel.12164&amp;pmid=24134651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) would very likely require enzymatic catalysis. The data with the cell extracts of WT and TKO mouse MEF cells demonstrate that endogenous LanCL, but not other proteins, can catalyze addition of GSH to Dha-MEK1. The high sequence tolerance with respect to dehydroamino-acid-containing peptides/proteins accepted as substrates by LanCL1/2 would ensure that electrophilic sites in the form of dehydroamino acids present in diverse proteins could be removed. It is possible that LanCL1–3 will have substrate selectivity in cells and organisms and indeed such selectivity is observed <i>in&nbsp;vitro</i> (e.g., <a id="crosref1060" href="#fig5">Figure&nbsp;5</a>C).</div><div id="p0200" role="paragraph">In addition to the glutathione adducts that have been detected previously, protein cross-links such as lanthionine (Lan), histidinoalanine (HAL), and lysinoalanine (LAL) have been identified in various settings including the human lens and the abundance of these cross-links are increased in aged human cataractous lens (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Friedman, 1999</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Friedman, M.</div><div><strong>Chemistry, biochemistry, nutrition, and microbiology of lysinoalanine, lanthionine, and histidinoalanine in food and other proteins</strong></div><div><em>J.&nbsp;Agric. Food Chem.</em> 1999; <strong>47</strong>:1295-1319</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jf981000%2B" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0032965144" target="_blank" aria-label="Open Scopus in new tab">Scopus (311)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10563973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjf981000%252B&amp;pmid=10563973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42-3" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-3" aria-controls="sref42-3">Linetsky et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref42-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Linetsky, M. ∙ Hill, J.M. ∙ LeGrand, R.D. ...</div><div><strong>Dehydroalanine crosslinks in human lens</strong></div><div><em>Exp. Eye Res.</em> 2004; <strong>79</strong>:499-512</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.exer.2004.06.026" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-4544350320" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15381034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.exer.2004.06.026&amp;pmid=15381034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These crosslinks are thought to be formed by nucleophilic addition of the side chains of Cys, His, and Lys to Dha and Dhb generated in turn by phosphate elimination of pSer and pThr (<span class="dropBlock reference-citations"><a id="body-ref-sref42-4" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-4" aria-controls="sref42-4">Linetsky et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref42-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Linetsky, M. ∙ Hill, J.M. ∙ LeGrand, R.D. ...</div><div><strong>Dehydroalanine crosslinks in human lens</strong></div><div><em>Exp. Eye Res.</em> 2004; <strong>79</strong>:499-512</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.exer.2004.06.026" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-4544350320" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15381034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.exer.2004.06.026&amp;pmid=15381034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69-5" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-5" aria-controls="sref69-5">Wang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref69-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Wang, Z. ∙ Lyons, B. ∙ Truscott, R.J. ...</div><div><strong>Human protein aging: modification and crosslinking through dehydroalanine and dehydrobutyrine intermediates</strong></div><div><em>Aging Cell.</em> 2014; <strong>13</strong>:226-234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/acel.12164" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84895822337" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24134651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Facel.12164&amp;pmid=24134651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It has been predicted that dehydroamino acids are likely generated in other tissues as well by β-elimination of activated Ser/Thr/Cys residues (<span class="dropBlock reference-citations"><a id="body-ref-sref12-6" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-6" aria-controls="sref12-6">Cooper et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref12-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cooper, A.J. ∙ Pinto, J.T. ∙ Callery, P.S.</div><div><strong>Reversible and irreversible protein glutathionylation: biological and clinical aspects</strong></div><div><em>Expert Opin. Drug Metab. Toxicol.</em> 2011; <strong>7</strong>:891-910</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1517/17425255.2011.577738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79959221560" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21557709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1517%2F17425255.2011.577738&amp;pmid=21557709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), leading to the suggestion that an eliminylome may be present in cells (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Brennan and Barford, 2009</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Brennan, D.F. ∙ Barford, D.</div><div><strong>Eliminylation: a post-translational modification catalyzed by phosphothreonine lyases</strong></div><div><em>Trends Biochem. Sci.</em> 2009; <strong>34</strong>:108-114</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-62849120767" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19233656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tibs.2008.11.005&amp;pmid=19233656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Indeed, our initial survey of proteomics data suggests that elimination at known Cys and phosphorylated Ser sites to Dha and phosphorylated Thr sites to Dhb may be quite common. It would be surprising if mammals would not have evolved a protection mechanism to prevent protein cross-linking and irreversible kinase (in)activation involving dehydroamino acids. Non-canonical glutathionylation catalyzed by LanCL proteins might serve this function. Glutathionylation does not repair eliminated proteins, unlike for instance the repair methyltransferases that convert isoAsp back to Asp (<span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">McFadden and Clarke, 1987</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>McFadden, P.N. ∙ Clarke, S.</div><div><strong>Conversion of isoaspartyl peptides to normal peptides: implications for the cellular repair of damaged proteins</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1987; <strong>84</strong>:2595-2599</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.84.9.2595" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0008123901" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3472227/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.84.9.2595&amp;pmid=3472227" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). But LanCL proteins would provide containment of the eliminylome. The importance of such containment is implied by the phenotype of LanCL KO mice. At present we do not know why the striking premature death phenotype is not observed continuously and suspect that a currently unknown environmental factor likely induces the phenotype when in a LanCL(–) background. The absence of a clear pathological cause of death makes it challenging to identify the molecular mechanism especially since a potential misregulated enzyme such as that shown for MEK1 could be formed in very small amounts, and yet, due to its catalytic signaling potential, have major and/or amplified effects. Future studies will need to investigate the many lines of research that may be able to provide more insights for such transient and/or amplified events.</div><div id="p0205" role="paragraph">Notably, the existence of an eliminylome was previously suggested to be part of a regulated <i>suppressing</i> mechanism (<span class="dropBlock reference-citations"><a id="body-ref-sref5-3" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-3" aria-controls="sref5-3">Brennan and Barford, 2009</a><div class="dropBlock__holder" data-db-target-of="sref5-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Brennan, D.F. ∙ Barford, D.</div><div><strong>Eliminylation: a post-translational modification catalyzed by phosphothreonine lyases</strong></div><div><em>Trends Biochem. Sci.</em> 2009; <strong>34</strong>:108-114</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-62849120767" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19233656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tibs.2008.11.005&amp;pmid=19233656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we reveal that it can be part of an <i>activating</i> mechanism. Given the natural occurrence of elimination and the existence of pathogens that utilize virulence-determining enzymes that drive phosphate elimination at kinase activation sites, we speculate that such elimination processes might have a further, dysregulatory effect in addition to primary, dynamic phosphorylation-dephosphoryation. Conflicting observations have been made on the effect of the pathogen effector protein OspF upon host kinase activity: while some suggest decreased activity (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Arbibe et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arbibe, L. ∙ Kim, D.W. ∙ Batsche, E. ...</div><div><strong>An injected bacterial effector targets chromatin access for transcription factor NF-kappaB to alter transcription of host genes involved in immune responses</strong></div><div><em>Nat. Immunol.</em> 2007; <strong>8</strong>:47-56</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni1423" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33846958783" target="_blank" aria-label="Open Scopus in new tab">Scopus (332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17159983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni1423&amp;pmid=17159983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), other studies suggest that OspF instead drives <i>higher</i> kinase activities (<span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Zurawski et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Zurawski, D.V. ∙ Mitsuhata, C. ∙ Mumy, K.L. ...</div><div><strong>OspF and OspC1 are Shigella flexneri type III secretion system effectors that are required for postinvasion aspects of virulence</strong></div><div><em>Infect. Immun.</em> 2006; <strong>74</strong>:5964-5976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/IAI.00594-06" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33749237700" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16988276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FIAI.00594-06&amp;pmid=16988276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similarly, in the phosphoproteomes of <i>Shigella</i>-infected human epithelial cells both decreases and increases of global phosphorylation levels have been reported (<span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Schmutz et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Schmutz, C. ∙ Ahrné, E. ∙ Kasper, C.A. ...</div><div><strong>Systems-level overview of host protein phosphorylation during <i>Shigella flexneri</i> infection revealed by phosphoproteomics</strong></div><div><em>Mol. Cell. Proteomics.</em> 2013; <strong>12</strong>:2952-2968</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.M113.029918&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.M113.029918&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84885122882" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23828894/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M113.029918&amp;pmid=23828894" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The basis of the previously unexplained activation of the MEK/ERK pathway <i>in cellulo</i> following <i>Shigella</i> infection (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Zurawski et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Zurawski, D.V. ∙ Mumy, K.L. ∙ Faherty, C.S. ...</div><div><strong>Shigella flexneri type III secretion system effectors OspB and OspF target the nucleus to downregulate the host inflammatory response via interactions with retinoblastoma protein</strong></div><div><em>Mol. Microbiol.</em> 2009; <strong>71</strong>:350-368</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2008.06524.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-58149313835" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19017275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2008.06524.x&amp;pmid=19017275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) might be explained by effects similar to the elimination-activation effects that we demonstrate <i>in&nbsp;vitro</i> for MEK1. Kinases that would be activated by phosphate elimination would no longer be sensitive to dephosphorylation by phosphatases, and therefore a subsequent glutathionylation event would be essential for preventing dysregulation.</div><section id="sec-3-1"><h3>Limitations of study</h3><div id="p0210" role="paragraph">The current study demonstrates that LanCL1 and -2 add GSH to dehydro amino acids in peptides and proteins. We cannot rule out that these enzymes also have activity with other electrophilic substrates. Our data show that elimination of phosphate in the activation loop of MEK1 leads to activation of its kinase activity, but the generality of this effect in other enzyme contexts needs to be investigated. The premature death phenotype of TKO mice appears triggered by one or more currently unknown factor(s) that need to be identified. We propose that the phenotype is caused by dysregulation caused by eliminative damage to one or more unknown proteins, but this needs to be confirmed in future studies.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0215" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT OR RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>mouse anti-His-tag</td><td>Cell Signaling Technology</td><td>Cat#2366<br>RRID:<a id="intref0030" href="https://antibodyregistry.org/AB_2115719" target="_blank">AB_2115719</a></td></tr><tr><td>anti-MBP-tag</td><td>Cell Signaling Technology</td><td>Cat#2396<br>RRID:<a id="intref0035" href="https://antibodyregistry.org/AB_2140060" target="_blank">AB_2140060</a></td></tr><tr><td>rabbit anti-Vps34/PI3K</td><td>Cell Signaling Technology</td><td>Cat#4263<br>RRID:<a id="intref0040" href="https://antibodyregistry.org/AB_2299765" target="_blank">AB_2299765</a></td></tr><tr><td>rabbit anti-p44/p42 MAPK(ERK1/2)</td><td>Cell Signaling Technology</td><td>Cat#9102<br>RRID:<a id="intref0045" href="https://antibodyregistry.org/AB_330744" target="_blank">AB_330744</a></td></tr><tr><td>rabbit anti-mTOR</td><td>Cell Signaling Technology</td><td>Cat#2972<br>RRID:<a id="intref0050" href="https://antibodyregistry.org/AB_330978" target="_blank">AB_330978</a></td></tr><tr><td>rabbit anti-PI3 Kinase p110 α</td><td>Cell Signaling Technology</td><td>Cat#4249<br>RRID:<a id="intref0055" href="https://antibodyregistry.org/AB_2165248" target="_blank">AB_2165248</a></td></tr><tr><td>rabbit anti-PIP4K2A</td><td>Cell Signaling Technology</td><td>Cat#5527<br>RRID:<a id="intref0060" href="https://antibodyregistry.org/AB_2722636" target="_blank">AB_2722636</a></td></tr><tr><td>rabbit anti-p70 S6 kinase</td><td>Cell Signaling Technology</td><td>Cat#9202<br>RRID:<a id="intref0065" href="https://antibodyregistry.org/AB_331676" target="_blank">AB_331676</a></td></tr><tr><td>rabbit anti-Akt</td><td>Cell Signaling Technology</td><td>Cat#9272<br>RRID:<a id="intref0070" href="https://antibodyregistry.org/AB_329827" target="_blank">AB_329827</a></td></tr><tr><td>rabbit anti-ATR</td><td>Cell Signaling Technology</td><td>Cat#2790<br>RRID:<a id="intref0075" href="https://antibodyregistry.org/AB_2227860" target="_blank">AB_2227860</a></td></tr><tr><td>rabbit anti-MEK1/2</td><td>Cell Signaling Technology</td><td>Cat#8727<br>RRID:<a id="intref0080" href="https://antibodyregistry.org/AB_10829473" target="_blank">AB_10829473</a></td></tr><tr><td>rabbit anti-GAPDH (HRP Conjugate)</td><td>Cell Signaling Technology</td><td>Cat#3683<br>RRID:<a id="intref0085" href="https://antibodyregistry.org/AB_1642205" target="_blank">AB_1642205</a></td></tr><tr><td>mouse anti-phospho-p44/42 MAPK (ERK1/2) (Thr202/Tyr204)</td><td>Cell Signaling Technology</td><td>Cat#9106<br>RRID:<a id="intref0090" href="https://antibodyregistry.org/AB_331768" target="_blank">AB_331768</a></td></tr><tr><td>anti-mouse IgG HRP-conjugated secondary antibody</td><td>Cell Signaling Technology</td><td>Cat#7076<br>RRID:<a id="intref0095" href="https://antibodyregistry.org/AB_330924" target="_blank">AB_330924</a></td></tr><tr><td>anti-rabbit IgG HRP conjugated secondary antibody</td><td>Cell Signaling Technology</td><td>Cat#7074<br>RRID:<a id="intref0100" href="https://antibodyregistry.org/AB_2099233" target="_blank">AB_2099233</a></td></tr><tr><td>Mouse anti-GSH</td><td>Virogen</td><td>Cat#101-A</td></tr><tr><td>Rabbit anti-LanCL1</td><td>ThermoFisher Scientific</td><td>Cat#A304-482A<br>RRID:<a id="intref0105" href="https://antibodyregistry.org/AB_2620676" target="_blank">AB_2620676</a></td></tr><tr class="rowsep"><td>Rabbit anti-LanCL2</td><td>Custom made by Proteintech Group (Chicago, IL) using full-length recombinant mouse LanCL2 protein as antigen</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td><i>E.&nbsp;coli</i> BL21 (DE3)</td><td>Novagen/MilliporeSigma</td><td>Cat#69451</td></tr><tr><td><i>E.&nbsp;coli</i> T7 <i>lysY</i></td><td>NEB</td><td>Cat#C3010I</td></tr><tr class="rowsep"><td><i>E.&nbsp;coli</i> Rosetta 2 (DE3) cells</td><td>Novagen</td><td>Cat#71397</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Dulbecco’s Modified Eagle’s Medium (DMEM)</td><td>GIBCO</td><td>Cat#11960-069</td></tr><tr><td>Phosphate Buffered Saline (PBS)</td><td>GIBCO</td><td>Cat#20012-050</td></tr><tr><td>Terrific Broth (TB)</td><td>Sigma</td><td>Cat#T9179</td></tr><tr><td>LB Broth (Miller)</td><td>Sigma</td><td>Cat#L3522</td></tr><tr><td>Fetal Bovine Serum (FBS)</td><td>GIBCO</td><td>Cat#10500064</td></tr><tr><td>Penicillin-Streptomycin</td><td>GIBCO</td><td>Cat# 15140163</td></tr><tr><td>Trypsin-EDTA</td><td>Thermo Scientific</td><td>Cat#25200056</td></tr><tr><td>Ampicillin</td><td>Fisher Scientific</td><td>Cat#A9518</td></tr><tr><td>Chloramphenicol</td><td>Sigma</td><td>Cat#C0378</td></tr><tr><td>Kanamycin</td><td>Sigma</td><td>Cat#K1377</td></tr><tr><td>Glutathione reduced</td><td>Sigma</td><td>Cat#G4251</td></tr><tr><td>2,5-Methyldibromopentonate</td><td>Fluorochem Ltd</td><td>Cat#320210</td></tr><tr><td>Adenosine 5ʹ-diphosphate</td><td>Sigma</td><td>Cat#A2754</td></tr><tr><td>Isopropyl β-D-1-thiolgalactopyranoside (IPTG)</td><td>Sigma</td><td>Cat#I6758</td></tr><tr><td>Tris(2-carboxyethyl)phosphine (TCEP)</td><td>Sigma</td><td>Cat#C4706</td></tr><tr><td>Sequencing grade modified trypsin</td><td>Promega</td><td>Cat#V5111</td></tr><tr><td>HRP-conjugated Streptavidin</td><td>Thermo Scientific</td><td>Cat#N100</td></tr><tr><td>Imidazole</td><td>Sigma</td><td>Cat#I5513</td></tr><tr><td>HEPES</td><td>Sigma</td><td>Cat#H3375</td></tr><tr><td>Thionyl chloride</td><td>Sigma</td><td>Cat#230464</td></tr><tr><td>Adipic acid</td><td>Sigma</td><td>Cat#A26357</td></tr><tr><td>Carbon tetrachloride</td><td>Sigma</td><td>Cat#289116</td></tr><tr><td><i>N</i>-Bromosuccinimide</td><td>Fluorochem Ltd</td><td>Cat#BR1130</td></tr><tr><td>2,5-Dibromohexanedioic acid</td><td>Fluorochem Ltd</td><td>Cat#242006</td></tr><tr class="rowsep"><td>Glycinamide hydrochloride</td><td>Fluorochem Ltd</td><td>Cat#078516</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Pierce BCA Protein Assay Kit</td><td>Thermo Scientific</td><td>Cat#23227</td></tr><tr><td>cOmplete, Mini, EDTA-free Protease Inhibitor Cocktail</td><td>Roche</td><td>Cat#11836170001</td></tr><tr><td>Halt Protease Inhibitor Cocktail</td><td>Thermo Scientific</td><td>Cat#78420</td></tr><tr><td>QuikChange II XL Site-Directed Mutagenesis Kit</td><td>Agilent Technologies</td><td>Cat#200521</td></tr><tr><td>HisTRap HP column</td><td>GE Healthcare</td><td>Cat#17-5248-01</td></tr><tr><td>GSTrap FF column</td><td>GE Healthcare</td><td>Cat#17-5131-01</td></tr><tr><td>PD SpinTrap G-25</td><td>GE Healthcare</td><td>Cat#28-9180-04</td></tr><tr><td>NUCLEODUR C18 HTec C18 column</td><td>Machery-Nagel</td><td>Cat#762566.100</td></tr><tr><td>NUCLEODUR 100-5 C18 ec column</td><td>Machery-Nagel</td><td>Cat#7600001.20</td></tr><tr><td>Jupiter 4&nbsp;μm Proteo 90&nbsp;Å LC column</td><td>Phenomenex</td><td>Cat#00G-4396-N0</td></tr><tr><td>CP-Chirasil-L-Val column</td><td>Agilent</td><td>Cat#CP749515</td></tr><tr><td>VP 250/10 Nucleodur C18 HTec, 5&nbsp;μm</td><td>Macherey-Nagel</td><td>Cat#762556.100</td></tr><tr><td>ProSwiftTM RP-2H column</td><td>Thermo Scientific</td><td>Cat#064296</td></tr><tr><td>HisPur Cobalt resin</td><td>Thermo Scientific</td><td>Cat#89964</td></tr><tr><td>Amylose resin</td><td>New England Biolabs</td><td>Cat#E8021S</td></tr><tr><td>His60 Ni Superflow resin</td><td>Takara</td><td>Cat#635661</td></tr><tr><td>Oasis HLB 1cc Vac Cartridge</td><td>Waters</td><td>Cat#WAT094225</td></tr><tr><td>Vivaspin 6 centrifugal concentrator</td><td>Fisher Scientific</td><td>Cat#10421243</td></tr><tr class="rowsep"><td>2X Laemmli Sample Buffer</td><td>Bio-rad</td><td>Cat#1610737</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Mass spectrometry data<br>Data for figures</td><td>This paper</td><td>Mendeley Data: <a id="intref0110" href="https://doi.org/10.17632/sgjzwcxgsr.1">https://doi.org/10.17632/sgjzwcxgsr.1</a></td></tr><tr><td>Mass spectrometry data<br>Data for figures</td><td>This paper</td><td><a id="intref0115" href="https://ora.ox.ac.uk/objects/uuid:d8729aad-ca98-4f06-b14a-0a75ea306584" target="_blank">https://ora.ox.ac.uk/objects/uuid:d8729aad-ca98-4f06-b14a-0a75ea306584</a></td></tr><tr><td>LanCL2 structure</td><td>This paper</td><td>PDB ID <a id="intref0120" href="http://firstglance.jmol.org/fg.htm?mol6WQ1" target="_blank">6WQ1</a></td></tr><tr><td>MS analysis of pulldown samples</td><td>Harvard Taplin Mass Spectrometry Facility</td><td><a id="crosref1125" href="#mmc1">Table S1</a></td></tr><tr><td>MEK1⋅ATPγS structure</td><td>Protein Data Bank</td><td>PDB ID <a id="intref0125" href="http://firstglance.jmol.org/fg.htm?mol3W8Q" target="_blank">3W8Q</a></td></tr><tr><td>MEK1⋅ATPγS⋅Mg<sup>2+</sup> structure</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref20-2" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-2" aria-controls="sref20-2">Fischmann et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref20-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Fischmann, T.O. ∙ Smith, C.K. ∙ Mayhood, T.W. ...</div><div><strong>Crystal structures of MEK1 binary and ternary complexes with nucleotides and inhibitors</strong></div><div><em>Biochemistry.</em> 2009; <strong>48</strong>:2661-2674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi801898e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-65249090229" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19161339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi801898e&amp;pmid=19161339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB ID <a id="intref0130" href="http://firstglance.jmol.org/fg.htm?mol3EQD" target="_blank">3EQD</a></td></tr><tr><td>MEK1⋅ADPMg<sup>2+</sup> structure</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref20-3" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-3" aria-controls="sref20-3">Fischmann et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref20-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Fischmann, T.O. ∙ Smith, C.K. ∙ Mayhood, T.W. ...</div><div><strong>Crystal structures of MEK1 binary and ternary complexes with nucleotides and inhibitors</strong></div><div><em>Biochemistry.</em> 2009; <strong>48</strong>:2661-2674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi801898e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-65249090229" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19161339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi801898e&amp;pmid=19161339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB ID <a id="intref0135" href="http://firstglance.jmol.org/fg.htm?mol3EQI" target="_blank">3EQI</a></td></tr><tr><td>CylM structure</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Dong et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dong, S.H. ∙ Tang, W. ∙ Lukk, T. ...</div><div><strong>The enterococcal cytolysin synthetase has an unanticipated lipid kinase fold</strong></div><div><em>eLife.</em> 2015; <strong>4</strong>:e07607</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.07607" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84940538541" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.07607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB ID <a id="intref0140" href="http://firstglance.jmol.org/fg.htm?mol5DZT" target="_blank">5DZT</a></td></tr><tr><td>LanCL1 structure</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref77-6" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-6" aria-controls="sref77-6">Zhang et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref77-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zhang, W. ∙ Wang, L. ∙ Liu, Y. ...</div><div><strong>Structure of human lanthionine synthetase C-like protein 1 and its interaction with Eps8 and glutathione</strong></div><div><em>Genes Dev.</em> 2009; <strong>23</strong>:1387-1392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1789209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-67149147296" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19528316/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1789209&amp;pmid=19528316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB ID <a id="intref0145" href="http://firstglance.jmol.org/fg.htm?mol3E73" target="_blank">3E73</a></td></tr><tr class="rowsep"><td>NisC structure</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref38-4" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-4" aria-controls="sref38-4">Li et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref38-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Li, B. ∙ Yu, J.P. ∙ Brunzelle, J.S. ...</div><div><strong>Structure and mechanism of the lantibiotic cyclase involved in nisin biosynthesis</strong></div><div><em>Science.</em> 2006; <strong>311</strong>:1464-1467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1121422" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33644854595" target="_blank" aria-label="Open Scopus in new tab">Scopus (263)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16527981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1121422&amp;pmid=16527981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB ID <a id="intref0150" href="http://firstglance.jmol.org/fg.htm?mol2G0D" target="_blank">2G0D</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr><td>LANCL1 KO mice in FVB background</td><td>Jackson labs</td><td>029695</td></tr><tr><td>LanCL2 KO mice in FVB background</td><td>Jackson labs</td><td>029694</td></tr><tr><td>LanCL3 KO mice are in process of being deposited</td><td>Jackson labs</td><td>N/A</td></tr><tr><td>Human: HEK293</td><td>ATCC</td><td>CRL-1573</td></tr><tr class="rowsep"><td>Mouse: MEF</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref26-4" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-4" aria-controls="sref26-4">He et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref26-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>He, C. ∙ Zeng, M. ∙ Dutta, D. ...</div><div><strong>LanCL proteins are not Involved in Lanthionine Synthesis in Mammals</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:40980</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep40980" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85010403537" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28106097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep40980&amp;pmid=28106097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>pMAL-p2x-LanCL2-FP 5′−cctcgg<br>gatcgagggaaggatttcagaatt<br>catgggcgagaccatgtc−3′</td><td>This paper</td><td>N/A</td></tr><tr><td>pMAL-p2x-LanCL2-RP 5′−ggccagt<br>gccaagcttgcctgcaggtcgact<br>taatccctcttcgaagagtc−3′</td><td>This paper</td><td>N/A</td></tr><tr><td>pET15b-OspF-FP 5′−caactcagct<br>tcctttcgggctttgttagcagccggatccct<br>actctatcatcaaacgataaaatgg−3′</td><td>This paper</td><td>N/A</td></tr><tr><td>pET15b-OspF-RP 5′−tcatcacagcag<br>cggcctggtgccgcgcggcagccatatga<br>tgcccataaaaaagccctgtctaaaac−3′</td><td>This paper</td><td>N/A</td></tr><tr><td>pET15b-SpvC-FP 5′−tcctttcgggct<br>ttgttagcagccggatccttactctgtc<br>atcaaacgataaaacg−3′</td><td>This paper</td><td>N/A</td></tr><tr><td>pET15b-SpvC-RP 5′−agcggcctggt<br>gccgcgcggcagccatatgatgccca<br>taaataggcctaatct−3′</td><td>This paper</td><td>N/A</td></tr><tr><td>pET15b-GSTA4-FP 5′−aactcagcttcc<br>tttcgggctttgttagcagccggatcc<br>ttatggcctaaagatgttgt−3′</td><td>This paper</td><td>N/A</td></tr><tr><td>pET15b-GSTA4-RP 5′−acagcagcggc<br>ctggtgccgcgcggcagccatatgctcg<br>agatggcagcaaggcccaag−3′</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td>Primers for LanCL, MEK1 an ERK1 variants</td><td><a id="crosref1160" href="#mmc4">Table S4</a></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Plasmids</b></td></tr><tr><td>pET-15b-LanCL1</td><td>This paper</td><td>RRID: Addgene 154189</td></tr><tr><td>pETDuet1-WT-LanCL2-C-Ter-His</td><td>This paper</td><td>RRID: Addgene 154186</td></tr><tr><td>pMAL-p2x-LanCL2</td><td>This paper</td><td>RRID: Addgene 154192</td></tr><tr><td>pETDuet1-LanCL2-H264A-C-Ter-His</td><td>This paper</td><td>RRID: Addgene 154187</td></tr><tr><td>pETDuet1-LanCL2-C321A/C367A-C-Ter-His</td><td>This paper</td><td>RRID: Addgene 154188,</td></tr><tr><td>pET-15b-LanCL1-H219A</td><td>This paper</td><td>RRID: Addgene 154190</td></tr><tr><td>pET-15b-LanCL1-C322A</td><td>This paper</td><td>RRID: Addgene 154191</td></tr><tr><td>pET28a-MEK1- G(−19)F/S222C/C277S/C376S</td><td>This paper</td><td>RRID: Addgene 164638</td></tr><tr><td>pET28a-MEK1- G(−19)F/S218C/C277S/C376S</td><td>This paper</td><td>RRID: Addgene 164639</td></tr><tr><td>pET28a-MEK1- G(−19)F/S222C/S218C/C277S/C376S</td><td>This paper</td><td>RRID: Addgene 164640</td></tr><tr><td>pET28a-MEK1- Δ<sub>1-60</sub>-S222C/C277S/C376S</td><td>This paper</td><td>RRID: Addgene 164641</td></tr><tr><td>pET28a-MEK1- Δ<sub>1-60</sub>-S218C/C277S/C376S</td><td>This paper</td><td>RRID: Addgene 164643</td></tr><tr><td>pET28a-MEK1- Δ<sub>1-60</sub>-S222C/S218C/C277S/C376S</td><td>This paper</td><td>RRID: Addgene 164645</td></tr><tr><td>ERK1-C202 (K71R/C82S/C144S/C178S/C183S/T202C/ C271S</td><td>This paper</td><td>RRID: Addgene 164647</td></tr><tr><td>ERK1-C82C202 (K71R/C144S/C178S/C183S/T202C/C271S</td><td>This paper</td><td>RRID: Addgene 164650</td></tr><tr><td>NpT7-5-ERK1</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Robbins et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Robbins, D.J. ∙ Zhen, E. ∙ Owaki, H. ...</div><div><strong>Regulation and properties of extracellular signal-regulated protein kinases 1 and 2 in&nbsp;vitro</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1993; <strong>268</strong>:5097-5106</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2FS0021-9258%2818%2953507-9&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2FS0021-9258%2818%2953507-9&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8444886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2953507-9&amp;pmid=8444886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>RRID: Addgene_39229</td></tr><tr><td>pGEX-4T-1-3XFlag-ERK2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Bose and Janes, 2013</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bose, A.K. ∙ Janes, K.A.</div><div><strong>A high-throughput assay for phosphoprotein-specific phosphatase activity in cellular extracts</strong></div><div><em>Mol. Cell. Proteomics.</em> 2013; <strong>12</strong>:797-806</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O112.024059&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O112.024059&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84874584886" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23233447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.O112.024059&amp;pmid=23233447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>RRID: Addgene_47573</td></tr><tr><td>pET-His6-ERK2-MEK1_R4F_coexpression</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref33-1" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-1" aria-controls="sref33-1">Khokhlatchev et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref33-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Khokhlatchev, A. ∙ Xu, S. ∙ English, J. ...</div><div><strong>Reconstitution of mitogen-activated protein kinase phosphorylation cascades in bacteria. Efficient synthesis of active protein kinases</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1997; <strong>272</strong>:11057-11062</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0030966367" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9110999/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.272.17.11057&amp;pmid=9110999" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>RRID: Addgene_39212</td></tr><tr class="rowsep"><td>pGEX-4T-1-3XFlag-ERK2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref33-2" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-2" aria-controls="sref33-2">Khokhlatchev et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref33-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Khokhlatchev, A. ∙ Xu, S. ∙ English, J. ...</div><div><strong>Reconstitution of mitogen-activated protein kinase phosphorylation cascades in bacteria. Efficient synthesis of active protein kinases</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1997; <strong>272</strong>:11057-11062</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0030966367" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9110999/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.272.17.11057&amp;pmid=9110999" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>RRID: Addgene_47573</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Origin Pro9.7</td><td>OriginLab Corporation</td><td>RRID:<a id="intref0155" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_014212" target="_blank">SCR_014212</a></td></tr><tr><td>GraphPad Prism6</td><td>GraphPad</td><td>RRID:<a id="intref0160" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_002798" target="_blank">SCR_002798</a></td></tr><tr><td>Sequest</td><td>Thermo Scientific</td><td>RRID:<a id="intref0165" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_014594" target="_blank">SCR_014594</a></td></tr><tr><td>PEAKS Studio 7.0</td><td>Bioinformatics Solutions Inc</td><td><a id="intref0170" href="https://www.bioinfor.com/peaks-studio/" target="_blank">https://www.bioinfor.com/peaks-studio/</a></td></tr><tr><td>MassLynx V4.1</td><td>Waters</td><td>RRID:<a id="intref0175" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_014271" target="_blank">SCR_014271</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2900436-0" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0220" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Wilfred van der Donk (<a id="intref0180" href="mailto:vddonk@illinois.edu">vddonk@illinois.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0225" role="paragraph">Plasmids generated in this study for preparation of LanCL1/2, their variants, and their protein substrates were deposited to Addgene.</div></section><section id="sec-4-2-3"><h4>Data and code availability</h4><div id="p0230" role="paragraph">The published article includes all datasets analyzed during this study. Structural data have been deposited in the PDB: 6WQ1. Mass spectral data and western blot data have been deposited to Mendeley Data: <a id="intref0185" href="https://dx.doi.org/10.17632/sgjzwcxgsr.1" target="_blank">https://dx.doi.org/10.17632/sgjzwcxgsr.1</a> and at <a id="intref0190" href="https://ora.ox.ac.uk/objects/uuid:d8729aad-ca98-4f06-b14a-0a75ea306584" target="_blank">https://ora.ox.ac.uk/objects/uuid:d8729aad-ca98-4f06-b14a-0a75ea306584</a>.</div></section></section><section id="sec-4-3"><h3>Experimental procedures</h3><section id="sec-4-3-1"><h4>Cell culture and primary cell isolation</h4><div id="p0235" role="paragraph">Human embryonic kidney (HEK) 293 cells and primary mouse embryonic fibroblasts (MEFs) were cultured in Dulbecco’s Modified Eagle Media (DMEM) with 10% (v/v) fetal bovine serum (FBS) (GIBCO) and 1X penicillin/streptomycin (final concentration for penicillin: 100 unit/mL, streptomycin: 100&nbsp;μg/mL) (DMEM complete media). MEFs were isolated from 13-day mouse embryos. In brief, the mouse embryos were harvested and rinsed with 70% ethanol. The embryos were transferred into PBS buffer and organs were removed for each embryo. After all embryos were dissected, the tissues were minced with a scalpel and then incubated with 0.25% trypsin-EDTA (Thermo Fisher) at 37°C for 15&nbsp;min. Trypsin was inactivated by adding DMEM complete media. The isolated cells were re-suspended in DMEM complete media and cultured in T-75 flasks. LanCL1/2/3 triple knockout (TKO) MEF cells were obtained from TKO mice (<span class="dropBlock reference-citations"><a id="body-ref-sref26-5" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-5" aria-controls="sref26-5">He et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref26-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>He, C. ∙ Zeng, M. ∙ Dutta, D. ...</div><div><strong>LanCL proteins are not Involved in Lanthionine Synthesis in Mammals</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:40980</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep40980" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85010403537" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28106097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep40980&amp;pmid=28106097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-3-2"><h4>Animal husbandry</h4><div id="p0240" role="paragraph">All animal strains were generated in a FVB background (<span class="dropBlock reference-citations"><a id="body-ref-sref26-6" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-6" aria-controls="sref26-6">He et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref26-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>He, C. ∙ Zeng, M. ∙ Dutta, D. ...</div><div><strong>LanCL proteins are not Involved in Lanthionine Synthesis in Mammals</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:40980</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep40980" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85010403537" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28106097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep40980&amp;pmid=28106097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All animal experiments in this study followed protocols approved by the Animal Care and Use Committee at the University of Illinois at Urbana-Champaign. FVB/NJ mice were purchased from the Jackson Laboratory (Stock Number #001800). LanCL1-3 TKO FVB mice were generated at the University of Illinois at Urbana Champaign. Neither wt nor TKO FVB mice were involved in previous procedures and were drug and test naive. All mice were housed and maintained in the Institute for Genomic Biology on a 12 h/12&nbsp;h light/dark cycle with access to clean water and food (2018 Teklad global 18% protein rodent diet). No special treatment was given to experimental animals. Sick animals were given moist food by mixing their diet with clean water.</div></section><section id="sec-4-3-3"><h4>Bacterial growth conditions for protein expression</h4><div id="p0245" role="paragraph">His<sub>6</sub>-SpvC, His<sub>6</sub>-OspF, His<sub>6</sub>-LanCL1, His<sub>6</sub>-LanCL1-H219, His<sub>6</sub>-LanCL1-C322A, LanCL2-His<sub>6</sub>, LanCL2-His<sub>6</sub>-H264A, LanCL2-His<sub>6</sub>-C321A/C367A and MBP-LanCL2 were expressed in <i>E.&nbsp;coli</i> Rosetta 2 (DE3) cells as either His<sub>6</sub>- or maltose binding protein (MBP) fusion proteins. Cells were grown in LB media with appropriate antibiotics (final concentrations: ampicillin, 100&nbsp;μg/mL; chloramphenicol, 25&nbsp;μg/mL) until OD<sub>600</sub> reached 0.7. Cells were then induced with 0.2&nbsp;mM IPTG after cooling on ice and incubated at 18°C for 18&nbsp;h unless otherwise noted. Cells were isolated by centrifugation and stored at −80°C until protein purification.</div><div id="p0250" role="paragraph">The His<sub>6</sub>-ERK2-MEK1-R4F plasmid was expressed in <i>E.&nbsp;coli</i> T7 <i>lysY</i> cells (New England Biolabs). Cells were grown in LB media with appropriate antibiotics (final concentrations: ampicillin, 100&nbsp;μg/mL; chloramphenicol, 25&nbsp;μg/mL) until the OD<sub>600</sub> reached 0.7. Cells were then induced with 0.5&nbsp;mM IPTG after cooling on ice and incubated at 18°C for 20 h. Cells were isolated by centrifugation and stored at −80°C until protein purification.</div><div id="p0255" role="paragraph">The mutant plasmids MEK-C218 and MEK-C222 were used to transform <i>E.&nbsp;coli</i> BL21 (DE3) competent cells and expressed as N-terminal His<sub>6</sub>-fusion proteins. Cells were grown in LB media with 50&nbsp;μg/mL kanamycin and induced with 1&nbsp;mM IPTG when the OD600 reached 0.6. The incubation was continued for another 12-16&nbsp;h at 25°C. Cells were isolated by centrifugation and stored at −80°C until protein purification.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Plasmids</h4><div id="p0260" role="paragraph">The gene encoding LanCL1 was cloned into pET15b using XhoI and NdeI cloning sites (pET-15b-LanCL1; Addgene 154189). For bacterial expression of C-terminal His-tagged LanCL2, the human LanCL2 cDNA was subcloned into pETDuet-1 using NcoI and NotI cloning sites in multiple cloning site I (MCSI; pETDuet1-WT-LanCL2-C-Ter-His; Addgene 154186). The gene encoding LanCL2 was cloned into pMAL-p2x vector using BamHI and SalI sites (pMAL-p2x-LanCL2; Addgene 154192). All LanCL1 and LanCL2 mutants were generated by site-directed mutagenesis (Addgene 154187, 154188, 154190, 154191). The cDNA encoding the phosphothreonine lyase OspF was synthesized by Integrated DNA technologies and amplified by polymerase chain reaction (PCR). The SpvC encoding gene was directly amplified by PCR using genomic DNA as template from <i>S.&nbsp;typhimurium</i> strain 14028 (gift from Dr. James M. Slauch at the University of Illinois at Urbana-Champaign). The cDNA encoding SpvC or OspF was cloned into the pET15b vector using NdeI and BamHI cloning sites. The human glutathione transferase A4 (GSTA4) cDNA was subcloned into the pET15b vector using NdeI and BamHI sites. All lanthipeptides were co-expressed using the pRSFDuet vector system with the corresponding modification enzyme as previously reported (<span class="dropBlock reference-citations"><a id="body-ref-sref17-3" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-3" aria-controls="sref17-3">Dong et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref17-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dong, S.H. ∙ Tang, W. ∙ Lukk, T. ...</div><div><strong>The enterococcal cytolysin synthetase has an unanticipated lipid kinase fold</strong></div><div><em>eLife.</em> 2015; <strong>4</strong>:e07607</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.07607" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84940538541" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.07607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Yang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Yang, X. ∙ Lennard, K.R. ∙ He, C. ...</div><div><strong>A lanthipeptide library used to identify a protein-protein interaction inhibitor</strong></div><div><em>Nat. Chem. Biol.</em> 2018; <strong>14</strong>:375-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41589-018-0008-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85042882496" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29507389/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41589-018-0008-5&amp;pmid=29507389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The ProcA2.8 mutant (C3A/C19A/S13A) was generated by site directed mutagenesis. A list of primers for generating all constructs is provided in the <a id="crosref1195" href="#sec-4-1">Key resources table</a> and <a id="crosref1200" href="#mmc4">Table S4</a>.</div><div id="p0265" role="paragraph">pET28a-MEK1-S222C/C277S/C376S plasmid was prepared by Genscript (1&nbsp;μL, 200&nbsp;ng/μL). This plasmid encodes for an N-terminal His<sub>6</sub> tag. In order to overcome gluconoylation, the plasmid was mutated at G(−19)F to afford MEK1 G(−19)F/S222C/C277S/C376S referred to as MEK1C222 herein. All desired mutations and deletions were performed from this original plasmid. MEK1-C218 (S218C/C277S/C376S; Addgene 164639), MEK1-C222 (S222C/C277S/C376S; Addgene 164638), MEK1-C218C222 (S218C/S222C/C277S/C376S: Addgene 164640), ERK1-C202 (K71R/C82S/C144S/C178S/C183S/T202C/ C271S; Addgene 164647) and ERK1-C82C202 (K71R/C144S/C178S/C183S/T202C/C271S; Addgene 164650) mutants were obtained by site-directed mutagenesis using standard protocols and were deposited at Addgene. A list of primers for generating all constructs is provided in the <a id="crosref1205" href="#sec-4-1">Key resources table</a> and <a id="crosref1210" href="#mmc4">Table S4</a>. The N-terminal truncated (residues 1-60) MEK1 C277SC377S, referred as Δ<sub>1-60</sub>MEK1 mutant with a C-terminal His<sub>6</sub> tag (residues 323-329), was constructed and cloned by SGC (Dr. Apirat Chaikuad) from the MEK1 plasmids described above. The gene encoding MEK1 was cloned into pET28a between NcoI and BamHI restriction sites (C222: Addgene 164641; C218 Addgene 164643; C218C222 Addgene 164645).</div><div id="p0270" role="paragraph">The following plasmids were obtained from Addgene: human ERK1 gene inserted between EcoRI and BamHI sites of NpT7-5 vector (gift from Melanie Cobb; RRID: Addgene_39229).pGEX-4T-1-3XFlag-ERK2 (gift from Kevin Janes; RRID:Addgene_47573) (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Bose and Janes, 2013</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bose, A.K. ∙ Janes, K.A.</div><div><strong>A high-throughput assay for phosphoprotein-specific phosphatase activity in cellular extracts</strong></div><div><em>Mol. Cell. Proteomics.</em> 2013; <strong>12</strong>:797-806</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O112.024059&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O112.024059&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84874584886" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23233447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.O112.024059&amp;pmid=23233447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); pET-His6-ERK2-MEK1_R4F_coexpression (gift from Melanie Cobb; RRID:Addgene_39212) (<span class="dropBlock reference-citations"><a id="body-ref-sref33-3" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-3" aria-controls="sref33-3">Khokhlatchev et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref33-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Khokhlatchev, A. ∙ Xu, S. ∙ English, J. ...</div><div><strong>Reconstitution of mitogen-activated protein kinase phosphorylation cascades in bacteria. Efficient synthesis of active protein kinases</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1997; <strong>272</strong>:11057-11062</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0030966367" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9110999/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.272.17.11057&amp;pmid=9110999" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-2"><h4>Solid-phase peptide synthesis</h4><div id="p0275" role="paragraph">Peptide synthesis was performed on a CEM Liberty microwave peptide synthesizer. Standard Fmoc protected amino acids were used unless otherwise noted. 2-(6-chloro-1<i>H</i>-benzotriazole-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate (HCTU) was used as activator, 2&nbsp;M <i>N,N</i>-diisopropylethylamine (DIPEA) in <i>N</i>-methyl-2-pyrrolidone (NMP) as activator base and 20% piperidine in dimethylformamide (DMF) as deprotection agent. Synthetic peptides were cleaved from the resin by adding cleavage cocktail (90/7.5/2.5 TFA/triisopropylsilane/H<sub>2</sub>O) for 2&nbsp;h and crude peptides were precipitated in ice-cold diethyl ether. Precipitants were then air-dried to remove residual diethyl ether. The dried precipitants were dissolved in water and purified by RP-HPLC (Macherey-Nagel, VP 250/10 Nucleodur C18 HTec, 5&nbsp;μm) at 1&nbsp;mL/min flow rate with the following gradient of solvent A (0.1% trifluoroacetic acid in H<sub>2</sub>O) and B (0.1% trifluoroacetic acid in MeCN): linear gradient from 2% to 85% B in 40&nbsp;min, followed by a gradient from 85% to 98% B in 2&nbsp;min.</div></section><section id="sec-4-4-3"><h4>Protein purification</h4><div id="p0280" role="paragraph">For His-tagged proteins, cell pellets from 1&nbsp;L of cell culture were resuspended and lysed in 25&nbsp;mL of lysis buffer (20&nbsp;mM Tris, 500&nbsp;mM NaCl, 1&nbsp;mM tris(2-carboxyethyl)phosphine (TCEP), 5&nbsp;mM imidazole, 10% glycerol, pH 8.0). Lysates were centrifuged at 4°C for 30&nbsp;min at 75,000&nbsp;× <i>g</i> and supernatants were filtered with centrifugal filters (Fisher Scientific). NiNTA purification was performed using a 5&nbsp;mL HisTrap coumn (GE Healthcare). After loading the clarified lysate, the column was applied to an Äkta Pure FPLC system (GE Healthcare) using solvent A (20&nbsp;mM Tris, 500&nbsp;mM NaCl, 1&nbsp;mM TCEP, 5&nbsp;mM imidazole, 10% glycerol, pH 8.0) and solvent B (20&nbsp;mM Tris, 500&nbsp;mM NaCl, 1&nbsp;mM TCEP, 500&nbsp;mM imidazole, 10% glycerol, pH 8.0). The loaded column was washed at 1.5&nbsp;mL/min with a linear gradient from 0 to 20% solvent B over 23 column volumes (CV) and eluted with 100% solvent B for 10 CV. The purest fraction was identified by SDS-PAGE and concentrated by 10&nbsp;kDa cutoff Amicon ultracentrifugal filters (Millipore). For MBP-tagged protein purification, cell pellets from 1&nbsp;L of cell culture were resuspended and lysed in 10&nbsp;mL of MBP purification buffer (20&nbsp;mM Tris, 150&nbsp;mM NaCl, pH 8.0). Lysates were centrifuged at 4°C for 30&nbsp;min at 75,000&nbsp;× <i>g</i> and supernatants were filtered with centrifugal filters (Fisher Scientific). The filtered lysate was applied to 3&nbsp;mL of amylose resin (New England Biolabs) and eluted with MBP elution buffer (20&nbsp;mM Tris, 150&nbsp;mM NaCl, 200&nbsp;mM maltose, pH 8.0). Fractions were identified by SDS-PAGE and concentrated using 10&nbsp;kDa cutoff Amicon ultracentrifugal filters (Millipore).</div><div id="p0285" role="paragraph">For purification of the phosphorylated His<sub>6</sub>-ERK2 protein, cell pellets from 1&nbsp;L of cell culture were resuspended and lysed in 25&nbsp;mL of lysis buffer (20&nbsp;mM Tris, 500&nbsp;mM NaCl, 1&nbsp;mM tris(2-carboxyethyl)phosphine (TCEP), 25&nbsp;mM NaF, 1&nbsp;mM NaVO<sub>3</sub>, 25&nbsp;mM β-glycerophosphate, 5&nbsp;mM imidazole, 10% glycerol, pH 8.0). Lysates were centrifuged at 4°C for 30&nbsp;min at 75,000&nbsp;× <i>g</i> and supernatants were filtered with centrifugal filters (Fisher Scientific). The His<sub>6</sub>-ERK2 was purified using the same procedure as described above, and its phosphorylation was confirmed by intact protein MS.</div><div id="p0290" role="paragraph">Pellets of cells expressing MEK-C218 and MEK-C222 were resuspended in 30&nbsp;mL of lysis buffer (50&nbsp;mM HEPES, 500&nbsp;mM NaCl, 20&nbsp;mM imidazole, 5% glycerol, 0.5&nbsp;mM TCEP, 1% Triton X-100, pH 7.5) for 1&nbsp;h and sonicated using a microtip. The supernatant was separated by centrifugation (20,000&nbsp;rpm, 30&nbsp;min at 4°C) and loaded onto a 5&nbsp;mL HisTrap column. The column was washed using HEPES buffer with increased amounts of imidazole (20-60&nbsp;mM). The target protein was eluted with 80 to 500&nbsp;mM imidazole in HEPES buffer, identified with SDS-PAGE and mass spectrometry, and concentrated by the use of Vivaspin 6 10kDa-cutoff concentrator.</div></section><section id="sec-4-4-4"><h4>Expression of ERK1-C202 and ERK1-C82C202 mutants</h4><div id="p0295" role="paragraph">For <i>in&nbsp;vitro</i> MEK1 activity assay and the generation of dehydroalanine-containing ERK1, a series of plasmids encoding wt ERK1 or mutants (ERK1-C202 and ERK1-C82C202) was constructed. ERK1 wt or mutant gene with a His<sub>6</sub>-tag at its N terminus was inserted into the expression vector NpT7-5 between EcoRI and BamHI sites, and the plasmid was used to transform <i>E.&nbsp;coli</i> BL21(DE3) competent cells.°C Then 1&nbsp;mM IPTG was added to induce protein expression, and after further 16-h incubation at 25°C, cell pellets were harvested. ERK1 protein was purified by the use of Ni-NTA column, and its purity verified through SDS-PAGE and LC-MS.</div></section><section id="sec-4-4-5"><h4>His-pulldown assays for western blot analysis</h4><div id="p0300" role="paragraph">HEK293 cells were rinsed with ice-cold PBS buffer and lysed in His-pulldown buffer (20&nbsp;mM Tris, 150&nbsp;mM NaCl, 25&nbsp;mM NaF, 25&nbsp;mM β-glycerolphosphate, 0.1&nbsp;mM NaVO<sub>3</sub>, 20&nbsp;mM imidazole, 0.3% Triton X-100, pH 8.0 and 1X Halt protease inhibitor cocktail [Thermo Fisher]). HEK293 cell extracts were micro-centrifuged at 14,100&nbsp;× <i>g</i> for 20&nbsp;min at 4°C and supernatants were transferred into clean vials. Recombinant His-tagged LanCL1 or LanCL2 (4&nbsp;μM) were added into HEK293 cell extracts and incubated at 4°C. After 2 h, cell extracts were treated with 30&nbsp;μL of HisPur Cobalt resin (Thermo Fisher) for His-pulldown at 4°C for 1 h. The cobalt resin was washed with 500&nbsp;μL of His-pulldown buffer three times and boiled with 30&nbsp;μL of 2X Laemmli buffer (Bio-Rad). All samples were analyzed by western blotting.</div></section><section id="sec-4-4-6"><h4>His-pulldown assays for unbiased MS-based proteomics</h4><div id="p0305" role="paragraph">HEK293 cells were rinsed with ice-cold PBS buffer and lysed in His-pulldown buffer (20&nbsp;mM Tris, 150&nbsp;mM NaCl, 25&nbsp;mM NaF, 25&nbsp;mM β-glycerolphosphate, 0.1&nbsp;mM NaVO<sub>3</sub>, 20&nbsp;mM imidazole, 0.3% Triton X-100, pH 8.0 and 1X Halt protease inhibitor cocktail [Thermo Fisher]). HEK293 cell extracts were micro-centrifuged at 14,100&nbsp;× <i>g</i> for 20&nbsp;min at 4°C and supernatants were pre-cleaned with 20&nbsp;μL of HisPur Cobalt resin (Thermo Fisher) for 30&nbsp;min at 4°C. The pre-cleaned cell extracts were transferred into clean vials. Recombinant His-tagged LanCL1 or LanCL2 (4&nbsp;μM) were added to the HEK293 cell extracts and incubated at 4°C. Equivalent amounts of LanCL&nbsp;storage buffer were added to the negative controls. After 2 h, cell extracts were treated with 30&nbsp;μL of HisPur Cobalt resin for His-pulldown at 4°C for 1 h. The cobalt resin was washed with 500&nbsp;μL of His-pulldown buffer three times and boiled with 30&nbsp;μL of 2X Laemmli buffer (Bio-Rad). A small fraction was used for western blot analysis as described above and most of the samples were resolved on a 4%–20% TGX gel (Bio-Rad). The gel was cut into slices that were submitted to the Harvard Taplin Mass Spectrometry Facility for proteomics analysis where they were treated with trypsin and the resulting digest analyzed by MS.</div></section><section id="sec-4-4-7"><h4>Proteomics sequence analysis</h4><div id="p0310" role="paragraph">Excised gel pieces were washed and dehydrated with acetonitrile and completely dried with a speed-vac. Acetonitrile-free gel pieces were rehydrated with 50&nbsp;mM ammonium bicarbonate solution containing 12.5&nbsp;ng/μL of modified sequencing-grade trypsin (Promega, Madison, WI) at 4°C for 45&nbsp;min. Trypsin solution was replaced with 50&nbsp;mM ammonium bicarbonate solution and samples were left at 37°C overnight. Peptides were extracted by removing the ammonium bicarbonate solution followed by one wash with 50% acetonitrile and 1% formic acid. The extracts were dried in a speed-vac for approximately 1&nbsp;h and stored at 4°C until analysis. Samples were reconstituted in 5 - 10&nbsp;μL of 5% acetonitrile and 0.1% formic acid solution. A nano-scale reverse-phase HPLC capillary column was created by packing 2.6&nbsp;μm C18 spherical silica beads into a fused silica capillary (100&nbsp;μm inner diameter x ∼30&nbsp;cm length) with a flame-drawn tip (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Peng and Gygi, 2001</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Peng, J. ∙ Gygi, S.P.</div><div><strong>Proteomics: the move to mixtures</strong></div><div><em>J.&nbsp;Mass Spectrom.</em> 2001; <strong>36</strong>:1083-1091</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jms.229" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0034757770" target="_blank" aria-label="Open Scopus in new tab">Scopus (528)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11747101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjms.229&amp;pmid=11747101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Samples were loaded on to the column via a Famos auto sampler (LC Packings, San Francisco CA). A gradient was formed, and peptides were eluted with increasing concentrations of 97.5% acetonitrile, 0.1% formic acid solution.</div><div id="p0315" role="paragraph">Eluted peptides were subjected to electrospray ionization and then entered into an LTQ Orbitrap Velos Pro ion-trap mass spectrometer (Thermo Fisher Scientific, Waltham, MA). Peptides were detected, isolated, and fragmented to produce a tandem mass spectrum of specific fragment ions for each peptide. Peptide sequences were determined by matching protein databases with the acquired fragmentation pattern by the software program Sequest (Thermo Fisher Scientific, Waltham, MA). All databases include a reversed version of all the sequences and the data was filtered to between a one and two percent peptide false discovery rate.</div><div id="p0320" role="paragraph">The intensity of ions corresponding to individual proteins were summed and compared to the negative control. If the protein was also observed in the negative control the data are expressed as fold-change compared to the negative control with a 10-fold change used as cut-off for inclusion. If the protein was not observed in the negative control, the protein is indicated as unique to the LanCL pulldown sample. The data are presented in <a id="crosref1230" href="#mmc1">Table S1</a>.</div></section><section id="sec-4-4-8"><h4><b>Purification of ProcA 2.8 mut, CylLs</b>’’ <b>and CylL</b><sub><b>L</b></sub>’’</h4><div id="p0325" role="paragraph"><i>E.&nbsp;coli</i> BL21(DE3) cells were transformed with the corresponding plasmid and protein expression was induced with 0.3&nbsp;mM IPTG in TB media at 18°C for 18&nbsp;h (<span class="dropBlock reference-citations"><a id="body-ref-sref17-4" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-4" aria-controls="sref17-4">Dong et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref17-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dong, S.H. ∙ Tang, W. ∙ Lukk, T. ...</div><div><strong>The enterococcal cytolysin synthetase has an unanticipated lipid kinase fold</strong></div><div><em>eLife.</em> 2015; <strong>4</strong>:e07607</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.07607" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84940538541" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.07607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Yang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Yang, X. ∙ Lennard, K.R. ∙ He, C. ...</div><div><strong>A lanthipeptide library used to identify a protein-protein interaction inhibitor</strong></div><div><em>Nat. Chem. Biol.</em> 2018; <strong>14</strong>:375-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41589-018-0008-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85042882496" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29507389/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41589-018-0008-5&amp;pmid=29507389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cell pellets were resuspended at 0.2&nbsp;mg/mL in LanA buffer B1 (4&nbsp;M guanidine-HCl, 20&nbsp;mM NaH<sub>2</sub>PO<sub>4</sub>, 500&nbsp;mM NaCl, 0.5&nbsp;mM imidazole, pH 7.5) and lysed by sonication. Insoluble debris was removed by centrifugation at 16,500&nbsp;× <i>g</i> for 30&nbsp;min at 4°C. The supernatant was purified by immobilized metal affinity chromatography using 2-4&nbsp;mL of His60 Ni Superflow Resin (Takara). Following 30&nbsp;min incubation at room temperature, the resin was washed with LanA B2 buffer (4&nbsp;M guanidine-HCl, 20&nbsp;mM NaH<sub>2</sub>PO<sub>4</sub>, 300&nbsp;mM NaCl, 30&nbsp;mM imidazole, pH 7.5). For the ProcA 2.8 mutant (ProcA 2.8 mut: C3A/S13A/C19A), the peptide was eluted with LanA elution buffer (4&nbsp;M guanidine hydrochloride, 20&nbsp;mM NaH<sub>2</sub>PO<sub>4</sub>, 100&nbsp;mM NaCl, 0.5&nbsp;M imidazole, pH 7.5). The elution fractions were desalted by C4 SPE column and purified on a Machery-Nagel NUCLEODUR® C18 HTec C18 column, (250&nbsp;mm × 7&nbsp;μm; part no. 762566.100). For CylL<sub>S</sub>’’ and CylL<sub>L</sub>," resins were washed again with LanA co-expression buffer (20&nbsp;mM NaH<sub>2</sub>PO<sub>4</sub>, 300&nbsp;mM NaCl, 30&nbsp;mM imidazole, pH 7.5) and eluted with LanA co-expression elution buffer (20&nbsp;mM NaH<sub>2</sub>PO<sub>4</sub>, 300&nbsp;mM NaCl, 0.5&nbsp;M imidazole, pH 7.5). To the eluted fractions, the protease CylA was added to a final concentration of 10&nbsp;μg/mL to remove the leader peptides (<span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">Tang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Tang, W. ∙ Bobeica, S.C. ∙ Wang, L. ...</div><div><strong>CylA is a sequence-specific protease involved in toxin biosynthesis</strong></div><div><em>J.&nbsp;Ind. Microbiol. Biotechnol.</em> 2019; <strong>46</strong>:537-549</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10295-018-2110-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85057230763" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30484123/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10295-018-2110-9&amp;pmid=30484123" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and produce CylL<sub>L</sub>’’ and CylL<sub>S</sub>." The core peptide was purified by preparatory HPLC (Agilent) equipped with a Phenomenex Jupiter® 4&nbsp;μm Proteo 90&nbsp;Å LC column (250&nbsp;× 10&nbsp;mm; Part No. 00G-4396-N0) using the gradient described in the section on peptide synthesis.</div></section><section id="sec-4-4-9"><h4>Generation of Dhb-containing ERK peptide by phosphothreonine lyase and glutathionylation by LanCL2</h4><div id="p0330" role="paragraph">Phosphate elimination was carried out using 5&nbsp;μM phosphothreonine lyase and 25&nbsp;μΜ peptide in reaction buffer (25&nbsp;mM Tris, 150&nbsp;mM NaCl, pH 8.0). For phospho-peptides that mimic the activation loop of ERK, peptides were treated with His<sub>6</sub>-SpvC or His<sub>6</sub>-OspF at 30°C for 90&nbsp;min. For the phospho-Akt peptide, peptide was treated with His<sub>6</sub>-OspF at 30°C for 18 h. The phosphate elimination process was monitored by matrix-assisted laser-desorption/ionization time-of-flight mass spectrometry (MALDI-TOF MS) and dehydroamino acid containing peptides were purified by RP-HPLC (Macherey-Nagel, VP 250/10 Nucleodur C18 HTec, 5&nbsp;μm). For glutathionylation of the ERK peptide, 25&nbsp;μM Dhb-containing peptide was dissolved in reaction buffer (25&nbsp;mM Tris, 150&nbsp;mM NaCl, 1&nbsp;mM glutathione (GSH), 1&nbsp;μM TCEP, pH 8.0) and incubated with 5&nbsp;μM LanCL2-His<sub>6</sub> at 25°C for 18 h. The glutathionylation process was monitored by MALDI-TOF MS and the resulting peptide was purified by RP-HPLC (Macherey-Nagel, VP 250/10 Nucleodur C18 HTec, 5&nbsp;μm). All peptides described in this section were purified by HPLC as described in the peptide synthesis section.</div></section><section id="sec-4-4-10"><h4>Hydrolysis of glutathionylated ERK peptide (GS-ERK), derivatization of the amino acids and GC-MS analysis</h4><div id="p0335" role="paragraph">GS-ERK peptide was enzymatically synthesized by LanCL2 from Dhb-ERK and purified by RP-HPLC (Macherey-Nagel, VP 250/10 Nucleodur C18 HTec, 5&nbsp;μm). GS-ERK peptide (1.5&nbsp;mg) was dissolved in 6&nbsp;M DCl in D<sub>2</sub>O (3&nbsp;mL). The reaction mixture was heated in an oil bath with stirring at 110°C for 24 h. After cooling down the reaction mixture to RT, the solvent was removed by rotary evaporator. Acetyl chloride (1.5&nbsp;mL) was added dropwise to pre-chilled MeOH (5&nbsp;mL). The mixed solution (3&nbsp;mL) was added to the hydrolyzed peptide and heated to 110°C for 1 h. The reaction mixture was cooled to RT and the solvent was removed by rotary evaporation. The residue was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (3&nbsp;mL) and pentafluoropropionic anhydride (1&nbsp;mL) was added into the reaction vessel. The reaction vessel was heated to 110°C for 1&nbsp;h with reflux. After cooling down the reaction vessel to RT, the sample was dried under a stream of argon. The dried residue was dissolved in MeOH and transferred to a clean vial. The samples were analyzed by GC-MS on an Agilent HP 6890N mass spectrometer with an Agilent CP-Chirasil-L-Val column (25&nbsp;m × 0.25&nbsp;mm × 0.12&nbsp;μm). The derivatized sample was diluted with MeOH and injected into the instrument with helium gas at a flow rate of 2.0&nbsp;mL/min. Samples were analyzed by the following gradient: the injection was held at 160°C for 5&nbsp;min and increased to 190°C at 3°C /min. The selected ion mode (SIM) was used to monitor 379&nbsp;Da for derivatized methyllanthionine (MeLan) (<span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Tang and van der Donk, 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Tang, W. ∙ van der Donk, W.A.</div><div><strong>Structural characterization of four prochlorosins: a novel class of lantipeptides produced by planktonic marine cyanobacteria</strong></div><div><em>Biochemistry.</em> 2012; <strong>51</strong>:4271-4279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi300255s" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84861624891" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22574919/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi300255s&amp;pmid=22574919" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The derivatized sample was co-injected with characterized synthetic standards of DL- and LL-methyllanthionine to identify the stereochemistry of GS-ERK peptide (<span class="dropBlock reference-citations"><a id="body-ref-sref60-3" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-3" aria-controls="sref60-3">Tang and van der Donk, 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Tang, W. ∙ van der Donk, W.A.</div><div><strong>Structural characterization of four prochlorosins: a novel class of lantipeptides produced by planktonic marine cyanobacteria</strong></div><div><em>Biochemistry.</em> 2012; <strong>51</strong>:4271-4279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi300255s" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84861624891" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22574919/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi300255s&amp;pmid=22574919" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" aria-label="Reference 2013" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">2013</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Tang, W. ∙ van der Donk, W.A.</div><div><strong>The sequence of the enterococcal cytolysin imparts unusual lanthionine stereochemistry</strong></div><div><em>Nat. Chem. Biol.</em> 2013; <strong>9</strong>:157-159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nchembio.1162" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84874115767" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23314913/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnchembio.1162&amp;pmid=23314913" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-11"><h4>LanCL1/2 kinetic assays with dehydroamino acid containing peptides</h4><div id="p0340" role="paragraph">Kinetic assays were performed with His<sub>6</sub>-LanCL1 or LanCL2-His<sub>6</sub> and Dhb-ERK/CylLs’’ purified as described previously. Assay mixtures consisted of 0.5&nbsp;mL containing 1&nbsp;mM GSH, different concentrations of Dhb-containing peptides (50/100/150/200/250/300/350&nbsp;μM), 0.5&nbsp;mM TCEP, 0.1&nbsp;mM ZnCl<sub>2</sub>, 150&nbsp;mM NaCl and 100&nbsp;mM Tris, pH 7.5. The assay was performed at room temperature for 30&nbsp;min. After addition of His<sub>6</sub>-LanCL2 at a final concentration of 0.5&nbsp;μM, reaction aliquots of 45&nbsp;μL were collected and quenched with 5&nbsp;μL of LanCL quench buffer (1.1&nbsp;M citric acid and 5.5&nbsp;mM EDTA) at 0, 1, 2, and 3&nbsp;min time points. Quenched reaction aliquots (5&nbsp;μL) were injected onto a Macherey-Nagel NUCLEODUR® 100-5 C18 ec column (125&nbsp;mm × 2&nbsp;mm; catalog no. 7600001.20) attached to a quadrupole/time-of-flight (Q/TOF) Synapt-G1 mass spectrometer (Waters). All kinetic and control experiments were performed in triplicate. The column was eluted with solvent A (100% H<sub>2</sub>O, 0.1% formic acid) and solvent B (100% MeCN, 0.1% formic acid) with a gradient of 2%–100% solvent B over 15&nbsp;min at a flow rate of 0.18&nbsp;mL/min. The Synapt instrument settings were as follows: positive ion mode, V optics, capillary voltage&nbsp;= 2.0 V, cone gas&nbsp;= 40 L/h, desolvation gas&nbsp;= 600 L/h, souce temperature&nbsp;= 120°C, desolvation temperature&nbsp;= 200°C. The instrument was externally calibrated with a 0.1% phosphoric acid standard. Data were collected over a m/z window of 50-2000&nbsp;Da in continuous mode with a 0.2&nbsp;s scan rate using a leucine-enkephalin lockspray solution for internal calibration. Spectra were smoothed and baselines were subtracted using the MassLynx software package (Waters). Extracted ion chromatograms (EICs) were generated by applying a mass window of 2.5&nbsp;Da around the center of the most intense peak of each product. The EICs were integrated using MassLynx software to generate peak areas. The EIC peak areas were used to calculate the concentration of glutathionylated product at each time point by comparison with standard curves. Initial rates at different substrate concentrations were calculated by linear regression fit of product concentration over time and fit to the Michaelis-Menten model using Prism 6 (GraphPad) for kinetic parameters <i>K</i><sub>M,peptide</sub> and <i>k</i><sub>cat</sub>.</div></section><section id="sec-4-4-12"><h4><i>In&nbsp;vitro</i> binding assays</h4><div id="p0345" role="paragraph">To determine the direct physical interaction of MEK1-Dha218 and LanCL2, 500&nbsp;nM of MBP or MBP-LanCL2 and 700&nbsp;nM of His<sub>6</sub>-MEK1-Dha218 were mixed at 4°C for 30&nbsp;min and subsequently incubated with 30&nbsp;μL HisPur cobalt resin (Thermo Fisher) for 7&nbsp;min. The resin was washed with 500&nbsp;μL of His-pulldown buffer three times and boiled with 30&nbsp;μL of 2X Laemmli buffer (Bio-Rad). The samples were analyzed by western blotting. MBP or MBP-LanCL2 was detected by anti-MBP tag antibody and His<sub>6</sub>-MEK1-Dha218 was detected by an anti-His tag antibody.</div></section><section id="sec-4-4-13"><h4>Preparation of MEF cell extracts for enzymatic analysis</h4><div id="p0350" role="paragraph">MEF cells (2x10<sup>6</sup>) were seeded in 100&nbsp;mm cell culture dishes and cultured with DMEM complete media for two days. The MEFs were then washed with ice-cold PBS buffer and cell pellets resuspended with enzymatic buffer (25&nbsp;mM Tris, 150&nbsp;mM NaCl, 1&nbsp;mM GSH, 1&nbsp;μM TCEP, pH 8.0). Cells were lysed by sonication followed by microcentrifugation at 14,100&nbsp;× <i>g</i> for 30&nbsp;min at 4°C. The supernatant was transferred into fresh tubes and the protein concentration was determined by bicinchoninic acid (BCA) assay (Thermo Fisher). Equal amounts of total protein (500&nbsp;μg) from wt and triple LanCL knockout MEFs were incubated with His<sub>6</sub>-MEK1-Dha222 (final concentration 1.5&nbsp;μM) at 25°C for 4 h. His<sub>6</sub>-MEK1-Dha222 was enriched by incubating with 30&nbsp;μL of HisPur Cobalt resin (Thermo Fisher) for 10&nbsp;min followed by washing the resin with 500&nbsp;μL of His-pulldown buffer three times and boiled with 30&nbsp;μL of 2X Laemmli buffer (Bio-Rad). All samples were analyzed by western blotting.</div></section><section id="sec-4-4-14"><h4>Site-selective formation of MEK-Dha218, MEK-Dha222, and MEK-Dha218Dha222</h4><div id="p0355" role="paragraph">MEK1C proteins in reaction buffer at pH 8.0 were treated with bisalkylating/elimination agents (added from a stock solution in DMSO,&nbsp;&lt; 2% v/v). The mixtures were shaken at 600&nbsp;rpm at 37°C or at room temperature. The reactions were monitored using LCMS-ESI-TOF. For more details see <a id="crosref1775" href="#figs2">Figures S2</a>, <a id="crosref1780" href="#figs3">S3</a>, <a id="crosref1785" href="#figs4">S4</a>, <a id="crosref1790" href="#figs5">S5</a>, and <a id="crosref1795" href="#figs6">S6</a> and <a id="crosref1265" href="#mmc2">Table S2</a>. For example, specifically for use of 2,5-methyldibromopentonate (MDBP), the purified MEK-C218 or MEK-C222 protein was desalted into reaction buffer (20&nbsp;mM Tris, 100&nbsp;mM NaCl, 25&nbsp;mM MgCl<sub>2</sub>, and 10&nbsp;mM ADP, pH 8.0) via PD SpinTrap G25 column. The concentration of desalted protein was measured by BCA protein assay. In order to obtain the dehydroalanine (Dha) form of MEK1 proteins through <i>in&nbsp;vitro</i> site-directed modification, MEK-C218 or MEK-C222 protein was incubated with 0.5 or 0.2&nbsp;mM TCEP, and 400 (or 200) molar equivalents of MDBP. The mixtures were kept at 25°C for 1 h, followed by increase of the reaction temperature to 37°C. After 2 h, MEK-C218 or MEK-C222 reached&nbsp;100% conversion to its Dha form (MEK-Dha218 or MEK-Dha222), as monitored by LC-MS using a XEVO mass spectrometer (Waters). The excess MDBP and TCEP were removed by the use of a SpinTrap G25 column. MEK-Dha proteins were stored in reaction buffer and kept at −80°C.</div></section><section id="sec-4-4-15"><h4>Generation of full length MEK-Dha/MEK-GSH ERK variants for phosphorylation assay</h4><div id="p0360" role="paragraph">Eliminated MEK1 variants, 30&nbsp;μM MEK-Dha222 or MEK-Dha218Dha222 were treated with 50&nbsp;mM glutathione (GSH) in Tris buffer (20&nbsp;mM Tris, 100&nbsp;mM NaCl, pH 8.0). After 2-hour incubation at 25°C, the complete conversion to glutathione adduct (MEKGS222 or MEKGS218GS222) was achieved and verified by LC-ESI-MS analysis (Xevo). The excess of glutathione in each MEK glutathione adduct was removed by a SpinTrap G25 column. The target product was obtained and confirmed by LC-ESI-MS analysis (Xevo).</div><div id="p0365" role="paragraph">For eliminated MEKGS218 variant, 30&nbsp;μM MEK-Dha218 was incubated with 20&nbsp;mM glutathione and 3.5&nbsp;μM LanCL1 protein. After 1-hour incubation at 25°C, the complete conversion to glutathione adduct was achieved and verified by LC-ESI-MS analysis (Xevo). The excess of glutathione in each MEK glutathione adduct was removed by a SpinTrap G25 column. The target product was obtained and confirmed by LC-ESI-MS analysis (Xevo).</div></section><section id="sec-4-4-16"><h4>MEK variants – ERK phosphorylation assay</h4><div id="p0370" role="paragraph">Purified ERK1 (0.5, 1.0 or 1.8&nbsp;μM) was incubated with MEK1 variants in kinase buffer (50&nbsp;mM Tris pH 7.5, 30&nbsp;mM NaCl, 10&nbsp;mM MgCl<sub>2</sub>, 0.2&nbsp;mM DTT) at 30°C. 1&nbsp;mM ATP was added to the reaction mixture. At the indicated time, an aliquot was removed and analyzed by LC-ESI-MS after the addition of 50&nbsp;mM EDTA to quench the reaction. The amounts of phosphorylated ERK were quantified using the intensities of ERK deconvoluted peaks from mass spectra and fitted by global least-square regression to solutions of kinetic models (see <a id="crosref1270" href="#figs6">Figure&nbsp;S6</a>) using OriginPro9.7. To control for the presence of residual potential LanCL1 protein (in any MEK-GS adducts that were synthesized using LanCL1) in phosphorylation assays, a control experiment of ERK phosphorylation also in the presence of 3.5&nbsp;μM LanCL1 protein was performed under the same conditions; the additional presence of LanCL1 had no significant effect on activity.</div></section><section id="sec-4-4-17"><h4>Formation of ERK1-Dha202 and ERK1-Dha82Dha202</h4><div id="p0375" role="paragraph">In order to generate ERK1-Dha202, the mutant protein ERK1-C202 was desalted into the reaction buffer consisting of 20&nbsp;mM Tris, 100&nbsp;mM NaCl, 0.1&nbsp;mM TCEP pH 8.0. After 1&nbsp;h incubation, TCEP was removed with a PD SpinTrap G25 column. ERK1-C202 protein was treated with 100 molar equivalents of MDBP immediately and the reaction was monitored by MS using a XEVO mass spectrometer. ERK1-C202 protein reached fully convertion to ERK-Dha202 within 24&nbsp;h at 25°C and excess MDBP was removed with a SpinTrap G25 desalting column. ERK1-Dha82Dha202 was generated using the same method as ERK1-Dha202.</div></section><section id="sec-4-4-18"><h4>MS/MS of eliminated MEK and ERK variants</h4><div id="p0380" role="paragraph">‘In-solution’ tryptic digestions were performed following an MS-compatible protocol. Specifically, for 10&nbsp;μg ERK1-Dha202 or ERK1-Dha82Dha202 protein sample was diluted into 100&nbsp;μl with 8&nbsp;M urea, 100&nbsp;mM Triethylammonium bicarbonate (TEAB) buffer and treated with 10&nbsp;mM TCEP at room temperature for 30&nbsp;mins. The mixture was incubated with 50&nbsp;mM freshly prepared iodoacetamide at room temperature for 30&nbsp;mins in dark. Then a ten-time dilution was performed with a buffer containing 8&nbsp;M urea and 50&nbsp;mM TEAB. The mixture was treated with trypsin (1:20 w/w trypsin:protein) at 37°C overnight. The salts within the digested samples were extracted by an Oasis HLB 1cc Vac cartridge (Waters) and the eluted peptide fragments were concentrated overnight on SpeedVac. The dried sample was dissolved in 10&nbsp;μl MilliQ water containing 0.1% formic acid and 2% acetonitrile for LC-MS/MS analysis on Orbitrap Elite mass spectrometer (Thermo Fisher).</div><div id="p0385" role="paragraph">For MEK-Dha218, MEK-Dha222 and MEK-Dha218Dha222 samples (25&nbsp;μL, around 1&nbsp;mg/mL) were diluted with 8&nbsp;M urea solution (75&nbsp;μL). 200&nbsp;mM aqueous DTT (2&nbsp;μL) was then added and the mixture was heated at 56°C for 15&nbsp;min. The mixtures were cooled down to room temperature and treated with a 36&nbsp;mg/mL iodoacetamide in 50&nbsp;mM ammonium carbonate solution (2&nbsp;μL). The mixture was left standing in the dark for 30&nbsp;min. 200&nbsp;mM aqueous DTT (2&nbsp;μL) was then added to neutralize the iodoacetamide. The mixture was diluted four times with 50&nbsp;mM ammonium bicarbonate solution and treated with trypsin (1:50 w/w ratio enzyme:protein). The mixture was incubated at 37°C overnight. Some samples were also treated with AspN (1:50) and incubated at 37°C for 4 h. The sample was diluted down to 200 fmol/μL with, either milliQ water or 1% formic acid before analysis LC-MS/MS analysis on an Orbitrap Elite mass spectrometer (Thermo Fisher). Data was processed and analyzed using PEAKS software v7.0.</div></section><section id="sec-4-4-19"><h4>Kinetics of glutathionylation of MEK-Dha by LanCLs</h4><div id="p0390" role="paragraph">LanCL1/2 were incubated with 0.1&nbsp;mM zinc chloride before adding to the reaction due to its Zn<sup>2+</sup>-binding property. The reaction mixtures, consisting of 6&nbsp;μM MEK-Dha218 protein and 1.2&nbsp;μM LanCL1/2, were incubated at 25°C with up to 120&nbsp;μM glutathione (GSH). At each time point, an aliquot (1&nbsp;μL) was diluted with 49&nbsp;μL of 100&nbsp;mM ammonium acetate solution (pH 8.0), and the conversion of MEK-Dha218 to MEK-GS218 was monitored and quantified by mass spectrometry. Initial rates for the formation of MEK-GS218 conjugate were plotted against the concentrations of substrate GSH, the apparent <i>K</i><sub>M,GSH</sub> and <i>k</i><sub>cat</sub> values were obtained using a fit of the data to the Michaelis-Menten equation using Origin9.7.</div><div id="p0395" role="paragraph">The following MS set-up was used: Xevo G2-S and G2-XS mass spectrometer systems (Waters) were coupled to an Acquity UPLC using a Thermo ProSwiftTM RP-2H column (4.6&nbsp;mm x 50&nbsp;mm; catalog no. 064296). All kinetic experiments were carried out in triplicate. Water (as solvent A) and acetonitrile (as solvent B) both containing 0.1% formic acid were used as the mobile phase with a gradient of 5-95%–5% solvent B over 10&nbsp;min at a flow rate of 0.3&nbsp;mL/min. The electrospray source was operated with a capillary voltage of 3 kV and a cone voltage of 20 V. Spectra were calibrated by use of an internal lock-spray. Desolvation temperature was 400°C and desolvation gas at a total flow of 700 L/h or 650 L/h for Xevo G2-S or G2-XS mass spectrometer, respectively. Total mass spectra were deconvoluted from multiply charged ion series using the MaxEnt algorithm preinstalled on MassLynx Software (v4.1).</div></section><section id="sec-4-4-20"><h4>Glutathionylation of ERK1-Dha202 and ERK1-Dha82Dha202 comparative time course</h4><div id="p0400" role="paragraph">LanCL1/2 wt or mutant proteins (LanCL1-C322A or LanCL2-C321A/C367A) were pre-treated with 0.1&nbsp;mM ZnCl<sub>2</sub> for 30&nbsp;mins. 6&nbsp;μM ERK1-Dha202 or ERK1-Dha82Dha202 was incubated with 1.0&nbsp;mM glutathione (GSH) and 1.2&nbsp;μM LanCL proteins at 25°C up to 3 h. A negative control experiment without the addition of LanCL proteins was performed under the same conditions. At each time interval, an aliquot from the above mixture was diluted with 100&nbsp;mM ammonia acetate (pH 8.0) buffer and analyzed by mass spectrometry. The deconvoluted spectra of LC-MS were used to quantify the conversion to conjugates ERK1-GS202 or ERK1-Dha82GS202.</div></section><section id="sec-4-4-21"><h4>Generation of dehydrobutyrine-containing ERK2 (ERK2-Dhb185pY187)</h4><section id="sec-4-4-21-1"><h5>GST-Flag-ERK2 purification</h5><div id="p0405" role="paragraph">The plasmid pGEX-4T-1-3xFlag-ERK2 with a GST-tag and triple Flag-tag at the N terminus was obtained from Addgene (plasmid no. 47573; gift from Kevin Janes). The 3xFlag tag was inserted at BamHI site, and a new BamHI site was regenerated at its downstream. <i>erk2</i> gene was inserted between BamHI and SalI sites of this pGEX-4T1 vector. GST-Flag-ERK2 protein was expressed in <i>E.&nbsp;coli</i> Rosetta 2 (DE3) cells. The overnight cultures were inoculated in a 1L fresh LB media with 100&nbsp;μg/ml ampicillin at 37°C. Cells were induced with 0.5&nbsp;mM IPTG until OD<sub>600</sub> reaches 0.7 and incubated at 18°C for further18 h. Cell pellets were harvested and then lysed in a 25&nbsp;mL ice-cold PBS buffer with 1X protease inhibitor. The supernatants were centrifugated at 76,000 x g for 30&nbsp;mins at 4°C, filtered by centrifugal filters and then applied onto a GSTrap column (GE healthcare) at a flow rate of 1.5&nbsp;mL/min. The loaded column was washed with 10 column volumes of PBS to remove non-bound components and the purified GST-Flag-ERK2 protein was recovered from the column with a elution buffer (1xPBS buffer containing 10&nbsp;mM reduced glutathione and 1X protease inhibitor). The fractions with desired protein were analyzed by SDS-PAGE and verified by mass spectrometry.</div></section><section id="sec-4-4-21-2"><h5>Phosphorylation of ERK2 using MEK-Dha218Dha222</h5><div id="p0410" role="paragraph">The mutant protein His<sub>6</sub>-MEK-C218/C222 was expressed in <i>E.&nbsp;coli</i> BL21(DE3) cells and purified by Ni<sup>2+</sup>-NTA column according to the same methods as shown for MEK-C218 and MEK-C222 proteins. The purified MEK-C218C222 protein in reaction buffer (20&nbsp;mM Tris, 100&nbsp;mM NaCl, 25&nbsp;mM MgCl<sub>2</sub>, 10&nbsp;mM ADP, pH 8.0) was incubated with 0.2&nbsp;mM TCEP for 1&nbsp;h at 25°C and then treated with 600 molar equivalents of bis-alkylating reagent MDBP gradually. The mixture was kept at 25°C for 1 h, followed by incubating at 37°C for 2 h. The fully converted MEK-Dha218Dha222 protein was verified by LC-ESI-MS using Xevo mass spectrometer (Waters) and desalted by a SpinTrap column to remove excess of MDBP and TCEP.</div><div id="p0415" role="paragraph">Double phosphorylation of GST-Flag-ERK2 at Thr185 and Tyr187 residues was achieved by incubating 200&nbsp;μM GST-Flag-ERK2 protein with 4&nbsp;μM His<sub>6</sub>-MEK-Dha218Dha222 in the reaction buffer (20&nbsp;mM Tris, 100&nbsp;mM NaCl, 25&nbsp;mM MgCl<sub>2</sub>, 25&nbsp;mM ATP, pH 8.0) at 25°C for 4&nbsp;h and then at 4°C for 16 h. The fully phosphorylated GST-Flag-ERK2-pT185pY187 was confirmed by mass spectrometry. To remove His<sub>6</sub>-MEK-Dha218Dha222 protein from double phosphorylated ERK, the reaction mixture was applied to a 1&nbsp;mL pre-equilibrated HisTrap column and incubated at 4°C for 1 h. The column was washed with 3 column volumes of Tris buffer (20&nbsp;mM Tris, 100&nbsp;mM NaCl, pH 8.0) and the proteins were eluted with increased amounts of imidazole (20 to 500&nbsp;mM). Each fraction was analyzed by SDS-PAGE. The purified GST-Flag-ERK2-pT185pY187 was in the fractions containing 20&nbsp;mM imidazole and concentrated by a Vivaspin6 10&nbsp;kDa-cutoff concentrator.</div></section><section id="sec-4-4-21-3"><h5>Elimination of ERK2-pThr185-using phosphothreonine lyase</h5><div id="p0420" role="paragraph">Phosphate elimination was achieved with 80&nbsp;μM GST-Flag-ERK2-pT185pY187 and 16&nbsp;μM His<sub>6</sub>-OspF at 25°C for 1 h. The reaction process was monitored by mass spectrometry. The dehydrobutyrine-containing ERK protein (GST-Flag-ERK2-Dhb185pY187) was separated from His<sub>6</sub>-OspF by a 1&nbsp;mL HisTrap column using the similar procedures as mentioned above. The fractions of purified GST-Flag-ERK2-Dhb185pY187 (containing 20&nbsp;mM imidazole in Tris buffer) were combined and desalted to remove imidazole.</div></section></section><section id="sec-4-4-22"><h4>Test of glutathionylation of Dhb-containing ERK2 (ERK2-Dhb185pY187) by LanCL1/2</h4><div id="p0425" role="paragraph">For the glutathionylation of ERK2-Dhb185pY187 protein, 10&nbsp;μM Dhb-containing ERK2 was incubated with 1&nbsp;mM GSH and 10&nbsp;μM LanCL protein (LanCL1 or LanCL2) for up to 7&nbsp;h at 25°C. At each time interval, an aliquot (2&nbsp;μL) of reaction mixture was diluted into 48&nbsp;μL 100&nbsp;mM ammonia acetate (pH 8.0) buffer (with 1&nbsp;mM DTT) and analyzed by mass spectrometry. The conversion of ERK2-Dhb185pY187 protein to ERK2-GS185pY187 was monitored and quantified via deconvoluted intact protein mass spectra.</div></section><section id="sec-4-4-23"><h4>Western blot analysis of phosphate elimination and glutathionylation of His<sub>6</sub>-ERK2</h4><div id="p0430" role="paragraph">Phosphate elimination was carried out using 5&nbsp;μM bisphosphorylated His<sub>6</sub>-ERK2 (obtained by co-expression with MEK1-R4F (<span class="dropBlock reference-citations"><a id="body-ref-sref33-4" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-4" aria-controls="sref33-4">Khokhlatchev et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref33-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Khokhlatchev, A. ∙ Xu, S. ∙ English, J. ...</div><div><strong>Reconstitution of mitogen-activated protein kinase phosphorylation cascades in bacteria. Efficient synthesis of active protein kinases</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1997; <strong>272</strong>:11057-11062</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0030966367" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9110999/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.272.17.11057&amp;pmid=9110999" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)) and 1&nbsp;μM His<sub>6</sub>-OspF and incubated at 30°C for 1 h. A total of 1&nbsp;μM MBP-LanCL2 was added into the reaction mixture and incubated at 25°C for 3 h. Phosphorylation of ERK2 and Dhb formation were first verified by MS. His<sub>6</sub>-ERK2-pThr185pTyr187 was analyzed with a Thermo Q Exactive Ultra High Mass Range (UHMR) Mass Spectrometer. First buffer exchange was performed for phospho-ERK2 (50&nbsp;μM) using a Zeba spin column with 25&nbsp;mM ammonium acetate. The resulting sample was analyzed. Calculated mass: 42329.53&nbsp;Da Observed mass: 42333.72 Da. For the Dhb-containing ERK2 in solution tryptic digestion was performed and the resulting peptides were analyzed by using a Thermo Fusion Orbitrap Tribrid Mass Spectrometer. Calculated mass for the relevant peptide phosphorylated on Tyr and containing Dhb at the position of the former pThr: 2204.9508&nbsp;Da Observed: 2204.9528 Da. All reaction mixtures were also boiled with 2X Laemmli buffer (Bio-Rad) and analyzed by western blotting. Phosphate elimination was detected using anti-phospho-ERK1/2 antibody (Cell Signaling Technology). Glutathionylation was monitored using anti-GSH antibody (Virogen). The amount of total ERK2 protein was detected using anti-ERK1/2 antibody (Cell Signaling Technology).</div></section><section id="sec-4-4-24"><h4>Accessibility analysis for Cys residues</h4><div id="p0435" role="paragraph">Accessibility analysis applied to unbound <i>apo</i>-MEK1 structure [generated from the binary MEK1-ATPγS structure (PDB <a id="intref0195" href="http://firstglance.jmol.org/fg.htm?mol3W8Q" target="_blank">3W8Q</a>) minus ligand ATPγS] predicted (<a id="crosref1280" href="#fig3">Figure&nbsp;3</a>B) that Cys might be added through SDM at sites 218 and/or 222 to give residues with highest predicted reactivity, as long as higher accessibility residues C277, C376 (93.5, 21.9% accessibility, respectively, <a id="crosref1285" href="#fig3">Figure&nbsp;3</a>B) were removed. Thus, mutation to near-isosteric, unreactive Ser yielded MEK1-C277S/C376S with essentially identical enzymatic properties to MEK1wt (<a id="crosref1290" href="#mmc2">Table S2</a>). This protein was used as the base mutant sequence for all subsequent alterations.</div><div id="p0440" role="paragraph">Critically, in the mutated construct used in this study, C207, C121, C142 and C341 were retained. C207 lies within the ATP-binding pocket; C121, 207, 341 mutants variously (<span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Wagle et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Wagle, N. ∙ Emery, C. ∙ Berger, M.F. ...</div><div><strong>Dissecting therapeutic resistance to RAF inhibition in melanoma by tumor genomic profiling</strong></div><div><em>J.&nbsp;Clin. Oncol.</em> 2011; <strong>29</strong>:3085-3096</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1200/JCO.2010.33.2312" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-80051625929" target="_blank" aria-label="Open Scopus in new tab">Scopus (830)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21383288/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1200%2FJCO.2010.33.2312&amp;pmid=21383288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Zhao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Zhao, K. ∙ Ju, Y. ∙ Li, S. ...</div><div><strong>S-sulfhydration of MEK1 leads to PARP-1 activation and DNA damage repair</strong></div><div><em>EMBO Rep.</em> 2014; <strong>15</strong>:792-800</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/embr.201338271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84899806842" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24778456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fembr.201338271&amp;pmid=24778456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) alter activity and C207 is a key target site for covalent inhibitors (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Liu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Liu, Q. ∙ Sabnis, Y. ∙ Zhao, Z. ...</div><div><strong>Developing irreversible inhibitors of the protein kinase cysteinome</strong></div><div><em>Chem. Biol.</em> 2013; <strong>20</strong>:146-159</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.chembiol.2012.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.chembiol.2012.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84874301754" target="_blank" aria-label="Open Scopus in new tab">Scopus (532)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23438744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chembiol.2012.12.006&amp;pmid=23438744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0445" role="paragraph">Importantly, the same predictive analysis of reactivity suggested that in binary (MEK1⋅ATPγS [PDB <a id="intref0200" href="http://firstglance.jmol.org/fg.htm?mol3W8Q" target="_blank">3W8Q</a>]) or ternary (MEK1⋅ATPγS⋅Mg [PDB <a id="intref0205" href="http://firstglance.jmol.org/fg.htm?mol3EQD" target="_blank">3EQD</a>] &amp; MEK1⋅ADP⋅Mg [PDB <a id="intref0210" href="http://firstglance.jmol.org/fg.htm?mol3EQI" target="_blank">3EQI</a>]) complexes accessibility to C207 is dramatically decreased (accessibility 22.4→∼0.1%, <a id="crosref1310" href="#fig3">Figures 3</a>B and 3C), consistent with C207’s location in the nucleotide binding site of MEK1. In these complexes, predicted accessibilities of proposed reaction sites C222 and C218 were also importantly either unaffected or enhanced (<a id="crosref1315" href="#fig3">Figure&nbsp;3</a>C). Notably, Mg(II) induces a conformational change that usefully enhances accessibility of both sites (C222: 38.4→93.4%, C218: 8.9→15.2%, respectively, <a id="crosref1320" href="#fig3">Figure&nbsp;3</a>C). These analyses prompted the additional consideration of nucleotides (ATPγS, ATP, ADP) (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Lamoureaux and Lee, 2011</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Lamoureaux, T.L. ∙ Lee, D.H.</div><div><strong>Chemical activation of MEK1--a redox trigger for evaluating the effects of phosphorylation</strong></div><div><em>Chem. Commun. (Camb.).</em> 2011; <strong>47</strong>:8623-8625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c1cc11745a" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79960474817" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21717004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc1cc11745a&amp;pmid=21717004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and metals (Mg(II)) as masking or enhancing ligands to usefully modulate regioselectivity (<a id="crosref1330" href="#fig3">Figure&nbsp;3</a>C). In this way, we designed and generated three key mutant variants of MEK1 for testing as substrates in regioselective elimination chemistry: MEK1-C218, MEK1-C222, MEK1-C218C222 (bearing five, five or six Cys residues, respectively).</div><div id="p0450" role="paragraph">Prior to reactivity testing, interactions with masking agents were confirmed: differential scanning fluorimetry (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Niesen et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Niesen, F.H. ∙ Berglund, H. ∙ Vedadi, M.</div><div><strong>The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</strong></div><div><em>Nat. Protoc.</em> 2007; <strong>2</strong>:2212-2221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2007.321" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-37249005205" target="_blank" aria-label="Open Scopus in new tab">Scopus (1888)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17853878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2007.321&amp;pmid=17853878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) showed strong binding of putative masking/enhancing ligands to form MEK1⋅ATP(±Mg) and MEK1⋅ATPγS(±Mg) (<a id="crosref1340" href="#figs2">Figure&nbsp;S2</a>).</div><div id="p0455" role="paragraph">Consistent with predictions, <i>all</i> of the MEK1 mutant proteins, in the <i>absence</i> of masking nucleotide/metal gave products of unwanted reaction at Cys207 with parent reagent DBHDA (<a id="crosref1345" href="#fig4">Figure&nbsp;4</a>A, <a id="crosref1350" href="#figs2">Figure&nbsp;S2</a>, <a id="crosref1355" href="#mmc2">Table S2</a>). For elimination of MEK1-C222, use of alternative alkylating agents did not prevent unwanted C207 reaction (<a id="crosref1360" href="#figs2">Figure&nbsp;S2</a>, <a id="crosref1365" href="#mmc2">Table S2</a>). However, use of nucleotide additive proved successful: thus, while ATP only partially masked C207 in MEK1-C222 (<a id="crosref1370" href="#figs2">Figure&nbsp;S2</a>), ATP⋅Mg with DBHDA masked it fully, successfully allowing elimination reaction at site 222 (<a id="crosref1375" href="#fig4">Figure&nbsp;4</a>A). Concomitant observation of ∼25% phosphorylated MEK1-Dha222 (plus ADP release, <a id="crosref1380" href="#fig4">Figure&nbsp;4</a>A) also revealed competitive autophosphorylation during reaction. To solve this problem, the same reaction was performed ‘masked’ instead by ADP⋅Mg and this led to clean, selective formation of desired MEK1-Dha222 after 5&nbsp;h (<a id="crosref1385" href="#fig4">Figure&nbsp;4</a>A, <a id="crosref1390" href="#figs3">Figure&nbsp;S3</a>). Use of other reagents from the panel similarly allowed clean formation of MEK1-Dha222 at tuneable rates (<a id="crosref1395" href="#figs3">Figure&nbsp;S3</a>, <a id="crosref1400" href="#mmc2">Table S2</a>).</div><div id="p0460" role="paragraph">Reaction of MEK1-C218 with less accessible site 218 was slower: DBHDA&nbsp;+ ADP⋅Mg gave fully converted MEK1-Dha218 in 15&nbsp;h (<a id="crosref1405" href="#fig4">Figure&nbsp;4</a>B) along with unwanted side-products of overalkylation without elimination that increased with longer reaction (<a id="crosref1410" href="#figs4">Figure&nbsp;S4</a>, likely pyrrolidine-Lys). Other conditions were therefore sought: screening of alternative reagents (<a id="crosref1415" href="#mmc2">Table S2</a>) revealed clean, site-selective Dha formation (<a id="crosref1420" href="#fig4">Figure&nbsp;4</a>B; <a id="crosref1425" href="#figs4">Figure&nbsp;S4</a>) using DBDGla (20 h, 37°C) or MDBP (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Morrison et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Morrison, P.M. ∙ Foley, P.J. ∙ Warriner, S.L. ...</div><div><strong>Chemical generation and modification of peptides containing multiple dehydroalanines</strong></div><div><em>Chem. Commun. (Camb.).</em> 2015; <strong>51</strong>:13470-13473</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/C5CC05469A" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26219458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FC5CC05469A&amp;pmid=26219458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (4 h, 37°C).</div><div id="p0465" role="paragraph">Following successful regio-selective formation of both MEK1-Dha218 and MEK1-Dha222 through single site elimination, we tested double elimination in MEK1-C218C222. DBHDA alone showed no Dha formation and only partial alkylation (<a id="crosref1435" href="#fig4">Figure&nbsp;4</a>C, <a id="crosref1440" href="#figs2">Figure&nbsp;S2</a>). Promisingly, DBHDA&nbsp;+ ATP⋅Mg gave partial conversion (∼80%) to MEK1 bearing Dha, albeit with concomitant autophosphorylation (<a id="crosref1445" href="#figs5">Figure&nbsp;S5</a>). Screening of reagents and masking agents (<a id="crosref1450" href="#mmc2">Table S2</a>) revealed that DBDGla&nbsp;+ ADP⋅Mg gave only low conversion to MEK1-Dha218Dha222 (23%, 14 h, <a id="crosref1455" href="#figs5">Figure&nbsp;S5</a>). Pleasingly, however, MDBP&nbsp;+ ADP⋅Mg cleanly gave MEK1-Dha218Dha222 (<a id="crosref1460" href="#fig4">Figure&nbsp;4</a>C, <a id="crosref1465" href="#figs5">Figure&nbsp;S5</a>).</div></section></section><section id="sec-4-5"><h3>Synthetic chemistry procedures</h3><section id="sec-4-5-1"><h4>General consideration</h4><section id="sec-4-5-1-1"><h5>Reagents</h5><div id="p0470" role="paragraph">Chemicals and solvents were purchased from Sigma-Aldrich UK, Acros UK, Alfa Aesar UK, or Fischer UK and were used as delivered unless stated otherwise. This included 2,5-methyldibromopentanoate (MDBP).</div></section><section id="sec-4-5-1-2"><h5>Small molecule characterization</h5><div id="p0475" role="paragraph">Proton nuclear magnetic resonance (δH) spectra were recorded on a Bruker AV400 (400 MHz), or on a Bruker AVII500 (500 MHz) spectrometer. Carbon nuclear magnetic resonance (δC) spectra were recorded on a Bruker AV400 (100.7 MHz) spectrometer or on a Bruker AVII500 (125.8 MHz) spectrometer. All chemical shifts are quoted on the δ scale in ppm using residual solvent as the internal standard (<sup>1</sup>H NMR: CDCl<sub>3</sub>&nbsp;= 7.26, CD<sub>3</sub>OD&nbsp;= 4.87; <sup>13</sup>C NMR: CDCl<sub>3</sub>&nbsp;= 77.0; CD<sub>3</sub>OD&nbsp;= 49.0; D<sub>2</sub>O&nbsp;= 4.80). The following splitting abbreviations were used: s&nbsp;= singlet, d&nbsp;= doublet, t&nbsp;= triplet, q&nbsp;= quartet, a&nbsp;= apparent. All NMR spectra are found in <a id="crosref1470" href="#mmc5">Methods S1</a>. Infrared spectra were recorded on a Bruker Tensor 27 Fourier Transform spectrophotometer by attenuated total reflectance (ATR) –oil or crystals of the compound were placed directly against the stage. A background measurement was subtracted from each spectrum and a new background measurement was recorded before each spectrum was recorded. Absorption maxima (υ<sub>max</sub>) are reported in wavenumbers (cm<sup>-1</sup>) and classified as strong (s) or broad (br). Only signals representing functional groups are reported; CH absorptions as well as the fingerprint region are not listed. Low resolution mass spectra were recorded on a Micromass Platform 1 spectrometer using electrospray ionization (ESI). High resolution mass spectra were recorded on a Walters 2790-Micromass LCT electrospray ionization mass spectrometer. m/z values are reported in Daltons. Thin layer chromatography (TLC) was carried out using Merck aluminum backed sheets coated with 60F254 silica gel. Visualization of the silica plates was achieved using a UV lamp (λmax&nbsp;= 254&nbsp;nm), and/or ammonium molybdate (5% in 2M H<sub>2</sub>SO<sub>4</sub>), or potassium permanganate (5% in 1M NaOH). Flash column chromatography was carried out using BDH PROLAB® 40-63&nbsp;mm silica gel (VWR).</div></section></section><section id="sec-4-5-2"><h4>Synthesis of bisalkylating/elimination agents</h4><section id="sec-4-5-2-1"><h5>2,5-Dibromohexanediamide, DBHDA</h5><div id="p0480" role="paragraph"><div class="figure-wrap"><figure id="undfig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/00ceb0d9-9a4f-4ca6-9ff9-8f5141b1f40e/main.assets/fx2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx2" alt="Figure undfig2" src="/cms/10.1016/j.cell.2021.04.001/asset/acc57732-f752-4653-89c6-a045ed4705bd/main.assets/fx2.jpg" height="53" width="156" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/00ceb0d9-9a4f-4ca6-9ff9-8f5141b1f40e/main.assets/fx2_lrg.jpg" loading="lazy"></a><div><button class="figure__open__ctrl">Figure viewer</button></div></figure></div></div><div id="p0485" role="paragraph">Adipic acid (25 g, 171.1&nbsp;mmol) was suspended in thionylchloride (75&nbsp;mL, 1034&nbsp;mmol). The reaction was heated to reflux (open to air, bath temp 80°C). After 30&nbsp;min at reflux, the solid had dissolved. The reaction was stirred for an additional 60&nbsp;min at reflux and then cooled to room temperature. Carbontetrachloride (100&nbsp;mL) was added to the reaction followed by <i>N</i>-bromosuccinimide (73 g, 411&nbsp;mmol). The reaction was stirred vigorously and 10 drops of HBr (48% aq.) was added by pipette. The reaction was heated to reflux, again open to air. The reaction gradually turns from red to black over the course of an hour. After 2&nbsp;h at reflux, the reaction was cooled to room temperature and then to 0°C. The mixture was stirred at 0°C to ensure all succinimide had precipitated. The solid was removed by filtration. Diethylether (50&nbsp;mL) was used to rinse and complete the filtration. The filtrate was concentrated under reduced pressure to give a thick, dark red liquid. In a 500&nbsp;mL round bottom flask, 200&nbsp;mL of NH<sub>4</sub>OH (25% aq.) was cooled to 0°C. The crude acid chloride was added dropwise over 20&nbsp;min to the ammonia solution with rapid stirring. After the addition was complete, the reaction was stirred vigorously at 0°C for 1 h. The bis-amide product precipitated from the reaction mixture. The dark solid was isolated by filtration and partially dried. The product was purified by triturating in MeOH/H<sub>2</sub>O: The solid was suspended in H<sub>2</sub>O (100&nbsp;mL) and MeOH (100&nbsp;mL) and heated to 60°C. The mixture was stirred rapidly at 60°C for 30&nbsp;min. After this time, the mixture was cooled to room temperature. The resulting white solid (a mixture of meso and D/L diastereomers) was isolated by filtration and washed with MeOH (200&nbsp;mL). The title compound was dried under high vacuum to afford 28.4&nbsp;g (55%) as an off-white solid.</div><div id="p0490" role="paragraph"><sup>1</sup>H NMR (400 MHz, DMSO-d6) δ ppm 1.75-2.08 (4H, m, H3, H4), 4.28-4.36 (2H, m, H2, H5), 7.30 (2H, s, H7, H8), 7.69 (2H, s, H7, H8). <sup>13</sup>C NMR (100 MHz, DMSO-d6): (both diastereomers reported), δ ppm 32.5 (s, C3, C4), 32.6 (s, C3, C4), 48.2 (s, C2, C5), 48.5 (s, C2, C5), 169.87 (s, C1), 169.92 (s, C5) (<a id="crosref1475" href="#mmc5">Methods S1</a>). IR (υ<sub>max</sub>, KBr): 3302, 2946, 2801, 1684, 1418, 1319, 1277, 1250, 1220, 1194, 978, 876. HRMS m/z (ESI+): Found 300.9178 [M+H]<sup>+</sup>; C<sub>8</sub>H<sub>11</sub>O<sub>2</sub>N<sub>2</sub><sup>79</sup>Br<sub>2</sub> requires 300.9182.</div></section><section id="sec-4-5-2-2"><h5>2,5-Dibromohexanedioic acid, DBHDAc</h5><div id="p0495" role="paragraph"><div class="figure-wrap"><figure id="undfig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/1b4c4521-2ea3-453f-aa41-2417ab752dbd/main.assets/fx3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx3" alt="Figure undfig3" src="/cms/10.1016/j.cell.2021.04.001/asset/8f8d90ba-8658-4790-acfb-6f8058fa3af7/main.assets/fx3.jpg" height="52" width="146" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/1b4c4521-2ea3-453f-aa41-2417ab752dbd/main.assets/fx3_lrg.jpg" loading="lazy"></a><div><button class="figure__open__ctrl">Figure viewer</button></div></figure></div></div><div id="p0500" role="paragraph">Adipic acid (2.0 g, 13.7&nbsp;mmol) and thionyl chloride (2.98&nbsp;mL, 41.1&nbsp;mmol) were stirred at reflux for 1 h. The mixture was cooled down to room temperature and treated with <i>N</i>-bromosuccinimide (7.3 g, 41.1&nbsp;mmol). The mixture was stirred at 75°C for 3 h. The mixture was cooled down to 0°C before addition of a 1:1 mixture of water/tetrahydrofuran (20&nbsp;mL). The mixture was stirred vigorously for 2 h. The mixture was filtered. The residue was stirred in dichloromethane for 1 h. Filtration afforded 1.3&nbsp;g (32%) of the title compound (mixture of racemate and meso diastereoisomer) as an off-white solid.</div><div id="p0505" role="paragraph"><sup>1</sup>H NMR (400 MHz, DMSO-d6) δ ppm 1.83 - 2.22 (m, 4H, H3, H4), 4.45 - 4.60 (m, 2H, H2, H5), 13.32 (br. s., 2H, H7, H8). <sup>13</sup>C NMR (101 MHz, DMSO-d6) δ ppm 32.05 (s, C3, C4), 32.17 (s, C3, C4), 46.74 (s, C2, C5), 46.88 (s, C2, C5), 170.44 (s, C1, C6). (<a id="crosref1480" href="#mmc5">Methods S1</a>). HRMS m/z (ES-): Found 300.8718 [M-H]<sup>-</sup>; C<sub>6</sub>H<sub>7</sub>O<sub>4</sub><sup>79</sup>Br<sub>2</sub> requires 300.8717.</div><div id="p0510" role="paragraph">The compound (50&nbsp;mg, 0.016&nbsp;mmol) was stirred in water (1&nbsp;mL) for 16 h. The solid was filtered to afford meso-2,5-dibromohexanedioic acid.</div></section><section id="sec-4-5-2-3"><h5>2,5-Dibromo-N,N’-dimethylhexanediamide, DBDMet</h5><div id="p0515" role="paragraph"><div class="figure-wrap"><figure id="undfig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/32af1d64-2fd1-466b-9c6e-401c233831cc/main.assets/fx4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx4" alt="Figure undfig4" src="/cms/10.1016/j.cell.2021.04.001/asset/f193a92b-e6d4-4175-8a12-c4e02bf23922/main.assets/fx4.jpg" height="61" width="164" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/32af1d64-2fd1-466b-9c6e-401c233831cc/main.assets/fx4_lrg.jpg" loading="lazy"></a><div><button class="figure__open__ctrl">Figure viewer</button></div></figure></div></div><div id="p0520" role="paragraph">2,5-Dibromohexanedioic acid (100&nbsp;mg, 0.33&nbsp;mmol) and thionyl chloride (239&nbsp;μL, 3.3&nbsp;mmol) was stirred at reflux for 2 h. The solution was cooled down to room temperature before evaporation under reduced pressure. The residue was then added to a stirred 40% wt aqueous solution of methyl amine (1&nbsp;mL, 13.6&nbsp;mmol) at 0°C. The mixture was stirred vigorously for 2 h. The precipitate was filtered and stirred in a minimum of dichloromethane. Filtration afforded 35&nbsp;mg (32%) of title compound as a white solid.</div><div id="p0525" role="paragraph"><sup>1</sup>H NMR (400 MHz, DMSO-d<sub>6</sub>) δ ppm 1.67 - 2.13 (m, 4H, H3, H4), 2.61 (d, J&nbsp;= 4.58&nbsp;Hz, 6H, H10, H12), 4.23 - 4.45 (m, 2H, H2, H5), 8.13 - 8.36 (m, 2H, H9, H11). <sup>13</sup>C NMR (101 MHz, DMSO-d<sub>6</sub>) δ ppm 25.87 (s, C10, C12), 32.67 (s, C3, C4), 48.24 (s, C2, C5), 168.31 (s, C1, C6) (<a id="crosref1485" href="#mmc5">Methods S1</a>). IR (υmax, KBr): 3279, 1650, 1571. HRMS m/z (ESI+): Found 328.9493 [M+H]<sup>+</sup>; C<sub>8</sub>H<sub>15</sub>O<sub>2</sub>N<sub>2</sub><sup>79</sup>Br<sub>2</sub> requires 328.9495.</div></section><section id="sec-4-5-2-4"><h5>meso-N,N’-bis(2-amino-2-oxoethyl)-2,5-dibromohexanediamide, DBDGla</h5><div id="p0530" role="paragraph"><div class="figure-wrap"><figure id="undfig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/8574f97c-9ea8-468c-8376-345f5c2acee5/main.assets/fx5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx5" alt="Figure undfig5" src="/cms/10.1016/j.cell.2021.04.001/asset/c1e3339d-df60-47be-9570-e77f75555d7f/main.assets/fx5.jpg" height="61" width="260" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/8574f97c-9ea8-468c-8376-345f5c2acee5/main.assets/fx5_lrg.jpg" loading="lazy"></a><div><button class="figure__open__ctrl">Figure viewer</button></div></figure></div></div><div id="p0535" role="paragraph">2,5-Dibromohexanedioic acid (100&nbsp;mg, 0.33&nbsp;mmol) and thionyl chloride (100&nbsp;μL, 1.38&nbsp;mmol) was stirred at reflux for 1.5 h. The solution was cooled down to room temperature before evaporation under reduced pressure. The residue was then diluted in diethyl ether (1&nbsp;mL). The solution was added dropwise to a stirred solution of glycinamide hydrochloride (72&nbsp;mg, 0.66&nbsp;mmol) in a mixture of diethyl ether (500&nbsp;μL) and 2&nbsp;M aqueous sodium hydroxide solution (660&nbsp;μL) at 0°C. The mixture was stirred for 1&nbsp;h at room temperature. The organic layer was separated. The aqueous was acidified to pH∼2 with addition of a 1&nbsp;N hydrochloric acid solution. The mixture was washed with ethyl acetate (2&nbsp;mL). The aqueous layer (suspension) was separated, evaporated under reduced pressure and azeotroped twice with toluene. The residue was stirred in acetonitrile for 1&nbsp;h at room temperature. The precipitate was filtered to afford 130&nbsp;mg of crude material. 40&nbsp;mg of this material were stirred in water for 16 h. Filtration afforded 7&nbsp;mg (17%) of the title compound as a white solid.</div><div id="p0540" role="paragraph"><sup>1</sup>H NMR (400 MHz, DMSO-d<sub>6</sub>) δ ppm 1.73 - 1.92 (m, 2H, H3, H4), 2.02 - 2.19 (m, 2 H, H3, H4), 3.55 - 3.83 (m, 4 H, H10, H12), 4.43 - 4.63 (m, 2H, H2, H5), 7.09 (br. s., 2 H, H14, H16), 7.39 (br. s., 2 H, H14, H16), 8.44 (t, J&nbsp;= 5.62&nbsp;Hz, 2H, H9, H11). <sup>13</sup>C NMR (101 MHz, DMSO-d<sub>6</sub>) δ ppm 32.65 (s, C3, C4), 42.04 (s, C10, C12), 48.01 (s, C2, C3), 168.27 (s, C13, C14), 170.24 (s, C1, C6) (<a id="crosref1490" href="#mmc5">Methods S1</a>). IR (υ<sub>max</sub>, KBr): 3292, 1679, 1651, 1600, 1556. HRMS m/z (ESI-): Found 412.9467 [M-H]<sup>-</sup>; C<sub>10</sub>H<sub>15</sub>O<sub>4</sub>N<sub>4</sub><sup>79</sup>Br<sub>2</sub> requires 412.9466.</div></section><section id="sec-4-5-2-5"><h5>({(meso)-2,5-dibromo-6-[(carboxymethyl)amino]-6-oxohexanoyl}amino)acetic acid, DBDGly</h5><div id="p0545" role="paragraph"><div class="figure-wrap"><figure id="undfig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.001/asset/7761b99d-e7ec-4bf7-9587-ce46c25e2aac/main.assets/fx6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx6" alt="Figure undfig6" src="/cms/10.1016/j.cell.2021.04.001/asset/424a46b3-29f6-4b26-8095-8169ef2732fe/main.assets/fx6.jpg" height="61" width="243" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/7761b99d-e7ec-4bf7-9587-ce46c25e2aac/main.assets/fx6_lrg.jpg" loading="lazy"></a><div><button class="figure__open__ctrl">Figure viewer</button></div></figure></div></div><div id="p0550" role="paragraph">2,5-Dibromohexanedioic acid (100&nbsp;mg, 0.33&nbsp;mmol) and thionyl chloride (100&nbsp;μL, 1.38&nbsp;mmol) was stirred at reflux for 1.5 h. The solution was cooled down to room temperature before evaporation under reduced pressure. The residue was then diluted in diethyl ether (1&nbsp;mL). The solution was added dropwise to a stirred solution of glycine (50&nbsp;mg, 0.66&nbsp;mmol) in a mixture of diethyl ether (500&nbsp;μL) and 2&nbsp;M aqueous sodium hydroxide solution (660&nbsp;μL) at 0°C. The mixture was stirred for 1&nbsp;h at room temperature. The organic layer was separated. The aqueous was acidified to pH∼2 with addition of a 1&nbsp;N hydrochloric acid solution. The mixture was washed with ethyl acetate (2&nbsp;mL). The aqueous layer (suspension) was separated, evaporated under reduced pressure and azeotroped twice with toluene. The residue was stirred in acetonitrile for 1&nbsp;h at room temperature. The precipitate was filtered to afford 100&nbsp;mg of crude material. 40&nbsp;mg of this material were stirred in water for 16 h. Filtration afforded 4&nbsp;mg (9%) of the title compound as a white solid. The solid was highly soluble in water.</div><div id="p0555" role="paragraph"><sup>1</sup>H NMR (400 MHz, DMSO-d6) δ ppm 1.77 - 1.93 (m, 2H, H3, H4), 2.01 - 2.15 (m, 2H, H3, H4), 3.80 (dd, J&nbsp;= 5.87, 3.42&nbsp;Hz, 4H, H10, H12), 4.38 - 4.59 (m, 2H, H2, H5), 8.60 (t, J&nbsp;= 5.75&nbsp;Hz, 2H, H9, H11), 12.66 (br. s, 2H, H14, H16). <sup>13</sup>C NMR (101 MHz, DMSO-d6) δ ppm 32.69 (s, C3, C4), 40.95 (s, C10, C12), 47.56 (s, C2, C5), 168.44 (s, C13, C15), 170.78 (s, C1, C6) (<a id="crosref1495" href="#mmc5">Methods S1</a>). IR (υmax, KBr): 3279, 1650, 1571. HRMS m/z (ESI+): Found 416.9292 [M+H]<sup>+</sup>; C<sub>10</sub>H<sub>15</sub>O<sub>6</sub>N<sub>2</sub><sup>79</sup>Br<sub>2</sub> requires 416.9291.</div></section></section><section id="sec-4-5-3"><h4>Eliminylome analysis</h4><div id="p0560" role="paragraph">Previously deposited MS raw data by Medard et&nbsp;al. (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Médard et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Médard, G. ∙ Pachl, F. ∙ Ruprecht, B. ...</div><div><strong>Optimized chemical proteomics assay for kinase inhibitor profiling</strong></div><div><em>J.&nbsp;Proteome Res.</em> 2015; <strong>14</strong>:1574-1586</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/pr5012608" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84924371112" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25660469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fpr5012608&amp;pmid=25660469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were processed using MaxQuant software integrated with the Andromeda search engine (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Cox and Mann, 2008</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Cox, J. ∙ Mann, M.</div><div><strong>MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</strong></div><div><em>Nat. Biotechnol.</em> 2008; <strong>26</strong>:1367-1372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1511" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-57449099865" target="_blank" aria-label="Open Scopus in new tab">Scopus (10772)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19029910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1511&amp;pmid=19029910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Cox et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Cox, J. ∙ Neuhauser, N. ∙ Michalski, A. ...</div><div><strong>Andromeda: a peptide search engine integrated into the MaxQuant environment</strong></div><div><em>J.&nbsp;Proteome Res.</em> 2011; <strong>10</strong>:1794-1805</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/pr101065j" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79953701087" target="_blank" aria-label="Open Scopus in new tab">Scopus (4049)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21254760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fpr101065j&amp;pmid=21254760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0565" role="paragraph">For identification of putative eliminylome components of serine and threonine residues, the peak lists were searched using MaxQuant software against the human protein database with carbamidomethyl as fixed modification for cysteine residues. Elimination of serine and threonine residues to form Dha and Dhb, respectively, as well as their Michael adducts with DTT, beta-mercaptoethanol (bME), and glutathione were used along with acetylation (at N terminus), oxidation (at M), and phosphorylation (at S, T, or Y) as variable modifications. Trypsin was selected as the proteolytic enzyme, and up to two missed cleavages were allowed. The mass tolerance of the precursor ion was set to 4.5 ppm and for fragmentation ion to 0.5 Da. Protein and PSM false discovery rate were set at 0.01. The threshold score for modified protein selected for inclusion was set at 40. A parallel search was performed for identification of putative eliminylome components of cysteine residues. Elimination of cysteine residues to form Dha as well as their Michael adducts with DTT, beta-mercaptoethanol (bME), and glutathione were used along with carbamidomethyl (at C), acetylation (at N terminus), oxidation (at M), and phosphorylation (at S,T, or Y) as variable modifications. All other processing parameters were kept same as previously mentioned.</div><div id="p0570" role="paragraph">The resulting data were then manually curated for correlation of elimination to Dha or Dhb at S or T phosphorylation sites in protein kinases, respectively, and Cys sites in kinase domains using at least one occurrence as noted under the post-translational modification function in UniProt (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">The Uniprot consortium, 2019</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>The Uniprot consortium</div><div><strong>UniProt: a worldwide hub of protein knowledge</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D506-D515</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85059797762" target="_blank" aria-label="Open Scopus in new tab">Scopus (5038)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30395287/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1049&amp;pmid=30395287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Veuthey et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Veuthey, A.L. ∙ Bridge, A. ∙ Gobeill, J. ...</div><div><strong>Application of text-mining for updating protein post-translational modification annotation in UniProtKB</strong></div><div><em>BMC Bioinformatics.</em> 2013; <strong>14</strong>:104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-14-104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84875129833" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23517090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-14-104&amp;pmid=23517090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and PhosphoSitePlus (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Hornbeck et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Hornbeck, P.V. ∙ Zhang, B. ∙ Murray, B. ...</div><div><strong>PhosphoSitePlus, 2014: mutations, PTMs and recalibrations</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:D512-D520</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku1267" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84941120469" target="_blank" aria-label="Open Scopus in new tab">Scopus (2082)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25514926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku1267&amp;pmid=25514926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-4"><h4>Generation of Dha-GPX by H<sub>2</sub>O<sub>2</sub> oxidation</h4><div id="p0575" role="paragraph">Glutathione peroxidase (GPX) from bovine erythrocytes was purchased from Sigma-Aldrich in the form of lyophilized powder and contained DTT. To remove DTT, the initial powder was dissolved in Tris buffer (25&nbsp;mM Tris, 150&nbsp;mM NaCl, pH 7.5) and purified by Zeba spin column. The concentration of the resulting GPX solution was quantified by nanodrop. Following a reported protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Cho et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Cho, C.S. ∙ Lee, S. ∙ Lee, G.T. ...</div><div><strong>Irreversible inactivation of glutathione peroxidase 1 and reversible inactivation of peroxiredoxin II by H<sub>2</sub>O<sub>2</sub> in red blood cells</strong></div><div><em>Antioxid. Redox Signal.</em> 2010; <strong>12</strong>:1235-1246</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ars.2009.2701" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-77951813369" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20070187/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Fars.2009.2701&amp;pmid=20070187" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Dha-GPX was generated by incubating the GPX solution with 1&nbsp;mM H<sub>2</sub>O<sub>2</sub> for 1&nbsp;h at 37°C. The oxidation product was purified by Zeba spin column to remove excess H<sub>2</sub>O<sub>2</sub> and quantified by nanodrop.</div></section><section id="sec-4-5-5"><h4>Synthesis of biotinylated cysteamine to detect dehydroalanine (Dha) in GPX</h4><div id="p0580" role="paragraph">Nonenzymatic thiol Michael addition was used to detect whether Dha was successfully generated in GPX upon H<sub>2</sub>O<sub>2</sub> oxidation (<span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Rhee and Cho, 2010</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Rhee, S.G. ∙ Cho, C.S.</div><div><strong>Blot-based detection of dehydroalanine-containing glutathione peroxidase with the use of biotin-conjugated cysteamine</strong></div><div><em>Methods Enzymol.</em> 2010; <strong>474</strong>:23-34</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0076-6879(10)74002-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79958041926" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20609902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0076-6879%2810%2974002-7&amp;pmid=20609902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For this purpose, biotinylated cysteamine was synthesized. Cysteamine hydrochloride (11.4&nbsp;mg, 0.1&nbsp;mmol) was treated with 0.5&nbsp;mL of PBS solution of NaHCO<sub>3</sub> (1 M) and 0.1&nbsp;mL of a DMF solution of EZ-linked <i>N</i>-hydroxysuccinimide biotin (0.5 M) for 3&nbsp;h at room temperature. To the above reaction mixture was added TCEP hydrochloride (45&nbsp;mg, 0.15&nbsp;mmol). The resulting solution was incubated at room temp for another 2 h. After lyophilizing the reaction mixture, the sample was dissolved in 0.1% TFA, filtered through a 0.2&nbsp;μM filter, and purified by reversed-phase HPLC on a C18 column. The mass of biotinylated cysteamine was confirmed by MS (ESI): [M&nbsp;+ H]<sup>+</sup> calculated, 304.1075; found, 304.1145. The detection of Dha in GPX was performed by following an established protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Rhee and Cho, 2010</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Rhee, S.G. ∙ Cho, C.S.</div><div><strong>Blot-based detection of dehydroalanine-containing glutathione peroxidase with the use of biotin-conjugated cysteamine</strong></div><div><em>Methods Enzymol.</em> 2010; <strong>474</strong>:23-34</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0076-6879(10)74002-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79958041926" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20609902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0076-6879%2810%2974002-7&amp;pmid=20609902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, Dha-containing GPX (∼3&nbsp;μg) was treated with a PBS solution containing 0.1&nbsp;M NaHCO<sub>3</sub>, 0.5&nbsp;mM biotinylated cysteamine, 1&nbsp;mM TCEP, and 1% SDS for 20&nbsp;h at 37°C. The reaction mixture was then boiled with 2X Laemmli buffer (Bio-Rad) and separated by SDS-PAGE. The separated proteins were transferred to a nitrocellulose membrane for blot analysis with HRP-conjugated streptavidin (Thermo Scientific). GPX was used as the control in the western blot analysis (<a id="crosref1545" href="#figs1">Figure&nbsp;S1</a>G).</div></section><section id="sec-4-5-6"><h4>Investigating the possibility of LanCL-catalyzed GSH addition to Dha-GPX</h4><div id="p0585" role="paragraph">Dha-containing GPX (10&nbsp;μM) was dissolved in reaction buffer (25&nbsp;mM Tris, 150&nbsp;mM NaCl, 1&nbsp;mM GSH, 1&nbsp;mM TCEP, pH 7.5) and incubated with 2&nbsp;μM His<sub>6</sub>-LanCL1 or LanCL2-His<sub>6</sub> (preactivated with 0.1&nbsp;mM ZnCl<sub>2</sub> at room temperature for 0.5 h) at room temperature for 2 h. The reaction mixtures were boiled with 2X Laemmli buffer and analyzed by western blotting. No protein bands were detected upon staining with anti-GSH antibody (Virogen) compared to positive control, which suggests that LanCL does not catalyze GSH addition to Dha-GPX.</div></section><section id="sec-4-5-7"><h4>Investigating the possibility of LanCL2-catalyzed crosslinking of thioredoxin and Dhb-ERK peptide</h4><div id="p0590" role="paragraph">To investigate whether LanCL may catalyze protein crosslinking, we performed a preliminary exploration based on a model system in which thioredoxin was chosen as the thiol nucleophile given its reactive and accessible Cys and Dhb-ERK was chosen as the electrophile. Dhb-ERK (25&nbsp;μM) and thioredoxin (Trx, 100&nbsp;μM) in reaction buffer (25&nbsp;mM Tris, 150&nbsp;mM NaCl, 1&nbsp;mM TCEP, pH 7.5) were incubated with 5&nbsp;μM LanCL2-His<sub>6</sub> (preactivated with 0.1&nbsp;mM ZnCl<sub>2</sub>) at room temperature. The reaction mixture was monitored by MALDI-TOF MS over 5 h. No crosslinked products were detected. To test whether LanCL2-His<sub>6</sub> was still active, 1&nbsp;mM GSH was introduced to the reaction at the end of the above incubation. The GSH-containing reaction mixture was further incubated at room temperature for an additional 1&nbsp;h and then analyzed by MALDI-TOF MS. The glutathionylation of Dhb-ERK was clearly detected in the presence of LanCL2-His<sub>6</sub>. Thus, LanCL2 was active but did not add Cys from Trx to Dhb-ERK.</div></section><section id="sec-4-5-8"><h4>Knock-out mice phenotype</h4><div id="p0595" role="paragraph">In 2011 we first generated LanCL2−/− KO mice. The first generation of these mice displayed the phenotype shown in <a id="crosref1550" href="#figs7">Figure&nbsp;S7</a>E where about 30% of the KO mice died between ∼2 and 6&nbsp;months. Histopathology analysis did not identify any common cause of death and for most mice no obvious cause of death was found. The surviving KO mice did not display any phenotype and subsequent generations did not display the premature death phenotype. Hence, we assumed that a subset of mice for unknown reasons possibly associated with other genetic differences died and that we had lost the phenotype in the surviving mice. In the subsequent years we generated LanCL1−/− and LanCL3−/− mice and through breeding generated TKO mice in 2016. These mice did not display any obvious phenotype when we used them to test the hypothesis that LanCL proteins are responsible for lanthionine formation in the brain (<span class="dropBlock reference-citations"><a id="body-ref-sref26-7" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-7" aria-controls="sref26-7">He et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref26-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>He, C. ∙ Zeng, M. ∙ Dutta, D. ...</div><div><strong>LanCL proteins are not Involved in Lanthionine Synthesis in Mammals</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:40980</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep40980" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85010403537" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28106097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep40980&amp;pmid=28106097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, a colony of TKO mice that were born in early 2020 displayed near-identical premature death phenotype (<a id="crosref1560" href="#fig6">Figures 6</a>H and <a id="crosref1565" href="#figs7">S7</a>D) as observed in 2011.</div><div id="p0600" role="paragraph">Multiple seizures were observed before death in several of the TKO mice that died prematurely. Necropsy specimen from nine mice were examined, including one TKO male, six TKO females, one sacrificed wt male, and one wt female. Four out of six TKO female mice examined had necrotic livers containing coagula of hypereosinophilic, hepatocellular debris with degenerative neutrophils. Two TKO female mice had necrotic endometrium with bacterial infection. Steiner stain of necrotizing mouse livers was negative. Gram stain on endometrium and placentae showed intralesional colonies of gram-positive cocci. No significant histopathological abnormality was seen in wt mice or TKO males. Overall no common cause of death was evident in TKO mice that died prematurely.</div></section><section id="sec-4-5-9"><h4>Survival analysis</h4><div id="p0605" role="paragraph">A total of 105 mice (58 WT and 47 LanCL TKO) were studied and used to generate Kaplan-Meier (KM) survival plots (<a id="crosref1570" href="#fig6">Figures 6</a>H and <a id="crosref1575" href="#figs7">S7</a>D). Survival times were monitored for up to 210&nbsp;days. The following data were collected, i) age (if mice were alive then, age&nbsp;= age at end of study, if mice died then, age&nbsp;= age at death), ii) sex; differences between wt and TKO survival were determined by the Gehan-Breslow-Wilcoxon test. Statistical significance in all tests was accepted at p&nbsp;&lt; 0.05. The KM plots were generated using GraphPad Prism 8.0.2. The older LanCL2 data involved 50 LanCL2&nbsp;−/− mice and 24 wt mice and provided similar overall numbers (<a id="crosref1580" href="#figs7">Figure&nbsp;S7</a>E). Censored subjects are healthy mice that were sacrificed to compare tissues containing high LanCL levels (based on previous studies) to monitor for potential differences in GSH-adduct in wt and TKO mice (see below). In addition, all surviving mice at the end of the time interval are considered censored as their long term survival was not known at the end of the study.</div></section><section id="sec-4-5-10"><h4>Protein extraction from mouse brain and heart tissue</h4><div id="p0610" role="paragraph">Mouse brain and heart from deceased TKO mice, sacrificed TKO, or wt mice were dissected and snap frozen immediately in liquid nitrogen. Frozen tissue was homogenized and lysed in tissue extraction buffer (200&nbsp;nM Tris HCl pH 7.4, 150&nbsp;mM NaCl, 1&nbsp;mM EGTA, 1&nbsp;mM EDTA, 2.5&nbsp;mM sodium pyrophosphate, 1&nbsp;mM β-glycerol phosphate, 1&nbsp;mM sodium orthovanadate, 2% Triton X-100 and a protease inhibitor) using a tissue homogenizer. Homogenized tissue was rotated for 30&nbsp;min and centrifuged at 14,100&nbsp;× g for 10&nbsp;min at 4°C. Supernatant was transferred to a clean vial and protein concentration was determined using Pierce BCA Protein Assay Kit. LanCL1/2 and GSH, or GSH only were added to the protein, and the samples were analyzed by western blot after SDS-PAGE under reducing conditions using anti-glutathione antibody (Virogen). No difference was observed between TKO and wt samples.</div></section><section id="sec-4-5-11"><h4>Immunoprecipitation of ERK1/2 from total mouse brain protein and reaction with LanCL2</h4><div id="p0615" role="paragraph">In an effort to potentially concentrate a dehydroamino acid containing protein, mouse brain total protein was pre-cleaned with Protein A Agarose Beads (Cell Signaling Technology) to remove endogenous IgG and incubated with ERK1/2 antibody (Cell Signaling Technology) at 4°C overnight. The solution was then incubated with Protein A Agarose Beads at 4°C for 4&nbsp;h and micro-centrifuged at 250&nbsp;× <i>g</i> for 1&nbsp;min. Supernatant was removed and beads were washed with tissue extraction buffer 3 times.</div><div id="p0620" role="paragraph">Beads with ERK1/2 immunoprecipitated from 200&nbsp;μg of WT or TKO mouse brain total protein were treated with 4&nbsp;μM LanCL2, 500&nbsp;μg glutathione (GSH) (Sigma-Aldrich), and 0.5&nbsp;mM tris(2-carboxyethyl) phosphine (TCEP), and incubated at 37°C for 1 h. Samples were washed three times with tissue extraction buffer and boiled with 20&nbsp;μL of 2X Laemmli buffer (Bio-Rad). All samples were analyzed by western blotting to detect <i>C-</i>glutathionylated ERK using anti-glutathione antibody (Virogen). No difference was observed between TKO and wt brain.</div></section></section><section id="sec-4-6"><h3>Quantification and statistical analysis</h3><div id="p0625" role="paragraph">For all non-quantitative experiments such as mass spectrometric analysis, experiments were performed in triplicate with a typical outcome shown that was representative of all three experiments. Kinetic experiments with dehydroamino acid containing peptides were performed in triplicate. The amounts of products formed were quantified by LC-MS as previously reported (<span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Thibodeaux et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Thibodeaux, C.J. ∙ Ha, T. ∙ van der Donk, W.A.</div><div><strong>A price to pay for relaxed substrate specificity: a comparative kinetic analysis of the class II lanthipeptide synthetases ProcM and HalM2</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2014; <strong>136</strong>:17513-17529</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja5089452" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84919344174" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja5089452&amp;pmid=25409537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The EICs were integrated using MassLynx software to generate peak areas. The EIC peak areas were used to calculate the concentration of glutathionylated product at each time point by comparison with standard curves. Initial rates at different substrate concentrations were calculated by linear regression fit of product concentration over time and these intitial rates were fit to the Michaelis-Menten model using Prism 6 (GraphPad) for kinetic parameters <i>K</i><sub>M,peptide</sub> and <i>k</i><sub>cat</sub>. For kinetic experiments to determine kinase activity of MEK variants, the amounts of phosphorylated ERK were quantified using the intensities of ERK observed in deconvoluted peaks from mass spectra obtained by LC-MS. The results, shown as mean ± standard deviation of triplicate experiments, were fitted by global least-square regression to solutions of kinetic models (see <a id="crosref1590" href="#figs6">Figure&nbsp;S6</a>) using OriginPro9.7.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0630" role="paragraph">This study was supported by the Howard Hughes Medical Institute (to W.A.V.), NIH GM089771 (to J.C.), NIH GM 079038 (to S.K.N.), the BBSRC with AstraZeneca (S.R.G.G.), the EPSRC UK Catalysis Hub (EP/R026815/1, EP/R026939/1 Y.Z., G.J.H., and B.G.D.), the EU Horizon 2020 Programme under the Marie Sklodowska-Curie program (700124, J.R.), the Felix Foundation (R.R.), and the EPSRC with Pfizer (K.P.C.). A MALDI TOF mass spectrometer was purchased with a grant from the National Institutes of Health (S10 RR027109 A). We thank Dr. Ian R. Bothwell and Dr. Alex V. Ulanov (Roy J. Carver Biotechnology Center, University of Illinois at Urbana-Champaign) for assistance with GC-MS, Elisabete Pires for assistance with MS/MS analysis, Dr. Shih-Hsuan Hsiao (Veterinary Medicine, UIUC) for histopathology of deceased TKO mice, and Drs. J. Read and G. Pairaudeau (AstraZeneca) for useful discussions.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0635" role="paragraph">K.-Y.L. performed pull-down, peptide activity, and cellular assays; Y.Z. and J.R. performed glutathionylation of MEK-Dha and ERK-Dha; S.R.G.G., K.P.C., R.R., J.R., L.H.J., and G.J.H. developed Dha installation technology; R.R. and S.R.G.G. performed protein MS/MS; T.H.Z. performed unbiased pull-down and mouse tissue analysis; T.H.Z. and M.Z. constructed survival curves; S.L. provided GPX. R.R., B.G.D., and S.M. conducted the eliminylome survey; N.G. and S.K.N. performed crystallography; C.H. performed kinetics with peptides. J.C., B.G.D., and W.A.V. designed the study. K.-Y.L., C.H., J.C., S.R.G.G., B.G.D., and W.A.V. wrote the manuscript; and all authors read and commented on the manuscript.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0640" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (5)</h2><div id="p0650" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.04.001/attachment/6f78b334-7b3e-4f75-a417-357cb3d027d8/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (569.78 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Unbiased protein pull-down and proteomics analysis, related to Figure&nbsp;1E</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.04.001/attachment/21b17e29-57d1-4163-9a21-4f57665537cb/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (22.57 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Chemical generation of Dha-containing MEK1 variants and their kinase activity, related to Figures 3, 4, S2, S3, S4, S5, and S6</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.04.001/attachment/a5fe6e10-bcdd-4bd6-bfa4-1713bbe628e4/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (66.17 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Analysis of the proteomic literature for the presence of Dha and Dhb proteins and their conjugates with GSH, β-mercaptoethanol, Cys, and dithiothreitol, related to Figures 5 and S7</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.04.001/attachment/8ca23b38-ab97-4112-a357-7115e90bc7b4/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (15.93 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Oligonucleotide sequences used in this study, related to the STAR Methods</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.04.001/attachment/fb8a904e-8e09-43d3-bc4c-bb092d41b911/mmc5.pdf" download="mmc5.pdf" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">PDF (138.45 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Methods S1. NMR spectra of bisalkylating agents used to introduce dehydro amino acids into proteins, related to Figures 3 and 4</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Alessi, D.R. ∙ Saito, Y. ∙ Campbell, D.G. ...</div><div><strong>Identification of the sites in MAP kinase kinase-1 phosphorylated by p74raf-1</strong></div><div><em>EMBO J.</em> 1994; <strong>13</strong>:1610-1619</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/j.1460-2075.1994.tb06424.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0028293931" target="_blank" aria-label="Open Scopus in new tab">Scopus (474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8157000/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fj.1460-2075.1994.tb06424.x&amp;pmid=8157000" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Arbibe, L. ∙ Kim, D.W. ∙ Batsche, E. ...</div><div><strong>An injected bacterial effector targets chromatin access for transcription factor NF-kappaB to alter transcription of host genes involved in immune responses</strong></div><div><em>Nat. Immunol.</em> 2007; <strong>8</strong>:47-56</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni1423" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33846958783" target="_blank" aria-label="Open Scopus in new tab">Scopus (332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17159983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni1423&amp;pmid=17159983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Bauer, H. ∙ Mayer, H. ∙ Marchler-Bauer, A. ...</div><div><strong>Characterization of p40/GPR69A as a peripheral membrane protein related to the lantibiotic synthetase component C</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 2000; <strong>275</strong>:69-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/bbrc.2000.3260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0034682901" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10944443/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fbbrc.2000.3260&amp;pmid=10944443" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Bose, A.K. ∙ Janes, K.A.</div><div><strong>A high-throughput assay for phosphoprotein-specific phosphatase activity in cellular extracts</strong></div><div><em>Mol. Cell. Proteomics.</em> 2013; <strong>12</strong>:797-806</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O112.024059&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O112.024059&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84874584886" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23233447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.O112.024059&amp;pmid=23233447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Brennan, D.F. ∙ Barford, D.</div><div><strong>Eliminylation: a post-translational modification catalyzed by phosphothreonine lyases</strong></div><div><em>Trends Biochem. Sci.</em> 2009; <strong>34</strong>:108-114</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-62849120767" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19233656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tibs.2008.11.005&amp;pmid=19233656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Bruns, C.M. ∙ Hubatsch, I. ∙ Ridderström, M. ...</div><div><strong>Human glutathione transferase A4-4 crystal structures and mutagenesis reveal the basis of high catalytic efficiency with toxic lipid peroxidation products</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1999; <strong>288</strong>:427-439</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jmbi.1999.2697" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0033531970" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10329152/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjmbi.1999.2697&amp;pmid=10329152" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7-1" title="View in article">7.</a></div><div>Chalker, J.M. ∙ Gunnoo, S.B. ∙ Boutureira, O. ...</div><div><em>Chem. Sci. (Camb.).</em> 2011; <strong>2</strong>:1666-1676</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c1sc00185j" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-81355138529" target="_blank" aria-label="Open Scopus in new tab">Scopus (284)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc1sc00185j" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Chambers, K.A. ∙ Abularrage, N.S. ∙ Scheck, R.A.</div><div><strong>Selectivity within a family of bacterial phosphothreonine lyases</strong></div><div><em>Biochemistry.</em> 2018; <strong>57</strong>:3790-3796</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.biochem.8b00534" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85047643839" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29792689/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.biochem.8b00534&amp;pmid=29792689" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Chatterjee, C. ∙ Paul, M. ∙ Xie, L. ...</div><div><strong>Biosynthesis and mode of action of lantibiotics</strong></div><div><em>Chem. Rev.</em> 2005; <strong>105</strong>:633-684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/cr030105v" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-14844340728" target="_blank" aria-label="Open Scopus in new tab">Scopus (644)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15700960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fcr030105v&amp;pmid=15700960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Cho, C.S. ∙ Lee, S. ∙ Lee, G.T. ...</div><div><strong>Irreversible inactivation of glutathione peroxidase 1 and reversible inactivation of peroxiredoxin II by H<sub>2</sub>O<sub>2</sub> in red blood cells</strong></div><div><em>Antioxid. Redox Signal.</em> 2010; <strong>12</strong>:1235-1246</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ars.2009.2701" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-77951813369" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20070187/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Fars.2009.2701&amp;pmid=20070187" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Chung, C.H. ∙ Kurien, B.T. ∙ Mehta, P. ...</div><div><strong>Identification of lanthionine synthase C-like protein-1 as a prominent glutathione binding protein expressed in the mammalian central nervous system</strong></div><div><em>Biochemistry.</em> 2007; <strong>46</strong>:3262-3269</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi061888s" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33947395638" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17305318/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi061888s&amp;pmid=17305318" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">12.</a></div><div>Cooper, A.J. ∙ Pinto, J.T. ∙ Callery, P.S.</div><div><strong>Reversible and irreversible protein glutathionylation: biological and clinical aspects</strong></div><div><em>Expert Opin. Drug Metab. Toxicol.</em> 2011; <strong>7</strong>:891-910</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1517/17425255.2011.577738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79959221560" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21557709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1517%2F17425255.2011.577738&amp;pmid=21557709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Cox, J. ∙ Mann, M.</div><div><strong>MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</strong></div><div><em>Nat. Biotechnol.</em> 2008; <strong>26</strong>:1367-1372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1511" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-57449099865" target="_blank" aria-label="Open Scopus in new tab">Scopus (10772)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19029910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1511&amp;pmid=19029910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Cox, J. ∙ Neuhauser, N. ∙ Michalski, A. ...</div><div><strong>Andromeda: a peptide search engine integrated into the MaxQuant environment</strong></div><div><em>J.&nbsp;Proteome Res.</em> 2011; <strong>10</strong>:1794-1805</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/pr101065j" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79953701087" target="_blank" aria-label="Open Scopus in new tab">Scopus (4049)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21254760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fpr101065j&amp;pmid=21254760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Dai, P. ∙ Zhang, C. ∙ Welborn, M. ...</div><div><strong>Salt effect accelerates site-selective cysteine bioconjugation</strong></div><div><em>ACS Cent. Sci.</em> 2016; <strong>2</strong>:637-646</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acscentsci.6b00180" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85019331701" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27725962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facscentsci.6b00180&amp;pmid=27725962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Dalle-Donne, I. ∙ Rossi, R. ∙ Colombo, G. ...</div><div><strong>Protein S-glutathionylation: a regulatory device from bacteria to humans</strong></div><div><em>Trends Biochem. Sci.</em> 2009; <strong>34</strong>:85-96</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tibs.2008.11.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-59849129012" target="_blank" aria-label="Open Scopus in new tab">Scopus (545)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19135374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tibs.2008.11.002&amp;pmid=19135374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Dong, S.H. ∙ Tang, W. ∙ Lukk, T. ...</div><div><strong>The enterococcal cytolysin synthetase has an unanticipated lipid kinase fold</strong></div><div><em>eLife.</em> 2015; <strong>4</strong>:e07607</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.07607" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84940538541" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.07607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Duan, Y. ∙ Llorente, C. ∙ Lang, S. ...</div><div><strong>Bacteriophage targeting of gut bacterium attenuates alcoholic liver disease</strong></div><div><em>Nature.</em> 2019; <strong>575</strong>:505-511</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1742-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85074974768" target="_blank" aria-label="Open Scopus in new tab">Scopus (512)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31723265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1742-x&amp;pmid=31723265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Eley, G.D. ∙ Reiter, J.L. ∙ Pandita, A. ...</div><div><strong>A chromosomal region 7p11.2 transcript map: its development and application to the study of EGFR amplicons in glioblastoma</strong></div><div><em>Neuro-oncol.</em> 2002; <strong>4</strong>:86-94</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/4.2.86" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11916499/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2F4.2.86&amp;pmid=11916499" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20-1" title="View in article">20.</a></div><div>Fischmann, T.O. ∙ Smith, C.K. ∙ Mayhood, T.W. ...</div><div><strong>Crystal structures of MEK1 binary and ternary complexes with nucleotides and inhibitors</strong></div><div><em>Biochemistry.</em> 2009; <strong>48</strong>:2661-2674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi801898e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-65249090229" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19161339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi801898e&amp;pmid=19161339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Forrest, A.R. ∙ Kawaji, H. ∙ Rehli, M. ..., FANTOM Consortium and the RIKEN PMI and CLST (DGT)</div><div><strong>A promoter-level mammalian expression atlas</strong></div><div><em>Nature.</em> 2014; <strong>507</strong>:462-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13182" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84897406127" target="_blank" aria-label="Open Scopus in new tab">Scopus (1401)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24670764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13182&amp;pmid=24670764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Friedman, M.</div><div><strong>Chemistry, biochemistry, nutrition, and microbiology of lysinoalanine, lanthionine, and histidinoalanine in food and other proteins</strong></div><div><em>J.&nbsp;Agric. Food Chem.</em> 1999; <strong>47</strong>:1295-1319</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jf981000%2B" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0032965144" target="_blank" aria-label="Open Scopus in new tab">Scopus (311)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10563973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjf981000%252B&amp;pmid=10563973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Geiger, T. ∙ Velic, A. ∙ Macek, B. ...</div><div><strong>Initial quantitative proteomic map of 28 mouse tissues using the SILAC mouse</strong></div><div><em>Mol. Cell. Proteomics.</em> 2013; <strong>12</strong>:1709-1722</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.M112.024919&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.M112.024919&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84878741907" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23436904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M112.024919&amp;pmid=23436904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Gopalbhai, K. ∙ Jansen, G. ∙ Beauregard, G. ...</div><div><strong>Negative regulation of MAPKK by phosphorylation of a conserved serine residue equivalent to Ser212 of MEK1</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2003; <strong>278</strong>:8118-8125</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M211870200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M211870200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0037424509" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12506122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M211870200&amp;pmid=12506122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">25.</a></div><div>Grek, C.L. ∙ Zhang, J. ∙ Manevich, Y. ...</div><div><strong>Causes and consequences of cysteine S-glutathionylation</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2013; <strong>288</strong>:26497-26504</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.R113.461368&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.R113.461368&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84884179149" target="_blank" aria-label="Open Scopus in new tab">Scopus (254)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23861399/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.R113.461368&amp;pmid=23861399" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">26.</a></div><div>He, C. ∙ Zeng, M. ∙ Dutta, D. ...</div><div><strong>LanCL proteins are not Involved in Lanthionine Synthesis in Mammals</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:40980</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep40980" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85010403537" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28106097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep40980&amp;pmid=28106097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Hornbeck, P.V. ∙ Zhang, B. ∙ Murray, B. ...</div><div><strong>PhosphoSitePlus, 2014: mutations, PTMs and recalibrations</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:D512-D520</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku1267" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84941120469" target="_blank" aria-label="Open Scopus in new tab">Scopus (2082)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25514926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku1267&amp;pmid=25514926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">28.</a></div><div>Huang, C. ∙ Chen, M. ∙ Pang, D. ...</div><div><strong>Developmental and activity-dependent expression of LanCL1 confers antioxidant activity required for neuronal survival</strong></div><div><em>Dev. Cell.</em> 2014; <strong>30</strong>:479-487</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.devcel.2014.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.devcel.2014.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84907304641" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25158856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2014.06.011&amp;pmid=25158856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Hubatsch, I. ∙ Ridderström, M. ∙ Mannervik, B.</div><div><strong>Human glutathione transferase A4-4: an alpha class enzyme with high catalytic efficiency in the conjugation of 4-hydroxynonenal and other genotoxic products of lipid peroxidation</strong></div><div><em>Biochem. J.</em> 1998; <strong>330</strong>:175-179</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1042/bj3300175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0032519517" target="_blank" aria-label="Open Scopus in new tab">Scopus (335)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9461507/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1042%2Fbj3300175&amp;pmid=9461507" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Hubbard, S. ∙ Thornton, J.</div><div><strong>NACCESS</strong></div><div>University College London, 1993</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.HubbardJ.ThorntonNACCESS1993University+College+London" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Huttlin, E.L. ∙ Jedrychowski, M.P. ∙ Elias, J.E. ...</div><div><strong>A tissue-specific atlas of mouse protein phosphorylation and expression</strong></div><div><em>Cell.</em> 2010; <strong>143</strong>:1174-1189</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.cell.2010.12.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.cell.2010.12.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-78650466243" target="_blank" aria-label="Open Scopus in new tab">Scopus (1348)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21183079/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2010.12.001&amp;pmid=21183079" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Kawaji, H. ∙ Kasukawa, T. ∙ Forrest, A. ...</div><div><strong>The FANTOM5 collection, a data series underpinning mammalian transcriptome atlases in diverse cell types</strong></div><div><em>Sci. Data.</em> 2017; <strong>4</strong>:170113</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/sdata.2017.113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85028458606" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28850107/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsdata.2017.113&amp;pmid=28850107" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33-1" title="View in article">33.</a></div><div>Khokhlatchev, A. ∙ Xu, S. ∙ English, J. ...</div><div><strong>Reconstitution of mitogen-activated protein kinase phosphorylation cascades in bacteria. Efficient synthesis of active protein kinases</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1997; <strong>272</strong>:11057-11062</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.272.17.11057&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0030966367" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9110999/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.272.17.11057&amp;pmid=9110999" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">34.</a></div><div>Kim, M.S. ∙ Pinto, S.M. ∙ Getnet, D. ...</div><div><strong>A draft map of the human proteome</strong></div><div><em>Nature.</em> 2014; <strong>509</strong>:575-581</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13302" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84901599553" target="_blank" aria-label="Open Scopus in new tab">Scopus (1678)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24870542/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13302&amp;pmid=24870542" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">35.</a></div><div>Lamoureaux, T.L. ∙ Lee, D.H.</div><div><strong>Chemical activation of MEK1--a redox trigger for evaluating the effects of phosphorylation</strong></div><div><em>Chem. Commun. (Camb.).</em> 2011; <strong>47</strong>:8623-8625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c1cc11745a" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79960474817" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21717004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc1cc11745a&amp;pmid=21717004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Lee, B. ∙ Richards, F.M.</div><div><strong>The interpretation of protein structures: estimation of static accessibility</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1971; <strong>55</strong>:379-400</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(71)90324-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0015222647" target="_blank" aria-label="Open Scopus in new tab">Scopus (5511)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5551392/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2871%2990324-X&amp;pmid=5551392" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">37.</a></div><div>Li, B. ∙ van der Donk, W.A.</div><div><strong>Identification of essential catalytic residues of the cyclase NisC involved in the biosynthesis of nisin</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:21169-21175</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.M701802200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-34547137156" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17513866/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M701802200&amp;pmid=17513866" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">38.</a></div><div>Li, B. ∙ Yu, J.P. ∙ Brunzelle, J.S. ...</div><div><strong>Structure and mechanism of the lantibiotic cyclase involved in nisin biosynthesis</strong></div><div><em>Science.</em> 2006; <strong>311</strong>:1464-1467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1121422" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33644854595" target="_blank" aria-label="Open Scopus in new tab">Scopus (263)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16527981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1121422&amp;pmid=16527981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39-1" title="View in article">39.</a></div><div>Li, H. ∙ Xu, H. ∙ Zhou, Y. ...</div><div><strong>The phosphothreonine lyase activity of a bacterial type III effector family</strong></div><div><em>Science.</em> 2007; <strong>315</strong>:1000-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1138960" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33847154642" target="_blank" aria-label="Open Scopus in new tab">Scopus (345)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17303758/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1138960&amp;pmid=17303758" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Li, B. ∙ Sher, D. ∙ Kelly, L. ...</div><div><strong>Catalytic promiscuity in the biosynthesis of cyclic peptide secondary metabolites in planktonic marine cyanobacteria</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2010; <strong>107</strong>:10430-10435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0913677107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-77953742146" target="_blank" aria-label="Open Scopus in new tab">Scopus (226)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20479271/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0913677107&amp;pmid=20479271" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">41.</a></div><div>Linetsky, M. ∙ LeGrand, R.D.</div><div><strong>Glutathionylation of lens proteins through the formation of thioether bond</strong></div><div><em>Mol. Cell. Biochem.</em> 2005; <strong>272</strong>:133-144</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s11010-005-6908-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-20944446301" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16010980/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs11010-005-6908-1&amp;pmid=16010980" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42-1" title="View in article">42.</a></div><div>Linetsky, M. ∙ Hill, J.M. ∙ LeGrand, R.D. ...</div><div><strong>Dehydroalanine crosslinks in human lens</strong></div><div><em>Exp. Eye Res.</em> 2004; <strong>79</strong>:499-512</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.exer.2004.06.026" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-4544350320" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15381034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.exer.2004.06.026&amp;pmid=15381034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Liu, W. ∙ Chan, A.S.H. ∙ Liu, H. ...</div><div><strong>Solid supported chemical syntheses of both components of the lantibiotic lacticin 3147</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2011; <strong>133</strong>:14216-14219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja206017p" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-80052580657" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21848315/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja206017p&amp;pmid=21848315" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>Liu, Q. ∙ Sabnis, Y. ∙ Zhao, Z. ...</div><div><strong>Developing irreversible inhibitors of the protein kinase cysteinome</strong></div><div><em>Chem. Biol.</em> 2013; <strong>20</strong>:146-159</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.chembiol.2012.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.chembiol.2012.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84874301754" target="_blank" aria-label="Open Scopus in new tab">Scopus (532)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23438744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chembiol.2012.12.006&amp;pmid=23438744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Ma, S. ∙ Caprioli, R.M. ∙ Hill, K.E. ...</div><div><strong>Loss of selenium from selenoproteins: conversion of selenocysteine to dehydroalanine in&nbsp;vitro</strong></div><div><em>J.&nbsp;Am. Soc. Mass Spectrom.</em> 2003; <strong>14</strong>:593-600</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S1044-0305(03)00141-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0037811713" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12781460/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1044-0305%2803%2900141-7&amp;pmid=12781460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">46.</a></div><div>Mayer, H. ∙ Salzer, U. ∙ Breuss, J. ...</div><div><strong>Isolation, molecular characterization, and tissue-specific expression of a novel putative G protein-coupled receptor</strong></div><div><em>Biochim. Biophys. Acta.</em> 1998; <strong>1395</strong>:301-308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0167-4781(97)00178-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0032506996" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9512664/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0167-4781%2897%2900178-4&amp;pmid=9512664" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Mayer, H. ∙ Bauer, H. ∙ Breuss, J. ...</div><div><strong>Characterization of rat LANCL1, a novel member of the lanthionine synthetase C-like protein family, highly expressed in testis and brain</strong></div><div><em>Gene.</em> 2001; <strong>269</strong>:73-80</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0378-1119(01)00463-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0035897569" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11376939/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0378-1119%2801%2900463-2&amp;pmid=11376939" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">48.</a></div><div>Mayer, H. ∙ Pongratz, M. ∙ Prohaska, R.</div><div><strong>Molecular cloning, characterization, and tissue-specific expression of human LANCL2, a novel member of the LanC-like protein family</strong></div><div><em>DNA Seq.</em> 2001; <strong>12</strong>:161-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/10425170109080770" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0035238674" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11762191/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F10425170109080770&amp;pmid=11762191" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>McFadden, P.N. ∙ Clarke, S.</div><div><strong>Conversion of isoaspartyl peptides to normal peptides: implications for the cellular repair of damaged proteins</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1987; <strong>84</strong>:2595-2599</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.84.9.2595" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0008123901" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3472227/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.84.9.2595&amp;pmid=3472227" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">50.</a></div><div>Médard, G. ∙ Pachl, F. ∙ Ruprecht, B. ...</div><div><strong>Optimized chemical proteomics assay for kinase inhibitor profiling</strong></div><div><em>J.&nbsp;Proteome Res.</em> 2015; <strong>14</strong>:1574-1586</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/pr5012608" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84924371112" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25660469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fpr5012608&amp;pmid=25660469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">51.</a></div><div>Meister, A. ∙ Anderson, M.E.</div><div><strong>Glutathione</strong></div><div><em>Annu. Rev. Biochem.</em> 1983; <strong>52</strong>:711-760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.bi.52.070183.003431" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0020972417" target="_blank" aria-label="Open Scopus in new tab">Scopus (6229)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6137189/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.bi.52.070183.003431&amp;pmid=6137189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">52.</a></div><div>Mladkova, J. ∙ Sanda, M. ∙ Matouskova, E. ...</div><div><strong>Phenotyping breast cancer cell lines EM-G3, HCC1937, MCF7 and MDA-MB-231 using 2-D electrophoresis and affinity chromatography for glutathione-binding proteins</strong></div><div><em>BMC Cancer.</em> 2010; <strong>10</strong>:449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2407-10-449" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-77955803289" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20731849/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2407-10-449&amp;pmid=20731849" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">53.</a></div><div>Morrison, P.M. ∙ Foley, P.J. ∙ Warriner, S.L. ...</div><div><strong>Chemical generation and modification of peptides containing multiple dehydroalanines</strong></div><div><em>Chem. Commun. (Camb.).</em> 2015; <strong>51</strong>:13470-13473</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/C5CC05469A" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26219458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FC5CC05469A&amp;pmid=26219458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#figs2" title="View in article">54.</a></div><div>Niesen, F.H. ∙ Berglund, H. ∙ Vedadi, M.</div><div><strong>The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</strong></div><div><em>Nat. Protoc.</em> 2007; <strong>2</strong>:2212-2221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2007.321" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-37249005205" target="_blank" aria-label="Open Scopus in new tab">Scopus (1888)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17853878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2007.321&amp;pmid=17853878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Peng, J. ∙ Gygi, S.P.</div><div><strong>Proteomics: the move to mixtures</strong></div><div><em>J.&nbsp;Mass Spectrom.</em> 2001; <strong>36</strong>:1083-1091</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jms.229" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0034757770" target="_blank" aria-label="Open Scopus in new tab">Scopus (528)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11747101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjms.229&amp;pmid=11747101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">56.</a></div><div>Repka, L.M. ∙ Chekan, J.R. ∙ Nair, S.K. ...</div><div><strong>Mechanistic understanding of lanthipeptide biosynthetic enzymes</strong></div><div><em>Chem. Rev.</em> 2017; <strong>117</strong>:5457-5520</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.chemrev.6b00591" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85017930087" target="_blank" aria-label="Open Scopus in new tab">Scopus (342)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28135077/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.chemrev.6b00591&amp;pmid=28135077" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">57.</a></div><div>Rhee, S.G. ∙ Cho, C.S.</div><div><strong>Blot-based detection of dehydroalanine-containing glutathione peroxidase with the use of biotin-conjugated cysteamine</strong></div><div><em>Methods Enzymol.</em> 2010; <strong>474</strong>:23-34</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0076-6879(10)74002-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79958041926" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20609902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0076-6879%2810%2974002-7&amp;pmid=20609902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">58.</a></div><div>Robbins, D.J. ∙ Zhen, E. ∙ Owaki, H. ...</div><div><strong>Regulation and properties of extracellular signal-regulated protein kinases 1 and 2 in&nbsp;vitro</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1993; <strong>268</strong>:5097-5106</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2FS0021-9258%2818%2953507-9&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2FS0021-9258%2818%2953507-9&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8444886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2953507-9&amp;pmid=8444886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">59.</a></div><div>Schmutz, C. ∙ Ahrné, E. ∙ Kasper, C.A. ...</div><div><strong>Systems-level overview of host protein phosphorylation during <i>Shigella flexneri</i> infection revealed by phosphoproteomics</strong></div><div><em>Mol. Cell. Proteomics.</em> 2013; <strong>12</strong>:2952-2968</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.M113.029918&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.M113.029918&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84885122882" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23828894/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M113.029918&amp;pmid=23828894" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">60.</a></div><div>Tang, W. ∙ van der Donk, W.A.</div><div><strong>Structural characterization of four prochlorosins: a novel class of lantipeptides produced by planktonic marine cyanobacteria</strong></div><div><em>Biochemistry.</em> 2012; <strong>51</strong>:4271-4279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi300255s" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84861624891" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22574919/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi300255s&amp;pmid=22574919" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">61.</a></div><div>Tang, W. ∙ van der Donk, W.A.</div><div><strong>The sequence of the enterococcal cytolysin imparts unusual lanthionine stereochemistry</strong></div><div><em>Nat. Chem. Biol.</em> 2013; <strong>9</strong>:157-159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nchembio.1162" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84874115767" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23314913/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnchembio.1162&amp;pmid=23314913" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">62.</a></div><div>Tang, W. ∙ Bobeica, S.C. ∙ Wang, L. ...</div><div><strong>CylA is a sequence-specific protease involved in toxin biosynthesis</strong></div><div><em>J.&nbsp;Ind. Microbiol. Biotechnol.</em> 2019; <strong>46</strong>:537-549</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10295-018-2110-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85057230763" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30484123/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10295-018-2110-9&amp;pmid=30484123" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>The Uniprot consortium</div><div><strong>UniProt: a worldwide hub of protein knowledge</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D506-D515</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85059797762" target="_blank" aria-label="Open Scopus in new tab">Scopus (5038)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30395287/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1049&amp;pmid=30395287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">64.</a></div><div>Thibodeaux, C.J. ∙ Ha, T. ∙ van der Donk, W.A.</div><div><strong>A price to pay for relaxed substrate specificity: a comparative kinetic analysis of the class II lanthipeptide synthetases ProcM and HalM2</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2014; <strong>136</strong>:17513-17529</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja5089452" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84919344174" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja5089452&amp;pmid=25409537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">65.</a></div><div>van Kasteren, S.I. ∙ Kramer, H.B. ∙ Jensen, H.H. ...</div><div><strong>Expanding the diversity of chemical protein modification allows post-translational mimicry</strong></div><div><em>Nature.</em> 2007; <strong>446</strong>:1105-1109</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05757" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-34247622775" target="_blank" aria-label="Open Scopus in new tab">Scopus (275)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17460675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05757&amp;pmid=17460675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">66.</a></div><div>Veuthey, A.L. ∙ Bridge, A. ∙ Gobeill, J. ...</div><div><strong>Application of text-mining for updating protein post-translational modification annotation in UniProtKB</strong></div><div><em>BMC Bioinformatics.</em> 2013; <strong>14</strong>:104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-14-104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84875129833" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23517090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-14-104&amp;pmid=23517090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">67.</a></div><div>Wagle, N. ∙ Emery, C. ∙ Berger, M.F. ...</div><div><strong>Dissecting therapeutic resistance to RAF inhibition in melanoma by tumor genomic profiling</strong></div><div><em>J.&nbsp;Clin. Oncol.</em> 2011; <strong>29</strong>:3085-3096</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1200/JCO.2010.33.2312" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-80051625929" target="_blank" aria-label="Open Scopus in new tab">Scopus (830)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21383288/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1200%2FJCO.2010.33.2312&amp;pmid=21383288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">68.</a></div><div>Wang, S.K. ∙ Weaver, J.D. ∙ Zhang, S. ...</div><div><strong>Knockout of SOD1 promotes conversion of selenocysteine to dehydroalanine in murine hepatic GPX1 protein</strong></div><div><em>Free Radic. Biol. Med.</em> 2011; <strong>51</strong>:197-204</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.freeradbiomed.2011.03.018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-79957968064" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21420488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.freeradbiomed.2011.03.018&amp;pmid=21420488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">69.</a></div><div>Wang, Z. ∙ Lyons, B. ∙ Truscott, R.J. ...</div><div><strong>Human protein aging: modification and crosslinking through dehydroalanine and dehydrobutyrine intermediates</strong></div><div><em>Aging Cell.</em> 2014; <strong>13</strong>:226-234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/acel.12164" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84895822337" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24134651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Facel.12164&amp;pmid=24134651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">70.</a></div><div>Willwacher, J. ∙ Raj, R. ∙ Mohammed, S. ...</div><div><strong>Selective metal-site-guided arylation of proteins</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2016; <strong>138</strong>:8678-8681</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jacs.6b04043" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84979263737" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27336299/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjacs.6b04043&amp;pmid=27336299" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">71.</a></div><div>Wu, B. ∙ Dong, D.</div><div><strong>Human cytosolic glutathione transferases: structure, function, and drug discovery</strong></div><div><em>Trends Pharmacol. Sci.</em> 2012; <strong>33</strong>:656-668</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tips.2012.09.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.tips.2012.09.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84869236062" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23121834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tips.2012.09.007&amp;pmid=23121834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">72.</a></div><div>Xu, B.e. ∙ Wilsbacher, J.L. ∙ Collisson, T. ...</div><div><strong>The N-terminal ERK-binding site of MEK1 is required for efficient feedback phosphorylation by&nbsp;ERK2 in&nbsp;vitro and ERK activation in&nbsp;vivo</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1999; <strong>274</strong>:34029-34035</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.274.48.34029&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fjbc.274.48.34029&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0033607522" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10567369/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.274.48.34029&amp;pmid=10567369" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">73.</a></div><div>Yang, X. ∙ van der Donk, W.A.</div><div><strong>Michael-type cyclizations in lantibiotic biosynthesis are reversible</strong></div><div><em>ACS Chem. Biol.</em> 2015; <strong>10</strong>:1234-1238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acschembio.5b00007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84929583013" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25723375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facschembio.5b00007&amp;pmid=25723375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74-1" title="View in article">74.</a></div><div>Yang, J. ∙ Carroll, K.S. ∙ Liebler, D.C.</div><div><strong>The expanding landscape of the thiol redox proteome</strong></div><div><em>Mol. Cell. Proteomics.</em> 2016; <strong>15</strong>:1-11</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O115.056051&amp;cf=fulltext&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1074%2Fmcp.O115.056051&amp;cf=pdf&amp;site=mcpro-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84955506189" target="_blank" aria-label="Open Scopus in new tab">Scopus (163)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26518762/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.O115.056051&amp;pmid=26518762" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">75.</a></div><div>Yang, X. ∙ Lennard, K.R. ∙ He, C. ...</div><div><strong>A lanthipeptide library used to identify a protein-protein interaction inhibitor</strong></div><div><em>Nat. Chem. Biol.</em> 2018; <strong>14</strong>:375-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41589-018-0008-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-85042882496" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29507389/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41589-018-0008-5&amp;pmid=29507389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">76.</a></div><div>Zeng, M. ∙ van der Donk, W.A. ∙ Chen, J.</div><div><strong>Lanthionine synthetase C-like protein 2 (LanCL2) is a novel regulator of Akt</strong></div><div><em>Mol. Biol. Cell.</em> 2014; <strong>25</strong>:3954-3961</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E14-01-0004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84913529605" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25273559/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E14-01-0004&amp;pmid=25273559" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">77.</a></div><div>Zhang, W. ∙ Wang, L. ∙ Liu, Y. ...</div><div><strong>Structure of human lanthionine synthetase C-like protein 1 and its interaction with Eps8 and glutathione</strong></div><div><em>Genes Dev.</em> 2009; <strong>23</strong>:1387-1392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1789209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-67149147296" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19528316/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1789209&amp;pmid=19528316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78-1" title="View in article">78.</a></div><div>Zhao, K. ∙ Ju, Y. ∙ Li, S. ...</div><div><strong>S-sulfhydration of MEK1 leads to PARP-1 activation and DNA damage repair</strong></div><div><em>EMBO Rep.</em> 2014; <strong>15</strong>:792-800</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/embr.201338271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-84899806842" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24778456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fembr.201338271&amp;pmid=24778456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">79.</a></div><div>Zheng, C.F. ∙ Guan, K.L.</div><div><strong>Activation of MEK family kinases requires&nbsp;phosphorylation of two conserved Ser/Thr residues</strong></div><div><em>EMBO J.</em> 1994; <strong>13</strong>:1123-1131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/j.1460-2075.1994.tb06361.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-0028329630" target="_blank" aria-label="Open Scopus in new tab">Scopus (329)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8131746/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fj.1460-2075.1994.tb06361.x&amp;pmid=8131746" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80-1" title="View in article">80.</a></div><div>Zhu, Y. ∙ Li, H. ∙ Long, C. ...</div><div><strong>Structural insights into the enzymatic mechanism of the pathogenic MAPK phosphothreonine lyase</strong></div><div><em>Mol. Cell.</em> 2007; <strong>28</strong>:899-913</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.molcel.2007.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=10.1016%2Fj.molcel.2007.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-36749011295" target="_blank" aria-label="Open Scopus in new tab">Scopus (109)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18060821/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2007.11.011&amp;pmid=18060821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">81.</a></div><div>Zurawski, D.V. ∙ Mitsuhata, C. ∙ Mumy, K.L. ...</div><div><strong>OspF and OspC1 are Shigella flexneri type III secretion system effectors that are required for postinvasion aspects of virulence</strong></div><div><em>Infect. Immun.</em> 2006; <strong>74</strong>:5964-5976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/IAI.00594-06" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-33749237700" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16988276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FIAI.00594-06&amp;pmid=16988276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">82.</a></div><div>Zurawski, D.V. ∙ Mumy, K.L. ∙ Faherty, C.S. ...</div><div><strong>Shigella flexneri type III secretion system effectors OspB and OspF target the nucleus to downregulate the host inflammatory response via interactions with retinoblastoma protein</strong></div><div><em>Mol. Microbiol.</em> 2009; <strong>71</strong>:350-368</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2008.06524.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.001&amp;key=2-s2.0-58149313835" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19017275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2008.06524.x&amp;pmid=19017275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (18)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/6be4ebdf-7882-4bda-9c1d-c1f188986d78/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/2f423bcc-01b8-45c1-86e9-0e11f063c223/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/e0271357-c929-4b58-91f9-7795921c5009/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/924e0fa0-dd39-4441-8352-3c8a71e7f237/main.assets/gr3.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/e3427577-2316-4f56-8334-db5e4d563c95/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/3efc3874-bac3-4c0b-ac56-0f725fc187b3/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/21a2ffbb-bcd0-4a0c-bc28-c0dc1c019f0a/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/756aa32a-c9fb-4084-b079-0b3a3e13c6b0/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/424a9c7f-1e31-4571-80fd-251f84b70ae0/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/9e392dcf-38a7-44ec-935a-b9d714b3b3f3/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/34c36566-0275-45e6-995f-5431f549fd32/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/57992bb5-7bea-45dc-a878-a21d6850e2e0/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/499c1e6f-3a33-4d47-9a25-4b7fd882d70e/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#undfig2" aria-label="Go to "><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/acc57732-f752-4653-89c6-a045ed4705bd/main.assets/fx2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#undfig3" aria-label="Go to "><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/8f8d90ba-8658-4790-acfb-6f8058fa3af7/main.assets/fx3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#undfig4" aria-label="Go to "><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/f193a92b-e6d4-4175-8a12-c4e02bf23922/main.assets/fx4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#undfig5" aria-label="Go to "><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/c1e3339d-df60-47be-9570-e77f75555d7f/main.assets/fx5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#undfig6" aria-label="Go to "><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/asset/424a46b3-29f6-4b26-8095-8169ef2732fe/main.assets/fx6.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00110%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Barticle%3Aarticle%3Apii%5C%3AS0092867421004360%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (5)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/attachment/6f78b334-7b3e-4f75-a417-357cb3d027d8/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (569.78 KB)</span></a><div class="core-description">Table S1. Unbiased protein pull-down and proteomics analysis, related to Figure&nbsp;1E</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/attachment/21b17e29-57d1-4163-9a21-4f57665537cb/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (22.57 KB)</span></a><div class="core-description">Table S2. Chemical generation of Dha-containing MEK1 variants and their kinase activity, related to Figures 3, 4, S2, S3, S4, S5, and S6</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/attachment/a5fe6e10-bcdd-4bd6-bfa4-1713bbe628e4/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (66.17 KB)</span></a><div class="core-description">Table S3. Analysis of the proteomic literature for the presence of Dha and Dhb proteins and their conjugates with GSH, β-mercaptoethanol, Cys, and dithiothreitol, related to Figures 5 and S7</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/attachment/8ca23b38-ab97-4112-a357-7115e90bc7b4/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (15.93 KB)</span></a><div class="core-description">Table S4. Oligonucleotide sequences used in this study, related to the STAR Methods</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.04.001/attachment/fb8a904e-8e09-43d3-bc4c-bb092d41b911/mmc5.pdf" download="mmc5.pdf"><span id="mmc5-link">PDF (138.45 KB)</span></a><div class="core-description">Methods S1. NMR spectra of bisalkylating agents used to introduce dehydro amino acids into proteins, related to Figures 3 and 4</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421004360" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.04.001","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">LanCLs add glutathione to dehydroamino acids generated at phosphorylated sites in the proteome</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.04.001/asset/6be4ebdf-7882-4bda-9c1d-c1f188986d78/main.assets/gr1.jpg" height="866" width="750" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/2771d953-4355-43cd-9341-0ad94fbb26d5/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.04.001/asset/2f423bcc-01b8-45c1-86e9-0e11f063c223/main.assets/figs1.jpg" height="898" width="637" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/783b9af0-0b39-4304-b76c-3f2bec54405d/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.04.001/asset/e0271357-c929-4b58-91f9-7795921c5009/main.assets/gr2.jpg" height="840" width="750" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/373da9f4-9ddc-454f-91dc-3c9ef8738584/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.04.001/asset/924e0fa0-dd39-4441-8352-3c8a71e7f237/main.assets/gr3.jpg" height="770" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/87e1c8d6-fb5d-4a3a-ad1a-dca8b837cac6/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.04.001/asset/e3427577-2316-4f56-8334-db5e4d563c95/main.assets/gr4.jpg" height="750" width="497" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/69c4dabd-06c5-4027-b9aa-798498854716/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.04.001/asset/3efc3874-bac3-4c0b-ac56-0f725fc187b3/main.assets/figs2.jpg" height="622" width="750" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/926b8367-92b7-42ad-95b5-a801a6ed17aa/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.04.001/asset/21a2ffbb-bcd0-4a0c-bc28-c0dc1c019f0a/main.assets/figs3.jpg" height="486" width="750" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/9faf8f42-ee32-46a8-b1a6-aa5ad10cfdd8/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.04.001/asset/756aa32a-c9fb-4084-b079-0b3a3e13c6b0/main.assets/figs4.jpg" height="618" width="750" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/5b4b43f5-1dd3-48aa-9359-74692910d851/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.04.001/asset/424a9c7f-1e31-4571-80fd-251f84b70ae0/main.assets/figs5.jpg" height="939" width="479" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/93fa428f-64e3-4af1-968c-3310734dc4f1/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.04.001/asset/9e392dcf-38a7-44ec-935a-b9d714b3b3f3/main.assets/figs6.jpg" height="939" width="413" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/d647826b-5433-414c-9279-7efa6452a19c/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.04.001/asset/34c36566-0275-45e6-995f-5431f549fd32/main.assets/gr5.jpg" height="750" width="750" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/38d0103d-2825-49e0-8e5e-fc032d5d54d8/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.04.001/asset/57992bb5-7bea-45dc-a878-a21d6850e2e0/main.assets/figs7.jpg" height="860" width="750" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/16fbeded-9ade-4158-8766-94ed33b4ae6d/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.04.001/asset/499c1e6f-3a33-4d47-9a25-4b7fd882d70e/main.assets/gr6.jpg" height="939" width="650" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/cdad30fd-8167-43c7-8fc4-b67941a8088a/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="fx2" alt="Figure undfig2" src="/cms/10.1016/j.cell.2021.04.001/asset/acc57732-f752-4653-89c6-a045ed4705bd/main.assets/fx2.jpg" height="53" width="156" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/00ceb0d9-9a4f-4ca6-9ff9-8f5141b1f40e/main.assets/fx2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="fx3" alt="Figure undfig3" src="/cms/10.1016/j.cell.2021.04.001/asset/8f8d90ba-8658-4790-acfb-6f8058fa3af7/main.assets/fx3.jpg" height="52" width="146" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/1b4c4521-2ea3-453f-aa41-2417ab752dbd/main.assets/fx3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="fx4" alt="Figure undfig4" src="/cms/10.1016/j.cell.2021.04.001/asset/f193a92b-e6d4-4175-8a12-c4e02bf23922/main.assets/fx4.jpg" height="61" width="164" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/32af1d64-2fd1-466b-9c6e-401c233831cc/main.assets/fx4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="fx5" alt="Figure undfig5" src="/cms/10.1016/j.cell.2021.04.001/asset/c1e3339d-df60-47be-9570-e77f75555d7f/main.assets/fx5.jpg" height="61" width="260" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/8574f97c-9ea8-468c-8376-345f5c2acee5/main.assets/fx5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="fx6" alt="Figure undfig6" src="/cms/10.1016/j.cell.2021.04.001/asset/424a46b3-29f6-4b26-8095-8169ef2732fe/main.assets/fx6.jpg" height="61" width="243" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/7761b99d-e7ec-4bf7-9587-ce46c25e2aac/main.assets/fx6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.04.001/asset/6be4ebdf-7882-4bda-9c1d-c1f188986d78/main.assets/gr1.jpg" height="866" width="750" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/2771d953-4355-43cd-9341-0ad94fbb26d5/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.04.001/asset/2f423bcc-01b8-45c1-86e9-0e11f063c223/main.assets/figs1.jpg" height="898" width="637" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/783b9af0-0b39-4304-b76c-3f2bec54405d/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.04.001/asset/e0271357-c929-4b58-91f9-7795921c5009/main.assets/gr2.jpg" height="840" width="750" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/373da9f4-9ddc-454f-91dc-3c9ef8738584/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.04.001/asset/924e0fa0-dd39-4441-8352-3c8a71e7f237/main.assets/gr3.jpg" height="770" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/87e1c8d6-fb5d-4a3a-ad1a-dca8b837cac6/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.04.001/asset/e3427577-2316-4f56-8334-db5e4d563c95/main.assets/gr4.jpg" height="750" width="497" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/69c4dabd-06c5-4027-b9aa-798498854716/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.04.001/asset/3efc3874-bac3-4c0b-ac56-0f725fc187b3/main.assets/figs2.jpg" height="622" width="750" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/926b8367-92b7-42ad-95b5-a801a6ed17aa/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.04.001/asset/21a2ffbb-bcd0-4a0c-bc28-c0dc1c019f0a/main.assets/figs3.jpg" height="486" width="750" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/9faf8f42-ee32-46a8-b1a6-aa5ad10cfdd8/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.04.001/asset/756aa32a-c9fb-4084-b079-0b3a3e13c6b0/main.assets/figs4.jpg" height="618" width="750" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/5b4b43f5-1dd3-48aa-9359-74692910d851/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.04.001/asset/424a9c7f-1e31-4571-80fd-251f84b70ae0/main.assets/figs5.jpg" height="939" width="479" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/93fa428f-64e3-4af1-968c-3310734dc4f1/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.04.001/asset/9e392dcf-38a7-44ec-935a-b9d714b3b3f3/main.assets/figs6.jpg" height="939" width="413" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/d647826b-5433-414c-9279-7efa6452a19c/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.04.001/asset/34c36566-0275-45e6-995f-5431f549fd32/main.assets/gr5.jpg" height="750" width="750" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/38d0103d-2825-49e0-8e5e-fc032d5d54d8/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.04.001/asset/57992bb5-7bea-45dc-a878-a21d6850e2e0/main.assets/figs7.jpg" height="860" width="750" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/16fbeded-9ade-4158-8766-94ed33b4ae6d/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.04.001/asset/499c1e6f-3a33-4d47-9a25-4b7fd882d70e/main.assets/gr6.jpg" height="939" width="650" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/cdad30fd-8167-43c7-8fc4-b67941a8088a/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure</div><div class="thumbnail"><div class="thumbnail__image"><img id="fx2" alt="Figure undfig2" src="/cms/10.1016/j.cell.2021.04.001/asset/acc57732-f752-4653-89c6-a045ed4705bd/main.assets/fx2.jpg" height="53" width="156" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/00ceb0d9-9a4f-4ca6-9ff9-8f5141b1f40e/main.assets/fx2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure</div><div class="thumbnail"><div class="thumbnail__image"><img id="fx3" alt="Figure undfig3" src="/cms/10.1016/j.cell.2021.04.001/asset/8f8d90ba-8658-4790-acfb-6f8058fa3af7/main.assets/fx3.jpg" height="52" width="146" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/1b4c4521-2ea3-453f-aa41-2417ab752dbd/main.assets/fx3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure</div><div class="thumbnail"><div class="thumbnail__image"><img id="fx4" alt="Figure undfig4" src="/cms/10.1016/j.cell.2021.04.001/asset/f193a92b-e6d4-4175-8a12-c4e02bf23922/main.assets/fx4.jpg" height="61" width="164" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/32af1d64-2fd1-466b-9c6e-401c233831cc/main.assets/fx4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure</div><div class="thumbnail"><div class="thumbnail__image"><img id="fx5" alt="Figure undfig5" src="/cms/10.1016/j.cell.2021.04.001/asset/c1e3339d-df60-47be-9570-e77f75555d7f/main.assets/fx5.jpg" height="61" width="260" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/8574f97c-9ea8-468c-8376-345f5c2acee5/main.assets/fx5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure</div><div class="thumbnail"><div class="thumbnail__image"><img id="fx6" alt="Figure undfig6" src="/cms/10.1016/j.cell.2021.04.001/asset/424a46b3-29f6-4b26-8095-8169ef2732fe/main.assets/fx6.jpg" height="61" width="243" data-viewer-src="/cms/10.1016/j.cell.2021.04.001/asset/7761b99d-e7ec-4bf7-9587-ce46c25e2aac/main.assets/fx6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00436-0';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00436-0';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749589987275","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"184","issueNumber":"10","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421004360","coverDate":"202105","type":"XOCS-JOURNAL:SCOPE-FULL","title":"LanCLs add glutathione to dehydroamino acids generated at phosphorylated sites in the proteome","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dbdee5fb5352eb',t:'MTc0OTU4OTk4Ni4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dbdee5fb5352eb&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.04.001&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>