<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="e9a990a2-8134-4ea2-8fd8-512320aafe2e" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S009286742030502X;issue:issue:pii\:S0092867419X00136;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(20)30502-X">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/68f46f42-e77a-4f2d-a05b-1105c9b64d38/fx1.jpg">
    


        <meta property="og:description" content="Genome-wide ancient DNA data from the Lake Baikal and its surroundings, comprising
a time transect of 10,000 years from the Upper Paleolithic to the Early Bronze Age,
reveals deeply divergent ancestry that links Upper Paleolithic Siberians and the First
Peoples of the Americas and delineates the complex transition between Early Neolithic
and Early Bronze Age populations in Siberia.">
    











        <meta name="twitter:title" content="Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/68f46f42-e77a-4f2d-a05b-1105c9b64d38/fx1.jpg">
    


        <meta name="twitter:description" content="Genome-wide ancient DNA data from the Lake Baikal and its surroundings, comprising
a time transect of 10,000 years from the Upper Paleolithic to the Early Bronze Age,
reveals deeply divergent ancestry that links Upper Paleolithic Siberians and the First
Peoples of the Americas and delineates the complex transition between Early Neolithic
and Early Bronze Age populations in Siberia.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(20)30502-X">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia"> <meta name="citation_date" content="2020/06/11"> <meta name="citation_online_date" content="2020/05/20"> <meta name="citation_volume" content="181"> <meta name="citation_issue" content="6"> <meta name="citation_firstpage" content="1232"> <meta name="citation_lastpage" content="1245.e20"> <meta name="citation_pmid" content="32437661"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S009286742030502X"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S009286742030502X"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S009286742030502X"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2020.04.037"> <meta name="citation_pmid" content="32437661"> <meta name="citation_author" content="He Yu"><meta name="citation_author_institution" content="Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author" content="Maria A. Spyrou"><meta name="citation_author_institution" content="Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author" content="Marina Karapetian"><meta name="citation_author_institution" content="Research Institute and Museum of Anthropology, Moscow State University, Moscow 125009, Russia"><meta name="citation_author" content="Svetlana Shnaider"><meta name="citation_author_institution" content="Institute of Archaeology and Ethnography of the Siberian Branch of the Russian Academy of Sciences, Novosibirsk 630090, Russia"><meta name="citation_author" content="Rita Radzevičiūtė"><meta name="citation_author_institution" content="Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author" content="Kathrin Nägele"><meta name="citation_author_institution" content="Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author" content="Gunnar U. Neumann"><meta name="citation_author_institution" content="Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author" content="Sandra Penske"><meta name="citation_author_institution" content="Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author" content="Jana Zech"><meta name="citation_author_institution" content="Department of Archaeology, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author" content="Mary Lucas"><meta name="citation_author_institution" content="Department of Archaeology, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author" content="Petrus LeRoux"><meta name="citation_author_institution" content="Department of Geological Sciences, University of Cape Town, Rondebosch 7701, South Africa"><meta name="citation_author" content="Patrick Roberts"><meta name="citation_author_institution" content="Department of Archaeology, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author" content="Galina Pavlenok"><meta name="citation_author_institution" content="Institute of Archaeology and Ethnography of the Siberian Branch of the Russian Academy of Sciences, Novosibirsk 630090, Russia"><meta name="citation_author" content="Alexandra Buzhilova"><meta name="citation_author_institution" content="Research Institute and Museum of Anthropology, Moscow State University, Moscow 125009, Russia"><meta name="citation_author" content="Cosimo Posth"><meta name="citation_author_email" content="posth@shh.mpg.de"><meta name="citation_author_institution" content="Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author_institution" content="Institute for Archaeological Sciences, Archaeo- and Palaeogenetics, University of Tübingen, Tübingen 72070, Germany"><meta name="citation_author" content="Choongwon Jeong"><meta name="citation_author_email" content="jeong@shh.mpg.de"><meta name="citation_author_institution" content="Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_author_institution" content="School of Biological Sciences, Seoul National University, Seoul 08826, Republic of Korea"><meta name="citation_author" content="Johannes Krause"><meta name="citation_author_email" content="krause@shh.mpg.de"><meta name="citation_author_orcid" content="0000-0001-9144-3920"><meta name="citation_author_institution" content="Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany"><meta name="citation_keywords" content="ancient genomics; human history; Siberia; Native Americans; mobility; Yersinia pestis; Upper Paleolithic; Neolithic; Bronze Age"><meta name="citation_reference" content="citation_title=K paleoantropologii epohi neolita verhov’ev Leny; citation_author=V.P. Alekseev; citation_author=N.N. Mamonova; citation_journal_title=Sov. Etnogr.; citation_volume=5; citation_firstpage=49; citation_lastpage=63; citation_publication_date=1979; "><meta name="citation_reference" content="citation_title=Fast model-based estimation of ancestry in unrelated individuals; citation_author=D.H. Alexander; citation_author=J. Novembre; citation_author=K. Lange; citation_journal_title=Genome Res.; citation_volume=19; citation_firstpage=1655; citation_lastpage=1664; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Strontium Isotopes from the Earth to the Archaeological Skeleton: A Review; citation_author=R. Alexander Bentley; citation_journal_title=J.&nbsp;Archaeol. Method Theory; citation_volume=13; citation_firstpage=135; citation_lastpage=187; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Population genomics of Bronze Age Eurasia; citation_author=M.E. Allentoft; citation_author=M. Sikora; citation_author=K.-G. Sjögren; citation_author=S. Rasmussen; citation_author=M. Rasmussen; citation_author=J. Stenderup; citation_author=P.B. Damgaard; citation_author=H. Schroeder; citation_author=T. Ahlström; citation_author=L. Vinner; citation_author=et al.; citation_journal_title=Nature; citation_volume=522; citation_firstpage=167; citation_lastpage=172; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Experimental Evidence for the Relationship of the Carbon Isotope Ratios of Whole Diet and Dietary Protein to Those of Bone Collagen and Carbonate; citation_inbook_title=Prehistoric Human Bone: Archaeology at the Molecular Level; citation_author=S.H. Ambrose; citation_author=L. Norr; citation_firstpage=1; citation_lastpage=37; citation_publication_date=1993; citation_publisher=Springer Berlin Heidelberg; "><meta name="citation_reference" content="citation_title=The Stone Age Plague and Its Persistence in Eurasia; citation_author=A. Andrades Valtueña; citation_author=A. Mittnik; citation_author=F.M. Key; citation_author=W. Haak; citation_author=R. Allmäe; citation_author=A. Belinskij; citation_author=M. Daubaras; citation_author=M. Feldman; citation_author=R. Jankauskas; citation_author=I. Janković; citation_author=et al.; citation_journal_title=Curr. Biol.; citation_volume=27; citation_firstpage=3683; citation_lastpage=3691; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A global reference for human genetic variation; citation_author=A. Auton; citation_author=L.D. Brooks; citation_author=R.M. Durbin; citation_author=E.P. Garrison; citation_author=H.M. Kang; citation_author=J.O. Korbel; citation_author=J.L. Marchini; citation_author=S. McCarthy; citation_author=G.A. McVean; citation_author=G.R. Abecasis; citation_author=1000 Genomes Project Consortium; citation_journal_title=Nature; citation_volume=526; citation_firstpage=68; citation_lastpage=74; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Pogrebal’nye kompleksy epohi pozdnego mezolita – neolita Bajkal’skoj Sibiri: tradicii pogrebenij, absolyutnyj vozrast. Izv. Lab. Drevnih Tekhnologij; citation_author=V.I. Bazalijskij; citation_journal_title=Irkutsk Izd-vo IrGTU.; citation_volume=9; citation_firstpage=43; citation_lastpage=101; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Human mobility at the early neolithic settlement of vaihingen, germany: evidence from strontium isotope analysis∗; citation_author=R.A. Bentley; citation_author=R. Krause; citation_author=T.D. Price; citation_author=B. Kaufmann; citation_journal_title=Archaeometry; citation_volume=45; citation_firstpage=471; citation_lastpage=486; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Determining the ‘local’ 87Sr/86Sr range for archaeological skeletons: a case study from Neolithic Europe; citation_author=R.A. Bentley; citation_author=T.D. Price; citation_author=E. Stephan; citation_journal_title=J.&nbsp;Archaeol. Sci.; citation_volume=31; citation_firstpage=365; citation_lastpage=375; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A draft genome of Yersinia pestis from victims of the Black Death; citation_author=K.I. Bos; citation_author=V.J. Schuenemann; citation_author=G.B. Golding; citation_author=H.A. Burbano; citation_author=N. Waglechner; citation_author=B.K. Coombes; citation_author=J.B. McPhee; citation_author=S.N. DeWitte; citation_author=M. Meyer; citation_author=S. Schmedes; citation_author=et al.; citation_journal_title=Nature; citation_volume=478; citation_firstpage=506; citation_lastpage=510; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Eighteenth century Yersinia pestis genomes reveal the long-term persistence of an historical plague focus; citation_author=K.I. Bos; citation_author=A. Herbig; citation_author=J. Sahl; citation_author=N. Waglechner; citation_author=M. Fourment; citation_author=S.A. Forrest; citation_author=J. Klunk; citation_author=V.J. Schuenemann; citation_author=D. Poinar; citation_author=M. Kuch; citation_author=et al.; citation_journal_title=eLife; citation_volume=5; citation_firstpage=e12994; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Removal of deaminated cytosines and detection of in&nbsp;vivo methylation in ancient DNA; citation_author=A.W. Briggs; citation_author=U. Stenzel; citation_author=M. Meyer; citation_author=J. Krause; citation_author=M. Kircher; citation_author=S. Pääbo; citation_journal_title=Nucleic Acids Res.; citation_volume=38; citation_firstpage=e87; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Comments on the Use of Ezee-Filters™ and Ultrafilters at Orau; citation_author=F. Brock; citation_author=T. Higham; citation_author=C.B. Ramsey; citation_journal_title=Radiocarbon; citation_volume=55; citation_firstpage=211; citation_lastpage=212; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Improving the accuracy and efficiency of identity-by-descent detection in population data; citation_author=B.L. Browning; citation_author=S.R. Browning; citation_journal_title=Genetics; citation_volume=194; citation_firstpage=459; citation_lastpage=471; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Strontium isotopes as tracers of ecosystem processes: theory and methods; citation_author=R.C. Capo; citation_author=B.W. Stewart; citation_author=O.A. Chadwick; citation_journal_title=Geoderma; citation_volume=82; citation_firstpage=197; citation_lastpage=225; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Historical variations in mutation rate in an epidemic pathogen, Yersinia pestis; citation_author=Y. Cui; citation_author=C. Yu; citation_author=Y. Yan; citation_author=D. Li; citation_author=Y. Li; citation_author=T. Jombart; citation_author=L.A. Weinert; citation_author=Z. Wang; citation_author=Z. Guo; citation_author=L. Xu; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=577; citation_lastpage=582; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments; citation_author=J. Dabney; citation_author=M. Knapp; citation_author=I. Glocke; citation_author=M.-T. Gansauge; citation_author=A. Weihmann; citation_author=B. Nickel; citation_author=C. Valdiosera; citation_author=N. García; citation_author=S. Pääbo; citation_author=J.-L. Arsuaga; citation_author=M. Meyer; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=15758; citation_lastpage=15763; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The first horse herders and the impact of early Bronze Age steppe expansions into Asia; citation_author=P. de B. Damgaard; citation_author=R. Martiniano; citation_author=J. Kamm; citation_author=J.V. Moreno-Mayar; citation_author=G. Kroonen; citation_author=M. Peyrot; citation_author=G. Barjamovic; citation_author=S. Rasmussen; citation_author=C. Zacho; citation_author=N. Baimukhanov; citation_author=et al.; citation_journal_title=Science; citation_volume=360; citation_firstpage=eaar7711; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=137 ancient human genomes from across the Eurasian steppes; citation_author=P.B. Damgaard; citation_author=N. Marchi; citation_author=S. Rasmussen; citation_author=M. Peyrot; citation_author=G. Renaud; citation_author=T. Korneliussen; citation_author=J.V. Moreno-Mayar; citation_author=M.W. Pedersen; citation_author=A. Goldberg; citation_author=E. Usmanova; citation_author=et al.; citation_journal_title=Nature; citation_volume=557; citation_firstpage=369; citation_lastpage=374; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Antropologicheskij sostav drevnego naseleniya Pribajkal’ya v epohu pozdnego neolita; citation_author=G.F. Debets; citation_journal_title=Rus. Antropol. Zhurnal; citation_volume=t. XIX; citation_firstpage=1; citation_lastpage=2; citation_publication_date=1930; "><meta name="citation_reference" content="citation_title=Paleoantropologiya SSSR / TIE, novaya seriya; citation_author=G.F. Debets; citation_publication_date=1948; citation_publisher=M.-L. T. IV.; "><meta name="citation_reference" content="citation_title=Improved whole-chromosome phasing for disease and population genetic studies; citation_author=O. Delaneau; citation_author=J.-F. Zagury; citation_author=J. Marchini; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=5; citation_lastpage=6; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Yersinia pestis and plague: an updated view on evolution, virulence determinants, immune subversion, vaccination, and diagnostics; citation_author=C.E. Demeure; citation_author=O. Dussurget; citation_author=G. Mas Fiol; citation_author=A.-S. Le Guern; citation_author=C. Savin; citation_author=J. Pizarro-Cerdá; citation_journal_title=Genes Immun.; citation_volume=20; citation_firstpage=357; citation_lastpage=370; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Influence of diet on the distribution of nitrogen isotopes in animals; citation_author=M.J. Deniro; citation_author=S. Epstein; citation_journal_title=Geochim. Cosmochim. Acta; citation_volume=45; citation_firstpage=341; citation_lastpage=351; citation_publication_date=1981; "><meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data; citation_author=M.A. DePristo; citation_author=E. Banks; citation_author=R. Poplin; citation_author=K.V. Garimella; citation_author=J.R. Maguire; citation_author=C. Hartl; citation_author=A.A. Philippakis; citation_author=G. del Angel; citation_author=M.A. Rivas; citation_author=M. Hanna; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=43; citation_firstpage=491; citation_lastpage=498; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=BEAST: Bayesian evolutionary analysis by sampling trees; citation_author=A.J. Drummond; citation_author=A. Rambaut; citation_journal_title=BMC Evol. Biol.; citation_volume=7; citation_firstpage=214; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Yersinia pestis strains of ancient phylogenetic branch 0.ANT are widely spread in the high-mountain plague foci of Kyrgyzstan; citation_author=G.A. Eroshenko; citation_author=N.Y. Nosov; citation_author=Y.M. Krasnov; citation_author=Y.G. Oglodin; citation_author=L.M. Kukleva; citation_author=N.P. Guseva; citation_author=A.A. Kuznetsov; citation_author=S.T. Abdikarimov; citation_author=A.K. Dzhaparova; citation_author=V.V. Kutyrev; citation_journal_title=PLoS ONE; citation_volume=12; citation_firstpage=e0187230; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A High-Coverage Yersinia pestis Genome from a Sixth-Century Justinianic Plague Victim; citation_author=M. Feldman; citation_author=M. Harbeck; citation_author=M. Keller; citation_author=M.A. Spyrou; citation_author=A. Rott; citation_author=B. Trautmann; citation_author=H.C. Scholz; citation_author=B. Päffgen; citation_author=J. Peters; citation_author=M. McCormick; citation_author=et al.; citation_journal_title=Mol. Biol. Evol.; citation_volume=33; citation_firstpage=2911; citation_lastpage=2923; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America; citation_author=P. Flegontov; citation_author=N.E. Altınışık; citation_author=P. Changmai; citation_author=N. Rohland; citation_author=S. Mallick; citation_author=N. Adamski; citation_author=D.A. Bolnick; citation_author=N. Broomandkhoshbacht; citation_author=F. Candilio; citation_author=B.J. Culleton; citation_author=et al.; citation_journal_title=Nature; citation_volume=570; citation_firstpage=236; citation_lastpage=240; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=DNA analysis of an early modern human from Tianyuan Cave, China; citation_author=Q. Fu; citation_author=M. Meyer; citation_author=X. Gao; citation_author=U. Stenzel; citation_author=H.A. Burbano; citation_author=J. Kelso; citation_author=S. Pääbo; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=2223; citation_lastpage=2227; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=An early modern human from Romania with a recent Neanderthal ancestor; citation_author=Q. Fu; citation_author=M. Hajdinjak; citation_author=O.T. Moldovan; citation_author=S. Constantin; citation_author=S. Mallick; citation_author=P. Skoglund; citation_author=N. Patterson; citation_author=N. Rohland; citation_author=I. Lazaridis; citation_author=B. Nickel; citation_author=et al.; citation_journal_title=Nature; citation_volume=524; citation_firstpage=216; citation_lastpage=219; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The genetic history of Ice Age Europe; citation_author=Q. Fu; citation_author=C. Posth; citation_author=M. Hajdinjak; citation_author=M. Petr; citation_author=S. Mallick; citation_author=D. Fernandes; citation_author=A. Furtwängler; citation_author=W. Haak; citation_author=M. Meyer; citation_author=A. Mittnik; citation_author=et al.; citation_journal_title=Nature; citation_volume=534; citation_firstpage=200; citation_lastpage=205; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Stable carbon and nitrogen isotope variations in tooth dentine serial sections from Wharram Percy; citation_author=B.T. Fuller; citation_author=M.P. Richards; citation_author=S.A. Mays; citation_journal_title=J.&nbsp;Archaeol. Sci.; citation_volume=30; citation_firstpage=1673; citation_lastpage=1684; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Single-stranded DNA library preparation from highly degraded DNA using T4 DNA ligase; citation_author=M.-T. Gansauge; citation_author=T. Gerber; citation_author=I. Glocke; citation_author=P. Korlević; citation_author=L. Lippik; citation_author=S. Nagel; citation_author=L.M. Riehl; citation_author=A. Schmidt; citation_author=M. Meyer; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Pestoides F, an Atypical Yersinia pestis Strain from the Former Soviet Union; citation_author=E. Garcia; citation_author=P. Chain; citation_author=P. Worsham; citation_author=S.W. Bearden; citation_author=S. Malfatti; citation_author=D. Lang; citation_author=F. Larimer; citation_author=L. Lindler; citation_journal_title=Adv. Exp. Med. Biol.; citation_volume=603; citation_firstpage=17; citation_lastpage=22; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The use of strontium-87/strontium-86 ratios to measure atmospheric transport into forested watersheds; citation_author=W.C. Graustein; citation_author=R.L. Armstrong; citation_journal_title=Science; citation_volume=219; citation_firstpage=289; citation_lastpage=292; citation_publication_date=1983; "><meta name="citation_reference" content="citation_title=Massive migration from the steppe was a source for Indo-European languages in Europe; citation_author=W. Haak; citation_author=I. Lazaridis; citation_author=N. Patterson; citation_author=N. Rohland; citation_author=S. Mallick; citation_author=B. Llamas; citation_author=G. Brandt; citation_author=S. Nordenfelt; citation_author=E. Harney; citation_author=K. Stewardson; citation_author=et al.; citation_journal_title=Nature; citation_volume=522; citation_firstpage=207; citation_lastpage=211; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Identifying hunter-gatherer mobility patterns using strontium isotopes; citation_inbook_title=Prehistoric Hunter-Gatherers of the Baikal Region, Siberia; citation_author=C.M. Haverkort; citation_author=V.I. Bazaliiskii; citation_author=N.A. Savel’ev; citation_firstpage=217; citation_lastpage=238; citation_publication_date=2010; citation_publisher=Penn Press; "><meta name="citation_reference" content="citation_title=Dental Anthropology; citation_author=S. Hillson; citation_publication_date=1996; citation_publisher=Cambridge University Press; "><meta name="citation_reference" content="citation_title=HOPS: automated detection and authentication of pathogen DNA in archaeological remains; citation_author=R. Hübler; citation_author=F.M. Key; citation_author=C. Warinner; citation_author=K.I. Bos; citation_author=J. Krause; citation_author=A. Herbig; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=280; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Bronze Age population dynamics and the rise of dairy pastoralism on the eastern Eurasian steppe; citation_author=C. Jeong; citation_author=S. Wilkin; citation_author=T. Amgalantugs; citation_author=A.S. Bouwman; citation_author=W.T.T. Taylor; citation_author=R.W. Hagan; citation_author=S. Bromage; citation_author=S. Tsolmon; citation_author=C. Trachsel; citation_author=J. Grossmann; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=115; citation_firstpage=E11248; citation_lastpage=E11255; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The genetic history of admixture across inner Eurasia; citation_author=C. Jeong; citation_author=O. Balanovsky; citation_author=E. Lukianova; citation_author=N. Kahbatkyzy; citation_author=P. Flegontov; citation_author=V. Zaporozhchenko; citation_author=A. Immel; citation_author=C.C. Wang; citation_author=O. Ixan; citation_author=E. Khussainova; citation_author=et al.; citation_journal_title=Nat. Ecol. Evol.; citation_volume=3; citation_firstpage=966; citation_lastpage=976; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Upper Palaeolithic genomes reveal deep roots of modern Eurasians; citation_author=E.R. Jones; citation_author=G. Gonzalez-Fortes; citation_author=S. Connell; citation_author=V. Siska; citation_author=A. Eriksson; citation_author=R. Martiniano; citation_author=R.L. McLaughlin; citation_author=M. Gallego Llorente; citation_author=L.M. Cassidy; citation_author=C. Gamba; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=6; citation_firstpage=8912; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=mapDamage2.0: fast approximate Bayesian estimates of ancient DNA damage parameters; citation_author=H. Jónsson; citation_author=A. Ginolhac; citation_author=M. Schubert; citation_author=P.L.F. Johnson; citation_author=L. Orlando; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=1682; citation_lastpage=1684; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The petrous bone--a new sampling site for identifying early dietary patterns in stable isotopic studies; citation_author=M.L.S. Jørkov; citation_author=J. Heinemeier; citation_author=N. Lynnerup; citation_journal_title=Am. J. Phys. Anthropol.; citation_volume=138; citation_firstpage=199; citation_lastpage=209; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Stable isotope ecology and palaeodiet in the Lake Baikal region of Siberia; citation_author=M.A. Katzenberg; citation_author=A. Weber; citation_journal_title=J.&nbsp;Archaeol. Sci.; citation_volume=26; citation_firstpage=651; citation_lastpage=659; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=8. Diet Reconstruction of Prehistoric Hunter-Gatherers in the Lake Baikal Region; citation_inbook_title=Prehistoric Hunter-Gatherers of the Baikal Region, Siberia; citation_author=M.A. Katzenberg; citation_author=V.I. Bazaliiskii; citation_author=O.I. Goriunova; citation_author=N.A. Savel’ev; citation_author=A.W. Weber; citation_firstpage=175; citation_lastpage=192; citation_publication_date=2010; citation_publisher=University of Pennsylvania Press; "><meta name="citation_reference" content="citation_title=Ancient Yersinia pestis genomes from across Western Europe reveal early diversification during the First Pandemic (541–750); citation_author=M. Keller; citation_author=M.A. Spyrou; citation_author=C.L. Scheib; citation_author=G.U. Neumann; citation_author=A. Kröpelin; citation_author=B. Haas-Gebhard; citation_author=B. Päffgen; citation_author=J. Haberstroh; citation_author=A. Ribera I Lacomba; citation_author=C. Raynaud; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_firstpage=12363; citation_lastpage=12372; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform; citation_author=M. Kircher; citation_author=S. Sawyer; citation_author=M. Meyer; citation_journal_title=Nucleic Acids Res.; citation_volume=40; citation_firstpage=e3; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Nineteen Whole-Genome Assemblies of Yersinia pestis subsp. microtus, Including Representatives of Biovars caucasica, talassica, hissarica, altaica, xilingolensis, and ulegeica; citation_author=A.A. Kislichkina; citation_author=A.G. Bogun; citation_author=L.A. Kadnikova; citation_author=N.V. Maiskaya; citation_author=M.E. Platonov; citation_author=N.V. Anisimov; citation_author=E.V. Galkina; citation_author=S.V. Dentovskaya; citation_author=A.P. Anisimov; citation_journal_title=Genome Announc.; citation_volume=3; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Nine Whole-Genome Assemblies of Yersinia pestis subsp. microtus bv. Altaica Strains Isolated from the Altai Mountain Natural Plague Focus (No. 36) in Russia; citation_author=A.A. Kislichkina; citation_author=A.G. Bogun; citation_author=L.A. Kadnikova; citation_author=N.V. Maiskaya; citation_author=V.I. Solomentsev; citation_author=S.V. Dentovskaya; citation_author=S.V. Balakhonov; citation_author=A.P. Anisimov; citation_journal_title=Genome Announc.; citation_volume=6; citation_firstpage=e01440-17; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Six Whole-Genome Assemblies of Yersinia pestis subsp. microtus bv. ulegeica (Phylogroup 0.PE5) Strains Isolated from Mongolian Natural Plague Foci; citation_author=A.A. Kislichkina; citation_author=A.G. Bogun; citation_author=L.A. Kadnikova; citation_author=N.V. Maiskaya; citation_author=V.I. Solomentsev; citation_author=A.A. Sizova; citation_author=S.V. Dentovskaya; citation_author=S.V. Balakhonov; citation_author=A.P. Anisimov; citation_journal_title=Genome Announc.; citation_volume=6; citation_firstpage=e00536-18; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Stable carbon isotope ratios differentiate autotrophs supporting animal diversity in Lake Baikal. Comptes Rendus l’Académie Des Sci. - Ser. III - Sci; citation_author=S.I. Kiyashko; citation_author=P. Richard; citation_author=T. Chandler; citation_author=T.A. Kozlova; citation_author=D.F. Williams; citation_journal_title=Vine; citation_volume=321; citation_firstpage=509; citation_lastpage=516; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Reducing microbial and human contamination in DNA extractions from ancient bones and teeth; citation_author=P. Korlević; citation_author=T. Gerber; citation_author=M.-T. Gansauge; citation_author=M. Hajdinjak; citation_author=S. Nagel; citation_author=A. Aximu-Petri; citation_author=M. Meyer; citation_journal_title=Biotechniques; citation_volume=59; citation_firstpage=87; citation_lastpage=93; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=ANGSD: Analysis of Next Generation Sequencing Data; citation_author=T.S. Korneliussen; citation_author=A. Albrechtsen; citation_author=R. Nielsen; citation_journal_title=BMC Bioinformatics; citation_volume=15; citation_firstpage=356; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Phylogeny and Classification of Yersinia pestis Through the Lens of Strains From the Plague Foci of Commonwealth of Independent States; citation_author=V.V. Kutyrev; citation_author=G.A. Eroshenko; citation_author=V.L. Motin; citation_author=N.Y. Nosov; citation_author=J.M. Krasnov; citation_author=L.M. Kukleva; citation_author=K.A. Nikiforov; citation_author=Z.V. Al’khova; citation_author=E.G. Oglodin; citation_author=N.P. Guseva; citation_journal_title=Front. Microbiol.; citation_volume=9; citation_firstpage=1106; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Ancient human genomes suggest three ancestral populations for present-day Europeans; citation_author=I. Lazaridis; citation_author=N. Patterson; citation_author=A. Mittnik; citation_author=G. Renaud; citation_author=S. Mallick; citation_author=K. Kirsanow; citation_author=P.H. Sudmant; citation_author=J.G. Schraiber; citation_author=S. Castellano; citation_author=M. Lipson; citation_author=et al.; citation_journal_title=Nature; citation_volume=513; citation_firstpage=409; citation_lastpage=413; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Genomic insights into the origin of farming in the ancient Near East; citation_author=I. Lazaridis; citation_author=D. Nadel; citation_author=G. Rollefson; citation_author=D.C. Merrett; citation_author=N. Rohland; citation_author=S. Mallick; citation_author=D. Fernandes; citation_author=M. Novak; citation_author=B. Gamarra; citation_author=K. Sirak; citation_author=et al.; citation_journal_title=Nature; citation_volume=536; citation_firstpage=419; citation_lastpage=424; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H. Li; citation_author=R. Durbin; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=1754; citation_lastpage=1760; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools; citation_author=H. Li; citation_author=B. Handsaker; citation_author=A. Wysoker; citation_author=T. Fennell; citation_author=J. Ruan; citation_author=N. Homer; citation_author=G. Marth; citation_author=G. Abecasis; citation_author=R. Durbin; citation_author=1000 Genome Project Data Processing Subgroup; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2078; citation_lastpage=2079; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=New method of collagen extraction for radiocarbon dating; citation_author=R. Longin; citation_journal_title=Nature; citation_volume=230; citation_firstpage=241; citation_lastpage=242; citation_publication_date=1971; "><meta name="citation_reference" content="citation_title=K voprosu o drevnem naselenii Priangar’ya po paleoantropologicheskim dannym; citation_inbook_title=Problemy Arheologii Urala i Sibiri; citation_author=N.N. Mamonova; citation_firstpage=8; citation_lastpage=29; citation_publication_date=1973; citation_publisher=M. Nauka; "><meta name="citation_reference" content="citation_title=Radiouglerodnaya khronologiya golocenovyh pogrebenij Pribajkal’ya i Zabajkal’ya po osteologicheskomu materialu iz mogil’nikov; citation_author=N.N. Mamonova; citation_author=L.D. Sulerzhickij; citation_journal_title=Sov. Arkheologiya; citation_volume=1; citation_firstpage=19; citation_lastpage=32; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=Radiouglerodnaya khronologiya golocenovyh pogrebenij Pribajkal’ya i Zabajkal’ya po osteologicheskomu materialu iz mogil’nikov; citation_inbook_title=Chelovek, Adaptaciya, Kul’tura; citation_author=N.N. Mamonova; citation_author=L.D. Sulerzhickij; citation_firstpage=127; citation_lastpage=138; citation_publication_date=2008; citation_publisher=M. IA RAN; "><meta name="citation_reference" content="citation_title=Genome-wide patterns of selection in 230 ancient Eurasians; citation_author=I. Mathieson; citation_author=I. Lazaridis; citation_author=N. Rohland; citation_author=S. Mallick; citation_author=N. Patterson; citation_author=S.A. Roodenberg; citation_author=E. Harney; citation_author=K. Stewardson; citation_author=D. Fernandes; citation_author=M. Novak; citation_author=et al.; citation_journal_title=Nature; citation_volume=528; citation_firstpage=499; citation_lastpage=503; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The genomic history of southeastern Europe; citation_author=I. Mathieson; citation_author=S. Alpaslan-Roodenberg; citation_author=C. Posth; citation_author=A. Szécsényi-Nagy; citation_author=N. Rohland; citation_author=S. Mallick; citation_author=I. Olalde; citation_author=N. Broomandkhoshbacht; citation_author=F. Candilio; citation_author=O. Cheronet; citation_author=et al.; citation_journal_title=Nature; citation_volume=555; citation_firstpage=197; citation_lastpage=203; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Illumina Sequencing Library Preparation for Highly Multiplexed Target Capture and Sequencing; citation_author=M. Meyer; citation_author=M. Kircher; citation_journal_title=Cold Spring Harb. Protoc.; citation_volume=2010; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Stepwise enrichment of 15N along food chains: Further evidence and the relation between δ15N and animal age; citation_author=M. Minagawa; citation_author=E. Wada; citation_journal_title=Geochim. Cosmochim. Acta; citation_volume=48; citation_firstpage=1135; citation_lastpage=1140; citation_publication_date=1984; "><meta name="citation_reference" content="citation_title=A rationale for repression and/or loss of motility by pathogenic yersinia in the mammalian host; citation_inbook_title=The Genus Yersinia: From Genomics to Function; citation_author=S.A. Minnich; citation_author=H.N. Rohde; citation_firstpage=298; citation_lastpage=311; citation_publication_date=2007; citation_publisher=Springer; "><meta name="citation_reference" content="citation_title=The genetic prehistory of the Baltic Sea region; citation_author=A. Mittnik; citation_author=C.C. Wang; citation_author=S. Pfrengle; citation_author=M. Daubaras; citation_author=G. Zariņa; citation_author=F. Hallgren; citation_author=R. Allmäe; citation_author=V. Khartanovich; citation_author=V. Moiseyev; citation_author=M. Tõrv; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=1; citation_lastpage=11; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Kinship-based social inequality in Bronze Age Europe; citation_author=A. Mittnik; citation_author=K. Massy; citation_author=C. Knipper; citation_author=F. Wittenborn; citation_author=R. Friedrich; citation_author=S. Pfrengle; citation_author=M. Burri; citation_author=N. Carlichi-Witjes; citation_author=H. Deeg; citation_author=A. Furtwängler; citation_author=et al.; citation_journal_title=Science; citation_volume=366; citation_firstpage=731; citation_lastpage=734; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Passports from the past: Investigating human dispersals using strontium isotope analysis of tooth enamel; citation_author=J. Montgomery; citation_journal_title=Ann. Hum. Biol.; citation_volume=37; citation_firstpage=325; citation_lastpage=346; citation_publication_date=2010; "><meta name="citation_reference" content="citation_author=P. Moorjani; citation_author=N. Patterson; "><meta name="citation_reference" content="citation_title=Yersinia pestis genome sequencing identifies patterns of global phylogenetic diversity; citation_author=G. Morelli; citation_author=Y. Song; citation_author=C.J. Mazzoni; citation_author=M. Eppinger; citation_author=P. Roumagnac; citation_author=D.M. Wagner; citation_author=M. Feldkamp; citation_author=B. Kusecek; citation_author=A.J. Vogler; citation_author=Y. Li; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=42; citation_firstpage=1140; citation_lastpage=1143; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans; citation_author=J.V. Moreno-Mayar; citation_author=B.A. Potter; citation_author=L. Vinner; citation_author=M. Steinrücken; citation_author=S. Rasmussen; citation_author=J. Terhorst; citation_author=J.A. Kamm; citation_author=A. Albrechtsen; citation_author=A.S. Malaspinas; citation_author=M. Sikora; citation_author=et al.; citation_journal_title=Nature; citation_volume=553; citation_firstpage=203; citation_lastpage=207; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Biological diversity and population history of Middle Holocene hunter-gatherers from the Cis-Baikal region of Siberia; citation_author=A.A. Movsesian; citation_author=V.Y. Bakholdina; citation_author=D.V. Pezhemsky; citation_journal_title=Am. J. Phys. Anthropol.; citation_volume=155; citation_firstpage=559; citation_lastpage=570; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Sushchestvovala li geneticheskaya preemstvennost’ mezhdu naseleniem razlichnyh etapov Pribajkal’skogo neolita?; citation_author=A.A. Movsesian; citation_author=V.Y. Bakholdina; citation_author=D.V. Pezhemsky; citation_journal_title=Vestn. Mosk. Univ. Seriya 23 Antropol.; citation_volume=3; citation_firstpage=94; citation_lastpage=104; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Integrative approach using Yersinia pestis genomes to revisit the historical landscape of plague during the Medieval Period; citation_author=A. Namouchi; citation_author=M. Guellil; citation_author=O. Kersten; citation_author=S. Hänsch; citation_author=C. Ottoni; citation_author=B.V. Schmid; citation_author=E. Pacciani; citation_author=L. Quaglia; citation_author=M. Vermunt; citation_author=E.L. Bauer; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=115; citation_firstpage=E11790; citation_lastpage=E11797; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The Isotopic Constitution of Strontium, Barium, Bismuth, Thallium and Mercury; citation_author=A.O. Nier; citation_journal_title=Phys. Rev.; citation_volume=54; citation_firstpage=275; citation_lastpage=278; citation_publication_date=1938; "><meta name="citation_reference" content="citation_title=A freshwater old carbon offset in Lake Baikal, Siberia and problems with the radiocarbon dating of archaeological sediments: Evidence from the Sagan-Zaba II site; citation_author=T. Nomokonova; citation_author=R.J. Losey; citation_author=O.I. Goriunova; citation_author=A.W. Weber; citation_journal_title=Quat. Int.; citation_volume=290–291; citation_firstpage=110; citation_lastpage=125; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Neoliticheskie stoyanki na Verhnej Lene. Kraeved. v Irkutsk. Gubernii Zap.Stud. Nauch; citation_author=A.P. Okladnikov; citation_journal_title=Kruzhka Kraeved. Pri Irkut. Gos. Un-Te; citation_volume=3; citation_firstpage=29; citation_lastpage=38; citation_publication_date=1926; "><meta name="citation_reference" content="citation_title=Neoliticheskij mogil’nik v mestnosti Hapcagaj (Verhnyaya Lena); citation_author=A.P. Okladnikov; citation_journal_title=Izv. VSORGO; citation_volume=53; citation_firstpage=125; citation_lastpage=134; citation_publication_date=1928; "><meta name="citation_reference" content="citation_title=Raskopki v Kachugskom rajone po pritokam Leny; citation_author=A.P. Okladnikov; citation_journal_title=Vlast’ Tr.; citation_volume=10; citation_publication_date=1929; "><meta name="citation_reference" content="citation_title=Neolit i bronzovyj vek Pribajkal’ya. Istoriko-arheologicheskoe issledovanie; citation_author=A.P. Okladnikov; citation_journal_title=CH. I, II. Mater. i Issled. Po Arheol. SSSR; citation_volume=18; citation_publication_date=1950; "><meta name="citation_reference" content="citation_title=Neolit i bronzovyj vek Pribajkal’ya. CH. III. (Glazkovskoe vremya); citation_author=A.P. Okladnikov; citation_journal_title=Mater. i Issled. Po Arheol. SSSR; citation_volume=43; citation_publication_date=1955; "><meta name="citation_reference" content="citation_title=Otchet ob issledovanii paleoliticheskogo poseleniia Ust’-Kyakhta v 1976 g; citation_inbook_title=Archive of the Institut Arkheologii i Etnografii Sibirskogo Otdeleniia Rossiiskoi Akademii Nauk; citation_author=A.P. Okladnikov; citation_firstpage=78; citation_publication_date=1977; citation_publisher=Novosibirsk; "><meta name="citation_reference" content="citation_title=Nauchnyi otchet o raskopkakh stoianki Ust’-Kyakhta 1 (Kyakhtinskii raion BurASSR) v 1978 g; citation_inbook_title=Archive of the Institut Arkheologii i Etnografii Sibirskogo Otdeleniia Rossiiskoi Akademii Nauk; citation_author=A.P. Okladnikov; citation_firstpage=21; citation_publication_date=1979; citation_publisher=Novosibirsk; "><meta name="citation_reference" content="citation_title=The Beaker phenomenon and the genomic transformation of northwest Europe; citation_author=I. Olalde; citation_author=S. Brace; citation_author=M.E. Allentoft; citation_author=I. Armit; citation_author=K. Kristiansen; citation_author=T. Booth; citation_author=N. Rohland; citation_author=S. Mallick; citation_author=A. Szécsényi-Nagy; citation_author=A. Mittnik; citation_author=et al.; citation_journal_title=Nature; citation_volume=555; citation_firstpage=190; citation_lastpage=196; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Radiocarbon dating of archaeological sites of Siberia and the Far East; citation_inbook_title=Methods of Natural Sciences in Archaeological Reconstructions; citation_author=L.A. Orlova; citation_firstpage=207; citation_lastpage=232; citation_publication_date=1995; citation_publisher=Publishing House of the Institute of Archeology and Ethnography SB RAS; "><meta name="citation_reference" content="citation_title=Population structure and eigenanalysis; citation_author=N. Patterson; citation_author=A.L. Price; citation_author=D. Reich; citation_journal_title=PLoS Genet.; citation_volume=2; citation_firstpage=e190; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Ancient admixture in human history; citation_author=N. Patterson; citation_author=P. Moorjani; citation_author=Y. Luo; citation_author=S. Mallick; citation_author=N. Rohland; citation_author=Y. Zhan; citation_author=T. Genschoreck; citation_author=T. Webster; citation_author=D. Reich; citation_journal_title=Genetics; citation_volume=192; citation_firstpage=1065; citation_lastpage=1093; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Kostyanaya industriya stoyanki Ust’-Kyahta-3 (Zapadnoe Zabajkal’e) [Bone industry of Ust’-Kyakhta Site (Western Trans-Baikalia)]. Gumanit. Nauk. v Sib; citation_author=G.D. Pavlenok; citation_journal_title=Humanities Sib.; citation_volume=2; citation_firstpage=14; citation_lastpage=18; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Tekhnologiya obrabotki kamnya v selenginskoi kulture Zapadnogo Zabaikaliya (po materialam stoyanki Ust-Kyakhta-3); citation_author=G.D. Pavlenok; citation_publication_date=2015; citation_publisher=Altai State University; "><meta name="citation_reference" content="citation_title=Tekhnologiya izgotovleniya klinovidnyh nukleusov v selenginskoj kul’ture Zapadnogo Zabajkal’ya (po materialam stoyanki Ust’-Kyahta-3). Izv. Altaiskogo Gos. Univ. Istor. Nauk. i Arkheologiia [Bulletin Altai State Univ; citation_author=G.D. Pavlenok; citation_journal_title=Hist. Sci. Archaeol.; citation_volume=87; citation_firstpage=178; citation_lastpage=184; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=New Dental Finds Associated with the Paleolithic Selenga Culture, Western Trans-Baikal Region; citation_author=G.D. Pavlenok; citation_author=A.V. Zubova; citation_journal_title=Archaeol. Ethnol. Anthropol. Eurasia; citation_volume=47; citation_firstpage=3; citation_lastpage=11; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=EAGER: efficient ancient genome reconstruction; citation_author=A. Peltzer; citation_author=G. Jäger; citation_author=A. Herbig; citation_author=A. Seitz; citation_author=C. Kniep; citation_author=J. Krause; citation_author=K. Nieselt; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=60; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Concomitant separation of strontium and samarium-neodymium for isotopic analysis in silicate samples, based on specific extraction chromatography; citation_author=C. Pin; citation_author=D. Briot; citation_author=C. Bassin; citation_author=F. Poitrasson; citation_journal_title=Anal. Chim. Acta; citation_volume=298; citation_firstpage=209; citation_lastpage=217; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=Reconstructing the Deep Population History of Central and South America; citation_author=C. Posth; citation_author=N. Nakatsuka; citation_author=I. Lazaridis; citation_author=P. Skoglund; citation_author=S. Mallick; citation_author=T.C. Lamnidis; citation_author=N. Rohland; citation_author=K. Nägele; citation_author=N. Adamski; citation_author=E. Bertolini; citation_author=et al.; citation_journal_title=Cell; citation_volume=175; citation_firstpage=1185; citation_lastpage=1197; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Identifying Y-chromosome haplogroups in arbitrarily large samples of sequenced or genotyped men; citation_author=G.D. Poznik; citation_journal_title=bioRxiv; citation_doi=10.1101/088716; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=New insights into how Yersinia pestis adapts to its mammalian host during bubonic plague; citation_author=E. Pradel; citation_author=N. Lemaître; citation_author=M. Merchez; citation_author=I. Ricard; citation_author=A. Reboul; citation_author=A. Dewitte; citation_author=F. Sebbane; citation_journal_title=PLoS Pathog.; citation_volume=10; citation_firstpage=e1004029; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Bone chemistry and prehistoric diet: strontium studies of laboratory rats; citation_author=T.D. Price; citation_author=R.W. Swick; citation_author=E.P. Chase; citation_journal_title=Am. J. Phys. Anthropol.; citation_volume=70; citation_firstpage=365; citation_lastpage=375; citation_publication_date=1986; "><meta name="citation_reference" content="citation_title=The Characterization of Biologically Available Strontium Isotope Ratios for the Study of Prehistoric Migration; citation_author=T.D. Price; citation_author=J.H. Burton; citation_author=R.A. Bentley; citation_journal_title=Archaeometry; citation_volume=44; citation_firstpage=117; citation_lastpage=135; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; citation_author=S. Purcell; citation_author=B. Neale; citation_author=K. Todd-Brown; citation_author=L. Thomas; citation_author=M.A.R. Ferreira; citation_author=D. Bender; citation_author=J. Maller; citation_author=P. Sklar; citation_author=P.I.W. de Bakker; citation_author=M.J. Daly; citation_author=P.C. Sham; citation_journal_title=Am. J. Hum. Genet.; citation_volume=81; citation_firstpage=559; citation_lastpage=575; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features; citation_author=A.R. Quinlan; citation_author=I.M. Hall; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=841; citation_lastpage=842; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Computational Many-Particle Physics; citation_author=R Development Core Team; citation_publication_date=2008; citation_publisher=Springer Berlin Heidelberg; "><meta name="citation_reference" content="citation_title=Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans; citation_author=M. Raghavan; citation_author=P. Skoglund; citation_author=K.E. Graf; citation_author=M. Metspalu; citation_author=A. Albrechtsen; citation_author=I. Moltke; citation_author=S. Rasmussen; citation_author=T.W. Stafford; citation_author=L. Orlando; citation_author=E. Metspalu; citation_author=et al.; citation_journal_title=Nature; citation_volume=505; citation_firstpage=87; citation_lastpage=91; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The genetic prehistory of the New World Arctic; citation_author=M. Raghavan; citation_author=M. DeGiorgio; citation_author=A. Albrechtsen; citation_author=I. Moltke; citation_author=P. Skoglund; citation_author=T.S. Korneliussen; citation_author=B. Grønnow; citation_author=M. Appelt; citation_author=H.C. Gulløv; citation_author=T.M. Friesen; citation_author=et al.; citation_journal_title=Science; citation_volume=345; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Population genetics. Genomic evidence for the Pleistocene and recent population history of Native Americans; citation_author=M. Raghavan; citation_author=M. Steinrücken; citation_author=K. Harris; citation_author=S. Schiffels; citation_author=S. Rasmussen; citation_author=M. DeGiorgio; citation_author=A. Albrechtsen; citation_author=C. Valdiosera; citation_author=M.C. Ávila-Arcos; citation_author=A.-S. Malaspinas; citation_author=et al.; citation_journal_title=Science; citation_volume=349; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Analyzing Radiocarbon Reservoir Offsets Through Stable Nitrogen Isotopes and Bayesian Modeling: A Case Study Using Paired Human and Faunal Remains from the Cis-Baikal Region, Siberia; citation_author=C.B. Ramsey; citation_author=R. Schulting; citation_author=O.I. Goriunova; citation_author=V.I. Bazaliiskii; citation_author=A.W. Weber; citation_journal_title=Radiocarbon; citation_volume=56; citation_firstpage=789; citation_lastpage=799; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Emergence and Spread of Basal Lineages of Yersinia pestis during the Neolithic Decline; citation_author=N. Rascovan; citation_author=K.-G. Sjögren; citation_author=K. Kristiansen; citation_author=R. Nielsen; citation_author=E. Willerslev; citation_author=C. Desnues; citation_author=S. Rasmussen; citation_journal_title=Cell; citation_volume=176; citation_firstpage=295; citation_lastpage=305; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Ancient human genome sequence of an extinct Palaeo-Eskimo; citation_author=M. Rasmussen; citation_author=Y. Li; citation_author=S. Lindgreen; citation_author=J.S. Pedersen; citation_author=A. Albrechtsen; citation_author=I. Moltke; citation_author=M. Metspalu; citation_author=E. Metspalu; citation_author=T. Kivisild; citation_author=R. Gupta; citation_author=et al.; citation_journal_title=Nature; citation_volume=463; citation_firstpage=757; citation_lastpage=762; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The genome of a Late Pleistocene human from a Clovis burial site in western Montana; citation_author=M. Rasmussen; citation_author=S.L. Anzick; citation_author=M.R. Waters; citation_author=P. Skoglund; citation_author=M. DeGiorgio; citation_author=T.W. Stafford; citation_author=S. Rasmussen; citation_author=I. Moltke; citation_author=A. Albrechtsen; citation_author=S.M. Doyle; citation_author=et al.; citation_journal_title=Nature; citation_volume=506; citation_firstpage=225; citation_lastpage=229; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The ancestry and affiliations of Kennewick Man; citation_author=M. Rasmussen; citation_author=M. Sikora; citation_author=A. Albrechtsen; citation_author=T.S. Korneliussen; citation_author=J.V. Moreno-Mayar; citation_author=G.D. Poznik; citation_author=C.P.E. Zollikofer; citation_author=M.P. de León; citation_author=M.E. Allentoft; citation_author=I. Moltke; citation_author=et al.; citation_journal_title=Nature; citation_volume=523; citation_firstpage=455; citation_lastpage=458; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago; citation_author=S. Rasmussen; citation_author=M.E. Allentoft; citation_author=K. Nielsen; citation_author=L. Orlando; citation_author=M. Sikora; citation_author=K.-G. Sjögren; citation_author=A.G. Pedersen; citation_author=M. Schubert; citation_author=A. Van Dam; citation_author=C.M.O. Kapel; citation_author=et al.; citation_journal_title=Cell; citation_volume=163; citation_firstpage=571; citation_lastpage=582; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Reconstructing Indian population history; citation_author=D. Reich; citation_author=K. Thangaraj; citation_author=N. Patterson; citation_author=A.L. Price; citation_author=L. Singh; citation_journal_title=Nature; citation_volume=461; citation_firstpage=489; citation_lastpage=494; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=IntCal13 and Marine13 Radiocarbon Age Calibration Curves 0–50,000 Years cal BP; citation_author=P.J. Reimer; citation_author=E. Bard; citation_author=A. Bayliss; citation_author=J.W. Beck; citation_author=P.G. Blackwell; citation_author=C.B. Ramsey; citation_author=C.E. Buck; citation_author=H. Cheng; citation_author=R.L. Edwards; citation_author=M. Friedrich; citation_author=et al.; citation_journal_title=Radiocarbon; citation_volume=55; citation_firstpage=1869; citation_lastpage=1887; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Schmutzi: estimation of contamination and endogenous mitochondrial consensus calling for ancient DNA; citation_author=G. Renaud; citation_author=V. Slon; citation_author=A.T. Duggan; citation_author=J. Kelso; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=224; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Ancient DNA extraction from bones and teeth; citation_author=N. Rohland; citation_author=M. Hofreiter; citation_journal_title=Nat. Protoc.; citation_volume=2; citation_firstpage=1756; citation_lastpage=1762; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Partial uracil-DNA-glycosylase treatment for screening of ancient DNA; citation_author=N. Rohland; citation_author=E. Harney; citation_author=S. Mallick; citation_author=S. Nordenfelt; citation_author=D. Reich; citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_volume=370; citation_firstpage=20130624; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Substitution of calcium by strontium within selected calcium phosphates; citation_author=E. Rokita; citation_author=C. Hermes; citation_author=H.-F. Nolting; citation_author=J. Ryczek; citation_journal_title=J.&nbsp;Cryst. Growth; citation_volume=130; citation_firstpage=543; citation_lastpage=552; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Extensive Farming in Estonia Started through a Sex-Biased Migration from the Steppe; citation_author=L. Saag; citation_author=L. Varul; citation_author=C.L. Scheib; citation_author=J. Stenderup; citation_author=M.E. Allentoft; citation_author=L. Saag; citation_author=L. Pagani; citation_author=M. Reidla; citation_author=K. Tambets; citation_author=E. Metspalu; citation_author=et al.; citation_journal_title=Curr. Biol.; citation_volume=27; citation_firstpage=2185; citation_lastpage=2193; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Ancient human parallel lineages within North America contributed to a coastal expansion; citation_author=C.L. Scheib; citation_author=H. Li; citation_author=T. Desai; citation_author=V. Link; citation_author=C. Kendall; citation_author=G. Dewar; citation_author=P.W. Griffith; citation_author=A. Mörseburg; citation_author=J.R. Johnson; citation_author=A. Potter; citation_author=et al.; citation_journal_title=Science; citation_volume=360; citation_firstpage=1024; citation_lastpage=1027; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=AdapterRemoval v2: rapid adapter trimming, identification, and read merging; citation_author=M. Schubert; citation_author=S. Lindgreen; citation_author=L. Orlando; citation_journal_title=BMC Res. Notes; citation_volume=9; citation_firstpage=88; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Targeted enrichment of ancient pathogens yielding the pPCP1 plasmid of Yersinia pestis from victims of the Black Death; citation_author=V.J. Schuenemann; citation_author=K. Bos; citation_author=S. DeWitte; citation_author=S. Schmedes; citation_author=J. Jamieson; citation_author=A. Mittnik; citation_author=S. Forrest; citation_author=B.K. Coombes; citation_author=J.W. Wood; citation_author=D.J.D. Earn; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=108; citation_firstpage=E746; citation_lastpage=E752; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Freshwater Reservoir Offsets Investigated Through Paired Human-Faunal 14C Dating and Stable Carbon and Nitrogen Isotope Analysis at Lake Baikal, Siberia; citation_author=R.J. Schulting; citation_author=C.B. Ramsey; citation_author=V.I. Bazaliiskii; citation_author=O.I. Goriunova; citation_author=A. Weber; citation_journal_title=Radiocarbon; citation_volume=56; citation_firstpage=991; citation_lastpage=1008; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Highly Variable Freshwater Reservoir Offsets Found along the Upper Lena Watershed, Cis-Baikal, Southeast Siberia; citation_author=R.J. Schulting; citation_author=C. Bronk Ramsey; citation_author=V.I. Bazaliiskii; citation_author=A. Weber; citation_journal_title=Radiocarbon; citation_volume=57; citation_firstpage=581; citation_lastpage=593; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Nitrogen isotopic ecology in southern Africa: Implications for environmental and dietary tracing; citation_author=J.C. Sealy; citation_author=N.J. van der Merwe; citation_author=J.A.L. Thorp; citation_author=J.L. Lanham; citation_journal_title=Geochim. Cosmochim. Acta; citation_volume=51; citation_firstpage=2707; citation_lastpage=2717; citation_publication_date=1987; "><meta name="citation_reference" content="citation_title=The population history of northeastern Siberia since the Pleistocene; citation_author=M. Sikora; citation_author=V.V. Pitulko; citation_author=V.C. Sousa; citation_author=M.E. Allentoft; citation_author=L. Vinner; citation_author=S. Rasmussen; citation_author=A. Margaryan; citation_author=P. de Barros Damgaard; citation_author=C. de la Fuente; citation_author=G. Renaud; citation_author=et al.; citation_journal_title=Nature; citation_volume=570; citation_firstpage=182; citation_lastpage=188; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genome-wide data from two early Neolithic East Asian individuals dating to 7700 years ago; citation_author=V. Siska; citation_author=E.R. Jones; citation_author=S. Jeon; citation_author=Y. Bhak; citation_author=H.-M. Kim; citation_author=Y.S. Cho; citation_author=H. Kim; citation_author=K. Lee; citation_author=E. Veselovskaya; citation_author=T. Balueva; citation_author=et al.; citation_journal_title=Sci. Adv.; citation_volume=3; citation_firstpage=e1601877; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Two categories of c/c ratios for higher plants; citation_author=B.N. Smith; citation_author=S. Epstein; citation_journal_title=Plant Physiol.; citation_volume=47; citation_firstpage=380; citation_lastpage=384; citation_publication_date=1971; "><meta name="citation_reference" content="citation_title=GeneImp: Fast imputation to large reference panels using genotype likelihoods from ultralow coverage sequencing; citation_author=A. Spiliopoulou; citation_author=M. Colombo; citation_author=P. Orchard; citation_author=F. Agakov; citation_author=P. McKeigue; citation_journal_title=Genetics; citation_volume=206; citation_firstpage=91; citation_lastpage=104; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Historical Y. pestis genomes reveal the European Black Death as the source of ancient and modern plague pandemics; citation_author=M.A. Spyrou; citation_author=R.I. Tukhbatova; citation_author=M. Feldman; citation_author=J. Drath; citation_author=S. Kacki; citation_author=J. Beltrán de Heredia; citation_author=S. Arnold; citation_author=A.G. Sitdikov; citation_author=D. Castex; citation_author=J. Wahl; citation_author=et al.; citation_journal_title=Cell Host &amp; Microbe; citation_volume=19; citation_firstpage=874; citation_lastpage=881; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Analysis of 3800-year-old Yersinia pestis genomes suggests Bronze Age origin for bubonic plague; citation_author=M.A. Spyrou; citation_author=R.I. Tukhbatova; citation_author=C.-C. Wang; citation_author=A.A. Valtueña; citation_author=A.K. Lankapalli; citation_author=V.V. Kondrashin; citation_author=V.A. Tsybin; citation_author=A. Khokhlov; citation_author=D. Kühnert; citation_author=A. Herbig; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=2234; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Phylogeography of the second plague pandemic revealed through analysis of historical Yersinia pestis genomes; citation_author=M.A. Spyrou; citation_author=M. Keller; citation_author=R.I. Tukhbatova; citation_author=C.L. Scheib; citation_author=E.A. Nelson; citation_author=A. Andrades Valtueña; citation_author=G.U. Neumann; citation_author=D. Walker; citation_author=A. Alterauge; citation_author=N. Carty; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=4470; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies; citation_author=A. Stamatakis; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=1312; citation_lastpage=1313; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Retracing the evolutionary path that led to flea-borne transmission of Yersinia pestis; citation_author=Y.-C. Sun; citation_author=C.O. Jarrett; citation_author=C.F. Bosio; citation_author=B.J. Hinnebusch; citation_journal_title=Cell Host Microbe; citation_volume=15; citation_firstpage=578; citation_lastpage=586; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Tortsovye klinovidnye nukleusy Zapadnogo Zabaikaliya v pozdnem paleolite i mezolite; citation_inbook_title=Kamennyi Vek Yuzhnoi Sibiri i Mongolii: Teoreticheskie Problemy I Novye Otkrytiya; citation_author=V.I. Tashak; citation_firstpage=59; citation_lastpage=74; citation_publication_date=2000; citation_publisher=Buryat scientific center SB RAS; "><meta name="citation_reference" content="citation_title=Paleoliticheskie i mezoliticheskie pamiatniki Ust’-Kiakhty; citation_author=V.I. Tashak; citation_publication_date=2005; citation_publisher=Buryatia Scientific Center of the Siberian Branch of the Russian Academy of Sciences; "><meta name="citation_reference" content="citation_title=Some probabilistic and statistical problems in the analysis of DNA sequences; citation_author=S. Tavaré; citation_journal_title=Lect. Math Life Sci.; citation_volume=17; citation_firstpage=57; citation_lastpage=86; citation_publication_date=1986; "><meta name="citation_reference" content="citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration; citation_author=H. Thorvaldsdóttir; citation_author=J.T. Robinson; citation_author=J.P. Mesirov; citation_journal_title=Brief. Bioinform.; citation_volume=14; citation_firstpage=178; citation_lastpage=192; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Salmonella enterica genomes from victims of a major sixteenth-century epidemic in Mexico; citation_author=Å.J. Vågene; citation_author=A. Herbig; citation_author=M.G. Campana; citation_author=N.M. Robles García; citation_author=C. Warinner; citation_author=S. Sabin; citation_author=M.A. Spyrou; citation_author=A. Andrades Valtueña; citation_author=D. Huson; citation_author=N. Tuross; citation_author=et al.; citation_journal_title=Nat. Ecol. Evol.; citation_volume=2; citation_firstpage=520; citation_lastpage=528; citation_publication_date=2018; "><meta name="citation_reference" content="Vlast’truda (1928). 21 oktyabrya. 246, 4."><meta name="citation_reference" content="Vlast’truda (1929). 28 iyulya. 17, 4."><meta name="citation_reference" content="citation_title=Ancient human genome-wide data from a 3000-year interval in the Caucasus corresponds with eco-geographic regions; citation_author=C.C. Wang; citation_author=S. Reinhold; citation_author=A. Kalmykov; citation_author=A. Wissgott; citation_author=G. Brandt; citation_author=C. Jeong; citation_author=O. Cheronet; citation_author=M. Ferry; citation_author=E. Harney; citation_author=D. Keating; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=590; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The neolithic and early bronze age of the Lake Baikal Region: A review of recent research; citation_author=A. Weber; citation_journal_title=J.&nbsp;World Prehist.; citation_volume=9; citation_firstpage=99; citation_lastpage=165; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Middle Holocene hunter-gatherers of Cis-Baikal, Siberia: An overview for the new century; citation_author=A.W. Weber; citation_author=R. Bettinger; citation_journal_title=J.&nbsp;Anthropol. Archaeol.; citation_volume=29; citation_firstpage=491; citation_lastpage=506; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Hunter-gatherer culture change and continuity in the middle holocene of the Cis-Baikal, Siberia; citation_author=A.W. Weber; citation_author=D.W. Link; citation_author=M.A. Katzenberg; citation_journal_title=J.&nbsp;Anthropol. Archaeol.; citation_volume=21; citation_firstpage=230; citation_lastpage=299; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Radiocarbon Dates from Neolithic and Bronze Age Hunter-Gatherer Cemeteries in the Cis-Baikal Region of Siberia; citation_author=A.W. Weber; citation_author=R.P. Beukens; citation_author=V.I. Bazaliiskii; citation_author=O.I. Goriunova; citation_author=N.A. Savel’ev; citation_journal_title=Radiocarbon; citation_volume=48; citation_firstpage=127; citation_lastpage=166; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Chronology of middle Holocene hunter–gatherers in the Cis-Baikal region of Siberia: Corrections based on examination of the freshwater reservoir effect; citation_author=A.W. Weber; citation_author=R.J. Schulting; citation_author=C. Bronk Ramsey; citation_author=V.I. Bazaliiskii; citation_author=O.I. Goriunova; citation_author=N.E. Berdnikova; citation_journal_title=Quat. Int.; citation_volume=419; citation_firstpage=74; citation_lastpage=98; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics For Data Analysis; citation_author=H. Wickham; citation_publication_date=2016; citation_publisher=Springer; "><meta name="citation_reference" content="citation_title=Genome Assemblies for 11 Yersinia pestis Strains Isolated in the Caucasus Region; citation_author=E. Zhgenti; citation_author=S.L. Johnson; citation_author=K.W. Davenport; citation_author=G. Chanturia; citation_author=H.E. Daligault; citation_author=P.S. Chain; citation_author=M.P. Nikolich; citation_journal_title=Genome Announc.; citation_volume=3; citation_firstpage=3; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Molecular Darwinian evolution of virulence in Yersinia pestis; citation_author=D. Zhou; citation_author=R. Yang; citation_journal_title=Infect. Immun.; citation_volume=77; citation_firstpage=2242; citation_lastpage=2250; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Early emergence of Yersinia pestis as a severe respiratory pathogen; citation_author=D.L. Zimbler; citation_author=J.A. Schroeder; citation_author=J.L. Eddy; citation_author=W.W. Lathem; citation_journal_title=Nat. Commun.; citation_volume=6; citation_firstpage=7487; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Tekhnologiya proizvodstva busin iz skorlupy yaic strausa v final’nom paleolite Zapadnogo Zabajkal’ya [Technology of Ostrich Eggshell Bead Production in the Final Palaeolithic of Western Transbaikalia]. Strat. plus; citation_author=L.V. Zotkina; citation_author=G.D. Pavlenok; citation_author=V.I. Tashak; citation_journal_title=Archaeol. Cult. Anthropol.; citation_volume=1; citation_firstpage=181; citation_lastpage=197; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Materialy iz drevnih mogil s territorii poselka Kachug i u derevni SHishkino na Verhnej Lene (raskopki A. P. Okladnikova 1928-1930, 1941 gg.). Izv. Lab. Drevnih Tekhnologij; citation_author=V.S. Zubkov; citation_journal_title=Irkutsk Izd-vo IrGTU.; citation_volume=8; citation_firstpage=135; citation_lastpage=146; citation_publication_date=2010; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Modern humans have inhabited the Lake Baikal region since the Upper Paleolithic, though the precise history of its peoples over this long time span is still largely unknown. Here, we report genome-wide data from 19&nbsp;Upper Paleolithic to Early Bronze Age individuals from this Siberian region. An Upper Paleolithic genome shows a direct link with the First Americans by sharing the admixed ancestry that gave rise to all non-Arctic Native Americans. We also demonstrate the formation of Early Neolithic and Bronze Age Baikal populations as the result of prolonged admixture throughout the eighth to sixth millennium BP. Moreover, we detect genetic interactions with western Eurasian steppe populations and reconstruct <i>Yersinia pestis</i> genomes from two Early Bronze Age individuals without western Eurasian ancestry. Overall, our study demonstrates the most deeply divergent connection between Upper Paleolithic Siberians and the First Americans and reveals human and pathogen mobility across Eurasia during the Bronze Age.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=5458&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(20)30502-X&amp;be=3848&amp;fe=5348&amp;dc=5097&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749586597681,%22n%22:0,%22f%22:3223,%22dn%22:3224,%22dne%22:3224,%22c%22:3224,%22s%22:3224,%22ce%22:3224,%22rq%22:3224,%22rp%22:3814,%22rpe%22:4654,%22dl%22:3820,%22di%22:5090,%22ds%22:5097,%22de%22:5113,%22dc%22:5347,%22l%22:5348,%22le%22:5380%7D,%22navigation%22:%7B%7D%7D&amp;fp=4884&amp;fcp=4884&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930502-X" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30502-X"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(20)30502-X"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930502-X" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-693568" aria-expanded="false" aria-controls="login-links-693568" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-693568" id="login-links--693568" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930502-X" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930502-X" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930502-X" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-693568" data-opens="#links-dialog-693568" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-693568" aria-modal="true"><button data-closes="#links-dialog-693568" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930502-X" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930502-X" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930502-X" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2020.04.037">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(20)30502-X" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(19)X0013-6"><span>Volume 181</span>, <span>Issue 6</span></a></span><span class="meta-panel__pages"><span>p1232-1245.e20</span></span><span class="meta-panel__onlineDate">June 11, 2020</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=6&amp;pii=S0092-8674%2819%29X0013-6" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">He</span> <span property="familyName">Yu</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">He</span> <span property="familyName">Yu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Yu/He" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Maria A.</span> <span property="familyName">Spyrou</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Maria A.</span> <span property="familyName">Spyrou</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Spyrou/Maria+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Marina</span> <span property="familyName">Karapetian</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marina</span> <span property="familyName">Karapetian</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Research Institute and Museum of Anthropology, Moscow State University, Moscow 125009, Russia</span></div></div><div class="core-author-link"><a href="/authored-by/Karapetian/Marina" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Svetlana</span> <span property="familyName">Shnaider</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Svetlana</span> <span property="familyName">Shnaider</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Archaeology and Ethnography of the Siberian Branch of the Russian Academy of Sciences, Novosibirsk 630090, Russia</span></div></div><div class="core-author-link"><a href="/authored-by/Shnaider/Svetlana" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Rita</span> <span property="familyName">Radzevičiūtė</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Rita</span> <span property="familyName">Radzevičiūtė</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Radzevi%C4%8Di%C5%ABt%C4%97/Rita" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Kathrin</span> <span property="familyName">Nägele</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kathrin</span> <span property="familyName">Nägele</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/N%C3%A4gele/Kathrin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Gunnar U.</span> <span property="familyName">Neumann</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gunnar U.</span> <span property="familyName">Neumann</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Neumann/Gunnar+U" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Sandra</span> <span property="familyName">Penske</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sandra</span> <span property="familyName">Penske</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Penske/Sandra" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Jana</span> <span property="familyName">Zech</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jana</span> <span property="familyName">Zech</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeology, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Zech/Jana" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Mary</span> <span property="familyName">Lucas</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mary</span> <span property="familyName">Lucas</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeology, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Lucas/Mary" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Petrus</span> <span property="familyName">LeRoux</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Petrus</span> <span property="familyName">LeRoux</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Geological Sciences, University of Cape Town, Rondebosch 7701, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/LeRoux/Petrus" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Patrick</span> <span property="familyName">Roberts</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Patrick</span> <span property="familyName">Roberts</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeology, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Roberts/Patrick" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Galina</span> <span property="familyName">Pavlenok</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Galina</span> <span property="familyName">Pavlenok</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Archaeology and Ethnography of the Siberian Branch of the Russian Academy of Sciences, Novosibirsk 630090, Russia</span></div></div><div class="core-author-link"><a href="/authored-by/Pavlenok/Galina" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Alexandra</span> <span property="familyName">Buzhilova</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alexandra</span> <span property="familyName">Buzhilova</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Research Institute and Museum of Anthropology, Moscow State University, Moscow 125009, Russia</span></div></div><div class="core-author-link"><a href="/authored-by/Buzhilova/Alexandra" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Cosimo</span> <span property="familyName">Posth</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Cosimo</span> <span property="familyName">Posth</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:posth@shh.mpg.de" property="email" title="Send email to Cosimo Posth">posth@shh.mpg.de</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute for Archaeological Sciences, Archaeo- and Palaeogenetics, University of Tübingen, Tübingen 72070, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Posth/Cosimo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup> <a id="eadd0010" href="mailto:posth@shh.mpg.de" property="email" title="Send email to Cosimo Posth">posth@shh.mpg.de</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Choongwon</span> <span property="familyName">Jeong</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Choongwon</span> <span property="familyName">Jeong</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:jeong@shh.mpg.de" property="email" title="Send email to Choongwon Jeong">jeong@shh.mpg.de</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">School of Biological Sciences, Seoul National University, Seoul 08826, Republic of Korea</span></div></div><div class="core-author-link"><a href="/authored-by/Jeong/Choongwon" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup> <a id="eadd0015" href="mailto:jeong@shh.mpg.de" property="email" title="Send email to Choongwon Jeong">jeong@shh.mpg.de</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Johannes</span> <span property="familyName">Krause</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Johannes</span> <span property="familyName">Krause</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9144-3920" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9144-3920</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:krause@shh.mpg.de" property="email" title="Send email to Johannes Krause">krause@shh.mpg.de</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead Contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Krause/Johannes" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">8</a></sup> <a id="eadd0020" href="mailto:krause@shh.mpg.de" property="email" title="Send email to Johannes Krause">krause@shh.mpg.de</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Archaeogenetics, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Research Institute and Museum of Anthropology, Moscow State University, Moscow 125009, Russia</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Institute of Archaeology and Ethnography of the Siberian Branch of the Russian Academy of Sciences, Novosibirsk 630090, Russia</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Archaeology, Max Planck Institute for the Science of Human History, Jena 07745, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Geological Sciences, University of Cape Town, Rondebosch 7701, South Africa</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Institute for Archaeological Sciences, Archaeo- and Palaeogenetics, University of Tübingen, Tübingen 72070, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">School of Biological Sciences, Seoul National University, Seoul 08826, Republic of Korea</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead Contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received November 18, 2019</span>; <span>Revised March 16, 2020</span>; <span>Accepted April 21, 2020</span>; <span>Published online May 20, 2020</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2020.04.037" property="sameAs" target="_blank">10.1016/j.cell.2020.04.037 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S009286742030502X" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2020 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2020.04.037" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0013-6"><img src="/cms/asset/693aff61-b843-4e36-9d8c-1f1580f96ce7/cov200h.gif" alt="Cover Image - Cell, Volume 181, Issue 6"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2820%2930502-X" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental Information (6)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930502-X" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.04.037">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2820%2930502-X" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2020.04.037&amp;pii=S0092-8674%2820%2930502-X" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930502-X" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.04.037">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2020.04.037" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S009286742030502X&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S009286742030502X&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=6&amp;pii=S0092-8674%2819%29X0013-6" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)30552-3" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)30497-9" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical Abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental Information (6)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">An Upper Paleolithic Siberian shows a deep link with the First Peoples of the Americas</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">A 10,000-year continuum of Ancient North Eurasian ancestry in the Lake Baikal region</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">The Neolithic to Bronze Age population formation occurred through prolonged local admixture</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Long-range human and <i>Y.&nbsp;pestis</i> mobility across Eurasia during the Early Bronze Age</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Modern humans have inhabited the Lake Baikal region since the Upper Paleolithic, though the precise history of its peoples over this long time span is still largely unknown. Here, we report genome-wide data from 19&nbsp;Upper Paleolithic to Early Bronze Age individuals from this Siberian region. An Upper Paleolithic genome shows a direct link with the First Americans by sharing the admixed ancestry that gave rise to all non-Arctic Native Americans. We also demonstrate the formation of Early Neolithic and Bronze Age Baikal populations as the result of prolonged admixture throughout the eighth to sixth millennium BP. Moreover, we detect genetic interactions with western Eurasian steppe populations and reconstruct <i>Yersinia pestis</i> genomes from two Early Bronze Age individuals without western Eurasian ancestry. Overall, our study demonstrates the most deeply divergent connection between Upper Paleolithic Siberians and the First Americans and reveals human and pathogen mobility across Eurasia during the Bronze Age.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/119cfa8c-3773-473d-aa46-76e6f972718b/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2020.04.037/asset/68f46f42-e77a-4f2d-a05b-1105c9b64d38/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/119cfa8c-3773-473d-aa46-76e6f972718b/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;ancient+genomics&quot;" alt="Search on this keyword" rel="nofollow">ancient genomics</a></li><li><a href="/action/doSearch?AllField=&quot;human+history&quot;" alt="Search on this keyword" rel="nofollow">human history</a></li><li><a href="/action/doSearch?AllField=&quot;Siberia&quot;" alt="Search on this keyword" rel="nofollow">Siberia</a></li><li><a href="/action/doSearch?AllField=&quot;Native+Americans&quot;" alt="Search on this keyword" rel="nofollow">Native Americans</a></li><li><a href="/action/doSearch?AllField=&quot;mobility&quot;" alt="Search on this keyword" rel="nofollow">mobility</a></li><li><a href="/action/doSearch?AllField=&quot;Yersinia+pestis&quot;" alt="Search on this keyword" rel="nofollow"><i>Yersinia pestis</i></a></li><li><a href="/action/doSearch?AllField=&quot;Upper+Paleolithic&quot;" alt="Search on this keyword" rel="nofollow">Upper Paleolithic</a></li><li><a href="/action/doSearch?AllField=&quot;Neolithic&quot;" alt="Search on this keyword" rel="nofollow">Neolithic</a></li><li><a href="/action/doSearch?AllField=&quot;Bronze+Age&quot;" alt="Search on this keyword" rel="nofollow">Bronze Age</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">The Lake Baikal region in Siberia has been inhabited by modern humans since the Upper Paleolithic and has a rich archaeological record (<span class="dropBlock reference-citations"><a id="body-ref-sref145" href="#" role="doc-biblioref" data-xml-rid="bib47 bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145" aria-controls="sref145">Katzenberg and Weber, 1999; Weber, 1995</a><div class="dropBlock__holder" data-db-target-of="sref145"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Katzenberg, M.A. ∙ Weber, A.</div><div><strong>Stable isotope ecology and palaeodiet in the Lake Baikal region of Siberia</strong></div><div><em>J.&nbsp;Archaeol. Sci.</em> 1999; <strong>26</strong>:651-659</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jasc.1998.0382" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0033150923" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjasc.1998.0382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Weber, A.</div><div><strong>The neolithic and early bronze age of the Lake Baikal Region: A review of recent research</strong></div><div><em>J.&nbsp;World Prehist.</em> 1995; <strong>9</strong>:99-165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF02221004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0029532484" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF02221004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the past 5 years, ancient genomic studies have revealed multiple genetic turnovers and admixture events in this region. The 24,000-year-old individual (MA1) from the Mal’ta site represents an ancestry referred to as “Ancient North Eurasian (ANE),” which was widespread across Siberia during the Paleolithic (<span class="dropBlock reference-citations"><a id="body-ref-sref129-1" href="#" role="doc-biblioref" data-xml-rid="bib33 bib107 bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129-1" aria-controls="sref129-1">Fu et&nbsp;al., 2016; Raghavan et&nbsp;al., 2014a; Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref129-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Fu, Q. ∙ Posth, C. ∙ Hajdinjak, M. ...</div><div><strong>The genetic history of Ice Age Europe</strong></div><div><em>Nature.</em> 2016; <strong>534</strong>:200-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84969820697" target="_blank" aria-label="Open Scopus in new tab">Scopus (302)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27135931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17993&amp;pmid=27135931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Raghavan, M. ∙ Skoglund, P. ∙ Graf, K.E. ...</div><div><strong>Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans</strong></div><div><em>Nature.</em> 2014; <strong>505</strong>:87-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12736" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84892367625" target="_blank" aria-label="Open Scopus in new tab">Scopus (334)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24256729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12736&amp;pmid=24256729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and that contributed to the genetic profile of a vast number of present-day Eurasian populations as well as Native Americans (<span class="dropBlock reference-citations"><a id="body-ref-sref109-1" href="#" role="doc-biblioref" data-xml-rid="bib38 bib59 bib60 bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-1" aria-controls="sref109-1">Haak et&nbsp;al., 2015; Lazaridis et&nbsp;al., 2014, 2016; Raghavan et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref109-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (615)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Lazaridis, I. ∙ Patterson, N. ∙ Mittnik, A. ...</div><div><strong>Ancient human genomes suggest three ancestral populations for present-day Europeans</strong></div><div><em>Nature.</em> 2014; <strong>513</strong>:409-413</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13673" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84907803675" target="_blank" aria-label="Open Scopus in new tab">Scopus (524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25230663/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13673&amp;pmid=25230663" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Lazaridis, I. ∙ Nadel, D. ∙ Rollefson, G. ...</div><div><strong>Genomic insights into the origin of farming in the ancient Near East</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:419-424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19310" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27459054/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19310&amp;pmid=27459054" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Raghavan, M. ∙ Steinrücken, M. ∙ Harris, K. ...</div><div><strong>Population genetics. Genomic evidence for the Pleistocene and recent population history of Native Americans</strong></div><div><em>Science.</em> 2015; <strong>349</strong></div><div>aab3884–aab3884</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aab3884" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84939864401" target="_blank" aria-label="Open Scopus in new tab">Scopus (182)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26198033/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aab3884&amp;pmid=26198033" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ANE ancestry was suggested to have been largely replaced in the Lake Baikal region during the Early Neolithic by a gene pool related to present-day northeast Asians, with a limited resurgence of ANE ancestry by the Early Bronze Age (<span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Damgaard et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0035" role="paragraph">Siberia has also been proposed as a source for multiple waves of dispersals into the Americas, the first of which was shown to be driven by a founding population estimated to have formed around 25,000–20,000 years before the present (BP) (<span class="dropBlock reference-citations"><a id="body-ref-sref109-2" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-2" aria-controls="sref109-2">Raghavan et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref109-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Raghavan, M. ∙ Steinrücken, M. ∙ Harris, K. ...</div><div><strong>Population genetics. Genomic evidence for the Pleistocene and recent population history of Native Americans</strong></div><div><em>Science.</em> 2015; <strong>349</strong></div><div>aab3884–aab3884</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aab3884" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84939864401" target="_blank" aria-label="Open Scopus in new tab">Scopus (182)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26198033/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aab3884&amp;pmid=26198033" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The so-called Ancient Beringian ancestry represented by a 11,500-year-old Alaskan individual (USR1) was shown to be part of this founding population, estimated to have split from other Native Americans around 23,000 BP (<span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Moreno-Mayar et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Moreno-Mayar, J.V. ∙ Potter, B.A. ∙ Vinner, L. ...</div><div><strong>Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans</strong></div><div><em>Nature.</em> 2018; <strong>553</strong>:203-207</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85040467287" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29323294/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25173&amp;pmid=29323294" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, the recently published 9,800-year-old Kolyma genome from northeastern Siberia was suggested to represent the closest relative to Native American populations outside of the Americas (<span class="dropBlock reference-citations"><a id="body-ref-sref129-2" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129-2" aria-controls="sref129-2">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref129-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, the Paleo-Eskimo ancestry represented by a 4,000-year-old Saqqaq individual from Greenland was also estimated to have split from northeastern Siberian groups and migrated to Arctic America around 6,000–5,000 BP (<span class="dropBlock reference-citations"><a id="body-ref-sref112-1" href="#" role="doc-biblioref" data-xml-rid="bib30 bib108 bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-1" aria-controls="sref112-1">Flegontov et&nbsp;al., 2019; Raghavan et&nbsp;al., 2014b; Rasmussen et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref112-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Flegontov, P. ∙ Altınışık, N.E. ∙ Changmai, P. ...</div><div><strong>Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:236-240</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1251-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85067075462" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1251-y&amp;pmid=31168094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Raghavan, M. ∙ DeGiorgio, M. ∙ Albrechtsen, A. ...</div><div><strong>The genetic prehistory of the New World Arctic</strong></div><div><em>Science.</em> 2014; <strong>345</strong></div><div>1255832–1255832</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1255832" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84907279761" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25170159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1255832&amp;pmid=25170159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Rasmussen, M. ∙ Li, Y. ∙ Lindgreen, S. ...</div><div><strong>Ancient human genome sequence of an extinct Palaeo-Eskimo</strong></div><div><em>Nature.</em> 2010; <strong>463</strong>:757-762</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08835" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-76749141381" target="_blank" aria-label="Open Scopus in new tab">Scopus (464)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20148029/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08835&amp;pmid=20148029" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although these waves of migration are generally linked to ancient Siberian populations, their origins in the context of the Siberian genetic history remain poorly understood. Further studies of the Siberian population history using ancient genomes are, therefore, critical for the better understanding of the formation of Native American populations.</div><div id="p0040" role="paragraph">Furthermore, the Neolithic to Bronze Age transition in Eurasia was marked by complex cultural and genetic changes facilitated by extensive population movements, though their impact in the Lake Baikal region is still unclear. Looking to the west, the Early Bronze Age groups from the Pontic-Caspian steppe associated with the Yamnaya complex spread both east and west along with their distinct genetic profile often referred to as “Steppe ancestry” (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib4 bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Allentoft et&nbsp;al., 2015; Haak et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Allentoft, M.E. ∙ Sikora, M. ∙ Sjögren, K.-G. ...</div><div><strong>Population genomics of Bronze Age Eurasia</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:167-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84930940396" target="_blank" aria-label="Open Scopus in new tab">Scopus (523)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26062507/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14507&amp;pmid=26062507" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (615)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The eastward expansion of this group is considered to be associated with the Early Bronze Age Afanasievo culture. However, the later Middle Bronze Age Okunevo-related population from the central steppe as well as the Late Bronze Age Khövsgöl-related population from the eastern steppe harbor only a limited proportion of Steppe ancestry (<span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib42 bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Jeong et&nbsp;al., 2018, 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jeong, C. ∙ Wilkin, S. ∙ Amgalantugs, T. ...</div><div><strong>Bronze Age population dynamics and the rise of dairy pastoralism on the eastern Eurasian steppe</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E11248-E11255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1813608115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85057166426" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30397125/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1813608115&amp;pmid=30397125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Jeong, C. ∙ Balanovsky, O. ∙ Lukianova, E. ...</div><div><strong>The genetic history of admixture across inner Eurasia</strong></div><div><em>Nat. Ecol. Evol.</em> 2019; <strong>3</strong>:966-976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-019-0878-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85065175002" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31036896/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-019-0878-2&amp;pmid=31036896" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Therefore, the effect of steppe migrations in eastern Eurasia, particularly the interactions of Bronze Age Baikal hunter-gatherers with the contemporaneous and geographically proximal Afanasievo population, is still largely unexplored.</div><div id="p0045" role="paragraph">In this study, we report 19 newly sequenced ancient hunter-gatherers from the Lake Baikal and its surrounding regions, spanning from the Upper Paleolithic to the Early Bronze Age. Their analyses alongside published data reveal the most deeply divergent ancestry that link Upper Paleolithic Siberians and the First Peoples of the Americas, and more clearly delineate the complex transition between Early Neolithic and Early Bronze Age populations in the Lake Baikal region. We also provide both human and pathogen genomic evidence demonstrating the influence of western Eurasian steppe populations in this region during the Early Bronze Age and discuss the genetic contribution of Lake Baikal hunter-gatherers to Siberian populations through time.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Ancient DNA Sequencing</h3><div id="p0050" role="paragraph">We generated genome-wide genotype data from 19 ancient humans, including one Upper Paleolithic individual (dated to 14,050–13,770 BP, see <span class="dropBlock reference-citations"><a id="body-ref-sref96-1" href="#" role="doc-biblioref" data-xml-rid="bib90 bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-1" aria-controls="sref96-1">Orlova, 1995; Pavlenok et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref96-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Orlova, L.A.</div><div><strong>Radiocarbon dating of archaeological sites of Siberia and the Far East</strong></div><div><div>Derevianko, A.P. ∙ Kholushkin, Y.P. (Editors)</div></div><div><strong>Methods of Natural Sciences in Archaeological Reconstructions</strong></div><div>Publishing House of the Institute of Archeology and Ethnography SB RAS, 1995; 207-232</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.A.OrlovaRadiocarbon+dating+of+archaeological+sites+of+Siberia+and+the+Far+EastA.P.DereviankoY.P.KholushkinMethods+of+Natural+Sciences+in+Archaeological+Reconstructions1995Publishing+House+of+the+Institute+of+Archeology+and+Ethnography+SB+RAS207232" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Pavlenok, G.D. ∙ Zubova, A.V.</div><div><strong>New Dental Finds Associated with the Paleolithic Selenga Culture, Western Trans-Baikal Region</strong></div><div><em>Archaeol. Ethnol. Anthropol. Eurasia.</em> 2019; <strong>47</strong>:3-11</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.17746/1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85076466976" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.17746%2F1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), four Early Neolithic individuals (7,320–6,500 BP), and 14 Late Neolithic to Early Bronze Age (LNBA) individuals (4,830–3,570 BP) from a total of 10 archaeological sites (<a id="crosref0145" href="#fig1">Figure&nbsp;1</a>; <a id="crosref0150" href="#mmc1">Table S1</a>). The radiocarbon date offsets caused by the local freshwater reservoir were estimated using the carbon and nitrogen isotopic values as described in previous studies on the same region (see <a id="crosref0155" href="#sec-4">STAR Methods</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref127-1" href="#" role="doc-biblioref" data-xml-rid="bib126 bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127-1" aria-controls="sref127-1">Schulting et&nbsp;al., 2014, 2015</a><div class="dropBlock__holder" data-db-target-of="sref127-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Schulting, R.J. ∙ Ramsey, C.B. ∙ Bazaliiskii, V.I. ...</div><div><strong>Freshwater Reservoir Offsets Investigated Through Paired Human-Faunal 14C Dating and Stable Carbon and Nitrogen Isotope Analysis at Lake Baikal, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:991-1008</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899833606" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Schulting, R.J. ∙ Bronk Ramsey, C. ∙ Bazaliiskii, V.I. ...</div><div><strong>Highly Variable Freshwater Reservoir Offsets Found along the Upper Lena Watershed, Cis-Baikal, Southeast Siberia</strong></div><div><em>Radiocarbon.</em> 2015; <strong>57</strong>:581-593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_rc.57.18458" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84942103801" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_rc.57.18458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We built single- and double-stranded DNA libraries from teeth or petrous portions of the temporal bone for the studied individuals, and shotgun sequencing revealed high levels of DNA preservation with endogenous DNA contents ranging from 0.12% to 50.54% (<a id="crosref0160" href="#mmc1">Table S1</a>). Subsequently, libraries were enriched for human DNA by SNP-capture targeting a set of 1.24 million variable sites (<span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Fu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (259)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and sequenced to mean coverage ranging from 0.04X to 2.07X. Pseudo-haploid genotypes were called on the targeted SNPs by randomly sampling a single allele at each position, with 34k to 886k SNPs covered by our samples. Additionally, we performed deep shotgun sequencing on eight individuals with high endogenous DNA levels (12%–51%), to achieve genomic coverage that ranged from 0.1X to 1.9X and refined their diploid genotypes by genotype likelihood-based imputation, resulting in 386k to 518k SNPs overlapping with the <i>Human Origins</i> dataset (<a id="crosref0170" href="#mmc1">Table S1</a>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/b5362c42-2e75-45e8-b53c-ed253fced4ef/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2020.04.037/asset/f3c5ec7d-3001-4f0d-acd0-0cf5f8457879/main.assets/gr1.jpg" height="671" width="753" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/b5362c42-2e75-45e8-b53c-ed253fced4ef/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Geographic Location, Time Period, and Genetic Profile of Studied Individuals</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Location of the 19 newly reported and published ancient individuals relevant in this study. Newly reported individuals are shown in outlined squares or circles.</div><div id="fspara0020" role="paragraph">(B) Ages of newly reported individuals from each site, with the x axis showing the median calibrated radiocarbon dates after correction for freshwater reservoir effect.</div><div id="fspara0025" role="paragraph">(C) PCA of Eurasian and Native American populations. The modern individuals are shown in light gray circles, with the name of several representative populations marking the positions of West Eurasians (Sardinian), East Asians (Ami and Uyghur), Siberians (Chukchi, Eskimo Naukan, Koryak, Nganasan, Selkup), and Native Americans (Chipewyan, Mixe, Karitiana). Ancient individuals are shown in colored symbols. The individual KPT005, which showed a significant shift toward west Eurasian populations, and individuals GLZ001&amp;GLZ002, which also represented outlier genetic profiles, are marked out with an arrow.</div><div id="fspara0030" role="paragraph">(D) Population clustering pattern of studied individuals together with representative modern and ancient populations, when K&nbsp;= 16. Most Early Neolithic to Bronze Age Lake Baikal region individuals are modeled as admixture of northeast Asian (dark red), ANE (dark blue), and Nganasan component (purple). The BZK002 individual has similar genetic profile as Okunevo, with a significantly larger ANE proportion compared with Baikal individuals, while the KPT005 individual shows a large component associated to WHG (light blue).</div><div id="fspara0035" role="paragraph">See also <a id="crosref1440" href="#figs1">Figure&nbsp;S1</a> and <a id="crosref1445" href="#mmc1">Table S1</a>.</div></div></div></div></figcaption></figure></div><div id="p0055" role="paragraph">We determined genetic sex by comparing the coverage on the sex chromosomes with the autosomal chromosomes, which revealed four females and 15 males. All individuals revealed low modern human DNA contamination at the mitochondrial level as well as through an estimation of X chromosomal heterozygosity on male individuals, except for KAG001 that showed 9.6% nuclear contamination (<a id="crosref0175" href="#mmc1">Table S1</a>). No kin relationship was found among these individuals. We finally intersected our genotypes with SNPs on the Affymetrix <i>Human Origins</i> array (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Lazaridis et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Lazaridis, I. ∙ Patterson, N. ∙ Mittnik, A. ...</div><div><strong>Ancient human genomes suggest three ancestral populations for present-day Europeans</strong></div><div><em>Nature.</em> 2014; <strong>513</strong>:409-413</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13673" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84907803675" target="_blank" aria-label="Open Scopus in new tab">Scopus (524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25230663/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13673&amp;pmid=25230663" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and combined with published genotype data from 3,014 present-day worldwide individuals and 453 ancient individuals for population genomic analysis (<a id="crosref0185" href="#mmc1">Table S1</a>).</div></section><section id="sec-2-2"><h3>Population Structure</h3><div id="p0060" role="paragraph">We first performed principal-component analysis (PCA) to understand the genetic background of the studied individuals, against modern Eurasian and Native American populations, and projected selected ancient individuals onto the PCs calculated with modern ones (<a id="crosref0190" href="#fig1">Figure&nbsp;1</a>C). Most of the Lake Baikal individuals occupied the space on a “ANE-NEA” cline running between “Northeast Asian” (NEA) ancestry represented by Neolithic hunter-gathers from the Devil’s Gate in the Russian Far East (<span class="dropBlock reference-citations"><a id="body-ref-sref130-1" href="#" role="doc-biblioref" data-xml-rid="bib129 bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130-1" aria-controls="sref130-1">Sikora et&nbsp;al., 2019; Siska et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref130-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Siska, V. ∙ Jones, E.R. ∙ Jeon, S. ...</div><div><strong>Genome-wide data from two early Neolithic East Asian individuals dating to 7700 years ago</strong></div><div><em>Sci. Adv.</em> 2017; <strong>3</strong>:e1601877</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1601877" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28164156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1601877&amp;pmid=28164156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and the ANE ancestry represented by Upper Paleolithic Siberian individuals MA1, AfontovaGora 2 (AG2), and AfontovaGora 3 (AG3) (<span class="dropBlock reference-citations"><a id="body-ref-sref107-2" href="#" role="doc-biblioref" data-xml-rid="bib33 bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-2" aria-controls="sref107-2">Fu et&nbsp;al., 2016; Raghavan et&nbsp;al., 2014a</a><div class="dropBlock__holder" data-db-target-of="sref107-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Fu, Q. ∙ Posth, C. ∙ Hajdinjak, M. ...</div><div><strong>The genetic history of Ice Age Europe</strong></div><div><em>Nature.</em> 2016; <strong>534</strong>:200-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84969820697" target="_blank" aria-label="Open Scopus in new tab">Scopus (302)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27135931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17993&amp;pmid=27135931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Raghavan, M. ∙ Skoglund, P. ∙ Graf, K.E. ...</div><div><strong>Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans</strong></div><div><em>Nature.</em> 2014; <strong>505</strong>:87-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12736" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84892367625" target="_blank" aria-label="Open Scopus in new tab">Scopus (334)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24256729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12736&amp;pmid=24256729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which was first described by <span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Damgaard et&nbsp;al. (2018a)</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Our newly sequenced Upper Paleolithic genome from the Ust-Kyakhta-3 site (UKY) just south to the Lake Baikal is placed close to the Mesolithic northeastern Siberian Kolyma individual (<span class="dropBlock reference-citations"><a id="body-ref-sref129-4" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129-4" aria-controls="sref129-4">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref129-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and is shifted toward Native American populations compared to the rest of the ancient Baikal individuals along PC2. All four Early Neolithic individuals cluster with published Early Neolithic groups from the same region (Shamanka_EN, Lokomotiv_EN, UstBelaya_EN) (<span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib19 bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Damgaard et&nbsp;al., 2018a; Flegontov et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Flegontov, P. ∙ Altınışık, N.E. ∙ Changmai, P. ...</div><div><strong>Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:236-240</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1251-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85067075462" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1251-y&amp;pmid=31168094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) designated as the “Baikal_EN” population. The LNBA individuals were divided into four groups. The major “Baikal_LNBA” group included 10 individuals and clustered with published Late Neolithic to Bronze Age Baikal populations (Shamanka_EBA, Kurma_EBA, UstIda_EBA, UstIda_LN, UstBelaya_BA). These individuals were positioned in PCA closer to ANE-related individuals compared with the Early Neolithic individuals from the same region, as well as closer to the Paleo-Eskimo Saqqaq individual (<span class="dropBlock reference-citations"><a id="body-ref-sref112-2" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-2" aria-controls="sref112-2">Rasmussen et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref112-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Rasmussen, M. ∙ Li, Y. ∙ Lindgreen, S. ...</div><div><strong>Ancient human genome sequence of an extinct Palaeo-Eskimo</strong></div><div><em>Nature.</em> 2010; <strong>463</strong>:757-762</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08835" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-76749141381" target="_blank" aria-label="Open Scopus in new tab">Scopus (464)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20148029/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08835&amp;pmid=20148029" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Another two individuals (GLZ001 and GLZ002) from the Glazkovskoe predmestie site, unlike the third individual from the same archaeological site (GLZ003), seemed shifted from the main cluster and showed closer genetic affinity to the Devil’s Gate and Early Neolithic Baikal individuals. One of the six individuals from the Kachug site (KPT005) was substantially displaced from the Baikal_LNBA group toward western Eurasians along PC1, not along the ANE-NEA cline but toward later Bronze Age populations, suggesting a potential introgression of the Steppe-related ancestry. Finally, an Early Bronze Age individual (BZK002) from the Bazaikha site in the Yenisei River region further to the west of the Lake Baikal was significantly displaced toward ANE-related individuals and located close to published Bronze Age individuals associated to the Okunevo culture (<span class="dropBlock reference-citations"><a id="body-ref-sref19-4" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-4" aria-controls="sref19-4">Damgaard et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref19-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0065" role="paragraph">Population clustering with ADMIXTURE based on worldwide populations also showed a similar clustering pattern. When selecting a K value of 16 (see <a id="crosref0215" href="#sec-4">STAR Methods</a>), the published and newly sequenced individuals belonging to main Early Neolithic to Bronze Age Baikal groups all showed genetic profiles composed of a mixture of three major components that were mostly enriched in ANE-related individuals, northeast Asians, and central Siberians represented by the Uralic-speaking Nganasan population (<a id="crosref0220" href="#fig1">Figure&nbsp;1</a>D). The ANE and central Siberian ancestries were both of higher proportion in most LNBA Baikal individuals than in the Early Neolithic ones, while GLZ001 and GLZ002 showed higher NEA ancestry, similar to the Early Neolithic population. The BZK002 individual presented a profile similar to the published Okunevo group (<span class="dropBlock reference-citations"><a id="body-ref-sref20-1" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-1" aria-controls="sref20-1">Damgaard et&nbsp;al., 2018b</a><div class="dropBlock__holder" data-db-target-of="sref20-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Damgaard, P.B. ∙ Marchi, N. ∙ Rasmussen, S. ...</div><div><strong>137 ancient human genomes from across the Eurasian steppes</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:369-374</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0094-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047137672" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0094-2&amp;pmid=29743675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with a much larger ANE component compared to other Lake Baikal individuals. The KPT005 individual also displayed a substantial contribution derived from European “Western Hunter-Gatherer” (WHG) ancestry, likely acquired through gene flow from the west.</div><div id="p0070" role="paragraph">We estimated the runs of homozygosity (ROH) of selected individuals together with published Baikal individuals (<a id="crosref0230" href="#mmc1">Table S1</a>) and did not identify an inbreeding signal in any individual. The Kolyma individual showed significantly more ROH compared with other individuals, suggesting a smaller population size in Mesolithic northeastern Siberia (<a id="crosref0235" href="#figs1">Figure&nbsp;S1</a>). The sharing of identity-by-descent (IBD) segments between individuals suggested a close relationship between UKY and Kolyma, supporting our analyses based on genome-wide SNP data, and also revealed that Early Neolithic and LNBA Baikal individuals shared genetic affinity with each other as well as with the older UKY and Kolyma genomes (<a id="crosref0240" href="#figs1">Figure&nbsp;S1</a>).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/d82ae6f7-aa7e-4398-9de7-1778cf898dc4/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2020.04.037/asset/20983171-5174-46af-bf6a-d14a7b8a597a/main.assets/figs1.jpg" height="841" width="691" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/d82ae6f7-aa7e-4398-9de7-1778cf898dc4/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Population Size and Relatedness of Lake Baikal Populations Revealed by ROH and IBD Segments, Related to <a id="crosref1545" href="#fig1">Figure&nbsp;1</a> and <a id="crosref1550" href="#mmc1">Table S1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0130" role="paragraph">This figure summarizes the accumulative ROH length detected in each individual (row 1), shared IBD segment length of individuals within population (row 2), and shared IBD segment length of UKY, Baikal_EN and Baikal_LNBA individuals with other population (row 3-5), respectively. The long segments (&gt; 8Mb) and short segments (&lt; 2Mb) are also summarized separately.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-3"><h3>Upper Paleolithic Baikal Ancestry Links with Non-Arctic Native Americans</h3><div id="p0075" role="paragraph">In the population structure analysis, we found the Upper Paleolithic UKY individual to be closely related with the northeastern Siberian Kolyma individual. This is further validated by outgroup <i>f</i><sub>3</sub> statistics (<a id="crosref0245" href="#fig2">Figure&nbsp;2</a>A) where, similarly to Kolyma, UKY showed close genetic affinity with Native American and Beringian populations (<a id="crosref0250" href="#fig2">Figure&nbsp;2</a>A). <i>F</i><sub>4</sub> statistics in the form of <i>f</i><sub>4</sub>(Mbuti, X; Kolyma, UKY) revealed that Kolyma is more closely related to populations from northeastern Siberia and North America compared with UKY (<a id="crosref0255" href="#figs2">Figure&nbsp;S2</a>). We further applied <i>f</i><sub>4</sub> statistics to explore the relationship of UKY and Kolyma with Native Americans and USR1 that was described as an outgroup to all non-Arctic Native Americans (<span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Moreno-Mayar et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Moreno-Mayar, J.V. ∙ Potter, B.A. ∙ Vinner, L. ...</div><div><strong>Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans</strong></div><div><em>Nature.</em> 2018; <strong>553</strong>:203-207</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85040467287" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29323294/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25173&amp;pmid=29323294" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Both UKY and Kolyma were symmetrically related with non-Arctic Native Americans and USR1, while USR1 shared significantly more genetic affinity with Native American populations compared to UKY and Kolyma (<a id="crosref0265" href="#figs2">Figure&nbsp;S2</a>; <a id="crosref0270" href="#mmc2">Table S2</a>).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/63026b29-4ac1-4614-8af0-437e85f2e07b/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2020.04.037/asset/0ef045e9-30d4-4452-a712-bbca7dd50a6b/main.assets/gr2.jpg" height="825" width="656" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/63026b29-4ac1-4614-8af0-437e85f2e07b/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Genetic Affinity between Upper Paleolithic UKY, Kolyma, and Native Americans</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0045" role="paragraph">(A) Genetic affinity between UKY and worldwide population assessed by <i>f</i><sub>3</sub>(Mbuti;X,UKY). The sampling location of UKY is shown with a green triangle. The 10 test populations with highest <i>f</i><sub>3</sub> values are shown in diamonds and other populations in circles.</div><div id="fspara0050" role="paragraph">(B) Graphic model of the relationship among UKY, Kolyma, and Native American populations. We first find the best fitted model with only UKY or only Kolyma as described in <a id="crosref1450" href="#figs2">Figure&nbsp;S2</a> and then add Kolyma on the selected model with UKY and choose the best model based on maximum <i>f</i>-statistics <i>Z</i> scores and final scores reported for each model. The lineages related with Native American population are colored orange, and the northeast Asian-related lineages are colored red.</div><div id="fspara0055" role="paragraph">See also <a id="crosref1455" href="#figs2">Figure&nbsp;S2</a> and <a id="crosref1460" href="#mmc2">Table S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/9defa4b1-fa0e-43e3-85a0-d60986db100e/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2020.04.037/asset/7bc69245-783e-4daa-a0b4-44ff5d18b6b6/main.assets/figs2.jpg" height="748" width="656" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/9defa4b1-fa0e-43e3-85a0-d60986db100e/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Relationship between UKY, Kolyma, and Modern-Day Populations Based on <i>f</i><sub>4</sub> Statistics and <i>qpGraph</i> Modeling, Related to <a id="crosref1555" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0140" role="paragraph">(A) This figure shows the different genetic affinities between UKY, Kolyma with worldwide population, assessed by <i>f</i><sub>4</sub>(Mbuti, X; Kolyma, UKY). The test populations with significant <i>f</i><sub>4</sub> values (|Z| &gt; 3) are shown in diamonds and other populations in circles.</div><div id="fspara0145" role="paragraph">(B) This figure shows the graphic modeling of UKY (left) and Kolyma (right) on the skeleton graph including Mbuti, AG3, Onge, Devil’s Gate, USR1, ASO and ESN described in <a id="crosref1560" href="#sec-4">STAR Methods</a>. The best fitted model for each individual is selected based on the maximum <i>f</i>-statistics Z scores and final scores reported for each model.</div></div></div></div></figcaption></figure></div><div id="p0080" role="paragraph">We also investigated their genetic composition using <i>qpAdm</i> modeling (<span class="dropBlock reference-citations"><a id="body-ref-sref38-3" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-3" aria-controls="sref38-3">Haak et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref38-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (615)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and found that both UKY and Kolyma possessed a similar level of ANE contribution, around 30%, when modeled as two-way mixture of Devil’s Gate (representing NEA ancestry) and AG3 (representing ANE ancestry) (<a id="crosref0280" href="#mmc3">Table S3</a>). Noticeably, this model did not fit well for both UKY (p&nbsp;= 1.45E-03) and Kolyma (p&nbsp;= 3.98E-08), as the Native American Karitiana population showed extra affinity with the tested individuals compared to the fitted model (<a id="crosref0285" href="#mmc3">Table S3</a>). This observation suggests that UKY and Kolyma shared a certain degree of genetic drift with Native American populations that occurred after the ancestors of Native Americans diverged from ANE and NEA ancestries.</div><div id="p0085" role="paragraph">We further explored the relationships among UKY, Kolyma, and ancient Native American groups using the graphic-based <i>qpGraph</i> modeling (<span class="dropBlock reference-citations"><a id="body-ref-sref116" href="#" role="doc-biblioref" data-xml-rid="bib92 bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116" aria-controls="sref116">Patterson et&nbsp;al., 2012; Reich et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref116"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (646)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Reich, D. ∙ Thangaraj, K. ∙ Patterson, N. ...</div><div><strong>Reconstructing Indian population history</strong></div><div><em>Nature.</em> 2009; <strong>461</strong>:489-494</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-70349472875" target="_blank" aria-label="Open Scopus in new tab">Scopus (824)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19779445/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08365&amp;pmid=19779445" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found that both UKY and Kolyma could be modeled as mixture between a northeast Asian lineage and a sister group of the Native American clade represented by USR1, Ancient Southwestern Ontario (ASO) individuals from Canada, and Early San Nicolas (ESN) individuals from the California Channel Islands in the USA (<span class="dropBlock reference-citations"><a id="body-ref-sref123-1" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-1" aria-controls="sref123-1">Scheib et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref123-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Scheib, C.L. ∙ Li, H. ∙ Desai, T. ...</div><div><strong>Ancient human parallel lineages within North America contributed to a coastal expansion</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:1024-1027</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar6851" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85048178453" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29853687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar6851&amp;pmid=29853687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0295" href="#figs2">Figure&nbsp;S2</a>). When UKY and Kolyma were included in the same graph, they were consistently modeled as descendants from two independent admixture events, with ancestral lineages of both deriving from the Native American-related and the northeast Asian-related clades (<a id="crosref0300" href="#fig2">Figure&nbsp;2</a>B). These findings confirm the close affinity of UKY and Kolyma to Native Americans but also highlight that both lineages contributing to UKY were ancestral to the groups contributing to Kolyma. In addition, aside from the first wave migrating into the Americas through Beringia, the admixture modeling suggests that the source of the Native American ancestry was more broadly spread across Siberia during the Upper Paleolithic, as UKY was found to be ∼4,000 years older and over 3,000&nbsp;km further to the southwest from Kolyma. In fact, our admixture graph indicates that this basal Native American group experienced multiple genetic contacts with northeast Asian populations giving rise to distinct ancient Siberian populations.</div></section><section id="sec-2-4"><h3>Complex Transition between the Early Neolithic and Bronze Age in the Lake Baikal Region</h3><div id="p0090" role="paragraph">A previous study described the transition between Early Neolithic and Bronze Age populations from the Lake Baikal region as the result of a discrete admixture event of ANE ancestry into the local gene pool (<span class="dropBlock reference-citations"><a id="body-ref-sref19-5" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-5" aria-controls="sref19-5">Damgaard et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref19-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this study, we combined the newly sequenced Baikal_EN and Baikal_LNBA individuals with published data from the same time period (<a id="crosref0310" href="#fig1">Figure&nbsp;1</a>C) and analyze these two combined datasets, Baikal_EN_all (n&nbsp;= 19) and Baikal_LNBA_all (n&nbsp;= 34), to better elucidate the genetic transition that occurred in this region. Prior to analyzing the combined groups, we confirmed the similarity between the new individuals and published groups using outgroup <i>f</i><sub>3</sub> statistics. Both Baikal_EN and Baikal_LNBA groups showed the highest genetic affinity with published Early Neolithic and LNBA Baikal populations, respectively (<a id="crosref0315" href="#mmc4">Table S4</a>).</div><div id="p0095" role="paragraph">From outgroup <i>f</i><sub>3</sub> statistics of the combined groups, we found both of the Baikal Early Neolithic and LNBA groups to be sharing high genetic affinity with ancient and modern northeast Asian and Siberian populations (<a id="crosref0320" href="#figs3">Figure&nbsp;S3</a>). The LNBA Baikal population also showed a high genetic affinity with the Paleo-Eskimo Saqqaq individual. Compared to their NEA proxy, they both carried extra genetic affinity with ANE-related populations while the LNBA population more so than the Early Neolithic population, as shown by <i>f</i><sub>4</sub> statistics (<a id="crosref0325" href="#figs3">Figure&nbsp;S3</a>). These results revealed the existence of ANE-related ancestry in the Early Neolithic population and, at the same time, validated the previous finding that an extra ANE ancestry gene flow is responsible for the genetic shift between Early Neolithic and Bronze Age Baikal populations.</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/113a2a61-115f-46df-9edb-64ff42a5b19c/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2020.04.037/asset/4075fab7-317c-445a-806d-eba8f732461d/main.assets/figs3.jpg" height="941" width="601" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/113a2a61-115f-46df-9edb-64ff42a5b19c/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Genetic Affinity of Combined Baikal Populations with Worldwide Population, Related to <a id="crosref1565" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0155" role="paragraph">The outgroup <i>f</i><sub>3</sub>-statistics in the form of (A) <i>f</i><sub>3</sub>(Mbuti, X; Baikal_EN_all) and (B) <i>f</i><sub>3</sub>(Mbuti, X; Baikal_LNBA_all) are applied to measure the genetic affinity of Early Neolithic and LNBA Baikal individuals with worldwide population. The ten population with highest <i>f</i><sub>3</sub> are shown in diamonds. Then (C) <i>f</i><sub>4</sub>(Mbuti, X; Devil’s Gate, Baikal_EN_all) and (D) <i>f</i><sub>4</sub>(Mbuti, X; Baikal_EN_all, Baikal_LNBA_all) are used to show the genetic difference between NEA ancestry, Early Neolithic Baikal population and LNBA Baikal population.</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">We further applied <i>qpAdm</i> modeling to quantify the proportion of ANE-related ancestry in Early Neolithic and LNBA Baikal populations, Saqqaq and Nganasan. Using Devil’s Gate as the NEA proxy, the Upper Paleolithic UKY was found to be a better fit than Kolyma as the ANE-related proxy for both Baikal populations, while AG3 provided a good fit for the Early Neolithic population (<a id="crosref0330" href="#mmc5">Table S5</a>). Using Devil’s Gate and AG3 as the two proxies of NEA and ANE ancestries, respectively, we estimated the ANE-related ancestry increasing from 14.3% in the Early Neolithic Baikal population to 22.7% in the LNBA population (<a id="crosref0335" href="#fig3">Figure&nbsp;3</a>A; <a id="crosref0340" href="#mmc3">Table S3</a>). Of note, the northeastern Siberian Kolyma individual could work as a sufficient ANE proxy for Saqqaq, as described in the study where this genome was first reported (<span class="dropBlock reference-citations"><a id="body-ref-sref129-5" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129-5" aria-controls="sref129-5">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref129-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) but did not provide a good fit for the Baikal populations and Nganasan. This suggests that the Baikal hunter-gatherer and Nganasan populations are more likely to have formed in central or southern Siberia while Paleo-Eskimo ancestry could have emerged in either central or northeastern Siberia.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/46e719e1-c622-46d7-b052-e1f3c0c7b729/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2020.04.037/asset/4f50b2ae-0dc4-4b33-ab05-f2172d2c1b68/main.assets/gr3.jpg" height="362" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/46e719e1-c622-46d7-b052-e1f3c0c7b729/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Genetic Modeling of Early Neolithic to Bronze Age Baikal Individuals and Admixture Dating</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0065" role="paragraph">(A) <i>qpAdm</i> modeling of Early Neolithic and LNBA Baikal populations, together with Nganasan and Saqqaq, as admixture between NEA ancestry represented by Devil’s Gate and different ANE ancestries. The error bars show the standard errors of estimated ancestry proportions. Details for the modeling are provided in <a id="crosref1465" href="#mmc3">Table S3</a>.</div><div id="fspara0070" role="paragraph">(B) Estimated dates of admixture events between ANE and NEA ancestries in Early Neolithic and LNBA Baikal population. The individual ages are the averages and standard deviations of median radiocarbon dates without correcting for freshwater reservoir effect, to be consistent with previously published individuals. The estimated admixture dates are calculated with generation time of 29 years, and the error bars show the sum of standard errors of DATES estimations and individual ages.</div><div id="fspara0075" role="paragraph">See also <a id="crosref1470" href="#figs3">Figures S3</a> and <a id="crosref1475" href="#figs4">S4</a> and <a id="crosref1480" href="#mmc3">Tables S3</a>, <a id="crosref1485" href="#mmc4">S4</a>, and <a id="crosref1490" href="#mmc5">S5</a>.</div></div></div></div></figcaption></figure></div><div id="p0105" role="paragraph">Furthermore, the program <i>DATES</i> was used to date the admixture events between ANE and NEA ancestries in the Baikal population based on the decay of ancestry covariance (<span class="dropBlock reference-citations"><a id="body-ref-sref158-1" href="#" role="doc-biblioref" data-xml-rid="bib158" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref158-1" aria-controls="sref158-1">Moorjani and Patterson, 2018</a><div class="dropBlock__holder" data-db-target-of="sref158-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Moorjani, P. ∙ Patterson, N.</div><div><a id="intref0250a" href="https://github.com/priyamoorjani/DATES" target="_blank">https://github.com/priyamoorjani/DATES</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MoorjaniN.Pattersonhttps%3A%2F%2Fgithub.com%2Fpriyamoorjani%2FDATES" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We detected a recent admixture signal in the Early Neolithic population, estimated to around 21 generations ago, while the admixture signal in LNBA population was dated to 71 generations ago, although this group harbored significantly more ANE-related ancestry (<a id="crosref0350" href="#mmc5">Table S5</a>). When considering the average radiocarbon date of each population and the standard errors of their admixture dates, we identified contiguous intervals for the admixture events that spanned ∼8,500–6,000 BP, considering a generation time of 29 years (<a id="crosref0355" href="#fig3">Figure&nbsp;3</a>B; <a id="crosref0360" href="#figs4">Figure&nbsp;S4</a>; <a id="crosref0365" href="#mmc1">Table S1</a>; <a id="crosref0370" href="#mmc5">Table S5</a>). Assuming a dating offset of 400–500 years due to freshwater reservoir effect estimated for the newly reported individuals, the admixture timings ranged between ∼8,000 and 5,500 BP. This suggests that both Baikal populations could have been formed through an extended admixture process between local groups and northeast Asian-related populations. The Early Neolithic groups were thus found to have experienced a prolonged admixture process, in contrast to the discrete and rather abrupt event suggested earlier (<span class="dropBlock reference-citations"><a id="body-ref-sref19-6" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-6" aria-controls="sref19-6">Damgaard et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref19-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This admixture, however, did not continue substantially in the Late Neolithic and Bronze Age, as suggested by the older admixture date for the LNBA population (<a id="crosref0380" href="#fig3">Figure&nbsp;3</a>B) and the relatively larger genetic variation among Early Neolithic individuals compared to the homogeneous LNBA cluster, as shown in the PCA plot (<a id="crosref0385" href="#fig1">Figure&nbsp;1</a>C).</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/7945273e-b457-4e20-8aa0-8c364af59b67/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2020.04.037/asset/3b13159e-698e-4288-9805-81698053401c/main.assets/figs4.jpg" height="862" width="753" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/7945273e-b457-4e20-8aa0-8c364af59b67/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Dating of the Admixture Events in Baikal, Okunevo Population, and the BZK002 Individual, Related to <a id="crosref1570" href="#fig3">Figures 3</a> and <a id="crosref1575" href="#fig5">5</a> and <a id="crosref1580" href="#mmc5">Table S5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0165" role="paragraph">This figure shows the <i>DATES</i> estimation of (A) time of admixture events in Early Neolithic and LNBA Baikal population and (B) time of admixture events in Okunevo population and BZK002 with different ancestor pairs. The red cross dots show the weighted ancestry covariance in different genetic distances, and the green curves show the exponential fitting starting at 0.5 cM. Details of the results are listed in <a id="crosref1585" href="#mmc5">Table S5</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-5"><h3>High Mobility in Bronze Age Siberia Revealed by Genetic Outliers</h3><div id="p0110" role="paragraph">Among the LNBA Baikal individuals, we identified three outlier individuals, which showed distinct genetic backgrounds from the major group. The genetic profile of the KPT005 individual suggested its affinity to western Eurasian populations inferred from PCA and ADMIXTURE results (<a id="crosref0390" href="#fig1">Figure&nbsp;1</a>). Using <i>qpAdm</i> modeling, we found that this individual could indeed be modeled as the mixture between the LNBA Baikal and multiple Bronze Age western steppe populations, with the Steppe ancestry contribution ranging between 42% and 48% (<a id="crosref0395" href="#fig4">Figure&nbsp;4</a>A; <a id="crosref0400" href="#mmc5">Table S5</a>). Considering geography and radiocarbon dates, we suggest the most likely candidate in our dataset for the source of admixture to be the Yamnaya-related Afanasievo population from nearby Altai-Sayan region.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/6df06745-7152-4de3-b331-0dac91a869c9/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2020.04.037/asset/340cc90d-1602-4577-9dfd-e596b7218561/main.assets/gr4.jpg" height="553" width="734" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/6df06745-7152-4de3-b331-0dac91a869c9/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Modeling of Genetic Outliers and the <i>Yersinia pestis</i> Phylogeny</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0085" role="paragraph">(A) <i>qpAdm</i> modeling of KPT005 as admixture between LNBA Baikal and steppe populations and GLZ001&amp;GLZ002 as admixture between different ANE ancestries and NEA. The error bars show the standard errors of estimated ancestry proportions. Details of the modeling are provided in <a id="crosref1495" href="#mmc3">Table S3</a>.</div><div id="fspara0090" role="paragraph">(B) Maximum likelihood phylogeny of <i>Y.&nbsp;pestis</i> genomes based on 4,368 SNPs, generated by applying a 99% partial deletion on the full SNP alignment. The newly reconstructed GLZ001 and GLZ002 genomes are analyzed alongside nine previously published Late Neolithic and Bronze Age genomes, two 6<sup>th</sup> century genomes from modern-day Germany and England associated with the first plague pandemic and a set of 14<sup>th</sup>-to-18<sup>th</sup>-century genomes from the second plague pandemic (ancient comparative dataset n&nbsp;= 28). In addition, they are analyzed alongside a worldwide modern dataset of 233 <i>Y.&nbsp;pestis</i> isolates (see full list; <span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Keller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Keller, M. ∙ Spyrou, M.A. ∙ Scheib, C.L. ...</div><div><strong>Ancient <i>Yersinia pestis</i> genomes from across Western Europe reveal early diversification during the First Pandemic (541–750)</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:12363-12372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1820447116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85067606359" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31164419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1820447116&amp;pmid=31164419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Branches of modern-day isolates were collapsed to enhance tree clarity. Nodes with ≥95 bootstrap support are indicated by asterisks (<sup>∗</sup>). Scale denotes substitutions per site.</div><div id="fspara0095" role="paragraph">(C) Overlapping posterior distributions of the Bayesian age estimates for GLZ001 and GLZ002, determined using BEAST v1.8. The median estimates for all tested models and their accompanying highest posterior densities (95%) are shown in <a id="crosref1505" href="#mmc6">Table S6</a>.</div><div id="fspara0100" role="paragraph">See also <a id="crosref1510" href="#figs5">Figures S5</a> and <a id="crosref1515" href="#figs6">S6</a> and <a id="crosref1520" href="#mmc3">Tables S3</a> and <a id="crosref1525" href="#mmc6">S6</a></div></div></div></div></figcaption></figure></div><div id="p0115" role="paragraph">Additionally, we also identified two Early Bronze Age Baikal individuals (GLZ001 and GLZ002) showing significantly lower ANE admixture levels compared to the major Baikal_LNBA group (<a id="crosref0405" href="#fig1">Figure&nbsp;1</a>C). We estimated the ANE contribution in these two individuals to be around 10%, using Devil’s Gate and AG3 as proxies of NEA and ANE ancestries, respectively (<a id="crosref0410" href="#fig4">Figure&nbsp;4</a>A; <a id="crosref0415" href="#mmc3">Table S3</a>). Furthermore, these two individuals both carried Y chromosome haplogroup C2b1, which was predominant in Early Neolithic Baikal populations (<a id="crosref0420" href="#mmc1">Table S1</a>), while the other LNBA individuals carried exclusively Q1a haplogroup. This could indicate either a relic population with ANE ancestry proportion similar to, or even less than, the Early Neolithic population preserved until the Bronze Age, or immigrants from surrounding regions carrying more NEA ancestry.</div><div id="p0120" role="paragraph">The mobility of the Baikal individuals during their lifetimes was thus investigated through the application of strontium isotope analysis to the enamel of eleven teeth samples (<span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib10 bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Bentley et&nbsp;al., 2004; Price et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bentley, R.A. ∙ Price, T.D. ∙ Stephan, E.</div><div><strong>Determining the ‘local’ 87Sr/86Sr range for archaeological skeletons: a case study from Neolithic Europe</strong></div><div><em>J.&nbsp;Archaeol. Sci.</em> 2004; <strong>31</strong>:365-375</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jas.2003.09.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-3042592719" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jas.2003.09.003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Price, T.D. ∙ Burton, J.H. ∙ Bentley, R.A.</div><div><strong>The Characterization of Biologically Available Strontium Isotope Ratios for the Study of Prehistoric Migration</strong></div><div><em>Archaeometry.</em> 2002; <strong>44</strong>:117-135</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/1475-4754.00047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0036462393" target="_blank" aria-label="Open Scopus in new tab">Scopus (496)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2F1475-4754.00047" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found that ten out of the eleven individuals all had values within the bioavailable <sup>87</sup>Sr/<sup>86</sup>Sr range (∼0.708–0.711) comprehensively reported for the Baikal region (<span class="dropBlock reference-citations"><a id="body-ref-sref39-1" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-1" aria-controls="sref39-1">Haverkort et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref39-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Haverkort, C.M. ∙ Bazaliiskii, V.I. ∙ Savel’ev, N.A.</div><div><strong>Identifying hunter-gatherer mobility patterns using strontium isotopes</strong></div><div><div>Weber, A. ∙ Katzenberg, A. ∙ Schurr, T.G. (Editors)</div></div><div><strong>Prehistoric Hunter-Gatherers of the Baikal Region, Siberia</strong></div><div>Penn Press, 2010; 217-238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.9783/9781934536391.217" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.9783%2F9781934536391.217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). By contrast, GLZ001 had a <sup>87</sup>Sr/<sup>86</sup>Sr value &gt;0.713 that suggested a non-local origin for this individual, followed by movement to the Angara River Valley region after the period of formation of the second molar (&gt;7 years of age) (<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Alexander Bentley, 2006</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Alexander Bentley, R.</div><div><strong>Strontium Isotopes from the Earth to the Archaeological Skeleton: A Review</strong></div><div><em>J.&nbsp;Archaeol. Method Theory.</em> 2006; <strong>13</strong>:135-187</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10816-006-9009-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-33749026059" target="_blank" aria-label="Open Scopus in new tab">Scopus (631)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10816-006-9009-x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Based on existing bioavailable strontium data, sources of origin plausibly include areas to the south and east of Lake Baikal on Archean and Proterozoic geology (<span class="dropBlock reference-citations"><a id="body-ref-sref39-2" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-2" aria-controls="sref39-2">Haverkort et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref39-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Haverkort, C.M. ∙ Bazaliiskii, V.I. ∙ Savel’ev, N.A.</div><div><strong>Identifying hunter-gatherer mobility patterns using strontium isotopes</strong></div><div><div>Weber, A. ∙ Katzenberg, A. ∙ Schurr, T.G. (Editors)</div></div><div><strong>Prehistoric Hunter-Gatherers of the Baikal Region, Siberia</strong></div><div>Penn Press, 2010; 217-238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.9783/9781934536391.217" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.9783%2F9781934536391.217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GLZ002 appears instead “local” to the Angara River Valley in terms of <sup>87</sup>Sr/<sup>86</sup>Sr, despite the genetic similarity to individual GLZ001. Though a speculation, it is possible that GLZ002 could have migrated to the region during childhood, prior to 7 years of age and, hence, prior to the formation of the second molar, or from a region with similar <sup>87</sup>Sr/<sup>86</sup>Sr range to the Angara River Valley.</div></section><section id="sec-2-6"><h3>Identification of <i>Y.&nbsp;pestis</i> Infections among Genetic Outlier Individuals</h3><div id="p0125" role="paragraph">Recent studies have reported evidence of <i>Yersinia pestis</i> infections in humans across multiple regions of Europe and central Asia between the Middle Neolithic and Bronze Age (∼4,900–3,500 BP) (<span class="dropBlock reference-citations"><a id="body-ref-sref133-1" href="#" role="doc-biblioref" data-xml-rid="bib6 bib111 bib115 bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133-1" aria-controls="sref133-1">Andrades Valtueña et&nbsp;al., 2017; Rascovan et&nbsp;al., 2019; Rasmussen et&nbsp;al., 2015b; Spyrou et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref133-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Rascovan, N. ∙ Sjögren, K.-G. ∙ Kristiansen, K. ...</div><div><strong>Emergence and Spread of Basal Lineages of Yersinia pestis during the Neolithic Decline</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:295-305</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2018.11.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2018.11.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85059906505" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30528431/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.11.005&amp;pmid=30528431" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Rasmussen, S. ∙ Allentoft, M.E. ∙ Nielsen, K. ...</div><div><strong>Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:571-582</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26496604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.10.009&amp;pmid=26496604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Spyrou, M.A. ∙ Tukhbatova, R.I. ∙ Wang, C.-C. ...</div><div><strong>Analysis of 3800-year-old Yersinia pestis genomes suggests Bronze Age origin for bubonic plague</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:2234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-04550-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85048310202" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29884871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-04550-9&amp;pmid=29884871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To date, the majority of identified strains group on an extinct phylogenetic lineage previously designated as the LNBA lineage (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Andrades Valtueña et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Such data have been interpreted alongside human population genetic frameworks, suggesting the spread of this lineage via Yamnaya-related human migrations across Eurasia during the Late Neolithic and Early Bronze Age (<span class="dropBlock reference-citations"><a id="body-ref-sref6-3" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-3" aria-controls="sref6-3">Andrades Valtueña et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0130" role="paragraph">To investigate the presence of pathogen DNA signatures among individuals from the Lake Baikal region, we screened all analyzed specimens (n&nbsp;= 19) using the pipeline HOPS (<span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Hübler et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Hübler, R. ∙ Key, F.M. ∙ Warinner, C. ...</div><div><strong>HOPS: automated detection and authentication of pathogen DNA in archaeological remains</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:280</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1903-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85076688072" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31842945/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1903-0&amp;pmid=31842945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our metagenomic analysis of shotgun sequenced data revealed evidence of ancient <i>Y.&nbsp;pestis</i> DNA in two Early Bronze Age individuals (GLZ001 and GLZ002) (<a id="crosref0455" href="#figs5">Figure&nbsp;S5</a>), which were dated to 4,556 and 4,430 BP (median dates), respectively, after accounting for the freshwater reservoir effect present in this region (<a id="crosref0460" href="#mmc1">Table S1</a>). Subsequent whole-<i>Y.&nbsp;pestis</i>-genome enrichment yielded a 7.2-and 12.8-fold genomic coverage for GLZ001 and GLZ002, respectively (<a id="crosref0465" href="#mmc6">Table S6</a>). We built a maximum likelihood phylogeny in order to compare the newly reconstructed genomes with previously published ancient and modern <i>Y.&nbsp;pestis</i> isolates (<a id="crosref0470" href="#fig4">Figure&nbsp;4</a>B). GLZ001 and GLZ002 grouped on the previously described LNBA <i>Y.&nbsp;pestis</i> lineage (<span class="dropBlock reference-citations"><a id="body-ref-sref115-2" href="#" role="doc-biblioref" data-xml-rid="bib6 bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-2" aria-controls="sref115-2">Andrades Valtueña et&nbsp;al., 2017; Rasmussen et&nbsp;al., 2015b</a><div class="dropBlock__holder" data-db-target-of="sref115-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Rasmussen, S. ∙ Allentoft, M.E. ∙ Nielsen, K. ...</div><div><strong>Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:571-582</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26496604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.10.009&amp;pmid=26496604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), together with 4,800-to-3,500-year-old isolates from across Eurasia (<a id="crosref0475" href="#fig4">Figure&nbsp;4</a>B). Specifically, they appeared most closely related (distance <i>d</i>&nbsp;= 17–19 SNPs) to a genome from the Baltic Sea region (Kunila2, 4,520–4,290 BP), retrieved from an individual culturally affiliated with the Corded Ware complex (<a id="crosref0480" href="#fig4">Figure&nbsp;4</a>B; <span class="dropBlock reference-citations"><a id="body-ref-sref6-5" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-5" aria-controls="sref6-5">Andrades Valtueña et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Such a result is intriguing since, based on human genetic analyses, individuals GLZ001 and GLZ002 appear as genetic outliers compared to other analyzed Bronze Age individuals from the same region. In addition, they lack Yamnaya-related Steppe ancestry (<a id="crosref0490" href="#fig1">Figure&nbsp;1</a>; <a id="crosref0495" href="#fig4">Figure&nbsp;4</a>A), common to all published plague victims with <i>Y.&nbsp;pestis</i> genomes grouping on the LNBA phylogenetic lineage (<span class="dropBlock reference-citations"><a id="body-ref-sref115-3" href="#" role="doc-biblioref" data-xml-rid="bib6 bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-3" aria-controls="sref115-3">Andrades Valtueña et&nbsp;al., 2017; Rasmussen et&nbsp;al., 2015b</a><div class="dropBlock__holder" data-db-target-of="sref115-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Rasmussen, S. ∙ Allentoft, M.E. ∙ Nielsen, K. ...</div><div><strong>Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:571-582</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26496604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.10.009&amp;pmid=26496604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/a0177473-6a74-421e-9bd6-59ee7fbed665/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2020.04.037/asset/a15d65a2-95dd-4b78-a9c2-1ef3bbc5fa64/main.assets/figs5.jpg" height="937" width="680" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/a0177473-6a74-421e-9bd6-59ee7fbed665/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Overview of <i>Y.&nbsp;pestis</i> Screening, Gene Content, and Temporal Signal, Related to <a id="crosref1590" href="#fig4">Figures 4</a> and <a id="crosref1595" href="#figs6">S6</a> and <a id="crosref1600" href="#mmc1">Tables S1</a> and <a id="crosref1605" href="#mmc6">S6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0175" role="paragraph">(A) <i>Y.&nbsp;pestis</i> screening results for samples GLZ001 and GLZ002. The histograms were calculated in HOPS and represent the edit distance of GLZ001 and GLZ002 MALT assigned reads to their closest matching reference within the <i>Y.&nbsp;pseudotuberculosis</i> complex. In addition, aDNA damage patterns were generated using MapDamage2.0, after BWA mapping of all reads against the <i>Y.&nbsp;pestis</i> reference genome CO92.</div><div id="fspara0180" role="paragraph">(B) Presence/absence analysis of virulence-associated and evolutionary-determinant genes across <i>Y.&nbsp;pestis</i> genomes. The genomic profiles of GLZ001 and GLZ002 were investigated by calculating the coverage across virulence-associated and evolutionary-determinant genes, in comparison to those from previously published ancient strains from the LNBA period (RISE509, RK1001, GEN72, Gyvakarai1, Kunila2, 1343UnTal85, 6Postillionstrasse, RISE505), the Late Bronze Age (RT5), as well as from the medieval and early modern periods (Altenerding 2148 and London BD 8124/8291/11972). In addition, representatives of modern lineages were also included for comparative purposes. The investigated genes were located on the <i>Y.&nbsp;pestis</i> chromosome and the pMT1, pPCP1 and pCD1 plasmids. The heatmap was constructed in R version 3.2.1 using the ggplot2 package.</div><div id="fspara0185" role="paragraph">(C) Calculating the temporal signal within the <i>Y.&nbsp;pestis</i> LNBA lineage. Regressions of root-to-tip genetic distance against specimen age were calculated using TempEst v1.5 for all published LNBA genomes using a maximum likelihood phylogeny of 374 overlapping variant positions for the analysis. The input ages were as follows: RISE509 (4729 BP), RK1.001 (4720 BP), GEN72 (4721 BP), Gyvakarai1 (4485 BP), Kunila2 (4427 BP), 1343UnTal (4203 BP), 6Post (3873 BP) and RISE505 (3635 BP). The left plot represents a linear regression of root-to-tip genetic distance against specimen age including a modern <i>Y.&nbsp;pestis</i> genome as outgroup (0.PE2 Pestoides F), whereas the right panel represents the regression including only previously published ancient genomes within the LNBA lineage (without the outgroup).</div></div></div></div></figcaption></figure></div><div id="p0135" role="paragraph">To further characterize the <i>Y.&nbsp;pestis</i> genomic contents of GLZ001 and GLZ002, we computed the coverage across previously defined virulence-associated and evolutionary-determinant genes (<span class="dropBlock reference-citations"><a id="body-ref-sref152-1" href="#" role="doc-biblioref" data-xml-rid="bib24 bib152" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref152-1" aria-controls="sref152-1">Demeure et&nbsp;al., 2019; Zhou and Yang, 2009</a><div class="dropBlock__holder" data-db-target-of="sref152-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Demeure, C.E. ∙ Dussurget, O. ∙ Mas Fiol, G. ...</div><div><strong>Yersinia pestis and plague: an updated view on evolution, virulence determinants, immune subversion, vaccination, and diagnostics</strong></div><div><em>Genes Immun.</em> 2019; <strong>20</strong>:357-370</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41435-019-0065-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85063771364" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30940874/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41435-019-0065-0&amp;pmid=30940874" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Zhou, D. ∙ Yang, R.</div><div><strong>Molecular Darwinian evolution of virulence in Yersinia pestis</strong></div><div><em>Infect. Immun.</em> 2009; <strong>77</strong>:2242-2250</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/IAI.01477-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-66549101505" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19289506/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FIAI.01477-08&amp;pmid=19289506" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our analysis revealed similar patterns to those previously observed in LNBA strains, which are consistent with a decreased efficiency in flea-mediated-transmission (<a id="crosref0500" href="#figs5">Figure&nbsp;S5</a>). These include the absence of <i>ymt</i> on the pMT1 plasmid and the presence of active PDE-2, PDE-3, <i>rcsA</i>, and <i>ureD</i> gene variants (<a id="crosref0505" href="#figs5">Figures S5</a> and <a id="crosref0510" href="#figs6">S6</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref136-1" href="#" role="doc-biblioref" data-xml-rid="bib6 bib115 bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136-1" aria-controls="sref136-1">Andrades Valtueña et&nbsp;al., 2017; Rasmussen et&nbsp;al., 2015b; Sun et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref136-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Rasmussen, S. ∙ Allentoft, M.E. ∙ Nielsen, K. ...</div><div><strong>Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:571-582</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26496604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.10.009&amp;pmid=26496604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Sun, Y.-C. ∙ Jarrett, C.O. ∙ Bosio, C.F. ...</div><div><strong>Retracing the evolutionary path that led to flea-borne transmission of Yersinia pestis</strong></div><div><em>Cell Host Microbe.</em> 2014; <strong>15</strong>:578-586</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2014.04.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2014.04.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24832452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2014.04.003&amp;pmid=24832452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally, we identified here the absence of <i>ympt1.66c</i> (<a id="crosref0515" href="#figs5">Figure&nbsp;S5</a>), a putative helicase considered to be involved in the bacterium’s initial within-macrophage survival and subsequent bubonic disease progression (<span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Pradel et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Pradel, E. ∙ Lemaître, N. ∙ Merchez, M. ...</div><div><strong>New insights into how Yersinia pestis adapts to its mammalian host during bubonic plague</strong></div><div><em>PLoS Pathog.</em> 2014; <strong>10</strong>:e1004029</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1004029" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84897393547" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24675805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1004029&amp;pmid=24675805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/8d3483f1-7c24-436a-a30a-c85e89d36bd9/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2020.04.037/asset/84abb332-ca65-44e6-bb12-80c5fbab8e2f/main.assets/figs6.jpg" height="760" width="767" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/8d3483f1-7c24-436a-a30a-c85e89d36bd9/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Visual Inspection of Functionally Informative <i>Y.&nbsp;pestis</i> SNPs in Virulence-Associated and Evolutionary-Determinant Genes, Related to <a id="crosref1610" href="#fig4">Figures 4</a> and <a id="crosref1615" href="#figs5">S5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0195" role="paragraph">Screenshots created in IGV (<span class="dropBlock reference-citations"><a id="body-ref-sref140-1" href="#" role="doc-biblioref" data-xml-rid="bib140" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref140-1" aria-controls="sref140-1">Thorvaldsdóttir et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref140-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Thorvaldsdóttir, H. ∙ Robinson, J.T. ∙ Mesirov, J.P.</div><div><strong>Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration</strong></div><div><em>Brief. Bioinform.</em> 2013; <strong>14</strong>:178-192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bib/bbs017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84875634162" target="_blank" aria-label="Open Scopus in new tab">Scopus (3379)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22517427/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbib%2Fbbs017&amp;pmid=22517427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for a virtualisation of virulence-associated genes affected by substitutions or InDels. Visualized genomic positions are as follows: (A) presence of a “T” at CO92 pPCP1 plasmid position 7500 suggests ancestral <i>pla</i> variant; (B) lack of “T” insertion at CO92 chromosomal position 1892659 suggest active <i>flhD</i> gene variant; (C) Presence of “G” at chromosomal <i>Y.&nbsp;pseudotuberculosis</i> IP32953 position 3944534 suggests ancestral (active) PDE-3 variant; (D) presence of “T” at chromosomal <i>Y.&nbsp;pseudotuberculosis</i> IP32953 position 3944166 suggests derived (pseudogenised) PDE-3-pe’variant; (E) lack of “T” insertion at CO92 chromosomal position 1434044 suggests active PDE-2 variant; (F) lack of “G” insertion at CO92 chromosomal position 2997296 suggests active <i>ureD</i> variant; (G) imprecise mappability of reads spanning a 30bp duplication within <i>rcsA</i> gene in GLZ001 and GLZ002 (mapped against the <i>rscA</i> [y1741] gene in KIM10), compared to the published ancient strain RT5 showing genetic evidence of flea adaptation (<span class="dropBlock reference-citations"><a id="body-ref-sref133-2" href="#" role="doc-biblioref" data-xml-rid="bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133-2" aria-controls="sref133-2">Spyrou et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref133-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Spyrou, M.A. ∙ Tukhbatova, R.I. ∙ Wang, C.-C. ...</div><div><strong>Analysis of 3800-year-old Yersinia pestis genomes suggests Bronze Age origin for bubonic plague</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:2234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-04550-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85048310202" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29884871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-04550-9&amp;pmid=29884871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); (H) exclusion of one 30bp duplicate from a constructed <i>rcsA</i> reference restores read mappability within this region in GLZ001 and GLZ002, suggesting an absence of the duplication in those genomes.</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph">Moreover, to investigate whether the radiocarbon date correction using stable isotope values could provide a reliable estimation of the freshwater reservoir offsets in GLZ001 and GLZ002, we explored an additional dating approach for these specimens based on the <i>Y.&nbsp;pestis</i> molecular phylogeny. For this, we first assessed the temporal phylogenetic signal across the LNBA lineage using previously published radiocarbon dates as tip calibrations (<span class="dropBlock reference-citations"><a id="body-ref-sref115-5" href="#" role="doc-biblioref" data-xml-rid="bib6 bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-5" aria-controls="sref115-5">Andrades Valtueña et&nbsp;al., 2017; Rasmussen et&nbsp;al., 2015b</a><div class="dropBlock__holder" data-db-target-of="sref115-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Rasmussen, S. ∙ Allentoft, M.E. ∙ Nielsen, K. ...</div><div><strong>Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:571-582</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26496604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.10.009&amp;pmid=26496604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The resulting correlation coefficient (<i>r</i>) and R<sup>2</sup> values were 0.88 and 0.78, respectively (<a id="crosref0525" href="#figs5">Figure&nbsp;S5</a>), which supported a linear relationship between root-to-tip genetic distance and specimen age for this dataset (including a modern 0.PE2 genome as outgroup). As such, we used the Bayesian framework BEAST (<span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">Drummond and Rambaut, 2007</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Drummond, A.J. ∙ Rambaut, A.</div><div><strong>BEAST: Bayesian evolutionary analysis by sampling trees</strong></div><div><em>BMC Evol. Biol.</em> 2007; <strong>7</strong>:214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-7-214" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-38649110663" target="_blank" aria-label="Open Scopus in new tab">Scopus (8871)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17996036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-7-214&amp;pmid=17996036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to independently re-estimate the tip dates of GLZ001 and GLZ002 and tested the coalescent constant size and coalescent skyline tree priors in combination with strict and lognormal relaxed clock models (see <a id="crosref0535" href="#sec-4">STAR Methods</a>). All runs produced overlapping estimates supporting median ages of ∼4,400 BP for both genomes (95% highest posterior density [HPD] range across all models: 4,115–4,590 BP) (<a id="crosref0540" href="#fig4">Figure&nbsp;4</a>C; <a id="crosref0545" href="#mmc6">Table S6</a>). As such, our analysis supports the estimated offset in the radiocarbon date estimates of both Glazkovskoe predmestie individuals, caused by the freshwater reservoir effect described previously for the Lake Baikal region (<span class="dropBlock reference-citations"><a id="body-ref-sref149" href="#" role="doc-biblioref" data-xml-rid="bib126 bib149" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref149" aria-controls="sref149">Schulting et&nbsp;al., 2014; Weber et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref149"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Schulting, R.J. ∙ Ramsey, C.B. ∙ Bazaliiskii, V.I. ...</div><div><strong>Freshwater Reservoir Offsets Investigated Through Paired Human-Faunal 14C Dating and Stable Carbon and Nitrogen Isotope Analysis at Lake Baikal, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:991-1008</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899833606" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">150.</div><div>Weber, A.W. ∙ Schulting, R.J. ∙ Bronk Ramsey, C. ...</div><div><strong>Chronology of middle Holocene hunter–gatherers in the Cis-Baikal region of Siberia: Corrections based on examination of the freshwater reservoir effect</strong></div><div><em>Quat. Int.</em> 2016; <strong>419</strong>:74-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2015.12.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84956468879" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2015.12.003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0145" role="paragraph">Taken together, these results provide evidence for the presence of human <i>Y.&nbsp;pestis</i> infections in the Lake Baikal region during the Early Bronze Age (<a id="crosref0550" href="#fig4">Figure&nbsp;4</a>B). To our knowledge, this is the easternmost appearance of <i>Y.&nbsp;pestis</i> strains associated with the LNBA lineage, despite the lack of Steppe-related ancestry in both affected individuals.</div></section><section id="sec-2-7"><h3>Genetic Influence on the Okunevo Culture</h3><div id="p0150" role="paragraph">The Okunevo is a Bronze Age culture of the central steppe, geographically located west of the Lake Baikal region (<a id="crosref0555" href="#fig1">Figure&nbsp;1</a>A). Previous studies have suggested a genetic relationship between Okunevo and ancient Lake Baikal individuals (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib20 bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Damgaard et&nbsp;al., 2018b; Jeong et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Damgaard, P.B. ∙ Marchi, N. ∙ Rasmussen, S. ...</div><div><strong>137 ancient human genomes from across the Eurasian steppes</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:369-374</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0094-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047137672" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0094-2&amp;pmid=29743675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Jeong, C. ∙ Balanovsky, O. ∙ Lukianova, E. ...</div><div><strong>The genetic history of admixture across inner Eurasia</strong></div><div><em>Nat. Ecol. Evol.</em> 2019; <strong>3</strong>:966-976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-019-0878-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85065175002" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31036896/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-019-0878-2&amp;pmid=31036896" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this study, we detected a high genetic affinity between the Early Bronze Age BZK002 genome from the region where the Okunevo culture was subsequently practiced and Okunevo-related individuals (<span class="dropBlock reference-citations"><a id="body-ref-sref19-7" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-7" aria-controls="sref19-7">Damgaard et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref19-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as revealed by PCA and outgroup <i>f</i><sub>3</sub> statistics (<a id="crosref0565" href="#fig1">Figure&nbsp;1</a>C; <a id="crosref0570" href="#mmc4">Table S4</a>). The BZK002 genome could be modeled with <i>qpAdm</i> as a two-way admixture between LNBA Baikal and AG3 or Botai-like populations, which are central Eurasian populations without any Steppe ancestry (<a id="crosref0575" href="#fig5">Figure&nbsp;5</a>A). BZK002 also performed as a good proxy for the populations that mixed with Yamnaya/Afanasievo-related groups to form the ancestry observed in Okunevo-associated individuals (<a id="crosref0580" href="#mmc3">Table S3</a>). Moreover, we verified the modeling of Okunevo as a three-way admixture among the LNBA Baikal group, AG3 or Botai-like individuals, and Steppe-related populations (<a id="crosref0585" href="#fig5">Figure&nbsp;5</a>A; <a id="crosref0590" href="#mmc3">Table S3</a>).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.04.037/asset/6a09fb12-3b6d-4e2d-987d-120f61a54b08/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2020.04.037/asset/11061e01-1bb2-4976-b68e-bd26a6841493/main.assets/gr5.jpg" height="323" width="301" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/6a09fb12-3b6d-4e2d-987d-120f61a54b08/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Genetic Relationship between BZK002 and Okunevo Population</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0110" role="paragraph">(A) <i>qpAdm</i> modeling of BZK002 as 2-way admixture between LNBA Baikal and Botai populations and Okunevo as 3-way admixture of LNBA Baikal, Botai, and Yamnaya. The error bars show the standard errors of estimated ancestry proportions. Details of the modeling are provided in <a id="crosref1530" href="#mmc3">Table S3</a>.</div><div id="fspara0115" role="paragraph">(B) Estimated admixture dates of LNBA Baikal, Botai, and Yamnaya ancestries in the Okunevo-related population and BZK002. The dashed lines show the median calibrated age of BZK002 and the average age of published Okunevo-related individuals. The shades show the 2-sigma range of BZK002 calibrated radiocarbon age and the range of Okunevo-related individual ages, respectively. The error bars of the estimated admixture dates show the cumulative range considering both standard errors of the admixture dating analysis and radiocarbon age ranges.</div><div id="fspara0120" role="paragraph">See also <a id="crosref1535" href="#figs4">Figure&nbsp;S4</a> and <a id="crosref1540" href="#mmc3">Table S3</a>.</div></div></div></div></figcaption></figure></div><div id="p0155" role="paragraph">Finally, we used <i>DATES</i> to estimate the timing of admixture events among those three ancestries in the Okunevo-related population, as well as to estimate the admixture timing of LNBA Baikal and Botai ancestries in the BZK002 individual. The estimated dates of these admixture events fell into two distinct time periods. Using LNBA Baikal and Botai-like populations as sources, the admixture events in Okunevo and BZK002 were dated to 42.9 and 23.4 generations ago, respectively, corresponding to an overlapping range of 6,000–5,000 BP, considering the specimens ages. The estimations including Yamnaya-related populations as one of the sources in Okunevo were instead both younger, around 17 generations, corresponding to 5,000–4,500 BP (<a id="crosref0595" href="#fig5">Figure&nbsp;5</a>B; <a id="crosref0600" href="#figs4">Figure&nbsp;S4</a>; <a id="crosref0605" href="#mmc5">Table S5</a>). Based on the dating results, we propose a scenario where the formation of the Okunevo-related gene pool resulted from an initial admixture of Botai-like and LNBA Baikal ancestries, followed by a gene flow of Yamnaya-related ancestry via its eastward expansion during the fifth millennium BP. BZK002 was directly radiocarbon dated to around 4,700 BP, predating the published Okunevo-related individuals by 200–800 years and overlapping with the estimated time range of the Steppe-ancestry admixture. Moreover, the Baikal-Botai admixture for BZK002 falls within the same period estimated for the Okunevo population, suggesting that this individual could represent an intermediate status during the formation of the Okunevo-associated genetic profile.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0160" role="paragraph">In this study, we demonstrated that multiple population turnovers took place in the Lake Baikal region from the Upper Paleolithic to the Bronze Age periods as a dynamic variation of ANE and NEA ancestry proportions. However, instead of the suggested complete replacement of the ANE-related ancestry by an East Asian gene pool (<span class="dropBlock reference-citations"><a id="body-ref-sref19-8" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-8" aria-controls="sref19-8">Damgaard et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref19-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we show that the ancestry first described in the 24,000-year-old MA1 individual survived in the region throughout the Upper Paleolithic until the Bronze Age, as revealed in the genome of the 14,000-year-old UKY individual as well as in the Early Neolithic population. Furthermore, the genetic transition from Early Neolithic to Bronze Age Baikal groups could be well explained by a prolonged gene flow between NEA- and ANE-related ancestries throughout the eighth to the sixth millennium BP (<a id="crosref0615" href="#fig3">Figure&nbsp;3</a>B), and the UKY genome was attested to be a better local source for the ANE ancestry than a possible southward expansion of Kolyma-related ancestry (<span class="dropBlock reference-citations"><a id="body-ref-sref129-6" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129-6" aria-controls="sref129-6">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref129-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In contrast, the Kolyma genome represents a good proxy for the Paleo-Eskimo Saqqaq genetic profile, supporting the view that the Paleo-Eskimo ancestry emerged around northeastern Siberia (<span class="dropBlock reference-citations"><a id="body-ref-sref30-3" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-3" aria-controls="sref30-3">Flegontov et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Flegontov, P. ∙ Altınışık, N.E. ∙ Changmai, P. ...</div><div><strong>Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:236-240</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1251-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85067075462" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1251-y&amp;pmid=31168094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), though we are unable to rule out the scenario that this ancestry formed in central or southern Siberia from an UKY-related gene pool.</div><div id="p0165" role="paragraph">Noticeably, we detect a strong genetic connection between Upper Paleolithic Siberians from the Lake Baikal region with non-Arctic Native Americans. According to our demographic modeling, Native Americans, the 14,000-year-old southern Siberian UKY individual and the 9,800-year-old northeastern Siberian Kolyma individual were all descendants, at least in part, of the same admixed population that carried both ANE and NEA ancestries. This population was likely widespread across Siberia during the Upper Paleolithic and experienced frequent genetic contacts with northeast Asian-related populations, which resulted in varying proportions of ANE/NEA ancestry through different areas and time periods. Considering the earlier population split of UKY compared with Kolyma and the Alaskan USR1 genome from the lineage leading to the rest of Native American populations, our result challenges the hypothesis that this basal Native American ancestry formed in northeastern Siberia (<span class="dropBlock reference-citations"><a id="body-ref-sref129-7" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129-7" aria-controls="sref129-7">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref129-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Further genetic evidence from Upper Paleolithic Siberian groups will be necessary to describe where and when exactly the ancestral gene pool of Native Americans came together.</div><div id="p0170" role="paragraph">Moreover, we provided evidence for the genetic contact between southern Siberia and the western steppe during the Early Bronze Age, based on both human and pathogen DNA data, which suggests a high mobility across Eurasia. The genetic influence of Yamnaya-related populations in the Lake Baikal region is evident by the presence of Steppe ancestry in individual KPT005 (<a id="crosref0635" href="#fig4">Figure&nbsp;4</a>A). In addition, previous studies have suggested that such migrations likely also facilitated the spread of the <i>Y.&nbsp;pestis</i> LNBA lineage across Eurasia (<span class="dropBlock reference-citations"><a id="body-ref-sref6-9" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-9" aria-controls="sref6-9">Andrades Valtueña et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as all individuals associated with this lineage carry Steppe ancestry (<span class="dropBlock reference-citations"><a id="body-ref-sref144" href="#" role="doc-biblioref" data-xml-rid="bib4 bib6 bib38 bib68 bib73 bib74 bib144" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref144" aria-controls="sref144">Allentoft et&nbsp;al., 2015; Andrades Valtueña et&nbsp;al., 2017; Haak et&nbsp;al., 2015; Mathieson et&nbsp;al., 2018; Mittnik et&nbsp;al., 2018, 2019; Wang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref144"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Allentoft, M.E. ∙ Sikora, M. ∙ Sjögren, K.-G. ...</div><div><strong>Population genomics of Bronze Age Eurasia</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:167-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84930940396" target="_blank" aria-label="Open Scopus in new tab">Scopus (523)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26062507/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14507&amp;pmid=26062507" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (615)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Mathieson, I. ∙ Alpaslan-Roodenberg, S. ∙ Posth, C. ...</div><div><strong>The genomic history of southeastern Europe</strong></div><div><em>Nature.</em> 2018; <strong>555</strong>:197-203</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25778" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85043332657" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29466330/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25778&amp;pmid=29466330" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Mittnik, A. ∙ Wang, C.C. ∙ Pfrengle, S. ...</div><div><strong>The genetic prehistory of the Baltic Sea region</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:1-11</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29317637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=29317637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Mittnik, A. ∙ Massy, K. ∙ Knipper, C. ...</div><div><strong>Kinship-based social inequality in Bronze Age Europe</strong></div><div><em>Science.</em> 2019; <strong>366</strong>:731-734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aax6219" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85074676786" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31601705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aax6219&amp;pmid=31601705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">145.</div><div>Wang, C.C. ∙ Reinhold, S. ∙ Kalmykov, A. ...</div><div><strong>Ancient human genome-wide data from a 3000-year interval in the Caucasus corresponds with eco-geographic regions</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:590</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-08220-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85061007525" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30713341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-08220-8&amp;pmid=30713341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this study, the two individuals shown to be infected with <i>Yersinia pestis</i> (GLZ001 and GLZ002) did not display genetic evidence of Yamnaya-related Steppe ancestry in their genomes but had substantially more NEA ancestry than all other Bronze Age individuals from the same and other surrounding sites (<a id="crosref0645" href="#fig1">Figure&nbsp;1</a>). Moreover, one of the two individuals (GLZ001) showed a non-local signal in strontium isotope analysis, supporting a homeland outside the Angara River Valley. Instead, based on the <i>Y.&nbsp;pestis</i> phylogeny, the newly reconstructed Baikal genomes were found to be genetically closest related to a strain from the Baltic region in northeastern Europe (<a id="crosref0650" href="#fig4">Figure&nbsp;4</a>B), isolated from an individual associated with the Corded Ware complex (<span class="dropBlock reference-citations"><a id="body-ref-sref73-2" href="#" role="doc-biblioref" data-xml-rid="bib6 bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-2" aria-controls="sref73-2">Andrades Valtueña et&nbsp;al., 2017; Mittnik et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref73-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Mittnik, A. ∙ Wang, C.C. ∙ Pfrengle, S. ...</div><div><strong>The genetic prehistory of the Baltic Sea region</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:1-11</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29317637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=29317637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although our current resolution is insufficient for inferring patterns of pathogen transmission between western Eurasia and southern Siberia, the presented data may rather be indicative of a century- or decade-long process that lead to the bacterium’s long-distance spread. Nevertheless, the phylogenetic topology of both Lake Baikal genomes within the LNBA lineage most parsimoniously suggests that <i>Y.&nbsp;pestis</i> spread into this region within the context of Yamnaya-related steppe expansions during the fifth millennium BP (<span class="dropBlock reference-citations"><a id="body-ref-sref6-12" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-12" aria-controls="sref6-12">Andrades Valtueña et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Importantly, our results provide prime evidence that strains of the LNBA <i>Y.&nbsp;pestis</i> lineage also affected individuals that were not genetically impacted by such migrations.</div><div id="p0175" role="paragraph">In conclusion, our study describes dynamic changes in the population structure of the Lake Baikal region and reveals a widespread occurrence of the genetic ancestry that gave rise to the First People of the Americas in Upper Paleolithic Siberia. In addition, we provide evidence of high human mobility across the Eurasian steppe revealing new insights about the spread of <i>Y.&nbsp;pestis</i> during the fifth millennium BP.</div></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key Resources Table</h3><div id="p0180" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Biological Samples</b></td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>ANG001</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>BZK002</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>GLZ001</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>GLZ002</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>GLZ003</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>IUO001</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>KAG001</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>KAG002</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>KPT001</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>KPT002</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>KPT003</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>KPT004</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>KPT005</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>KPT006</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>STB001</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>STB002</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>ZPL001</td></tr><tr><td>Ancient skeletal element</td><td>This study</td><td>ZPL002</td></tr><tr class="rowsep"><td>Ancient skeletal element</td><td>This study</td><td>UKY001</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td></tr><tr><td>D1000 ScreenTapes</td><td>Agilent Technologies</td><td>Cat# 5067-5582</td></tr><tr><td>D1000 Reagents</td><td>Agilent Technologies</td><td>Cat# 5067-5583</td></tr><tr><td>PfuTurbo Cx Hotstart DNA Polymerase</td><td>Agilent Technologies</td><td>Cat# 600412</td></tr><tr><td>Herculase II Fusion DNA Polymerase</td><td>Agilent Technologies</td><td>Cat# 600679</td></tr><tr><td>1x Tris-EDTA pH 8.0</td><td>AppliChem</td><td>Cat# A8569,0500</td></tr><tr><td>Sodiumhydroxide Pellets</td><td>Fisher Scientific</td><td>Cat# 10306200</td></tr><tr><td>Sera-Mag Magnetic Speed-beads Carboxylate-Modified (1&nbsp;mm, 3EDAC/PA5)</td><td>GE LifeScience</td><td>Cat# 65152105050250</td></tr><tr><td>0.5&nbsp;M EDTA pH 8.0</td><td>Life Technologies</td><td>Cat# AM9261</td></tr><tr><td>10x Buffer Tango</td><td>Life Technologies</td><td>Cat# BY5</td></tr><tr><td>GeneRuler Ultra Low Range DNA Ladder</td><td>Life Technologies</td><td>Cat# SM1211</td></tr><tr><td>Isopropanol</td><td>Merck</td><td>Cat# 1070222511</td></tr><tr><td>Ethanol</td><td>Merck</td><td>Cat# 1009832511</td></tr><tr><td>USER enzyme</td><td>New England Biolabs</td><td>Cat# M5505</td></tr><tr><td>Uracil Glycosylase inhibitor (UGI)</td><td>New England Biolabs</td><td>Cat# M0281</td></tr><tr><td>Bst DNA Polymerase2.0, large frag.</td><td>New England Biolabs</td><td>Cat# M0537</td></tr><tr><td>BSA 20mg/mL</td><td>New England Biolabs</td><td>Cat# B9000</td></tr><tr><td>T4 Polynucleotide Kinase</td><td>New England Biolabs</td><td>Cat# M0201</td></tr><tr><td>T4 DNA Polymerase</td><td>New England Biolabs</td><td>Cat# M0203</td></tr><tr><td>PEG-8000</td><td>Promega</td><td>Cat# V3011</td></tr><tr><td>20% SDS</td><td>Serva</td><td>Cat# 39575.01</td></tr><tr><td>Proteinase K</td><td>Sigma Aldrich</td><td>Cat# P2308</td></tr><tr><td>Guanidine hydrochloride</td><td>Sigma Aldrich</td><td>Cat# G3272</td></tr><tr><td>3M Sodium Acetate (pH 5.2)</td><td>Sigma Aldrich</td><td>Cat# S7899</td></tr><tr><td>Water</td><td>Sigma Aldrich</td><td>Cat# 34877</td></tr><tr><td>Tween-20</td><td>Sigma Aldrich</td><td>Cat# P9416</td></tr><tr><td>5M NaCl</td><td>Sigma Aldrich</td><td>Cat# S5150</td></tr><tr><td>Denhardt’s solution</td><td>Sigma Aldrich</td><td>Cat# D9905</td></tr><tr><td>ATP 100&nbsp;mM</td><td>Thermo Fisher Scientific</td><td>Cat# R0441</td></tr><tr><td>1&nbsp;M Tris-HCl pH 8.0</td><td>Thermo Fisher Scientific</td><td>Cat# 15568025</td></tr><tr><td>dNTP Mix</td><td>Thermo Fisher Scientific</td><td>Cat# R1121</td></tr><tr><td>SSC Buffer (20x)</td><td>Thermo Fisher Scientific</td><td>Cat# AM9770</td></tr><tr><td>GeneAmp 10x PCR Gold Buffer</td><td>Thermo Fisher Scientific</td><td>Cat# 4379874</td></tr><tr><td>Dynabeads MyOne Streptavidin T1</td><td>Thermo Fisher Scientific</td><td>Cat# 65602</td></tr><tr><td>Salmon sperm DNA</td><td>Thermo Fisher Scientific</td><td>Cat# 15632-011</td></tr><tr><td>Human Cot-I DNA</td><td>Thermo Fisher Scientific</td><td>Cat#15279011</td></tr><tr><td>0.5M HCl</td><td>Carl Roth</td><td>Cat# 9277.1</td></tr><tr class="rowsep"><td>HNO<sub>3</sub></td><td>Merck</td><td>Cat# 1.00456.2500</td></tr><tr class="rowsep"><td colspan="3"><b>Critical Commercial Assays</b></td></tr><tr><td>High Pure Viral Nucleic Acid Large Volume Kit</td><td>Roche</td><td>Cat# 5114403001</td></tr><tr><td>HiSeq 4000 SBS Kit (50/75 cycles)</td><td>Illumina</td><td>Cat# FC-410-1001/2</td></tr><tr><td>DyNAmo Flash SYBR Green qPCR Kit</td><td>Thermo Fisher Scientific</td><td>Cat# F415L</td></tr><tr><td>MinElute PCR Purification Kit</td><td>QIAGEN</td><td>Cat# 28006</td></tr><tr><td>Quick Ligation Kit</td><td>New England Biolabs</td><td>Cat# M2200L</td></tr><tr class="rowsep"><td>Oligo aCGH/Chip-on-Chip Hybridization Kit</td><td>Agilent Technologies</td><td>Cat# 5188-5220</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited Data</b></td></tr><tr><td>Raw and analyzed data (European nucleotide archive)</td><td>This study</td><td>ENA: PRJEB37007</td></tr><tr class="rowsep"><td>1240K Genotype data (Edmond Data Repository of Max Planck Society)</td><td>This study</td><td><a id="intref0010" href="https://edmond.mpdl.mpg.de/imeji/collection/z9C4DN1vFlqsmbHO" target="_blank">https://edmond.mpdl.mpg.de/imeji/collection/z9C4DN1vFlqsmbHO</a></td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>EAGER 1.92.55</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref97-1" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-1" aria-controls="sref97-1">Peltzer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref97-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Peltzer, A. ∙ Jäger, G. ∙ Herbig, A. ...</div><div><strong>EAGER: efficient ancient genome reconstruction</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:60</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0918-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27036623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0918-z&amp;pmid=27036623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0015" href="https://eager.readthedocs.io/en/latest/" target="_blank">https://eager.readthedocs.io/en/latest/</a></td></tr><tr><td>AdapterRemoval 2.2.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref124-1" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-1" aria-controls="sref124-1">Schubert et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref124-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Schubert, M. ∙ Lindgreen, S. ∙ Orlando, L.</div><div><strong>AdapterRemoval v2: rapid adapter trimming, identification, and read merging</strong></div><div><em>BMC Res. Notes.</em> 2016; <strong>9</strong>:88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13104-016-1900-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84958153152" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26868221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13104-016-1900-2&amp;pmid=26868221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0020" href="https://github.com/MikkelSchubert/adapterremoval" target="_blank">https://github.com/MikkelSchubert/adapterremoval</a></td></tr><tr><td>BWA 0.7.12</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (17717)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0025" href="http://bio-bwa.sourceforge.net/" target="_blank">http://bio-bwa.sourceforge.net/</a></td></tr><tr><td>DeDup 0.12.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref97-2" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-2" aria-controls="sref97-2">Peltzer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref97-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Peltzer, A. ∙ Jäger, G. ∙ Herbig, A. ...</div><div><strong>EAGER: efficient ancient genome reconstruction</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:60</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0918-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27036623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0918-z&amp;pmid=27036623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0030" href="https://github.com/apeltzer/DeDup" target="_blank">https://github.com/apeltzer/DeDup</a></td></tr><tr><td>mapDamage 2.0.6</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Jónsson et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Jónsson, H. ∙ Ginolhac, A. ∙ Schubert, M. ...</div><div><strong>mapDamage2.0: fast approximate Bayesian estimates of ancient DNA damage parameters</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1682-1684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84879910635" target="_blank" aria-label="Open Scopus in new tab">Scopus (398)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23613487/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt193&amp;pmid=23613487" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0035" href="https://github.com/ginolhac/mapDamage" target="_blank">https://github.com/ginolhac/mapDamage</a></td></tr><tr><td>bamUtil 1.0.13</td><td><a id="intref0040" href="https://github.com/statgen/bamUtil" target="_blank">https://github.com/statgen/bamUtil</a></td><td><a id="intref0045" href="https://github.com/statgen/bamUtil" target="_blank">https://github.com/statgen/bamUtil</a></td></tr><tr><td>CircularMapper</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref97-3" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-3" aria-controls="sref97-3">Peltzer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref97-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Peltzer, A. ∙ Jäger, G. ∙ Herbig, A. ...</div><div><strong>EAGER: efficient ancient genome reconstruction</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:60</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0918-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27036623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0918-z&amp;pmid=27036623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="https://github.com/apeltzer/CircularMapper" target="_blank">https://github.com/apeltzer/CircularMapper</a></td></tr><tr><td>Schmutzi</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref118-1" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-1" aria-controls="sref118-1">Renaud et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref118-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Renaud, G. ∙ Slon, V. ∙ Duggan, A.T. ...</div><div><strong>Schmutzi: estimation of contamination and endogenous mitochondrial consensus calling for ancient DNA</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0776-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84943738237" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26458810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0776-0&amp;pmid=26458810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0055" href="https://github.com/grenaud/schmutzi" target="_blank">https://github.com/grenaud/schmutzi</a></td></tr><tr><td>SAMtools 1.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Li et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The sequence alignment/map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (19724)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="http://www.htslib.org/doc/samtools.html" target="_blank">http://www.htslib.org/doc/samtools.html</a></td></tr><tr><td>pileupCaller</td><td><a id="intref0065" href="https://github.com/stschiff/sequenceTools" target="_blank">https://github.com/stschiff/sequenceTools</a></td><td><a id="intref0070" href="https://github.com/stschiff/sequenceTools" target="_blank">https://github.com/stschiff/sequenceTools</a></td></tr><tr><td>GATK 3.5</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">DePristo et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>DePristo, M.A. ∙ Banks, E. ∙ Poplin, R. ...</div><div><strong>A framework for variation discovery and genotyping using next-generation DNA sequencing data</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:491-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.806" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-79955483667" target="_blank" aria-label="Open Scopus in new tab">Scopus (5329)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21478889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.806&amp;pmid=21478889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="https://gatk.broadinstitute.org/hc/en-us" target="_blank">https://gatk.broadinstitute.org/hc/en-us</a></td></tr><tr><td>GeneImp 1.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref132-1" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132-1" aria-controls="sref132-1">Spiliopoulou et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref132-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Spiliopoulou, A. ∙ Colombo, M. ∙ Orchard, P. ...</div><div><strong>GeneImp: Fast imputation to large reference panels using genotype likelihoods from ultralow coverage sequencing</strong></div><div><em>Genetics.</em> 2017; <strong>206</strong>:91-104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.117.200063" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85020715759" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28348060/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.117.200063&amp;pmid=28348060" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0080" href="https://pm2.phs.ed.ac.uk/geneimp/" target="_blank">https://pm2.phs.ed.ac.uk/geneimp/</a></td></tr><tr><td>SHAPEIT v2.r837</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref23-1" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-1" aria-controls="sref23-1">Delaneau et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref23-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Delaneau, O. ∙ Zagury, J.-F. ∙ Marchini, J.</div><div><strong>Improved whole-chromosome phasing for disease and population genetic studies</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:5-6</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2307" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84871952176" target="_blank" aria-label="Open Scopus in new tab">Scopus (649)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23269371/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2307&amp;pmid=23269371" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0085" href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html" target="_blank">https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</a></td></tr><tr><td>yHaplo</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref100-1" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-1" aria-controls="sref100-1">Poznik, 2016</a><div class="dropBlock__holder" data-db-target-of="sref100-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Poznik, G.D.</div><div><strong>Identifying Y-chromosome haplogroups in arbitrarily large samples of sequenced or genotyped men</strong></div><div><em>bioRxiv.</em> 2016;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/088716" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F088716" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0090" href="https://github.com/23andMe/yhaplo" target="_blank">https://github.com/23andMe/yhaplo</a></td></tr><tr><td>ANGSD 0.910</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Korneliussen et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Korneliussen, T.S. ∙ Albrechtsen, A. ∙ Nielsen, R.</div><div><strong>ANGSD: Analysis of Next Generation Sequencing Data</strong></div><div><em>BMC Bioinformatics.</em> 2014; <strong>15</strong>:356</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-014-0356-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84920729412" target="_blank" aria-label="Open Scopus in new tab">Scopus (517)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25420514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-014-0356-4&amp;pmid=25420514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0095" href="http://www.popgen.dk/angsd/index.php/ANGSD" target="_blank">http://www.popgen.dk/angsd/index.php/ANGSD</a></td></tr><tr><td>EIGENSOFT 6.0.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref91-1" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-1" aria-controls="sref91-1">Patterson et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref91-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Patterson, N. ∙ Price, A.L. ∙ Reich, D.</div><div><strong>Population structure and eigenanalysis</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020190" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-33846006923" target="_blank" aria-label="Open Scopus in new tab">Scopus (2283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17194218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020190&amp;pmid=17194218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0100" href="https://github.com/DReichLab/EIG" target="_blank">https://github.com/DReichLab/EIG</a></td></tr><tr><td>ADMIXTURE 1.3.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Alexander et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alexander, D.H. ∙ Novembre, J. ∙ Lange, K.</div><div><strong>Fast model-based estimation of ancestry in unrelated individuals</strong></div><div><em>Genome Res.</em> 2009; <strong>19</strong>:1655-1664</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.094052.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-69749099417" target="_blank" aria-label="Open Scopus in new tab">Scopus (2279)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.094052.109&amp;pmid=19648217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0105" href="http://software.genetics.ucla.edu/admixture/" target="_blank">http://software.genetics.ucla.edu/admixture/</a></td></tr><tr><td>PLINK v1.90b3.29</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref104-1" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-1" aria-controls="sref104-1">Purcell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref104-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (14939)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0110" href="http://www.cog-genomics.org/plink2/" target="_blank">http://www.cog-genomics.org/plink2/</a></td></tr><tr><td>BEAGLE v4.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Browning and Browning, 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Browning, B.L. ∙ Browning, S.R.</div><div><strong>Improving the accuracy and efficiency of identity-by-descent detection in population data</strong></div><div><em>Genetics.</em> 2013; <strong>194</strong>:459-471</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.113.150029" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84878589007" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23535385/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.113.150029&amp;pmid=23535385" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0115" href="https://faculty.washington.edu/browning/beagle/b4_1.html" target="_blank">https://faculty.washington.edu/browning/beagle/b4_1.html</a></td></tr><tr><td>ADMIXTOOLS 5.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref92-2" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-2" aria-controls="sref92-2">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref92-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (646)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0120" href="https://github.com/DReichLab/AdmixTools" target="_blank">https://github.com/DReichLab/AdmixTools</a></td></tr><tr><td>DATES 763</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref158-2" href="#" role="doc-biblioref" data-xml-rid="bib158" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref158-2" aria-controls="sref158-2">Moorjani and Patterson, 2018</a><div class="dropBlock__holder" data-db-target-of="sref158-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Moorjani, P. ∙ Patterson, N.</div><div><a id="intref0250a" href="https://github.com/priyamoorjani/DATES" target="_blank">https://github.com/priyamoorjani/DATES</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MoorjaniN.Pattersonhttps%3A%2F%2Fgithub.com%2Fpriyamoorjani%2FDATES" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0125" href="https://github.com/priyamoorjani/DATES" target="_blank">https://github.com/priyamoorjani/DATES</a></td></tr><tr><td>MALT 0.4.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref141-1" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-1" aria-controls="sref141-1">Vågene et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref141-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">142.</div><div>Vågene, Å.J. ∙ Herbig, A. ∙ Campana, M.G. ...</div><div><strong>Salmonella enterica genomes from victims of a major sixteenth-century epidemic in Mexico</strong></div><div><em>Nat. Ecol. Evol.</em> 2018; <strong>2</strong>:520-528</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-017-0446-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85040669654" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29335577/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-017-0446-6&amp;pmid=29335577" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0130" href="https://github.com/husonlab/malt" target="_blank">https://github.com/husonlab/malt</a></td></tr><tr><td>HOPS</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Hübler et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Hübler, R. ∙ Key, F.M. ∙ Warinner, C. ...</div><div><strong>HOPS: automated detection and authentication of pathogen DNA in archaeological remains</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:280</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1903-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85076688072" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31842945/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1903-0&amp;pmid=31842945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0135" href="https://github.com/rhuebler/HOPS" target="_blank">https://github.com/rhuebler/HOPS</a></td></tr><tr><td>picard MarkDuplicates 1.1.40</td><td><a id="intref0140" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a></td><td><a id="intref0145" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a></td></tr><tr><td>MultiVCFAnalyzer v0.85</td><td><a id="intref0150" href="https://github.com/alexherbig/MultiVCFAnalyzer" target="_blank">https://github.com/alexherbig/MultiVCFAnalyzer</a></td><td><a id="intref0155" href="https://github.com/alexherbig/MultiVCFAnalyzer" target="_blank">https://github.com/alexherbig/MultiVCFAnalyzer</a></td></tr><tr><td>RaxML 8.2.9</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref135-1" href="#" role="doc-biblioref" data-xml-rid="bib135" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref135-1" aria-controls="sref135-1">Stamatakis, 2014</a><div class="dropBlock__holder" data-db-target-of="sref135-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Stamatakis, A.</div><div><strong>RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:1312-1313</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu033" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899553363" target="_blank" aria-label="Open Scopus in new tab">Scopus (11412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24451623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu033&amp;pmid=24451623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0160" href="https://cme.h-its.org/exelixis/web/software/raxml/index.html" target="_blank">https://cme.h-its.org/exelixis/web/software/raxml/index.html</a></td></tr><tr><td>TempEst v1.5</td><td><a id="intref0165" href="http://tree.bio.ed.ac.uk/software/tempest/" target="_blank">http://tree.bio.ed.ac.uk/software/tempest/</a></td><td><a id="intref0170" href="http://tree.bio.ed.ac.uk/software/tempest/" target="_blank">http://tree.bio.ed.ac.uk/software/tempest/</a></td></tr><tr><td>BEAST v1.8</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">Drummond and Rambaut, 2007</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Drummond, A.J. ∙ Rambaut, A.</div><div><strong>BEAST: Bayesian evolutionary analysis by sampling trees</strong></div><div><em>BMC Evol. Biol.</em> 2007; <strong>7</strong>:214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-7-214" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-38649110663" target="_blank" aria-label="Open Scopus in new tab">Scopus (8871)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17996036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-7-214&amp;pmid=17996036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0175" href="http://beast.community/" target="_blank">http://beast.community/</a></td></tr><tr><td>Tracer v1.6</td><td><a id="intref0180" href="http://tree.bio.ed.ac.uk/software/tracer" target="_blank">http://tree.bio.ed.ac.uk/software/tracer</a></td><td><a id="intref0185" href="http://tree.bio.ed.ac.uk/software/tracer/" target="_blank">http://tree.bio.ed.ac.uk/software/tracer/</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2820%2930502-X" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource Availability</h3><section id="sec-4-2-1"><h4>Lead Contact</h4><div id="p0185" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Johannes Krause (<a id="intref0190" href="mailto:krause@shh.mpg.de">krause@shh.mpg.de</a>).</div></section><section id="sec-4-2-2"><h4>Materials Availability</h4><div id="p0190" role="paragraph">This study did not generate new unique reagents.</div></section><section id="sec-4-2-3"><h4>Data and Code Availability</h4><div id="p0195" role="paragraph">Aligned reads to the human reference genome and mtDNA from the 19 newly reported ancient individuals and the <i>Y.&nbsp;pestis</i> capture sequencing reads of two individuals are available at the ENA archive under the accession number PRJEB37007. Haploid genotype data for the 1240K panel are available in the eigenstrat format at the Edmond Data Repository of Max Planck Society (<a id="intref0195" href="https://edmond.mpdl.mpg.de/imeji/collection/z9C4DN1vFlqsmbHO" target="_blank">https://edmond.mpdl.mpg.de/imeji/collection/z9C4DN1vFlqsmbHO</a>)</div></section></section><section id="sec-4-3"><h3>Experimental Model and Subject Details</h3><section id="sec-4-3-1"><h4>Archaeological information</h4><div id="p0200" role="paragraph">In this study we collected 7 petrous bones and 13 teeth from 19 individuals excavated from 10 different sites, including one Ust-Kyakhta-3 site from the south of Lake Baikal and the other nine sites in <i>cis</i>-Baikal region.</div><section id="sec-4-3-1-1"><h5>Sample from the Western Trans-Baikal</h5><section id="sec-4-3-1-1-1"><h6>Ust-Kyakhta-3 Site</h6><div id="p0205" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0210" role="paragraph">UKY001, tooth fragment (Ust’Kyakhta-3 1976 Excavation area 1 Layer 1 Square B-3)</div></div></div></div></div><div id="p0215" role="paragraph">The stratified Ust-Kyakhta-3 site is located on the right bank of the Selenga River in the vicinity of Ust-Kyakhta village in the Kyakhtinski Region of the Republic of Buryatia (Russian Federation). The site was discovered in 1947 by an archaeological team led by Academician A. P. Okladnikov. Initial excavations at Ust-Kyakhta-3 were carried out in 1976 and 1978, revealing two culture-bearing horizons. More than 40,000 stone artifacts were recovered; abundant faunal remains were also collected (<span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib87 bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Okladnikov, 1977, 1979</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Okladnikov, A.P.</div><div><strong>Otchet ob issledovanii paleoliticheskogo poseleniia Ust’-Kyakhta v 1976 g</strong></div><div><strong>Archive of the Institut Arkheologii i Etnografii Sibirskogo Otdeleniia Rossiiskoi Akademii Nauk</strong></div><div>Novosibirsk, 1977; 78</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovOtchet+ob+issledovanii+paleoliticheskogo+poseleniia+Ust%E2%80%99-Kyakhta+v+1976+gArchive+of+the+Institut+Arkheologii+i+Etnografii+Sibirskogo+Otdeleniia+Rossiiskoi+Akademii+Nauk1977Novosibirsk78" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Okladnikov, A.P.</div><div><strong>Nauchnyi otchet o raskopkakh stoianki Ust’-Kyakhta 1 (Kyakhtinskii raion BurASSR) v 1978 g</strong></div><div><strong>Archive of the Institut Arkheologii i Etnografii Sibirskogo Otdeleniia Rossiiskoi Akademii Nauk</strong></div><div>Novosibirsk, 1979; 21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNauchnyi+otchet+o+raskopkakh+stoianki+Ust%E2%80%99-Kyakhta+1+%28Kyakhtinskii+raion+BurASSR%29+v+1978+gArchive+of+the+Institut+Arkheologii+i+Etnografii+Sibirskogo+Otdeleniia+Rossiiskoi+Akademii+Nauk1979Novosibirsk21" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0220" role="paragraph">Archaeological work undertaken at the site in 2012 (<span class="dropBlock reference-citations"><a id="body-ref-sref94-1" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-1" aria-controls="sref94-1">Pavlenok, 2015a</a><div class="dropBlock__holder" data-db-target-of="sref94-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Pavlenok, G.D.</div><div><strong>Tekhnologiya obrabotki kamnya v selenginskoi kulture Zapadnogo Zabaikaliya (po materialam stoyanki Ust-Kyakhta-3)</strong></div><div>Altai State University, 2015</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.D.PavlenokTekhnologiya+obrabotki+kamnya+v+selenginskoi+kulture+Zapadnogo+Zabaikaliya+%28po+materialam+stoyanki+Ust-Kyakhta-3%292015Altai+State+University" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) yielded two human tooth fragments among archaeological materials collected in the course of sieving the sediments of Layer 1 (<span class="dropBlock reference-citations"><a id="body-ref-sref96-2" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-2" aria-controls="sref96-2">Pavlenok et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref96-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Pavlenok, G.D. ∙ Zubova, A.V.</div><div><strong>New Dental Finds Associated with the Paleolithic Selenga Culture, Western Trans-Baikal Region</strong></div><div><em>Archaeol. Ethnol. Anthropol. Eurasia.</em> 2019; <strong>47</strong>:3-11</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.17746/1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85076466976" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.17746%2F1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0225" role="paragraph">Excavations at Ust-Kyakhta-3 in 2012 provided additional information on the stratigraphy of the site. The stratigraphic column was subdivided into 12 lithological strata. Strata from 1 to 5 represent diluvial and aeolian sediments. The lower segment of the column, including Strata 6 through 12, was formed by flood-plain alluvium. Archaeological materials were noted in association with two lithological&nbsp;strata. Archaeological Layer 1 is enclosed within lithological Stratum 9, comprising a humus-containing fine-grained sandy loam from 4 to 12&nbsp;cm thick. Archaeological Layer 2 is associated with lithological Stratum 11, composed of light brown sandy loam from 15 to 20&nbsp;cm thick.</div><div id="p0230" role="paragraph">Cultural remains associated with Ust-Kyakhta-3 Layer 1 were accumulated during the terminal Pleistocene with associated dates of 14,165 ± 236 – 14,183 ± 234 cal BP. Archaeological materials from Layer 2 yielded similar dates ranging from 13,758 ± 138 – 14,326 ± 266 cal BP (calibration was carried out following <a id="intref0200" href="http://www.calpal-online.de/" target="_blank">http://www.calpal-online.de/</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref96-3" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-3" aria-controls="sref96-3">Pavlenok et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref96-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Pavlenok, G.D. ∙ Zubova, A.V.</div><div><strong>New Dental Finds Associated with the Paleolithic Selenga Culture, Western Trans-Baikal Region</strong></div><div><em>Archaeol. Ethnol. Anthropol. Eurasia.</em> 2019; <strong>47</strong>:3-11</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.17746/1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85076466976" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.17746%2F1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0235" role="paragraph">The lithic industry is characterized by flaking technology of typical wedge-shaped cores aimed at the production of micro-blades; blades were produced through flaking flat and sub-prismatic cores. The archaeological assemblage from Layer 1 is larger and contains a greater variety of various typologically and technologically significant artifacts. All the major inferences concerning stone reduction at Ust-Kyakhta-3 are based on the collection from Layer 1. The materials from Layer 2 do not contradict these inferences. One specific feature of this culture is a special strategy of core reduction. Upon exhaustion of the flaking surface, some flat cores were modified into blanks for wedge-shaped cores together with pebbles and thick spalls. Two modification strategies were reconstructed (<span class="dropBlock reference-citations"><a id="body-ref-sref137" href="#" role="doc-biblioref" data-xml-rid="bib95 bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137" aria-controls="sref137">Pavlenok, 2015b; Tashak, 2000</a><div class="dropBlock__holder" data-db-target-of="sref137"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Pavlenok, G.D.</div><div><strong>Tekhnologiya izgotovleniya klinovidnyh nukleusov v selenginskoj kul’ture Zapadnogo Zabajkal’ya (po materialam stoyanki Ust’-Kyahta-3). Izv. Altaiskogo Gos. Univ. Istor. Nauk. i Arkheologiia [Bulletin Altai State Univ</strong></div><div><em>Hist. Sci. Archaeol.</em> 2015; <strong>87</strong>:178-184</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.D.PavlenokTekhnologiya+izgotovleniya+klinovidnyh+nukleusov+v+selenginskoj+kul%E2%80%99ture+Zapadnogo+Zabajkal%E2%80%99ya+%28po+materialam+stoyanki+Ust%E2%80%99-Kyahta-3%29.+Izv.+Altaiskogo+Gos.+Univ.+Istor.+Nauk.+i+Arkheologiia+%5BBulletin+Altai+State+UnivHist.+Sci.+Archaeol.872015178184" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">138.</div><div>Tashak, V.I.</div><div><strong>Tortsovye klinovidnye nukleusy Zapadnogo Zabaikaliya v pozdnem paleolite i mezolite</strong></div><div><strong>Kamennyi Vek Yuzhnoi Sibiri i Mongolii: Teoreticheskie Problemy I Novye Otkrytiya</strong></div><div>Buryat scientific center SB RAS, 2000; 59-74</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.I.TashakTortsovye+klinovidnye+nukleusy+Zapadnogo+Zabaikaliya+v+pozdnem+paleolite+i+mezoliteKamennyi+Vek+Yuzhnoi+Sibiri+i+Mongolii%3A+Teoreticheskie+Problemy+I+Novye+Otkrytiya2000Buryat+scientific+center+SB+RAS5974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Analyses of the tool kit have shown that the targets of primary reduction; blades and micro-blades; were primarily selected for secondary working. The tool kit is dominated by knives, side- and end-scrapers and specific Ust-Kyakhta points (<span class="dropBlock reference-citations"><a id="body-ref-sref138-1" href="#" role="doc-biblioref" data-xml-rid="bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138-1" aria-controls="sref138-1">Tashak, 2005</a><div class="dropBlock__holder" data-db-target-of="sref138-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Tashak, V.I.</div><div><strong>Paleoliticheskie i mezoliticheskie pamiatniki Ust’-Kiakhty</strong></div><div>Buryatia Scientific Center of the Siberian Branch of the Russian Academy of Sciences, 2005</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.I.TashakPaleoliticheskie+i+mezoliticheskie+pamiatniki+Ust%E2%80%99-Kiakhty2005Buryatia+Scientific+Center+of+the+Siberian+Branch+of+the+Russian+Academy+of+Sciences" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); burins fashioned mostly on blades are less numerous. Borers made on micro-blades (<span class="dropBlock reference-citations"><a id="body-ref-sref94-2" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-2" aria-controls="sref94-2">Pavlenok, 2015a</a><div class="dropBlock__holder" data-db-target-of="sref94-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Pavlenok, G.D.</div><div><strong>Tekhnologiya obrabotki kamnya v selenginskoi kulture Zapadnogo Zabaikaliya (po materialam stoyanki Ust-Kyakhta-3)</strong></div><div>Altai State University, 2015</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.D.PavlenokTekhnologiya+obrabotki+kamnya+v+selenginskoi+kulture+Zapadnogo+Zabaikaliya+%28po+materialam+stoyanki+Ust-Kyakhta-3%292015Altai+State+University" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) constitute a characteristic series in the tool kit.</div><div id="p0240" role="paragraph">In addition to lithic artifacts, bone implements were also recovered, including points, awls, needles, hafts of composite tools with one or two cutting edges, fishhooks (<span class="dropBlock reference-citations"><a id="body-ref-sref138-2" href="#" role="doc-biblioref" data-xml-rid="bib93 bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138-2" aria-controls="sref138-2">Pavlenok, 2014; Tashak, 2005</a><div class="dropBlock__holder" data-db-target-of="sref138-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Pavlenok, G.D.</div><div><strong>Kostyanaya industriya stoyanki Ust’-Kyahta-3 (Zapadnoe Zabajkal’e) [Bone industry of Ust’-Kyakhta Site (Western Trans-Baikalia)]. Gumanit. Nauk. v Sib</strong></div><div><em>Humanities Sib.</em> 2014; <strong>2</strong>:14-18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.D.PavlenokKostyanaya+industriya+stoyanki+Ust%E2%80%99-Kyahta-3+%28Zapadnoe+Zabajkal%E2%80%99e%29+%5BBone+industry+of+Ust%E2%80%99-Kyakhta+Site+%28Western+Trans-Baikalia%29%5D.+Gumanit.+Nauk.+v+SibHumanities+Sib.220141418" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Tashak, V.I.</div><div><strong>Paleoliticheskie i mezoliticheskie pamiatniki Ust’-Kiakhty</strong></div><div>Buryatia Scientific Center of the Siberian Branch of the Russian Academy of Sciences, 2005</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.I.TashakPaleoliticheskie+i+mezoliticheskie+pamiatniki+Ust%E2%80%99-Kiakhty2005Buryatia+Scientific+Center+of+the+Siberian+Branch+of+the+Russian+Academy+of+Sciences" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and a series of ostrich eggshell beads (<span class="dropBlock reference-citations"><a id="body-ref-sref154" href="#" role="doc-biblioref" data-xml-rid="bib154" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref154" aria-controls="sref154">Zotkina et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref154"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">155.</div><div>Zotkina, L.V. ∙ Pavlenok, G.D. ∙ Tashak, V.I.</div><div><strong>Tekhnologiya proizvodstva busin iz skorlupy yaic strausa v final’nom paleolite Zapadnogo Zabajkal’ya [Technology of Ostrich Eggshell Bead Production in the Final Palaeolithic of Western Transbaikalia]. Strat. plus</strong></div><div><em>Archaeol. Cult. Anthropol.</em> 2018; <strong>1</strong>:181-197</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.V.ZotkinaG.D.PavlenokV.I.TashakTekhnologiya+proizvodstva+busin+iz+skorlupy+yaic+strausa+v+final%E2%80%99nom+paleolite+Zapadnogo+Zabajkal%E2%80%99ya+%5BTechnology+of+Ostrich+Eggshell+Bead+Production+in+the+Final+Palaeolithic+of+Western+Transbaikalia%5D.+Strat.+plusArchaeol.+Cult.+Anthropol.12018181197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0245" role="paragraph">Analytical data from the Ust-Kyakhta-3 archaeological collection provide solid evidence for their attribution to the Selenga Terminal Paleolithic culture (<span class="dropBlock reference-citations"><a id="body-ref-sref94-3" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-3" aria-controls="sref94-3">Pavlenok, 2015a</a><div class="dropBlock__holder" data-db-target-of="sref94-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Pavlenok, G.D.</div><div><strong>Tekhnologiya obrabotki kamnya v selenginskoi kulture Zapadnogo Zabaikaliya (po materialam stoyanki Ust-Kyakhta-3)</strong></div><div>Altai State University, 2015</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.D.PavlenokTekhnologiya+obrabotki+kamnya+v+selenginskoi+kulture+Zapadnogo+Zabaikaliya+%28po+materialam+stoyanki+Ust-Kyakhta-3%292015Altai+State+University" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The origin of this culture is associated with older, autochthonous complexes in Trans-Baikalia.</div></section></section><section id="sec-4-3-1-2"><h5>Individuals from the cis-Baikal region</h5><div id="p0250" role="paragraph">These human remains are currently housed in the Research Institute and Museum of Anthropology (RIMA) of the Lomonosov Moscow State University (MSU). Most of these remains were collected by Alexey Okladnikov during his first few independent archaeological excavations as an employee of the Irkutsk museum of regional studies between 1926 and 1929 in the Verkholenski and Irkutski uyezds of the Irkutsk Governorate. Now these territories form several districts of the East Siberian region. The works were regularly reported by the local press, informing general public not only about the archaeological excavations, but also about the establishment of a special department in the Irkutsk museum of regional studies to store the excavation materials, as well as the establishment of the District committee for the protection of the Paleo-Ethnological heritage of the region (<span class="dropBlock reference-citations"><a id="body-ref-sref142" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142" aria-controls="sref142">Vlast’truda, 1928</a><div class="dropBlock__holder" data-db-target-of="sref142"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">143.</div>Vlast’truda (1928). 21 oktyabrya. <i>246</i>, 4.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Vlast%E2%80%99truda+%281928%29.+21+oktyabrya.+246%2C+4." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0255" role="paragraph">In 1926 Okladnikov documented about ten Late Stone Age dwelling sites in the Verkholenski uyezd, and he also discovered a Neolithic cemetery nearby Iushino village (<span class="dropBlock reference-citations"><a id="body-ref-sref82-1" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-1" aria-controls="sref82-1">Okladnikov, 1926</a><div class="dropBlock__holder" data-db-target-of="sref82-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Okladnikov, A.P.</div><div><strong>Neoliticheskie stoyanki na Verhnej Lene. Kraeved. v Irkutsk. Gubernii Zap.Stud. Nauch</strong></div><div><em>Kruzhka Kraeved. Pri Irkut. Gos. Un-Te.</em> 1926; <strong>3</strong>:29-38</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeoliticheskie+stoyanki+na+Verhnej+Lene.+Kraeved.+v+Irkutsk.+Gubernii+Zap.Stud.+NauchKruzhka+Kraeved.+Pri+Irkut.+Gos.+Un-Te319262938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). During 1927 an archaeological exploration took place in Kachugsky District along the Lena river tributaries, including Biryulka, Anga, Manzurka and some other rivers. As a result, several Neolithic dwelling sites were discovered as well as a cemetery in Khaptsagai area near Bayraki village on the right bank of the Manzurka river (<span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">Okladnikov, 1928</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Okladnikov, A.P.</div><div><strong>Neoliticheskij mogil’nik v mestnosti Hapcagaj (Verhnyaya Lena)</strong></div><div><em>Izv. VSORGO.</em> 1928; <strong>53</strong>:125-134</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeoliticheskij+mogil%E2%80%99nik+v+mestnosti+Hapcagaj+%28Verhnyaya+Lena%29Izv.+VSORGO531928125134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, 1929). One month prior to this, local press distributed an information about excavations in the vicinities of the Stepno-Baltaiskii ulus, by Okladnikov and Kveiko, resulting in the discovery of human remains, stone scrapers, polished green nephrite knife, bone ornaments and more (<span class="dropBlock reference-citations"><a id="body-ref-sref143-1" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-1" aria-controls="sref143-1">Vlast’truda, 1929</a><div class="dropBlock__holder" data-db-target-of="sref143-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">144.</div>Vlast’truda (1929). 28 iyulya. <i>17</i>, 4.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Vlast%E2%80%99truda+%281929%29.+28+iyulya.+17%2C+4." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The same year Okladnikov conducted archaeological excavations near Zapleskino village in the Zhigalovsky District. The resulting findings were also transferred to the Irkutsk museum of regional studies.</div><div id="p0260" role="paragraph">In 1929 most of the anthropological materials from Okladnikov’s excavations were transferred to the Anthropological museum of the Lomonosov Moscow State University.</div><div id="p0265" role="paragraph">Through the work of several generations of Russian specialists (N.I. Vitkovsky, M.P. Ovchinnikov, G.F. Debets, A.P. Okladnikov, M.M. Gerasimov, L.P., L.P. Chlobystin and others), characteristic archaeological cultures of the Neolithic and the Early Bonze Age of the Cisbaikalia were defined. Over the past decades, the chronology and periodization of the Neolithic and the Bronze Age of the Cisbaikalia were refined (<span class="dropBlock reference-citations"><a id="body-ref-sref148" href="#" role="doc-biblioref" data-xml-rid="bib8 bib65 bib66 bib147 bib148" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref148" aria-controls="sref148">Bazalijskij, 2012; Mamonova and Sulerzhickij, 1989, 2008; Weber et&nbsp;al., 2002, 2006</a><div class="dropBlock__holder" data-db-target-of="sref148"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Bazalijskij, V.I.</div><div><strong>Pogrebal’nye kompleksy epohi pozdnego mezolita – neolita Bajkal’skoj Sibiri: tradicii pogrebenij, absolyutnyj vozrast. Izv. Lab. Drevnih Tekhnologij</strong></div><div><em>Irkutsk Izd-vo IrGTU.</em> 2012; <strong>9</strong>:43-101</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.I.BazalijskijPogrebal%E2%80%99nye+kompleksy+epohi+pozdnego+mezolita+%E2%80%93+neolita+Bajkal%E2%80%99skoj+Sibiri%3A+tradicii+pogrebenij%2C+absolyutnyj+vozrast.+Izv.+Lab.+Drevnih+TekhnologijIrkutsk+Izd-vo+IrGTU.9201243101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Mamonova, N.N. ∙ Sulerzhickij, L.D.</div><div><strong>Radiouglerodnaya khronologiya golocenovyh pogrebenij Pribajkal’ya i Zabajkal’ya po osteologicheskomu materialu iz mogil’nikov</strong></div><div><em>Sov. Arkheologiya.</em> 1989; <strong>1</strong>:19-32</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.N.MamonovaL.D.SulerzhickijRadiouglerodnaya+khronologiya+golocenovyh+pogrebenij+Pribajkal%E2%80%99ya+i+Zabajkal%E2%80%99ya+po+osteologicheskomu+materialu+iz+mogil%E2%80%99nikovSov.+Arkheologiya119891932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Mamonova, N.N. ∙ Sulerzhickij, L.D.</div><div><strong>Radiouglerodnaya khronologiya golocenovyh pogrebenij Pribajkal’ya i Zabajkal’ya po osteologicheskomu materialu iz mogil’nikov</strong></div><div><strong>Chelovek, Adaptaciya, Kul’tura</strong></div><div>M. IA RAN, 2008; 127-138</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.N.MamonovaL.D.SulerzhickijRadiouglerodnaya+khronologiya+golocenovyh+pogrebenij+Pribajkal%E2%80%99ya+i+Zabajkal%E2%80%99ya+po+osteologicheskomu+materialu+iz+mogil%E2%80%99nikovChelovek%2C+Adaptaciya%2C+Kul%E2%80%99tura2008M.+IA+RAN127138" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">148.</div><div>Weber, A.W. ∙ Link, D.W. ∙ Katzenberg, M.A.</div><div><strong>Hunter-gatherer culture change and continuity in the middle holocene of the Cis-Baikal, Siberia</strong></div><div><em>J.&nbsp;Anthropol. Archaeol.</em> 2002; <strong>21</strong>:230-299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jaar.2001.0395" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0036324599" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjaar.2001.0395" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">149.</div><div>Weber, A.W. ∙ Beukens, R.P. ∙ Bazaliiskii, V.I. ...</div><div><strong>Radiocarbon Dates from Neolithic and Bronze Age Hunter-Gatherer Cemeteries in the Cis-Baikal Region of Siberia</strong></div><div><em>Radiocarbon.</em> 2006; <strong>48</strong>:127-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/S0033822200035463" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-33645215976" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FS0033822200035463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As a result, two clusters of the Early Neolithic (EN) sites and a multicomponent cluster of the Late Neolithic and Early Bronze Age (LN-EBA) sites were defined. The first Early Neolithic group includes sites of the Kitoi mortuary tradition, represented by large well-known cemeteries in South Angara region of Southern Baikal, such as the Shamanka II cemetery. The second cluster of the Early Neolithic sites is located at adjacent regions in the Upper Lena River, Olkhon and Western Transbaikal area, different from the Kitoi group mainly in artifact composition and mortuary tradition. This cluster is represented by several small cemeteries and few burials (<span class="dropBlock reference-citations"><a id="body-ref-sref79-1" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-1" aria-controls="sref79-1">Movsesian et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref79-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Movsesian, A.A. ∙ Bakholdina, V.Y. ∙ Pezhemsky, D.V.</div><div><strong>Sushchestvovala li geneticheskaya preemstvennost’ mezhdu naseleniem razlichnyh etapov Pribajkal’skogo neolita?</strong></div><div><em>Vestn. Mosk. Univ. Seriya 23 Antropol.</em> 2015; <strong>3</strong>:94-104</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.A.MovsesianV.Y.BakholdinaD.V.PezhemskySushchestvovala+li+geneticheskaya+preemstvennost%E2%80%99+mezhdu+naseleniem+razlichnyh+etapov+Pribajkal%E2%80%99skogo+neolita%3FVestn.+Mosk.+Univ.+Seriya+23+Antropol.3201594104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The EN groups practiced hunting, fishing and sealing, with large and unevenly distributed population, differential mobility and substantial social differentiation. The LN-EBA (Serovo-Isakovo-Glazkovo) cultures are represented by various local variants spread over large territory. Similar with EN population, they had formal cemeteries, practiced hunting, fishing and sealing, but are represented by larger and evenly distributed populations that had moderate mobility and social differentiation (<span class="dropBlock reference-citations"><a id="body-ref-sref146" href="#" role="doc-biblioref" data-xml-rid="bib146" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref146" aria-controls="sref146">Weber and Bettinger, 2010</a><div class="dropBlock__holder" data-db-target-of="sref146"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Weber, A.W. ∙ Bettinger, R.</div><div><strong>Middle Holocene hunter-gatherers of Cis-Baikal, Siberia: An overview for the new century</strong></div><div><em>J.&nbsp;Anthropol. Archaeol.</em> 2010; <strong>29</strong>:491-506</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jaa.2010.08.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-78349304828" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaa.2010.08.002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0270" role="paragraph">Earlier craniometric studies by Russian anthropologist Debets suggested that the EN and the LN-EBA populations were genetically distinct (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib21 bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Debets, 1930, 1948</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Debets, G.F.</div><div><strong>Antropologicheskij sostav drevnego naseleniya Pribajkal’ya v epohu pozdnego neolita</strong></div><div><em>Rus. Antropol. Zhurnal.</em> 1930; <strong>t. XIX</strong>:1-2</div><div>7–50</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.F.DebetsAntropologicheskij+sostav+drevnego+naseleniya+Pribajkal%E2%80%99ya+v+epohu+pozdnego+neolitaRus.+Antropol.+Zhurnalt.+XIX1930127%E2%80%9350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Debets, G.F.</div><div><strong>Paleoantropologiya SSSR / TIE, novaya seriya</strong></div><div>M.-L. T. IV., 1948</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.F.DebetsPaleoantropologiya+SSSR+%2F+TIE%2C+novaya+seriya1948M.-L.+T.+IV." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mamonova shown that while differences between Serovo and Glazkovo craniological samples were minor, Kitoi crania differed substantially from both of these samples, and the extent of the difference between Kitoi and Serovo crania&nbsp;was greater than between Kitoi and Glazkovo crania (<span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Mamonova, 1973</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Mamonova, N.N.</div><div><strong>K voprosu o drevnem naselenii Priangar’ya po paleoantropologicheskim dannym</strong></div><div><div>Smirnov, A.P. (Editor)</div></div><div><strong>Problemy Arheologii Urala i Sibiri</strong></div><div>M. Nauka, 1973; 8-29</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.N.MamonovaK+voprosu+o+drevnem+naselenii+Priangar%E2%80%99ya+po+paleoantropologicheskim+dannymA.P.SmirnovProblemy+Arheologii+Urala+i+Sibiri1973M.+Nauka829" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It is likely that Serovo and Glazkovo series represent single craniological complex with local variants. Overall, this is the same craniological complex present in Angara (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Alekseev and Mamonova, 1979</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alekseev, V.P. ∙ Mamonova, N.N.</div><div><strong>K paleoantropologii epohi neolita verhov’ev Leny</strong></div><div><em>Sov. Etnogr.</em> 1979; <strong>5</strong>:49-63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.P.AlekseevN.N.MamonovaK+paleoantropologii+epohi+neolita+verhov%E2%80%99ev+LenySov.+Etnogr.519794963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0275" role="paragraph">Recent craniometric studies as well as studies of non-metric cranial traits suggest that the Early Neolithic Kitoi groups and the Late Neolithic Serovo groups belong to different gene pools. However, at the same time similarities were found between the Kitoi group from Angara region and chronologically distant Bronze Age Glazkovo groups from Angara, Upper Lena and Transbaikal territories. It is hypothesized that the morphological diversity of Glazkovo groups could be due to their mixture with the descendants of the Kitoi groups who left the Cisbaikalia and later migrated back from the neighboring territories. It is assumed that the population of the Transbaikalia, which exhibited extremely stable anthropological profile from the Early Neolithic to the Bronze Age time period, penetrated Cisbaikalia during the Glazkovo stage and influenced the formation of the Early Bronze Age population’s gene pool (<span class="dropBlock reference-citations"><a id="body-ref-sref79-2" href="#" role="doc-biblioref" data-xml-rid="bib78 bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-2" aria-controls="sref79-2">Movsesian et&nbsp;al., 2014, 2015</a><div class="dropBlock__holder" data-db-target-of="sref79-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Movsesian, A.A. ∙ Bakholdina, V.Y. ∙ Pezhemsky, D.V.</div><div><strong>Biological diversity and population history of Middle Holocene hunter-gatherers from the Cis-Baikal region of Siberia</strong></div><div><em>Am. J. Phys. Anthropol.</em> 2014; <strong>155</strong>:559-570</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.22608" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84910107273" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25176172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.22608&amp;pmid=25176172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Movsesian, A.A. ∙ Bakholdina, V.Y. ∙ Pezhemsky, D.V.</div><div><strong>Sushchestvovala li geneticheskaya preemstvennost’ mezhdu naseleniem razlichnyh etapov Pribajkal’skogo neolita?</strong></div><div><em>Vestn. Mosk. Univ. Seriya 23 Antropol.</em> 2015; <strong>3</strong>:94-104</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.A.MovsesianV.Y.BakholdinaD.V.PezhemskySushchestvovala+li+geneticheskaya+preemstvennost%E2%80%99+mezhdu+naseleniem+razlichnyh+etapov+Pribajkal%E2%80%99skogo+neolita%3FVestn.+Mosk.+Univ.+Seriya+23+Antropol.3201594104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0280" role="paragraph">Five skulls (RIMA number 4620 and 4571) were donated to the Anthropological museum of the MSU in 1885 by Siberian archaeologist I.T. Savenkov. In 1884 he conducted archaeological research in the villages: Ladeyka, Nyasha, Bazaikha and Sobakino (Krasnoyarsk district).</div><section id="sec-4-3-1-2-1"><h6>Angara Site</h6><div id="p0285" role="paragraph"><div id="ulist0020" role="list"><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0290" role="paragraph">ANG001, tooth M3 lower right (RIMA number 4620)</div></div></div></div></div><div id="p0295" role="paragraph">Savenkov found this cranium while excavating ancient graves on the banks of Angara River. No archaeological context is available.</div></section><section id="sec-4-3-1-2-2"><h6>Bazaikha Site</h6><div id="p0300" role="paragraph"><div id="ulist0025" role="list"><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0305" role="paragraph">BZK002, tooth M2 lower right (RIMA number 4571)</div></div></div></div></div><div id="p0310" role="paragraph">Savenkov found two skulls near the river Bazaikha on the right banks of Yenisei river in Krasnoyarsk district. No archaeological context is available. According to <span class="dropBlock reference-citations"><a id="body-ref-sref85-1" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-1" aria-controls="sref85-1">Okladnikov (1950)</a><div class="dropBlock__holder" data-db-target-of="sref85-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Okladnikov, A.P.</div><div><strong>Neolit i bronzovyj vek Pribajkal’ya. Istoriko-arheologicheskoe issledovanie</strong></div><div><em>CH. I, II. Mater. i Issled. Po Arheol. SSSR.</em> 1950; <strong>18</strong></div><div>M.-L. 412s</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeolit+i+bronzovyj+vek+Pribajkal%E2%80%99ya.+Istoriko-arheologicheskoe+issledovanieCH.+I%2C+II.+Mater.+i+Issled.+Po+Arheol.+SSSR181950M.-L.+412s" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, the skeletal remains may be attributed to the Bronze Age Karasuk culture between late second to early first millennium BC, based on the archaeological findings described by Savenkov.</div></section><section id="sec-4-3-1-2-3"><h6>Glazkovskoe Predmestie Site</h6><div id="p0315" role="paragraph"><div id="ulist0030" role="list"><div id="u0045" role="listitem"><div class="label">•</div><div class="content"><div id="p0320" role="paragraph">GLZ001, tooth M2 upper left, (RIMA number 4616),</div></div></div><div id="u0050" role="listitem"><div class="label">•</div><div class="content"><div id="p0325" role="paragraph">GLZ002, tooth M2 upper right, (RIMA number 4617),</div></div></div><div id="u0055" role="listitem"><div class="label">•</div><div class="content"><div id="p0330" role="paragraph">GLZ003, tooth M1 lower right, (RIMA number 4618)</div></div></div></div></div><div id="p0335" role="paragraph">Archaeological and anthropological findings were from the Glazkovo outskirts of Irkutsk city found during excavations for the foundation of a new infant asylum. Construction works were conducted under the lead of V.P. Sukachov from the summer of 1887. An ancient burial was discovered in the autumn of the same year. N.I. Vitkovski, an employee of the museum of the East Siberian division of the Russian Geographical society, was called in upon discovery. This researcher was known for his excavations on the Kitoi cemetery between 1880-1881 that gave the name to the Early Neolithic culture of the region. By the time Vitkovski arrived at the site, construction workers had destroyed the burials. He collected six crania and a few artifacts, including two rings made from white nephrite. According to several eyewitnesses, the interred individuals were in the sitting position, facing East. Under Vitkovski’s supervision, excavation continued and an undamaged burial was discovered. The skeleton was in flexed position with the legs bent at the knees. Two perforated pendants from cleaved wild boar tusk were found on the cranium. The findings were transported to the museum of the Russian Geographical society’s division (<span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Debets, 1930</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Debets, G.F.</div><div><strong>Antropologicheskij sostav drevnego naseleniya Pribajkal’ya v epohu pozdnego neolita</strong></div><div><em>Rus. Antropol. Zhurnal.</em> 1930; <strong>t. XIX</strong>:1-2</div><div>7–50</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.F.DebetsAntropologicheskij+sostav+drevnego+naseleniya+Pribajkal%E2%80%99ya+v+epohu+pozdnego+neolitaRus.+Antropol.+Zhurnalt.+XIX1930127%E2%80%9350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Among these, three crania were later transferred to the MSU and became part of its osteological collections.</div></section><section id="sec-4-3-1-2-4"><h6>Iushino Site</h6><div id="p0340" role="paragraph"><div id="ulist0035" role="list"><div id="u0060" role="listitem"><div class="label">•</div><div class="content"><div id="p0345" role="paragraph">IUO001, part of left <i>os temporale</i> (<i>pars petrosa</i>), (RIMA number 8311)</div></div></div></div></div><div id="p0350" role="paragraph">In 1926 Okladnikov discovered a Neolithic burial ground near Iushino village (<span class="dropBlock reference-citations"><a id="body-ref-sref82-2" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-2" aria-controls="sref82-2">Okladnikov, 1926</a><div class="dropBlock__holder" data-db-target-of="sref82-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Okladnikov, A.P.</div><div><strong>Neoliticheskie stoyanki na Verhnej Lene. Kraeved. v Irkutsk. Gubernii Zap.Stud. Nauch</strong></div><div><em>Kruzhka Kraeved. Pri Irkut. Gos. Un-Te.</em> 1926; <strong>3</strong>:29-38</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeoliticheskie+stoyanki+na+Verhnej+Lene.+Kraeved.+v+Irkutsk.+Gubernii+Zap.Stud.+NauchKruzhka+Kraeved.+Pri+Irkut.+Gos.+Un-Te319262938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Human skeletal remains and artifacts from three burials were transferred to the Irkutsk museum. Based on these, the remains are attributed to the Early Neolithic period of the Kitoi culture (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Alekseev and Mamonova, 1979</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alekseev, V.P. ∙ Mamonova, N.N.</div><div><strong>K paleoantropologii epohi neolita verhov’ev Leny</strong></div><div><em>Sov. Etnogr.</em> 1979; <strong>5</strong>:49-63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.P.AlekseevN.N.MamonovaK+paleoantropologii+epohi+neolita+verhov%E2%80%99ev+LenySov.+Etnogr.519794963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The cranium from the Grave 3 was later transferred to the MSU.</div></section><section id="sec-4-3-1-2-5"><h6>Kachug Site</h6><div id="p0355" role="paragraph"><div id="ulist0040" role="list"><div id="u0065" role="listitem"><div class="label">•</div><div class="content"><div id="p0360" role="paragraph">KAG001, part of left <i>os temporale</i> (<i>pars petrosa</i>), (RIMA number 8307)</div></div></div><div id="u0070" role="listitem"><div class="label">•</div><div class="content"><div id="p0365" role="paragraph">KAG002, tooth M1 lower left, (RIMA number 8308)</div></div></div></div></div><div id="p0370" role="paragraph">The Kachug burial ground was located on the right bank of the Lena river, above the Kachug village. Currently, the exact location of this burial ground cannot be identified. Here in 1927 three burials were excavated, dated to the Eneolithic time (<span class="dropBlock reference-citations"><a id="body-ref-sref86-1" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-1" aria-controls="sref86-1">Okladnikov, 1955</a><div class="dropBlock__holder" data-db-target-of="sref86-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Okladnikov, A.P.</div><div><strong>Neolit i bronzovyj vek Pribajkal’ya. CH. III. (Glazkovskoe vremya)</strong></div><div><em>Mater. i Issled. Po Arheol. SSSR.</em> 1955; <strong>43</strong></div><div>M.-L. 373 s</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeolit+i+bronzovyj+vek+Pribajkal%E2%80%99ya.+CH.+III.+%28Glazkovskoe+vremya%29Mater.+i+Issled.+Po+Arheol.+SSSR431955M.-L.+373+s" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Skeletal remains from the grave 2 (# 8307) and grave 3 (# 8308) were transferred to the Anthropological museum. Grave 2 (# 8307) contained polished nephrite ring and a black stone disc (<span class="dropBlock reference-citations"><a id="body-ref-sref86-2" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-2" aria-controls="sref86-2">Okladnikov, 1955</a><div class="dropBlock__holder" data-db-target-of="sref86-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Okladnikov, A.P.</div><div><strong>Neolit i bronzovyj vek Pribajkal’ya. CH. III. (Glazkovskoe vremya)</strong></div><div><em>Mater. i Issled. Po Arheol. SSSR.</em> 1955; <strong>43</strong></div><div>M.-L. 373 s</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeolit+i+bronzovyj+vek+Pribajkal%E2%80%99ya.+CH.+III.+%28Glazkovskoe+vremya%29Mater.+i+Issled.+Po+Arheol.+SSSR431955M.-L.+373+s" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). From grave 3 (# 8308) Okladnikov described the following artifacts: a small copper or bronze knife “firmly attached to the massive horn handle,” “this knife is flat, with double edged blade, its protruding part has triangular outline,” and two copper tubes (<span class="dropBlock reference-citations"><a id="body-ref-sref86-3" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-3" aria-controls="sref86-3">Okladnikov, 1955</a><div class="dropBlock__holder" data-db-target-of="sref86-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Okladnikov, A.P.</div><div><strong>Neolit i bronzovyj vek Pribajkal’ya. CH. III. (Glazkovskoe vremya)</strong></div><div><em>Mater. i Issled. Po Arheol. SSSR.</em> 1955; <strong>43</strong></div><div>M.-L. 373 s</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeolit+i+bronzovyj+vek+Pribajkal%E2%80%99ya.+CH.+III.+%28Glazkovskoe+vremya%29Mater.+i+Issled.+Po+Arheol.+SSSR431955M.-L.+373+s" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Items found in these burials have analogies in well-documented and dated burial complexes of Angara, Upper Lena and Olkhon areas, attributed to the Glazkovo culture of the Late Neolithic – Early Bronze Age (<span class="dropBlock reference-citations"><a id="body-ref-sref155" href="#" role="doc-biblioref" data-xml-rid="bib1 bib155" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref155" aria-controls="sref155">Alekseev and Mamonova, 1979; Zubkov, 2010</a><div class="dropBlock__holder" data-db-target-of="sref155"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alekseev, V.P. ∙ Mamonova, N.N.</div><div><strong>K paleoantropologii epohi neolita verhov’ev Leny</strong></div><div><em>Sov. Etnogr.</em> 1979; <strong>5</strong>:49-63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.P.AlekseevN.N.MamonovaK+paleoantropologii+epohi+neolita+verhov%E2%80%99ev+LenySov.+Etnogr.519794963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">156.</div><div>Zubkov, V.S.</div><div><strong>Materialy iz drevnih mogil s territorii poselka Kachug i u derevni SHishkino na Verhnej Lene (raskopki A. P. Okladnikova 1928-1930, 1941 gg.). Izv. Lab. Drevnih Tekhnologij</strong></div><div><em>Irkutsk Izd-vo IrGTU.</em> 2010; <strong>8</strong>:135-146</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.S.ZubkovMaterialy+iz+drevnih+mogil+s+territorii+poselka+Kachug+i+u+derevni+SHishkino+na+Verhnej+Lene+%28raskopki+A.+P.+Okladnikova+1928-1930%2C+1941+gg.%29.+Izv.+Lab.+Drevnih+TekhnologijIrkutsk+Izd-vo+IrGTU.82010135146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-3-1-2-6"><h6>Khaptsagai Site</h6><div id="p0375" role="paragraph"><div id="ulist0045" role="list"><div id="u0075" role="listitem"><div class="label">•</div><div class="content"><div id="p0380" role="paragraph">KPT001, part of left <i>os temporale</i> (<i>pars petrosa</i>), (RIMA number 8313),</div></div></div><div id="u0080" role="listitem"><div class="label">•</div><div class="content"><div id="p0385" role="paragraph">KPT002, part of left <i>os temporale</i> (<i>pars petrosa</i>), (RIMA number 8300),</div></div></div><div id="u0085" role="listitem"><div class="label">•</div><div class="content"><div id="p0390" role="paragraph">KPT003, M2 upper right, (RIMA number 8302),</div></div></div><div id="u0090" role="listitem"><div class="label">•</div><div class="content"><div id="p0395" role="paragraph">KPT004, M2 upper left, (RIMA number 8304),</div></div></div><div id="u0095" role="listitem"><div class="label">•</div><div class="content"><div id="p0400" role="paragraph">KPT005, M2 lower right, (RIMA number 8301),</div></div></div><div id="u0100" role="listitem"><div class="label">•</div><div class="content"><div id="p0405" role="paragraph">KPT006, M1 lower left, (RIMA number 8305)</div></div></div></div></div><div id="p0410" role="paragraph">In 1927 an archaeological exploration in Kachug area took place. As a result, several Neolithic dwelling sites were discovered and a burial ground in Khaptsagay area near Bayraki village, on the right bank of Manzurka river (<span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">Okladnikov, 1928</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Okladnikov, A.P.</div><div><strong>Neoliticheskij mogil’nik v mestnosti Hapcagaj (Verhnyaya Lena)</strong></div><div><em>Izv. VSORGO.</em> 1928; <strong>53</strong>:125-134</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeoliticheskij+mogil%E2%80%99nik+v+mestnosti+Hapcagaj+%28Verhnyaya+Lena%29Izv.+VSORGO531928125134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Twenty ancient burials were excavated. Skeletal remains from six graves were transferred to the Anthropological museum: from grave 7 (inventory #8300), grave 12 (#8304), grave 13 (#8305), grave 14 (#8302), grave 19 (#8301) and from a looted grave (#8313). Some of the individuals were in a flexed position within the interment, under a stone vault. Numerous artifacts were unearthed such as white nephrite rings, green nephrite knives, stone arrow heads and spear points, stone chisel, awls and stone needles in ornamented needle cases (<span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Okladnikov, 1929</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Okladnikov, A.P.</div><div><strong>Raskopki v Kachugskom rajone po pritokam Leny</strong></div><div><em>Vlast’ Tr.</em> 1929; <strong>10</strong></div><div>Avg <i>185</i>, 4</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovRaskopki+v+Kachugskom+rajone+po+pritokam+LenyVlast%E2%80%99+Tr.101929Avg+185%2C+4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). According to the researchers, human skeletal remains are dated to the Glazkovo stage of the Late Neolithic based on the artifacts (<span class="dropBlock reference-citations"><a id="body-ref-sref1-4" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-4" aria-controls="sref1-4">Alekseev and Mamonova, 1979</a><div class="dropBlock__holder" data-db-target-of="sref1-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alekseev, V.P. ∙ Mamonova, N.N.</div><div><strong>K paleoantropologii epohi neolita verhov’ev Leny</strong></div><div><em>Sov. Etnogr.</em> 1979; <strong>5</strong>:49-63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.P.AlekseevN.N.MamonovaK+paleoantropologii+epohi+neolita+verhov%E2%80%99ev+LenySov.+Etnogr.519794963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-3-1-2-7"><h6>Zhigalovo Site</h6><div id="p0415" role="paragraph"><div id="ulist0050" role="list"><div id="u0105" role="listitem"><div class="label">•</div><div class="content"><div id="p0420" role="paragraph">STB001, part of right <i>os temporale (pars petrosa)</i>, (RIMA number 8315)</div></div></div></div></div><div id="p0425" role="paragraph">This cranium has an abbreviation “Zh. 1” written on it, which likely stands for Zhigalovo, grave 1, where the cranium was presumably discovered. Based on the finding date (year 1929) this may be a cranium from a disintegrated grave, discovered by Okladnikov on the Lena river bank on the opposite side of the Zhigalovo village, the district center. The grave was severely destroyed due to displacement and erosion of clay layers of the ancient terrace. Here, stone bow plates attributed to Late Neolithic Serovo culture were discovered. This was the first finding of the bow remnants from this period (<span class="dropBlock reference-citations"><a id="body-ref-sref85-2" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-2" aria-controls="sref85-2">Okladnikov, 1950</a><div class="dropBlock__holder" data-db-target-of="sref85-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Okladnikov, A.P.</div><div><strong>Neolit i bronzovyj vek Pribajkal’ya. Istoriko-arheologicheskoe issledovanie</strong></div><div><em>CH. I, II. Mater. i Issled. Po Arheol. SSSR.</em> 1950; <strong>18</strong></div><div>M.-L. 412s</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeolit+i+bronzovyj+vek+Pribajkal%E2%80%99ya.+Istoriko-arheologicheskoe+issledovanieCH.+I%2C+II.+Mater.+i+Issled.+Po+Arheol.+SSSR181950M.-L.+412s" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-3-1-2-8"><h6>Stepno-Baltaiskii ulus Site</h6><div id="p0430" role="paragraph"><div id="ulist0055" role="list"><div id="u0110" role="listitem"><div class="label">•</div><div class="content"><div id="p0435" role="paragraph">STB002, part of left <i>os temporale (pars petrosa)</i>, (RIMA number 8314)</div></div></div></div></div><div id="p0440" role="paragraph">Okladnikov and Kveiko found Stone Age burials in the vicinity of the Stepno-Baltaiskii ulus. Stone scrapers, a polished green jade knife, bone ornaments and patterned sawn elk shoulder blades were found within these burials (<span class="dropBlock reference-citations"><a id="body-ref-sref143-2" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-2" aria-controls="sref143-2">Vlast’truda, 1929</a><div class="dropBlock__holder" data-db-target-of="sref143-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">144.</div>Vlast’truda (1929). 28 iyulya. <i>17</i>, 4.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Vlast%E2%80%99truda+%281929%29.+28+iyulya.+17%2C+4." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The skeletal remains (the cranium and postcranial skeleton) from the grave 1 were transferred to the Anthropological museum of the MSU. According to the researcher, this skeleton may belong to the Early Neolithic period of the Kitoi culture (<span class="dropBlock reference-citations"><a id="body-ref-sref1-5" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-5" aria-controls="sref1-5">Alekseev and Mamonova, 1979</a><div class="dropBlock__holder" data-db-target-of="sref1-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alekseev, V.P. ∙ Mamonova, N.N.</div><div><strong>K paleoantropologii epohi neolita verhov’ev Leny</strong></div><div><em>Sov. Etnogr.</em> 1979; <strong>5</strong>:49-63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.P.AlekseevN.N.MamonovaK+paleoantropologii+epohi+neolita+verhov%E2%80%99ev+LenySov.+Etnogr.519794963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-3-1-2-9"><h6>Zapleskino Site</h6><div id="p0445" role="paragraph"><div id="ulist0060" role="list"><div id="u0115" role="listitem"><div class="label">•</div><div class="content"><div id="p0450" role="paragraph">ZPL001, part of left <i>os temporale (pars petrosa)</i>, (RIMA number 8309),</div></div></div><div id="u0120" role="listitem"><div class="label">•</div><div class="content"><div id="p0455" role="paragraph">ZPL002, M2 lower right, (RIMA number 8310)</div></div></div></div></div><div id="p0460" role="paragraph">These remains are from burials in the vicinity of Zapleskino village in the Zhigalovsky District. Okladnikov transferred two crania to the Anthropological museum, including crania from grave 1 (#8309) and grave 4 (#8310). These findings are dated to the Late Neolithic – Early Bronze Age stage of the Glazkovo culture (<span class="dropBlock reference-citations"><a id="body-ref-sref1-6" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-6" aria-controls="sref1-6">Alekseev and Mamonova, 1979</a><div class="dropBlock__holder" data-db-target-of="sref1-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alekseev, V.P. ∙ Mamonova, N.N.</div><div><strong>K paleoantropologii epohi neolita verhov’ev Leny</strong></div><div><em>Sov. Etnogr.</em> 1979; <strong>5</strong>:49-63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.P.AlekseevN.N.MamonovaK+paleoantropologii+epohi+neolita+verhov%E2%80%99ev+LenySov.+Etnogr.519794963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section></section></section><section id="sec-4-4"><h3>Method Details</h3><section id="sec-4-4-1"><h4>Radiocarbon dating and calibration</h4><div id="p0465" role="paragraph">Except for UKY001, all the other 19 samples were analyzed by accelerator mass spectrometry (AMS) at Manheim (MAMS). For UKY001 we used the three published <sup>14</sup>C dates of charcoal and bone materials from the same layer reported as references of the sample age (<span class="dropBlock reference-citations"><a id="body-ref-sref96-4" href="#" role="doc-biblioref" data-xml-rid="bib90 bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-4" aria-controls="sref96-4">Orlova, 1995; Pavlenok et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref96-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Orlova, L.A.</div><div><strong>Radiocarbon dating of archaeological sites of Siberia and the Far East</strong></div><div><div>Derevianko, A.P. ∙ Kholushkin, Y.P. (Editors)</div></div><div><strong>Methods of Natural Sciences in Archaeological Reconstructions</strong></div><div>Publishing House of the Institute of Archeology and Ethnography SB RAS, 1995; 207-232</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.A.OrlovaRadiocarbon+dating+of+archaeological+sites+of+Siberia+and+the+Far+EastA.P.DereviankoY.P.KholushkinMethods+of+Natural+Sciences+in+Archaeological+Reconstructions1995Publishing+House+of+the+Institute+of+Archeology+and+Ethnography+SB+RAS207232" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Pavlenok, G.D. ∙ Zubova, A.V.</div><div><strong>New Dental Finds Associated with the Paleolithic Selenga Culture, Western Trans-Baikal Region</strong></div><div><em>Archaeol. Ethnol. Anthropol. Eurasia.</em> 2019; <strong>47</strong>:3-11</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.17746/1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85076466976" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.17746%2F1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and combined them in calibration. All the radiocarbon dates were calibrated using the dataset IntCal 13 (<span class="dropBlock reference-citations"><a id="body-ref-sref117" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117" aria-controls="sref117">Reimer et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref117"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Reimer, P.J. ∙ Bard, E. ∙ Bayliss, A. ...</div><div><strong>IntCal13 and Marine13 Radiocarbon Age Calibration Curves 0–50,000 Years cal BP</strong></div><div><em>Radiocarbon.</em> 2013; <strong>55</strong>:1869-1887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_js_rc.55.16947" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84887538928" target="_blank" aria-label="Open Scopus in new tab">Scopus (7346)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_js_rc.55.16947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Prior to conversion to a calendar age it was important to determine whether there were any dietary offsets that could influence the calibrated ages, as freshwater reservoir effects, including those from the Lake Baikal region, are known to make radiocarbon ages appear as much as &gt; 1,000 years in some systems (<span class="dropBlock reference-citations"><a id="body-ref-sref127-2" href="#" role="doc-biblioref" data-xml-rid="bib81 bib110 bib126 bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127-2" aria-controls="sref127-2">Nomokonova et&nbsp;al., 2013; Ramsey et&nbsp;al., 2014; Schulting et&nbsp;al., 2014, 2015</a><div class="dropBlock__holder" data-db-target-of="sref127-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Nomokonova, T. ∙ Losey, R.J. ∙ Goriunova, O.I. ...</div><div><strong>A freshwater old carbon offset in Lake Baikal, Siberia and problems with the radiocarbon dating of archaeological sediments: Evidence from the Sagan-Zaba II site</strong></div><div><em>Quat. Int.</em> 2013; <strong>290–291</strong>:110-125</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2012.06.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84875540228" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2012.06.007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Ramsey, C.B. ∙ Schulting, R. ∙ Goriunova, O.I. ...</div><div><strong>Analyzing Radiocarbon Reservoir Offsets Through Stable Nitrogen Isotopes and Bayesian Modeling: A Case Study Using Paired Human and Faunal Remains from the Cis-Baikal Region, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:789-799</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17160" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899887915" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17160" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Schulting, R.J. ∙ Ramsey, C.B. ∙ Bazaliiskii, V.I. ...</div><div><strong>Freshwater Reservoir Offsets Investigated Through Paired Human-Faunal 14C Dating and Stable Carbon and Nitrogen Isotope Analysis at Lake Baikal, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:991-1008</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899833606" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Schulting, R.J. ∙ Bronk Ramsey, C. ∙ Bazaliiskii, V.I. ...</div><div><strong>Highly Variable Freshwater Reservoir Offsets Found along the Upper Lena Watershed, Cis-Baikal, Southeast Siberia</strong></div><div><em>Radiocarbon.</em> 2015; <strong>57</strong>:581-593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_rc.57.18458" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84942103801" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_rc.57.18458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We used stable carbon (δ<sup>13</sup>C) and nitrogen (δ<sup>15</sup>N) measurements of bone collagen from the same individuals, as well as published regression equations to estimate the offset between conventional and terrestrial radiocarbon dates (<span class="dropBlock reference-citations"><a id="body-ref-sref127-3" href="#" role="doc-biblioref" data-xml-rid="bib126 bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127-3" aria-controls="sref127-3">Schulting et&nbsp;al., 2014, 2015</a><div class="dropBlock__holder" data-db-target-of="sref127-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Schulting, R.J. ∙ Ramsey, C.B. ∙ Bazaliiskii, V.I. ...</div><div><strong>Freshwater Reservoir Offsets Investigated Through Paired Human-Faunal 14C Dating and Stable Carbon and Nitrogen Isotope Analysis at Lake Baikal, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:991-1008</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899833606" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Schulting, R.J. ∙ Bronk Ramsey, C. ∙ Bazaliiskii, V.I. ...</div><div><strong>Highly Variable Freshwater Reservoir Offsets Found along the Upper Lena Watershed, Cis-Baikal, Southeast Siberia</strong></div><div><em>Radiocarbon.</em> 2015; <strong>57</strong>:581-593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_rc.57.18458" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84942103801" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_rc.57.18458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The estimated offsets are given in <a id="crosref0920" href="#mmc1">Table S1</a>.</div></section><section id="sec-4-4-2"><h4>Stable isotope analysis</h4><section id="sec-4-4-2-1"><h5>Background</h5><div id="p0470" role="paragraph">δ<sup>13</sup>C variability in terrestrial ecosystems is primarily driven by plants that variously utilize two dominant photosynthetic pathways, C<sub>3</sub> and C<sub>4</sub> (<span class="dropBlock reference-citations"><a id="body-ref-sref131-1" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131-1" aria-controls="sref131-1">Smith and Epstein, 1971</a><div class="dropBlock__holder" data-db-target-of="sref131-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Smith, B.N. ∙ Epstein, S.</div><div><strong>Two categories of c/c ratios for higher plants</strong></div><div><em>Plant Physiol.</em> 1971; <strong>47</strong>:380-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1104/pp.47.3.380" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16657626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1104%2Fpp.47.3.380&amp;pmid=16657626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). C<sub>3</sub> plants, including virtually all trees, shrubs, and temperate grasses, and domesticates such as wheat and barley, have values from <i>c.</i> −24 to −36‰ (global mean −26.5‰). C<sub>4</sub> plants, including domesticated grasses such as millet, have values ranging from <i>c</i>. −9 to −17‰ (global mean −12‰ (<span class="dropBlock reference-citations"><a id="body-ref-sref131-2" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131-2" aria-controls="sref131-2">Smith and Epstein, 1971</a><div class="dropBlock__holder" data-db-target-of="sref131-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Smith, B.N. ∙ Epstein, S.</div><div><strong>Two categories of c/c ratios for higher plants</strong></div><div><em>Plant Physiol.</em> 1971; <strong>47</strong>:380-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1104/pp.47.3.380" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16657626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1104%2Fpp.47.3.380&amp;pmid=16657626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These non-overlapping distinctions are reflected in the tissues of consumers (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Ambrose and Norr, 1993</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Ambrose, S.H. ∙ Norr, L.</div><div><strong>Experimental Evidence for the Relationship of the Carbon Isotope Ratios of Whole Diet and Dietary Protein to Those of Bone Collagen and Carbonate</strong></div><div><div>Lambert, J.B. ∙ Grupe, G. (Editors)</div></div><div><strong>Prehistoric Human Bone: Archaeology at the Molecular Level</strong></div><div>Springer Berlin Heidelberg, 1993; 1-37</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-3-662-02894-0_1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-3-662-02894-0_1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). δ<sup>15</sup>N varies with trophic level, and δ<sup>15</sup>N trophic shifts of&nbsp;+2-6‰ being documented in terrestrial ecosystems (<span class="dropBlock reference-citations"><a id="body-ref-sref128" href="#" role="doc-biblioref" data-xml-rid="bib25 bib128" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref128" aria-controls="sref128">Deniro and Epstein, 1981; Sealy et&nbsp;al., 1987</a><div class="dropBlock__holder" data-db-target-of="sref128"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Deniro, M.J. ∙ Epstein, S.</div><div><strong>Influence of diet on the distribution of nitrogen isotopes in animals</strong></div><div><em>Geochim. Cosmochim. Acta.</em> 1981; <strong>45</strong>:341-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0016-7037(81)90244-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0019681135" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0016-7037%2881%2990244-1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>Sealy, J.C. ∙ van der Merwe, N.J. ∙ Thorp, J.A.L. ...</div><div><strong>Nitrogen isotopic ecology in southern Africa: Implications for environmental and dietary tracing</strong></div><div><em>Geochim. Cosmochim. Acta.</em> 1987; <strong>51</strong>:2707-2717</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0016-7037(87)90151-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0016-7037%2887%2990151-7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The long length of aquatic foodchains, leads to distinctively high δ<sup>15</sup>N in consumers (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Minagawa and Wada, 1984</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Minagawa, M. ∙ Wada, E.</div><div><strong>Stepwise enrichment of 15N along food chains: Further evidence and the relation between δ15N and animal age</strong></div><div><em>Geochim. Cosmochim. Acta.</em> 1984; <strong>48</strong>:1135-1140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0016-7037(84)90204-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0021342781" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0016-7037%2884%2990204-7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, while marine consumers have both high δ<sup>13</sup>C and δ<sup>15</sup>N, δ<sup>13</sup>C is less predictable in freshwater ecosystems necessitating detailed baseline reconstruction (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib47 bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Katzenberg and Weber, 1999; Kiyashko et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Katzenberg, M.A. ∙ Weber, A.</div><div><strong>Stable isotope ecology and palaeodiet in the Lake Baikal region of Siberia</strong></div><div><em>J.&nbsp;Archaeol. Sci.</em> 1999; <strong>26</strong>:651-659</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jasc.1998.0382" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0033150923" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjasc.1998.0382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Kiyashko, S.I. ∙ Richard, P. ∙ Chandler, T. ...</div><div><strong>Stable carbon isotope ratios differentiate autotrophs supporting animal diversity in Lake Baikal. Comptes Rendus l’Académie Des Sci. - Ser. III - Sci</strong></div><div><em>Vine.</em> 1998; <strong>321</strong>:509-516</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.I.KiyashkoP.RichardT.ChandlerT.A.KozlovaD.F.WilliamsStable+carbon+isotope+ratios+differentiate+autotrophs+supporting+animal+diversity+in+Lake+Baikal.+Comptes+Rendus+l%E2%80%99Acad%C3%A9mie+Des+Sci.+-+Ser.+III+-+SciVine3211998509516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). δ<sup>13</sup>C and δ<sup>15</sup>N analysis of human bone collagen primarily determines the isotopic values of the protein input to the diet, with a much more minor contribution of lipid and carbohydrate sources (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Ambrose and Norr, 1993</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Ambrose, S.H. ∙ Norr, L.</div><div><strong>Experimental Evidence for the Relationship of the Carbon Isotope Ratios of Whole Diet and Dietary Protein to Those of Bone Collagen and Carbonate</strong></div><div><div>Lambert, J.B. ∙ Grupe, G. (Editors)</div></div><div><strong>Prehistoric Human Bone: Archaeology at the Molecular Level</strong></div><div>Springer Berlin Heidelberg, 1993; 1-37</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-3-662-02894-0_1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-3-662-02894-0_1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This means that the δ<sup>13</sup>C and δ<sup>15</sup>N values of bone and dentine collagen will be heavily affected by foods that are high in protein (<span class="dropBlock reference-citations"><a id="body-ref-sref5-3" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-3" aria-controls="sref5-3">Ambrose and Norr, 1993</a><div class="dropBlock__holder" data-db-target-of="sref5-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Ambrose, S.H. ∙ Norr, L.</div><div><strong>Experimental Evidence for the Relationship of the Carbon Isotope Ratios of Whole Diet and Dietary Protein to Those of Bone Collagen and Carbonate</strong></div><div><div>Lambert, J.B. ∙ Grupe, G. (Editors)</div></div><div><strong>Prehistoric Human Bone: Archaeology at the Molecular Level</strong></div><div>Springer Berlin Heidelberg, 1993; 1-37</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-3-662-02894-0_1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-3-662-02894-0_1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, tooth dentine forming during a period of breastfeeding (e.g., permanent M1) will be elevated in δ<sup>15</sup>N to appear a whole trophic level higher (<span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Fuller et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Fuller, B.T. ∙ Richards, M.P. ∙ Mays, S.A.</div><div><strong>Stable carbon and nitrogen isotope variations in tooth dentine serial sections from Wharram Percy</strong></div><div><em>J.&nbsp;Archaeol. Sci.</em> 2003; <strong>30</strong>:1673-1684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0305-4403(03)00073-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0344962345" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0305-4403%2803%2900073-6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The same may also apply to certain portions of the skull including the petrus (<span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Jørkov et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Jørkov, M.L.S. ∙ Heinemeier, J. ∙ Lynnerup, N.</div><div><strong>The petrous bone--a new sampling site for identifying early dietary patterns in stable isotopic studies</strong></div><div><em>Am. J. Phys. Anthropol.</em> 2009; <strong>138</strong>:199-209</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.20919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-59949101061" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18773469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.20919&amp;pmid=18773469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-2-2"><h5>Stable isotope analysis</h5><div id="p0475" role="paragraph">We sampled the petrous bones of seven individuals and tooth dentine of 12 individuals (<a id="crosref0965" href="#mmc1">Table S1</a>) for stable carbon and nitrogen isotope analysis in order to determine differences in diet and the potential impact of the reservoir effect on the ages of individuals analyzed.</div><div id="p0480" role="paragraph">We obtained <i>c.</i> 0.5&nbsp;g of dentine or bone powder using a handheld Dremel drill. All specimens were first cleaned using abrasion by sandblaster. We extracted collagen from the resulting powder using a modified Longin methodology (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Longin, 1971</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Longin, R.</div><div><strong>New method of collagen extraction for radiocarbon dating</strong></div><div><em>Nature.</em> 1971; <strong>230</strong>:241-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/230241a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0015240867" target="_blank" aria-label="Open Scopus in new tab">Scopus (975)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4926713/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F230241a0&amp;pmid=4926713" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Samples were demineralised by immersion in 0.5M HCl for 1-7&nbsp;days. Once demineralisation was complete, samples were rinsed three times with ultra-pure H<sub>2</sub>O. The residue was gelatinized in pH3 HCl at 70°C for 48 hours and Ezee-filters were used to remove the insoluble residues from the soluble collagen solution (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Brock et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Brock, F. ∙ Higham, T. ∙ Ramsey, C.B.</div><div><strong>Comments on the Use of Ezee-Filters™ and Ultrafilters at Orau</strong></div><div><em>Radiocarbon.</em> 2013; <strong>55</strong>:211-212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_js_rc.v55i1.16480" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84874546554" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_js_rc.v55i1.16480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Samples were lyophilized for 48hrs. 1.0&nbsp;mg of the resulting purified collagen was weighed in duplicates into tin capsules for analysis.</div><div id="p0485" role="paragraph">The δ<sup>13</sup>C and δ<sup>15</sup>N ratios of the weighed-out bone collagen were measured using a Thermo Scientific Flash 2000 Elemental Analyzer coupled to a Thermo Delta V Advantage Isotope Ratio Mass Spectrometer at the Stable Isotope Laboratory of the Department of Archaeology, Max Planck Institute for the Science of Human History, Jena. Values are reported as the ratio of the heavier isotope to the lighter isotope (<sup>13</sup>C/<sup>12</sup>C or <sup>15</sup>N/<sup>14</sup>N) as δ values in parts per mill (‰) relative to international standards, VPDB for δ<sup>13</sup>C and atmospheric N2 (AIR) for δ<sup>15</sup>N. The results reported for the samples were calibrated against international standards of (IAEA-CH-6: δ<sup>13</sup>C&nbsp;= −10.80 ± 0.47 ‰, IAEA-N-2: δ<sup>15</sup>N&nbsp;= 20.3 ± 0.2 ‰, and USGS40: δ<sup>13</sup>C&nbsp;= −26.38 ± 0.042 ‰, δ<sup>15</sup>N&nbsp;= 4.5 ± 0.1 ‰). Machine error was determined using repeat runs of a laboratory standard (fish gelatin: δ<sup>13</sup>C&nbsp;= ∼-15.1 ‰, δ<sup>15</sup>N&nbsp;= ∼14.3 ‰). Based on replicate analyses machine error over the course of a year is ± 0.2‰ for δ<sup>13</sup>C and ± 0.2‰ for δ<sup>15</sup>N. Overall measurement precision was studied through the measurement of repeats of fish gelatin (n&nbsp;= 80, ± 0.2‰ for δ<sup>13</sup>C and ± 0.2‰ for δ<sup>15</sup>N).</div><div id="p0490" role="paragraph">The resulting δ<sup>13</sup>C and δ<sup>15</sup>N values were used to independently estimate the potential impacts on the ‘reservoir effect’ of radiocarbon ages from the skeletons caused by the consumption of freshwater fish in the Lena River and Angara River regions. There has been a long history of research into the ‘reservoir’ effects of prehistoric individuals living in this region based on the paired radiocarbon dating of human bone and associated animal remains from the same graves, and correlations have been found between the extent of the offset in radiocarbon (<sup>14</sup>C) years and human stable isotope values (<span class="dropBlock reference-citations"><a id="body-ref-sref127-4" href="#" role="doc-biblioref" data-xml-rid="bib110 bib126 bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127-4" aria-controls="sref127-4">Ramsey et&nbsp;al., 2014; Schulting et&nbsp;al., 2014, 2015</a><div class="dropBlock__holder" data-db-target-of="sref127-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Ramsey, C.B. ∙ Schulting, R. ∙ Goriunova, O.I. ...</div><div><strong>Analyzing Radiocarbon Reservoir Offsets Through Stable Nitrogen Isotopes and Bayesian Modeling: A Case Study Using Paired Human and Faunal Remains from the Cis-Baikal Region, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:789-799</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17160" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899887915" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17160" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Schulting, R.J. ∙ Ramsey, C.B. ∙ Bazaliiskii, V.I. ...</div><div><strong>Freshwater Reservoir Offsets Investigated Through Paired Human-Faunal 14C Dating and Stable Carbon and Nitrogen Isotope Analysis at Lake Baikal, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:991-1008</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899833606" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Schulting, R.J. ∙ Bronk Ramsey, C. ∙ Bazaliiskii, V.I. ...</div><div><strong>Highly Variable Freshwater Reservoir Offsets Found along the Upper Lena Watershed, Cis-Baikal, Southeast Siberia</strong></div><div><em>Radiocarbon.</em> 2015; <strong>57</strong>:581-593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_rc.57.18458" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84942103801" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_rc.57.18458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we used established equations from <span class="dropBlock reference-citations"><a id="body-ref-sref127-5" href="#" role="doc-biblioref" data-xml-rid="bib126 bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127-5" aria-controls="sref127-5">Schulting et&nbsp;al. (2014, 2015</a><div class="dropBlock__holder" data-db-target-of="sref127-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Schulting, R.J. ∙ Ramsey, C.B. ∙ Bazaliiskii, V.I. ...</div><div><strong>Freshwater Reservoir Offsets Investigated Through Paired Human-Faunal 14C Dating and Stable Carbon and Nitrogen Isotope Analysis at Lake Baikal, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:991-1008</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899833606" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Schulting, R.J. ∙ Bronk Ramsey, C. ∙ Bazaliiskii, V.I. ...</div><div><strong>Highly Variable Freshwater Reservoir Offsets Found along the Upper Lena Watershed, Cis-Baikal, Southeast Siberia</strong></div><div><em>Radiocarbon.</em> 2015; <strong>57</strong>:581-593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_rc.57.18458" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84942103801" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_rc.57.18458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to estimate the radiocarbon offset based on our data. For individuals from the Lena River we have used Schulting et&nbsp;al.’s (2015, Table 3 pp. 586) “Full data set, δ<sup>13</sup>C” equation for the Early Neolithic samples and the “EBA sites, δ<sup>13</sup>C and δ<sup>15</sup>N” equation for Early Bronze Age samples. For individuals from the Angara River we have used Schulting et&nbsp;al.’s “SW Baikal/Angara<sup>2</sup>” Equation (2014, Table 4 pp. 998) that is based on their treating analyzed graves separately. Due to the fact that we are analyzing petrous bones and dentine, however, it should be noted that our reservoir estimates are best treated as ‘maximum’ estimates given the potential impact of the ‘weaning effect’ on δ<sup>13</sup>C and δ<sup>15</sup>N, particularly on First Molars (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Fuller et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Fuller, B.T. ∙ Richards, M.P. ∙ Mays, S.A.</div><div><strong>Stable carbon and nitrogen isotope variations in tooth dentine serial sections from Wharram Percy</strong></div><div><em>J.&nbsp;Archaeol. Sci.</em> 2003; <strong>30</strong>:1673-1684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0305-4403(03)00073-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0344962345" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0305-4403%2803%2900073-6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and even perhaps petrous bones (<span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Jørkov et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Jørkov, M.L.S. ∙ Heinemeier, J. ∙ Lynnerup, N.</div><div><strong>The petrous bone--a new sampling site for identifying early dietary patterns in stable isotopic studies</strong></div><div><em>Am. J. Phys. Anthropol.</em> 2009; <strong>138</strong>:199-209</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.20919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-59949101061" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18773469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.20919&amp;pmid=18773469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0495" role="paragraph">The δ<sup>13</sup>C and δ<sup>15</sup>N results of the samples analyzed are shown in <a id="crosref0990" href="#mmc1">Table S1</a>. The data is broadly consistent with individuals living in a predominantly C<sub>3</sub> environment and consuming significant amounts of freshwater resources (see also <span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Katzenberg et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Katzenberg, M.A. ∙ Bazaliiskii, V.I. ∙ Goriunova, O.I. ...</div><div><strong>8. Diet Reconstruction of Prehistoric Hunter-Gatherers in the Lake Baikal Region</strong></div><div><div>Weber, A.W. ∙ Katzenberg, M.A. ∙ Schurr, T.G. (Editors)</div></div><div><strong>Prehistoric Hunter-Gatherers of the Baikal Region, Siberia</strong></div><div>University of Pennsylvania Press, 2010; 175-192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.9783/9781934536391.175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.9783%2F9781934536391.175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). There is a distinction between individuals excavated along the Lena and Angara Rivers, with the latter having higher δ<sup>13</sup>C and δ<sup>15</sup>N suggesting increasing amounts of freshwater resources in the diets of these individuals (<a id="crosref1000" href="#mmc1">Table S1</a>). <span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Katzenberg et&nbsp;al. (2010)</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Katzenberg, M.A. ∙ Bazaliiskii, V.I. ∙ Goriunova, O.I. ...</div><div><strong>8. Diet Reconstruction of Prehistoric Hunter-Gatherers in the Lake Baikal Region</strong></div><div><div>Weber, A.W. ∙ Katzenberg, M.A. ∙ Schurr, T.G. (Editors)</div></div><div><strong>Prehistoric Hunter-Gatherers of the Baikal Region, Siberia</strong></div><div>University of Pennsylvania Press, 2010; 175-192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.9783/9781934536391.175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.9783%2F9781934536391.175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> reported Late Neolithic and Bronze Age human δ<sup>13</sup>C of between −20 and −19 ‰ for the Lena and Angara Rivers, with δ<sup>15</sup>N of <i>c.</i> 10.0 ‰. Our human data (δ<sup>13</sup>C&nbsp;= <i>c.</i> −20 to −16 ‰; δ<sup>15</sup>N <i>c.</i> 10 to 16 ‰) is consistent with interpretations of a C<sub>3</sub>-dominated diet, though with significantly varying, and often greater, contributions of freshwater fish to the diet relative to Katzenberg et&nbsp;al.’s dataset for the same region. While some of this difference is likely a product of the fact that one of the individuals with the highest δ<sup>13</sup>C and δ<sup>15</sup>N is dated to the Early Neolithic at Angara (where more fish consumption was also noted by <span class="dropBlock reference-citations"><a id="body-ref-sref48-3" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-3" aria-controls="sref48-3">Katzenberg et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref48-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Katzenberg, M.A. ∙ Bazaliiskii, V.I. ∙ Goriunova, O.I. ...</div><div><strong>8. Diet Reconstruction of Prehistoric Hunter-Gatherers in the Lake Baikal Region</strong></div><div><div>Weber, A.W. ∙ Katzenberg, M.A. ∙ Schurr, T.G. (Editors)</div></div><div><strong>Prehistoric Hunter-Gatherers of the Baikal Region, Siberia</strong></div><div>University of Pennsylvania Press, 2010; 175-192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.9783/9781934536391.175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.9783%2F9781934536391.175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and another individual with highest δ<sup>13</sup>C and δ<sup>15</sup>N is represented by an M1 that is likely influenced by the breastfeeding effect, it is clear that even on a cross-tissue basis, freshwater resources may have resulted in radiocarbon offsets in a number of these individuals (<a id="crosref1015" href="#mmc1">Table S1</a>). The calculated offsets are shown in <a id="crosref1020" href="#mmc1">Table S1</a>, with individuals from the Lena River having offsets between 161 and 618 years (with many centering around 300 years) and those from the Angara River having offsets from between 246 and 485 years (with 3 of the 4 individuals centering around 400 years) (<a id="crosref1025" href="#mmc1">Table S1</a>).</div></section></section><section id="sec-4-4-3"><h4>Strontium isotope analysis</h4><section id="sec-4-4-3-1"><h5>Background</h5><div id="p0500" role="paragraph">The use of strontium isotope ratios (<sup>87</sup>Sr/<sup>86</sup>Sr) to determine human mobility rests on the basic concept that rocks of different ages and compositions have distinctive values that do not fractionate from the bedrock into the biosphere (including the tissues of any individual measured) (<span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib37 bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Graustein and Armstrong, 1983; Montgomery, 2010</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Graustein, W.C. ∙ Armstrong, R.L.</div><div><strong>The use of strontium-87/strontium-86 ratios to measure atmospheric transport into forested watersheds</strong></div><div><em>Science.</em> 1983; <strong>219</strong>:289-292</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.219.4582.289" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0020641791" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17798277/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.219.4582.289&amp;pmid=17798277" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Montgomery, J.</div><div><strong>Passports from the past: Investigating human dispersals using strontium isotope analysis of tooth enamel</strong></div><div><em>Ann. Hum. Biol.</em> 2010; <strong>37</strong>:325-346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/03014461003649297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951685399" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20367186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F03014461003649297&amp;pmid=20367186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Older rocks have relatively higher <sup>87</sup>Sr/<sup>86</sup>Sr when compared to younger rocks, while different mineral contents (with different <sup>87</sup>Sr/<sup>86</sup>Sr ratios) leads to characteristic <sup>87</sup>Sr/<sup>86</sup>Sr based on rock ‘type’ (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Capo et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Capo, R.C. ∙ Stewart, B.W. ∙ Chadwick, O.A.</div><div><strong>Strontium isotopes as tracers of ecosystem processes: theory and methods</strong></div><div><em>Geoderma.</em> 1998; <strong>82</strong>:197-225</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0016-7061(97)00102-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0031943786" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0016-7061%2897%2900102-X" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example limestones have lower Rb/Sr ratios than continental granites (<span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Montgomery, 2010</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Montgomery, J.</div><div><strong>Passports from the past: Investigating human dispersals using strontium isotope analysis of tooth enamel</strong></div><div><em>Ann. Hum. Biol.</em> 2010; <strong>37</strong>:325-346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/03014461003649297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951685399" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20367186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F03014461003649297&amp;pmid=20367186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While there is a lack of mass-dependent fractionation from the bedrock into the foodchain, bedrock <sup>87</sup>Sr/<sup>86</sup>Sr does not always correlate with bioavailable <sup>87</sup>Sr/<sup>86</sup>Sr. Differential weathering of rocks with different <sup>87</sup>Sr/<sup>86</sup>Sr ratios, as well as the geographic variability of hydrology and aeolian transport, lead to variations in the <sup>87</sup>Sr/<sup>86</sup>Sr of soils and plants overlying a given distribution of rocks (<span class="dropBlock reference-citations"><a id="body-ref-sref75-3" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-3" aria-controls="sref75-3">Montgomery, 2010</a><div class="dropBlock__holder" data-db-target-of="sref75-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Montgomery, J.</div><div><strong>Passports from the past: Investigating human dispersals using strontium isotope analysis of tooth enamel</strong></div><div><em>Ann. Hum. Biol.</em> 2010; <strong>37</strong>:325-346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/03014461003649297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951685399" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20367186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F03014461003649297&amp;pmid=20367186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0505" role="paragraph">From plants, Sr<sup>2+</sup> actively replaces Ca<sup>2+</sup> in consumer tissues as part of the process of nutrient uptake and excretion (<span class="dropBlock reference-citations"><a id="body-ref-sref121" href="#" role="doc-biblioref" data-xml-rid="bib75 bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121" aria-controls="sref121">Montgomery, 2010; Rokita et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref121"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Montgomery, J.</div><div><strong>Passports from the past: Investigating human dispersals using strontium isotope analysis of tooth enamel</strong></div><div><em>Ann. Hum. Biol.</em> 2010; <strong>37</strong>:325-346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/03014461003649297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951685399" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20367186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F03014461003649297&amp;pmid=20367186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Rokita, E. ∙ Hermes, C. ∙ Nolting, H.-F. ...</div><div><strong>Substitution of calcium by strontium within selected calcium phosphates</strong></div><div><em>J.&nbsp;Cryst. Growth.</em> 1993; <strong>130</strong>:543-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-0248(93)90543-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0027608865" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-0248%2893%2990543-6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with the amount of strontium incorporated into the skeleton believed to be directly reflective of that available from the diet (and environment) (<span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib75 bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Montgomery, 2010; Price et&nbsp;al., 1986</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Montgomery, J.</div><div><strong>Passports from the past: Investigating human dispersals using strontium isotope analysis of tooth enamel</strong></div><div><em>Ann. Hum. Biol.</em> 2010; <strong>37</strong>:325-346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/03014461003649297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951685399" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20367186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F03014461003649297&amp;pmid=20367186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Price, T.D. ∙ Swick, R.W. ∙ Chase, E.P.</div><div><strong>Bone chemistry and prehistoric diet: strontium studies of laboratory rats</strong></div><div><em>Am. J. Phys. Anthropol.</em> 1986; <strong>70</strong>:365-375</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.1330700311" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0022743649" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3752231/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.1330700311&amp;pmid=3752231" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the context of tooth enamel from permanent teeth, Sr<sup>2+</sup> (and <sup>87</sup>Sr/<sup>86</sup>Sr ratios) will be incorporated during the process of mineralizaton which will vary depending on the tooth sampled anywhere between ten weeks before birth and 16 years of age (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Hillson, 1996</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Hillson, S.</div><div><strong>Dental Anthropology</strong></div><div>Cambridge University Press, 1996</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/CBO9781139170697" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FCBO9781139170697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). By sampling teeth that form during different periods of life, and comparing the <sup>87</sup>Sr/<sup>86</sup>Sr values of these teeth to bioavailable <sup>87</sup>Sr/<sup>86</sup>Sr values in the burial locale of an individuals, it is possible to determine whether someone grew up in a different location to that in which they were buried, thus determining potential migration during the life of an individual (<span class="dropBlock reference-citations"><a id="body-ref-sref75-6" href="#" role="doc-biblioref" data-xml-rid="bib9 bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-6" aria-controls="sref75-6">Bentley et&nbsp;al., 2003; Montgomery, 2010</a><div class="dropBlock__holder" data-db-target-of="sref75-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Bentley, R.A. ∙ Krause, R. ∙ Price, T.D. ...</div><div><strong>Human mobility at the early neolithic settlement of vaihingen, germany: evidence from strontium isotope analysis<sup>∗</sup></strong></div><div><em>Archaeometry.</em> 2003; <strong>45</strong>:471-486</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/1475-4754.00122" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0013196242" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2F1475-4754.00122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Montgomery, J.</div><div><strong>Passports from the past: Investigating human dispersals using strontium isotope analysis of tooth enamel</strong></div><div><em>Ann. Hum. Biol.</em> 2010; <strong>37</strong>:325-346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/03014461003649297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951685399" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20367186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F03014461003649297&amp;pmid=20367186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Robustness and detail of the bioavailable baseline created is important as the <sup>87</sup>Sr/<sup>86</sup>Sr of an individual will be the combined consequence of all food consumed and all water drunk, as well as the extent of the range over which it obtains food and drink (significant in the context of trade in food or presence of long rivers) (<span class="dropBlock reference-citations"><a id="body-ref-sref75-7" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-7" aria-controls="sref75-7">Montgomery, 2010</a><div class="dropBlock__holder" data-db-target-of="sref75-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Montgomery, J.</div><div><strong>Passports from the past: Investigating human dispersals using strontium isotope analysis of tooth enamel</strong></div><div><em>Ann. Hum. Biol.</em> 2010; <strong>37</strong>:325-346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/03014461003649297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951685399" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20367186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F03014461003649297&amp;pmid=20367186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which can complicate interpretation.</div></section><section id="sec-4-4-3-2"><h5>Strontium isotope analysis</h5><div id="p0510" role="paragraph">We sampled a total of 11 human teeth (<a id="crosref1055" href="#mmc1">Table S1</a>) from the Angara River Valley (GLZ001, GLZ002, GLZ003, ANG001) and the broader Baikal region (KAG002, KPT002-KPT006, ZPL002) for strontium isotope analysis. The teeth sampled were permanent 1<sup>st</sup>, 2<sup>nd</sup> and 3<sup>rd</sup> molars, which represent different periods of life (0-3 years, 3-7 years and adult respectively as per <span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Alexander Bentley, 2006</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Alexander Bentley, R.</div><div><strong>Strontium Isotopes from the Earth to the Archaeological Skeleton: A Review</strong></div><div><em>J.&nbsp;Archaeol. Method Theory.</em> 2006; <strong>13</strong>:135-187</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10816-006-9009-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-33749026059" target="_blank" aria-label="Open Scopus in new tab">Scopus (631)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10816-006-9009-x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), to see if there were any distinctions between their <sup>87</sup>Sr/<sup>86</sup>Sr ratios during childhood and the bioavailable <sup>87</sup>Sr/<sup>86</sup>Sr of their burial location. We used the comprehensive dataset of <span class="dropBlock reference-citations"><a id="body-ref-sref39-3" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-3" aria-controls="sref39-3">Haverkort et&nbsp;al. (2010)</a><div class="dropBlock__holder" data-db-target-of="sref39-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Haverkort, C.M. ∙ Bazaliiskii, V.I. ∙ Savel’ev, N.A.</div><div><strong>Identifying hunter-gatherer mobility patterns using strontium isotopes</strong></div><div><div>Weber, A. ∙ Katzenberg, A. ∙ Schurr, T.G. (Editors)</div></div><div><strong>Prehistoric Hunter-Gatherers of the Baikal Region, Siberia</strong></div><div>Penn Press, 2010; 217-238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.9783/9781934536391.217" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.9783%2F9781934536391.217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> that includes modern and archaeological fauna from terrestrial and aquatic settings as an indication of the local baseline.</div><div id="p0515" role="paragraph">For each tooth, adhering sediments were removed and teeth were cleaned ultrasonically in ultra-pure water (deionized water). Teeth were then air-dried overnight and a groove was made along the buccal edge of the tooth, in order to average the signal for the period of enamel formation (<span class="dropBlock reference-citations"><a id="body-ref-sref3-3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-3" aria-controls="sref3-3">Alexander Bentley, 2006</a><div class="dropBlock__holder" data-db-target-of="sref3-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Alexander Bentley, R.</div><div><strong>Strontium Isotopes from the Earth to the Archaeological Skeleton: A Review</strong></div><div><em>J.&nbsp;Archaeol. Method Theory.</em> 2006; <strong>13</strong>:135-187</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10816-006-9009-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-33749026059" target="_blank" aria-label="Open Scopus in new tab">Scopus (631)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10816-006-9009-x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), using a Dremel drill at the Stable Isotope Laboratory of the Department of Archaeology, Max Planck Institute for the Science of Human History, Jena, Germany.</div><div id="p0520" role="paragraph">Drill powdered samples of 20&nbsp;mg were then shipped to the clean laboratory in the Department of Geological Sciences at the University of Cape Town. Samples were dissolved in 2&nbsp;mL 65% 2B HNO<sub>3</sub> in a closed Teflon beaker and placed for one hour on a hotplate at 140°C. The samples were then dried down and re-dissolved in 1.5&nbsp;mL 2&nbsp;M HNO<sub>3</sub>. Strontium separation chemistry followed methods discussed in <span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Pin et&nbsp;al. (1994)</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Pin, C. ∙ Briot, D. ∙ Bassin, C. ...</div><div><strong>Concomitant separation of strontium and samarium-neodymium for isotopic analysis in silicate samples, based on specific extraction chromatography</strong></div><div><em>Anal. Chim. Acta.</em> 1994; <strong>298</strong>:209-217</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0003-2670(94)00274-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0028127258" target="_blank" aria-label="Open Scopus in new tab">Scopus (393)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0003-2670%2894%2900274-6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. After separation, the solutions for each sample were dried, dissolved in 2&nbsp;mL 0.2% HNO<sub>3</sub> and diluted to 200 ppb Sr concentrations for strontium isotope analysis. Radiogenic <sup>87</sup>Sr/<sup>86</sup>Sr ratios were measured using a Nu Instruments NuPlasma HR MC-ICP-MS in the Department of Geological Sciences at the University of Cape Town. Sample values were corrected for instrumental mass fractionation using an <sup>86</sup>Sr/<sup>88</sup>Sr ratio of 0.1194 (<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Nier, 1938</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Nier, A.O.</div><div><strong>The Isotopic Constitution of Strontium, Barium, Bismuth, Thallium and Mercury</strong></div><div><em>Phys. Rev.</em> 1938; <strong>54</strong>:275-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRev.54.275" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0006029013" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRev.54.275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and isobaric <sup>87</sup>Rb interference using the measured <sup>85</sup>Rb signal and natural Rb isotope ratios. All data presented are referenced to bracketing analyses of NIST SRM987 (<sup>87</sup>Sr/<sup>86</sup>Sr reference value of 0.710255). Results for repeat analyses of an in-house carbonate standard (NM95) processed and measured with the batches of unknown samples in this study gave an <sup>87</sup>Sr/<sup>86</sup>Sr ratio of 0.708885 (2σ&nbsp;= 0.000010; n&nbsp;= 40) are in agreement with long-term results for this in-house standard having an average <sup>87</sup>Sr/<sup>86</sup>Sr ratio of 0.708911 (2σ&nbsp;= 0.000040; n&nbsp;= 414).</div></section></section><section id="sec-4-4-4"><h4>Ancient DNA processing</h4><div id="p0525" role="paragraph">All samples were processed in dedicated laboratories at the Max Planck Institute for the Science of Human History in Jena, Germany, except the single-stranded library of sample UKY001.B that was generated at the Max Planck Institute for Evolutionary Anthropology in Leipzig, Germany. Bone powder for DNA extraction was obtained from petrous bones by drilling the densest osseous matter around the cochlea, and from teeth by cutting at the junction between the root and crown, sampling the dental pulp. The sample UKY001 was already fragmented and two fragments were milled using the MM200 mixer mill (Retsch). The resulting bone powder was split in three equal aliquots. For detailed information on the analyzed samples, their archaeological context and radiocarbon age see <a id="crosref1085" href="#mmc1">Table S1</a> and <a id="crosref1090" href="#sec-4">STAR Methods</a>.</div><div id="p0530" role="paragraph">DNA from the 19 ancient individuals was extracted following established protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Dabney et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Dabney, J. ∙ Knapp, M. ∙ Glocke, I. ...</div><div><strong>Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:15758-15763</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1314445110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84884624229" target="_blank" aria-label="Open Scopus in new tab">Scopus (459)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24019490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1314445110&amp;pmid=24019490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The sample UKY001.B was pre-treated with a washing step to reduce surface contamination (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Korlević et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Korlević, P. ∙ Gerber, T. ∙ Gansauge, M.-T. ...</div><div><strong>Reducing microbial and human contamination in DNA extractions from ancient bones and teeth</strong></div><div><em>Biotechniques.</em> 2015; <strong>59</strong>:87-93</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2144/000114320" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84938877820" target="_blank" aria-label="Open Scopus in new tab">Scopus (88)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26260087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2144%2F000114320&amp;pmid=26260087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A negative and cave bear positive controls were included. To release DNA from 30–100&nbsp;mg of bone powder, a solution of 900&nbsp;μL EDTA, 75&nbsp;μL H2O and 25&nbsp;μL Proteinase K was added. In a rotator, samples were digested for at least 16&nbsp;h at 37°C, for UKY001.B this was followed by an additional hour at 56°C (<span class="dropBlock reference-citations"><a id="body-ref-sref119" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119" aria-controls="sref119">Rohland and Hofreiter, 2007</a><div class="dropBlock__holder" data-db-target-of="sref119"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Rohland, N. ∙ Hofreiter, M.</div><div><strong>Ancient DNA extraction from bones and teeth</strong></div><div><em>Nat. Protoc.</em> 2007; <strong>2</strong>:1756-1762</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2007.247" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-34548579114" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17641642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2007.247&amp;pmid=17641642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The suspension was then centrifuged and transferred into a binding buffer as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Dabney et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Dabney, J. ∙ Knapp, M. ∙ Glocke, I. ...</div><div><strong>Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:15758-15763</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1314445110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84884624229" target="_blank" aria-label="Open Scopus in new tab">Scopus (459)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24019490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1314445110&amp;pmid=24019490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To bind DNA, silica columns for high volumes (High Pure Viral Nucleic Acid Large Volume Kit; Roche) were used. After two washing steps using the manufacturer’s wash buffer, DNA was eluted in TET (10&nbsp;mM Tris, 1&nbsp;mM EDTA and 0.05% Tween) in two steps for a final volume of 100&nbsp;μl.</div><div id="p0535" role="paragraph">Double-stranded DNA libraries were built from 25&nbsp;μL of DNA extract in the presence of uracil DNA glycosylase, following a double-stranded ‘UDG-half’ library preparation to reduce, but not eliminate, the amount of deamination-induced damage toward the ends of ancient DNA (aDNA) fragments (<span class="dropBlock reference-citations"><a id="body-ref-sref120" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120" aria-controls="sref120">Rohland et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref120"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Rohland, N. ∙ Harney, E. ∙ Mallick, S. ...</div><div><strong>Partial uracil-DNA-glycosylase treatment for screening of ancient DNA</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2015; <strong>370</strong>:20130624</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2013.0624" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84916210143" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25487342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2013.0624&amp;pmid=25487342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Negative and positive controls were carried alongside each experiment. Libraries were quantified using the IS7 and IS8 primers (<span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Meyer and Kircher, 2010</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina Sequencing Library Preparation for Highly Multiplexed Target Capture and Sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb.prot5448-pdb.prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (825)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in a quantification assay using a DyNAmo SYBP Green qPCR Kit (Thermo Fisher Scientific) on the LightCycler 480 (Roche). Each aDNA library was double indexed (<span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Kircher et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kircher, M. ∙ Sawyer, S. ∙ Meyer, M.</div><div><strong>Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:e3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr771" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84855295457" target="_blank" aria-label="Open Scopus in new tab">Scopus (461)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22021376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr771&amp;pmid=22021376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in 1–4 parallel 100&nbsp;μ l reactions using PfuTurbo DNA Polymerase (Agilent). The indexed products for each library were pooled, purified over MinElute columns (QIAGEN), eluted in 50&nbsp;μL TET and again quantified using the IS5 and IS6 primers (<span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Meyer and Kircher, 2010</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina Sequencing Library Preparation for Highly Multiplexed Target Capture and Sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb.prot5448-pdb.prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (825)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using the quantification method described above. 4&nbsp;μ l of the purified product were amplified in multiple 100&nbsp;μL reactions using Herculase II Fusion DNA Polymerase (Agilent) following the manufacturer’s specifications with 0.3&nbsp;μ M of the IS5/IS6 primers. After another MinElute purification, the product was quantified using the Agilent 2100 Bioanalyzer DNA 1000 chip. An equimolar pool of all libraries was then prepared for shotgun sequencing on Illumina Hiseq4000 platform using 75bp single-end reads for screening.</div><div id="p0540" role="paragraph">The single-stranded DNA library for sample UKY001.B was built from 30&nbsp;μL of DNA extract in the absence of uracil DNA glycosylase (non-UDG library) followed by double indexing, using an automated version of the protocols described in <span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Gansauge et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Gansauge, M.-T. ∙ Gerber, T. ∙ Glocke, I. ...</div><div><strong>Single-stranded DNA library preparation from highly degraded DNA using T4 DNA ligase</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong></div><div>e79–e79</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28119419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=28119419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and <span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Kircher et&nbsp;al. (2012)</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kircher, M. ∙ Sawyer, S. ∙ Meyer, M.</div><div><strong>Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:e3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr771" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84855295457" target="_blank" aria-label="Open Scopus in new tab">Scopus (461)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22021376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr771&amp;pmid=22021376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> on a liquid handling system (Agilent Technologies Bravo NGS Workstation).</div></section><section id="sec-4-4-5"><h4>Human genome enrichment and sequencing</h4><div id="p0545" role="paragraph">Both double-stranded UDG-half and single-stranded non-UDG-treated libraries were further amplified with IS5/IS6 primers to reach a concentration of 200–400&nbsp;ng μ l–1 as measured on a NanoDrop spectrophotometer (Thermo Fisher Scientific). Mitochondrial DNA capture (<span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Fu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (222)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was performed on screened double-stranded libraries, after shotgun sequencing, showed the presence of aDNA, highlighted by the typical C-to-T and G-to-A substitution pattern toward 5′ and 3′ molecule ends, respectively (<a id="crosref1150" href="#mmc1">Table S1</a>). Eight of the libraries with percentages of human DNA over 10% were deep shotgun sequenced on an Illumina HiSeq 4000 instrument with 75 pair-end-run cycles using the manufacturer’s protocol. Furthermore, samples with a percentage of human DNA in shotgun data around 0.1% or greater were enriched for a set of 1,237,207 targeted SNPs (1240K capture) across the human genome (<span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Fu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (259)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The enriched DNA product was sequenced on an Illumina HiSeq 4000 instrument with 75 single-end-run cycles using the manufacturer’s protocol.</div><div id="p0550" role="paragraph">The de-multiplexed capture sequencing reads were cleaned and mapped to human reference genome hs37d5 using EAGER pipeline 1.92.55 (<span class="dropBlock reference-citations"><a id="body-ref-sref97-4" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-4" aria-controls="sref97-4">Peltzer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref97-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Peltzer, A. ∙ Jäger, G. ∙ Herbig, A. ...</div><div><strong>EAGER: efficient ancient genome reconstruction</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:60</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0918-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27036623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0918-z&amp;pmid=27036623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Within the pipeline, the adapters were removed by AdapterRemoval 2.2.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref124-2" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-2" aria-controls="sref124-2">Schubert et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref124-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Schubert, M. ∙ Lindgreen, S. ∙ Orlando, L.</div><div><strong>AdapterRemoval v2: rapid adapter trimming, identification, and read merging</strong></div><div><em>BMC Res. Notes.</em> 2016; <strong>9</strong>:88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13104-016-1900-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84958153152" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26868221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13104-016-1900-2&amp;pmid=26868221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), reads were mapped with BWA 0.7.12 aln/samse algorithm (<span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (17717)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), duplications were removed by DeDup 0.12.1 (<a id="intref0205" href="https://github.com/apeltzer/DeDup" target="_blank">https://github.com/apeltzer/DeDup</a>) and damage patterns of each library were checked with mapDamage 2.0.6 (<span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Jónsson et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Jónsson, H. ∙ Ginolhac, A. ∙ Schubert, M. ...</div><div><strong>mapDamage2.0: fast approximate Bayesian estimates of ancient DNA damage parameters</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1682-1684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84879910635" target="_blank" aria-label="Open Scopus in new tab">Scopus (398)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23613487/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt193&amp;pmid=23613487" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Then we masked 2bp from both ends of the reads from double-stranded libraries with trimBam in bamUtil 1.0.13 (<a id="intref0210" href="https://github.com/statgen/bamUtil" target="_blank">https://github.com/statgen/bamUtil</a>) to remove the damaged sites.</div><div id="p0555" role="paragraph">The mitochondrial capture sequencing reads were cleaned by AdapterRemoval 2.2.0 to remove the adapters. Then the cleaned reads were mapped to human reference mitochondrial sequence GenBank: <a id="interref255" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC_012920.1" target="_blank">NC_012920.1</a> with BWA 0.7.12 aln/samse algorithm and realigned with CircularMapper (<span class="dropBlock reference-citations"><a id="body-ref-sref97-5" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-5" aria-controls="sref97-5">Peltzer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref97-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Peltzer, A. ∙ Jäger, G. ∙ Herbig, A. ...</div><div><strong>EAGER: efficient ancient genome reconstruction</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:60</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0918-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27036623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0918-z&amp;pmid=27036623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). After removing duplication with DeDup, the consensus sequences were generated by Schmutzi (<span class="dropBlock reference-citations"><a id="body-ref-sref118-2" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-2" aria-controls="sref118-2">Renaud et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref118-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Renaud, G. ∙ Slon, V. ∙ Duggan, A.T. ...</div><div><strong>Schmutzi: estimation of contamination and endogenous mitochondrial consensus calling for ancient DNA</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0776-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84943738237" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26458810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0776-0&amp;pmid=26458810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-6"><h4><i>Y. pestis</i> genome enrichment</h4><div id="p0560" role="paragraph">Samples GLZ001 and GLZ002 that were putatively positive for <i>Y.&nbsp;pestis</i> were also converted into Illumina double-stranded libraries with 50&nbsp;μL of input DNA extract. As post-mortem aDNA cytosine deamination can influence read mapping and SNP identification, we used an initial uracil-DNA-glycosylase (UDG) and endonuclease VIII treatment (USER enzyme, New England Biolabs) for uracil excision and subsequent DNA repair (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Briggs et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Briggs, A.W. ∙ Stenzel, U. ∙ Meyer, M. ...</div><div><strong>Removal of deaminated cytosines and detection of in&nbsp;vivo methylation in ancient DNA</strong></div><div><em>Nucleic Acids Res.</em> 2010; <strong>38</strong>:e87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkp1163" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77950862213" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20028723/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkp1163&amp;pmid=20028723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The rest of the library preparation steps were proceeded with according to a published protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref69-3" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-3" aria-controls="sref69-3">Meyer and Kircher, 2010</a><div class="dropBlock__holder" data-db-target-of="sref69-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina Sequencing Library Preparation for Highly Multiplexed Target Capture and Sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb.prot5448-pdb.prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (825)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Subsequently, the primer combination IS7/IS8 was used to quantify the resulting libraries using a qPCR (Roche). For a double-indexing step (<span class="dropBlock reference-citations"><a id="body-ref-sref51-3" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-3" aria-controls="sref51-3">Kircher et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref51-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kircher, M. ∙ Sawyer, S. ∙ Meyer, M.</div><div><strong>Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:e3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr771" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84855295457" target="_blank" aria-label="Open Scopus in new tab">Scopus (461)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22021376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr771&amp;pmid=22021376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), each library was assigned a unique combination of indexing primers (each primer containing a unique 8bp identifier), and was split into multiple PCR reactions based on the initial quantification (IS7/IS8) so that each indexing reaction does not exceed 2&nbsp;× 10<sup>10</sup> input DNA copies. The indexing PCR reaction was carried out for 10-cycles using the enzyme Pfu Turbo Cx Hotstart DNA Polymerase (Agilent). Reaction products were cleaned up using the MinElute DNA purification kit (QIAGEN), eluted in TET (10mM Tris-HCl, 1mM EDTA pH 8.0, 0.05% Tween20), and quantified using a qPCR (Roche) with the IS5/IS6 primer combination (<span class="dropBlock reference-citations"><a id="body-ref-sref69-4" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-4" aria-controls="sref69-4">Meyer and Kircher, 2010</a><div class="dropBlock__holder" data-db-target-of="sref69-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina Sequencing Library Preparation for Highly Multiplexed Target Capture and Sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb.prot5448-pdb.prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (825)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, indexed libraries were amplified using the enzyme Herculase II Fusion DNA Polymerase (Agilent) to achieve 1-2&nbsp;μg of total DNA in 7&nbsp;μl, which would be used as input for whole-genome <i>Y.&nbsp;pestis</i> enrichment. PCR products were purified with the MinElute DNA purification kit (QIAGEN) and eluted in EBT (QIAGEN EB, 0.05% Tween20). In-solution <i>Y.&nbsp;pestis</i> captures of both full-UDG and partially-UDG-treated libraries were performed using a published protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Fu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (222)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), adapted for <i>Y.&nbsp;pestis</i> as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref6-13" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-13" aria-controls="sref6-13">Andrades Valtueña et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section><section id="sec-4-5"><h3>Quantification and Statistical Analysis</h3><section id="sec-4-5-1"><h4>Genotyping and dataset preparation for analysis</h4><div id="p0565" role="paragraph">The cleaned reads with base quality and mapping quality over 30 were piled up with mpileup in SAMtools 1.3 (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Li et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The sequence alignment/map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (19724)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For double-stranded libraries, we called pseudo-haploid genotypes with pileupCaller 1.2.2 (<a id="intref0215" href="https://github.com/stschiff/sequenceTools" target="_blank">https://github.com/stschiff/sequenceTools</a>) under random haploid calling mode. For the single-stranded library UKY001.B0102, we called pseudo-haploid genotypes with pileupCaller 1.4.0.3 under single-strand mode using both shotgun and capture sequencing reads, which ignores forward reads at C/T polymorphisms and reverse reads at G/A polymorphisms. Then we combined our genotypes with published ancient individuals (<span class="dropBlock reference-citations"><a id="body-ref-sref130-2" href="#" role="doc-biblioref" data-xml-rid="bib4 bib20 bib19 bib30 bib42 bib43 bib44 bib60 bib67 bib77 bib89 bib112 bib113 bib114 bib122 bib123 bib129 bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130-2" aria-controls="sref130-2">Allentoft et&nbsp;al., 2015; Damgaard et&nbsp;al., 2018b, 2018a; Flegontov et&nbsp;al., 2019; Jeong et&nbsp;al., 2018, 2019; Jones et&nbsp;al., 2015; Lazaridis et&nbsp;al., 2016; Mathieson et&nbsp;al., 2015; Moreno-Mayar et&nbsp;al., 2018; Olalde et&nbsp;al., 2018; Rasmussen et&nbsp;al., 2010, 2014, 2015a; Saag et&nbsp;al., 2017; Scheib et&nbsp;al., 2018; Sikora et&nbsp;al., 2019; Siska et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref130-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Allentoft, M.E. ∙ Sikora, M. ∙ Sjögren, K.-G. ...</div><div><strong>Population genomics of Bronze Age Eurasia</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:167-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84930940396" target="_blank" aria-label="Open Scopus in new tab">Scopus (523)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26062507/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14507&amp;pmid=26062507" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Damgaard, P.B. ∙ Marchi, N. ∙ Rasmussen, S. ...</div><div><strong>137 ancient human genomes from across the Eurasian steppes</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:369-374</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0094-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047137672" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0094-2&amp;pmid=29743675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Flegontov, P. ∙ Altınışık, N.E. ∙ Changmai, P. ...</div><div><strong>Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:236-240</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1251-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85067075462" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1251-y&amp;pmid=31168094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jeong, C. ∙ Wilkin, S. ∙ Amgalantugs, T. ...</div><div><strong>Bronze Age population dynamics and the rise of dairy pastoralism on the eastern Eurasian steppe</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E11248-E11255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1813608115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85057166426" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30397125/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1813608115&amp;pmid=30397125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Jeong, C. ∙ Balanovsky, O. ∙ Lukianova, E. ...</div><div><strong>The genetic history of admixture across inner Eurasia</strong></div><div><em>Nat. Ecol. Evol.</em> 2019; <strong>3</strong>:966-976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-019-0878-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85065175002" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31036896/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-019-0878-2&amp;pmid=31036896" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Jones, E.R. ∙ Gonzalez-Fortes, G. ∙ Connell, S. ...</div><div><strong>Upper Palaeolithic genomes reveal deep roots of modern Eurasians</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:8912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms9912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84947563586" target="_blank" aria-label="Open Scopus in new tab">Scopus (148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26567969/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms9912&amp;pmid=26567969" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Lazaridis, I. ∙ Nadel, D. ∙ Rollefson, G. ...</div><div><strong>Genomic insights into the origin of farming in the ancient Near East</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:419-424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19310" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27459054/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19310&amp;pmid=27459054" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Mathieson, I. ∙ Lazaridis, I. ∙ Rohland, N. ...</div><div><strong>Genome-wide patterns of selection in 230 ancient Eurasians</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:499-503</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84951286136" target="_blank" aria-label="Open Scopus in new tab">Scopus (453)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26595274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16152&amp;pmid=26595274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Moreno-Mayar, J.V. ∙ Potter, B.A. ∙ Vinner, L. ...</div><div><strong>Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans</strong></div><div><em>Nature.</em> 2018; <strong>553</strong>:203-207</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85040467287" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29323294/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25173&amp;pmid=29323294" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Olalde, I. ∙ Brace, S. ∙ Allentoft, M.E. ...</div><div><strong>The Beaker phenomenon and the genomic transformation of northwest Europe</strong></div><div><em>Nature.</em> 2018; <strong>555</strong>:190-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85043325398" target="_blank" aria-label="Open Scopus in new tab">Scopus (150)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29466337/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25738&amp;pmid=29466337" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Rasmussen, M. ∙ Li, Y. ∙ Lindgreen, S. ...</div><div><strong>Ancient human genome sequence of an extinct Palaeo-Eskimo</strong></div><div><em>Nature.</em> 2010; <strong>463</strong>:757-762</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08835" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-76749141381" target="_blank" aria-label="Open Scopus in new tab">Scopus (464)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20148029/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08835&amp;pmid=20148029" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Rasmussen, M. ∙ Anzick, S.L. ∙ Waters, M.R. ...</div><div><strong>The genome of a Late Pleistocene human from a Clovis burial site in western Montana</strong></div><div><em>Nature.</em> 2014; <strong>506</strong>:225-229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84893961036" target="_blank" aria-label="Open Scopus in new tab">Scopus (260)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24522598/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13025&amp;pmid=24522598" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Rasmussen, M. ∙ Sikora, M. ∙ Albrechtsen, A. ...</div><div><strong>The ancestry and affiliations of Kennewick Man</strong></div><div><em>Nature.</em> 2015; <strong>523</strong>:455-458</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84937475681" target="_blank" aria-label="Open Scopus in new tab">Scopus (103)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26087396/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14625&amp;pmid=26087396" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Saag, L. ∙ Varul, L. ∙ Scheib, C.L. ...</div><div><strong>Extensive Farming in Estonia Started through a Sex-Biased Migration from the Steppe</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:2185-2193</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.06.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.06.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85023634728" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28712569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.06.022&amp;pmid=28712569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Scheib, C.L. ∙ Li, H. ∙ Desai, T. ...</div><div><strong>Ancient human parallel lineages within North America contributed to a coastal expansion</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:1024-1027</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar6851" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85048178453" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29853687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar6851&amp;pmid=29853687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Siska, V. ∙ Jones, E.R. ∙ Jeon, S. ...</div><div><strong>Genome-wide data from two early Neolithic East Asian individuals dating to 7700 years ago</strong></div><div><em>Sci. Adv.</em> 2017; <strong>3</strong>:e1601877</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1601877" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28164156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1601877&amp;pmid=28164156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and further intersected the dataset with Affymetrix Axiom Human Origins array SNPs in order to compare with worldwide modern populations. This final dataset of 539,124 SNPs on autosomes is used for downstream population analysis.</div><div id="p0570" role="paragraph">With shotgun sequencing reads of base quality over 30, we individually calculated the genotype likelihood on 27.9 million bi-allelic SNPs with minor allele count 5 or higher published in the 1000 Genomes Project Phase 3 dataset (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Auton et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (4710)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), using GATK UnifiedGenotyper 3.5 (<span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">DePristo et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>DePristo, M.A. ∙ Banks, E. ∙ Poplin, R. ...</div><div><strong>A framework for variation discovery and genotyping using next-generation DNA sequencing data</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:491-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.806" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-79955483667" target="_blank" aria-label="Open Scopus in new tab">Scopus (5329)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21478889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.806&amp;pmid=21478889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For half-UDG-treated double-stranded libraries, we used 2bp-trimmed reads for genotype likelihood calculation. For the single-stranded library, we ignored forward reads at C/T polymorphisms and reverse reads at G/A polymorphisms. For non-UDG-treated libraries of published ancient individuals from the Lake Baikal region (<span class="dropBlock reference-citations"><a id="body-ref-sref129-9" href="#" role="doc-biblioref" data-xml-rid="bib19 bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129-9" aria-controls="sref129-9">Damgaard et&nbsp;al., 2018a; Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref129-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we only calculated the genotype likelihood on transversion sites.</div><div id="p0575" role="paragraph">Then we applied GeneImp 1.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref132-2" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132-2" aria-controls="sref132-2">Spiliopoulou et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref132-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Spiliopoulou, A. ∙ Colombo, M. ∙ Orchard, P. ...</div><div><strong>GeneImp: Fast imputation to large reference panels using genotype likelihoods from ultralow coverage sequencing</strong></div><div><em>Genetics.</em> 2017; <strong>206</strong>:91-104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.117.200063" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85020715759" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28348060/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.117.200063&amp;pmid=28348060" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to impute diploid genotypes on the 1000 Genomes Phase 3 variations, with the genotype likelihood as input and the 2504 individuals in 1000 Genomes Phase 3 as reference. Following the developer’s guideline in the original publication, we ran imputations with three different window sizes, kl&nbsp;= 15, 20 and 25 and took the average GP (genotype call probabilities) as posterior probability for each imputed genotype.</div><div id="p0580" role="paragraph">Then all the genotypes with posterior probability of 0.99 or higher were extracted and 24 individuals with genotyping rate over 75% were included in downstream analysis (<a id="crosref1240" href="#mmc1">Table S1</a>), including one Upper Paleolithic UKY individual, one Mesolithic Kolyma individual, 12 Early Neolithic individuals and 10 Late Neolithic to Bronze Age individuals. We intersected the imputed SNPs with 1240K and HumanOrigins array SNPs, and kept 520,289 sites with over 50% genotyping rate in our 24 individuals for downstream phasing.</div><div id="p0585" role="paragraph">For the IBD analysis that requires phased genotype data, we jointly phased the 24 imputed individuals with 3014 worldwide modern individuals (<a id="crosref1245" href="#mmc1">Table S1</a>) using SHAPEIT v2.r837 (<span class="dropBlock reference-citations"><a id="body-ref-sref23-2" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-2" aria-controls="sref23-2">Delaneau et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref23-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Delaneau, O. ∙ Zagury, J.-F. ∙ Marchini, J.</div><div><strong>Improved whole-chromosome phasing for disease and population genetic studies</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:5-6</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2307" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84871952176" target="_blank" aria-label="Open Scopus in new tab">Scopus (649)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23269371/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2307&amp;pmid=23269371" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with a genetic map from 1000 Genomes Phase 3 data, under its default settings.</div><div id="p0590" role="paragraph">The sex of each individual was determined by coverage on sex chromosomes (<a id="crosref1255" href="#mmc1">Table S1</a>). Then Y chromosome haplogroups of male individuals were determined by yHaplo program (<span class="dropBlock reference-citations"><a id="body-ref-sref100-2" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-2" aria-controls="sref100-2">Poznik, 2016</a><div class="dropBlock__holder" data-db-target-of="sref100-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Poznik, G.D.</div><div><strong>Identifying Y-chromosome haplogroups in arbitrarily large samples of sequenced or genotyped men</strong></div><div><em>bioRxiv.</em> 2016;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/088716" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F088716" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0595" role="paragraph">The nuclear DNA contamination of male individuals was estimated by ANGSD 0.910 (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Korneliussen et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Korneliussen, T.S. ∙ Albrechtsen, A. ∙ Nielsen, R.</div><div><strong>ANGSD: Analysis of Next Generation Sequencing Data</strong></div><div><em>BMC Bioinformatics.</em> 2014; <strong>15</strong>:356</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-014-0356-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84920729412" target="_blank" aria-label="Open Scopus in new tab">Scopus (517)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25420514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-014-0356-4&amp;pmid=25420514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) based on mismatch rate on the X chromosome. For the single-stranded library without UDG-treatment, we kept only transversion sites for the estimation. The mitochondrial DNA contamination was estimated by schmutzi (<span class="dropBlock reference-citations"><a id="body-ref-sref118-3" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-3" aria-controls="sref118-3">Renaud et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref118-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Renaud, G. ∙ Slon, V. ∙ Duggan, A.T. ...</div><div><strong>Schmutzi: estimation of contamination and endogenous mitochondrial consensus calling for ancient DNA</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0776-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84943738237" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26458810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0776-0&amp;pmid=26458810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-2"><h4>Population genetics analysis</h4><div id="p0600" role="paragraph">The principal component analysis was carried out by smartpca in EIGENSOFT 6.0.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref91-2" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-2" aria-controls="sref91-2">Patterson et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref91-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Patterson, N. ∙ Price, A.L. ∙ Reich, D.</div><div><strong>Population structure and eigenanalysis</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020190" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-33846006923" target="_blank" aria-label="Open Scopus in new tab">Scopus (2283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17194218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020190&amp;pmid=17194218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with modern individuals used for calculation and all the ancient individuals projected on the calculated PCs. The “lsqproject: YES” parameter was used to minimize the effect of missing data in ancient individuals. The PCA was calculated with 2182 individuals from 181 Eurasian and Native American modern populations (<a id="crosref1280" href="#fig1">Figure&nbsp;1</a>C; <a id="crosref1285" href="#mmc1">Table S1</a>).</div><div id="p0605" role="paragraph">The unsupervised population clustering pattern was estimated by ADMIXTURE 1.3.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Alexander et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alexander, D.H. ∙ Novembre, J. ∙ Lange, K.</div><div><strong>Fast model-based estimation of ancestry in unrelated individuals</strong></div><div><em>Genome Res.</em> 2009; <strong>19</strong>:1655-1664</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.094052.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-69749099417" target="_blank" aria-label="Open Scopus in new tab">Scopus (2279)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.094052.109&amp;pmid=19648217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We included 3488 individuals in the ADMIXTURE analysis, including 2986 worldwide modern individuals, 483 selected ancient individuals and 19 individuals newly genotyped in this study. Before ADMIXTURE analysis, the dataset was filtered to 120,599 SNPs with PLINK 1.90 (<span class="dropBlock reference-citations"><a id="body-ref-sref104-2" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-2" aria-controls="sref104-2">Purcell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref104-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (14939)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to remove SNPs with minor allele frequency lower than 0.01 (–maf 0.01) and linked SNPs with r<sup>2</sup> &gt; 0.2 (–indep-pairwise 200 25 0.2). The ADMIXTURE analysis was carried out with population number (K) from 2 to 20, five duplications for each K value and we chose K&nbsp;= 16 as it gave the lowest cross-validation (CV) error.</div><div id="p0610" role="paragraph">We identified the ROH and shared IBD segments in the 24 shotgun sequenced individuals and 81 individuals from several modern Siberian populations (Buryat, Even, Evenk_FarEast, Evenk_Transbaikal, Nganasan, see <a id="crosref1300" href="#mmc1">Table S1</a>). The 24 ancient individuals are divided into four groups, namely UKY, Kolyma, Baikal EN and Baikal LNBA (<a id="crosref1305" href="#mmc1">Table S1</a>). The ROH was detected using PLINK v1.90b3.29 (<span class="dropBlock reference-citations"><a id="body-ref-sref104-3" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-3" aria-controls="sref104-3">Purcell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref104-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (14939)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with the “–homozyg” flag and default settings. The shared IBD segments between individuals were detected using Refined IBD algorithm in BEAGLE v4.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Browning and Browning, 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Browning, B.L. ∙ Browning, S.R.</div><div><strong>Improving the accuracy and efficiency of identity-by-descent detection in population data</strong></div><div><em>Genetics.</em> 2013; <strong>194</strong>:459-471</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.113.150029" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84878589007" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23535385/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.113.150029&amp;pmid=23535385" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), under the default parameter settings with the HapMap GRCh37 genetic map provided by BEAGLE (<a id="intref0220" href="http://bochet.gcc.biostat.washington.edu/beagle/genetic_maps/" target="_blank">http://bochet.gcc.biostat.washington.edu/beagle/genetic_maps/</a>).</div><div id="p0615" role="paragraph">The <i>f</i><sub>3</sub> statistics was calculated with <i>qp3Pop</i> 435 in ADMIXTOOLS 5.1 package (<span class="dropBlock reference-citations"><a id="body-ref-sref92-3" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-3" aria-controls="sref92-3">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref92-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (646)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and the <i>f</i><sub>4</sub> statistics was calculated by <i>qpDstat</i> 755 with parameter “f4 mode: YES.” The Mbuti population from Africa was used as outgroup in all <i>f</i>-statistics analysis.</div></section><section id="sec-4-5-3"><h4>Admixture modeling with <i>qpAdm</i></h4><div id="p0620" role="paragraph">The <i>qpAdm</i> 810 in ADMIXTOOLS 5.1 was applied to model the ancestries of admixed populations. We chose an outgroup set of seven populations (stdright), including modern African Mbuti (n&nbsp;= 10), Paleolithic European Villabruna (n&nbsp;= 1), Levantine hunter-gather Natufian (n&nbsp;= 6), Neolithic Iranian farmer Iran_N (n&nbsp;= 5), modern South Asian Onge (n&nbsp;= 11), Southeast Asian Ami (n&nbsp;= 10) and Native American Karitiana (n&nbsp;= 16), to distinguish ancestries across Eurasian continent. For the modeling of KPT005 with possible West Eurasian ancestry, we also included Neolithic Anatolian farmer Anatolia_N (n&nbsp;= 25) in the outgroup (stdright_Anatolia) in order to distinguish the Yamnaya-related and Sintashta-related ancestries. Before running <i>qpAdm</i>, all the reference combinations were tested by <i>qpWave</i> 410, to make sure the outgroup set could distinguish all sources (<a id="crosref1325" href="#mmc3">Table S3</a>).</div></section><section id="sec-4-5-4"><h4>Demographic modeling with <i>qpGraph</i></h4><div id="p0625" role="paragraph">We used <i>qpGraph</i> 6450 to reconstruct a demographic model of studied populations. For the resolution of graphic modeling, we applied “useallsnps” mode to correct for low-coverage sample and carried out this part of analysis using 1240K SNP dataset. Starting from (Mbuti, (AG3, Onge)), we first iteratively added the Devil’s Gate, USR1, Ancient Southwestern Ontario (ASO, n&nbsp;= 6) and Early San Nicolas (ESN, n&nbsp;= 17) to build a skeleton graph based on previous publications (<span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Posth et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Posth, C. ∙ Nakatsuka, N. ∙ Lazaridis, I. ...</div><div><strong>Reconstructing the Deep Population History of Central and South America</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:1185-1197</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2018.10.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2018.10.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85056451563" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30415837/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.10.027&amp;pmid=30415837" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with USR1, ASO and ESN forming a single group which is the mixture of AG3 and Devil’s Gate, and USR1 outgroup to ASO and ESN.</div><div id="p0630" role="paragraph">Then we added UKY and Kolyma independently to this graph, considering them as either sister lineage of existing node, or two-way admixture of the existing lineages. For each population, we tested 91 different models and chose the best fitted model based on the worst Z scores and the final likelihood scores reported by each model. After obtaining the best fitted model with UKY or Kolyma (<a id="crosref1335" href="#figs2">Figure&nbsp;S2</a>), we further tested 340 different models by adding Kolyma onto the best UKY model, or vice versa, to find the best fitted graph including both of these individuals.</div></section><section id="sec-4-5-5"><h4>Admixture dating with <i>DATES</i></h4><div id="p0635" role="paragraph">We applied the ancestry covariance pattern-based <i>DATES</i> 753 program (<span class="dropBlock reference-citations"><a id="body-ref-sref158-3" href="#" role="doc-biblioref" data-xml-rid="bib158" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref158-3" aria-controls="sref158-3">Moorjani and Patterson, 2018</a><div class="dropBlock__holder" data-db-target-of="sref158-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Moorjani, P. ∙ Patterson, N.</div><div><a id="intref0250a" href="https://github.com/priyamoorjani/DATES" target="_blank">https://github.com/priyamoorjani/DATES</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MoorjaniN.Pattersonhttps%3A%2F%2Fgithub.com%2Fpriyamoorjani%2FDATES" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to estimate the time of admixture events in Lake Baikal and Okunevo populations. For the Baikal populations, we used a pooled population ANE_pool (n&nbsp;= 15, including MA1, AG2, AG3, EHG, Botai, West_Siberia_N) to represent the ANE-related ancestry, and population EA_pool (n&nbsp;= 39, including Devil’s Gate, Ulchi and Nanai) to represent the NEA ancestry. For the Okunevo population, the three ancestries were represented by Baikal_LNBA_all (n&nbsp;= 34), Yamnaya_pool (n&nbsp;= 43, including Yamnaya_Samara, Yamnaya_Karagash, Yamnaya_Kalmkia and Afanasievo) and Botai_pool (n&nbsp;= 8, including Botai and West_Siberia_N), respectively. For the BZK002 individual, we used the same Baikal_LNBA_all and Botai_pool to represent the two ancestries. The bin size for covariance calculation was 0.1cM and exponential fitting started at d ≥ 0.5cM.</div></section><section id="sec-4-5-6"><h4>Pathogen DNA screening with HOPS</h4><div id="p0640" role="paragraph">The Megan Alignment Tool (MALT version 0.4.0) (<span class="dropBlock reference-citations"><a id="body-ref-sref141-2" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-2" aria-controls="sref141-2">Vågene et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref141-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">142.</div><div>Vågene, Å.J. ∙ Herbig, A. ∙ Campana, M.G. ...</div><div><strong>Salmonella enterica genomes from victims of a major sixteenth-century epidemic in Mexico</strong></div><div><em>Nat. Ecol. Evol.</em> 2018; <strong>2</strong>:520-528</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-017-0446-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85040669654" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29335577/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-017-0446-6&amp;pmid=29335577" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used within the pathogen screening pipeline HOPS (<span class="dropBlock reference-citations"><a id="body-ref-sref41-3" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-3" aria-controls="sref41-3">Hübler et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref41-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Hübler, R. ∙ Key, F.M. ∙ Warinner, C. ...</div><div><strong>HOPS: automated detection and authentication of pathogen DNA in archaeological remains</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:280</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1903-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85076688072" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31842945/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1903-0&amp;pmid=31842945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), for the assessment of pathogen DNA in all Lake Baikal specimens analyzed in this study. Pre-processed shotgun NGS reads were used as input for MALT, against a custom database of all NCBI RefSeq bacterial and viral assemblies marked as complete, as well as a selection of eukaryotic pathogen genomes and the human GRCh38 reference (RefSeq, November 2017). Genomes with description keywords such as “unknown” were removed from the database, retaining a total of 15,361 entries. MALT was run with an 85&nbsp;minimum percentage identity filter (–minPercentIdentity), 1 for the minimum support filer (–minSupport), a top percentage parameter of 1 (–topPercent) and a semi-global alignment mode, with all other parameters remaining as default. The MALT output was filtered with MALTExtract (within HOPS) based on a predefined list of bacterial, viral and eukaryotic pathogen candidates, where assigned reads were evaluated based on different parameters such as their coverage distributions, their edit distance and their aDNA damage profiles.</div></section><section id="sec-4-5-7"><h4>Read processing, SNP calling and phylogenetic analysis of <i>Yersinia pestis</i></h4><div id="p0645" role="paragraph">All captured libraries were sequenced on an Illumina HiSeq4000 (2x76+8+8 cycles). Processing of de-multiplexed NGS reads was performed within the automated pipeline EAGER v1.92.55 (<span class="dropBlock reference-citations"><a id="body-ref-sref97-6" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-6" aria-controls="sref97-6">Peltzer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref97-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Peltzer, A. ∙ Jäger, G. ∙ Herbig, A. ...</div><div><strong>EAGER: efficient ancient genome reconstruction</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:60</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0918-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27036623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0918-z&amp;pmid=27036623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, AdapterRemoval v2 (<span class="dropBlock reference-citations"><a id="body-ref-sref124-3" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-3" aria-controls="sref124-3">Schubert et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref124-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Schubert, M. ∙ Lindgreen, S. ∙ Orlando, L.</div><div><strong>AdapterRemoval v2: rapid adapter trimming, identification, and read merging</strong></div><div><em>BMC Res. Notes.</em> 2016; <strong>9</strong>:88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13104-016-1900-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84958153152" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26868221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13104-016-1900-2&amp;pmid=26868221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used for removing Illumina adapters and for read filtering based on sequencing quality (minimum base quality of 20) and length (≥30&nbsp;bp). Moreover, for partially-UDG-treated libraries, both ends of NGS reads were trimmed to remove remaining deaminated sites, and the resulting files were concatenated with pre-processed read data from full-UDG-treated libraries. Furthermore, BWA (<span class="dropBlock reference-citations"><a id="body-ref-sref61-3" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-3" aria-controls="sref61-3">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref61-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (17717)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used for read mapping against the <i>Y.&nbsp;pestis</i> chromosomal reference genome (GenBank: <a id="interref260" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC_003143.1" target="_blank">NC_003143.1</a>), using a –n parameter of 0.1 and -l 32. The identical parameters were also used for mapping against the plasmid reference sequences (GenBank: <a id="interref265" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC_003131.1" target="_blank">NC_003131.1</a>, GenBank: <a id="interref270" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC_003134.1" target="_blank">NC_003134.1</a> and GenBank: <a id="interref275" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC_003132.1" target="_blank">NC_003132.1</a>), where nucleotide positions 3,000 - 4,200 were masked in pPCP1 as they were previously shown to display homology with an expression vector present in laboratory reagents (<span class="dropBlock reference-citations"><a id="body-ref-sref125" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125" aria-controls="sref125">Schuenemann et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref125"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Schuenemann, V.J. ∙ Bos, K. ∙ DeWitte, S. ...</div><div><strong>Targeted enrichment of ancient pathogens yielding the pPCP1 plasmid of Yersinia pestis from victims of the Black Death</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:E746-E752</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1105107108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-80053137140" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21876176/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1105107108&amp;pmid=21876176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Subsequently, SAMtools was used to remove reads with mapping quality lower than 37 and MarkDuplicates was used to remove PCR duplicates (<a id="intref0225" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a>).</div><div id="p0650" role="paragraph">SNPs were called on the chromosomal <i>Y.&nbsp;pestis</i> mapped reads using the UnifiedGenotyper in GATK (<span class="dropBlock reference-citations"><a id="body-ref-sref26-3" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-3" aria-controls="sref26-3">DePristo et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref26-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>DePristo, M.A. ∙ Banks, E. ∙ Poplin, R. ...</div><div><strong>A framework for variation discovery and genotyping using next-generation DNA sequencing data</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:491-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.806" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-79955483667" target="_blank" aria-label="Open Scopus in new tab">Scopus (5329)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21478889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.806&amp;pmid=21478889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GLZ001 and GLZ002 were analyzed alongside a dataset of modern <i>Y.&nbsp;pesti</i>s genomes (n&nbsp;= 233) (<span class="dropBlock reference-citations"><a id="body-ref-sref151" href="#" role="doc-biblioref" data-xml-rid="bib17 bib28 bib52 bib53 bib54 bib58 bib76 bib151" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref151" aria-controls="sref151">Cui et&nbsp;al., 2013; Eroshenko et&nbsp;al., 2017; Kislichkina et&nbsp;al., 2015, 2018a, 2018b; Kutyrev et&nbsp;al., 2018; Morelli et&nbsp;al., 2010; Zhgenti et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref151"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Cui, Y. ∙ Yu, C. ∙ Yan, Y. ...</div><div><strong>Historical variations in mutation rate in an epidemic pathogen, Yersinia pestis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:577-582</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1205750110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84872185929" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23271803/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1205750110&amp;pmid=23271803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Eroshenko, G.A. ∙ Nosov, N.Y. ∙ Krasnov, Y.M. ...</div><div><strong>Yersinia pestis strains of ancient phylogenetic branch 0.ANT are widely spread in the high-mountain plague foci of Kyrgyzstan</strong></div><div><em>PLoS ONE.</em> 2017; <strong>12</strong>:e0187230</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0187230" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85032187730" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29073248/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0187230&amp;pmid=29073248" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Kislichkina, A.A. ∙ Bogun, A.G. ∙ Kadnikova, L.A. ...</div><div><strong>Nineteen Whole-Genome Assemblies of Yersinia pestis subsp. microtus, Including Representatives of Biovars caucasica, talassica, hissarica, altaica, xilingolensis, and ulegeica</strong></div><div><em>Genome Announc.</em> 2015; <strong>3</strong></div><div>e01342-15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/genomeA.01342-15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84990011322" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26634751/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FgenomeA.01342-15&amp;pmid=26634751" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Kislichkina, A.A. ∙ Bogun, A.G. ∙ Kadnikova, L.A. ...</div><div><strong>Nine Whole-Genome Assemblies of <i>Yersinia pestis</i> subsp. <i>microtus</i> bv. Altaica Strains Isolated from the Altai Mountain Natural Plague Focus (No. 36) in Russia</strong></div><div><em>Genome Announc.</em> 2018; <strong>6</strong>:e01440-17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/genomeA.01440-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85041172977" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29348336/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FgenomeA.01440-17&amp;pmid=29348336" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Kislichkina, A.A. ∙ Bogun, A.G. ∙ Kadnikova, L.A. ...</div><div><strong>Six Whole-Genome Assemblies of Yersinia pestis subsp. microtus bv. ulegeica (Phylogroup 0.PE5) Strains Isolated from Mongolian Natural Plague Foci</strong></div><div><em>Genome Announc.</em> 2018; <strong>6</strong>:e00536-18</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/genomeA.00536-18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85048939325" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29930050/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FgenomeA.00536-18&amp;pmid=29930050" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Kutyrev, V.V. ∙ Eroshenko, G.A. ∙ Motin, V.L. ...</div><div><strong>Phylogeny and Classification of <i>Yersinia pestis</i> Through the Lens of Strains From the Plague Foci of Commonwealth of Independent States</strong></div><div><em>Front. Microbiol.</em> 2018; <strong>9</strong>:1106</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fmicb.2018.01106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047535772" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29887859/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffmicb.2018.01106&amp;pmid=29887859" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Morelli, G. ∙ Song, Y. ∙ Mazzoni, C.J. ...</div><div><strong>Yersinia pestis genome sequencing identifies patterns of global phylogenetic diversity</strong></div><div><em>Nat. Genet.</em> 2010; <strong>42</strong>:1140-1143</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.705" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-78649470145" target="_blank" aria-label="Open Scopus in new tab">Scopus (331)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21037571/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.705&amp;pmid=21037571" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Zhgenti, E. ∙ Johnson, S.L. ∙ Davenport, K.W. ...</div><div><strong>Genome Assemblies for 11 Yersinia pestis Strains Isolated in the Caucasus Region</strong></div><div><em>Genome Announc.</em> 2015; <strong>3</strong>:3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/genomeA.01030-15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FgenomeA.01030-15" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (as listed in (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Keller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Keller, M. ∙ Spyrou, M.A. ∙ Scheib, C.L. ...</div><div><strong>Ancient <i>Yersinia pestis</i> genomes from across Western Europe reveal early diversification during the First Pandemic (541–750)</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:12363-12372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1820447116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85067606359" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31164419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1820447116&amp;pmid=31164419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)). In addition, we included previously published historical <i>Y.&nbsp;pestis</i> genomes with ≥ 10-fold coverage (<span class="dropBlock reference-citations"><a id="body-ref-sref134" href="#" role="doc-biblioref" data-xml-rid="bib11 bib12 bib29 bib50 bib156 bib157 bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134" aria-controls="sref134">Bos et al., 2011, 2016; Feldman et al., 2016; Keller et al., 2019; Namouchi et al., 2018; Spyrou et al., 2016, 2019</a><div class="dropBlock__holder" data-db-target-of="sref134"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bos, K.I. ∙ Schuenemann, V.J. ∙ Golding, G.B. ...</div><div><strong>A draft genome of Yersinia pestis from victims of the Black Death</strong></div><div><em>Nature.</em> 2011; <strong>478</strong>:506-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10549" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-80054987989" target="_blank" aria-label="Open Scopus in new tab">Scopus (344)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21993626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10549&amp;pmid=21993626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Bos, K.I. ∙ Herbig, A. ∙ Sahl, J. ...</div><div><strong>Eighteenth century Yersinia pestis genomes reveal the long-term persistence of an historical plague focus</strong></div><div><em>eLife.</em> 2016; <strong>5</strong>:e12994</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.12994" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26795402/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.12994&amp;pmid=26795402" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Feldman, M. ∙ Harbeck, M. ∙ Keller, M. ...</div><div><strong>A High-Coverage Yersinia pestis Genome from a Sixth-Century Justinianic Plague Victim</strong></div><div><em>Mol. Biol. Evol.</em> 2016; <strong>33</strong>:2911-2923</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msw170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84994519286" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27578768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsw170&amp;pmid=27578768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Keller, M. ∙ Spyrou, M.A. ∙ Scheib, C.L. ...</div><div><strong>Ancient <i>Yersinia pestis</i> genomes from across Western Europe reveal early diversification during the First Pandemic (541–750)</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:12363-12372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1820447116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85067606359" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31164419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1820447116&amp;pmid=31164419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Namouchi, A. ∙ Guellil, M. ∙ Kersten, O. ...</div><div><strong>Integrative approach using <i>Yersinia pestis</i> genomes to revisit the historical landscape of plague during the Medieval Period</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E11790-E11797</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1812865115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85058318857" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30478041/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1812865115&amp;pmid=30478041" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">133.</div><div>Spyrou, M.A. ∙ Tukhbatova, R.I. ∙ Feldman, M. ...</div><div><strong>Historical <i>Y. pestis</i> genomes reveal the European Black Death as the source of ancient and modern plague pandemics</strong></div><div><em>Cell Host &amp; Microbe.</em> 2016; <strong>19</strong>:874-881</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2016.05.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2016.05.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27281573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2016.05.012&amp;pmid=27281573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">135.</div><div>Spyrou, M.A. ∙ Keller, M. ∙ Tukhbatova, R.I. ...</div><div><strong>Phylogeography of the second plague pandemic revealed through analysis of historical Yersinia pestis genomes</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12154-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85072923513" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31578321/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12154-0&amp;pmid=31578321" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and all previously published Late Neolithic and Bronze Age genomes with ≥ 3-fold coverage (<span class="dropBlock reference-citations"><a id="body-ref-sref133-3" href="#" role="doc-biblioref" data-xml-rid="bib6 bib115 bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133-3" aria-controls="sref133-3">Andrades Valtueña et&nbsp;al., 2017; Rasmussen et&nbsp;al., 2015b; Spyrou et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref133-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Rasmussen, S. ∙ Allentoft, M.E. ∙ Nielsen, K. ...</div><div><strong>Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:571-582</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26496604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.10.009&amp;pmid=26496604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Spyrou, M.A. ∙ Tukhbatova, R.I. ∙ Wang, C.-C. ...</div><div><strong>Analysis of 3800-year-old Yersinia pestis genomes suggests Bronze Age origin for bubonic plague</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:2234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-04550-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85048310202" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29884871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-04550-9&amp;pmid=29884871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Geographic isolation locations of genomes within the comparative dataset are as follows: China (CHN), United States of America (USA), Madagascar (MDG), India (IND), Myanmar (MNM), Congo (COG), Uganda (UGA), Mongolia (MNG), Nepal (NPL), Iran (IRN), Kazakhstan (KAZ), Kyrgyzstan (KGZ), Armenia (ARM), Georgia (GEO), Azerbaijan (AZE), Uzbekistan (UZB), Turkmenistan (TKM), Tajikistan (TJK), Russia (RUS), unspecified regions of the Former Soviet Union (FSU), Switzerland (CHE), Germany (DEU), Spain (ESP), France (FRA), United Kingdom (GBR), Netherlands (NLD), Norway (NOR), Lithuania (LTU), Croatia (CRO), Estonia (EST).</div><div id="p0655" role="paragraph">SNPs were called using the ‘EMIT_ALL_SITES’ option in GATK, which generated a call for every position in the chromosomal reference genome (CO92). Subsequently, a custom java tool, MultiVCFAnalyzer v0.85 (<a id="intref0230" href="https://github.com/alexherbig/MultiVCFAnalyzer" target="_blank">https://github.com/alexherbig/MultiVCFAnalyzer</a>), was used to produce a SNP table of variant positions across the entire dataset, excluding previously defined non-core regions, as well as mRNAs, tRNAs and tmRNAs (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib17 bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Cui et&nbsp;al., 2013; Morelli et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Cui, Y. ∙ Yu, C. ∙ Yan, Y. ...</div><div><strong>Historical variations in mutation rate in an epidemic pathogen, Yersinia pestis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:577-582</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1205750110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84872185929" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23271803/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1205750110&amp;pmid=23271803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Morelli, G. ∙ Song, Y. ∙ Mazzoni, C.J. ...</div><div><strong>Yersinia pestis genome sequencing identifies patterns of global phylogenetic diversity</strong></div><div><em>Nat. Genet.</em> 2010; <strong>42</strong>:1140-1143</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.705" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-78649470145" target="_blank" aria-label="Open Scopus in new tab">Scopus (331)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21037571/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.705&amp;pmid=21037571" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Variants were retained based on the following conditions: homozygous SNP positions were called with a minimum coverage of 3-fold and a minimum genotyping quality of 30. For heterozygous positions, SNPs were retained when at least 90% of reads supported the call. When none of the conditions were met, an “N” would be inserted in the respective genomic position. A total of 7,052 SNPs were identified in the present dataset. The SNP alignment produced by MultiVCFAnalyzer v0.85 was used as input for a maximum likelihood (ML) phylogeny with 99% partial deletion filter, which retained 4,368 SNP positions. ML trees were built using the program RAxML (version 8.2.9) (<span class="dropBlock reference-citations"><a id="body-ref-sref135-2" href="#" role="doc-biblioref" data-xml-rid="bib135" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref135-2" aria-controls="sref135-2">Stamatakis, 2014</a><div class="dropBlock__holder" data-db-target-of="sref135-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Stamatakis, A.</div><div><strong>RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:1312-1313</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu033" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899553363" target="_blank" aria-label="Open Scopus in new tab">Scopus (11412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24451623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu033&amp;pmid=24451623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), under the Generalized Time Reversible (GTR) substitution model (<span class="dropBlock reference-citations"><a id="body-ref-sref139" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139" aria-controls="sref139">Tavaré, 1986</a><div class="dropBlock__holder" data-db-target-of="sref139"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">140.</div><div>Tavaré, S.</div><div><strong>Some probabilistic and statistical problems in the analysis of DNA sequences</strong></div><div><em>Lect. Math Life Sci.</em> 1986; <strong>17</strong>:57-86</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.Tavar%C3%A9Some+probabilistic+and+statistical+problems+in+the+analysis+of+DNA+sequencesLect.+Math+Life+Sci.1719865786" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (four gamma rate categories). 1,000 bootstrap replicates were used to determine node support.</div></section><section id="sec-4-5-8"><h4>BEAST v1.8 tip dating of GLZ001 and GLZ002</h4><div id="p0660" role="paragraph">The program TempEst v1.5 (<a id="intref0235" href="http://tree.bio.ed.ac.uk/software/tempest/" target="_blank">http://tree.bio.ed.ac.uk/software/tempest/</a>) was used to assess of the temporal signal across previously published genomes and calibration points on the <i>Y.&nbsp;pestis</i> LNBA lineage (<span class="dropBlock reference-citations"><a id="body-ref-sref115-7" href="#" role="doc-biblioref" data-xml-rid="bib6 bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-7" aria-controls="sref115-7">Andrades Valtueña et&nbsp;al., 2017; Rasmussen et&nbsp;al., 2015b</a><div class="dropBlock__holder" data-db-target-of="sref115-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Rasmussen, S. ∙ Allentoft, M.E. ∙ Nielsen, K. ...</div><div><strong>Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:571-582</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26496604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.10.009&amp;pmid=26496604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), including the modern isolate 0.PE.2 Pestoides F (<span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Garcia et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Garcia, E. ∙ Chain, P. ∙ Worsham, P. ...</div><div><strong>Pestoides F, an Atypical Yersinia pestis Strain from the Former Soviet Union</strong></div><div><em>Adv. Exp. Med. Biol.</em> 2007; <strong>603</strong>:17-22</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-0-387-72124-8_2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17966401/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-0-387-72124-8_2&amp;pmid=17966401" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as outgroup. All overlapping variant positions across this set of genomes were used for the analysis, retaining a total of 374 SNPs in the dataset. The resulting correlation coefficient (<i>r</i>) was 0.88 and R<sup>2</sup> was 0.78 when including the 0.PE2 outgroup and 0.98 and 0.97, respectively, for the LNBA lineage alone. Such result suggests a strong temporal signal in the dataset.</div><div id="p0665" role="paragraph">Subsequently, we used BEAST v1.8 (<span class="dropBlock reference-citations"><a id="body-ref-sref27-3" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-3" aria-controls="sref27-3">Drummond and Rambaut, 2007</a><div class="dropBlock__holder" data-db-target-of="sref27-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Drummond, A.J. ∙ Rambaut, A.</div><div><strong>BEAST: Bayesian evolutionary analysis by sampling trees</strong></div><div><em>BMC Evol. Biol.</em> 2007; <strong>7</strong>:214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-7-214" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-38649110663" target="_blank" aria-label="Open Scopus in new tab">Scopus (8871)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17996036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-7-214&amp;pmid=17996036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to estimate the tip dates of GLZ001 and GLZ002, using published radiocarbon date ranges across the LNBA lineage as uniform priors (in years before the present) as follows: RISE509 (uniform prior: 4836–4625, tip date: 4729), RK1.001 (uniform prior: 4828–4622, tip date: 4720), GEN72 (uniform prior: 4833–4592, tip date: 4721), Gyvakarai1 (uniform prior: 4571–4422, tip date: 4485), Kunila2 (uniform prior: 4524–4290, tip date: 4427), 1343UnTal (uniform prior: 4346–4098, tip date: 4203), 6Post (uniform prior: 3957–3832, tip date: 3873), RT5 (uniform prior: 3868–3704, tip date: 3789), RISE505 (uniform prior: 3694–3575, tip date: 3635). The tip date of the 0.PE2 outgroup was set to 0 (present). The input date range for GLZ001 and GLZ002 was constrained to be between 5000 and 3500 years before the present, which spans the entire temporal range of the LNBA lineage, and the starting value tip date was set to 4250 (midpoint). The graphical user interface BEAUti v1.8 (<span class="dropBlock reference-citations"><a id="body-ref-sref27-4" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-4" aria-controls="sref27-4">Drummond and Rambaut, 2007</a><div class="dropBlock__holder" data-db-target-of="sref27-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Drummond, A.J. ∙ Rambaut, A.</div><div><strong>BEAST: Bayesian evolutionary analysis by sampling trees</strong></div><div><em>BMC Evol. Biol.</em> 2007; <strong>7</strong>:214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-7-214" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-38649110663" target="_blank" aria-label="Open Scopus in new tab">Scopus (8871)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17996036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-7-214&amp;pmid=17996036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to set up multiple runs using the coalescent constant size and coalescent skyline tree priors in combination with both a strict and a lognormal relaxed clock model. In addition, a GTR substitution model with four gamma rate categories was used for all runs. All set-ups were run in BEAST v1.8 (<span class="dropBlock reference-citations"><a id="body-ref-sref27-5" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-5" aria-controls="sref27-5">Drummond and Rambaut, 2007</a><div class="dropBlock__holder" data-db-target-of="sref27-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Drummond, A.J. ∙ Rambaut, A.</div><div><strong>BEAST: Bayesian evolutionary analysis by sampling trees</strong></div><div><em>BMC Evol. Biol.</em> 2007; <strong>7</strong>:214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-7-214" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-38649110663" target="_blank" aria-label="Open Scopus in new tab">Scopus (8871)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17996036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-7-214&amp;pmid=17996036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using two independent chains of 25,000,000 states each. After run completion, chains were combined using LogCombiner (<span class="dropBlock reference-citations"><a id="body-ref-sref27-6" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-6" aria-controls="sref27-6">Drummond and Rambaut, 2007</a><div class="dropBlock__holder" data-db-target-of="sref27-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Drummond, A.J. ∙ Rambaut, A.</div><div><strong>BEAST: Bayesian evolutionary analysis by sampling trees</strong></div><div><em>BMC Evol. Biol.</em> 2007; <strong>7</strong>:214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-7-214" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-38649110663" target="_blank" aria-label="Open Scopus in new tab">Scopus (8871)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17996036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-7-214&amp;pmid=17996036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with 10% burn-in and were then inspected in Tracer v1.6 to ensure run convergence (<a id="intref0240" href="http://tree.bio.ed.ac.uk/software/tracer/" target="_blank">http://tree.bio.ed.ac.uk/software/tracer/</a>) with all ESS values being &gt; 200. The resulting mean tip dates and HPD95% intervals produced for GLZ001 and GLZ002 across all runs were viewed and analyzed in Tracer v1.6 (<a id="intref0245" href="http://tree.bio.ed.ac.uk/software/tracer/" target="_blank">http://tree.bio.ed.ac.uk/software/tracer/</a>).</div></section><section id="sec-4-5-9"><h4>Assessment of <i>Y. pestis</i> gene profiles</h4><div id="p0670" role="paragraph">In order to assess the gene profiles of GLZ001 and GLZ002, we computed the coverage across of 150 previously defended virulence-associated and evolutionary-determinant <i>Y.&nbsp;pestis</i> genes (<span class="dropBlock reference-citations"><a id="body-ref-sref152-2" href="#" role="doc-biblioref" data-xml-rid="bib24 bib152" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref152-2" aria-controls="sref152-2">Demeure et&nbsp;al., 2019; Zhou and Yang, 2009</a><div class="dropBlock__holder" data-db-target-of="sref152-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Demeure, C.E. ∙ Dussurget, O. ∙ Mas Fiol, G. ...</div><div><strong>Yersinia pestis and plague: an updated view on evolution, virulence determinants, immune subversion, vaccination, and diagnostics</strong></div><div><em>Genes Immun.</em> 2019; <strong>20</strong>:357-370</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41435-019-0065-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85063771364" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30940874/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41435-019-0065-0&amp;pmid=30940874" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Zhou, D. ∙ Yang, R.</div><div><strong>Molecular Darwinian evolution of virulence in Yersinia pestis</strong></div><div><em>Infect. Immun.</em> 2009; <strong>77</strong>:2242-2250</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/IAI.01477-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-66549101505" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19289506/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FIAI.01477-08&amp;pmid=19289506" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For this, the two newly reconstructed genomes were comparatively assessed alongside previously published representatives of the LNBA lineage (RISE509, RK1001, GEN72, Gyvakarai1, Kunila2, 1343UnTal85, 6Postillionstrasse, RISE505), a Bronze Age isolate showing signatures of flea adaptation (RT5), historical <i>Y.</i> pestis genomes from the first and second plague pandemics (Altenerding 2148, London ES 8124/8291/11972), representatives of modern isolates (0.PE2 Pestoides F, 0.PE4 Microtus 91001, 1.ORI CO92), as well as a <i>Y.&nbsp;pseudotuberculosis</i> strain (IP31953). Raw pre-processed reads from all aforementioned datasets were mapped against the CO92 reference chromosome and plasmids using BWA (-n 0.1, -l 32), without the subsequent use of a mapping quality filter. The coverage across the defined gene regions was calculated using BEDTools (<span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Quinlan and Hall, 2010</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Quinlan, A.R. ∙ Hall, I.M.</div><div><strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:841-842</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq033" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951770756" target="_blank" aria-label="Open Scopus in new tab">Scopus (7022)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq033&amp;pmid=20110278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The computed coverages were plotted in the form of a heatmap using the ggplot2 package within R version 3.2.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref150" href="#" role="doc-biblioref" data-xml-rid="bib106 bib150" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150" aria-controls="sref150">R Development Core Team, 2008; Wickham, 2016</a><div class="dropBlock__holder" data-db-target-of="sref150"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>R Development Core Team</div><div><strong>Computational Many-Particle Physics</strong></div><div>Springer Berlin Heidelberg, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Development+Core+TeamComputational+Many-Particle+Physics2008Springer+Berlin+Heidelberg" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">151.</div><div>Wickham, H.</div><div><strong>ggplot2: Elegant Graphics For Data Analysis</strong></div><div>Springer, 2016</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-3-319-24277-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-3-319-24277-4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, previously defined substitutions or InDels within functionally relevant genes were manually inspected in GLZ001 and GLZ002 using the program IGV (<span class="dropBlock reference-citations"><a id="body-ref-sref140-2" href="#" role="doc-biblioref" data-xml-rid="bib140" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref140-2" aria-controls="sref140-2">Thorvaldsdóttir et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref140-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Thorvaldsdóttir, H. ∙ Robinson, J.T. ∙ Mesirov, J.P.</div><div><strong>Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration</strong></div><div><em>Brief. Bioinform.</em> 2013; <strong>14</strong>:178-192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bib/bbs017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84875634162" target="_blank" aria-label="Open Scopus in new tab">Scopus (3379)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22517427/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbib%2Fbbs017&amp;pmid=22517427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These were: a non-synonymous substitution defining the <i>pla</i> gene variant involved in within-host bacterial dissemination (<span class="dropBlock reference-citations"><a id="body-ref-sref153" href="#" role="doc-biblioref" data-xml-rid="bib153" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153" aria-controls="sref153">Zimbler et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref153"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">154.</div><div>Zimbler, D.L. ∙ Schroeder, J.A. ∙ Eddy, J.L. ...</div><div><strong>Early emergence of Yersinia pestis as a severe respiratory pathogen</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:7487</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms8487" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26123398/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms8487&amp;pmid=26123398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), pseudogenization mutations in genes PDE2, PDE3, <i>ureD</i> and a 30bp duplication in <i>rcsA</i> associated with <i>Y.&nbsp;pestis</i> flea-colonization (<span class="dropBlock reference-citations"><a id="body-ref-sref136-2" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136-2" aria-controls="sref136-2">Sun et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref136-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Sun, Y.-C. ∙ Jarrett, C.O. ∙ Bosio, C.F. ...</div><div><strong>Retracing the evolutionary path that led to flea-borne transmission of Yersinia pestis</strong></div><div><em>Cell Host Microbe.</em> 2014; <strong>15</strong>:578-586</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2014.04.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2014.04.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24832452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2014.04.003&amp;pmid=24832452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as well as a pseudogenization mutation in <i>flhD</i> associated with immune evasion (<span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Minnich and Rohde, 2007</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Minnich, S.A. ∙ Rohde, H.N.</div><div><strong>A rationale for repression and/or loss of motility by pathogenic yersinia in the mammalian host</strong></div><div><div>Perry, R.D. ∙ Fetherston, J.D. (Editors)</div></div><div><strong>The Genus Yersinia: From Genomics to Function</strong></div><div>Springer, 2007; 298-311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-0-387-72124-8_27" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84934436026" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-0-387-72124-8_27" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0675" role="paragraph">We thank the wet laboratory and computational teams at MPI-SHH. We are grateful to Matthias Meyer and Sarah Nagel from the Max Planck Institute for Evolutionary Anthropology in Leipzig for generating the single-stranded library of UKY. We thank Stephan Schiffels for his suggestions and comments and Aida Andrades Valtueña for computational support. G.P. was supported by the Russian Foundation for Basic Research (Project No. 19-59-44010). The human and pathogen genomics research were funded by the Max Planck Society and the European Research Council ERC-CoG 771234 PALEoRIDER.</div><section id="sec-5"><h3>Author Contributions</h3><div id="p0680" role="paragraph">Conceptualization, H.Y., M.A.S., C.P., C.J., and J.K.; Formal Analysis, H.Y., M.A.S., and P.R.; Investigation, R.R., K.N., G.U.N., S.P., J.Z., M.L., and P.L.; Resources, A.B., M.K., G.P., and S.S.; Writing – Original Draft, H.Y., M.A.S., and P.R.; Writing – Review &amp; Editing, C.P. and C.J.; Supervision, C.P., C.J., and J.K.</div></section><section id="sec-6"><h3>Declaration of Interests</h3><div id="p0685" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental Information (6)</h2><div id="p0695" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.04.037/attachment/48aa7473-9937-4e3e-84f6-00a27dce72ef/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (34.81 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Sample Statistics and Reference Information, Related to Figures 1 and S1 and STAR Methods</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.04.037/attachment/e28e1521-bc51-4104-8099-9e18e9e27b70/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (39.38 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. The Genetic Affinity among UKY, Kolyma, USR1, and the First American People Revealed by <i>f</i><sub>4</sub> Statistics, Related to Figures 2 and S2</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.04.037/attachment/b50425d1-f129-4239-8282-eff087b8dd64/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (29.94 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. <i>qpAdm</i> and <i>qpWave</i> Modeling of Our Newly Reported Individuals and Related Population, Related to Figures 3, 4, and 5</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.04.037/attachment/ae848b68-0b21-4b2f-846a-560deb56891e/mmc4.pdf" download="mmc4.pdf" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">PDF (97.27 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Genetic Affinity of Newly Genotyped Early Neolithic to Bronze Age Individuals with Worldwide Population, Revealed by Outgroup <i>f</i><sub>3</sub> Statistics, Related to Figure&nbsp;3</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.04.037/attachment/25484f6f-581b-4e00-8fda-ed97fbeaa3ec/mmc5.pdf" download="mmc5.pdf" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">PDF (64.29 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. Admixture Dating of Early Neolithic and Bronze Age Baikal Individuals, as well as Okunevo and BZK002, Using <i>DATES</i>, Related to Figures 3, 5, and S4</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.04.037/attachment/08562660-e4c4-4173-8232-3d905bf42a3d/mmc6.xlsx" download="mmc6.xlsx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Spreadsheet (13.10 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Table S6. <i>Yersinia pestis</i> Post-capture Mapping Statistics and BEAST Tip Dates Inference of GLZ001 and GLZ002, Related to Figures 4 and S5</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Alekseev, V.P. ∙ Mamonova, N.N.</div><div><strong>K paleoantropologii epohi neolita verhov’ev Leny</strong></div><div><em>Sov. Etnogr.</em> 1979; <strong>5</strong>:49-63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.P.AlekseevN.N.MamonovaK+paleoantropologii+epohi+neolita+verhov%E2%80%99ev+LenySov.+Etnogr.519794963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Alexander, D.H. ∙ Novembre, J. ∙ Lange, K.</div><div><strong>Fast model-based estimation of ancestry in unrelated individuals</strong></div><div><em>Genome Res.</em> 2009; <strong>19</strong>:1655-1664</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.094052.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-69749099417" target="_blank" aria-label="Open Scopus in new tab">Scopus (2279)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.094052.109&amp;pmid=19648217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Alexander Bentley, R.</div><div><strong>Strontium Isotopes from the Earth to the Archaeological Skeleton: A Review</strong></div><div><em>J.&nbsp;Archaeol. Method Theory.</em> 2006; <strong>13</strong>:135-187</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10816-006-9009-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-33749026059" target="_blank" aria-label="Open Scopus in new tab">Scopus (631)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10816-006-9009-x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-2" title="View in article">4.</a></div><div>Allentoft, M.E. ∙ Sikora, M. ∙ Sjögren, K.-G. ...</div><div><strong>Population genomics of Bronze Age Eurasia</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:167-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84930940396" target="_blank" aria-label="Open Scopus in new tab">Scopus (523)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26062507/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14507&amp;pmid=26062507" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Ambrose, S.H. ∙ Norr, L.</div><div><strong>Experimental Evidence for the Relationship of the Carbon Isotope Ratios of Whole Diet and Dietary Protein to Those of Bone Collagen and Carbonate</strong></div><div><div>Lambert, J.B. ∙ Grupe, G. (Editors)</div></div><div><strong>Prehistoric Human Bone: Archaeology at the Molecular Level</strong></div><div>Springer Berlin Heidelberg, 1993; 1-37</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-3-662-02894-0_1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-3-662-02894-0_1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref133-1" title="View in article">6.</a></div><div>Andrades Valtueña, A. ∙ Mittnik, A. ∙ Key, F.M. ...</div><div><strong>The Stone Age Plague and Its Persistence in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3683-3691</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.10.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85034632738" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29174893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.10.025&amp;pmid=29174893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (4710)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148" title="View in article">8.</a></div><div>Bazalijskij, V.I.</div><div><strong>Pogrebal’nye kompleksy epohi pozdnego mezolita – neolita Bajkal’skoj Sibiri: tradicii pogrebenij, absolyutnyj vozrast. Izv. Lab. Drevnih Tekhnologij</strong></div><div><em>Irkutsk Izd-vo IrGTU.</em> 2012; <strong>9</strong>:43-101</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.I.BazalijskijPogrebal%E2%80%99nye+kompleksy+epohi+pozdnego+mezolita+%E2%80%93+neolita+Bajkal%E2%80%99skoj+Sibiri%3A+tradicii+pogrebenij%2C+absolyutnyj+vozrast.+Izv.+Lab.+Drevnih+TekhnologijIrkutsk+Izd-vo+IrGTU.9201243101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-6" title="View in article">9.</a></div><div>Bentley, R.A. ∙ Krause, R. ∙ Price, T.D. ...</div><div><strong>Human mobility at the early neolithic settlement of vaihingen, germany: evidence from strontium isotope analysis<sup>∗</sup></strong></div><div><em>Archaeometry.</em> 2003; <strong>45</strong>:471-486</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/1475-4754.00122" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0013196242" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2F1475-4754.00122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">10.</a></div><div>Bentley, R.A. ∙ Price, T.D. ∙ Stephan, E.</div><div><strong>Determining the ‘local’ 87Sr/86Sr range for archaeological skeletons: a case study from Neolithic Europe</strong></div><div><em>J.&nbsp;Archaeol. Sci.</em> 2004; <strong>31</strong>:365-375</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jas.2003.09.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-3042592719" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jas.2003.09.003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">11.</a></div><div>Bos, K.I. ∙ Schuenemann, V.J. ∙ Golding, G.B. ...</div><div><strong>A draft genome of Yersinia pestis from victims of the Black Death</strong></div><div><em>Nature.</em> 2011; <strong>478</strong>:506-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10549" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-80054987989" target="_blank" aria-label="Open Scopus in new tab">Scopus (344)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21993626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10549&amp;pmid=21993626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">12.</a></div><div>Bos, K.I. ∙ Herbig, A. ∙ Sahl, J. ...</div><div><strong>Eighteenth century Yersinia pestis genomes reveal the long-term persistence of an historical plague focus</strong></div><div><em>eLife.</em> 2016; <strong>5</strong>:e12994</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.12994" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26795402/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.12994&amp;pmid=26795402" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Briggs, A.W. ∙ Stenzel, U. ∙ Meyer, M. ...</div><div><strong>Removal of deaminated cytosines and detection of in&nbsp;vivo methylation in ancient DNA</strong></div><div><em>Nucleic Acids Res.</em> 2010; <strong>38</strong>:e87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkp1163" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77950862213" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20028723/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkp1163&amp;pmid=20028723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Brock, F. ∙ Higham, T. ∙ Ramsey, C.B.</div><div><strong>Comments on the Use of Ezee-Filters™ and Ultrafilters at Orau</strong></div><div><em>Radiocarbon.</em> 2013; <strong>55</strong>:211-212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_js_rc.v55i1.16480" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84874546554" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_js_rc.v55i1.16480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Browning, B.L. ∙ Browning, S.R.</div><div><strong>Improving the accuracy and efficiency of identity-by-descent detection in population data</strong></div><div><em>Genetics.</em> 2013; <strong>194</strong>:459-471</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.113.150029" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84878589007" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23535385/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.113.150029&amp;pmid=23535385" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Capo, R.C. ∙ Stewart, B.W. ∙ Chadwick, O.A.</div><div><strong>Strontium isotopes as tracers of ecosystem processes: theory and methods</strong></div><div><em>Geoderma.</em> 1998; <strong>82</strong>:197-225</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0016-7061(97)00102-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0031943786" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0016-7061%2897%2900102-X" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">17.</a></div><div>Cui, Y. ∙ Yu, C. ∙ Yan, Y. ...</div><div><strong>Historical variations in mutation rate in an epidemic pathogen, Yersinia pestis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:577-582</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1205750110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84872185929" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23271803/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1205750110&amp;pmid=23271803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Dabney, J. ∙ Knapp, M. ∙ Glocke, I. ...</div><div><strong>Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:15758-15763</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1314445110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84884624229" target="_blank" aria-label="Open Scopus in new tab">Scopus (459)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24019490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1314445110&amp;pmid=24019490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">19.</a></div><div>Damgaard, P. de B. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20-1" title="View in article">20.</a></div><div>Damgaard, P.B. ∙ Marchi, N. ∙ Rasmussen, S. ...</div><div><strong>137 ancient human genomes from across the Eurasian steppes</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:369-374</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0094-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047137672" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743675/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0094-2&amp;pmid=29743675" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">21.</a></div><div>Debets, G.F.</div><div><strong>Antropologicheskij sostav drevnego naseleniya Pribajkal’ya v epohu pozdnego neolita</strong></div><div><em>Rus. Antropol. Zhurnal.</em> 1930; <strong>t. XIX</strong>:1-2</div><div>7–50</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.F.DebetsAntropologicheskij+sostav+drevnego+naseleniya+Pribajkal%E2%80%99ya+v+epohu+pozdnego+neolitaRus.+Antropol.+Zhurnalt.+XIX1930127%E2%80%9350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Debets, G.F.</div><div><strong>Paleoantropologiya SSSR / TIE, novaya seriya</strong></div><div>M.-L. T. IV., 1948</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.F.DebetsPaleoantropologiya+SSSR+%2F+TIE%2C+novaya+seriya1948M.-L.+T.+IV." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23-1" title="View in article">23.</a></div><div>Delaneau, O. ∙ Zagury, J.-F. ∙ Marchini, J.</div><div><strong>Improved whole-chromosome phasing for disease and population genetic studies</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:5-6</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2307" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84871952176" target="_blank" aria-label="Open Scopus in new tab">Scopus (649)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23269371/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2307&amp;pmid=23269371" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref152-1" title="View in article">24.</a></div><div>Demeure, C.E. ∙ Dussurget, O. ∙ Mas Fiol, G. ...</div><div><strong>Yersinia pestis and plague: an updated view on evolution, virulence determinants, immune subversion, vaccination, and diagnostics</strong></div><div><em>Genes Immun.</em> 2019; <strong>20</strong>:357-370</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41435-019-0065-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85063771364" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30940874/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41435-019-0065-0&amp;pmid=30940874" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref128" title="View in article">25.</a></div><div>Deniro, M.J. ∙ Epstein, S.</div><div><strong>Influence of diet on the distribution of nitrogen isotopes in animals</strong></div><div><em>Geochim. Cosmochim. Acta.</em> 1981; <strong>45</strong>:341-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0016-7037(81)90244-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0019681135" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0016-7037%2881%2990244-1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">26.</a></div><div>DePristo, M.A. ∙ Banks, E. ∙ Poplin, R. ...</div><div><strong>A framework for variation discovery and genotyping using next-generation DNA sequencing data</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:491-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.806" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-79955483667" target="_blank" aria-label="Open Scopus in new tab">Scopus (5329)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21478889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.806&amp;pmid=21478889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">27.</a></div><div>Drummond, A.J. ∙ Rambaut, A.</div><div><strong>BEAST: Bayesian evolutionary analysis by sampling trees</strong></div><div><em>BMC Evol. Biol.</em> 2007; <strong>7</strong>:214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-7-214" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-38649110663" target="_blank" aria-label="Open Scopus in new tab">Scopus (8871)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17996036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-7-214&amp;pmid=17996036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">28.</a></div><div>Eroshenko, G.A. ∙ Nosov, N.Y. ∙ Krasnov, Y.M. ...</div><div><strong>Yersinia pestis strains of ancient phylogenetic branch 0.ANT are widely spread in the high-mountain plague foci of Kyrgyzstan</strong></div><div><em>PLoS ONE.</em> 2017; <strong>12</strong>:e0187230</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0187230" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85032187730" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29073248/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0187230&amp;pmid=29073248" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">29.</a></div><div>Feldman, M. ∙ Harbeck, M. ∙ Keller, M. ...</div><div><strong>A High-Coverage Yersinia pestis Genome from a Sixth-Century Justinianic Plague Victim</strong></div><div><em>Mol. Biol. Evol.</em> 2016; <strong>33</strong>:2911-2923</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msw170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84994519286" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27578768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsw170&amp;pmid=27578768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112-1" title="View in article">30.</a></div><div>Flegontov, P. ∙ Altınışık, N.E. ∙ Changmai, P. ...</div><div><strong>Palaeo-Eskimo genetic ancestry and the peopling of Chukotka and North America</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:236-240</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1251-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85067075462" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1251-y&amp;pmid=31168094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">31.</a></div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (222)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">32.</a></div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (259)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref129-1" title="View in article">33.</a></div><div>Fu, Q. ∙ Posth, C. ∙ Hajdinjak, M. ...</div><div><strong>The genetic history of Ice Age Europe</strong></div><div><em>Nature.</em> 2016; <strong>534</strong>:200-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84969820697" target="_blank" aria-label="Open Scopus in new tab">Scopus (302)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27135931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17993&amp;pmid=27135931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Fuller, B.T. ∙ Richards, M.P. ∙ Mays, S.A.</div><div><strong>Stable carbon and nitrogen isotope variations in tooth dentine serial sections from Wharram Percy</strong></div><div><em>J.&nbsp;Archaeol. Sci.</em> 2003; <strong>30</strong>:1673-1684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0305-4403(03)00073-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0344962345" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0305-4403%2803%2900073-6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">35.</a></div><div>Gansauge, M.-T. ∙ Gerber, T. ∙ Glocke, I. ...</div><div><strong>Single-stranded DNA library preparation from highly degraded DNA using T4 DNA ligase</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong></div><div>e79–e79</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28119419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=28119419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Garcia, E. ∙ Chain, P. ∙ Worsham, P. ...</div><div><strong>Pestoides F, an Atypical Yersinia pestis Strain from the Former Soviet Union</strong></div><div><em>Adv. Exp. Med. Biol.</em> 2007; <strong>603</strong>:17-22</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-0-387-72124-8_2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17966401/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-0-387-72124-8_2&amp;pmid=17966401" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">37.</a></div><div>Graustein, W.C. ∙ Armstrong, R.L.</div><div><strong>The use of strontium-87/strontium-86 ratios to measure atmospheric transport into forested watersheds</strong></div><div><em>Science.</em> 1983; <strong>219</strong>:289-292</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.219.4582.289" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0020641791" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17798277/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.219.4582.289&amp;pmid=17798277" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">38.</a></div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (615)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39-1" title="View in article">39.</a></div><div>Haverkort, C.M. ∙ Bazaliiskii, V.I. ∙ Savel’ev, N.A.</div><div><strong>Identifying hunter-gatherer mobility patterns using strontium isotopes</strong></div><div><div>Weber, A. ∙ Katzenberg, A. ∙ Schurr, T.G. (Editors)</div></div><div><strong>Prehistoric Hunter-Gatherers of the Baikal Region, Siberia</strong></div><div>Penn Press, 2010; 217-238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.9783/9781934536391.217" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.9783%2F9781934536391.217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Hillson, S.</div><div><strong>Dental Anthropology</strong></div><div>Cambridge University Press, 1996</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/CBO9781139170697" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FCBO9781139170697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Hübler, R. ∙ Key, F.M. ∙ Warinner, C. ...</div><div><strong>HOPS: automated detection and authentication of pathogen DNA in archaeological remains</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:280</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1903-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85076688072" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31842945/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1903-0&amp;pmid=31842945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">42.</a></div><div>Jeong, C. ∙ Wilkin, S. ∙ Amgalantugs, T. ...</div><div><strong>Bronze Age population dynamics and the rise of dairy pastoralism on the eastern Eurasian steppe</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E11248-E11255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1813608115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85057166426" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30397125/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1813608115&amp;pmid=30397125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">43.</a></div><div>Jeong, C. ∙ Balanovsky, O. ∙ Lukianova, E. ...</div><div><strong>The genetic history of admixture across inner Eurasia</strong></div><div><em>Nat. Ecol. Evol.</em> 2019; <strong>3</strong>:966-976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-019-0878-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85065175002" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31036896/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-019-0878-2&amp;pmid=31036896" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130-2" title="View in article">44.</a></div><div>Jones, E.R. ∙ Gonzalez-Fortes, G. ∙ Connell, S. ...</div><div><strong>Upper Palaeolithic genomes reveal deep roots of modern Eurasians</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:8912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms9912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84947563586" target="_blank" aria-label="Open Scopus in new tab">Scopus (148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26567969/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms9912&amp;pmid=26567969" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">45.</a></div><div>Jónsson, H. ∙ Ginolhac, A. ∙ Schubert, M. ...</div><div><strong>mapDamage2.0: fast approximate Bayesian estimates of ancient DNA damage parameters</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1682-1684</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84879910635" target="_blank" aria-label="Open Scopus in new tab">Scopus (398)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23613487/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt193&amp;pmid=23613487" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">46.</a></div><div>Jørkov, M.L.S. ∙ Heinemeier, J. ∙ Lynnerup, N.</div><div><strong>The petrous bone--a new sampling site for identifying early dietary patterns in stable isotopic studies</strong></div><div><em>Am. J. Phys. Anthropol.</em> 2009; <strong>138</strong>:199-209</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.20919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-59949101061" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18773469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.20919&amp;pmid=18773469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref145" title="View in article">47.</a></div><div>Katzenberg, M.A. ∙ Weber, A.</div><div><strong>Stable isotope ecology and palaeodiet in the Lake Baikal region of Siberia</strong></div><div><em>J.&nbsp;Archaeol. Sci.</em> 1999; <strong>26</strong>:651-659</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jasc.1998.0382" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0033150923" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjasc.1998.0382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">48.</a></div><div>Katzenberg, M.A. ∙ Bazaliiskii, V.I. ∙ Goriunova, O.I. ...</div><div><strong>8. Diet Reconstruction of Prehistoric Hunter-Gatherers in the Lake Baikal Region</strong></div><div><div>Weber, A.W. ∙ Katzenberg, M.A. ∙ Schurr, T.G. (Editors)</div></div><div><strong>Prehistoric Hunter-Gatherers of the Baikal Region, Siberia</strong></div><div>University of Pennsylvania Press, 2010; 175-192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.9783/9781934536391.175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.9783%2F9781934536391.175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#fig4" title="View in article">49.</a></div><div>Keller, M. ∙ Spyrou, M.A. ∙ Scheib, C.L. ...</div><div><strong>Ancient <i>Yersinia pestis</i> genomes from across Western Europe reveal early diversification during the First Pandemic (541–750)</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:12363-12372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1820447116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85067606359" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31164419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1820447116&amp;pmid=31164419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">50.</a></div><div>Kircher, M. ∙ Sawyer, S. ∙ Meyer, M.</div><div><strong>Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:e3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr771" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84855295457" target="_blank" aria-label="Open Scopus in new tab">Scopus (461)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22021376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr771&amp;pmid=22021376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">51.</a></div><div>Kislichkina, A.A. ∙ Bogun, A.G. ∙ Kadnikova, L.A. ...</div><div><strong>Nineteen Whole-Genome Assemblies of Yersinia pestis subsp. microtus, Including Representatives of Biovars caucasica, talassica, hissarica, altaica, xilingolensis, and ulegeica</strong></div><div><em>Genome Announc.</em> 2015; <strong>3</strong></div><div>e01342-15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/genomeA.01342-15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84990011322" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26634751/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FgenomeA.01342-15&amp;pmid=26634751" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">52.</a></div><div>Kislichkina, A.A. ∙ Bogun, A.G. ∙ Kadnikova, L.A. ...</div><div><strong>Nine Whole-Genome Assemblies of <i>Yersinia pestis</i> subsp. <i>microtus</i> bv. Altaica Strains Isolated from the Altai Mountain Natural Plague Focus (No. 36) in Russia</strong></div><div><em>Genome Announc.</em> 2018; <strong>6</strong>:e01440-17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/genomeA.01440-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85041172977" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29348336/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FgenomeA.01440-17&amp;pmid=29348336" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">53.</a></div><div>Kislichkina, A.A. ∙ Bogun, A.G. ∙ Kadnikova, L.A. ...</div><div><strong>Six Whole-Genome Assemblies of Yersinia pestis subsp. microtus bv. ulegeica (Phylogroup 0.PE5) Strains Isolated from Mongolian Natural Plague Foci</strong></div><div><em>Genome Announc.</em> 2018; <strong>6</strong>:e00536-18</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/genomeA.00536-18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85048939325" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29930050/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FgenomeA.00536-18&amp;pmid=29930050" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">54.</a></div><div>Kiyashko, S.I. ∙ Richard, P. ∙ Chandler, T. ...</div><div><strong>Stable carbon isotope ratios differentiate autotrophs supporting animal diversity in Lake Baikal. Comptes Rendus l’Académie Des Sci. - Ser. III - Sci</strong></div><div><em>Vine.</em> 1998; <strong>321</strong>:509-516</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.I.KiyashkoP.RichardT.ChandlerT.A.KozlovaD.F.WilliamsStable+carbon+isotope+ratios+differentiate+autotrophs+supporting+animal+diversity+in+Lake+Baikal.+Comptes+Rendus+l%E2%80%99Acad%C3%A9mie+Des+Sci.+-+Ser.+III+-+SciVine3211998509516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">55.</a></div><div>Korlević, P. ∙ Gerber, T. ∙ Gansauge, M.-T. ...</div><div><strong>Reducing microbial and human contamination in DNA extractions from ancient bones and teeth</strong></div><div><em>Biotechniques.</em> 2015; <strong>59</strong>:87-93</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2144/000114320" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84938877820" target="_blank" aria-label="Open Scopus in new tab">Scopus (88)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26260087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2144%2F000114320&amp;pmid=26260087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">56.</a></div><div>Korneliussen, T.S. ∙ Albrechtsen, A. ∙ Nielsen, R.</div><div><strong>ANGSD: Analysis of Next Generation Sequencing Data</strong></div><div><em>BMC Bioinformatics.</em> 2014; <strong>15</strong>:356</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-014-0356-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84920729412" target="_blank" aria-label="Open Scopus in new tab">Scopus (517)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25420514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-014-0356-4&amp;pmid=25420514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">57.</a></div><div>Kutyrev, V.V. ∙ Eroshenko, G.A. ∙ Motin, V.L. ...</div><div><strong>Phylogeny and Classification of <i>Yersinia pestis</i> Through the Lens of Strains From the Plague Foci of Commonwealth of Independent States</strong></div><div><em>Front. Microbiol.</em> 2018; <strong>9</strong>:1106</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fmicb.2018.01106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85047535772" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29887859/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffmicb.2018.01106&amp;pmid=29887859" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">58.</a></div><div>Lazaridis, I. ∙ Patterson, N. ∙ Mittnik, A. ...</div><div><strong>Ancient human genomes suggest three ancestral populations for present-day Europeans</strong></div><div><em>Nature.</em> 2014; <strong>513</strong>:409-413</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13673" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84907803675" target="_blank" aria-label="Open Scopus in new tab">Scopus (524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25230663/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13673&amp;pmid=25230663" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">59.</a></div><div>Lazaridis, I. ∙ Nadel, D. ∙ Rollefson, G. ...</div><div><strong>Genomic insights into the origin of farming in the ancient Near East</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:419-424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19310" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27459054/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19310&amp;pmid=27459054" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">60.</a></div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (17717)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">61.</a></div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The sequence alignment/map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (19724)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">62.</a></div><div>Longin, R.</div><div><strong>New method of collagen extraction for radiocarbon dating</strong></div><div><em>Nature.</em> 1971; <strong>230</strong>:241-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/230241a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0015240867" target="_blank" aria-label="Open Scopus in new tab">Scopus (975)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4926713/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F230241a0&amp;pmid=4926713" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">63.</a></div><div>Mamonova, N.N.</div><div><strong>K voprosu o drevnem naselenii Priangar’ya po paleoantropologicheskim dannym</strong></div><div><div>Smirnov, A.P. (Editor)</div></div><div><strong>Problemy Arheologii Urala i Sibiri</strong></div><div>M. Nauka, 1973; 8-29</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.N.MamonovaK+voprosu+o+drevnem+naselenii+Priangar%E2%80%99ya+po+paleoantropologicheskim+dannymA.P.SmirnovProblemy+Arheologii+Urala+i+Sibiri1973M.+Nauka829" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148" title="View in article">64.</a></div><div>Mamonova, N.N. ∙ Sulerzhickij, L.D.</div><div><strong>Radiouglerodnaya khronologiya golocenovyh pogrebenij Pribajkal’ya i Zabajkal’ya po osteologicheskomu materialu iz mogil’nikov</strong></div><div><em>Sov. Arkheologiya.</em> 1989; <strong>1</strong>:19-32</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.N.MamonovaL.D.SulerzhickijRadiouglerodnaya+khronologiya+golocenovyh+pogrebenij+Pribajkal%E2%80%99ya+i+Zabajkal%E2%80%99ya+po+osteologicheskomu+materialu+iz+mogil%E2%80%99nikovSov.+Arkheologiya119891932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148" title="View in article">65.</a></div><div>Mamonova, N.N. ∙ Sulerzhickij, L.D.</div><div><strong>Radiouglerodnaya khronologiya golocenovyh pogrebenij Pribajkal’ya i Zabajkal’ya po osteologicheskomu materialu iz mogil’nikov</strong></div><div><strong>Chelovek, Adaptaciya, Kul’tura</strong></div><div>M. IA RAN, 2008; 127-138</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.N.MamonovaL.D.SulerzhickijRadiouglerodnaya+khronologiya+golocenovyh+pogrebenij+Pribajkal%E2%80%99ya+i+Zabajkal%E2%80%99ya+po+osteologicheskomu+materialu+iz+mogil%E2%80%99nikovChelovek%2C+Adaptaciya%2C+Kul%E2%80%99tura2008M.+IA+RAN127138" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130-2" title="View in article">66.</a></div><div>Mathieson, I. ∙ Lazaridis, I. ∙ Rohland, N. ...</div><div><strong>Genome-wide patterns of selection in 230 ancient Eurasians</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:499-503</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84951286136" target="_blank" aria-label="Open Scopus in new tab">Scopus (453)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26595274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16152&amp;pmid=26595274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref144" title="View in article">67.</a></div><div>Mathieson, I. ∙ Alpaslan-Roodenberg, S. ∙ Posth, C. ...</div><div><strong>The genomic history of southeastern Europe</strong></div><div><em>Nature.</em> 2018; <strong>555</strong>:197-203</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25778" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85043332657" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29466330/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25778&amp;pmid=29466330" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">68.</a></div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina Sequencing Library Preparation for Highly Multiplexed Target Capture and Sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb.prot5448-pdb.prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (825)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">69.</a></div><div>Minagawa, M. ∙ Wada, E.</div><div><strong>Stepwise enrichment of 15N along food chains: Further evidence and the relation between δ15N and animal age</strong></div><div><em>Geochim. Cosmochim. Acta.</em> 1984; <strong>48</strong>:1135-1140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0016-7037(84)90204-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0021342781" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0016-7037%2884%2990204-7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">70.</a></div><div>Minnich, S.A. ∙ Rohde, H.N.</div><div><strong>A rationale for repression and/or loss of motility by pathogenic yersinia in the mammalian host</strong></div><div><div>Perry, R.D. ∙ Fetherston, J.D. (Editors)</div></div><div><strong>The Genus Yersinia: From Genomics to Function</strong></div><div>Springer, 2007; 298-311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-0-387-72124-8_27" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84934436026" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-0-387-72124-8_27" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref144" title="View in article">71.</a></div><div>Mittnik, A. ∙ Wang, C.C. ∙ Pfrengle, S. ...</div><div><strong>The genetic prehistory of the Baltic Sea region</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:1-11</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29317637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=29317637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref144" title="View in article">72.</a></div><div>Mittnik, A. ∙ Massy, K. ∙ Knipper, C. ...</div><div><strong>Kinship-based social inequality in Bronze Age Europe</strong></div><div><em>Science.</em> 2019; <strong>366</strong>:731-734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aax6219" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85074676786" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31601705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aax6219&amp;pmid=31601705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">73.</a></div><div>Montgomery, J.</div><div><strong>Passports from the past: Investigating human dispersals using strontium isotope analysis of tooth enamel</strong></div><div><em>Ann. Hum. Biol.</em> 2010; <strong>37</strong>:325-346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3109/03014461003649297" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951685399" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20367186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3109%2F03014461003649297&amp;pmid=20367186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib158" class="citations"><div id="sref158" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref158-1" title="View in article">74.</a></div><div>Moorjani, P. ∙ Patterson, N.</div><div><a id="intref0250a" href="https://github.com/priyamoorjani/DATES" target="_blank">https://github.com/priyamoorjani/DATES</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MoorjaniN.Pattersonhttps%3A%2F%2Fgithub.com%2Fpriyamoorjani%2FDATES" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">75.</a></div><div>Morelli, G. ∙ Song, Y. ∙ Mazzoni, C.J. ...</div><div><strong>Yersinia pestis genome sequencing identifies patterns of global phylogenetic diversity</strong></div><div><em>Nat. Genet.</em> 2010; <strong>42</strong>:1140-1143</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.705" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-78649470145" target="_blank" aria-label="Open Scopus in new tab">Scopus (331)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21037571/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.705&amp;pmid=21037571" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">76.</a></div><div>Moreno-Mayar, J.V. ∙ Potter, B.A. ∙ Vinner, L. ...</div><div><strong>Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans</strong></div><div><em>Nature.</em> 2018; <strong>553</strong>:203-207</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85040467287" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29323294/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25173&amp;pmid=29323294" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79-2" title="View in article">77.</a></div><div>Movsesian, A.A. ∙ Bakholdina, V.Y. ∙ Pezhemsky, D.V.</div><div><strong>Biological diversity and population history of Middle Holocene hunter-gatherers from the Cis-Baikal region of Siberia</strong></div><div><em>Am. J. Phys. Anthropol.</em> 2014; <strong>155</strong>:559-570</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.22608" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84910107273" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25176172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.22608&amp;pmid=25176172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79-1" title="View in article">78.</a></div><div>Movsesian, A.A. ∙ Bakholdina, V.Y. ∙ Pezhemsky, D.V.</div><div><strong>Sushchestvovala li geneticheskaya preemstvennost’ mezhdu naseleniem razlichnyh etapov Pribajkal’skogo neolita?</strong></div><div><em>Vestn. Mosk. Univ. Seriya 23 Antropol.</em> 2015; <strong>3</strong>:94-104</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.A.MovsesianV.Y.BakholdinaD.V.PezhemskySushchestvovala+li+geneticheskaya+preemstvennost%E2%80%99+mezhdu+naseleniem+razlichnyh+etapov+Pribajkal%E2%80%99skogo+neolita%3FVestn.+Mosk.+Univ.+Seriya+23+Antropol.3201594104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib156" class="citations"><div id="sref156" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">79.</a></div><div>Namouchi, A. ∙ Guellil, M. ∙ Kersten, O. ...</div><div><strong>Integrative approach using <i>Yersinia pestis</i> genomes to revisit the historical landscape of plague during the Medieval Period</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E11790-E11797</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1812865115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85058318857" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30478041/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1812865115&amp;pmid=30478041" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">80.</a></div><div>Nier, A.O.</div><div><strong>The Isotopic Constitution of Strontium, Barium, Bismuth, Thallium and Mercury</strong></div><div><em>Phys. Rev.</em> 1938; <strong>54</strong>:275-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRev.54.275" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0006029013" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRev.54.275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127-2" title="View in article">81.</a></div><div>Nomokonova, T. ∙ Losey, R.J. ∙ Goriunova, O.I. ...</div><div><strong>A freshwater old carbon offset in Lake Baikal, Siberia and problems with the radiocarbon dating of archaeological sediments: Evidence from the Sagan-Zaba II site</strong></div><div><em>Quat. Int.</em> 2013; <strong>290–291</strong>:110-125</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2012.06.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84875540228" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2012.06.007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82-1" title="View in article">82.</a></div><div>Okladnikov, A.P.</div><div><strong>Neoliticheskie stoyanki na Verhnej Lene. Kraeved. v Irkutsk. Gubernii Zap.Stud. Nauch</strong></div><div><em>Kruzhka Kraeved. Pri Irkut. Gos. Un-Te.</em> 1926; <strong>3</strong>:29-38</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeoliticheskie+stoyanki+na+Verhnej+Lene.+Kraeved.+v+Irkutsk.+Gubernii+Zap.Stud.+NauchKruzhka+Kraeved.+Pri+Irkut.+Gos.+Un-Te319262938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">83.</a></div><div>Okladnikov, A.P.</div><div><strong>Neoliticheskij mogil’nik v mestnosti Hapcagaj (Verhnyaya Lena)</strong></div><div><em>Izv. VSORGO.</em> 1928; <strong>53</strong>:125-134</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeoliticheskij+mogil%E2%80%99nik+v+mestnosti+Hapcagaj+%28Verhnyaya+Lena%29Izv.+VSORGO531928125134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">84.</a></div><div>Okladnikov, A.P.</div><div><strong>Raskopki v Kachugskom rajone po pritokam Leny</strong></div><div><em>Vlast’ Tr.</em> 1929; <strong>10</strong></div><div>Avg <i>185</i>, 4</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovRaskopki+v+Kachugskom+rajone+po+pritokam+LenyVlast%E2%80%99+Tr.101929Avg+185%2C+4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85-1" title="View in article">85.</a></div><div>Okladnikov, A.P.</div><div><strong>Neolit i bronzovyj vek Pribajkal’ya. Istoriko-arheologicheskoe issledovanie</strong></div><div><em>CH. I, II. Mater. i Issled. Po Arheol. SSSR.</em> 1950; <strong>18</strong></div><div>M.-L. 412s</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeolit+i+bronzovyj+vek+Pribajkal%E2%80%99ya.+Istoriko-arheologicheskoe+issledovanieCH.+I%2C+II.+Mater.+i+Issled.+Po+Arheol.+SSSR181950M.-L.+412s" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86-1" title="View in article">86.</a></div><div>Okladnikov, A.P.</div><div><strong>Neolit i bronzovyj vek Pribajkal’ya. CH. III. (Glazkovskoe vremya)</strong></div><div><em>Mater. i Issled. Po Arheol. SSSR.</em> 1955; <strong>43</strong></div><div>M.-L. 373 s</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNeolit+i+bronzovyj+vek+Pribajkal%E2%80%99ya.+CH.+III.+%28Glazkovskoe+vremya%29Mater.+i+Issled.+Po+Arheol.+SSSR431955M.-L.+373+s" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">87.</a></div><div>Okladnikov, A.P.</div><div><strong>Otchet ob issledovanii paleoliticheskogo poseleniia Ust’-Kyakhta v 1976 g</strong></div><div><strong>Archive of the Institut Arkheologii i Etnografii Sibirskogo Otdeleniia Rossiiskoi Akademii Nauk</strong></div><div>Novosibirsk, 1977; 78</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovOtchet+ob+issledovanii+paleoliticheskogo+poseleniia+Ust%E2%80%99-Kyakhta+v+1976+gArchive+of+the+Institut+Arkheologii+i+Etnografii+Sibirskogo+Otdeleniia+Rossiiskoi+Akademii+Nauk1977Novosibirsk78" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">88.</a></div><div>Okladnikov, A.P.</div><div><strong>Nauchnyi otchet o raskopkakh stoianki Ust’-Kyakhta 1 (Kyakhtinskii raion BurASSR) v 1978 g</strong></div><div><strong>Archive of the Institut Arkheologii i Etnografii Sibirskogo Otdeleniia Rossiiskoi Akademii Nauk</strong></div><div>Novosibirsk, 1979; 21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.OkladnikovNauchnyi+otchet+o+raskopkakh+stoianki+Ust%E2%80%99-Kyakhta+1+%28Kyakhtinskii+raion+BurASSR%29+v+1978+gArchive+of+the+Institut+Arkheologii+i+Etnografii+Sibirskogo+Otdeleniia+Rossiiskoi+Akademii+Nauk1979Novosibirsk21" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130-2" title="View in article">89.</a></div><div>Olalde, I. ∙ Brace, S. ∙ Allentoft, M.E. ...</div><div><strong>The Beaker phenomenon and the genomic transformation of northwest Europe</strong></div><div><em>Nature.</em> 2018; <strong>555</strong>:190-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature25738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85043325398" target="_blank" aria-label="Open Scopus in new tab">Scopus (150)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29466337/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature25738&amp;pmid=29466337" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96-1" title="View in article">90.</a></div><div>Orlova, L.A.</div><div><strong>Radiocarbon dating of archaeological sites of Siberia and the Far East</strong></div><div><div>Derevianko, A.P. ∙ Kholushkin, Y.P. (Editors)</div></div><div><strong>Methods of Natural Sciences in Archaeological Reconstructions</strong></div><div>Publishing House of the Institute of Archeology and Ethnography SB RAS, 1995; 207-232</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.A.OrlovaRadiocarbon+dating+of+archaeological+sites+of+Siberia+and+the+Far+EastA.P.DereviankoY.P.KholushkinMethods+of+Natural+Sciences+in+Archaeological+Reconstructions1995Publishing+House+of+the+Institute+of+Archeology+and+Ethnography+SB+RAS207232" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91-1" title="View in article">91.</a></div><div>Patterson, N. ∙ Price, A.L. ∙ Reich, D.</div><div><strong>Population structure and eigenanalysis</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020190" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-33846006923" target="_blank" aria-label="Open Scopus in new tab">Scopus (2283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17194218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020190&amp;pmid=17194218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">92.</a></div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (646)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref138-2" title="View in article">93.</a></div><div>Pavlenok, G.D.</div><div><strong>Kostyanaya industriya stoyanki Ust’-Kyahta-3 (Zapadnoe Zabajkal’e) [Bone industry of Ust’-Kyakhta Site (Western Trans-Baikalia)]. Gumanit. Nauk. v Sib</strong></div><div><em>Humanities Sib.</em> 2014; <strong>2</strong>:14-18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.D.PavlenokKostyanaya+industriya+stoyanki+Ust%E2%80%99-Kyahta-3+%28Zapadnoe+Zabajkal%E2%80%99e%29+%5BBone+industry+of+Ust%E2%80%99-Kyakhta+Site+%28Western+Trans-Baikalia%29%5D.+Gumanit.+Nauk.+v+SibHumanities+Sib.220141418" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94-1" title="View in article">94.</a></div><div>Pavlenok, G.D.</div><div><strong>Tekhnologiya obrabotki kamnya v selenginskoi kulture Zapadnogo Zabaikaliya (po materialam stoyanki Ust-Kyakhta-3)</strong></div><div>Altai State University, 2015</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.D.PavlenokTekhnologiya+obrabotki+kamnya+v+selenginskoi+kulture+Zapadnogo+Zabaikaliya+%28po+materialam+stoyanki+Ust-Kyakhta-3%292015Altai+State+University" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref137" title="View in article">95.</a></div><div>Pavlenok, G.D.</div><div><strong>Tekhnologiya izgotovleniya klinovidnyh nukleusov v selenginskoj kul’ture Zapadnogo Zabajkal’ya (po materialam stoyanki Ust’-Kyahta-3). Izv. Altaiskogo Gos. Univ. Istor. Nauk. i Arkheologiia [Bulletin Altai State Univ</strong></div><div><em>Hist. Sci. Archaeol.</em> 2015; <strong>87</strong>:178-184</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.D.PavlenokTekhnologiya+izgotovleniya+klinovidnyh+nukleusov+v+selenginskoj+kul%E2%80%99ture+Zapadnogo+Zabajkal%E2%80%99ya+%28po+materialam+stoyanki+Ust%E2%80%99-Kyahta-3%29.+Izv.+Altaiskogo+Gos.+Univ.+Istor.+Nauk.+i+Arkheologiia+%5BBulletin+Altai+State+UnivHist.+Sci.+Archaeol.872015178184" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96-1" title="View in article">96.</a></div><div>Pavlenok, G.D. ∙ Zubova, A.V.</div><div><strong>New Dental Finds Associated with the Paleolithic Selenga Culture, Western Trans-Baikal Region</strong></div><div><em>Archaeol. Ethnol. Anthropol. Eurasia.</em> 2019; <strong>47</strong>:3-11</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.17746/1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85076466976" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.17746%2F1563-0110.2019.47.3.003-011" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97-1" title="View in article">97.</a></div><div>Peltzer, A. ∙ Jäger, G. ∙ Herbig, A. ...</div><div><strong>EAGER: efficient ancient genome reconstruction</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:60</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0918-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27036623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0918-z&amp;pmid=27036623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">98.</a></div><div>Pin, C. ∙ Briot, D. ∙ Bassin, C. ...</div><div><strong>Concomitant separation of strontium and samarium-neodymium for isotopic analysis in silicate samples, based on specific extraction chromatography</strong></div><div><em>Anal. Chim. Acta.</em> 1994; <strong>298</strong>:209-217</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0003-2670(94)00274-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0028127258" target="_blank" aria-label="Open Scopus in new tab">Scopus (393)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0003-2670%2894%2900274-6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">99.</a></div><div>Posth, C. ∙ Nakatsuka, N. ∙ Lazaridis, I. ...</div><div><strong>Reconstructing the Deep Population History of Central and South America</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:1185-1197</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2018.10.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2018.10.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85056451563" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30415837/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.10.027&amp;pmid=30415837" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100-1" title="View in article">100.</a></div><div>Poznik, G.D.</div><div><strong>Identifying Y-chromosome haplogroups in arbitrarily large samples of sequenced or genotyped men</strong></div><div><em>bioRxiv.</em> 2016;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/088716" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F088716" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">101.</a></div><div>Pradel, E. ∙ Lemaître, N. ∙ Merchez, M. ...</div><div><strong>New insights into how Yersinia pestis adapts to its mammalian host during bubonic plague</strong></div><div><em>PLoS Pathog.</em> 2014; <strong>10</strong>:e1004029</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1004029" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84897393547" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24675805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1004029&amp;pmid=24675805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">102.</a></div><div>Price, T.D. ∙ Swick, R.W. ∙ Chase, E.P.</div><div><strong>Bone chemistry and prehistoric diet: strontium studies of laboratory rats</strong></div><div><em>Am. J. Phys. Anthropol.</em> 1986; <strong>70</strong>:365-375</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.1330700311" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0022743649" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3752231/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.1330700311&amp;pmid=3752231" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">103.</a></div><div>Price, T.D. ∙ Burton, J.H. ∙ Bentley, R.A.</div><div><strong>The Characterization of Biologically Available Strontium Isotope Ratios for the Study of Prehistoric Migration</strong></div><div><em>Archaeometry.</em> 2002; <strong>44</strong>:117-135</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/1475-4754.00047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0036462393" target="_blank" aria-label="Open Scopus in new tab">Scopus (496)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2F1475-4754.00047" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104-1" title="View in article">104.</a></div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (14939)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">105.</a></div><div>Quinlan, A.R. ∙ Hall, I.M.</div><div><strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:841-842</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq033" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-77951770756" target="_blank" aria-label="Open Scopus in new tab">Scopus (7022)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq033&amp;pmid=20110278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref150" title="View in article">106.</a></div><div>R Development Core Team</div><div><strong>Computational Many-Particle Physics</strong></div><div>Springer Berlin Heidelberg, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Development+Core+TeamComputational+Many-Particle+Physics2008Springer+Berlin+Heidelberg" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref129-1" title="View in article">107.</a></div><div>Raghavan, M. ∙ Skoglund, P. ∙ Graf, K.E. ...</div><div><strong>Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans</strong></div><div><em>Nature.</em> 2014; <strong>505</strong>:87-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12736" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84892367625" target="_blank" aria-label="Open Scopus in new tab">Scopus (334)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24256729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12736&amp;pmid=24256729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112-1" title="View in article">108.</a></div><div>Raghavan, M. ∙ DeGiorgio, M. ∙ Albrechtsen, A. ...</div><div><strong>The genetic prehistory of the New World Arctic</strong></div><div><em>Science.</em> 2014; <strong>345</strong></div><div>1255832–1255832</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1255832" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84907279761" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25170159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1255832&amp;pmid=25170159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">109.</a></div><div>Raghavan, M. ∙ Steinrücken, M. ∙ Harris, K. ...</div><div><strong>Population genetics. Genomic evidence for the Pleistocene and recent population history of Native Americans</strong></div><div><em>Science.</em> 2015; <strong>349</strong></div><div>aab3884–aab3884</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aab3884" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84939864401" target="_blank" aria-label="Open Scopus in new tab">Scopus (182)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26198033/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aab3884&amp;pmid=26198033" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127-2" title="View in article">110.</a></div><div>Ramsey, C.B. ∙ Schulting, R. ∙ Goriunova, O.I. ...</div><div><strong>Analyzing Radiocarbon Reservoir Offsets Through Stable Nitrogen Isotopes and Bayesian Modeling: A Case Study Using Paired Human and Faunal Remains from the Cis-Baikal Region, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:789-799</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17160" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899887915" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17160" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref133-1" title="View in article">111.</a></div><div>Rascovan, N. ∙ Sjögren, K.-G. ∙ Kristiansen, K. ...</div><div><strong>Emergence and Spread of Basal Lineages of Yersinia pestis during the Neolithic Decline</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:295-305</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2018.11.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2018.11.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85059906505" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30528431/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.11.005&amp;pmid=30528431" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112-1" title="View in article">112.</a></div><div>Rasmussen, M. ∙ Li, Y. ∙ Lindgreen, S. ...</div><div><strong>Ancient human genome sequence of an extinct Palaeo-Eskimo</strong></div><div><em>Nature.</em> 2010; <strong>463</strong>:757-762</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08835" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-76749141381" target="_blank" aria-label="Open Scopus in new tab">Scopus (464)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20148029/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08835&amp;pmid=20148029" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130-2" title="View in article">113.</a></div><div>Rasmussen, M. ∙ Anzick, S.L. ∙ Waters, M.R. ...</div><div><strong>The genome of a Late Pleistocene human from a Clovis burial site in western Montana</strong></div><div><em>Nature.</em> 2014; <strong>506</strong>:225-229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84893961036" target="_blank" aria-label="Open Scopus in new tab">Scopus (260)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24522598/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13025&amp;pmid=24522598" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130-2" title="View in article">114.</a></div><div>Rasmussen, M. ∙ Sikora, M. ∙ Albrechtsen, A. ...</div><div><strong>The ancestry and affiliations of Kennewick Man</strong></div><div><em>Nature.</em> 2015; <strong>523</strong>:455-458</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84937475681" target="_blank" aria-label="Open Scopus in new tab">Scopus (103)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26087396/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14625&amp;pmid=26087396" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref133-1" title="View in article">115.</a></div><div>Rasmussen, S. ∙ Allentoft, M.E. ∙ Nielsen, K. ...</div><div><strong>Early divergent strains of Yersinia pestis in Eurasia 5,000 years ago</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:571-582</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cell.2015.10.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26496604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.10.009&amp;pmid=26496604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">116.</a></div><div>Reich, D. ∙ Thangaraj, K. ∙ Patterson, N. ...</div><div><strong>Reconstructing Indian population history</strong></div><div><em>Nature.</em> 2009; <strong>461</strong>:489-494</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-70349472875" target="_blank" aria-label="Open Scopus in new tab">Scopus (824)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19779445/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08365&amp;pmid=19779445" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117" title="View in article">117.</a></div><div>Reimer, P.J. ∙ Bard, E. ∙ Bayliss, A. ...</div><div><strong>IntCal13 and Marine13 Radiocarbon Age Calibration Curves 0–50,000 Years cal BP</strong></div><div><em>Radiocarbon.</em> 2013; <strong>55</strong>:1869-1887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_js_rc.55.16947" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84887538928" target="_blank" aria-label="Open Scopus in new tab">Scopus (7346)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_js_rc.55.16947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118-1" title="View in article">118.</a></div><div>Renaud, G. ∙ Slon, V. ∙ Duggan, A.T. ...</div><div><strong>Schmutzi: estimation of contamination and endogenous mitochondrial consensus calling for ancient DNA</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0776-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84943738237" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26458810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0776-0&amp;pmid=26458810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119" title="View in article">119.</a></div><div>Rohland, N. ∙ Hofreiter, M.</div><div><strong>Ancient DNA extraction from bones and teeth</strong></div><div><em>Nat. Protoc.</em> 2007; <strong>2</strong>:1756-1762</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2007.247" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-34548579114" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17641642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2007.247&amp;pmid=17641642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="sref120" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref120" title="View in article">120.</a></div><div>Rohland, N. ∙ Harney, E. ∙ Mallick, S. ...</div><div><strong>Partial uracil-DNA-glycosylase treatment for screening of ancient DNA</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2015; <strong>370</strong>:20130624</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2013.0624" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84916210143" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25487342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2013.0624&amp;pmid=25487342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="sref121" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref121" title="View in article">121.</a></div><div>Rokita, E. ∙ Hermes, C. ∙ Nolting, H.-F. ...</div><div><strong>Substitution of calcium by strontium within selected calcium phosphates</strong></div><div><em>J.&nbsp;Cryst. Growth.</em> 1993; <strong>130</strong>:543-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-0248(93)90543-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0027608865" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-0248%2893%2990543-6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="sref122" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130-2" title="View in article">122.</a></div><div>Saag, L. ∙ Varul, L. ∙ Scheib, C.L. ...</div><div><strong>Extensive Farming in Estonia Started through a Sex-Biased Migration from the Steppe</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:2185-2193</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.06.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.cub.2017.06.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85023634728" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28712569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.06.022&amp;pmid=28712569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib123" class="citations"><div id="sref123" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref123-1" title="View in article">123.</a></div><div>Scheib, C.L. ∙ Li, H. ∙ Desai, T. ...</div><div><strong>Ancient human parallel lineages within North America contributed to a coastal expansion</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:1024-1027</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar6851" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85048178453" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29853687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar6851&amp;pmid=29853687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib124" class="citations"><div id="sref124" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref124-1" title="View in article">124.</a></div><div>Schubert, M. ∙ Lindgreen, S. ∙ Orlando, L.</div><div><strong>AdapterRemoval v2: rapid adapter trimming, identification, and read merging</strong></div><div><em>BMC Res. Notes.</em> 2016; <strong>9</strong>:88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13104-016-1900-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84958153152" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26868221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13104-016-1900-2&amp;pmid=26868221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib125" class="citations"><div id="sref125" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref125" title="View in article">125.</a></div><div>Schuenemann, V.J. ∙ Bos, K. ∙ DeWitte, S. ...</div><div><strong>Targeted enrichment of ancient pathogens yielding the pPCP1 plasmid of Yersinia pestis from victims of the Black Death</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:E746-E752</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1105107108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-80053137140" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21876176/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1105107108&amp;pmid=21876176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib126" class="citations"><div id="sref126" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127-1" title="View in article">126.</a></div><div>Schulting, R.J. ∙ Ramsey, C.B. ∙ Bazaliiskii, V.I. ...</div><div><strong>Freshwater Reservoir Offsets Investigated Through Paired Human-Faunal 14C Dating and Stable Carbon and Nitrogen Isotope Analysis at Lake Baikal, Siberia</strong></div><div><em>Radiocarbon.</em> 2014; <strong>56</strong>:991-1008</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/56.17963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899833606" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2F56.17963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib127" class="citations"><div id="sref127" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127-1" title="View in article">127.</a></div><div>Schulting, R.J. ∙ Bronk Ramsey, C. ∙ Bazaliiskii, V.I. ...</div><div><strong>Highly Variable Freshwater Reservoir Offsets Found along the Upper Lena Watershed, Cis-Baikal, Southeast Siberia</strong></div><div><em>Radiocarbon.</em> 2015; <strong>57</strong>:581-593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2458/azu_rc.57.18458" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84942103801" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2458%2Fazu_rc.57.18458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib128" class="citations"><div id="sref128" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref128" title="View in article">128.</a></div><div>Sealy, J.C. ∙ van der Merwe, N.J. ∙ Thorp, J.A.L. ...</div><div><strong>Nitrogen isotopic ecology in southern Africa: Implications for environmental and dietary tracing</strong></div><div><em>Geochim. Cosmochim. Acta.</em> 1987; <strong>51</strong>:2707-2717</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0016-7037(87)90151-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0016-7037%2887%2990151-7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib129" class="citations"><div id="sref129" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref129-1" title="View in article">129.</a></div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib130" class="citations"><div id="sref130" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130-1" title="View in article">130.</a></div><div>Siska, V. ∙ Jones, E.R. ∙ Jeon, S. ...</div><div><strong>Genome-wide data from two early Neolithic East Asian individuals dating to 7700 years ago</strong></div><div><em>Sci. Adv.</em> 2017; <strong>3</strong>:e1601877</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1601877" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28164156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1601877&amp;pmid=28164156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib131" class="citations"><div id="sref131" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref131-1" title="View in article">131.</a></div><div>Smith, B.N. ∙ Epstein, S.</div><div><strong>Two categories of c/c ratios for higher plants</strong></div><div><em>Plant Physiol.</em> 1971; <strong>47</strong>:380-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1104/pp.47.3.380" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16657626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1104%2Fpp.47.3.380&amp;pmid=16657626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib132" class="citations"><div id="sref132" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132-1" title="View in article">132.</a></div><div>Spiliopoulou, A. ∙ Colombo, M. ∙ Orchard, P. ...</div><div><strong>GeneImp: Fast imputation to large reference panels using genotype likelihoods from ultralow coverage sequencing</strong></div><div><em>Genetics.</em> 2017; <strong>206</strong>:91-104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.117.200063" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85020715759" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28348060/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.117.200063&amp;pmid=28348060" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib157" class="citations"><div id="sref157" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">133.</a></div><div>Spyrou, M.A. ∙ Tukhbatova, R.I. ∙ Feldman, M. ...</div><div><strong>Historical <i>Y. pestis</i> genomes reveal the European Black Death as the source of ancient and modern plague pandemics</strong></div><div><em>Cell Host &amp; Microbe.</em> 2016; <strong>19</strong>:874-881</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2016.05.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2016.05.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27281573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2016.05.012&amp;pmid=27281573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib133" class="citations"><div id="sref133" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref133-1" title="View in article">134.</a></div><div>Spyrou, M.A. ∙ Tukhbatova, R.I. ∙ Wang, C.-C. ...</div><div><strong>Analysis of 3800-year-old Yersinia pestis genomes suggests Bronze Age origin for bubonic plague</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:2234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-04550-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85048310202" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29884871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-04550-9&amp;pmid=29884871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib134" class="citations"><div id="sref134" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">135.</a></div><div>Spyrou, M.A. ∙ Keller, M. ∙ Tukhbatova, R.I. ...</div><div><strong>Phylogeography of the second plague pandemic revealed through analysis of historical Yersinia pestis genomes</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12154-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85072923513" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31578321/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12154-0&amp;pmid=31578321" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib135" class="citations"><div id="sref135" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref135-1" title="View in article">136.</a></div><div>Stamatakis, A.</div><div><strong>RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:1312-1313</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu033" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84899553363" target="_blank" aria-label="Open Scopus in new tab">Scopus (11412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24451623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu033&amp;pmid=24451623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib136" class="citations"><div id="sref136" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref136-1" title="View in article">137.</a></div><div>Sun, Y.-C. ∙ Jarrett, C.O. ∙ Bosio, C.F. ...</div><div><strong>Retracing the evolutionary path that led to flea-borne transmission of Yersinia pestis</strong></div><div><em>Cell Host Microbe.</em> 2014; <strong>15</strong>:578-586</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2014.04.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=10.1016%2Fj.chom.2014.04.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24832452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2014.04.003&amp;pmid=24832452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib137" class="citations"><div id="sref137" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref137" title="View in article">138.</a></div><div>Tashak, V.I.</div><div><strong>Tortsovye klinovidnye nukleusy Zapadnogo Zabaikaliya v pozdnem paleolite i mezolite</strong></div><div><strong>Kamennyi Vek Yuzhnoi Sibiri i Mongolii: Teoreticheskie Problemy I Novye Otkrytiya</strong></div><div>Buryat scientific center SB RAS, 2000; 59-74</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.I.TashakTortsovye+klinovidnye+nukleusy+Zapadnogo+Zabaikaliya+v+pozdnem+paleolite+i+mezoliteKamennyi+Vek+Yuzhnoi+Sibiri+i+Mongolii%3A+Teoreticheskie+Problemy+I+Novye+Otkrytiya2000Buryat+scientific+center+SB+RAS5974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib138" class="citations"><div id="sref138" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref138-1" title="View in article">139.</a></div><div>Tashak, V.I.</div><div><strong>Paleoliticheskie i mezoliticheskie pamiatniki Ust’-Kiakhty</strong></div><div>Buryatia Scientific Center of the Siberian Branch of the Russian Academy of Sciences, 2005</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.I.TashakPaleoliticheskie+i+mezoliticheskie+pamiatniki+Ust%E2%80%99-Kiakhty2005Buryatia+Scientific+Center+of+the+Siberian+Branch+of+the+Russian+Academy+of+Sciences" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib139" class="citations"><div id="sref139" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref139" title="View in article">140.</a></div><div>Tavaré, S.</div><div><strong>Some probabilistic and statistical problems in the analysis of DNA sequences</strong></div><div><em>Lect. Math Life Sci.</em> 1986; <strong>17</strong>:57-86</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.Tavar%C3%A9Some+probabilistic+and+statistical+problems+in+the+analysis+of+DNA+sequencesLect.+Math+Life+Sci.1719865786" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib140" class="citations"><div id="sref140" class="citation"><div class="citation-content"><div class="label"><a href="#figs6" title="View in article">141.</a></div><div>Thorvaldsdóttir, H. ∙ Robinson, J.T. ∙ Mesirov, J.P.</div><div><strong>Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration</strong></div><div><em>Brief. Bioinform.</em> 2013; <strong>14</strong>:178-192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bib/bbs017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84875634162" target="_blank" aria-label="Open Scopus in new tab">Scopus (3379)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22517427/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbib%2Fbbs017&amp;pmid=22517427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib141" class="citations"><div id="sref141" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref141-1" title="View in article">142.</a></div><div>Vågene, Å.J. ∙ Herbig, A. ∙ Campana, M.G. ...</div><div><strong>Salmonella enterica genomes from victims of a major sixteenth-century epidemic in Mexico</strong></div><div><em>Nat. Ecol. Evol.</em> 2018; <strong>2</strong>:520-528</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-017-0446-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85040669654" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29335577/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-017-0446-6&amp;pmid=29335577" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib142" class="citations"><div id="sref142" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref142" title="View in article">143.</a></div>Vlast’truda (1928). 21 oktyabrya. <i>246</i>, 4.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Vlast%E2%80%99truda+%281928%29.+21+oktyabrya.+246%2C+4." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib143" class="citations"><div id="sref143" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref143-1" title="View in article">144.</a></div>Vlast’truda (1929). 28 iyulya. <i>17</i>, 4.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Vlast%E2%80%99truda+%281929%29.+28+iyulya.+17%2C+4." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib144" class="citations"><div id="sref144" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref144" title="View in article">145.</a></div><div>Wang, C.C. ∙ Reinhold, S. ∙ Kalmykov, A. ...</div><div><strong>Ancient human genome-wide data from a 3000-year interval in the Caucasus corresponds with eco-geographic regions</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:590</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-08220-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-85061007525" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30713341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-08220-8&amp;pmid=30713341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib145" class="citations"><div id="sref145" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref145" title="View in article">146.</a></div><div>Weber, A.</div><div><strong>The neolithic and early bronze age of the Lake Baikal Region: A review of recent research</strong></div><div><em>J.&nbsp;World Prehist.</em> 1995; <strong>9</strong>:99-165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF02221004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0029532484" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF02221004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib146" class="citations"><div id="sref146" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref146" title="View in article">147.</a></div><div>Weber, A.W. ∙ Bettinger, R.</div><div><strong>Middle Holocene hunter-gatherers of Cis-Baikal, Siberia: An overview for the new century</strong></div><div><em>J.&nbsp;Anthropol. Archaeol.</em> 2010; <strong>29</strong>:491-506</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jaa.2010.08.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-78349304828" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaa.2010.08.002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib147" class="citations"><div id="sref147" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148" title="View in article">148.</a></div><div>Weber, A.W. ∙ Link, D.W. ∙ Katzenberg, M.A.</div><div><strong>Hunter-gatherer culture change and continuity in the middle holocene of the Cis-Baikal, Siberia</strong></div><div><em>J.&nbsp;Anthropol. Archaeol.</em> 2002; <strong>21</strong>:230-299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jaar.2001.0395" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-0036324599" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjaar.2001.0395" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib148" class="citations"><div id="sref148" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148" title="View in article">149.</a></div><div>Weber, A.W. ∙ Beukens, R.P. ∙ Bazaliiskii, V.I. ...</div><div><strong>Radiocarbon Dates from Neolithic and Bronze Age Hunter-Gatherer Cemeteries in the Cis-Baikal Region of Siberia</strong></div><div><em>Radiocarbon.</em> 2006; <strong>48</strong>:127-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/S0033822200035463" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-33645215976" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FS0033822200035463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib149" class="citations"><div id="sref149" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref149" title="View in article">150.</a></div><div>Weber, A.W. ∙ Schulting, R.J. ∙ Bronk Ramsey, C. ...</div><div><strong>Chronology of middle Holocene hunter–gatherers in the Cis-Baikal region of Siberia: Corrections based on examination of the freshwater reservoir effect</strong></div><div><em>Quat. Int.</em> 2016; <strong>419</strong>:74-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2015.12.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-84956468879" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2015.12.003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib150" class="citations"><div id="sref150" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref150" title="View in article">151.</a></div><div>Wickham, H.</div><div><strong>ggplot2: Elegant Graphics For Data Analysis</strong></div><div>Springer, 2016</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-3-319-24277-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-3-319-24277-4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib151" class="citations"><div id="sref151" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">152.</a></div><div>Zhgenti, E. ∙ Johnson, S.L. ∙ Davenport, K.W. ...</div><div><strong>Genome Assemblies for 11 Yersinia pestis Strains Isolated in the Caucasus Region</strong></div><div><em>Genome Announc.</em> 2015; <strong>3</strong>:3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/genomeA.01030-15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FgenomeA.01030-15" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib152" class="citations"><div id="sref152" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref152-1" title="View in article">153.</a></div><div>Zhou, D. ∙ Yang, R.</div><div><strong>Molecular Darwinian evolution of virulence in Yersinia pestis</strong></div><div><em>Infect. Immun.</em> 2009; <strong>77</strong>:2242-2250</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/IAI.01477-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.04.037&amp;key=2-s2.0-66549101505" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19289506/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FIAI.01477-08&amp;pmid=19289506" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib153" class="citations"><div id="sref153" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref153" title="View in article">154.</a></div><div>Zimbler, D.L. ∙ Schroeder, J.A. ∙ Eddy, J.L. ...</div><div><strong>Early emergence of Yersinia pestis as a severe respiratory pathogen</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:7487</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms8487" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26123398/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms8487&amp;pmid=26123398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib154" class="citations"><div id="sref154" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref154" title="View in article">155.</a></div><div>Zotkina, L.V. ∙ Pavlenok, G.D. ∙ Tashak, V.I.</div><div><strong>Tekhnologiya proizvodstva busin iz skorlupy yaic strausa v final’nom paleolite Zapadnogo Zabajkal’ya [Technology of Ostrich Eggshell Bead Production in the Final Palaeolithic of Western Transbaikalia]. Strat. plus</strong></div><div><em>Archaeol. Cult. Anthropol.</em> 2018; <strong>1</strong>:181-197</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.V.ZotkinaG.D.PavlenokV.I.TashakTekhnologiya+proizvodstva+busin+iz+skorlupy+yaic+strausa+v+final%E2%80%99nom+paleolite+Zapadnogo+Zabajkal%E2%80%99ya+%5BTechnology+of+Ostrich+Eggshell+Bead+Production+in+the+Final+Palaeolithic+of+Western+Transbaikalia%5D.+Strat.+plusArchaeol.+Cult.+Anthropol.12018181197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib155" class="citations"><div id="sref155" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref155" title="View in article">156.</a></div><div>Zubkov, V.S.</div><div><strong>Materialy iz drevnih mogil s territorii poselka Kachug i u derevni SHishkino na Verhnej Lene (raskopki A. P. Okladnikova 1928-1930, 1941 gg.). Izv. Lab. Drevnih Tekhnologij</strong></div><div><em>Irkutsk Izd-vo IrGTU.</em> 2010; <strong>8</strong>:135-146</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.S.ZubkovMaterialy+iz+drevnih+mogil+s+territorii+poselka+Kachug+i+u+derevni+SHishkino+na+Verhnej+Lene+%28raskopki+A.+P.+Okladnikova+1928-1930%2C+1941+gg.%29.+Izv.+Lab.+Drevnih+TekhnologijIrkutsk+Izd-vo+IrGTU.82010135146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (11)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/f3c5ec7d-3001-4f0d-acd0-0cf5f8457879/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/20983171-5174-46af-bf6a-d14a7b8a597a/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/0ef045e9-30d4-4452-a712-bbca7dd50a6b/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/7bc69245-783e-4daa-a0b4-44ff5d18b6b6/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/4075fab7-317c-445a-806d-eba8f732461d/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/4f50b2ae-0dc4-4b33-ab05-f2172d2c1b68/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/3b13159e-698e-4288-9805-81698053401c/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/340cc90d-1602-4577-9dfd-e596b7218561/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/a15d65a2-95dd-4b78-a9c2-1ef3bbc5fa64/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/84abb332-ca65-44e6-bb12-80c5fbab8e2f/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/asset/11061e01-1bb2-4976-b68e-bd26a6841493/main.assets/gr5.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS009286742030502X%3Bissue%3Aissue%3Apii%5C%3AS0092867419X00136%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">71</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">126</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">46</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">421</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2020.04.037&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (6)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/attachment/48aa7473-9937-4e3e-84f6-00a27dce72ef/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (34.81 KB)</span></a><div class="core-description">Table S1. Sample Statistics and Reference Information, Related to Figures 1 and S1 and STAR Methods</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/attachment/e28e1521-bc51-4104-8099-9e18e9e27b70/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (39.38 KB)</span></a><div class="core-description">Table S2. The Genetic Affinity among UKY, Kolyma, USR1, and the First American People Revealed by <i>f</i><sub>4</sub> Statistics, Related to Figures 2 and S2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/attachment/b50425d1-f129-4239-8282-eff087b8dd64/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (29.94 KB)</span></a><div class="core-description">Table S3. <i>qpAdm</i> and <i>qpWave</i> Modeling of Our Newly Reported Individuals and Related Population, Related to Figures 3, 4, and 5</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/attachment/ae848b68-0b21-4b2f-846a-560deb56891e/mmc4.pdf" download="mmc4.pdf"><span id="mmc4-link">PDF (97.27 KB)</span></a><div class="core-description">Table S4. Genetic Affinity of Newly Genotyped Early Neolithic to Bronze Age Individuals with Worldwide Population, Revealed by Outgroup <i>f</i><sub>3</sub> Statistics, Related to Figure&nbsp;3</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/attachment/25484f6f-581b-4e00-8fda-ed97fbeaa3ec/mmc5.pdf" download="mmc5.pdf"><span id="mmc5-link">PDF (64.29 KB)</span></a><div class="core-description">Table S5. Admixture Dating of Early Neolithic and Bronze Age Baikal Individuals, as well as Okunevo and BZK002, Using <i>DATES</i>, Related to Figures 3, 5, and S4</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.04.037/attachment/08562660-e4c4-4173-8232-3d905bf42a3d/mmc6.xlsx" download="mmc6.xlsx"><span id="mmc6-link">Spreadsheet (13.10 KB)</span></a><div class="core-description">Table S6. <i>Yersinia pestis</i> Post-capture Mapping Statistics and BEAST Tip Dates Inference of GLZ001 and GLZ002, Related to Figures 4 and S5</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S009286742030502X" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(22)01892-9" target="_blank" title="Middle Holocene Siberian genomes reveal highly connected gene pools throughout North Asia">Middle Holocene Siberian genomes reveal highly connected gene pools throughout North Asia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">January 12, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(24)00899-7" target="_blank" title="Bronze Age cheese reveals human-Lactobacillus interactions over evolutionary history">Bronze Age cheese reveals human-Lactobacillus interactions over evolutionary history</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">September 25, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(20)31506-2" target="_blank" title="Genomes of Pleistocene Siberian Wolves Uncover Multiple Extinct Wolf Lineages">Genomes of Pleistocene Siberian Wolves Uncover Multiple Extinct Wolf Lineages</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ramos-Madrigal et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">October 29, 2020</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(24)01559-8" target="_blank" title="Motion detection: Specific thalamocortical connections revealed">Motion detection: Specific thalamocortical connections revealed</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Matsumoto et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">January 06, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(19)30424-5" target="_blank" title="The Arrival of Siberian Ancestry Connecting the Eastern Baltic to Uralic Speakers further East">The Arrival of Siberian Ancestry Connecting the Eastern Baltic to Uralic Speakers further East</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Saag et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">May 9, 2019</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(22)00955-0" target="_blank" title="Repeated intraperitoneal injection of ovine growth hormone accelerates growth in sub-yearling Siberian sturgeon Acipenser baerii">Repeated intraperitoneal injection of ovine growth hormone accelerates growth in sub-yearling Siberian sturgeon Acipenser baerii</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Falahatkar et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">June 7, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(22)01101-0" target="_blank" title="Ancient Yersinia pestis and Salmonella enterica genomes from Bronze Age Crete">Ancient Yersinia pestis and Salmonella enterica genomes from Bronze Age Crete</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Neumann et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">July 25, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/matter/fulltext/S2590-2385(24)00635-0" target="_blank" title="Long-span delivery of differentiable hybrid robots for restoration of neural connections">Long-span delivery of differentiable hybrid robots for restoration of neural connections</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Shen et al.</li></ul></div><div class="linked-article__details"><span><i>Matter</i></span><span id="item_date">January 13, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)01058-1" target="_blank" title="Horizontal cortical connections shape intrinsic traveling waves into feature-selective motifs that regulate perceptual sensitivity">Horizontal cortical connections shape intrinsic traveling waves into feature-selective motifs that regulate perceptual sensitivity</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Davis et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">September 6, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)02151-0" target="_blank" title="Salinity-induced succession of phototrophic communities in a southwestern Siberian soda lake during the solar activity cycle">Salinity-induced succession of phototrophic communities in a southwestern Siberian soda lake during the solar activity cycle</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Samylina et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">February 17, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)07457-7" target="_blank" title="Rainbow connections of bioriented graphs">Rainbow connections of bioriented graphs</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">May 20, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-chemical-biology/fulltext/S2451-9456(25)00064-9" target="_blank" title="Disrupting neuron-tumor networking connections">Disrupting neuron-tumor networking connections</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Shen et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Chemical Biology</i></span><span id="item_date">March 20, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)05984-X" target="_blank" title="Corporate social responsibility, political connections, and barrier industry diversification: Evidence from China">Corporate social responsibility, political connections, and barrier industry diversification: Evidence from China</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">April 20, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)11917-2" target="_blank" title="Development and systematic review of connection techniques for RC Precast structural elements: Beam and Column Connection">Development and systematic review of connection techniques for RC Precast structural elements: Beam and Column Connection</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Xiao et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 8, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)04983-6" target="_blank" title="Characterization of atypical pathogenic Aeromonas salmonicida isolated from a diseased Siberian sturgeon (Acipenser baerii)">Characterization of atypical pathogenic Aeromonas salmonicida isolated from a diseased Siberian sturgeon (Acipenser baerii)</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Bakiyev et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 3, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(22)00748-9" target="_blank" title="Helicobacter pylori genomes reveal Paleolithic human migration to the east end of Asia">Helicobacter pylori genomes reveal Paleolithic human migration to the east end of Asia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Suzuki et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">May 29, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)06103-6" target="_blank" title="First record of well-preserved canid coprolites from Eurasia: New insights into the unique ecological niche of Yuanmou Basin">First record of well-preserved canid coprolites from Eurasia: New insights into the unique ecological niche of Yuanmou Basin</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Farjand et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">April 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(25)00518-5" target="_blank" title="Infrequent strong connections constrain connectomic predictions of neuronal function">Infrequent strong connections constrain connectomic predictions of neuronal function</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Currier et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">June 2, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)05670-0" target="_blank" title="Anomalous venoatrial connections – CT and MRI assessment">Anomalous venoatrial connections – CT and MRI assessment</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Celona et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 26, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(22)00459-5" target="_blank" title="Technological insights on the Early-Middle Bronze Age pottery of Monte Meana cave (Sardinia, Italy)">Technological insights on the Early-Middle Bronze Age pottery of Monte Meana cave (Sardinia, Italy)</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Paglietti et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">March 25, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)06846-4" target="_blank" title="Evolution characteristics and performance evaluation of extended end-plate joints with single and double side connections under cyclic loading">Evolution characteristics and performance evaluation of extended end-plate joints with single and double side connections under cyclic loading</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Luo et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">May 6, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)03933-1" target="_blank" title="The loss of political connections and the fluctuation of corporate stock prices: Moderating effect based on media attention">The loss of political connections and the fluctuation of corporate stock prices: Moderating effect based on media attention</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sun et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">March 9, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)10655-X" target="_blank" title="Strengthening mechanism of sub-nano Ni3P and Cu3P phases particles on the Pb–Sn bronze alloy">Strengthening mechanism of sub-nano Ni3P and Cu3P phases particles on the Pb–Sn bronze alloy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ren et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 13, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/one-earth/fulltext/S2590-3322(25)00020-X" target="_blank" title="Cross-country variation in people’s connection to nature">Cross-country variation in people’s connection to nature</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Soga et al.</li></ul></div><div class="linked-article__details"><span><i>One Earth</i></span><span id="item_date">February 10, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(23)01266-4" target="_blank" title="Blocking of ebolavirus spread through intercellular connections by an MPER-specific antibody depends on BST2/tetherin">Blocking of ebolavirus spread through intercellular connections by an MPER-specific antibody depends on BST2/tetherin</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Santos et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">October 17, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)17265-9" target="_blank" title="Mathematical modeling and QSPR analysis of hepatitis treatment drugs through connection indices an innovative approach">Mathematical modeling and QSPR analysis of hepatitis treatment drugs through connection indices an innovative approach</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hakami et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">December 17, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)09476-8" target="_blank" title="How is the effect of social media on intention to outdoor recreations? A study using personal social connections as a moderator">How is the effect of social media on intention to outdoor recreations? A study using personal social connections as a moderator</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hung et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">November 12, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)15715-5" target="_blank" title="Improving understanding of ferroptosis: Molecular mechanisms, connection with cellular senescence and implications for aging">Improving understanding of ferroptosis: Molecular mechanisms, connection with cellular senescence and implications for aging</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>De Leon-Oliva et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">October 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)05898-5" target="_blank" title="Genetic history of esophageal cancer group in southwestern China revealed by Y‐chromosome STRs and genomic evolutionary connection analysis">Genetic history of esophageal cancer group in southwestern China revealed by Y‐chromosome STRs and genomic evolutionary connection analysis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jia et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">April 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/ajhg/fulltext/S0002-9297(22)00539-0" target="_blank" title="Leveraging drug perturbation to reveal genetic regulators of hepatic gene expression in African Americans">Leveraging drug perturbation to reveal genetic regulators of hepatic gene expression in African Americans</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhong et al.</li></ul></div><div class="linked-article__details"><span><i>The American Journal of Human Genetics</i></span><span id="item_date">January 05, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(23)00546-0" target="_blank" title="TIMely connections: APOE4, aging, and Alzheimer’s">TIMely connections: APOE4, aging, and Alzheimer’s</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Qingyun Li</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">January 09, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/cell-biology/fulltext/S0962-8924(24)00118-1" target="_blank" title="Lipids and chromatin: a tale of intriguing connections shaping genomic landscapes">Lipids and chromatin: a tale of intriguing connections shaping genomic landscapes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sosa Ponce et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Cell Biology</i></span><span id="item_date">July 25, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/one-earth/fulltext/S2590-3322(23)00035-0" target="_blank" title="Global synthesis reveals heterogeneous changes in connection of humans to nature">Global synthesis reveals heterogeneous changes in connection of humans to nature</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Soga et al.</li></ul></div><div class="linked-article__details"><span><i>One Earth</i></span><span id="item_date">February 17, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)10610-X" target="_blank" title="Inspection of wind turbine bolted connections using the ultrasonic phased array system">Inspection of wind turbine bolted connections using the ultrasonic phased array system</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Mills et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 13, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(20)31187-8" target="_blank" title="Low Prevalence of Lactase Persistence in Bronze Age Europe Indicates Ongoing Strong Selection over the Last 3,000 Years">Low Prevalence of Lactase Persistence in Bronze Age Europe Indicates Ongoing Strong Selection over the Last 3,000 Years</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Burger et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">September 3, 2020</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)06834-2" target="_blank" title="Reproducing bronze archaeological patinas through intentional burial: A comparison between short- and long-term interactions with soil">Reproducing bronze archaeological patinas through intentional burial: A comparison between short- and long-term interactions with soil</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Boccaccini et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 29, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(17)31328-3" target="_blank" title="The Stone Age Plague and Its Persistence in Eurasia">The Stone Age Plague and Its Persistence in Eurasia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Andrades Valtueña et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">November 22, 2017</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-systems/fulltext/S2405-4712(24)00303-X" target="_blank" title="Systematic analysis identifies a connection between spatial and genomic variations of chromatin states">Systematic analysis identifies a connection between spatial and genomic variations of chromatin states</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Cao et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Systems</i></span><span id="item_date">November 13, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(22)01443-8" target="_blank" title="Political connections, investment opportunity sets, tax avoidance: does corporate social responsibility disclosure in Indonesia have a role?">Political connections, investment opportunity sets, tax avoidance: does corporate social responsibility disclosure in Indonesia have a role?</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Firmansyah et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 9, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/joule/fulltext/S2542-4351(24)00503-8" target="_blank" title="Grid connection barriers to renewable energy deployment in the United States">Grid connection barriers to renewable energy deployment in the United States</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gorman et al.</li></ul></div><div class="linked-article__details"><span><i>Joule</i></span><span id="item_date">December 6, 2024</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2020.04.037","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.04.037/asset/f3c5ec7d-3001-4f0d-acd0-0cf5f8457879/main.assets/gr1.jpg" height="671" width="753" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/b5362c42-2e75-45e8-b53c-ed253fced4ef/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.04.037/asset/20983171-5174-46af-bf6a-d14a7b8a597a/main.assets/figs1.jpg" height="841" width="691" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/d82ae6f7-aa7e-4398-9de7-1778cf898dc4/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.04.037/asset/0ef045e9-30d4-4452-a712-bbca7dd50a6b/main.assets/gr2.jpg" height="825" width="656" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/63026b29-4ac1-4614-8af0-437e85f2e07b/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.04.037/asset/7bc69245-783e-4daa-a0b4-44ff5d18b6b6/main.assets/figs2.jpg" height="748" width="656" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/9defa4b1-fa0e-43e3-85a0-d60986db100e/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.04.037/asset/4075fab7-317c-445a-806d-eba8f732461d/main.assets/figs3.jpg" height="941" width="601" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/113a2a61-115f-46df-9edb-64ff42a5b19c/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.04.037/asset/4f50b2ae-0dc4-4b33-ab05-f2172d2c1b68/main.assets/gr3.jpg" height="362" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/46e719e1-c622-46d7-b052-e1f3c0c7b729/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.04.037/asset/3b13159e-698e-4288-9805-81698053401c/main.assets/figs4.jpg" height="862" width="753" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/7945273e-b457-4e20-8aa0-8c364af59b67/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.04.037/asset/340cc90d-1602-4577-9dfd-e596b7218561/main.assets/gr4.jpg" height="553" width="734" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/6df06745-7152-4de3-b331-0dac91a869c9/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.04.037/asset/a15d65a2-95dd-4b78-a9c2-1ef3bbc5fa64/main.assets/figs5.jpg" height="937" width="680" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/a0177473-6a74-421e-9bd6-59ee7fbed665/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.04.037/asset/84abb332-ca65-44e6-bb12-80c5fbab8e2f/main.assets/figs6.jpg" height="760" width="767" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/8d3483f1-7c24-436a-a30a-c85e89d36bd9/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.04.037/asset/11061e01-1bb2-4976-b68e-bd26a6841493/main.assets/gr5.jpg" height="323" width="301" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/6a09fb12-3b6d-4e2d-987d-120f61a54b08/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.04.037/asset/f3c5ec7d-3001-4f0d-acd0-0cf5f8457879/main.assets/gr1.jpg" height="671" width="753" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/b5362c42-2e75-45e8-b53c-ed253fced4ef/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.04.037/asset/20983171-5174-46af-bf6a-d14a7b8a597a/main.assets/figs1.jpg" height="841" width="691" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/d82ae6f7-aa7e-4398-9de7-1778cf898dc4/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.04.037/asset/0ef045e9-30d4-4452-a712-bbca7dd50a6b/main.assets/gr2.jpg" height="825" width="656" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/63026b29-4ac1-4614-8af0-437e85f2e07b/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.04.037/asset/7bc69245-783e-4daa-a0b4-44ff5d18b6b6/main.assets/figs2.jpg" height="748" width="656" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/9defa4b1-fa0e-43e3-85a0-d60986db100e/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.04.037/asset/4075fab7-317c-445a-806d-eba8f732461d/main.assets/figs3.jpg" height="941" width="601" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/113a2a61-115f-46df-9edb-64ff42a5b19c/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.04.037/asset/4f50b2ae-0dc4-4b33-ab05-f2172d2c1b68/main.assets/gr3.jpg" height="362" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/46e719e1-c622-46d7-b052-e1f3c0c7b729/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.04.037/asset/3b13159e-698e-4288-9805-81698053401c/main.assets/figs4.jpg" height="862" width="753" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/7945273e-b457-4e20-8aa0-8c364af59b67/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.04.037/asset/340cc90d-1602-4577-9dfd-e596b7218561/main.assets/gr4.jpg" height="553" width="734" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/6df06745-7152-4de3-b331-0dac91a869c9/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.04.037/asset/a15d65a2-95dd-4b78-a9c2-1ef3bbc5fa64/main.assets/figs5.jpg" height="937" width="680" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/a0177473-6a74-421e-9bd6-59ee7fbed665/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.04.037/asset/84abb332-ca65-44e6-bb12-80c5fbab8e2f/main.assets/figs6.jpg" height="760" width="767" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/8d3483f1-7c24-436a-a30a-c85e89d36bd9/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.04.037/asset/11061e01-1bb2-4976-b68e-bd26a6841493/main.assets/gr5.jpg" height="323" width="301" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.04.037/asset/6a09fb12-3b6d-4e2d-987d-120f61a54b08/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30502-X';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30502-X';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749586602127","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"181","issueNumber":"6","viewState":"fulltext","format":"MIME-HTML","id":"pii:S009286742030502X","coverDate":"202006","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94db8c3fbf79428c',t:'MTc0OTU4NjYwMS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94db8c3fbf79428c&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2020.04.037&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>