<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="efa2abaa-2e8f-4cda-b8b6-9b5adcfe2ab9" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0092867421012757;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;issue:issue:pii\:S0092867420X00262;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Xist nucleates local protein gradients to propagate silencing across the X chromosome">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)01275-7">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/b4a52ff0-6201-406a-81fd-ef90a19b5d67/fx1.jpg">
    


        <meta property="og:description" content="Instead of spreading across an X chromosome to inactivate it, Xist nucleates dynamic
protein complexes, allowing protein expansion across the X to induce higher-order
chromatin changes and gene silencing.">
    











        <meta name="twitter:title" content="Xist nucleates local protein gradients to propagate silencing across the X chromosome">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/b4a52ff0-6201-406a-81fd-ef90a19b5d67/fx1.jpg">
    


        <meta name="twitter:description" content="Instead of spreading across an X chromosome to inactivate it, Xist nucleates dynamic
protein complexes, allowing protein expansion across the X to induce higher-order
chromatin changes and gene silencing.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Xist nucleates local protein gradients to propagate silencing across the X chromosome: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Xist nucleates local protein gradients to propagate silencing across the X chromosome"> <meta name="citation_date" content="2021/12/09"> <meta name="citation_online_date" content="2021/11/04"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="25"> <meta name="citation_firstpage" content="6174"> <meta name="citation_lastpage" content="6192.e32"> <meta name="citation_pmid" content="34813726"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421012757"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421012757"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421012757"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.10.022"> <meta name="citation_pmid" content="34813726"> <meta name="citation_author" content="Yolanda Markaki"><meta name="citation_author_email" content="gmarkaki@mednet.ucla.edu"><meta name="citation_author_institution" content="Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author" content="Johnny Gan Chong"><meta name="citation_author_institution" content="Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author" content="Yuying Wang"><meta name="citation_author_institution" content="Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author" content="Elsie C. Jacobson"><meta name="citation_author_orcid" content="0000-0002-6094-5490"><meta name="citation_author_institution" content="Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author" content="Christy Luong"><meta name="citation_author_orcid" content="0000-0003-2841-9930"><meta name="citation_author_institution" content="Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author" content="Shawn Y.X. Tan"><meta name="citation_author_institution" content="Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author" content="Joanna W. Jachowicz"><meta name="citation_author_institution" content="Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_author" content="Mackenzie Strehle"><meta name="citation_author_institution" content="Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_author" content="Davide Maestrini"><meta name="citation_author_institution" content="Department of Computational Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author" content="Abhik K. Banerjee"><meta name="citation_author_orcid" content="0000-0002-9797-0104"><meta name="citation_author_institution" content="Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_author" content="Bhaven A. Mistry"><meta name="citation_author_institution" content="Department of Computational Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author_institution" content="Claremont McKenna College, Claremont, CA 91711, USA"><meta name="citation_author" content="Iris Dror"><meta name="citation_author_orcid" content="0000-0001-7144-2250"><meta name="citation_author_institution" content="Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author" content="Francois Dossin"><meta name="citation_author_orcid" content="0000-0002-4085-1446"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, Director’s Unit, Heidelberg 69117, Germany"><meta name="citation_author" content="Johannes Schöneberg"><meta name="citation_author_institution" content="Departments of Pharmacology &amp; Chemistry and Biochemistry, University of California San Diego, San Diego, CA 92093, USA"><meta name="citation_author" content="Edith Heard"><meta name="citation_author_institution" content="European Molecular Biology Laboratory, Director’s Unit, Heidelberg 69117, Germany"><meta name="citation_author" content="Mitchell Guttman"><meta name="citation_author_institution" content="Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_author" content="Tom Chou"><meta name="citation_author_email" content="tomchou@ucla.edu"><meta name="citation_author_orcid" content="0000-0003-0785-6349"><meta name="citation_author_institution" content="Department of Computational Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_author" content="Kathrin Plath"><meta name="citation_author_email" content="kplath@mednet.ucla.edu"><meta name="citation_author_orcid" content="0000-0001-7796-3372"><meta name="citation_author_institution" content="Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA"><meta name="citation_keywords" content="heterochromatin; X chromosome inactivation; Xist RNA; RNA-binding proteins; biomolecular condensates; supramolecular complexes; chromatin organization; macromolecular dynamics; super-resolution microscopy; quantitative imaging"><meta name="citation_reference" content="citation_title=PCGF3/5-PRC1 initiates Polycomb recruitment in X chromosome inactivation; citation_author=M. Almeida; citation_author=G. Pintacuda; citation_author=O. Masui; citation_author=Y. Koseki; citation_author=M. Gdula; citation_author=A. Cerase; citation_author=D. Brown; citation_author=A. Mould; citation_author=C. Innocent; citation_author=M. Nakayama; citation_author=et al.; citation_journal_title=Science; citation_volume=356; citation_firstpage=1081; citation_lastpage=1084; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Biomolecular condensates: organizers of cellular biochemistry; citation_author=S.F. Banani; citation_author=H.O. Lee; citation_author=A.A. Hyman; citation_author=M.K. Rosen; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=18; citation_firstpage=285; citation_lastpage=298; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Kinetics of Xist-induced gene silencing can be predicted from combinations of epigenetic and genomic features; citation_author=L. Barros de Andrade E. Sousa; citation_author=I. Jonkers; citation_author=L. Syx; citation_author=I. Dunkel; citation_author=J. Chaumeil; citation_author=C. Picard; citation_author=B. Foret; citation_author=C.J. Chen; citation_author=J.T. Lis; citation_author=E. Heard; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=29; citation_firstpage=1087; citation_lastpage=1099; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Efficient method to generate single-copy transgenic mice by site-specific integration in embryonic stem cells; citation_author=C. Beard; citation_author=K. Hochedlinger; citation_author=K. Plath; citation_author=A. Wutz; citation_author=R. Jaenisch; citation_journal_title=Genesis; citation_volume=44; citation_firstpage=23; citation_lastpage=28; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=From data to probability densities without histograms; citation_author=B.A. Berg; citation_author=R.C. Harris; citation_journal_title=Comput. Phys. Commun.; citation_volume=179; citation_firstpage=443; citation_lastpage=448; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Localization of ASH1 mRNA particles in living yeast; citation_author=E. Bertrand; citation_author=P. Chartrand; citation_author=M. Schaefer; citation_author=S.M. Shenoy; citation_author=R.H. Singer; citation_author=R.M. Long; citation_journal_title=Mol. Cell; citation_volume=2; citation_firstpage=437; citation_lastpage=445; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Nuclear cloning of embryonal carcinoma cells; citation_author=R.H. Blelloch; citation_author=K. Hochedlinger; citation_author=Y. Yamada; citation_author=C. Brennan; citation_author=M. Kim; citation_author=B. Mintz; citation_author=L. Chin; citation_author=R. Jaenisch; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=101; citation_firstpage=13985; citation_lastpage=13990; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=SmcHD1, containing a structural-maintenance-of-chromosomes hinge domain, has a critical role in X inactivation; citation_author=M.E. Blewitt; citation_author=A.V. Gendrel; citation_author=Z. Pang; citation_author=D.B. Sparrow; citation_author=N. Whitelaw; citation_author=J.M. Craig; citation_author=A. Apedaile; citation_author=D.J. Hilton; citation_author=S.L. Dunwoodie; citation_author=N. Brockdorff; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=40; citation_firstpage=663; citation_lastpage=669; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=The role of Xist-mediated Polycomb recruitment in the initiation of X-chromosome inactivation; citation_author=A. Bousard; citation_author=A.C. Raposo; citation_author=J.J. Żylicz; citation_author=C. Picard; citation_author=V.B. Pires; citation_author=Y. Qi; citation_author=C. Gil; citation_author=L. Syx; citation_author=H.Y. Chang; citation_author=E. Heard; citation_author=S.T. da Rocha; citation_journal_title=EMBO Rep.; citation_volume=20; citation_firstpage=e48019; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A central role for canonical PRC1 in shaping the 3D nuclear landscape; citation_author=S. Boyle; citation_author=I.M. Flyamer; citation_author=I. Williamson; citation_author=D. Sengupta; citation_author=W.A. Bickmore; citation_author=R.S. Illingworth; citation_journal_title=Genes Dev.; citation_volume=34; citation_firstpage=931; citation_lastpage=949; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Bridging-induced microphase separation: photobleaching experiments, chromatin domains and the need for active reactions; citation_author=C.A. Brackley; citation_author=D. Marenduzzo; citation_journal_title=Brief. Funct. Genomics; citation_volume=19; citation_firstpage=111; citation_lastpage=118; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A reaction-diffusion model to study RNA motion by quantitative fluorescence recovery after photobleaching; citation_author=J. Braga; citation_author=J.G. McNally; citation_author=M. Carmo-Fonseca; citation_journal_title=Biophys. J.; citation_volume=92; citation_firstpage=2694; citation_lastpage=2703; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Polycomb complexes in X chromosome inactivation; citation_author=N. Brockdorff; citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_volume=372; citation_firstpage=20170021; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Progress toward understanding chromosome silencing by Xist RNA; citation_author=N. Brockdorff; citation_author=J.S. Bowness; citation_author=G. Wei; citation_journal_title=Genes Dev.; citation_volume=34; citation_firstpage=733; citation_lastpage=744; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution; citation_author=M.N. Cabili; citation_author=M.C. Dunagin; citation_author=P.D. McClanahan; citation_author=A. Biaesch; citation_author=O. Padovan-Merhar; citation_author=A. Regev; citation_author=J.L. Rinn; citation_author=A. Raj; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=20; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Spatial separation of Xist RNA and polycomb proteins revealed by superresolution microscopy; citation_author=A. Cerase; citation_author=D. Smeets; citation_author=Y.A. Tang; citation_author=M. Gdula; citation_author=F. Kraus; citation_author=M. Spivakov; citation_author=B. Moindrot; citation_author=M. Leleu; citation_author=A. Tattermusch; citation_author=J. Demmerle; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=111; citation_firstpage=2235; citation_lastpage=2240; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Phase separation drives X-chromosome inactivation: a hypothesis; citation_author=A. Cerase; citation_author=A. Armaos; citation_author=C. Neumayer; citation_author=P. Avner; citation_author=M. Guttman; citation_author=G.G. Tartaglia; citation_journal_title=Nat. Struct. Mol. Biol.; citation_volume=26; citation_firstpage=331; citation_lastpage=334; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A path-integral approach to bayesian inference for inverse problems using the semiclassical approximation; citation_author=J.C. Chang; citation_author=V.M. Savage; citation_author=T. Chou; citation_journal_title=J.&nbsp;Stat. Phys.; citation_volume=109; citation_firstpage=966; citation_lastpage=974; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Bayesian Uncertainty Quantification for Bond Energies and Mobilities Using Path Integral Analysis; citation_author=J.C. Chang; citation_author=P.W. Fok; citation_author=T. Chou; citation_journal_title=Biophys. J.; citation_volume=109; citation_firstpage=966; citation_lastpage=974; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A novel role for Xist RNA in the formation of a repressive nuclear compartment into which genes are recruited when silenced; citation_author=J. Chaumeil; citation_author=P. Le Baccon; citation_author=A. Wutz; citation_author=E. Heard; citation_journal_title=Genes Dev.; citation_volume=20; citation_firstpage=2223; citation_lastpage=2237; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Dynamic imaging of genomic loci in living human cells by an optimized CRISPR/Cas system; citation_author=B. Chen; citation_author=L.A. Gilbert; citation_author=B.A. Cimini; citation_author=J. Schnitzbauer; citation_author=W. Zhang; citation_author=G.W. Li; citation_author=J. Park; citation_author=E.H. Blackburn; citation_author=J.S. Weissman; citation_author=L.S. Qi; citation_author=B. Huang; citation_journal_title=Cell; citation_volume=155; citation_firstpage=1479; citation_lastpage=1491; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Systematic discovery of Xist RNA binding proteins; citation_author=C. Chu; citation_author=Q.C. Zhang; citation_author=S.T. da Rocha; citation_author=R.A. Flynn; citation_author=M. Bharadwaj; citation_author=J.M. Calabrese; citation_author=T. Magnuson; citation_author=E. Heard; citation_author=H.Y. Chang; citation_journal_title=Cell; citation_volume=161; citation_firstpage=404; citation_lastpage=416; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=XIST RNA paints the inactive X chromosome at interphase: evidence for a novel RNA involved in nuclear/chromosome structure; citation_author=C.M. Clemson; citation_author=J.A. McNeil; citation_author=H.F. Willard; citation_author=J.B. Lawrence; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=132; citation_firstpage=259; citation_lastpage=275; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Xist Deletional Analysis Reveals an Interdependency between Xist RNA and Polycomb Complexes for Spreading along the Inactive X; citation_author=D. Colognori; citation_author=H. Sunwoo; citation_author=A.J. Kriz; citation_author=C.Y. Wang; citation_author=J.T. Lee; citation_journal_title=Mol. Cell; citation_volume=74; citation_firstpage=101; citation_lastpage=117.e10; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Multicolor 3D fluorescence in&nbsp;situ hybridization for imaging interphase chromosomes; citation_author=M. Cremer; citation_author=F. Grasser; citation_author=C. Lanctôt; citation_author=S. Müller; citation_author=M. Neusser; citation_author=R. Zinner; citation_author=I. Solovei; citation_author=T. Cremer; citation_journal_title=Methods Mol. Biol.; citation_volume=463; citation_firstpage=205; citation_lastpage=239; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Twelve years of SAMtools and BCFtools; citation_author=P. Danecek; citation_author=J.K. Bonfield; citation_author=J. Liddle; citation_author=J. Marshall; citation_author=V. Ohan; citation_author=M.O. Pollard; citation_author=A. Whitwham; citation_author=T. Keane; citation_author=S.A. McCarthy; citation_author=R.M. Davies; citation_author=H. Li; citation_journal_title=Gigascience; citation_volume=10; citation_firstpage=giab008; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Deletion of DXZ4 on the human inactive X chromosome alters higher-order genome architecture; citation_author=E.M. Darrow; citation_author=M.H. Huntley; citation_author=O. Dudchenko; citation_author=E.K. Stamenova; citation_author=N.C. Durand; citation_author=Z. Sun; citation_author=S.C. Huang; citation_author=A.L. Sanborn; citation_author=I. Machol; citation_author=M. Shamim; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=113; citation_firstpage=E4504; citation_lastpage=E4512; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Strategic and practical guidelines for successful structured illumination microscopy; citation_author=J. Demmerle; citation_author=C. Innocent; citation_author=A.J. North; citation_author=G. Ball; citation_author=M. Müller; citation_author=E. Miron; citation_author=A. Matsuda; citation_author=I.M. Dobbie; citation_author=Y. Markaki; citation_author=L. Schermelleh; citation_journal_title=Nat. Protoc.; citation_volume=12; citation_firstpage=988; citation_lastpage=1010; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression; citation_author=T. Derrien; citation_author=R. Johnson; citation_author=G. Bussotti; citation_author=A. Tanzer; citation_author=S. Djebali; citation_author=H. Tilgner; citation_author=G. Guernec; citation_author=D. Martin; citation_author=A. Merkel; citation_author=D.G. Knowles; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=1775; citation_lastpage=1789; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A. Dobin; citation_author=C.A. Davis; citation_author=F. Schlesinger; citation_author=J. Drenkow; citation_author=C. Zaleski; citation_author=S. Jha; citation_author=P. Batut; citation_author=M. Chaisson; citation_author=T.R. Gingeras; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=15; citation_lastpage=21; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=SPEN integrates transcriptional and epigenetic control of X-inactivation; citation_author=F. Dossin; citation_author=I. Pinheiro; citation_author=J.J. Żylicz; citation_author=J. Roensch; citation_author=S. Collombet; citation_author=A. Le Saux; citation_author=T. Chelmicki; citation_author=M. Attia; citation_author=V. Kapoor; citation_author=Y. Zhan; citation_author=et al.; citation_journal_title=Nature; citation_volume=578; citation_firstpage=455; citation_lastpage=460; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Measuring dynamics of nuclear proteins by photobleaching; citation_author=M. Dundr; citation_author=T. Misteli; citation_journal_title=Curr. Protoc. Cell Biol.; citation_doi=10.1002/0471143030.cb1305s18; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=mRNA on&nbsp;the move: the road to its biological destiny; citation_author=C. Eliscovich; citation_author=A.R. Buxbaum; citation_author=Z.B. Katz; citation_author=R.H. Singer; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=288; citation_firstpage=20361; citation_lastpage=20368; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome; citation_author=J.M. Engreitz; citation_author=A. Pandya-Jones; citation_author=P. McDonel; citation_author=A. Shishkin; citation_author=K. Sirokman; citation_author=C. Surka; citation_author=S. Kadri; citation_author=J. Xing; citation_author=A. Goren; citation_author=E.S. Lander; citation_author=et al.; citation_journal_title=Science; citation_volume=341; citation_firstpage=1237973; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Long non-coding RNAs: spatial amplifiers that control nuclear structure and gene expression; citation_author=J.M. Engreitz; citation_author=N. Ollikainen; citation_author=M. Guttman; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=17; citation_firstpage=756; citation_lastpage=770; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Comparative analysis of 2D and 3D distance measurements to study spatial genome organization; citation_author=E.H. Finn; citation_author=G. Pegoraro; citation_author=S. Shachar; citation_author=T. Misteli; citation_journal_title=Methods; citation_volume=123; citation_firstpage=47; citation_lastpage=55; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Chromatin compaction by a polycomb group protein complex; citation_author=N.J. Francis; citation_author=R.E. Kingston; citation_author=C.L. Woodcock; citation_journal_title=Science; citation_volume=306; citation_firstpage=1574; citation_lastpage=1577; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Repetitive RNAs as Regulators of Chromatin-Associated Subcompartment Formation by Phase Separation; citation_author=L. Frank; citation_author=K. Rippe; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=432; citation_firstpage=4270; citation_lastpage=4286; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Single mRNA molecules demonstrate probabilistic movement in living mammalian cells; citation_author=D. Fusco; citation_author=N. Accornero; citation_author=B. Lavoie; citation_author=S.M. Shenoy; citation_author=J.M. Blanchard; citation_author=R.H. Singer; citation_author=E. Bertrand; citation_journal_title=Curr. Biol.; citation_volume=13; citation_firstpage=161; citation_lastpage=167; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=X-Chromosome Inactivation: A Crossroads Between Chromosome Architecture and Gene Regulation; citation_author=R. Galupa; citation_author=E. Heard; citation_journal_title=Annu. Rev. Genet.; citation_volume=52; citation_firstpage=535; citation_lastpage=566; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Structural organization of the inactive X chromosome in the mouse; citation_author=L. Giorgetti; citation_author=B.R. Lajoie; citation_author=A.C. Carter; citation_author=M. Attia; citation_author=Y. Zhan; citation_author=J. Xu; citation_author=C.J. Chen; citation_author=N. Kaplan; citation_author=H.Y. Chang; citation_author=E. Heard; citation_author=J. Dekker; citation_journal_title=Nature; citation_volume=535; citation_firstpage=575; citation_lastpage=579; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Compaction of chromatin by diverse Polycomb group proteins requires localized regions of high charge; citation_author=D.J. Grau; citation_author=B.A. Chapman; citation_author=J.D. Garlick; citation_author=M. Borowsky; citation_author=N.J. Francis; citation_author=R.E. Kingston; citation_journal_title=Genes Dev.; citation_volume=25; citation_firstpage=2210; citation_lastpage=2221; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Efficient generation of targeted large insertions by microinjection into two-cell-stage mouse embryos; citation_author=B. Gu; citation_author=E. Posfai; citation_author=J. Rossant; citation_journal_title=Nat. Biotechnol.; citation_volume=36; citation_firstpage=632; citation_lastpage=637; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=EnrichedHeatmap: an R/Bioconductor package for comprehensive visualization of genomic signal associations; citation_author=Z. Gu; citation_author=R. Eils; citation_author=M. Schlesner; citation_author=N. Ishaque; citation_journal_title=BMC Genomics; citation_volume=19; citation_firstpage=234; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Array programming with NumPy; citation_author=C.R. Harris; citation_author=K.J. Millman; citation_author=S.J. van der Walt; citation_author=R. Gommers; citation_author=P. Virtanen; citation_author=D. Cournapeau; citation_author=E. Wieser; citation_author=J. Taylor; citation_author=S. Berg; citation_author=N.J. Smith; citation_author=et al.; citation_journal_title=Nature; citation_volume=585; citation_firstpage=357; citation_lastpage=362; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Generation of eggs from mouse embryonic stem cells and induced pluripotent stem cells; citation_author=K. Hayashi; citation_author=M. Saitou; citation_journal_title=Nat. Protoc.; citation_volume=8; citation_firstpage=1513; citation_lastpage=1524; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Identification and characterization of the human XIST gene promoter: implications for models of X&nbsp;chromosome inactivation; citation_author=B.D. Hendrich; citation_author=R.M. Plenge; citation_author=H.F. Willard; citation_journal_title=Nucleic Acids Res.; citation_volume=25; citation_firstpage=2661; citation_lastpage=2671; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Design of a hyperstable 60-subunit protein dodecahedron. [corrected]; citation_author=Y. Hsia; citation_author=J.B. Bale; citation_author=S. Gonen; citation_author=D. Shi; citation_author=W. Sheffler; citation_author=K.K. Fong; citation_author=U. Nattermann; citation_author=C. Xu; citation_author=P.S. Huang; citation_author=R. Ravichandran; citation_author=et al.; citation_journal_title=Nature; citation_volume=535; citation_firstpage=136; citation_lastpage=139; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Matplotlib: A 2D Graphics Environment; citation_author=J.D. Hunter; citation_journal_title=Comput. Sci. Eng.; citation_volume=9; citation_firstpage=90; citation_lastpage=95; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Chromatin folding and nuclear architecture: PRC1 function in 3D; citation_author=R.S. Illingworth; citation_journal_title=Curr. Opin. Genet. Dev.; citation_volume=55; citation_firstpage=82; citation_lastpage=90; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Xist spatially amplifies SHARP recruitment to balance chromosome-wide silencing and specificity to the X chromosome; citation_author=J.W. Jachowicz; citation_author=M. Strehle; citation_author=A.K. Banerjee; citation_author=J. Thai; citation_author=M.R. Blanco; citation_author=M. Guttman; citation_journal_title=bioRxiv; citation_doi=10.1101/2021.10.27.466149; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Smchd1 Targeting to the Inactive X Is Dependent on the Xist-HnrnpK-PRC1 Pathway; citation_author=N. Jansz; citation_author=T. Nesterova; citation_author=A. Keniry; citation_author=M. Iminitoff; citation_author=P.F. Hickey; citation_author=G. Pintacuda; citation_author=O. Masui; citation_author=S. Kobelke; citation_author=N. Geoghegan; citation_author=K.A. Breslin; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=25; citation_firstpage=1912; citation_lastpage=1923.e9; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The X chromosome in space; citation_author=T. Jégu; citation_author=E. Aeby; citation_author=J.T. Lee; citation_journal_title=Nat. Rev. Genet.; citation_volume=18; citation_firstpage=377; citation_lastpage=389; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Xist RNA is confined to the nuclear territory of the silenced X chromosome throughout the cell cycle; citation_author=I. Jonkers; citation_author=K. Monkhorst; citation_author=E. Rentmeester; citation_author=J.A. Grootegoed; citation_author=F. Grosveld; citation_author=J. Gribnau; citation_journal_title=Mol. Cell. Biol.; citation_volume=28; citation_firstpage=5583; citation_lastpage=5594; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=A quantitative approach to analyze binding diffusion kinetics by confocal FRAP; citation_author=M. Kang; citation_author=C.A. Day; citation_author=E. DiBenedetto; citation_author=A.K. Kenworthy; citation_journal_title=Biophys. J.; citation_volume=99; citation_firstpage=2737; citation_lastpage=2747; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=An improved Flp deleter mouse in C57Bl/6 based on Flpo recombinase; citation_author=A. Kranz; citation_author=J. Fu; citation_author=K. Duerschke; citation_author=S. Weidlich; citation_author=R. Naumann; citation_author=A.F. Stewart; citation_author=K. Anastassiadis; citation_journal_title=Genesis; citation_volume=48; citation_firstpage=512; citation_lastpage=520; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Quantitative 3D structured illumination microscopy of nuclear structures; citation_author=F. Kraus; citation_author=E. Miron; citation_author=J. Demmerle; citation_author=T. Chitiashvili; citation_author=A. Budco; citation_author=Q. Alle; citation_author=A. Matsuda; citation_author=H. Leonhardt; citation_author=L. Schermelleh; citation_author=Y. Markaki; citation_journal_title=Nat. Protoc.; citation_volume=12; citation_firstpage=1011; citation_lastpage=1028; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=What macromolecular crowding can do to a protein; citation_author=I.M. Kuznetsova; citation_author=K.K. Turoverov; citation_author=V.N. Uversky; citation_journal_title=Int. J. Mol. Sci.; citation_volume=15; citation_firstpage=23090; citation_lastpage=23140; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; citation_author=B. Langmead; citation_author=C. Trapnell; citation_author=M. Pop; citation_author=S.L. Salzberg; citation_journal_title=Genome Biol.; citation_volume=10; citation_firstpage=R25; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=plyranges: a grammar of genomic data transformation; citation_author=S. Lee; citation_author=D. Cook; citation_author=M. Lawrence; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=4; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Xist RNA in action: Past, present, and future; citation_author=A. Loda; citation_author=E. Heard; citation_journal_title=PLoS Genet.; citation_volume=15; citation_firstpage=e1008333; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The potential of 3D-FISH and super-resolution structured illumination microscopy for studies of 3D nuclear architecture: 3D structured illumination microscopy of defined chromosomal structures visualized by 3D (immuno)-FISH opens new perspectives for studies of nuclear architecture; citation_author=Y. Markaki; citation_author=D. Smeets; citation_author=S. Fiedler; citation_author=V.J. Schmid; citation_author=L. Schermelleh; citation_author=T. Cremer; citation_author=M. Cremer; citation_journal_title=BioEssays; citation_volume=34; citation_firstpage=412; citation_lastpage=426; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Fluorescence in&nbsp;situ hybridization applications for super-resolution 3D structured illumination microscopy; citation_author=Y. Markaki; citation_author=D. Smeets; citation_author=M. Cremer; citation_author=L. Schermelleh; citation_journal_title=Methods Mol. Biol.; citation_volume=950; citation_firstpage=43; citation_lastpage=64; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The Xist lncRNA interacts directly with SHARP to silence transcription through HDAC3; citation_author=C.A. McHugh; citation_author=C.K. Chen; citation_author=A. Chow; citation_author=C.F. Surka; citation_author=C. Tran; citation_author=P. McDonel; citation_author=A. Pandya-Jones; citation_author=M. Blanco; citation_author=C. Burghard; citation_author=A. Moradian; citation_author=et al.; citation_journal_title=Nature; citation_volume=521; citation_firstpage=232; citation_lastpage=236; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Quantitative FRAP in analysis of molecular binding dynamics in&nbsp;vivo; citation_author=J.G. McNally; citation_journal_title=Methods Cell Biol.; citation_volume=85; citation_firstpage=329; citation_lastpage=351; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Evaluating phase separation in live cells: diagnosis, caveats, and functional consequences; citation_author=D.T. McSwiggen; citation_author=M. Mir; citation_author=X. Darzacq; citation_author=R. Tjian; citation_journal_title=Genes Dev.; citation_volume=33; citation_firstpage=1619; citation_lastpage=1634; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Chromosomes. A comprehensive Xist interactome reveals cohesin repulsion and an RNA-directed chromosome conformation; citation_author=A. Minajigi; citation_author=J. Froberg; citation_author=C. Wei; citation_author=H. Sunwoo; citation_author=B. Kesner; citation_author=D. Colognori; citation_author=D. Lessing; citation_author=B. Payer; citation_author=M. Boukhali; citation_author=W. Haas; citation_author=J.T. Lee; citation_journal_title=Science; citation_doi=10.1126/science.aab2276; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The Mbd1-Atf7ip-Setdb1 pathway contributes to the maintenance of X chromosome inactivation; citation_author=A. Minkovsky; citation_author=A. Sahakyan; citation_author=E. Rankin-Gee; citation_author=G. Bonora; citation_author=S. Patel; citation_author=K. Plath; citation_journal_title=Epigenetics Chromatin; citation_volume=7; citation_firstpage=12; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Atomic-level characterization of disordered protein ensembles; citation_author=T. Mittag; citation_author=J.D. Forman-Kay; citation_journal_title=Curr. Opin. Struct. Biol.; citation_volume=17; citation_firstpage=3; citation_lastpage=14; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Rapid immunoprecipitation mass spectrometry of endogenous proteins (RIME) for analysis of chromatin complexes; citation_author=H. Mohammed; citation_author=C. Taylor; citation_author=G.D. Brown; citation_author=E.K. Papachristou; citation_author=J.S. Carroll; citation_author=C.S. D’Santos; citation_journal_title=Nat. Protoc.; citation_volume=11; citation_firstpage=316; citation_lastpage=326; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A Pooled shRNA Screen Identifies Rbm15, Spen, and Wtap as Factors Required for Xist RNA-Mediated Silencing; citation_author=B. Moindrot; citation_author=A. Cerase; citation_author=H. Coker; citation_author=O. Masui; citation_author=A. Grijzenhout; citation_author=G. Pintacuda; citation_author=L. Schermelleh; citation_author=T.B. Nesterova; citation_author=N. Brockdorff; citation_journal_title=Cell Rep.; citation_volume=12; citation_firstpage=562; citation_lastpage=572; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Identification of Spen as a Crucial Factor for Xist Function through Forward Genetic Screening in Haploid Embryonic Stem Cells; citation_author=A. Monfort; citation_author=G. Di Minin; citation_author=A. Postlmayr; citation_author=R. Freimann; citation_author=F. Arieti; citation_author=S. Thore; citation_author=A. Wutz; citation_journal_title=Cell Rep.; citation_volume=12; citation_firstpage=554; citation_lastpage=561; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Dynamics of single mRNP nucleocytoplasmic transport and export through the nuclear pore in living cells; citation_author=A. Mor; citation_author=S. Suliman; citation_author=R. Ben-Yishay; citation_author=S. Yunger; citation_author=Y. Brody; citation_author=Y. Shav-Tal; citation_journal_title=Nat. Cell Biol.; citation_volume=12; citation_firstpage=543; citation_lastpage=552; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=High levels of Id1 expression define B1 type adult neural stem cells; citation_author=H.S. Nam; citation_author=R. Benezra; citation_journal_title=Cell Stem Cell; citation_volume=5; citation_firstpage=515; citation_lastpage=526; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Systematic allelic analysis defines the interplay of key pathways in X chromosome inactivation; citation_author=T.B. Nesterova; citation_author=G. Wei; citation_author=H. Coker; citation_author=G. Pintacuda; citation_author=J.S. Bowness; citation_author=T. Zhang; citation_author=M. Almeida; citation_author=B. Bloechl; citation_author=B. Moindrot; citation_author=E.J. Carter; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=3129; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A system for imaging the regulatory noncoding Xist RNA in living mouse embryonic stem cells; citation_author=K. Ng; citation_author=N. Daigle; citation_author=A. Bancaud; citation_author=T. Ohhata; citation_author=P. Humphreys; citation_author=R. Walker; citation_author=J. Ellenberg; citation_author=A. Wutz; citation_journal_title=Mol. Biol. Cell; citation_volume=22; citation_firstpage=2634; citation_lastpage=2645; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Dynamic Organization of Chromatin Domains Revealed by Super-Resolution Live-Cell Imaging; citation_author=T. Nozaki; citation_author=R. Imai; citation_author=M. Tanbo; citation_author=R. Nagashima; citation_author=S. Tamura; citation_author=T. Tani; citation_author=Y. Joti; citation_author=M. Tomita; citation_author=K. Hibino; citation_author=M.T. Kanemaki; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=67; citation_firstpage=282; citation_lastpage=293.e7; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=TANGO: a generic tool for high-throughput 3D image analysis for studying nuclear organization; citation_author=J. Ollion; citation_author=J. Cochennec; citation_author=F. Loll; citation_author=C. Escudé; citation_author=T. Boudier; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=1840; citation_lastpage=1841; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Integrated analysis of Xist upregulation and X-chromosome inactivation with single-cell and single-allele resolution; citation_author=G. Pacini; citation_author=I. Dunkel; citation_author=N. Mages; citation_author=V. Mutzel; citation_author=B. Timmermann; citation_author=A. Marsico; citation_author=E.G. Schulz; citation_journal_title=Nat. Commun.; citation_volume=12; citation_firstpage=3638; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A protein assembly mediates Xist localization and gene silencing; citation_author=A. Pandya-Jones; citation_author=Y. Markaki; citation_author=J. Serizay; citation_author=T. Chitiashvili; citation_author=W.R. Mancia Leon; citation_author=A. Damianov; citation_author=C. Chronis; citation_author=B. Papp; citation_author=C.K. Chen; citation_author=R. McKee; citation_author=et al.; citation_journal_title=Nature; citation_volume=587; citation_firstpage=145; citation_lastpage=151; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=X chromosome inactivation is mediated by Xist RNA stabilization; citation_author=B. Panning; citation_author=J. Dausman; citation_author=R. Jaenisch; citation_journal_title=Cell; citation_volume=90; citation_firstpage=907; citation_lastpage=916; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=hnRNPK Recruits PCGF3/5-PRC1 to the Xist RNA B-Repeat to Establish Polycomb-Mediated Chromosomal Silencing; citation_author=G. Pintacuda; citation_author=G. Wei; citation_author=C. Roustan; citation_author=B.A. Kirmizitas; citation_author=N. Solcan; citation_author=A. Cerase; citation_author=A. Castello; citation_author=S. Mohammed; citation_author=B. Moindrot; citation_author=T.B. Nesterova; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=68; citation_firstpage=955; citation_lastpage=969.e10; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Xist RNA and the mechanism of X chromosome inactivation; citation_author=K. Plath; citation_author=S. Mlynarczyk-Evans; citation_author=D.A. Nusinow; citation_author=B. Panning; citation_journal_title=Annu. Rev. Genet.; citation_volume=36; citation_firstpage=233; citation_lastpage=278; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Role of histone H3 lysine 27 methylation in X inactivation; citation_author=K. Plath; citation_author=J. Fang; citation_author=S.K. Mlynarczyk-Evans; citation_author=R. Cao; citation_author=K.A. Worringer; citation_author=H. Wang; citation_author=C.C. de la Cruz; citation_author=A.P. Otte; citation_author=B. Panning; citation_author=Y. Zhang; citation_journal_title=Science; citation_volume=300; citation_firstpage=131; citation_lastpage=135; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features; citation_author=A.R. Quinlan; citation_author=I.M. Hall; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=841; citation_lastpage=842; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=RNA promotes the formation of spatial compartments in the nucleus; citation_author=S.A. Quinodoz; citation_author=P. Bhat; citation_author=N. Ollikainen; citation_author=J.W. Jachowicz; citation_author=A.K. Banerjee; citation_author=P. Chovanec; citation_author=M.R. Blanco; citation_author=A. Chow; citation_author=Y. Markaki; citation_author=K. Plath; citation_author=M. Guttman; citation_journal_title=bioRxiv; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing; citation_author=R Core Team; citation_publication_date=2021; citation_publisher=R Foundation for Statistical Computing, Vienna, Austria; "><meta name="citation_reference" content="citation_title=deepTools2: a next generation web server for deep-sequencing data analysis; citation_author=F. Ramírez; citation_author=D.P. Ryan; citation_author=B. Grüning; citation_author=V. Bhardwaj; citation_author=F. Kilpert; citation_author=A.S. Richter; citation_author=S. Heyne; citation_author=F. Dündar; citation_author=T. Manke; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_number=W1; citation_firstpage=W160; citation_lastpage=W165; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The nuclear matrix protein CIZ1 facilitates localization of Xist RNA to the inactive X-chromosome territory; citation_author=R. Ridings-Figueroa; citation_author=E.R. Stewart; citation_author=T.B. Nesterova; citation_author=H. Coker; citation_author=G. Pintacuda; citation_author=J. Godwin; citation_author=R. Wilson; citation_author=A. Haslam; citation_author=F. Lilley; citation_author=R. Ruigrok; citation_author=et al.; citation_journal_title=Genes Dev.; citation_volume=31; citation_firstpage=876; citation_lastpage=888; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Genome regulation by long noncoding RNAs; citation_author=J.L. Rinn; citation_author=H.Y. Chang; citation_journal_title=Annu. Rev. Biochem.; citation_volume=81; citation_firstpage=145; citation_lastpage=166; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=RStudio: Integrated Development for R. RStudio; citation_author=RStudio Team; citation_publication_date=2020; citation_publisher=PBC, Boston, MA; "><meta name="citation_reference" content="citation_title=ImageJ2: ImageJ for the next generation of scientific image data; citation_author=C.T. Rueden; citation_author=J. Schindelin; citation_author=M.C. Hiner; citation_author=B.E. DeZonia; citation_author=A.E. Walter; citation_author=E.T. Arena; citation_author=K.W. Eliceiri; citation_journal_title=BMC Bioinformatics; citation_volume=18; citation_firstpage=529; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=lncRNA-Induced Spread of Polycomb Controlled by Genome Architecture, RNA Abundance, and CpG Island DNA; citation_author=M.D. Schertzer; citation_author=K.C.A. Braceros; citation_author=J. Starmer; citation_author=R.E. Cherney; citation_author=D.M. Lee; citation_author=G. Salazar; citation_author=M. Justice; citation_author=S.R. Bischoff; citation_author=D.O. Cowley; citation_author=P. Ariel; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=75; citation_firstpage=523; citation_lastpage=537.e10; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Fiji: an open-source platform for biological-image analysis; citation_author=J. Schindelin; citation_author=I. Arganda-Carreras; citation_author=E. Frise; citation_author=V. Kaynig; citation_author=M. Longair; citation_author=T. Pietzsch; citation_author=S. Preibisch; citation_author=C. Rueden; citation_author=S. Saalfeld; citation_author=B. Schmid; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=676; citation_lastpage=682; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Establishment of histone h3 methylation on the inactive X chromosome requires transient recruitment of Eed-Enx1 polycomb group complexes; citation_author=J. Silva; citation_author=W. Mak; citation_author=I. Zvetkova; citation_author=R. Appanah; citation_author=T.B. Nesterova; citation_author=Z. Webster; citation_author=A.H. Peters; citation_author=T. Jenuwein; citation_author=A.P. Otte; citation_author=N. Brockdorff; citation_journal_title=Dev. Cell; citation_volume=4; citation_firstpage=481; citation_lastpage=495; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=High-resolution Xist binding maps reveal two-step spreading during X-chromosome inactivation; citation_author=M.D. Simon; citation_author=S.F. Pinter; citation_author=R. Fang; citation_author=K. Sarma; citation_author=M. Rutenberg-Schoenberg; citation_author=S.K. Bowman; citation_author=B.A. Kesner; citation_author=V.K. Maier; citation_author=R.E. Kingston; citation_author=J.T. Lee; citation_journal_title=Nature; citation_volume=504; citation_firstpage=465; citation_lastpage=469; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=MXS-Chaining: A Highly Efficient Cloning Platform for Imaging and Flow Cytometry Approaches in Mammalian Systems; citation_author=H.L. Sladitschek; citation_author=P.A. Neveu; citation_journal_title=PLoS ONE; citation_volume=10; citation_firstpage=e0124958; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Three-dimensional super-resolution microscopy of the inactive X chromosome territory reveals a collapse of its active nuclear compartment harboring distinct Xist RNA foci; citation_author=D. Smeets; citation_author=Y. Markaki; citation_author=V.J. Schmid; citation_author=F. Kraus; citation_author=A. Tattermusch; citation_author=A. Cerase; citation_author=M. Sterr; citation_author=S. Fiedler; citation_author=J. Demmerle; citation_author=J. Popken; citation_author=et al.; citation_journal_title=Epigenetics Chromatin; citation_volume=7; citation_firstpage=8; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Gene regulation by long non-coding RNAs and its biological functions; citation_author=L. Statello; citation_author=C.J. Guo; citation_author=L.L. Chen; citation_author=M. Huarte; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=22; citation_firstpage=96; citation_lastpage=118; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=The Xist RNA-PRC2 complex at 20-nm resolution reveals a low Xist stoichiometry and suggests a hit-and-run mechanism in mouse cells; citation_author=H. Sunwoo; citation_author=J.Y. Wu; citation_author=J.T. Lee; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=E4216; citation_lastpage=E4225; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Repeat E anchors Xist RNA to the inactive X chromosomal compartment through&nbsp;CDKN1A-interacting protein (CIZ1); citation_author=H. Sunwoo; citation_author=D. Colognori; citation_author=J.E. Froberg; citation_author=Y. Jeon; citation_author=J.T. Lee; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=114; citation_firstpage=10654; citation_lastpage=10659; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=RYBP-PRC1 complexes mediate H2A ubiquitylation at polycomb target sites independently of PRC2 and H3K27me3; citation_author=L. Tavares; citation_author=E. Dimitrova; citation_author=D. Oxley; citation_author=J. Webster; citation_author=R. Poot; citation_author=J. Demmers; citation_author=K. Bezstarosti; citation_author=S. Taylor; citation_author=H. Ura; citation_author=H. Koide; citation_author=et al.; citation_journal_title=Cell; citation_volume=148; citation_firstpage=664; citation_lastpage=678; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=A top-down analysis of Xa- and Xi-territories reveals differences of higher order structure at 3 20 Mb genomic length scales; citation_author=K. Teller; citation_author=D. Illner; citation_author=S. Thamm; citation_author=C.S. Casas-Delucchi; citation_author=R. Versteeg; citation_author=M. Indemans; citation_author=T. Cremer; citation_author=M. Cremer; citation_journal_title=Nucleus; citation_volume=2; citation_firstpage=465; citation_lastpage=477; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=pandas-dev/pandas: Pandas; citation_author=J. Reback; citation_author=W. McKinney; citation_author=B. Mendel; citation_author=J. Van den Bossche; citation_author=T. Augspurger; citation_author=P. Cloud; citation_author=gfyoung; citation_author=S. Hawkins; citation_author=Sinhrks; citation_author=M. Roeschke; citation_author=et al.; citation_journal_title=Zenodo; citation_doi=10.5281/zenodo.3509134; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=TrackMate: An open and extensible platform for single-particle tracking; citation_author=J.Y. Tinevez; citation_author=N. Perry; citation_author=J. Schindelin; citation_author=G.M. Hoopes; citation_author=G.D. Reynolds; citation_author=E. Laplantine; citation_author=S.Y. Bednarek; citation_author=S.L. Shorte; citation_author=K.W. Eliceiri; citation_journal_title=Methods; citation_volume=115; citation_firstpage=80; citation_lastpage=90; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The multifaceted roles of intrinsic disorder in protein complexes; citation_author=V.N. Uversky; citation_journal_title=FEBS Lett.; citation_volume=589; citation_number=19 Pt A; citation_firstpage=2498; citation_lastpage=2506; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Genomics in the Cloud: Using Docker, GATK, and WDL in Terra; citation_author=G.A. Van der Auwera; citation_author=B.D. O’Connor; citation_publication_date=2020; citation_publisher=O’Reilly Media; "><meta name="citation_reference" content="citation_title=Python 3 Reference Manual; citation_author=G. van Rossum; citation_author=F.L. Drake; citation_publication_date=2009; citation_publisher=Scotts Valley, CA; "><meta name="citation_reference" content="citation_title=Constructing smooth potentials of mean force, radial distribution functions, and probability densities from sampled data; citation_author=R. van Zon; citation_author=J. Schofield; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=132; citation_firstpage=154110; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=SciPy 1.0: fundamental algorithms for scientific computing in Python; citation_author=P. Virtanen; citation_author=R. Gommers; citation_author=T.E. Oliphant; citation_author=M. Haberland; citation_author=T. Reddy; citation_author=D. Cournapeau; citation_author=E. Burovski; citation_author=P. Peterson; citation_author=W. Weckesser; citation_author=J. Bright; citation_author=SciPy 1.0 Contributors; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=17; citation_firstpage=261; citation_lastpage=272; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=SMCHD1 Merges Chromosome Compartments and Assists Formation of Super-Structures on the Inactive X; citation_author=C.Y. Wang; citation_author=T. Jegu; citation_author=H.P. Chu; citation_author=H.J. Oh; citation_author=J.T. Lee; citation_journal_title=Cell; citation_volume=174; citation_firstpage=406; citation_lastpage=421.e25; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=PRC1 collaborates with SMCHD1 to fold the X-chromosome and spread Xist RNA between chromosome compartments; citation_author=C.Y. Wang; citation_author=D. Colognori; citation_author=H. Sunwoo; citation_author=D. Wang; citation_author=J.T. Lee; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=2950; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=seaborn: statistical data visualization; citation_author=M.L. Waskom; citation_journal_title=Journal of Open Source Software; citation_volume=6; citation_firstpage=3021; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Welcome to the tidyverse; citation_author=H. Wickham; citation_author=M. Averick; citation_author=J. Bryan; citation_author=W. Chang; citation_author=L. D’Agostino McGowan; citation_author=R. François; citation_author=G. Grolemund; citation_author=A. Hayes; citation_author=L. Henry; citation_author=J. Hester; citation_author=et al.; citation_journal_title=Journal of Open Source Software; citation_volume=4; citation_firstpage=1686; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Fluorescence fluctuation spectroscopy enables quantitative imaging of single mRNAs in living cells; citation_author=B. Wu; citation_author=J.A. Chao; citation_author=R.H. Singer; citation_journal_title=Biophys. J.; citation_volume=102; citation_firstpage=2936; citation_lastpage=2944; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Gene silencing in X-chromosome inactivation: advances in understanding facultative heterochromatin formation; citation_author=A. Wutz; citation_journal_title=Nat. Rev. Genet.; citation_volume=12; citation_firstpage=542; citation_lastpage=553; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=A shift from reversible to irreversible X inactivation is triggered during ES cell differentiation; citation_author=A. Wutz; citation_author=R. Jaenisch; citation_journal_title=Mol. Cell; citation_volume=5; citation_firstpage=695; citation_lastpage=705; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Single-cell imaging of genome organization and dynamics; citation_author=L. Xie; citation_author=Z. Liu; citation_journal_title=Mol. Syst. Biol.; citation_volume=17; citation_firstpage=e9653; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Defined conditions for neural commitment and differentiation; citation_author=Q.L. Ying; citation_author=A.G. Smith; citation_journal_title=Methods Enzymol.; citation_volume=365; citation_firstpage=327; citation_lastpage=341; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Xist RNA repeat E is essential for ASH2L recruitment to the inactive X&nbsp;and regulates histone modifications and escape gene expression; citation_author=M. Yue; citation_author=A. Ogawa; citation_author=N. Yamada; citation_author=J.L. Charles Richard; citation_author=A. Barski; citation_author=Y. Ogawa; citation_journal_title=PLoS Genet.; citation_volume=13; citation_firstpage=e1006890; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Model-based analysis of ChIP-Seq (MACS); citation_author=Y. Zhang; citation_author=T. Liu; citation_author=C.A. Meyer; citation_author=J. Eeckhoute; citation_author=D.S. Johnson; citation_author=B.E. Bernstein; citation_author=C. Nusbaum; citation_author=R.M. Myers; citation_author=M. Brown; citation_author=W. Li; citation_author=X.S. Liu; citation_journal_title=Genome Biol.; citation_volume=9; citation_firstpage=R137; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=ChIPpeakAnno: a Bioconductor package to annotate ChIP-seq and ChIP-chip data; citation_author=L.J. Zhu; citation_author=C. Gazin; citation_author=N.D. Lawson; citation_author=H. Pagès; citation_author=S.M. Lin; citation_author=D.S. Lapointe; citation_author=M.R. Green; citation_journal_title=BMC Bioinformatics; citation_volume=11; citation_firstpage=237; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Macromolecular crowding allows blunt-end ligation by DNA ligases from rat liver or Escherichia coli; citation_author=S.B. Zimmerman; citation_author=B.H. Pheiffer; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=80; citation_firstpage=5852; citation_lastpage=5856; citation_publication_date=1983; "><meta name="citation_reference" content="citation_title=The Implication of Early Chromatin Changes in X Chromosome Inactivation; citation_author=J.J. Zylicz; citation_author=A. Bousard; citation_author=K. Zumer; citation_author=F. Dossin; citation_author=E. Mohammad; citation_author=S.T. da Rocha; citation_author=B. Schwalb; citation_author=L. Syx; citation_author=F. Dingli; citation_author=D. Loew; citation_author=et al.; citation_journal_title=Cell; citation_volume=176; citation_firstpage=182; citation_lastpage=197.e23; citation_publication_date=2019; "><meta name="citation_abstract" content="<h2>Summary</h2><p>The lncRNA <i>Xist</i> forms ∼50 diffraction-limited foci to transcriptionally silence one X chromosome. How this small number of RNA foci and interacting proteins regulate a much larger number of X-linked genes is unknown. We show that <i>Xist</i> foci are locally confined, contain ∼2 RNA molecules, and nucleate supramolecular complexes (SMACs) that include many copies of the critical silencing protein SPEN. Aggregation and exchange of SMAC proteins generate local protein gradients that regulate broad, proximal chromatin regions. Partitioning of numerous SPEN molecules into SMACs is mediated by their intrinsically disordered regions and essential for transcriptional repression. Polycomb deposition via SMACs induces chromatin compaction and the increase in SMACs density around genes, which propagates silencing across the X chromosome. Our findings introduce a mechanism for functional nuclear compartmentalization whereby crowding of transcriptional and architectural regulators enables the silencing of many target genes by few RNA molecules.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=9909&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7&amp;be=3998&amp;fe=8547&amp;dc=7582&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749592129878,%22n%22:0,%22f%22:3368,%22dn%22:3369,%22dne%22:3369,%22c%22:3369,%22s%22:3369,%22ce%22:3369,%22rq%22:3369,%22rp%22:3982,%22rpe%22:4854,%22dl%22:3987,%22di%22:5299,%22ds%22:7583,%22de%22:7812,%22dc%22:8547,%22l%22:8548,%22le%22:8665%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><style>stream {
  position: relative;
  display: block;
}
</style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c1D6F7 {
  padding: 0.683em 0.667em 0 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c221E.MM-I {
  padding: 0.456em 0.942em 0.025em 0;
}

mjx-c.mjx-c2E {
  padding: 0.106em 0.278em 0 0;
}

mjx-c.mjx-c1D43A {
  padding: 0.705em 0.786em 0.022em 0;
}

mjx-c.mjx-c1D465 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c2C {
  padding: 0.106em 0.278em 0.193em 0;
}

mjx-c.mjx-c1D466 {
  padding: 0.442em 0.49em 0.205em 0;
}

mjx-c.mjx-c1D467 {
  padding: 0.442em 0.467em 0.011em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c28.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c33 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c2026 {
  padding: 0.106em 0.837em 0 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c29.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901275-7" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)01275-7"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901275-7" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-984304" aria-expanded="false" aria-controls="login-links-984304" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-984304" id="login-links--984304" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901275-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901275-7" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901275-7" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-984304" data-opens="#links-dialog-984304" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-984304" aria-modal="true"><button data-closes="#links-dialog-984304" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901275-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901275-7" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901275-7" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.10.022">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)01275-7" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0026-2"><span>Volume 184</span>, <span>Issue 25</span></a></span><span class="meta-panel__pages"><span>p6174-6192.e32</span></span><span class="meta-panel__onlineDate">December 09, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=25&amp;pii=S0092-8674%2820%29X0026-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name"><i>Xist</i> nucleates local protein gradients to propagate silencing across the X chromosome</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Yolanda</span> <span property="familyName">Markaki</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yolanda</span> <span property="familyName">Markaki</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:gmarkaki@mednet.ucla.edu" property="email" title="Send email to Yolanda Markaki">gmarkaki@mednet.ucla.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Markaki/Yolanda" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup> <a id="eadd0010" href="mailto:gmarkaki@mednet.ucla.edu" property="email" title="Send email to Yolanda Markaki">gmarkaki@mednet.ucla.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Johnny</span> <span property="familyName">Gan Chong</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Johnny</span> <span property="familyName">Gan Chong</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gan+Chong/Johnny" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Yuying</span> <span property="familyName">Wang</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yuying</span> <span property="familyName">Wang</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Yuying" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">7</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Elsie C.</span> <span property="familyName">Jacobson</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Elsie C.</span> <span property="familyName">Jacobson</span> <a class="orcid-id" href="https://orcid.org/0000-0002-6094-5490" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-6094-5490</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jacobson/Elsie+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Christy</span> <span property="familyName">Luong</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Christy</span> <span property="familyName">Luong</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2841-9930" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2841-9930</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Luong/Christy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Shawn Y.X.</span> <span property="familyName">Tan</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shawn Y.X.</span> <span property="familyName">Tan</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Tan/Shawn+Y+X" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Joanna W.</span> <span property="familyName">Jachowicz</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Joanna W.</span> <span property="familyName">Jachowicz</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jachowicz/Joanna+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Mackenzie</span> <span property="familyName">Strehle</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mackenzie</span> <span property="familyName">Strehle</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Strehle/Mackenzie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Davide</span> <span property="familyName">Maestrini</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Davide</span> <span property="familyName">Maestrini</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computational Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Maestrini/Davide" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Abhik K.</span> <span property="familyName">Banerjee</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Abhik K.</span> <span property="familyName">Banerjee</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9797-0104" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9797-0104</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Banerjee/Abhik+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Bhaven A.</span> <span property="familyName">Mistry</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bhaven A.</span> <span property="familyName">Mistry</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computational Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Claremont McKenna College, Claremont, CA 91711, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Mistry/Bhaven+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Iris</span> <span property="familyName">Dror</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Iris</span> <span property="familyName">Dror</span> <a class="orcid-id" href="https://orcid.org/0000-0001-7144-2250" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-7144-2250</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Dror/Iris" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Francois</span> <span property="familyName">Dossin</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Francois</span> <span property="familyName">Dossin</span> <a class="orcid-id" href="https://orcid.org/0000-0002-4085-1446" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-4085-1446</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, Director’s Unit, Heidelberg 69117, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Dossin/Francois" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Johannes</span> <span property="familyName">Schöneberg</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Johannes</span> <span property="familyName">Schöneberg</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Departments of Pharmacology &amp; Chemistry and Biochemistry, University of California San Diego, San Diego, CA 92093, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Sch%C3%B6neberg/Johannes" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Edith</span> <span property="familyName">Heard</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Edith</span> <span property="familyName">Heard</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">European Molecular Biology Laboratory, Director’s Unit, Heidelberg 69117, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Heard/Edith" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Mitchell</span> <span property="familyName">Guttman</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mitchell</span> <span property="familyName">Guttman</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Guttman/Mitchell" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Tom</span> <span property="familyName">Chou</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tom</span> <span property="familyName">Chou</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0785-6349" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0785-6349</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:tomchou@ucla.edu" property="email" title="Send email to Tom Chou">tomchou@ucla.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computational Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chou/Tom" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup> <a id="eadd0015" href="mailto:tomchou@ucla.edu" property="email" title="Send email to Tom Chou">tomchou@ucla.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Kathrin</span> <span property="familyName">Plath</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kathrin</span> <span property="familyName">Plath</span> <a class="orcid-id" href="https://orcid.org/0000-0001-7796-3372" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-7796-3372</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:kplath@mednet.ucla.edu" property="email" title="Send email to Kathrin Plath">kplath@mednet.ucla.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Plath/Kathrin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">8</a></sup> <a id="eadd0020" href="mailto:kplath@mednet.ucla.edu" property="email" title="Send email to Kathrin Plath">kplath@mednet.ucla.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-linked-content" aria-expanded="false" tabindex="0" role="button"><span>Linked Articles (1)</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Biological Chemistry, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Computational Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Claremont McKenna College, Claremont, CA 91711, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">European Molecular Biology Laboratory, Director’s Unit, Heidelberg 69117, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Departments of Pharmacology &amp; Chemistry and Biochemistry, University of California San Diego, San Diego, CA 92093, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received December 15, 2020</span>; <span>Revised July 29, 2021</span>; <span>Accepted October 11, 2021</span>; <span>Published online November 4, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.10.022" property="sameAs" target="_blank">10.1016/j.cell.2021.10.022 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421012757" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.10.022" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div><div id="core-linked-content" role="region" data-expandable="item" style="display: none;">
    

        


    

            

            

            <section data-widget-id="e587233e-22aa-4221-8ae1-22df489d46f7" data-widget-name="UX3linkedContentList" class="axel-linked-articles"><div class="section-paragraph linked-article"><ul class="rlist"><li class="linked-article__item"><div class="linked-article__article-type">Article</div><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(21)01230-7" target="_blank" title="RNA promotes the formation of spatial compartments in the nucleus">RNA promotes the formation of spatial compartments in the nucleus</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Quinodoz et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">November 4, 2021</span></div></li></ul></div></section>

            

        
</div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0026-2"><img src="/cms/asset/0540b9ed-f4ad-4863-9830-19ca05143514/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 25"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2901275-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (3)</a></li><li><a href="#app-2">Supporting citations</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901275-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.10.022">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2901275-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.10.022&amp;pii=S0092-8674%2821%2901275-7" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901275-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.10.022">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.10.022" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421012757&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421012757&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=25&amp;pii=S0092-8674%2820%29X0026-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)01327-1" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)01320-9" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (3)</a></li><li><a href="#app-2">Supporting citations</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-relations"><div class="relation"><div class="relation--head">Correction: </div><div class="relation--body">Errata<div class="content--title"><a href="/cell/fulltext/S0092-8674(21)01376-3"><em>Xist</em> nucleates local protein gradients to propagate silencing across the X chromosome</a></div><div class="content--contributors">Yolanda Markaki, Johnny Gan Chong, Yuying Wang ...</div><div class="content--publishDate"> December 9, 2021</div></div></div></div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">∼50 locally confined <i>Xist</i> foci of ∼2 RNA molecules initiate X-inactivation</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph"><i>Xist</i> foci seed dynamic supramolecular complexes (SMACs) of interacting proteins</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">IDR-mediated crowding of SPEN within SMACs is required for X-inactivation</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Silencing on the X proceeds through chromatin compaction and clustering of SMACs</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">The lncRNA <i>Xist</i> forms ∼50 diffraction-limited foci to transcriptionally silence one X chromosome. How this small number of RNA foci and interacting proteins regulate a much larger number of X-linked genes is unknown. We show that <i>Xist</i> foci are locally confined, contain ∼2 RNA molecules, and nucleate supramolecular complexes (SMACs) that include many copies of the critical silencing protein SPEN. Aggregation and exchange of SMAC proteins generate local protein gradients that regulate broad, proximal chromatin regions. Partitioning of numerous SPEN molecules into SMACs is mediated by their intrinsically disordered regions and essential for transcriptional repression. Polycomb deposition via SMACs induces chromatin compaction and the increase in SMACs density around genes, which propagates silencing across the X chromosome. Our findings introduce a mechanism for functional nuclear compartmentalization whereby crowding of transcriptional and architectural regulators enables the silencing of many target genes by few RNA molecules.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/1887748e-282e-48cc-940d-307b9e55e7b5/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.10.022/asset/b4a52ff0-6201-406a-81fd-ef90a19b5d67/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/1887748e-282e-48cc-940d-307b9e55e7b5/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;heterochromatin&quot;" alt="Search on this keyword" rel="nofollow">heterochromatin</a></li><li><a href="/action/doSearch?AllField=&quot;X+chromosome+inactivation&quot;" alt="Search on this keyword" rel="nofollow">X chromosome inactivation</a></li><li><a href="/action/doSearch?AllField=&quot;Xist+RNA&quot;" alt="Search on this keyword" rel="nofollow">Xist RNA</a></li><li><a href="/action/doSearch?AllField=&quot;RNA-binding+proteins&quot;" alt="Search on this keyword" rel="nofollow">RNA-binding proteins</a></li><li><a href="/action/doSearch?AllField=&quot;biomolecular+condensates&quot;" alt="Search on this keyword" rel="nofollow">biomolecular condensates</a></li><li><a href="/action/doSearch?AllField=&quot;supramolecular+complexes&quot;" alt="Search on this keyword" rel="nofollow">supramolecular complexes</a></li><li><a href="/action/doSearch?AllField=&quot;chromatin+organization&quot;" alt="Search on this keyword" rel="nofollow">chromatin organization</a></li><li><a href="/action/doSearch?AllField=&quot;macromolecular+dynamics&quot;" alt="Search on this keyword" rel="nofollow">macromolecular dynamics</a></li><li><a href="/action/doSearch?AllField=&quot;super-resolution+microscopy&quot;" alt="Search on this keyword" rel="nofollow">super-resolution microscopy</a></li><li><a href="/action/doSearch?AllField=&quot;quantitative+imaging&quot;" alt="Search on this keyword" rel="nofollow">quantitative imaging</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0050" role="paragraph">Mammalian genomes encode thousands of long non-coding (lnc) RNAs, many of which play key roles in regulating gene expression by localizing effector proteins to genomic targets (<span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Engreitz et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Engreitz, J.M. ∙ Ollikainen, N. ∙ Guttman, M.</div><div><strong>Long non-coding RNAs: spatial amplifiers that control nuclear structure and gene expression</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2016; <strong>17</strong>:756-770</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.EngreitzN.OllikainenM.GuttmanLong+non-coding+RNAs%3A+spatial+amplifiers+that+control+nuclear+structure+and+gene+expressionNat.+Rev.+Mol.+Cell+Biol.172016756770" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Rinn and Chang, 2012</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Rinn, J.L. ∙ Chang, H.Y.</div><div><strong>Genome regulation by long noncoding RNAs</strong></div><div><em>Annu. Rev. Biochem.</em> 2012; <strong>81</strong>:145-166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.RinnH.Y.ChangGenome+regulation+by+long+noncoding+RNAsAnnu.+Rev.+Biochem.812012145166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Statello et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Statello, L. ∙ Guo, C.J. ∙ Chen, L.L. ...</div><div><strong>Gene regulation by long non-coding RNAs and its biological functions</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2021; <strong>22</strong>:96-118</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.StatelloC.J.GuoL.L.ChenM.HuarteGene+regulation+by+long+non-coding+RNAs+and+its+biological+functionsNat.+Rev.+Mol.+Cell+Biol.22202196118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). There has been considerable debate about how lncRNAs can robustly regulate gene expression, given that they are often expressed at low levels (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Cabili et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cabili, M.N. ∙ Dunagin, M.C. ∙ McClanahan, P.D. ...</div><div><strong>Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:20</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.N.CabiliM.C.DunaginP.D.McClanahanA.BiaeschO.Padovan-MerharA.RegevJ.L.RinnA.RajLocalization+and+abundance+analysis+of+human+lncRNAs+at+single-cell+and+single-molecule+resolutionGenome+Biol.16201520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29-1" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-1" aria-controls="sref29-1">Derrien et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref29-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Derrien, T. ∙ Johnson, R. ∙ Bussotti, G. ...</div><div><strong>The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:1775-1789</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.DerrienR.JohnsonG.BussottiA.TanzerS.DjebaliH.TilgnerG.GuernecD.MartinA.MerkelD.G.KnowlesThe+GENCODE+v7+catalog+of+human+long+noncoding+RNAs%3A+analysis+of+their+gene+structure%2C+evolution%2C+and+expressionGenome+Res.22201217751789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). One such example is the <i>Xist</i> lncRNA, which silences transcription of a large number of genes across an entire chromosome.</div><div id="p0055" role="paragraph"><i>Xist</i> is transcribed from, coats, and silences one of the two X&nbsp;chromosomes during development of female placental mammals in a process referred to as X chromosome inactivation (XCI) (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Brockdorff et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Brockdorff, N. ∙ Bowness, J.S. ∙ Wei, G.</div><div><strong>Progress toward understanding chromosome silencing by Xist RNA</strong></div><div><em>Genes Dev.</em> 2020; <strong>34</strong>:733-744</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.BrockdorffJ.S.BownessG.WeiProgress+toward+understanding+chromosome+silencing+by+Xist+RNAGenes+Dev.342020733744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Galupa and Heard, 2018</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Galupa, R. ∙ Heard, E.</div><div><strong>X-Chromosome Inactivation: A Crossroads Between Chromosome Architecture and Gene Regulation</strong></div><div><em>Annu. Rev. Genet.</em> 2018; <strong>52</strong>:535-566</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.GalupaE.HeardX-Chromosome+Inactivation%3A+A+Crossroads+Between+Chromosome+Architecture+and+Gene+RegulationAnnu.+Rev.+Genet.522018535566" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Jégu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Jégu, T. ∙ Aeby, E. ∙ Lee, J.T.</div><div><strong>The X chromosome in space</strong></div><div><em>Nat. Rev. Genet.</em> 2017; <strong>18</strong>:377-389</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.J%C3%A9guE.AebyJ.T.LeeThe+X+chromosome+in+spaceNat.+Rev.+Genet.182017377389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref82-1" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-1" aria-controls="sref82-1">Plath et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref82-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Plath, K. ∙ Mlynarczyk-Evans, S. ∙ Nusinow, D.A. ...</div><div><strong>Xist RNA and the mechanism of X chromosome inactivation</strong></div><div><em>Annu. Rev. Genet.</em> 2002; <strong>36</strong>:233-278</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.PlathS.Mlynarczyk-EvansD.A.NusinowB.PanningXist+RNA+and+the+mechanism+of+X+chromosome+inactivationAnnu.+Rev.+Genet.362002233278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref115" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115" aria-controls="sref115">Wutz, 2011</a><div class="dropBlock__holder" data-db-target-of="sref115"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Wutz, A.</div><div><strong>Gene silencing in X-chromosome inactivation: advances in understanding facultative heterochromatin formation</strong></div><div><em>Nat. Rev. Genet.</em> 2011; <strong>12</strong>:542-553</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.WutzGene+silencing+in+X-chromosome+inactivation%3A+advances+in+understanding+facultative+heterochromatin+formationNat.+Rev.+Genet.122011542553" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>Xist</i> initiates gene silencing, large-scale chromatin remodeling, and formation of a unique nuclear compartment, the inactive X chromosome (Xi), through the recruitment of chromatin-modifying proteins, transcriptional silencers, and other RNA-binding proteins (<span class="dropBlock reference-citations"><a id="body-ref-sref20-1" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-1" aria-controls="sref20-1">Chaumeil et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref20-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Chaumeil, J. ∙ Le Baccon, P. ∙ Wutz, A. ...</div><div><strong>A novel role for Xist RNA in the formation of a repressive nuclear compartment into which genes are recruited when silenced</strong></div><div><em>Genes Dev.</em> 2006; <strong>20</strong>:2223-2237</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ChaumeilP.Le+BacconA.WutzE.HeardA+novel+role+for+Xist+RNA+in+the+formation+of+a+repressive+nuclear+compartment+into+which+genes+are+recruited+when+silencedGenes+Dev.20200622232237" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Chu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Chu, C. ∙ Zhang, Q.C. ∙ da Rocha, S.T. ...</div><div><strong>Systematic discovery of Xist RNA binding proteins</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:404-416</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.ChuQ.C.ZhangS.T.da+RochaR.A.FlynnM.BharadwajJ.M.CalabreseT.MagnusonE.HeardH.Y.ChangSystematic+discovery+of+Xist+RNA+binding+proteinsCell1612015404416" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Giorgetti et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Giorgetti, L. ∙ Lajoie, B.R. ∙ Carter, A.C. ...</div><div><strong>Structural organization of the inactive X chromosome in the mouse</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:575-579</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.GiorgettiB.R.LajoieA.C.CarterM.AttiaY.ZhanJ.XuC.J.ChenN.KaplanH.Y.ChangE.HeardJ.DekkerStructural+organization+of+the+inactive+X+chromosome+in+the+mouseNature5352016575579" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63-1" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-1" aria-controls="sref63-1">McHugh et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref63-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>McHugh, C.A. ∙ Chen, C.K. ∙ Chow, A. ...</div><div><strong>The Xist lncRNA interacts directly with SHARP to silence transcription through HDAC3</strong></div><div><em>Nature.</em> 2015; <strong>521</strong>:232-236</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.A.McHughC.K.ChenA.ChowC.F.SurkaC.TranP.McDonelA.Pandya-JonesM.BlancoC.BurghardA.MoradianThe+Xist+lncRNA+interacts+directly+with+SHARP+to+silence+transcription+through+HDAC3Nature5212015232236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Minajigi et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Minajigi, A. ∙ Froberg, J. ∙ Wei, C. ...</div><div><strong>Chromosomes. A comprehensive Xist interactome reveals cohesin repulsion and an RNA-directed chromosome conformation</strong></div><div><em>Science.</em> 2015;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MinajigiJ.FrobergC.WeiH.SunwooB.KesnerD.ColognoriD.LessingB.PayerM.BoukhaliW.HaasJ.T.LeeChromosomes.+A+comprehensive+Xist+interactome+reveals+cohesin+repulsion+and+an+RNA-directed+chromosome+conformationScience201510.1126%2Fscience.aab2276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110-1" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-1" aria-controls="sref110-1">Wang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref110-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Wang, C.Y. ∙ Jegu, T. ∙ Chu, H.P. ...</div><div><strong>SMCHD1 Merges Chromosome Compartments and Assists Formation of Super-Structures on the Inactive X</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:406-421.e25</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.Y.WangT.JeguH.P.ChuH.J.OhJ.T.LeeSMCHD1+Merges+Chromosome+Compartments+and+Assists+Formation+of+Super-Structures+on+the+Inactive+XCell1742018406421.e25" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The protein SPEN is essential for initiating the silencing of virtually all X-linked genes (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Chu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Chu, C. ∙ Zhang, Q.C. ∙ da Rocha, S.T. ...</div><div><strong>Systematic discovery of Xist RNA binding proteins</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:404-416</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.ChuQ.C.ZhangS.T.da+RochaR.A.FlynnM.BharadwajJ.M.CalabreseT.MagnusonE.HeardH.Y.ChangSystematic+discovery+of+Xist+RNA+binding+proteinsCell1612015404416" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63-2" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-2" aria-controls="sref63-2">McHugh et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref63-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>McHugh, C.A. ∙ Chen, C.K. ∙ Chow, A. ...</div><div><strong>The Xist lncRNA interacts directly with SHARP to silence transcription through HDAC3</strong></div><div><em>Nature.</em> 2015; <strong>521</strong>:232-236</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.A.McHughC.K.ChenA.ChowC.F.SurkaC.TranP.McDonelA.Pandya-JonesM.BlancoC.BurghardA.MoradianThe+Xist+lncRNA+interacts+directly+with+SHARP+to+silence+transcription+through+HDAC3Nature5212015232236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Moindrot et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Moindrot, B. ∙ Cerase, A. ∙ Coker, H. ...</div><div><strong>A Pooled shRNA Screen Identifies Rbm15, Spen, and Wtap as Factors Required for Xist RNA-Mediated Silencing</strong></div><div><em>Cell Rep.</em> 2015; <strong>12</strong>:562-572</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.MoindrotA.CeraseH.CokerO.MasuiA.GrijzenhoutG.PintacudaL.SchermellehT.B.NesterovaN.BrockdorffA+Pooled+shRNA+Screen+Identifies+Rbm15%2C+Spen%2C+and+Wtap+as+Factors+Required+for+Xist+RNA-Mediated+SilencingCell+Rep.122015562572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Monfort et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Monfort, A. ∙ Di Minin, G. ∙ Postlmayr, A. ...</div><div><strong>Identification of Spen as a Crucial Factor for Xist Function through Forward Genetic Screening in Haploid Embryonic Stem Cells</strong></div><div><em>Cell Rep.</em> 2015; <strong>12</strong>:554-561</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MonfortG.Di+MininA.PostlmayrR.FreimannF.ArietiS.ThoreA.WutzIdentification+of+Spen+as+a+Crucial+Factor+for+Xist+Function+through+Forward+Genetic+Screening+in+Haploid+Embryonic+Stem+CellsCell+Rep.122015554561" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, a subset of X-linked genes also requires other <i>Xist</i>-interactors for silencing, including non-canonical PRC1-type Polycomb group protein complexes and the architectural protein structural-maintenance of chromosomes hinge domain containing 1 (SMCHD1) (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Almeida et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Almeida, M. ∙ Pintacuda, G. ∙ Masui, O. ...</div><div><strong>PCGF3/5-PRC1 initiates Polycomb recruitment in X chromosome inactivation</strong></div><div><em>Science.</em> 2017; <strong>356</strong>:1081-1084</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.AlmeidaG.PintacudaO.MasuiY.KosekiM.GdulaA.CeraseD.BrownA.MouldC.InnocentM.NakayamaPCGF3%2F5-PRC1+initiates+Polycomb+recruitment+in+X+chromosome+inactivationScience356201710811084" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Blewitt et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Blewitt, M.E. ∙ Gendrel, A.V. ∙ Pang, Z. ...</div><div><strong>SmcHD1, containing a structural-maintenance-of-chromosomes hinge domain, has a critical role in X inactivation</strong></div><div><em>Nat. Genet.</em> 2008; <strong>40</strong>:663-669</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.E.BlewittA.V.GendrelZ.PangD.B.SparrowN.WhitelawJ.M.CraigA.ApedaileD.J.HiltonS.L.DunwoodieN.BrockdorffSmcHD1%2C+containing+a+structural-maintenance-of-chromosomes+hinge+domain%2C+has+a+critical+role+in+X+inactivationNat.+Genet.402008663669" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Jansz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Jansz, N. ∙ Nesterova, T. ∙ Keniry, A. ...</div><div><strong>Smchd1 Targeting to the Inactive X Is Dependent on the Xist-HnrnpK-PRC1 Pathway</strong></div><div><em>Cell Rep.</em> 2018; <strong>25</strong>:1912-1923.e9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.JanszT.NesterovaA.KeniryM.IminitoffP.F.HickeyG.PintacudaO.MasuiS.KobelkeN.GeogheganK.A.BreslinSmchd1+Targeting+to+the+Inactive+X+Is+Dependent+on+the+Xist-HnrnpK-PRC1+PathwayCell+Rep.25201819121923.e9" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-1" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-1" aria-controls="sref74-1">Nesterova et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref74-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Nesterova, T.B. ∙ Wei, G. ∙ Coker, H. ...</div><div><strong>Systematic allelic analysis defines the interplay of key pathways in X chromosome inactivation</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:3129</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.B.NesterovaG.WeiH.CokerG.PintacudaJ.S.BownessT.ZhangM.AlmeidaB.BloechlB.MoindrotE.J.CarterSystematic+allelic+analysis+defines+the+interplay+of+key+pathways+in+X+chromosome+inactivationNat.+Commun.1020193129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-1" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-1" aria-controls="sref81-1">Pintacuda et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref81-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Pintacuda, G. ∙ Wei, G. ∙ Roustan, C. ...</div><div><strong>hnRNPK Recruits PCGF3/5-PRC1 to the Xist RNA B-Repeat to Establish Polycomb-Mediated Chromosomal Silencing</strong></div><div><em>Mol. Cell.</em> 2017; <strong>68</strong>:955-969.e10</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.PintacudaG.WeiC.RoustanB.A.KirmizitasN.SolcanA.CeraseA.CastelloS.MohammedB.MoindrotT.B.NesterovahnRNPK+Recruits+PCGF3%2F5-PRC1+to+the+Xist+RNA+B-Repeat+to+Establish+Polycomb-Mediated+Chromosomal+SilencingMol.+Cell682017955969.e10" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref111-1" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-1" aria-controls="sref111-1">Wang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Wang, C.Y. ∙ Colognori, D. ∙ Sunwoo, H. ...</div><div><strong>PRC1 collaborates with SMCHD1 to fold the X-chromosome and spread Xist RNA between chromosome compartments</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2950</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.Y.WangD.ColognoriH.SunwooD.WangJ.T.LeePRC1+collaborates+with+SMCHD1+to+fold+the+X-chromosome+and+spread+Xist+RNA+between+chromosome+compartmentsNat.+Commun.1020192950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, some X-linked genes become repressed sooner than others (<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Barros de Andrade E. Sousa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Barros de Andrade E. Sousa, L. ∙ Jonkers, I. ∙ Syx, L. ...</div><div><strong>Kinetics of <i>Xist</i>-induced gene silencing can be predicted from combinations of epigenetic and genomic features</strong></div><div><em>Genome Res.</em> 2019; <strong>29</strong>:1087-1099</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.Barros+de+Andrade+E.+SousaI.JonkersL.SyxI.DunkelJ.ChaumeilC.PicardB.ForetC.J.ChenJ.T.LisE.HeardKinetics+of+Xist-induced+gene+silencing+can+be+predicted+from+combinations+of+epigenetic+and+genomic+featuresGenome+Res.29201910871099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Why X-linked genes differ in their silencing dynamics and require multiple repressive pathways for inactivation remain major questions.</div><div id="p0060" role="paragraph">Based on conventional fluorescence microscopy and genomic methods that measure an ensemble of millions of cells at high resolution, <i>Xist</i>, its effector proteins, and Xi chromatin modifications appear to accumulate along the entire chromosome (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Clemson et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Clemson, C.M. ∙ McNeil, J.A. ∙ Willard, H.F. ...</div><div><strong>XIST RNA paints the inactive X chromosome at interphase: evidence for a novel RNA involved in nuclear/chromosome structure</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1996; <strong>132</strong>:259-275</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.ClemsonJ.A.McNeilH.F.WillardJ.B.LawrenceXIST+RNA+paints+the+inactive+X+chromosome+at+interphase%3A+evidence+for+a+novel+RNA+involved+in+nuclear%2Fchromosome+structureJ.%C2%A0Cell+Biol.1321996259275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Engreitz et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Engreitz, J.M. ∙ Pandya-Jones, A. ∙ McDonel, P. ...</div><div><strong>The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome</strong></div><div><em>Science.</em> 2013; <strong>341</strong>:1237973</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.EngreitzA.Pandya-JonesP.McDonelA.ShishkinK.SirokmanC.SurkaS.KadriJ.XingA.GorenE.S.LanderThe+Xist+lncRNA+exploits+three-dimensional+genome+architecture+to+spread+across+the+X+chromosomeScience34120131237973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">Plath et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Plath, K. ∙ Fang, J. ∙ Mlynarczyk-Evans, S.K. ...</div><div><strong>Role of histone H3 lysine 27 methylation in X inactivation</strong></div><div><em>Science.</em> 2003; <strong>300</strong>:131-135</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.PlathJ.FangS.K.Mlynarczyk-EvansR.CaoK.A.WorringerH.WangC.C.de+la+CruzA.P.OtteB.PanningY.ZhangRole+of+histone+H3+lysine+27+methylation+in+X+inactivationScience3002003131135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-1" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-1" aria-controls="sref94-1">Silva et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref94-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Silva, J. ∙ Mak, W. ∙ Zvetkova, I. ...</div><div><strong>Establishment of histone h3 methylation on the inactive X chromosome requires transient recruitment of Eed-Enx1 polycomb group complexes</strong></div><div><em>Dev. Cell.</em> 2003; <strong>4</strong>:481-495</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.SilvaW.MakI.ZvetkovaR.AppanahT.B.NesterovaZ.WebsterA.H.PetersT.JenuweinA.P.OtteN.BrockdorffEstablishment+of+histone+h3+methylation+on+the+inactive+X+chromosome+requires+transient+recruitment+of+Eed-Enx1+polycomb+group+complexesDev.+Cell42003481495" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">Simon et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Simon, M.D. ∙ Pinter, S.F. ∙ Fang, R. ...</div><div><strong>High-resolution Xist binding maps reveal two-step spreading during X-chromosome inactivation</strong></div><div><em>Nature.</em> 2013; <strong>504</strong>:465-469</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.D.SimonS.F.PinterR.FangK.SarmaM.Rutenberg-SchoenbergS.K.BowmanB.A.KesnerV.K.MaierR.E.KingstonJ.T.LeeHigh-resolution+Xist+binding+maps+reveal+two-step+spreading+during+X-chromosome+inactivationNature5042013465469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref123-1" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-1" aria-controls="sref123-1">Zylicz et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref123-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Zylicz, J.J. ∙ Bousard, A. ∙ Zumer, K. ...</div><div><strong>The Implication of Early Chromatin Changes in X Chromosome Inactivation</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:182-197.e23</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.J.ZyliczA.BousardK.ZumerF.DossinE.MohammadS.T.da+RochaB.SchwalbL.SyxF.DingliD.LoewThe+Implication+of+Early+Chromatin+Changes+in+X+Chromosome+InactivationCell1762019182197.e23" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These observations have led to a model in which <i>Xist</i> and its interacting proteins form ribonucleoprotein complexes that are distributed across all X-linked genes to control gene expression. However, super-resolution microscopy has shown that <i>Xist</i> distributes as only 50 to 150 diffraction-limited foci on the Xi in differentiated cells (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Cerase et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Cerase, A. ∙ Smeets, D. ∙ Tang, Y.A. ...</div><div><strong>Spatial separation of Xist RNA and polycomb proteins revealed by superresolution microscopy</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:2235-2240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.CeraseD.SmeetsY.A.TangM.GdulaF.KrausM.SpivakovB.MoindrotM.LeleuA.TattermuschJ.DemmerleSpatial+separation+of+Xist+RNA+and+polycomb+proteins+revealed+by+superresolution+microscopyProc.+Natl.+Acad.+Sci.+USA111201422352240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Markaki et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Markaki, Y. ∙ Smeets, D. ∙ Fiedler, S. ...</div><div><strong>The potential of 3D-FISH and super-resolution structured illumination microscopy for studies of 3D nuclear architecture: 3D structured illumination microscopy of defined chromosomal structures visualized by 3D (immuno)-FISH opens new perspectives for studies of nuclear architecture</strong></div><div><em>BioEssays.</em> 2012; <strong>34</strong>:412-426</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.MarkakiD.SmeetsS.FiedlerV.J.SchmidL.SchermellehT.CremerM.CremerThe+potential+of+3D-FISH+and+super-resolution+structured+illumination+microscopy+for+studies+of+3D+nuclear+architecture%3A+3D+structured+illumination+microscopy+of+defined+chromosomal+structures+visualized+by+3D+%28immuno%29-FISH+opens+new+perspectives+for+studies+of+nuclear+architectureBioEssays342012412426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Smeets et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Smeets, D. ∙ Markaki, Y. ∙ Schmid, V.J. ...</div><div><strong>Three-dimensional super-resolution microscopy of the inactive X chromosome territory reveals a collapse of its active nuclear compartment harboring distinct Xist RNA foci</strong></div><div><em>Epigenetics Chromatin.</em> 2014; <strong>7</strong>:8</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.SmeetsY.MarkakiV.J.SchmidF.KrausA.TattermuschA.CeraseM.SterrS.FiedlerJ.DemmerleJ.PopkenThree-dimensional+super-resolution+microscopy+of+the+inactive+X+chromosome+territory+reveals+a+collapse+of+its+active+nuclear+compartment+harboring+distinct+Xist+RNA+fociEpigenetics+Chromatin720148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99-1" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-1" aria-controls="sref99-1">Sunwoo et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref99-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Sunwoo, H. ∙ Wu, J.Y. ∙ Lee, J.T.</div><div><strong>The Xist RNA-PRC2 complex at 20-nm resolution reveals a low Xist stoichiometry and suggests a hit-and-run mechanism in mouse cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4216-E4225</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.SunwooJ.Y.WuJ.T.LeeThe+Xist+RNA-PRC2+complex+at+20-nm+resolution+reveals+a+low+Xist+stoichiometry+and+suggests+a+hit-and-run+mechanism+in+mouse+cellsProc.+Natl.+Acad.+Sci.+USA1122015E4216E4225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Therefore, these foci cannot regulate gene expression through simultaneous accumulation at each target gene. Thus, even though we know the effector proteins of <i>Xist</i>, the mechanism by which RNA foci exploit these proteins to induce silencing of ∼1,000 genes distributed over 167 million base pairs remains unknown. Here, we addressed this fundamental problem by developing quantitative super-resolution microscopy approaches to interrogate the stoichiometry and spatial relationship of <i>Xist</i> to its effector proteins and target genes during the initiation of XCI. We also performed kinetic measurements and single-particle tracking to explore the dynamics and mobility of <i>Xist</i> and associated proteins.</div><div id="p0065" role="paragraph">We discovered that <i>Xist</i> foci are locally confined and that they induce the <i>de novo</i> formation of local protein compartments that encompass <i>Xist</i>-interactors at concentrations exceeding those of the RNA. We refer to these compartments as supramolecular complexes (SMACs). SMACs are dynamic structures formed by transient protein interactions around a slowly exchanging <i>Xist</i> core. The rapid binding and dissociation of most <i>Xist</i>-interacting proteins in SMACs create local protein concentration gradients over broad chromosomal regions. We show that the intrinsically disordered regions of SPEN are essential for its integration into SMACs and for gene silencing, and that Polycomb-mediated chromatin reconfiguration propagates silencing across the X chromosome. In summary, our work reveals that protein crowding enables a limited number of locally confined seeding RNA molecules to silence a much larger number of target genes.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Progressive gene silencing during XCI is associated with chromatin compaction</h3><div id="p0070" role="paragraph">To explore how <i>Xist</i> orchestrates the formation of the Xi, we differentiated female mouse embryonic stem cells (ESCs) to epiblast-like cells (EpiLCs), which leads to the induction of <i>Xist</i> expression and initiation of XCI (<a id="crosref0330" href="#fig1">Figures 1</a>A and <a id="crosref0335" href="#figs1">S1</a>A). Gene silencing occurs predominantly during the transition from day 2 (D2) to D4, as shown by nascent transcript detection of rapidly (<i>Rlim</i>) and slowly (<i>AtrX</i> and <i>Mecp2</i>) silencing genes (<a id="crosref0340" href="#fig1">Figures 1</a>B, 1C, and <a id="crosref0345" href="#figs1">S1</a>B). Single-cell (sc) RNA sequencing (RNA-seq) analyses extended this result to all X-linked genes (<a id="crosref0350" href="#fig1">Figures 1</a>D and <a id="crosref0355" href="#figs1">S1</a>C). These data confirm that <i>Xist</i> coating and gene silencing are stepwise processes (<span class="dropBlock reference-citations"><a id="body-ref-sref20-2" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-2" aria-controls="sref20-2">Chaumeil et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref20-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Chaumeil, J. ∙ Le Baccon, P. ∙ Wutz, A. ...</div><div><strong>A novel role for Xist RNA in the formation of a repressive nuclear compartment into which genes are recruited when silenced</strong></div><div><em>Genes Dev.</em> 2006; <strong>20</strong>:2223-2237</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ChaumeilP.Le+BacconA.WutzE.HeardA+novel+role+for+Xist+RNA+in+the+formation+of+a+repressive+nuclear+compartment+into+which+genes+are+recruited+when+silencedGenes+Dev.20200622232237" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and establish the D2 to D4 transition as a critical window for dissecting the relationship between <i>Xist</i>, its interacting proteins, and gradual gene silencing. Henceforth, we refer to the D2 X chromosome as the “pre-Xi” and the D4 X chromosome as the “Xi.”</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/8f06d774-4912-42d1-a5ad-de7455b82a77/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.10.022/asset/c19b267e-71ac-4696-b50d-4957eeae7fd7/main.assets/gr1.jpg" height="710" width="752" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/8f06d774-4912-42d1-a5ad-de7455b82a77/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">∼50 <i>Xist</i> foci initiate XCI</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Schematic of the ESC differentiation protocol.</div><div id="fspara0020" role="paragraph">(B) RNA FISH for <i>Xist/Tsix</i> (green), <i>Rlim</i> (yellow), and <i>Atrx</i> (magenta) transcripts during differentiation. DAPI staining is shown in gray. Right panels show magnifications of the <i>Tsix</i> signals on the Xa or the <i>Xist</i> signals on the Xi and nascent gene transcripts.</div><div id="fspara0025" role="paragraph">(C) Percentage of cells with given nascent transcripts of <i>Rlim</i>, <i>Atrx</i>, and <i>Mecp2</i> under <i>Xist</i> clouds during differentiation from two replicates. Error bars denote standard deviation; n is the number of cells analyzed.</div><div id="fspara0030" role="paragraph">(D) Violin plots of the Xi expression ratios of X-linked genes averaged across single cells expressing <i>Xist</i> from the 129 allele at D2 and D4. The ratios of <i>Rlim</i>, <i>Atrx</i>, and <i>Mecp2</i> are highlighted.</div><div id="fspara0035" role="paragraph">(E) DNA/RNA FISH with X chromosome paints (mmX, green) and Xist probes (red) of cells at D2 and D4. DAPI is shown in gray. n is the number of cells analyzed in (F).</div><div id="fspara0040" role="paragraph">(F) Boxplots showing the volume and sphericity ratio of Xa/Xa (blue) in cells not expressing <i>Xist</i> at D2 and Xa/Xi (green) in cells with an <i>Xist</i> cloud at D2 and D4. Mann-Whitney-Wilcoxon (MWW) p values are given.</div><div id="fspara0045" role="paragraph">(G) Schematic of the spectral barcoding strategy applied to map X chromosome configuration.</div><div id="fspara0050" role="paragraph">(H) DNA FISH of the spectrally barcoded genomic regions described in (G). Overlay with <i>Xist</i> RNA FISH signals (far right, yellow) was used to score for the Xi at D2 (top) and D4 (bottom).</div><div id="fspara0055" role="paragraph">(I) 2D configuration plots of average coordinates of genomic barcodes from (H), extracted with 95% confidence. n is the number of cells analyzed from three experiments.</div><div id="fspara0060" role="paragraph">(J) Illustration of live-cell <i>Xist</i> labeling strategy.</div><div id="fspara0065" role="paragraph">(K) 3D-SIM projections showing <i>Xist</i><sup>MS2-GFP</sup> signals (green) and DAPI staining (gray) at the indicated differentiation day.</div><div id="fspara0070" role="paragraph">(L) Violin plots of the 3D-SIM quantification of <i>Xist</i><sup>MS2-GFP</sup> foci number at D2 and D4. n denotes the number of Xist foci measured, followed by the number of cells analyzed from two replicates. MWW p value is given.</div><div id="fspara0075" role="paragraph">(M) As in (L) at D4 after scoring for cell cycle stages.</div><div id="fspara0080" role="paragraph">(N) As in (L), except for showing integrated density (AFU) and volume (μm<sup>3</sup>) of <i>Xist</i><sup>MS2-GFP</sup> foci at D2 and D4 for the same sets of foci.</div><div id="fspara0085" role="paragraph">(O) Histograms depicting integrated density (AFU) of <i>Xist</i><sup>MS2-GFP</sup> foci (∼30 GFP molecules per <i>Xist</i>) and GFP nanocages (cage<sup>60GFP</sup>) per pixel kernel density, detected by 3D-SIM. n denotes the number of foci measured followed by the number of cells analyzed from two replicates.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/f848c78d-eaae-42e2-b03c-e0ddd9c6ac4c/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.10.022/asset/99a82c67-aef8-4a7d-83af-ffab46c09f96/main.assets/figs1.jpg" height="939" width="695" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/f848c78d-eaae-42e2-b03c-e0ddd9c6ac4c/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Progressive compaction of the X through a conserved <i>Xist</i> cluster, related to <a id="crosref2015" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0430" role="paragraph">(A) Quantification of the proportion of cells with <i>Xist</i> signals on one or both X chromosomes at the indicated differentiation day, based on RNA FISH from two experiments. Error bars indicate standard deviation and n the number of cells analyzed.</div><div id="fspara0435" role="paragraph">(B) Wide-field projections of RNA FISH experiments with <i>Xist</i> RNA (green) and <i>Mecp2</i> (magenta) probes at D2 and D4, with DAPI counterstaining in gray. Yellow and blue arrows indicate the presence and lack of gene expression, respectively.</div><div id="fspara0440" role="paragraph">(C) Violin plots of the Xi expression ratios of X-linked genes previously classified as early, late, very late silencing and escaping from XCI (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Barros de Andrade E. Sousa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Barros de Andrade E. Sousa, L. ∙ Jonkers, I. ∙ Syx, L. ...</div><div><strong>Kinetics of <i>Xist</i>-induced gene silencing can be predicted from combinations of epigenetic and genomic features</strong></div><div><em>Genome Res.</em> 2019; <strong>29</strong>:1087-1099</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.Barros+de+Andrade+E.+SousaI.JonkersL.SyxI.DunkelJ.ChaumeilC.PicardB.ForetC.J.ChenJ.T.LisE.HeardKinetics+of+Xist-induced+gene+silencing+can+be+predicted+from+combinations+of+epigenetic+and+genomic+featuresGenome+Res.29201910871099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), averaged across single cells expressing <i>Xist</i> from either the 129 or Cas X chromosome in female WT cells at D2 and D4.</div><div id="fspara0445" role="paragraph">(D) Ellipsoids of two-dimensional (2D) coordinates of X chromosome (mmX) barcodes on the Xa and Xi at D2 and D4, showing a radius of 95% confidence in the allocated positions. For each probe, the position along the X chromosome is given in megabases (Mb).</div><div id="fspara0450" role="paragraph">(E) Heatmaps showing the average three-dimensional (3D) spatial distances of genomic barcodes across the X chromosome described in (D) on the Xa and Xi at D2 and D4 of differentiation.</div><div id="fspara0455" role="paragraph">(F) Heatmaps as in (E), showing changes in distances between probes on the Xa and Xi on D2 and D4.</div><div id="fspara0460" role="paragraph">(G) Demonstration of effective silencing of <i>Atrx</i> in differentiating unmodified cells (<i>Xist</i><sup>wt</sup>) and <i>Xist</i><sup>MS2-GFP</sup> cells at D4. The proportion of cells displaying <i>Atrx</i> silencing is given (left graph). For <i>Xist</i><sup>MS2-GFP</sup> cells, silencing was assessed before and after induction of MCP-GFP expression with doxycycline. An representative image of <i>Atrx</i> silencing in an <i>Xist</i><sup>MS2-GFP</sup> cell with expression of <i>Xist</i>-MS2 is shown. Additionally, the proportion of <i>Xist</i><sup>MS2-GFP</sup> cells with <i>Xist</i>-MS2 clouds was quantified (right graph). Error bars indicate the standard deviation. n is the number of cells analyzed from three experiments.</div><div id="fspara0465" role="paragraph">(H) 3D-SIM projections showing <i>Xist</i> RNA FISH signals (green) and DAPI counterstaining (gray) at the indicated differentiation day. Bar: 5&nbsp;μm.</div><div id="fspara0470" role="paragraph">(I) Violin plots of the 3D-SIM quantification of the number of <i>Xist</i> foci (#), their integrated density of fluorescence (AFU), volume (μm<sup>3</sup>), and Feret diameter (μm) at indicated differentiation days. White dots denote the median, thick black bars the interquartile range, thin bars show upper/lower values. n denotes the number of <i>Xist</i> foci measured followed by the number of cells analyzed from three experiments. MWW p values are given for comparisons indicated by horizontal black lines. NS, Not Significant.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">Although architectural differences between the active X chromosome (Xa) and Xi are well known (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Darrow et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Darrow, E.M. ∙ Huntley, M.H. ∙ Dudchenko, O. ...</div><div><strong>Deletion of DXZ4 on the human inactive X chromosome alters higher-order genome architecture</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:E4504-E4512</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.M.DarrowM.H.HuntleyO.DudchenkoE.K.StamenovaN.C.DurandZ.SunS.C.HuangA.L.SanbornI.MacholM.ShamimDeletion+of+DXZ4+on+the+human+inactive+X+chromosome+alters+higher-order+genome+architectureProc.+Natl.+Acad.+Sci.+USA1132016E4504E4512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Giorgetti et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Giorgetti, L. ∙ Lajoie, B.R. ∙ Carter, A.C. ...</div><div><strong>Structural organization of the inactive X chromosome in the mouse</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:575-579</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.GiorgettiB.R.LajoieA.C.CarterM.AttiaY.ZhanJ.XuC.J.ChenN.KaplanH.Y.ChangE.HeardJ.DekkerStructural+organization+of+the+inactive+X+chromosome+in+the+mouseNature5352016575579" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref102-1" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-1" aria-controls="sref102-1">Teller et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref102-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Teller, K. ∙ Illner, D. ∙ Thamm, S. ...</div><div><strong>A top-down analysis of Xa- and Xi-territories reveals differences of higher order structure at <sup>3</sup> 20 Mb genomic length scales</strong></div><div><em>Nucleus.</em> 2011; <strong>2</strong>:465-477</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.TellerD.IllnerS.ThammC.S.Casas-DelucchiR.VersteegM.IndemansT.CremerM.CremerA+top-down+analysis+of+Xa-+and+Xi-territories+reveals+differences+of+higher+order+structure+at+3+20+Mb+genomic+length+scalesNucleus22011465477" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref111-2" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-2" aria-controls="sref111-2">Wang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Wang, C.Y. ∙ Colognori, D. ∙ Sunwoo, H. ...</div><div><strong>PRC1 collaborates with SMCHD1 to fold the X-chromosome and spread Xist RNA between chromosome compartments</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2950</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.Y.WangD.ColognoriH.SunwooD.WangJ.T.LeePRC1+collaborates+with+SMCHD1+to+fold+the+X-chromosome+and+spread+Xist+RNA+between+chromosome+compartmentsNat.+Commun.1020192950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), it remains unclear when they arise during XCI. Volume and sphericity measurements upon X-painting showed that the pre-Xi is similar to the Xa and that the Xi at D4 is as compact and spherical as in somatic cells (<a id="crosref0385" href="#fig1">Figures 1</a>E and 1F) (<span class="dropBlock reference-citations"><a id="body-ref-sref102-2" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-2" aria-controls="sref102-2">Teller et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref102-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Teller, K. ∙ Illner, D. ∙ Thamm, S. ...</div><div><strong>A top-down analysis of Xa- and Xi-territories reveals differences of higher order structure at <sup>3</sup> 20 Mb genomic length scales</strong></div><div><em>Nucleus.</em> 2011; <strong>2</strong>:465-477</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.TellerD.IllnerS.ThammC.S.Casas-DelucchiR.VersteegM.IndemansT.CremerM.CremerA+top-down+analysis+of+Xa-+and+Xi-territories+reveals+differences+of+higher+order+structure+at+3+20+Mb+genomic+length+scalesNucleus22011465477" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Accordingly, assessing the conformation of seven loci on the X through DNA FISH revealed a moderate change of the higher-order configuration in the pre-Xi compared to the Xa and a dramatic difference between the Xa and Xi (<a id="crosref0395" href="#fig1">Figures 1</a>G–1I and <a id="crosref0400" href="#figs1">S1</a>D–S1F). Thus, gene silencing is associated with major changes in higher-order chromatin structure and both processes need to be considered to understand the mechanism through which <i>Xist</i> foci form the Xi.</div></section><section id="sec-2-2"><h3>∼50 <i>Xist</i> foci of two transcripts induce XCI</h3><div id="p0080" role="paragraph">As the number of Xist foci during the initiation of XCI is unknown, we employed super-resolution three-dimensional structured illumination microscopy (3D-SIM) to quantify them during the D2 to D4 transition. To this end, we generated a female mouse ESC line that allows for <i>Xist</i> detection in fixed as well as living cells. Specifically, exploiting the MS2 RNA-MS2 coat protein (MCP) interaction (<span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Bertrand et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bertrand, E. ∙ Chartrand, P. ∙ Schaefer, M. ...</div><div><strong>Localization of ASH1 mRNA particles in living yeast</strong></div><div><em>Mol. Cell.</em> 1998; <strong>2</strong>:437-445</div></div><div class="external-links"><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.10.022&amp;key=2-s2.0-0032185810" target="_blank" aria-label="Open Scopus in new tab">Scopus (1168)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.BertrandP.ChartrandM.SchaeferS.M.ShenoyR.H.SingerR.M.LongLocalization+of+ASH1+mRNA+particles+in+living+yeastMol.+Cell21998437445" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we tagged the Xist gene on one of the two X chromosomes with 24 MS2-repeats. We then expressed MCP-GFP to label <i>Xist</i> with GFP and confirmed the functionality of the Xist<sup>MS2-GFP</sup> allele (<a id="crosref0410" href="#fig1">Figures 1</a>J, 1K, and <a id="crosref0415" href="#figs1">S1</a>G). Quantitative 3D-SIM analysis of <i>Xist</i><sup>MS2-GFP</sup> signals showed that the <i>Xist</i> territory consists of, on average, 74 diffraction-limited foci on the pre-Xi and 60 on the Xi (<a id="crosref0420" href="#fig1">Figure&nbsp;1</a>L), which we corroborated by RNA FISH (<a id="crosref0425" href="#figs1">Figures S1</a>H and S1I). We also found that the doubling of the X chromosome with DNA replication is accompanied by the doubling of <i>Xist</i> foci from ∼50 in G1 to ∼100 in G2 and that the number of foci correlates with chromosome length (<a id="crosref0430" href="#fig1">Figures 1</a>M and <a id="crosref0435" href="#figs2">S2</a>A–S2D). Thus, the variability in the number of <i>Xist</i> foci is largely due to differences in cell cycle across the cell population. <i>Xist</i> foci maintain their integrated density and volume, which is consistent with the constitutive transcription of the <i>Xist</i> locus, suggesting that RNA levels on the Xi are stable throughout XCI (<a id="crosref0440" href="#fig1">Figures 1</a>N, <a id="crosref0445" href="#figs1">S1</a>I, <a id="crosref0450" href="#figs2">S2</a>E, and S2F). Taken together, these data reveal that XCI is induced by only ∼50 <i>Xist</i> foci and that the pre-Xi to Xi transition occurs without a dramatic change in their number.</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/751fb368-e5f8-4ce6-becc-55e342033636/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.10.022/asset/21395ca2-4f88-4fae-9dd9-b175525fbee6/main.assets/figs2.jpg" height="883" width="750" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/751fb368-e5f8-4ce6-becc-55e342033636/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Features of <i>Xist</i> foci across cell types and X chromosome lengths, related to <a id="crosref2025" href="#fig1">Figures 1</a> and <a id="crosref2030" href="#fig2">2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0480" role="paragraph">(A) 3D-SIM projections of immuno-RNA FISH experiments detecting <i>Xist</i> RNA, EdU and H3 phospho-serine10 in C127 cells to score for the cell cycle stages (G1, early S-phase (Early S), mid S-phase (Mid S), late S-phase (Late S) and G2). DAPI counterstaining is shown in gray. Stainings for EdU (to score S-phase cells) and anti-H3 phospho-serine10 antibodies (to score cells in G2) were detected in the same channel shown in magenta. Bar: 5&nbsp;μm.</div><div id="fspara0485" role="paragraph">(B) Violin plots showing the number of <i>Xist</i> RNA foci in cells in different cell cycle stages from (A). Dots denote the median, thick black bars the interquartile range, thin bars show upper/lower values. <i>n</i> denotes the number of <i>Xist</i> foci measured followed by the number of cells scored for each cell cycle stage. 47 cells were analyzed in total from two replicates. A separate experiment of 35 cells from two replicates without scoring for cell cycle is shown for comparison (C127). The Xi is known to replicate around mid-S.</div><div id="fspara0490" role="paragraph">(C) 3D-SIM projections of RNA FISH experiments in human cell lines (named by their catalog number) harboring abnormal Xi-chromosomes. XIST RNA probes are shown in green and DAPI counterstaining in magenta. Bar: 5&nbsp;μm. n denotes the number of cells used for quantification in (D) from two replicates.</div><div id="fspara0495" role="paragraph">(D) Graph showing the average number of XIST foci in each human cell line from (C) (y axis), ordered by the length of the abnormal Xi-chromosome in Mb (x axis). Dots denote the median, bars the 95% confidence interval. <i>p-value</i>s were derived from a MWW test.</div><div id="fspara0500" role="paragraph">(E) RNA FISH with Xist (green) and intron 1 of <i>Xist</i> (magenta) probes to detect nascent transcripts of <i>Xist</i> at D2 and D4. Chromatin is counterstained with DAPI (gray). The images on the right show only the intron 1 and DAPI signals. Bar: 5&nbsp;μm.</div><div id="fspara0505" role="paragraph">(F) Bar graphs showing the percentages of cells exhibiting a signal for the nascent transcript of <i>Xist</i> under the <i>Xist</i> signal-filled nuclear regions (i.e., the pre-Xi/Xi) from the experiment in (E). Error bars denote the standard deviation. n denotes the number of cells analyzed from two replicates.</div><div id="fspara0510" role="paragraph">(G) 3D-SIM projections showing visualization of cages<sup>60GFP</sup> (gray) expressed in the cytosol and nucleus (demarcated by a dashed pink line) at D2. Cells were transfected with plasmids expressing cages<sup>60GFP</sup> 24&nbsp;h prior to fixation. Inset shows a magnification of the framed region including cytosolic (Cy) and nuclear (Nu) cages<sup>60GFP</sup>.</div><div id="fspara0515" role="paragraph">(H) Violin plots showing comparison of the integrated density of fluorescence (AFU) of nuclear and cytoplasmic cages from (G). Dots denote the median, thick black bars the interquartile range, thin bars show upper/lower values. <i>n</i> denotes the number of cells analyzed from two replicates. MWW <i>p-value</i> is given. NS: Non-Significant.</div><div id="fspara0520" role="paragraph">(I) 3D-SIM projections showing visualization of <i>Xist</i><sup>MS2-GFP</sup> signals (green) at D2 and D4 after addition of 1&nbsp;μg/mL doxycycline for 2&nbsp;h to induce MCP-GFP expression. Cells were transfected with plasmids expressing cages<sup>60GFP</sup> (green) 24&nbsp;h prior to fixation. Chromatin is counterstained with DAPI (gray). Bar: 5&nbsp;μm.</div><div id="fspara0525" role="paragraph">(J) Comparison of integrated density of fluorescence of <i>Xist</i> foci at D2 and D4 and cage<sup>60GFP</sup> from (I) after image segmentation. Dots denote the median, thick black bars the interquartile range, thin bars show upper/lower values. Data are also shown in histogram in <a id="crosref2035" href="#fig1">Figure&nbsp;1</a>O. The calculated integrated density of cages was 64577 ± 14467, <i>Xist</i> at D2: 59413 ± 19555, <i>Xist</i> at D4: 74174 ± 9933 AFU. The <i>p-value</i>s from the MWW test were non-significant (NS).</div><div id="fspara0530" role="paragraph">(K) Quantification of the percentages of chromatin density classes under <i>Xist</i> masks. Error bars denote the standard deviation. n is the number of <i>Xist</i> foci analyzed followed by the number of cells from three experiments.</div><div id="fspara0535" role="paragraph">(L) Graph indicating the transition between chromatin densities. For each H2B-Halo<sup>JF646</sup> pixel, the intensity of all neighboring pixels was determined and plotted across the 7 density classes (y axis). The data show that the neighbors (placed in the bin of their associated chromatin density and color-coded by the chromatin class of the pixel of origin) is found in the same or the directly incrementing density class to the pixel of origin.</div><div id="fspara0540" role="paragraph">(M) Overlap of <i>Xist</i> RAP-seq peaks on the X chromosome at D2 and D4.</div><div id="fspara0545" role="paragraph">(N) Metaplot of the <i>Xist</i> enrichment at D2 and D4 at the genomic locations of D2 <i>Xist</i> peak summits. Similar results were obtained for different cutoffs for peak calling.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">To estimate the number of <i>Xist</i> molecules per focus, we transiently expressed nanocages consisting of 60 GFP molecules (cage<sup>60GFP</sup>) (<span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Hsia et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.HsiaJ.B.BaleS.GonenD.ShiW.ShefflerK.K.FongU.NattermannC.XuP.S.HuangR.RavichandranDesign+of+a+hyperstable+60-subunit+protein+dodecahedron.+%5Bcorrected%5DNature5352016136139" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as internal fluorescence standards in <i>Xist</i><sup>MS2-GFP</sup> cells and confirmed similar intensity profiles of cages<sup>60GFP</sup> in the nucleus and cytoplasm (<a id="crosref0460" href="#figs2">Figures S2</a>G and S2H). The integrated density of one <i>Xist</i><sup>MS2-GFP</sup> focus on the pre-Xi and Xi corresponds to that of one cage<sup>60GFP</sup> (<a id="crosref0465" href="#fig1">Figures 1</a>O, <a id="crosref0470" href="#figs2">S2</a>I, and S2J). Since ∼30 MCP-GFP molecules bind to 24 MS2-repeats (<span class="dropBlock reference-citations"><a id="body-ref-sref114" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114" aria-controls="sref114">Wu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref114"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Wu, B. ∙ Chao, J.A. ∙ Singer, R.H.</div><div><strong>Fluorescence fluctuation spectroscopy enables quantitative imaging of single mRNAs in living cells</strong></div><div><em>Biophys. J.</em> 2012; <strong>102</strong>:2936-2944</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.WuJ.A.ChaoR.H.SingerFluorescence+fluctuation+spectroscopy+enables+quantitative+imaging+of+single+mRNAs+in+living+cellsBiophys.+J.102201229362944" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we infer that each focus contains ∼2 <i>Xist</i> molecules. This result is consistent with measured levels of <i>Xist</i> RNA in single differentiating ESCs (<span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Pacini et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Pacini, G. ∙ Dunkel, I. ∙ Mages, N. ...</div><div><strong>Integrated analysis of Xist upregulation and X-chromosome inactivation with single-cell and single-allele resolution</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:3638</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.PaciniI.DunkelN.MagesV.MutzelB.TimmermannA.MarsicoE.G.SchulzIntegrated+analysis+of+Xist+upregulation+and+X-chromosome+inactivation+with+single-cell+and+single-allele+resolutionNat.+Commun.1220213638" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and estimated numbers of <i>Xist</i> molecules in differentiated cells (<span class="dropBlock reference-citations"><a id="body-ref-sref99-2" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-2" aria-controls="sref99-2">Sunwoo et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref99-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Sunwoo, H. ∙ Wu, J.Y. ∙ Lee, J.T.</div><div><strong>The Xist RNA-PRC2 complex at 20-nm resolution reveals a low Xist stoichiometry and suggests a hit-and-run mechanism in mouse cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4216-E4225</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.SunwooJ.Y.WuJ.T.LeeThe+Xist+RNA-PRC2+complex+at+20-nm+resolution+reveals+a+low+Xist+stoichiometry+and+suggests+a+hit-and-run+mechanism+in+mouse+cellsProc.+Natl.+Acad.+Sci.+USA1122015E4216E4225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, only ∼100 <i>Xist</i> molecules orchestrate the initiation and maintenance of XCI.</div></section><section id="sec-2-3"><h3><i>Xist</i> foci are locally confined and form at open chromatin regions</h3><div id="p0090" role="paragraph">Hence, how the limited number of <i>Xist</i> foci can silence the ∼1000 X-linked genes remains a puzzle. One possibility is that they regulate target genes via rapid diffusion and transient contacts. To investigate the mobility of <i>Xist</i> foci, we developed conditions for live-cell 3D-SIM of <i>Xist</i><sup>MS2-GFP</sup> at D2 and D4. Single-particle tracking for at least 2&nbsp;min (<a id="crosref0490" href="#mmc4">Videos S1</a> and <a id="crosref0495" href="#mmc5">S2</a>) showed that <i>Xist</i> foci exhibit restricted motion without fission or fusion (<a id="crosref0500" href="#fig2">Figures 2</a>A and 2B). In 90% of cases, the displacement of each <i>Xist</i> focus over time was less than 200&nbsp;nm and foci movement was characterized by diffusion in a local confining potential (<a id="crosref0505" href="#fig2">Figures 2</a>C–2E). The confined motion of <i>Xist</i> foci is highly correlated with the motion of chromatin loci (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Chen et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Chen, B. ∙ Gilbert, L.A. ∙ Cimini, B.A. ...</div><div><strong>Dynamic imaging of genomic loci in living human cells by an optimized CRISPR/Cas system</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1479-1491</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.ChenL.A.GilbertB.A.CiminiJ.SchnitzbauerW.ZhangG.W.LiJ.ParkE.H.BlackburnJ.S.WeissmanL.S.QiB.HuangDynamic+imaging+of+genomic+loci+in+living+human+cells+by+an+optimized+CRISPR%2FCas+systemCell155201314791491" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Nozaki et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Nozaki, T. ∙ Imai, R. ∙ Tanbo, M. ...</div><div><strong>Dynamic Organization of Chromatin Domains Revealed by Super-Resolution Live-Cell Imaging</strong></div><div><em>Mol. Cell.</em> 2017; <strong>67</strong>:282-293.e7</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.NozakiR.ImaiM.TanboR.NagashimaS.TamuraT.TaniY.JotiM.TomitaK.HibinoM.T.KanemakiDynamic+Organization+of+Chromatin+Domains+Revealed+by+Super-Resolution+Live-Cell+ImagingMol.+Cell672017282293.e7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We conclude that <i>Xist</i> foci are tethered to chromatin with high affinity, constraining their movement to the local Brownian motion of chromatin. Thus, XCI is initiated through ∼50 sites at which <i>Xist</i> molecules are locally confined.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/75a8691d-5579-4491-920b-3aa78abdd669/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.10.022/asset/c4ef57c8-83d0-42f5-93e1-cee376cf8414/main.assets/gr2.jpg" height="436" width="752" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/75a8691d-5579-4491-920b-3aa78abdd669/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text"><i>Xist</i> foci are locally confined at open chromatin regions</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0095" role="paragraph">(A) Trajectories of <i>Xist</i><sup>MS2-GFP</sup> foci from live-cell 3D-SIM imaging for 2&nbsp;min (5 sec/frame) at D4. Inset: projection of one frame showing a <i>Xist</i><sup>MS2-GFP</sup> cluster.</div><div id="fspara0100" role="paragraph">(B) Selected trajectories from (A) showing the displacement of <i>Xist</i> foci over time (color-gradient) in top (xyz, top) and side (zyx, bottom) views.</div><div id="fspara0105" role="paragraph">(C) D4 <i>Xist</i> foci displacements derived from ∼100 trajectories, each centered about their centers of mass. n denotes the number of foci analyzed from four experiments.</div><div id="fspara0110" role="paragraph">(D) Cumulative distribution function <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Phi left parenthesis r right parenthesis" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-children="0,4" data-semantic-content="6,0" data-semantic-owns="0 6 4" data-semantic-speech="upper Phi left parenthesis r right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7" data-semantic-operator="appl"><mjx-c class="mjx-c1D6F7">𝛷</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="7" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-math></mjx-container>of the number of displacement positions at D4 with distance from origin&nbsp;&lt; <i>r.</i> The distance marked with the dashed line at <i>r</i>&nbsp;= 0.22&nbsp;μm corresponds to the radius of the shaded sphere in (C) where ∼80% of all distances lie (<a id="crosref2005" href="#mmc3">Methods S1</a>). n denotes the number of foci analyzed from ∼800 trajectories from four experiments.</div><div id="fspara0115" role="paragraph">(E) Effective spherically symmetric confining potential inferred from the spatial distribution of displacement distances of <i>Xist</i> foci at D2 and D4. We assume an equilibrium Boltzmann distribution over an effective potential energy well that is a function of <i>r</i>.</div><div id="fspara0120" role="paragraph">(F) Image sequence from <i>t</i>&nbsp;= 0 to <i>t</i>&nbsp;= 28sec and <i>Z-</i>projection (from <i>t</i>&nbsp;= 0) of <i>Xist</i><sup>MS2-GFP</sup> (green) and H2B-Halo<sup>JF646</sup> (magenta) from live-cell 3D-SIM.</div><div id="fspara0125" role="paragraph">(G) Segmentation of H2B-Halo<sup>JF646</sup> from live-cell 3D-SIM data into seven density classes with overlay of <i>Xist</i> masks.</div><div id="fspara0130" role="paragraph">(H) Left: schematic for the assessment of the chromatin landscape around one <i>Xist</i> focus. Mask (bright pink) of one <i>Xist</i> focus showing radial distances denoted by circles from its centroid (dark green). Inset: magnification showing the outline of the <i>Xist</i> mask. Right: plot of <i>Xist</i> foci trajectories showing the average radial maxima of chromatin density reached at indicated time points. Light shaded areas show 95% confidence interval. n denotes the number of foci and cells analyzed from three experiments.</div><div id="fspara0135" role="paragraph">(I) Correlation of <i>Xist</i> enrichment determined by RAP-seq at D2 and D4 to the first principal component of ESC (D0) and D4 Hi-C data (A-compartment&nbsp;= positive values, B-compartment&nbsp;= negative values). Far right panel shows the correlation between D2 and D4 <i>Xist</i> RAP-seq data. Pearson correlation r coefficients and associated p values are given.</div><div id="fspara0140" role="paragraph">(J) <i>Xist</i> enrichment along the X chromosome, defined based on RAP-seq data for <i>Xist</i> over the input, at D2 and D4, with peak calls below. The <i>Xist</i> locus is indicated.</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph"><div id="mmc4" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc4" data-type="mp4"><span id="video_stream_mmc4_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlZTRjYzdiYjQ1YTMwZjViMTdjMGExYzJkNWRmMDdkNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.l1AJKf0HXwNAzAJ5lufX2Imku6RFIJ4fhqzWNTtam6PTW7sqybpRIqkelj4OVVXVP2gH7v7iwP5SwZFsdd6fHx6WLeJJ9wPxTUG7XmuzG2GFid_6W-doSLXc_yMskNCvzOK6NmkxsULBhMJMVSJ9Z42ur4eh9iLzaXTT8HV1GooU8HYXOm8-RRQYf1X02EQPB-C5kjhPa4MazZpOwCxrGOr55f2ez7XN3x5XC2QbqppoJ1_6L1_l01j7LPIlTih_wr1JPCQJz1tzE9E5fmCXkquTNt3r2kEEhN3f1uea1r3o43k4GxiZU-VVMFfS-pNFJEM3YZSnRDTg50kRa9E87g</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlZTRjYzdiYjQ1YTMwZjViMTdjMGExYzJkNWRmMDdkNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.l1AJKf0HXwNAzAJ5lufX2Imku6RFIJ4fhqzWNTtam6PTW7sqybpRIqkelj4OVVXVP2gH7v7iwP5SwZFsdd6fHx6WLeJJ9wPxTUG7XmuzG2GFid_6W-doSLXc_yMskNCvzOK6NmkxsULBhMJMVSJ9Z42ur4eh9iLzaXTT8HV1GooU8HYXOm8-RRQYf1X02EQPB-C5kjhPa4MazZpOwCxrGOr55f2ez7XN3x5XC2QbqppoJ1_6L1_l01j7LPIlTih_wr1JPCQJz1tzE9E5fmCXkquTNt3r2kEEhN3f1uea1r3o43k4GxiZU-VVMFfS-pNFJEM3YZSnRDTg50kRa9E87g" class="cloudflare-stream-player ready tracked" id="video_stream_mmc4" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 100%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlZTRjYzdiYjQ1YTMwZjViMTdjMGExYzJkNWRmMDdkNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.l1AJKf0HXwNAzAJ5lufX2Imku6RFIJ4fhqzWNTtam6PTW7sqybpRIqkelj4OVVXVP2gH7v7iwP5SwZFsdd6fHx6WLeJJ9wPxTUG7XmuzG2GFid_6W-doSLXc_yMskNCvzOK6NmkxsULBhMJMVSJ9Z42ur4eh9iLzaXTT8HV1GooU8HYXOm8-RRQYf1X02EQPB-C5kjhPa4MazZpOwCxrGOr55f2ez7XN3x5XC2QbqppoJ1_6L1_l01j7LPIlTih_wr1JPCQJz1tzE9E5fmCXkquTNt3r2kEEhN3f1uea1r3o43k4GxiZU-VVMFfS-pNFJEM3YZSnRDTg50kRa9E87g" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.10.022/attachment/8dddff02-d38b-4bcc-a4b4-62cde7c93386/mmc4.mp4" download="mmc4.mp4" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Video (105.56 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Video S1. Xist foci are locally confined within a ∼250 nm radius, related to Figure&nbsp;2</div><div id="espara0010" role="paragraph">Live-cell 3D-SIM imaging of <i>Xist</i><sup>MS2-GFP</sup> signals at D4 of differentiation. Projections of optical stacks acquired through time are shown.</div></div></div></div><div id="p0100" role="paragraph"><div id="mmc5" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc5" data-type="mp4"><span id="video_stream_mmc5_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMTNjMjcxNWNiZGVkYzQ4ODk2NTBiMTFlNWYyNjUyYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.BSOsNhlJhMx9Nv-fnsHIxp1dNybk106ZosaDyEybiu-sUc4su0HbOFpqLfw_lJu_UQ8em6loRYsZWNv7Q9hYupshwiYvVVQ4SpKA6AxS0BjlYnz3H_CUpkvuJk9ZdtvWCizYolrE5AL00H1DI3GkYbfSYD_chT9mK__LmvYoWspWXXHN9v92gz5iy1vQH6Lz1sc0lBwWcnt9yu5gs0CdoIVwtbiZAe5CRNk3sNuAWDtKKf3ZyY2S3P1EpeEnZgmRXe3NO28_ylieoqrvpAJFIFBqGZmucDLQgqEbXa7A62KLoRkIzAEL04shiSKeqiBeLWh143MO9r0NOGAckklpuA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMTNjMjcxNWNiZGVkYzQ4ODk2NTBiMTFlNWYyNjUyYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.BSOsNhlJhMx9Nv-fnsHIxp1dNybk106ZosaDyEybiu-sUc4su0HbOFpqLfw_lJu_UQ8em6loRYsZWNv7Q9hYupshwiYvVVQ4SpKA6AxS0BjlYnz3H_CUpkvuJk9ZdtvWCizYolrE5AL00H1DI3GkYbfSYD_chT9mK__LmvYoWspWXXHN9v92gz5iy1vQH6Lz1sc0lBwWcnt9yu5gs0CdoIVwtbiZAe5CRNk3sNuAWDtKKf3ZyY2S3P1EpeEnZgmRXe3NO28_ylieoqrvpAJFIFBqGZmucDLQgqEbXa7A62KLoRkIzAEL04shiSKeqiBeLWh143MO9r0NOGAckklpuA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc5" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 100%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMTNjMjcxNWNiZGVkYzQ4ODk2NTBiMTFlNWYyNjUyYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.BSOsNhlJhMx9Nv-fnsHIxp1dNybk106ZosaDyEybiu-sUc4su0HbOFpqLfw_lJu_UQ8em6loRYsZWNv7Q9hYupshwiYvVVQ4SpKA6AxS0BjlYnz3H_CUpkvuJk9ZdtvWCizYolrE5AL00H1DI3GkYbfSYD_chT9mK__LmvYoWspWXXHN9v92gz5iy1vQH6Lz1sc0lBwWcnt9yu5gs0CdoIVwtbiZAe5CRNk3sNuAWDtKKf3ZyY2S3P1EpeEnZgmRXe3NO28_ylieoqrvpAJFIFBqGZmucDLQgqEbXa7A62KLoRkIzAEL04shiSKeqiBeLWh143MO9r0NOGAckklpuA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.10.022/attachment/e227ca3b-2a23-49a7-b6ea-a51341cb425a/mmc5.mp4" download="mmc5.mp4" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Video (15.56 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Video S2. Xist foci are persistent through time, related to Figure&nbsp;2</div><div id="espara0015" role="paragraph"><i>Xist</i> foci from Video S1 after magnification with bilinear pixel size interpolation for clarity.</div></div></div></div><div id="p0105" role="paragraph">To investigate if the “wiggling” of <i>Xist</i> foci around their centers occurs within a specific chromatin environment, we introduced a histone H2B-Halo transgene into <i>Xist</i><sup>MS2-GFP</sup> ESCs and performed live-cell 3D-SIM (<a id="crosref0520" href="#fig2">Figure&nbsp;2</a>F; <a id="crosref0525" href="#mmc6">Video S3</a>). H2B signals were segmented into intensity levels that correspond to chromatin density classes, with class 1 representing DNA-free interchromatin channels (ICs) and classes 2 to 7 capturing increasing chromatin densities (<span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Markaki et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Markaki, Y. ∙ Smeets, D. ∙ Fiedler, S. ...</div><div><strong>The potential of 3D-FISH and super-resolution structured illumination microscopy for studies of 3D nuclear architecture: 3D structured illumination microscopy of defined chromosomal structures visualized by 3D (immuno)-FISH opens new perspectives for studies of nuclear architecture</strong></div><div><em>BioEssays.</em> 2012; <strong>34</strong>:412-426</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.MarkakiD.SmeetsS.FiedlerV.J.SchmidL.SchermellehT.CremerM.CremerThe+potential+of+3D-FISH+and+super-resolution+structured+illumination+microscopy+for+studies+of+3D+nuclear+architecture%3A+3D+structured+illumination+microscopy+of+defined+chromosomal+structures+visualized+by+3D+%28immuno%29-FISH+opens+new+perspectives+for+studies+of+nuclear+architectureBioEssays342012412426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0535" href="#fig2">Figure&nbsp;2</a>G). <i>Xist</i> foci covered predominantly classes 1 to 3 (<a id="crosref0540" href="#figs2">Figure&nbsp;S2</a>K). Over time, the chromatin densities underlying the footprint of <i>Xist</i> foci never surpassed class 3 and the centroids remained within chromatin class 2, consistent with the linearly and incrementally increasing chromatin density (<a id="crosref0545" href="#fig2">Figures 2</a>H and <a id="crosref0550" href="#figs2">S2</a>L). Thus, <i>Xist</i> foci are spatially confined to the periphery of chromatin domains and stably maintain their positions relative to chromatin.</div><div id="p0110" role="paragraph"><div id="mmc6" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc6" data-type="mp4"><span id="video_stream_mmc6_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5MjdjNDIyZmYxZDgzYjU4Y2U0YzQ0NjY5MjJmMzJiYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.iZeYXeSf4SngaaTKdNXMwTB2YHca5nkvszeoWfJ0e3B6h16si2Y6xhAhWCkXh0ijiyqThUHP2bQ6w795C3iDLPoK7dGfESthxefX0Nd7MExxqoouNg--LsMSxAGmhb4Qwv9y_Aw6FGvNMqdwGQRnA2SNv-Y-g7XwiTyjQrM3Sdaby-glGT5tyApJ37UeBDpXN01trUB7A4VLl9HX8Mv32zuxI0At3PWV5hLjhp-D_c_syNSUWjk0EgAT-qcQjIPTfUiOsHLRcm5-EAKfXl_ZKbahoXYau6aZ1iVUf3D1EbQg0p2Z6WLAaCy-i658uwhFZ_s2zT5FfdTm-gIEUDOLDw</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5MjdjNDIyZmYxZDgzYjU4Y2U0YzQ0NjY5MjJmMzJiYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.iZeYXeSf4SngaaTKdNXMwTB2YHca5nkvszeoWfJ0e3B6h16si2Y6xhAhWCkXh0ijiyqThUHP2bQ6w795C3iDLPoK7dGfESthxefX0Nd7MExxqoouNg--LsMSxAGmhb4Qwv9y_Aw6FGvNMqdwGQRnA2SNv-Y-g7XwiTyjQrM3Sdaby-glGT5tyApJ37UeBDpXN01trUB7A4VLl9HX8Mv32zuxI0At3PWV5hLjhp-D_c_syNSUWjk0EgAT-qcQjIPTfUiOsHLRcm5-EAKfXl_ZKbahoXYau6aZ1iVUf3D1EbQg0p2Z6WLAaCy-i658uwhFZ_s2zT5FfdTm-gIEUDOLDw" class="cloudflare-stream-player ready tracked" id="video_stream_mmc6" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 98.3603%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5MjdjNDIyZmYxZDgzYjU4Y2U0YzQ0NjY5MjJmMzJiYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.iZeYXeSf4SngaaTKdNXMwTB2YHca5nkvszeoWfJ0e3B6h16si2Y6xhAhWCkXh0ijiyqThUHP2bQ6w795C3iDLPoK7dGfESthxefX0Nd7MExxqoouNg--LsMSxAGmhb4Qwv9y_Aw6FGvNMqdwGQRnA2SNv-Y-g7XwiTyjQrM3Sdaby-glGT5tyApJ37UeBDpXN01trUB7A4VLl9HX8Mv32zuxI0At3PWV5hLjhp-D_c_syNSUWjk0EgAT-qcQjIPTfUiOsHLRcm5-EAKfXl_ZKbahoXYau6aZ1iVUf3D1EbQg0p2Z6WLAaCy-i658uwhFZ_s2zT5FfdTm-gIEUDOLDw" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.10.022/attachment/0ecfc6d4-085e-4611-9d27-c8b5d97fe740/mmc6.mp4" download="mmc6.mp4" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Video (2.34 MB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Video S3. Xist foci are locally confined at chromatin borders facing the IC space, related to Figure&nbsp;2</div><div id="espara0020" role="paragraph">Live-cell 3D-SIM imaging of <i>Xist</i><sup>MS2-GFP</sup> (green) and H2B-Halo<sup>JF647</sup> (magenta) at D4 of differentiation. Projections of optical stacks acquired through time are shown.</div></div></div></div><div id="p0115" role="paragraph">In agreement with these observations, RNA antisense purification (RAP) of <i>Xist</i> followed by DNA sequencing of associated chromatin (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Engreitz et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Engreitz, J.M. ∙ Pandya-Jones, A. ∙ McDonel, P. ...</div><div><strong>The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome</strong></div><div><em>Science.</em> 2013; <strong>341</strong>:1237973</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.EngreitzA.Pandya-JonesP.McDonelA.ShishkinK.SirokmanC.SurkaS.KadriJ.XingA.GorenE.S.LanderThe+Xist+lncRNA+exploits+three-dimensional+genome+architecture+to+spread+across+the+X+chromosomeScience34120131237973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) showed that <i>Xist</i> localizes to gene-rich, open chromatin regions of the A-compartment (<a id="crosref0560" href="#fig2">Figure&nbsp;2</a>I). We identified 65 and 63 highly overlapping peaks of <i>Xist</i> enrichment on the pre-Xi and Xi, respectively (<a id="crosref0565" href="#fig2">Figures 2</a>J and <a id="crosref0570" href="#figs2">S2</a>M), similar to the number of foci detected by 3D-SIM. These peaks cover broad genomic regions of 1–5 megabases (<a id="crosref0575" href="#figs2">Figure&nbsp;S2</a>N), indicating variability in <i>Xist</i> foci locations between cells. <i>Xist</i> peaks in the Xi are broader than those in the pre-Xi despite the overall similar distributions (Pearson’s correlation r&nbsp;= 0.83) (<a id="crosref0580" href="#fig2">Figures 2</a>I, 2J, and <a id="crosref0585" href="#figs2">S2</a>N), suggesting that the chromatin contacts of <i>Xist</i> foci change over time due to chromatin compaction.</div></section><section id="sec-2-4"><h3><i>Xist</i> nucleates supramolecular complexes</h3><div id="p0120" role="paragraph">To explore how <i>Xist</i> effector proteins accumulate relative to ∼50 locally confined <i>Xist</i> foci, we set out to quantify, at sub-diffraction resolution, their spatial relationship to <i>Xist</i> and to each other. We initially focused on SPEN, PCGF5, CELF1, and CIZ1, four proteins that bind to distinct repeat sequences of <i>Xist</i> and have different roles in XCI (<a id="crosref0590" href="#fig3">Figure&nbsp;3</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Loda and Heard, 2019</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Loda, A. ∙ Heard, E.</div><div><strong>Xist RNA in action: Past, present, and future</strong></div><div><em>PLoS Genet.</em> 2019; <strong>15</strong>:e1008333</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.LodaE.HeardXist+RNA+in+action%3A+Past%2C+present%2C+and+futurePLoS+Genet.152019e1008333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SPEN binds the A-repeat of <i>Xist</i> and activates HDAC3 on chromatin to induce gene silencing (<span class="dropBlock reference-citations"><a id="body-ref-sref22-3" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-3" aria-controls="sref22-3">Chu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref22-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Chu, C. ∙ Zhang, Q.C. ∙ da Rocha, S.T. ...</div><div><strong>Systematic discovery of Xist RNA binding proteins</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:404-416</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.ChuQ.C.ZhangS.T.da+RochaR.A.FlynnM.BharadwajJ.M.CalabreseT.MagnusonE.HeardH.Y.ChangSystematic+discovery+of+Xist+RNA+binding+proteinsCell1612015404416" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63-3" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-3" aria-controls="sref63-3">McHugh et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref63-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>McHugh, C.A. ∙ Chen, C.K. ∙ Chow, A. ...</div><div><strong>The Xist lncRNA interacts directly with SHARP to silence transcription through HDAC3</strong></div><div><em>Nature.</em> 2015; <strong>521</strong>:232-236</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.A.McHughC.K.ChenA.ChowC.F.SurkaC.TranP.McDonelA.Pandya-JonesM.BlancoC.BurghardA.MoradianThe+Xist+lncRNA+interacts+directly+with+SHARP+to+silence+transcription+through+HDAC3Nature5212015232236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The non-canonical PRC1 subunit PCGF5 is recruited to the Xi via the B-repeat and supports silencing of some X-linked genes (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Almeida et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Almeida, M. ∙ Pintacuda, G. ∙ Masui, O. ...</div><div><strong>PCGF3/5-PRC1 initiates Polycomb recruitment in X chromosome inactivation</strong></div><div><em>Science.</em> 2017; <strong>356</strong>:1081-1084</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.AlmeidaG.PintacudaO.MasuiY.KosekiM.GdulaA.CeraseD.BrownA.MouldC.InnocentM.NakayamaPCGF3%2F5-PRC1+initiates+Polycomb+recruitment+in+X+chromosome+inactivationScience356201710811084" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref9-1" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-1" aria-controls="sref9-1">Bousard et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Bousard, A. ∙ Raposo, A.C. ∙ Żylicz, J.J. ...</div><div><strong>The role of Xist-mediated Polycomb recruitment in the initiation of X-chromosome inactivation</strong></div><div><em>EMBO Rep.</em> 2019; <strong>20</strong>:e48019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.BousardA.C.RaposoJ.J.%C5%BByliczC.PicardV.B.PiresY.QiC.GilL.SyxH.Y.ChangE.HeardS.T.da+RochaThe+role+of+Xist-mediated+Polycomb+recruitment+in+the+initiation+of+X-chromosome+inactivationEMBO+Rep.202019e48019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-2" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-2" aria-controls="sref74-2">Nesterova et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref74-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Nesterova, T.B. ∙ Wei, G. ∙ Coker, H. ...</div><div><strong>Systematic allelic analysis defines the interplay of key pathways in X chromosome inactivation</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:3129</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.B.NesterovaG.WeiH.CokerG.PintacudaJ.S.BownessT.ZhangM.AlmeidaB.BloechlB.MoindrotE.J.CarterSystematic+allelic+analysis+defines+the+interplay+of+key+pathways+in+X+chromosome+inactivationNat.+Commun.1020193129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-2" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-2" aria-controls="sref81-2">Pintacuda et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref81-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Pintacuda, G. ∙ Wei, G. ∙ Roustan, C. ...</div><div><strong>hnRNPK Recruits PCGF3/5-PRC1 to the Xist RNA B-Repeat to Establish Polycomb-Mediated Chromosomal Silencing</strong></div><div><em>Mol. Cell.</em> 2017; <strong>68</strong>:955-969.e10</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.PintacudaG.WeiC.RoustanB.A.KirmizitasN.SolcanA.CeraseA.CastelloS.MohammedB.MoindrotT.B.NesterovahnRNPK+Recruits+PCGF3%2F5-PRC1+to+the+Xist+RNA+B-Repeat+to+Establish+Polycomb-Mediated+Chromosomal+SilencingMol.+Cell682017955969.e10" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CELF1 and CIZ1 bind to the E-repeat and restrict <i>Xist</i> to the Xi (<span class="dropBlock reference-citations"><a id="body-ref-sref79-1" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-1" aria-controls="sref79-1">Pandya-Jones et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref79-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Pandya-Jones, A. ∙ Markaki, Y. ∙ Serizay, J. ...</div><div><strong>A protein assembly mediates Xist localization and gene silencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:145-151</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Pandya-JonesY.MarkakiJ.SerizayT.ChitiashviliW.R.Mancia+LeonA.DamianovC.ChronisB.PappC.K.ChenR.McKeeA+protein+assembly+mediates+Xist+localization+and+gene+silencingNature5872020145151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Ridings-Figueroa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Ridings-Figueroa, R. ∙ Stewart, E.R. ∙ Nesterova, T.B. ...</div><div><strong>The nuclear matrix protein CIZ1 facilitates localization of Xist RNA to the inactive X-chromosome territory</strong></div><div><em>Genes Dev.</em> 2017; <strong>31</strong>:876-888</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.Ridings-FigueroaE.R.StewartT.B.NesterovaH.CokerG.PintacudaJ.GodwinR.WilsonA.HaslamF.LilleyR.RuigrokThe+nuclear+matrix+protein+CIZ1+facilitates+localization+of+Xist+RNA+to+the+inactive+X-chromosome+territoryGenes+Dev.312017876888" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Sunwoo et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Sunwoo, H. ∙ Colognori, D. ∙ Froberg, J.E. ...</div><div><strong>Repeat E anchors Xist RNA to the inactive X chromosomal compartment through&nbsp;CDKN1A-interacting protein (CIZ1)</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:10654-10659</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.SunwooD.ColognoriJ.E.FrobergY.JeonJ.T.LeeRepeat+E+anchors+Xist+RNA+to+the+inactive+X+chromosomal+compartment+through%C2%A0CDKN1A-interacting+protein+%28CIZ1%29Proc.+Natl.+Acad.+Sci.+USA11420171065410659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119" aria-controls="sref119">Yue et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref119"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Yue, M. ∙ Ogawa, A. ∙ Yamada, N. ...</div><div><strong>Xist RNA repeat E is essential for ASH2L recruitment to the inactive X&nbsp;and regulates histone modifications and escape gene expression</strong></div><div><em>PLoS Genet.</em> 2017; <strong>13</strong>:e1006890</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.YueA.OgawaN.YamadaJ.L.Charles+RichardA.BarskiY.OgawaXist+RNA+repeat+E+is+essential+for+ASH2L+recruitment+to+the+inactive+X%C2%A0and+regulates+histone+modifications+and+escape+gene+expressionPLoS+Genet.132017e1006890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/63a6f9b3-acef-449e-ba18-552cd83fd9b4/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.10.022/asset/d11d2385-8963-44e4-b02c-7d96d756b4ae/main.assets/gr3.jpg" height="770" width="750" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/63a6f9b3-acef-449e-ba18-552cd83fd9b4/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text"><i>Xist</i> nucleates supramolecular complexes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0150" role="paragraph">(A) Schematic of <i>Xist</i> RNA with its repeat sequences (A–F), different repeat-binding proteins, and repeat functions. Proteins examined in (B) are indicated.</div><div id="fspara0155" role="paragraph">(B) 125&nbsp;nm 3D-SIM optical sections showing detection of indicated Halo protein-fusions labeled with JF549 (yellow) and immunodetected proteins (magenta) in <i>Xist</i><sup>MS2-GFP</sup> cells at D2 and D4. Top panels show the <i>Xist</i>-demarcated territory (pre-Xi/Xi); bottom panels a nuclear region (Nuclear). Note the distinctive enrichment of all pairs of interactors around <i>Xist</i> foci.</div><div id="fspara0160" role="paragraph">(C) Overview of inter-protein particle distance measurements for protein foci associated with one <i>Xist</i> focus, based on data in (B). Overlay: <i>Xist</i> (green), SPEN (yellow), and CIZ1 (magenta). Bottom right panel shows mask outlines after image segmentation, depiction of protein and <i>Xist</i> foci centroids (crosses) and measurement of inter-particle distances performed in (D). Circles denote a 200&nbsp;nm radius.</div><div id="fspara0165" role="paragraph">(D) Boxplots from data in (B) showing the nearest-neighbor distances between the indicated pairs of protein particles in nuclear and <i>Xist</i>-associated fractions obtained as shown in (C) for D2 and D4. n denotes the number of protein particles followed by the number of cells analyzed. MWW p values are given.</div><div id="fspara0170" role="paragraph">(E) Boxplots of the distribution of the density of indicated protein particles (number of particles per μm<sup>3</sup>) in the Xi and in nuclear regions on D2 and D4. n denotes the number of cells analyzed. MWW p values are given.</div><div id="fspara0175" role="paragraph">(F) Point-plots showing the average ratio of the number of indicated protein particles per <i>Xist</i> particle within 250&nbsp;nm radial search (left) and their nearest distance (right) on D2 and D4. The bars denote the standard deviation, and n the number of particles followed by number of cells analyzed.</div><div id="fspara0180" role="paragraph">(G) Schematic of a <i>Xist</i>-supramolecular complex.</div><div id="fspara0185" role="paragraph">(H) Point-plots from data in (F) showing the integrated density of fluorescence of indicated protein particles in <i>Xist</i>-associated (green) compared to nuclear (magenta) fractions, on D2 and D4 from two experiments. Dots denote the median, bars the standard deviation. Dotted lines are included to visualize changes. Data are normalized to the highest signal observed across the entire population of each protein. Absolute values are shown in <a id="crosref2010" href="#figs4">Figure&nbsp;S4</a>E. MWW p values are given.</div><div id="fspara0190" role="paragraph">(I) Projection of a nucleus imaged with 3D-SIM expressing SPEN-GFP from the endogenous locus (magenta) and <i>Xist</i>-Bgl-mCherry (green) at 18&nbsp;h post tetO-<i>Xist</i> induction. Inset shows SPEN signals in the Xi.</div><div id="fspara0195" role="paragraph">(J) Boxplots showing integrated densities of cages<sup>60GFP</sup>, <i>Xist</i>-associated SPEN at 6 or 18&nbsp;h after tetO-<i>Xist</i> induction. n denotes the number of cells analyzed from two replicates.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">To interrogate the localization of <i>Xist</i>-interactors, we introduced Halo-tagged transgenes into <i>Xist</i><sup>MS2-GFP</sup> cells, allowing imaging of an antibody-stained and a stably expressed Halo-fusion protein together with <i>Xist</i> by multispectral 3D-SIM. We observed the formation of diffraction-limited protein assemblies in proximity to <i>Xist</i> foci in both the pre-Xi and Xi that are larger than nuclear accumulations (<a id="crosref0660" href="#fig3">Figures 3</a>B, <a id="crosref0665" href="#figs3">S3</a>A, and S3B). To quantitatively define these distributions, we extracted nucleus-wide spatial coordinates of thousands of segmented diffraction-limited protein particles (<a id="crosref0670" href="#figs3">Figures S3</a>C–S3E). We paired protein to <i>Xist</i> foci and measured nearest neighbor distances between pairs of different interactors associated with the same <i>Xist</i> focus (<a id="crosref0675" href="#fig3">Figure&nbsp;3</a>C) or protein pairs located in the remainder of the nucleus. For all pairs, protein foci are on average within ∼150–200&nbsp;nm of each other when associated with <i>Xist</i>, both on the pre-Xi and Xi, and are separated by &gt;350&nbsp;nm in the rest of the nucleus (<a id="crosref0680" href="#fig3">Figure&nbsp;3</a>D). Thus, <i>Xist</i> foci induce the <i>de novo</i> formation of unique protein complexes that locally concentrate effector proteins more than elsewhere in the nucleus. The density of SPEN, CELF1, PCGF5, and CIZ1 particles is significantly higher in the pre-Xi and Xi than in the nucleus, consistent with their decreased nearest and average distances in the X-territory (<a id="crosref0685" href="#fig3">Figures 3</a>E, <a id="crosref0690" href="#figs4">S4</a>A, and S4B). Furthermore, their concentration increases from the pre-Xi to the Xi, along with the observed chromatin compaction. Hence, large multi-protein assemblies that are not typically found outside the Xi form around <i>Xist</i> foci. In this way, <i>Xist</i> recruitment increases the concentration of proteins within the forming Xi. We refer to the <i>Xist</i>-nucleated proteinaceous nanostructures as supramolecular complexes (SMACs). We also observed their formation when XCI is ectopically induced on an autosome (<a id="crosref0695" href="#figs3">Figure&nbsp;S3</a>F) consistent with SMACs being a fundamental feature of the XCI process.</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/9895d2ca-0ac2-4e5b-8a3a-3457319f131f/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.10.022/asset/1fa7e093-5e9d-4586-a402-f59235d458c7/main.assets/figs3.jpg" height="940" width="645" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/9895d2ca-0ac2-4e5b-8a3a-3457319f131f/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Identification of <i>Xist</i> SMACs, related to <a id="crosref2040" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0555" role="paragraph">(A) 125nm 3D-SIM optical sections showing the detection of Halo protein-fusions labeled with JF549 (yellow) and immunodetected proteins (magenta) in <i>Xist</i><sup>MS2-GFP</sup> cells (<i>Xis</i>t, green) at D2 and D4. Data are whole nucleus images of regions shown in <a id="crosref2045" href="#fig3">Figure&nbsp;3</a>B.</div><div id="fspara0560" role="paragraph">(B) Comparison of the CIZ1-Halo<sup>JF549</sup> pattern to the localization of the endogenous CIZ1 detected with anti-CIZ1 primary and AlexaFluor647-conjugated secondary antibodies from data in <a id="crosref2050" href="#fig3">Figure&nbsp;3</a>B, showing the same trend in both endogenous and transgenic protein populations after quantitative 3D-SIM analyses. MWW <i>p-value</i>s are given. Thus, endogenous proteins detected by antibody staining or Halo-fused transgenes displayed similar distributions.</div><div id="fspara0565" role="paragraph">(C) Overview of image segmentation pipeline of 3D-SIM data for assignment of objects (particles) for downstream measurements of features.</div><div id="fspara0570" role="paragraph">(D) Outline of image segmentation pipeline on the example of CIZ1 signals, showing the output of pre- and post-filtering steps for assigning protein particles. 3D Gaussian and TopHat kernels are applied for contrast enhancement and a 3D-seeded watershed algorithm is used to obtain segmented particles. Histograms indicate the intensity profiles obtained under the dashed line during the processing steps to achieve determination of individual particle signals.</div><div id="fspara0575" role="paragraph">(E) Pipeline for the creation of masks to include all protein signals quantified in 3D-SIM particles analyses.</div><div id="fspara0580" role="paragraph">(F) 3D-SIM optical sections of immuno-RNA FISH in a male ESC line carrying an autosomal tetO-<i>Xist</i> transgene (on chromosome 11) showing the distribution of <i>Xist</i> RNA (green) and representative <i>Xist</i>-repeat binding proteins (magenta). Greyscale images show individual channels as indicated. Blue arrows show the enrichment of SPEN, RYBP and CIZ1 in the <i>Xist</i>-coated Xi-territory. DAPI counterstaining is shown in gray. Insets show two-times magnifications of the <i>Xist</i>-demarcated regions.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/ba01631a-c92c-467c-94cf-32aeeb53fb29/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.10.022/asset/f15882a8-f37c-47a0-8151-994e0cda80d0/main.assets/figs4.jpg" height="939" width="693" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/ba01631a-c92c-467c-94cf-32aeeb53fb29/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Macromolecular crowding of <i>Xist</i>-associated proteins in SMACs, related to <a id="crosref2055" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0590" role="paragraph">(A) Boxplots showing nuclear (blue) and <i>Xist</i>-associated (orange) average intra-particle distances of indicated XCI proteins at D2 and D4 from data in <a id="crosref2060" href="#fig3">Figure&nbsp;3</a>E.</div><div id="fspara0595" role="paragraph">(B) Same as in (A) showing the nearest neighbor distances.</div><div id="fspara0600" role="paragraph">(C) Left column: 3D-SIM 125nm optical sections of D2 and D4 cells showing <i>Xist</i><sup>MS2-GFP</sup> signals (green) and immunodetected or Halo-fused proteins labeled with primary and secondary CF568-conjugated antibodies or JF549 Halo ligands (magenta) (see methods). DAPI counterstaining is shown in gray. Three right columns: magnifications of the Xi and nuclear regions with or without DAPI.</div><div id="fspara0605" role="paragraph">(D) Point-plots showing the nearest neighbor distances of <i>Xist</i>-associated protein particles to <i>Xist</i> foci (orange) (same data as shown in <a id="crosref2065" href="#fig3">Figure&nbsp;3</a>F) and the nearest neighbor distances of the same number of randomized protein positions to <i>Xist</i> foci (blue), at D2 and D4. MWW p values comparing the experimentally observed and randomized protein distances are given.</div><div id="fspara0610" role="paragraph">(E) Violin plots of integrated densities and volumes of nuclear and <i>Xist</i>-associated protein particles from experiment shown in <a id="crosref2070" href="#fig3">Figures 3</a>H, at D2 and D4. Long-dashed black lines denote the median, short- dashed lines denote the upper/lower quartiles.</div><div id="fspara0615" role="paragraph">(F) Point-plots showing the integrated density of fluorescence of <i>Xist</i>-associated and nuclear SPEN-GFP levels across a time course of <i>Xist</i> induction. Here, doxycycline was added at t&nbsp;= 0&nbsp;h to induce <i>Xist</i> expression from the tetO-<i>Xist</i> allele in B6<sup>tetOXist-Bgl-mCherry</sup>Cas<sup>WTXist</sup> ESCs expressing GFP-tagged SPEN from the endogenous <i>Spen</i> locus. n denotes the number of nuclei analyzed for each time point from two replicates.</div><div id="fspara0620" role="paragraph">(G) Left: Point-plots comparing the integrated density of nuclear and Xist-associated SPEN-Halo particles at D2 and D4 of differentiation. In this experiment, we used B6<sup>tetOXist-Bgl-mCherry</sup>Cas<sup>WTXist</sup> ESCs expressing Halo-tagged SPEN from the endogenous <i>Spen</i> locus. Xist expression was induced from the tetO-Xist allele by addition of doxycycline during differentiation (+dox) or differentiation-induced (-dox). Differentiation-induced <i>Xist</i> SMACs display the increase of SPEN levels from D2 to D4, whereas SPEN levels have already plateaued in doxycycline-induced <i>Xist</i> SMACs, likely due to faster <i>Xist</i> induction. Right: Comparison of the number of <i>Xist</i> foci in both <i>Xist</i> induction conditions at D2 and D4. n denotes the number of cells analyzed from two experiments. MWW p values are given.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-5"><h3>SMACs contain a wide spectrum of <i>Xist</i>-interacting proteins</h3><div id="p0130" role="paragraph">To explore whether integration into SMACs is the main mechanism of protein recruitment in the Xi, we probed the distribution of additional XCI effectors (<a id="crosref0700" href="#figs4">Figure&nbsp;S4</a>C), including the PRC1 subunit RYBP (<span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Tavares et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Tavares, L. ∙ Dimitrova, E. ∙ Oxley, D. ...</div><div><strong>RYBP-PRC1 complexes mediate H2A ubiquitylation at polycomb target sites independently of PRC2 and H3K27me3</strong></div><div><em>Cell.</em> 2012; <strong>148</strong>:664-678</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.TavaresE.DimitrovaD.OxleyJ.WebsterR.PootJ.DemmersK.BezstarostiS.TaylorH.UraH.KoideRYBP-PRC1+complexes+mediate+H2A+ubiquitylation+at+polycomb+target+sites+independently+of+PRC2+and+H3K27me3Cell1482012664678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); the EZH2 subunit of PRC2 (<span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">Plath et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Plath, K. ∙ Fang, J. ∙ Mlynarczyk-Evans, S.K. ...</div><div><strong>Role of histone H3 lysine 27 methylation in X inactivation</strong></div><div><em>Science.</em> 2003; <strong>300</strong>:131-135</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.PlathJ.FangS.K.Mlynarczyk-EvansR.CaoK.A.WorringerH.WangC.C.de+la+CruzA.P.OtteB.PanningY.ZhangRole+of+histone+H3+lysine+27+methylation+in+X+inactivationScience3002003131135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-2" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-2" aria-controls="sref94-2">Silva et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref94-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Silva, J. ∙ Mak, W. ∙ Zvetkova, I. ...</div><div><strong>Establishment of histone h3 methylation on the inactive X chromosome requires transient recruitment of Eed-Enx1 polycomb group complexes</strong></div><div><em>Dev. Cell.</em> 2003; <strong>4</strong>:481-495</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.SilvaW.MakI.ZvetkovaR.AppanahT.B.NesterovaZ.WebsterA.H.PetersT.JenuweinA.P.OtteN.BrockdorffEstablishment+of+histone+h3+methylation+on+the+inactive+X+chromosome+requires+transient+recruitment+of+Eed-Enx1+polycomb+group+complexesDev.+Cell42003481495" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); hnRNP-K, which binds the <i>Xist</i> B-repeat to recruit PCGF5 (<span class="dropBlock reference-citations"><a id="body-ref-sref81-3" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-3" aria-controls="sref81-3">Pintacuda et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref81-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Pintacuda, G. ∙ Wei, G. ∙ Roustan, C. ...</div><div><strong>hnRNPK Recruits PCGF3/5-PRC1 to the Xist RNA B-Repeat to Establish Polycomb-Mediated Chromosomal Silencing</strong></div><div><em>Mol. Cell.</em> 2017; <strong>68</strong>:955-969.e10</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.PintacudaG.WeiC.RoustanB.A.KirmizitasN.SolcanA.CeraseA.CastelloS.MohammedB.MoindrotT.B.NesterovahnRNPK+Recruits+PCGF3%2F5-PRC1+to+the+Xist+RNA+B-Repeat+to+Establish+Polycomb-Mediated+Chromosomal+SilencingMol.+Cell682017955969.e10" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); PTBP1 and MATR3, which regulate <i>Xist</i> localization with CELF1 (<span class="dropBlock reference-citations"><a id="body-ref-sref79-2" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-2" aria-controls="sref79-2">Pandya-Jones et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref79-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Pandya-Jones, A. ∙ Markaki, Y. ∙ Serizay, J. ...</div><div><strong>A protein assembly mediates Xist localization and gene silencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:145-151</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Pandya-JonesY.MarkakiJ.SerizayT.ChitiashviliW.R.Mancia+LeonA.DamianovC.ChronisB.PappC.K.ChenR.McKeeA+protein+assembly+mediates+Xist+localization+and+gene+silencingNature5872020145151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For all examined proteins, we detected a particle associated with a <i>Xist</i> focus in a near 1:1 ratio and nearest neighbor measurements revealed their presence within a 200&nbsp;nm zone from the centroid of <i>Xist</i>, significantly more proximal than randomized protein populations (<a id="crosref0730" href="#fig3">Figures 3</a>F and <a id="crosref0735" href="#figs4">S4</a>D; <a id="crosref0740" href="#mmc1">Table S1</a>). These data corroborate the <i>de novo</i> formation of a multi-protein macromolecular cloud around <i>Xist</i> foci at the onset of XCI (<a id="crosref0745" href="#fig3">Figure&nbsp;3</a>G).</div><div id="p0135" role="paragraph">We next quantified the concentration of proteins in SMACs in relation to nuclear accumulations (<a id="crosref0750" href="#fig3">Figures 3</a>H and <a id="crosref0755" href="#figs4">S4</a>E). Integrated density and particle volume measurements showed that the levels of CIZ1, CELF1, SPEN, PCGF5, EZH2, and RYBP in SMACs significantly exceed those in nuclear assemblies. Protein accumulation in SMACs varies moderately across the pre-Xi to Xi&nbsp;transition. CIZ1 levels remain relatively stable; PCGF5, EZH2, and RYBP levels follow nuclear changes; CELF1 levels decrease; and SPEN levels dramatically increase. Thus, gene silencing is correlated with more SPEN molecules in SMACs. MATR3, PTBP1, and hnRNP-K exhibit baseline concentrations in SMACs, suggesting that their recruitment, rather than increased accumulation is essential in XCI.</div><div id="p0140" role="paragraph">To estimate numbers of protein molecules incorporated into SMACs, we focused on the critical silencing factor SPEN. We exploited our cage<sup>60GFP</sup>-standard approach and a cell line in which endogenously encoded SPEN is GFP-tagged and <i>Xist</i> can be induced with doxycycline (dox) (<span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0765" href="#fig3">Figure&nbsp;3</a>I). Compared to developmentally induced XCI, dox-induction results in a larger number of <i>Xist</i> foci, yet yields similar levels of SPEN in SMACs. Moreover, before plateauing, SPEN-SMAC levels increase between 6 and 18&nbsp;h of dox-addition as seen for the pre-Xi to Xi transition (<a id="crosref0770" href="#figs4">Figures S4</a>F and S4G). This result is consistent with the observation that SPEN increases over the Xi-territory across time upon dox-induction of <i>Xist</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref124-1" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-1" aria-controls="sref124-1">Jachowicz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref124-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Jachowicz, J.W. ∙ Strehle, M. ∙ Banerjee, A.K. ...</div><div><strong>Xist spatially amplifies SHARP recruitment to balance chromosome-wide silencing and specificity to the X chromosome</strong></div><div><em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.W.JachowiczM.StrehleA.K.BanerjeeJ.ThaiM.R.BlancoM.GuttmanXist+spatially+amplifies+SHARP+recruitment+to+balance+chromosome-wide+silencing+and+specificity+to+the+X+chromosomebioRxiv202110.1101%2F2021.10.27.466149" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Comparing the integrated density of SPEN-GFP to that of cage<sup>60GFP</sup>, we infer that there are up to 35 SPEN molecules per SMAC (<a id="crosref0775" href="#fig3">Figure&nbsp;3</a>J). This finding suggests that each complex may consist of hundreds to thousands of protein molecules and that many effector proteins likely are significantly enriched relative to the number of RNA molecules (<a id="crosref0780" href="#fig3">Figure&nbsp;3</a>G).</div></section><section id="sec-2-6"><h3>Binding to <i>Xist</i> alters the kinetic behavior of interacting proteins</h3><div id="p0145" role="paragraph">To explore the kinetic behavior of SMAC protein components, we introduced Halo or mCherry-tagged SPEN, PCGF5, CIZ1, CELF1, or PTBP1 fusions into <i>Xist</i><sup>MS2-GFP</sup> ESCs and performed fluorescence recovery after photobleaching (FRAP) over the <i>Xist</i><sup>MS2-GFP</sup>-demarcated Xi-territory and other same-size nuclear regions. We also examined <i>Xist</i> RNA dynamics after confirming that no recovery of the <i>Xist</i><sup>MS2-GFP</sup> signals occurred in the absence of transcription (<a id="crosref0785" href="#figs5">Figures S5</a>A–S5E). We observed a slow exchange of photobleached <i>Xist</i><sup>MS2-GFP</sup> in the Xi (<a id="crosref0790" href="#fig4">Figure&nbsp;4</a>A), comparable to that of ectopically expressed <i>Xist</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Ng et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Ng, K. ∙ Daigle, N. ∙ Bancaud, A. ...</div><div><strong>A system for imaging the regulatory noncoding Xist RNA in living mouse embryonic stem cells</strong></div><div><em>Mol. Biol. Cell.</em> 2011; <strong>22</strong>:2634-2645</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.NgN.DaigleA.BancaudT.OhhataP.HumphreysR.WalkerJ.EllenbergA.WutzA+system+for+imaging+the+regulatory+noncoding+Xist+RNA+in+living+mouse+embryonic+stem+cellsMol.+Biol.+Cell22201126342645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A single-exponential kinetic model provided the best fit to the measured FRAP curve and inferred a slow dissociation rate (0.05/min) and an average lifetime of ∼22&nbsp;min. This result is consistent with a single type of high-affinity interaction between <i>Xist</i> and chromatin (<a id="crosref0800" href="#fig4">Figures 4</a>B and 4C). Similar to <i>Xist</i>, CIZ1 has a ∼19&nbsp;min recovery time in the Xi, which is much longer than that of all other interrogated proteins (<a id="crosref0805" href="#fig4">Figures 4</a>D–F). The tight kinetic and spatial relationship between <i>Xist</i> and CIZ1 (<a id="crosref0815" href="#fig3">Figures 3</a>F and <a id="crosref0820" href="#fig4">4</a>D; <a id="crosref0825" href="#mmc7">Video S4</a>) suggests that CIZ1 and <i>Xist</i> molecules form a relatively stable core within a SMAC.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/0c4c651a-032e-403f-a478-d31684701085/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.10.022/asset/aaa684a4-a2b2-4e6a-8f63-c1622f867719/main.assets/gr4.jpg" height="853" width="750" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/0c4c651a-032e-403f-a478-d31684701085/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">FRAP of <i>Xist</i> interactors identifies diverse protein behaviors in the Xi</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0205" role="paragraph">(A) Top: schematic of <i>Xist</i> live-cell labeling. Bottom: image sequence from an Airyscan FRAP experiment of <i>Xist</i><sup>MS2-GFP</sup> at D4. Insets show the <i>Xist</i> territory.</div><div id="fspara0210" role="paragraph">(B) <i>Xist</i><sup>MS2-GFP</sup> FRAP recovery at D4 and fitting. Error bars indicate the standard error. Dissociation rate and lifetime inferred from fitting are given. n denotes the number of cells analyzed from four experiments.</div><div id="fspara0215" role="paragraph">(C) Model for the <i>Xist</i> FRAP process. The expression and replenishment of <i>Xist</i> from its expression site is assumed to be fast and free MCP-GFP replenishment is assumed almost instantaneous after <i>t</i>&nbsp;= 0. The exchange of photobleached with fluorescing <i>Xist</i> is assumed fast in the Xi-territory outside <i>Xist</i>-SMACs (free pool) and slow within <i>Xist</i>-SMACs. <i>Xist</i>-SMACs with zero, one, and two fluorescing <i>Xist</i> molecules are denoted <i>Xist</i>-SMAC-0, -1, and -2. Binding of <i>Xist</i> to sites in SMACs occurs at rate <i>b</i> and dissociation at rate <i>d,</i> which sets the timescale for FRAP recovery. The FRAP curves for <i>Xist</i> were fit with a single exponential.</div><div id="fspara0220" role="paragraph">(D) Image sequence showing a FRAP experiment of <i>Xist</i><sup>MS2-GFP</sup> (green) and CIZ1-mCherry (magenta) at D4. Dashed circles indicate bleached <i>Xist</i>-territories and yellow arrows monitor recovery.</div><div id="fspara0225" role="paragraph">(E) FRAP recovery and fitting (dashed black lines) of the nuclear and <i>Xist</i>-associated populations of CIZ1-mCherry. Error bars denote the standard error. Parameters from fitting with a single exponential are given. n denotes the number of cells analyzed from four experiments.</div><div id="fspara0230" role="paragraph">(F) FRAP recovery and fitting (dashed black lines) of the nuclear and <i>Xist</i>-associated populations of SPEN-Halo<sup>TMR</sup>, PCGF5-Halo<sup>TMR</sup>, CELF1-mCherry, and PTBP1-Halo<sup>TMR</sup> at D4. Error bars denote the standard error. Every fifth time point is shown. Lifetimes for the slow (<i>f</i><sub><i>1</i></sub>) and fast (<i>f</i><sub>2</sub>) detaching fractions inferred for each protein from bi-exponential fitting are indicated. n denotes the number of cells analyzed from two experiments.</div><div id="fspara0235" role="paragraph">(G) Bar graphs showing the lifetimes for <i>Xist</i> and CIZ1 (left) and for the two subpopulations (<i>f</i><sub>1</sub>, <i>f</i><sub>2</sub>) of bi-exponentially fitted proteins (right). Error bars denote the standard error.</div><div id="fspara0240" role="paragraph">(H) Schematic showing an <i>Xist</i>-SMAC and its dynamic regulatory compartment. The increased accumulation of proteins surrounding <i>Xist</i> (red) and their rapid cycling results in gradients over broad chromosomal regions in the vicinity to the SMAC. SPEN is depicted in purple, accumulation of PCGF5 in blue, and purple arrows indicate the gene silencing function of SPEN.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/c81108cf-006d-49dc-afce-5ababe24d8ac/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.10.022/asset/11c66072-78b4-4f32-9259-64d85c508dcf/main.assets/figs5.jpg" height="937" width="726" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/c81108cf-006d-49dc-afce-5ababe24d8ac/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Binding to <i>Xist</i> alters the kinetic behavior of interacting proteins, related to <a id="crosref2075" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0630" role="paragraph">(A) Projections of confocal optical stacks from RNA FISH experiments with Xist probes (magenta) at D4 with or without treatment with the transcriptional inhibitor actinomycinD. DAPI counterstaining is shown in gray.</div><div id="fspara0635" role="paragraph">(B) As in (A) showing one nucleus after actinomycinD treatment with high-resolution Airyscan imaging.</div><div id="fspara0640" role="paragraph">(C) FRAP experiment of <i>Xist</i><sup>MS2-GFP</sup> at D4, after actinomycinD treatment for 20min, showing no recovery of <i>Xist</i><sup>MS2-GFP</sup> signals.</div><div id="fspara0645" role="paragraph">(D) Time series from FRAP experiments showing bleaching (dashed circles) of nuclear regions and the Xi demarcated by <i>Xist</i><sup>MS2-GFP</sup> signals, for CIZ1-mCherry, CELF1-mCherry, SPEN-Halo, PCGF5-Halo and PTBP1-Halo transgenes at D4. Bar: 5&nbsp;μm.</div><div id="fspara0650" role="paragraph">(E) Bar graphs showing percentages of mobile (high opacity) and immobile (low opacity) protein fractions during the time course of each FRAP experiment in <a id="crosref2080" href="#fig4">Figure&nbsp;4</a>. Black bars denote the standard deviation. Results for nuclear and <i>Xist</i>-demarcated fractions are shown in magenta and green, respectively.</div><div id="fspara0655" role="paragraph">(F) Equation used to infer parameters and kinetic modeling of FRAP data (see “Mass action binding and dissociation model of FRAP dynamics” in <a id="crosref2085" href="#mmc3">Methods S1</a>).</div><div id="fspara0660" role="paragraph">(G) Percentage of the slow (<i>f</i><sub><i>1</i></sub>) and fast (<i>f</i><sub>2</sub>) detaching fractions inferred for the indicated proteins, based on the fitting of the FRAP data in the nucleus and the Xi.</div><div id="fspara0665" role="paragraph">(H) Bar graphs showing the percentage of protein particles detected in the pre-Xi/Xi outside <i>Xist</i>-SMACs (labeled as Xi) or in <i>Xist</i>-SMACs, at D2 and D4. Error bars indicate standard deviation.</div><div id="fspara0670" role="paragraph">(I) 3D-SIM projections showing magnifications of the Xi region for <i>Xist</i>, PCGF5 and CIZ1 (top) and <i>Xist</i>, SPEN and CIZ1 (bottom). Note the extended population of PCGF5 and SPEN in the Xi compared to <i>Xist</i> and CIZ1.</div><div id="fspara0675" role="paragraph">(J) Point-plots showing the integrated densities of PCGF5 (left) and SPEN (right) protein particles within the nuclear fraction, the Xi (outside <i>Xist</i> SMACs) or in <i>Xist</i> SMACs at D2 and D4 from data in <a id="crosref2090" href="#fig3">Figure&nbsp;3</a>H. Dots denote the median, bars the 95% confidence interval. The medians at D2 and D4 are connected by dotted lines to visualize any changes. Data are normalized to the highest signal observed across the entire population. MWW <i>p-value</i>s for the comparison between the nucleus and the Xi or <i>Xist</i> SMACs at D2 and D4 4 are given.</div><div id="fspara0680" role="paragraph">(K) Same as (J), except showing volume measurements.</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph"><div id="mmc7" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc7" data-type="mp4"><span id="video_stream_mmc7_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIwNzUxNTMxZmU2ZWMxYzhkZGQzNDMyOWU2NTQ4ODg3YiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.Cqd-OUFaAW7eMJOjiDrgsxQDvXSs79mHo-lLdTG1Oqe8FrGIv-VYFnyaSpZ7GH_9wcP72JuBSxgBppbUqFbTzse9H2hxF5f9kurTHhplDEaeFp2KYvf2GjdoyX0R_s42Tlla7Tj0P6mxKA9_Kg8W4Hdkpy3uvIvjo-2fF6y59_Ggd9gHJHvE1DVkI0J2X8XV0PiytfFRmrMd0g3vxML1UJ5UZKqS8VOVTpb1sgMoJ9TSHeF27y7tX0xDAo5B-mx6pMkUIcyXr4ybs90_x3lTq2Yw7NVdklJbDaamU2_m2ElKHKTKZ5j3LG3DjL4pml3Ko3rdoNw8F0i08nlEmcfvEA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIwNzUxNTMxZmU2ZWMxYzhkZGQzNDMyOWU2NTQ4ODg3YiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.Cqd-OUFaAW7eMJOjiDrgsxQDvXSs79mHo-lLdTG1Oqe8FrGIv-VYFnyaSpZ7GH_9wcP72JuBSxgBppbUqFbTzse9H2hxF5f9kurTHhplDEaeFp2KYvf2GjdoyX0R_s42Tlla7Tj0P6mxKA9_Kg8W4Hdkpy3uvIvjo-2fF6y59_Ggd9gHJHvE1DVkI0J2X8XV0PiytfFRmrMd0g3vxML1UJ5UZKqS8VOVTpb1sgMoJ9TSHeF27y7tX0xDAo5B-mx6pMkUIcyXr4ybs90_x3lTq2Yw7NVdklJbDaamU2_m2ElKHKTKZ5j3LG3DjL4pml3Ko3rdoNw8F0i08nlEmcfvEA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc7" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 100%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIwNzUxNTMxZmU2ZWMxYzhkZGQzNDMyOWU2NTQ4ODg3YiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEzNzM0fQ.Cqd-OUFaAW7eMJOjiDrgsxQDvXSs79mHo-lLdTG1Oqe8FrGIv-VYFnyaSpZ7GH_9wcP72JuBSxgBppbUqFbTzse9H2hxF5f9kurTHhplDEaeFp2KYvf2GjdoyX0R_s42Tlla7Tj0P6mxKA9_Kg8W4Hdkpy3uvIvjo-2fF6y59_Ggd9gHJHvE1DVkI0J2X8XV0PiytfFRmrMd0g3vxML1UJ5UZKqS8VOVTpb1sgMoJ9TSHeF27y7tX0xDAo5B-mx6pMkUIcyXr4ybs90_x3lTq2Yw7NVdklJbDaamU2_m2ElKHKTKZ5j3LG3DjL4pml3Ko3rdoNw8F0i08nlEmcfvEA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.10.022/attachment/d01b36a6-abb0-4316-9ade-a4f27cf3b220/mmc7.mp4" download="mmc7.mp4" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Video (611.07 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Video S4. Xist and CIZ1 exhibit a highly correlated motion in the Xi-territory, related to Figure&nbsp;4</div><div id="espara0025" role="paragraph">Live-cell 3D-SIM imaging of <i>Xist</i><sup>MS2-GFP</sup> (green) and CIZ1-Halo<sup>JF647</sup> (magenta) at D4 of differentiation. Projections of optical stacks acquired through time are shown.</div></div></div></div><div id="p0155" role="paragraph">Kinetic modeling of the SPEN, PCGF5, CELF1, and PTBP1 FRAP curves yielded faster exchange rates than CIZ1 and <i>Xist</i> and two effective types of binding sites (<a id="crosref0830" href="#fig4">Figures 4</a>F, 4G, and <a id="crosref0835" href="#figs5">S5</a>D–S5F). Using two-exponential fits, we inferred parameters for the short-lived (<i>f</i><sub>1</sub>) and long-lived (<i>f</i><sub>2</sub>) bound fractions within and outside of the Xi. For each protein, the rapid binding events occurred within seconds, while slow dissociation required several minutes (<a id="crosref0840" href="#fig4">Figures 4</a>G and <a id="crosref0845" href="#figs5">S5</a>G). SPEN was the most dynamic among the examined proteins. Recruitment by <i>Xist</i> extended the residence time of these proteins compared to that in the nucleus, indicating that the Xi forms a unique nuclear compartment within which proteins exhibit distinct kinetic behaviors (<a id="crosref0850" href="#fig4">Figure&nbsp;4</a>G). The kinetic assays indicate that <i>Xist</i> effector proteins with short residence times bind to the slowly exchanging <i>Xist</i>-CIZ1 core. Thus, SMACs are rapidly exchanging dynamic complexes that form local, high-affinity concentration platforms. Accordingly, examination of SPEN and PCGF5 populations in the Xi, outside SMACs, revealed higher protein concentrations than in nuclear accumulations, demonstrating that recruitment to SMACs leads to enrichment of constituent proteins across extended local neighborhoods in the X-territory (<a id="crosref0855" href="#fig4">Figures 4</a>H and <a id="crosref0860" href="#figs5">S5</a>H–S5K).</div></section><section id="sec-2-7"><h3>Crowding of SPEN in SMACs is IDR-dependent</h3><div id="p0160" role="paragraph">The formation of SMACs is consistent with a requirement for extensive protein-protein interactions (<a id="crosref0865" href="#fig3">Figure&nbsp;3</a>G). SPEN contains intrinsically disordered regions (IDRs), which typically mediate weak, multivalent interactions (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Banani et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Banani, S.F. ∙ Lee, H.O. ∙ Hyman, A.A. ...</div><div><strong>Biomolecular condensates: organizers of cellular biochemistry</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2017; <strong>18</strong>:285-298</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.F.BananiH.O.LeeA.A.HymanM.K.RosenBiomolecular+condensates%3A+organizers+of+cellular+biochemistryNat.+Rev.+Mol.+Cell+Biol.182017285298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Cerase et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Cerase, A. ∙ Armaos, A. ∙ Neumayer, C. ...</div><div><strong>Phase separation drives X-chromosome inactivation: a hypothesis</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:331-334</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.CeraseA.ArmaosC.NeumayerP.AvnerM.GuttmanG.G.TartagliaPhase+separation+drives+X-chromosome+inactivation%3A+a+hypothesisNat.+Struct.+Mol.+Biol.262019331334" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Mittag and Forman-Kay, 2007</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Mittag, T. ∙ Forman-Kay, J.D.</div><div><strong>Atomic-level characterization of disordered protein ensembles</strong></div><div><em>Curr. Opin. Struct. Biol.</em> 2007; <strong>17</strong>:3-14</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.MittagJ.D.Forman-KayAtomic-level+characterization+of+disordered+protein+ensemblesCurr.+Opin.+Struct.+Biol.172007314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105-1" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-1" aria-controls="sref105-1">Uversky, 2015</a><div class="dropBlock__holder" data-db-target-of="sref105-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Uversky, V.N.</div><div><strong>The multifaceted roles of intrinsic disorder in protein complexes</strong></div><div><em>FEBS Lett.</em> 2015; <strong>589</strong>:2498-2506</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.N.UverskyThe+multifaceted+roles+of+intrinsic+disorder+in+protein+complexesFEBS+Lett.58919+Pt+A201524982506" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We homozygously deleted the IDRs within the endogenous SPEN alleles in female ESCs in which SPEN is tagged with GFP (<span class="dropBlock reference-citations"><a id="body-ref-sref31-3" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-3" aria-controls="sref31-3">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and showed that ΔIDR SPEN expression did not disrupt the formation of the <i>Xist</i> cloud (<a id="crosref0895" href="#fig5">Figures 5</a>A, 5B, <a id="crosref0900" href="#figs6">S6</a>A, and S6B). Deletion of the IDRs does not interfere with the binding of the protein to <i>Xist</i> but eliminates the increased SPEN levels in SMACs such that ΔIDR SPEN levels in the pre-Xi and Xi are close to those of the wild-type (WT) protein within the nucleus (<a id="crosref0905" href="#fig5">Figures 5</a>B and 5C). We conclude that the accumulation of SPEN in SMACs is driven exclusively by their IDRs. A recent study similarly showed that exogenous expression of SPEN lacking its IDRs impacts SPEN localization over the Xi-territory upon dox induction of <i>Xist</i> without affecting <i>Xist</i> binding (<span class="dropBlock reference-citations"><a id="body-ref-sref124-2" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-2" aria-controls="sref124-2">Jachowicz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref124-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Jachowicz, J.W. ∙ Strehle, M. ∙ Banerjee, A.K. ...</div><div><strong>Xist spatially amplifies SHARP recruitment to balance chromosome-wide silencing and specificity to the X chromosome</strong></div><div><em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.W.JachowiczM.StrehleA.K.BanerjeeJ.ThaiM.R.BlancoM.GuttmanXist+spatially+amplifies+SHARP+recruitment+to+balance+chromosome-wide+silencing+and+specificity+to+the+X+chromosomebioRxiv202110.1101%2F2021.10.27.466149" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, binding to <i>Xist</i> through its RNA binding (RRM) domains is required for the IDR-mediated concentration of SPEN into SMACs (<a id="crosref0915" href="#figs6">Figures S6</a>C–S6E). FRAP experiments showed that the deletion of the IDRs or RRMs abolishes the characteristic Xi-immobile fraction of SPEN and dramatically alters residence times, with ΔIDR SPEN exhibiting very long unbinding times in both the Xi and nuclear fractions, possibly due to the tight binding to the RNA through the RRMs (<a id="crosref0920" href="#figs6">Figures S6</a>F–S6H). Therefore, the IDRs of SPEN are also critical for creating a dynamic protein assembly (<a id="crosref0925" href="#fig5">Figure&nbsp;5</a>K).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/ac1d871f-f53b-42ef-ac2f-25bc8de904a8/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.10.022/asset/4b5346e3-2243-4821-8deb-53b4fa1d27ed/main.assets/gr5.jpg" height="939" width="695" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/ac1d871f-f53b-42ef-ac2f-25bc8de904a8/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">IDR-mediated crowding of SPEN in SMACs is required for gene silencing</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0250" role="paragraph">(A) Schematic of the domains of WT and ΔIDR SPEN.</div><div id="fspara0255" role="paragraph">(B) 3D-SIM optical sections of immuno-RNA-FISH with Xist probes (magenta) and GFP antibodies (green) at D2 and D4 in cells homozygously expressing GFP-tagged WT- or ΔIDR-SPEN. Second columns show the Xi. Staining by DAPI is shown in gray.</div><div id="fspara0260" role="paragraph">(C) Point-plots of integrated density and volume measurements for WT- and ΔIDR-SPEN particles that are <i>Xist</i>-associated (green) or in the nuclear (purple) fraction, on D2 and D4 from (B). n denotes the number of cells analyzed from two experiments. MWW p values are given.</div><div id="fspara0265" role="paragraph">(D) RNA FISH of <i>Xist</i> (green) and nascent transcripts of <i>Rlim</i> or <i>Atrx</i> (magenta) at 24&nbsp;h after doxycycline induction of tetO-<i>Xist</i> expression in female Xist<sup>wt/tetO</sup> cells homozygously expressing WT-, ΔIDR-, and ΔSPOC-SPEN. Chromatin is stained with DAPI (gray). Second row images show <i>Rlim</i> or <i>Atrx</i> signals (gray) and nuclei masks (dashed lines).</div><div id="fspara0270" role="paragraph">(E) Quantification of experiment in (D) showing percentage of <i>Xist</i> clouds with no nascent X-linked gene transcript spot (purple, monoallelic) versus detection of transcripts from both X chromosomes (gray, biallelic). n&nbsp;denotes the number of cells analyzed from two replicates.</div><div id="fspara0275" role="paragraph">(F) Violin plots of Xi expression ratios of X-linked genes averaged across single Xist<sup>wt/tetO</sup> cells homozygously expressing WT-, ΔIDR-, or ΔSPOC-SPEN without and with 24&nbsp;h of doxycycline addition. MWW p values are given.</div><div id="fspara0280" role="paragraph">(G) Violin plots of the change in Xi expression ratio for data in (F), grouped according to gene silencing dynamics in normal cells. Kruskal-Wallis p values are given.</div><div id="fspara0285" role="paragraph">(H) Schematic of rescue assay used in (I) and (J). SPEN-AID-GFP encoded from the endogenous locus is depleted with addition of auxin and FL- or ΔIDR-SPEN-Halo are constitutively expressed from the R26 locus. <i>Xist</i> expression and XCI are induced by addition of doxycycline for 24&nbsp;h in the presence of auxin.</div><div id="fspara0290" role="paragraph">(I) Images of SPEN-AID-GFP (green) with transgenic FL-SPEN (top) and ΔIDR-SPEN (bottom) rescue proteins (magenta). Columns from left to right: untreated; 12&nbsp;h auxin treated and 24&nbsp;h doxycycline and 36&nbsp;h auxin treated cells. This strategy was used in (J) to explore rescue of XCI by transgenically encoded SPEN proteins after depletion of endogenously encoded SPEN and induction of tetO-<i>Xist</i>.</div><div id="fspara0295" role="paragraph">(J) Violin plots of the change in Xi expression ratios of X-linked genes over 24&nbsp;h of doxycycline-induced <i>Xist</i> expression, grouped according to gene silencing dynamics in normal cells, for conditions described in (H). The Xi ratio was averaged across three replicates. Kruskal-Wallis p values are given.</div><div id="fspara0300" role="paragraph">(K) Model of the augmented and dynamic distribution of WT-SPEN (top) in a <i>Xist</i>-SMAC compared to ΔIDR-SPEN (bottom). <i>Xist</i> is shown in red and SPEN domains are annotated as in (A). Silent and active X-linked genes are indicated with black and gray arrows.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/88f9b675-fb01-47ee-aea6-8fb1e83be1f9/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.10.022/asset/4ef9c592-535b-45a9-932b-7b2510be73ee/main.assets/figs6.jpg" height="940" width="713" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/88f9b675-fb01-47ee-aea6-8fb1e83be1f9/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Gene silencing by SPEN is mediated by IDR-aggregation, related to <a id="crosref2095" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0690" role="paragraph">(A) Schematic of the CRISPR/Cas9-based genome editing strategy used to delete the IDRs of SPEN with two guide RNAs (gRNA1, gRNA2) and annotation of the genotyping primers (black single-headed arrowed lines) and PCR amplicons (black double headed arrowed lines/dashed lines) in female Bl6-Xist<sup>tetO</sup> Cas-Xist<sup>wt</sup> ESCs (left). PCR products after genotyping of SPEN<sup>WT/WT</sup> (WT) and SPEN<sup>ΔIDR/ΔIDR</sup> (ΔIDR) cells, showing the expected band sizes (right).</div><div id="fspara0695" role="paragraph">(B) Illustration of the SPEN<sup>ΔIDR/ΔIDR</sup> locus after genome editing described in <b>A</b>, with indicated exons, introns and deletion site. Below, histograms of reads obtained by Sanger sequencing for cDNA from a RT reaction with oligodT-primers from RNA derived from SPEN<sup>ΔIDR/ΔIDR</sup> cells and PCR across exons 5 to 14, confirm the presence of deleted transcripts for both the B6 and Cas alleles, based on single nucleotide polymorphisms in exons 5 and 7.</div><div id="fspara0700" role="paragraph">(C) 3D-SIM projections of nuclei of female cells stably expressing FL-SPEN-Halo<sup>JF646</sup>, ΔIDR-SPEN-Halo<sup>JF646</sup> or ΔRRM-SPEN-Halo<sup>JF646</sup> together with <i>Xist</i><sup>MS2-GFP</sup> on D2 and D4. Second columns show magnifications of the <i>Xist</i>-territory. Bars: 5&nbsp;μm; insets: 1&nbsp;μm.</div><div id="fspara0705" role="paragraph">(D) Halo-based CLAP-seq with cells described in <b>C</b>. Shown is the fold change of IP/input RPKMs in 100bp bins, smoothed over 300bp, across the <i>Xist</i> genomic locus. <i>Xist</i> RNA repeat elements are labeled. Data are averaged from two replicates of FL- and ΔIDR-, and from one replicate for ΔRRM-SPEN-Halo. Note the different y axis labels, which indicates the lack of Xist A-repeat binding specifically for ΔRRM-SPEN. Together, these data demonstrate that ΔIDR SPEN binds to the A-repeat region of <i>Xist</i> similar to FL SPEN, whereas ΔRRM SPEN does not interact significantly.</div><div id="fspara0710" role="paragraph">(E) Point-plots of particle integrated density of fluorescence and volume measurements in <i>Xist</i>-associated (green) and nuclear (magenta) fractions on D2 and D4, comparing FL-, ΔIDR- and ΔRRM-SPEN levels in cells described in (C). Dots denote the median, bars the standard deviation. The medians at D2 and D4 are connected by dotted lines to visualize any changes. Data are normalized to the highest signal observed across the entire population. n is the number of cells from two replicates. MWW <i>p-value</i>s for the comparison between the nucleus and the Xi at D2 or D4 are given.</div><div id="fspara0715" role="paragraph">(F) Time-sequence from FRAP experiments of <i>Xist</i><sup>MS2-GFP</sup> cells stably expressing ΔIDR- or ΔRRM-SPEN-Halo at D4 of differentiation. Bar: 5&nbsp;μm.</div><div id="fspara0720" role="paragraph">(G) FRAP recovery curves comparing ΔIDR- to ΔRRM-SPEN-Halo in the nucleus and <i>Xist</i>-territory from data in (F). Error bars denote the standard error. Every fifth time point is shown for clarity. The fitted curves for the first 10&nbsp;s of this graph are shown in the magnified inset in comparison to FL-SPEN-Halo from <a id="crosref2100" href="#fig4">Figure&nbsp;4</a>F. n is the number of cells analyzed from two experiments.</div><div id="fspara0725" role="paragraph">(H) Bar graphs showing parameters extracted from the fitting of FRAP data shown in (G) for ΔIDR- and ΔRRM-SPEN in the nucleus and the Xi, respectively, showing the lifetimes of slow (<i>f</i><sub><i>1</i></sub>) and fast (<i>f</i><sub>2</sub>) detaching fractions in comparison to FL-SPEN. Note that ΔIDR-SPEN exhibits very long lifetimes for the <i>f</i><sub>2</sub> fraction denoted as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="infinity period" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="2" data-semantic-children="0,1" data-semantic-content="1" data-semantic-owns="0 1" data-semantic-speech="infinity period"><mjx-mi data-semantic-type="identifier" data-semantic-role="infty" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c221E MM-I">∞</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="1" data-semantic-parent="2" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> Error bars denote the standard error. Percentages indicate the proportion of each population.</div><div id="fspara0730" role="paragraph">(I) Schematic showing recruitment of the SPEN silencing domain SPOC to <i>Xist</i> under physiological conditions (via the full length SPEN protein) or by aptamer tethering (via BglG/SL) as described in (<span class="dropBlock reference-citations"><a id="body-ref-sref31-4" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-4" aria-controls="sref31-4">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0735" role="paragraph">(J) Violin plots of the silencing defects of X-linked genes in Bl6-Xist<sup>tetO, BglG</sup> Cas-Xist<sup>wt</sup> cells upon loss of endogenously encoded SPEN-AID (upon auxin addition) and rescue with GFP (BglG-GFP) or <i>Xist</i>-tethered SPOC (BglG-GFP-SPOC) shown in (I) (re-analyzing published data (<span class="dropBlock reference-citations"><a id="body-ref-sref31-5" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-5" aria-controls="sref31-5">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)). Genes are grouped by published silencing dynamics. A silencing defect of 0 indicates no difference in silencing efficacy between WT cells and the respective rescue cells, whereas a value of 1 indicates a complete loss of silencing in rescue lines compared to the WT cells. MWW <i>p-value</i>s given.</div><div id="fspara0740" role="paragraph">(K) Same as in (J), but directly showing genes in each silencing group side-by-side.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-8"><h3>IDR-mediated crowding of SPEN in SMACs is required for XCI</h3><div id="p0165" role="paragraph">The SPOC domain of SPEN is essential for silencing (<span class="dropBlock reference-citations"><a id="body-ref-sref31-6" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-6" aria-controls="sref31-6">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but whether the crowding of SPEN is necessary for the functionality of SPOC is unknown. RNA FISH for nascent transcription of five X-linked genes revealed a striking silencing defect when&nbsp;IDR-mediated crowding was ablated, similar to the lack of silencing caused by the deletion of SPOC (<span class="dropBlock reference-citations"><a id="body-ref-sref31-7" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-7" aria-controls="sref31-7">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0940" href="#fig5">Figures 5</a>D and 5E). This is consistent with a recent report that showed, that the IDRs of SPEN are required for transcriptional silencing of four X-linked genes upon dox-induction of <i>Xist</i>, based on RNA FISH (<span class="dropBlock reference-citations"><a id="body-ref-sref124-3" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-3" aria-controls="sref124-3">Jachowicz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref124-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Jachowicz, J.W. ∙ Strehle, M. ∙ Banerjee, A.K. ...</div><div><strong>Xist spatially amplifies SHARP recruitment to balance chromosome-wide silencing and specificity to the X chromosome</strong></div><div><em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.W.JachowiczM.StrehleA.K.BanerjeeJ.ThaiM.R.BlancoM.GuttmanXist+spatially+amplifies+SHARP+recruitment+to+balance+chromosome-wide+silencing+and+specificity+to+the+X+chromosomebioRxiv202110.1101%2F2021.10.27.466149" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To explore if the loss of gene&nbsp;silencing by ΔIDR SPEN extends to the entire X chromosome, we&nbsp;performed scRNA-seq before and 24&nbsp;h after <i>Xist</i> induction in&nbsp;ΔIDR-, ΔSPOC-, and WT SPEN-expressing cells. Although X-linked gene repression was observed in WT cells, both ΔIDR- and ΔSPOC SPEN-expressing cells displayed a dramatic X chromosome-wide loss of gene silencing, affecting both rapidly and slowly silencing genes (<span class="dropBlock reference-citations"><a id="body-ref-sref3-3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-3" aria-controls="sref3-3">Barros de Andrade E. Sousa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Barros de Andrade E. Sousa, L. ∙ Jonkers, I. ∙ Syx, L. ...</div><div><strong>Kinetics of <i>Xist</i>-induced gene silencing can be predicted from combinations of epigenetic and genomic features</strong></div><div><em>Genome Res.</em> 2019; <strong>29</strong>:1087-1099</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.Barros+de+Andrade+E.+SousaI.JonkersL.SyxI.DunkelJ.ChaumeilC.PicardB.ForetC.J.ChenJ.T.LisE.HeardKinetics+of+Xist-induced+gene+silencing+can+be+predicted+from+combinations+of+epigenetic+and+genomic+featuresGenome+Res.29201910871099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0950" href="#fig5">Figures 5</a>F and 5G). Next, we constitutively expressed Halo-tagged ΔIDR or full-length (FL) SPEN as rescue constructs in ESCs in which the endogenously encoded SPEN is fused to the AID degron tag and can be depleted by addition of auxin (<span class="dropBlock reference-citations"><a id="body-ref-sref31-8" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-8" aria-controls="sref31-8">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0960" href="#fig5">Figures 5</a>H and 5I). Bulk RNA-seq showed that FL but not ΔIDR SPEN can rescue X-linked gene silencing (<a id="crosref0965" href="#fig5">Figure&nbsp;5</a>J). Interestingly, when only SPOC is tightly tethered to <i>Xist</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref31-9" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-9" aria-controls="sref31-9">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), X-linked genes are inefficiently silenced (<a id="crosref0975" href="#figs6">Figures S6</a>I–S6K), consistent with a dynamic SPEN protein being required for XCI. In summary, our findings demonstrate that the concentration of SPEN in SMACs and its rapid kinetic behavior, both mediated by the IDRs, are required for the protein to exert its silencing function through the SPOC domain (<a id="crosref0980" href="#fig5">Figure&nbsp;5</a>K).</div></section><section id="sec-2-9"><h3>The B-repeat is critical for Xi compaction and late gene silencing</h3><div id="p0170" role="paragraph">Non-canonical PRC1 induces the recruitment of canonical PRC1 and downstream accumulation of PRC2 (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Brockdorff, 2017</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Brockdorff, N.</div><div><strong>Polycomb complexes in X chromosome inactivation</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2017; <strong>372</strong>:20170021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.BrockdorffPolycomb+complexes+in+X+chromosome+inactivationPhilos.+Trans.+R.+Soc.+Lond.+B+Biol.+Sci.372201720170021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and is implicated in the silencing of a subset of X-linked genes (<span class="dropBlock reference-citations"><a id="body-ref-sref9-2" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-2" aria-controls="sref9-2">Bousard et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Bousard, A. ∙ Raposo, A.C. ∙ Żylicz, J.J. ...</div><div><strong>The role of Xist-mediated Polycomb recruitment in the initiation of X-chromosome inactivation</strong></div><div><em>EMBO Rep.</em> 2019; <strong>20</strong>:e48019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.BousardA.C.RaposoJ.J.%C5%BByliczC.PicardV.B.PiresY.QiC.GilL.SyxH.Y.ChangE.HeardS.T.da+RochaThe+role+of+Xist-mediated+Polycomb+recruitment+in+the+initiation+of+X-chromosome+inactivationEMBO+Rep.202019e48019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Colognori et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Colognori, D. ∙ Sunwoo, H. ∙ Kriz, A.J. ...</div><div><strong>Xist Deletional Analysis Reveals an Interdependency between Xist RNA and Polycomb Complexes for Spreading along the Inactive X</strong></div><div><em>Mol. Cell.</em> 2019; <strong>74</strong>:101-117.e10</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.ColognoriH.SunwooA.J.KrizC.Y.WangJ.T.LeeXist+Deletional+Analysis+Reveals+an+Interdependency+between+Xist+RNA+and+Polycomb+Complexes+for+Spreading+along+the+Inactive+XMol.+Cell742019101117.e10" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-3" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-3" aria-controls="sref74-3">Nesterova et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref74-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Nesterova, T.B. ∙ Wei, G. ∙ Coker, H. ...</div><div><strong>Systematic allelic analysis defines the interplay of key pathways in X chromosome inactivation</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:3129</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.B.NesterovaG.WeiH.CokerG.PintacudaJ.S.BownessT.ZhangM.AlmeidaB.BloechlB.MoindrotE.J.CarterSystematic+allelic+analysis+defines+the+interplay+of+key+pathways+in+X+chromosome+inactivationNat.+Commun.1020193129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref123-2" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-2" aria-controls="sref123-2">Zylicz et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref123-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Zylicz, J.J. ∙ Bousard, A. ∙ Zumer, K. ...</div><div><strong>The Implication of Early Chromatin Changes in X Chromosome Inactivation</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:182-197.e23</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.J.ZyliczA.BousardK.ZumerF.DossinE.MohammadS.T.da+RochaB.SchwalbL.SyxF.DingliD.LoewThe+Implication+of+Early+Chromatin+Changes+in+X+Chromosome+InactivationCell1762019182197.e23" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Yet, PRC1 spreads into genes only after silencing has occurred (<span class="dropBlock reference-citations"><a id="body-ref-sref123-3" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-3" aria-controls="sref123-3">Zylicz et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref123-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Zylicz, J.J. ∙ Bousard, A. ∙ Zumer, K. ...</div><div><strong>The Implication of Early Chromatin Changes in X Chromosome Inactivation</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:182-197.e23</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.J.ZyliczA.BousardK.ZumerF.DossinE.MohammadS.T.da+RochaB.SchwalbL.SyxF.DingliD.LoewThe+Implication+of+Early+Chromatin+Changes+in+X+Chromosome+InactivationCell1762019182197.e23" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), raising the question of how it contributes to XCI. Since PRC1 is critical for chromatin compaction in various developmental contexts (<span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Boyle et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Boyle, S. ∙ Flyamer, I.M. ∙ Williamson, I. ...</div><div><strong>A central role for canonical PRC1 in shaping the 3D nuclear landscape</strong></div><div><em>Genes Dev.</em> 2020; <strong>34</strong>:931-949</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.BoyleI.M.FlyamerI.WilliamsonD.SenguptaW.A.BickmoreR.S.IllingworthA+central+role+for+canonical+PRC1+in+shaping+the+3D+nuclear+landscapeGenes+Dev.342020931949" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Francis et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Francis, N.J. ∙ Kingston, R.E. ∙ Woodcock, C.L.</div><div><strong>Chromatin compaction by a polycomb group protein complex</strong></div><div><em>Science.</em> 2004; <strong>306</strong>:1574-1577</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.J.FrancisR.E.KingstonC.L.WoodcockChromatin+compaction+by+a+polycomb+group+protein+complexScience306200415741577" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Grau et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Grau, D.J. ∙ Chapman, B.A. ∙ Garlick, J.D. ...</div><div><strong>Compaction of chromatin by diverse Polycomb group proteins requires localized regions of high charge</strong></div><div><em>Genes Dev.</em> 2011; <strong>25</strong>:2210-2221</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.J.GrauB.A.ChapmanJ.D.GarlickM.BorowskyN.J.FrancisR.E.KingstonCompaction+of+chromatin+by+diverse+Polycomb+group+proteins+requires+localized+regions+of+high+chargeGenes+Dev.25201122102221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Illingworth, 2019</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Illingworth, R.S.</div><div><strong>Chromatin folding and nuclear architecture: PRC1 function in 3D</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2019; <strong>55</strong>:82-90</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.S.IllingworthChromatin+folding+and+nuclear+architecture%3A+PRC1+function+in+3DCurr.+Opin.+Genet.+Dev.5520198290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we explored whether the B-repeat is required for the structural reorganization of the Xi.</div><div id="p0175" role="paragraph">We perturbed PRC1 recruitment to the X by heterozygously deleting the B-repeat (ΔB-<i>Xist</i>) on the MS2-tagged 129 Xist allele in female ESCs derived from a 129 × castaneous (Cas) cross and compared the compaction of the Xi<sup>Cas</sup> formed by FL-<i>Xist</i> to the Xi<sup>129</sup> induced by ΔB-<i>Xist</i> (<a id="crosref1035" href="#fig6">Figures 6</a>A and 6B). Deletion of the B-repeat results in less compacted pre-Xi and Xi territories, larger distances between <i>Xist</i> foci and an expansion of the <i>Xist</i> cluster (<a id="crosref1040" href="#fig6">Figures 6</a>B–6C and <a id="crosref1045" href="#figs7">S7</a>A–S7C). Accordingly, the density of SPEN-decorated SMACs is far lower in the ΔΒ-Xi than in the WT-Xi, although the concentration of SPEN in their respective SMACs is similar (<a id="crosref1050" href="#figs7">Figures S7</a>D and S7E). Together, these results uncover a role of the B-repeat, and in turn of PRC1 and its downstream effectors, in driving the compaction of the X chromosome and densification of SMACs during XCI initiation.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/63ee43ac-1abf-4cad-8d45-1f8dfdad4f81/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.10.022/asset/b1820108-2d53-4388-b563-1ad19068c872/main.assets/gr6.jpg" height="681" width="752" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/63ee43ac-1abf-4cad-8d45-1f8dfdad4f81/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">The B-repeat is critical for Xi compaction and late gene silencing</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0310" role="paragraph">(A) Schematic of the heterozygous deletion of the B-repeat of Xist and insertion of a MS2 tag on the 129 X chromosome in female mouse 129/Cas ESCs (Xist<sup>ΔΒ+MS2/WT</sup> ESCs). The effect of the B-repeat deletion is indicated.</div><div id="fspara0315" role="paragraph">(B) RNA/DNA FISH of Xist<sup>ΔΒ+MS2/WT</sup> cells at D4 using X chromosome paints (mmX, green), Xist (magenta) and MS2 probes (yellow). DAPI staining is shown in blue. Greyscale images show individual channels as indicated.</div><div id="fspara0320" role="paragraph">(C) Boxplots showing the ratio of Xa/Xa at D2 in cells not expressing <i>Xist</i> and of Xa/Xi in FL-<i>Xist</i> (purple) or ΔB-<i>Xist</i> (gray) expressing Xist<sup>ΔΒ+MS2/WT</sup> cells at D2 and D4. n is the number of cells analyzed from two replicates.</div><div id="fspara0325" role="paragraph">(D) Boxplots showing the minimal (left) and average (right) distance between <i>Xist</i> foci in FL- or ΔB-<i>Xist</i>-expressing cells at D2 and D4. n is the number of cells analyzed from three experiments.</div><div id="fspara0330" role="paragraph">(E) Violin plots of Xi ratios of X-linked gene expression at D2 and D4 in Xist<sup>ΔΒ+MS2/WT</sup> cells or parent WT (Xist<sup>WT-MS2/WT</sup>) cells silencing the WT- or ΔB-<i>Xist</i> 129 X&nbsp;chromosome. Mean Xi ratio per gene was averaged across single cells based on scRNA-seq data. MWW p values are given.</div><div id="fspara0335" role="paragraph">(F) As in (E), except that genes are grouped by gene-silencing dynamics. MWW p values are given.</div><div id="fspara0340" role="paragraph">(G) Violin plots of the difference in Xi ratio between the ΔB- and WT-<i>Xist</i> expressing Xi shown in (F). MWW p values are given.</div><div id="fspara0345" role="paragraph">(H) Bar graphs showing the proportion of ΔB-sensitive or insensitive genes from (F). Genes were considered ΔB-sensitive based on a one-sided Welch t test comparing ΔB and WT Xi<sup>129</sup> ratios, p value&nbsp;&lt; 0.05.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/519f0220-fc46-4fca-a2be-74b927dbadb9/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.10.022/asset/582e4f5d-02cd-4309-8b18-bde7a9e6e16e/main.assets/figs7.jpg" height="939" width="688" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/519f0220-fc46-4fca-a2be-74b927dbadb9/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Progressive X chromosome compaction links PRC1/SMCHD1 function to late gene silencing, related to <a id="crosref2115" href="#fig6">Figures 6</a> and <a id="crosref2120" href="#fig7">7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0750" role="paragraph">(A) Volume measurements of X chromosome territories formed by FL- or ΔB-<i>Xist</i> in Xist<sup>ΔΒ+MS2/WT</sup> cells at D4 upon X chromosome painting shown in <a id="crosref2125" href="#fig6">Figure&nbsp;6</a>B. Error bars denote the standard deviation.</div><div id="fspara0755" role="paragraph">(B) 3D-SIM projections of RNA FISH with Xist probes (green) for the Xi formed by FL-<i>Xist</i> or the Xi formed at D4. DAPI counterstaining is shown in blue. Magnified insets showing <i>Xist</i> signals (<i>Xist</i> cluster, gray).</div><div id="fspara0760" role="paragraph">(C) As in (B) showing only the DAPI channel. The Xi regions are indicated by arrows. Note the characteristic compaction of the Xi territory evident in FL-<i>Xist</i>-expressing cells, which is not present in cells expressing ΔB-<i>Xist</i>.</div><div id="fspara0765" role="paragraph">(D) 3D-SIM optical sections of RNA FISH with Xist probes (green) in Xist<sup>ΔΒ+MS2/WT</sup> Xist- cells that express either FL-<i>Xist</i> or ΔΒ/MS2-<i>Xist</i> together with stably expressed SPEN-Halo, detected with JF549 ligand (magenta), at D2 and D4. DAPI counterstaining is shown in gray. Second columns show magnifications of the Xi regions. Far right columns show z-projections at D4 and magnifications of the Xi region without <i>Xist</i> signals.</div><div id="fspara0770" role="paragraph">(E) Point-plots for data in (D) showing the integrated density of fluorescence of SPEN particles in <i>Xist</i>-associated (red) and nuclear (green) fractions, on D2 and D4, from two experiments. Dots denote the median, bars the standard deviation. The medians at D2 and D4 are connected by dotted lines to visualize any changes. Data are normalized to the highest signal observed across the entire population. MWW <i>p-value</i>s for the comparison between the nuclear and <i>Xist</i>-associated fractions are given. n denotes the number of cells analyzed from two experiments.</div><div id="fspara0775" role="paragraph">(F) Violin plots showing the silencing half-times (in days) of SMCHD1-sensitive and insensitive X-linked genes based on a reanalysis of published data (<span class="dropBlock reference-citations"><a id="body-ref-sref110-2" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-2" aria-controls="sref110-2">Wang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref110-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Wang, C.Y. ∙ Jegu, T. ∙ Chu, H.P. ...</div><div><strong>SMCHD1 Merges Chromosome Compartments and Assists Formation of Super-Structures on the Inactive X</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:406-421.e25</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.Y.WangT.JeguH.P.ChuH.J.OhJ.T.LeeSMCHD1+Merges+Chromosome+Compartments+and+Assists+Formation+of+Super-Structures+on+the+Inactive+XCell1742018406421.e25" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Genes that lose silencing upon <i>Smchd1</i> knockout (KO) are silencing more slowly under normal conditions than genes that continue to silence. MWW <i>p-value</i> is given.</div><div id="fspara0780" role="paragraph">(G) Boxplots showing the median distances of early (red) and late (green) genes from data in <a id="crosref2135" href="#fig7">Figure&nbsp;7</a>G to the center of the <i>Xist</i> cluster at D2 and D4, without dividing the genes according to expression status. MWW <i>p-value</i>s are given.</div><div id="fspara0785" role="paragraph">(H) Boxplots showing the nearest neighbor distances of active (red) and silent (green) early and late genes to <i>Xist</i> foci from data in <a id="crosref2140" href="#fig7">Figure&nbsp;7</a>G. MWW <i>p-value</i>s are given.</div></div></div></div></figcaption></figure></div><div id="p0180" role="paragraph">To explore if X-linked gene-silencing dynamics are altered in the absence of compaction, we performed scRNA-seq at D2 and D4. Upon deletion of the B-repeat, silencing is more impaired on the Xi than on the pre-Xi (<a id="crosref1055" href="#fig6">Figure&nbsp;6</a>E). The silencing defect is strongest for slowly silencing genes (<a id="crosref1060" href="#fig6">Figures 6</a>F–6H). These results extend to cells lacking SMCHD1 that controls the compartmentalization of the Xi and is recruited to the Xi by PRC1 (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Jansz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Jansz, N. ∙ Nesterova, T. ∙ Keniry, A. ...</div><div><strong>Smchd1 Targeting to the Inactive X Is Dependent on the Xist-HnrnpK-PRC1 Pathway</strong></div><div><em>Cell Rep.</em> 2018; <strong>25</strong>:1912-1923.e9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.JanszT.NesterovaA.KeniryM.IminitoffP.F.HickeyG.PintacudaO.MasuiS.KobelkeN.GeogheganK.A.BreslinSmchd1+Targeting+to+the+Inactive+X+Is+Dependent+on+the+Xist-HnrnpK-PRC1+PathwayCell+Rep.25201819121923.e9" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref111-3" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-3" aria-controls="sref111-3">Wang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Wang, C.Y. ∙ Colognori, D. ∙ Sunwoo, H. ...</div><div><strong>PRC1 collaborates with SMCHD1 to fold the X-chromosome and spread Xist RNA between chromosome compartments</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2950</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.Y.WangD.ColognoriH.SunwooD.WangJ.T.LeePRC1+collaborates+with+SMCHD1+to+fold+the+X-chromosome+and+spread+Xist+RNA+between+chromosome+compartmentsNat.+Commun.1020192950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1075" href="#figs7">Figure&nbsp;S7</a>E). Thus, compaction by PRC1 and SMCHD1 and the further clustering of the <i>Xist</i>-SMACs allows SPEN to act on all genes.</div></section><section id="sec-2-10"><h3><i>Xist</i>-SMACs progressively re-configure and silence the Xi</h3><div id="p0185" role="paragraph">We next explored how genes with different silencing kinetics, i.e., rapidly (early) and slowly (late) silencing genes, localize relative to <i>Xist</i> foci. To determine these spatial relationships, we applied multiple distance metrics to 20 simultaneously detected early or late genes distributed across the entire X chromosome, their nascent transcripts, and <i>Xist</i> by RNA/DNA FISH and 3D-SIM (<a id="crosref1080" href="#fig7">Figures 7</a>A–7D).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.10.022/asset/4fec8c78-644e-4c86-a3ac-a140380e87f1/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.10.022/asset/684cc1a4-5d2e-4da2-99a0-6db9103b5649/main.assets/gr7.jpg" height="939" width="684" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/4fec8c78-644e-4c86-a3ac-a140380e87f1/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text"><i>Xist</i>-SMACs progressively re-configure and silence the Xi</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0355" role="paragraph">(A) Annotation of the position of early (yellow) and late (purple) genes on the X chromosome simultaneously detected with oligo probes in RNA/DNA FISH experiments and their silencing half-time.</div><div id="fspara0360" role="paragraph">(B) 3D-SIM projections of nuclei after RNA/DNA FISH, showing indicated gene sets (yellow), their corresponding transcripts (magenta), and <i>Xist</i> signals (cyan) at D2 and D4. Xa, pre-Xi, and Xi are indicated. Insets show magnifications of pre-Xi or Xi areas with high (top) and low (bottom) <i>Xist</i> density. Images are smoothed with a 3&nbsp;× 3 px for clarity.</div><div id="fspara0365" role="paragraph">(C) 3D-SIM projections of Xa, pre-Xi, or Xi regions after RNA/DNA FISH for <i>Xist</i> (cyan), early (yellow), and late (magenta) genes at D2 and D4.</div><div id="fspara0370" role="paragraph">(D) Schematic of different types of distance metrics performed in this figure.</div><div id="fspara0375" role="paragraph">(E) Boxplots of distances of early (yellow) or late (purple) genes relative to Tsix signals on the Xa. n denotes the number of cells analyzed from three experiments. MWW p value is given.</div><div id="fspara0380" role="paragraph">(F) Boxplots of the nearest distance of <i>Xist</i> foci to early (yellow) or late (purple) genes at D2 and D4. n denotes the number of cells analyzed from three experiments. MWW p values are given.</div><div id="fspara0385" role="paragraph">(G) Boxplots of distances of early (yellow) or late (purple) genes to the center of the <i>Xist</i> cluster at D2 and D4, divided into silent or active based on nascent transcripts detection. n is the number of cells analyzed from three experiments. MWW p values are given.</div><div id="fspara0390" role="paragraph">(H) Intra-genic distances of early (yellow) or late (purple) genes on the Xa, pre-Xi, or Xi at D2 and D4. n denotes the number of cells analyzed from three experiments. Kruskal-Wallis p values are given.</div><div id="fspara0395" role="paragraph">(I) 3D-SIM projections of the Xi after RNA/DNA FISH for <i>Xist</i> (cyan), early (yellow), and late (magenta) genes in male FL- or ΔΒ-<i>Xist</i>-expressing ESCs after 18&nbsp;h doxycycline induction of tetO-<i>Xist</i>. n denotes the number of cells analyzed in (J) to (M) from two experiments. MWW p values are given.</div><div id="fspara0400" role="paragraph">(J) Boxplots of distances of early (yellow) or late (purple) genes to the center of the <i>Xist</i> cluster in FL- or ΔΒ-<i>Xist</i> expressing cells described in (I).</div><div id="fspara0405" role="paragraph">(K) As in (J), except for intra-genic distances of early (yellow) or late (purple) genes.</div><div id="fspara0410" role="paragraph">(L) As in (J), except for the distances of early to late silencing genes.</div><div id="fspara0415" role="paragraph">(M) As in (J), except for nearest neighbor distance of early (yellow) or late (purple) genes to <i>Xist</i> foci.</div><div id="fspara0420" role="paragraph">(N) SMAC-based model of XCI. Left column shows the changes in the higher-order chromatin organization between the Xa (top), pre-Xi (middle), and Xi (bottom). The <i>Xist</i> production site is shown in blue, SMACs in dark red, and genomic regions harboring early and late silencing genes with orange and purple lines, respectively. Insets indicate regions of early (top) or late (bottom) silencing genes magnified in the second and third columns to show the progression of silencing. Arrows indicate active (green) and silent (brown) genes. Fourth column shows the increase in protein concentration upon establishment of <i>Xist</i>-SMACs. Pink dots indicate Polycomb-group proteins and purple dots SPEN. Free protein dots indicate increased concentrations in the Xi due to the presence of SMACs. Architectural protein-mediated chromosomal compaction is depicted by pink islets on the DNA fiber.</div></div></div></div></figcaption></figure></div><div id="p0190" role="paragraph">We first monitored the distribution of early and late gene loci on the Xa relative to the <i>Xist</i> transcription locus by exploiting the detection of <i>Tsix</i> RNA as a proxy for the X-inactivation center (Xic) where the <i>Xist</i> gene is located (<span class="dropBlock reference-citations"><a id="body-ref-sref82-2" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-2" aria-controls="sref82-2">Plath et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref82-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Plath, K. ∙ Mlynarczyk-Evans, S. ∙ Nusinow, D.A. ...</div><div><strong>Xist RNA and the mechanism of X chromosome inactivation</strong></div><div><em>Annu. Rev. Genet.</em> 2002; <strong>36</strong>:233-278</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.PlathS.Mlynarczyk-EvansD.A.NusinowB.PanningXist+RNA+and+the+mechanism+of+X+chromosome+inactivationAnnu.+Rev.+Genet.362002233278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This analysis showed that early genes are closer to the <i>Xist</i> locus than late genes (<a id="crosref1090" href="#fig7">Figures 7</a>C and 7E). Accordingly, upon differentiation, early genes are closer to individual <i>Xist</i> foci or to the center of the entire <i>Xist</i> cluster than late genes, which is more pronounced on the pre-Xi (<a id="crosref1095" href="#fig7">Figures 7</a>F and <a id="crosref1100" href="#figs7">S7</a>G). Thus, genomic regions containing early genes are spatially more proximal to the <i>Xist</i> locus at the onset of XCI and more likely to be populated by the <i>Xist</i> cluster. Intriguingly, there is no significant difference in the nearest distance of active and silenced genes to <i>Xist</i> foci within each group, yet active genes tend to be more distal to the center of the <i>Xist</i> foci cluster (<a id="crosref1105" href="#fig7">Figures 7</a>G and <a id="crosref1110" href="#figs7">S7</a>H). This finding is consistent with late genes being further away from the <i>Xist</i> cluster in the pre-Xi.</div><div id="p0195" role="paragraph">Chromosomal compaction significantly decreases the distances of both early and late genes to <i>Xist</i> foci in the pre-Xi to Xi transition, with a higher impact on late genes, which also exhibit the most dramatic repositioning (<a id="crosref1115" href="#fig7">Figures 7</a>F and 7H). The result of the conformational change is that early and late genes congregate and move closer to the center of the <i>Xist</i> cluster (<a id="crosref1120" href="#fig7">Figures 7</a>C and <a id="crosref1125" href="#figs7">S7</a>G). Consequently, the same number of <i>Xist</i> foci can progressively silence more genes. The gradual gene silencing during the XCI process can therefore be explained by the spatial organization and reconfiguration of the X chromosome that dictate the relationship of genes to the <i>Xist</i> cluster.</div><div id="p0200" role="paragraph">Finally, we explored how loss of compaction affects the organization of genes relative to <i>Xist</i> in cells expressing ΔB-<i>Xist</i>. All genes on the ΔB-Xi, regardless of their silencing kinetics, are at larger distances from the centroid of the <i>Xist</i> cluster and to each other, and the overall distances between early and late genes are enlarged compared to WT-Xi (<a id="crosref1130" href="#fig7">Figures 7</a>I–7L). However, nearest neighbor measurements between <i>Xist</i> foci and early or late genes revealed no significant difference for the ΔB and WT Xi, suggesting that <i>Xist</i> foci localize similarly to target regions (<a id="crosref1135" href="#fig7">Figure&nbsp;7</a>M). These findings indicate that the lack in compaction affects the reorganization of genes, which results in poor clustering of SMACs and inefficient silencing, particularly of late genes.</div></section></section><section id="sec-3"><h2>Discussion</h2><section id="sec-3-1"><h3>SMACs are the functional units of <i>Xist</i>-mediated XCI</h3><div id="p0205" role="paragraph">XCI is a powerful model for interrogating how lncRNA molecules can establish a functional nuclear compartment. Since its discovery, it has been thought that <i>Xist</i> progressively spreads on chromatin to associate with all target genes. This view was refined by the observation that <i>Xist</i> first localizes to sites in close spatial proximity to its transcription locus and then spreads chromosome wide (<span class="dropBlock reference-citations"><a id="body-ref-sref34-3" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-3" aria-controls="sref34-3">Engreitz et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref34-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Engreitz, J.M. ∙ Pandya-Jones, A. ∙ McDonel, P. ...</div><div><strong>The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome</strong></div><div><em>Science.</em> 2013; <strong>341</strong>:1237973</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.EngreitzA.Pandya-JonesP.McDonelA.ShishkinK.SirokmanC.SurkaS.KadriJ.XingA.GorenE.S.LanderThe+Xist+lncRNA+exploits+three-dimensional+genome+architecture+to+spread+across+the+X+chromosomeScience34120131237973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Owing to the focal accumulation of <i>Xist</i> revealed by super-resolution microscopy studies, it was proposed that the RNA and interacting proteins form ribonucleoprotein complexes that sample genes along the chromosome through a “hit-and-run” model (<span class="dropBlock reference-citations"><a id="body-ref-sref99-3" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-3" aria-controls="sref99-3">Sunwoo et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref99-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Sunwoo, H. ∙ Wu, J.Y. ∙ Lee, J.T.</div><div><strong>The Xist RNA-PRC2 complex at 20-nm resolution reveals a low Xist stoichiometry and suggests a hit-and-run mechanism in mouse cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4216-E4225</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.SunwooJ.Y.WuJ.T.LeeThe+Xist+RNA-PRC2+complex+at+20-nm+resolution+reveals+a+low+Xist+stoichiometry+and+suggests+a+hit-and-run+mechanism+in+mouse+cellsProc.+Natl.+Acad.+Sci.+USA1122015E4216E4225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our study shows that <i>Xist</i> foci are instead stably bound to chromatin and that they induce the <i>de novo</i> formation of SMACs. Each <i>Xist</i>-SMAC accumulates ∼35 copies of the ∼400 KDa protein SPEN. A comparison with other protein levels in SMACs suggests that other <i>Xist</i> effectors likely concentrate to much higher levels. The formation of SMACs induces a phase transition in the Xi, as SMACs surrounding stably bound <i>Xist</i> molecules create a sharp increase in protein density at the boundary of the Xi. Whether SMACs exhibit features of liquid-liquid phase separation and whether the progressive coalescence of chromatin regions induces polymer-polymer phase separation (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Frank and Rippe, 2020</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Frank, L. ∙ Rippe, K.</div><div><strong>Repetitive RNAs as Regulators of Chromatin-Associated Subcompartment Formation by Phase Separation</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 2020; <strong>432</strong>:4270-4286</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.FrankK.RippeRepetitive+RNAs+as+Regulators+of+Chromatin-Associated+Subcompartment+Formation+by+Phase+SeparationJ.%C2%A0Mol.+Biol.432202042704286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) remains to be determined.</div><div id="p0210" role="paragraph">Our results suggest that different binding environments in SMACs allow for both topological retention of proteins as well as their rapid exchange. IDRs are critical for the dynamic supramolecular aggregation of SPEN in SMACs, which is necessary for its catalytic domain SPOC to exert gene silencing. This finding is consistent with reports that catalytic rates of IDR-containing DNA modifying enzymes increase with crowding (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Kuznetsova et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Kuznetsova, I.M. ∙ Turoverov, K.K. ∙ Uversky, V.N.</div><div><strong>What macromolecular crowding can do to a protein</strong></div><div><em>Int. J. Mol. Sci.</em> 2014; <strong>15</strong>:23090-23140</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.M.KuznetsovaK.K.TuroverovV.N.UverskyWhat+macromolecular+crowding+can+do+to+a+proteinInt.+J.+Mol.+Sci.1520142309023140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref122" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122" aria-controls="sref122">Zimmerman and Pheiffer, 1983</a><div class="dropBlock__holder" data-db-target-of="sref122"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Zimmerman, S.B. ∙ Pheiffer, B.H.</div><div><strong>Macromolecular crowding allows blunt-end ligation by DNA ligases from rat liver or Escherichia coli</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1983; <strong>80</strong>:5852-5856</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.B.ZimmermanB.H.PheifferMacromolecular+crowding+allows+blunt-end+ligation+by+DNA+ligases+from+rat+liver+or+Escherichia+coliProc.+Natl.+Acad.+Sci.+USA80198358525856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Many <i>Xist</i>-interacting proteins contain IDRs and have the propensity to self-aggregate (<span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Cerase et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Cerase, A. ∙ Armaos, A. ∙ Neumayer, C. ...</div><div><strong>Phase separation drives X-chromosome inactivation: a hypothesis</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:331-334</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.CeraseA.ArmaosC.NeumayerP.AvnerM.GuttmanG.G.TartagliaPhase+separation+drives+X-chromosome+inactivation%3A+a+hypothesisNat.+Struct.+Mol.+Biol.262019331334" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79-3" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-3" aria-controls="sref79-3">Pandya-Jones et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref79-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Pandya-Jones, A. ∙ Markaki, Y. ∙ Serizay, J. ...</div><div><strong>A protein assembly mediates Xist localization and gene silencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:145-151</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Pandya-JonesY.MarkakiJ.SerizayT.ChitiashviliW.R.Mancia+LeonA.DamianovC.ChronisB.PappC.K.ChenR.McKeeA+protein+assembly+mediates+Xist+localization+and+gene+silencingNature5872020145151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Whether the IDRs of SPEN are involved solely in homotypic interactions and how IDRs of other <i>Xist</i>-interactors contribute to the formation and function of SMACs remain open questions. Interestingly, the binding to <i>Xist</i> is required for the IDR-dependent integration of SPEN into SMACs and may impart specificity to protein interactions within the Xi. RNA binding may induce folding of unstructured IDRs and enable “entry” into SMACs, consistent with observations for other IDR-containing proteins (<span class="dropBlock reference-citations"><a id="body-ref-sref105-2" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-2" aria-controls="sref105-2">Uversky, 2015</a><div class="dropBlock__holder" data-db-target-of="sref105-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Uversky, V.N.</div><div><strong>The multifaceted roles of intrinsic disorder in protein complexes</strong></div><div><em>FEBS Lett.</em> 2015; <strong>589</strong>:2498-2506</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.N.UverskyThe+multifaceted+roles+of+intrinsic+disorder+in+protein+complexesFEBS+Lett.58919+Pt+A201524982506" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-3-2"><h3>A supramolecular aggregation-based model of XCI</h3><div id="p0215" role="paragraph">Our work yields a revised model of how <i>Xist</i> establishes the Xi&nbsp;compartment and orchestrates gradual transcriptional silencing (<a id="crosref1180" href="#fig7">Figure&nbsp;7</a>N). Through expression, diffusion, sequestration, and degradation (see section “Expression-diffusion-degradation model for <i>Xist</i> confinement” in <a id="crosref1185" href="#mmc3">Methods S1</a>), two <i>Xist</i> transcripts become localized and tightly bound to chromatin, seeding SMACs at 50 regions that are proximal to the <i>Xist</i> locus and enriched for rapidly silencing genes. By establishing high local concentrations of transiently binding effector proteins in SMACs, <i>Xist</i> induces gradients of silencing proteins, most importantly of SPEN, that can act at genomic locations on the X without their continuous association with <i>Xist</i>. This process initiates silencing on the pre-Xi. The high concentration of PRC1 and likely other architectural regulators brought about by SMACs progressively induces higher-order chromatin changes and compaction. Compaction promotes the overall densification of genes under the SMACs cluster, enabling a constant number of SMACs to gain access to an increasing number of genes. Consequently, a higher concentration of SPEN is present in the vicinity of more genes and silencing expands across the entire X. However, the presence of a SMAC per se is not sufficient to induce effective silencing as genomic regions that are poorly crowded by SMACs silence less efficiently. By showing that SMAC formation and chromatin reconfiguration are interdependent mechanisms to achieve robust gene silencing, our model fills the knowledge gap of how different repressive pathways cooperate in XCI.</div></section><section id="sec-3-3"><h3>Implications beyond <i>Xist</i></h3><div id="p0220" role="paragraph">Phase separation has recently emerged as a much-debated mechanism in the field of gene regulation (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">McSwiggen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>McSwiggen, D.T. ∙ Mir, M. ∙ Darzacq, X. ...</div><div><strong>Evaluating phase separation in live cells: diagnosis, caveats, and functional consequences</strong></div><div><em>Genes Dev.</em> 2019; <strong>33</strong>:1619-1634</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.T.McSwiggenM.MirX.DarzacqR.TjianEvaluating+phase+separation+in+live+cells%3A+diagnosis%2C+caveats%2C+and+functional+consequencesGenes+Dev.33201916191634" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Yet, the functional role of condensates in gene regulation remains largely undefined. Macromolecular crowding as the mode of heterochromatin formation, described here, expands transcriptional control beyond the seeding molecule. This mechanism may be particularly important for gene regulatory RNAs, typically expressed at low numbers relative to their targets (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Cabili et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cabili, M.N. ∙ Dunagin, M.C. ∙ McClanahan, P.D. ...</div><div><strong>Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:20</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.N.CabiliM.C.DunaginP.D.McClanahanA.BiaeschO.Padovan-MerharA.RegevJ.L.RinnA.RajLocalization+and+abundance+analysis+of+human+lncRNAs+at+single-cell+and+single-molecule+resolutionGenome+Biol.16201520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29-2" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-2" aria-controls="sref29-2">Derrien et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref29-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Derrien, T. ∙ Johnson, R. ∙ Bussotti, G. ...</div><div><strong>The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:1775-1789</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.DerrienR.JohnsonG.BussottiA.TanzerS.DjebaliH.TilgnerG.GuernecD.MartinA.MerkelD.G.KnowlesThe+GENCODE+v7+catalog+of+human+long+noncoding+RNAs%3A+analysis+of+their+gene+structure%2C+evolution%2C+and+expressionGenome+Res.22201217751789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Intriguingly, other lncRNAs have also been found to induce spreading of Polycomb complexes (<span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Schertzer et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Schertzer, M.D. ∙ Braceros, K.C.A. ∙ Starmer, J. ...</div><div><strong>lncRNA-Induced Spread of Polycomb Controlled by Genome Architecture, RNA Abundance, and CpG Island DNA</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:523-537.e10</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.D.SchertzerK.C.A.BracerosJ.StarmerR.E.CherneyD.M.LeeG.SalazarM.JusticeS.R.BischoffD.O.CowleyP.AriellncRNA-Induced+Spread+of+Polycomb+Controlled+by+Genome+Architecture%2C+RNA+Abundance%2C+and+CpG+Island+DNAMol.+Cell752019523537.e10" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), suggesting that a common mechanism in the organization of an efficient repressive nuclear compartment may involve enzymes that induce transcriptional repression together with regulators of chromatin architecture.</div></section><section id="sec-3-4"><h3>Limitations of the study</h3><div id="p0225" role="paragraph">In this study, we applied super-resolution imaging in combination with kinetic modeling, genomic approaches, and functional perturbations to investigate fundamental principles of RNA-seeded nuclear compartmentalization. While multi-spectral quantitative super-resolution imaging allowed us to explore the spatial relationships of RNA, DNA, and protein in individual cells, it is subject to physical limitations. With 3D-SIM, we can resolve the distribution of XCI effectors down to few hundreds of kb along the genome that does not allow us to examine enrichment at specific genes, which would require even higher resolution (<span class="dropBlock reference-citations"><a id="body-ref-sref117" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117" aria-controls="sref117">Xie and Liu, 2021</a><div class="dropBlock__holder" data-db-target-of="sref117"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Xie, L. ∙ Liu, Z.</div><div><strong>Single-cell imaging of genome organization and dynamics</strong></div><div><em>Mol. Syst. Biol.</em> 2021; <strong>17</strong>:e9653</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.XieZ.LiuSingle-cell+imaging+of+genome+organization+and+dynamicsMol.+Syst.+Biol.172021e9653" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, due to the limited number of individual fluorophores that can be employed, it is not possible to simultaneously detect several SMAC proteins <i>Xist</i>, genes, and gene transcripts to directly determine what changes trigger the switch from an active to a repressed state. Additionally, fixation, permeabilization, and heat-denaturation applied in FISH or immunodetection experiments solubilize a considerable amount&nbsp;of protein, thus, the protein levels in SMACs is likely underestimated. Finally, although a sharp boundary needs to be determined in segmentation-based image analysis, as employed in our study, SMACs are dynamic, formed by rapidly exchanging proteins and likely do not adopt defined, but rather graded, distributed structures. Despite these limitations, super-resolution microscopy was critical for disentangling the processes of gene silencing and chromatin reconfiguration and allowed us to distinguish <i>Xist</i> from its protein interactors as functionally distinct entities in the Xi space.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0230" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Rabbit monoclonal anti-CUG-BP1</td><td>Abcam</td><td>RRID:<a id="intref0010" href="https://antibodyregistry.org/AB_11141441" target="_blank">AB_11141441</a>;<br>Cat #: ab129115</td></tr><tr><td>Rabbit polyclonal anti-hnRNP-K</td><td>Bethyl</td><td>RRID:<a id="intref0015" href="https://antibodyregistry.org/AB_530281" target="_blank">AB_530281</a>;<br>Cat #: A300-678A</td></tr><tr><td>Rabbit polyclonal anti-MATR3</td><td>Bethyl</td><td>RRID:<a id="intref0020" href="https://antibodyregistry.org/AB_2141651" target="_blank">AB_2141651</a>;<br>Cat #: IHC-00081</td></tr><tr><td>Rabbit polyclonal anti-RYBP (DEDAF)</td><td>Millipore Sigma</td><td>RRID:<a id="intref0025" href="https://antibodyregistry.org/AB_2285466" target="_blank">AB_2285466</a>;<br>Cat #: AB3637</td></tr><tr><td>Rabbit monoclonal anti-EZH2</td><td>Cell signaling Technology</td><td>RRID:<a id="intref0030" href="https://antibodyregistry.org/AB_10694683" target="_blank">AB_10694683</a>;<br>Cat #: 5246</td></tr><tr><td>Rabbit polyclonal anti-CIZ1</td><td>Novus Biologicals</td><td>RRID:<a id="intref0035" href="https://antibodyregistry.org/AB_1048573" target="_blank">AB_1048573</a>;<br>Cat #: NB100-74624</td></tr><tr><td>Rabbit polyclonal anti-histone H3 phospho-Serine 10</td><td>Active Motif</td><td>RRID:<a id="intref0040" href="https://antibodyregistry.org/AB_2793206" target="_blank">AB_2793206</a>;<br>Cat #: 39253</td></tr><tr><td>Rabbit polyclonal anti-GFP</td><td>Abcam</td><td>Cat #: ab6556</td></tr><tr><td>Donkey anti-rabbit IgG Alexa Fluor 488</td><td>Invitrogen</td><td>RRID: <a id="intref0045" href="https://antibodyregistry.org/AB_2535792" target="_blank">AB_2535792</a>;<br>Cat #: A-21206</td></tr><tr><td>Donkey anti-rabbit IgG CF568</td><td>Sigma</td><td>Cat #: SAB4600076</td></tr><tr class="rowsep"><td>Goat anti-rabbit IgG Alexa Fluor 647</td><td>Life Technologies</td><td>RRID: <a id="intref0050" href="https://antibodyregistry.org/AB_2535813" target="_blank">AB_2535813</a>;<br>Cat #: A21245</td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td>Stellar Competent Cells</td><td>Clontech</td><td>Cat #: 636766</td></tr><tr><td>One Shot TOP10 Chemically Competent</td><td>Thermo Fisher Scientific</td><td>Cat #: C404003</td></tr><tr class="rowsep"><td>10-beta Competent E.coli (High Efficiency)</td><td>NEB</td><td>Cat #: C3019H</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>HaloTag Ligands for Super Resolution Microscopy JF 549</td><td>Promega</td><td>Cat #: GA1111</td></tr><tr><td>HaloTag Ligands for Super Resolution Microscopy JF 646</td><td>Promega</td><td>Cat #: GA1121</td></tr><tr><td>HaloTag TMR Ligand</td><td>Promega</td><td>Cat #: PRG8252</td></tr><tr><td>HaloLink Resin</td><td>Promega</td><td>Cat #: G1912</td></tr><tr><td>Aminoallyl dUTP</td><td>Sigma-Aldrich</td><td>Cat #: A 0410</td></tr><tr><td>ATTO 488-NHS Ester</td><td>Sigma-Aldrich</td><td>Cat #: 41698-1MG-F</td></tr><tr><td>Alexa Fluor 568 NHS Ester</td><td>Thermo Fisher Scientific</td><td>Cat #: A20003</td></tr><tr><td>Cy3 Mono-NHS Ester</td><td>VWR</td><td>Cat #:PA13101</td></tr><tr><td>Cy5 NHS-Ester</td><td>VWR</td><td>Cat #: 95017-506</td></tr><tr><td>CF Dye Azide 568</td><td>Biotium</td><td>Cat #: 92082</td></tr><tr><td>Texas Red-X, Succinimidyl Ester, mixed isomers</td><td>Thermo Fisher Scientific</td><td>Cat #: T6134</td></tr><tr><td>ProLong Live Antifade Reagent</td><td>Thermo Fisher Scientific</td><td>Cat #: P36975</td></tr><tr><td>DNase I recombinant, RNase-free</td><td>Sigma-Aldrich</td><td>Cat #: 4716728001</td></tr><tr><td>DNA Polymerase I (10&nbsp;U/μL)</td><td>Thermo Fisher Scientific</td><td>Cat #: EP0042</td></tr><tr><td>Phusion High-Fidelity DNA Polymerase (2&nbsp;U/μL)</td><td>Thermo Fisher Scientific</td><td>Cat #: F530L</td></tr><tr><td>SuperScript III Reverse Transcriptase</td><td>Life Technologies</td><td>Cat #: 18080-044</td></tr><tr><td>RNase H</td><td>NEB</td><td>Cat #: M0297L</td></tr><tr><td>TURBO DNase (2&nbsp;U/μL)</td><td>Thermo Fisher Scientific</td><td>Cat #: AM2238</td></tr><tr><td>KAPA HiFi hot start Taq</td><td>Kapa Biosystems</td><td>Cat #: kk2502</td></tr><tr><td>Dynabeads MyOne Streptavidin C1</td><td>Life Technologies</td><td>Cat #: 65002</td></tr><tr><td>Cytiva Sera-Mag SpeedBeads Carboxyl Magnetic Beads, hydrophobic</td><td>Fisher Scientific</td><td>Cat #: 09-981-123</td></tr><tr><td>DSG (disuccinimidyl glutarate)</td><td>Thermo Fisher Scientific</td><td>Cat #: 20593</td></tr><tr><td>Proteinase K, Molecular Biology Grade</td><td>NEB</td><td>Cat #: P8107S</td></tr><tr><td>RNase Inhibitor, Murine</td><td>NEB</td><td>Cat #: M0314L</td></tr><tr><td>T4 Polynucleotide Kinase</td><td>NEB</td><td>Cat #: M0201L</td></tr><tr><td>FastAP Thermosensitive Alkaline Phosphatase</td><td>Thermo Fisher Scientific</td><td>Cat #: EF0654</td></tr><tr><td>Protease Inhibitor Cocktail</td><td>Promega</td><td>Cat #: G6521</td></tr><tr><td>ProTEV Plus</td><td>Promega</td><td>Cat #: V6101</td></tr><tr><td>Vectashield</td><td>Vector Labs</td><td>Cat #: H-1000</td></tr><tr><td>DAPI</td><td>Thermo Fisher Scientific</td><td>Cat #: D1306</td></tr><tr><td>SpCas9 2NLS Nuclease (1000 pmol)</td><td>Synthego</td><td>N/A</td></tr><tr><td>DMSO</td><td>Sigma-Aldrich</td><td>Cat #: D2650</td></tr><tr><td>DMEM/F-12, HEPES, no phenol red</td><td>Life Technologies</td><td>Cat #: 11039021</td></tr><tr><td>Geltrex LDEV-Free Reduced Growth Factor Basement Membrane Matrix</td><td>Life Technologies</td><td>Cat #: A1413202</td></tr><tr><td>N2 Supplement (100X)</td><td>Thermo Fisher Scientific</td><td>Cat #: 17502048</td></tr><tr><td>B27 Supplement (50X), minus Vitamin A</td><td>Thermo Fisher Scientific</td><td>Cat #:12587010</td></tr><tr><td>Animal Free Human Activin-A(e.coli)</td><td>PeproTech</td><td>Cat #: AF-120-14E</td></tr><tr><td>Recombinant Human FGF-basic (FGF)</td><td>PeproTech</td><td>Cat #: 100-18B</td></tr><tr><td>Recombinant Human EGF Protein, CF</td><td>RnD (Perseus Proteomics)</td><td>Cat #: 2028-EG-200</td></tr><tr><td>Mouse LIF</td><td>Homemade</td><td>N/A</td></tr><tr><td>PD 0325901</td><td>Fisher Scientific</td><td>Cat #: 4192</td></tr><tr><td>CHIR99021</td><td>Stemgent</td><td>Cat #: 04-0004</td></tr><tr><td>Neurobasal Medium</td><td>Life Technologies</td><td>Cat #: 21103-049</td></tr><tr><td>DMEM: F12</td><td>Life Technologies</td><td>Cat #: 11320-082</td></tr><tr><td>Knockout DMEM</td><td>Life Technologies</td><td>Cat #: 10829018</td></tr><tr><td>DMEM</td><td>Life Technologies</td><td>Cat #: 11995073</td></tr><tr><td>FBS</td><td>Thermo Fisher Scientific</td><td>Cat #: 10437028</td></tr><tr><td>Glutamax I</td><td>Life Technologies</td><td>Cat #: 35050061</td></tr><tr><td>MEM NEAA</td><td>Life Technologies</td><td>Cat #: 11140-050</td></tr><tr><td>20x Penicillin/Streptomycin</td><td>Life Technologies</td><td>Cat #: 15140-163</td></tr><tr><td>Gelatin from porcine skin, Type A</td><td>Sigma-Aldrich</td><td>Cat #: G2500</td></tr><tr><td>Accutase (cell dissociation)</td><td>Life Technologies</td><td>Cat #: A11105-01</td></tr><tr><td>Trypsin</td><td>Life Technologies</td><td>Cat #: 25200114</td></tr><tr><td>DPBS</td><td>Fisher Scientific</td><td>Cat #: SH3002802</td></tr><tr><td>UltraPure BSA (50&nbsp;mg/mL)</td><td>Thermo Fisher Scientific</td><td>Cat #: AM2616</td></tr><tr><td>Bovine Serum Albumin (BSA)</td><td>Sigma-Aldrich</td><td>Cat #: A7906</td></tr><tr><td>Fish Skin Gelatin</td><td>Sigma-Aldrich</td><td>Cat #: G7765</td></tr><tr><td>Triton X-100</td><td>Sigma-Aldrich</td><td>Cat #: T8787</td></tr><tr><td>Tween 20</td><td>Sigma-Aldrich</td><td>Cat #: P9416</td></tr><tr><td>20X SSC</td><td>Life Technologies</td><td>Cat #:AM9765</td></tr><tr><td>PBS (10X), pH 7.4</td><td>Life Technologies</td><td>Cat #: 70011069</td></tr><tr><td>Dextran sulfate sodium salt</td><td>Sigma-Aldrich</td><td>Cat #: D8906</td></tr><tr><td>Formamide</td><td>Fisher Scientific</td><td>Cat #: F84-1</td></tr><tr><td>Omnipur deionized formamide</td><td>VWR</td><td>Cat #: EM-4610</td></tr><tr><td>Lipofectamine 3000</td><td>Life Technologies</td><td>Cat #: L3000015</td></tr><tr><td>RNaseOUT</td><td>Life Technologies</td><td>Cat #: 10777019</td></tr><tr><td>Opti-MEM I Reduced Serum Medium</td><td>Thermo Fisher Scientific</td><td>Cat #: 31985070</td></tr><tr><td>Tri Reagent</td><td>Zymo Research</td><td>Cat #: R2050-1-200</td></tr><tr><td>Thermo Scientific Pierce Methanol free Formaldehyde Ampules</td><td>Thermo Fisher Scientific</td><td>Cat #: 28908</td></tr><tr><td>Ribonucleoside Vanadyl Complex</td><td>NEB</td><td>Cat #: S1402S</td></tr><tr><td>Actinomycin D</td><td>Sigma-Aldrich</td><td>Cat #: A9415</td></tr><tr><td>Indole-3-Acetic Acid</td><td>Cayman Chemical Company</td><td>Cat #: 16954</td></tr><tr class="rowsep"><td>Glycine,bioultra, for molecular biology, <sup>3</sup>99.0% (NT)</td><td>Sigma-Aldrich</td><td>Cat #: 50046</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>TrueSeq Stranded mRNA Library Prep Kit</td><td>Illumina</td><td>Cat #: 20020594</td></tr><tr><td>Chromium single cell 3¢ reagent kit V3.1</td><td>10xGenomics</td><td>Cat #: PN-1000121</td></tr><tr><td>Click-IT EdU Cell Proliferation Kit for Imaging</td><td>Thermo Fisher Scientific</td><td>Cat #: C10337</td></tr><tr><td>In-Fusion HD Cloning</td><td>Clontech</td><td>Cat #: 639649</td></tr><tr><td>BioPrime Array CGH Labeling System</td><td>Life Technologies</td><td>Cat #: 18095011</td></tr><tr><td>Quick Ligation Kit</td><td>NEB</td><td>Cat #: M2200L</td></tr><tr><td>NEBNext Ultra End Repair/dA-tailing</td><td>NEB</td><td>Cat #: E7442L</td></tr><tr><td>4D-NucleofectorTM X Kit</td><td>Lonza</td><td>Cat #: V4XP-3024</td></tr><tr><td>P3 Primary Cell 4D-Nucleofector Kit S</td><td>Lonza</td><td>Cat #: V4XP-3032</td></tr><tr><td>GeneJET Plasmid Miniprep Kit</td><td>Thermo Fisher Scientific</td><td>Cat #: FERK0503</td></tr><tr><td>Direct-zol RNA MiniPrep Kit with TRI-Reagent, Zymo-Spin IIC Columns</td><td>Zymo Research</td><td>Cat #: R2051</td></tr><tr><td>MinElute Gel Extraction Kit</td><td>QIAGEN</td><td>Cat #: 28606</td></tr><tr><td>NucleoBond Xtra BAC</td><td>Clontech</td><td>Cat #: 740436.25</td></tr><tr class="rowsep"><td>NucleoBond Xtra Maxi</td><td>Clontech</td><td>Cat #: 740414.50</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Bulk RNA-seq, scRNA-seq, CLAP-seq, RAP-seq</td><td>This study</td><td>GEO: GSE181236</td></tr><tr><td><i>Mus musculus</i> Cas genome sequence</td><td>EMBL-EBI</td><td>ENA: ERP000042</td></tr><tr><td><i>Mus musculus</i> 129 genome sequence</td><td>EMBL-EBI</td><td>SRA-XML: SRX037820</td></tr><tr><td>Hi-C and RNA-seq allelic counts in Smchd1−/− female NPCs</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref110-3" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-3" aria-controls="sref110-3">Wang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref110-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Wang, C.Y. ∙ Jegu, T. ∙ Chu, H.P. ...</div><div><strong>SMCHD1 Merges Chromosome Compartments and Assists Formation of Super-Structures on the Inactive X</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:406-421.e25</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.Y.WangT.JeguH.P.ChuH.J.OhJ.T.LeeSMCHD1+Merges+Chromosome+Compartments+and+Assists+Formation+of+Super-Structures+on+the+Inactive+XCell1742018406421.e25" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>GEO: GSE99991</td></tr><tr class="rowsep"><td>RNA-seq from SPOC-Bgl tethering to Xist</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref31-10" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-10" aria-controls="sref31-10">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>GEO: GSE131784</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>Mouse ESCs 129S4/SvJae/castaneus F1 2-1</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref80-1" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-1" aria-controls="sref80-1">Panning et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref80-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Panning, B. ∙ Dausman, J. ∙ Jaenisch, R.</div><div><strong>X chromosome inactivation is mediated by Xist RNA stabilization</strong></div><div><em>Cell.</em> 1997; <strong>90</strong>:907-916</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.PanningJ.DausmanR.JaenischX+chromosome+inactivation+is+mediated+by+Xist+RNA+stabilizationCell901997907916" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>XistMS2-GFP ESCs (F1 2-1-XIST24MS2/MCP-GFP)</td><td>This study</td><td>N/A</td></tr><tr><td>XistMS2-GFP/R26CIZ1mCherry</td><td>This study</td><td>N/A</td></tr><tr><td>XistMS2-GFP/R26CIZ1Halo</td><td>This study</td><td>N/A</td></tr><tr><td>XistMS2-GFP/R26CELF1mCherry</td><td>This study</td><td>N/A</td></tr><tr><td>XistMS2-GFP/R26PCGF5Halo</td><td>This study</td><td>N/A</td></tr><tr><td>XistMS2-GFP/R26PTBP1Halo</td><td>This study</td><td>N/A</td></tr><tr><td>XistMS2-GFP/PyP-CAG-HaloSPEN</td><td>This study</td><td>N/A</td></tr><tr><td>XistMS2-GFP/PyP-CAG-HaloΔIDRSPEN</td><td>This study</td><td>N/A</td></tr><tr><td>XistMS2-GFP/PyP-CAG-HaloΔRRMSPEN</td><td>This study</td><td>N/A</td></tr><tr><td>Mouse ESCs 129S4/SvJae/castaneus F1 2-1-XIST12MS2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Jonkers et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Jonkers, I. ∙ Monkhorst, K. ∙ Rentmeester, E. ...</div><div><strong>Xist RNA is confined to the nuclear territory of the silenced X chromosome throughout the cell cycle</strong></div><div><em>Mol. Cell. Biol.</em> 2008; <strong>28</strong>:5583-5594</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.JonkersK.MonkhorstE.RentmeesterJ.A.GrootegoedF.GrosveldJ.GribnauXist+RNA+is+confined+to+the+nuclear+territory+of+the+silenced+X+chromosome+throughout+the+cell+cycleMol.+Cell.+Biol.28200855835594" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>129S4ΔΒ/SvJae/castaneus F1 2-1 ESCs-XIST12MS2</td><td>This study</td><td>N/A</td></tr><tr><td>129S4ΔΒ/SvJae/castaneus F1 2-1 ESCs-XIST12MS2-R26SPENHalo</td><td>This study</td><td>N/A</td></tr><tr><td>pSM33 tetO-Xist V6.5 male mouse ESCs</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref34-4" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-4" aria-controls="sref34-4">Engreitz et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref34-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Engreitz, J.M. ∙ Pandya-Jones, A. ∙ McDonel, P. ...</div><div><strong>The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome</strong></div><div><em>Science.</em> 2013; <strong>341</strong>:1237973</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.EngreitzA.Pandya-JonesP.McDonelA.ShishkinK.SirokmanC.SurkaS.KadriJ.XingA.GorenE.S.LanderThe+Xist+lncRNA+exploits+three-dimensional+genome+architecture+to+spread+across+the+X+chromosomeScience34120131237973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>pSM9 tetO-XistΔΒ V6.5 male mouse ESCs</td><td>This study</td><td>N/A</td></tr><tr><td>36.11 tetO-Xist cDNA transgene chr 11, male mouse ESCs</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref116-1" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116-1" aria-controls="sref116-1">Wutz and Jaenisch, 2000</a><div class="dropBlock__holder" data-db-target-of="sref116-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Wutz, A. ∙ Jaenisch, R.</div><div><strong>A shift from reversible to irreversible X inactivation is triggered during ES cell differentiation</strong></div><div><em>Mol. Cell.</em> 2000; <strong>5</strong>:695-705</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.WutzR.JaenischA+shift+from+reversible+to+irreversible+X+inactivation+is+triggered+during+ES+cell+differentiationMol.+Cell52000695705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>36.11 tetO-Xist cDNA transgene chr 11, male mouse ESCs- R26SPENHalo</td><td>This study</td><td>N/A</td></tr><tr><td>Mouse ESCs TX1072-Spen-GFP/Spen-GFP-BglXist-mCherry</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref31-11" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-11" aria-controls="sref31-11">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>TX1072 ESCs -Spen-Halo/Spen-Halo</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref31-12" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-12" aria-controls="sref31-12">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>TX1072 ESCs-Spen-GFP/Spen-GFP</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref31-13" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-13" aria-controls="sref31-13">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>TX1072 ESCs-Spen-AID-GFP/Spen-AID-GFP</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref31-14" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-14" aria-controls="sref31-14">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>TX1072 ESCs -ΔIDRSpen-GFP/ΔIDRSpen-GFP</td><td>This study</td><td>N/A</td></tr><tr><td>TX1072 ESCs-Spen-AID-GFP/Spen-AID-GFP-R26Spen-Halo</td><td>This study</td><td>N/A</td></tr><tr><td>TX1072 ESCs-Spen-AID-GFP/Spen-AID-GFP-R26ΔIDRSpen-Halo</td><td>This study</td><td>N/A</td></tr><tr><td>C127I</td><td>ATCC</td><td>Cat #: CRL-1616</td></tr><tr><td>Human fibroblasts - Abnormal Xi-Chromosome deletion</td><td>Coriell</td><td>Cat #: GM3827</td></tr><tr><td>Human fibroblasts - Abnormal Xi-Turner Syndrome</td><td>Coriell</td><td>Cat #: GM00735</td></tr><tr><td>Human fibroblasts - Abnormal Xi-Dicentric chromosome</td><td>Coriell</td><td>Cat #: GM06960</td></tr><tr class="rowsep"><td>Human fibroblasts - Abnormal Xi-Dicentric chromosome</td><td>Coriell</td><td>Cat #: GM07213</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr class="rowsep"><td>DR4 mice (for feeders)</td><td>The Jackson Laboratory</td><td>Cat #: 003208</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr class="rowsep"><td>Fluorescently labeled oligonucleotides used in oligoFISH, probes for RAP-seq and gRNAs for the IDRs deletion of SPEN</td><td>This study</td><td>See <a id="crosref1265" href="#mmc2">Table S2</a></td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>pMS2-GFP</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Fusco et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Fusco, D. ∙ Accornero, N. ∙ Lavoie, B. ...</div><div><strong>Single mRNA molecules demonstrate probabilistic movement in living mammalian cells</strong></div><div><em>Curr. Biol.</em> 2003; <strong>13</strong>:161-167</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.FuscoN.AccorneroB.LavoieS.M.ShenoyJ.M.BlanchardR.H.SingerE.BertrandSingle+mRNA+molecules+demonstrate+probabilistic+movement+in+living+mammalian+cellsCurr.+Biol.132003161167" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene plasmid cat #: 27121</td></tr><tr><td>pCR4-24XMS2SL-stable</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Bertrand et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bertrand, E. ∙ Chartrand, P. ∙ Schaefer, M. ...</div><div><strong>Localization of ASH1 mRNA particles in living yeast</strong></div><div><em>Mol. Cell.</em> 1998; <strong>2</strong>:437-445</div></div><div class="external-links"><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.10.022&amp;key=2-s2.0-0032185810" target="_blank" aria-label="Open Scopus in new tab">Scopus (1168)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.BertrandP.ChartrandM.SchaeferS.M.ShenoyR.H.SingerR.M.LongLocalization+of+ASH1+mRNA+particles+in+living+yeastMol.+Cell21998437445" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene plasmid cat #: 31865</td></tr><tr><td>pBglII5k plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Jonkers et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Jonkers, I. ∙ Monkhorst, K. ∙ Rentmeester, E. ...</div><div><strong>Xist RNA is confined to the nuclear territory of the silenced X chromosome throughout the cell cycle</strong></div><div><em>Mol. Cell. Biol.</em> 2008; <strong>28</strong>:5583-5594</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.JonkersK.MonkhorstE.RentmeesterJ.A.GrootegoedF.GrosveldJ.GribnauXist+RNA+is+confined+to+the+nuclear+territory+of+the+silenced+X+chromosome+throughout+the+cell+cycleMol.+Cell.+Biol.28200855835594" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>pBglII5k-24xMS2 plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>pBS31 (pgkATGfrt) plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Beard et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Beard, C. ∙ Hochedlinger, K. ∙ Plath, K. ...</div><div><strong>Efficient method to generate single-copy transgenic mice by site-specific integration in embryonic stem cells</strong></div><div><em>Genesis.</em> 2006; <strong>44</strong>:23-28</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.BeardK.HochedlingerK.PlathA.WutzR.JaenischEfficient+method+to+generate+single-copy+transgenic+mice+by+site-specific+integration+in+embryonic+stem+cellsGenesis4420062328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>pBS32 plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Minkovsky et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Minkovsky, A. ∙ Sahakyan, A. ∙ Rankin-Gee, E. ...</div><div><strong>The Mbd1-Atf7ip-Setdb1 pathway contributes to the maintenance of X chromosome inactivation</strong></div><div><em>Epigenetics Chromatin.</em> 2014; <strong>7</strong>:12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MinkovskyA.SahakyanE.Rankin-GeeG.BonoraS.PatelK.PlathThe+Mbd1-Atf7ip-Setdb1+pathway+contributes+to+the+maintenance+of+X+chromosome+inactivationEpigenetics+Chromatin7201412" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>pBS32-MCP-CIZ1 plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref79-4" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-4" aria-controls="sref79-4">Pandya-Jones et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref79-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Pandya-Jones, A. ∙ Markaki, Y. ∙ Serizay, J. ...</div><div><strong>A protein assembly mediates Xist localization and gene silencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:145-151</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Pandya-JonesY.MarkakiJ.SerizayT.ChitiashviliW.R.Mancia+LeonA.DamianovC.ChronisB.PappC.K.ChenR.McKeeA+protein+assembly+mediates+Xist+localization+and+gene+silencingNature5872020145151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>FRT-neo plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Beard et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Beard, C. ∙ Hochedlinger, K. ∙ Plath, K. ...</div><div><strong>Efficient method to generate single-copy transgenic mice by site-specific integration in embryonic stem cells</strong></div><div><em>Genesis.</em> 2006; <strong>44</strong>:23-28</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.BeardK.HochedlingerK.PlathA.WutzR.JaenischEfficient+method+to+generate+single-copy+transgenic+mice+by+site-specific+integration+in+embryonic+stem+cellsGenesis4420062328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>I3-01-ct60GFP plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Hsia et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.HsiaJ.B.BaleS.GonenD.ShiW.ShefflerK.K.FongU.NattermannC.XuP.S.HuangR.RavichandranDesign+of+a+hyperstable+60-subunit+protein+dodecahedron.+%5Bcorrected%5DNature5352016136139" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>pBS32-cage-60GFP plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>MXS_PGK::rtTA3-bGHpA</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Sladitschek and Neveu, 2015</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Sladitschek, H.L. ∙ Neveu, P.A.</div><div><strong>MXS-Chaining: A Highly Efficient Cloning Platform for Imaging and Flow Cytometry Approaches in Mammalian Systems</strong></div><div><em>PLoS ONE.</em> 2015; <strong>10</strong>:e0124958</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.L.SladitschekP.A.NeveuMXS-Chaining%3A+A+Highly+Efficient+Cloning+Platform+for+Imaging+and+Flow+Cytometry+Approaches+in+Mammalian+SystemsPLoS+ONE102015e0124958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene plasmid cat #: 62446</td></tr><tr><td>pBS31-MCP-GFP-rtTA3 plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>H2B-mCherry</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Nam and Benezra, 2009</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Nam, H.S. ∙ Benezra, R.</div><div><strong>High levels of Id1 expression define B1 type adult neural stem cells</strong></div><div><em>Cell Stem Cell.</em> 2009; <strong>5</strong>:515-526</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.S.NamR.BenezraHigh+levels+of+Id1+expression+define+B1+type+adult+neural+stem+cellsCell+Stem+Cell52009515526" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene plasmid cat #: 20972</td></tr><tr><td>EasyFusion Halo-mAID</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Gu et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Gu, B. ∙ Posfai, E. ∙ Rossant, J.</div><div><strong>Efficient generation of targeted large insertions by microinjection into two-cell-stage mouse embryos</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:632-637</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.GuE.PosfaiJ.RossantEfficient+generation+of+targeted+large+insertions+by+microinjection+into+two-cell-stage+mouse+embryosNat.+Biotechnol.362018632637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene plasmid cat #: 112852</td></tr><tr><td>R26-SA-EGFP-puro</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Blelloch et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Blelloch, R.H. ∙ Hochedlinger, K. ∙ Yamada, Y. ...</div><div><strong>Nuclear cloning of embryonal carcinoma cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:13985-13990</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.H.BlellochK.HochedlingerY.YamadaC.BrennanM.KimB.MintzL.ChinR.JaenischNuclear+cloning+of+embryonal+carcinoma+cellsProc.+Natl.+Acad.+Sci.+USA10120041398513990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene plasmid cat #: 26890</td></tr><tr><td>pYM215-R26-SA/SD-puro</td><td>This study</td><td>N/A</td></tr><tr><td>R26-CELF1-mCherry-puro plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>R26-PCGF5-Halo-puro plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>R26-PTBP1-Halo-puro plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>R26-CIZ1-Halo-puro plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>R26-CIZ1-mCherry-puro plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>R26-H2B-Halo-puro plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>pYM300-R26-Halo-SPEN-hygro plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>pYM301-R26-Halo-ΔIDR-SPEN-hygro plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>PyPP-CAG-Halo-full-length-Spen-V5 plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>PyPP-CAG-Halo-Spen-ΔIDR-V5 plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>PyPP-CAG-Halo-Spen-ΔRRM-V5 plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>full-length mSpen Entry Clone (Sp22)</td><td>Alexander Shiskin</td><td>N/A</td></tr><tr><td>pFD46-R26-SPEN-hygro plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref31-15" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-15" aria-controls="sref31-15">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>pFD82-Cas9D10A-gRNA1 plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref31-16" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-16" aria-controls="sref31-16">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>pFD83-Cas9D10A-gRNA2 plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref31-17" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-17" aria-controls="sref31-17">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>p15A-31-17.9kb Xist plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref79-5" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-5" aria-controls="sref79-5">Pandya-Jones et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref79-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Pandya-Jones, A. ∙ Markaki, Y. ∙ Serizay, J. ...</div><div><strong>A protein assembly mediates Xist localization and gene silencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:145-151</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Pandya-JonesY.MarkakiJ.SerizayT.ChitiashviliW.R.Mancia+LeonA.DamianovC.ChronisB.PappC.K.ChenR.McKeeA+protein+assembly+mediates+Xist+localization+and+gene+silencingNature5872020145151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>pCMV-Xist-PA</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref116-2" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116-2" aria-controls="sref116-2">Wutz and Jaenisch, 2000</a><div class="dropBlock__holder" data-db-target-of="sref116-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Wutz, A. ∙ Jaenisch, R.</div><div><strong>A shift from reversible to irreversible X inactivation is triggered during ES cell differentiation</strong></div><div><em>Mol. Cell.</em> 2000; <strong>5</strong>:695-705</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.WutzR.JaenischA+shift+from+reversible+to+irreversible+X+inactivation+is+triggered+during+ES+cell+differentiationMol.+Cell52000695705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene plasmid cat #: 26760</td></tr><tr><td>p13-5-Xist-Bdel plasmid</td><td>This study</td><td>N/A</td></tr><tr><td>pPGK-Cre-bpA</td><td>Klaus Rajewsky</td><td>Addgene plasmid cat #: 11543</td></tr><tr><td>FlpO plasmid</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Kranz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kranz, A. ∙ Fu, J. ∙ Duerschke, K. ...</div><div><strong>An improved Flp deleter mouse in C57Bl/6 based on Flpo recombinase</strong></div><div><em>Genesis.</em> 2010; <strong>48</strong>:512-520</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.KranzJ.FuK.DuerschkeS.WeidlichR.NaumannA.F.StewartK.AnastassiadisAn+improved+Flp+deleter+mouse+in+C57Bl%2F6+based+on+Flpo+recombinaseGenesis482010512520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>BAC plasmid used to generate AtrX probe</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: RP23-265D6</td></tr><tr><td>Fosmid plasmid used to generate Mecp2 probe</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: WI-894A5</td></tr><tr><td>Fosmid plasmid used to generate Rlim probe</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: WI1-2704K12</td></tr><tr><td>BAC plasmid used to generate multispectral X chromosome barcoding</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: RP23-53H15</td></tr><tr><td>BAC plasmid used to generate multispectral X chromosome barcoding</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: RP23-83J1</td></tr><tr><td>BAC plasmid used to generate multispectral X chromosome barcoding</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: RP23-451D5</td></tr><tr><td>BAC plasmid used to generate multispectral X chromosome barcoding</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: RP24-81K23</td></tr><tr><td>BAC plasmid used to generate multispectral X chromosome barcoding</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: RP24-374B8</td></tr><tr><td>BAC plasmid used to generate multispectral X chromosome barcoding</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: RP23-401G5</td></tr><tr><td>BAC plasmid used to generate multispectral X chromosome barcoding</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: RP23-104K18</td></tr><tr class="rowsep"><td>BAC plasmid to generate the Xist intron 1 probe</td><td>BacPac Consortium at Children’s Hospital Oakland Research Institute</td><td>Cat #: RP23-223G18</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Fiji</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref93-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-1" aria-controls="sref93-1">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref93-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.SchindelinI.Arganda-CarrerasE.FriseV.KaynigM.LongairT.PietzschS.PreibischC.RuedenS.SaalfeldB.SchmidFiji%3A+an+open-source+platform+for+biological-image+analysisNat.+Methods92012676682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0055" href="https://fiji.sc/" target="_blank">https://fiji.sc/</a></td></tr><tr><td>ImageJ</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Rueden et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Rueden, C.T. ∙ Schindelin, J. ∙ Hiner, M.C. ...</div><div><strong>ImageJ2: ImageJ for the next generation of scientific image data</strong></div><div><em>BMC Bioinformatics.</em> 2017; <strong>18</strong>:529</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.T.RuedenJ.SchindelinM.C.HinerB.E.DeZoniaA.E.WalterE.T.ArenaK.W.EliceiriImageJ2%3A+ImageJ+for+the+next+generation+of+scientific+image+dataBMC+Bioinformatics182017529" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0060" href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a></td></tr><tr><td>TrackMate</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref104-1" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-1" aria-controls="sref104-1">Tinevez et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref104-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Tinevez, J.Y. ∙ Perry, N. ∙ Schindelin, J. ...</div><div><strong>TrackMate: An open and extensible platform for single-particle tracking</strong></div><div><em>Methods.</em> 2017; <strong>115</strong>:80-90</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.Y.TinevezN.PerryJ.SchindelinG.M.HoopesG.D.ReynoldsE.LaplantineS.Y.BednarekS.L.ShorteK.W.EliceiriTrackMate%3A+An+open+and+extensible+platform+for+single-particle+trackingMethods11520178090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0065" href="https://imagej.net/plugins/trackmate/" target="_blank">https://imagej.net/plugins/trackmate/</a></td></tr><tr><td>3D ImageJ Suite</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Ollion et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Ollion, J. ∙ Cochennec, J. ∙ Loll, F. ...</div><div><strong>TANGO: a generic tool for high-throughput 3D image analysis for studying nuclear organization</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1840-1841</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.OllionJ.CochennecF.LollC.Escud%C3%A9T.BoudierTANGO%3A+a+generic+tool+for+high-throughput+3D+image+analysis+for+studying+nuclear+organizationBioinformatics29201318401841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0070" href="https://imagej.net/plugins/3d-imagej-suite/" target="_blank">https://imagej.net/plugins/3d-imagej-suite/</a></td></tr><tr><td>Python</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref107" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107" aria-controls="sref107">van Rossum and Drake, 2009</a><div class="dropBlock__holder" data-db-target-of="sref107"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>van Rossum, G. ∙ Drake, F.L.</div><div><strong>Python 3 Reference Manual</strong></div><div>Scotts Valley, CA, 2009</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.van+RossumF.L.DrakePython+3+Reference+Manual2009Scotts+ValleyCA" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0075" href="https://www.python.org/" target="_blank">https://www.python.org/</a></td></tr><tr><td>Google Colaboratory</td><td>Google Research</td><td><a id="intref0080" href="https://research.google.com/colaboratory" target="_blank">https://research.google.com/colaboratory</a></td></tr><tr><td>PyTrackmate</td><td>Hadrien Mary</td><td><a id="intref0085" href="https://github.com/hadim/pytrackmate" target="_blank">https://github.com/hadim/pytrackmate</a></td></tr><tr><td>pandas</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Reback et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Reback, J. ∙ McKinney, W. ∙ Mendel, B. ...</div><div><strong>pandas-dev/pandas: Pandas</strong></div><div><em>Zenodo.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.RebackW.McKinneyB.MendelJ.Van+den+BosscheT.AugspurgerP.CloudgfyoungS.HawkinsSinhrksM.Roeschkepandas-dev%2Fpandas%3A+PandasZenodo202010.5281%2Fzenodo.3509134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0090" href="https://pandas.pydata.org/" target="_blank">https://pandas.pydata.org/</a></td></tr><tr><td>NumPy</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Harris et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Harris, C.R. ∙ Millman, K.J. ∙ van der Walt, S.J. ...</div><div><strong>Array programming with NumPy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:357-362</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.R.HarrisK.J.MillmanS.J.van+der+WaltR.GommersP.VirtanenD.CournapeauE.WieserJ.TaylorS.BergN.J.SmithArray+programming+with+NumPyNature5852020357362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0095" href="https://numpy.org/" target="_blank">https://numpy.org/</a></td></tr><tr><td>SciPy</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref109" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109" aria-controls="sref109">Virtanen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref109"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Virtanen, P. ∙ Gommers, R. ∙ Oliphant, T.E. ..., SciPy 1.0 Contributors</div><div><strong>SciPy 1.0: fundamental algorithms for scientific computing in Python</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:261-272</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.VirtanenR.GommersT.E.OliphantM.HaberlandT.ReddyD.CournapeauE.BurovskiP.PetersonW.WeckesserJ.BrightSciPy+1.0+ContributorsSciPy+1.0%3A+fundamental+algorithms+for+scientific+computing+in+PythonNat.+Methods172020261272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0100" href="https://www.scipy.org/index.html" target="_blank">https://www.scipy.org/index.html</a></td></tr><tr><td>Matplotlib</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Hunter, 2007</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Hunter, J.D.</div><div><strong>Matplotlib: A 2D Graphics Environment</strong></div><div><em>Comput. Sci. Eng.</em> 2007; <strong>9</strong>:90-95</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.D.HunterMatplotlib%3A+A+2D+Graphics+EnvironmentComput.+Sci.+Eng.920079095" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0105" href="https://matplotlib.org/" target="_blank">https://matplotlib.org/</a></td></tr><tr><td>Seaborn</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref112" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112" aria-controls="sref112">Waskom, 2021</a><div class="dropBlock__holder" data-db-target-of="sref112"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Waskom, M.L.</div><div><strong>seaborn: statistical data visualization</strong></div><div><em>Journal of Open Source Software.</em> 2021; <strong>6</strong>:3021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.L.Waskomseaborn%3A+statistical+data+visualizationJournal+of+Open+Source+Software620213021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0110" href="https://seaborn.pydata.org/index.html" target="_blank">https://seaborn.pydata.org/index.html</a></td></tr><tr><td>Benchling</td><td>The Benchling Life Sciences R&amp;D</td><td><a id="intref0115" href="https://benchling.com" target="_blank">https://benchling.com</a>.</td></tr><tr><td>MACS2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref120-1" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-1" aria-controls="sref120-1">Zhang et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref120-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.ZhangT.LiuC.A.MeyerJ.EeckhouteD.S.JohnsonB.E.BernsteinC.NusbaumR.M.MyersM.BrownW.LiX.S.LiuModel-based+analysis+of+ChIP-Seq+%28MACS%29Genome+Biol.92008R137" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0120" href="https://github.com/macs3-project/MACS" target="_blank">https://github.com/macs3-project/MACS</a></td></tr><tr><td>Bowtie2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Langmead et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Langmead, B. ∙ Trapnell, C. ∙ Pop, M. ...</div><div><strong>Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</strong></div><div><em>Genome Biol.</em> 2009; <strong>10</strong>:R25</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.LangmeadC.TrapnellM.PopS.L.SalzbergUltrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genomeGenome+Biol.102009R25" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0125" href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td></tr><tr><td>TrimGalore (v0.4.1)</td><td>Babraham Bioinformatics</td><td><a id="intref0130" href="https://github.com/FelixKrueger/TrimGalore" target="_blank">https://github.com/FelixKrueger/TrimGalore</a></td></tr><tr><td>bedtools (2.26.0)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Quinlan and Hall, 2010</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Quinlan, A.R. ∙ Hall, I.M.</div><div><strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:841-842</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.R.QuinlanI.M.HallBEDTools%3A+a+flexible+suite+of+utilities+for+comparing+genomic+featuresBioinformatics262010841842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0135" href="https://bedtools.readthedocs.io/en/latest" target="_blank">https://bedtools.readthedocs.io/en/latest</a></td></tr><tr><td>samtools (v1.7)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Danecek et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Danecek, P. ∙ Bonfield, J.K. ∙ Liddle, J. ...</div><div><strong>Twelve years of SAMtools and BCFtools</strong></div><div><em>Gigascience.</em> 2021; <strong>10</strong>:giab008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.DanecekJ.K.BonfieldJ.LiddleJ.MarshallV.OhanM.O.PollardA.WhitwhamT.KeaneS.A.McCarthyR.M.DaviesH.LiTwelve+years+of+SAMtools+and+BCFtoolsGigascience102021giab008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0140" href="http://www.htslib.org/" target="_blank">http://www.htslib.org/</a></td></tr><tr><td>bcftools</td><td>Wellcome Sanger Institute</td><td><a id="intref0145" href="http://www.htslib.org/" target="_blank">http://www.htslib.org/</a></td></tr><tr><td>Picard (v2.1.0)</td><td>The Broad Institute</td><td><a id="intref0150" href="https://broadinstitute.github.io/picard/" target="_blank">https://broadinstitute.github.io/picard/</a></td></tr><tr><td>Plyranges (v1.4.4)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Lee et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Lee, S. ∙ Cook, D. ∙ Lawrence, M.</div><div><strong>plyranges: a grammar of genomic data transformation</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:4</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.LeeD.CookM.Lawrenceplyranges%3A+a+grammar+of+genomic+data+transformationGenome+Biol.2020194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0155" href="https://bioconductor.org/packages/release/bioc/html/plyranges.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/plyranges.html</a></td></tr><tr><td>Cellranger (v5.0.1)</td><td>10xGenomics</td><td><a id="intref0160" href="https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/installation" target="_blank">https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/installation</a></td></tr><tr><td>Vartrix (v1.1.14)</td><td>10xGenomics</td><td><a id="intref0165" href="https://github.com/10XGenomics/vartrix/releases" target="_blank">https://github.com/10XGenomics/vartrix/releases</a></td></tr><tr><td>STAR (v2.7.1a)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.DobinC.A.DavisF.SchlesingerJ.DrenkowC.ZaleskiS.JhaP.BatutM.ChaissonT.R.GingerasSTAR%3A+ultrafast+universal+RNA-seq+alignerBioinformatics2920131521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0170" href="https://github.com/alexdobin/STAR" target="_blank">https://github.com/alexdobin/STAR</a></td></tr><tr><td>GATK (v4.1.4.1)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref106" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106" aria-controls="sref106">Van der Auwera and O’Connor, 2020</a><div class="dropBlock__holder" data-db-target-of="sref106"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Van der Auwera, G.A. ∙ O’Connor, B.D.</div><div><strong>Genomics in the Cloud: Using Docker, GATK, and WDL in Terra</strong></div><div>O’Reilly Media, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.A.Van+der+AuweraB.D.O%E2%80%99ConnorGenomics+in+the+Cloud%3A+Using+Docker%2C+GATK%2C+and+WDL+in+TerraFirst+Edition2020O%E2%80%99Reilly+Media" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0175" href="https://gatk.broadinstitute.org/hc/en-us" target="_blank">https://gatk.broadinstitute.org/hc/en-us</a></td></tr><tr><td>R Software Package (v3.6)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">R Core Team, 2021</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing</strong></div><div>R Foundation for Statistical Computing, Vienna, Austria, 2021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing2021R+Foundation+for+Statistical+ComputingVienna%2C+Austria" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0180" href="https://www.r-project.org/" target="_blank">https://www.r-project.org/</a></td></tr><tr><td>RStudio</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">RStudio Team, 2020</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>RStudio Team</div><div><strong>RStudio: Integrated Development for R. RStudio</strong></div><div>PBC, Boston, MA, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=RStudio+TeamRStudio%3A+Integrated+Development+for+R.+RStudio2020PBCBoston%2C+MA" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0185" href="https://www.rstudio.com/" target="_blank">https://www.rstudio.com/</a></td></tr><tr><td>Tidyverse</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref113" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113" aria-controls="sref113">Wickham et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref113"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Wickham, H. ∙ Averick, M. ∙ Bryan, J. ...</div><div><strong>Welcome to the tidyverse</strong></div><div><em>Journal of Open Source Software.</em> 2019; <strong>4</strong>:1686</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamM.AverickJ.BryanW.ChangL.D%E2%80%99Agostino+McGowanR.Fran%C3%A7oisG.GrolemundA.HayesL.HenryJ.HesterWelcome+to+the+tidyverseJournal+of+Open+Source+Software420191686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0190" href="https://www.tidyverse.org/" target="_blank">https://www.tidyverse.org/</a></td></tr><tr><td>ggpubr</td><td>CRAN</td><td><a id="intref0195" href="https://cran.r-project.org/web/packages/ggpubr/index.html" target="_blank">https://cran.r-project.org/web/packages/ggpubr/index.html</a></td></tr><tr><td>Deeptools</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Ramírez et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Ramírez, F. ∙ Ryan, D.P. ∙ Grüning, B. ...</div><div><strong>deepTools2: a next generation web server for deep-sequencing data analysis</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W160-W165</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.Ram%C3%ADrezD.P.RyanB.Gr%C3%BCningV.BhardwajF.KilpertA.S.RichterS.HeyneF.D%C3%BCndarT.MankedeepTools2%3A+a+next+generation+web+server+for+deep-sequencing+data+analysisNucleic+Acids+Res.44W12016W160W165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0200" href="https://deeptools.readthedocs.io/en/3.4.3/index.html" target="_blank">https://deeptools.readthedocs.io/en/3.4.3/index.html</a></td></tr><tr><td>ChIPpeakAnno</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref121" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121" aria-controls="sref121">Zhu et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref121"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Zhu, L.J. ∙ Gazin, C. ∙ Lawson, N.D. ...</div><div><strong>ChIPpeakAnno: a Bioconductor package to annotate ChIP-seq and ChIP-chip data</strong></div><div><em>BMC Bioinformatics.</em> 2010; <strong>11</strong>:237</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.J.ZhuC.GazinN.D.LawsonH.Pag%C3%A8sS.M.LinD.S.LapointeM.R.GreenChIPpeakAnno%3A+a+Bioconductor+package+to+annotate+ChIP-seq+and+ChIP-chip+dataBMC+Bioinformatics112010237" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0205" href="http://bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html</a></td></tr><tr><td>EnrichedHeatmap</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Gu et&nbsp;al., 2018b</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Gu, Z. ∙ Eils, R. ∙ Schlesner, M. ...</div><div><strong>EnrichedHeatmap: an R/Bioconductor package for comprehensive visualization of genomic signal associations</strong></div><div><em>BMC Genomics.</em> 2018; <strong>19</strong>:234</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Z.GuR.EilsM.SchlesnerN.IshaqueEnrichedHeatmap%3A+an+R%2FBioconductor+package+for+comprehensive+visualization+of+genomic+signal+associationsBMC+Genomics192018234" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0210" href="https://bioconductor.org/packages/release/bioc/html/EnrichedHeatmap.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/EnrichedHeatmap.html</a></td></tr><tr><td>MATLAB</td><td>Mathworks</td><td><a id="intref0215" href="https://www.mathworks.com/products/matlab.html" target="_blank">https://www.mathworks.com/products/matlab.html</a></td></tr><tr class="rowsep"><td>Mathematica (v10.1)</td><td>Wolfram Research, Inc.</td><td><a id="intref0220" href="https://www.wolfram.com/mathematica" target="_blank">https://www.wolfram.com/mathematica</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Salmon Sperm DNA</td><td>Sigma-Aldrich</td><td>Cat #: D9156</td></tr><tr><td>XMP X Green (mmX probe)</td><td>MetaSystems Probes</td><td>Cat #: D-1420-050-FI</td></tr><tr><td>Mouse Cot1 DNA</td><td>Life Technologies</td><td>Cat #: 18440016</td></tr><tr><td>Mouse flow sorted chromosome X</td><td>Gift form I. Solovei</td><td>N/A</td></tr><tr><td>high precision coverslips 12&nbsp;mm round</td><td>Azer Scientific</td><td>Cat #: ES0117520</td></tr><tr><td>Correlative microscopy coverslips</td><td>Ted Pella</td><td>Cat #: 260511</td></tr><tr><td>Fixogum rubber cement</td><td>Fisher Scientific</td><td>Cat #: 11FIXO0125</td></tr><tr><td>μ-Slide 8 Well Glass Bottom</td><td>ibidi</td><td>Cat #: 80827</td></tr><tr><td>μ-Slide 4-well Glass Bottom</td><td>ibidi</td><td>Cat #: 80427</td></tr><tr><td>CoverGrip Coverslip Sealant</td><td>VWR</td><td>Cat #: 89411-108</td></tr><tr><td>Gene Pulser/MicroPulser Electroporation Cuvettes, 0.4&nbsp;cm gap</td><td>Bio-rad</td><td>Cat #: 1652088</td></tr><tr><td>TetraSpeck Microspheres, 0.1&nbsp;μm, fluorescent blue/green/orange/dark red</td><td>Thermo Fisher Scientific</td><td>Cat #: T7279</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2901275-7" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0235" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kathrin Plath (<a id="intref0225" href="mailto:KPlath@mednet.ucla.edu">KPlath@mednet.ucla.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0240" role="paragraph">All unique materials generated in this study, such as plasmids and cell lines will be available to researchers from the lead contact with a completed Materials Transfer Agreement.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Cell culture</h4><div id="p0245" role="paragraph">Female mouse polymorphic <i>129S4/SvJae / castaneus</i> F1 2-1 ESCs (<span class="dropBlock reference-citations"><a id="body-ref-sref80-2" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-2" aria-controls="sref80-2">Panning et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref80-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Panning, B. ∙ Dausman, J. ∙ Jaenisch, R.</div><div><strong>X chromosome inactivation is mediated by Xist RNA stabilization</strong></div><div><em>Cell.</em> 1997; <strong>90</strong>:907-916</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.PanningJ.DausmanR.JaenischX+chromosome+inactivation+is+mediated+by+Xist+RNA+stabilizationCell901997907916" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and its engineered derivatives were grown on 0.5% gelatin-coated flasks seeded with irradiated DR4 feeders (obtained from day 14.5 embryos with appropriate animal protocols in place). Cultures were maintained in mouse ESC medium containing knockout medium DMEM (Life Technologies), 15% FBS (Omega), 2mM L-glutamine (Life Technologies), 1x NEAA (Life Technologies), 0.1mM β-Mercaptoethanol (Sigma), 1x Penicillin/Streptomycin (Life Technologies), and 1000U/mL mouse LIF (homemade) in 5% CO2, 37°C incubators.</div><div id="p0250" role="paragraph">Epiblast-like (EpiLC) differentiation was performed as described (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Hayashi and Saitou, 2013</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Hayashi, K. ∙ Saitou, M.</div><div><strong>Generation of eggs from mouse embryonic stem cells and induced pluripotent stem cells</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:1513-1524</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.HayashiM.SaitouGeneration+of+eggs+from+mouse+embryonic+stem+cells+and+induced+pluripotent+stem+cellsNat.+Protoc.8201315131524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, prior to induction of EpiLC differentiation cells were adjusted for 3 passages to feeder-free conditions in the presence of 1000U/mL LIF and two inhibitors, CHIR99021 (3&nbsp;μM) and PD0325901 (0.4&nbsp;μM) (2i+LIF) in serum-free N2B27 medium containing 1x N2 supplement and 1x B27 supplement (Thermo Fischer), 2mM L-glutamine (Life Technologies), 1x NEAA (Life Technologies), 0.1mM β-Mercaptoethanol (Sigma), 0.5 x Penicillin/Streptomycin (Life Technologies). To induce differentiation, cells were dissociated and seeded at a density of 2&nbsp;× 10<sup>5</sup> cells/mL in N2B27 media containing 20&nbsp;ng/mL Activin A and 12&nbsp;ng/mL bFGF on geltrex-coated flasks or coverslips. For experiments extending beyond day 4 of differentiation, we applied a protocol previously described in (<span class="dropBlock reference-citations"><a id="body-ref-sref118" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118" aria-controls="sref118">Ying and Smith, 2003</a><div class="dropBlock__holder" data-db-target-of="sref118"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Ying, Q.L. ∙ Smith, A.G.</div><div><strong>Defined conditions for neural commitment and differentiation</strong></div><div><em>Methods Enzymol.</em> 2003; <strong>365</strong>:327-341</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Q.L.YingA.G.SmithDefined+conditions+for+neural+commitment+and+differentiationMethods+Enzymol.3652003327341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, at day 4 of differentiation, EpiLCs were dissociated with accutase and seeded on geltrex-coated coverslips at a density of 5&nbsp;× 10<sup>5</sup> cells/cm<sup>2</sup>. Cells were then grown in N2B27 media supplemented with EGF and FGF (10&nbsp;ng/mL each), on geltrex-coated coverslips for 4 more days (day 8 of differentiation). At this developmental stage, cells have lost Tsix expression as observed in <a id="crosref1490" href="#fig1">Figure&nbsp;1</a>B. Media was exchanged daily.</div><div id="p0255" role="paragraph">TX1072, female polymorphic <i>B6/castaneus</i> mouse ESCs lines, carrying a tetO-promoter driving the endogenous Xist allele on the B6 X chromosome (B6<sup>tetO-Xist</sup> Cas<sup>WT-Xist</sup>), further modified by a homozygous insertion of the GFP or Halo tag in the endogenous <i>Spen</i> loci (<span class="dropBlock reference-citations"><a id="body-ref-sref31-18" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-18" aria-controls="sref31-18">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or the deletion of the SPOC domain in the endogenous locus (ΔSPOC-SPEN-GFP) (<span class="dropBlock reference-citations"><a id="body-ref-sref31-19" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-19" aria-controls="sref31-19">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as well as derivative cell lines generated in this study carrying ΔIDR-SPEN-GFP or <i>Rosa26</i> knockins, were grown on gelatin-coated flasks in feeder-free conditions (2i+LIF in mouse ESC medium) and EpiLC differentiation was performed as described for the F1 2-1 ESCs. When induction of XCI was performed from the tetO-<i>Xist</i>, 0.5&nbsp;μg/mL doxycycline were added to ESC media without 2i+LIF for 24 h. Similarly, male tetO-<i>Xist</i> ESCs were grown on gelatin-coated flasks in feeder-free conditions and induction of <i>Xist</i> expression was performed with addition of 0.5&nbsp;μg/mL doxycycline in ESC media for 18 to 24 h.</div><div id="p0260" role="paragraph">For auxin-mediated depletion experiments of the TX1072 ESCs expressing SPEN-AID-GFP (<span class="dropBlock reference-citations"><a id="body-ref-sref31-20" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-20" aria-controls="sref31-20">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and <i>Rosa26 SPEN&nbsp;rescue knockins,</i> auxin was added to 2i+LIF ESC medium at 500&nbsp;μM for 12 h. Following, cells were dissociated by trypsinization&nbsp;and seeded in ESC medium without 2i+LIF including 500&nbsp;μM auxin and 0.5&nbsp;μg/mL doxycycline for 24&nbsp;h to induce <i>Xist</i> expression.</div><div id="p0265" role="paragraph">C127 cells were purchased from ATCC and human fibroblasts containing abnormal X-chromosomes (GM3827, GM00735, GM06960, GM07213) were obtained from Coriell Institute. These cell lines were cultured in DMEM (Life Technologies), 15% FBS (Omega), 2mM L-glutamine (Life Technologies) and 1x Penicillin/Streptomycin (Life Technologies).</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Plasmid construction for engineered cell lines</h4><div id="p0270" role="paragraph">Plasmids containing the 24xMS2 repeats (#31865) and MS2-Coat-Protein-GFP (MCP-GFP) coding sequence (#27121) were obtained from Addgene. The pBglII5k plasmid (<span class="dropBlock reference-citations"><a id="body-ref-sref53-3" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-3" aria-controls="sref53-3">Jonkers et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref53-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Jonkers, I. ∙ Monkhorst, K. ∙ Rentmeester, E. ...</div><div><strong>Xist RNA is confined to the nuclear territory of the silenced X chromosome throughout the cell cycle</strong></div><div><em>Mol. Cell. Biol.</em> 2008; <strong>28</strong>:5583-5594</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.JonkersK.MonkhorstE.RentmeesterJ.A.GrootegoedF.GrosveldJ.GribnauXist+RNA+is+confined+to+the+nuclear+territory+of+the+silenced+X+chromosome+throughout+the+cell+cycleMol.+Cell.+Biol.28200855835594" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used for targeting the 24xMS2 repeats into Xist and contains homology arms for insertion into exon 7 of Xist, downstream of the E-repeat sequence, and a floxed neomycin resistance cassette. The 24xMS2 repeats were excised from plasmid #31865 by restriction digest with BglII and BamHI and inserted into the pBglII5k plasmid by Infusion cloning, yielding the pBglII5k-24xMS2 plasmid (which replaces the 16xMS2 repeat array originally contained in the pBglII5k plasmid). The coding region for MCP-GFP was amplified by PCR from plasmid #27121 and introduced under control of a tetracycline-inducible promoter (tetO) into the pBS31 plasmid (pgkATGfrt) (<span class="dropBlock reference-citations"><a id="body-ref-sref4-3" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-3" aria-controls="sref4-3">Beard et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref4-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Beard, C. ∙ Hochedlinger, K. ∙ Plath, K. ...</div><div><strong>Efficient method to generate single-copy transgenic mice by site-specific integration in embryonic stem cells</strong></div><div><em>Genesis.</em> 2006; <strong>44</strong>:23-28</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.BeardK.HochedlingerK.PlathA.WutzR.JaenischEfficient+method+to+generate+single-copy+transgenic+mice+by+site-specific+integration+in+embryonic+stem+cellsGenesis4420062328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) by Infusion cloning, yielding pBS31-MCP-GFP. A reverse tetracycline TransActivator (rtTA3) cassette containing the PGK promoter and a BGH polyA element was amplified by PCR from the MXS_PGK::rtTA3-bGHpA plasmid (&nbsp;Addgene, #62446) and introduced into the unique AscI site of pBS31-MCP-GFP, downstream of the tetO-MCP-GFP-polyA insert, by Infusion cloning, resulting in the pBS31-MCP-GFP-rtTA3 plasmid. For deletion of the B-repeat of Xist the p13-5-Xist-Bdel plasmid was constructed from PCR-amplified 5′ and 3′ homology regions obtained from the PCV-Xist-PA plasmid and a loxP-flanked hygroTK cassette that replaces the B-repeat sequence (chrX: 103480156-103480430, mm10), inserted into a PBS-KS (+) plasmid.</div></section><section id="sec-4-4-2"><h4>Plasmid construction to generate transgenic lines</h4><div id="p0275" role="paragraph">For the integration of transgenes expressing various mCherry or Halo protein fusions under the control of the endogenous <i>Rosa26</i> promoter in Xist<sup>MS2-GFP</sup> ESCs, we employed a parent plasmid harboring homology arms for targeting the <i>Rosa26</i> locus and a loxP-flanked puromycin cassette for antibiotic selection (R26-SA-EGFP-puro). A splice-acceptor (SA) sequence and a splice-donor (SD) coding sequence were synthesized (Genewiz) and inserted into the R26-SA-EGFP-puro plasmid after MluI/MfeI restriction digest to remove the GFP, by Infusion reaction. The resulting pYM215-R26-SA/SD-puro plasmid was used as the parent plasmid for insertion of all protein fusions in three-piece Infusion reactions. The coding sequence for CIZ1 was amplified from the donor plasmid pBS32-MCP-CIZ1(<span class="dropBlock reference-citations"><a id="body-ref-sref79-6" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-6" aria-controls="sref79-6">Pandya-Jones et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref79-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Pandya-Jones, A. ∙ Markaki, Y. ∙ Serizay, J. ...</div><div><strong>A protein assembly mediates Xist localization and gene silencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:145-151</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Pandya-JonesY.MarkakiJ.SerizayT.ChitiashviliW.R.Mancia+LeonA.DamianovC.ChronisB.PappC.K.ChenR.McKeeA+protein+assembly+mediates+Xist+localization+and+gene+silencingNature5872020145151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Coding sequences for histone H2B and mCherry were amplified from a H2B-mCherry plasmid (Addgene, #20972) and the Halo cDNA was obtained from the Halo-EasyFusion plasmid (Addgene, #112852). The coding sequences for PTBP1, PCGF5, and CELF1 were synthesized (Genewiz).</div><div id="p0280" role="paragraph">To generate the Spen-FL-Halo, Spen-ΔRRM-Halo and Spen-ΔIDR-Halo plasmids, introduced into Xist<sup>MS2-GFP</sup> ESCs or TX1072 ESCs, the full-length Spen Entry Clone (Sp22) was modified using Polymerase Incomplete Primer Extension-based mutagenesis with primers designed to delete amino acids 639-3460 or 1-591, respectively. Sp22 and the Spen-ΔIDR or Spen-ΔRRM entry clones, respectively, were inserted into the PyPP-CAG-Halo-V5-IRES-Puro destination vector using Gateway LR Recombination, generating PyPP-CAG-Halo-full-length-Spen-V5, PyPP-CAG-Halo-Spen-ΔIDR-V5 and PyPP-CAG-Halo-Spen-ΔRRM-V5, respectively, also containing an IRES-puromycin resistance cassette for selection. These plasmids enable constitutive expression of Spen variants with an N-terminal Halo tag and a C-terminal V5 tag and contain a polyoma episomal origin of replication for efficient propagation in mammalian cell culture.</div><div id="p0285" role="paragraph">For integration of the full length (FL-) and ΔIDR-Spen-Halo transgenes into the SPEN-AID-GFP female ESC lines the plasmid pFD46 for targeting into the <i>Rosa26</i> locus was used (previously described in (<span class="dropBlock reference-citations"><a id="body-ref-sref31-21" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-21" aria-controls="sref31-21">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)). Halo was amplified from plasmid Halo-EasyFusion (Addgene, #112852) and inserted upstream the Spen coding sequence of pFD46 by Infusion cloning, resulting in the plasmid pYM300. Similarly, the FL-Spen coding sequence was excised from pFD46 by restriction digest and Infusion cloning was performed to insert Halo-ΔIDR-Spen amplified from plasmid PyPP-CAG-Halo-Spen-ΔIDR-V5, resulting in plasmid pYM301. For the integration of FL-Spen-Halo into the Xist-FL<sup>Cas</sup>Xist-ΔΒ<sup>129</sup> female F1 2-1 ESCs and 36.11 cells, a male ESC line carrying an autosomal tetO-Xist transgene (on chromosome 11)(<span class="dropBlock reference-citations"><a id="body-ref-sref116-3" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116-3" aria-controls="sref116-3">Wutz and Jaenisch, 2000</a><div class="dropBlock__holder" data-db-target-of="sref116-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Wutz, A. ∙ Jaenisch, R.</div><div><strong>A shift from reversible to irreversible X inactivation is triggered during ES cell differentiation</strong></div><div><em>Mol. Cell.</em> 2000; <strong>5</strong>:695-705</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.WutzR.JaenischA+shift+from+reversible+to+irreversible+X+inactivation+is+triggered+during+ES+cell+differentiationMol.+Cell52000695705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the same strategy was employed. All plasmids were verified by restriction digests and sequencing.</div></section><section id="sec-4-4-3"><h4>Targeting and cell line generation</h4><div id="p0290" role="paragraph">For targeting, F1 2-1 female ESC lines were grown on DR4 feeders. All targetings were performed by electroporation using the GenePulserII (Biorad). Approximately 2&nbsp;× 10<sup>7</sup> cells and 50&nbsp;μg of DNA were resuspended in 400&nbsp;μL PBS in 4mm diameter cuvettes and pulsed twice for 0.2msec at 800V. To target the <i>Rosa26</i> locus with plasmids pYM300 or pYM301 and pFD82, pFD83 in TX1072 ESCs expressing SPEN-AID-GFP, female F1 2-1 Xist<sup>129ΔB/CasWT</sup> ESCs and male ESCs carrying an autosomal tetO-Xist transgene (36.11), the 3D Nucleofector (Lonza) was used with program CG-104 and 100&nbsp;μL cuvettes according to the manufacturer’s instructions. Antibiotics were added to the growth media 24-36&nbsp;h after electroporation. Puromycin was used at 1.5&nbsp;μg/mL, hygromycin at 130&nbsp;μg/mL and G418 at 400&nbsp;μg/mL. The culture medium containing the respective antibiotics was exchanged every 2 to 3&nbsp;days. Once adequate colony growth was observed (1-2&nbsp;weeks), 100-200 colonies were picked under a stereoscope, dissociated by trypsinization and seeded in 96-well plate replicates. One replicate plate was used for genomic DNA extraction and subsequent genotyping PCR. All positive clones used in this study were screened to ensure that they maintain two X chromosomes in the undifferentiated state indicated by the presence of two Tsix transcripts in RNA FISH experiments. In addition, we confirmed gene silencing by <i>Xist</i> and normal <i>Xist</i> distribution across the X-territory upon induction of differentiation as applicable (<a id="crosref1535" href="#fig1">Figures 1</a>K and <a id="crosref1540" href="#figs1">S1</a>G).</div></section><section id="sec-4-4-4"><h4>Integration of 24xMS2 repeats into the <i>Xist</i> locus</h4><div id="p0295" role="paragraph">The pBglII5k-24xMS2 plasmid was electroporated into the F1 2-1 ESC line after linearization with XhoI. The cell culture was exposed to neomycin selection 36&nbsp;h post-electroporation. Colonies were picked and expanded for screening by genotyping PCR and RNA FISH with Xist and MS2 probes was performed on EpiLCs at day 4 as shown in <a id="crosref1545" href="#figs1">Figure&nbsp;S1</a>G. We confirmed that the 24xMS2-repeat unit was introduced into the 129 allele (<a id="crosref1550" href="#figs1">Figure&nbsp;S1</a>G, right). This observation comes in line with the known skewing of XCI in 129/Cas female cells, where ∼80% of cells inactivate the 129 allele. The loxP-flanked neomycin resistance cassette was removed from targeted ESC clones by transient expression of Cre-recombinase. Subsequently, a FRT-recombination site-containing a landing pad (FRT-neo plasmid) (<span class="dropBlock reference-citations"><a id="body-ref-sref4-4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-4" aria-controls="sref4-4">Beard et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref4-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Beard, C. ∙ Hochedlinger, K. ∙ Plath, K. ...</div><div><strong>Efficient method to generate single-copy transgenic mice by site-specific integration in embryonic stem cells</strong></div><div><em>Genesis.</em> 2006; <strong>44</strong>:23-28</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.BeardK.HochedlingerK.PlathA.WutzR.JaenischEfficient+method+to+generate+single-copy+transgenic+mice+by+site-specific+integration+in+embryonic+stem+cellsGenesis4420062328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was targeted into the inert <i>Col1A</i> locus (on chromosome 11) in F1 2-1<sup>24xMS2-Xist</sup> ESCs. The MCP-GFP-rtTA3 expression cassette was then inserted into the FRT site by electroporation of a FlpO-recombinase-encoding plasmid and the pBS31-MCP-GFP-rtTA3 plasmid. The resulting ESC line was denoted as Xist<sup>MS2-GFP</sup><sub>.</sub></div></section><section id="sec-4-4-5"><h4>Engineering strategy to delete the IDRs of SPEN</h4><div id="p0300" role="paragraph">CRISPR/Cas9-based genome editing was used for the deletion of the IDRs of SPEN with two guide RNAs targeting intronic sequences flanking the IDR-encoding exons, which were synthesized by Synthego. Targeting was performed in the TX1072-SPEN-GFP cell line (<span class="dropBlock reference-citations"><a id="body-ref-sref31-22" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-22" aria-controls="sref31-22">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells were targeted using recombinant Cas9 protein (Synthego) and gRNAs with the 4D Nucleofector (Lonza). ∼2.5&nbsp;× 10<sup>5</sup> cells/mL were resuspended in 20&nbsp;μL Lonza solution and electroporated with 1.8&nbsp;μL of gRNA1, 1.8&nbsp;μL gRNA2 and 2.4&nbsp;μL Cas9 using program CG-104. gRNA sequences are given in <a id="crosref1565" href="#mmc2">Table S2</a><b>.</b> Nucleofected cells were serially diluted and plated onto 10cm dishes. Once adequate colony growth was observed (1-2&nbsp;weeks), 100 colonies were picked under a stereoscope, dissociated by trypsinization, and seeded in 96-well plate replicates. One replicate plate was used for genomic DNA extraction and subsequent genotyping PCR. Genotyping PCR was performed as shown in <a id="crosref1570" href="#figs6">Figure&nbsp;S6</a>A. Positive clones were selected based on an expected 605bp band and sequenced to verify deletion. Biallelic expression was confirmed using RT-PCR with oligo-dT primers and sequencing as shown in <a id="crosref1575" href="#figs6">Figure&nbsp;S6</a>B, scoring for SNPs in exon 5 (rs27580268) and exon 7 (rs223335536).</div></section><section id="sec-4-4-6"><h4>Engineering strategy to delete the B-repeat of Xist</h4><div id="p0305" role="paragraph">F1 2-1 ESC line previously targeted with an 16xMS2 tag in the large final Xist exon on the 129 allele (<span class="dropBlock reference-citations"><a id="body-ref-sref53-4" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-4" aria-controls="sref53-4">Jonkers et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref53-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Jonkers, I. ∙ Monkhorst, K. ∙ Rentmeester, E. ...</div><div><strong>Xist RNA is confined to the nuclear territory of the silenced X chromosome throughout the cell cycle</strong></div><div><em>Mol. Cell. Biol.</em> 2008; <strong>28</strong>:5583-5594</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.JonkersK.MonkhorstE.RentmeesterJ.A.GrootegoedF.GrosveldJ.GribnauXist+RNA+is+confined+to+the+nuclear+territory+of+the+silenced+X+chromosome+throughout+the+cell+cycleMol.+Cell.+Biol.28200855835594" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or male V6.5 ESCs expressing Xist under a tetO promoter (<span class="dropBlock reference-citations"><a id="body-ref-sref34-5" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-5" aria-controls="sref34-5">Engreitz et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref34-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Engreitz, J.M. ∙ Pandya-Jones, A. ∙ McDonel, P. ...</div><div><strong>The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome</strong></div><div><em>Science.</em> 2013; <strong>341</strong>:1237973</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.EngreitzA.Pandya-JonesP.McDonelA.ShishkinK.SirokmanC.SurkaS.KadriJ.XingA.GorenE.S.LanderThe+Xist+lncRNA+exploits+three-dimensional+genome+architecture+to+spread+across+the+X+chromosomeScience34120131237973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were electroporated with the linearized plasmid 13-5 harboring homology arms for targeting into the B-repeat region of XIST and replacing it with a loxP-flanked hygroTK cassette for antibiotic selection. The loxP-flanked hygroTK resistance cassette was removed from targeted clones by transient expression of the Cre-recombinase and ganciclovir treatment. Genotyping and confirmation of deletion of the B-repeat in both cell lines and targeting on the 129 allele in F1 2-1 ESCs were performed by Southern blotting (not shown).</div></section><section id="sec-4-4-7"><h4>Expression of cage<sup>60GFP</sup> in ESCs</h4><div id="p0310" role="paragraph">The gene encoding ct-60 (cage<sup>60GFP</sup>) was amplified by PCR from plasmid I3-01-ct60GFP (<span class="dropBlock reference-citations"><a id="body-ref-sref48-3" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-3" aria-controls="sref48-3">Hsia et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref48-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.HsiaJ.B.BaleS.GonenD.ShiW.ShefflerK.K.FongU.NattermannC.XuP.S.HuangR.RavichandranDesign+of+a+hyperstable+60-subunit+protein+dodecahedron.+%5Bcorrected%5DNature5352016136139" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The fragment was introduced under control of the CACGS promoter into the pBS32 plasmid by Infusion reaction yielding pBS32-cage<sup>60GFP</sup> and positive clones were confirmed by restriction digests and sequencing. The pBS32 plasmid was derived from the pBS31 plasmid upon replacement of the tetO promoter with a CAGGS promoter. To visualize both Xist<sup>MS2-GFP</sup> and cage<sup>60GFP</sup>, Xist<sup>MS2-GFP</sup> ESCs were differentiated into EpiLCs to induce endogenous <i>Xist</i> expression and doxycycline was added at 0.5&nbsp;μg/mL for 2&nbsp;h to induce MCP-GFP expression. Expression of the cage<sup>60GFP</sup> was achieved by transient transfection of the pBS32-cage<sup>60GFP</sup> plasmid into differentiating cells by Lipofectamine3000 24&nbsp;h prior to imaging, according to the manufacturer’s instructions.</div></section><section id="sec-4-4-8"><h4>Halo labeling</h4><div id="p0315" role="paragraph">For FRAP experiments of Halo-fused proteins, 5&nbsp;μM of TMR Halo ligand was added to the culture medium for 30min following a 30min incubation in media without added ligand to wash-off unbound ligand. For fixed and live-cell 3D-SIM imaging, 1&nbsp;μM JF549 or JF646 Halo ligands were introduced to the media for 15min, washed-off twice with PBS and exchanged with fresh medium which was incubated for another 15&nbsp;min. Live-cell imaging or fixation was done as described in the corresponding sections.</div></section><section id="sec-4-4-9"><h4>Immunofluorescence staining</h4><div id="p0320" role="paragraph">Immunodetection was performed as described in (<span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Kraus et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Kraus, F. ∙ Miron, E. ∙ Demmerle, J. ...</div><div><strong>Quantitative 3D structured illumination microscopy of nuclear structures</strong></div><div><em>Nat. Protoc.</em> 2017; <strong>12</strong>:1011-1028</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KrausE.MironJ.DemmerleT.ChitiashviliA.BudcoQ.AlleA.MatsudaH.LeonhardtL.SchermellehY.MarkakiQuantitative+3D+structured+illumination+microscopy+of+nuclear+structuresNat.+Protoc.12201710111028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, cells were grown on geltrex-coated high precision coverslips at the desired differentiation state. Coverslips were then transferred to new multi-well plates, washed three times with PBS and fixed with 2% formaldehyde dissolved in PBS for 10min, followed by two washes with PBST (1xPBS, 0.05% Tween 20). Samples were then quenched for 10min with 20mM glycine in PBS. Following samples were washed with PBS and permeabilized with 0.5% Triton X-100 dissolved in PBS for 10min and washed once with PBST. Samples were then blocked for 1&nbsp;h in blocking buffer (PBST, 2% BSA, 0.5% Fish skin gelatin) and incubated with primary antibodies diluted in blocking buffer for 1&nbsp;h in a humidified chamber at RT. Samples where then washed three times with PBST followed by incubation with secondary antibodies for 45min and another round of PBST washes. Samples were then washed once with PBS, post-fixed with 4% formaldehyde dissolved in PBS for 10min, followed by two washes with PBST. Chromatin counterstaining was performed using DAPI dissolved in PBST at a concentration of 2&nbsp;μg/mL for 5min. Samples were then washed four times with PBS, mounted on slides with Vectashield and sealed with Covergrip. For combined Halo ligand and antibody detection, cells were labeled with the Halo ligands as described in the Halo labeling section, fixed and processed by immunostaining. For the 4-color 3D-SIM imaging where we detect combinations of proteins together with Xist<sup>MS2-GFP</sup> (<a id="crosref1600" href="#fig3">Figures 3</a>B and <a id="crosref1605" href="#figs3">S3</a>A) we used CIZ1-Halo and CELF1 antibody staining, SPEN-Halo and CIZ1 antibody staining, PCGF5-Halo and CIZ1/CELF1 antibody staining. Halo transgenes were detected with the Halo ligand JF549 and primary antibodies with secondary antibodies conjugated to AlexaFluor647. In <a id="crosref1610" href="#fig3">Figures 3</a>H and <a id="crosref1615" href="#figs4">S4</a>C we used endogenous Halo-tagged SPEN (<span class="dropBlock reference-citations"><a id="body-ref-sref31-23" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-23" aria-controls="sref31-23">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Halo transgenes for detection of CIZ1, PCGF5 and PTBP1 labeled by the Halo ligand JF549 and antibody stainings with primary and secondary antibodies conjugated to CF568 dye for CELF1, RYBP, EZH2 and hnRNP-K. In <a id="crosrefs0010" href="#figs6">Figures S6</a>C and <a id="crosref1625" href="#figs7">S7</a>D<b>,</b> we used Halo transgenes FL-/ΔIDR-/ΔRRM-SPEN expressed in Xist<sup>MS2-GFP</sup> or F1 2-1 Xist<sup>129ΔB/CasWT</sup> EpiLCs labeled by the Halo ligand JF549. In <a id="crosref1630" href="#figs3">Figure&nbsp;S3</a>F, RYBP and CIZ1 are detected with antibodies and secondaries conjugated to CF568, while SPEN is stably integrated into the Rosa26 locus (plasmid YM301) and detected with the JF549 Halo ligand. In <a id="crosref1635" href="#fig5">Figure&nbsp;5</a>C, we detect the endogenous WT-SPEN-GFP or ΔIDR-SPEN-GFP with anti-GFP antibodies. We compared the distribution of the CIZ1-Halo fusion protein and the endogenous (antibody-stained) CIZ1 protein and show the same trend (<a id="crosref1640" href="#figs3">Figure&nbsp;S3</a>B) similarly to the distribution of endogenously Halo- or GFP-tagged SPEN proteins to Halo tagged transgenes (<a id="crosref1645" href="#fig3">Figures 3</a>H, <a id="crosref1650" href="#fig5">5</a>C, and <a id="crosref1655" href="#figs6">S6</a>E).</div></section><section id="sec-4-4-10"><h4>Antibodies and dilutions</h4><div id="p0325" role="paragraph">Endogenous CELF1 was detected with monoclonal rabbit anti-CUG-BP1 antibody ab129115 (1/800; Abcam); hnRNP-K with polyclonal rabbit antibody A300-678A (1/800, Bethyl); MATR3 with polyclonal rabbit antibody IHC-00081 (1/200, Bethyl); RYBP (DEDAF) with polyclonal rabbit antibody AB3637 (1/1000; Sigma); Ezh2 with monoclonal rabbit antibody #5246 (1/500; Cell Signaling Technology); CIZ1 with a polyclonal rabbit antibody NB100-74624 (1/800; Novus Biologicals), and histone H3 phospho-Serine 10 with the polyclonal rabbit anti-histone H3-phospho-Serine10 #39253 (1/1000; Active Motif). GFP with a polyclonal rabbit antibody ab6556 (1/500, Abcam). Two secondary antibodies were used, including high cross-absorbed donkey anti-rabbit IgG CF568 antibody SAB4600076 (1/400; Sigma) and high cross-absorbed goat anti-rabbit IgG Alexa Fluor 647 antibody A21245 (1/400; Life Technologies).</div></section><section id="sec-4-4-11"><h4>FISH Probe synthesis</h4><div id="p0330" role="paragraph">Probes for DNA and RNA FISH experiments were labeled by Nick Translation (NT) as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Cremer et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Cremer, M. ∙ Grasser, F. ∙ Lanctôt, C. ...</div><div><strong>Multicolor 3D fluorescence in&nbsp;situ hybridization for imaging interphase chromosomes</strong></div><div><em>Methods Mol. Biol.</em> 2008; <strong>463</strong>:205-239</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.CremerF.GrasserC.Lanct%C3%B4tS.M%C3%BCllerM.NeusserR.ZinnerI.SoloveiT.CremerMulticolor+3D+fluorescence+in%C2%A0situ+hybridization+for+imaging+interphase+chromosomesMethods+Mol.+Biol.4632008205239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, 1&nbsp;μg of DNA was labeled in a 50&nbsp;μL NT reaction for 8&nbsp;h using 1.3-2.5&nbsp;μL fluorescently-labeled dUTPs, 1&nbsp;μL of DNA Polymerase (Thermo Fisher Scientific) and 2&nbsp;μL of DNase I (Sigma-Aldrich) from a stock which was freshly prepared by a 1:200 dilution in ice-cold H<sub>2</sub>O. NT reactions were purified using magnetic beads, probes were then resuspended in Nuclease-free H<sub>2</sub>O and ethanol-precipitated together with Salmon sperm and Cot1 DNA at −80°C ON. After precipitation and washes with ethanol series (70%–100%), probes were resuspended in deionized formamide with shaking at 37°C ON. Probes were then adjusted in hybridization buffer (50%formamide, 2&nbsp;× SSC, 10% dextran sulfate) and stored at −20°C. To create mouse Xist probes, we used a full-length mouse Xist cDNA plasmid (p15A-31-17.9kb Xist). Human XIST probes were created from a full-length XIST cDNA construct. For assessing X-linked gene silencing, <i>Atrx</i> probes were synthesized using BAC RP23-265D6, <i>Rlim</i> probes using fosmid WI1-2704K12 and <i>Mecp2</i> probes using fosmid WI-894A5. For the chromosome barcoding experiment, we used BACs RP23-53H15, RP23-83J1, RP23-451D5, RP24-81K23, RP24-374B8, RP23-401G5, RP23-104K18. To create an intronic probe against the first intron of <i>Xist</i>, the corresponding region was amplified from the Xist-encoding BAC RP23-223G18 and was labeled by NT. To create MS2 probes the corresponding region was amplified from plasmid #31865 (Addgene) and labeled by NT. For multispectral chromosome barcoding experiments, individual BACs were labeled separately, pooled in a 1:1 ratio and used at 0.1&nbsp;μg/cm<sup>2</sup>. DNA from flow sorted mouse X-chromosomes was a gift from Irina Solovei and labeled using the Bioprimer kit according to the manufacturer’s instructions. NT products were labeled with Atto488-dUTP, Alexa Fluor 568-dUTP, Cy3-dUTP, Cy5-dUTP, Texas Red-dUTP and chromosome paints were labeled with Atto448-dUTPs or Cy3-dUTPs.</div><div id="p0335" role="paragraph">Probes used in this study include: <a id="crosref1665" href="#fig1">Figure&nbsp;1</a>B: Xist-Atto488, Rlim-AlexaFluor568, Atrx-Cy5; <a id="crosref1670" href="#fig1">Figure&nbsp;1</a>E: mmX-Atto488, Xist-Alexa568;&nbsp;<a id="crosref1675" href="#fig1">Figure&nbsp;1</a>H: Color-coded as green-Atto488, yellow-Cy3, red-Texas Red, magenta-Cy5; <a id="crosref1680" href="#fig5">Figure&nbsp;5</a>B: Xist-AlexaFluore568; <a id="crosref1685" href="#fig5">Figure&nbsp;5</a>D: Xist-Atto488, Rlim-, AtrX-AlexaFluor568; <a id="crosref1690" href="#fig6">Figure&nbsp;6</a>B: mmX-Atto488, MS2-AlexaFluor568, Xist-Cy5; <a id="crosref1695" href="#fig7">Figure&nbsp;7</a>B: Genes-Atto550, Transcript-Alexa647N; <a id="crosref1700" href="#fig7">Figures 7</a>C and 7I: Early genes-Atto550, Late-genes-Alexa647N; <a id="crosref1705" href="#figs1">Figure&nbsp;S1</a>B: Xist-Atto488, Mecp2-Cy3; <a id="crosref1710" href="#figs1">Figure&nbsp;S1</a>G: MS2-Atto488, Xist-Cy3, Atrx-Cy5; <a id="crosref1715" href="#figs1">Figure&nbsp;S1</a>H: Xist-Atto488; <a id="crosref1720" href="#figs2">Figure&nbsp;S2</a>A: Xist-Atto488; <a id="crosref1725" href="#figs2">Figure&nbsp;S2</a>C: XIST-Atto488; <a id="crosref1730" href="#figs2">Figure&nbsp;S2</a>E: Xist-Atto488, Xist Intron1-Cy3; <a id="crosref1735" href="#figs5">Figures S5</a>A and S5B: Xist-Atto488; <a id="crosref1740" href="#figs6">Figures S6</a>B and S6D: Xist-Atto488. All BACs and fosmids used in this study were purchased from CHORI-BACPAC.</div></section><section id="sec-4-4-12"><h4>Oligo FISH Probe design and synthesis</h4><div id="p0340" role="paragraph">We selected 20 early (half time [0-0.4] days) and 20 very late (half time [1-1.6] days) silencing genes from published silencing kinetics data (<span class="dropBlock reference-citations"><a id="body-ref-sref3-4" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-4" aria-controls="sref3-4">Barros de Andrade E. Sousa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Barros de Andrade E. Sousa, L. ∙ Jonkers, I. ∙ Syx, L. ...</div><div><strong>Kinetics of <i>Xist</i>-induced gene silencing can be predicted from combinations of epigenetic and genomic features</strong></div><div><em>Genome Res.</em> 2019; <strong>29</strong>:1087-1099</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.Barros+de+Andrade+E.+SousaI.JonkersL.SyxI.DunkelJ.ChaumeilC.PicardB.ForetC.J.ChenJ.T.LisE.HeardKinetics+of+Xist-induced+gene+silencing+can+be+predicted+from+combinations+of+epigenetic+and+genomic+featuresGenome+Res.29201910871099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) that are longer than 10kb and have high expression levels in ESCs. To ensure the assessment of genes across the length of the chromosome, we selected no more than five genes per early or late silencing group from each 10Mb region of the X chromosome.</div><div id="p0345" role="paragraph">The resulting early silencing genes were: ENSMUSG00000025862, ENSMUSG00000055780, ENSMUSG00000036022, ENSMUSG00000023092, ENSMUSG00000000838, ENSMUSG00000016382, ENSMUSG00000025246, ENSMUSG00000025059, ENSMUSG00000035232, ENSMUSG00000050332, ENSMUSG00000079487, ENSMUSG00000034055, ENSMUSG00000056537, ENSMUSG00000046449, ENSMUSG00000031333, ENSMUSG00000031232, ENSMUSG00000025531, ENSMUSG00000025271, ENSMUSG00000041649, ENSMUSG00000025289.</div><div id="p0350" role="paragraph">The late silencing genes were: ENSMUSG00000031161, ENSMUSG00000040363, ENSMUSG00000031012, ENSMUSG00000031060, ENSMUSG00000001173, ENSMUSG00000063785, ENSMUSG00000025630, ENSMUSG00000031351, ENSMUSG00000002015, ENSMUSG00000031328, ENSMUSG00000031197, ENSMUSG00000006678, ENSMUSG00000035150, ENSMUSG00000034480, ENSMUSG00000041229, ENSMUSG00000045180, ENSMUSG00000046873, ENSMUSG00000067194, ENSMUSG00000079316, ENSMUSG00000031352.</div><div id="p0355" role="paragraph">We note that the first 40Mb of the chromosome are poor in early-silencing genes that provide high expression levels (RPKMs) for sufficient detection by oligonucleotide FISH. Custom fluorescent oligonucleotide probe pools (MyTags) targeting either genes or their corresponding transcripts were designed and synthesized by Daicel Arbor Biosciences. To detect nascent gene transcripts, ∼500 45bp oligonucleotide probes targeting gene introns downstream of the transcriptional start site of the gene were synthesized spanning 31 to 68kb. To detect genes, ∼500 45bp anti-sense oligonucleotide probes targeting upstream of the transcriptional start site were synthesized, spanning between 31 to 61kb. Probe sequences are given in <a id="crosref1750" href="#mmc2">Table S2</a>.</div></section><section id="sec-4-4-13"><h4>RNA/DNA and immuno-RNA FISH</h4><div id="p0360" role="paragraph">RNA and DNA FISH experiments were conducted as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Markaki et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Markaki, Y. ∙ Smeets, D. ∙ Cremer, M. ...</div><div><strong>Fluorescence in&nbsp;situ hybridization applications for super-resolution 3D structured illumination microscopy</strong></div><div><em>Methods Mol. Biol.</em> 2013; <strong>950</strong>:43-64</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.MarkakiD.SmeetsM.CremerL.SchermellehFluorescence+in%C2%A0situ+hybridization+applications+for+super-resolution+3D+structured+illumination+microscopyMethods+Mol.+Biol.95020134364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, cells were grown on geltrex-coated high precision coverslips at the desired differentiation state. Coverslips were then transferred to new multi-well plates, washed three times with PBS and fixed with 3% formaldehyde dissolved in PBS for 10min, followed by two washes with PBS. Samples were then quenched for 10min with 20mM glycine in PBS. Following, samples were washed with PBS and permeabilized with 0.5% Triton X-100 dissolved in PBS for 15min, washed twice with PBST, equilibrated in 2xSSC for 10min and incubated for 30min to 2&nbsp;h with 50%formamide/ 2xSSC. Probes were denatured at 76°C for 7min and kept on ice. To hybridize the specimens, probes were spotted on slides and coverslips were placed on the probes and sealed with rubber cement. All probes were used at 0.1&nbsp;μg/cm<sup>2</sup> and oligonucleotide probes were used at 10pmol/cm<sup>2</sup>. For DNA FISH, a denaturation step was performed for 2min at 76°C. Samples were then hybridized in a humidified chamber at 37°C ON. After demounting coverslips, unbound probes were washed-off with three 20min washes with 2xSSCT (2xSSC, 0.5% Tween 20) under mild shaking, followed by three 5min washes with 4xSSCT. For DNA FISH samples were washed with three additional 5min washes with 0.1xSSC. Following, samples were post-fixed and chromatin was counterstained with DAPI as described in the Immunofluorescence staining section.</div><div id="p0365" role="paragraph">For sequential RNA and DNA FISH experiments with X chromosome paints (mmX paints) or oligonucleotide probes and Xist probes, RNA FISH was performed first, samples were post-fixed and DNA FISH followed. For the detection of SPEN proteins fused to GFP together with Xist RNA, detected by FISH probes, immunodetection was performed first, samples were post-fixed and RNA FISH followed.</div></section><section id="sec-4-4-14"><h4>Confocal laser scanning microscopy</h4><div id="p0370" role="paragraph">Confocal and improved confocal (Airyscan detector) laser scanning microscopy was performed on the LSM880 platform equipped with 100x/1.46NA or 63x/1.4 NA plan Apochromat oil objectives and 405/488 diode and 594 Helium-Neon lasers (Carl Zeiss Microscopy, Thornwood, NY). To optimize imaging and reduce photobleaching, the regions of interest (ROIs) in each case were marked and the appropriate magnifications were used. The pixel size and z-optical sectioning were set to meet the Nyquist sampling criterion in each case. Airyscan raw data were reconstructed using the ZEN Black (v2.3) software.</div><div id="p0375" role="paragraph">For the detection of genomic regions across the X chromosome with spectral barcoding, cells were seeded on gridded coverslips and DNA FISH was performed first. 5-color optical z stacks of 0.35&nbsp;μm were acquired on a confocal Zeiss LSM880 system. Grid coordinates were recorded, and spatial coordinates of the acquired positions were registered on the ZEN Black software and saved. Following, samples were equilibrated with 50% formamide in 2xSSC pH 7.2 solution for 3&nbsp;h at 37°C followed by RNA FISH with Cy3-labeled Xist probes. Specimens were returned to the microscope stage and saved spatial coordinates were revisited to acquire the <i>Xist</i> RNA signal and discriminate between the Xi and Xa in downstream analyses. Although hybridization of RNA usually precedes DNA FISH, we have found that <i>Xist</i> RNA is remarkably stable during the sequential process. Since the sequential hybridization for this experiment was only necessary for the scoring of the Xi, without the need for harsh probe strip-off steps, RNA FISH was performed last.</div></section><section id="sec-4-4-15"><h4>Detection of cell cycle stages</h4><div id="p0380" role="paragraph">To discriminate between different cell cycle stages, we used a combination of EdU pulse labeling, to detect S-phase cells, and anti-histone H3-phospho-Serine10 (Active Motif, #39253), to detect G2/M phase cells, while G1 cells remained marker-free. EdU and click-iT labeling were performed according to the manufacturer’s instructions. A 10mM EdU stock solution was diluted 1:1000 in growth media and cells were pulsed for 20min prior to fixation. RNA FISH with Xist probes was performed in the 488 channel and detection of EdU by click-iT reaction with CF dye Azide 568 (Biotium, #92082) was combined with immunodetection of phospho-histone H3 Serine 10 and secondary antibodies conjugated to CF568, where RNA FISH was performed first (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Markaki et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Markaki, Y. ∙ Smeets, D. ∙ Cremer, M. ...</div><div><strong>Fluorescence in&nbsp;situ hybridization applications for super-resolution 3D structured illumination microscopy</strong></div><div><em>Methods Mol. Biol.</em> 2013; <strong>950</strong>:43-64</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.MarkakiD.SmeetsM.CremerL.SchermellehFluorescence+in%C2%A0situ+hybridization+applications+for+super-resolution+3D+structured+illumination+microscopyMethods+Mol.+Biol.95020134364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For the assessment of <i>Xist</i> foci features and number throughout the cell cycle in EpiLCs (at day 4 of differentiation), we used the Xist<sup>MS2-GFP</sup> cell line and detected Xist<sup>MS2-GFP</sup> signals after addition of 0.5&nbsp;μg/mL doxycycline for 2&nbsp;h (for MCP-GFP induction).</div></section><section id="sec-4-4-16"><h4>Super-resolution microscopy</h4><div id="p0385" role="paragraph">3D-Structured Illumination Microscopy (3D-SIM) was performed on a DeltaVision OMX-SR system (Cytiva, Marlborough, MA, USA) equipped with a 60x/1.42 NA Plan Apo oil immersion objective (Olympus, Tokyo, Japan), sCMOS cameras (PCO, Kelheim, Germany) and 405, 488, 642nm diode lasers and a 568nm DPSS laser. Image stacks were acquired on the OMX AcquireSR software package 4.4.9934 with a z-steps of 125nm and with 15 raw images per plane (five phases, three angles). Raw data were computationally reconstructed with the soft-WoRx 7.0.0 software package (Cytiva, Marlborough, MA, USA) using a Wiener filter set at 0.001 to 0.002 (up to 0.006 for DAPI) and optical transfer functions (OTFs) measured specifically for each channel using immersion oil with different refractive indices (RIs) as described in (<span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Demmerle et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Demmerle, J. ∙ Innocent, C. ∙ North, A.J. ...</div><div><strong>Strategic and practical guidelines for successful structured illumination microscopy</strong></div><div><em>Nat. Protoc.</em> 2017; <strong>12</strong>:988-1010</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.DemmerleC.InnocentA.J.NorthG.BallM.M%C3%BCllerE.MironA.MatsudaI.M.DobbieY.MarkakiL.SchermellehStrategic+and+practical+guidelines+for+successful+structured+illumination+microscopyNat.+Protoc.1220179881010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Kraus et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Kraus, F. ∙ Miron, E. ∙ Demmerle, J. ...</div><div><strong>Quantitative 3D structured illumination microscopy of nuclear structures</strong></div><div><em>Nat. Protoc.</em> 2017; <strong>12</strong>:1011-1028</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KrausE.MironJ.DemmerleT.ChitiashviliA.BudcoQ.AlleA.MatsudaH.LeonhardtL.SchermellehY.MarkakiQuantitative+3D+structured+illumination+microscopy+of+nuclear+structuresNat.+Protoc.12201710111028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Images from different channels were registered using alignment parameters obtained from a calibration slide of 100nm gold grid holes and a second calibration for axial alignment using 100nm diameter Tetraspeck beads according to established procedures (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Demmerle et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Demmerle, J. ∙ Innocent, C. ∙ North, A.J. ...</div><div><strong>Strategic and practical guidelines for successful structured illumination microscopy</strong></div><div><em>Nat. Protoc.</em> 2017; <strong>12</strong>:988-1010</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.DemmerleC.InnocentA.J.NorthG.BallM.M%C3%BCllerE.MironA.MatsudaI.M.DobbieY.MarkakiL.SchermellehStrategic+and+practical+guidelines+for+successful+structured+illumination+microscopyNat.+Protoc.1220179881010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-17"><h4>Live-cell imaging</h4><div id="p0390" role="paragraph">Wide-field and confocal scanning microscopy (for FRAP experiments) or 3D-SIM live-cell imaging (4D-SIM) were performed at 37°C (for 3D-SIM in conjunction with an objective heater), with 5% CO<sub>2</sub>, controlled humidity and 10%O<sub>2</sub>, having equilibrated the system and immersion oils for at least five h prior to acquisitions. This equilibration was particularly important for obtaining artifact-free 3D-SIM datasets and minimize stage drift. Cells were differentiated in geltrex-coated chambers fitted with a high precision glass (ibidi) with daily exchange of media. To induce MCP-GFP expression, doxycycline was added to the cells two h prior to acquisitions at a concentration of 1&nbsp;μg/mL. Imaging was performed in media containing no phenol red and supplemented with ProlongLive Antifade reagent (Thermo Fisher). For live-cell 3D-SIM imaging, typically 1&nbsp;μm to 2&nbsp;μm stacks of 125nm z sections were acquired in 1- or 2-color 3D-SIM imaging to obtain 240-500 raw images per frame in 5-8&nbsp;s intervals depending on exposure times and z-depth. Photobleaching over time was corrected by using histogram matching on the BleachCorrection plugin in ImageJ/Fiji.</div></section><section id="sec-4-4-18"><h4>FRAP experiments</h4><div id="p0395" role="paragraph">FRAP experiments with z sectioning for Xist<sup>MS2-GFP</sup> and CIZ1-mCherry were performed on an LSM880 equipped with an Airyscan on a Plan-Apochromat 63x1.4NA oil immersion objective, an image size of 67.5&nbsp;μm x 67.5&nbsp;μm with a pixel size of 0.085&nbsp;μm. Z-optical stacks of 0.5&nbsp;μm were obtained through a 15&nbsp;μm z-depth. Bleaching was performed in ROIs demarcating the <i>Xist</i> territory or corresponding nuclear (control) regions at full laser power and 4 iterations with a pixel dwell time of 4.04&nbsp;μsec. The first post-bleach frame was acquired immediately after bleaching. Time series were acquired every 1.3min up to 10 frames and every 2min thereafter for a total of 30min with an Argon ion 488nm laser or a DPSS 561nm laser set to 1% laser power.</div><div id="p0400" role="paragraph">Single-plane FRAP experiments for all other proteins were performed on the OMX-SR platform in widefield mode and an image size of 512x512 pixels with a pixel size of 0.08&nbsp;μm. In these experiments, we employed transgenic cells lines carrying mCherry-tagged CIZ1 and CELF1 and Halo-tagged FL-/ΔIDR-/ΔRMM-SPEN, PCGF5 and PTBP1, respectively (<a id="crosref1780" href="#fig4">Figures 4</a>F, <a id="crosref1785" href="#figs5">S5</a>D, and <a id="crosref1790" href="#figs6">S6</a>F). Images were acquired for Xist<sup>MS2-GFP</sup> in the 488nm channel (95MHz- 6% amplitude, 20msec) and for all mCherry- or Halo-fused-TMR labeled proteins in the 568nm channel (272MHz, 6% amplitude, 50-100ms exposure). Bleaching in ROIs demarcating the <i>Xist</i> territory or nuclear (control) regions was performed by using the 568nm laser line in the Ring-TIRF/PK photokinetics module with a bleach spot of 1&nbsp;μm for one iteration for 0.1sec.</div></section><section id="sec-4-4-19"><h4>RNA-Antisense purification (RAP)-seq</h4><div id="p0405" role="paragraph">F1 2-1 female mouse ESCs were seeded on geltrex-coated plates and differentiated for 2 or 4&nbsp;days. 5&nbsp;× 10<sup>6</sup> cells were collected per condition after dissociation by accutase and RAP-seq was performed (<span class="dropBlock reference-citations"><a id="body-ref-sref34-6" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-6" aria-controls="sref34-6">Engreitz et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref34-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Engreitz, J.M. ∙ Pandya-Jones, A. ∙ McDonel, P. ...</div><div><strong>The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome</strong></div><div><em>Science.</em> 2013; <strong>341</strong>:1237973</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.EngreitzA.Pandya-JonesP.McDonelA.ShishkinK.SirokmanC.SurkaS.KadriJ.XingA.GorenE.S.LanderThe+Xist+lncRNA+exploits+three-dimensional+genome+architecture+to+spread+across+the+X+chromosomeScience34120131237973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, harvested cells were incubated for 45min with 2mM DSG in PBS at RT, crosslinked with 3% formaldehyde for 10min and quenched with 500mM glycine. Following, cells were pelleted at 1,500xg for 5min and flash frozen. Pellets were resuspended in 10ml nuclear extraction buffer LB1 containing&nbsp;50mM HEPES-KOH (pH 7.5), 140mM NaCl, 1mM EDTA, 10% (vol/vol) glycerol, 0.5% (vol/vol) NP-40/Igepal CA-630 and 0.25% (vol/vol) Triton X-100 and incubated for 10min with rotation at 4°C, then pelleted at 2000xg for 5min and the same procedure was followed with buffer LB2, containing 10mM Tris-HCL (pH 8.0), 200mM NaCl, 1mM EDTA and 0.5mM EGTA (<span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Mohammed et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Mohammed, H. ∙ Taylor, C. ∙ Brown, G.D. ...</div><div><strong>Rapid immunoprecipitation mass spectrometry of endogenous proteins (RIME) for analysis of chromatin complexes</strong></div><div><em>Nat. Protoc.</em> 2016; <strong>11</strong>:316-326</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.MohammedC.TaylorG.D.BrownE.K.PapachristouJ.S.CarrollC.S.D%E2%80%99SantosRapid+immunoprecipitation+mass+spectrometry+of+endogenous+proteins+%28RIME%29+for+analysis+of+chromatin+complexesNat.+Protoc.112016316326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For cell lysis, nuclei were resuspended in 300&nbsp;μL buffer LB3 containing 10mM Tris-HCl (pH 8.0), 100mM NaCl, 1mM EDTA, 0.5mM EGTA, 0.1% (wt/vol) sodium deoxycholate and 0.5% (vol/vol) N-lauroylsarcosine and sonicated on ice using a Misonix S-400 sonicator with microtip for 2min with 1sec pulses intermitted by 3sec pauses. Chromatin was then digested using TURBO DNase at a concentration of 0.1-0.4U/μl at 37°C for 15min. For each pulldown library, an input library was also generated. The RNA pulldown was performed using 50pmol of 90nt long biotinylated oligonucleotide probes for every 5&nbsp;× 10<sup>6</sup> cells and Streptavidin C1 beads. DNA was eluted by RNase H digestion, and the crosslinks were reversed by proteinase K digestion of the eluted DNA at 60þC. The DNA libraries were prepared using NEB Next Ultra End Repair/dA-Tailing Module (NEB) and TruSeq DNA adapters (Illumina) were ligated using Quick Ligase (NEB). Libraries were amplified by KAPA HiFi Polymerase, pooled and sequenced on the Illlumina HiSeq 6000 platform to generate 50bp pair-end reads. Probe sequences are given in <a id="crosref1805" href="#mmc2">Table S2</a><b>.</b></div></section><section id="sec-4-4-20"><h4>Single cell RNA-seq</h4><div id="p0410" role="paragraph">scRNA-seq was performed in F1 2-1 Xist<sup>129-WT/Cas-WT</sup> and Xist<sup>129-ΔB/Cas-WT</sup> ESC lines at days 2 and 4 of EpiLC differentiation. For the B6<sup>tetO-Xist</sup>Cas<sup>WT-Xist</sup> TX1072 ESC lines expressing SPEN-GFP or ΔSPOC-SPEN-GFP (<span class="dropBlock reference-citations"><a id="body-ref-sref31-24" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-24" aria-controls="sref31-24">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and ΔIDR-SPEN-GFP generated in this study, scRNA-seq was performed without (0 h) or with (24 h) of 0.5&nbsp;μg/mL doxycycline to induce tetO-Xist expression.</div><div id="p0415" role="paragraph">Cells were dissociated with accutase for 5min, washed 3 times with PBS and resuspended in PBS containing 0.04% BSA. Cell concentration was adjusted between 800 to 1200 cells/μl and cell suspension was kept on ice before loading on the 10x Genomics Chromium instrument. scRNA-seq libraries were generated using the Chromium single cell 3′¢ reagent kit V3.1. Individual libraries were designed to target up to 10,000 cells. Libraries were generated following manufacturer’s instructions and library fragment size distribution was determined by BioAnalyzer. Libraries were pooled and sequenced on the Illumina Novaseq 6000 platform to generate 100bp pair-end reads.</div></section><section id="sec-4-4-21"><h4>Bulk RNA-seq</h4><div id="p0420" role="paragraph">Bulk mRNA-seq libraries were generated from B6<sup>tetO-Xist</sup>Cas<sup>WT-Xist</sup> TX1072 ESC lines homozygously expressing WT-SPEN-AID-GFP (<span class="dropBlock reference-citations"><a id="body-ref-sref31-25" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-25" aria-controls="sref31-25">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with no <i>Rosa26</i> knockin rescue, full length (FL)-SPEN <i>Rosa26</i> knockin rescue, or ΔIDR-SPEN <i>Rosa26</i> knockin rescue, at 0&nbsp;h and 24&nbsp;h dox treatment and upon auxin treatment. Harvested cells were washed with PBS and collected into TRI reagent. Lysates were processed immediately or snap-frozen in liquid nitrogen and stored at −80°C for up to 3&nbsp;days. All conditions were collected from three biological replicates where samples for each replicate were processed at the same time. RNA was isolated using the Zymo Research RNA miniprep isolation kit according to manufacturer’s instructions. RNA-seq libraries were prepared using the TrueSeq Stranded mRNA Library Prep Kit according to manufacturer’s instructions. Libraries were pooled and sequenced on the Illumina Novaseq 6000 platform to generate 100bp pair-end reads.</div></section><section id="sec-4-4-22"><h4>CLAP-seq</h4><div id="p0425" role="paragraph">CLAP-seq libraries were generated from B6<sup>tetO-Xist</sup> Cas<sup>WT-Xist</sup> TX1072 derivative ESC lines expressing the FL-, ΔIDR- or ΔRRM-Halo-SPEN at day 4 of differentiation with the addition of 0.5&nbsp;μg/mL doxycycline for the last 24&nbsp;h to induce tetO-Xist expression. For each pulldown library, an input library was also generated. ∼5&nbsp;× 10<sup>7</sup> cells were collected for each replicate. Cells grown on gelatin-coated 150cm<sup>2</sup> culture dishes were washed three times with ice-cold PBS and UV-cross-linked on ice using 0.25 Jcm−2 (UV2.5k) at 254&nbsp;nm in a Spectrolinker UV Crosslinker. Cells were then collected through scraping on ice and centrifuged at 1,500&nbsp;g for 5min, washed once with PBS and pelleted by centrifugation in aliquots of 5&nbsp;× 10<sup>6</sup> cells. Pellets were snap frozen in liquid nitrogen for storage at −80°C. CLAP-seq using the HaloLink Resin to pulldown the Halo-tagged proteins was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Quinodoz et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Quinodoz, S.A. ∙ Bhat, P. ∙ Ollikainen, N. ...</div><div><strong>RNA promotes the formation of spatial compartments in the nucleus</strong></div><div><em>bioRxiv.</em> 2020;</div><div>2020.2008.2025.267435</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.A.QuinodozP.BhatN.OllikainenJ.W.JachowiczA.K.BanerjeeP.ChovanecM.R.BlancoA.ChowY.MarkakiK.PlathM.GuttmanRNA+promotes+the+formation+of+spatial+compartments+in+the+nucleusbioRxiv20202020.2008.2025.267435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, each pellet was lysed for 10min in 1ml Lysis buffer containing 50mM HEPES, pH 7.4, 100mM NaCl, 1% NP-40, 0.1% SDS, 0.5% Sodium Deoxycholate, 1x Protease Inhibitor Cocktail (Promega), 200U of Murine RNase Inhibitor, 20U Turbo DNase and 1X Manganese/Calcium Mix (0.5mM CaCl2, 2.5mM MnCl2) at 37°C. Lysates were pelleted at 800xg for 8min at 4°C, the supernatant was removed and pellets were resuspended in 1ml Lysis buffer and sonicated for 30sec with 0.7sec pulses intermitted by 2.3sec pauses. After removal a 10min incubation at 37°C, samples were pelleted at 15,000xg for 2min and the resulting supernatant was collected and stored on ice. The HaloLink Resin was washed three times in Wash buffer (1xPBS, 0.1% Triton) and blocked for 20min in Blocking buffer (50mM HEPES, pH 7.5, 10&nbsp;μg/mL Random 9-mer, 100&nbsp;μg/mL BSA). Following the HaloLink Resin was incubated with the supernatant for 3-16&nbsp;h under rotation at 4°C. After incubation, three washes in Wash buffer were performed at RT and three additional washes at 90C for 2min with the following buffers: NLS buffer (50mM HEPES pH 7.5, 2% NLS, 10mM EDTA, 0.1% NP-40, 10mM DTT), High Salt Buffer (50mM HEPES pH 7.5, 10mM EDTA, 0.1% NP-40, 1M NaCl), 8M Urea Buffer (50mM HEPES pH&nbsp;7.5, 10mM EDTA, 0.1% NP-40, 8M Urea) and Tween buffer (50mM HEPES pH 7.5, 0.1% Tween 20, 10mM EDTA). The HaloLink Resin was then equilibrated by washing three times with Elution buffer (50mM HEPES pH 7.5, 0.5mM EDTA, 0.1% NP-40) at 30C and elution was performed by resuspension in 100&nbsp;μL of NLS buffer containing 10% Proteinase K and shaking at 50°C for 30min. Successful elution of the Halo-fused proteins was tested by western blotting. After elution RNA overhangs were repaired by treatment with FastAP and T4 Polynucleotide Kinase with no ATP at 37°C for 15min and 1 h, respectively. The RNA was then reverse transcribed using Superscript III according to the manufacturer’s instructions. Following, the cDNA was amplified by PCR using illumina sequencing adaptors libraries, pooled and sequenced on the Illlumina HiSeq 6000 platform to generate 100bp pair-end reads.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><section id="sec-4-5-1"><h4>Quantitative 3D-SIM analyses</h4><div id="p0430" role="paragraph">For image segmentation and image objects (particles) determination, 32-bit raw datasets were imported into ImageJ/Fiji (<span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.SchindelinI.Arganda-CarrerasE.FriseV.KaynigM.LongairT.PietzschS.PreibischC.RuedenS.SaalfeldB.SchmidFiji%3A+an+open-source+platform+for+biological-image+analysisNat.+Methods92012676682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and converted to 16-bit tiff composite stacks. The segmentation of <i>Xist</i> and protein particles (foci) was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref56-3" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-3" aria-controls="sref56-3">Kraus et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Kraus, F. ∙ Miron, E. ∙ Demmerle, J. ...</div><div><strong>Quantitative 3D structured illumination microscopy of nuclear structures</strong></div><div><em>Nat. Protoc.</em> 2017; <strong>12</strong>:1011-1028</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KrausE.MironJ.DemmerleT.ChitiashviliA.BudcoQ.AlleA.MatsudaH.LeonhardtL.SchermellehY.MarkakiQuantitative+3D+structured+illumination+microscopy+of+nuclear+structuresNat.+Protoc.12201710111028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using the TANGO suite (<span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Ollion et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Ollion, J. ∙ Cochennec, J. ∙ Loll, F. ...</div><div><strong>TANGO: a generic tool for high-throughput 3D image analysis for studying nuclear organization</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1840-1841</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.OllionJ.CochennecF.LollC.Escud%C3%A9T.BoudierTANGO%3A+a+generic+tool+for+high-throughput+3D+image+analysis+for+studying+nuclear+organizationBioinformatics29201318401841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Specifically, raw datasets without filtering or subtraction of signals were imported into the segmentation pipeline. Image segmentation pipelines, adjustment of thresholds and creation of seeds were performed using high-throughput batch-processing and without manual intervention. Resulting masks of segmented particles were inspected by overlays over the raw data to ensure that the majority of signals was contained in the area to be analyzed. Nuclear masks were created using the DAPI channel as the segmentation volume. For each channel, a duplicate was generated and filtered with a 3D Gaussian filter with standard deviation of 1 (σ&nbsp;= 1) and a Tophat filter with a radius of two pixels in xy and a one-pixel radius in z. The filtered image was segmented using the 3D Suite’s Watershed method. Seed threshold and image threshold for watershed were calculated by equations Mean+StdDev<sup>∗</sup>2<sup>∗</sup>seed multiplier and (Mean+StdDev<sup>∗</sup>2<sup>∗</sup>seed multiplier)/image multiplier (Signal-to-Noise Ratio, SNR), respectively, where seed multiplier and image multiplier were determined and inspected manually to ensure the inclusion of all the regions of interest (ROIs) and the removal of background noise. Object features and distance measurements were performed using the 3D ImageJ Suite’s “Measure 3D,” “Quantif 3D” and “Distance” option plugins for ImageJ/Fiji.</div><div id="p0435" role="paragraph">For the assessment of the cage<sup>60GFP</sup> versus <i>Xist</i> signals, <i>Xist</i><sup>MS2-GFP</sup> cells expressing the cage<sup>60GFP</sup> plasmid were typically imaged in the same Field of View (FOV) as cells with the <i>Xist</i><sup>MS2-GFP</sup> signal, allowing us to obtain data that could be directly compared. When cells expressed both entities, since the cages are located in the cytosol in the majority of cells, nuclear masks from the DAPI channel were created and <i>Xist</i><sup>MS2-GFP</sup> signals were measured inside the masked regions, whereas the signal from the GFP-expressing cages was measured outside the nuclear masks. For the comparison of intensities of cages expressed in the cytosol or in the nucleus the same masking procedure was applied. The variability of the integrated density of fluorescent cages imaged under 3D-SIM conditions was compared to that obtained by a similar analysis in the original publication (<span class="dropBlock reference-citations"><a id="body-ref-sref48-4" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-4" aria-controls="sref48-4">Hsia et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref48-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.HsiaJ.B.BaleS.GonenD.ShiW.ShefflerK.K.FongU.NattermannC.XuP.S.HuangR.RavichandranDesign+of+a+hyperstable+60-subunit+protein+dodecahedron.+%5Bcorrected%5DNature5352016136139" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and found that results are within the same range.</div><div id="p0440" role="paragraph">The number of <i>Xist</i> foci is typically lower when detecting the RNA via MCP-GFP compared to detection by RNA FISH using probes that cover the entire spliced transcript. Our labeling with FISH probes captures the entire RNA molecule in contrast to MCP-GFP, which detects a region in the last large exon of <i>Xist</i> downstream to the E-repeat sequence. Therefore, RNA FISH-based detection results in slightly larger and possibly more complex <i>Xist</i> structures. Moreover, the different average number of <i>Xist</i> foci between cells at D2, D4, D8 of differentiation or in C127 cells, defined across the cell population without discriminating cell cycle stage (<a id="crosref1845" href="#fig1">Figures 1</a>L, <a id="crosref1850" href="#figs1">S1</a>I, and <a id="crosref1855" href="#figs2">S2</a>B), is likely a reflection of different populations of cells across the different cell states.</div><div id="p0445" role="paragraph">For the comparison of the integrated densities of SPEN-GFP and cage<sup>60GFP</sup>, cells expressing the cage<sup>60GFP</sup> plasmid and TX1072 (B6<sup>tetO-Xist</sup>Cas<sup>WT-Xist</sup>) derivative cells expressing SPEN-GFP after addition of 0.5&nbsp;μg/mL doxycycline, to induce expression of the tetO-Bgl-mCherry-<i>Xist</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref31-26" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-26" aria-controls="sref31-26">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), were imaged under the same settings and datasets processed with the same threshold to define seeds and objects (particles). We found that the increase in <i>Xist</i>-associated SPEN accumulation during differentiation, observed from the D2 to D4 transition, also occurs similarly upon doxycycline induction of <i>Xist</i> expression between 6 and 18&nbsp;h of <i>Xist</i> induction. Moreover, the levels of SPEN in SMACs are the same upon tetO-Xist induction at 18&nbsp;h as for WT-Xist at D4 of differentiation (not shown).</div><div id="p0450" role="paragraph">To extract global nuclear protein particle features (in and outside the Xi) such as mean intensity, integrated density (amount of fluorescence per defined particle volume) and volume, masks of the protein signals of interest were created by filtering raw data with a 3D Gaussian blur followed by automatic thresholding to include all signals and exclude nucleoli. ROIs within a 4&nbsp;μm radius of <i>Xist</i> centroids were selected for features extraction to limit computation time. Nearest neighbor centroid distances and all distances between ROIs within each channel and across different channels were extracted using the 3D ImageJ Suite for minimal distance and average distance analysis, respectively. Distance averaging was performed in Python. Assignment of <i>Xist</i>-associated signals was based on a proximity threshold to <i>Xist</i> centroids with a radius of 250nm. Signals 500nm away from <i>Xist</i> centroids, resulting in a ‘rim’ around the Xi due to the scattering of many <i>Xist</i> foci throughout the Xi, were defined as the nuclear fraction. To test the specificity of the tight spatial association of <i>Xist</i> and its associated protein particles in the Xi, we performed randomized controls (<a id="crosref1865" href="#figs4">Figure&nbsp;S4</a>D). We first determined the Xi territory by generating voxel arrays corresponding to a 350nm radius from <i>Xist</i> foci centroids. The overlapping (double-called) voxels were removed. We then generated random positions equal to <i>Xist</i>-associated protein particles within the Xi masks of each nucleus using Python’s numpy.random.choice function. We found a statistically significant difference between the nearest-neighbor <i>Xist</i>-associated protein particle to <i>Xist</i> compared to the randomized positions to <i>Xist</i>. The comparison of protein features, such as integrated density of fluorescence and volume, was performed by measurements acquired in the same laser line (568nm) for all proteins detected either with the Halo ligand JF549 or primary and secondary antibodies conjugated to CF568 dye. For each experiment, ROIs with integrated density and volume values below the 10th percentile or above the 90th percentile of the dataset were removed as outliers.</div><div id="p0455" role="paragraph">For the oligo-FISH analysis, signal centroid from 3D-SIM data were extracted as described above using the 3D ImageJ Suite. Signal coordinates were imported into Python using Pandas and separated by cell and signal type (late, early, DNA, RNA, <i>Xist</i>) into NumPy arrays. We detected on average ∼10-30 foci for each gene pool per X chromosome. Numbers in the higher range can likely be explained by the detection of some genomic regions as doublets after DNA replication or the segmentation of extended structures. <i>Xist</i> cluster centroids were calculated for each nucleus using the corresponding <i>Xist</i> array and centroid formula:<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper G Subscript x comma y comma z Baseline equals left parenthesis StartFraction x 1 plus x 2 plus x 3 plus ellipsis plus x Subscript n Baseline Over n EndFraction comma StartFraction y 1 plus y 2 plus y 3 plus ellipsis plus y Subscript n Baseline Over n EndFraction comma StartFraction z 1 plus z 2 plus z 3 plus ellipsis plus z Subscript n Baseline Over n EndFraction right parenthesis" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(75 (7 0 (6 1 2 3 4 5)) 8 (74 9 (72 (29 (27 (12 10 11) 13 (16 14 15) 17 (20 18 19) 21 22 23 (26 24 25)) 28) 30 (50 (48 (33 31 32) 34 (37 35 36) 38 (41 39 40) 42 43 44 (47 45 46)) 49) 51 (71 (69 (54 52 53) 55 (58 56 57) 59 (62 60 61) 63 64 65 (68 66 67)) 70)) 73))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="75" data-semantic-children="7,74" data-semantic-content="8" data-semantic-owns="7 8 74" data-semantic-speech="upper G Subscript x comma y comma z Baseline equals left parenthesis StartFraction x 1 plus x 2 plus x 3 plus ellipsis plus x Subscript n Baseline Over n EndFraction comma StartFraction y 1 plus y 2 plus y 3 plus ellipsis plus y Subscript n Baseline Over n EndFraction comma StartFraction z 1 plus z 2 plus z 3 plus ellipsis plus z Subscript n Baseline Over n EndFraction right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="0,6" data-semantic-parent="75" data-semantic-owns="0 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="1,2,3,4,5" data-semantic-content="2,4" data-semantic-parent="7" data-semantic-owns="1 2 3 4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="8" data-semantic-parent="75" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="74" data-semantic-children="72" data-semantic-content="9,73" data-semantic-parent="75" data-semantic-owns="9 72 73"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="9" data-semantic-parent="74" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="72" data-semantic-children="29,30,50,51,71" data-semantic-content="30,51" data-semantic-parent="74" data-semantic-owns="29 30 50 51 71"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="29" data-semantic-children="27,28" data-semantic-parent="72" data-semantic-owns="27 28"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="27" data-semantic-children="12,16,20,22,26" data-semantic-content="13,17,21,23" data-semantic-parent="29" data-semantic-owns="12 13 16 17 20 21 22 23 26"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="27" data-semantic-owns="10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="13" data-semantic-parent="27" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="27" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="17" data-semantic-parent="27" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="27" data-semantic-owns="18 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="20"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="21" data-semantic-parent="27" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="22" data-semantic-parent="27"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="23" data-semantic-parent="27" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="24,25" data-semantic-parent="27" data-semantic-owns="24 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="26"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="29"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="30" data-semantic-parent="72" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mfrac space="2" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="50" data-semantic-children="48,49" data-semantic-parent="72" data-semantic-owns="48 49"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="48" data-semantic-children="33,37,41,43,47" data-semantic-content="34,38,42,44" data-semantic-parent="50" data-semantic-owns="33 34 37 38 41 42 43 44 47"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="33" data-semantic-children="31,32" data-semantic-parent="48" data-semantic-owns="31 32"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="33"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="33"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="34" data-semantic-parent="48" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="37" data-semantic-children="35,36" data-semantic-parent="48" data-semantic-owns="35 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="37"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="37"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="38" data-semantic-parent="48" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="41" data-semantic-children="39,40" data-semantic-parent="48" data-semantic-owns="39 40"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="41"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="41"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="42" data-semantic-parent="48" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="43" data-semantic-parent="48"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="44" data-semantic-parent="48" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="47" data-semantic-children="45,46" data-semantic-parent="48" data-semantic-owns="45 46"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="47"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-parent="47"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="50"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="51" data-semantic-parent="72" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mfrac space="2" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="71" data-semantic-children="69,70" data-semantic-parent="72" data-semantic-owns="69 70"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="69" data-semantic-children="54,58,62,64,68" data-semantic-content="55,59,63,65" data-semantic-parent="71" data-semantic-owns="54 55 58 59 62 63 64 65 68"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="54" data-semantic-children="52,53" data-semantic-parent="69" data-semantic-owns="52 53"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="54"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.002em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="53" data-semantic-parent="54"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="55" data-semantic-parent="69" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="58" data-semantic-children="56,57" data-semantic-parent="69" data-semantic-owns="56 57"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="56" data-semantic-parent="58"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.002em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="57" data-semantic-parent="58"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="59" data-semantic-parent="69" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="62" data-semantic-children="60,61" data-semantic-parent="69" data-semantic-owns="60 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="62"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.002em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="62"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="63" data-semantic-parent="69" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="64" data-semantic-parent="69"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="65" data-semantic-parent="69" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="68" data-semantic-children="66,67" data-semantic-parent="69" data-semantic-owns="66 67"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="66" data-semantic-parent="68"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.002em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="67" data-semantic-parent="68"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="70" data-semantic-parent="71"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="73" data-semantic-parent="74" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div></div><div id="p0460" role="paragraph">Distance to <i>Xist</i> centroid was then calculated using the spatial coordinates of each gene centroid to <i>Xist</i> centroid for the corresponding nucleus using the distance formula. Gene transcriptional activity was determined by pairing DNA signals to RNA signals in each nucleus. Nearest neighbors (NN) of DNA and RNA were determined by calculating the distance between DNA and RNA centroids. DNA centroids with a NN RNA closer than 350&nbsp;nm were paired and labeled as “active genes” while those with a NN farther away were labeled as “silent genes.”</div></section><section id="sec-4-5-2"><h4>X-territory volume and sphericity measurements</h4><div id="p0465" role="paragraph">Confocal optical stacks were imported to ImageJ/Fiji and converted to 16-bit tiffs. Raw data were processed using the “Smooth” function and an automatic threshold, using either the Yen or Otsu method, was set to create 3D masks for the X chromosome territories. Assignment of the Xi was based on RNA FISH signals from the <i>Xist</i> channel. Masks were imported into 3D Suite and the volume and sphericity measurements of the X chromosomes (Xa and Xi) were extracted. Sphericity is defined as the length of the object over its width, with a maximum value of one.</div></section><section id="sec-4-5-3"><h4>Extraction of X chromosome configurations</h4><div id="p0470" role="paragraph">Confocal optical stacks from sequential rounds were imported into Fiji/ImageJ and superimposed and alignment of the two sequential rounds was performed with the affine transformation of the StackReg plugin based on the DAPI channel. Data were smoothed with a 3D Gaussian blur with a standard deviation of 1 (σ&nbsp;= 1) and background removal was performed using the “Subtract Background” plugin with a rolling ball radius of 10 pixels. The Xa and Xi (scored by the presence of <i>Xist</i> RNA) were identified and saved as separate stacks. Subsequently, each probe signal centroid was extracted using the “3D Object Counter” plugin. The 3D Object Counter generated a list of coordinates of probe signals for each channel. To assign signals to multi-spectral barcodes consisting of two labels, a nearest neighbor search between the two corresponding channels was applied based on all spatial coordinates in each channel. Once pairs of signals were assigned to the multi-spectral barcodes the coordinates obtained in the shortest wavelength were used. In cases where two adjacent signals were detected per probe, potentially due to the presence of transcripts or DNA replication, only one of the signals was used.</div><div id="p0475" role="paragraph">The coordinates of individual barcodes for the Xa and Xi at days 2 and 4 of EpiLC differentiation were reoriented in 3D space to compute spatial statistics across all cells. To obtain configurations of chromosomal backbones, for each set of probe coordinates, principal component analysis (PCA) was performed in the x and y axes using MATLAB. The z axis was unused as the segmentation resolution in that axis is significantly lower, contributing to large variations in the z coordinate (<span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Finn et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Finn, E.H. ∙ Pegoraro, G. ∙ Shachar, S. ...</div><div><strong>Comparative analysis of 2D and 3D distance measurements to study spatial genome organization</strong></div><div><em>Methods.</em> 2017; <strong>123</strong>:47-55</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.H.FinnG.PegoraroS.ShacharT.MisteliComparative+analysis+of+2D+and+3D+distance+measurements+to+study+spatial+genome+organizationMethods12320174755" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and confounding the reorientation method used, which is highly sensitive to anisotropic error. The principal component is assumed to be the “backbone” of the chromosome: the expected orientation of a chromosome if initially stretched out along that component’s direction before entropically relaxing into an equilibrium configuration. Each set of probes is rotated in order to align its corresponding principal components with the y axis and translated such that the probes’ centroid is aligned with the coordinate origin. Probes of the same loci were then statistically compared to locate their local spatial centroid and 95% confidence interval for Xa day 2, Xi day 2, Xa day 4, and Xi day 4 separately. Ellipsoids encompassing the 95% confidence interval were plotted around each loci centroid. In order to quantify the relative compaction between Xa and Xi from day 2 to day 4, the pairwise distances of 3D coordinates (x,y,z) between each barcode location were measured and averaged over all cells. Averages of Xa distances were subtracted from those of Xi at day 2 and the same was done for day 4 in order to measure the absolute change between chromosomes. A heatmap of this change was plotted, where large negative numbers indicate a higher compaction.</div></section><section id="sec-4-5-4"><h4>Single-Particle Tracking (SPT) of <i>Xist</i> foci</h4><div id="p0480" role="paragraph">Individual <i>Xist</i> particles from live-cell 3D-SIM data were extracted using TrackMate (<span class="dropBlock reference-citations"><a id="body-ref-sref104-2" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-2" aria-controls="sref104-2">Tinevez et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref104-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Tinevez, J.Y. ∙ Perry, N. ∙ Schindelin, J. ...</div><div><strong>TrackMate: An open and extensible platform for single-particle tracking</strong></div><div><em>Methods.</em> 2017; <strong>115</strong>:80-90</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.Y.TinevezN.PerryJ.SchindelinG.M.HoopesG.D.ReynoldsE.LaplantineS.Y.BednarekS.L.ShorteK.W.EliceiriTrackMate%3A+An+open+and+extensible+platform+for+single-particle+trackingMethods11520178090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), an ImageJ plugin. DoG Detector with a 0.2&nbsp;μm diameter was used to define the particles and the Simple LAP Tracker with 0.25 max linking distance, 0.3 gap-closing max distance and 2 gap-closing max frame gap were used to track the particles and generate trajectories. Trajectories that were not possible to track for over 10 consecutive frames were not used. Over 850 trajectories from 30 cells were analyzed and approximately 50% of all <i>Xist</i> foci could be tracked without manual intervention per nucleus for an average of 2min. To characterize the motion of <i>Xist</i> foci, the data extracted from the software were fed into downstream confinement analyses described in <a id="crosref1880" href="#mmc3">Methods S1</a> (see section on <b>“</b><i>Xist</i> foci position trajectories and effective confining potentials”).</div></section><section id="sec-4-5-5"><h4>Live-cell 3D-SIM image registration</h4><div id="p0485" role="paragraph">Two types of motion are captured simultaneously in live-cell 3D-SIM microscopy: a) the developmental motion of the cell and the nucleus, and b) the individual motion of <i>Xist</i> foci within the nucleus. To specifically extract (b) from live-cell 3D-SIM images we implemented the following processing pipeline using Python: 1) A set of <i>Xist</i> foci were tracked using TrackMate and spatial coordinates were extracted and imported into Python using PyTrackmate. 2) For each timestep <i>t</i>, the individual displacement vectors <b><i>x</i></b><sub><i>i,t</i></sub> of each <i>Xist</i> focus were calculated using NumPy and Pandas. 3) For each time-step, individual displacement vectors were averaged to obtain <b><i>X</i></b><sub><i>t,</i></sub> an approximation of the developmental motion in that time-step. 4) This developmental motion was then subtracted from the displacement vector of each <i>Xist</i> focus to arrive at an approximation to granule <i>i</i> ‘s motion, <b><i>x</i></b><sub><i>i,t</i></sub> -<b><i>X</i></b><sub><i>t</i></sub>.</div></section><section id="sec-4-5-6"><h4>H2B density classification and <i>Xist</i> localization</h4><div id="p0490" role="paragraph">Histone H2B-Halo<sup>JF646</sup> intensities were extracted on ImageJ/Fiji plugin using the “getValue” macro command, that iterates over every pixel in the image to get the intensity value of each pixel, generating a list of all the pixel intensities and their corresponding coordinates. The list of intensities was imported to Python. Then, seven intensity/density classes of equal variance were determined. The 3D Suite was used to create <i>Xist</i> masks, and <i>Xist</i> trajectories were extracted from TrackMate to obtain spatial coordinates (centroids) from each time point. The matrices were paired within the radius of one pixel and chromatin density classes were measured under the masks. Radial distances were measured at all pixels within the respective 100, 250, 500&nbsp;nm radius of the <i>Xist</i> centroid and the maximal intensity value within that range was defined. Averaged values were then plotted in a line graph as a function of time. To extract the nearest neighbors (NN) in chromatin density maps, neighboring intensities for each H2B pixel were determined as the average intensity of all adjacent pixels and stored in an array. A strip plot was used to plot the averaged intensity values where each value was assigned to one of the 7 classes based on the class of the origin pixel (<a id="crosref1885" href="#figs2">Figure&nbsp;S2</a>L).</div></section><section id="sec-4-5-7"><h4>FRAP data analysis</h4><div id="p0495" role="paragraph">FRAP time series were imported into Fiji and converted to 16-bit tiffs. Datasets from <i>Xist</i> and CIZ1 derived from z stacking were projected for each time point. To correct for drift, images were registered using the Correct3DDrift plugin and datasets that could not be registered were discarded. To measure FRAP recoveries data were normalized for fluorescence decay as described (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Dundr and Misteli, 2003</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Dundr, M. ∙ Misteli, T.</div><div><strong>Measuring dynamics of nuclear proteins by photobleaching</strong></div><div><em>Curr. Protoc. Cell Biol.</em> 2003;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.DundrT.MisteliMeasuring+dynamics+of+nuclear+proteins+by+photobleachingCurr.+Protoc.+Cell+Biol.200310.1002%2F0471143030.cb1305s18" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, ∼2&nbsp;μm user-defined ROIs were created to define and measure the bleached region (<i>I</i><sub>t</sub>), a randomly selected unbleached nuclear region (<i>T</i><sub>t</sub>), and a randomly selected region outside the cell (<i>B</i><sub>t</sub>) for each time point. A relative intensity for each time point was calculated using the equation: <i>I</i><sub>rel</sub>&nbsp;= (<i>T</i><sub>0</sub> − <i>B</i><sub>0</sub>) × (<i>I</i><sub>(t)</sub>– <i>B</i><sub>(t)</sub>)/(<i>T</i><sub>(t)</sub> − <i>B</i><sub>(t)</sub>) × (<i>I</i><sub>0</sub> – <i>B</i><sub>0</sub>) where <i>T</i><sub>0</sub> and <i>I</i><sub>0</sub> are derived from the average intensity of the region of interest during prebleach. To derive the FRAP recovery, <i>I</i><sub>rel</sub> was measured through time. For compiling figures, FRAP time series were bleach-corrected using the BleachCorrect ImageJ/Fiji plugin. To infer dissociation rates and residence times, FRAP curves for <i>Xist</i> and all proteins were fit to single or double exponential models derived from mass-action kinetics. Squared errors were minimized to obtain best-fit detachment rates, binding site densities, and freely diffusing fractions described in the <a id="crosref1895" href="#mmc3">Methods S1</a> (“Mass-action binding and dissociation model of FRAP dynamics”).</div></section><section id="sec-4-5-8"><h4>Image data statistical analysis and visualization</h4><div id="p0500" role="paragraph">Data analysis and visualization were performed using Python and executed in Google Colaboratory. All violin plots, boxplots, bar plots and point-plots were generated using Seaborn and Matplotlib. NumPy and SciPy were used for mathematical computation and Pandas for data analysis. Unless stated otherwise, all graphs show the median as the central point or the central line, and bars on point plots represent the standard deviation. Point plots of protein integrated density and volume in <a id="crosref1900" href="#fig3">Figures 3</a>H, <a id="crosref1905" href="#figs3">S3</a>B, <a id="crosref1910" href="#figs5">S5</a>J, S5K, and <a id="crosref1915" href="#figs6">S6</a>D show the percentage of the maximum absolute value in each group. Statistical differences between two groups were analyzed by the two-sided Wilcoxon’s or Mann-Whitney rank-sum test (scipy.stats.mannwhitneyu). The Kruskal-Wallis H-test was used for statistical comparisons between multiple groups (scipy.stats.kruskal). Statistical significance was defined as a p value less than 0.05.</div></section><section id="sec-4-5-9"><h4>RAP-seq data analysis</h4><div id="p0505" role="paragraph">RAP-seq reads were trimmed using trim_galore (v0.4.1) with default parameters to remove the standard Illumina adaptor sequences. Bowtie2 (v2.2.9) was used to align reads to the mouse genome (mm9) with the default parameters. Reads with mapping quality less than 30 were removed using samtools (v1.2). Picard MarkDuplicates (v2.1.0) was used to remove PCR duplicates. Bedtools intersect (v2.26.0) was used to count reads in sliding windows (100Kb every 25Kb) along the X chromosome. <i>Xist</i> localization across the X&nbsp;chromosome was defined by calculating the <i>Xist</i> enrichment scores (<i>Xist</i> pulldown/input) in the sliding windows. Unmappable regions were masked.</div><div id="p0510" role="paragraph">MACS2 (<span class="dropBlock reference-citations"><a id="body-ref-sref120-2" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-2" aria-controls="sref120-2">Zhang et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref120-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.ZhangT.LiuC.A.MeyerJ.EeckhouteD.S.JohnsonB.E.BernsteinC.NusbaumR.M.MyersM.BrownW.LiX.S.LiuModel-based+analysis+of+ChIP-Seq+%28MACS%29Genome+Biol.92008R137" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to call broadPeaks in D2 and D4 <i>Xist</i> pulldown data, using the input as control. Peaks within 50kb were merged. To identify overlapping <i>Xist</i> peaks between D2 and D4, the respective X-linked peak sets were intersected using bedtools intersect. The most significant peaks were selected with a log<sub>10</sub>(qvalue)&gt;13 cutoff. The R package makeVennDiagram with connectedPeaks&nbsp;= min was used to make the Venn Diagram from the two peak ranges.</div><div id="p0515" role="paragraph">To plot the average <i>Xist</i> enrichment around the summits of <i>Xist</i> peaks, <i>Xist</i> peak summits defined for D2 were used and filtered for summit scores &gt; 100 on the X chromosome. The RAP-seq enrichment scores for 2500kb up and downstream of these D2 summits were extracted from the D2 and D4 data using normalizeToMatrix function in the R package EnrichedHeatmap.</div></section><section id="sec-4-5-10"><h4>Hi-C compartments</h4><div id="p0520" role="paragraph">To compare Hi-C compartmentalization and <i>Xist</i> enrichment, Hi-C PC1 values were downloaded from GSE99991 (<span class="dropBlock reference-citations"><a id="body-ref-sref110-4" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-4" aria-controls="sref110-4">Wang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref110-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Wang, C.Y. ∙ Jegu, T. ∙ Chu, H.P. ...</div><div><strong>SMCHD1 Merges Chromosome Compartments and Assists Formation of Super-Structures on the Inactive X</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:406-421.e25</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.Y.WangT.JeguH.P.ChuH.J.OhJ.T.LeeSMCHD1+Merges+Chromosome+Compartments+and+Assists+Formation+of+Super-Structures+on+the+Inactive+XCell1742018406421.e25" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Analysis was performed in R (v3.6.0). Hi-C PC1 values from undifferentiated ESCs were correlated with <i>Xist</i> enrichment at D2 and Hi-C PC1 values at differentiation D4 with <i>Xist</i> enrichment at D4. In addition, <i>Xist</i> enrichment at D2 and D4 were correlated. Datasets were intersected with plyranges (v1.4.4) (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Lee et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Lee, S. ∙ Cook, D. ∙ Lawrence, M.</div><div><strong>plyranges: a grammar of genomic data transformation</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:4</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.LeeD.CookM.Lawrenceplyranges%3A+a+grammar+of+genomic+data+transformationGenome+Biol.2020194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Plots were made with ggplot2 (v3.3.2) and pearson’s correlation coefficients and p values (two-sided t test, r≠0) were calculated with the function stat_cor in the R package ggpubr (v0.4.0).</div></section><section id="sec-4-5-11"><h4>SNP Calling</h4><div id="p0525" role="paragraph">Single nucleotide polymorphisms (SNPs) were identified to distinguish the <i>mus castaneous</i> (Cas) genome from the 129S4/SvJae (129) or C57BL/6J (B6, reference) strains of <i>Mus musculus</i>. Parental genome sequencing data were downloaded from publicly available databases (Cas genome sequence (EMBL-EBI: ERP000042); 129 genome sequence (SRA: SRX037820). In order to identify distinguishing SNPs, we first aligned whole genome sequencing reads of each strain to the mm10 genome with bowtie2 (v2.3.5.1), filtered multimapped reads (mapq&nbsp;&lt; 10) with samtools (v1.7), and removed duplicates with GATK (v4.1.4.1) MarkDuplicates. For allelic analysis in Cas/129 F1 hybrid cells, we jointly called SNPs with the aligned Cas data and the aligned 129 data and filtered out low quality SNPs with bcftools (v1.8). For allelic analysis in Cas/B6 F1 hybrid cells, we called SNPs with the aligned Cas data alone and filtered out low quality SNPs with bcftools. SNPs were required to have a minimum depth of 5, with at least 90% of the reads supporting the alternate allele, a minimum of 3 reads supporting the alternate allele and no more than 2 reads supporting the reference allele. Indels and complex SNPs were not included. To filter the Cas/129 SNPs, SNPs that were alternate in both strains were excluded. When a SNP is alternate in one strain, the other strain must have fewer than 10% of reads supporting the alternate allele, at least 2 reads supporting the reference allele, and fewer than 2 reads supporting the alternate allele. The resulting Cas/129 and Cas/B6 variant call files (VCFs) were used in subsequent allelic analysis.</div></section><section id="sec-4-5-12"><h4>Single cell RNA-seq data analysis</h4><div id="p0530" role="paragraph">Cellranger count (v5.0.1) was used to align and process scRNA-seq reads to the mm10 genome with the option [–include-introns]. Seurat (v3.9.9.9024) in R (v3.6.1) to identify and exclude low quality cells, defined by a high percent of reads aligning to the mitochondrial genome (&gt; 7%) or a low number of unique feature counts. As the feature count per cell is partially a function of read depth per library, the threshold used was based on the number of features in the bulk of the cells in each library and ranged between 1500 and 2000. Vartrix (v1.1.14) was used to identify and count reads with informative SNPs (allelic reads) with the options [–scoring-method coverage –umi]. Either the Cas/129 VCF or the Cas/B6 VCF described above were used, as appropriate for the strain. All subsequent analyses were done in R. Allelic reads from the X chromosome were summarized for each gene to generate Xi allelic reads ratio Xi/(Xi+Xa) per X-linked gene. Lowly expressed genes were filtered out, by requiring a minimum of 10 cells in each library to have at least one allelic read of that gene. Xi allelic ratios for each gene were calculated per cell, requiring a minimum of 3 allelic reads to get an Xi ratio for a given gene in a cell.</div><div id="p0535" role="paragraph">For scRNA-seq data of the female F1 2-1 Xist<sup>129 WT / Cas WT</sup> or Xist<sup>129 ΔB / Cas WT</sup> EpiLCs, cells with biallelic or undetectable <i>Xist</i> expression (&lt; 3 allelic reads), and XO cells, were excluded, and the Xi was determined based on monoallelic <i>Xist</i> expression. Xi allelic ratios between WT<sup>129</sup> Xi and ΔΒ<sup>129</sup> Xi were compared using a one-sided welch t test (p&nbsp;&lt; 0.05) with the R function t.test, where each cell is a replicate.</div><div id="p0540" role="paragraph">For the scRNA-seq data of TX1072 (B6<sup>tetO-Xist</sup> Cas<sup>WT-Xist</sup>) ESC lines expressing WT-SPEN-GFP, ΔIDR-SPEN-GFP, and ΔSPOC-SPEN-GFP at 0&nbsp;h and 24&nbsp;h of dox treatment, where <i>Xist</i> is induced by doxycycline from the B6 allele, XO cells were excluded. All cells were assumed to inactivate the B6 X chromosome. The Xi ratios between 0 and 24&nbsp;h dox induction of <i>Xist</i> for the respective cell line were compared using a one-sided welch t test (p&nbsp;&lt; 0.05) with the R function t.test, where each cell is a replicate.</div><div id="p0545" role="paragraph">The mean Xi allelic ratios per gene were found for each set of cells and plotted with ggplot2. Statistical tests comparing distributions were performed with the function stat_compare_means in the R package ggpubr. The Wilcoxon rank sum test was used for pairwise comparisons, while the Kruskal-Wallis test was used to compare multiple groups.</div></section><section id="sec-4-5-13"><h4>Bulk RNA-seq data analysis</h4><div id="p0550" role="paragraph">Reads were aligned to the mm10 genome with STAR (v2.7.1a), filtered multimapped reads (mapq&nbsp;&lt; 10) with samtools (v1.7) and allelic ratios were found using GATK (v4.1.4.1) ASEReadCounter and the Cas/B6 VCF described above. Allelic reads were assigned to genes using bedtools intersect with all genes from the gencode mm10 annotation file (gencode.vM24.annotation.gtf). All subsequent analyses were done in R. Due to the tetO Xist promoter being on the B6 allele, the B6 X chromosome was inactivated in all cells. Allelic reads were summed across genes and the allelic reads ratio Xi/(Xi+Xa) were found. Genes with fewer than 10 reads with informative SNPs (allelic reads) were filtered out from each sample. Xi ratios between 0&nbsp;h and 24&nbsp;h dox were compared with a one-sided welch t&nbsp;test (p&nbsp;&lt; 0.05) with the R function t.test, with 3 replicates per condition. The mean Xi allelic ratios per gene were found for each condition and plotted with ggplot2. Statistical tests comparing distributions were performed with the function stat_compare_means in the R package ggpubr. The Wilcoxon rank sum test was used for pairwise comparisons, while the Kruskal-Wallis test was used to compare multiple groups.</div></section><section id="sec-4-5-14"><h4>X-linked gene silencing dynamics</h4><div id="p0555" role="paragraph">The silencing half-times of genes during XCI were downloaded from (<span class="dropBlock reference-citations"><a id="body-ref-sref3-5" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-5" aria-controls="sref3-5">Barros de Andrade E. Sousa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Barros de Andrade E. Sousa, L. ∙ Jonkers, I. ∙ Syx, L. ...</div><div><strong>Kinetics of <i>Xist</i>-induced gene silencing can be predicted from combinations of epigenetic and genomic features</strong></div><div><em>Genome Res.</em> 2019; <strong>29</strong>:1087-1099</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.Barros+de+Andrade+E.+SousaI.JonkersL.SyxI.DunkelJ.ChaumeilC.PicardB.ForetC.J.ChenJ.T.LisE.HeardKinetics+of+Xist-induced+gene+silencing+can+be+predicted+from+combinations+of+epigenetic+and+genomic+featuresGenome+Res.29201910871099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and analyzed in R. X-linked genes with a half-time range of 0-0.4&nbsp;days were classified as early-silencing genes, of 0.4–1&nbsp;days as late-silencing genes, of 1-2&nbsp;days as very late silencing genes, and &gt; 2&nbsp;days as escapee genes.</div></section><section id="sec-4-5-15"><h4>CLAP-seq data analysis</h4><div id="p0560" role="paragraph">CLAP-seq libraries of FL, ΔΙDR and ΔRRM SPEN were aligned to the mm10 genome with STAR (v2.7.1a), filtered multimapped reads (mapq&nbsp;&lt; 10) with samtools (v1.7), duplicate reads were removed with GATK (v4.1.4.1) MarkDuplicates. RPKM values were calculated for each sample in 100bp bins smoothed across 300bp with deeptools, only counting the forward strand in order to avoid double counting fragments, (v3.5.0) with the options [–normalizeUsing RPKM–binSize 100–samFlagInclude 64–skipNAs]. Enrichment was calculated in R by dividing SPEN pulldown RPKM over input RPKM in each bin. The R package ggplot2 was used to plot the enrichment scores.</div></section><section id="sec-4-5-16"><h4><i>Xist</i>-tethered SPOC silencing</h4><div id="p0565" role="paragraph">To identify genes that were repressed by SPOC tethered to <i>Xist</i> in the absence of SPEN, we used previously published allele-specific RNA-seq count data from GSE131784 of TX1072 B6<sup>tetO-Xist</sup>Cas<sup>WT-Xist</sup> ESC lines expressing SPEN-AID-GFP, where depletion of SPEN was rescued through tethering of BglG-GFP-SPOC or BglG-GFP (as a control) to tetO-<i>Xist</i>–Bgl stem–loop RNA via BglG (<span class="dropBlock reference-citations"><a id="body-ref-sref31-27" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-27" aria-controls="sref31-27">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0570" role="paragraph">SPOC silencing values for each gene were calculated as described (<span class="dropBlock reference-citations"><a id="body-ref-sref31-28" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-28" aria-controls="sref31-28">Dossin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31-28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, we filtered out genes that were skewed or not silenced under control conditions. We then calculated a silencing index under normal conditions (silencing_index<sub>DOX</sub>&nbsp;= 1&nbsp;− (allelic_ratio<sub>DOX/</sub>allelic_ratio<sub>control</sub>) and after degrading SPEN and expressing SPOC tethered to <i>Xist</i> (silencing_index<sub>SPOC</sub>&nbsp;= 1&nbsp;− (allelic_ratio<sub>SPOC</sub>/allelic_ratio<sub>control</sub>). To quantify the silencing defect in cells expressing only <i>Xist</i>-tethered SPOC, we calculated the silencing defect (silencing_defect&nbsp;= 1 – (silencing_index<sub>SPOC</sub>/silencing_index<sub>DOX</sub>). We calculated the silencing defects for the control BglG-GFP rescue identically. The silencing defect per gene was found for each condition and plotted with ggplot2. Statistical tests comparing distributions were performed with the function stat_compare_means in the R package ggpubr. The Wilcoxon rank sum test was used for pairwise comparisons, while the Kruskal-Wallis test was used to compare multiple groups.</div></section><section id="sec-4-5-17"><h4>SMCHD1 sensitivity</h4><div id="p0575" role="paragraph">To identify genes with a silencing defect as a result of the knockout of <i>Smchd1</i>, we downloaded RNA-seq allelic counts from GSE99991 and classified genes as SMCHD1-sensitive or -insensitive following the previously analysis pipeline (Wang et&nbsp;al.). All analysis was performed in R. Briefly, the Xi allelic reads ratio Xi/(Xi+Xa) was determined per X-linked gene. Genes with fewer than 13 reads with informative SNPs (allelic reads), a skewed allelic ratio, or that escape X inactivation were filtered out. SMCHD1-sensitive genes were defined as having Xi allelic ratio in SMCHD1 knockout neural progenitor cells (NPCs) of 3-fold greater than Xi allelic ratio in WT NPCs. The silencing half-time from <span class="dropBlock reference-citations"><a id="body-ref-sref3-6" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-6" aria-controls="sref3-6">Barros de Andrade E. Sousa et&nbsp;al., (2019)</a><div class="dropBlock__holder" data-db-target-of="sref3-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Barros de Andrade E. Sousa, L. ∙ Jonkers, I. ∙ Syx, L. ...</div><div><strong>Kinetics of <i>Xist</i>-induced gene silencing can be predicted from combinations of epigenetic and genomic features</strong></div><div><em>Genome Res.</em> 2019; <strong>29</strong>:1087-1099</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.Barros+de+Andrade+E.+SousaI.JonkersL.SyxI.DunkelJ.ChaumeilC.PicardB.ForetC.J.ChenJ.T.LisE.HeardKinetics+of+Xist-induced+gene+silencing+can+be+predicted+from+combinations+of+epigenetic+and+genomic+featuresGenome+Res.29201910871099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> was plotted for SMCHD1-sensitive and -insensitive genes with the R package ggplot2, and distributions were compared with a Wilcoxon rank sum test using the function stat_compare_means from the R&nbsp;package ggpubr.</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0580" role="paragraph">We thank David Baker for sharing the ct-60 gene and Yi-Yun Ho, Tsotne Chitiashvili, Amy Pandya-Jones, and Mario Blanco for help with the study. We thank Lars Dreier, Douglas Black, Emilie Marcus, and Plath lab members for critical discussions. We thank the DGSOM at UCLA, David Williams, and the Department of Biological Chemistry for support. The imaging was supported by the NIH (R01GM115233 and P30EY000331); Y.M. by NIH (R03HD095086); K.P. by an Innovation Award from the BSCRC at UCLA, NIH (R01GM115233, 1R01MH109166, R21HD094172), the Keck Foundation, and a HHMI Faculty Scholar grant; D.M. and T.C. by the NSF (DMS-1814364) and NIH (R01HL146552); and A.B. by NIH (F30HL136080).</div><section id="sec-5"><h3>Author contributions</h3><div id="p0585" role="paragraph">Conceptualization, Y.M., M.G., and K.P.; methodology, Y.M. and T.C.; software, J.G.C., E.C.J, Y.M., Y.W., D.M., I.D., B.A.M., and J.S.; validation, Y.M.; formal analysis, Y.M., J.G.C., Y.W., E.C.J., D.M., B.A.M, I.D., and C.L.; investigation, Y.M., J.G.C., Y.W., C.L., S.Y.X.T., K.P. A.K.B., and F.D.; contribution of initial IDR-dependent silencing observation, J.W.J. and M.S.; data curation, Y.M., J.G.C., Y.W., and E.C.J.; writing – original draft, Y.M., K.P., and T.C.; writing – review &amp; editing, Y.M., K.P., and T.C.; visualization, Y.M., J.C.G., Y.W., and E.C.J.; supervision, K.P., Y.M., T.C., M.G., and E.H.; project administration, Y.M. and K.P.; funding acquisition, K.P., Y.M. and T.C.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0590" role="paragraph">K.P. is a member of <i>Cell</i>’s advisory board. The authors have a patent pending related to this work.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (3)</h2><div id="p0600" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.10.022/attachment/a1d09d35-2602-408a-85ae-391f3cadaae6/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (11.91 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. A list of the number of cells analyzed and nuclear or <i>Xist</i>-associated protein particles for all investigated proteins, related to Figures 3 and S4</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.10.022/attachment/a5dcc2c3-328d-414b-92a0-e60e45bac9a5/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (1.63 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. A list of the oligonucleotides used in this study, related to STAR Methods</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.10.022/attachment/b932441e-c9ed-4c6a-8f36-1ba569a747c4/mmc3.pdf" download="mmc3.pdf" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">PDF (163.66 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Methods S1. Mathematical appendices, related to STAR Methods</div></div></div></div></section><section id="app-2"><h2>Supporting citations</h2><div id="p0605" role="paragraph">The following references appear in the supplemental information: <span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Berg and Harris, 2008</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Berg, B.A. ∙ Harris, R.C.</div><div><strong>From data to probability densities without histograms</strong></div><div><em>Comput. Phys. Commun.</em> 2008; <strong>179</strong>:443-448</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.A.BergR.C.HarrisFrom+data+to+probability+densities+without+histogramsComput.+Phys.+Commun.1792008443448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Brackley and Marenduzzo, 2020</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Brackley, C.A. ∙ Marenduzzo, D.</div><div><strong>Bridging-induced microphase separation: photobleaching experiments, chromatin domains and the need for active reactions</strong></div><div><em>Brief. Funct. Genomics.</em> 2020; <strong>19</strong>:111-118</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.A.BrackleyD.MarenduzzoBridging-induced+microphase+separation%3A+photobleaching+experiments%2C+chromatin+domains+and+the+need+for+active+reactionsBrief.+Funct.+Genomics192020111118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Braga et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Braga, J. ∙ McNally, J.G. ∙ Carmo-Fonseca, M.</div><div><strong>A reaction-diffusion model to study RNA motion by quantitative fluorescence recovery after photobleaching</strong></div><div><em>Biophys. J.</em> 2007; <strong>92</strong>:2694-2703</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.BragaJ.G.McNallyM.Carmo-FonsecaA+reaction-diffusion+model+to+study+RNA+motion+by+quantitative+fluorescence+recovery+after+photobleachingBiophys.+J.92200726942703" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Chang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Chang, J.C. ∙ Savage, V.M. ∙ Chou, T.</div><div><strong>A path-integral approach to bayesian inference for inverse problems using the semiclassical approximation</strong></div><div><em>J.&nbsp;Stat. Phys.</em> 2014; <strong>109</strong>:966-974</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.C.ChangV.M.SavageT.ChouA+path-integral+approach+to+bayesian+inference+for+inverse+problems+using+the+semiclassical+approximationJ.%C2%A0Stat.+Phys.1092014966974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">2015</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Chang, J.C. ∙ Fok, P.W. ∙ Chou, T.</div><div><strong>Bayesian Uncertainty Quantification for Bond Energies and Mobilities Using Path Integral Analysis</strong></div><div><em>Biophys. J.</em> 2015; <strong>109</strong>:966-974</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.C.ChangP.W.FokT.ChouBayesian+Uncertainty+Quantification+for+Bond+Energies+and+Mobilities+Using+Path+Integral+AnalysisBiophys.+J.1092015966974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Eliscovich et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Eliscovich, C. ∙ Buxbaum, A.R. ∙ Katz, Z.B. ...</div><div><strong>mRNA on&nbsp;the move: the road to its biological destiny</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2013; <strong>288</strong>:20361-20368</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.EliscovichA.R.BuxbaumZ.B.KatzR.H.SingermRNA+on%C2%A0the+move%3A+the+road+to+its+biological+destinyJ.%C2%A0Biol.+Chem.28820132036120368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Hendrich et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Hendrich, B.D. ∙ Plenge, R.M. ∙ Willard, H.F.</div><div><strong>Identification and characterization of the human XIST gene promoter: implications for models of X&nbsp;chromosome inactivation</strong></div><div><em>Nucleic Acids Res.</em> 1997; <strong>25</strong>:2661-2671</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.D.HendrichR.M.PlengeH.F.WillardIdentification+and+characterization+of+the+human+XIST+gene+promoter%3A+implications+for+models+of+X%C2%A0chromosome+inactivationNucleic+Acids+Res.25199726612671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Kang et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Kang, M. ∙ Day, C.A. ∙ DiBenedetto, E. ...</div><div><strong>A quantitative approach to analyze binding diffusion kinetics by confocal FRAP</strong></div><div><em>Biophys. J.</em> 2010; <strong>99</strong>:2737-2747</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.KangC.A.DayE.DiBenedettoA.K.KenworthyA+quantitative+approach+to+analyze+binding+diffusion+kinetics+by+confocal+FRAPBiophys.+J.99201027372747" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">McNally, 2008</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>McNally, J.G.</div><div><strong>Quantitative FRAP in analysis of molecular binding dynamics in&nbsp;vivo</strong></div><div><em>Methods Cell Biol.</em> 2008; <strong>85</strong>:329-351</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.G.McNallyQuantitative+FRAP+in+analysis+of+molecular+binding+dynamics+in%C2%A0vivoMethods+Cell+Biol.852008329351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Mor et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Mor, A. ∙ Suliman, S. ∙ Ben-Yishay, R. ...</div><div><strong>Dynamics of single mRNP nucleocytoplasmic transport and export through the nuclear pore in living cells</strong></div><div><em>Nat. Cell Biol.</em> 2010; <strong>12</strong>:543-552</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MorS.SulimanR.Ben-YishayS.YungerY.BrodyY.Shav-TalDynamics+of+single+mRNP+nucleocytoplasmic+transport+and+export+through+the+nuclear+pore+in+living+cellsNat.+Cell+Biol.122010543552" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">van Zon and Schofield, 2010</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>van Zon, R. ∙ Schofield, J.</div><div><strong>Constructing smooth potentials of mean force, radial distribution functions, and probability densities from sampled data</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 2010; <strong>132</strong>:154110</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.van+ZonJ.SchofieldConstructing+smooth+potentials+of+mean+force%2C+radial+distribution+functions%2C+and+probability+densities+from+sampled+dataJ.%C2%A0Chem.+Phys.1322010154110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">All genomic data (bulk mRNA-seq, scRNA-seq, CLAP-seq, RAP-seq) generated in this study have been deposited in the Gene Expression Omnibus (GEO) database. The accession number is listed in the key resources table. Accession numbers of reanalyzed publicly available data are also listed in the key resources table. Super-resolution microscopy image data, segmented masks and derived features of nuclear particles will be shared by the lead contact upon request.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">This study did not generate original code. All computational approaches and software used are described in the <a id="crosref0135" href="#sec-4">STAR Methods</a> and listed in the key resources table.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Almeida, M. ∙ Pintacuda, G. ∙ Masui, O. ...</div><div><strong>PCGF3/5-PRC1 initiates Polycomb recruitment in X chromosome inactivation</strong></div><div><em>Science.</em> 2017; <strong>356</strong>:1081-1084</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.AlmeidaG.PintacudaO.MasuiY.KosekiM.GdulaA.CeraseD.BrownA.MouldC.InnocentM.NakayamaPCGF3%2F5-PRC1+initiates+Polycomb+recruitment+in+X+chromosome+inactivationScience356201710811084" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Banani, S.F. ∙ Lee, H.O. ∙ Hyman, A.A. ...</div><div><strong>Biomolecular condensates: organizers of cellular biochemistry</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2017; <strong>18</strong>:285-298</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.F.BananiH.O.LeeA.A.HymanM.K.RosenBiomolecular+condensates%3A+organizers+of+cellular+biochemistryNat.+Rev.+Mol.+Cell+Biol.182017285298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Barros de Andrade E. Sousa, L. ∙ Jonkers, I. ∙ Syx, L. ...</div><div><strong>Kinetics of <i>Xist</i>-induced gene silencing can be predicted from combinations of epigenetic and genomic features</strong></div><div><em>Genome Res.</em> 2019; <strong>29</strong>:1087-1099</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.Barros+de+Andrade+E.+SousaI.JonkersL.SyxI.DunkelJ.ChaumeilC.PicardB.ForetC.J.ChenJ.T.LisE.HeardKinetics+of+Xist-induced+gene+silencing+can+be+predicted+from+combinations+of+epigenetic+and+genomic+featuresGenome+Res.29201910871099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Beard, C. ∙ Hochedlinger, K. ∙ Plath, K. ...</div><div><strong>Efficient method to generate single-copy transgenic mice by site-specific integration in embryonic stem cells</strong></div><div><em>Genesis.</em> 2006; <strong>44</strong>:23-28</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.BeardK.HochedlingerK.PlathA.WutzR.JaenischEfficient+method+to+generate+single-copy+transgenic+mice+by+site-specific+integration+in+embryonic+stem+cellsGenesis4420062328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Berg, B.A. ∙ Harris, R.C.</div><div><strong>From data to probability densities without histograms</strong></div><div><em>Comput. Phys. Commun.</em> 2008; <strong>179</strong>:443-448</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.A.BergR.C.HarrisFrom+data+to+probability+densities+without+histogramsComput.+Phys.+Commun.1792008443448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Bertrand, E. ∙ Chartrand, P. ∙ Schaefer, M. ...</div><div><strong>Localization of ASH1 mRNA particles in living yeast</strong></div><div><em>Mol. Cell.</em> 1998; <strong>2</strong>:437-445</div></div><div class="external-links"><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.10.022&amp;key=2-s2.0-0032185810" target="_blank" aria-label="Open Scopus in new tab">Scopus (1168)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.BertrandP.ChartrandM.SchaeferS.M.ShenoyR.H.SingerR.M.LongLocalization+of+ASH1+mRNA+particles+in+living+yeastMol.+Cell21998437445" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Blelloch, R.H. ∙ Hochedlinger, K. ∙ Yamada, Y. ...</div><div><strong>Nuclear cloning of embryonal carcinoma cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:13985-13990</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.H.BlellochK.HochedlingerY.YamadaC.BrennanM.KimB.MintzL.ChinR.JaenischNuclear+cloning+of+embryonal+carcinoma+cellsProc.+Natl.+Acad.+Sci.+USA10120041398513990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Blewitt, M.E. ∙ Gendrel, A.V. ∙ Pang, Z. ...</div><div><strong>SmcHD1, containing a structural-maintenance-of-chromosomes hinge domain, has a critical role in X inactivation</strong></div><div><em>Nat. Genet.</em> 2008; <strong>40</strong>:663-669</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.E.BlewittA.V.GendrelZ.PangD.B.SparrowN.WhitelawJ.M.CraigA.ApedaileD.J.HiltonS.L.DunwoodieN.BrockdorffSmcHD1%2C+containing+a+structural-maintenance-of-chromosomes+hinge+domain%2C+has+a+critical+role+in+X+inactivationNat.+Genet.402008663669" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9-1" title="View in article">9.</a></div><div>Bousard, A. ∙ Raposo, A.C. ∙ Żylicz, J.J. ...</div><div><strong>The role of Xist-mediated Polycomb recruitment in the initiation of X-chromosome inactivation</strong></div><div><em>EMBO Rep.</em> 2019; <strong>20</strong>:e48019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.BousardA.C.RaposoJ.J.%C5%BByliczC.PicardV.B.PiresY.QiC.GilL.SyxH.Y.ChangE.HeardS.T.da+RochaThe+role+of+Xist-mediated+Polycomb+recruitment+in+the+initiation+of+X-chromosome+inactivationEMBO+Rep.202019e48019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Boyle, S. ∙ Flyamer, I.M. ∙ Williamson, I. ...</div><div><strong>A central role for canonical PRC1 in shaping the 3D nuclear landscape</strong></div><div><em>Genes Dev.</em> 2020; <strong>34</strong>:931-949</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.BoyleI.M.FlyamerI.WilliamsonD.SenguptaW.A.BickmoreR.S.IllingworthA+central+role+for+canonical+PRC1+in+shaping+the+3D+nuclear+landscapeGenes+Dev.342020931949" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Brackley, C.A. ∙ Marenduzzo, D.</div><div><strong>Bridging-induced microphase separation: photobleaching experiments, chromatin domains and the need for active reactions</strong></div><div><em>Brief. Funct. Genomics.</em> 2020; <strong>19</strong>:111-118</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.A.BrackleyD.MarenduzzoBridging-induced+microphase+separation%3A+photobleaching+experiments%2C+chromatin+domains+and+the+need+for+active+reactionsBrief.+Funct.+Genomics192020111118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Braga, J. ∙ McNally, J.G. ∙ Carmo-Fonseca, M.</div><div><strong>A reaction-diffusion model to study RNA motion by quantitative fluorescence recovery after photobleaching</strong></div><div><em>Biophys. J.</em> 2007; <strong>92</strong>:2694-2703</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.BragaJ.G.McNallyM.Carmo-FonsecaA+reaction-diffusion+model+to+study+RNA+motion+by+quantitative+fluorescence+recovery+after+photobleachingBiophys.+J.92200726942703" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Brockdorff, N.</div><div><strong>Polycomb complexes in X chromosome inactivation</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2017; <strong>372</strong>:20170021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.BrockdorffPolycomb+complexes+in+X+chromosome+inactivationPhilos.+Trans.+R.+Soc.+Lond.+B+Biol.+Sci.372201720170021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Brockdorff, N. ∙ Bowness, J.S. ∙ Wei, G.</div><div><strong>Progress toward understanding chromosome silencing by Xist RNA</strong></div><div><em>Genes Dev.</em> 2020; <strong>34</strong>:733-744</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.BrockdorffJ.S.BownessG.WeiProgress+toward+understanding+chromosome+silencing+by+Xist+RNAGenes+Dev.342020733744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Cabili, M.N. ∙ Dunagin, M.C. ∙ McClanahan, P.D. ...</div><div><strong>Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:20</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.N.CabiliM.C.DunaginP.D.McClanahanA.BiaeschO.Padovan-MerharA.RegevJ.L.RinnA.RajLocalization+and+abundance+analysis+of+human+lncRNAs+at+single-cell+and+single-molecule+resolutionGenome+Biol.16201520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Cerase, A. ∙ Smeets, D. ∙ Tang, Y.A. ...</div><div><strong>Spatial separation of Xist RNA and polycomb proteins revealed by superresolution microscopy</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:2235-2240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.CeraseD.SmeetsY.A.TangM.GdulaF.KrausM.SpivakovB.MoindrotM.LeleuA.TattermuschJ.DemmerleSpatial+separation+of+Xist+RNA+and+polycomb+proteins+revealed+by+superresolution+microscopyProc.+Natl.+Acad.+Sci.+USA111201422352240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Cerase, A. ∙ Armaos, A. ∙ Neumayer, C. ...</div><div><strong>Phase separation drives X-chromosome inactivation: a hypothesis</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2019; <strong>26</strong>:331-334</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.CeraseA.ArmaosC.NeumayerP.AvnerM.GuttmanG.G.TartagliaPhase+separation+drives+X-chromosome+inactivation%3A+a+hypothesisNat.+Struct.+Mol.+Biol.262019331334" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Chang, J.C. ∙ Savage, V.M. ∙ Chou, T.</div><div><strong>A path-integral approach to bayesian inference for inverse problems using the semiclassical approximation</strong></div><div><em>J.&nbsp;Stat. Phys.</em> 2014; <strong>109</strong>:966-974</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.C.ChangV.M.SavageT.ChouA+path-integral+approach+to+bayesian+inference+for+inverse+problems+using+the+semiclassical+approximationJ.%C2%A0Stat.+Phys.1092014966974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Chang, J.C. ∙ Fok, P.W. ∙ Chou, T.</div><div><strong>Bayesian Uncertainty Quantification for Bond Energies and Mobilities Using Path Integral Analysis</strong></div><div><em>Biophys. J.</em> 2015; <strong>109</strong>:966-974</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.C.ChangP.W.FokT.ChouBayesian+Uncertainty+Quantification+for+Bond+Energies+and+Mobilities+Using+Path+Integral+AnalysisBiophys.+J.1092015966974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20-1" title="View in article">20.</a></div><div>Chaumeil, J. ∙ Le Baccon, P. ∙ Wutz, A. ...</div><div><strong>A novel role for Xist RNA in the formation of a repressive nuclear compartment into which genes are recruited when silenced</strong></div><div><em>Genes Dev.</em> 2006; <strong>20</strong>:2223-2237</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ChaumeilP.Le+BacconA.WutzE.HeardA+novel+role+for+Xist+RNA+in+the+formation+of+a+repressive+nuclear+compartment+into+which+genes+are+recruited+when+silencedGenes+Dev.20200622232237" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Chen, B. ∙ Gilbert, L.A. ∙ Cimini, B.A. ...</div><div><strong>Dynamic imaging of genomic loci in living human cells by an optimized CRISPR/Cas system</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1479-1491</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.ChenL.A.GilbertB.A.CiminiJ.SchnitzbauerW.ZhangG.W.LiJ.ParkE.H.BlackburnJ.S.WeissmanL.S.QiB.HuangDynamic+imaging+of+genomic+loci+in+living+human+cells+by+an+optimized+CRISPR%2FCas+systemCell155201314791491" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Chu, C. ∙ Zhang, Q.C. ∙ da Rocha, S.T. ...</div><div><strong>Systematic discovery of Xist RNA binding proteins</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:404-416</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.ChuQ.C.ZhangS.T.da+RochaR.A.FlynnM.BharadwajJ.M.CalabreseT.MagnusonE.HeardH.Y.ChangSystematic+discovery+of+Xist+RNA+binding+proteinsCell1612015404416" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Clemson, C.M. ∙ McNeil, J.A. ∙ Willard, H.F. ...</div><div><strong>XIST RNA paints the inactive X chromosome at interphase: evidence for a novel RNA involved in nuclear/chromosome structure</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1996; <strong>132</strong>:259-275</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.ClemsonJ.A.McNeilH.F.WillardJ.B.LawrenceXIST+RNA+paints+the+inactive+X+chromosome+at+interphase%3A+evidence+for+a+novel+RNA+involved+in+nuclear%2Fchromosome+structureJ.%C2%A0Cell+Biol.1321996259275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Colognori, D. ∙ Sunwoo, H. ∙ Kriz, A.J. ...</div><div><strong>Xist Deletional Analysis Reveals an Interdependency between Xist RNA and Polycomb Complexes for Spreading along the Inactive X</strong></div><div><em>Mol. Cell.</em> 2019; <strong>74</strong>:101-117.e10</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.ColognoriH.SunwooA.J.KrizC.Y.WangJ.T.LeeXist+Deletional+Analysis+Reveals+an+Interdependency+between+Xist+RNA+and+Polycomb+Complexes+for+Spreading+along+the+Inactive+XMol.+Cell742019101117.e10" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Cremer, M. ∙ Grasser, F. ∙ Lanctôt, C. ...</div><div><strong>Multicolor 3D fluorescence in&nbsp;situ hybridization for imaging interphase chromosomes</strong></div><div><em>Methods Mol. Biol.</em> 2008; <strong>463</strong>:205-239</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.CremerF.GrasserC.Lanct%C3%B4tS.M%C3%BCllerM.NeusserR.ZinnerI.SoloveiT.CremerMulticolor+3D+fluorescence+in%C2%A0situ+hybridization+for+imaging+interphase+chromosomesMethods+Mol.+Biol.4632008205239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Danecek, P. ∙ Bonfield, J.K. ∙ Liddle, J. ...</div><div><strong>Twelve years of SAMtools and BCFtools</strong></div><div><em>Gigascience.</em> 2021; <strong>10</strong>:giab008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.DanecekJ.K.BonfieldJ.LiddleJ.MarshallV.OhanM.O.PollardA.WhitwhamT.KeaneS.A.McCarthyR.M.DaviesH.LiTwelve+years+of+SAMtools+and+BCFtoolsGigascience102021giab008" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Darrow, E.M. ∙ Huntley, M.H. ∙ Dudchenko, O. ...</div><div><strong>Deletion of DXZ4 on the human inactive X chromosome alters higher-order genome architecture</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:E4504-E4512</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.M.DarrowM.H.HuntleyO.DudchenkoE.K.StamenovaN.C.DurandZ.SunS.C.HuangA.L.SanbornI.MacholM.ShamimDeletion+of+DXZ4+on+the+human+inactive+X+chromosome+alters+higher-order+genome+architectureProc.+Natl.+Acad.+Sci.+USA1132016E4504E4512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">28.</a></div><div>Demmerle, J. ∙ Innocent, C. ∙ North, A.J. ...</div><div><strong>Strategic and practical guidelines for successful structured illumination microscopy</strong></div><div><em>Nat. Protoc.</em> 2017; <strong>12</strong>:988-1010</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.DemmerleC.InnocentA.J.NorthG.BallM.M%C3%BCllerE.MironA.MatsudaI.M.DobbieY.MarkakiL.SchermellehStrategic+and+practical+guidelines+for+successful+structured+illumination+microscopyNat.+Protoc.1220179881010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29-1" title="View in article">29.</a></div><div>Derrien, T. ∙ Johnson, R. ∙ Bussotti, G. ...</div><div><strong>The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:1775-1789</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.DerrienR.JohnsonG.BussottiA.TanzerS.DjebaliH.TilgnerG.GuernecD.MartinA.MerkelD.G.KnowlesThe+GENCODE+v7+catalog+of+human+long+noncoding+RNAs%3A+analysis+of+their+gene+structure%2C+evolution%2C+and+expressionGenome+Res.22201217751789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.DobinC.A.DavisF.SchlesingerJ.DrenkowC.ZaleskiS.JhaP.BatutM.ChaissonT.R.GingerasSTAR%3A+ultrafast+universal+RNA-seq+alignerBioinformatics2920131521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">31.</a></div><div>Dossin, F. ∙ Pinheiro, I. ∙ Żylicz, J.J. ...</div><div><strong>SPEN integrates transcriptional and epigenetic control of X-inactivation</strong></div><div><em>Nature.</em> 2020; <strong>578</strong>:455-460</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.DossinI.PinheiroJ.J.%C5%BByliczJ.RoenschS.CollombetA.Le+SauxT.ChelmickiM.AttiaV.KapoorY.ZhanSPEN+integrates+transcriptional+and+epigenetic+control+of+X-inactivationNature5782020455460" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Dundr, M. ∙ Misteli, T.</div><div><strong>Measuring dynamics of nuclear proteins by photobleaching</strong></div><div><em>Curr. Protoc. Cell Biol.</em> 2003;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.DundrT.MisteliMeasuring+dynamics+of+nuclear+proteins+by+photobleachingCurr.+Protoc.+Cell+Biol.200310.1002%2F0471143030.cb1305s18" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Eliscovich, C. ∙ Buxbaum, A.R. ∙ Katz, Z.B. ...</div><div><strong>mRNA on&nbsp;the move: the road to its biological destiny</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2013; <strong>288</strong>:20361-20368</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.EliscovichA.R.BuxbaumZ.B.KatzR.H.SingermRNA+on%C2%A0the+move%3A+the+road+to+its+biological+destinyJ.%C2%A0Biol.+Chem.28820132036120368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Engreitz, J.M. ∙ Pandya-Jones, A. ∙ McDonel, P. ...</div><div><strong>The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome</strong></div><div><em>Science.</em> 2013; <strong>341</strong>:1237973</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.EngreitzA.Pandya-JonesP.McDonelA.ShishkinK.SirokmanC.SurkaS.KadriJ.XingA.GorenE.S.LanderThe+Xist+lncRNA+exploits+three-dimensional+genome+architecture+to+spread+across+the+X+chromosomeScience34120131237973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">35.</a></div><div>Engreitz, J.M. ∙ Ollikainen, N. ∙ Guttman, M.</div><div><strong>Long non-coding RNAs: spatial amplifiers that control nuclear structure and gene expression</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2016; <strong>17</strong>:756-770</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.EngreitzN.OllikainenM.GuttmanLong+non-coding+RNAs%3A+spatial+amplifiers+that+control+nuclear+structure+and+gene+expressionNat.+Rev.+Mol.+Cell+Biol.172016756770" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Finn, E.H. ∙ Pegoraro, G. ∙ Shachar, S. ...</div><div><strong>Comparative analysis of 2D and 3D distance measurements to study spatial genome organization</strong></div><div><em>Methods.</em> 2017; <strong>123</strong>:47-55</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.H.FinnG.PegoraroS.ShacharT.MisteliComparative+analysis+of+2D+and+3D+distance+measurements+to+study+spatial+genome+organizationMethods12320174755" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Francis, N.J. ∙ Kingston, R.E. ∙ Woodcock, C.L.</div><div><strong>Chromatin compaction by a polycomb group protein complex</strong></div><div><em>Science.</em> 2004; <strong>306</strong>:1574-1577</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.J.FrancisR.E.KingstonC.L.WoodcockChromatin+compaction+by+a+polycomb+group+protein+complexScience306200415741577" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>Frank, L. ∙ Rippe, K.</div><div><strong>Repetitive RNAs as Regulators of Chromatin-Associated Subcompartment Formation by Phase Separation</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 2020; <strong>432</strong>:4270-4286</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.FrankK.RippeRepetitive+RNAs+as+Regulators+of+Chromatin-Associated+Subcompartment+Formation+by+Phase+SeparationJ.%C2%A0Mol.+Biol.432202042704286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Fusco, D. ∙ Accornero, N. ∙ Lavoie, B. ...</div><div><strong>Single mRNA molecules demonstrate probabilistic movement in living mammalian cells</strong></div><div><em>Curr. Biol.</em> 2003; <strong>13</strong>:161-167</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.FuscoN.AccorneroB.LavoieS.M.ShenoyJ.M.BlanchardR.H.SingerE.BertrandSingle+mRNA+molecules+demonstrate+probabilistic+movement+in+living+mammalian+cellsCurr.+Biol.132003161167" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Galupa, R. ∙ Heard, E.</div><div><strong>X-Chromosome Inactivation: A Crossroads Between Chromosome Architecture and Gene Regulation</strong></div><div><em>Annu. Rev. Genet.</em> 2018; <strong>52</strong>:535-566</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.GalupaE.HeardX-Chromosome+Inactivation%3A+A+Crossroads+Between+Chromosome+Architecture+and+Gene+RegulationAnnu.+Rev.+Genet.522018535566" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Giorgetti, L. ∙ Lajoie, B.R. ∙ Carter, A.C. ...</div><div><strong>Structural organization of the inactive X chromosome in the mouse</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:575-579</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.GiorgettiB.R.LajoieA.C.CarterM.AttiaY.ZhanJ.XuC.J.ChenN.KaplanH.Y.ChangE.HeardJ.DekkerStructural+organization+of+the+inactive+X+chromosome+in+the+mouseNature5352016575579" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Grau, D.J. ∙ Chapman, B.A. ∙ Garlick, J.D. ...</div><div><strong>Compaction of chromatin by diverse Polycomb group proteins requires localized regions of high charge</strong></div><div><em>Genes Dev.</em> 2011; <strong>25</strong>:2210-2221</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.J.GrauB.A.ChapmanJ.D.GarlickM.BorowskyN.J.FrancisR.E.KingstonCompaction+of+chromatin+by+diverse+Polycomb+group+proteins+requires+localized+regions+of+high+chargeGenes+Dev.25201122102221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Gu, B. ∙ Posfai, E. ∙ Rossant, J.</div><div><strong>Efficient generation of targeted large insertions by microinjection into two-cell-stage mouse embryos</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:632-637</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.GuE.PosfaiJ.RossantEfficient+generation+of+targeted+large+insertions+by+microinjection+into+two-cell-stage+mouse+embryosNat.+Biotechnol.362018632637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>Gu, Z. ∙ Eils, R. ∙ Schlesner, M. ...</div><div><strong>EnrichedHeatmap: an R/Bioconductor package for comprehensive visualization of genomic signal associations</strong></div><div><em>BMC Genomics.</em> 2018; <strong>19</strong>:234</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Z.GuR.EilsM.SchlesnerN.IshaqueEnrichedHeatmap%3A+an+R%2FBioconductor+package+for+comprehensive+visualization+of+genomic+signal+associationsBMC+Genomics192018234" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Harris, C.R. ∙ Millman, K.J. ∙ van der Walt, S.J. ...</div><div><strong>Array programming with NumPy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:357-362</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.R.HarrisK.J.MillmanS.J.van+der+WaltR.GommersP.VirtanenD.CournapeauE.WieserJ.TaylorS.BergN.J.SmithArray+programming+with+NumPyNature5852020357362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">46.</a></div><div>Hayashi, K. ∙ Saitou, M.</div><div><strong>Generation of eggs from mouse embryonic stem cells and induced pluripotent stem cells</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:1513-1524</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.HayashiM.SaitouGeneration+of+eggs+from+mouse+embryonic+stem+cells+and+induced+pluripotent+stem+cellsNat.+Protoc.8201315131524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Hendrich, B.D. ∙ Plenge, R.M. ∙ Willard, H.F.</div><div><strong>Identification and characterization of the human XIST gene promoter: implications for models of X&nbsp;chromosome inactivation</strong></div><div><em>Nucleic Acids Res.</em> 1997; <strong>25</strong>:2661-2671</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.D.HendrichR.M.PlengeH.F.WillardIdentification+and+characterization+of+the+human+XIST+gene+promoter%3A+implications+for+models+of+X%C2%A0chromosome+inactivationNucleic+Acids+Res.25199726612671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">48.</a></div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.HsiaJ.B.BaleS.GonenD.ShiW.ShefflerK.K.FongU.NattermannC.XuP.S.HuangR.RavichandranDesign+of+a+hyperstable+60-subunit+protein+dodecahedron.+%5Bcorrected%5DNature5352016136139" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>Hunter, J.D.</div><div><strong>Matplotlib: A 2D Graphics Environment</strong></div><div><em>Comput. Sci. Eng.</em> 2007; <strong>9</strong>:90-95</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.D.HunterMatplotlib%3A+A+2D+Graphics+EnvironmentComput.+Sci.+Eng.920079095" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">50.</a></div><div>Illingworth, R.S.</div><div><strong>Chromatin folding and nuclear architecture: PRC1 function in 3D</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2019; <strong>55</strong>:82-90</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.S.IllingworthChromatin+folding+and+nuclear+architecture%3A+PRC1+function+in+3DCurr.+Opin.+Genet.+Dev.5520198290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib124" class="citations"><div id="sref124" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref124-1" title="View in article">51.</a></div><div>Jachowicz, J.W. ∙ Strehle, M. ∙ Banerjee, A.K. ...</div><div><strong>Xist spatially amplifies SHARP recruitment to balance chromosome-wide silencing and specificity to the X chromosome</strong></div><div><em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.W.JachowiczM.StrehleA.K.BanerjeeJ.ThaiM.R.BlancoM.GuttmanXist+spatially+amplifies+SHARP+recruitment+to+balance+chromosome-wide+silencing+and+specificity+to+the+X+chromosomebioRxiv202110.1101%2F2021.10.27.466149" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">52.</a></div><div>Jansz, N. ∙ Nesterova, T. ∙ Keniry, A. ...</div><div><strong>Smchd1 Targeting to the Inactive X Is Dependent on the Xist-HnrnpK-PRC1 Pathway</strong></div><div><em>Cell Rep.</em> 2018; <strong>25</strong>:1912-1923.e9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.JanszT.NesterovaA.KeniryM.IminitoffP.F.HickeyG.PintacudaO.MasuiS.KobelkeN.GeogheganK.A.BreslinSmchd1+Targeting+to+the+Inactive+X+Is+Dependent+on+the+Xist-HnrnpK-PRC1+PathwayCell+Rep.25201819121923.e9" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">53.</a></div><div>Jégu, T. ∙ Aeby, E. ∙ Lee, J.T.</div><div><strong>The X chromosome in space</strong></div><div><em>Nat. Rev. Genet.</em> 2017; <strong>18</strong>:377-389</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.J%C3%A9guE.AebyJ.T.LeeThe+X+chromosome+in+spaceNat.+Rev.+Genet.182017377389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">54.</a></div><div>Jonkers, I. ∙ Monkhorst, K. ∙ Rentmeester, E. ...</div><div><strong>Xist RNA is confined to the nuclear territory of the silenced X chromosome throughout the cell cycle</strong></div><div><em>Mol. Cell. Biol.</em> 2008; <strong>28</strong>:5583-5594</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.JonkersK.MonkhorstE.RentmeesterJ.A.GrootegoedF.GrosveldJ.GribnauXist+RNA+is+confined+to+the+nuclear+territory+of+the+silenced+X+chromosome+throughout+the+cell+cycleMol.+Cell.+Biol.28200855835594" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">55.</a></div><div>Kang, M. ∙ Day, C.A. ∙ DiBenedetto, E. ...</div><div><strong>A quantitative approach to analyze binding diffusion kinetics by confocal FRAP</strong></div><div><em>Biophys. J.</em> 2010; <strong>99</strong>:2737-2747</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.KangC.A.DayE.DiBenedettoA.K.KenworthyA+quantitative+approach+to+analyze+binding+diffusion+kinetics+by+confocal+FRAPBiophys.+J.99201027372747" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">56.</a></div><div>Kranz, A. ∙ Fu, J. ∙ Duerschke, K. ...</div><div><strong>An improved Flp deleter mouse in C57Bl/6 based on Flpo recombinase</strong></div><div><em>Genesis.</em> 2010; <strong>48</strong>:512-520</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.KranzJ.FuK.DuerschkeS.WeidlichR.NaumannA.F.StewartK.AnastassiadisAn+improved+Flp+deleter+mouse+in+C57Bl%2F6+based+on+Flpo+recombinaseGenesis482010512520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">57.</a></div><div>Kraus, F. ∙ Miron, E. ∙ Demmerle, J. ...</div><div><strong>Quantitative 3D structured illumination microscopy of nuclear structures</strong></div><div><em>Nat. Protoc.</em> 2017; <strong>12</strong>:1011-1028</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KrausE.MironJ.DemmerleT.ChitiashviliA.BudcoQ.AlleA.MatsudaH.LeonhardtL.SchermellehY.MarkakiQuantitative+3D+structured+illumination+microscopy+of+nuclear+structuresNat.+Protoc.12201710111028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">58.</a></div><div>Kuznetsova, I.M. ∙ Turoverov, K.K. ∙ Uversky, V.N.</div><div><strong>What macromolecular crowding can do to a protein</strong></div><div><em>Int. J. Mol. Sci.</em> 2014; <strong>15</strong>:23090-23140</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.M.KuznetsovaK.K.TuroverovV.N.UverskyWhat+macromolecular+crowding+can+do+to+a+proteinInt.+J.+Mol.+Sci.1520142309023140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">59.</a></div><div>Langmead, B. ∙ Trapnell, C. ∙ Pop, M. ...</div><div><strong>Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</strong></div><div><em>Genome Biol.</em> 2009; <strong>10</strong>:R25</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.LangmeadC.TrapnellM.PopS.L.SalzbergUltrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genomeGenome+Biol.102009R25" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">60.</a></div><div>Lee, S. ∙ Cook, D. ∙ Lawrence, M.</div><div><strong>plyranges: a grammar of genomic data transformation</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:4</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.LeeD.CookM.Lawrenceplyranges%3A+a+grammar+of+genomic+data+transformationGenome+Biol.2020194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">61.</a></div><div>Loda, A. ∙ Heard, E.</div><div><strong>Xist RNA in action: Past, present, and future</strong></div><div><em>PLoS Genet.</em> 2019; <strong>15</strong>:e1008333</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.LodaE.HeardXist+RNA+in+action%3A+Past%2C+present%2C+and+futurePLoS+Genet.152019e1008333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">62.</a></div><div>Markaki, Y. ∙ Smeets, D. ∙ Fiedler, S. ...</div><div><strong>The potential of 3D-FISH and super-resolution structured illumination microscopy for studies of 3D nuclear architecture: 3D structured illumination microscopy of defined chromosomal structures visualized by 3D (immuno)-FISH opens new perspectives for studies of nuclear architecture</strong></div><div><em>BioEssays.</em> 2012; <strong>34</strong>:412-426</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.MarkakiD.SmeetsS.FiedlerV.J.SchmidL.SchermellehT.CremerM.CremerThe+potential+of+3D-FISH+and+super-resolution+structured+illumination+microscopy+for+studies+of+3D+nuclear+architecture%3A+3D+structured+illumination+microscopy+of+defined+chromosomal+structures+visualized+by+3D+%28immuno%29-FISH+opens+new+perspectives+for+studies+of+nuclear+architectureBioEssays342012412426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">63.</a></div><div>Markaki, Y. ∙ Smeets, D. ∙ Cremer, M. ...</div><div><strong>Fluorescence in&nbsp;situ hybridization applications for super-resolution 3D structured illumination microscopy</strong></div><div><em>Methods Mol. Biol.</em> 2013; <strong>950</strong>:43-64</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.MarkakiD.SmeetsM.CremerL.SchermellehFluorescence+in%C2%A0situ+hybridization+applications+for+super-resolution+3D+structured+illumination+microscopyMethods+Mol.+Biol.95020134364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63-1" title="View in article">64.</a></div><div>McHugh, C.A. ∙ Chen, C.K. ∙ Chow, A. ...</div><div><strong>The Xist lncRNA interacts directly with SHARP to silence transcription through HDAC3</strong></div><div><em>Nature.</em> 2015; <strong>521</strong>:232-236</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.A.McHughC.K.ChenA.ChowC.F.SurkaC.TranP.McDonelA.Pandya-JonesM.BlancoC.BurghardA.MoradianThe+Xist+lncRNA+interacts+directly+with+SHARP+to+silence+transcription+through+HDAC3Nature5212015232236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">65.</a></div><div>McNally, J.G.</div><div><strong>Quantitative FRAP in analysis of molecular binding dynamics in&nbsp;vivo</strong></div><div><em>Methods Cell Biol.</em> 2008; <strong>85</strong>:329-351</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.G.McNallyQuantitative+FRAP+in+analysis+of+molecular+binding+dynamics+in%C2%A0vivoMethods+Cell+Biol.852008329351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">66.</a></div><div>McSwiggen, D.T. ∙ Mir, M. ∙ Darzacq, X. ...</div><div><strong>Evaluating phase separation in live cells: diagnosis, caveats, and functional consequences</strong></div><div><em>Genes Dev.</em> 2019; <strong>33</strong>:1619-1634</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.T.McSwiggenM.MirX.DarzacqR.TjianEvaluating+phase+separation+in+live+cells%3A+diagnosis%2C+caveats%2C+and+functional+consequencesGenes+Dev.33201916191634" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">67.</a></div><div>Minajigi, A. ∙ Froberg, J. ∙ Wei, C. ...</div><div><strong>Chromosomes. A comprehensive Xist interactome reveals cohesin repulsion and an RNA-directed chromosome conformation</strong></div><div><em>Science.</em> 2015;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MinajigiJ.FrobergC.WeiH.SunwooB.KesnerD.ColognoriD.LessingB.PayerM.BoukhaliW.HaasJ.T.LeeChromosomes.+A+comprehensive+Xist+interactome+reveals+cohesin+repulsion+and+an+RNA-directed+chromosome+conformationScience201510.1126%2Fscience.aab2276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">68.</a></div><div>Minkovsky, A. ∙ Sahakyan, A. ∙ Rankin-Gee, E. ...</div><div><strong>The Mbd1-Atf7ip-Setdb1 pathway contributes to the maintenance of X chromosome inactivation</strong></div><div><em>Epigenetics Chromatin.</em> 2014; <strong>7</strong>:12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MinkovskyA.SahakyanE.Rankin-GeeG.BonoraS.PatelK.PlathThe+Mbd1-Atf7ip-Setdb1+pathway+contributes+to+the+maintenance+of+X+chromosome+inactivationEpigenetics+Chromatin7201412" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">69.</a></div><div>Mittag, T. ∙ Forman-Kay, J.D.</div><div><strong>Atomic-level characterization of disordered protein ensembles</strong></div><div><em>Curr. Opin. Struct. Biol.</em> 2007; <strong>17</strong>:3-14</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.MittagJ.D.Forman-KayAtomic-level+characterization+of+disordered+protein+ensemblesCurr.+Opin.+Struct.+Biol.172007314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">70.</a></div><div>Mohammed, H. ∙ Taylor, C. ∙ Brown, G.D. ...</div><div><strong>Rapid immunoprecipitation mass spectrometry of endogenous proteins (RIME) for analysis of chromatin complexes</strong></div><div><em>Nat. Protoc.</em> 2016; <strong>11</strong>:316-326</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.MohammedC.TaylorG.D.BrownE.K.PapachristouJ.S.CarrollC.S.D%E2%80%99SantosRapid+immunoprecipitation+mass+spectrometry+of+endogenous+proteins+%28RIME%29+for+analysis+of+chromatin+complexesNat.+Protoc.112016316326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">71.</a></div><div>Moindrot, B. ∙ Cerase, A. ∙ Coker, H. ...</div><div><strong>A Pooled shRNA Screen Identifies Rbm15, Spen, and Wtap as Factors Required for Xist RNA-Mediated Silencing</strong></div><div><em>Cell Rep.</em> 2015; <strong>12</strong>:562-572</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.MoindrotA.CeraseH.CokerO.MasuiA.GrijzenhoutG.PintacudaL.SchermellehT.B.NesterovaN.BrockdorffA+Pooled+shRNA+Screen+Identifies+Rbm15%2C+Spen%2C+and+Wtap+as+Factors+Required+for+Xist+RNA-Mediated+SilencingCell+Rep.122015562572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">72.</a></div><div>Monfort, A. ∙ Di Minin, G. ∙ Postlmayr, A. ...</div><div><strong>Identification of Spen as a Crucial Factor for Xist Function through Forward Genetic Screening in Haploid Embryonic Stem Cells</strong></div><div><em>Cell Rep.</em> 2015; <strong>12</strong>:554-561</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MonfortG.Di+MininA.PostlmayrR.FreimannF.ArietiS.ThoreA.WutzIdentification+of+Spen+as+a+Crucial+Factor+for+Xist+Function+through+Forward+Genetic+Screening+in+Haploid+Embryonic+Stem+CellsCell+Rep.122015554561" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">73.</a></div><div>Mor, A. ∙ Suliman, S. ∙ Ben-Yishay, R. ...</div><div><strong>Dynamics of single mRNP nucleocytoplasmic transport and export through the nuclear pore in living cells</strong></div><div><em>Nat. Cell Biol.</em> 2010; <strong>12</strong>:543-552</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.MorS.SulimanR.Ben-YishayS.YungerY.BrodyY.Shav-TalDynamics+of+single+mRNP+nucleocytoplasmic+transport+and+export+through+the+nuclear+pore+in+living+cellsNat.+Cell+Biol.122010543552" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">74.</a></div><div>Nam, H.S. ∙ Benezra, R.</div><div><strong>High levels of Id1 expression define B1 type adult neural stem cells</strong></div><div><em>Cell Stem Cell.</em> 2009; <strong>5</strong>:515-526</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.S.NamR.BenezraHigh+levels+of+Id1+expression+define+B1+type+adult+neural+stem+cellsCell+Stem+Cell52009515526" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74-1" title="View in article">75.</a></div><div>Nesterova, T.B. ∙ Wei, G. ∙ Coker, H. ...</div><div><strong>Systematic allelic analysis defines the interplay of key pathways in X chromosome inactivation</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:3129</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.B.NesterovaG.WeiH.CokerG.PintacudaJ.S.BownessT.ZhangM.AlmeidaB.BloechlB.MoindrotE.J.CarterSystematic+allelic+analysis+defines+the+interplay+of+key+pathways+in+X+chromosome+inactivationNat.+Commun.1020193129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">76.</a></div><div>Ng, K. ∙ Daigle, N. ∙ Bancaud, A. ...</div><div><strong>A system for imaging the regulatory noncoding Xist RNA in living mouse embryonic stem cells</strong></div><div><em>Mol. Biol. Cell.</em> 2011; <strong>22</strong>:2634-2645</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.NgN.DaigleA.BancaudT.OhhataP.HumphreysR.WalkerJ.EllenbergA.WutzA+system+for+imaging+the+regulatory+noncoding+Xist+RNA+in+living+mouse+embryonic+stem+cellsMol.+Biol.+Cell22201126342645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">77.</a></div><div>Nozaki, T. ∙ Imai, R. ∙ Tanbo, M. ...</div><div><strong>Dynamic Organization of Chromatin Domains Revealed by Super-Resolution Live-Cell Imaging</strong></div><div><em>Mol. Cell.</em> 2017; <strong>67</strong>:282-293.e7</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.NozakiR.ImaiM.TanboR.NagashimaS.TamuraT.TaniY.JotiM.TomitaK.HibinoM.T.KanemakiDynamic+Organization+of+Chromatin+Domains+Revealed+by+Super-Resolution+Live-Cell+ImagingMol.+Cell672017282293.e7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">78.</a></div><div>Ollion, J. ∙ Cochennec, J. ∙ Loll, F. ...</div><div><strong>TANGO: a generic tool for high-throughput 3D image analysis for studying nuclear organization</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1840-1841</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.OllionJ.CochennecF.LollC.Escud%C3%A9T.BoudierTANGO%3A+a+generic+tool+for+high-throughput+3D+image+analysis+for+studying+nuclear+organizationBioinformatics29201318401841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">79.</a></div><div>Pacini, G. ∙ Dunkel, I. ∙ Mages, N. ...</div><div><strong>Integrated analysis of Xist upregulation and X-chromosome inactivation with single-cell and single-allele resolution</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:3638</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.PaciniI.DunkelN.MagesV.MutzelB.TimmermannA.MarsicoE.G.SchulzIntegrated+analysis+of+Xist+upregulation+and+X-chromosome+inactivation+with+single-cell+and+single-allele+resolutionNat.+Commun.1220213638" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79-1" title="View in article">80.</a></div><div>Pandya-Jones, A. ∙ Markaki, Y. ∙ Serizay, J. ...</div><div><strong>A protein assembly mediates Xist localization and gene silencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:145-151</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Pandya-JonesY.MarkakiJ.SerizayT.ChitiashviliW.R.Mancia+LeonA.DamianovC.ChronisB.PappC.K.ChenR.McKeeA+protein+assembly+mediates+Xist+localization+and+gene+silencingNature5872020145151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80-1" title="View in article">81.</a></div><div>Panning, B. ∙ Dausman, J. ∙ Jaenisch, R.</div><div><strong>X chromosome inactivation is mediated by Xist RNA stabilization</strong></div><div><em>Cell.</em> 1997; <strong>90</strong>:907-916</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.PanningJ.DausmanR.JaenischX+chromosome+inactivation+is+mediated+by+Xist+RNA+stabilizationCell901997907916" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81-1" title="View in article">82.</a></div><div>Pintacuda, G. ∙ Wei, G. ∙ Roustan, C. ...</div><div><strong>hnRNPK Recruits PCGF3/5-PRC1 to the Xist RNA B-Repeat to Establish Polycomb-Mediated Chromosomal Silencing</strong></div><div><em>Mol. Cell.</em> 2017; <strong>68</strong>:955-969.e10</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.PintacudaG.WeiC.RoustanB.A.KirmizitasN.SolcanA.CeraseA.CastelloS.MohammedB.MoindrotT.B.NesterovahnRNPK+Recruits+PCGF3%2F5-PRC1+to+the+Xist+RNA+B-Repeat+to+Establish+Polycomb-Mediated+Chromosomal+SilencingMol.+Cell682017955969.e10" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82-1" title="View in article">83.</a></div><div>Plath, K. ∙ Mlynarczyk-Evans, S. ∙ Nusinow, D.A. ...</div><div><strong>Xist RNA and the mechanism of X chromosome inactivation</strong></div><div><em>Annu. Rev. Genet.</em> 2002; <strong>36</strong>:233-278</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.PlathS.Mlynarczyk-EvansD.A.NusinowB.PanningXist+RNA+and+the+mechanism+of+X+chromosome+inactivationAnnu.+Rev.+Genet.362002233278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">84.</a></div><div>Plath, K. ∙ Fang, J. ∙ Mlynarczyk-Evans, S.K. ...</div><div><strong>Role of histone H3 lysine 27 methylation in X inactivation</strong></div><div><em>Science.</em> 2003; <strong>300</strong>:131-135</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.PlathJ.FangS.K.Mlynarczyk-EvansR.CaoK.A.WorringerH.WangC.C.de+la+CruzA.P.OtteB.PanningY.ZhangRole+of+histone+H3+lysine+27+methylation+in+X+inactivationScience3002003131135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">85.</a></div><div>Quinlan, A.R. ∙ Hall, I.M.</div><div><strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:841-842</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.R.QuinlanI.M.HallBEDTools%3A+a+flexible+suite+of+utilities+for+comparing+genomic+featuresBioinformatics262010841842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">86.</a></div><div>Quinodoz, S.A. ∙ Bhat, P. ∙ Ollikainen, N. ...</div><div><strong>RNA promotes the formation of spatial compartments in the nucleus</strong></div><div><em>bioRxiv.</em> 2020;</div><div>2020.2008.2025.267435</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.A.QuinodozP.BhatN.OllikainenJ.W.JachowiczA.K.BanerjeeP.ChovanecM.R.BlancoA.ChowY.MarkakiK.PlathM.GuttmanRNA+promotes+the+formation+of+spatial+compartments+in+the+nucleusbioRxiv20202020.2008.2025.267435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">87.</a></div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing</strong></div><div>R Foundation for Statistical Computing, Vienna, Austria, 2021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing2021R+Foundation+for+Statistical+ComputingVienna%2C+Austria" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">88.</a></div><div>Ramírez, F. ∙ Ryan, D.P. ∙ Grüning, B. ...</div><div><strong>deepTools2: a next generation web server for deep-sequencing data analysis</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W160-W165</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.Ram%C3%ADrezD.P.RyanB.Gr%C3%BCningV.BhardwajF.KilpertA.S.RichterS.HeyneF.D%C3%BCndarT.MankedeepTools2%3A+a+next+generation+web+server+for+deep-sequencing+data+analysisNucleic+Acids+Res.44W12016W160W165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">89.</a></div><div>Ridings-Figueroa, R. ∙ Stewart, E.R. ∙ Nesterova, T.B. ...</div><div><strong>The nuclear matrix protein CIZ1 facilitates localization of Xist RNA to the inactive X-chromosome territory</strong></div><div><em>Genes Dev.</em> 2017; <strong>31</strong>:876-888</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.Ridings-FigueroaE.R.StewartT.B.NesterovaH.CokerG.PintacudaJ.GodwinR.WilsonA.HaslamF.LilleyR.RuigrokThe+nuclear+matrix+protein+CIZ1+facilitates+localization+of+Xist+RNA+to+the+inactive+X-chromosome+territoryGenes+Dev.312017876888" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">90.</a></div><div>Rinn, J.L. ∙ Chang, H.Y.</div><div><strong>Genome regulation by long noncoding RNAs</strong></div><div><em>Annu. Rev. Biochem.</em> 2012; <strong>81</strong>:145-166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.RinnH.Y.ChangGenome+regulation+by+long+noncoding+RNAsAnnu.+Rev.+Biochem.812012145166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">91.</a></div><div>RStudio Team</div><div><strong>RStudio: Integrated Development for R. RStudio</strong></div><div>PBC, Boston, MA, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=RStudio+TeamRStudio%3A+Integrated+Development+for+R.+RStudio2020PBCBoston%2C+MA" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">92.</a></div><div>Rueden, C.T. ∙ Schindelin, J. ∙ Hiner, M.C. ...</div><div><strong>ImageJ2: ImageJ for the next generation of scientific image data</strong></div><div><em>BMC Bioinformatics.</em> 2017; <strong>18</strong>:529</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.T.RuedenJ.SchindelinM.C.HinerB.E.DeZoniaA.E.WalterE.T.ArenaK.W.EliceiriImageJ2%3A+ImageJ+for+the+next+generation+of+scientific+image+dataBMC+Bioinformatics182017529" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">93.</a></div><div>Schertzer, M.D. ∙ Braceros, K.C.A. ∙ Starmer, J. ...</div><div><strong>lncRNA-Induced Spread of Polycomb Controlled by Genome Architecture, RNA Abundance, and CpG Island DNA</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:523-537.e10</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.D.SchertzerK.C.A.BracerosJ.StarmerR.E.CherneyD.M.LeeG.SalazarM.JusticeS.R.BischoffD.O.CowleyP.AriellncRNA-Induced+Spread+of+Polycomb+Controlled+by+Genome+Architecture%2C+RNA+Abundance%2C+and+CpG+Island+DNAMol.+Cell752019523537.e10" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-1" title="View in article">94.</a></div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.SchindelinI.Arganda-CarrerasE.FriseV.KaynigM.LongairT.PietzschS.PreibischC.RuedenS.SaalfeldB.SchmidFiji%3A+an+open-source+platform+for+biological-image+analysisNat.+Methods92012676682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94-1" title="View in article">95.</a></div><div>Silva, J. ∙ Mak, W. ∙ Zvetkova, I. ...</div><div><strong>Establishment of histone h3 methylation on the inactive X chromosome requires transient recruitment of Eed-Enx1 polycomb group complexes</strong></div><div><em>Dev. Cell.</em> 2003; <strong>4</strong>:481-495</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.SilvaW.MakI.ZvetkovaR.AppanahT.B.NesterovaZ.WebsterA.H.PetersT.JenuweinA.P.OtteN.BrockdorffEstablishment+of+histone+h3+methylation+on+the+inactive+X+chromosome+requires+transient+recruitment+of+Eed-Enx1+polycomb+group+complexesDev.+Cell42003481495" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">96.</a></div><div>Simon, M.D. ∙ Pinter, S.F. ∙ Fang, R. ...</div><div><strong>High-resolution Xist binding maps reveal two-step spreading during X-chromosome inactivation</strong></div><div><em>Nature.</em> 2013; <strong>504</strong>:465-469</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.D.SimonS.F.PinterR.FangK.SarmaM.Rutenberg-SchoenbergS.K.BowmanB.A.KesnerV.K.MaierR.E.KingstonJ.T.LeeHigh-resolution+Xist+binding+maps+reveal+two-step+spreading+during+X-chromosome+inactivationNature5042013465469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">97.</a></div><div>Sladitschek, H.L. ∙ Neveu, P.A.</div><div><strong>MXS-Chaining: A Highly Efficient Cloning Platform for Imaging and Flow Cytometry Approaches in Mammalian Systems</strong></div><div><em>PLoS ONE.</em> 2015; <strong>10</strong>:e0124958</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.L.SladitschekP.A.NeveuMXS-Chaining%3A+A+Highly+Efficient+Cloning+Platform+for+Imaging+and+Flow+Cytometry+Approaches+in+Mammalian+SystemsPLoS+ONE102015e0124958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">98.</a></div><div>Smeets, D. ∙ Markaki, Y. ∙ Schmid, V.J. ...</div><div><strong>Three-dimensional super-resolution microscopy of the inactive X chromosome territory reveals a collapse of its active nuclear compartment harboring distinct Xist RNA foci</strong></div><div><em>Epigenetics Chromatin.</em> 2014; <strong>7</strong>:8</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.SmeetsY.MarkakiV.J.SchmidF.KrausA.TattermuschA.CeraseM.SterrS.FiedlerJ.DemmerleJ.PopkenThree-dimensional+super-resolution+microscopy+of+the+inactive+X+chromosome+territory+reveals+a+collapse+of+its+active+nuclear+compartment+harboring+distinct+Xist+RNA+fociEpigenetics+Chromatin720148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">99.</a></div><div>Statello, L. ∙ Guo, C.J. ∙ Chen, L.L. ...</div><div><strong>Gene regulation by long non-coding RNAs and its biological functions</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2021; <strong>22</strong>:96-118</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.StatelloC.J.GuoL.L.ChenM.HuarteGene+regulation+by+long+non-coding+RNAs+and+its+biological+functionsNat.+Rev.+Mol.+Cell+Biol.22202196118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99-1" title="View in article">100.</a></div><div>Sunwoo, H. ∙ Wu, J.Y. ∙ Lee, J.T.</div><div><strong>The Xist RNA-PRC2 complex at 20-nm resolution reveals a low Xist stoichiometry and suggests a hit-and-run mechanism in mouse cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4216-E4225</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.SunwooJ.Y.WuJ.T.LeeThe+Xist+RNA-PRC2+complex+at+20-nm+resolution+reveals+a+low+Xist+stoichiometry+and+suggests+a+hit-and-run+mechanism+in+mouse+cellsProc.+Natl.+Acad.+Sci.+USA1122015E4216E4225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">101.</a></div><div>Sunwoo, H. ∙ Colognori, D. ∙ Froberg, J.E. ...</div><div><strong>Repeat E anchors Xist RNA to the inactive X chromosomal compartment through&nbsp;CDKN1A-interacting protein (CIZ1)</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:10654-10659</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.SunwooD.ColognoriJ.E.FrobergY.JeonJ.T.LeeRepeat+E+anchors+Xist+RNA+to+the+inactive+X+chromosomal+compartment+through%C2%A0CDKN1A-interacting+protein+%28CIZ1%29Proc.+Natl.+Acad.+Sci.+USA11420171065410659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">102.</a></div><div>Tavares, L. ∙ Dimitrova, E. ∙ Oxley, D. ...</div><div><strong>RYBP-PRC1 complexes mediate H2A ubiquitylation at polycomb target sites independently of PRC2 and H3K27me3</strong></div><div><em>Cell.</em> 2012; <strong>148</strong>:664-678</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.TavaresE.DimitrovaD.OxleyJ.WebsterR.PootJ.DemmersK.BezstarostiS.TaylorH.UraH.KoideRYBP-PRC1+complexes+mediate+H2A+ubiquitylation+at+polycomb+target+sites+independently+of+PRC2+and+H3K27me3Cell1482012664678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102-1" title="View in article">103.</a></div><div>Teller, K. ∙ Illner, D. ∙ Thamm, S. ...</div><div><strong>A top-down analysis of Xa- and Xi-territories reveals differences of higher order structure at <sup>3</sup> 20 Mb genomic length scales</strong></div><div><em>Nucleus.</em> 2011; <strong>2</strong>:465-477</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.TellerD.IllnerS.ThammC.S.Casas-DelucchiR.VersteegM.IndemansT.CremerM.CremerA+top-down+analysis+of+Xa-+and+Xi-territories+reveals+differences+of+higher+order+structure+at+3+20+Mb+genomic+length+scalesNucleus22011465477" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">104.</a></div><div>Reback, J. ∙ McKinney, W. ∙ Mendel, B. ...</div><div><strong>pandas-dev/pandas: Pandas</strong></div><div><em>Zenodo.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.RebackW.McKinneyB.MendelJ.Van+den+BosscheT.AugspurgerP.CloudgfyoungS.HawkinsSinhrksM.Roeschkepandas-dev%2Fpandas%3A+PandasZenodo202010.5281%2Fzenodo.3509134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104-1" title="View in article">105.</a></div><div>Tinevez, J.Y. ∙ Perry, N. ∙ Schindelin, J. ...</div><div><strong>TrackMate: An open and extensible platform for single-particle tracking</strong></div><div><em>Methods.</em> 2017; <strong>115</strong>:80-90</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.Y.TinevezN.PerryJ.SchindelinG.M.HoopesG.D.ReynoldsE.LaplantineS.Y.BednarekS.L.ShorteK.W.EliceiriTrackMate%3A+An+open+and+extensible+platform+for+single-particle+trackingMethods11520178090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105-1" title="View in article">106.</a></div><div>Uversky, V.N.</div><div><strong>The multifaceted roles of intrinsic disorder in protein complexes</strong></div><div><em>FEBS Lett.</em> 2015; <strong>589</strong>:2498-2506</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.N.UverskyThe+multifaceted+roles+of+intrinsic+disorder+in+protein+complexesFEBS+Lett.58919+Pt+A201524982506" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106" title="View in article">107.</a></div><div>Van der Auwera, G.A. ∙ O’Connor, B.D.</div><div><strong>Genomics in the Cloud: Using Docker, GATK, and WDL in Terra</strong></div><div>O’Reilly Media, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.A.Van+der+AuweraB.D.O%E2%80%99ConnorGenomics+in+the+Cloud%3A+Using+Docker%2C+GATK%2C+and+WDL+in+TerraFirst+Edition2020O%E2%80%99Reilly+Media" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107" title="View in article">108.</a></div><div>van Rossum, G. ∙ Drake, F.L.</div><div><strong>Python 3 Reference Manual</strong></div><div>Scotts Valley, CA, 2009</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.van+RossumF.L.DrakePython+3+Reference+Manual2009Scotts+ValleyCA" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">109.</a></div><div>van Zon, R. ∙ Schofield, J.</div><div><strong>Constructing smooth potentials of mean force, radial distribution functions, and probability densities from sampled data</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 2010; <strong>132</strong>:154110</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.van+ZonJ.SchofieldConstructing+smooth+potentials+of+mean+force%2C+radial+distribution+functions%2C+and+probability+densities+from+sampled+dataJ.%C2%A0Chem.+Phys.1322010154110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109" title="View in article">110.</a></div><div>Virtanen, P. ∙ Gommers, R. ∙ Oliphant, T.E. ..., SciPy 1.0 Contributors</div><div><strong>SciPy 1.0: fundamental algorithms for scientific computing in Python</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:261-272</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.VirtanenR.GommersT.E.OliphantM.HaberlandT.ReddyD.CournapeauE.BurovskiP.PetersonW.WeckesserJ.BrightSciPy+1.0+ContributorsSciPy+1.0%3A+fundamental+algorithms+for+scientific+computing+in+PythonNat.+Methods172020261272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110-1" title="View in article">111.</a></div><div>Wang, C.Y. ∙ Jegu, T. ∙ Chu, H.P. ...</div><div><strong>SMCHD1 Merges Chromosome Compartments and Assists Formation of Super-Structures on the Inactive X</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:406-421.e25</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.Y.WangT.JeguH.P.ChuH.J.OhJ.T.LeeSMCHD1+Merges+Chromosome+Compartments+and+Assists+Formation+of+Super-Structures+on+the+Inactive+XCell1742018406421.e25" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111-1" title="View in article">112.</a></div><div>Wang, C.Y. ∙ Colognori, D. ∙ Sunwoo, H. ...</div><div><strong>PRC1 collaborates with SMCHD1 to fold the X-chromosome and spread Xist RNA between chromosome compartments</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2950</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.Y.WangD.ColognoriH.SunwooD.WangJ.T.LeePRC1+collaborates+with+SMCHD1+to+fold+the+X-chromosome+and+spread+Xist+RNA+between+chromosome+compartmentsNat.+Commun.1020192950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">113.</a></div><div>Waskom, M.L.</div><div><strong>seaborn: statistical data visualization</strong></div><div><em>Journal of Open Source Software.</em> 2021; <strong>6</strong>:3021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.L.Waskomseaborn%3A+statistical+data+visualizationJournal+of+Open+Source+Software620213021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113" title="View in article">114.</a></div><div>Wickham, H. ∙ Averick, M. ∙ Bryan, J. ...</div><div><strong>Welcome to the tidyverse</strong></div><div><em>Journal of Open Source Software.</em> 2019; <strong>4</strong>:1686</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamM.AverickJ.BryanW.ChangL.D%E2%80%99Agostino+McGowanR.Fran%C3%A7oisG.GrolemundA.HayesL.HenryJ.HesterWelcome+to+the+tidyverseJournal+of+Open+Source+Software420191686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114" title="View in article">115.</a></div><div>Wu, B. ∙ Chao, J.A. ∙ Singer, R.H.</div><div><strong>Fluorescence fluctuation spectroscopy enables quantitative imaging of single mRNAs in living cells</strong></div><div><em>Biophys. J.</em> 2012; <strong>102</strong>:2936-2944</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.WuJ.A.ChaoR.H.SingerFluorescence+fluctuation+spectroscopy+enables+quantitative+imaging+of+single+mRNAs+in+living+cellsBiophys.+J.102201229362944" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115" title="View in article">116.</a></div><div>Wutz, A.</div><div><strong>Gene silencing in X-chromosome inactivation: advances in understanding facultative heterochromatin formation</strong></div><div><em>Nat. Rev. Genet.</em> 2011; <strong>12</strong>:542-553</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.WutzGene+silencing+in+X-chromosome+inactivation%3A+advances+in+understanding+facultative+heterochromatin+formationNat.+Rev.+Genet.122011542553" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116-1" title="View in article">117.</a></div><div>Wutz, A. ∙ Jaenisch, R.</div><div><strong>A shift from reversible to irreversible X inactivation is triggered during ES cell differentiation</strong></div><div><em>Mol. Cell.</em> 2000; <strong>5</strong>:695-705</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.WutzR.JaenischA+shift+from+reversible+to+irreversible+X+inactivation+is+triggered+during+ES+cell+differentiationMol.+Cell52000695705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117" title="View in article">118.</a></div><div>Xie, L. ∙ Liu, Z.</div><div><strong>Single-cell imaging of genome organization and dynamics</strong></div><div><em>Mol. Syst. Biol.</em> 2021; <strong>17</strong>:e9653</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.XieZ.LiuSingle-cell+imaging+of+genome+organization+and+dynamicsMol.+Syst.+Biol.172021e9653" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118" title="View in article">119.</a></div><div>Ying, Q.L. ∙ Smith, A.G.</div><div><strong>Defined conditions for neural commitment and differentiation</strong></div><div><em>Methods Enzymol.</em> 2003; <strong>365</strong>:327-341</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Q.L.YingA.G.SmithDefined+conditions+for+neural+commitment+and+differentiationMethods+Enzymol.3652003327341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119" title="View in article">120.</a></div><div>Yue, M. ∙ Ogawa, A. ∙ Yamada, N. ...</div><div><strong>Xist RNA repeat E is essential for ASH2L recruitment to the inactive X&nbsp;and regulates histone modifications and escape gene expression</strong></div><div><em>PLoS Genet.</em> 2017; <strong>13</strong>:e1006890</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.YueA.OgawaN.YamadaJ.L.Charles+RichardA.BarskiY.OgawaXist+RNA+repeat+E+is+essential+for+ASH2L+recruitment+to+the+inactive+X%C2%A0and+regulates+histone+modifications+and+escape+gene+expressionPLoS+Genet.132017e1006890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="sref120" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref120-1" title="View in article">121.</a></div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.ZhangT.LiuC.A.MeyerJ.EeckhouteD.S.JohnsonB.E.BernsteinC.NusbaumR.M.MyersM.BrownW.LiX.S.LiuModel-based+analysis+of+ChIP-Seq+%28MACS%29Genome+Biol.92008R137" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="sref121" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref121" title="View in article">122.</a></div><div>Zhu, L.J. ∙ Gazin, C. ∙ Lawson, N.D. ...</div><div><strong>ChIPpeakAnno: a Bioconductor package to annotate ChIP-seq and ChIP-chip data</strong></div><div><em>BMC Bioinformatics.</em> 2010; <strong>11</strong>:237</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.J.ZhuC.GazinN.D.LawsonH.Pag%C3%A8sS.M.LinD.S.LapointeM.R.GreenChIPpeakAnno%3A+a+Bioconductor+package+to+annotate+ChIP-seq+and+ChIP-chip+dataBMC+Bioinformatics112010237" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="sref122" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref122" title="View in article">123.</a></div><div>Zimmerman, S.B. ∙ Pheiffer, B.H.</div><div><strong>Macromolecular crowding allows blunt-end ligation by DNA ligases from rat liver or Escherichia coli</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1983; <strong>80</strong>:5852-5856</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.B.ZimmermanB.H.PheifferMacromolecular+crowding+allows+blunt-end+ligation+by+DNA+ligases+from+rat+liver+or+Escherichia+coliProc.+Natl.+Acad.+Sci.+USA80198358525856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib123" class="citations"><div id="sref123" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref123-1" title="View in article">124.</a></div><div>Zylicz, J.J. ∙ Bousard, A. ∙ Zumer, K. ...</div><div><strong>The Implication of Early Chromatin Changes in X Chromosome Inactivation</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:182-197.e23</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.J.ZyliczA.BousardK.ZumerF.DossinE.MohammadS.T.da+RochaB.SchwalbL.SyxF.DingliD.LoewThe+Implication+of+Early+Chromatin+Changes+in+X+Chromosome+InactivationCell1762019182197.e23" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/c19b267e-71ac-4696-b50d-4957eeae7fd7/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/99a82c67-aef8-4a7d-83af-ffab46c09f96/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/21395ca2-4f88-4fae-9dd9-b175525fbee6/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/c4ef57c8-83d0-42f5-93e1-cee376cf8414/main.assets/gr2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/d11d2385-8963-44e4-b02c-7d96d756b4ae/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/1fa7e093-5e9d-4586-a402-f59235d458c7/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/f15882a8-f37c-47a0-8151-994e0cda80d0/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/aaa684a4-a2b2-4e6a-8f63-c1622f867719/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/11c66072-78b4-4f32-9259-64d85c508dcf/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/4b5346e3-2243-4821-8deb-53b4fa1d27ed/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/4ef9c592-535b-45a9-932b-7b2510be73ee/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/b1820108-2d53-4388-b563-1ad19068c872/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/582e4f5d-02cd-4309-8b18-bde7a9e6e16e/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/asset/684cc1a4-5d2e-4da2-99a0-6db9103b5649/main.assets/gr7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS0092867421012757%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00262%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (3)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/attachment/a1d09d35-2602-408a-85ae-391f3cadaae6/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (11.91 KB)</span></a><div class="core-description">Table S1. A list of the number of cells analyzed and nuclear or <i>Xist</i>-associated protein particles for all investigated proteins, related to Figures 3 and S4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/attachment/a5dcc2c3-328d-414b-92a0-e60e45bac9a5/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (1.63 MB)</span></a><div class="core-description">Table S2. A list of the oligonucleotides used in this study, related to STAR Methods</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.10.022/attachment/b932441e-c9ed-4c6a-8f36-1ba569a747c4/mmc3.pdf" download="mmc3.pdf"><span id="mmc3-link">PDF (163.66 KB)</span></a><div class="core-description">Methods S1. Mathematical appendices, related to STAR Methods</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421012757" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.10.022","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Xist nucleates local protein gradients to propagate silencing across the X chromosome</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.10.022/asset/c19b267e-71ac-4696-b50d-4957eeae7fd7/main.assets/gr1.jpg" height="710" width="752" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/8f06d774-4912-42d1-a5ad-de7455b82a77/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.10.022/asset/99a82c67-aef8-4a7d-83af-ffab46c09f96/main.assets/figs1.jpg" height="939" width="695" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/f848c78d-eaae-42e2-b03c-e0ddd9c6ac4c/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.10.022/asset/21395ca2-4f88-4fae-9dd9-b175525fbee6/main.assets/figs2.jpg" height="883" width="750" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/751fb368-e5f8-4ce6-becc-55e342033636/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.10.022/asset/c4ef57c8-83d0-42f5-93e1-cee376cf8414/main.assets/gr2.jpg" height="436" width="752" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/75a8691d-5579-4491-920b-3aa78abdd669/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.10.022/asset/d11d2385-8963-44e4-b02c-7d96d756b4ae/main.assets/gr3.jpg" height="770" width="750" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/63a6f9b3-acef-449e-ba18-552cd83fd9b4/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.10.022/asset/1fa7e093-5e9d-4586-a402-f59235d458c7/main.assets/figs3.jpg" height="940" width="645" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/9895d2ca-0ac2-4e5b-8a3a-3457319f131f/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.10.022/asset/f15882a8-f37c-47a0-8151-994e0cda80d0/main.assets/figs4.jpg" height="939" width="693" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/ba01631a-c92c-467c-94cf-32aeeb53fb29/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.10.022/asset/aaa684a4-a2b2-4e6a-8f63-c1622f867719/main.assets/gr4.jpg" height="853" width="750" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/0c4c651a-032e-403f-a478-d31684701085/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.10.022/asset/11c66072-78b4-4f32-9259-64d85c508dcf/main.assets/figs5.jpg" height="937" width="726" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/c81108cf-006d-49dc-afce-5ababe24d8ac/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.10.022/asset/4b5346e3-2243-4821-8deb-53b4fa1d27ed/main.assets/gr5.jpg" height="939" width="695" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/ac1d871f-f53b-42ef-ac2f-25bc8de904a8/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.10.022/asset/4ef9c592-535b-45a9-932b-7b2510be73ee/main.assets/figs6.jpg" height="940" width="713" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/88f9b675-fb01-47ee-aea6-8fb1e83be1f9/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.10.022/asset/b1820108-2d53-4388-b563-1ad19068c872/main.assets/gr6.jpg" height="681" width="752" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/63ee43ac-1abf-4cad-8d45-1f8dfdad4f81/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.10.022/asset/582e4f5d-02cd-4309-8b18-bde7a9e6e16e/main.assets/figs7.jpg" height="939" width="688" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/519f0220-fc46-4fca-a2be-74b927dbadb9/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.10.022/asset/684cc1a4-5d2e-4da2-99a0-6db9103b5649/main.assets/gr7.jpg" height="939" width="684" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/4fec8c78-644e-4c86-a3ac-a140380e87f1/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.10.022/asset/c19b267e-71ac-4696-b50d-4957eeae7fd7/main.assets/gr1.jpg" height="710" width="752" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/8f06d774-4912-42d1-a5ad-de7455b82a77/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.10.022/asset/99a82c67-aef8-4a7d-83af-ffab46c09f96/main.assets/figs1.jpg" height="939" width="695" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/f848c78d-eaae-42e2-b03c-e0ddd9c6ac4c/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.10.022/asset/21395ca2-4f88-4fae-9dd9-b175525fbee6/main.assets/figs2.jpg" height="883" width="750" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/751fb368-e5f8-4ce6-becc-55e342033636/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.10.022/asset/c4ef57c8-83d0-42f5-93e1-cee376cf8414/main.assets/gr2.jpg" height="436" width="752" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/75a8691d-5579-4491-920b-3aa78abdd669/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.10.022/asset/d11d2385-8963-44e4-b02c-7d96d756b4ae/main.assets/gr3.jpg" height="770" width="750" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/63a6f9b3-acef-449e-ba18-552cd83fd9b4/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.10.022/asset/1fa7e093-5e9d-4586-a402-f59235d458c7/main.assets/figs3.jpg" height="940" width="645" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/9895d2ca-0ac2-4e5b-8a3a-3457319f131f/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.10.022/asset/f15882a8-f37c-47a0-8151-994e0cda80d0/main.assets/figs4.jpg" height="939" width="693" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/ba01631a-c92c-467c-94cf-32aeeb53fb29/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.10.022/asset/aaa684a4-a2b2-4e6a-8f63-c1622f867719/main.assets/gr4.jpg" height="853" width="750" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/0c4c651a-032e-403f-a478-d31684701085/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.10.022/asset/11c66072-78b4-4f32-9259-64d85c508dcf/main.assets/figs5.jpg" height="937" width="726" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/c81108cf-006d-49dc-afce-5ababe24d8ac/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.10.022/asset/4b5346e3-2243-4821-8deb-53b4fa1d27ed/main.assets/gr5.jpg" height="939" width="695" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/ac1d871f-f53b-42ef-ac2f-25bc8de904a8/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.10.022/asset/4ef9c592-535b-45a9-932b-7b2510be73ee/main.assets/figs6.jpg" height="940" width="713" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/88f9b675-fb01-47ee-aea6-8fb1e83be1f9/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.10.022/asset/b1820108-2d53-4388-b563-1ad19068c872/main.assets/gr6.jpg" height="681" width="752" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/63ee43ac-1abf-4cad-8d45-1f8dfdad4f81/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.10.022/asset/582e4f5d-02cd-4309-8b18-bde7a9e6e16e/main.assets/figs7.jpg" height="939" width="688" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/519f0220-fc46-4fca-a2be-74b927dbadb9/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.10.022/asset/684cc1a4-5d2e-4da2-99a0-6db9103b5649/main.assets/gr7.jpg" height="939" width="684" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.10.022/asset/4fec8c78-644e-4c86-a3ac-a140380e87f1/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01275-7';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749592134648","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"184","issueNumber":"25","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421012757","coverDate":"202112","type":"XOCS-JOURNAL:SCOPE-FULL","title":"emXistem nucleates local protein gradients to propagate silencing across the X chromosome","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc135179e34ed1',t:'MTc0OTU5MjEzMy4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc135179e34ed1&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.10.022&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>