<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="7e9880f4-e699-4d81-a4e0-0d303b23dbf1" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;issue:issue:pii\:S0092867420X00262;wgroup:string:Default Website Group;article:article:pii\:S0092867421013313;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="An NK-like CAR T&nbsp;cell transition in CAR T&nbsp;cell dysfunction">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)01331-3">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/9bac390f-f40c-49b3-b07e-db7abbc341a8/fx1.jpg">
    


        <meta property="og:description" content="Continuous antigen exposure drives CAR T cell exhaustion and promotes CD8+ T-to-NK-like
T cell transition. Transcription factors ID3 and SOX4 are upregulated during CAR dysfunction
and regulate genes associated with exhaustion, including NK receptors. Knocking out
ID3 and SOX4 in CAR T cells slows dysfunction and improves anti-tumor immunity.">
    











        <meta name="twitter:title" content="An NK-like CAR T&nbsp;cell transition in CAR T&nbsp;cell dysfunction">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/9bac390f-f40c-49b3-b07e-db7abbc341a8/fx1.jpg">
    


        <meta name="twitter:description" content="Continuous antigen exposure drives CAR T cell exhaustion and promotes CD8+ T-to-NK-like
T cell transition. Transcription factors ID3 and SOX4 are upregulated during CAR dysfunction
and regulate genes associated with exhaustion, including NK receptors. Knocking out
ID3 and SOX4 in CAR T cells slows dysfunction and improves anti-tumor immunity.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)01331-3">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>An NK-like CAR T&nbsp;cell transition in CAR T&nbsp;cell dysfunction: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="An NK-like CAR T&nbsp;cell transition in CAR T&nbsp;cell dysfunction"> <meta name="citation_date" content="2021/12/09"> <meta name="citation_online_date" content="2021/12/02"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="25"> <meta name="citation_firstpage" content="6081"> <meta name="citation_lastpage" content="6100.e26"> <meta name="citation_pmid" content="34861191"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421013313"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421013313"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421013313"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.11.016"> <meta name="citation_pmid" content="34861191"> <meta name="citation_author" content="Charly R. Good"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="M. Angela Aznar"><meta name="citation_author_orcid" content="0000-0002-7970-1672"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Shunichiro Kuramitsu"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Parisa Samareh"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Sangya Agarwal"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Greg Donahue"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Kenichi Ishiyama"><meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of California San Francisco and the Parker Institute for Cancer Immunotherapy at the University of California San Francisco, San Francisco, California 94143, USA"><meta name="citation_author" content="Nils Wellhausen"><meta name="citation_author_orcid" content="0000-0001-8955-7582"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Austin K. Rennels"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Yujie Ma"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Lifeng Tian"><meta name="citation_author_orcid" content="0000-0001-9740-5327"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Sonia Guedan"><meta name="citation_author_orcid" content="0000-0002-9605-0760"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Katherine A. Alexander"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Zhen Zhang"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Philipp C. Rommel"><meta name="citation_author_orcid" content="0000-0002-9790-4086"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Nathan Singh"><meta name="citation_author_orcid" content="0000-0002-0350-7574"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Karl M. Glastad"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Max W. Richardson"><meta name="citation_author_institution" content="Department of Microbiology, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Keisuke Watanabe"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Janos L. Tanyi"><meta name="citation_author_institution" content="Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Mark H. O’Hara"><meta name="citation_author_institution" content="Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Marco Ruella"><meta name="citation_author_orcid" content="0000-0003-4301-5811"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Lymphoma Program, Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Simon F. Lacey"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Edmund K. Moon"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Division of Pulmonary, Allergy, and Critical Care, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Stephen J. Schuster"><meta name="citation_author_institution" content="Lymphoma Program, Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Steven M. Albelda"><meta name="citation_author_orcid" content="0000-0002-6598-3752"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Division of Pulmonary, Allergy, and Critical Care, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Lewis L. Lanier"><meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of California San Francisco and the Parker Institute for Cancer Immunotherapy at the University of California San Francisco, San Francisco, California 94143, USA"><meta name="citation_author" content="Regina M. Young"><meta name="citation_author_email" content="ryoung@upenn.edu"><meta name="citation_author_orcid" content="0000-0002-0158-9449"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Shelley L. Berger"><meta name="citation_author_email" content="bergers@pennmedicine.upenn.edu"><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author" content="Carl H. June"><meta name="citation_author_email" content="cjune@upenn.edu"><meta name="citation_author_institution" content="Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA"><meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA 19104, USA"><meta name="citation_keywords" content="CAR T&nbsp;cell; pancreatic cancer; T cell dysfunction; immunotherapy; T cell exhaustion; single-cell RNA-seq; immunology; NK-like T&nbsp;cell; cancer; cell transfer therapy; ID3; SOX4"><meta name="citation_reference" content="citation_title=Enriched HLA-E and CD94/NKG2A Interaction Limits Antitumor CD8+ Tumor-Infiltrating T Lymphocyte Responses; citation_author=M. Abd Hamid; citation_author=R.Z. Wang; citation_author=X. Yao; citation_author=P. Fan; citation_author=X. Li; citation_author=X.M. Chang; citation_author=Y. Feng; citation_author=S. Jones; citation_author=D. Maldonado-Perez; citation_author=C. Waugh; citation_author=et al.; citation_journal_title=Cancer Immunol. Res.; citation_volume=7; citation_firstpage=1293; citation_lastpage=1306; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Production of Human CRISPR-Engineered CAR-T Cells; citation_author=S. Agarwal; citation_author=N. Wellhausen; citation_author=B.L. Levine; citation_author=C.H. June; citation_journal_title=J.&nbsp;Vis. Exp.; citation_doi=10.3791/62299; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=HTSeq--a Python framework to work with high-throughput sequencing data; citation_author=S. Anders; citation_author=P.T. Pyl; citation_author=W. Huber; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=166; citation_lastpage=169; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Anti-NKG2A mAb Is a Checkpoint Inhibitor that Promotes Anti-tumor Immunity by Unleashing Both T and NK Cells; citation_author=P. Andre; citation_author=C. Denis; citation_author=C. Soulas; citation_author=C. Bourbon-Caillet; citation_author=J. Lopez; citation_author=T. Arnoux; citation_author=M. Blery; citation_author=C. Bonnafous; citation_author=L. Gauthier; citation_author=A. Morel; citation_author=et al.; citation_journal_title=Cell; citation_volume=175; citation_firstpage=1731; citation_lastpage=1743.e1713; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Human antimicrobial cytotoxic T lymphocytes, defined by NK receptors and antimicrobial proteins, kill intracellular bacteria; citation_author=S.J. Balin; citation_author=M. Pellegrini; citation_author=E. Klechevsky; citation_author=S.T. Won; citation_author=D.I. Weiss; citation_author=A.W. Choi; citation_author=J. Hakimian; citation_author=J. Lu; citation_author=M.T. Ochoa; citation_author=B.R. Bloom; citation_author=et al.; citation_journal_title=Sci. Immunol.; citation_volume=3; citation_firstpage=eaat7668; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Phenotype of NK-Like CD8(+) T Cells with Innate Features in Humans and Their Relevance in Cancer Diseases; citation_author=A. Barbarin; citation_author=E. Cayssials; citation_author=F. Jacomet; citation_author=N.G. Nunez; citation_author=S. Basbous; citation_author=L. Lefèvre; citation_author=M. Abdallah; citation_author=N. Piccirilli; citation_author=B. Morin; citation_author=V. Lavoue; citation_author=et al.; citation_journal_title=Front. Immunol.; citation_volume=8; citation_firstpage=316; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Developmental Relationships of Four Exhausted CD8(+) T Cell Subsets Reveals Underlying Transcriptional and Epigenetic Landscape Control Mechanisms; citation_author=J.C. Beltra; citation_author=S. Manne; citation_author=M.S. Abdel-Hakeem; citation_author=M. Kurachi; citation_author=J.R. Giles; citation_author=Z. Chen; citation_author=V. Casella; citation_author=S.F. Ngiow; citation_author=O. Khan; citation_author=Y.J. Huang; citation_author=et al.; citation_journal_title=Immunity; citation_volume=52; citation_firstpage=825; citation_lastpage=841.e828; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The protein Id: a negative regulator of helix-loop-helix DNA binding proteins; citation_author=R. Benezra; citation_author=R.L. Davis; citation_author=D. Lockshon; citation_author=D.L. Turner; citation_author=H. Weintraub; citation_journal_title=Cell; citation_volume=61; citation_firstpage=49; citation_lastpage=59; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=Defining ‘T cell exhaustion’; citation_author=C.U. Blank; citation_author=W.N. Haining; citation_author=W. Held; citation_author=P.G. Hogan; citation_author=A. Kallies; citation_author=E. Lugli; citation_author=R.C. Lynn; citation_author=M. Philip; citation_author=A. Rao; citation_author=N.P. Restifo; citation_author=et al.; citation_journal_title=Nat. Rev. Immunol.; citation_volume=19; citation_firstpage=665; citation_lastpage=674; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A Distinct Transcriptional Program in Human CAR T Cells Bearing the 4-1BB Signaling Domain Revealed by scRNA-Seq; citation_author=A.C. Boroughs; citation_author=R.C. Larson; citation_author=N.D. Marjanovic; citation_author=K. Gosik; citation_author=A.P. Castano; citation_author=C.B.M. Porter; citation_author=S.J. Lorrey; citation_author=O. Ashenberg; citation_author=L. Jerby; citation_author=M. Hofree; citation_author=et al.; citation_journal_title=Mol. Ther.; citation_volume=28; citation_firstpage=2577; citation_lastpage=2592; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position; citation_author=J.D. Buenrostro; citation_author=P.G. Giresi; citation_author=L.C. Zaba; citation_author=H.Y. Chang; citation_author=W.J. Greenleaf; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=1213; citation_lastpage=1218; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Integrating single-cell transcriptomic data across different conditions, technologies, and species; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=P. Smibert; citation_author=E. Papalexi; citation_author=R. Satija; citation_journal_title=Nat. Biotechnol.; citation_volume=36; citation_firstpage=411; citation_lastpage=420; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Control of large, established tumor xenografts with genetically retargeted human T&nbsp;cells containing CD28 and CD137 domains; citation_author=C. Carpenito; citation_author=M.C. Milone; citation_author=R. Hassan; citation_author=J.C. Simonet; citation_author=M. Lakhal; citation_author=M.M. Suhoski; citation_author=A. Varela-Rohena; citation_author=K.M. Haines; citation_author=D.F. Heitjan; citation_author=S.M. Albelda; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=106; citation_firstpage=3360; citation_lastpage=3365; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Gene Regulatory Network Inference from Single-Cell Data Using Multivariate Information Measures; citation_author=T.E. Chan; citation_author=M.P.H. Stumpf; citation_author=A.C. Babtie; citation_journal_title=Cell Syst.; citation_volume=5; citation_firstpage=251; citation_lastpage=267.e3; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=NR4A transcription factors limit CAR T&nbsp;cell function in solid tumours; citation_author=J. Chen; citation_author=I.F. López-Moyado; citation_author=H. Seo; citation_author=C.J. Lio; citation_author=L.J. Hempleman; citation_author=T. Sekiya; citation_author=A. Yoshimura; citation_author=J.P. Scott-Browne; citation_author=A. Rao; citation_journal_title=Nature; citation_volume=567; citation_firstpage=530; citation_lastpage=534; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues; citation_author=M.R. Corces; citation_author=A.E. Trevino; citation_author=E.G. Hamilton; citation_author=P.G. Greenside; citation_author=N.A. Sinnott-Armstrong; citation_author=S. Vesuna; citation_author=A.T. Satpathy; citation_author=A.J. Rubin; citation_author=K.S. Montine; citation_author=B. Wu; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=959; citation_lastpage=962; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A. Dobin; citation_author=C.A. Davis; citation_author=F. Schlesinger; citation_author=J. Drenkow; citation_author=C. Zaleski; citation_author=S. Jha; citation_author=P. Batut; citation_author=M. Chaisson; citation_author=T.R. Gingeras; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=15; citation_lastpage=21; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Systematic Immunotherapy Target Discovery Using Genome-Scale In&nbsp;Vivo CRISPR Screens in CD8&nbsp;T Cells; citation_author=M.B. Dong; citation_author=G. Wang; citation_author=R.D. Chow; citation_author=L. Ye; citation_author=L. Zhu; citation_author=X. Dai; citation_author=J.J. Park; citation_author=H.R. Kim; citation_author=Y. Errami; citation_author=C.D. Guzman; citation_author=et al.; citation_journal_title=Cell; citation_volume=178; citation_firstpage=1189; citation_lastpage=1204.e23; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=JASPAR 2020: update of the open-access database of transcription factor binding profiles; citation_author=O. Fornes; citation_author=J.A. Castro-Mondragon; citation_author=A. Khan; citation_author=R. van der Lee; citation_author=X. Zhang; citation_author=P.A. Richmond; citation_author=B.P. Modi; citation_author=S. Correard; citation_author=M. Gheorghe; citation_author=D. Baranašić; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=48; citation_number=D1; citation_firstpage=D87; citation_lastpage=D92; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Determinants of response and resistance to CD19 chimeric antigen receptor (CAR) T&nbsp;cell therapy of chronic lymphocytic leukemia; citation_author=J.A. Fraietta; citation_author=S.F. Lacey; citation_author=E.J. Orlando; citation_author=I. Pruteanu-Malinici; citation_author=M. Gohil; citation_author=S. Lundh; citation_author=A.C. Boesteanu; citation_author=Y. Wang; citation_author=R.S. O’Connor; citation_author=W.T. Hwang; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=563; citation_lastpage=571; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Disruption of TET2 promotes the therapeutic efficacy of CD19-targeted T&nbsp;cells; citation_author=J.A. Fraietta; citation_author=C.L. Nobles; citation_author=M.A. Sammons; citation_author=S. Lundh; citation_author=S.A. Carty; citation_author=T.J. Reich; citation_author=A.P. Cogdill; citation_author=J.J.D. Morrissette; citation_author=J.E. DeNizio; citation_author=S. Reddy; citation_author=et al.; citation_journal_title=Nature; citation_volume=558; citation_firstpage=307; citation_lastpage=312; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=NKT cells: what’s in a name?; citation_author=D.I. Godfrey; citation_author=H.R. MacDonald; citation_author=M. Kronenberg; citation_author=M.J. Smyth; citation_author=L. Van Kaer; citation_journal_title=Nat. Rev. Immunol.; citation_volume=4; citation_firstpage=231; citation_lastpage=237; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Screening for the Next-Generation T Cell Therapies; citation_author=J. Guo; citation_author=C. Xu; citation_journal_title=Cancer Cell; citation_volume=37; citation_firstpage=627; citation_lastpage=629; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing; citation_author=X. Guo; citation_author=Y. Zhang; citation_author=L. Zheng; citation_author=C. Zheng; citation_author=J. Song; citation_author=Q. Zhang; citation_author=B. Kang; citation_author=Z. Liu; citation_author=L. Jin; citation_author=R. Xing; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=978; citation_lastpage=985; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Multi-phenotype CRISPR-Cas9 Screen Identifies p38 Kinase as a Target for Adoptive Immunotherapies; citation_author=D. Gurusamy; citation_author=A.N. Henning; citation_author=T.N. Yamamoto; citation_author=Z. Yu; citation_author=N. Zacharakis; citation_author=S. Krishna; citation_author=R.J. Kishton; citation_author=S.K. Vodnala; citation_author=A. Eidizadeh; citation_author=L. Jia; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=37; citation_firstpage=818; citation_lastpage=833.e9; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression; citation_author=C. Hafemeister; citation_author=R. Satija; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=296; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Inhibition of T&nbsp;cell and promotion of natural killer cell development by the dominant negative helix loop helix factor Id3; citation_author=M.H. Heemskerk; citation_author=B. Blom; citation_author=G. Nolan; citation_author=A.P. Stegmann; citation_author=A.Q. Bakker; citation_author=K. Weijer; citation_author=P.C. Res; citation_author=H. Spits; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=186; citation_firstpage=1597; citation_lastpage=1602; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities; citation_author=S. Heinz; citation_author=C. Benner; citation_author=N. Spann; citation_author=E. Bertolino; citation_author=Y.C. Lin; citation_author=P. Laslo; citation_author=J.X. Cheng; citation_author=C. Murre; citation_author=H. Singh; citation_author=C.K. Glass; citation_journal_title=Mol. Cell; citation_volume=38; citation_firstpage=576; citation_lastpage=589; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A genome-wide regulatory network identifies key transcription factors for memory CD8+ T-cell development; citation_author=G. Hu; citation_author=J. Chen; citation_journal_title=Nat. Commun.; citation_volume=4; citation_firstpage=2830; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Repression of the DNA-binding inhibitor Id3 by Blimp-1 limits the formation of memory CD8+ T&nbsp;cells; citation_author=Y. Ji; citation_author=Z. Pos; citation_author=M. Rao; citation_author=C.A. Klebanoff; citation_author=Z. Yu; citation_author=M. Sukumar; citation_author=R.N. Reger; citation_author=D.C. Palmer; citation_author=Z.A. Borman; citation_author=P. Muranski; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=12; citation_firstpage=1230; citation_lastpage=1237; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=TOX transcriptionally and epigenetically programs CD8+ T&nbsp;cell exhaustion; citation_author=O. Khan; citation_author=J.R. Giles; citation_author=S. McDonald; citation_author=S. Manne; citation_author=S.F. Ngiow; citation_author=K.P. Patel; citation_author=M.T. Werner; citation_author=A.C. Huang; citation_author=K.A. Alexander; citation_author=J.E. Wu; citation_author=et al.; citation_journal_title=Nature; citation_volume=571; citation_firstpage=211; citation_lastpage=218; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Innate-like CD8+ T-cells and NK cells: converging functions and phenotypes; citation_author=A. Kurioka; citation_author=P. Klenerman; citation_author=C.B. Willberg; citation_journal_title=Immunology; citation_volume=154; citation_firstpage=547; citation_lastpage=556; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Production, concentration and titration of pseudotyped HIV-1-based lentiviral vectors; citation_author=R.H. Kutner; citation_author=X.Y. Zhang; citation_author=J. Reiser; citation_journal_title=Nat. Protoc.; citation_volume=4; citation_firstpage=495; citation_lastpage=505; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B. Langmead; citation_author=S.L. Salzberg; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=357; citation_lastpage=359; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Transcriptional and post-transcriptional regulation of NK cell development and function; citation_author=J.W. Leong; citation_author=J.A. Wagner; citation_author=A.R. Ireland; citation_author=T.A. Fehniger; citation_journal_title=Clin. Immunol.; citation_volume=177; citation_firstpage=60; citation_lastpage=69; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools; citation_author=H. Li; citation_author=B. Handsaker; citation_author=A. Wysoker; citation_author=T. Fennell; citation_author=J. Ruan; citation_author=N. Homer; citation_author=G. Marth; citation_author=G. Abecasis; citation_author=R. Durbin; citation_author=1000 Genome Project Data Processing Subgroup; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2078; citation_lastpage=2079; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Dysfunctional CD8&nbsp;T Cells Form a Proliferative, Dynamically Regulated Compartment within Human Melanoma; citation_author=H. Li; citation_author=A.M. van der Leun; citation_author=I. Yofe; citation_author=Y. Lubling; citation_author=D. Gelbard-Solodkin; citation_author=A.C.J. van Akkooi; citation_author=M. van den Braber; citation_author=E.A. Rozeman; citation_author=J. Haanen; citation_author=C.U. Blank; citation_author=et al.; citation_journal_title=Cell; citation_volume=176; citation_firstpage=775; citation_lastpage=789.e718; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Chimeric Antigen Receptor Designed to Prevent Ubiquitination and Downregulation Showed Durable Antitumor Efficacy; citation_author=W. Li; citation_author=S. Qiu; citation_author=J. Chen; citation_author=S. Jiang; citation_author=W. Chen; citation_author=J. Jiang; citation_author=F. Wang; citation_author=W. Si; citation_author=Y. Shu; citation_author=P. Wei; citation_author=et al.; citation_journal_title=Immunity; citation_volume=53; citation_firstpage=456; citation_lastpage=470.e456; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=4-1BB costimulation ameliorates T&nbsp;cell exhaustion induced by tonic signaling of chimeric antigen receptors; citation_author=A.H. Long; citation_author=W.M. Haso; citation_author=J.F. Shern; citation_author=K.M. Wanhainen; citation_author=M. Murgai; citation_author=M. Ingaramo; citation_author=J.P. Smith; citation_author=A.J. Walker; citation_author=M.E. Kohler; citation_author=V.R. Venkateshwara; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=21; citation_firstpage=581; citation_lastpage=590; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Single-Cell Transcriptome Analysis Reveals Gene Signatures Associated with T-cell Persistence Following Adoptive Cell Therapy; citation_author=Y.C. Lu; citation_author=L. Jia; citation_author=Z. Zheng; citation_author=E. Tran; citation_author=P.F. Robbins; citation_author=S.A. Rosenberg; citation_journal_title=Cancer Immunol. Res.; citation_volume=7; citation_firstpage=1824; citation_lastpage=1836; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=c-Jun overexpression in CAR T&nbsp;cells induces exhaustion resistance; citation_author=R.C. Lynn; citation_author=E.W. Weber; citation_author=E. Sotillo; citation_author=D. Gennert; citation_author=P. Xu; citation_author=Z. Good; citation_author=H. Anbunathan; citation_author=J. Lattin; citation_author=R. Jones; citation_author=V. Tieu; citation_author=et al.; citation_journal_title=Nature; citation_volume=576; citation_firstpage=293; citation_lastpage=300; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=SOX4 controls invariant NKT cell differentiation by tuning TCR signaling; citation_author=N. Malhotra; citation_author=Y. Qi; citation_author=N.A. Spidale; citation_author=M. Frascoli; citation_author=B. Miu; citation_author=O. Cho; citation_author=K. Sylvia; citation_author=J. Kang; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=215; citation_firstpage=2887; citation_lastpage=2900; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=In&nbsp;vivo CRISPR screening identifies Ptpn2 as a cancer immunotherapy target; citation_author=R.T. Manguso; citation_author=H.W. Pope; citation_author=M.D. Zimmer; citation_author=F.D. Brown; citation_author=K.B. Yates; citation_author=B.C. Miller; citation_author=N.B. Collins; citation_author=K. Bi; citation_author=M.W. LaFleur; citation_author=V.R. Juneja; citation_author=et al.; citation_journal_title=Nature; citation_volume=547; citation_firstpage=413; citation_lastpage=418; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Inhibitory CD161 receptor identified in glioma-infiltrating T&nbsp;cells by single-cell analysis; citation_author=N.D. Mathewson; citation_author=O. Ashenberg; citation_author=I. Tirosh; citation_author=S. Gritsch; citation_author=E.M. Perez; citation_author=S. Marx; citation_author=L. Jerby-Arnon; citation_author=R. Chanoch-Myers; citation_author=T. Hara; citation_author=A.R. Richman; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_firstpage=1281; citation_lastpage=1298.e1226; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Viral and bacterial infections induce expression of multiple NK cell receptors in responding CD8(+) T&nbsp;cells; citation_author=C.W. McMahon; citation_author=A.J. Zajac; citation_author=A.M. Jamieson; citation_author=L. Corral; citation_author=G.E. Hammer; citation_author=R. Ahmed; citation_author=D.H. Raulet; citation_journal_title=J.&nbsp;Immunol.; citation_volume=169; citation_firstpage=1444; citation_lastpage=1452; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Coordinated induction by IL15 of a TCR-independent NKG2D signaling pathway converts CTL into lymphokine-activated killer cells in celiac disease; citation_author=B. Meresse; citation_author=Z. Chen; citation_author=C. Ciszewski; citation_author=M. Tretiakova; citation_author=G. Bhagat; citation_author=T.N. Krausz; citation_author=D.H. Raulet; citation_author=L.L. Lanier; citation_author=V. Groh; citation_author=T. Spies; citation_author=et al.; citation_journal_title=Immunity; citation_volume=21; citation_firstpage=357; citation_lastpage=366; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Heterogenous Populations of Tissue-Resident CD8+ T Cells Are Generated in Response to Infection and Malignancy; citation_author=J.J. Milner; citation_author=C. Toma; citation_author=Z. He; citation_author=N.S. Kurd; citation_author=Q.P. Nguyen; citation_author=B. McDonald; citation_author=L. Quezada; citation_author=C.E. Widjaja; citation_author=D.A. Witherden; citation_author=J.T. Crowl; citation_author=et al.; citation_journal_title=Immunity; citation_volume=52; citation_firstpage=808; citation_lastpage=824.e7; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Exhaustion-associated regulatory regions in CD8+ tumor-infiltrating T&nbsp;cells; citation_author=G.P. Mognol; citation_author=R. Spreafico; citation_author=V. Wong; citation_author=J.P. Scott-Browne; citation_author=S. Togher; citation_author=A. Hoffmann; citation_author=P.G. Hogan; citation_author=A. Rao; citation_author=S. Trifari; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=114; citation_firstpage=E2776; citation_lastpage=E2785; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Multifactorial T-cell hypofunction that is reversible can limit the efficacy of chimeric antigen receptor-transduced human T&nbsp;cells in solid tumors; citation_author=E.K. Moon; citation_author=L.C. Wang; citation_author=D.V. Dolfi; citation_author=C.B. Wilson; citation_author=R. Ranganathan; citation_author=J. Sun; citation_author=V. Kapoor; citation_author=J. Scholler; citation_author=E. Puré; citation_author=M.C. Milone; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=20; citation_firstpage=4262; citation_lastpage=4273; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Blockade of Programmed Death 1 Augments the Ability of Human T Cells Engineered to Target NY-ESO-1 to Control Tumor Growth after Adoptive Transfer; citation_author=E.K. Moon; citation_author=R. Ranganathan; citation_author=E. Eruslanov; citation_author=S. Kim; citation_author=K. Newick; citation_author=S. O’Brien; citation_author=A. Lo; citation_author=X. Liu; citation_author=Y. Zhao; citation_author=S.M. Albelda; citation_journal_title=Clin. Cancer Res.; citation_volume=22; citation_firstpage=436; citation_lastpage=447; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Overcoming T&nbsp;cell exhaustion in infection and cancer; citation_author=K.E. Pauken; citation_author=E.J. Wherry; citation_journal_title=Trends Immunol.; citation_volume=36; citation_firstpage=265; citation_lastpage=276; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Epigenetic stability of exhausted T&nbsp;cells limits durability of reinvigoration by PD-1 blockade; citation_author=K.E. Pauken; citation_author=M.A. Sammons; citation_author=P.M. Odorizzi; citation_author=S. Manne; citation_author=J. Godec; citation_author=O. Khan; citation_author=A.M. Drake; citation_author=Z. Chen; citation_author=D.R. Sen; citation_author=M. Kurachi; citation_author=et al.; citation_journal_title=Science; citation_volume=354; citation_firstpage=1160; citation_lastpage=1165; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Chromatin states define tumour-specific T&nbsp;cell dysfunction and reprogramming; citation_author=M. Philip; citation_author=L. Fairchild; citation_author=L. Sun; citation_author=E.L. Horste; citation_author=S. Camara; citation_author=M. Shakiba; citation_author=A.C. Scott; citation_author=A. Viale; citation_author=P. Lauer; citation_author=T. Merghoub; citation_author=et al.; citation_journal_title=Nature; citation_volume=545; citation_firstpage=452; citation_lastpage=456; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Full-length RNA-seq from single cells using Smart-seq2; citation_author=S. Picelli; citation_author=O.R. Faridani; citation_author=A.K. Björklund; citation_author=G. Winberg; citation_author=S. Sagasser; citation_author=R. Sandberg; citation_journal_title=Nat. Protoc.; citation_volume=9; citation_firstpage=171; citation_lastpage=181; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Counteracting CAR T&nbsp;cell dysfunction; citation_author=M. Poorebrahim; citation_author=J. Melief; citation_author=Y. Pico de Coaña; citation_author=S.L. Wickström; citation_author=A. Cid-Arregui; citation_author=R. Kiessling; citation_journal_title=Oncogene; citation_volume=40; citation_firstpage=421; citation_lastpage=435; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Single-cell mRNA quantification and differential analysis with Census; citation_author=X. Qiu; citation_author=A. Hill; citation_author=J. Packer; citation_author=D. Lin; citation_author=Y.A. Ma; citation_author=C. Trapnell; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=309; citation_lastpage=315; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Pooled Knockin Targeting for Genome Engineering of Cellular Immunotherapies; citation_author=T.L. Roth; citation_author=P.J. Li; citation_author=F. Blaeschke; citation_author=J.F. Nies; citation_author=R. Apathy; citation_author=C. Mowery; citation_author=R. Yu; citation_author=M.L.T. Nguyen; citation_author=Y. Lee; citation_author=A. Truong; citation_author=et al.; citation_journal_title=Cell; citation_volume=181; citation_firstpage=728; citation_lastpage=744.e21; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Targeting Tim-3 and PD-1 pathways to reverse T&nbsp;cell exhaustion and restore anti-tumor immunity; citation_author=K. Sakuishi; citation_author=L. Apetoh; citation_author=J.M. Sullivan; citation_author=B.R. Blazar; citation_author=V.K. Kuchroo; citation_author=A.C. Anderson; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=207; citation_firstpage=2187; citation_lastpage=2194; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing; citation_author=K. Sato; citation_author=K. Tsuyuzaki; citation_author=K. Shimizu; citation_author=I. Nikaido; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=31; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Chimeric Antigen Receptor T Cells in Refractory B-Cell Lymphomas; citation_author=S.J. Schuster; citation_author=J. Svoboda; citation_author=E.A. Chong; citation_author=S.D. Nasta; citation_author=A.R. Mato; citation_author=Ö. Anak; citation_author=J.L. Brogdon; citation_author=I. Pruteanu-Malinici; citation_author=V. Bhoj; citation_author=D. Landsburg; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=377; citation_firstpage=2545; citation_lastpage=2554; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=T Cells Going Innate; citation_author=M. Seyda; citation_author=A. Elkhal; citation_author=M. Quante; citation_author=C.S. Falk; citation_author=S.G. Tullius; citation_journal_title=Trends Immunol.; citation_volume=37; citation_firstpage=546; citation_lastpage=556; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Genome-wide CRISPR Screens in Primary Human T Cells Reveal Key Regulators of Immune Function; citation_author=E. Shifrut; citation_author=J. Carnevale; citation_author=V. Tobin; citation_author=T.L. Roth; citation_author=J.M. Woo; citation_author=C.T. Bui; citation_author=P.J. Li; citation_author=M.E. Diolaiti; citation_author=A. Ashworth; citation_author=A. Marson; citation_journal_title=Cell; citation_volume=175; citation_firstpage=1958; citation_lastpage=1971.e1915; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A Distinct Gene Module for Dysfunction Uncoupled from Activation in Tumor-Infiltrating T Cells; citation_author=M. Singer; citation_author=C. Wang; citation_author=L. Cong; citation_author=N.D. Marjanovic; citation_author=M.S. Kowalczyk; citation_author=H. Zhang; citation_author=J. Nyman; citation_author=K. Sakuishi; citation_author=S. Kurtulus; citation_author=D. Gennert; citation_author=et al.; citation_journal_title=Cell; citation_volume=166; citation_firstpage=1500; citation_lastpage=1511.e1509; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=CRISPR-engineered T&nbsp;cells in patients with refractory cancer; citation_author=E.A. Stadtmauer; citation_author=J.A. Fraietta; citation_author=M.M. Davis; citation_author=A.D. Cohen; citation_author=K.L. Weber; citation_author=E. Lancaster; citation_author=P.A. Mangan; citation_author=I. Kulikovskaya; citation_author=M. Gupta; citation_author=F. Chen; citation_author=et al.; citation_journal_title=Science; citation_volume=367; citation_firstpage=eaba7365; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=T Cells Engineered against a Native Antigen Can Surmount Immunologic and Physical Barriers to Treat Pancreatic Ductal Adenocarcinoma; citation_author=I.M. Stromnes; citation_author=T.M. Schmitt; citation_author=A. Hulbert; citation_author=J.S. Brockenbrough; citation_author=H. Nguyen; citation_author=C. Cuevas; citation_author=A.M. Dotson; citation_author=X. Tan; citation_author=J.L. Hotes; citation_author=P.D. Greenberg; citation_author=S.R. Hingorani; citation_journal_title=Cancer Cell; citation_volume=28; citation_firstpage=638; citation_lastpage=652; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Comprehensive Integration of Single-Cell Data; citation_author=T. Stuart; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=C. Hafemeister; citation_author=E. Papalexi; citation_author=W.M. Mauck; citation_author=Y. Hao; citation_author=M. Stoeckius; citation_author=P. Smibert; citation_author=R. Satija; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1888; citation_lastpage=1902.e1821; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=T Cell Dysfunction in Cancer; citation_author=D.S. Thommen; citation_author=T.N. Schumacher; citation_journal_title=Cancer Cell; citation_volume=33; citation_firstpage=547; citation_lastpage=562; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells; citation_author=C. Trapnell; citation_author=D. Cacchiarelli; citation_author=J. Grimsby; citation_author=P. Pokharel; citation_author=S. Li; citation_author=M. Morse; citation_author=N.J. Lennon; citation_author=K.J. Livak; citation_author=T.S. Mikkelsen; citation_author=J.L. Rinn; citation_journal_title=Nat. Biotechnol.; citation_volume=32; citation_firstpage=381; citation_lastpage=386; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=UniProt: a worldwide hub of protein knowledge; citation_author=UniProt Consortium; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=D1; citation_firstpage=D506; citation_lastpage=D515; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Early precursor T&nbsp;cells establish and propagate T&nbsp;cell exhaustion in chronic infection; citation_author=D.T. Utzschneider; citation_author=S.S. Gabriel; citation_author=D. Chisanga; citation_author=R. Gloury; citation_author=P.M. Gubser; citation_author=A. Vasanthakumar; citation_author=W. Shi; citation_author=A. Kallies; citation_journal_title=Nat. Immunol.; citation_volume=21; citation_firstpage=1256; citation_lastpage=1266; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=NKG2A Blockade Potentiates CD8&nbsp;T Cell Immunity Induced by Cancer Vaccines; citation_author=N. van Montfoort; citation_author=L. Borst; citation_author=M.J. Korrer; citation_author=M. Sluijter; citation_author=K.A. Marijt; citation_author=S.J. Santegoets; citation_author=V.J. van Ham; citation_author=I. Ehsan; citation_author=P. Charoentong; citation_author=P. André; citation_author=et al.; citation_journal_title=Cell; citation_volume=175; citation_firstpage=1744; citation_lastpage=1755.e1715; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Targeting REGNASE-1 programs long-lived effector T&nbsp;cells for cancer therapy; citation_author=J. Wei; citation_author=L. Long; citation_author=W. Zheng; citation_author=Y. Dhungana; citation_author=S.A. Lim; citation_author=C. Guy; citation_author=Y. Wang; citation_author=Y.D. Wang; citation_author=C. Qian; citation_author=B. Xu; citation_author=et al.; citation_journal_title=Nature; citation_volume=576; citation_firstpage=471; citation_lastpage=476; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Innate-like T&nbsp;cells straddle innate and adaptive immunity by altering antigen-receptor responsiveness; citation_author=M. Wencker; citation_author=G. Turchinovich; citation_author=R. Di Marco Barros; citation_author=L. Deban; citation_author=A. Jandke; citation_author=A. Cope; citation_author=A.C. Hayday; citation_journal_title=Nat. Immunol.; citation_volume=15; citation_firstpage=80; citation_lastpage=87; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Molecular and cellular insights into T&nbsp;cell exhaustion; citation_author=E.J. Wherry; citation_author=M. Kurachi; citation_journal_title=Nat. Rev. Immunol.; citation_volume=15; citation_firstpage=486; citation_lastpage=499; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The transcriptional regulators Id2 and Id3 control the formation of distinct memory CD8+ T&nbsp;cell subsets; citation_author=C.Y. Yang; citation_author=J.A. Best; citation_author=J. Knell; citation_author=E. Yang; citation_author=A.D. Sheridan; citation_author=A.K. Jesionek; citation_author=H.S. Li; citation_author=R.R. Rivera; citation_author=K.C. Lind; citation_author=L.M. D’Cruz; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=12; citation_firstpage=1221; citation_lastpage=1229; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Lineage tracking reveals dynamic relationships of T&nbsp;cells in colorectal cancer; citation_author=L. Zhang; citation_author=X. Yu; citation_author=L. Zheng; citation_author=Y. Zhang; citation_author=Y. Li; citation_author=Q. Fang; citation_author=R. Gao; citation_author=B. Kang; citation_author=Q. Zhang; citation_author=J.Y. Huang; citation_author=et al.; citation_journal_title=Nature; citation_volume=564; citation_firstpage=268; citation_lastpage=272; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing; citation_author=C. Zheng; citation_author=L. Zheng; citation_author=J.K. Yoo; citation_author=H. Guo; citation_author=Y. Zhang; citation_author=X. Guo; citation_author=B. Kang; citation_author=R. Hu; citation_author=J.Y. Huang; citation_author=Q. Zhang; citation_author=et al.; citation_journal_title=Cell; citation_volume=169; citation_firstpage=1342; citation_lastpage=1356.e1316; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Metascape provides a biologist-oriented resource for the analysis of systems-level datasets; citation_author=Y. Zhou; citation_author=B. Zhou; citation_author=L. Pache; citation_author=M. Chang; citation_author=A.H. Khodabakhshi; citation_author=O. Tanaseichuk; citation_author=C. Benner; citation_author=S.K. Chanda; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=1523; citation_publication_date=2019; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Chimeric antigen receptor (CAR) T&nbsp;cell therapy has achieved remarkable success in hematological malignancies but remains ineffective in solid tumors, due in part to CAR T&nbsp;cell exhaustion in the solid tumor microenvironment. To study dysfunction of mesothelin-redirected CAR T&nbsp;cells in pancreatic cancer, we establish a robust model of continuous antigen exposure that recapitulates hallmark features of T&nbsp;cell exhaustion and discover, both <i>in&nbsp;vitro</i> and in CAR T&nbsp;cell patients, that CAR dysregulation is associated with a CD8+ T-to-NK-like T&nbsp;cell transition. Furthermore, we identify a gene signature defining CAR and TCR dysregulation and transcription factors, including SOX4 and ID3 as key regulators of CAR T&nbsp;cell exhaustion. Our findings shed light on the plasticity of human CAR T&nbsp;cells and demonstrate that genetic downmodulation of ID3 and SOX4 expression can improve the efficacy of CAR T&nbsp;cell therapy in solid tumors by preventing or delaying CAR T&nbsp;cell dysfunction.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7927&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)01331-3&amp;be=4162&amp;fe=7637&amp;dc=7271&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749592061548,%22n%22:0,%22f%22:3430,%22dn%22:3431,%22dne%22:3431,%22c%22:3431,%22s%22:3431,%22ce%22:3431,%22rq%22:3431,%22rp%22:4146,%22rpe%22:4886,%22dl%22:4152,%22di%22:5295,%22ds%22:7271,%22de%22:7481,%22dc%22:7637,%22l%22:7637,%22le%22:7738%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c1D443 {
  padding: 0.683em 0.754em 0 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D447 {
  padding: 0.677em 0.704em 0 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-cD7 {
  padding: 0.491em 0.778em 0 0;
}

mjx-c.mjx-c1D458 {
  padding: 0.694em 0.521em 0.011em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D459 {
  padding: 0.694em 0.298em 0.011em 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c1D454 {
  padding: 0.442em 0.477em 0.205em 0;
}

mjx-c.mjx-c210E {
  padding: 0.694em 0.576em 0.011em 0;
}

mjx-c.mjx-c1D464 {
  padding: 0.442em 0.716em 0.011em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-cF7 {
  padding: 0.504em 0.778em 0.004em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901331-3" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01331-3"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)01331-3"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901331-3" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-641358" aria-expanded="false" aria-controls="login-links-641358" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-641358" id="login-links--641358" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901331-3" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901331-3" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901331-3" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-641358" data-opens="#links-dialog-641358" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-641358" aria-modal="true"><button data-closes="#links-dialog-641358" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901331-3" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901331-3" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901331-3" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.11.016">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)01331-3" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0026-2"><span>Volume 184</span>, <span>Issue 25</span></a></span><span class="meta-panel__pages"><span>p6081-6100.e26</span></span><span class="meta-panel__onlineDate">December 09, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=25&amp;pii=S0092-8674%2820%29X0026-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">An NK-like CAR T&nbsp;cell transition in CAR T&nbsp;cell dysfunction</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Charly R.</span> <span property="familyName">Good</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Charly R.</span> <span property="familyName">Good</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Good/Charly+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">11</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">M. Angela</span> <span property="familyName">Aznar</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">M. Angela</span> <span property="familyName">Aznar</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7970-1672" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7970-1672</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Aznar/M+Angela" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">11</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Shunichiro</span> <span property="familyName">Kuramitsu</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shunichiro</span> <span property="familyName">Kuramitsu</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kuramitsu/Shunichiro" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">11</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Parisa</span> <span property="familyName">Samareh</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Parisa</span> <span property="familyName">Samareh</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Samareh/Parisa" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Sangya</span> <span property="familyName">Agarwal</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sangya</span> <span property="familyName">Agarwal</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Agarwal/Sangya" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Greg</span> <span property="familyName">Donahue</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Greg</span> <span property="familyName">Donahue</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Donahue/Greg" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Kenichi</span> <span property="familyName">Ishiyama</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kenichi</span> <span property="familyName">Ishiyama</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology and Immunology, University of California San Francisco and the Parker Institute for Cancer Immunotherapy at the University of California San Francisco, San Francisco, California 94143, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ishiyama/Kenichi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Nils</span> <span property="familyName">Wellhausen</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nils</span> <span property="familyName">Wellhausen</span> <a class="orcid-id" href="https://orcid.org/0000-0001-8955-7582" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-8955-7582</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wellhausen/Nils" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Austin K.</span> <span property="familyName">Rennels</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Austin K.</span> <span property="familyName">Rennels</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rennels/Austin+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Yujie</span> <span property="familyName">Ma</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yujie</span> <span property="familyName">Ma</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ma/Yujie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Lifeng</span> <span property="familyName">Tian</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lifeng</span> <span property="familyName">Tian</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9740-5327" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9740-5327</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Tian/Lifeng" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Sonia</span> <span property="familyName">Guedan</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sonia</span> <span property="familyName">Guedan</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9605-0760" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9605-0760</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Guedan/Sonia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Katherine A.</span> <span property="familyName">Alexander</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Katherine A.</span> <span property="familyName">Alexander</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Alexander/Katherine+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Zhen</span> <span property="familyName">Zhang</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Zhen</span> <span property="familyName">Zhang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Zhen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Philipp C.</span> <span property="familyName">Rommel</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Philipp C.</span> <span property="familyName">Rommel</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9790-4086" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9790-4086</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rommel/Philipp+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Nathan</span> <span property="familyName">Singh</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nathan</span> <span property="familyName">Singh</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0350-7574" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0350-7574</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Singh/Nathan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Karl M.</span> <span property="familyName">Glastad</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Karl M.</span> <span property="familyName">Glastad</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Glastad/Karl+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Max W.</span> <span property="familyName">Richardson</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Max W.</span> <span property="familyName">Richardson</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Richardson/Max+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Keisuke</span> <span property="familyName">Watanabe</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Keisuke</span> <span property="familyName">Watanabe</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Watanabe/Keisuke" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Janos L.</span> <span property="familyName">Tanyi</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Janos L.</span> <span property="familyName">Tanyi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Tanyi/Janos+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Mark H.</span> <span property="familyName">O’Hara</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mark H.</span> <span property="familyName">O’Hara</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/O%E2%80%99Hara/Mark+H" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Marco</span> <span property="familyName">Ruella</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marco</span> <span property="familyName">Ruella</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4301-5811" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4301-5811</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Lymphoma Program, Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ruella/Marco" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Simon F.</span> <span property="familyName">Lacey</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Simon F.</span> <span property="familyName">Lacey</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lacey/Simon+F" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Edmund K.</span> <span property="familyName">Moon</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Edmund K.</span> <span property="familyName">Moon</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Division of Pulmonary, Allergy, and Critical Care, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Moon/Edmund+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Stephen J.</span> <span property="familyName">Schuster</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stephen J.</span> <span property="familyName">Schuster</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Lymphoma Program, Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Schuster/Stephen+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">Steven M.</span> <span property="familyName">Albelda</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Steven M.</span> <span property="familyName">Albelda</span> <a class="orcid-id" href="https://orcid.org/0000-0002-6598-3752" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-6598-3752</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Division of Pulmonary, Allergy, and Critical Care, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Albelda/Steven+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au27" aria-controls="axel_author_au27"><span property="givenName">Lewis L.</span> <span property="familyName">Lanier</span></a><div id="au27" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lewis L.</span> <span property="familyName">Lanier</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology and Immunology, University of California San Francisco and the Parker Institute for Cancer Immunotherapy at the University of California San Francisco, San Francisco, California 94143, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lanier/Lewis+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au28" aria-controls="axel_author_au28"><span property="givenName">Regina M.</span> <span property="familyName">Young</span></a><div id="au28" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Regina M.</span> <span property="familyName">Young</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0158-9449" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0158-9449</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:ryoung@upenn.edu" property="email" title="Send email to Regina M. Young">ryoung@upenn.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Young/Regina+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup> <a id="eadd0010" href="mailto:ryoung@upenn.edu" property="email" title="Send email to Regina M. Young">ryoung@upenn.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au29" aria-controls="axel_author_au29"><span property="givenName">Shelley L.</span> <span property="familyName">Berger</span></a><div id="au29" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shelley L.</span> <span property="familyName">Berger</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:bergers@pennmedicine.upenn.edu" property="email" title="Send email to Shelley L. Berger">bergers@pennmedicine.upenn.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Berger/Shelley+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup> <a id="eadd0015" href="mailto:bergers@pennmedicine.upenn.edu" property="email" title="Send email to Shelley L. Berger">bergers@pennmedicine.upenn.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au30" aria-controls="axel_author_au30"><span property="givenName">Carl H.</span> <span property="familyName">June</span></a><div id="au30" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Carl H.</span> <span property="familyName">June</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:cjune@upenn.edu" property="email" title="Send email to Carl H. June">cjune@upenn.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-author-link"><a href="/authored-by/June/Carl+H" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a role="doc-noteref">12</a></sup> <a id="eadd0020" href="mailto:cjune@upenn.edu" property="email" title="Send email to Carl H. June">cjune@upenn.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Cell and Developmental Biology, Penn Institute of Epigenetics, Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Center for Cellular Immunotherapies, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Microbiology and Immunology, University of California San Francisco and the Parker Institute for Cancer Immunotherapy at the University of California San Francisco, San Francisco, California 94143, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Department of Microbiology, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Department of Obstetrics and Gynecology, Division of Gynecologic Oncology, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Division of Pulmonary, Allergy, and Critical Care, University of Pennsylvania Perelman School of Medicine, Philadelphia, PA 19104, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Lymphoma Program, Abramson Cancer Center, University of Pennsylvania, Philadelphia, PA 19104, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received November 4, 2020</span>; <span>Revised August 13, 2021</span>; <span>Accepted November 11, 2021</span>; <span>Published online December 2, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.11.016" property="sameAs" target="_blank">10.1016/j.cell.2021.11.016 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421013313" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.11.016" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0026-2"><img src="/cms/asset/0540b9ed-f4ad-4863-9830-19ca05143514/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 25"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2901331-3" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (6)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901331-3" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.11.016">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2901331-3" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.11.016&amp;pii=S0092-8674%2821%2901331-3" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901331-3" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.11.016">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.11.016" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421013313&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421013313&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=25&amp;pii=S0092-8674%2820%29X0026-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)01324-6" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)01322-2" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (6)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">CAR T&nbsp;cells under chronic antigen stimulation show hallmarks of T&nbsp;cell exhaustion</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">CAR dysregulation is associated with a CD8+ T-to-NK-like T&nbsp;cell transition</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">CAR T&nbsp;cells with NK-like transition were identified in patients after treatment</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Unlike WT CAR T&nbsp;cells, ID3 and SOX4 knockout CAR T&nbsp;cells retain anti-tumor immunity</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Chimeric antigen receptor (CAR) T&nbsp;cell therapy has achieved remarkable success in hematological malignancies but remains ineffective in solid tumors, due in part to CAR T&nbsp;cell exhaustion in the solid tumor microenvironment. To study dysfunction of mesothelin-redirected CAR T&nbsp;cells in pancreatic cancer, we establish a robust model of continuous antigen exposure that recapitulates hallmark features of T&nbsp;cell exhaustion and discover, both <i>in&nbsp;vitro</i> and in CAR T&nbsp;cell patients, that CAR dysregulation is associated with a CD8+ T-to-NK-like T&nbsp;cell transition. Furthermore, we identify a gene signature defining CAR and TCR dysregulation and transcription factors, including SOX4 and ID3 as key regulators of CAR T&nbsp;cell exhaustion. Our findings shed light on the plasticity of human CAR T&nbsp;cells and demonstrate that genetic downmodulation of ID3 and SOX4 expression can improve the efficacy of CAR T&nbsp;cell therapy in solid tumors by preventing or delaying CAR T&nbsp;cell dysfunction.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/2564e8f0-76d5-4b36-9d0f-2110ed7312eb/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.11.016/asset/9bac390f-f40c-49b3-b07e-db7abbc341a8/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/2564e8f0-76d5-4b36-9d0f-2110ed7312eb/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;CAR+T%C2%A0cell&quot;" alt="Search on this keyword" rel="nofollow">CAR T&nbsp;cell</a></li><li><a href="/action/doSearch?AllField=&quot;pancreatic+cancer&quot;" alt="Search on this keyword" rel="nofollow">pancreatic cancer</a></li><li><a href="/action/doSearch?AllField=&quot;T+cell+dysfunction&quot;" alt="Search on this keyword" rel="nofollow">T cell dysfunction</a></li><li><a href="/action/doSearch?AllField=&quot;immunotherapy&quot;" alt="Search on this keyword" rel="nofollow">immunotherapy</a></li><li><a href="/action/doSearch?AllField=&quot;T+cell+exhaustion&quot;" alt="Search on this keyword" rel="nofollow">T cell exhaustion</a></li><li><a href="/action/doSearch?AllField=&quot;single-cell+RNA-seq&quot;" alt="Search on this keyword" rel="nofollow">single-cell RNA-seq</a></li><li><a href="/action/doSearch?AllField=&quot;immunology&quot;" alt="Search on this keyword" rel="nofollow">immunology</a></li><li><a href="/action/doSearch?AllField=&quot;NK-like+T%C2%A0cell&quot;" alt="Search on this keyword" rel="nofollow">NK-like T&nbsp;cell</a></li><li><a href="/action/doSearch?AllField=&quot;cancer&quot;" alt="Search on this keyword" rel="nofollow">cancer</a></li><li><a href="/action/doSearch?AllField=&quot;cell+transfer+therapy&quot;" alt="Search on this keyword" rel="nofollow">cell transfer therapy</a></li><li><a href="/action/doSearch?AllField=&quot;ID3&quot;" alt="Search on this keyword" rel="nofollow">ID3</a></li><li><a href="/action/doSearch?AllField=&quot;SOX4&quot;" alt="Search on this keyword" rel="nofollow">SOX4</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0050" role="paragraph">T cell exhaustion is a differentiation state acquired when T&nbsp;cells are exposed to persistent antigen stimulation in the setting of chronic viral infection or in response to tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref9-1" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-1" aria-controls="sref9-1">Blank et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Blank, C.U. ∙ Haining, W.N. ∙ Held, W. ...</div><div><strong>Defining ‘T cell exhaustion’</strong></div><div><em>Nat. Rev. Immunol.</em> 2019; <strong>19</strong>:665-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-019-0221-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85074300816" target="_blank" aria-label="Open Scopus in new tab">Scopus (857)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-019-0221-9&amp;pmid=31570879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Failure to eliminate antigen results in a progressive loss of effector functions or dysregulation (<span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Pauken and Wherry, 2015</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Pauken, K.E. ∙ Wherry, E.J.</div><div><strong>Overcoming T&nbsp;cell exhaustion in infection and cancer</strong></div><div><em>Trends Immunol.</em> 2015; <strong>36</strong>:265-276</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2015.02.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2015.02.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84926528302" target="_blank" aria-label="Open Scopus in new tab">Scopus (792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25797516/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2015.02.008&amp;pmid=25797516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hallmarks of T&nbsp;cell exhaustion include reduced effector function, distinct epigenetic and transcriptional gene signatures, sustained expression of multiple inhibitory receptors, defective cytokine production, increased chemokine expression, and limited proliferative capacity (<span class="dropBlock reference-citations"><a id="body-ref-sref9-2" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-2" aria-controls="sref9-2">Blank et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Blank, C.U. ∙ Haining, W.N. ∙ Held, W. ...</div><div><strong>Defining ‘T cell exhaustion’</strong></div><div><em>Nat. Rev. Immunol.</em> 2019; <strong>19</strong>:665-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-019-0221-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85074300816" target="_blank" aria-label="Open Scopus in new tab">Scopus (857)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-019-0221-9&amp;pmid=31570879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Pauken and Wherry, 2015</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Pauken, K.E. ∙ Wherry, E.J.</div><div><strong>Overcoming T&nbsp;cell exhaustion in infection and cancer</strong></div><div><em>Trends Immunol.</em> 2015; <strong>36</strong>:265-276</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2015.02.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2015.02.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84926528302" target="_blank" aria-label="Open Scopus in new tab">Scopus (792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25797516/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2015.02.008&amp;pmid=25797516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Thommen and Schumacher, 2018</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Thommen, D.S. ∙ Schumacher, T.N.</div><div><strong>T Cell Dysfunction in Cancer</strong></div><div><em>Cancer Cell.</em> 2018; <strong>33</strong>:547-562</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2018.03.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2018.03.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85043771297" target="_blank" aria-label="Open Scopus in new tab">Scopus (768)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29634943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2018.03.012&amp;pmid=29634943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Examination of genes upregulated in exhausted CD8+ tumor-infiltrating lymphocytes (TILs) from patients (<span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Guo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, H. ∙ van der Leun, A.M. ∙ Yofe, I. ...</div><div><strong>Dysfunctional CD8&nbsp;T Cells Form a Proliferative, Dynamically Regulated Compartment within Human Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:775-789.e718</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061004091" target="_blank" aria-label="Open Scopus in new tab">Scopus (661)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30595452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.11.043&amp;pmid=30595452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Zhang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Zhang, L. ∙ Yu, X. ∙ Zheng, L. ...</div><div><strong>Lineage tracking reveals dynamic relationships of T&nbsp;cells in colorectal cancer</strong></div><div><em>Nature.</em> 2018; <strong>564</strong>:268-272</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0694-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85058518221" target="_blank" aria-label="Open Scopus in new tab">Scopus (694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30479382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0694-x&amp;pmid=30479382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zheng, C. ∙ Zheng, L. ∙ Yoo, J.K. ...</div><div><strong>Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1342-1356.e1316</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85020894317" target="_blank" aria-label="Open Scopus in new tab">Scopus (1383)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.035&amp;pmid=28622514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and TILs from mouse models (<span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Khan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Khan, O. ∙ Giles, J.R. ∙ McDonald, S. ...</div><div><strong>TOX transcriptionally and epigenetically programs CD8<sup>+</sup> T&nbsp;cell exhaustion</strong></div><div><em>Nature.</em> 2019; <strong>571</strong>:211-218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1325-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85068336352" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31207603/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1325-x&amp;pmid=31207603" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63-1" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-1" aria-controls="sref63-1">Singer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref63-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Singer, M. ∙ Wang, C. ∙ Cong, L. ...</div><div><strong>A Distinct Gene Module for Dysfunction Uncoupled from Activation in Tumor-Infiltrating T Cells</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:1500-1511.e1509</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84986285632" target="_blank" aria-label="Open Scopus in new tab">Scopus (269)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27610572/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.052&amp;pmid=27610572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) has led to the identification of genes that restrain tumor immunity, including <i>LAYN</i>, <i>Tox</i>, and <i>Gata3</i>. Furthermore, genome-wide CRISPR-Cas9 knockout and knockin screens in mouse and human CD8+ T&nbsp;cells revealed additional targets such as <i>Mapk14</i>, <i>Dhx37</i>, <i>NR4A</i>, <i>ZC3H12A</i>, <i>Ptpn2</i>, <i>SOSCS1</i>, and <i>TGFBR2</i> that modulate T&nbsp;cell function (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Dong et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Dong, M.B. ∙ Wang, G. ∙ Chow, R.D. ...</div><div><strong>Systematic Immunotherapy Target Discovery Using Genome-Scale In&nbsp;Vivo CRISPR Screens in CD8&nbsp;T Cells</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1189-1204.e23</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.07.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.07.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85070563455" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31442407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.044&amp;pmid=31442407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Guo and Xu, 2020</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Guo, J. ∙ Xu, C.</div><div><strong>Screening for the Next-Generation T Cell Therapies</strong></div><div><em>Cancer Cell.</em> 2020; <strong>37</strong>:627-629</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2020.04.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2020.04.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084206672" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32396857/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2020.04.009&amp;pmid=32396857" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Gurusamy et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Gurusamy, D. ∙ Henning, A.N. ∙ Yamamoto, T.N. ...</div><div><strong>Multi-phenotype CRISPR-Cas9 Screen Identifies p38 Kinase as a Target for Adoptive Immunotherapies</strong></div><div><em>Cancer Cell.</em> 2020; <strong>37</strong>:818-833.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2020.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2020.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85085757009" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32516591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2020.05.004&amp;pmid=32516591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Manguso et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Manguso, R.T. ∙ Pope, H.W. ∙ Zimmer, M.D. ...</div><div><strong>In&nbsp;vivo CRISPR screening identifies Ptpn2 as a cancer immunotherapy target</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:413-418</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23270" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85026403229" target="_blank" aria-label="Open Scopus in new tab">Scopus (748)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28723893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23270&amp;pmid=28723893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Roth et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Roth, T.L. ∙ Li, P.J. ∙ Blaeschke, F. ...</div><div><strong>Pooled Knockin Targeting for Genome Engineering of Cellular Immunotherapies</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:728-744.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2020.03.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2020.03.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85083775641" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32302591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.03.039&amp;pmid=32302591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">Shifrut et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Shifrut, E. ∙ Carnevale, J. ∙ Tobin, V. ...</div><div><strong>Genome-wide CRISPR Screens in Primary Human T Cells Reveal Key Regulators of Immune Function</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:1958-1971.e1915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85059173331" target="_blank" aria-label="Open Scopus in new tab">Scopus (333)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30449619/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.10.024&amp;pmid=30449619" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Wei et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Wei, J. ∙ Long, L. ∙ Zheng, W. ...</div><div><strong>Targeting REGNASE-1 programs long-lived effector T&nbsp;cells for cancer therapy</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:471-476</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1821-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85076392269" target="_blank" aria-label="Open Scopus in new tab">Scopus (250)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31827283/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1821-z&amp;pmid=31827283" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Importantly, engineered CAR and T&nbsp;cell receptor (TCR) T&nbsp;cells also acquire an exhausted phenotype when they enter the tumor microenvironment (TME) in <i>in&nbsp;vivo</i> models (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Chen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Chen, J. ∙ López-Moyado, I.F. ∙ Seo, H. ...</div><div><strong>NR4A transcription factors limit CAR T&nbsp;cell function in solid tumours</strong></div><div><em>Nature.</em> 2019; <strong>567</strong>:530-534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0985-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85062871387" target="_blank" aria-label="Open Scopus in new tab">Scopus (493)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30814732/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0985-x&amp;pmid=30814732" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Moon et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Moon, E.K. ∙ Wang, L.C. ∙ Dolfi, D.V. ...</div><div><strong>Multifactorial T-cell hypofunction that is reversible can limit the efficacy of chimeric antigen receptor-transduced human T&nbsp;cells in solid tumors</strong></div><div><em>Clin. Cancer Res.</em> 2014; <strong>20</strong>:4262-4273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-13-2627" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84905993539" target="_blank" aria-label="Open Scopus in new tab">Scopus (334)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24919573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-13-2627&amp;pmid=24919573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Stromnes et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Stromnes, I.M. ∙ Schmitt, T.M. ∙ Hulbert, A. ...</div><div><strong>T Cells Engineered against a Native Antigen Can Surmount Immunologic and Physical Barriers to Treat Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2015; <strong>28</strong>:638-652</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2015.09.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2015.09.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84946547794" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26525103/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2015.09.022&amp;pmid=26525103" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), leading to the hypothesis that CAR T&nbsp;cell exhaustion/dysfunction is a major hurdle for CAR T&nbsp;cell therapy (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Fraietta et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Fraietta, J.A. ∙ Lacey, S.F. ∙ Orlando, E.J. ...</div><div><strong>Determinants of response and resistance to CD19 chimeric antigen receptor (CAR) T&nbsp;cell therapy of chronic lymphocytic leukemia</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:563-571</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0010-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85046093409" target="_blank" aria-label="Open Scopus in new tab">Scopus (1116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0010-1&amp;pmid=29713085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">2018b</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Fraietta, J.A. ∙ Nobles, C.L. ∙ Sammons, M.A. ...</div><div><strong>Disruption of TET2 promotes the therapeutic efficacy of CD19-targeted T&nbsp;cells</strong></div><div><em>Nature.</em> 2018; <strong>558</strong>:307-312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0178-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85048364639" target="_blank" aria-label="Open Scopus in new tab">Scopus (577)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29849141/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0178-z&amp;pmid=29849141" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Long et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Long, A.H. ∙ Haso, W.M. ∙ Shern, J.F. ...</div><div><strong>4-1BB costimulation ameliorates T&nbsp;cell exhaustion induced by tonic signaling of chimeric antigen receptors</strong></div><div><em>Nat. Med.</em> 2015; <strong>21</strong>:581-590</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nm.3838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84930765209" target="_blank" aria-label="Open Scopus in new tab">Scopus (1240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25939063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnm.3838&amp;pmid=25939063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Lynn et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Lynn, R.C. ∙ Weber, E.W. ∙ Sotillo, E. ...</div><div><strong>c-Jun overexpression in CAR T&nbsp;cells induces exhaustion resistance</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:293-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1805-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85075931233" target="_blank" aria-label="Open Scopus in new tab">Scopus (452)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31802004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1805-z&amp;pmid=31802004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0055" role="paragraph">We hypothesized that the development of an <i>in&nbsp;vitro</i> CAR T&nbsp;cell model that employs prolonged continuous antigen exposure to drive CAR T&nbsp;cell exhaustion or dysfunction would uncover new perspectives of CAR T&nbsp;cell dysfunction. As opposed to <i>in&nbsp;vivo</i> models, an <i>in&nbsp;vitro</i> model allows for scalability, ease of manipulation, and the ability to study dynamic changes across multiple time points of T&nbsp;cell dysfunction. Despite the tremendous success of CAR T&nbsp;cells in hematological malignancies, patient responses to CAR T&nbsp;cell therapy in solid tumors are not curative. We therefore focused on solid tumors, in particular, pancreatic cancer using pancreatic cancer cells to stimulate mesothelin-directed CAR (M5CAR) T&nbsp;cells. Currently, phase 1 studies are underway evaluating the safety and feasibility of intravenous administration of M5CAR T&nbsp;cells in patients with mesothelin-positive tumors, including mesothelioma, lung, ovarian, and pancreatic cancers (NCT03054298, NCT03323944). Given that CRISPR-Cas9 technology now permits safe multiplex gene-editing of human T&nbsp;cells (<span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Stadtmauer et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Stadtmauer, E.A. ∙ Fraietta, J.A. ∙ Davis, M.M. ...</div><div><strong>CRISPR-engineered T&nbsp;cells in patients with refractory cancer</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:eaba7365</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba7365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85080829577" target="_blank" aria-label="Open Scopus in new tab">Scopus (894)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32029687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba7365&amp;pmid=32029687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), finding inducers of exhaustion in CAR T&nbsp;cells could permit in principle—via inactivation of the inducers—the development of synthetically enhanced CAR T&nbsp;cell therapies designed to treat solid tumors.</div><div id="p0060" role="paragraph">Here, we developed and validated an <i>in&nbsp;vitro</i> model of CAR T&nbsp;cell dysfunction that not only recapitulates defined characteristics of T&nbsp;cell exhaustion, but also identifies previously unknown hallmarks of CAR T&nbsp;cell dysfunction: expression of transcription factors and the transition of conventional CD8+ T-to-NK-like T&nbsp;cells. The relevance of these hallmarks of T&nbsp;cell dysfunction is further highlighted by the demonstration of loss of surface CAR and the presence of NK-like CAR T&nbsp;cells in patient samples from CAR T clinical trials. We also employed our <i>in&nbsp;vitro</i> model to identify a gene signature of dysfunction and to reveal that ID3 and SOX4 transcription factors potentiate this dysfunctional gene signature and the associated reduction in CAR T cytotoxicity. Importantly, such reduction in cytotoxicity can be attenuated by disruption of ID3 or SOX4, revealing a potential strategy to enhance the efficacy of CAR T&nbsp;cell therapy in solid tumors.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Establishment and validation of an <i>in&nbsp;vitro</i> model of CAR T dysfunction induced by prolonged and continuous antigen exposure (CAE)</h3><div id="p0065" role="paragraph">To gain a deeper understanding of CAR T&nbsp;cell exhaustion, we developed an <i>in&nbsp;vitro</i> model in which anti-mesothelin CAR (M5CAR) T&nbsp;cells were driven to a dysfunctional state through continuous antigen exposure (CAE). M5CAR contains a human MSLN-binding scFv and CD8α hinge and transmembrane domains fused to 4-1BB and CD3-ζ cytoplasmic signaling domains. To achieve CAE, M5CAR T&nbsp;cells were manufactured from normal donor (ND) peripheral blood mononuclear cells (PBMCs) and repeatedly stimulated with a mesothelin-expressing pancreatic cancer cell line (AsPC-1) such that tumor cells were never cleared by the CAR T&nbsp;cells (<a id="crosref0405" href="#fig1">Figures 1</a>A and <a id="crosref0410" href="#figs1">S1</a>A). AsPC-1 express low levels of mesothelin (<a id="crosref0415" href="#figs1">Figure&nbsp;S1</a>B). After prolonged&nbsp;stimulation (20–35&nbsp;days), M5CAR T&nbsp;cells lost or decreased doubling capacity—although the time to onset of this dysfunction varied between donors (<a id="crosref0420" href="#fig1">Figure&nbsp;1</a>B). Furthermore, although the viability of CAR T&nbsp;cells remained stable at 70% to 80%, the phenotype of apoptotic CAR T&nbsp;cells shifted from early apoptotic to late apoptotic after 18&nbsp;days of CAE (<a id="crosref0425" href="#figs1">Figure&nbsp;S1</a>C). We directly measured changes in the number of CD8+ M5CAR T&nbsp;cells by staining for CAR expression on the T&nbsp;cell surface (surCARpos) and observed increasingly reduced levels of surCARpos T&nbsp;cells undergoing prolonged CAE in most donors, similar to <span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Li et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Li, W. ∙ Qiu, S. ∙ Chen, J. ...</div><div><strong>Chimeric Antigen Receptor Designed to Prevent Ubiquitination and Downregulation Showed Durable Antitumor Efficacy</strong></div><div><em>Immunity.</em> 2020; <strong>53</strong>:456-470.e456</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.07.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.07.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85089400433" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32758419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.07.011&amp;pmid=32758419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> (<a id="crosref0435" href="#fig1">Figure&nbsp;1</a>C).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/de9cfe42-908e-4460-a881-87da4644f22c/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.016/asset/789df3bd-f72b-45fe-8867-8c6fdc2775e0/main.assets/gr1.jpg" height="942" width="740" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/de9cfe42-908e-4460-a881-87da4644f22c/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">CAR T&nbsp;cell dysfunction develops during chronic antigenic stimulation with reversible loss of cell surface expression of the CAR <i>in&nbsp;vitro</i> and in patients</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Experimental design of CAR T&nbsp;cell dysfunction <i>in&nbsp;vitro</i> model.</div><div id="fspara0020" role="paragraph">(B) Population doubling level of M5CAR transduced T&nbsp;cells during CAE, measured by changes in absolute Epcam-CD45+ counts. Five normal donors (ND) were tested.</div><div id="fspara0025" role="paragraph">(C) Time-related changes in surface expression of M5CAR on CD8+ T&nbsp;cells. Data from six donors are shown.</div><div id="fspara0030" role="paragraph">(D) Percent of sorted CD8+ CAR+ T&nbsp;cells expressing PD-1 and CTLA-4 during CAE. Two donors are shown.</div><div id="fspara0035" role="paragraph">(E) M5CAR T&nbsp;cell lysis of AsPC-1 pancreatic tumor cell line before and after CAE measured by xCelligence as real-time impedance (4:1 E:T ratio). Media and non-specific CD19BBz T&nbsp;cells are used as controls. Data are representative of four donors (see <a id="crosref1870" href="#figs1">Figure&nbsp;S1</a>C).</div><div id="fspara0040" role="paragraph">(F) Cytokine profile of CD8+ surCAR pos T&nbsp;cells (day 28 CAE, day 0 product and control CD19BBz) co-cultured with AsPC-1 cells. Significance by two-way ANOVA with Tukey’s post hoc test. Data are shown as mean ± SEM. Two additional donors were tested (see <a id="crosref1875" href="#figs1">Figure&nbsp;S1</a>I).</div><div id="fspara0045" role="paragraph">(G) M5CAR genomic DNA detection in CD8+ surface CAR-positive and -negative T&nbsp;cells (right) during CAE. Data from ND150 are shown.</div><div id="fspara0050" role="paragraph">(H) Surface CAR expression on CAE CD8+ CAR T&nbsp;cells before and after rest with IL-15. Data from ND150 are shown.</div><div id="fspara0055" role="paragraph">(I) Cell killing capacity of CD8+ M5CAR transduced T&nbsp;cells against AsPC-1 cells after 26&nbsp;days of CAE before and after 24&nbsp;h of rest with IL-15 (7:1 E:T ratio). Data representative of two donors are shown as mean ± SEM (see <a id="crosref1880" href="#figs2">Figure&nbsp;S2</a>C). Significance by Student’s t test.</div><div id="fspara0060" role="paragraph">(J) Surface (top) and intracellular (bottom) M5CAR expression on CD8+ T&nbsp;cells from pleural fluid 36&nbsp;days post-M5CAR T&nbsp;cell infusion (patient #02916-06). M5CAR FMO is shown as negative control (left).</div><div id="fspara0065" role="paragraph">See also <a id="crosref1885" href="#figs1">Figures S1</a> and <a id="crosref1890" href="#figs2">S2</a>. ∗∗∗∗p&nbsp;&lt; 0.0001, ∗∗∗p&nbsp;&lt; 0.001, ∗∗p&nbsp;&lt; 0.01.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/a184ead6-5536-4209-93e0-b143109e1ccd/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.016/asset/c829819e-c741-4233-8bbc-8ac085b847ef/main.assets/figs1.jpg" height="849" width="657" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/a184ead6-5536-4209-93e0-b143109e1ccd/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">CAR T&nbsp;cells continuously exposed to antigen lose effector function, related to <a id="crosref2035" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0390" role="paragraph"><b>(A)</b> Detailed experimental design of CAR T&nbsp;cell dysfunction <i>in&nbsp;vitro</i> model. <b>(B)</b> Mesothelin expression on AsPC-1 cells, measured by flow cytometry. <b>(C)</b> Apoptosis characterization by Annexin V binding assessment and loss of plasma membrane integrity (L/D Aqua staining), showing live, early apoptotic and late apoptotic M5CAR transduced T&nbsp;cells during CAE. A single normal donor was tested. Data shown as mean with standard deviation <b>(D)</b> Frequency of TIM3+ CD8+ M5CAR T&nbsp;cells on day 0 product and after CAE. Data is representative of two different donors. <sup>∗∗</sup>P &lt;0.01 by Mann-Whitney test. <b>(E)</b> Tumor cytotoxicity of CD8+ surCARpos T&nbsp;cells (CAE and day 0 product) against AsPC-1 cells using 3 different donor T&nbsp;cells (ND388, ND534, and ND516) measured by xCelligence. <b>(F)</b> Cell killing capacity of CD8+ M5CAR transduced T&nbsp;cells (CAE and day 0 product) against AsPC-1 cells using 2 different donor T&nbsp;cells (ND150 8:1 E:T ratio, ND538 7:1 E:T ratio) measured by Celigo. <sup>∗∗∗∗</sup><i>P</i> &lt;0.0001, <sup>∗∗∗</sup><i>P</i> &lt;0.001, <sup>∗∗</sup><i>P</i> &lt;0.01 by Student’s <i>t</i> test. <b>(G)</b> Mesothelin expression on mesothelin transduced K562 cells (K562-meso), measured by flow cytometry. <b>(H)</b> Kinetics of K562-meso cell lysis incubated with CD8+ M5CAR transduced T&nbsp;cells (day 26 CAE and day 0 product) using 2 different donor T&nbsp;cells (ND150 8:1 E:T ratio, ND538 7:1 E:T ratio) measured by Celigo. <sup>∗∗∗</sup><i>P</i> &lt;0.001, <sup>∗∗</sup><i>P</i> &lt;0.01, <sup>∗</sup><i>P</i>&nbsp;&lt; 0.05 by Student’s <i>t</i> test. <b>(I)</b> Cytokine secretion of CD8+ surCARpos T&nbsp;cells (day 28 CAE and day 0 product) stimulated with AsPC-1 for 24&nbsp;hrs using 2 different donor T&nbsp;cells (ND388 and ND534). CD19BBz CAR T&nbsp;cells were tested as a control for allogeneic recognition of AsPC-1. <sup>∗∗∗∗</sup><i>P</i> &lt;0.0001, <sup>∗</sup><i>P</i>&nbsp;&lt; 0.05 by two-way ANOVA with Tukey’s post hoc test. <b>(J)</b> Cytokine profile of CD8+ surCARpos T&nbsp;cells (day 24 CAE and day 0 product). Sorted CD8+ surCARpos T&nbsp;cells were stimulated with PMA&nbsp;+ ionomycin or AsPC-1, and media was used as a control. Two donors were analyzed (ND150 and ND534). <sup>∗∗∗∗</sup><i>P</i> &lt;0.0001, <sup>∗∗∗</sup><i>P</i> &lt;0.001, <sup>∗∗</sup><i>P</i> &lt;0.01, by two-way ANOVA with Tukey’s post hoc test.</div></div></div></div></figcaption></figure></div><div id="p0070" role="paragraph">At baseline, CD8+ M5CAR T&nbsp;cells did not express immune checkpoint inhibitors PD-1 or CTLA-4; however, this population exhibited high levels after initial stimulation (day 3) and, as expected, remained elevated above baseline in dysfunctional T&nbsp;cells (<a id="crosref0440" href="#fig1">Figure&nbsp;1</a>D). In addition, CAR T&nbsp;cells upregulated the exhaustion marker TIM3 upon prolonged antigen stimulation (<a id="crosref0445" href="#figs1">Figure&nbsp;S1</a>D). Moreover, we examined tumor cytotoxicity of CAR T&nbsp;cells following CAE (<a id="crosref0450" href="#fig1">Figures 1</a>E, <a id="crosref0455" href="#figs1">S1</a>E, and S1F). While day 0 (unstimulated) CD8+ surCARpos M5CAR T&nbsp;cells eliminated tumor cells, day 28 CD8+ surCARpos T&nbsp;cells and non-specific control CD8+ CD19CAR (BBz)-positive T&nbsp;cells did not control tumor growth, revealing that surCARpos T&nbsp;cells become dysfunctional after tumor recognition and CAE. Loss of effector function was not specific to co-culture with the AsPC-1 tumor cell line; similar results were observed when CD8+ M5CAR T&nbsp;cells were continuously stimulated with K562-meso tumor cells, a human myelogenous leukemia cell line engineered to express mesothelin (<a id="crosref0460" href="#figs1">Figures S1</a>G and S1H). Further, while day 0 CD8+ M5CAR T&nbsp;cells produced high levels of TNF-α and IL-2, CAE CD8+ M5CAR T&nbsp;cells and day 0 CD19BBz antigen control CAR T&nbsp;cells lacked cytokine production (<a id="crosref0465" href="#fig1">Figures 1</a>F and <a id="crosref0470" href="#figs1">S1</a>I). Together, these data demonstrate that our <i>in&nbsp;vitro</i> model induces progressive CAR T&nbsp;cell dysfunction that is dependent on antigen recognition.</div><div id="p0075" role="paragraph">Next, we examined whether this dysfunctional phenotype of CAR T&nbsp;cells in our model is specific to CAR signaling. We collected CD8+ M5CAR T&nbsp;cells following 24&nbsp;days of CAE, then stimulated with PMA&nbsp;+ ionomycin or AsPC-1 cells to measure cytokine production capacity. Both CAE and day 0 cells produced large amounts of IL-2 and IFN-γ after being stimulated with PMA&nbsp;+ ionomycin. However, when stimulated with AsPC-1 cells, cytokine production by the CAE cells was significantly reduced (<a id="crosref0475" href="#figs1">Figure&nbsp;S1</a>J). CAE M5CAR T&nbsp;cells failed to secrete cytokines after prolonged CAR engagement, but still retained the ability to produce cytokines through pharmacologic stimulation by a CAR bypass mechanism, suggesting that downstream signaling remains intact.</div></section><section id="sec-2-2"><h3>Rest restores surface CAR expression and improves cytotoxicity</h3><div id="p0080" role="paragraph">To further explore the decline in surface CAR expression with CAE, we sorted surface CAR-positive (surCARpos) and surface CAR-negative (surCARneg) M5CAR T&nbsp;cell populations at 4, 7, and 17&nbsp;days of CAE. Importantly, by day 17 of CAE, these two populations demonstrated equivalent amounts of genomic CAR DNA by qPCR, indicating that most surface CAR-negative cells are transduced CAR T&nbsp;cells with the CAR ligand internalized (<a id="crosref0480" href="#fig1">Figure&nbsp;1</a>G, left). To test whether CAE-induced loss of surface CAR is reversible in our model, transduced M5CAR T&nbsp;cells were cultured under CAE (<a id="crosref0485" href="#figs2">Figure&nbsp;S2</a>A), sorted for surCARneg cells (<a id="crosref0490" href="#fig1">Figure&nbsp;1</a>H, left), and then rested with fresh media plus IL-15 for a day. 38% of surCARneg CD8+ T&nbsp;cells regained surface CAR expression (<a id="crosref0495" href="#fig1">Figure&nbsp;1</a>H). We next investigated the impact of CAE-induced surface CAR loss on M5CAR T&nbsp;cell effector function by measuring cell killing capacity. Bulk CD8+ T cells collected after CAE could not control tumor growth; however, 24-h rest with IL-15 dramatically rescued their cytotoxic ability (<a id="crosref0500" href="#fig1">Figures 1</a>I, <a id="crosref0505" href="#figs2">S2</a>B, and S2C). Taken together, these results suggest that although loss of surface CAR expression is observed after several weeks of CAE, M5CAR T&nbsp;cells can recover effector function and surface CAR expression with rest and IL-15 supplement.</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/5ef7d8fa-1fc6-44a5-8e26-2033b832d5fc/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.016/asset/86e7b4e6-4675-4056-93eb-8f59571f3249/main.assets/figs2.jpg" height="589" width="649" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/5ef7d8fa-1fc6-44a5-8e26-2033b832d5fc/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Rest improves CAR T effector function and CAR internalization in patient samples, related to <a id="crosref2040" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0400" role="paragraph"><b>(A)</b> Surface CAR expression on day 23 CAE CD8+ M5CAR-transduced T&nbsp;cells before sorting, associated with <a id="crosref2045" href="#fig1">Figure&nbsp;1</a>H. <b>(B)</b> Frequency of surCARpos CD8+ T&nbsp;cells used for cell killing assay (day 26), associated with <a id="crosref2050" href="#fig1">Figure&nbsp;1</a>I. <b>(C)</b> Cell killing capacity of donor ND538 CD8+ M5CAR-transduced T&nbsp;cells against AsPC-1 cells after 26&nbsp;days of CAE before and after 24&nbsp;hrs of rest with IL-15 (7:1 E:T ratio). <sup>∗∗</sup><i>P</i> &lt;0.01, <sup>∗</sup><i>P</i> &lt;0.05 by Student’s <i>t</i> test, data is shown as mean ± SEM. <b>(D)</b> Tumor cells (mesothelin+ CD45-) in human pleural fluid on day 36 after CAR T&nbsp;cell infusion (patient #02916-06) and in peritoneal fluid on day 21 after CAR T&nbsp;cell infusion (patient #02916-01). <b>(E)</b> Surface CAR expression (top) and intracellular M5CAR expression (bottom) on human CD8+ T&nbsp;cells in peritoneal fluid (patient #02916-01) after 21&nbsp;days of M5CAR T&nbsp;cell infusion. CAR staining (right) and M5CAR FMO control (left).</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Having demonstrated reduced surface expression of the M5CAR <i>in&nbsp;vitro</i> under CAE, we examined the clinical relevance of this phenomenon in the human TME. We obtained peritoneal/pleural fluid samples collected after M5CAR T&nbsp;cell infusion from two ovarian cancer patients enrolled on a M5CAR T&nbsp;cell trial (NCT03054298). We identified tumor cells (<a id="crosref0510" href="#figs2">Figure&nbsp;S2</a>D) and M5CAR CD8+ T&nbsp;cells post-CAR intravenous infusion (<a id="crosref0515" href="#fig1">Figures 1</a>J and <a id="crosref0520" href="#figs2">S2</a>E). Although the levels of M5CAR T&nbsp;cells were low as determined by qPCR (data not shown), we were able to detect CAR T&nbsp;cells by flow cytometry. Notably, the frequency of intracellular CAR-positive T&nbsp;cells (<a id="crosref0525" href="#fig1">Figure&nbsp;1</a>J, bottom right), which represents both surCARpos and surCARneg T&nbsp;cells, was higher than surCARpos T&nbsp;cells alone (<a id="crosref0530" href="#fig1">Figure&nbsp;1</a>J, top right), confirming that M5CAR T&nbsp;cells exhibit reduced expression of CAR on the cell surface after infusion in the human TME (<a id="crosref0535" href="#fig1">Figures 1</a>J and <a id="crosref0540" href="#figs2">S2</a>E).</div></section><section id="sec-2-3"><h3>Transcriptional dynamics of dysfunctional CAR T&nbsp;cells</h3><div id="p0090" role="paragraph">To better understand the mechanisms driving loss of CAR T effector function, we performed bulk RNA-seq on CD8+ day 0 product and day 28 CAE surCARpos cells (<a id="crosref0545" href="#fig2">Figure&nbsp;2</a>A; <a id="crosref0550" href="#mmc1">Table S1</a>). In parallel, we performed RNA-seq on day 0 and day 28 CAE surCARneg CD8+ T&nbsp;cells (comprising both untransduced T&nbsp;cells and internalized CAR T&nbsp;cells). There was strong correlation of the gene expression signatures for surCARpos and surCARneg populations (<a id="crosref0555" href="#fig2">Figures 2</a>B, <a id="crosref0560" href="#figs3">S3</a>A, and S3B), suggesting that CAR T&nbsp;cells acquire the dysregulation signature before developing impaired expression of surface CAR. Since our phenotypic studies were performed in surCARpos cells (see <a id="crosref0565" href="#fig1">Figure&nbsp;1</a>) and the mechanisms of dysfunction in this population are unexplored, we decided to focus on this population for the remainder of the bulk RNA-seq analyses.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/59263fd7-8743-4ed2-860e-f1319d408528/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.016/asset/e564fb8b-cd94-443f-8b5d-30f5fe9065b3/main.assets/gr2.jpg" height="920" width="753" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/59263fd7-8743-4ed2-860e-f1319d408528/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Transcriptional dynamics of dysfunctional CAR T&nbsp;cells</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0075" role="paragraph">(A) Differentially expressed genes between day 0 and 28 CAE surCARpos cells. Genes on the right are upregulated at day 28 (N&nbsp;= 521) and genes on the left are downregulated (N&nbsp;= 517). Red dots indicate significant genes with adjusted p values &lt;0.05 and fold change &gt;2. Analysis includes four biological replicates.</div><div id="fspara0080" role="paragraph">(B) Average gene expression values (TPMs) for day 28 surCARpos compared to day 28 surCARneg for differentially expressed genes defined in <a id="crosref1895" href="#fig2">Figure&nbsp;2</a>A (top) and all genes (bottom).</div><div id="fspara0085" role="paragraph">(C) Ingenuity Pathway Analysis (IPA) of significant genes from 2A. Red denotes NK and blue denotes exhaustion pathways.</div><div id="fspara0090" role="paragraph">(D) Normalized RNA-seq counts of representative NK-related genes. Average of four biological replicates with standard deviation depicted. Statistics by Mann-Whitney U test.</div><div id="fspara0095" role="paragraph">(E) Heatmap of genes differentially expressed between day 0, 16, and 28 CAE surCARpos cells (N&nbsp;= 762 genes). Average of two biological replicates.</div><div id="fspara0100" role="paragraph">(F) IPA upstream regulator analysis of transcription factors predicted to regulate the differentially expressed genes between days 0 and 28, ranked by -log(p value). Gene expression log<sub>2</sub> FC (day 28/day 0) is shown on the right. Only transcription factors dysregulated upon CAE are shown.</div><div id="fspara0105" role="paragraph">(G and H) Representative ATAC-seq tracks (top) and pooled RNA-seq tracks (bottom) from day 0 and 28 samples at <i>ID3</i> (G) and <i>KLF2</i> (H) regulatory regions. Analysis includes four biological replicates.</div><div id="fspara0110" role="paragraph">See also <a id="crosref1900" href="#figs3">Figures S3</a> and <a id="crosref1905" href="#figs4">S4</a> and <a id="crosref1910" href="#mmc1">Tables S1</a>, <a id="crosref1915" href="#mmc2">S2</a>, and <a id="crosref1920" href="#mmc3">S3</a>. ∗∗∗p&nbsp;&lt; 0.001, ∗∗p&nbsp;&lt; 0.01.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/ee965794-8cd6-40a4-b8c2-cebff52133aa/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.016/asset/eca3dc33-7ec0-4644-86c2-4fb7d3f6a3f7/main.assets/figs3.jpg" height="941" width="752" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/ee965794-8cd6-40a4-b8c2-cebff52133aa/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Overlap of CD8+ CAR T&nbsp;cell dysfunction gene signature with <i>in&nbsp;vivo</i> models of T&nbsp;cell exhaustion and CD4+ CAR T&nbsp;cell signature, related to <a id="crosref2055" href="#fig2">Figure&nbsp;2</a> and <a id="crosref2060" href="#mmc5">Table S5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0410" role="paragraph">Average gene expression values (TPMs) for day 0 surCARpos compared to day 0 surCARneg cells for differentially expressed genes defined in <a id="crosref2065" href="#fig2">Figure&nbsp;2</a>A <b>(A)</b> and for all genes <b>(B). (C)</b> Venn diagram displaying overlap between genes upregulated in day 28 CAE surCARpos CD8+ T&nbsp;cells (see <a id="crosref2070" href="#fig2">Figure&nbsp;2</a>A) and genes upregulated in exhausted CD8+ T&nbsp;cells from the LCMV clone-13 mouse model of chronic viral infection. Only genes with mouse to human orthologs were included. <b>(D)</b> Overlap of genes downregulated in day 28 CAE surCARpos cells (see <a id="crosref2075" href="#fig2">Figure&nbsp;2</a>A) and genes downregulated in exhausted T&nbsp;cells from the LCMV clone-13 mouse model of chronic viral infection. <b>(E)</b> GSEA analysis of day 0 and 28 CAE surCARpos RNA-seq samples with exhausted T&nbsp;cell populations from (<span class="dropBlock reference-citations"><a id="body-ref-sref7-1" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-1" aria-controls="sref7-1">Beltra et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref7-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Beltra, J.C. ∙ Manne, S. ∙ Abdel-Hakeem, M.S. ...</div><div><strong>Developmental Relationships of Four Exhausted CD8(+) T Cell Subsets Reveals Underlying Transcriptional and Epigenetic Landscape Control Mechanisms</strong></div><div><em>Immunity.</em> 2020; <strong>52</strong>:825-841.e828</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084517361" target="_blank" aria-label="Open Scopus in new tab">Scopus (486)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32396847/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.04.014&amp;pmid=32396847" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for TEX-progenitor 1, TEX-progenitor 2, TEX-intermediate, and TEX-terminal from the LCMV mouse model of chronic viral infection. Overlap of genes upregulated in day 28 CAE surCARpos cells (see <a id="crosref2085" href="#fig2">Figure&nbsp;2</a>A) and genes that define dysfunctional CD8+ TILs from hepatocellular carcinoma patients [HCC] <b>(F)</b>, melanoma patients <b>(G)</b>, non-small-cell lung cancer patients [NSCLC] <b>(H),</b> and colorectal cancer patients [CRC] <b>(I)</b>. <b>(J)</b> Overlap of HCC, melanoma, NSCLC, and CRC dysfunctional TIL signature genes. <b>(K)</b> Heatmap of day 0 and 28 CAE surCARpos RNA-seq samples on CD8+ TIL dysfunction gene signature common between hepatocellular carcinoma, non-small-cell lung cancer, colorectal cancer, and melanoma (18 genes in common from <a id="crosref2090" href="#figs3">Figure&nbsp;S3</a>J). <b>(L)</b> GSEA analysis of day 0 and 28 CAE surCARpos RNA-seq samples with genes upregulated in CD8+ HA GD2 CAR T&nbsp;cells from (<span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Lynn et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Lynn, R.C. ∙ Weber, E.W. ∙ Sotillo, E. ...</div><div><strong>c-Jun overexpression in CAR T&nbsp;cells induces exhaustion resistance</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:293-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1805-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85075931233" target="_blank" aria-label="Open Scopus in new tab">Scopus (452)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31802004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1805-z&amp;pmid=31802004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <b>(M)</b> Overlap of genes upregulated in CD4+ T&nbsp;cells and CD8+ T&nbsp;cells in day 28 CAE surCARpos cells (compared to day 0 cells) (left) and for genes downregulated (right). <b>(N)</b> Normalized RNA-seq counts of representative dysfunction signature genes in CD4+ T&nbsp;cells at day 0 and day 28 CAE. Average of two biological replicates and data is shown as mean ± SEM. Statistical significance of overlap between two groups (in panels C-D, F-I, M) was calculated using hypergeometric test.</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph">Next, we investigated how well our model correlates with established <i>in&nbsp;vivo</i> models of T&nbsp;cell exhaustion. 27% of genes upregulated in CAE CD8+ surCARpos T&nbsp;cells overlapped with genes upregulated in exhausted T&nbsp;cells from the chronic lymphocytic choriomeningitis virus (LCMV) mouse model (<span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Pauken et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Pauken, K.E. ∙ Sammons, M.A. ∙ Odorizzi, P.M. ...</div><div><strong>Epigenetic stability of exhausted T&nbsp;cells limits durability of reinvigoration by PD-1 blockade</strong></div><div><em>Science.</em> 2016; <strong>354</strong>:1160-1165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf2807" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84994364943" target="_blank" aria-label="Open Scopus in new tab">Scopus (893)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27789795/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf2807&amp;pmid=27789795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), including genes implicated in T&nbsp;cell exhaustion (<i>CTLA4</i>, <i>TOX</i>, <i>TIGIT</i>, <i>NR4A2</i>, <i>NR4A3</i>, <i>HAVCR2</i> [TIM3], <i>ENTPD1</i> [CD39], <i>TNFRSF9</i> [4-1BB]) (<a id="crosref0575" href="#figs3">Figure&nbsp;S3</a>C). There was also significant overlap between genes downregulated in CAE and exhausted T&nbsp;cells, which included genes known to be expressed in naïve or memory CD8+ T&nbsp;cells (<i>IL7R</i>, <i>LEF1</i>, <i>SELL</i>) (<a id="crosref0580" href="#figs3">Figure&nbsp;S3</a>D). Further, GSEA analysis of our data with the four transient states of T&nbsp;cell exhaustion identified in the LCMV mouse model (<span class="dropBlock reference-citations"><a id="body-ref-sref7-2" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-2" aria-controls="sref7-2">Beltra et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref7-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Beltra, J.C. ∙ Manne, S. ∙ Abdel-Hakeem, M.S. ...</div><div><strong>Developmental Relationships of Four Exhausted CD8(+) T Cell Subsets Reveals Underlying Transcriptional and Epigenetic Landscape Control Mechanisms</strong></div><div><em>Immunity.</em> 2020; <strong>52</strong>:825-841.e828</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084517361" target="_blank" aria-label="Open Scopus in new tab">Scopus (486)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32396847/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.04.014&amp;pmid=32396847" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) revealed significant enrichment with the intermediate and terminally exhausted T&nbsp;cell populations (<a id="crosref0590" href="#figs3">Figure&nbsp;S3</a>E), indicating that our model recapitulates features of the later stages of T&nbsp;cell exhaustion in mouse T&nbsp;cells.</div><div id="p0100" role="paragraph">We also compared our model to TILs. The single-cell RNA-seq (scRNA-seq) gene signatures of dysfunctional human CD8+ TILs isolated from patients with melanoma (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, H. ∙ van der Leun, A.M. ∙ Yofe, I. ...</div><div><strong>Dysfunctional CD8&nbsp;T Cells Form a Proliferative, Dynamically Regulated Compartment within Human Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:775-789.e718</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061004091" target="_blank" aria-label="Open Scopus in new tab">Scopus (661)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30595452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.11.043&amp;pmid=30595452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), hepatocellular carcinoma (<span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zheng, C. ∙ Zheng, L. ∙ Yoo, J.K. ...</div><div><strong>Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1342-1356.e1316</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85020894317" target="_blank" aria-label="Open Scopus in new tab">Scopus (1383)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.035&amp;pmid=28622514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and colorectal (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Zhang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Zhang, L. ∙ Yu, X. ∙ Zheng, L. ...</div><div><strong>Lineage tracking reveals dynamic relationships of T&nbsp;cells in colorectal cancer</strong></div><div><em>Nature.</em> 2018; <strong>564</strong>:268-272</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0694-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85058518221" target="_blank" aria-label="Open Scopus in new tab">Scopus (694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30479382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0694-x&amp;pmid=30479382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and non-small cell lung cancer (<span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Guo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) significantly overlapped with genes upregulated in CAE surCARpos T&nbsp;cells (<a id="crosref0615" href="#figs3">Figures S3</a>F–S3I). We overlapped datasets from the four cancer types and found a common group of 18 TIL marker genes (<a id="crosref0620" href="#figs3">Figure&nbsp;S3</a>J), and most of these genes were upregulated in CAE surCARpos T&nbsp;cells (<a id="crosref0625" href="#figs3">Figure&nbsp;S3</a>K). To determine how applicable our signature is to other CARs, we performed GSEA analysis of the exhaustion signature curated in GD2-directed CARs (<span class="dropBlock reference-citations"><a id="body-ref-sref41-3" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-3" aria-controls="sref41-3">Lynn et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref41-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Lynn, R.C. ∙ Weber, E.W. ∙ Sotillo, E. ...</div><div><strong>c-Jun overexpression in CAR T&nbsp;cells induces exhaustion resistance</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:293-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1805-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85075931233" target="_blank" aria-label="Open Scopus in new tab">Scopus (452)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31802004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1805-z&amp;pmid=31802004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Genes upregulated in the exhausted CD8+ GD2 CAR T&nbsp;cells were significantly enriched with genes up in day 28 CAE M5CAR T&nbsp;cells, suggesting that at least some of the signaling observed in the 4-1BB mesothelin-directed dysfunctional CAR T&nbsp;cells is conserved in the exhausted GD2-28z CAR T&nbsp;cells (<a id="crosref0635" href="#figs3">Figure&nbsp;S3</a>L). Taken together, these analyses provide further evidence that our <i>in&nbsp;vitro</i> model of CAR T&nbsp;cell dysfunction aligns with many features of <i>in&nbsp;vivo</i> human and mouse models of T&nbsp;cell exhaustion and dysfunction.</div><div id="p0105" role="paragraph">To further illuminate the biological functions of the entire dysregulated gene expression signature identified in <a id="crosref0640" href="#fig2">Figure&nbsp;2</a>A, we performed Ingenuity Pathway Analysis (IPA). As expected, T&nbsp;cell exhaustion, PD-1/PD-L1 cancer immunotherapy, and CTLA4 signaling pathways were enriched (<a id="crosref0645" href="#fig2">Figure&nbsp;2</a>C, blue; <a id="crosref0650" href="#mmc2">Table S2</a>). Interestingly, several pathways related to natural killer (NK) cells were also enriched in the gene expression signature of CAE CD8+ surCARpos T&nbsp;cells (<a id="crosref0655" href="#fig2">Figure&nbsp;2</a>C, red). In fact, we noted that multiple NK receptors were upregulated, including <i>KLRC1</i>, <i>KLRC2</i>, <i>KLRC3</i>, <i>KLRB1</i>, <i>KLRD1</i>, and <i>KIR2DL4</i> (<a id="crosref0660" href="#fig2">Figure&nbsp;2</a>D). ⍺β T&nbsp;cells often upregulate receptors constitutively expressed by NK cells, potentially due to chronic activation by antigens and cytokines (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Balin et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Balin, S.J. ∙ Pellegrini, M. ∙ Klechevsky, E. ...</div><div><strong>Human antimicrobial cytotoxic T lymphocytes, defined by NK receptors and antimicrobial proteins, kill intracellular bacteria</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaat7668</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aat7668" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85059961035" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30171080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aat7668&amp;pmid=30171080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">McMahon et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>McMahon, C.W. ∙ Zajac, A.J. ∙ Jamieson, A.M. ...</div><div><strong>Viral and bacterial infections induce expression of multiple NK cell receptors in responding CD8(+) T&nbsp;cells</strong></div><div><em>J.&nbsp;Immunol.</em> 2002; <strong>169</strong>:1444-1452</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.169.3.1444" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12133970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.169.3.1444&amp;pmid=12133970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Meresse et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Meresse, B. ∙ Chen, Z. ∙ Ciszewski, C. ...</div><div><strong>Coordinated induction by IL15 of a TCR-independent NKG2D signaling pathway converts CTL into lymphokine-activated killer cells in celiac disease</strong></div><div><em>Immunity.</em> 2004; <strong>21</strong>:357-366</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2004.06.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2004.06.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-4444240035" target="_blank" aria-label="Open Scopus in new tab">Scopus (695)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15357947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2004.06.020&amp;pmid=15357947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To identify whether CAE drives a similar gene expression program in CD4+ T&nbsp;cells, we performed RNA-seq on day 0 and day 28 CAE surCARpos CD4+ T&nbsp;cells and found significant overlap between the CD4+ and CD8+ T&nbsp;cell signatures following CAE, including the upregulation of NK receptors (<i>KLRB1</i>, <i>KLRC1</i>, <i>KLRC2</i>, <i>KLRC3</i>, <i>KLRD1</i>) and other genes in our signature including <i>GNLY</i>, <i>LAYN</i>, <i>CD9</i>, <i>PHLDA1</i>, <i>SOX4</i>, and <i>TNFRSF9</i>, among others (<a id="crosref0680" href="#figs3">Figures S3</a>M and S3N).</div><div id="p0110" role="paragraph">To better understand how gene expression changes over time in our model, we performed RNA-seq on CAE surCARpos CD8+ T&nbsp;cells at day 16 (a middle time point). We identified genes that showed temporal changes in expression between days 0, 16, and 28 (<a id="crosref0685" href="#fig2">Figure&nbsp;2</a>E). For example, many NK receptors and exhaustion markers gradually turned on, with moderate expression by day 16 and highest expression by day 28 (cluster 5: <i>KLRD1</i>, <i>KLRC1</i>, <i>KLRC2</i>, <i>KLRC3</i>, <i>TOX</i>, <i>HAVCR2</i>, <i>TIGIT</i>), while other markers remained off or lowly expressed until dramatic upregulation at day 28 (cluster 4: <i>KLRB1</i>, <i>KLRK1</i>). Cluster 6 genes displayed robust activation on day 16 with slight downregulation by day 28, and included inhibitory molecules (<i>CTLA4</i>, <i>LAG3</i>), genes encoding chemokines (<i>CCL3</i>, <i>CCL4</i>, <i>CXCL8</i>), cytotoxic molecules (<i>PRF1</i>, <i>GZMB</i>, <i>NKG7</i>), and T&nbsp;cell activation genes (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Boroughs et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Boroughs, A.C. ∙ Larson, R.C. ∙ Marjanovic, N.D. ...</div><div><strong>A Distinct Transcriptional Program in Human CAR T Cells Bearing the 4-1BB Signaling Domain Revealed by scRNA-Seq</strong></div><div><em>Mol. Ther.</em> 2020; <strong>28</strong>:2577-2592</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ymthe.2020.07.023&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ymthe.2020.07.023&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85088996471" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32755564/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymthe.2020.07.023&amp;pmid=32755564" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0115" role="paragraph">Next, we identified potential transcription factors that control the dysregulated gene expression signature in CAE surCARpos T&nbsp;cells. This list included genes that were upregulated (<i>EGR1</i>, <i>ID3</i>, <i>SOX4</i>, <i>RBPJ</i>) as well as downregulated (<i>KLF2</i>, <i>BCL6</i>, <i>LEF1</i>) in CAE surCARpos cells (<a id="crosref0695" href="#fig2">Figure&nbsp;2</a>F; <a id="crosref0700" href="#mmc3">Table S3</a>).</div><div id="p0120" role="paragraph">We performed ATAC-seq (assay for transposase-accessible chromatin with sequencing) to explore CAE-specific regulatory changes in surCARpos cells. Overall, there was a closing of chromatin upon CAE (<a id="crosref0705" href="#figs4">Figure&nbsp;S4</a>A). Of the sites that opened in CAE, most were in introns, intergenic, and promoter regions consistent with a regulatory role (<a id="crosref0710" href="#figs4">Figure&nbsp;S4</a>B). We integrated our RNA-seq and ATAC-seq datasets and found that genes upregulated in CAE displayed an opening of chromatin, while genes downregulated in CAE displayed a closing of chromatin (<a id="crosref0715" href="#figs4">Figure&nbsp;S4</a>C). For example, the upregulated gene <i>ID3</i> and the downregulated gene <i>KLF2</i> displayed opening and closing of chromatin at nearby regulatory regions, respectively (<a id="crosref0720" href="#fig2">Figures 2</a>G and 2H).</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/3732359c-b9e9-4142-9477-1f18ce7a753c/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.016/asset/a1846af5-4119-4b0a-9070-33d5402b9e2f/main.assets/figs4.jpg" height="941" width="728" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/3732359c-b9e9-4142-9477-1f18ce7a753c/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Chromatin dynamics of dysfunctional CAR T&nbsp;cells and scRNA-seq analyses, related to <a id="crosref2100" href="#fig2">Figures 2</a> and <a id="crosref2105" href="#fig3">3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0420" role="paragraph"><b>(A)</b> ATAC-seq open chromatin regions specific to day 0 (left) or day 28 CAE (right) surCARpos cells. ATAC-seq analyses include 4 biological replicates. <b>(B)</b> Genomic location of open chromatin regions for day 0 (blue) and day 28 CAE (red) surCARpos cells. <b>(C)</b> Relation of gene expression and chromatin changes during CAE. Average ATAC-seq signal of genes upregulated at day 28 (left) and genes downregulated (right) in day 0 and day 28 CAE surCARpos cells. Average of 4 biological replicates represented with box plots. Statistics assessed by Mann-Whitney. <b>(D)</b> Heatmap of sub-nucleosome ATAC-seq tags for naïve T&nbsp;cells and PD1-high TILs from (<span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Philip et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Philip, M. ∙ Fairchild, L. ∙ Sun, L. ...</div><div><strong>Chromatin states define tumour-specific T&nbsp;cell dysfunction and reprogramming</strong></div><div><em>Nature.</em> 2017; <strong>545</strong>:452-456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22367" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85019673982" target="_blank" aria-label="Open Scopus in new tab">Scopus (602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28514453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22367&amp;pmid=28514453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) at sites opening in day 28 CAE surCARpos cells compared to day 0 cells. <b>(E)</b> Box plot of PD1-high sub-nucleosome ATAC-seq tags from (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Philip et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Philip, M. ∙ Fairchild, L. ∙ Sun, L. ...</div><div><strong>Chromatin states define tumour-specific T&nbsp;cell dysfunction and reprogramming</strong></div><div><em>Nature.</em> 2017; <strong>545</strong>:452-456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22367" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85019673982" target="_blank" aria-label="Open Scopus in new tab">Scopus (602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28514453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22367&amp;pmid=28514453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) normalized to naïve T&nbsp;cells at day 28 CAE specific sites versus an equal number of randomly sampled other open chromatin regions (OCRs). Difference assessed by permutation test. <b>(F)</b> ATAC-seq tracks from day 0 and day 28 CAE cells at <i>CD5</i>, <i>CD28</i>, and <i>TCF7</i> regulatory regions. Analysis includes four biological replicates. <b>(G)</b> Decile plot showing correlation between bulk RNA-seq and single-cell RNA-seq datasets. Genes differentially expressed in single-cell data between day 20 CAE and day 0, defined by cellfishing.jl, were divided into 10 groups and sorted by single-cell fold-change, going from lowest to highest (x axis). Y-axis plots the fold change (day 28 CAE/day 0) in the bulk RNA-seq for genes in each group. <b>(H)</b> Heatmap of top 10 marker genes for each day 0 single-cell cluster as defined in <a id="crosref2120" href="#fig3">Figure&nbsp;3</a>A, donor ND388. Columns correspond to cells and rows correspond to gene names. <b>(I)</b> Gene ontology determined by metascape pathway analysis for each single-cell day 0 cluster, donor ND388. Columns are cell clusters (defined in <a id="crosref2125" href="#fig3">Figure&nbsp;3</a>A) and rows are enriched pathways color coded by level of significance. <b>(J)</b> Venn diagram displaying overlap between genes upregulated in day 28 CAE surCARpos CD8+ T&nbsp;cells from bulk RNA-seq analyses (see <a id="crosref2130" href="#fig2">Figure&nbsp;2</a>A) and genes that define single cell clusters D20-1, D20-2, D20-3, and D20-4 from day 20 CAE cells (see <a id="crosref2135" href="#fig3">Figure&nbsp;3</a>B). Significance calculated using hypergeometric test. <b>(K)</b> Violin plots depicting gene expression levels from day 20 CAE cells (donor ND388) for <i>SRGAP3, DUSP4, and CSF1</i>. X axis is cell clusters defined in <a id="crosref2140" href="#fig3">Figure&nbsp;3</a>B. <b>(L)</b> Violin plot of day 20 CAE cells for <i>CTLA4</i>, donor ND388 (padj 1.2e-11, log2FC 0.20).</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">To determine if the epigenetic landscape of the dysfunctional CAR T&nbsp;cells is similar to TCR-mediated exhaustion, we queried ATAC-seq datasets from exhausted human PD1-high TILs (<span class="dropBlock reference-citations"><a id="body-ref-sref53-3" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-3" aria-controls="sref53-3">Philip et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Philip, M. ∙ Fairchild, L. ∙ Sun, L. ...</div><div><strong>Chromatin states define tumour-specific T&nbsp;cell dysfunction and reprogramming</strong></div><div><em>Nature.</em> 2017; <strong>545</strong>:452-456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22367" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85019673982" target="_blank" aria-label="Open Scopus in new tab">Scopus (602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28514453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22367&amp;pmid=28514453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and found that chromatin sites opening in day 28 CAE cells are also open in exhausted TILs (<a id="crosref0730" href="#figs4">Figures S4</a>D and S4E). We also observed closing of chromatin in day 28 CAE cells at <i>CD5</i>, <i>CD28</i>, and <i>TCF7</i>, similar to PD1-high human TILs or dysfunctional mouse T&nbsp;cells, as previously reported (<span class="dropBlock reference-citations"><a id="body-ref-sref53-4" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-4" aria-controls="sref53-4">Philip et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Philip, M. ∙ Fairchild, L. ∙ Sun, L. ...</div><div><strong>Chromatin states define tumour-specific T&nbsp;cell dysfunction and reprogramming</strong></div><div><em>Nature.</em> 2017; <strong>545</strong>:452-456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22367" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85019673982" target="_blank" aria-label="Open Scopus in new tab">Scopus (602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28514453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22367&amp;pmid=28514453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0740" href="#figs4">Figure&nbsp;S4</a>F).</div></section><section id="sec-2-4"><h3>Single-cell analysis of CAE CD8+ T&nbsp;cells reveals co-expression of dysfunction signature genes</h3><div id="p0130" role="paragraph">We performed scRNA-seq for day 0 and day 20 CAE cells. Of note, this experiment was performed in CAR-transduced CD8+ T&nbsp;cells and thus includes a mixed population of surCARpos, surCARneg, and untransduced CD8+ T&nbsp;cells. We first identified differentially expressed genes (DEGs) between day 0 and 20 CAE cells using “cellfishing” (<span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Sato et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Sato, K. ∙ Tsuyuzaki, K. ∙ Shimizu, K. ...</div><div><strong>CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1639-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061352687" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30744683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1639-x&amp;pmid=30744683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and found a strong correlation with our findings using bulk RNA-seq (<a id="crosref0750" href="#figs4">Figure&nbsp;S4</a>G). Next, we performed a nonlinear dimensionality-reduction technique (uniform manifold approximation and projection, UMAP) followed by unsupervised clustering on cells from days 0 (<a id="crosref0755" href="#fig3">Figure&nbsp;3</a>A) and 20 (<a id="crosref0760" href="#fig3">Figure&nbsp;3</a>B). The program identified three distinct clusters on day 0 (D0-1, D0-2, D0-3) and four clusters on day&nbsp;20 (D20-1, D20-2, D20-3, D20-4). Top marker genes were identified for day 20 CAE (<a id="crosref0765" href="#fig3">Figure&nbsp;3</a>C) and day 0 cell clusters (<a id="crosref0770" href="#figs4">Figure&nbsp;S4</a>H). Interestingly, a group of genes upregulated in surCARpos CAE cells identified via bulk genomics (<a id="crosref0775" href="#fig2">Figure&nbsp;2</a>) (<i>KLRC1</i>, <i>SOX4</i>, <i>TNFRSF18</i>, <i>RBPJ</i>, <i>RGS16</i>, <i>CCL3</i>) were found to be top marker genes for single-cell clusters D20-1 and D20-4. Furthermore, gene pathway analysis using all DEGs for each cluster revealed enrichment of the term “natural killer signaling” in day 20 CAE cell clusters D20-1 and D20-4, but not D20-2 and D20-3 clusters or day 0 clusters (<a id="crosref0780" href="#fig3">Figures 3</a>D and <a id="crosref0785" href="#figs4">S4</a>I). Overlap of the top marker genes for each single-cell cluster revealed that genes defining clusters D20-1 and D20-4 significantly overlapped with genes upregulated in day 28 CAE cells via bulk genomics (<a id="crosref0790" href="#figs4">Figure&nbsp;S4</a>J). Thus, D20-1 and D20-4 clusters likely represent a subpopulation of CAE cells consisting of dysfunctional CD8+ T&nbsp;cells that express NK-associated genes. Genes that were highly expressed in day 0 cells (<i>IL7R</i>, <i>LTB</i>, <i>CD48</i>, <i>HLA-DRB1</i>) were top marker genes for clusters D20-2 and D20-3, suggesting that the cells in these clusters have attributes similar to day 0 cells. Of note, clusters D20-1 and D20-3 were highly enriched for cell-cycle regulated pathways (see <a id="crosref0795" href="#fig3">Figure&nbsp;3</a>D).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/92f2db90-9841-4ba6-9b7f-1688f44659c3/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.016/asset/d9d760e5-080d-4611-9300-a8e8ff6ae3f8/main.assets/gr3.jpg" height="942" width="732" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/92f2db90-9841-4ba6-9b7f-1688f44659c3/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Single-cell analysis of CAE CD8+ T&nbsp;cells reveals co-expression of dysfunction signature genes.</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0120" role="paragraph">(A and B) UMAP projection of scRNA-seq data from day 0 product (A) and day 20 CAE cells (B) for donor ND388.</div><div id="fspara0125" role="paragraph">(C) Heatmap of top 10 marker genes for each cluster defined in (B).</div><div id="fspara0130" role="paragraph">(D) Gene ontology determined by metascape pathway analysis for each single-cell cluster from the day 20 CAE sample. Columns are cell clusters (from B) and rows are enriched pathways color-coded by level of significance.</div><div id="fspara0135" role="paragraph">(E) Volcano plot depicting differentially expressed genes between day 20 CAE clusters 1 and 4 (dysfunctional) and clusters 2 and 3 (non-dysfunctional). Genes upregulated in the dysfunctional clusters are on the right side. Red dots indicate significant genes with p&nbsp;&lt; 0.05 and log<sub>2</sub>FC &gt; 0.2.</div><div id="fspara0140" role="paragraph">(F) Dot plot illustrating the expression level of dysfunction signature, naïve/memory, cell cycle, and control genes in day 0 (left) and day 20 CAE (right), donor ND388. Each column represents one cluster as depicted in (A) and (B).</div><div id="fspara0145" role="paragraph">(G) Gene regulatory network analysis (PIDC) for day 20 CAE cells. Columns and rows are the top 500 most variable genes determined by Seurat. Depicted on the right are select genes found within the same community, boxed in red.</div><div id="fspara0150" role="paragraph">(H) Normalized counts of CAR transcripts from scRNA-seq data for day 20 and 28 CAE cells. Pooled cells from dysfunctional and non-dysfunctional clusters from three CAR T donors. Data shown as mean with standard deviation. Significance by Mann-Whitney U test.</div><div id="fspara0155" role="paragraph">(I) Percent of cells that express the CAR transcript in dysfunctional and non-dysfunctional clusters. Average of three CAR T donors. Data shown as mean ±SEM.</div><div id="fspara0160" role="paragraph">See also <a id="crosref1925" href="#figs4">Figures S4</a> and <a id="crosref1930" href="#figs5">S5</a> and <a id="crosref1935" href="#mmc4">Tables S4</a> and <a id="crosref1940" href="#mmc5">S5</a>. ∗∗p&nbsp;&lt; 0.01.</div></div></div></div></figcaption></figure></div><div id="p0135" role="paragraph">We identified all genes specifically expressed in the presumptive dysfunctional clusters (D20-1, D20-4), compared to clusters D20-2 and D20-3 (<a id="crosref0800" href="#fig3">Figure&nbsp;3</a>E; <a id="crosref0805" href="#mmc4">Table S4</a>). Genes with known links to exhaustion, including <i>HAVCR2</i>, <i>ENTPD1</i>, <i>LAYN</i>, <i>CTLA4</i>, <i>PHLDA1</i>, <i>TNFRSF9</i>, <i>NR4A1</i>, <i>PRDM1</i>, and <i>LAG3</i> were upregulated in the dysfunctional clusters (<a id="crosref0810" href="#fig3">Figure&nbsp;3</a>E volcano plot, right side). We then curated an unbiased dysfunction gene signature consisting of the top 30 genes most highly upregulated in day 20 dysfunctional clusters (<a id="crosref0815" href="#fig3">Figure&nbsp;3</a>F), of which 24/30 genes were also upregulated in bulk CAE surCARpos T&nbsp;cells (<a id="crosref0820" href="#fig2">Figure&nbsp;2</a>A). Genes identified exclusively in scRNA-seq included <i>SRGAP3</i>, <i>DUSP4</i>, and <i>CSF1</i>- genes not currently linked to T&nbsp;cell exhaustion (<a id="crosref0825" href="#figs4">Figure&nbsp;S4</a>K). Clusters that emerged that were not dysfunctional (D20-2,D20-3) highly expressed HLA molecules (HLA-DRB1, HLA-DQB1, HLA-DRA, HLA-DPB1) and IL7R, TC2N, and FYB1 (see <a id="crosref0830" href="#fig3">Figure&nbsp;3</a>E, left side).</div><div id="p0140" role="paragraph">We generated dot plots containing the 30 signature genes, as well as naïve/memory markers, cell cycle genes, and control genes (<a id="crosref0835" href="#fig3">Figure&nbsp;3</a>F). Of note, many of the dysfunction signature genes were also present in the gene expression signature described for other models of T&nbsp;cell dysfunction (<a id="crosref0840" href="#mmc5">Table S5</a>). As expected, day 20 CAE cells (<a id="crosref0845" href="#fig3">Figure&nbsp;3</a>F, right) had two cell clusters that highly expressed the dysfunction signature (clusters D20-1, D20-4), while clusters D20-2 and D20-3 and day 0 cell clusters (<a id="crosref0850" href="#fig3">Figure&nbsp;3</a>F, left) did not express this signature. Although not part of our 30-gene signature, <i>CTLA4</i> was upregulated in D20-1 and D20-4 clusters (<a id="crosref0855" href="#figs4">Figure&nbsp;S4</a>L). Select T&nbsp;cell activation genes identified in CD19 CAR T&nbsp;cells (CCL3, CCL4, GZMB, TNFRSF9) (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Boroughs et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Boroughs, A.C. ∙ Larson, R.C. ∙ Marjanovic, N.D. ...</div><div><strong>A Distinct Transcriptional Program in Human CAR T Cells Bearing the 4-1BB Signaling Domain Revealed by scRNA-Seq</strong></div><div><em>Mol. Ther.</em> 2020; <strong>28</strong>:2577-2592</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ymthe.2020.07.023&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ymthe.2020.07.023&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85088996471" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32755564/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymthe.2020.07.023&amp;pmid=32755564" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) are in our 30-gene signature; however, many inhibitory receptors are also T&nbsp;cell activation genes, and their sustained expression is a hallmark feature of T&nbsp;cell exhaustion (<span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Wherry and Kurachi, 2015</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Wherry, E.J. ∙ Kurachi, M.</div><div><strong>Molecular and cellular insights into T&nbsp;cell exhaustion</strong></div><div><em>Nat. Rev. Immunol.</em> 2015; <strong>15</strong>:486-499</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nri3862" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84937837169" target="_blank" aria-label="Open Scopus in new tab">Scopus (2951)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26205583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnri3862&amp;pmid=26205583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0145" role="paragraph">We investigated whether the dysfunction signature genes were co-expressed within the same single cell using an unbiased gene regulatory network analysis (PIDC) (<span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Chan et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Chan, T.E. ∙ Stumpf, M.P.H. ∙ Babtie, A.C.</div><div><strong>Gene Regulatory Network Inference from Single-Cell Data Using Multivariate Information Measures</strong></div><div><em>Cell Syst.</em> 2017; <strong>5</strong>:251-267.e3</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cels.2017.08.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cels.2017.08.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85031092991" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28957658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2017.08.014&amp;pmid=28957658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). One community in day 20 CAE cells included 34 genes that were co-expressed (<a id="crosref0875" href="#fig3">Figure&nbsp;3</a>G; boxed in red). Strikingly, 27/30 of our defined dysfunction signature genes (<a id="crosref0880" href="#fig3">Figure&nbsp;3</a>F) were contained within this community, confirming that these genes were co-expressed in the same subset of cells and that they had a common regulatory network (<a id="crosref0885" href="#fig3">Figure&nbsp;3</a>G).</div><div id="p0150" role="paragraph">Importantly, to confirm our single-cell findings, we performed scRNA-seq in two additional donors (ND538 and ND150) for day 0 and 28 CAE cells and found remarkably similar gene expression signatures, despite these cells being collected at later timepoints of CAE (<a id="crosref0890" href="#figs5">Figures S5</a>A–S5J). Human donors have variability in the number of days required to reach a dysfunctional state; however, most CAR T donors are dysfunctional by 20&nbsp;days of CAE.</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/b98f4ef2-4c2d-4f4d-8146-9a2465470733/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.016/asset/b7a221ac-275d-4693-b226-3395f4709c87/main.assets/figs5.jpg" height="941" width="692" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/b98f4ef2-4c2d-4f4d-8146-9a2465470733/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Validation of scRNA-seq analyses in two additional CAR T donors, related to <a id="crosref2145" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0430" role="paragraph">Single-cell analysis of day 0 product and day 28 CAE cells for donor ND538 <b>(A-E)</b> and donor ND150 <b>(F-J)</b>. <b>(A)</b> UMAP projection of single-cell gene expression data from day 0 cells, made using Seurat. Each dot corresponds to one cell and cell clusters are color coded. <b>(B)</b> UMAP projection of single-cell gene expression data from day 28 CAE T&nbsp;cells, made using Seurat. Each dot corresponds to one cell and cell clusters are color-coded. <b>(C)</b> Heatmap of top 10 marker genes for each day 28 CAE cluster as defined in (B). Columns correspond to cells and rows correspond to gene names. <b>(D)</b> Volcano plot depicting differentially expressed genes between day 28 CAE cluster 1 (dysfunctional) and clusters 2 and 3 (non-dysfunctional), also see (E). Genes upregulated in the dysfunctional cluster are on the right side and genes downregulated are on the left. The x-axis is log<sub>2</sub>(fold change) and y-axis is -log<sub>10</sub>(p value). Red dots indicate significant genes with p&lt;0.05 and log<sub>2</sub>FC &gt;0.5. <b>(E)</b> Dot plot illustrating the expression level of genes in day 0 (left) and day 28 CAE cell clusters (right). Genes included are dysfunction signature genes (N=30), naïve/memory, cell cycle, and control genes. Each column represents one cluster. The size of the circle represents the percent of cells expressing the gene in each cluster and the color depicts how highly expressed the gene is within that cluster. The number of cells in each cluster is written beneath cluster identity. Donor ND150 <b>(F)</b> UMAP projection of single-cell gene expression data from day 0 cells. <b>(G)</b> UMAP projection of single-cell gene expression data from day 28 CAE cells. <b>(H)</b> Heatmap of top 10 marker genes for each day 28 CAE cluster as defined in (G). Columns correspond to cells and rows correspond to gene&nbsp;names. <b>(I)</b> Volcano plot depicting differentially expressed genes between day 28 cluster 3 (non-dysfunctional) and clusters 1,2,4, and 5 (dysfunctional), also see&nbsp;(J). Genes upregulated in the dysfunctional clusters are on the right side and genes downregulated are on the left. The x axis is log<sub>2</sub> (fold change) and y axis is -log<sub>10</sub>(p value). Red dots indicate significant genes with p&lt;0.05 and log<sub>2</sub>FC &gt;0.5. <b>(J)</b> Dot plot illustrating the expression level of genes in day 0 (left) and day 28 CAE (right). Genes included are dysfunction signature genes (N=30), naïve/memory, cell cycle and control genes. Each column represents one cluster. The number of cells in each cluster is written beneath cluster identity.</div></div></div></div></figcaption></figure></div><div id="p0155" role="paragraph">Given that CAE results in dysfunctional CAR T&nbsp;cells with reduced effector function, we next asked whether we could detect CAR transcripts in our single-cell datasets, and if so, whether cells that express the CAR are preferentially expressed in the dysfunctional cell clusters. We found that the dysfunctional cell clusters expressed significantly more CAR (<a id="crosref0895" href="#fig3">Figure&nbsp;3</a>H) and had a higher percentage of cells overall that expressed the CAR (<a id="crosref0900" href="#fig3">Figure&nbsp;3</a>I).</div></section><section id="sec-2-5"><h3>Mass and flow cytometry profiling reveals NK-like phenotype of CD8+ CAR T&nbsp;cells under CAE</h3><div id="p0160" role="paragraph">Next, we examined expression of NK-associated proteins by flow cytometry on surCARpos and surCARneg CD8+ T&nbsp;cells throughout CAE. CD8+ CAR T&nbsp;cells did not express high levels of NK-associated molecules and exhaustion markers before CAE, but exhibited increased expression after CAE with concurrent loss of CD28 (<a id="crosref0905" href="#fig4">Figure&nbsp;4</a>A). While most NK receptors increased over time, NKG2C was expressed early, followed by a rapid decline in expression during CAE. Importantly, we could not identify invariant NKT cells (<a id="crosref0910" href="#fig4">Figure&nbsp;4</a>A), suggesting that NK-like T&nbsp;cells identified in this model need to be separately classified from iNKT cells (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Godfrey et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Godfrey, D.I. ∙ MacDonald, H.R. ∙ Kronenberg, M. ...</div><div><strong>NKT cells: what’s in a name?</strong></div><div><em>Nat. Rev. Immunol.</em> 2004; <strong>4</strong>:231-237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nri1309" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15039760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnri1309&amp;pmid=15039760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/65dc4b4b-ec84-49ed-b5d5-144c660ac7b3/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.016/asset/acbe6176-62a4-4e69-8020-45b91556ee12/main.assets/gr4.jpg" height="941" width="726" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/65dc4b4b-ec84-49ed-b5d5-144c660ac7b3/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text"><i>In&nbsp;vivo</i> relevance of CAR and TCR T&nbsp;cell dysfunction signature and the NK-like phenotype</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0170" role="paragraph">(A) Time-related changes in NK-associated molecules and PD-1 and CD28 on surCARpos and surCARneg CD8+ T&nbsp;cells during CAE. iNKT are defined as cells with Vα24-Jα18-specific TCRs. Data from ND150 are shown.</div><div id="fspara0175" role="paragraph">(B) Experimental design of the recurrent AsPC-1 mouse model.</div><div id="fspara0180" role="paragraph">(C) AsPC-1 tumor growth volumes in M5CAR T-treated mice. Red arrows indicate tumors analyzed after recurrence.</div><div id="fspara0185" role="paragraph">(D) NK-associated molecules expression in CD8&nbsp;day 0 product (top) and TILs from a representative AsPC-1 recurrent tumor (bottom).</div><div id="fspara0190" role="paragraph">(E) Average expression of NK-associated molecules on CD8 T&nbsp;cells in day 0 product and in three recurrent tumors. Each datapoint represents a single mouse for recurrent tumor data and a single technical replicate staining for day 0 product. Color code for mice data is matched with <a id="crosref1945" href="#fig4">Figure&nbsp;4</a>C.</div><div id="fspara0195" role="paragraph">(F) PD-1, LAG3, and TIM3 expression in CD8&nbsp;day 0 product (top) and TILs from a representative AsPC-1 recurrent tumor (bottom).</div><div id="fspara0200" role="paragraph">(G) Average expression of checkpoint receptors PD-1, LAG3, and TIM3 in CD8 T&nbsp;cells. Each datapoint represents a single mouse for recurrent tumor data and a single technical replicate staining for day 0 product. Color code for mice data is matched with <a id="crosref1950" href="#fig4">Figure&nbsp;4</a>C.</div><div id="fspara0205" role="paragraph">(H) CD56 expression in CD8+ surCARpos T&nbsp;cells isolated from DLBCL patients at the peak of CTL019 expansion.</div><div id="fspara0210" role="paragraph">(I) Expression of NK-associated molecules and PD-1 on CD8+ surCARpos T&nbsp;cells in day 0 product and day 27 peripheral blood T&nbsp;cells from a patient with DLBCL (#13413-39).</div><div id="fspara0215" role="paragraph">(J) Timeline showing the experimental design of NY-ESO-1 TIL mouse model.</div><div id="fspara0220" role="paragraph">(K) Heatmap of dysfunction signature genes in NY-ESO-1 reactive CD8+ TILs along with blood (CD8+CD45RO+ T&nbsp;cells) and day 0 infused product.</div><div id="fspara0225" role="paragraph">See also <a id="crosref1955" href="#figs6">Figure&nbsp;S6</a>. Data from (E) and (G) are shown as mean ± SEM, and significance was assessed by two-way ANOVA plus Sidak test. ∗∗∗∗p&nbsp;&lt; 0.0001, ∗∗∗p&nbsp;&lt; 0.001, ∗∗p&nbsp;&lt; 0.01, n.s.: not significant.</div></div></div></div></figcaption></figure></div><div id="p0165" role="paragraph">We performed an NK focused cytometry by time-of-flight (CyTOF or mass cytometry) to explore how the dysfunction signature identified by scRNA-seq relates to protein expression levels on CAR T&nbsp;cells. t-distributed stochastic neighbor embedding (t-SNE) plots revealed twenty subpopulations of CD8+ T&nbsp;cells, where CAE CAR T&nbsp;cells had markedly different clusters compared to day 0 product (<a id="crosref0920" href="#figs6">Figure&nbsp;S6</a>A, red circle denotes cell populations more abundant in CAE T&nbsp;cells). Notably, many NK receptors and NK-related proteins were increased in the CAE-specific clusters, including the inhibitory receptors (KLRB1, TIGIT, NKG2A, PD-1) and NK-related proteins CD56 and granulysin (<a id="crosref0925" href="#figs6">Figure&nbsp;S6</a>B). The mass cytometry data closely aligned with our flow cytometry profiling of CD8+ CAR T&nbsp;cells under CAE as shown in <a id="crosref0930" href="#fig4">Figure&nbsp;4</a>A. The various subpopulations identified in the CAE cells revealed that the NK-like phenotype was heterogeneous. There were two distinct subpopulations of cells that expressed CD56, one group that was KLRB1+ and another group that was KLRB1−. In agreement with our genomics data (see <a id="crosref0935" href="#fig2">Figure&nbsp;2</a>B), NK-like phenotypes emerged in both surCARpos and surCARneg cells (<a id="crosref0940" href="#fig4">Figures 4</a>A and <a id="crosref0945" href="#figs6">S6</a>B). Overall, these data suggest that a subset of day 0 CD8+ T&nbsp;cells dynamically evolve into NK-like T&nbsp;cells with a distinct phenotype marked by KLRB1 and/or CD56 expression.</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/4861d17a-ca65-489d-8a7d-5e346ed66dc1/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.016/asset/5df5355f-7131-4595-ad45-4ec8096d258b/main.assets/figs6.jpg" height="872" width="657" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/4861d17a-ca65-489d-8a7d-5e346ed66dc1/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">NK-like-T cell phenotype in dysfunctional CAR T&nbsp;cells and validation of dysfunction signature in <i>in&nbsp;vivo</i> models, related to <a id="crosref2150" href="#fig4">Figures 4</a>, <a id="crosref2155" href="#fig5">5</a>, and <a id="crosref2160" href="#fig6">6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0440" role="paragraph"><b>(A)</b> Phenotypical change between day 0 products (left) and day 29 CAE samples (right) profiled by mass cytometry using a NK flow panel. Data from 2 donors (ND150 and ND538) are shown. <b>(B)</b> Expression of surface M5CAR and NK-associated molecules (CD161, TIGIT, CD56, NKG2A, NKG2C) and granulysin on day 0 product (top) and day 29 CAE CD8+ T&nbsp;cells (bottom). Red circles highlight subpopulations of CD8+ T&nbsp;cells more abundant under CAE<b>. (C)</b> Representative plot showing mesothelin expression from an AsPC-1 recurrent tumor associated with <a id="crosref2165" href="#fig4">Figure&nbsp;4</a>B, and the FMO control (right). <b>(D)</b> Frequency of CD8+ T&nbsp;cells infiltrating recurrent tumors (n=3 mice). Data is shown as mean ± SEM. <b>(E)</b> Representative dot plot of surface CAR (sCAR) and intracellular (iCAR) staining (left) and FMO controls (middle). Percentage of CD8 T&nbsp;cells expressing surface and intracellular CAR isolated from recurrent tumors from <a id="crosref2170" href="#fig4">Figure&nbsp;4</a>B (n=3), (right). <b>(F)</b> Model for CD56 depletion assay. Expected percentage of NK-like T&nbsp;cells (CD56+, y-axis) and time (x-axis) during continuous antigen stimulation for out competition model (left) and transition model (right). Control cells start with regular CAR T&nbsp;cell population at day 0 and CD56-depleted starts with day 0 CAR T&nbsp;cells depleted of CD56. <b>(G)</b> TCR single-cell sequencing data for day 0 and day 28 CAE for donors ND150 and ND538. Y axis is the percentage of CDR3 sequences, and x-axis is the number of cells that have that CDR3 sequence. Illustrates that between 96-99% of CDR3 sequences are unique. <b>(H)</b> Single-cell transcript levels of FOS, EGR1, FOXP3, MYB, LEF1, CREM, TWIST1 illustrated by UMAP plots, corresponding to clusters from <a id="crosref2175" href="#fig3">Figure&nbsp;3</a>B (day 20 CAE cells). Top two clusters are dysfunctional. <b>(I)</b> Transcription factor motif for SOX4 from Jaspar database (top) and SOX4 and SOX17 motifs from UniProt database (bottom).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-6"><h3><i>In&nbsp;vivo</i> NK receptor upregulation and dysfunction signature gene expression in CAR T&nbsp;cells and TILs</h3><div id="p0170" role="paragraph">Our observations above of upregulation of NK molecules on CD8+ CAR T&nbsp;cells <i>in&nbsp;vitro</i> during CAE prompted us to test whether this expansion occurs <i>in&nbsp;vivo</i>. AsPC-1 tumors were established in mice and M5CAR T&nbsp;cells were able to eliminate large mesothelin-expressing flank tumors within 2&nbsp;weeks after CAR T injection (<a id="crosref0950" href="#fig4">Figures 4</a>B and 4C). However, 2 to 4&nbsp;months after initial injection of the CAR T&nbsp;cells, several of the mice relapsed. We analyzed the recurrent tumors and found that the mechanism of tumor relapse was not due to loss of the mesothelin target antigen (<a id="crosref0955" href="#figs6">Figure&nbsp;S6</a>C). Therefore, we analyzed the infiltrating human T&nbsp;cells in the relapsed tumors and found that nearly all the infiltrating T&nbsp;cells were CD8+ CAR T&nbsp;cells (<a id="crosref0960" href="#figs6">Figures S6</a>D and S6E). Intriguingly, the CAR T&nbsp;cells from the recurrent tumors expressed our dysfunction signature with high levels of NK receptors (<a id="crosref0965" href="#fig4">Figures 4</a>D and 4E) and checkpoint receptors (<a id="crosref0970" href="#fig4">Figures 4</a>F and 4G), unlike the day 0 CAR T product. Further, since the tumors were progressing without losing mesothelin expression, we can be confident that the T&nbsp;cells had lost the ability to control the tumor and are thus dysfunctional.</div><div id="p0175" role="paragraph">This finding prompted us to test whether this expansion occurs in patients undergoing CAR T therapy. Diffuse large B-cell lymphoma (DLBCL) patients treated with CD19-directed CAR T&nbsp;cells (CTL019) were retrospectively assessed in a clinical trial (NCT02030834) to determine whether any of their circulating CAR T&nbsp;cells exhibited NK-like features. Three of seventeen analyzed DLBCL patients exhibited greater than 5% expansion of the CAR+ NK-like T&nbsp;cell population as early as 10&nbsp;days post-CAR T infusion of a CD19-directed CAR, and other patients showed detectable expansion (<a id="crosref0975" href="#fig4">Figure&nbsp;4</a>H). Notably, the patient with the highest level of NK-like CAR T&nbsp;cells (13413-39) had progressive tumor and failed to respond to the therapy (<span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Schuster et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Schuster, S.J. ∙ Svoboda, J. ∙ Chong, E.A. ...</div><div><strong>Chimeric Antigen Receptor T Cells in Refractory B-Cell Lymphomas</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2017; <strong>377</strong>:2545-2554</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1708566" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85040171486" target="_blank" aria-label="Open Scopus in new tab">Scopus (1380)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29226764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1708566&amp;pmid=29226764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). There was sufficient material from patient 13413-39 to analyze additional NK markers in CAR+ T&nbsp;cells. The percentage of NK-like T&nbsp;cells in the day 0 CAR T product was low, but the NK-like CD8+ T&nbsp;cell phenotype was upregulated at day 27 post-CAR T infusion as determined by increased levels of NKG2A, CD94, and CD56 (<a id="crosref0985" href="#fig4">Figure&nbsp;4</a>I). We did not detect increased KLRB1 levels; however, this could be explained by the late expression of this marker upon CAE (<a id="crosref0990" href="#fig4">Figure&nbsp;4</a>A). In conclusion, these data provide evidence for the acquisition of an NK-like CAR T&nbsp;cell phenotype in some CAR T&nbsp;cell patients.</div><div id="p0180" role="paragraph">To determine whether the CAR T dysfunction signature is CAR-specific or more broadly applicable to T&nbsp;cells chronically exposed to antigen, we generated lung tumors that expressed the antigen NY-ESO-1 in a xenograft mouse model, and then injected human T&nbsp;cells specifically engineered to express NY-ESO-1-reactive Ly95 TCR into the tumor (<a id="crosref0995" href="#fig4">Figure&nbsp;4</a>J). This generates hypofunctional Ly95 TILs that are unable to eradicate tumor (<span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Moon et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Moon, E.K. ∙ Ranganathan, R. ∙ Eruslanov, E. ...</div><div><strong>Blockade of Programmed Death 1 Augments the Ability of Human T Cells Engineered to Target NY-ESO-1 to Control Tumor Growth after Adoptive Transfer</strong></div><div><em>Clin. Cancer Res.</em> 2016; <strong>22</strong>:436-447</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-15-1070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84959036332" target="_blank" aria-label="Open Scopus in new tab">Scopus (105)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26324743/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-15-1070&amp;pmid=26324743" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our dysfunction gene signature was expressed at a low level in the infused product and blood CD8+ T&nbsp;cells, but strikingly, 28/30 of the exhaustion and NK signature genes were upregulated in the NY-ESO-1-reactive TCR TILs, including the transcription factors ID3 and SOX4 (<a id="crosref1005" href="#fig4">Figure&nbsp;4</a>K).</div></section><section id="sec-2-7"><h3>Transition of CD8+ T&nbsp;cells to NK-like T&nbsp;cells upon continuous antigen stimulation</h3><div id="p0185" role="paragraph">NK-like T&nbsp;cells have been shown to express both T&nbsp;cell and NK cell markers and are frequently defined as CD3+CD56+ or CD3+KLRB1+, and they often express KLRC1 (<span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Barbarin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Barbarin, A. ∙ Cayssials, E. ∙ Jacomet, F. ...</div><div><strong>Phenotype of NK-Like CD8(+) T Cells with Innate Features in Humans and Their Relevance in Cancer Diseases</strong></div><div><em>Front. Immunol.</em> 2017; <strong>8</strong>:316</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2017.00316" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85017100637" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28396661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2017.00316&amp;pmid=28396661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Kurioka et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Kurioka, A. ∙ Klenerman, P. ∙ Willberg, C.B.</div><div><strong>Innate-like CD8+ T-cells and NK cells: converging functions and phenotypes</strong></div><div><em>Immunology.</em> 2018; <strong>154</strong>:547-556</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/imm.12925" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85050148100" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fimm.12925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). UMAP plots of scRNA-seq day 0 versus day 20 CAE cells showed enrichment of cells that co-express <i>CD3</i>, <i>KLRB1</i>, and <i>KLRC1</i> (<a id="crosref1020" href="#fig5">Figure&nbsp;5</a>A, related to UMAPs in <a id="crosref1025" href="#fig3">Figures 3</a>A and 3B). In addition, flow cytometry analysis using two separate markers for NK-like T&nbsp;cells (CD3+CD56+ and CD3+KLRB1+) revealed a robust expansion of this NK-like T&nbsp;cell population during CAE (<a id="crosref1030" href="#fig5">Figure&nbsp;5</a>B).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/e3ce779e-2096-4093-a3c7-5e14eedfe7a3/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.016/asset/6e3f535d-b895-4b5e-b769-522f17500fc4/main.assets/gr5.jpg" height="933" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/e3ce779e-2096-4093-a3c7-5e14eedfe7a3/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Transition of CD8+ T&nbsp;cells to NK-like T&nbsp;cells upon continuous antigen stimulation</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0235" role="paragraph">(A) NK-like T&nbsp;cell population (CD3+, KLRB1+, and KLRC1) at day 0 (left) and day 20 CAE (right) overlayed on UMAP graphs from <a id="crosref1960" href="#fig3">Figures 3</a>A and 3B.</div><div id="fspara0240" role="paragraph">(B) Identification of NK-like T&nbsp;cell populations (CD56+ CD3+ and CD3+ KLRB1) during CAE.</div><div id="fspara0245" role="paragraph">(C) On left, NK-like T&nbsp;cell frequency (CD3+CD56+) at day 0 and following CD56 depletion. On right, NK-like T&nbsp;cell frequency (CD3+CD56+) with or without CD56 depletion during CAE. Data representative of two donors are shown as mean ± SEM.</div><div id="fspara0250" role="paragraph">(D) Single-cell TCR fingerprinting&nbsp;+ gene expression analysis in ND150 (left) and ND538 (right). Results are filtered for CD8+ T&nbsp;cells that have the same CDR3 TCR sequence at day 0 and at day 28. Cells were classified as either KLRB1 negative or positive at day 0 and at day 28, and total number of cells in each category is depicted.</div><div id="fspara0255" role="paragraph">(E) Monocle trajectory analysis of ND388&nbsp;day 20 CAE cells, with single-cell clusters labeled according to their defined clusters in <a id="crosref1965" href="#fig3">Figure&nbsp;3</a>B (left). On right, same monocle trajectory but with cells labeled according to expression of the dysfunction gene signature (N&nbsp;= 30 genes, see <a id="crosref1970" href="#fig3">Figure&nbsp;3</a>F).</div><div id="fspara0260" role="paragraph">(F) Monocle trajectory analysis of ND150 and ND538&nbsp;day 0 and day 28 CAE cells combined, corresponding to <a id="crosref1975" href="#figs5">Figure&nbsp;S5</a>. Cells are labeled according to sample ID (left) or by how highly each cell expresses the dysfunction signature genes (right).</div><div id="fspara0265" role="paragraph">See also <a id="crosref1980" href="#figs6">Figure&nbsp;S6</a>.</div></div></div></div></figcaption></figure></div><div id="p0190" role="paragraph">Our findings overall demonstrate expansion of an NK-like T&nbsp;cell population upon CAE; however, it is unclear whether these are clonally expanded cells from an NK-like T population existing at day 0, or, in contrast, whether CD8+ T&nbsp;cells acquire NK receptors via plasticity during prolonged antigen exposure. To test this in our <i>in&nbsp;vitro</i> model of CAR T&nbsp;cell dysfunction, we depleted the CD56+ cells from the input day 0 population and repeated the CAE experiment. We note that CD56 is the most frequently used marker to identify human NK and NK-like T&nbsp;cells, and hence CD56 depletion is expected to remove both populations from the day 0 product (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Barbarin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Barbarin, A. ∙ Cayssials, E. ∙ Jacomet, F. ...</div><div><strong>Phenotype of NK-Like CD8(+) T Cells with Innate Features in Humans and Their Relevance in Cancer Diseases</strong></div><div><em>Front. Immunol.</em> 2017; <strong>8</strong>:316</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2017.00316" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85017100637" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28396661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2017.00316&amp;pmid=28396661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Seyda et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Seyda, M. ∙ Elkhal, A. ∙ Quante, M. ...</div><div><strong>T Cells Going Innate</strong></div><div><em>Trends Immunol.</em> 2016; <strong>37</strong>:546-556</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2016.06.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2016.06.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84991745789" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27402226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2016.06.004&amp;pmid=27402226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At day 0, the percentage of NK-like T&nbsp;cells was very low (0.69%–2.23%, <a id="crosref1045" href="#fig5">Figures 5</a>A and 5B, left). Strikingly, CD56<sup>+</sup> depletion had no effect on the percent of NK-like T&nbsp;cells that emerged upon CAE (<a id="crosref1050" href="#fig5">Figure&nbsp;5</a>C, right; also see theoretical model, <a id="crosref1055" href="#figs6">Figure&nbsp;S6</a>F), consistent with transition of CD8+ T&nbsp;cells to NK-like T&nbsp;cells rather than expansion.</div><div id="p0195" role="paragraph">To confirm the T&nbsp;cell to NK-like CAR T&nbsp;cell transition, we performed scRNA-seq alongside lineage tracing using TCR sequencing at day 0 and 28 CAE (<a id="crosref1060" href="#fig5">Figure&nbsp;5</a>D), reasoning that the specific TCR allele would be the same after transition. We filtered for CD8+ cells with TCRs in common between days 0 and 28 (<a id="crosref1065" href="#fig5">Figure&nbsp;5</a>D, left). Of these, 36 were KLRB1− at day 0 and by day 28, 17/36 (47%) transitioned to KLRB1+. This was validated independently using another CAR T donor (<a id="crosref1070" href="#fig5">Figure&nbsp;5</a>D, right). These results confirm that the NK-like T&nbsp;cells are undergoing transition, and not simply expanding. We note that 96%–99% of the TCRs were unique in each sample, providing additional evidence against clonal expansion in our <i>in&nbsp;vitro</i> model (<a id="crosref1075" href="#figs6">Figure&nbsp;S6</a>G).</div><div id="p0200" role="paragraph">To model the changes in transcription that occur as CD8+ T&nbsp;cells transition to NK-like T&nbsp;cells, we performed pseudotime analysis, which showed that day 20 CAE clusters (D20-2, D20-3) separated from dysfunctional clusters (D20-1, D20-4), with transcriptional progression from D20-3, D20-2, and D20-4 to the D20-1 cluster (<a id="crosref1080" href="#fig5">Figure&nbsp;5</a>E, left). Consistent with this progression, cells expressing the dysfunction signature (see <a id="crosref1085" href="#fig3">Figure&nbsp;3</a>F, N&nbsp;= 30 genes) prominently occupied the end of the trajectory (<a id="crosref1090" href="#fig5">Figure&nbsp;5</a>E, right). We used two additional donors to validate these findings and importantly, combined day 0 and 28 CAE samples from both donors together for pseudotime analysis. As expected, day 0 samples clustered together (red and blue cells) on the right side of the trajectory, while day 28 samples (green and purple) clustered together on the left (<a id="crosref1095" href="#fig5">Figure&nbsp;5</a>F, left). Furthermore, cells expressing the highest level of dysfunction signature genes (green) clustered on the left side of the trajectory with day 28 CAE cells (<a id="crosref1100" href="#fig5">Figure&nbsp;5</a>F, right). Taken together, our dysfunction signature genes associate with transitioned NK-like T&nbsp;cells.</div></section><section id="sec-2-8"><h3>ID3 and SOX4 are potential regulators of the dysfunction signature</h3><div id="p0205" role="paragraph">Identification of a common transcription factor(s) that controls this CAR T dysfunction signature and NK-like T&nbsp;cell transition could provide an approach to prevent and/or reverse loss of effector function. DEGs identified in our scRNA-seq datasets between day 0 and 20 CAE cells were analyzed by IPA to identify potential transcription factors that regulate the signature. All transcription factors highlighted in the bulk RNA-seq experiment (<a id="crosref1105" href="#fig2">Figure&nbsp;2</a>F) were also regulators of the single-cell signature and some, but not all, were themselves differentially expressed in the single-cell dataset (FC indicated to the right; <a id="crosref1110" href="#fig6">Figure&nbsp;6</a>A; <a id="crosref1115" href="#mmc6">Table S6</a>). Importantly, <i>ID3</i> and <i>SOX4</i> were specifically expressed in the dysfunction clusters (<a id="crosref1120" href="#fig6">Figures 6</a>B and 6C; related to <a id="crosref1125" href="#fig3">Figure&nbsp;3</a>B), while other transcription factors, with the possible exception of TWIST1 that was expressed at low levels, lacked specificity or had less dramatic changes between dysfunctional and non-dysfunctional clusters (<a id="crosref1130" href="#fig6">Figures 6</a>D and <a id="crosref1135" href="#figs6">S6</a>H). Consistently, <i>ID3</i> and <i>SOX4</i> were co-expressed with the other dysfunction signature genes in CAE T&nbsp;cells (see <a id="crosref1140" href="#fig3">Figure&nbsp;3</a>G), suggesting that these transcription factors may help to orchestrate the dysregulated gene expression signature.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/dce210f6-685b-4a7d-9f7c-fb4fa917bc9a/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.016/asset/34554139-10e0-4f29-a08e-9ca403e4bd3b/main.assets/gr6.jpg" height="902" width="753" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/dce210f6-685b-4a7d-9f7c-fb4fa917bc9a/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">ID3 and SOX4 are potential regulators of the dysfunction signature</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0275" role="paragraph">(A) Select transcription factors predicted to regulate differentially expressed genes between day 0 and day 20 CAE cells in single-cell sequencing datasets, identified using IPA upstream regulator analysis. Depicted are transcription factors that overlap with factors from <a id="crosref1985" href="#fig2">Figure&nbsp;2</a>F. On right, gene expression log<sub>2</sub> FC (day 20 CAE/day 0) for each transcription factor. NA depicts genes that are not differentially expressed between day 0 and day 20 cells.</div><div id="fspara0280" role="paragraph">(B) UMAP plots from <a id="crosref1990" href="#fig3">Figure&nbsp;3</a>B showing single-cell transcript levels of ID3 and SOX4 on day 20 CAE cells. Top two clusters are dysfunctional.</div><div id="fspara0285" role="paragraph">(C) Violin plots depicting gene expression levels for ID3 and SOX4 for each cluster from day 20 CAE cells (see <a id="crosref1995" href="#fig3">Figure&nbsp;3</a>B).</div><div id="fspara0290" role="paragraph">(D) Single-cell transcript levels of CDKN2A, BCL6, RBPJ, ID2, and KLF2 illustrated by UMAP plots, corresponding to clusters from <a id="crosref2000" href="#fig3">Figure&nbsp;3</a>B (day 20 CAE cells).</div><div id="fspara0295" role="paragraph">(E) HOMER motif analysis depicting top 10 enriched transcription factor motifs in bulk ATAC-seq dataset for day 0 samples (left) and day 28 samples (right). Analysis includes four biological replicates.</div><div id="fspara0300" role="paragraph">(F) Box plots illustrating the ATAC-seq signal at unchanged peaks (left) and peaks that change between day 0 and day 28 (right). Data are further subdivided depending on whether a SOX4 motif is present. Statistics assessed by Mann-Whitney U test.</div><div id="fspara0305" role="paragraph">(G–I) ATAC-seq tracks in regulatory regions at SOX4 motifs from day 0 and 28 CAE samples at dysfunction genes <i>AFAP1L2</i> (G), <i>CDK6</i> (H), and <i>CSF1</i> (I). SOX4 motifs labeled with red bars above tracks. Analysis includes four biological replicates.</div><div id="fspara0310" role="paragraph">See also <a id="crosref2005" href="#figs6">Figures S6</a> and <a id="crosref2010" href="#figs7">S7</a> and <a id="crosref2015" href="#mmc6">Table S6</a>.</div></div></div></div></figcaption></figure></div><div id="p0210" role="paragraph">ID3 is a member of a family of helix-loop-helix transcription factors that do not bind DNA directly, but rather inhibit other transcription factors from binding DNA (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Benezra et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Benezra, R. ∙ Davis, R.L. ∙ Lockshon, D. ...</div><div><strong>The protein Id: a negative regulator of helix-loop-helix DNA binding proteins</strong></div><div><em>Cell.</em> 1990; <strong>61</strong>:49-59</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2F0092-8674%2890%2990214-Y&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2F0092-8674%2890%2990214-Y&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-0025238437" target="_blank" aria-label="Open Scopus in new tab">Scopus (2050)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2156629/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2890%2990214-Y&amp;pmid=2156629" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and thus, ID3 lacks a specific DNA-binding motif. However, SOX4, a member of the SRY-related HMG-box family, has a known DNA motif (<span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Fornes et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Fornes, O. ∙ Castro-Mondragon, J.A. ∙ Khan, A. ...</div><div><strong>JASPAR 2020: update of the open-access database of transcription factor binding profiles</strong></div><div><em>Nucleic Acids Res.</em> 2020; <strong>48</strong>:D87-D92</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31701148/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31701148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">UniProt Consortium., 2019</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>UniProt Consortium</div><div><strong>UniProt: a worldwide hub of protein knowledge</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D506-D515</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85059797762" target="_blank" aria-label="Open Scopus in new tab">Scopus (5045)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30395287/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1049&amp;pmid=30395287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We identified top transcription factor motifs enriched in day 0 samples (left) and day 28 samples (right) using our bulk ATAC-seq datasets (<a id="crosref1160" href="#fig6">Figure&nbsp;6</a>E). Day-28-specific peaks were enriched for the SOX17 motif, which is identical to the SOX4 motif (<a id="crosref1165" href="#figs6">Figure&nbsp;S6</a>I), whereas day 0 peaks displayed no SOX enrichment. Day-28-specific ATAC-seq peaks with a SOX4 motif displayed increased ATAC-seq signal (p&nbsp;= 7.9e-07) compared to ATAC-seq peaks that lacked a SOX4 motif, while day 0 samples showed no significant difference (p&nbsp;= .09) (<a id="crosref1170" href="#fig6">Figure&nbsp;6</a>F, right). We note that ATAC-seq peaks that did not change between day 0 and 28 (<a id="crosref1175" href="#fig6">Figure&nbsp;6</a>F, left, unchanged peaks) showed no specific enrichment for SOX4 motifs. Further, 18/30 of our dysfunction signature genes had chromatin opening at SOX4 motifs in day 28 CAE cells—including <i>AFAP1L2</i>, <i>CDK6</i>, and <i>CSF1</i> (<a id="crosref1180" href="#fig6">Figures 6</a>G–6I), and NK receptor genes <i>KLRC1</i> and <i>KLRB1</i> (<a id="crosref1185" href="#figs7">Figures S7</a>A and S7B). Our results indicate that CAR T&nbsp;cells develop an opening of chromatin at SOX4 sites upon CAE.</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/6741ca22-e60f-4dbd-a345-be84f3011308/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.016/asset/36f14f38-52a6-4acb-b344-77e67fe9e02e/main.assets/figs7.jpg" height="852" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/6741ca22-e60f-4dbd-a345-be84f3011308/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Chromatin changes at NK receptor genes in dysfunctional CAR T&nbsp;cells, and ID3 and SOX4 KO CAR T&nbsp;cells have improved CAR T effector function, related to <a id="crosref2180" href="#fig6">Figures 6</a> and <a id="crosref2185" href="#fig7">7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0450" role="paragraph"><b>(A-B)</b> Representative ATAC-seq tracks in regulatory regions at SOX4 motifs from day 0 and day 28 CAE samples at NK receptor and dysfunction genes <i>KLRC1</i><b>(A)</b> and <i>KLRB1</i><b>(B)</b>. SOX4 motifs labeled with red bars above tracks. Analysis includes four biological replicates. <b>(C)</b> Agarose gel (top) and KO efficiency by genomic DNA sequencing (bottom) showing CRISPR edits on KO T&nbsp;cells. ID3-specific and SOX4-specific PCR targeting the edited region of each transcription factor shows the appearance of two bands in KO UTD and KO M5CAR T&nbsp;cells, corresponding to two edited populations derived from different sgRNA hits as depicted in <a id="crosref2190" href="#fig7">Figure&nbsp;7</a>A. The KO efficiency quantification from two donors (ND566 and ND539). WT: wildtype, UTD: non-electroporated un-transduced T&nbsp;cells (no M5CAR), M5: T&nbsp;cells electroporated with M5CAR. Data is shown as mean ± SEM from two CAR T donors. <b>(D)</b> Cytotoxicity assessment of day 0 products at 1:1 (left) and 1:10 (right) E:T ratio on ND539 (top) and ND566 (bottom) in WT, ID3 KO, and SOX4 KO M5CAR T&nbsp;cells. Media used as a control. Data is shown as mean ± SEM. <b>(E)</b> Flow cytometry characterization of naïve, central memory, effector memory and effector subsets on day 0 product with CD45RO and CCR7 expression for WT, ID3 KO, and SOX4 KO M5CAR T&nbsp;cells for donor ND566 (left) and ND539 (right). <b>(F)</b> Percentage of NK-like T&nbsp;cells in WT and ID3 KO cells, relative to WT (donor ND539). Significance measured by Fisher’s exact test. <b>(G)</b><i>In&nbsp;vitro</i> killing assay of ND539 WT, ID3 KO, and SOX4 KO M5CAR T&nbsp;cells. Cells were collected on day 18 of CAE and seeded at 1:8 E:T ratio with AsPC-1 on day 18. WT day 0 cells are used as a positive control and media is used as a negative control. Data is shown as mean ± SEM. <b>(H)</b><i>In&nbsp;vitro</i> killing assay of ND566 WT, ID3 KO, and SOX4 KO M5CAR T&nbsp;cells. Cells were collected on day 21 of CAE and seeded at 1:8 E:T ratio. Data is shown as mean ± SEM. <b>(I)</b><i>In&nbsp;vitro</i> killing assay of ND566 WT, ID3 KO, and SOX4 KO M5CAR T&nbsp;cells. Cells were collected on day 28 of CAE and seeded at 1:8 E:T ratio. Data is shown as mean ± SEM. ∗∗p&nbsp;&lt; 0.01.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-9"><h3>Disruption of ID3 and SOX4 improves CAR T effector function</h3><div id="p0215" role="paragraph">To investigate whether ID3 and SOX4 regulate the dysfunction signature and T-to-NK-like T transition, as well as drive CAR T dysfunction, we generated ID3 and SOX4 KO CAR T&nbsp;cells using CRISPR-Cas9 (<a id="crosref1190" href="#fig7">Figures 7</a>A and <a id="crosref1195" href="#figs7">S7</a>C). We validated the efficiency of KO cells in the day 0 product (<a id="crosref1200" href="#figs7">Figure&nbsp;S7</a>C). No differences in cytotoxicity (<a id="crosref1205" href="#figs7">Figure&nbsp;S7</a>D) or T&nbsp;cell subset distribution (naïve, effector, and memory populations) were observed at baseline between WT and KO day 0 CAR T&nbsp;cells (<a id="crosref1210" href="#figs7">Figure&nbsp;S7</a>E); however, as expected, there were minor differences in T&nbsp;cell subsets between the CAR T donors.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.016/asset/44e130bc-ba94-4348-8679-54b729f0455c/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.016/asset/2c9ef0f3-2156-468a-8fe6-e6eec9176d3e/main.assets/gr7.jpg" height="941" width="717" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/44e130bc-ba94-4348-8679-54b729f0455c/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Disruption of ID3 and SOX4 improves CAR T effector function</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0320" role="paragraph">(A) Schematic representation of the CRISPR strategy to generate ID3 and SOX4 KO M5CAR T&nbsp;cells.</div><div id="fspara0325" role="paragraph">(B) Experimental design for WT, ID3 KO, and SOX4 KO analyses for donors ND566 and ND539.</div><div id="fspara0330" role="paragraph">(C) Agarose gel showing ID3 and SOX4 KO detection on cDNA from CD8 sorted populations after CAE for donor ND566. <i>ID3</i>: ID3 PCR; <i>SOX4</i>: SOX4 PCR; <i>Positive Control</i>: histone H3.3; <i>WT</i>: Mock M5CAR; <i>W</i>: water negative control; <i>KO</i>: ID3 KO (in ID3 PCR) and SOX4 KO (in SOX4 PCR).</div><div id="fspara0335" role="paragraph">(D) KO quantification of ID3 (ND566 and ND539) and SOX4 (ND566) by cDNA sequencing. Percent indels and fragment deletions upon CAE are shown as mean with standard deviation.</div><div id="fspara0340" role="paragraph">(E) UMAP projection of scRNA-seq data from sorted CD8+ WT, ID3 KO, or SOX4 KO day 24 CAE cells for donor ND566—cells are color-coded by KO status.</div><div id="fspara0345" role="paragraph">(F) NK-like T&nbsp;cell population at day 24 CAE for donor ND566, depicted by co-expression of CD3, KLRB1, and KLRC1, overlayed on UMAP graphs from <a id="crosref2020" href="#fig7">Figure&nbsp;7</a>E.</div><div id="fspara0350" role="paragraph">(G) Percentage of NK-like T&nbsp;cells in WT, ID3 KO, and SOX4 KO cells, relative to WT (donor ND566). Significance by Fisher’s exact test.</div><div id="fspara0355" role="paragraph">(H) UMAP graph from <a id="crosref2025" href="#fig7">Figure&nbsp;7</a>E with cells labeled according to expression of the dysfunction gene signature for donor ND566.</div><div id="fspara0360" role="paragraph">(I and J) Dysfunction score for WT, ID3 KO, and SOX4 KO cells for donor ND566 (I) and WT and ID3 KO cells for donor ND539 (J). Significance measured by Mann-Whitney U test.</div><div id="fspara0365" role="paragraph">(K) Dot plot illustrating the expression level of dysfunction signature genes in WT, ID3 KO, and SOX4 KO day 24 CAE cells, donor ND566.</div><div id="fspara0370" role="paragraph">(L-T) Violin plots depicting gene expression levels from WT, ID3 KO, and SOX4 KO day 24 CAE cells for SOX4 (L), AFAP1L2 (M), CSF1 (N), ID3 (O), LAYN (P), CD9 (Q), TNFRSF18 (R), GNLY (S), and KLRC1 (T) for donor ND566.</div><div id="fspara0375" role="paragraph">(U) Cell killing capacity of WT, ID3 KO, and SOX4 KO M5CAR T CAE cells, with controls media alone and day 0 CAR T product. Cells were collected and seeded at 1:8 E:T ratio with AsPC-1 on day 18 (ND539) and day 21 (ND566). Data are presented as mean ± SEM. Significance by two-way ANOVA with Geisser-Greenhouse correction and Dunnet’s post hoc test.</div><div id="fspara0380" role="paragraph">See also <a id="crosref2030" href="#figs7">Figure&nbsp;S7</a>. ∗∗∗∗p&nbsp;&lt; 0.0001, ∗∗∗p&nbsp;&lt; 0.001, ∗∗p&nbsp;&lt; 0.01.</div></div></div></div></figcaption></figure></div><div id="p0220" role="paragraph">To study the role of the transcription factors in driving CAR T dysfunction, we challenged WT, ID3 KO, and SOX4 KO CAR T&nbsp;cells with CAE for 20–28&nbsp;days and analyzed their transcriptional profile and cytotoxic capacity (<a id="crosref1215" href="#fig7">Figure&nbsp;7</a>B). Of note, day 0 and CAE conditions showed a similar KO efficiency, suggesting that there was no enrichment or depletion of SOX4 or ID3 KO cells during CAE (<a id="crosref1220" href="#fig7">Figures 7</a>C, 7D, and <a id="crosref1225" href="#figs7">S7</a>C). To identify whether the transcription factors regulate the NK phenotype and/or the dysfunction signature genes, we performed scRNA-seq. WT cells clustered predominantly on the right side, while ID3 and SOX4 KO cells clustered largely on the left (<a id="crosref1230" href="#fig7">Figure&nbsp;7</a>E). Interestingly, the KO cluster on the left was depleted of NK-like T&nbsp;cells (<a id="crosref1235" href="#fig7">Figure&nbsp;7</a>F) and overall, KO cells showed a significant reduction in the frequency of NK-like T&nbsp;cells compared to WT cells at day 24 (<a id="crosref1240" href="#fig7">Figure&nbsp;7</a>G). This finding was validated in an independent CAR T donor for ID3 KO cells at day 20 CAE (<a id="crosref1245" href="#figs7">Figure&nbsp;S7</a>F).</div><div id="p0225" role="paragraph">We calculated a “dysfunction score” for each cell by taking the average expression level of the 30 genes in our signature. Importantly, we found cells that expressed the highest dysfunction score (in red) were clustered to the right (<a id="crosref1250" href="#fig7">Figure&nbsp;7</a>H), coincident with the cluster of NK-like T&nbsp;cells (<a id="crosref1255" href="#fig7">Figure&nbsp;7</a>F); overall, the KO conditions displayed a significant decrease in the dysfunction score per cell (<a id="crosref1260" href="#fig7">Figure&nbsp;7</a>I). This finding was reproduced in an independent CAR T donor for WT and ID3 KO conditions at day 20 CAE (<a id="crosref1265" href="#fig7">Figure&nbsp;7</a>J). A dot plot also revealed downregulation of the dysfunction signature in ID3 and SOX4 KO cells (<a id="crosref1270" href="#fig7">Figure&nbsp;7</a>K). Interestingly, we detected significant loss of SOX4 expression in the ID3 KO cells, suggesting that SOX4 is a putative ID3 target (<a id="crosref1275" href="#fig7">Figure&nbsp;7</a>L). Hence, the ID3 KO cells resembled a double KO as they lacked both ID3 and SOX4 expression. <i>AFAP1L2</i> and <i>CSF1</i> (genes upregulated in CAE) displayed chromatin opening in day 28 CAE cells at SOX4 motifs (see <a id="crosref1280" href="#fig6">Figures 6</a>G and 6I), and these genes were significantly downregulated in KO cells and are thus putative SOX4 target genes (<a id="crosref1285" href="#fig7">Figures 7</a>M and 7N). Of note, ID3 was significantly downregulated in SOX4 KO cells (<a id="crosref1290" href="#fig7">Figure&nbsp;7</a>O), although expression was not abrogated, suggesting ID3 may have additional transcriptional regulators. Select genes significantly downregulated in both KO conditions include <i>LAYN</i>, <i>CD9</i>, <i>TNFRSF18</i>, <i>GNLY</i>, and <i>KLRC1</i> (<a id="crosref1295" href="#fig7">Figures 7</a>P–7T).</div><div id="p0230" role="paragraph">To determine whether KO of ID3 or SOX4 associated with increased effector function, we performed cytotoxicity assays following CAE with WT, ID3 KO, and SOX4 KO cells. Importantly, ID3 and SOX4 KO cells showed enhanced CAR T killing of tumor cells after CAE compared to WT cells (<a id="crosref1300" href="#fig7">Figures 7</a>U and <a id="crosref1305" href="#figs7">S7</a>G–S7I).</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0235" role="paragraph">Several recent studies have suggested that T&nbsp;cell dysfunction is a major contributor to ineffective CAR T&nbsp;cell therapy in solid tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Poorebrahim et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Poorebrahim, M. ∙ Melief, J. ∙ Pico de Coaña, Y. ...</div><div><strong>Counteracting CAR T&nbsp;cell dysfunction</strong></div><div><em>Oncogene.</em> 2021; <strong>40</strong>:421-435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41388-020-01501-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85095708741" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33168929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41388-020-01501-x&amp;pmid=33168929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, little is known about the mechanisms mediating loss of CAR T&nbsp;cell function. Here, we examine how prolonged exposure to tumor antigen (CAE) in an <i>in&nbsp;vitro</i> model, as similarly encountered by CAR T&nbsp;cells in the TME, impacts the efficacy, surface expression, and phenotype of CAR T&nbsp;cells. We show the acquisition of a CAR T dysfunction or exhaustion gene signature and the transcription factors that regulate this transition. Moreover, we identify multiple mechanisms of CAR T dysfunction and demonstrate their relevance to patients treated with CAR T&nbsp;cell therapy.</div><div id="p0240" role="paragraph">Indeed, we identified a mechanism of CAR T&nbsp;cell dysfunction whereby cells undergo a transition from T&nbsp;cells to NK-like T&nbsp;cells. Our findings are supported by reports that CD8+ T&nbsp;cells acquire innate like characteristics by expressing NK receptors during chronic antigen exposure (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Balin et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Balin, S.J. ∙ Pellegrini, M. ∙ Klechevsky, E. ...</div><div><strong>Human antimicrobial cytotoxic T lymphocytes, defined by NK receptors and antimicrobial proteins, kill intracellular bacteria</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaat7668</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aat7668" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85059961035" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30171080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aat7668&amp;pmid=30171080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Seyda et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Seyda, M. ∙ Elkhal, A. ∙ Quante, M. ...</div><div><strong>T Cells Going Innate</strong></div><div><em>Trends Immunol.</em> 2016; <strong>37</strong>:546-556</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2016.06.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2016.06.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84991745789" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27402226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2016.06.004&amp;pmid=27402226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Wencker et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Wencker, M. ∙ Turchinovich, G. ∙ Di Marco Barros, R. ...</div><div><strong>Innate-like T&nbsp;cells straddle innate and adaptive immunity by altering antigen-receptor responsiveness</strong></div><div><em>Nat. Immunol.</em> 2014; <strong>15</strong>:80-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2773" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84890937867" target="_blank" aria-label="Open Scopus in new tab">Scopus (167)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24241693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2773&amp;pmid=24241693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and by observations of increased expression of NK receptors on tumor-infiltrating CD8+ T&nbsp;cells isolated from patients with hematological malignancy and solid tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref6-3" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-3" aria-controls="sref6-3">Barbarin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Barbarin, A. ∙ Cayssials, E. ∙ Jacomet, F. ...</div><div><strong>Phenotype of NK-Like CD8(+) T Cells with Innate Features in Humans and Their Relevance in Cancer Diseases</strong></div><div><em>Front. Immunol.</em> 2017; <strong>8</strong>:316</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2017.00316" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85017100637" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28396661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2017.00316&amp;pmid=28396661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Mathewson et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Mathewson, N.D. ∙ Ashenberg, O. ∙ Tirosh, I. ...</div><div><strong>Inhibitory CD161 receptor identified in glioma-infiltrating T&nbsp;cells by single-cell analysis</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:1281-1298.e1226</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85101668184" target="_blank" aria-label="Open Scopus in new tab">Scopus (216)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592174/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.01.022&amp;pmid=33592174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Several studies have shown that NKG2A and KLRB1 act as immune checkpoints and that blocking these receptors improves the efficacy of immunotherapies (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Abd Hamid et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Abd Hamid, M. ∙ Wang, R.Z. ∙ Yao, X. ...</div><div><strong>Enriched HLA-E and CD94/NKG2A Interaction Limits Antitumor CD8<sup>+</sup> Tumor-Infiltrating T Lymphocyte Responses</strong></div><div><em>Cancer Immunol. Res.</em> 2019; <strong>7</strong>:1293-1306</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2326-6066.CIR-18-0885" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85070565916" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31213473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2326-6066.CIR-18-0885&amp;pmid=31213473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Andre et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Andre, P. ∙ Denis, C. ∙ Soulas, C. ...</div><div><strong>Anti-NKG2A mAb Is a Checkpoint Inhibitor that Promotes Anti-tumor Immunity by Unleashing Both T and NK Cells</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:1731-1743.e1713</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85058479063" target="_blank" aria-label="Open Scopus in new tab">Scopus (800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30503213/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.10.014&amp;pmid=30503213" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Mathewson et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Mathewson, N.D. ∙ Ashenberg, O. ∙ Tirosh, I. ...</div><div><strong>Inhibitory CD161 receptor identified in glioma-infiltrating T&nbsp;cells by single-cell analysis</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:1281-1298.e1226</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85101668184" target="_blank" aria-label="Open Scopus in new tab">Scopus (216)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592174/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.01.022&amp;pmid=33592174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">van Montfoort et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>van Montfoort, N. ∙ Borst, L. ∙ Korrer, M.J. ...</div><div><strong>NKG2A Blockade Potentiates CD8&nbsp;T Cell Immunity Induced by Cancer Vaccines</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:1744-1755.e1715</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85058015117" target="_blank" aria-label="Open Scopus in new tab">Scopus (232)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30503208/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.10.028&amp;pmid=30503208" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Further, CD8+ cytotoxic T lymphocytes (CTLs) expressing cytotoxic granule proteins perforin, granzyme B, granulysin, and NK receptor NKG2C mediate TCR-dependent and independent anti-microbial activity (<span class="dropBlock reference-citations"><a id="body-ref-sref5-3" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-3" aria-controls="sref5-3">Balin et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Balin, S.J. ∙ Pellegrini, M. ∙ Klechevsky, E. ...</div><div><strong>Human antimicrobial cytotoxic T lymphocytes, defined by NK receptors and antimicrobial proteins, kill intracellular bacteria</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaat7668</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aat7668" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85059961035" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30171080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aat7668&amp;pmid=30171080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Interestingly, in addition to NK receptors, CAR T&nbsp;cells in our <i>in&nbsp;vitro</i> model express all three cytotoxic granule protein genes. Furthermore, plasticity of CTLs to NK-like cells has been observed in celiac disease (<span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Meresse et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Meresse, B. ∙ Chen, Z. ∙ Ciszewski, C. ...</div><div><strong>Coordinated induction by IL15 of a TCR-independent NKG2D signaling pathway converts CTL into lymphokine-activated killer cells in celiac disease</strong></div><div><em>Immunity.</em> 2004; <strong>21</strong>:357-366</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2004.06.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2004.06.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-4444240035" target="_blank" aria-label="Open Scopus in new tab">Scopus (695)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15357947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2004.06.020&amp;pmid=15357947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Together, these data support that NK-like T&nbsp;cells have an important role in immunity and that T&nbsp;cells can undergo a transition to NK-like T&nbsp;cells. Under prolonged CAE, CAR T&nbsp;cells both fail to re-express surface CAR and exhibit a significant decrease in the expression of genes involved in the antigen presentation pathway (see <a id="crosref1370" href="#fig2">Figure&nbsp;2</a>C), leading us to speculate that these conditions may select for T&nbsp;cells that transition to NK-like T&nbsp;cells because NK receptors provide needed signals required for T&nbsp;cell survival. Expression of the inhibitory NK receptors, such as CD94-NKG2A, KLRB1 (CD161), TIGIT, and inhibitory KIR may initially serve as a feedback mechanism to dampen excessive stimulatory signaling to avoid activation-induced cell death induced by TCR or CAR.</div><div id="p0245" role="paragraph">Single-cell gene expression data from CAE CAR-transduced CD8+ T&nbsp;cells uncovers both non-dysfunctional and dysfunctional clusters. The dysfunctional T&nbsp;cell clusters are defined by a robust gene expression signature that includes genes implicated in T&nbsp;cell exhaustion such as <i>HAVCR2</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Sakuishi et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Sakuishi, K. ∙ Apetoh, L. ∙ Sullivan, J.M. ...</div><div><strong>Targeting Tim-3 and PD-1 pathways to reverse T&nbsp;cell exhaustion and restore anti-tumor immunity</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2010; <strong>207</strong>:2187-2194</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20100643" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-77957744369" target="_blank" aria-label="Open Scopus in new tab">Scopus (1546)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20819927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20100643&amp;pmid=20819927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63-2" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-2" aria-controls="sref63-2">Singer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref63-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Singer, M. ∙ Wang, C. ∙ Cong, L. ...</div><div><strong>A Distinct Gene Module for Dysfunction Uncoupled from Activation in Tumor-Infiltrating T Cells</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:1500-1511.e1509</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84986285632" target="_blank" aria-label="Open Scopus in new tab">Scopus (269)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27610572/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.052&amp;pmid=27610572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>LAYN</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref77-3" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-3" aria-controls="sref77-3">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref77-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zheng, C. ∙ Zheng, L. ∙ Yoo, J.K. ...</div><div><strong>Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1342-1356.e1316</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85020894317" target="_blank" aria-label="Open Scopus in new tab">Scopus (1383)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.035&amp;pmid=28622514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>PHLDA1</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref37-3" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-3" aria-controls="sref37-3">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, H. ∙ van der Leun, A.M. ∙ Yofe, I. ...</div><div><strong>Dysfunctional CD8&nbsp;T Cells Form a Proliferative, Dynamically Regulated Compartment within Human Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:775-789.e718</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061004091" target="_blank" aria-label="Open Scopus in new tab">Scopus (661)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30595452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.11.043&amp;pmid=30595452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and <i>TNFRSF9</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Mognol et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Mognol, G.P. ∙ Spreafico, R. ∙ Wong, V. ...</div><div><strong>Exhaustion-associated regulatory regions in CD8<sup>+</sup> tumor-infiltrating T&nbsp;cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:E2776-E2785</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1620498114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85016431802" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28283662/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1620498114&amp;pmid=28283662" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and genes with no known connection to dysfunction including <i>RGS16</i>, <i>SRGAP3</i>, <i>DUSP4</i>, <i>NDFIP2</i>, and <i>CD9</i>. CAR expression was predominately detected in the dysfunctional clusters, with minimal expression in the non-dysfunctional clusters, indicating that chronic stimulation of CAR T&nbsp;cells is driving the dysfunction phenotype. Strikingly, we observed robust alignment of the dysfunction gene signature identified in our <i>in&nbsp;vitro</i> CAR T CAE model with gene expression changes in hypofunctional NY-ESO-1 TILs isolated from <i>in&nbsp;vivo</i> tumors; this important correlation suggests that our dysfunction signature is relevant to gene-engineered cell therapy, independent of whether CAR- or TCR mediated. Further, hypofunctional TILs isolated from mice with relapsing mesothelin-positive AsPC-1 tumors following M5CAR T&nbsp;cell injection also expressed NK receptors and exhaustion markers, similar to our <i>in&nbsp;vitro</i> data in <a id="crosref1400" href="#fig4">Figure&nbsp;4</a>A. This data, in conjunction with NY-ESO-1 TIL data, provides <i>in&nbsp;vivo</i> demonstration that cells exhibiting the exhaustion signatures are dysfunctional <i>in&nbsp;vivo</i>.</div><div id="p0250" role="paragraph">We further investigated the regulatory mechanisms driving CAR T&nbsp;cell dysfunction. We find that the transcription factors <i>SOX4</i> and <i>ID3</i> regulate genes in the dysfunction signature. Notably, our finding of improved CAR T&nbsp;cell killing in ID3 and SOX4 KO human CAR T&nbsp;cells demonstrates a role for these transcription factors in the dysfunction of CAR T&nbsp;cells. ID3 is important for promoting the thymic development of bipotential NK/T progenitors to an NK cell fate (<span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Leong et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Leong, J.W. ∙ Wagner, J.A. ∙ Ireland, A.R. ...</div><div><strong>Transcriptional and post-transcriptional regulation of NK cell development and function</strong></div><div><em>Clin. Immunol.</em> 2017; <strong>177</strong>:60-69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.clim.2016.03.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84977090602" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26948928/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.clim.2016.03.003&amp;pmid=26948928" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and forced expression of ID3 blocks T&nbsp;cell and promotes NK cell development in a fetal thymic organ culture system (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Heemskerk et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Heemskerk, M.H. ∙ Blom, B. ∙ Nolan, G. ...</div><div><strong>Inhibition of T&nbsp;cell and promotion of natural killer cell development by the dominant negative helix loop helix factor Id3</strong></div><div><em>J.&nbsp;Exp. Med.</em> 1997; <strong>186</strong>:1597-1602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.186.9.1597" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-0030665673" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9348318/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.186.9.1597&amp;pmid=9348318" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SOX4 has been shown to control thymic production of iNKT cells by inducing microRNA-181 (Mir181) to enhance TCR signaling (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Malhotra et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Malhotra, N. ∙ Qi, Y. ∙ Spidale, N.A. ...</div><div><strong>SOX4 controls invariant NKT cell differentiation by tuning TCR signaling</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2018; <strong>215</strong>:2887-2900</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20172021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85056287865" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30287480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20172021&amp;pmid=30287480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ID3 and SOX4 are also key transcription factors in memory CD8+ T&nbsp;cell development (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Hu and Chen, 2013</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Hu, G. ∙ Chen, J.</div><div><strong>A genome-wide regulatory network identifies key transcription factors for memory CD8<sup>+</sup> T-cell development</strong></div><div><em>Nat. Commun.</em> 2013; <strong>4</strong>:2830</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms3830" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84890638806" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24335726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms3830&amp;pmid=24335726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Ji et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Ji, Y. ∙ Pos, Z. ∙ Rao, M. ...</div><div><strong>Repression of the DNA-binding inhibitor Id3 by Blimp-1 limits the formation of memory CD8+ T&nbsp;cells</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:1230-1237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2153" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-81255144750" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22057288/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2153&amp;pmid=22057288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Yang et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Yang, C.Y. ∙ Best, J.A. ∙ Knell, J. ...</div><div><strong>The transcriptional regulators Id2 and Id3 control the formation of distinct memory CD8+ T&nbsp;cell subsets</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:1221-1229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-81255144662" target="_blank" aria-label="Open Scopus in new tab">Scopus (305)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22057289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2158&amp;pmid=22057289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and Prdm1 and Id3 expression distinguish distinct CD8+ T&nbsp;cell subsets in acute viral and bacterial infections and tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref47-1" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-1" aria-controls="sref47-1">Milner et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref47-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Milner, J.J. ∙ Toma, C. ∙ He, Z. ...</div><div><strong>Heterogenous Populations of Tissue-Resident CD8<sup>+</sup> T Cells Are Generated in Response to Infection and Malignancy</strong></div><div><em>Immunity.</em> 2020; <strong>52</strong>:808-824.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084480841" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32433949/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.04.007&amp;pmid=32433949" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our observation that ID3 plays a role in T&nbsp;cell dysfunction is supported by Li et&nbsp;al., who identify ID3 as one of 19 transcription factors computationally predicted to regulate dysfunctional melanoma TILs isolated from human patients (<span class="dropBlock reference-citations"><a id="body-ref-sref37-4" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-4" aria-controls="sref37-4">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, H. ∙ van der Leun, A.M. ∙ Yofe, I. ...</div><div><strong>Dysfunctional CD8&nbsp;T Cells Form a Proliferative, Dynamically Regulated Compartment within Human Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:775-789.e718</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061004091" target="_blank" aria-label="Open Scopus in new tab">Scopus (661)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30595452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.11.043&amp;pmid=30595452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Further, Id3<sup>hi</sup>/Prdm1<sup>lo</sup> mouse TILs show enrichment of gene-expression signatures associated with progenitor exhausted T&nbsp;cells (<span class="dropBlock reference-citations"><a id="body-ref-sref47-2" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-2" aria-controls="sref47-2">Milner et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref47-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Milner, J.J. ∙ Toma, C. ∙ He, Z. ...</div><div><strong>Heterogenous Populations of Tissue-Resident CD8<sup>+</sup> T Cells Are Generated in Response to Infection and Malignancy</strong></div><div><em>Immunity.</em> 2020; <strong>52</strong>:808-824.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084480841" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32433949/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.04.007&amp;pmid=32433949" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and Id3 expression delineates progenitor exhausted T&nbsp;cells in an LCMV model of chronic viral infection (<span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Utzschneider et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Utzschneider, D.T. ∙ Gabriel, S.S. ∙ Chisanga, D. ...</div><div><strong>Early precursor T&nbsp;cells establish and propagate T&nbsp;cell exhaustion in chronic infection</strong></div><div><em>Nat. Immunol.</em> 2020; <strong>21</strong>:1256-1266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-020-0760-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85089743781" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32839610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-020-0760-z&amp;pmid=32839610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, GSEA analysis of our CAE CAR T&nbsp;cells, which express both ID3 and PRDM1, reveals significant enrichment with genes upregulated in intermediate and terminally exhausted T&nbsp;cells, but not progenitor populations (<span class="dropBlock reference-citations"><a id="body-ref-sref7-3" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-3" aria-controls="sref7-3">Beltra et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref7-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Beltra, J.C. ∙ Manne, S. ∙ Abdel-Hakeem, M.S. ...</div><div><strong>Developmental Relationships of Four Exhausted CD8(+) T Cell Subsets Reveals Underlying Transcriptional and Epigenetic Landscape Control Mechanisms</strong></div><div><em>Immunity.</em> 2020; <strong>52</strong>:825-841.e828</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084517361" target="_blank" aria-label="Open Scopus in new tab">Scopus (486)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32396847/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.04.014&amp;pmid=32396847" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, compared to exhausted WT cells, chronically infected Tox-deficient T&nbsp;cells are negatively enriched for the SOX4 transcription factor network, indicating that Sox4 may collaborate with Tox and other transcription factors in the development of exhaustion upon chronic infection (<span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Khan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Khan, O. ∙ Giles, J.R. ∙ McDonald, S. ...</div><div><strong>TOX transcriptionally and epigenetically programs CD8<sup>+</sup> T&nbsp;cell exhaustion</strong></div><div><em>Nature.</em> 2019; <strong>571</strong>:211-218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1325-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85068336352" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31207603/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1325-x&amp;pmid=31207603" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, SOX4 is downregulated in two persistent clonotypes of a mutated KRAS (G12D)-reactive TIL infusion product from a patient with metastatic colorectal cancer (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Lu et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Lu, Y.C. ∙ Jia, L. ∙ Zheng, Z. ...</div><div><strong>Single-Cell Transcriptome Analysis Reveals Gene Signatures Associated with T-cell Persistence Following Adoptive Cell Therapy</strong></div><div><em>Cancer Immunol. Res.</em> 2019; <strong>7</strong>:1824-1836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2326-6066.CIR-19-0299" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85074444784" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31484655/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2326-6066.CIR-19-0299&amp;pmid=31484655" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), suggesting that its downregulation may contribute to persistence in adoptive cell therapy. Importantly, our finding of improved CAR T&nbsp;cell killing in ID3 and SOX4 KO human CAR T&nbsp;cells demonstrates a role for these transcription factors in the dysfunction of CAR T&nbsp;cells.</div><div id="p0255" role="paragraph">In summary, our robust <i>in&nbsp;vitro</i> model of dysfunction in pancreatic cancer reveals multiple mechanisms of CAR and TCR T&nbsp;cell dysfunction, including features of exhaustion and transition of CD8+ T&nbsp;cells to an NK-like T&nbsp;cell state. Importantly, we demonstrate that these <i>in&nbsp;vitro</i> observations are relevant <i>in&nbsp;vivo</i>, both in mouse models of CAR T and TCR dysfunction and in patients after CAR T&nbsp;cell infusion. We further confirm the predictive value of the model whereby disruption of the transcription factors ID3 and SOX4 in CAR T&nbsp;cells diminishes the dysfunctional gene expression signature and, importantly, enhances tumor killing. In conclusion, our <i>in&nbsp;vitro</i> model of human T&nbsp;cell dysfunction provides a validated platform that can lead to the development of new strategies to improve the efficacy of CAR and TCR T&nbsp;cell therapy in solid tumors.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0260" role="paragraph">While gene expression signatures obtained from dysfunctional CAR T&nbsp;cells <i>in&nbsp;vitro</i> significantly overlaps with <i>in&nbsp;vivo</i> models of T&nbsp;cell dysfunction, our model does not recapitulate other facets of T&nbsp;cell exhaustion, including the immunosuppressive effects of the TME or tumor-cell-line-specific effects contributing to exhaustion. Further, while our data show that KO of ID3 and SOX4 improves effector function <i>in&nbsp;vitro</i>, this study does not test KO CAR T&nbsp;cells using <i>in&nbsp;vivo</i> models, and thus follow-up studies are needed to determine whether these transcription factors function similarly <i>in&nbsp;vivo</i>.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0275" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>anti-human CD45</td><td>Biolegend</td><td>Cat# 304032; RRID:<a id="interref190" href="https://antibodyregistry.org/AB_2561357" target="_blank">AB_2561357</a></td></tr><tr><td>anti-human CD45</td><td>Biolegend</td><td>Cat# 304017; RRID:<a id="interref195" href="https://antibodyregistry.org/AB_389314" target="_blank">AB_389314</a></td></tr><tr><td>anti-human CD45</td><td>Biolegend</td><td>Cat# 304028; RRID:<a id="interref200" href="https://antibodyregistry.org/AB_893338" target="_blank">AB_893338</a></td></tr><tr><td>anti-human CD3</td><td>Biolegend</td><td>Cat# 317322; RRID:<a id="interref205" href="https://antibodyregistry.org/AB_2561911" target="_blank">AB_2561911</a></td></tr><tr><td>anti-human CD8</td><td>Biolegend</td><td>Cat# 344748; RRID:<a id="interref210" href="https://antibodyregistry.org/AB_2629584" target="_blank">AB_2629584</a></td></tr><tr><td>anti-human CD4</td><td>Biolegend</td><td>Cat# 357412; RRID:<a id="interref215" href="https://antibodyregistry.org/AB_2565664" target="_blank">AB_2565664</a></td></tr><tr><td>anti-human CD4</td><td>Biolegend</td><td>Cat# 317440; RRID:<a id="interref220" href="https://antibodyregistry.org/AB_2562912" target="_blank">AB_2562912</a></td></tr><tr><td>anti-human CD4</td><td>Biolegend</td><td>Cat# 317428; RRID:<a id="interref225" href="https://antibodyregistry.org/AB_1186122" target="_blank">AB_1186122</a></td></tr><tr><td>anti-human CD56</td><td>Biolegend</td><td>Cat# 304608; RRID:<a id="interref230" href="https://antibodyregistry.org/AB_314450" target="_blank">AB_314450</a></td></tr><tr><td>anti-human EpCAM</td><td>Biolegend</td><td>Cat# 324226; RRID:<a id="interref235" href="https://antibodyregistry.org/AB_256273" target="_blank">AB_256273</a></td></tr><tr><td>anti-human EpCAM</td><td>Biolegend</td><td>Cat# 324238; RRID:<a id="interref240" href="https://antibodyregistry.org/AB_2632937" target="_blank">AB_2632937</a></td></tr><tr><td>anti-human anti-human CD94</td><td>Biolegend</td><td>Cat# 305520; RRID:<a id="interref245" href="https://antibodyregistry.org/AB_2734277" target="_blank">AB_2734277</a></td></tr><tr><td>anti-human KLRB1</td><td>Biolegend</td><td>Cat# 339918;RRID:<a id="interref250" href="https://antibodyregistry.org/AB_11126745" target="_blank">AB_11126745</a></td></tr><tr><td>anti-human TIGIT</td><td>Biolegend</td><td>Cat# 372716, RRID:<a id="interref255" href="https://antibodyregistry.org/AB_2632931" target="_blank">AB_2632931</a></td></tr><tr><td>anti-human TCR Va24-Ja18</td><td>Biolegend</td><td>Cat# 342922; RRID:<a id="interref260" href="https://antibodyregistry.org/AB_2572068" target="_blank">AB_2572068</a></td></tr><tr><td>anti-human PD-1</td><td>Biolegend</td><td>Cat# 329928; RRID:<a id="interref265" href="https://antibodyregistry.org/AB_2562911" target="_blank">AB_2562911</a></td></tr><tr><td>anti-human TIM3</td><td>Biolegend</td><td>Cat# 345014; RRID:<a id="interref270" href="https://antibodyregistry.org/AB_2561720" target="_blank">AB_2561720</a></td></tr><tr><td>anti-human LAG3</td><td>Biolegend</td><td>Cat# 369315; RRID:<a id="interref275" href="https://antibodyregistry.org/AB_2632950" target="_blank">AB_2632950</a></td></tr><tr><td>anti-human Mesothelin</td><td>Biolegend</td><td>Cat# 530203; RRID:<a id="interref280" href="https://antibodyregistry.org/AB_2571909" target="_blank">AB_2571909</a></td></tr><tr><td>anti-human CD45RO</td><td>Biolegend</td><td>Cat# 304244; RRID:<a id="interref285" href="https://antibodyregistry.org/AB_2564160" target="_blank">AB_2564160</a></td></tr><tr><td>anti-human CD8</td><td>BD Biosciences</td><td>Cat# 560179; RRID:<a id="interref290" href="https://antibodyregistry.org/AB_1645481" target="_blank">AB_1645481</a></td></tr><tr><td>anti-human CCR7</td><td>BD Biosciences</td><td>Cat# 561271; RRID:<a id="interref295" href="https://antibodyregistry.org/AB_10561679" target="_blank">AB_10561679</a></td></tr><tr><td>anti-human NKG2A</td><td>R&amp;D Systems</td><td>Cat# FAB1059P; RRID:<a id="interref300" href="https://antibodyregistry.org/AB_2132978" target="_blank">AB_2132978</a></td></tr><tr><td>anti-human Mesothelin</td><td>R&amp;D Systems</td><td>Cat# FAB32652P; RRID:<a id="interref305" href="https://antibodyregistry.org/AB_1151946" target="_blank">AB_1151946</a></td></tr><tr><td>anti-human CTLA-4</td><td>eBioscience</td><td>Cat# 12-1529-42; RRID:<a id="interref310" href="https://antibodyregistry.org/AB_10805626" target="_blank">AB_10805626</a></td></tr><tr><td>anti-human NKG2C</td><td>Miltenyi Biotec</td><td>Cat# 130-103-636; RRID:<a id="interref315" href="https://antibodyregistry.org/AB_2655396" target="_blank">AB_2655396</a></td></tr><tr><td>anti-human IgG F(ab’)2</td><td>Jackson ImmunoResearch</td><td>Cat# 109-066-006; RRID:<a id="interref320" href="https://antibodyregistry.org/AB_2337634" target="_blank">AB_2337634</a></td></tr><tr><td>mouse IgG2a, κ Isotype control Antibody</td><td>Biolegend</td><td>Cat# 400269; RRID:<a id="interref325" href="https://antibodyregistry.org/AB_2864282" target="_blank">AB_2864282</a></td></tr><tr><td>mouse IgG1, κ Isotype control Antibody</td><td>Biolegend</td><td>Cat# 400126; RRID:<a id="interref330" href="https://antibodyregistry.org/AB_326448" target="_blank">AB_326448</a></td></tr><tr class="rowsep"><td>mouse IgG1, κ Isotype control Antibody</td><td>Biolegend</td><td>Cat# 400168; RRID:<a id="interref335" href="https://antibodyregistry.org/AB_11218607" target="_blank">AB_11218607</a></td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr class="rowsep"><td>One Shot™ Stbl3™ Chemically Competent E.&nbsp;coli</td><td>Invitrogen</td><td>C7373-03</td></tr><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr class="rowsep"><td>T&nbsp;lymphocytes from human healthy donors</td><td>UPenn Human Immunology Core</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Live/Dead Aqua</td><td>ThermoFisher</td><td>Cat# L34957</td></tr><tr><td>Zombie NIR Fixable Viability Kits</td><td>Biolegend</td><td>Cat# 423106</td></tr><tr><td>Apotracker™ Green</td><td>Biolegend</td><td>Cat# 427403</td></tr><tr><td>Alt-R® Cas9 Electroporation Enhancer, 10&nbsp;nmol</td><td>Integrated DNA Technologies</td><td>Cat# 1075916</td></tr><tr><td>SpyFi Cas9</td><td>Aldeveron</td><td>Cat# 9214</td></tr><tr><td>P3 Primary cell 4D-nucleofactor X Kit L</td><td>Lonza</td><td>Cat# V4XP-3024</td></tr><tr><td>OpTmizer T Cell Expansion SFM</td><td>Gibco</td><td>Cat# A1048501</td></tr><tr><td>Human AB Serum</td><td>GeminiBio</td><td>Cat#100-512</td></tr><tr><td>Recombinant Human IL-7</td><td>Peprotech</td><td>Cat#200-07</td></tr><tr><td>Recombinant Human IL-15</td><td>Peprotech</td><td>Cat#200-15</td></tr><tr><td>DNAse I roche</td><td>Sigma</td><td>Cat#10104159001</td></tr><tr><td>Lipofectamine 2000</td><td>Thermo Fisher Scientific</td><td>Cat#1166801</td></tr><tr><td>Lipofectamine 3000</td><td>Thermo Fisher Scientific</td><td>Cat#L3000015</td></tr><tr><td>Matrigel Membrane Matrix</td><td>Corning</td><td>Cat#356234</td></tr><tr><td>Collagenase D</td><td>Sigma</td><td>Cat# 11088866001</td></tr><tr><td>DNase I from bovine pancreas</td><td>Sigma</td><td>Cat# 11284932001</td></tr><tr><td>EDTA (0.5 M), pH 8.0, RNase-free</td><td>Thermo Fisher</td><td>Cat# AM9261</td></tr><tr><td>FITC Streptavidin</td><td>Biolegend</td><td>Cat# 405202</td></tr><tr><td>Alexa Fluor® 488 Streptavidin</td><td>Biolegend</td><td>Cat# 405235</td></tr><tr class="rowsep"><td>APC Streptavidin</td><td>Biolegend</td><td>Cat# 405235</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>RNA Clean &amp; Concentrator™-5</td><td>ZYMO</td><td>R1016</td></tr><tr><td>EZ-Tn5™ Transposase</td><td>Lucigen</td><td>TNP92110</td></tr><tr><td>SMARTScribe™ Reverse Transcriptase</td><td>Takara</td><td>639536</td></tr><tr><td>AGENCOURT® AMPURE® XP</td><td>beckmancoulter</td><td>A63881</td></tr><tr><td>DNA Clean &amp; Concentrator™-5</td><td>ZYMO</td><td>D4014</td></tr><tr><td>TAGMENT DNA BUFFER</td><td>Illumina</td><td>15027866</td></tr><tr><td>TDE1,TAGMENT DNA ENZYME</td><td>Illumina</td><td>15027865</td></tr><tr><td>NEBNext® Library Quant Kit for Illumina®</td><td>New England Biolabs</td><td>E7630L</td></tr><tr><td>NextSeq 500/550 High Output Kit (75 cycles) v2.5 kit</td><td>Illumina</td><td>20024906</td></tr><tr><td>NextSeq 500/550 High Output Kit v2.5 (150 Cycles)</td><td>Illumina</td><td>20024907</td></tr><tr><td>Chromium Next GEM Single Cell 3’ GEM, Library &amp; Gel Bead Kit v3.1</td><td>10X Genomics</td><td>1000128</td></tr><tr><td>Single Index Kit T Set A</td><td>10X Genomics</td><td>1000213</td></tr><tr><td>Chromium Single Cell 5’ Library &amp; Gel Bead Kit</td><td>10X Genomics</td><td>1000014</td></tr><tr><td>Chromium Single Cell V(D)J Enrichment Kit, Human T Cell</td><td>10X Genomics</td><td>1000005</td></tr><tr><td>Chromium Single Cell 3’ Library &amp; Gel Bead Kit v3</td><td>10X Genomics</td><td>1000092</td></tr><tr><td>DynaBeads CD3x28 (Human)</td><td>ThermoFisher</td><td>Cat# 11131D</td></tr><tr><td>NucleoSpin Gel and PCR Clean-up</td><td>Macherey-Nagel</td><td>Cat# 74609.50</td></tr><tr><td>QIAGEN Plasmid Plus Maxi Kit</td><td>QIAGEN</td><td>Cat# 12963</td></tr><tr><td>True-Nuclear™ Transcription Factor Buffer Set</td><td>Biolegend</td><td>Cat# 424401</td></tr><tr><td>Foxp3 / Transcription Factor Staining Buffer Set</td><td>Life Technologies</td><td>Cat# 00-5523-00</td></tr><tr><td>CD56 Microbeads</td><td>Miltenyi Biotec</td><td>Cat# 130-050-401</td></tr><tr><td>CountBright Absolute Counting Beads, (ThermoFisher)</td><td>Thermo Fisher</td><td>Cat# C36950</td></tr><tr><td>LongAmp™ Taq 2X Master Mix</td><td>New England Biolabs</td><td>Cat# M0287S</td></tr><tr><td>Vacuum Filter/Storage Systems</td><td>Corning</td><td>Cat# 430770</td></tr><tr class="rowsep"><td>Dead cell removal kit</td><td>Miltenyi Biotec</td><td>Cat# 130-090-101</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Raw and analyzed data</td><td>This paper</td><td>GEO: <a id="intref0015" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE160174" target="_blank">GSE160174</a></td></tr><tr><td>LCMV mouse model naïve and exhausted T&nbsp;cell RNA-seq datasets</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Pauken et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Pauken, K.E. ∙ Sammons, M.A. ∙ Odorizzi, P.M. ...</div><div><strong>Epigenetic stability of exhausted T&nbsp;cells limits durability of reinvigoration by PD-1 blockade</strong></div><div><em>Science.</em> 2016; <strong>354</strong>:1160-1165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf2807" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84994364943" target="_blank" aria-label="Open Scopus in new tab">Scopus (893)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27789795/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf2807&amp;pmid=27789795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>GEO: <a id="intref0020" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE86881" target="_blank">GSE86881</a></td></tr><tr class="rowsep"><td>Human PD1 high CD8 T&nbsp;cell ATAC-seq datasets</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref53-5" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-5" aria-controls="sref53-5">Philip et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Philip, M. ∙ Fairchild, L. ∙ Sun, L. ...</div><div><strong>Chromatin states define tumour-specific T&nbsp;cell dysfunction and reprogramming</strong></div><div><em>Nature.</em> 2017; <strong>545</strong>:452-456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22367" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85019673982" target="_blank" aria-label="Open Scopus in new tab">Scopus (602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28514453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22367&amp;pmid=28514453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>GEO: <a id="intref0025" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89308" target="_blank">GSE89308</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>Human (female) HEK293T</td><td>ATCC</td><td>CRL-11268</td></tr><tr><td>Human (female) K562</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td>Human (female) ASPC-1</td><td>ATCC</td><td>CRL-1682</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr class="rowsep"><td>NOD/scid/IL2rγ−/− (NSG)</td><td>Jackson Laboratory</td><td>Cat# 5557</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>TSO (SMARTseq2): AAG CAG TGG TAT CAA CGC AGA GTA CAT rGrGrG</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Picelli et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Picelli, S. ∙ Faridani, O.R. ∙ Björklund, A.K. ...</div><div><strong>Full-length RNA-seq from single cells using Smart-seq2</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:171-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84891677425" target="_blank" aria-label="Open Scopus in new tab">Scopus (2542)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.006&amp;pmid=24385147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>Oligo-dT30VN (SMARTseq2): AAG CAG TGG TAT CAA CGC AGA GTA CTT TTT TTT TTT TTT TTT TTT TTT TTT TTT TVN</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Picelli et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Picelli, S. ∙ Faridani, O.R. ∙ Björklund, A.K. ...</div><div><strong>Full-length RNA-seq from single cells using Smart-seq2</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:171-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84891677425" target="_blank" aria-label="Open Scopus in new tab">Scopus (2542)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.006&amp;pmid=24385147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>ISPCR (SMARTseq2): AAG CAG TGG TAT CAA CGC AGA GT</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref54-3" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-3" aria-controls="sref54-3">Picelli et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Picelli, S. ∙ Faridani, O.R. ∙ Björklund, A.K. ...</div><div><strong>Full-length RNA-seq from single cells using Smart-seq2</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:171-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84891677425" target="_blank" aria-label="Open Scopus in new tab">Scopus (2542)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.006&amp;pmid=24385147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>Tn5MErev (SMARTseq2): /5Phos/CT GTC TCT TAT ACA CAT CT</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref54-4" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-4" aria-controls="sref54-4">Picelli et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Picelli, S. ∙ Faridani, O.R. ∙ Björklund, A.K. ...</div><div><strong>Full-length RNA-seq from single cells using Smart-seq2</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:171-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84891677425" target="_blank" aria-label="Open Scopus in new tab">Scopus (2542)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.006&amp;pmid=24385147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>Tn5ME-A (SMARTseq2): TCG TCG GCA GCG TCA GAT GTG TAT AAG AGA CAG</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref54-5" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-5" aria-controls="sref54-5">Picelli et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Picelli, S. ∙ Faridani, O.R. ∙ Björklund, A.K. ...</div><div><strong>Full-length RNA-seq from single cells using Smart-seq2</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:171-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84891677425" target="_blank" aria-label="Open Scopus in new tab">Scopus (2542)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.006&amp;pmid=24385147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>Tn5ME-B (SMARTseq2): GTC TCG TGG GCT CGG AGA TGT GTA TAA GAG ACA G</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref54-6" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-6" aria-controls="sref54-6">Picelli et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Picelli, S. ∙ Faridani, O.R. ∙ Björklund, A.K. ...</div><div><strong>Full-length RNA-seq from single cells using Smart-seq2</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:171-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84891677425" target="_blank" aria-label="Open Scopus in new tab">Scopus (2542)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.006&amp;pmid=24385147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>Ad1_noMX (ATAC-seq): AATGATACGG<br>CGACCACCGAGATCTACACTCGTCGG<br>CAGCGTCAGATGTG</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Corces et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Corces, M.R. ∙ Trevino, A.E. ∙ Hamilton, E.G. ...</div><div><strong>An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:959-962</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85030994641" target="_blank" aria-label="Open Scopus in new tab">Scopus (1214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28846090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4396&amp;pmid=28846090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>ID3sgRNA#1: 5’-TGGCTAAGCTGAGTG<br>CCTCT-3’</td><td>Integrated DNA Technologies</td><td>Hs.Cas9.ID3.1.AA</td></tr><tr><td>ID3sgRNA#2: 5’-TGGCCAGACTGCGTT<br>CCGAC-3’</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td>SOX4 sgRNA #1: 5’-GCTGGTGCAAGA<br>CCCCGAGT-3’</td><td>Integrated DNA Technologies</td><td>Hs.Cas9.SOX4.1.AL</td></tr><tr><td>SOX4 sgRNA #2: 5’-AGGAGGCGATTC<br>CCAGCTCG-3’</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td>ID3.PCR F(genomic DNA): 5’-ATAAAGA<br>GGCGTGCCTTCCA-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>ID3.PCR.R(gDNA): R 5’- CATCCTTGCC<br>TGGGTGTTCA-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>ID3.Seq.F (gDNA): 5’-TTCTCTTTGGGG<br>CACCTCTG-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>ID3.Seq.R (gDNA): 5’-GAAGGTGGGG<br>GCCATCAG-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>SOX4.PCR.F (gDNA and cDNA): 5’-CG<br>GAGAACTCCTTCCCCAAATC-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>SOX4.PCR.R (gDNA and cDNA): 5’-CT<br>CTTTTTCTGCGCCGGTTTG-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>SOX4.Seq.F (gDNA and cDNA): 5’-CC<br>GCGAGGGTGTGAGC-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>SOX4.Seq.R (gDNA and cDNA): 5’-TG<br>TAGTCGGGGTAGTCAGCC-3’.</td><td>Genewiz</td><td>N/A</td></tr><tr><td>ID3.cDNAPCR.F (gDNA): 5’- TTGCAG<br>GTCACTGTAGCGG-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>ID3.cDNAPCR.R (gDNA): 5’- AGGCCA<br>CAAGTTCACAGTCC-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>ID3.cDNASeq.F (gDNA): 5’- TCTTTCT<br>CTTTGGGGCACCTC-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>ID3.cDNASeq.R (gDNA): 5’- TGGTGAA<br>GTCAAGTGGGCAG-3’</td><td>Genewiz</td><td>N/A</td></tr><tr><td>H3Histone Poscntrl Human F 5’-AAAGC<br>CGCTCGCAAGAGTGCG-3’</td><td>Genewiz</td><td>N/A</td></tr><tr class="rowsep"><td>H3Histone Poscntrl Human R 5’-ACTT<br>GCCTCCTGCAAAGCAC-3’</td><td>Genewiz</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr class="rowsep"><td>pTRPE M5BBz</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Unique code</td><td>This paper</td><td><a id="intref0030" href="https://github.com/bergerlabupenn/InVitroCARTexh_code_2020" target="_blank">https://github.com/bergerlabupenn/InVitroCARTexh_code_2020</a></td></tr><tr><td>R version 3.6.2</td><td>CRAN</td><td><a id="intref0035" href="https://cran.r-project.org/" target="_blank">https://cran.r-project.org/</a></td></tr><tr><td>Seurat_3.2.3</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e1821</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (7328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0040" href="https://satijalab.org/seurat/" target="_blank">https://satijalab.org/seurat/</a></td></tr><tr><td>Cell Ranger v3.1.0</td><td>10X Genomics</td><td><a id="intref0045" href="https://www.10xgenomics.com/" target="_blank">https://www.10xgenomics.com/</a></td></tr><tr><td>sctransform_0.3.2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">Hafemeister and Satija, 2019</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0050" href="https://cran.r-project.org/web/packages/sctransform/index.html" target="_blank">https://cran.r-project.org/web/packages/sctransform/index.html</a></td></tr><tr><td>Metascape</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Zhou et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Zhou, Y. ∙ Zhou, B. ∙ Pache, L. ...</div><div><strong>Metascape provides a biologist-oriented resource for the analysis of systems-level datasets</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:1523</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-09234-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85063948265" target="_blank" aria-label="Open Scopus in new tab">Scopus (7477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30944313/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-09234-6&amp;pmid=30944313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0055" href="https://metascape.org/gp/index.html#/main/step1" target="_blank">https://metascape.org/gp/index.html#/main/step1</a></td></tr><tr><td>Monocle 3</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Qiu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Qiu, X. ∙ Hill, A. ∙ Packer, J. ...</div><div><strong>Single-cell mRNA quantification and differential analysis with Census</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:309-315</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4150" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85010878111" target="_blank" aria-label="Open Scopus in new tab">Scopus (902)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28114287/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4150&amp;pmid=28114287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68-1" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-1" aria-controls="sref68-1">Trapnell et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref68-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Trapnell, C. ∙ Cacchiarelli, D. ∙ Grimsby, J. ...</div><div><strong>The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:381-386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84900873950" target="_blank" aria-label="Open Scopus in new tab">Scopus (3557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2859&amp;pmid=24658644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0060" href="https://www.bioconductor.org/packages/release/bioc/html/monocle.html" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/monocle.html</a></td></tr><tr><td>Cellfishing.jl</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Sato et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Sato, K. ∙ Tsuyuzaki, K. ∙ Shimizu, K. ...</div><div><strong>CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1639-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061352687" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30744683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1639-x&amp;pmid=30744683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0065" href="https://github.com/bicycle1885/CellFishing.jl" target="_blank">https://github.com/bicycle1885/CellFishing.jl</a></td></tr><tr><td>samtools v1.1</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Li et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The Sequence Alignment/Map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (38872)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0070" href="http://www.htslib.org/download/" target="_blank">http://www.htslib.org/download/</a></td></tr><tr><td>STAR v2.5.2a</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (27081)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0075" href="https://github.com/alexdobin/STAR" target="_blank">https://github.com/alexdobin/STAR</a></td></tr><tr><td>HTSeq v0.6.1</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Anders et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anders, S. ∙ Pyl, P.T. ∙ Huber, W.</div><div><strong>HTSeq--a Python framework to work with high-throughput sequencing data</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:166-169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu638" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84928987900" target="_blank" aria-label="Open Scopus in new tab">Scopus (13514)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25260700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu638&amp;pmid=25260700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0080" href="https://htseq.readthedocs.io/en/master/install.html" target="_blank">https://htseq.readthedocs.io/en/master/install.html</a></td></tr><tr><td>HOMER v4.6</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Heinz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (8027)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0085" href="http://homer.ucsd.edu/homer/introduction/install.html" target="_blank">http://homer.ucsd.edu/homer/introduction/install.html</a></td></tr><tr><td>bowtie2 v2.3.4.1</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Langmead and Salzberg, 2012</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (33805)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0090" href="http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml</a></td></tr><tr><td>FlowJo™ v10.8 Software</td><td>BD Life Sciences</td><td><a id="intref0095" href="https://www.flowjo.com" target="_blank">https://www.flowjo.com</a></td></tr><tr class="rowsep"><td>Ingenuity Pathway Analysis Software</td><td>QIAGEN</td><td><a id="intref0100" href="https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/analysis-and-visualization/qiagen-ipa/" target="_blank">https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/analysis-and-visualization/qiagen-ipa/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>CRISPick sgRNA designer ([2020])</td><td>The Broad Institute</td><td><a id="intref0105" href="https://portals.broadinstitute.org/gppx/crispick/public" target="_blank">https://portals.broadinstitute.org/gppx/crispick/public</a></td></tr><tr><td>Benchling sgRNA designer tool (<a id="intref0110" href="https://www.benchling.com" target="_blank">https://www.benchling.com</a>, [2020])</td><td>Benchling</td><td><a id="intref0115" href="https://benchling.com/" target="_blank">https://benchling.com/</a></td></tr><tr><td>Synthego’s Performance Analysis ICE (short for Inference of CRISPR Edits) tool</td><td>Synthego</td><td><a id="intref0120" href="https://ice.synthego.com/[2021])" target="_blank">https://ice.synthego.com/[2021])</a>.</td></tr><tr><td>BioRender illustration design tool</td><td>BioRender</td><td><a id="intref0125" href="https://biorender.com/" target="_blank">https://biorender.com/</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2901331-3" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0280" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Dr. Carl June (<a id="intref0130" href="mailto:cjune@upenn.edu">cjune@upenn.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0285" role="paragraph">CAR construct used in this study will be provided under a material transfer agreement. sgRNAs and primer sequences generated in this study are provided in the Key resources table. Anti M5 idiotype antibody was provided by Novartis.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Cell lines</h4><div id="p0290" role="paragraph">AsPC-1, K562, and HEK293T cells were obtained from American Type Culture Collection (ATCC). AsPC-1 cells were grown in D20 media consisting of DMEM/F12 (1:1) (Gibco, Life Technologies), 20% fetal bovine serum (FBS) and 1% penicillin/streptomycin (Gibco, Life Technologies) and K562 and HEK293T cells were cultured in R10 media consisting of RPMI-1640 (Gibco, Life Technologies) with 10% FBS, 2% HEPES (Gibco), 1% of GlutaMAX™ (Gibco), and 1% of penicillin/streptomycin. GFP-expressing cell lines were generated by lentiviral transduction for cell killing assays. All cell lines were routinely authenticated by the University of Arizona Genetics Core and tested for mycoplasma contamination (MycoAlert Mycoplasma Detection Kit, Lonza).</div></section><section id="sec-4-3-2"><h4>Mice</h4><div id="p0295" role="paragraph">Animal experiments were performed according to protocols approved by the Institutional Animal Care and Use Committee of the University of Pennsylvania. Six- to eight week-old male NOD/scid/IL2rγ−/− (NSG) were procured from Jackson Laboratories and bred in the vivarium at the University of Pennsylvania in pathogen-free conditions. Mice were maintained under pathogen free conditions.</div></section><section id="sec-4-3-3"><h4>Human samples</h4><div id="p0300" role="paragraph">Healthy donor primary T lymphocytes were provided by the University of Pennsylvania Human Immunology Core. Samples are deidentified for compliance with HIPAA rules. Donor sex and age is shown below: ND516 (female, age 37), ND538 (female, 48), ND388 (male, 53), ND534 (male, 28), ND150 (male, 40), ND552 (female, 26), ND539 (male, 39), ND566 (female, 26).</div><div id="p0305" role="paragraph">Post-CAR19 infusion PBMCs samples were collected from DLBCL patients who were enrolled in CTL019 clinical trial NCT02030834. Patients enrolled in this trial had received previous primary and salvage therapies, relapsed/residual disease after autologous stem-cell transplantation, or were not eligible for autologous or allogeneic stem-cell transplantation. Post-M5CAR infusion peritoneal/pleural fluid samples were collected from ovarian cancer patients (02916-01 and 02916-06) enrolled on a M5CAR T&nbsp;cell trial (NCT03054298). Patients enrolled in this trial had recurrent disease after at least one prior standard of care chemotherapy for advanced stage disease.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>General cell culture</h4><div id="p0310" role="paragraph">AsPC-1, K562, and HEK293T cells were obtained from American Type Culture Collection (ATCC). AsPC-1 cells were grown in D20 media consisting of DMEM/F12 (1:1) (Gibco, Life Technologies), 20% fetal bovine serum (FBS) and 1% penicillin/streptomycin (Gibco, Life Technologies) and K562 and HEK293T cells were cultured in R10 media consisting of RPMI-1640 (Gibco, Life Technologies) with 10% FBS, 2% HEPES (Gibco), 1% of GlutaMAX™ (Gibco), and 1% of penicillin/streptomycin. GFP-expressing cell lines were generated by lentiviral transduction for cell killing assays. All cell lines were routinely authenticated by the University of Arizona Genetics Core and tested for mycoplasma contamination (MycoAlert Mycoplasma Detection Kit, Lonza).</div></section><section id="sec-4-4-2"><h4>Lentiviral vector production</h4><div id="p0315" role="paragraph">Lentiviral vector production was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Kutner et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Kutner, R.H. ∙ Zhang, X.Y. ∙ Reiser, J.</div><div><strong>Production, concentration and titration of pseudotyped HIV-1-based lentiviral vectors</strong></div><div><em>Nat. Protoc.</em> 2009; <strong>4</strong>:495-505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2009.22" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-64749107970" target="_blank" aria-label="Open Scopus in new tab">Scopus (509)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19300443/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2009.22&amp;pmid=19300443" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, HEK293T cells were transfected with lentiviral CAR and packaging plasmids using Lipofectamine 2000 or Lipofectamine 3000 (Invitrogen) following the manufacturer’s protocol. Lentiviral supernatants were collected at 24- and 48-hours post-transfection and concentrated using high-speed ultracentrifugation. To generate the lentiviral stocks, the resulting concentrated lentivirus batches were resuspended in cold R10 media and stored at -80°C.</div></section><section id="sec-4-4-3"><h4>Transduction of CAR-redirected human T&nbsp;cells</h4><div id="p0320" role="paragraph">The M5CAR is a second-generation CAR containing a human MSLN-binding scFv and CD8α hinge and transmembrane domains fused to 4-1BB and CD3-ζ cytoplasmic signaling domains. Primary human CD4+ T and CD8+ T&nbsp;cells from normal donors were provided by University of Pennsylvania Human Immunology Core. CAR T&nbsp;cells were generated as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Carpenito et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Carpenito, C. ∙ Milone, M.C. ∙ Hassan, R. ...</div><div><strong>Control of large, established tumor xenografts with genetically retargeted human T&nbsp;cells containing CD28 and CD137 domains</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2009; <strong>106</strong>:3360-3365</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0813101106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-62549097817" target="_blank" aria-label="Open Scopus in new tab">Scopus (710)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19211796/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0813101106&amp;pmid=19211796" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, CD4+ and CD8+ T at 1 : 1 ratio at 1&nbsp;× 10e<sup>6</sup> cells/ml were activated with Dynabeads® CD3/CD28 CTS™ (Thermofisher) at a 3 : 1 bead-to-cell ratio. Approximately after 24 hours, T&nbsp;cells were transduced at a multiplicity of infection (MOI) of 3 to 5. At day 5 beads were removed from cultures. T&nbsp;cell cultures were maintained at 8&nbsp;× 10e<sup>5</sup> cells/ml. Cell number and volume were monitored daily using Multisizer 4 Coulter Counter (Beckman). Transduced T&nbsp;cells were cryopreserved when reached the resting state, as determined by cell size.</div></section><section id="sec-4-4-4"><h4>CAR T&nbsp;cell in&nbsp;vitro dysfunction model</h4><div id="p0325" role="paragraph">AsPC-1 cells were routinely seeded in 6-well plates at 1&nbsp;× 10e<sup>6</sup> cells/well the day preceding T&nbsp;cell seeding. M5CAR T&nbsp;cells (30 - 50% of transduction efficiency) were thawed and rested at 1&nbsp;× 10e<sup>6</sup> cells/ml in T75 flasks with R10 media. After 24 hours, the T&nbsp;cell number (CD45+EpCAM-) was calculated and 2.5&nbsp;× 10e<sup>5</sup> T&nbsp;cells/well were transferred to the AsPC-1 plates. After 3 - 4&nbsp;days, the cocultures were thoroughly suspended by frequent pipetting and 300 - 400μl of the cell suspension was used for T&nbsp;cell counting assessment and flow cytometry staining. The remaining cell suspension was spun down and the supernatant (conditioned media) was collected and filtered with a 0.45&nbsp;μm filter (Corning). The cells were resuspended in media containing equal amounts of conditioned and fresh R10. The resulting T&nbsp;cell suspension was transferred into AsPC-1-coated plates cells (2.5&nbsp;× 10e<sup>5</sup> T&nbsp;cells/well) for continuous co-culture. This process was repeated for 20-35&nbsp;days.</div></section><section id="sec-4-4-5"><h4>Flow cytometry and sorting</h4><div id="p0330" role="paragraph">For flow cytometry and sorting assays of CAE, cell suspensions from M5CAR T&nbsp;cell expansions, in&nbsp;vitro cocultures and recurrent AsPC-1 tumors were stained in fluorescence-activated cell sorting (FACS) buffer consisting of PBS (Gibco), 0.5% bovine serum albumin (BSA) (GEMINI), 2&nbsp;mM EDTA (Invitrogen), and 100&nbsp;μg/ml DNase (Roche). CountBright™ Absolute Counting Beads, (ThermoFisher) were used as an internal standard according to the manufacturer's instructions to calculate absolute cell counts in cell suspensions. Antibodies used for surface and intracellular stainings are detailed in the Key resources table. M5CAR expression was assessed using biotinylated goat anti-human IgG F(ab’)<sub>2</sub> (Jackson ImmunoResearch, 109-066-006) followed by streptavidin (FITC-, AF488- or APC-conjugated, see the Key resources table) or using an anti-idiotype antibody provided by Novartis Pharmaceuticals. Live/dead staining was performed using a Live/Dead Aqua (Life Technologies) or Zombie NIR (Biolegend) Fixable Viability Kits following manufacturer’s protocol followed by cell surface staining for 15&nbsp;min at 4°C in the dark. Apoptosis was assessed using Live/Dead Aqua and Apotracker™ Green according to manufacturer’s instructions Intracellular staining was performed with the True Nuclear and Foxp3/Transcription Factor Staining Buffer set (Thermo Fisher) according to the manufacturer’s instructions. Samples were acquired on an LSRII Fortessa Cytometer (BD Bioscience) and analyzed with FlowJo v10 software (FlowJo, LLC). Sorting assays were performed using a FACS Aria Cytometer (BD Bioscience).</div></section><section id="sec-4-4-6"><h4>CD56+ cell depletion</h4><div id="p0335" role="paragraph">MACS Dead cell removal kit and CD56 MicroBeads (Miltenyi Biotec) were used for CD56-positive cell depletion on day 0 CAR T&nbsp;cell products. The CD56-depleted CAR T&nbsp;cell product was subjected to CAE protocol as described above and the frequency of CD56+ T&nbsp;cells was assessed by flow cytometry.</div><div id="p0340" role="paragraph">The out-competition model assumes that initial depletion of the NK-like-T cell population would result in altered kinetics of NK-like-T cell abundance over time compared to a non-depleted control group, whereas transitioning assumes similar kinetics between the control and depleted groups. As shown in <a id="crosref1585" href="#figs6">Figure&nbsp;S6</a>F, in case of out-competition by the CD56-positive cell subset (left panel), the frequency of CD56 in the CD56-depleted cultures increase at a lower rate than in the controls. This growth can be expressed by the formula <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P upper T equals left parenthesis upper P Baseline 0 minus d right parenthesis times k t" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(21 (17 0 16 1) 2 (20 (12 3 (10 (9 4 8 5) 6 7) 11) 13 (19 14 18 15)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="21" data-semantic-children="17,20" data-semantic-content="2" data-semantic-owns="17 2 20" data-semantic-speech="upper P upper T equals left parenthesis upper P Baseline 0 minus d right parenthesis times k t"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="17" data-semantic-children="0,1" data-semantic-content="16" data-semantic-parent="21" data-semantic-owns="0 16 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="17"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="17"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="21" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-children="12,19" data-semantic-content="13" data-semantic-parent="21" data-semantic-owns="12 13 19"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="3,11" data-semantic-parent="20" data-semantic-owns="3 10 11"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="10" data-semantic-children="9,7" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="9 6 7"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-children="4,5" data-semantic-content="8" data-semantic-parent="10" data-semantic-owns="4 8 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="9"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="10" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="10"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="20" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="19" data-semantic-children="14,15" data-semantic-content="18" data-semantic-parent="20" data-semantic-owns="14 18 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="19"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="19"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. On the other hand, if T&nbsp;cell are transitioning into NK-like-T cells, (S6F, right panel), the frequency of CD56 in the cocultures would increase at the same rate over the time, independently of the initial depletion of the CD56 at the start of the coculture, which can be expressed as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P upper T equals left parenthesis upper P Baseline 0 minus d right parenthesis plus k times t" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(21 (18 0 17 1) 2 (19 (12 3 (10 (9 4 8 5) 6 7) 11) 13 (20 14 15 16)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="21" data-semantic-children="18,19" data-semantic-content="2" data-semantic-owns="18 2 19" data-semantic-speech="upper P upper T equals left parenthesis upper P Baseline 0 minus d right parenthesis plus k times t"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="18" data-semantic-children="0,1" data-semantic-content="17" data-semantic-parent="21" data-semantic-owns="0 17 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="18"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="18"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="21" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="19" data-semantic-children="12,20" data-semantic-content="13" data-semantic-parent="21" data-semantic-owns="12 13 20"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="3,11" data-semantic-parent="19" data-semantic-owns="3 10 11"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="10" data-semantic-children="9,7" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="9 6 7"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-children="4,5" data-semantic-content="8" data-semantic-parent="10" data-semantic-owns="4 8 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="9"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="10" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="10"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="13" data-semantic-parent="19" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-children="14,16" data-semantic-content="15" data-semantic-parent="19" data-semantic-owns="14 15 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="20"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="20" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="20"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. PT: percentage CD56-positive cells at time “T”. P0: Percentage CD56-positive cells at time zero. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="t"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>: time of in&nbsp;vitro stimulation [Days]. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="k" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="k"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>: transition constant. D: percentage CD56-positive cells depleted.</div></section><section id="sec-4-4-7"><h4>Clinical trial design and research participants</h4><div id="p0345" role="paragraph">Single-institution pilot safety and feasibility trial was conducted at University of Pennsylvania.</div><div id="p0350" role="paragraph">This study is registered at <a id="interref0010" href="https://clinicaltrials.gov/" target="_blank">https://clinicaltrials.gov/</a> as #<a id="interref0015" href="https://clinicaltrials.gov/show/NCT03054298" target="_blank">NCT03054298</a>. 1 – 3&nbsp;× 10e<sup>7</sup> M5CAR T&nbsp;cells/m<sup>2</sup> were intravenously infused into patients who were diagnosed with ovarian cancer. Pleural fluid (patient 06) or peritoneal fluid (patient 01) were collected (06: day 36, 01: day 21) and surface and intracellular CAR expression was analyzed by flow cytometry. PBMCs collected from patients who received CD19CAR (CTL019) T&nbsp;cells to treat DLBCL (<a id="PC_link9nzRYPhoiS" href="https://clinicaltrials.gov/ct2/results?cond=&amp;term=NCT02030834&amp;cntry=&amp;state=&amp;city=&amp;dist=" target="_blank">NCT02030834</a>) and CTL019 T&nbsp;cell products were used for identifying NK-like CAR T&nbsp;cells in human. Fifty-two DLBCL patients were enrolled and 35 patients were excluded as CD56 expression was not examined. CTL019 T&nbsp;cell expansion in the patient’s blood was analyzed by qPCR and the peak time point of expansion was selected to examine the frequency of NK-like CAR T&nbsp;cells. To investigate the expression of NK-related molecules on CAR T&nbsp;cells, cryopreserved materials from patient 13413-39 (CTL019 T&nbsp;cell product and PBMCs collected 27&nbsp;days after CAR T infusion) were thawed and analyzed by flow cytometry.</div></section><section id="sec-4-4-8"><h4>Production of Human CRISPR-engineered CAR-T cells</h4><div id="p0355" role="paragraph">Single guide RNA (sgRNA) sequences targeting ID3 and SOX4 were designed using CRISPick sgRNA designer (<a id="interref0020" href="https://portals.broadinstitute.org/gppx/crispick/public" target="_blank">https://portals.broadinstitute.org/gppx/crispick/public</a>) and Benchling online software (<a id="intref0155" href="https://www.benchling.com" target="_blank">https://www.benchling.com</a>) and were synthesized by Integrated DNA Technologies (IDT). Two of five sgRNAs targeting each gene were selected for further experiments after pre-validation. Gene disruption, T&nbsp;cell activation, transduction, expansion, and knockout validation of ID3KO and SOX4KO M5 CAR T&nbsp;cells were performed following an optimized protocol previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Agarwal et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Agarwal, S. ∙ Wellhausen, N. ∙ Levine, B.L. ...</div><div><strong>Production of Human CRISPR-Engineered CAR-T Cells</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3791/62299" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85103610405" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33779622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3791%2F62299&amp;pmid=33779622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, CD4+ and CD8+ T at 1 : 1 ratio were incubated in OpTmizer T Cell expansion media (Gibco) supplemented with 5&nbsp;ng/mL of huIL-7 and huIL-15 each(Preprotech) (OPT 7/15 media). After 24h, cells were collected and resuspended at 1x 10e<sup>8</sup> cells/mL in P3 nucleofection solution (Lonza). The ribonucleoprotein (RNP) complexes were generated by incubating each sgRNA (5&nbsp;μg per 10 x 10e<sup>6</sup> cells) individually with the Cas9 nuclease (Aldevron, 10&nbsp;μg per 10 x 10e<sup>6</sup> cells) for 10&nbsp;min at room temperature. Cells were electroporated in batches of 10 x 10e<sup>6</sup> cells (100&nbsp;μL) with a mixture of RNP complex plus 16.8 pmol of electroporation enhancer (IDT) into electroporation cuvettes (electroporation code EH111) in a 4D-Nucleofactor X-Unit (Lonza). After electroporation cells were grown in OPT 7/15 media at 5 x 10e<sup>6</sup>cells/mL at 37°C and activated 4 to 6h later with anti-CD3/anti-CD28 monoclonal antibody-coated magnetic beads. After 24&nbsp;h, T-cell were lentivirally transduced and expanded as described above.</div><div id="p0360" role="paragraph">Since each target locus was defined by two sgRNA cut sites (spanned 100 and 130&nbsp;bp for SOX4 and ID3, respectively), PCR primers and sequencing primers were designed to detect each target locus. LongAmp™ Taq 2X Master Mix (NEB) was used for target sequence amplification and used following manufacturer’s protocol and NucleoSpin Gel and PCR Clean-up (Macherey-Nagel) was used for DNA purification. Analysis of gene editing efficiency was assessed by Sanger sequencing. We obtained two sets of KO T&nbsp;cells per group: one bearing small insertions and deletions due to a single sgRNA hit, and a second population of CAR T&nbsp;cells bearing a large fragment deletion as a result of a double sgRNA hit. Synthego’s Performance Analysis ICE (short for Inference of CRISPR Edits) tool, was used to calculate the editing efficiency (<a id="intref0160" href="https://ice.synthego.com/" target="_blank">https://ice.synthego.com/</a> [2021]). The sequences used for KO generation and editing efficiency validation are listed in the Key resources table. The schematic representation of the in&nbsp;vivo experiments of <a id="crosref1595" href="#fig4">figure 4</a> and the knockout strategy in figure panel 7A were created using <a id="intref0165" href="http://BioRender.com" target="_blank">BioRender.com</a>.</div></section><section id="sec-4-4-9"><h4>Cytotoxicity assays</h4><div id="p0365" role="paragraph">Cytotoxic killing of target cells was assessed using a real-time, impedance-based assay with xCELLigence Real-Time Cell Analyzer System (ACEA Biosciences). Briefly, 1 x 10e<sup>4</sup> AsPC-1 cells were seeded to the 96-well E-plate. After 24 hours, sorted CD8+ CAE surCARpos T&nbsp;cells (day 28 CAE, day 0 product and CD19BBz) were added to the wells in 4 : 1 E:T ratio. Tumor killing was monitored every 20&nbsp;min over 4&nbsp;days. To evaluate the cell killing capacity of WT, ID3 KO and SOX4 KO M5CAR T&nbsp;cells upon CAE, cells were collected on day 18 for ND539 and day 21 and day 28 for ND566 and seeded at 1:8 E:T ratio with AsPC-1. Tumor killing was monitored every 20&nbsp;min over 8&nbsp;days. Significant differences between groups were assessed by two-way ANOVA and Dunnet’s post hoc test.</div><div id="p0370" role="paragraph">High-throughput cytotoxicity assay using Celigo Image Cytometer (Nexcelom Bioscience) was used to investigate the effects of the resting with cytokine supplement on cytotoxicity of CAR T&nbsp;cells. CD8+ M5CAR T&nbsp;cells were sorted after CAE, counted and the viability assessed using Moxi Flow System (Orflo Technologies). Part of the cell suspension was cocultured with 1.5&nbsp;× 10e<sup>3</sup>-2&nbsp;× 10e<sup>3</sup> AsPC-1-GFP cells immediately after sorting in a 7 : 1 E:T ratio and the rest was left resting at 1.0× 10e<sup>6</sup> cells/ml in fresh R10 media with IL-15 supplement (20&nbsp;ng/ml). After 24&nbsp;hr, cell viability was examined and rested T&nbsp;cells were cocultured with AsPC-1-GFP cells in identical conditions as the non-rested counterparts. The % lysis was calculated by direct cell counting of live fluorescent target cells. % Lysis&nbsp;= (1 – count # of live target cells (GFP) in wells with effector cells / count # of live target cells (GFP) in wells without effector cells) × 100.</div></section><section id="sec-4-4-10"><h4>Cytokine production</h4><div id="p0375" role="paragraph">Fifty thousand CD8+ surCARpos T&nbsp;cells (day 28 CAE, day 0 product and CD19BBz) were cocultured with 5&nbsp;× 10e<sup>4</sup> AsPC-1 cells or left in R10 media in 48 well plate. After 48&nbsp;hr, supernatant was collected and analyzed by high-sensitivity LUMINEX assay according to manufacturer’s instructions (Merck Millipore).</div></section><section id="sec-4-4-11"><h4>Quantitative real-time PCR (qPCR)</h4><div id="p0380" role="paragraph">Surface CAR-positive and -negative CD8+ T&nbsp;cells were sorted on days 4, 7, and 17 after CAE and genomic DNA was isolated from sorted cell pellets using an ArcturusTM PicoPureTM DNA Extraction Kit (Applied Biosystems). qPCR was performed in triplicate with TaqMan Gene Expression Master Mix on a 7500Fast Real-Time PCR System (Applied Biosystems) per the manufacturer’s instructions. The validated primers specific to the 4–1BB and CD3ζ fusion gene and probes specific for the fusion fragment and labeled with compatible reporter dyes (FAM or VIC) were used to detect the CAR. The average plasmid copy per cell was calculated based on the factor 0.0063&nbsp;ng /cell. Nine μL DNA was loaded directly for quantitation by p21 qPCR. A correction factor (CF) was not used for calculating the average % marking and copies/μg DNA as the amount of actual DNA loaded was accurately quantified by p21.</div></section><section id="sec-4-4-12"><h4>CAR re-expression assay</h4><div id="p0385" role="paragraph">SurCARneg CD8+ T&nbsp;cells were sorted after 23&nbsp;days of CAE, rested in fresh R10 media with IL-15 supplement (20&nbsp;ng/ml) for 24&nbsp;hrs and examined for surface CAR expression by flow cytometry.</div></section><section id="sec-4-4-13"><h4>CyTOF</h4><div id="p0390" role="paragraph">Mass cytometry antibodies were obtained as pre-conjugated metal-tagged antibodies from Fluidigm or prepared using the Maxpar antibody conjugation kit (Fluidigm) according to the manufacturer’s protocol. Following labeling, antibodies were diluted in Candor PBS Antibody Stabilization solution (Candor Bioscience GmbH, Wangen, Germany) supplemented with 0.02% NaN<sub>3</sub> to 0.25&nbsp;mg/mL and stored long-term at 4º C. Each antibody was titrated to optimal staining concentrations using primary human PBMCs.</div><div id="p0395" role="paragraph">CAE CD8<sup>+</sup> CART cells and CD8<sup>+</sup> CART product were washed and resuspended 1:1 with PBS containing EDTA and 20&nbsp;μM cisplatin for 2&nbsp;min before quenching 1:1 with CSM (cell staining medium: PBS with 0.5% BSA and 0.02% NaN<sub>3</sub>) for dead cell discrimination. After washed, the cells were fixed for 10&nbsp;min at RT using 1.6% paraformaldehyde (PFA) in PBS and frozen in CSM with 10% DMSO at -80ºC. CAE CD8<sup>+</sup> CART cells and CD8<sup>+</sup> CART product were barcoded with distinct combinations of stable Pd isotypes in Barcode Perm Buffer (Fluidigm). Cells were washed twice with CSM, and once with PBS, and pooled into a single tube. Cells were blocked with human FcR blocking reagent (BD Bioscience) for 10&nbsp;min at RT. Cells were then incubated with all antibodies targeting cell surface markers for 30&nbsp;min at RT. After washed, cells were fixed with 1.6% PFA and permeabilized with Perm-S buffer (Fluidigm). Fixed/permeabilized cells were incubated with all antibodies targeting intracellular antigens for 30&nbsp;min at room temperature. After washed with CSM, cells were incubated in 4% PFA in PBS with 191/193-iridium intercalator (Fluidigm) for 48&nbsp;hrs. Cells were washed in CSM, PBS, and then deionized H<sub>2</sub>O. Cells were resuspended in deionized H<sub>2</sub>O containing EQ four-element beads (Fluidigm) to approximately 1 x 10<sup>6</sup> cells and then analyzed on Helios CyTOF system (Fluidigm) at Flow Cytometry Core, University of California, San Francisco. The acquisition data were normalized with premessa package and analyzed with cytofkit package (27662185) in R software 3.6.1 (The R foundation for Statistical Computing, Vienna, Austria).</div></section><section id="sec-4-4-14"><h4>Mouse experiments</h4><div id="p0400" role="paragraph">NOD/scid/IL2rγ−/− (NSG) mice were purchased from The Jackson Laboratory and bred and housed in the vivarium at the University of Pennsylvania in pathogen-free conditions. Animal studies were approved by the University of Pennsylvania Institutional Animal Care and Use Committee.</div><div id="p0405" role="paragraph">For the NY-ESO-1 TIL mouse model, five million A549-A2-ESO tumor cells in 150&nbsp;μl of Matrigel:PBS (1:1) solution were subcutaneously injected in the flanks of NSG mice. 2&nbsp;× 10e<sup>7</sup> human T&nbsp;cells were activated with anti-CD3&nbsp;+ anti-CD28 microbeads 3:1 and subsequently transduced with 3<sup>rd</sup> generation high titer lentivirus encoding for the Ly95 TCR. Transduced cells (50% of which were positive for Ly95 TCR) were intravenously injected when tumors reached a mean volume of 150&nbsp;mm<sup>3</sup>. Thirty days after T&nbsp;cell injection, mice were sacrificed, tumors were harvested, digested, and processed. The single-cell suspension obtained was then treated with Dead Cell Removal Kit (Miltenyi Biotec) following manufacturer’s protocol, and CD3+ cells were positively selected by using an EasySep cell isolation kit (Stem Cell Technologies). The non-transduced CD8+ T&nbsp;cells from the same donor and the transduced NY-ESO-1 redirected infusion product were also subjected to the same digestion and processing protocols.</div><div id="p0410" role="paragraph">T cells from the tumor cell suspension were stained with anti-human CD8 and anti-TCRVβ13.1 The donor’s CD8+ T&nbsp;cells were stained with anti-CD8 and anti-CD45RO. NY-ESO-1 T&nbsp;cell infusion product was stained with anti-CD8 and anti-TCRVβ13.1. All three specimens were flow sorted on the BD FACS Aria on the same day for the following populations: CD45 cells isolated from tumor digest – CD8+/ TCRVβ13.1+, donor’s untransduced CD8+ T&nbsp;cells – CD8+/CD45RO+, NY-ESO-1 T&nbsp;cell infusion product – CD8+/ TCRVβ13.1+. Sorted samples were snap frozen, subjected to RNA extraction with Qiazol (Qiagen) and gene expression microarray analyses. For genes with multiple probes, average expression values were used to make the heatmap in R (pheatmap).</div><div id="p0415" role="paragraph">For the AsPC-1 recurrence model, NSG mice were subcutaneously injected with 2&nbsp;× 10e<sup>6</sup> AsPC-1 cells suspended in 200&nbsp;μl Matrigel:PBS (1:1) into the right flank. When the mean of tumor volumes reached 300&nbsp;mm<sup>3</sup>, mice were treated with 1 x 10e<sup>6</sup> ND552 M5CAR+ T&nbsp;cells. Tumor volumes were calculated as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l e n g t h times w i t h Superscript 2 divided by 2" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 (21 0 16 1 17 2 18 3 19 4 20 5) 6 (25 7 22 8 23 9 24 (15 10 (14 11 12 13))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="26" data-semantic-children="21,25" data-semantic-content="6" data-semantic-owns="21 6 25" data-semantic-speech="l e n g t h times w i t h Superscript 2 divided by 2"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="21" data-semantic-children="0,1,2,3,4,5" data-semantic-content="16,17,18,19,20" data-semantic-parent="26" data-semantic-owns="0 16 1 17 2 18 3 19 4 20 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="21"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="21"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="21"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="21"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="21"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="21"><mjx-c class="mjx-c210E">ℎ</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="26" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="25" data-semantic-children="7,8,9,15" data-semantic-content="22,23,24" data-semantic-parent="26" data-semantic-owns="7 22 8 23 9 24 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="25"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="25"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="25"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="10,14" data-semantic-parent="25" data-semantic-owns="10 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="15"><mjx-c class="mjx-c210E">ℎ</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="14" data-semantic-children="11,13" data-semantic-content="12" data-semantic-parent="15" data-semantic-owns="11 12 13"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="12" data-semantic-parent="14" data-semantic-operator="infixop,÷"><mjx-c class="mjx-cF7">÷</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Tumor growth was weekly assessed by caliper measurement. After primary antitumor response mice were monitored for recurrence. Mice bearing recurrent tumors were sacrificed when reached the maximum size or showed evident signs of disease, and tumors were collected. Fresh tumors were excised and digested in RPMI containing collagenase D (400 Mandl units/mL, Sigma) and DNase I (50&nbsp;mg/mL, Sigma) for 15&nbsp;minutes at 37ºC. Enzymatic digestion was stopped with 12&nbsp;μL/mL EDTA d 0.5 M, pH 8. Tumors were mechanically disrupted and filtered through a 0.7&nbsp;μm cell strainer (Corning). For flow cytometry stainings, single-cell suspensions were stained with Fixable Dead Cell Dyes followed by FcR-Block treatment (Fc Receptor Blocking Solution, Biolegend) following manufacturer’s recommendations. Positive NK receptor cell subsets in D0 and recurrent samples were determined in sample-matched tumor and Day 0 FMO controls. Positive checkpoint receptor subsets were determined sample-matched tumor and Day 0 isotype controls. for checkpoint receptors. All the isotype controls were incubated at the same final concentration as their corresponding test antibody.</div></section><section id="sec-4-4-15"><h4>Single-cell RNA-seq and TCR-seq</h4><div id="p0420" role="paragraph">ScRNA-seq libraries were generated using a Chromium Single-Cell 3’ Library and Gel Bead Kit (10x Genomics) using v3 for CAR T donor ND388 and v3.1 for donors ND539 and ND566 following the manufacturer’s protocol. Briefly, 16,000 CD8+ T&nbsp;cells were sorted by flow cytometry and washed with ice cold PBS&nbsp;+ 0.04% BSA. After washing, cells were used to generate single-cell gel beads in emulsion. Following reverse transcription, gel beads in emulsion were disrupted and barcoded complementary DNA was isolated and amplified by PCR for 12 cycles. After fragmentation, end repair, and poly A tailing, samples indexes were added and amplified following the manufacturer’s protocol. The final libraries were quality control checked and sequenced on an Illumina NextSeq 500 with a 150-cycle kit with parameters Read 1: 28, Read 2: 130, Index 1: 8, Index 2: 0. One sample was sequenced per flow cell. For CAR T donors ND150 and ND538, scRNA-seq libraries were generated using Chromium Single-Cell 5’ Library and Gel Bead Kit and TCR libraries were generated using Chromium Single-Cell V(D)J Reagent Kits (10x Genomics) according to the manufacturer’s protocol. Followed same brief protocol as above except amplified cDNA by PCR for 13 cycles. Two uL of post amplified cDNA was used to generate TCR libraries and 50ng of cDNA was used to generate 5’ gene expression libraries. After fragmentation, end repair and poly A tailing, sample indexes were added and amplified following manufacturer’s protocol. The libraries were sequenced on an Illumina NextSeq 500 with a 150-cycle kit with parameters Read 1: 26, Read 2: 130, Index 1: 8, Index 2: 0. One RNA library and one TCR library (8:1 ratio) were pooled and sequenced on one flow cell.</div></section><section id="sec-4-4-16"><h4>Single-cell RNA-seq analysis</h4><div id="p0425" role="paragraph">Sequencing data were aligned to the GRCh38 genome, filtered, and then barcodes and unique molecular identifiers were counted using the Cell Ranger v3.1.0 command cellranger count. Data were further analyzed in R using Seurat version 3.1.2 (<span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e1821</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (7328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, genes that were not detected in at least 3 cells and cells with &gt;5% mitochondrial reads were excluded, as well as cells that express &lt;200 genes or &gt;5000 genes. Data were normalized using sctransform (<span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">Hafemeister and Satija, 2019</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). PCA was performed on the most variable genes which were found based on average expression and variance. Clusters and UMAP were generated from the first 10 PCA dimensions using the default parameter settings in Seurat. Marker genes were determined using the FindAllMarkers function in Seurat where at least 25% of the cells must be expressing the gene. Sctransform normalized expression was used for the heatmap of marker genes, UMAP feature plots, and dot plots. Metascape was used with cluster marker genes for gene ontology analysis (<span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Zhou et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Zhou, Y. ∙ Zhou, B. ∙ Pache, L. ...</div><div><strong>Metascape provides a biologist-oriented resource for the analysis of systems-level datasets</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:1523</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-09234-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85063948265" target="_blank" aria-label="Open Scopus in new tab">Scopus (7477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30944313/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-09234-6&amp;pmid=30944313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Monocle 3 was used for trajectory analysis with the default parameter setting and 100 PCs (<span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Qiu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Qiu, X. ∙ Hill, A. ∙ Packer, J. ...</div><div><strong>Single-cell mRNA quantification and differential analysis with Census</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:309-315</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4150" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85010878111" target="_blank" aria-label="Open Scopus in new tab">Scopus (902)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28114287/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4150&amp;pmid=28114287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68-2" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-2" aria-controls="sref68-2">Trapnell et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref68-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Trapnell, C. ∙ Cacchiarelli, D. ∙ Grimsby, J. ...</div><div><strong>The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:381-386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84900873950" target="_blank" aria-label="Open Scopus in new tab">Scopus (3557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2859&amp;pmid=24658644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). AddModuleScore was used to project expression of the dysfunction signature genes (N=30) onto the Monocle trajectory. Gene regulatory network inference was performed using the partial information decomposition algorithm, PIDC, on the top 500 variable genes (identified via Seurat) with a threshold for edge inclusion of 15% (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Chan et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Chan, T.E. ∙ Stumpf, M.P.H. ∙ Babtie, A.C.</div><div><strong>Gene Regulatory Network Inference from Single-Cell Data Using Multivariate Information Measures</strong></div><div><em>Cell Syst.</em> 2017; <strong>5</strong>:251-267.e3</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cels.2017.08.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cels.2017.08.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85031092991" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28957658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2017.08.014&amp;pmid=28957658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cellfishing.jl, a software that builds a database from single-cell data to then be queried, was used for differential expression analysis between single cell data sets (day 0 product versus day 20 CAE cells) with the default of 10 k-nearest neighbors (<span class="dropBlock reference-citations"><a id="body-ref-sref59-3" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-3" aria-controls="sref59-3">Sato et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Sato, K. ∙ Tsuyuzaki, K. ∙ Shimizu, K. ...</div><div><strong>CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1639-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061352687" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30744683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1639-x&amp;pmid=30744683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). 1,834 genes were found to be differentially expressed. Data were analyzed using IPA (QIAGEN Inc., <a id="intref0170" href="https://www.qiagenbioinformatics.com/products/ingenuitypathway-analysis" target="_blank">https://www.qiagenbioinformatics.com/products/ingenuitypathway-analysis</a>). For IPA analysis, mitochondrial genes were filtered out and only genes with fold change &gt; 2 (N=1,442 genes) were included. Fold change was calculated as the number of cells at day 20 that upregulate the gene divided by the number of cells at day 20 that downregulate the gene compared to day 0 cells. NK-like T&nbsp;cells were identified using raw_counts ["KLRC1",]&gt;0 &amp; raw_counts["KLRB1",]&gt;0 &amp; raw_counts["CD3E",]&gt;0. Significant differences in changes in the NK-like T&nbsp;cell populations between WT and KO conditions were measured by Fisher’s exact test.</div><div id="p0430" role="paragraph">To identify our 30 gene dysfunction signature, we identified all genes differentially expressed between dysfunctional and non-dysfunctional clusters using Seurat’s FindMarkers function. For donor ND388, differentially expressed genes were identified between dysfunctional clusters D20-1 and D20-4 versus non-dysfunctional clusters D20-2 and D20-3. This list was further filtered by log2FC&gt;0.64 and padj&lt;1.0e22 (padj with Bonferroni correction using all genes in the dataset).</div><div id="p0435" role="paragraph">WT, SOX4 KO, and ID3 KO Seurat objects were combined for analysis using the merge function (for donor ND566) and WT and ID3 KO samples were combined for donor ND539. Genes that were not detected in at least 3 cells and cells with &gt;5% mitochondrial reads were excluded, as well as cells that express&nbsp;&lt; 200 genes or &gt;5000 genes. EPCAM expression (tumor marker) was used to identify a contaminating tumor cell cluster which was subsequently removed using seurat’s subset function. CellCycleScoring was used to regress out cell cycle specific clustering using SCTransform vars.to.regress (S.Score, G2M.Score) function. SCT counts of the dysfunction signature genes (N=30) were averaged per cell to create the dysfunction score. Mann-Whitney U test was used to test significance of dysfunction score between WT and KO conditions.</div><div id="p0440" role="paragraph">To assess the expression of M5CAR in the scRNA-seq data, the cellranger reference was reindexed (mkref) by adding a single contig for the 627&nbsp;bp WPRE sequence (a unique sequence in the CAR plasmid) to assembly GRCh38 of the human genome (the gene annotation GTF file was appended with CDS and exon entries spanning the entire sequence and gene_id “Ligand”). To analyze expression of CAR and to determine the percent of cells expressing the CAR (related to <a id="crosref1640" href="#fig3">Figures 3</a>H and 3I), we pooled data from three scRNA-seq experiments (ND388&nbsp;day 20 CAE cells, ND538 and ND150&nbsp;day 28 CAE cells). Cells belonging to the dysfunctional clusters and non-dysfunctional clusters were defined for each donor separately, see <a id="crosref1645" href="#fig3">Figures 3</a> and <a id="crosref1650" href="#figs5">S5</a>.</div></section><section id="sec-4-4-17"><h4>Single-cell TCR-seq analysis</h4><div id="p0445" role="paragraph">Sequencing data were aligned to the vdj-GRCh38-alts-ensembl-3.1.0 genome and processed using the cellranger vdj command in Cell Ranger v3.1.0. To assess receptor persistence, a map of full-length receptor peptide sequences to cell barcodes was loaded at both time points from filtered coverage annotation (FCA) files. Cell barcodes associated with peptide sequences in common to both time points were screened against lists of cell barcodes that express CD8A at both time points; cells without persistent CD8A expression were removed. Remaining cells were screened against barcodes of cells that express KLRB1 at either day 0 or day 28, or not at all. Sankey plots of this distribution were created using the plotly library in R. Maps were also analyzed for the number of cell barcodes associated to each full-length peptide sequence to insure that the data largely obey a one peptide : one barcode rule.</div></section><section id="sec-4-4-18"><h4>Bulk RNA-seq</h4><div id="p0450" role="paragraph">RNA-seq libraries were made following the previously established SMARTseq2 protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref54-7" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-7" aria-controls="sref54-7">Picelli et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Picelli, S. ∙ Faridani, O.R. ∙ Björklund, A.K. ...</div><div><strong>Full-length RNA-seq from single cells using Smart-seq2</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:171-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84891677425" target="_blank" aria-label="Open Scopus in new tab">Scopus (2542)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.006&amp;pmid=24385147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, total RNA was extracted using Qiazol (Qiagen) from 300 cells for day 0, day 16, and day 28 for CD8+ T&nbsp;cells continuously stimulated with antigen (two sorted populations including surface CAR-positive and surface CAR-negative cells). From the same experiment, CD4+ T&nbsp;cells were sorted, and RNA extracted for surface CAR positive cells at day 0 and day 28 CAE. Cells were recovered by RNA Clean and Concentrator spin columns (Zymo), followed by incubation with oligo-dT. The transcription reaction was carried&nbsp;out on 100 pg of cDNA for 1min at 55° C. Libraries were uniquely barcoded (<span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Buenrostro et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (4014)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and amplified for 14 cycles. Fragment size distribution was verified and paired-end sequencing (2 x 75&nbsp;bp reads) was carried out on an Illumina NextSeq 500.</div></section><section id="sec-4-4-19"><h4>Bulk RNA-seq analysis</h4><div id="p0455" role="paragraph">Paired-end data were aligned to human genome assembly GRCh37/hg19 using STAR v2.5.2a (<span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (27081)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with command-line parameters --outFilterType BySJout --outFilterMultimapNmax 20 --alignSJoverhangMin 8 --alignSJDBoverhangMin 1 --outFilterMismatchNmax 999 --alignIntronMin 20 --alignIntronMax 1000000 --alignMatesGapMax 1000000. Resulting SAM files were converted to BAM format using samtools v1.1 (samtools view -bS) and BAM files were sorted by position using samtools sort. For CD8+ T&nbsp;cell replicate R1, several libraries were pooled after alignment to enhance coverage using samtools merge as below:<div id="ulist0020" role="list"><div id="u0045" role="listitem"><div class="content"><div id="p0460" role="paragraph">R1 Control Day 0 CAR+ T1: 4-Day0-CD8-CARpos_S7, 16-Day0-CD8-CARpos_S5</div></div></div><div id="u0050" role="listitem"><div class="content"><div id="p0465" role="paragraph">R1 Control Day 0 CAR- T1: 3-Day0-CD8-CARneg_S2, 15-Day0-CD8-CARneg_S6</div></div></div><div id="u0055" role="listitem"><div class="content"><div id="p0470" role="paragraph">R1 other CAR+ T1: 21-other-CD8-CARpos-10-24-2018_S11, 12-other-CD8-CARpos_S12, 24-other-CD8-CARpos_S10</div></div></div><div id="u0060" role="listitem"><div class="content"><div id="p0475" role="paragraph">R1 CAE CAR+ T1: 8-CAE-CD8-CARpos_S9, 20-CAE-CD8-CARpos_S12</div></div></div><div id="u0065" role="listitem"><div class="content"><div id="p0480" role="paragraph">R1 CAE CAR- T1: 7-CAE-CD8-CARneg_S6, 19-CAE-CD8-CARneg_S3</div></div></div></div></div><div id="p0485" role="paragraph">HTSeq v0.6.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Anders et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anders, S. ∙ Pyl, P.T. ∙ Huber, W.</div><div><strong>HTSeq--a Python framework to work with high-throughput sequencing data</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:166-169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu638" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84928987900" target="_blank" aria-label="Open Scopus in new tab">Scopus (13514)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25260700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu638&amp;pmid=25260700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to count aligned tags over gene features with command-line python -m HTSeq.scripts.count -f bam -r pos -s no -t exon -i gene_id BAM_FILE GTF. The GTF was constructed from RefSeq transcripts and UCSC Genome Browser's annotation of RefSeq transcript IDs to gene symbols. For the antigen exposure and time series analysis, DESeq2 was used to adjust library size and estimate significant differences at an FDR of 0.05. The Wald test was used to assess differences between control day 0 and CAE. Other samples were included to adjust dispersions and library sizes but were not used for the contrast. LRT was used to assess differences along the time course (day 0, day 16, and day 28), with a full model of ∼Replicate+Time and a reduced model of ∼Replicate. For this analysis other exposure samples were not included. For the antigen exposure analysis (day 0 versus day 28 CAE, see <a id="crosref1675" href="#fig2">Figure&nbsp;2</a>A), some genes were filtered out which register as significantly different, but which may be artifacts of the SMART-seq library construction; these fall along the arcs of a parabola in a volcano plot of the data. Lines were interpolated on the plot using genes along the arcs: between <i>IL22</i> and <i>WDR63</i> (negative) and between <i>ALK</i> and <i>INBHE</i> (positive). Genes with a perpendicular distance &lt;1 to the lines were removed. For IPA pathway analysis and to overlap DEGs with other datasets, genes were further filtered for padj &lt;0.05 and fold-change &gt;2, (N=1,038 DEGs for CD8+ T&nbsp;cells and N=1,477 DEGs for CD4+ T&nbsp;cells). DESeq2 adjusted counts were used to generate gene expression plots of NK-associated molecules and DEGs (see <a id="crosref1680" href="#fig2">Figures 2</a>D and <a id="crosref1685" href="#figs3">S3</a>N). Statistics assessed by Mann-Whitney test (<sup>∗∗∗∗</sup><i>P</i> &lt;0.0001, <sup>∗∗∗</sup><i>P</i> &lt;0.001, <sup>∗∗</sup><i>P</i> &lt;0.01, <sup>∗</sup>P &lt;0.05). Statistics to compute significance of overlaps were assessed by hypergeometric test.</div><div id="p0490" role="paragraph">For IPA upstream regulator analysis, only transcription factors differentially expressed in CAE are shown (p value &lt;2e-5 and log<sub>2</sub> fold change &gt;1). However, a full list of upstream regulators can be found in <a id="crosref1690" href="#mmc3">Table S3</a>. For GSEA analyses, DESeq2 adjusted counts for all genes from CD8+ T&nbsp;cell bulk RNA-seq datasets for day 0 and day 28 CAE samples were uploaded. To perform GSEA with the four transient states of exhausted T&nbsp;cells (<a id="crosref1695" href="#figs3">Figure&nbsp;S3</a>E) we downloaded data from <a id="crosref1700" href="#mmc1">Table S1</a> (<span class="dropBlock reference-citations"><a id="body-ref-sref7-4" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-4" aria-controls="sref7-4">Beltra et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref7-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Beltra, J.C. ∙ Manne, S. ∙ Abdel-Hakeem, M.S. ...</div><div><strong>Developmental Relationships of Four Exhausted CD8(+) T Cell Subsets Reveals Underlying Transcriptional and Epigenetic Landscape Control Mechanisms</strong></div><div><em>Immunity.</em> 2020; <strong>52</strong>:825-841.e828</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084517361" target="_blank" aria-label="Open Scopus in new tab">Scopus (486)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32396847/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.04.014&amp;pmid=32396847" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to identify genes upregulated compared to at least one of the other 3 transient exhausted T&nbsp;cell states. Only genes with mouse to human orthologues were used for the analysis. GSEA max size parameter of 550 was used. For GSEA analysis of HA GD2 exhausted CAR T&nbsp;cell signature, gene lists related to <a id="crosref1710" href="#mmc1">Table S1</a> was downloaded from (<span class="dropBlock reference-citations"><a id="body-ref-sref41-4" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-4" aria-controls="sref41-4">Lynn et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref41-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Lynn, R.C. ∙ Weber, E.W. ∙ Sotillo, E. ...</div><div><strong>c-Jun overexpression in CAR T&nbsp;cells induces exhaustion resistance</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:293-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1805-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85075931233" target="_blank" aria-label="Open Scopus in new tab">Scopus (452)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31802004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1805-z&amp;pmid=31802004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We took the average expression of CD19 or HA samples for CD8+ T&nbsp;cell populations and filtered for genes that are upregulated &gt;2 fold in HA samples compared to CD19 to generate the CD8+ HA GD2 exhausted CAR T signature (N=91 genes) used for GSEA analysis.</div><div id="p0495" role="paragraph">To contrast CAR+ and CAR- samples (see <a id="crosref1720" href="#fig2">Figures 2</a>B, <a id="crosref1725" href="#figs3">S3</a>A, and S3B), transcripts per kilobase million (TPMs) were calculated for each gene using the bioinfokit.analys module in python. Gene lengths were calculated from the gene models used to run HTSeq, taking the maximum of all summed exon lengths across multiple isoforms as the length of the gene. For illustration purposes, we removed outlying genes with high expression (&gt;15,000) in surCARpos versus surCARneg plots to more easily see where &gt;99% of the genes fall on the correlation plot. However, all genes were included to make calculations, including spearman r.</div><div id="p0500" role="paragraph">Tracks were created for RNA-seq by pooling CAR+ samples across all replicates for control day 0 or CAE samples. BED files were filtered to remove alignments extending over 100bp, primarily removing intron-spanning alignments. Coverage maps were created using BEDtools genomeCoverageBed -bg and these were adjusted by multiplying by the RPM coefficient. Resulting bedGraphs were converted to bigWigs using UCSC Genome Browser Tools’ bedGraphToBigWig.</div><div id="p0505" role="paragraph">Bulk RNA-seq was compared to single-cell RNA-seq by taking all genes with significant differences in the single-cell data (between day 0 and day 20 CAE, identified using cellfishing.jl software, see description above) and rank-ordering them into ten deciles by log<sub>2</sub>(day 20 CAE/day 0 control), then representing the bulk RNA-seq log<sub>2</sub>(day 28 CAE/day 0 control) for each decile by box and whisker. Boxes are heated by the median value. DEGs identified by cellfishing.jl were filtered for genes with low signal- genes had to be expressed in at least 100 cells to be considered for analysis (see <a id="crosref1730" href="#figs4">Figure&nbsp;S4</a>G).</div></section><section id="sec-4-4-20"><h4>IPA analysis</h4><div id="p0510" role="paragraph">Full list of enriched pathways and transcription factors can be found in tables S2, S3, and S6.</div></section><section id="sec-4-4-21"><h4>ATAC-seq</h4><div id="p0515" role="paragraph">Omni ATAC-seq libraries were made as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Corces et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Corces, M.R. ∙ Trevino, A.E. ∙ Hamilton, E.G. ...</div><div><strong>An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:959-962</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85030994641" target="_blank" aria-label="Open Scopus in new tab">Scopus (1214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28846090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4396&amp;pmid=28846090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, nuclei were isolated from 30,000 sorted CD8+ surface CAR+ T&nbsp;cells, followed by the transposition reaction using Tn5 transposase (Illumina) for 30&nbsp;min at 37° C with 1000&nbsp;rpm mixing. Purification of transposed DNA was completed with DNA Clean and Concentrator (Zymo) and fragments were barcoded with ATAC-seq indices (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Buenrostro et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (4014)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Final libraries were double size selected using AMPure beads prior to sequencing. Paired-end sequencing (2 x 75&nbsp;bp reads) was carried out on an Illumina NextSeq 500 platform.</div></section><section id="sec-4-4-22"><h4>ATAC-seq analysis</h4><div id="p0520" role="paragraph">Paired-end data were aligned to human genome assembly GRCh37/hg19 using bowtie2 v2.3.4.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Langmead and Salzberg, 2012</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (33805)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with parameters --local -X 1000. Resulting SAM files were converted to BAM and filtered for match quality using samtools view -q 5 -bS (samtools v1.1). BAM files across four NextSeq lanes were merged and sorted by read name using samtools merge -n, then PCR de-duplicated with PICARD MarkDuplicates REMOVE_DUPLICATES=True ASSUME_SORT_ORDER=queryname. BAM files were converted to BEDs using BEDtools bamToBed and processed to remove all alignments on chrM. Alignments with a mate distance under 100&nbsp;bp were kept as sub-nucleosome fragment size signal and others were discarded.</div><div id="p0525" role="paragraph">For replicates R1 and R4, re-sequenced libraries were pooled using UNIX cat as follows:<div id="ulist0025" role="list"><div id="u0070" role="listitem"><div class="content"><div id="p0530" role="paragraph">R1 Day 0: 4-Day0-CD8-CARpos-R_S13, 4-Day0-CD8-CARpos-10-24-2018_S6</div></div></div><div id="u0075" role="listitem"><div class="content"><div id="p0535" role="paragraph">R1 CAE: 8-CAE-CD8-CARpos-R_S7, 8-CAE-CD8-CARpos-10-24-2018_S5)</div></div></div><div id="u0080" role="listitem"><div class="content"><div id="p0540" role="paragraph">R4 Day 0: 2-Day0-CD8-CARpos-REP4-ATAC-re_S17, 2-Day0-CD8-CARpos-REP4-ATAC_S17</div></div></div><div id="u0085" role="listitem"><div class="content"><div id="p0545" role="paragraph">R4 CAE: 4-CAE-CD8-CARpos-REP4-ATAC-re_S15, 4-CAE-CD8-CARpos-REP4-ATAC_S18</div></div></div></div></div><div id="p0550" role="paragraph">Peaks were called in the sub-nucleosome fragment fraction of alignments using MACS2 callpeak with parameters -s 42 -q 0.01 and no explicit background control sample. The FDR was subsequently controlled at 0.001.</div><div id="p0555" role="paragraph">Robust peak sets for control and CAE were identified in the following way. Peaks in either condition were combined across replicates, merging overlapping loci. Merged peaks without representation (BEDtools intersect) in all four replicates were removed.</div><div id="p0560" role="paragraph">To make track visualizations of the ATAC-seq data, an appropriate library size adjustment is necessary. DESeq2 was used to calculate size factors (coefficients for library size adjustment for each sample) from a set of pan-conditional peaks. The robust peak sets for control and CAE were combined, merging overlapping loci. Tag counts were calculated for all pan-conditional peaks across all samples and the resulting table was input to DESeq2 to estimate size factors and get adjusted tag counts at each peak. For each sample, sub-nucleosome sized fragment alignments were converted into a coverage map using BEDtools genomeCoverageBed -bg. Resulting bedGraph files were adjusted for library size by dividing coverage tallies by the DESeq2 size factors. Files were then sorted using UCSC Genome Browser Tools' bedSort and converted to bigwig format using bedGraphToBigWig.</div><div id="p0565" role="paragraph">To compare ATAC-seq to bulk RNA-seq, pan-conditional peaks were filtered to remove peaks overlapping ENCODE blacklisted regions. Remaining peaks were mapped to the nearest RefSeq transcript by TSS. The set of genes up- or down-regulated at FDR 0.05 in the antigen exposure contrast was used to identify mapped peaks, and their DESeq2-adjusted counts were plotted by box-and-whisker. Statistics assessed by Mann-Whitney.</div><div id="p0570" role="paragraph">Enriched motifs were identified in peaks specific to control day 0 or CAE using HOMER v4.6 findMotifsGenome.pl with command-line parameters -size 200 -mask (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Heinz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (8027)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Robust peak sets were filtered for any overlap with ENCODE blacklisted regions or with peaks from the other condition (e.g., control day 0 peaks without overlap to CAE peaks) using BEDtools intersect, and these specific peak sets were input to HOMER. The HOMER background (-bg) was set as robust peaks specific to the other condition.</div><div id="p0575" role="paragraph">To analyze the enrichment of SOX4 at ATAC-seq peaks, the SOX4 position weight matrix was downloaded from JASPAR (MA0867.2) and scanned against robust CAE-specific peaks (those without overlaps to ENCODE blacklisted regions or control day 0 peaks) or peaks common to control day 0 and CAE stimulation using PWMSCAN, with the FDR controlled at 1E-8. Peaks were divided into those with or without the motif and DESeq2-adjusted values are shown for these peak sets in box-and-whisker (<a id="crosref1755" href="#fig6">Figure&nbsp;6</a>F). Statistics assessed by Mann-Whitney. Day 28 CAE peaks present in at least two biological replicates were associated to nearby genes using HOMER v4.6 annotatePeaks.pl. Peaks were examined for the presence of the human SOX4 motif (JASPAR MA0867.2) using PWM_SCAN, with p-value cutoff 1E-6. Our CAR T dysfunction signature genes (N=30) were then queried to determine how many had associated peaks with a SOX4 motif hit.</div></section><section id="sec-4-4-23"><h4>ATAC-seq analysis of exhausted human TILs</h4><div id="p0580" role="paragraph">Published data from (<span class="dropBlock reference-citations"><a id="body-ref-sref53-6" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-6" aria-controls="sref53-6">Philip et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Philip, M. ∙ Fairchild, L. ∙ Sun, L. ...</div><div><strong>Chromatin states define tumour-specific T&nbsp;cell dysfunction and reprogramming</strong></div><div><em>Nature.</em> 2017; <strong>545</strong>:452-456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22367" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85019673982" target="_blank" aria-label="Open Scopus in new tab">Scopus (602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28514453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22367&amp;pmid=28514453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)(TIL model) were downloaded from NCBI GEO. FASTQs were trimmed using TrimGalore! v0.6.6 (relying on FASTQC v0.11.2 and cutadapt v2.10) with command-line arguments --paired --fastqc, then aligned to the hg19 assembly of the human genome using bowtie2 v2.3.4.1 with command-line parameters --local -X 1000. Data were filtered for poor alignments using samtools view -q 5 (samtools v1.1, (<span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Li et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The Sequence Alignment/Map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (38872)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)), then sorted with samtools sort -n and filtered for PCR duplicates using PICARD MarkDuplicates REMOVE_DUPLICATES=True ASSUME_SORT_ORDER=queryname (PICARD v2.21.3-SNAPSHOT). Data were rendered as BED files using bamToBed v2.27.1-65-gc2af1e7-dirty, then processed using python to exclude chrM alignments and filtered to find paired-end alignments smaller than 100&nbsp;bp (“sub-nucleosome fragments”). Remaining fragments from three biological replicates were pooled. Day 28 CAE-specific peaks present in at least two biological replicates (N&nbsp;= 4,766) were scanned for ATAC-seq enrichment in both our data and in the Philip et&nbsp;al Nature 2017 TIL model (naïve and PD1hi cells) in a 5kb window around the peak center using 50bp increments. Peaks were sorted vertically by the summed Day 28 CAE ATAC-seq signal in descending order and all ATAC-seq traces were rendered as a heatmap using python and the PIL imaging library. To assess the dynamics of the TIL model at day 28 CAE-specific peaks, a background peak set was randomly sampled from all ATAC-seq OCRs from day 0 or day 28 CAE (size&nbsp;= 4,766 with random seed 3399039292705153955). Peak enrichment for our data and the Philip et&nbsp;al Nature 2017 TIL model (naïve and PD1hi cells) was measured over day 28 CAE-specific peaks and background peaks, and then the difference PD1hi-naïve was rendered as a box-and-whisker for the two peak groups (<a id="crosref1770" href="#figs4">Figure&nbsp;S4</a>E). Statistical significance of the difference between day 28 CAE-specific peaks and background peaks was assessed using a permutation test (coin library in R).</div></section><section id="sec-4-4-24"><h4>LCMV chronic viral infection data analysis</h4><div id="p0585" role="paragraph">RNA-seq FASTQ files were downloaded from GEO submission GSE86881 for naïve mouse CD8+ T&nbsp;cells (GSM2309810, GSM2309811) and exhausted CD8+ T&nbsp;cells (GSM2309812, GSM2309813, GSM2309814). FASTQ files were aligned to the mm10 reference genome using STAR and differentially expressed genes between naïve CD8+ T&nbsp;cells and exhausted CD8+ T&nbsp;cells were identified using DESeq2. Only genes with mouse to human homologs were overlapped with CAR T dysfunction gene signature. Homologs were obtained from the Mouse Genome Informatics (MGI) database.</div></section><section id="sec-4-4-25"><h4>Human cancer TIL overlap analysis</h4><div id="p0590" role="paragraph">The following published single-cell datasets were overlapped with the CAR T&nbsp;cell dysfunction gene signature. Colorectal cancer exhausted CD8 TIL associated genes were downloaded from <a id="crosref1775" href="#mmc5">Table S5</a> for the CD8_C07-LAYN specific genes (N=714 genes, including LAYN) (<span class="dropBlock reference-citations"><a id="body-ref-sref76-3" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-3" aria-controls="sref76-3">Zhang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref76-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Zhang, L. ∙ Yu, X. ∙ Zheng, L. ...</div><div><strong>Lineage tracking reveals dynamic relationships of T&nbsp;cells in colorectal cancer</strong></div><div><em>Nature.</em> 2018; <strong>564</strong>:268-272</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0694-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85058518221" target="_blank" aria-label="Open Scopus in new tab">Scopus (694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30479382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0694-x&amp;pmid=30479382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Non-small-cell lung cancer exhausted CD8 TIL-associated genes were downloaded from <a id="crosref1785" href="#mmc3">Table S3</a> for the CD8-C6-LAYN specific genes (N=399 genes) (<span class="dropBlock reference-citations"><a id="body-ref-sref24-3" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-3" aria-controls="sref24-3">Guo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref24-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hepatocellular carcinoma exhausted CD8 TIL associated genes were downloaded from <a id="crosref1795" href="#mmc4">Table S4</a> (N=82 genes) (<span class="dropBlock reference-citations"><a id="body-ref-sref77-4" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-4" aria-controls="sref77-4">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref77-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zheng, C. ∙ Zheng, L. ∙ Yoo, J.K. ...</div><div><strong>Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1342-1356.e1316</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85020894317" target="_blank" aria-label="Open Scopus in new tab">Scopus (1383)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.035&amp;pmid=28622514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Melanoma exhausted CD8 TIL associated genes were obtained from <a id="crosref1805" href="#fig2">Figure&nbsp;2</a>B (genes most correlated with LAG3) and supplemental <a id="crosref1810" href="#figs2">Figure&nbsp;S2</a>E (genes most correlated with HAVCR2) (N=34) (<span class="dropBlock reference-citations"><a id="body-ref-sref37-5" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-5" aria-controls="sref37-5">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, H. ∙ van der Leun, A.M. ∙ Yofe, I. ...</div><div><strong>Dysfunctional CD8&nbsp;T Cells Form a Proliferative, Dynamically Regulated Compartment within Human Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:775-789.e718</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061004091" target="_blank" aria-label="Open Scopus in new tab">Scopus (661)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30595452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.11.043&amp;pmid=30595452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0595" role="paragraph">Statistical analyses were performed using GraphPad Prism software or R and are represented by the following <sup>∗∗∗∗</sup><i>P</i>&nbsp;&lt; 0.0001, <sup>∗∗∗</sup><i>P</i>&nbsp;&lt; 0.001, <sup>∗∗</sup><i>P</i>&nbsp;&lt; 0.01, <sup>∗</sup>P&nbsp;&lt; 0.05. Statistical significance between two groups was determined using two-tailed unpaired Student’s t test (parametric) or Mann-Whitney (non-parametric). The Benjamini–Hochberg procedure (FDR) was used to adjust p&nbsp;values for multiple testing, unless otherwise noted. Statistical significance between multiple groups of two variables was assessed by two-way ANOVA with post hoc tests. Statistical significance of Venn diagram overlaps between two groups was calculated using hypergeometric tests. The specific tests used for the analyses shown in the supplementary figure are indicated in supplementary figure legends. The specifical tests used for analyses in the main figure legends are detailed in the main figure legends and summarized below. Cytokine profile analysis of CD8+ surCARpos T&nbsp;cells (day 28 CAE, day 0 product and CD19BBz) of <a id="crosref1820" href="#fig1">Figure&nbsp;1</a>F was assessed by two-way ANOVA with Tukey’s post hoc test. Cytokine secretion of day 26 CAR T&nbsp;cells before and after 24&nbsp;hrs of rest in presence of IL-15 (<a id="crosref1825" href="#fig1">Figure&nbsp;1</a>H) was calculated with two-tailed unpaired Student’s t test. Differences between day 0 and Day 28 normalized RNA-seq counts of representative NK receptor/marker genes (<a id="crosref1830" href="#fig2">Figure&nbsp;2</a>D) was analyzed with Mann-Whitney U test. Significance of the differences on normalized counts of CAR transcripts from sc-RNA-seq data for between non dysfunctional and dysfunctional clusters (<a id="crosref1835" href="#fig3">Figure&nbsp;3</a>H) was calculated with Mann-Whitney U test. Statistical differences in protein levels of NK-associated molecules and checkpoint receptors in CD8&nbsp;T infiltrating recurrent tumors versus day 0 product (<a id="crosref1840" href="#fig4">Figures 4</a>E and 4G) were assessed by two-way ANOVA with Sidak test for multiple comparisons. Differences in ATAC-seq signal at peaks specific to day 28 (<a id="crosref1845" href="#fig6">Figure&nbsp;6</a>F) were assessed by Mann-Whitney. Differences on the percentage of NK-like T&nbsp;cells between WT, ID3 KO, and SOX4 KO M5CAR T&nbsp;cells (<a id="crosref1850" href="#fig7">Figure&nbsp;7</a>G) was measured by Fisher’s exact test. Statistical differences in the dysfunctional score of WT, ID3 KO, and SOX4 KO M5CAR T&nbsp;cells for donor ND566 (<a id="crosref1855" href="#fig7">Figure&nbsp;7</a>I) and WT and ID3 KO M5CAR T&nbsp;cells for donor ND539 (<a id="crosref1860" href="#fig7">Figure&nbsp;7</a>J) were measured by Mann-Whitney U test. Cell killing statistical differences in WT versus ID3 KO, and SOX4 KO M5CAR T&nbsp;cells was assessed by two-way ANOVA with Geisser-Greenhouse correction and Dunnet’s post hoc test. Data were visualized using the following R packages and functions ggplot2, EnhancedVolcano, pheatmap, RColorBrewer, gplots, dplyr, plotly. See <a id="crosref1865" href="#sec-4-1">Key Resource Table</a> for versions of R packages utilized in this study.</div></section><section id="sec-4-6"><h3>Additional resources</h3><div id="p0600" role="paragraph">Clinical samples analyzed in this study were obtained from clinical trials conducted at University of Pennsylvania and registered at <a id="intref0175" href="http://www.clinical" target="_blank">www.clinical</a> <a id="intref0180" href="http://trial.gov" target="_blank">trial.gov</a> as NCT03054298 and NCT02030834.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0605" role="paragraph">Research supported by a SU2C-Lustgarten Foundation Translational Cancer Research Team Grant, grant numbers SU2-CAACR-DT21-17 and SU2C-RT6162. S.L.B. is supported by NIH grant CA078831. C.R.G. is supported by NIH grant CA232466 and the American Cancer Society – Rob Kugler – Postdoctoral Fellowship. R.M.Y. and C.H.J. are supported by NIH grant P01CA214278. C.H.J. and L.L.L. are supported by the Parker Institute for Cancer Immunotherapy. P.C.R. is supported by the National Institutes of Health/National Cancer Institute grant 5T32CA009140. We acknowledge the Parnassus Flow Cytometry Core, in part supported by the NIH DRC Center grants P30 DK063720 and S10 1S10OD018040-01, for use of the CyTOF instrument. The authors wish to thank E. John Wherry and Austin L. Good for their valuable feedback on this manuscript.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0610" role="paragraph">C.R.G., M.A.A., and S.K. designed and conducted the experiments, interpreted the data, and wrote and prepared the manuscript. A.K.R., Y.M., L.T., S.A., S.G., N.W., M.W.R., N.S., P.C.R., and K.W. assisted with <i>in&nbsp;vitro</i> and <i>in&nbsp;vivo</i> assays. J.L.T., M.H.O., and S.J.S. provided clinical samples and expertise. S.M.A. and E.K.M. designed and performed the NY-ESO-1 models. M.R. assisted with supervision of <i>in&nbsp;vitro</i> experiments, and S.F.L. assisted with supervision of analyses of clinical samples. C.R.G., P.S., and G.D. performed bioinformatic analyses. K.A.A., K.M.G., and Z.Z. provided feedback on genomics experiments. K.I. designed and performed the mass cytometry assays. C.H.J., S.L.B., L.L.L., and R.M.Y. led the design, interpretation, and analysis of all experiments, and helped with the writing and preparation of the manuscript.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0615" role="paragraph">R.M.Y., S.G., S.F.L., S.M.A., M.R., and C.H.J. are inventors on patents and/or patent applications licensed to Novartis Institutes of Biomedical Research and receive license revenue from such licenses. R.M.Y. is an inventor on patents and/or patent applications licensed to Tmunity Therapeutics and receives license revenue from such licenses. C.H.J. is a scientific founder of Tmunity Therapeutics and DeCART Therapeutics, and is a member of the scientific advisory boards of AC Immune, BluesphereBio, Cabaletta, Carisma, Cartography, Cellares, Celldex, Decheng, Poseida, Verismo, WIRB-Copernicus, and Ziopharm. S.J.S. is a consultant, on the scientific advisory board, and receives research support from Genentech/Roche, Novartis, and Juno Therapeutics. S.J.S. is a consultant for AlloGene, AstraZeneca, BeiGene, Regeneron, and Tesa Therapeutics. S.J.S. is a consultant and on the scientific advisory board for Loxo Oncology. S.J.S. is on the scientific advisory board for Nordic Nanovector. S.J.S. is a consultant and receives research support from Celgene. S.F.L. receives research funding from Tmunity Therapeutics and Cabaletta. M.R. is on the scientific advisory board of AbClon Inc. and consulted for BMS, nanoString, GSK, and Bayer. M.R. is the scientific founder of viTToria Biotherapeutics. L.L.L. is on the scientific advisory boards for Alector, Atreca, Dragonfly, DrenBio, Morphosys, Nkarta, Obsidian Therapeutics, Rubius, SBI, and Innovent. S.M.A. receives research funding from Tmunity Therapeutics, RAPT, and Incyte Corporation and is scientific advisor for Trizell, BioArdis, and Verismo.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (6)</h2><div id="p0625" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.016/attachment/6e338615-7ea3-40c1-9312-2332133e5419/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (96.43 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Differentially expressed genes between day 0 and day 28 CAE CAR T&nbsp;cells from bulk RNA-seq experiments, related to Figure&nbsp;2A</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.016/attachment/60b88dbc-b15a-4535-bbc7-b364733375f6/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (39.33 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Full list of Ingenuity Pathway Analysis pathways enriched in differentially expressed genes between day 0 and day 28 CAE CAR T&nbsp;cells from bulk RNA-seq experiments, related to Figure&nbsp;2C</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.016/attachment/01423c9f-d98a-471b-b1ce-17ef12bef6ba/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (44.34 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. All upstream regulators (transcription factors) predicted to regulate differentially expressed genes between day 0 and day 28 CAE CAR T&nbsp;cells from bulk RNA-seq experiments, related to Figure&nbsp;2F</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.016/attachment/7af3028b-01a6-4d17-ab09-e3eb7779c4f0/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (549.98 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Differentially expressed genes between dysfunctional clusters (D20-1, D20-4) and non-dysfunctional clusters (D20-2, D20-3) from scRNA-seq experiment in donor ND388, related to Figure&nbsp;3E</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.016/attachment/ef785896-554e-476a-82fa-d2774da912bb/mmc5.pdf" download="mmc5.pdf" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">PDF (51.30 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. Table of dysfunction signature genes (N&nbsp;= 30), related to Figures 3 and S3</div><div id="espara0010" role="paragraph">X marks genes that are upregulated in LCMV exhausted T&nbsp;cells or defines dysfunctional TILs in scRNA-seq experiments. HCC (hepatocellular carcinoma), NSCLC (non-small-cell lung cancer), CRC (colorectal carcinoma), LCMV (lymphocytic choriomeningitis virus).</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.016/attachment/d5b4e66f-2da7-4269-b791-4a4bee89e524/mmc6.xlsx" download="mmc6.xlsx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Spreadsheet (32.84 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Table S6. All upstream regulators (transcription factors) predicted to regulate differentially expressed genes between day 0 and day 20 CAE CAR T&nbsp;cells from scRNA-seq experiment in donor ND388, related to Figure&nbsp;6A</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">All genomics data have been submitted to the gene expression omnibus database and are publicly available as of the date of publication. Accession numbers are listed in the key resources table. This paper analyzes existing, publicly available data. The accession numbers for the datasets are listed in the key resources table.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">Unique computer code used in this manuscript has been submitted to GitHub and can be accessed using the following link <a id="intref0010" href="https://github.com/bergerlabupenn/InVitroCARTexh_code_2020" target="_blank">https://github.com/bergerlabupenn/InVitroCARTexh_code_2020</a>.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Abd Hamid, M. ∙ Wang, R.Z. ∙ Yao, X. ...</div><div><strong>Enriched HLA-E and CD94/NKG2A Interaction Limits Antitumor CD8<sup>+</sup> Tumor-Infiltrating T Lymphocyte Responses</strong></div><div><em>Cancer Immunol. Res.</em> 2019; <strong>7</strong>:1293-1306</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2326-6066.CIR-18-0885" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85070565916" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31213473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2326-6066.CIR-18-0885&amp;pmid=31213473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Agarwal, S. ∙ Wellhausen, N. ∙ Levine, B.L. ...</div><div><strong>Production of Human CRISPR-Engineered CAR-T Cells</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3791/62299" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85103610405" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33779622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3791%2F62299&amp;pmid=33779622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Anders, S. ∙ Pyl, P.T. ∙ Huber, W.</div><div><strong>HTSeq--a Python framework to work with high-throughput sequencing data</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:166-169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu638" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84928987900" target="_blank" aria-label="Open Scopus in new tab">Scopus (13514)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25260700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu638&amp;pmid=25260700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Andre, P. ∙ Denis, C. ∙ Soulas, C. ...</div><div><strong>Anti-NKG2A mAb Is a Checkpoint Inhibitor that Promotes Anti-tumor Immunity by Unleashing Both T and NK Cells</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:1731-1743.e1713</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85058479063" target="_blank" aria-label="Open Scopus in new tab">Scopus (800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30503213/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.10.014&amp;pmid=30503213" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Balin, S.J. ∙ Pellegrini, M. ∙ Klechevsky, E. ...</div><div><strong>Human antimicrobial cytotoxic T lymphocytes, defined by NK receptors and antimicrobial proteins, kill intracellular bacteria</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaat7668</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aat7668" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85059961035" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30171080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aat7668&amp;pmid=30171080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Barbarin, A. ∙ Cayssials, E. ∙ Jacomet, F. ...</div><div><strong>Phenotype of NK-Like CD8(+) T Cells with Innate Features in Humans and Their Relevance in Cancer Diseases</strong></div><div><em>Front. Immunol.</em> 2017; <strong>8</strong>:316</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2017.00316" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85017100637" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28396661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2017.00316&amp;pmid=28396661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">7.</a></div><div>Beltra, J.C. ∙ Manne, S. ∙ Abdel-Hakeem, M.S. ...</div><div><strong>Developmental Relationships of Four Exhausted CD8(+) T Cell Subsets Reveals Underlying Transcriptional and Epigenetic Landscape Control Mechanisms</strong></div><div><em>Immunity.</em> 2020; <strong>52</strong>:825-841.e828</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084517361" target="_blank" aria-label="Open Scopus in new tab">Scopus (486)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32396847/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.04.014&amp;pmid=32396847" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Benezra, R. ∙ Davis, R.L. ∙ Lockshon, D. ...</div><div><strong>The protein Id: a negative regulator of helix-loop-helix DNA binding proteins</strong></div><div><em>Cell.</em> 1990; <strong>61</strong>:49-59</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2F0092-8674%2890%2990214-Y&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2F0092-8674%2890%2990214-Y&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-0025238437" target="_blank" aria-label="Open Scopus in new tab">Scopus (2050)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2156629/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2890%2990214-Y&amp;pmid=2156629" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9-1" title="View in article">9.</a></div><div>Blank, C.U. ∙ Haining, W.N. ∙ Held, W. ...</div><div><strong>Defining ‘T cell exhaustion’</strong></div><div><em>Nat. Rev. Immunol.</em> 2019; <strong>19</strong>:665-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-019-0221-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85074300816" target="_blank" aria-label="Open Scopus in new tab">Scopus (857)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-019-0221-9&amp;pmid=31570879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Boroughs, A.C. ∙ Larson, R.C. ∙ Marjanovic, N.D. ...</div><div><strong>A Distinct Transcriptional Program in Human CAR T Cells Bearing the 4-1BB Signaling Domain Revealed by scRNA-Seq</strong></div><div><em>Mol. Ther.</em> 2020; <strong>28</strong>:2577-2592</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ymthe.2020.07.023&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ymthe.2020.07.023&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85088996471" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32755564/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymthe.2020.07.023&amp;pmid=32755564" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (4014)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">12.</a></div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85046298440" target="_blank" aria-label="Open Scopus in new tab">Scopus (6227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4096&amp;pmid=29608179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Carpenito, C. ∙ Milone, M.C. ∙ Hassan, R. ...</div><div><strong>Control of large, established tumor xenografts with genetically retargeted human T&nbsp;cells containing CD28 and CD137 domains</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2009; <strong>106</strong>:3360-3365</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0813101106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-62549097817" target="_blank" aria-label="Open Scopus in new tab">Scopus (710)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19211796/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0813101106&amp;pmid=19211796" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Chan, T.E. ∙ Stumpf, M.P.H. ∙ Babtie, A.C.</div><div><strong>Gene Regulatory Network Inference from Single-Cell Data Using Multivariate Information Measures</strong></div><div><em>Cell Syst.</em> 2017; <strong>5</strong>:251-267.e3</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cels.2017.08.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cels.2017.08.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85031092991" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28957658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2017.08.014&amp;pmid=28957658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Chen, J. ∙ López-Moyado, I.F. ∙ Seo, H. ...</div><div><strong>NR4A transcription factors limit CAR T&nbsp;cell function in solid tumours</strong></div><div><em>Nature.</em> 2019; <strong>567</strong>:530-534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0985-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85062871387" target="_blank" aria-label="Open Scopus in new tab">Scopus (493)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30814732/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0985-x&amp;pmid=30814732" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Corces, M.R. ∙ Trevino, A.E. ∙ Hamilton, E.G. ...</div><div><strong>An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:959-962</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85030994641" target="_blank" aria-label="Open Scopus in new tab">Scopus (1214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28846090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4396&amp;pmid=28846090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (27081)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Dong, M.B. ∙ Wang, G. ∙ Chow, R.D. ...</div><div><strong>Systematic Immunotherapy Target Discovery Using Genome-Scale In&nbsp;Vivo CRISPR Screens in CD8&nbsp;T Cells</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:1189-1204.e23</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.07.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.07.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85070563455" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31442407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.044&amp;pmid=31442407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Fornes, O. ∙ Castro-Mondragon, J.A. ∙ Khan, A. ...</div><div><strong>JASPAR 2020: update of the open-access database of transcription factor binding profiles</strong></div><div><em>Nucleic Acids Res.</em> 2020; <strong>48</strong>:D87-D92</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31701148/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31701148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Fraietta, J.A. ∙ Lacey, S.F. ∙ Orlando, E.J. ...</div><div><strong>Determinants of response and resistance to CD19 chimeric antigen receptor (CAR) T&nbsp;cell therapy of chronic lymphocytic leukemia</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:563-571</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0010-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85046093409" target="_blank" aria-label="Open Scopus in new tab">Scopus (1116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0010-1&amp;pmid=29713085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Fraietta, J.A. ∙ Nobles, C.L. ∙ Sammons, M.A. ...</div><div><strong>Disruption of TET2 promotes the therapeutic efficacy of CD19-targeted T&nbsp;cells</strong></div><div><em>Nature.</em> 2018; <strong>558</strong>:307-312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0178-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85048364639" target="_blank" aria-label="Open Scopus in new tab">Scopus (577)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29849141/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0178-z&amp;pmid=29849141" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Godfrey, D.I. ∙ MacDonald, H.R. ∙ Kronenberg, M. ...</div><div><strong>NKT cells: what’s in a name?</strong></div><div><em>Nat. Rev. Immunol.</em> 2004; <strong>4</strong>:231-237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nri1309" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15039760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnri1309&amp;pmid=15039760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Guo, J. ∙ Xu, C.</div><div><strong>Screening for the Next-Generation T Cell Therapies</strong></div><div><em>Cancer Cell.</em> 2020; <strong>37</strong>:627-629</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2020.04.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2020.04.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084206672" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32396857/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2020.04.009&amp;pmid=32396857" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">24.</a></div><div>Guo, X. ∙ Zhang, Y. ∙ Zheng, L. ...</div><div><strong>Global characterization of T&nbsp;cells in non-small-cell lung cancer by single-cell sequencing</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:978-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0045-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85049041084" target="_blank" aria-label="Open Scopus in new tab">Scopus (967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0045-3&amp;pmid=29942094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Gurusamy, D. ∙ Henning, A.N. ∙ Yamamoto, T.N. ...</div><div><strong>Multi-phenotype CRISPR-Cas9 Screen Identifies p38 Kinase as a Target for Adoptive Immunotherapies</strong></div><div><em>Cancer Cell.</em> 2020; <strong>37</strong>:818-833.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2020.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2020.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85085757009" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32516591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2020.05.004&amp;pmid=32516591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">26.</a></div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (1792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Heemskerk, M.H. ∙ Blom, B. ∙ Nolan, G. ...</div><div><strong>Inhibition of T&nbsp;cell and promotion of natural killer cell development by the dominant negative helix loop helix factor Id3</strong></div><div><em>J.&nbsp;Exp. Med.</em> 1997; <strong>186</strong>:1597-1602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.186.9.1597" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-0030665673" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9348318/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.186.9.1597&amp;pmid=9348318" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">28.</a></div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (8027)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Hu, G. ∙ Chen, J.</div><div><strong>A genome-wide regulatory network identifies key transcription factors for memory CD8<sup>+</sup> T-cell development</strong></div><div><em>Nat. Commun.</em> 2013; <strong>4</strong>:2830</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms3830" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84890638806" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24335726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms3830&amp;pmid=24335726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Ji, Y. ∙ Pos, Z. ∙ Rao, M. ...</div><div><strong>Repression of the DNA-binding inhibitor Id3 by Blimp-1 limits the formation of memory CD8+ T&nbsp;cells</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:1230-1237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2153" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-81255144750" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22057288/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2153&amp;pmid=22057288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">31.</a></div><div>Khan, O. ∙ Giles, J.R. ∙ McDonald, S. ...</div><div><strong>TOX transcriptionally and epigenetically programs CD8<sup>+</sup> T&nbsp;cell exhaustion</strong></div><div><em>Nature.</em> 2019; <strong>571</strong>:211-218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1325-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85068336352" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31207603/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1325-x&amp;pmid=31207603" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Kurioka, A. ∙ Klenerman, P. ∙ Willberg, C.B.</div><div><strong>Innate-like CD8+ T-cells and NK cells: converging functions and phenotypes</strong></div><div><em>Immunology.</em> 2018; <strong>154</strong>:547-556</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/imm.12925" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85050148100" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fimm.12925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Kutner, R.H. ∙ Zhang, X.Y. ∙ Reiser, J.</div><div><strong>Production, concentration and titration of pseudotyped HIV-1-based lentiviral vectors</strong></div><div><em>Nat. Protoc.</em> 2009; <strong>4</strong>:495-505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2009.22" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-64749107970" target="_blank" aria-label="Open Scopus in new tab">Scopus (509)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19300443/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2009.22&amp;pmid=19300443" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (33805)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">35.</a></div><div>Leong, J.W. ∙ Wagner, J.A. ∙ Ireland, A.R. ...</div><div><strong>Transcriptional and post-transcriptional regulation of NK cell development and function</strong></div><div><em>Clin. Immunol.</em> 2017; <strong>177</strong>:60-69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.clim.2016.03.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84977090602" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26948928/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.clim.2016.03.003&amp;pmid=26948928" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">36.</a></div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The Sequence Alignment/Map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (38872)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">37.</a></div><div>Li, H. ∙ van der Leun, A.M. ∙ Yofe, I. ...</div><div><strong>Dysfunctional CD8&nbsp;T Cells Form a Proliferative, Dynamically Regulated Compartment within Human Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:775-789.e718</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.11.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061004091" target="_blank" aria-label="Open Scopus in new tab">Scopus (661)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30595452/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.11.043&amp;pmid=30595452" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>Li, W. ∙ Qiu, S. ∙ Chen, J. ...</div><div><strong>Chimeric Antigen Receptor Designed to Prevent Ubiquitination and Downregulation Showed Durable Antitumor Efficacy</strong></div><div><em>Immunity.</em> 2020; <strong>53</strong>:456-470.e456</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.07.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.07.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85089400433" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32758419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.07.011&amp;pmid=32758419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Long, A.H. ∙ Haso, W.M. ∙ Shern, J.F. ...</div><div><strong>4-1BB costimulation ameliorates T&nbsp;cell exhaustion induced by tonic signaling of chimeric antigen receptors</strong></div><div><em>Nat. Med.</em> 2015; <strong>21</strong>:581-590</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nm.3838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84930765209" target="_blank" aria-label="Open Scopus in new tab">Scopus (1240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25939063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnm.3838&amp;pmid=25939063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Lu, Y.C. ∙ Jia, L. ∙ Zheng, Z. ...</div><div><strong>Single-Cell Transcriptome Analysis Reveals Gene Signatures Associated with T-cell Persistence Following Adoptive Cell Therapy</strong></div><div><em>Cancer Immunol. Res.</em> 2019; <strong>7</strong>:1824-1836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2326-6066.CIR-19-0299" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85074444784" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31484655/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2326-6066.CIR-19-0299&amp;pmid=31484655" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Lynn, R.C. ∙ Weber, E.W. ∙ Sotillo, E. ...</div><div><strong>c-Jun overexpression in CAR T&nbsp;cells induces exhaustion resistance</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:293-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1805-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85075931233" target="_blank" aria-label="Open Scopus in new tab">Scopus (452)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31802004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1805-z&amp;pmid=31802004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Malhotra, N. ∙ Qi, Y. ∙ Spidale, N.A. ...</div><div><strong>SOX4 controls invariant NKT cell differentiation by tuning TCR signaling</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2018; <strong>215</strong>:2887-2900</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20172021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85056287865" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30287480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20172021&amp;pmid=30287480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Manguso, R.T. ∙ Pope, H.W. ∙ Zimmer, M.D. ...</div><div><strong>In&nbsp;vivo CRISPR screening identifies Ptpn2 as a cancer immunotherapy target</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:413-418</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23270" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85026403229" target="_blank" aria-label="Open Scopus in new tab">Scopus (748)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28723893/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23270&amp;pmid=28723893" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">44.</a></div><div>Mathewson, N.D. ∙ Ashenberg, O. ∙ Tirosh, I. ...</div><div><strong>Inhibitory CD161 receptor identified in glioma-infiltrating T&nbsp;cells by single-cell analysis</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:1281-1298.e1226</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2021.01.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85101668184" target="_blank" aria-label="Open Scopus in new tab">Scopus (216)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592174/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.01.022&amp;pmid=33592174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>McMahon, C.W. ∙ Zajac, A.J. ∙ Jamieson, A.M. ...</div><div><strong>Viral and bacterial infections induce expression of multiple NK cell receptors in responding CD8(+) T&nbsp;cells</strong></div><div><em>J.&nbsp;Immunol.</em> 2002; <strong>169</strong>:1444-1452</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.169.3.1444" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12133970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.169.3.1444&amp;pmid=12133970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">46.</a></div><div>Meresse, B. ∙ Chen, Z. ∙ Ciszewski, C. ...</div><div><strong>Coordinated induction by IL15 of a TCR-independent NKG2D signaling pathway converts CTL into lymphokine-activated killer cells in celiac disease</strong></div><div><em>Immunity.</em> 2004; <strong>21</strong>:357-366</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2004.06.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2004.06.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-4444240035" target="_blank" aria-label="Open Scopus in new tab">Scopus (695)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15357947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2004.06.020&amp;pmid=15357947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47-1" title="View in article">47.</a></div><div>Milner, J.J. ∙ Toma, C. ∙ He, Z. ...</div><div><strong>Heterogenous Populations of Tissue-Resident CD8<sup>+</sup> T Cells Are Generated in Response to Infection and Malignancy</strong></div><div><em>Immunity.</em> 2020; <strong>52</strong>:808-824.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.immuni.2020.04.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85084480841" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32433949/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2020.04.007&amp;pmid=32433949" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Mognol, G.P. ∙ Spreafico, R. ∙ Wong, V. ...</div><div><strong>Exhaustion-associated regulatory regions in CD8<sup>+</sup> tumor-infiltrating T&nbsp;cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:E2776-E2785</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1620498114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85016431802" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28283662/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1620498114&amp;pmid=28283662" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>Moon, E.K. ∙ Wang, L.C. ∙ Dolfi, D.V. ...</div><div><strong>Multifactorial T-cell hypofunction that is reversible can limit the efficacy of chimeric antigen receptor-transduced human T&nbsp;cells in solid tumors</strong></div><div><em>Clin. Cancer Res.</em> 2014; <strong>20</strong>:4262-4273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-13-2627" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84905993539" target="_blank" aria-label="Open Scopus in new tab">Scopus (334)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24919573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-13-2627&amp;pmid=24919573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">50.</a></div><div>Moon, E.K. ∙ Ranganathan, R. ∙ Eruslanov, E. ...</div><div><strong>Blockade of Programmed Death 1 Augments the Ability of Human T Cells Engineered to Target NY-ESO-1 to Control Tumor Growth after Adoptive Transfer</strong></div><div><em>Clin. Cancer Res.</em> 2016; <strong>22</strong>:436-447</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-15-1070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84959036332" target="_blank" aria-label="Open Scopus in new tab">Scopus (105)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26324743/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-15-1070&amp;pmid=26324743" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">51.</a></div><div>Pauken, K.E. ∙ Wherry, E.J.</div><div><strong>Overcoming T&nbsp;cell exhaustion in infection and cancer</strong></div><div><em>Trends Immunol.</em> 2015; <strong>36</strong>:265-276</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2015.02.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2015.02.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84926528302" target="_blank" aria-label="Open Scopus in new tab">Scopus (792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25797516/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2015.02.008&amp;pmid=25797516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52-1" title="View in article">52.</a></div><div>Pauken, K.E. ∙ Sammons, M.A. ∙ Odorizzi, P.M. ...</div><div><strong>Epigenetic stability of exhausted T&nbsp;cells limits durability of reinvigoration by PD-1 blockade</strong></div><div><em>Science.</em> 2016; <strong>354</strong>:1160-1165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf2807" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84994364943" target="_blank" aria-label="Open Scopus in new tab">Scopus (893)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27789795/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf2807&amp;pmid=27789795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#figs4" title="View in article">53.</a></div><div>Philip, M. ∙ Fairchild, L. ∙ Sun, L. ...</div><div><strong>Chromatin states define tumour-specific T&nbsp;cell dysfunction and reprogramming</strong></div><div><em>Nature.</em> 2017; <strong>545</strong>:452-456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22367" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85019673982" target="_blank" aria-label="Open Scopus in new tab">Scopus (602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28514453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22367&amp;pmid=28514453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">54.</a></div><div>Picelli, S. ∙ Faridani, O.R. ∙ Björklund, A.K. ...</div><div><strong>Full-length RNA-seq from single cells using Smart-seq2</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:171-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84891677425" target="_blank" aria-label="Open Scopus in new tab">Scopus (2542)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24385147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.006&amp;pmid=24385147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Poorebrahim, M. ∙ Melief, J. ∙ Pico de Coaña, Y. ...</div><div><strong>Counteracting CAR T&nbsp;cell dysfunction</strong></div><div><em>Oncogene.</em> 2021; <strong>40</strong>:421-435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41388-020-01501-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85095708741" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33168929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41388-020-01501-x&amp;pmid=33168929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">56.</a></div><div>Qiu, X. ∙ Hill, A. ∙ Packer, J. ...</div><div><strong>Single-cell mRNA quantification and differential analysis with Census</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:309-315</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4150" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85010878111" target="_blank" aria-label="Open Scopus in new tab">Scopus (902)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28114287/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4150&amp;pmid=28114287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">57.</a></div><div>Roth, T.L. ∙ Li, P.J. ∙ Blaeschke, F. ...</div><div><strong>Pooled Knockin Targeting for Genome Engineering of Cellular Immunotherapies</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:728-744.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2020.03.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2020.03.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85083775641" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32302591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.03.039&amp;pmid=32302591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">58.</a></div><div>Sakuishi, K. ∙ Apetoh, L. ∙ Sullivan, J.M. ...</div><div><strong>Targeting Tim-3 and PD-1 pathways to reverse T&nbsp;cell exhaustion and restore anti-tumor immunity</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2010; <strong>207</strong>:2187-2194</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20100643" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-77957744369" target="_blank" aria-label="Open Scopus in new tab">Scopus (1546)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20819927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20100643&amp;pmid=20819927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">59.</a></div><div>Sato, K. ∙ Tsuyuzaki, K. ∙ Shimizu, K. ...</div><div><strong>CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1639-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85061352687" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30744683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1639-x&amp;pmid=30744683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">60.</a></div><div>Schuster, S.J. ∙ Svoboda, J. ∙ Chong, E.A. ...</div><div><strong>Chimeric Antigen Receptor T Cells in Refractory B-Cell Lymphomas</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2017; <strong>377</strong>:2545-2554</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1708566" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85040171486" target="_blank" aria-label="Open Scopus in new tab">Scopus (1380)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29226764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1708566&amp;pmid=29226764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">61.</a></div><div>Seyda, M. ∙ Elkhal, A. ∙ Quante, M. ...</div><div><strong>T Cells Going Innate</strong></div><div><em>Trends Immunol.</em> 2016; <strong>37</strong>:546-556</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2016.06.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.it.2016.06.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84991745789" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27402226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2016.06.004&amp;pmid=27402226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">62.</a></div><div>Shifrut, E. ∙ Carnevale, J. ∙ Tobin, V. ...</div><div><strong>Genome-wide CRISPR Screens in Primary Human T Cells Reveal Key Regulators of Immune Function</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:1958-1971.e1915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85059173331" target="_blank" aria-label="Open Scopus in new tab">Scopus (333)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30449619/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.10.024&amp;pmid=30449619" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63-1" title="View in article">63.</a></div><div>Singer, M. ∙ Wang, C. ∙ Cong, L. ...</div><div><strong>A Distinct Gene Module for Dysfunction Uncoupled from Activation in Tumor-Infiltrating T Cells</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:1500-1511.e1509</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2016.08.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84986285632" target="_blank" aria-label="Open Scopus in new tab">Scopus (269)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27610572/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.052&amp;pmid=27610572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">64.</a></div><div>Stadtmauer, E.A. ∙ Fraietta, J.A. ∙ Davis, M.M. ...</div><div><strong>CRISPR-engineered T&nbsp;cells in patients with refractory cancer</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:eaba7365</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba7365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85080829577" target="_blank" aria-label="Open Scopus in new tab">Scopus (894)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32029687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba7365&amp;pmid=32029687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">65.</a></div><div>Stromnes, I.M. ∙ Schmitt, T.M. ∙ Hulbert, A. ...</div><div><strong>T Cells Engineered against a Native Antigen Can Surmount Immunologic and Physical Barriers to Treat Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2015; <strong>28</strong>:638-652</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2015.09.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2015.09.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84946547794" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26525103/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2015.09.022&amp;pmid=26525103" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">66.</a></div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e1821</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (7328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">67.</a></div><div>Thommen, D.S. ∙ Schumacher, T.N.</div><div><strong>T Cell Dysfunction in Cancer</strong></div><div><em>Cancer Cell.</em> 2018; <strong>33</strong>:547-562</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2018.03.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.ccell.2018.03.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85043771297" target="_blank" aria-label="Open Scopus in new tab">Scopus (768)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29634943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2018.03.012&amp;pmid=29634943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68-1" title="View in article">68.</a></div><div>Trapnell, C. ∙ Cacchiarelli, D. ∙ Grimsby, J. ...</div><div><strong>The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells</strong></div><div><em>Nat. Biotechnol.</em> 2014; <strong>32</strong>:381-386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84900873950" target="_blank" aria-label="Open Scopus in new tab">Scopus (3557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2859&amp;pmid=24658644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">69.</a></div><div>UniProt Consortium</div><div><strong>UniProt: a worldwide hub of protein knowledge</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D506-D515</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85059797762" target="_blank" aria-label="Open Scopus in new tab">Scopus (5045)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30395287/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1049&amp;pmid=30395287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">70.</a></div><div>Utzschneider, D.T. ∙ Gabriel, S.S. ∙ Chisanga, D. ...</div><div><strong>Early precursor T&nbsp;cells establish and propagate T&nbsp;cell exhaustion in chronic infection</strong></div><div><em>Nat. Immunol.</em> 2020; <strong>21</strong>:1256-1266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-020-0760-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85089743781" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32839610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-020-0760-z&amp;pmid=32839610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">71.</a></div><div>van Montfoort, N. ∙ Borst, L. ∙ Korrer, M.J. ...</div><div><strong>NKG2A Blockade Potentiates CD8&nbsp;T Cell Immunity Induced by Cancer Vaccines</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:1744-1755.e1715</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2018.10.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85058015117" target="_blank" aria-label="Open Scopus in new tab">Scopus (232)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30503208/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.10.028&amp;pmid=30503208" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">72.</a></div><div>Wei, J. ∙ Long, L. ∙ Zheng, W. ...</div><div><strong>Targeting REGNASE-1 programs long-lived effector T&nbsp;cells for cancer therapy</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:471-476</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1821-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85076392269" target="_blank" aria-label="Open Scopus in new tab">Scopus (250)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31827283/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1821-z&amp;pmid=31827283" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">73.</a></div><div>Wencker, M. ∙ Turchinovich, G. ∙ Di Marco Barros, R. ...</div><div><strong>Innate-like T&nbsp;cells straddle innate and adaptive immunity by altering antigen-receptor responsiveness</strong></div><div><em>Nat. Immunol.</em> 2014; <strong>15</strong>:80-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2773" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84890937867" target="_blank" aria-label="Open Scopus in new tab">Scopus (167)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24241693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2773&amp;pmid=24241693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">74.</a></div><div>Wherry, E.J. ∙ Kurachi, M.</div><div><strong>Molecular and cellular insights into T&nbsp;cell exhaustion</strong></div><div><em>Nat. Rev. Immunol.</em> 2015; <strong>15</strong>:486-499</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nri3862" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-84937837169" target="_blank" aria-label="Open Scopus in new tab">Scopus (2951)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26205583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnri3862&amp;pmid=26205583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">75.</a></div><div>Yang, C.Y. ∙ Best, J.A. ∙ Knell, J. ...</div><div><strong>The transcriptional regulators Id2 and Id3 control the formation of distinct memory CD8+ T&nbsp;cell subsets</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:1221-1229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-81255144662" target="_blank" aria-label="Open Scopus in new tab">Scopus (305)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22057289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2158&amp;pmid=22057289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">76.</a></div><div>Zhang, L. ∙ Yu, X. ∙ Zheng, L. ...</div><div><strong>Lineage tracking reveals dynamic relationships of T&nbsp;cells in colorectal cancer</strong></div><div><em>Nature.</em> 2018; <strong>564</strong>:268-272</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0694-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85058518221" target="_blank" aria-label="Open Scopus in new tab">Scopus (694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30479382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0694-x&amp;pmid=30479382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">77.</a></div><div>Zheng, C. ∙ Zheng, L. ∙ Yoo, J.K. ...</div><div><strong>Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1342-1356.e1316</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=10.1016%2Fj.cell.2017.05.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85020894317" target="_blank" aria-label="Open Scopus in new tab">Scopus (1383)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.035&amp;pmid=28622514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78-1" title="View in article">78.</a></div><div>Zhou, Y. ∙ Zhou, B. ∙ Pache, L. ...</div><div><strong>Metascape provides a biologist-oriented resource for the analysis of systems-level datasets</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:1523</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-09234-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.016&amp;key=2-s2.0-85063948265" target="_blank" aria-label="Open Scopus in new tab">Scopus (7477)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30944313/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-09234-6&amp;pmid=30944313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/789df3bd-f72b-45fe-8867-8c6fdc2775e0/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/c829819e-c741-4233-8bbc-8ac085b847ef/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/86e7b4e6-4675-4056-93eb-8f59571f3249/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/e564fb8b-cd94-443f-8b5d-30f5fe9065b3/main.assets/gr2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/eca3dc33-7ec0-4644-86c2-4fb7d3f6a3f7/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/a1846af5-4119-4b0a-9070-33d5402b9e2f/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/d9d760e5-080d-4611-9300-a8e8ff6ae3f8/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/b7a221ac-275d-4693-b226-3395f4709c87/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/acbe6176-62a4-4e69-8020-45b91556ee12/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/5df5355f-7131-4595-ad45-4ec8096d258b/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/6e3f535d-b895-4b5e-b769-522f17500fc4/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/34554139-10e0-4f29-a08e-9ca403e4bd3b/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/36f14f38-52a6-4acb-b344-77e67fe9e02e/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/asset/2c9ef0f3-2156-468a-8fe6-e6eec9176d3e/main.assets/gr7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00262%3Bwgroup%3Astring%3ADefault+Website+Group%3Barticle%3Aarticle%3Apii%5C%3AS0092867421013313%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (6)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/attachment/6e338615-7ea3-40c1-9312-2332133e5419/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (96.43 KB)</span></a><div class="core-description">Table S1. Differentially expressed genes between day 0 and day 28 CAE CAR T&nbsp;cells from bulk RNA-seq experiments, related to Figure&nbsp;2A</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/attachment/60b88dbc-b15a-4535-bbc7-b364733375f6/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (39.33 KB)</span></a><div class="core-description">Table S2. Full list of Ingenuity Pathway Analysis pathways enriched in differentially expressed genes between day 0 and day 28 CAE CAR T&nbsp;cells from bulk RNA-seq experiments, related to Figure&nbsp;2C</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/attachment/01423c9f-d98a-471b-b1ce-17ef12bef6ba/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (44.34 KB)</span></a><div class="core-description">Table S3. All upstream regulators (transcription factors) predicted to regulate differentially expressed genes between day 0 and day 28 CAE CAR T&nbsp;cells from bulk RNA-seq experiments, related to Figure&nbsp;2F</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/attachment/7af3028b-01a6-4d17-ab09-e3eb7779c4f0/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (549.98 KB)</span></a><div class="core-description">Table S4. Differentially expressed genes between dysfunctional clusters (D20-1, D20-4) and non-dysfunctional clusters (D20-2, D20-3) from scRNA-seq experiment in donor ND388, related to Figure&nbsp;3E</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/attachment/ef785896-554e-476a-82fa-d2774da912bb/mmc5.pdf" download="mmc5.pdf"><span id="mmc5-link">PDF (51.30 KB)</span></a><div class="core-description">Table S5. Table of dysfunction signature genes (N&nbsp;= 30), related to Figures 3 and S3</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.016/attachment/d5b4e66f-2da7-4269-b791-4a4bee89e524/mmc6.xlsx" download="mmc6.xlsx"><span id="mmc6-link">Spreadsheet (32.84 KB)</span></a><div class="core-description">Table S6. All upstream regulators (transcription factors) predicted to regulate differentially expressed genes between day 0 and day 20 CAE CAR T&nbsp;cells from scRNA-seq experiment in donor ND388, related to Figure&nbsp;6A</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421013313" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(25)00075-X" target="_blank" title="Oncolytic adeno-immunotherapy improves allogeneic adoptive HER2.CAR-NK function against pancreatic ductal adenocarcinoma">Oncolytic adeno-immunotherapy improves allogeneic adoptive HER2.CAR-NK function against pancreatic ductal adenocarcinoma</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Biegert et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">May 26, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(25)00004-7" target="_blank" title="CAR T&nbsp;cells, CAR NK cells, and CAR macrophages exhibit distinct traits in glioma models but are similarly enhanced when combined with cytokines">CAR T&nbsp;cells, CAR NK cells, and CAR macrophages exhibit distinct traits in glioma models but are similarly enhanced when combined with cytokines</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Look et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">January 30, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(25)00114-5" target="_blank" title="Targeting BCL11B in CAR-engineered lymphoid progenitors drives NK-like cell development with prolonged anti-leukemic activity">Targeting BCL11B in CAR-engineered lymphoid progenitors drives NK-like cell development with prolonged anti-leukemic activity</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Baatz et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">February 14, 2025</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(25)00026-8" target="_blank" title="CD147-CAR-NK cell therapy shows minimal toxicities in human CD147 transgenic mouse model with solid tumors">CD147-CAR-NK cell therapy shows minimal toxicities in human CD147 transgenic mouse model with solid tumors</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sabha et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">February 26, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(25)00396-X" target="_blank" title="Intraperitoneal infusion of NKG2D CAR-NK cells induces endogenous CD8+ T cell activation in patients with advanced colorectal cancer">Intraperitoneal infusion of NKG2D CAR-NK cells induces endogenous CD8+ T cell activation in patients with advanced colorectal cancer</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Li et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">May 27, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(25)00312-0" target="_blank" title="ErbB2/HER2-targeted CAR-NK cells eliminate breast cancer cells in an organoid model that recapitulates tumor progression">ErbB2/HER2-targeted CAR-NK cells eliminate breast cancer cells in an organoid model that recapitulates tumor progression</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Röder et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">April 24, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/methods/fulltext/S2329-0501(24)00190-6" target="_blank" title="CAR-NK cells derived from cord blood originate mainly from CD56−CD7+CD34−HLA-DR−Lin− NK progenitor cells">CAR-NK cells derived from cord blood originate mainly from CD56−CD7+CD34−HLA-DR−Lin− NK progenitor cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wibowo et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Methods &amp; Clinical Development</i></span><span id="item_date">November 11, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(24)00895-5" target="_blank" title="Emerging combined CAR-NK cell therapies in cancer treatment: Finding a dancing partner">Emerging combined CAR-NK cell therapies in cancer treatment: Finding a dancing partner</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hosseinalizadeh et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">January 2, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)01473-6" target="_blank" title="Overcoming CD226-related immune evasion in acute myeloid leukemia with CD38 CAR-engineered NK cells">Overcoming CD226-related immune evasion in acute myeloid leukemia with CD38 CAR-engineered NK cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Melo Garcia et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">January 3, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(24)00169-3" target="_blank" title="Combinatorial immunotherapy with anti-ROR1 CAR NK cells and an IL-21 secreting oncolytic virus against neuroblastoma">Combinatorial immunotherapy with anti-ROR1 CAR NK cells and an IL-21 secreting oncolytic virus against neuroblastoma</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chu et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">December 21, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/methods/fulltext/S2329-0501(24)00144-X" target="_blank" title="Linker-specific monoclonal antibodies present a simple and reliable detection method for scFv-based CAR NK cells">Linker-specific monoclonal antibodies present a simple and reliable detection method for scFv-based CAR NK cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Schindler et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Methods &amp; Clinical Development</i></span><span id="item_date">August 21, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(21)01330-1" target="_blank" title="Killer fatigue: Transition to NK-cell-like phenotype is a signature of CAR-T cell exhaustion">Killer fatigue: Transition to NK-cell-like phenotype is a signature of CAR-T cell exhaustion</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hong et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">December 09, 2021</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(24)00320-4" target="_blank" title="Engineering of potent CAR NK cells using non-viral Sleeping Beauty transposition from minimalistic DNA vectors">Engineering of potent CAR NK cells using non-viral Sleeping Beauty transposition from minimalistic DNA vectors</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Bexte et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">May 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(24)00019-5" target="_blank" title="Empowering pancreatic tumor homing with augmented anti-tumor potency of CXCR2-tethered CAR-NK cells">Empowering pancreatic tumor homing with augmented anti-tumor potency of CXCR2-tethered CAR-NK cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yoon et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">February 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(25)00809-0" target="_blank" title="CD28 signaling domain boosts persistence and in vivo anti-tumor activity of stem cell-derived CD19-CAR-NK cells">CD28 signaling domain boosts persistence and in vivo anti-tumor activity of stem cell-derived CD19-CAR-NK cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Kok et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">April 29, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(25)00175-3" target="_blank" title="Myeloma cell-intrinsic ANXA1 elevation and T&nbsp;cell dysfunction contribute to BCMA-negative relapse after CAR-T therapy">Myeloma cell-intrinsic ANXA1 elevation and T&nbsp;cell dysfunction contribute to BCMA-negative relapse after CAR-T therapy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yang et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">March 7, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(25)00152-1" target="_blank" title="Microneedle delivery of CAR-M-like engineered macrophages alleviates intervertebral disc degeneration through enhanced efferocytosis capacity">Microneedle delivery of CAR-M-like engineered macrophages alleviates intervertebral disc degeneration through enhanced efferocytosis capacity</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhou et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">April 7, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/med/fulltext/S2666-6340(25)00104-7" target="_blank" title="Emerging trends in clinical allogeneic CAR cell therapy">Emerging trends in clinical allogeneic CAR cell therapy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Li et al.</li></ul></div><div class="linked-article__details"><span><i>Med</i></span><span id="item_date">May 13, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(24)00660-8" target="_blank" title="CD70-targeted iPSC-derived CAR-NK cells display potent function against tumors and alloreactive T&nbsp;cells">CD70-targeted iPSC-derived CAR-NK cells display potent function against tumors and alloreactive T&nbsp;cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">January 9, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(24)00139-5" target="_blank" title="A novel mitochondrial pyruvate carrier inhibitor drives stem cell-like memory CAR T cell generation and enhances antitumor efficacy">A novel mitochondrial pyruvate carrier inhibitor drives stem cell-like memory CAR T cell generation and enhances antitumor efficacy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wenes et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">October 18, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(25)00093-X" target="_blank" title="CD4+ anti-TGF-β CAR T&nbsp;cells and CD8+ conventional CAR T&nbsp;cells exhibit synergistic antitumor effects">CD4+ anti-TGF-β CAR T&nbsp;cells and CD8+ conventional CAR T&nbsp;cells exhibit synergistic antitumor effects</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zheng et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">March 18, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(25)00280-1" target="_blank" title="High-dimensional temporal mapping of CAR T cells reveals phenotypic and functional remodeling during manufacturing">High-dimensional temporal mapping of CAR T cells reveals phenotypic and functional remodeling during manufacturing</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Cadinanos-Garai et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">May 1, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(25)00271-5" target="_blank" title="Engineering sonogenetic EchoBack-CAR T cells">Engineering sonogenetic EchoBack-CAR T cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">April 2, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)03227-4" target="_blank" title="CAR-NK cells in combination therapy against cancer: A potential paradigm">CAR-NK cells in combination therapy against cancer: A potential paradigm</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Li et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">February 29, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(24)00044-4" target="_blank" title="Bortezomib promotes the TRAIL-mediated killing of resistant rhabdomyosarcoma by ErbB2/Her2-targeted CAR-NK-92 cells via DR5 upregulation">Bortezomib promotes the TRAIL-mediated killing of resistant rhabdomyosarcoma by ErbB2/Her2-targeted CAR-NK-92 cells via DR5 upregulation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Heim et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">April 10, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2372-7705(22)00133-4" target="_blank" title="Gene-edited and CAR-NK cells: Opportunities and challenges with engineering of NK cells for immunotherapy">Gene-edited and CAR-NK cells: Opportunities and challenges with engineering of NK cells for immunotherapy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wu et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy - Oncolytics</i></span><span id="item_date">November 3, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(24)00075-4" target="_blank" title="Regulation of CAR transgene expression to design semiautonomous CAR-T">Regulation of CAR transgene expression to design semiautonomous CAR-T</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Głowacki et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">June 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(24)00057-2" target="_blank" title="Toll-like receptor 4 signaling activation domains promote CAR T&nbsp;cell function against solid tumors">Toll-like receptor 4 signaling activation domains promote CAR T&nbsp;cell function against solid tumors</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Mikolič et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">May 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/methods/fulltext/S2329-0501(22)00152-8" target="_blank" title="Co-transducing B7H3 CAR-NK cells with the DNR preserves their cytolytic function against GBM in the presence of exogenous TGF-β">Co-transducing B7H3 CAR-NK cells with the DNR preserves their cytolytic function against GBM in the presence of exogenous TGF-β</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chaudhry et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy - Methods &amp; Clinical Development</i></span><span id="item_date">October 20, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(25)00125-X" target="_blank" title="CAR-mediated target recognition limits TCR-mediated target recognition of TCR- and CAR-dual-receptor-edited T&nbsp;cells">CAR-mediated target recognition limits TCR-mediated target recognition of TCR- and CAR-dual-receptor-edited T&nbsp;cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wachsmann et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">February 27, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cancer-cell/fulltext/S1535-6108(25)00120-5" target="_blank" title="CAR-T therapy in solid tumors">CAR-T therapy in solid tumors</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Du et al.</li></ul></div><div class="linked-article__details"><span><i>Cancer Cell</i></span><span id="item_date">April 14, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(25)00193-5" target="_blank" title="Immunophenotype of CAR T&nbsp;cells and apheresis products predicts response in CD22 CAR T&nbsp;cell trial for B cell acute lymphoblastic leukemia">Immunophenotype of CAR T&nbsp;cells and apheresis products predicts response in CD22 CAR T&nbsp;cell trial for B cell acute lymphoblastic leukemia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Dreyzin et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">March 14, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(21)00518-9" target="_blank" title="IL12 integrated into the CAR exodomain converts CD8+ T&nbsp;cells to poly-functional NK-like cells with superior killing of antigen-loss tumors">IL12 integrated into the CAR exodomain converts CD8+ T&nbsp;cells to poly-functional NK-like cells with superior killing of antigen-loss tumors</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hombach et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">October 19, 2021</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/immunology/fulltext/S1471-4906(25)00120-6" target="_blank" title="Using TCR-CAR dual signaling for precise cancer targeting">Using TCR-CAR dual signaling for precise cancer targeting</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Shahbazy et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Immunology</i></span><span id="item_date">May 13, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/pharmacological-sciences/fulltext/S0165-6147(24)00270-0" target="_blank" title="Boosting CAR-T cell therapy through vaccine synergy">Boosting CAR-T cell therapy through vaccine synergy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Li et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Pharmacological Sciences</i></span><span id="item_date">January 3, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(25)00049-9" target="_blank" title="Cost-effective strategies for CAR-T cell therapy manufacturing">Cost-effective strategies for CAR-T cell therapy manufacturing</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Abdo et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">April 3, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-biomaterials/fulltext/S3050-5623(25)00002-9" target="_blank" title="Leveraging T&nbsp;cell-specific fusogenicity of HIV for in&nbsp;vivo mRNA delivery to produce human CAR-T cells">Leveraging T&nbsp;cell-specific fusogenicity of HIV for in&nbsp;vivo mRNA delivery to produce human CAR-T cells</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Biomaterials</i></span><span id="item_date">March 10, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(25)00122-4" target="_blank" title="Presetting CAR-T cells during ex&nbsp;vivo biomanufacturing">Presetting CAR-T cells during ex&nbsp;vivo biomanufacturing</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy</i></span><span id="item_date">February 22, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/oncology/fulltext/S2950-3299(24)00062-6" target="_blank" title="Efficiently targeting neuroblastoma with the combination of anti-ROR1 CAR NK cells and N-803 in&nbsp;vitro and in&nbsp;vivo in NB xenografts">Efficiently targeting neuroblastoma with the combination of anti-ROR1 CAR NK cells and N-803 in&nbsp;vitro and in&nbsp;vivo in NB xenografts</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chu et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Oncology</i></span><span id="item_date">May 24, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(25)00391-5" target="_blank" title="Immunotherapy-related cognitive impairment after CAR T cell therapy in mice">Immunotherapy-related cognitive impairment after CAR T cell therapy in mice</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Geraghty et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">May 12, 2025</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.11.016","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">An NK-like CAR T&nbsp;cell transition in CAR T&nbsp;cell dysfunction</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.016/asset/789df3bd-f72b-45fe-8867-8c6fdc2775e0/main.assets/gr1.jpg" height="942" width="740" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/de9cfe42-908e-4460-a881-87da4644f22c/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.016/asset/c829819e-c741-4233-8bbc-8ac085b847ef/main.assets/figs1.jpg" height="849" width="657" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/a184ead6-5536-4209-93e0-b143109e1ccd/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.016/asset/86e7b4e6-4675-4056-93eb-8f59571f3249/main.assets/figs2.jpg" height="589" width="649" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/5ef7d8fa-1fc6-44a5-8e26-2033b832d5fc/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.016/asset/e564fb8b-cd94-443f-8b5d-30f5fe9065b3/main.assets/gr2.jpg" height="920" width="753" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/59263fd7-8743-4ed2-860e-f1319d408528/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.016/asset/eca3dc33-7ec0-4644-86c2-4fb7d3f6a3f7/main.assets/figs3.jpg" height="941" width="752" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/ee965794-8cd6-40a4-b8c2-cebff52133aa/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.016/asset/a1846af5-4119-4b0a-9070-33d5402b9e2f/main.assets/figs4.jpg" height="941" width="728" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/3732359c-b9e9-4142-9477-1f18ce7a753c/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.016/asset/d9d760e5-080d-4611-9300-a8e8ff6ae3f8/main.assets/gr3.jpg" height="942" width="732" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/92f2db90-9841-4ba6-9b7f-1688f44659c3/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.016/asset/b7a221ac-275d-4693-b226-3395f4709c87/main.assets/figs5.jpg" height="941" width="692" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/b98f4ef2-4c2d-4f4d-8146-9a2465470733/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.016/asset/acbe6176-62a4-4e69-8020-45b91556ee12/main.assets/gr4.jpg" height="941" width="726" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/65dc4b4b-ec84-49ed-b5d5-144c660ac7b3/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.016/asset/5df5355f-7131-4595-ad45-4ec8096d258b/main.assets/figs6.jpg" height="872" width="657" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/4861d17a-ca65-489d-8a7d-5e346ed66dc1/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.016/asset/6e3f535d-b895-4b5e-b769-522f17500fc4/main.assets/gr5.jpg" height="933" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/e3ce779e-2096-4093-a3c7-5e14eedfe7a3/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.016/asset/34554139-10e0-4f29-a08e-9ca403e4bd3b/main.assets/gr6.jpg" height="902" width="753" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/dce210f6-685b-4a7d-9f7c-fb4fa917bc9a/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.016/asset/36f14f38-52a6-4acb-b344-77e67fe9e02e/main.assets/figs7.jpg" height="852" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/6741ca22-e60f-4dbd-a345-be84f3011308/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.016/asset/2c9ef0f3-2156-468a-8fe6-e6eec9176d3e/main.assets/gr7.jpg" height="941" width="717" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/44e130bc-ba94-4348-8679-54b729f0455c/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.016/asset/789df3bd-f72b-45fe-8867-8c6fdc2775e0/main.assets/gr1.jpg" height="942" width="740" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/de9cfe42-908e-4460-a881-87da4644f22c/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.016/asset/c829819e-c741-4233-8bbc-8ac085b847ef/main.assets/figs1.jpg" height="849" width="657" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/a184ead6-5536-4209-93e0-b143109e1ccd/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.016/asset/86e7b4e6-4675-4056-93eb-8f59571f3249/main.assets/figs2.jpg" height="589" width="649" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/5ef7d8fa-1fc6-44a5-8e26-2033b832d5fc/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.016/asset/e564fb8b-cd94-443f-8b5d-30f5fe9065b3/main.assets/gr2.jpg" height="920" width="753" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/59263fd7-8743-4ed2-860e-f1319d408528/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.016/asset/eca3dc33-7ec0-4644-86c2-4fb7d3f6a3f7/main.assets/figs3.jpg" height="941" width="752" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/ee965794-8cd6-40a4-b8c2-cebff52133aa/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.016/asset/a1846af5-4119-4b0a-9070-33d5402b9e2f/main.assets/figs4.jpg" height="941" width="728" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/3732359c-b9e9-4142-9477-1f18ce7a753c/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.016/asset/d9d760e5-080d-4611-9300-a8e8ff6ae3f8/main.assets/gr3.jpg" height="942" width="732" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/92f2db90-9841-4ba6-9b7f-1688f44659c3/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.016/asset/b7a221ac-275d-4693-b226-3395f4709c87/main.assets/figs5.jpg" height="941" width="692" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/b98f4ef2-4c2d-4f4d-8146-9a2465470733/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.016/asset/acbe6176-62a4-4e69-8020-45b91556ee12/main.assets/gr4.jpg" height="941" width="726" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/65dc4b4b-ec84-49ed-b5d5-144c660ac7b3/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.016/asset/5df5355f-7131-4595-ad45-4ec8096d258b/main.assets/figs6.jpg" height="872" width="657" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/4861d17a-ca65-489d-8a7d-5e346ed66dc1/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.016/asset/6e3f535d-b895-4b5e-b769-522f17500fc4/main.assets/gr5.jpg" height="933" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/e3ce779e-2096-4093-a3c7-5e14eedfe7a3/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.016/asset/34554139-10e0-4f29-a08e-9ca403e4bd3b/main.assets/gr6.jpg" height="902" width="753" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/dce210f6-685b-4a7d-9f7c-fb4fa917bc9a/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.016/asset/36f14f38-52a6-4acb-b344-77e67fe9e02e/main.assets/figs7.jpg" height="852" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/6741ca22-e60f-4dbd-a345-be84f3011308/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.016/asset/2c9ef0f3-2156-468a-8fe6-e6eec9176d3e/main.assets/gr7.jpg" height="941" width="717" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.016/asset/44e130bc-ba94-4348-8679-54b729f0455c/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01331-3';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01331-3';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749592066436","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"184","issueNumber":"25","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421013313","coverDate":"202112","type":"XOCS-JOURNAL:SCOPE-FULL","title":"An NKlike CAR Txa0cell transition in CAR Txa0cell dysfunction","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc11a74ea252eb',t:'MTc0OTU5MjA2NS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc11a74ea252eb&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.11.016&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>