<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="9cd3a54a-24c6-41e4-b0a0-74024b18f468" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;article:article:pii\:S0092867421015427;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text;issue:issue:pii\:S0092867421X00049">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Genomic structure predicts metabolite dynamics in microbial communities">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)01542-7">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/d5296a79-1be1-4045-bded-e10f51ff5a60/fx1.jpg">
    


        <meta property="og:description" content="The presence or absence of specific genes within communities of wild bacterial isolates
is sufficient to predict community-level metabolite dynamics without detailed knowledge
of pathway regulation or complex ecological processes.">
    











        <meta name="twitter:title" content="Genomic structure predicts metabolite dynamics in microbial communities">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/d5296a79-1be1-4045-bded-e10f51ff5a60/fx1.jpg">
    


        <meta name="twitter:description" content="The presence or absence of specific genes within communities of wild bacterial isolates
is sufficient to predict community-level metabolite dynamics without detailed knowledge
of pathway regulation or complex ecological processes.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)01542-7">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Genomic structure predicts metabolite dynamics in microbial communities: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Genomic structure predicts metabolite dynamics in microbial communities"> <meta name="citation_date" content="2022/02/03"> <meta name="citation_online_date" content="2022/01/26"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="3"> <meta name="citation_firstpage" content="530"> <meta name="citation_lastpage" content="546.e25"> <meta name="citation_pmid" content="35085485"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421015427"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421015427"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421015427"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.12.036"> <meta name="citation_pmid" content="35085485"> <meta name="citation_author" content="Karna Gowda"><meta name="citation_author_institution" content="Department of Ecology and Evolution, University of Chicago, Chicago, IL 60637, USA"><meta name="citation_author_institution" content="Center for the Physics of Evolving Systems, University of Chicago, Chicago, IL 60637, USA"><meta name="citation_author" content="Derek Ping"><meta name="citation_author_institution" content="Department of Physics, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA"><meta name="citation_author" content="Madhav Mani"><meta name="citation_author_email" content="madhav.mani@gmail.com"><meta name="citation_author_institution" content="Department of Engineering Sciences and Applied Mathematics, Northwestern University, Evanston, IL 60208, USA"><meta name="citation_author_institution" content="Department of Molecular Biosciences, Northwestern University, Evanston, IL 60208, USA"><meta name="citation_author_institution" content="NSF-Simons Center for Quantitative Biology, Northwestern University, Northwestern University, Evanston, IL 60208, USA"><meta name="citation_author" content="Seppe Kuehn"><meta name="citation_author_email" content="seppe.kuehn@gmail.com"><meta name="citation_author_institution" content="Department of Ecology and Evolution, University of Chicago, Chicago, IL 60637, USA"><meta name="citation_author_institution" content="Center for the Physics of Evolving Systems, University of Chicago, Chicago, IL 60637, USA"><meta name="citation_keywords" content="microbial ecology; microbial communities; biological physics; statistical genomics; microbial community metabolism; denitrification; population dynamics; microbial interactions; community assembly; theoretical ecology"><meta name="citation_reference" content="citation_title=Purification and characterization of the dissimilatory nitrite reductase from Alcaligenes xylosoxidans subsp. xylosoxidans (N.C.I.M.B. 11015): evidence for the presence of both type 1 and type 2 copper centres; citation_author=Z.H.L. Abraham; citation_author=D.J. Lowe; citation_author=B.E. Smith; citation_journal_title=Biochem. J.; citation_volume=295; citation_firstpage=587; citation_lastpage=593; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Thousands of microbial genomes shed light on interconnected biogeochemical processes in an aquifer system; citation_author=K. Anantharaman; citation_author=C.T. Brown; citation_author=L.A. Hug; citation_author=I. Sharon; citation_author=C.J. Castelle; citation_author=A.J. Probst; citation_author=B.C. Thomas; citation_author=A. Singh; citation_author=M.J. Wilkins; citation_author=U. Karaoz; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=7; citation_firstpage=13219; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing; citation_author=A. Bankevich; citation_author=S. Nurk; citation_author=D. Antipov; citation_author=A.A. Gurevich; citation_author=M. Dvorkin; citation_author=A.S. Kulikov; citation_author=V.M. Lesin; citation_author=S.I. Nikolenko; citation_author=S. Pham; citation_author=A.D. Prjibelski; citation_author=et al.; citation_journal_title=J.&nbsp;Comput. Biol.; citation_volume=19; citation_firstpage=455; citation_lastpage=477; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Kinetics of dissimilatory nitrate and nitrite reduction in suspended growth culture; citation_author=M. Beccari; citation_author=R. Passino; citation_author=R. Ramadori; citation_author=V. Tandoi; citation_journal_title=J.&nbsp;Water Pollut. Control Fed.; citation_volume=55; citation_firstpage=58; citation_lastpage=64; citation_publication_date=1983; "><meta name="citation_reference" content="citation_title=Periplasmic and membrane-bound respiratory nitrate reductases in Thiosphaera pantotropha: the periplasmic enzyme catalyzes the first step in aerobic denitrification; citation_author=L.C. Bell; citation_author=D.J. Richardson; citation_author=S.J. Ferguson; citation_journal_title=FEBS Lett.; citation_volume=265; citation_firstpage=85; citation_lastpage=87; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=Mapping the stereotyped behaviour of freely moving fruit flies; citation_author=G.J. Berman; citation_author=D.M. Choi; citation_author=W. Bialek; citation_author=J.W. Shaevitz; citation_journal_title=J.&nbsp;R. Soc. Interface; citation_volume=11; citation_firstpage=20140672; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A cytochrome cd1-type nitrite reductase isolated from the marine denitrifier Pseudomonas nautica 617: purification and characterization; citation_author=S. Besson; citation_author=C. Carneiro; citation_author=J.J. Moura; citation_author=I. Moura; citation_author=G. Fauque; citation_journal_title=Anaerobe; citation_volume=1; citation_firstpage=219; citation_lastpage=226; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data; citation_author=A.M. Bolger; citation_author=M. Lohse; citation_author=B. Usadel; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=2114; citation_lastpage=2120; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Marker exchange of the structural genes for nitric oxide reductase blocks the denitrification pathway of Pseudomonas stutzeri at nitric oxide; citation_author=C. Braun; citation_author=W.G. Zumft; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=266; citation_firstpage=22785; citation_lastpage=22788; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=RASTtk: a modular and extensible implementation of the RAST algorithm for building custom annotation pipelines and annotating batches of genomes; citation_author=T. Brettin; citation_author=J.J. Davis; citation_author=T. Disz; citation_author=R.A. Edwards; citation_author=S. Gerdes; citation_author=G.J. Olsen; citation_author=R. Olson; citation_author=R. Overbeek; citation_author=B. Parrello; citation_author=G.D. Pusch; citation_author=et al.; citation_journal_title=Sci. Rep.; citation_volume=5; citation_firstpage=8365; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The evolution and future of earth's nitrogen cycle; citation_author=D.E. Canfield; citation_author=A.N. Glazer; citation_author=P.G. Falkowski; citation_journal_title=Science; citation_volume=330; citation_firstpage=192; citation_lastpage=196; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Rapid, inexpensive measurement of synthetic bacterial community composition by Sanger sequencing of amplicon mixtures; citation_author=N. Cermak; citation_author=M.S. Datta; citation_author=A. Conwill; citation_journal_title=iScience; citation_volume=23; citation_firstpage=100915; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Physiology and kinetics of autotrophic denitrification by Thiobacillus denitrificans; citation_author=G. Claus; citation_author=H. Kutzner; citation_journal_title=Appl. Microbiol. Biotechnol.; citation_volume=22; citation_firstpage=283; citation_lastpage=288; citation_publication_date=1985; "><meta name="citation_reference" content="citation_title=High-throughput methods for culturing microorganisms in very-low-nutrient media yield diverse new marine isolates; citation_author=S.A. Connon; citation_author=S.J. Giovannoni; citation_journal_title=Appl. Environ. Microbiol.; citation_volume=68; citation_firstpage=3878; citation_lastpage=3885; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Public good dynamics drive evolution of iron acquisition strategies in natural bacterioplankton populations; citation_author=O.X. Cordero; citation_author=L.A. Ventouras; citation_author=E.F. DeLong; citation_author=M.F. Polz; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_firstpage=20059; citation_lastpage=20064; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Energetics of Bacillus stearothermophilus growth: molar growth yield and temperature effects on growth efficiency; citation_author=T.P. Coultate; citation_author=T.K. Sundaram; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=121; citation_firstpage=55; citation_lastpage=64; citation_publication_date=1975; "><meta name="citation_reference" content="citation_title=Automated adaptive inference of phenomenological dynamical models; citation_author=B.C. Daniels; citation_author=I. Nemenman; citation_journal_title=Nat. Commun.; citation_volume=6; citation_firstpage=8133; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Microbial interactions lead to rapid micro-scale successions on model marine particles; citation_author=M.S. Datta; citation_author=E. Sliwerska; citation_author=J. Gore; citation_author=M.F. Polz; citation_author=O.X. Cordero; citation_journal_title=Nat. Commun.; citation_volume=7; citation_firstpage=11965; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The denitrifying nitrite reductase of Bacillus halodenitrificans; citation_author=G. Denariaz; citation_author=W.J. Payne; citation_author=J. LeGall; citation_journal_title=Biochim. Biophys. Acta Bioenerg.; citation_volume=1056; citation_firstpage=225; citation_lastpage=232; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Kinetics of sequential nitrification and denitrification processes; citation_author=A.R. Dinçer; citation_author=F. Kargı; citation_journal_title=Enzyme Microb. Technol.; citation_volume=27; citation_firstpage=37; citation_lastpage=42; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Dynamic substrate preferences predict metabolic properties of a simple microbial consortium; citation_author=O. Erbilgin; citation_author=B.P. Bowen; citation_author=S.M. Kosina; citation_author=S. Jenkins; citation_author=R.K. Lau; citation_author=T.R. Northen; citation_journal_title=BMC Bioinformatics; citation_volume=18; citation_firstpage=57; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The microbial engines that drive earth's biogeochemical cycles; citation_author=P.G. Falkowski; citation_author=T. Fenchel; citation_author=E.F. Delong; citation_journal_title=Science; citation_volume=320; citation_firstpage=1034; citation_lastpage=1039; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=The enzymes and bioenergetics of bacterial nitrate, nitrite, nitric oxide and nitrous oxide respiration; citation_inbook_title=Respiration in Archaea and Bacteria: Diversity of Prokaryotic Respiratory Systems; citation_author=S.J. Ferguson; citation_author=D.J. Richardson; citation_firstpage=169; citation_lastpage=206; citation_publication_date=2004; citation_publisher=Springer; "><meta name="citation_reference" content="citation_title=Comparative metagenomic, phylogenetic and physiological analyses of soil microbial communities across nitrogen gradients; citation_author=N. Fierer; citation_author=C.L. Lauber; citation_author=K.S. Ramirez; citation_author=J. Zaneveld; citation_author=M.A. Bradford; citation_author=R. Knight; citation_journal_title=ISME J.; citation_volume=6; citation_firstpage=1007; citation_lastpage=1017; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Growth rate of polypeptide chains as a function of the cell growth rate in a mutant of Escherichia coli 15; citation_author=J. Forchhammer; citation_author=L. Lindahl; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=55; citation_firstpage=563; citation_lastpage=568; citation_publication_date=1971; "><meta name="citation_reference" content="citation_title=Evolution of generalists by phenotypic plasticity; citation_author=D.T. Fraebel; citation_author=K. Gowda; citation_author=M. Mani; citation_author=S. Kuehn; citation_journal_title=iScience; citation_volume=23; citation_firstpage=101678; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Relationships between genomic G+C content, RNA secondary structures, and optimal growth temperature in prokaryotes; citation_author=N. Galtier; citation_author=J.R. Lobry; citation_journal_title=J.&nbsp;Mol. Evol.; citation_volume=44; citation_firstpage=632; citation_lastpage=636; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Copper-containing nitrite reductase from Pseudomonas aureofaciens is functional in a mutationally cytochrome cd1-free background (NirS-) of Pseudomonas stutzeri; citation_author=A.B. Glockner; citation_author=A. Jüngst; citation_author=W.G. Zumft; citation_journal_title=Arch. Microbiol.; citation_volume=160; citation_firstpage=18; citation_lastpage=26; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=The Paracoccus denitrificans NarK-like nitrate and nitrite transporters-probing nitrate uptake and nitrate/nitrite exchange mechanisms; citation_author=A.D. Goddard; citation_author=S. Bali; citation_author=D.A. Mavridou; citation_author=V.M. Luque-Almagro; citation_author=A.J. Gates; citation_author=M. Dolores Roldán; citation_author=S. Newstead; citation_author=D.J. Richardson; citation_author=S.J. Ferguson; citation_journal_title=Mol. Microbiol.; citation_volume=103; citation_firstpage=117; citation_lastpage=133; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Interdependence of two NarK domains in a fused nitrate/nitrite transporter; citation_author=A.D. Goddard; citation_author=J.W.B. Moir; citation_author=D.J. Richardson; citation_author=S.J. Ferguson; citation_journal_title=Mol. Microbiol.; citation_volume=70; citation_firstpage=667; citation_lastpage=681; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Emergent simplicity in microbial community assembly; citation_author=J.E. Goldford; citation_author=N. Lu; citation_author=D. Bajić; citation_author=S. Estrela; citation_author=M. Tikhonov; citation_author=A. Sanchez-Gorostiaga; citation_author=D. Segrè; citation_author=P. Mehta; citation_author=A. Sanchez; citation_journal_title=Science; citation_volume=361; citation_firstpage=469; citation_lastpage=474; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Structure and kinetic properties of Paracoccus pantotrophus cytochrome cd1 nitrite reductase with the d1 heme active site ligand tyrosine 25 replaced by serine; citation_author=E.H.J. Gordon; citation_author=T. Sjögren; citation_author=M. Löfqvist; citation_author=C.D. Richter; citation_author=J.W. Allen; citation_author=C.W. Higham; citation_author=J. Hajdu; citation_author=V. Fülöp; citation_author=S.J. Ferguson; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=278; citation_firstpage=11773; citation_lastpage=11781; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Intergenomic comparisons highlight modularity of the denitrification pathway and underpin the importance of community structure for N2O emissions; citation_author=D.R.H. Graf; citation_author=C.M. Jones; citation_author=S. Hallin; citation_journal_title=PLoS ONE; citation_volume=9; citation_firstpage=e114118; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=QUAST: quality assessment tool for genome assemblies; citation_author=A. Gurevich; citation_author=V. Saveliev; citation_author=N. Vyahhi; citation_author=G. Tesler; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=1072; citation_lastpage=1075; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Metabolic resource allocation in individual microbes determines ecosystem interactions and spatial dynamics; citation_author=W.R. Harcombe; citation_author=W.J. Riehl; citation_author=I. Dukovski; citation_author=B.R. Granger; citation_author=A. Betts; citation_author=A.H. Lang; citation_author=G. Bonilla; citation_author=A. Kar; citation_author=N. Leiby; citation_author=P. Mehta; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=7; citation_firstpage=1104; citation_lastpage=1115; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Effects of heavy metals on Pseudomonas aeruginosa and Bacillus thuringiensis; citation_author=A. Hassen; citation_author=N. Saidi; citation_author=M. Cherif; citation_author=A. Boudabous; citation_journal_title=Bioresour. Technol.; citation_volume=65; citation_firstpage=73; citation_lastpage=82; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=The Elements of Statistical Learning; citation_author=T. Hastie; citation_author=R. Tibshirani; citation_author=J. Friedman; citation_publication_date=2008; citation_publisher=Springer; "><meta name="citation_reference" content="citation_title=Statistical learning with sparsity; citation_author=T. Hastie; citation_author=R.J. Tibshirani; citation_author=R. Wainwright; citation_publication_date=2016; citation_publisher=CRC Press; "><meta name="citation_reference" content="citation_title=The evolution of genomic base composition in bacteria; citation_author=E. Haywood-Farmer; citation_author=S.P. Otto; citation_journal_title=Evolution; citation_volume=57; citation_firstpage=1783; citation_lastpage=1792; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Effects of varying the carbon source limiting growth on yield and maintenance characteristics of Escherichia coli in continuous culture; citation_author=W.P. Hempfling; citation_author=S.E. Mainzer; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=123; citation_firstpage=1076; citation_lastpage=1087; citation_publication_date=1975; "><meta name="citation_reference" content="citation_title=Genome streamlining and the elemental costs of growth; citation_author=D.O. Hessen; citation_author=P.D. Jeyasingh; citation_author=M. Neiman; citation_author=L.J. Weider; citation_journal_title=Trends Ecol. Evol.; citation_volume=25; citation_firstpage=75; citation_lastpage=80; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The incidence of nirS and nirK and their genetic heterogeneity in cultivated denitrifiers; citation_author=K. Heylen; citation_author=D. Gevers; citation_author=B. Vanparys; citation_author=L. Wittebolle; citation_author=J. Geets; citation_author=N. Boon; citation_author=P. De Vos; citation_journal_title=Environ. Microbiol.; citation_volume=8; citation_firstpage=2012; citation_lastpage=2021; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Cultivation of denitrifying bacteria: optimization of isolation conditions and diversity study; citation_author=K. Heylen; citation_author=B. Vanparys; citation_author=L. Wittebolle; citation_author=W. Verstraete; citation_author=N. Boon; citation_author=P. De Vos; citation_journal_title=Appl. Environ. Microbiol.; citation_volume=72; citation_firstpage=2637; citation_lastpage=2643; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Structure, function and diversity of the healthy human microbiome; citation_author=Human Microbiome Project Consortium; citation_journal_title=Nature; citation_volume=486; citation_firstpage=207; citation_lastpage=214; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Physiological roles of two dissimilatory nitrate reductases in the deep-sea denitrifier Pseudomonas sp. strain MT-1; citation_author=E. Ikeda; citation_author=S. Andou; citation_author=U. Iwama; citation_author=C. Kato; citation_author=K. Horikoshi; citation_author=H. Tamegai; citation_journal_title=Biosci. Biotechnol. Biochem.; citation_volume=73; citation_firstpage=896; citation_lastpage=900; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The multifaceted role of the intestinal microbiota in colon cancer; citation_author=T. Irrazábal; citation_author=A. Belcheva; citation_author=S.E. Girardin; citation_author=A. Martin; citation_author=D.J. Philpott; citation_journal_title=Mol. Cell; citation_volume=54; citation_firstpage=309; citation_lastpage=320; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A nitrite reductase from Achromobacter cyclastes; citation_author=H. Iwasaki; citation_author=T. Matsubara; citation_journal_title=J.&nbsp;Biochem.; citation_volume=71; citation_firstpage=645; citation_lastpage=652; citation_publication_date=1972; "><meta name="citation_reference" content="citation_title=Ecological and evolutionary factors underlying global and local assembly of denitrifier communities; citation_author=C.M. Jones; citation_author=S. Hallin; citation_journal_title=ISME J.; citation_volume=4; citation_firstpage=633; citation_lastpage=641; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Phylogenetic analysis of nitrite, nitric oxide, and nitrous oxide respiratory enzymes reveal a complex evolutionary history for denitrification; citation_author=C.M. Jones; citation_author=B. Stres; citation_author=M. Rosenquist; citation_author=S. Hallin; citation_journal_title=Mol. Biol. Evol.; citation_volume=25; citation_firstpage=1955; citation_lastpage=1966; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Purification and properties of a copper-containing nitrite reductase from a denitrifying bacterium, Alcaligenes faecalis strain S-6; citation_author=T. Kakutani; citation_author=H. Watanabe; citation_author=K. Arima; citation_author=T. Beppu; citation_journal_title=J.&nbsp;Biochem.; citation_volume=89; citation_firstpage=453; citation_lastpage=461; citation_publication_date=1981; "><meta name="citation_reference" content="citation_title=phylosignal: an R package to measure, test, and explore the phylogenetic signal; citation_author=F. Keck; citation_author=F. Rimet; citation_author=A. Bouchez; citation_author=A. Franc; citation_journal_title=Ecol. Evol.; citation_volume=6; citation_firstpage=2774; citation_lastpage=2780; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Massively parallel screening of synthetic microbial communities; citation_author=J. Kehe; citation_author=A. Kulesa; citation_author=A. Ortiz; citation_author=C.M. Ackerman; citation_author=S.G. Thakku; citation_author=D. Sellers; citation_author=S. Kuehn; citation_author=J. Gore; citation_author=J. Friedman; citation_author=P.C. Blainey; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_firstpage=12804; citation_lastpage=12809; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Environments that induce synthetic microbial ecosystems; citation_author=N. Klitgord; citation_author=D. Segrè; citation_journal_title=PLoS Comput. Biol.; citation_volume=6; citation_firstpage=e1001002; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Kinetics of denitrification by Pseudomonas denitrificans under growth conditions limited by carbon and/or nitrate or nitrite; citation_author=M. Kornaros; citation_author=C. Zafiri; citation_author=G. Lyberatos; citation_journal_title=Water Environ. Res.; citation_volume=68; citation_firstpage=934; citation_lastpage=945; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=X-ray structure and site-directed mutagenesis of a nitrite reductase from Alcaligenes faecalis S-6: roles of two copper atoms in nitrite reduction; citation_author=M. Kukimoto; citation_author=M. Nishiyama; citation_author=M.E. Murphy; citation_author=S. Turley; citation_author=E.T. Adman; citation_author=S. Horinouchi; citation_author=T. Beppu; citation_journal_title=Biochemistry; citation_volume=33; citation_firstpage=5246; citation_lastpage=5252; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=MEGA X: molecular evolutionary genetics analysis across computing platforms; citation_author=S. Kumar; citation_author=G. Stecher; citation_author=M. Li; citation_author=C. Knyaz; citation_author=K. Tamura; citation_journal_title=Mol. Biol. Evol.; citation_volume=35; citation_firstpage=1547; citation_lastpage=1549; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Systems metabolic engineering of Escherichia coli for L-threonine production; citation_author=K.H. Lee; citation_author=J.H. Park; citation_author=T.Y. Kim; citation_author=H.U. Kim; citation_author=S.Y. Lee; citation_journal_title=Mol. Syst. Biol.; citation_volume=3; citation_number=149; citation_firstpage=149; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Denitrifiers and denitrifying activity in size fractions of a mollisol under permanent pasture and continuous cultivation; citation_author=R. Lensi; citation_author=A. Clays-Josserand; citation_author=L. Jocteur Monrozier; citation_journal_title=Soil Biol. Biochem.; citation_volume=27; citation_firstpage=61; citation_lastpage=69; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Predictive genomic traits for bacterial growth in culture versus actual growth in soil; citation_author=J. Li; citation_author=R.L. Mau; citation_author=P. Dijkstra; citation_author=B.J. Koch; citation_author=E. Schwartz; citation_author=X.A. Liu; citation_author=E.M. Morrissey; citation_author=S.J. Blazewicz; citation_author=J. Pett-Ridge; citation_author=B.W. Stone; citation_author=et al.; citation_journal_title=ISME J.; citation_volume=13; citation_firstpage=2162; citation_lastpage=2172; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Segregating metabolic processes into different microbial cells accelerates the consumption of inhibitory substrates; citation_author=E.E. Lilja; citation_author=D.R. Johnson; citation_journal_title=ISME J.; citation_volume=10; citation_firstpage=1568; citation_lastpage=1578; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Strains in the genus Thauera exhibit remarkably different denitrification regulatory phenotypes; citation_author=B. Liu; citation_author=Y. Mao; citation_author=L. Bergaust; citation_author=L.R. Bakken; citation_author=A. Frostegård; citation_journal_title=Environ. Microbiol.; citation_volume=15; citation_firstpage=2816; citation_lastpage=2828; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Properties and electron transfer specificity of copper proteins from the denitrifier “Achromobacter cycloclastes”; citation_author=M.Y. Liu; citation_author=M.C. Liu; citation_author=W.J. Payne; citation_author=J. Legall; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=166; citation_firstpage=604; citation_lastpage=608; citation_publication_date=1986; "><meta name="citation_reference" content="citation_title=High taxonomic variability despite stable functional structure across microbial communities; citation_author=S. Louca; citation_author=S.M.S. Jacques; citation_author=A.P.F. Pires; citation_author=J.S. Leal; citation_author=D.S. Srivastava; citation_author=L.W. Parfrey; citation_author=V.F. Farjalla; citation_author=M. Doebeli; citation_journal_title=Nat. Ecol. Evol.; citation_volume=1; citation_firstpage=15; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Decoupling function and taxonomy in the global ocean microbiome; citation_author=S. Louca; citation_author=L.W. Parfrey; citation_author=M. Doebeli; citation_journal_title=Science; citation_volume=353; citation_firstpage=1272; citation_lastpage=1277; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Function and functional redundancy in microbial systems; citation_author=S. Louca; citation_author=M.F. Polz; citation_author=F. Mazel; citation_author=M.B.N. Albright; citation_author=J.A. Huber; citation_author=M.I. O’Connor; citation_author=M. Ackermann; citation_author=A.S. Hahn; citation_author=D.S. Srivastava; citation_author=S.A. Crowe; citation_author=et al.; citation_journal_title=Nat. Ecol. Evol.; citation_volume=2; citation_firstpage=936; citation_lastpage=943; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Microbial ecology of denitrification in biological wastewater treatment; citation_author=H. Lu; citation_author=K. Chandran; citation_author=D. Stensel; citation_journal_title=Water Res.; citation_volume=64; citation_firstpage=237; citation_lastpage=254; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Phenotypic and genotypic richness of denitrifiers revealed by a novel isolation strategy; citation_author=P. Lycus; citation_author=K. Lovise Bøthun; citation_author=L. Bergaust; citation_author=J. Peele Shapleigh; citation_author=L. Reier Bakken; citation_author=Å. Frostegård; citation_journal_title=ISME J.; citation_volume=11; citation_firstpage=2219; citation_lastpage=2232; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Streamlining and simplification of microbial genome architecture; citation_author=M. Lynch; citation_journal_title=Annu. Rev. Microbiol.; citation_volume=60; citation_firstpage=327; citation_lastpage=349; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Microbiomes in light of traits: a phylogenetic perspective; citation_author=J.B.H. Martiny; citation_author=S.E. Jones; citation_author=J.T. Lennon; citation_author=A.C. Martiny; citation_journal_title=Science; citation_volume=350; citation_firstpage=aac9323; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Characterization of nitrite reductase from a denitrifier, Alcaligenes sp. NCIB 11015. A novel copper protein; citation_author=M. Masuko; citation_author=H. Iwasaki; citation_author=T. Sakurai; citation_author=S. Suzuki; citation_author=A. Nakahara; citation_journal_title=J.&nbsp;Biochem.; citation_volume=96; citation_firstpage=447; citation_lastpage=454; citation_publication_date=1984; "><meta name="citation_reference" content="citation_title=A lab assembled microcontroller-based sensor module for continuous oxygen measurement in portable hypoxia chambers; citation_author=S.P. Mathupala; citation_author=S. Kiousis; citation_author=N.J. Szerlip; citation_journal_title=PLoS ONE; citation_volume=11; citation_firstpage=e0148923; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Inferring metabolic mechanisms of interaction within a defined gut microbiota; citation_author=G.L. Medlock; citation_author=M.A. Carey; citation_author=D.G. McDuffie; citation_author=M.B. Mundy; citation_author=N. Giallourou; citation_author=J.R. Swann; citation_author=G.L. Kolling; citation_author=J.A. Papin; citation_journal_title=Cell Syst.; citation_volume=7; citation_firstpage=245; citation_lastpage=257.e7; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Molecular characterization of a copper-containing nitrite reductase from Rhodopseudomonas sphaeroides forma sp. denitrificans; citation_author=W.P. Michalski; citation_author=D. Nicholas; citation_journal_title=Biochim. Biophys. Acta Protein Struct. Mol. Enzymol.; citation_volume=828; citation_firstpage=130; citation_lastpage=137; citation_publication_date=1985; "><meta name="citation_reference" content="citation_title=Higher-order interaction between species inhibits bacterial invasion of a phototroph-predator microbial community; citation_author=H. Mickalide; citation_author=S. Kuehn; citation_journal_title=Cell Syst.; citation_volume=9; citation_firstpage=521; citation_lastpage=533.e10; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A rapid, simple spectrophotometric method for simultaneous detection of nitrate and nitrite; citation_author=K.M. Miranda; citation_author=M.G. Espey; citation_author=D.A. Wink; citation_journal_title=Nitric Oxide; citation_volume=5; citation_firstpage=62; citation_lastpage=71; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Nitrate and nitrite transport in bacteria; citation_author=J.W.B. Moir; citation_author=N.J. Wood; citation_journal_title=Cell. Mol. Life Sci.; citation_volume=58; citation_firstpage=215; citation_lastpage=224; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Scaling laws in functional genome content across prokaryotic clades and lifestyles; citation_author=N. Molina; citation_author=E. van. Nimwegen; citation_journal_title=Trends Genet.; citation_volume=25; citation_firstpage=243; citation_lastpage=247; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The growth of bacterial cultures; citation_author=J. Monod; citation_journal_title=Annu. Rev. Microbiol.; citation_volume=3; citation_firstpage=371; citation_lastpage=394; citation_publication_date=1949; "><meta name="citation_reference" content="citation_title=Constrained allocation flux balance analysis; citation_author=M. Mori; citation_author=T. Hwa; citation_author=O.C. Martin; citation_author=A. De Martino; citation_author=E. Marinari; citation_journal_title=PLoS Comput. Biol.; citation_volume=12; citation_firstpage=e1004913; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Phenotypic landscape of a bacterial cell; citation_author=R.J. Nichols; citation_author=S. Sen; citation_author=Y.J. Choo; citation_author=P. Beltrao; citation_author=M. Zietek; citation_author=R. Chaba; citation_author=S. Lee; citation_author=K.M. Kazmierczak; citation_author=K.J. Lee; citation_author=A. Wong; citation_author=et al.; citation_journal_title=Cell; citation_volume=144; citation_firstpage=143; citation_lastpage=156; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Reconfiguration of metabolic fluxes in Pseudomonas putida as a response to sub-lethal oxidative stress; citation_author=P.I. Nikel; citation_author=T. Fuhrer; citation_author=M. Chavarría; citation_author=A. Sánchez-Pascuala; citation_author=U. Sauer; citation_author=V. de Lorenzo; citation_journal_title=ISME J.; citation_volume=15; citation_firstpage=1751; citation_lastpage=1766; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A workflow for generating multi-strain genome-scale metabolic models of prokaryotes; citation_author=C.J. Norsigian; citation_author=X. Fang; citation_author=Y. Seif; citation_author=J.M. Monk; citation_author=B.O. Palsson; citation_journal_title=Nat. Protoc.; citation_volume=15; citation_firstpage=1; citation_lastpage=14; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=What is flux balance analysis?; citation_author=J.D. Orth; citation_author=I. Thiele; citation_author=B.Ø. Palsson; citation_journal_title=Nat. Biotechnol.; citation_volume=28; citation_firstpage=245; citation_lastpage=248; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A spectrophotometric method for determination of bacterial biomass in the presence of a polymer; citation_author=C. Péquignot; citation_author=C. Larroche; citation_author=J.B. Gros; citation_journal_title=Biotechnol. Tech.; citation_volume=12; citation_firstpage=899; citation_lastpage=903; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=16Stimator: statistical estimation of ribosomal gene copy numbers from draft genome assemblies; citation_author=M. Perisin; citation_author=M. Vetter; citation_author=J.A. Gilbert; citation_author=J. Bergelson; citation_journal_title=ISME J.; citation_volume=10; citation_firstpage=1020; citation_lastpage=1024; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Survival kinetics of starving bacteria is biphasic and density-dependent; citation_author=A. Phaiboun; citation_author=Y. Zhang; citation_author=B. Park; citation_author=M. Kim; citation_journal_title=PLoS Comput. Biol.; citation_volume=11; citation_firstpage=e1004198; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Mutant phenotypes for thousands of bacterial genes of unknown function; citation_author=M.N. Price; citation_author=K.M. Wetmore; citation_author=R.J. Waters; citation_author=M. Callaghan; citation_author=J. Ray; citation_author=H. Liu; citation_author=J.V. Kuehl; citation_author=R.A. Melnyk; citation_author=J.S. Lamson; citation_author=Y. Suh; citation_author=et al.; citation_journal_title=Nature; citation_volume=557; citation_firstpage=503; citation_lastpage=509; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes; citation_author=E. Pruesse; citation_author=J. Peplies; citation_author=F.O. Glöckner; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=1823; citation_lastpage=1829; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Dissimilatory metabolism of nitrogen oxides in bacteria: comparative reconstruction of transcriptional networks; citation_author=D.A. Rodionov; citation_author=I.L. Dubchak; citation_author=A.P. Arkin; citation_author=E.J. Alm; citation_author=M.S. Gelfand; citation_journal_title=PLoS Comput. Biol.; citation_volume=1; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Exploiting rRNA operon copy number to investigate bacterial reproductive strategies; citation_author=B.R.K. Roller; citation_author=S.F. Stoddard; citation_author=T.M. Schmidt; citation_journal_title=Nat. Microbiol.; citation_volume=1; citation_firstpage=16160; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The yields of Streptococcus faecalis grown in continuous culture; citation_author=R.F. Rosenberger; citation_author=S.R. Elsden; citation_journal_title=J.&nbsp;Gen. Microbiol.; citation_volume=22; citation_firstpage=726; citation_lastpage=739; citation_publication_date=1960; "><meta name="citation_reference" content="citation_title=Whole genome phylogenies reflect the distributions of recombination rates for many bacterial species; citation_author=T. Sakoparnig; citation_author=C. Field; citation_author=E. van. Nimwegen; citation_journal_title=eLife; citation_volume=10; citation_firstpage=e65366; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=High-order interactions distort the functional landscape of microbial consortia; citation_author=A. Sanchez-Gorostiaga; citation_author=D. Bajić; citation_author=M.L. Osborne; citation_author=J.F. Poyatos; citation_author=A. Sanchez; citation_journal_title=PLoS Biol.; citation_volume=17; citation_firstpage=e3000550; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Purification and properties of a dissimilatory nitrite reductase of a denitrifying phototrophic bacterium; citation_author=E. Sawada; citation_author=S. Toshio; citation_author=H. Kitamura; citation_journal_title=Plant Cell Physiol.; citation_volume=19; citation_firstpage=1339; citation_lastpage=1351; citation_publication_date=1978; "><meta name="citation_reference" content="citation_title=Sulphide-linked nitrite reductase from Thiobacillus denitrificans with cytochrome oxidase activity: purification and properties; citation_author=V. Sawhney; citation_author=D.J.D. Nicholas; citation_journal_title=J.&nbsp;Gen. Microbiol.; citation_volume=106; citation_firstpage=119; citation_lastpage=128; citation_publication_date=1978; "><meta name="citation_reference" content="citation_title=A two-enzyme adaptive unit within bacterial folate metabolism; citation_author=A.F. Schober; citation_author=A.D. Mathis; citation_author=C. Ingle; citation_author=J.O. Park; citation_author=L. Chen; citation_author=J.D. Rabinowitz; citation_author=I. Junier; citation_author=O. Rivoire; citation_author=K.A. Reynolds; citation_journal_title=Cell Rep.; citation_volume=27; citation_firstpage=3359; citation_lastpage=3370.e7; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Interdependence of cell growth and gene expression: origins and consequences; citation_author=M. Scott; citation_author=C.W. Gunderson; citation_author=E.M. Mateescu; citation_author=Z. Zhang; citation_author=T. Hwa; citation_journal_title=Science; citation_volume=330; citation_firstpage=1099; citation_lastpage=1102; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Denitrification across landscapes and waterscapes: a synthesis; citation_author=S. Seitzinger; citation_author=J.A. Harrison; citation_author=J.K. Böhlke; citation_author=A.F. Bouwman; citation_author=R. Lowrance; citation_author=B. Peterson; citation_author=C. Tobias; citation_author=G. Van Drecht; citation_journal_title=Ecol. Appl.; citation_volume=16; citation_firstpage=2064; citation_lastpage=2090; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Selection and genome plasticity as the key factors in the evolution of bacteria; citation_author=I. Sela; citation_author=Y.I. Wolf; citation_author=E.V. Koonin; citation_journal_title=Phys. Rev. X; citation_volume=9; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Synthetic cooperation in engineered yeast populations; citation_author=W. Shou; citation_author=S. Ram; citation_author=J.M.G. Vilar; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=104; citation_firstpage=1877; citation_lastpage=1882; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Complete genome sequence of Paracoccus denitrificans ATCC 19367 and its denitrification characteristics; citation_author=Y.-Y. Si; citation_author=K.-H. Xu; citation_author=X.-Y. Yu; citation_author=M.-F. Wang; citation_author=X.-H. Chen; citation_journal_title=Can. J. Microbiol.; citation_volume=65; citation_firstpage=486; citation_lastpage=495; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Recovery of genomes from metagenomes via a dereplication, aggregation and scoring strategy; citation_author=C.M.K. Sieber; citation_author=A.J. Probst; citation_author=A. Sharrar; citation_author=B.C. Thomas; citation_author=M. Hess; citation_author=S.G. Tringe; citation_author=J.F. Banfield; citation_journal_title=Nat. Microbiol.; citation_volume=3; citation_firstpage=836; citation_lastpage=843; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The nitrogen cycle; citation_author=L.Y. Stein; citation_author=M.G. Klotz; citation_journal_title=Curr. Biol.; citation_volume=26; citation_firstpage=R94; citation_lastpage=R98; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Periplasmic nitrate reductase (NapABC enzyme) supports anaerobic respiration by Escherichia coli K-12; citation_author=V. Stewart; citation_author=Y. Lu; citation_author=A.J. Darwin; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=184; citation_firstpage=1314; citation_lastpage=1323; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Growth yields in bacterial denitrification and nitrate ammonification; citation_author=T.O. Strohm; citation_author=B. Griffin; citation_author=W.G. Zumft; citation_author=B. Schink; citation_journal_title=Appl. Environ. Microbiol.; citation_volume=73; citation_firstpage=1420; citation_lastpage=1424; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Persistent gut microbiota immaturity in malnourished Bangladeshi children; citation_author=S. Subramanian; citation_author=S. Huq; citation_author=T. Yatsunenko; citation_author=R. Haque; citation_author=M. Mahfuz; citation_author=M.A. Alam; citation_author=A. Benezra; citation_author=J. DeStefano; citation_author=M.F. Meier; citation_author=B.D. Muegge; citation_author=et al.; citation_journal_title=Nature; citation_volume=510; citation_firstpage=417; citation_lastpage=421; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Statistical learning and selective inference; citation_author=J. Taylor; citation_author=R.J. Tibshirani; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=7629; citation_lastpage=7634; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Denitrification: ecological niches, competition and survival; citation_author=J.M. Tiedje; citation_author=A.J. Sexstone; citation_author=D.D. Myrold; citation_author=J.A. Robinson; citation_journal_title=Antonie van Leeuwenhoek; citation_volume=48; citation_firstpage=569; citation_lastpage=583; citation_publication_date=1982; "><meta name="citation_reference" content="citation_title=Isolation of Paracoccus denitrificans cytochrome cd1: comparative kinetics with other nitrite reductases; citation_author=R. Timkovich; citation_author=R. Dhesi; citation_author=K.J. Martinkus; citation_author=M.K. Robinson; citation_author=T.M. Rea; citation_journal_title=Arch. Biochem. Biophys.; citation_volume=215; citation_firstpage=47; citation_lastpage=58; citation_publication_date=1982; "><meta name="citation_reference" content="citation_title=Determining roles of accessory genes in denitrification by mutant fitness analyses; citation_author=B.J. Vaccaro; citation_author=M.P. Thorgersen; citation_author=W.A. Lancaster; citation_author=M.N. Price; citation_author=K.M. Wetmore; citation_author=F.L. Poole; citation_author=A. Deutschbauer; citation_author=A.P. Arkin; citation_author=M.W. Adams; citation_journal_title=Appl. Environ. Microbiol.; citation_volume=82; citation_firstpage=51; citation_lastpage=61; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Compensatory periplasmic nitrate reductase activity supports anaerobic growth of Pseudomonas aeruginosa PAO1 in the absence of membrane nitrate reductase; citation_author=N.E. Van Alst; citation_author=L.A. Sherrill; citation_author=B.H. Iglewski; citation_author=C.G. Haidaris; citation_journal_title=Can. J. Microbiol.; citation_volume=55; citation_firstpage=1133; citation_lastpage=1144; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Denitrification in Gram-positive bacteria: an underexplored trait; citation_author=I. Verbaendert; citation_author=P. De Vos; citation_author=N. Boon; citation_author=K. Heylen; citation_journal_title=Biochem. Soc. Trans.; citation_volume=39; citation_firstpage=254; citation_lastpage=258; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Cyanobacterial ecotypes in the microbial mat community of Mushroom Spring (Yellowstone National Park, Wyoming) as species-like units linking microbial community composition, structure and function; citation_author=D.M. Ward; citation_author=M.M. Bateson; citation_author=M.J. Ferris; citation_author=M. Kühl; citation_author=A. Wieland; citation_author=A. Koeppel; citation_author=F.M. Cohan; citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_volume=361; citation_firstpage=1997; citation_lastpage=2008; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Three nitrate reductase activities in Alcaligenes eutrophus; citation_author=U. Warnecke-Eberz; citation_author=B. Friedrich; citation_journal_title=Arch. Microbiol.; citation_volume=159; citation_firstpage=405; citation_lastpage=409; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=The Physiology and Biochemistry of Prokaryotes; citation_author=D. White; citation_author=J. Drummond; citation_author=C. Fuqua; citation_publication_date=2012; citation_publisher=Oxford University Press; "><meta name="citation_reference" content="citation_title=Challenges in microbial ecology: building predictive understanding of community function and dynamics; citation_author=S. Widder; citation_author=R.J. Allen; citation_author=T. Pfeiffer; citation_author=T.P. Curtis; citation_author=C. Wiuf; citation_author=W.T. Sloan; citation_author=O.X. Cordero; citation_author=S.P. Brown; citation_author=B. Momeni; citation_author=W. Shou; citation_author=et al.; citation_journal_title=ISME J.; citation_volume=10; citation_firstpage=2557; citation_lastpage=2568; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Cell biology and molecular basis of denitrification; citation_author=W.G. Zumft; citation_journal_title=Microbiol. Mol. Biol. Rev.; citation_volume=61; citation_firstpage=533; citation_lastpage=616; citation_publication_date=1997; "><meta name="citation_abstract" content="<h2>Summary</h2><p>The metabolic activities of microbial communities play a defining role in the evolution and persistence of life on Earth, driving redox reactions that give rise to global biogeochemical cycles. Community metabolism emerges from a hierarchy of processes, including gene expression, ecological interactions, and environmental factors. In wild communities, gene content is correlated with environmental context, but predicting metabolite dynamics from genomes remains elusive. Here, we show, for the process of denitrification, that metabolite dynamics of a community are predictable from the genes each member of the community possesses. A simple linear regression reveals a sparse and generalizable mapping from gene content to metabolite dynamics for genomically diverse bacteria. A consumer-resource model correctly predicts community metabolite dynamics from single-strain phenotypes. Our results demonstrate that the conserved impacts of metabolic genes can predict community metabolite dynamics, enabling the prediction of metabolite dynamics from metagenomes, designing denitrifying communities, and discovering how genome evolution impacts metabolism.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7777&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)01542-7&amp;be=3575&amp;fe=7419&amp;dc=6816&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749592564885,%22n%22:0,%22f%22:2944,%22dn%22:2945,%22dne%22:2945,%22c%22:2945,%22s%22:2945,%22ce%22:2945,%22rq%22:2946,%22rp%22:3558,%22rpe%22:4515,%22dl%22:3564,%22di%22:5082,%22ds%22:6816,%22de%22:7140,%22dc%22:7420,%22l%22:7420,%22le%22:7560%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><script src="https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/dynamic/script.js" charset="UTF-8"></script><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-mover {
  display: inline-block;
  text-align: left;
}

mjx-mover:not([limits="false"]) {
  padding-top: .1em;
}

mjx-mover:not([limits="false"]) > * {
  display: block;
  text-align: left;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-mtable {
  display: inline-block;
  text-align: center;
  vertical-align: .25em;
  position: relative;
  box-sizing: border-box;
  border-spacing: 0;
  border-collapse: collapse;
}

mjx-mstyle[size="s"] mjx-mtable {
  vertical-align: .354em;
}

mjx-labels {
  position: absolute;
  left: 0;
  top: 0;
}

mjx-table {
  display: inline-block;
  vertical-align: -.5ex;
  box-sizing: border-box;
}

mjx-table > mjx-itable {
  vertical-align: middle;
  text-align: left;
  box-sizing: border-box;
}

mjx-labels > mjx-itable {
  position: absolute;
  top: 0;
}

mjx-mtable[justify="left"] {
  text-align: left;
}

mjx-mtable[justify="right"] {
  text-align: right;
}

mjx-mtable[justify="left"][side="left"] {
  padding-right: 0 ! important;
}

mjx-mtable[justify="left"][side="right"] {
  padding-left: 0 ! important;
}

mjx-mtable[justify="right"][side="left"] {
  padding-right: 0 ! important;
}

mjx-mtable[justify="right"][side="right"] {
  padding-left: 0 ! important;
}

mjx-mtable[align] {
  vertical-align: baseline;
}

mjx-mtable[align="top"] > mjx-table {
  vertical-align: top;
}

mjx-mtable[align="bottom"] > mjx-table {
  vertical-align: bottom;
}

mjx-mtable[side="right"] mjx-labels {
  min-width: 100%;
}

mjx-mtr {
  display: table-row;
  text-align: left;
}

mjx-mtr[rowalign="top"] > mjx-mtd {
  vertical-align: top;
}

mjx-mtr[rowalign="center"] > mjx-mtd {
  vertical-align: middle;
}

mjx-mtr[rowalign="bottom"] > mjx-mtd {
  vertical-align: bottom;
}

mjx-mtr[rowalign="baseline"] > mjx-mtd {
  vertical-align: baseline;
}

mjx-mtr[rowalign="axis"] > mjx-mtd {
  vertical-align: .25em;
}

mjx-mtd {
  display: table-cell;
  text-align: center;
  padding: .215em .4em;
}

mjx-mtd:first-child {
  padding-left: 0;
}

mjx-mtd:last-child {
  padding-right: 0;
}

mjx-mtable > * > mjx-itable > *:first-child > mjx-mtd {
  padding-top: 0;
}

mjx-mtable > * > mjx-itable > *:last-child > mjx-mtd {
  padding-bottom: 0;
}

mjx-tstrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-labels[align="left"] > mjx-mtr > mjx-mtd {
  text-align: left;
}

mjx-labels[align="right"] > mjx-mtr > mjx-mtd {
  text-align: right;
}

mjx-mtd[extra] {
  padding: 0;
}

mjx-mtd[rowalign="top"] {
  vertical-align: top;
}

mjx-mtd[rowalign="center"] {
  vertical-align: middle;
}

mjx-mtd[rowalign="bottom"] {
  vertical-align: bottom;
}

mjx-mtd[rowalign="baseline"] {
  vertical-align: baseline;
}

mjx-mtd[rowalign="axis"] {
  vertical-align: .25em;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-munderover {
  display: inline-block;
  text-align: left;
}

mjx-munderover:not([limits="false"]) {
  padding-top: .1em;
}

mjx-munderover:not([limits="false"]) > * {
  display: block;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

mjx-munder {
  display: inline-block;
  text-align: left;
}

mjx-over {
  text-align: left;
}

mjx-munder:not([limits="false"]) {
  display: inline-table;
}

mjx-munder > mjx-row {
  text-align: left;
}

mjx-under {
  padding-bottom: .1em;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-stretchy-h.mjx-c305 mjx-ext mjx-c {
  padding: 0.67em 0 0 0;
  width: 0.19em;
}

mjx-c.mjx-c4E {
  padding: 0.683em 0.75em 0 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c4F {
  padding: 0.705em 0.778em 0.022em 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c33 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c2192 {
  padding: 0.51em 1em 0.01em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c28.MM-SO {
  padding: 0.797em 0.422em 0.297em 0;
}

mjx-c.mjx-c29.MM-SO {
  padding: 0.797em 0.422em 0.297em 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c1D434 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c1D43C {
  padding: 0.683em 0.497em 0 0;
}

mjx-c.mjx-c1D6FE {
  padding: 0.442em 0.543em 0.215em 0;
}

mjx-c.mjx-c1D43E {
  padding: 0.683em 0.889em 0 0;
}

mjx-c.mjx-c2217 {
  padding: 0.462em 0.5em 0 0;
}

mjx-c.mjx-c2C {
  padding: 0.106em 0.278em 0.193em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c394 {
  padding: 0.716em 0.833em 0 0;
}

mjx-c.mjx-c36 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c34 {
  padding: 0.677em 0.5em 0 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c2E {
  padding: 0.106em 0.278em 0 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c1D441 {
  padding: 0.683em 0.881em 0 0;
}

mjx-c.mjx-c37 {
  padding: 0.676em 0.5em 0.022em 0;
}

mjx-c.mjx-c39 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c1D45D {
  padding: 0.442em 0.503em 0.194em 0;
}

mjx-c.mjx-c38 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c35 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c1D43F {
  padding: 0.683em 0.681em 0 0;
}

mjx-c.mjx-c1D6FD {
  padding: 0.706em 0.574em 0.194em 0;
}

mjx-c.mjx-c1D457 {
  padding: 0.661em 0.412em 0.205em 0;
}

mjx-c.mjx-c1D700 {
  padding: 0.453em 0.466em 0.022em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c28.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}

mjx-c.mjx-c1D445 {
  padding: 0.683em 0.759em 0.022em 0;
}

mjx-c.mjx-c1D453 {
  padding: 0.705em 0.552em 0.205em 0;
}

mjx-c.mjx-c29.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}

mjx-c.mjx-c1D436 {
  padding: 0.705em 0.76em 0.022em 0;
}

mjx-c.mjx-c1D449 {
  padding: 0.683em 0.769em 0.022em 0;
}

mjx-c.mjx-c2264 {
  padding: 0.631em 0.778em 0.119em 0;
}

mjx-c.mjx-cD7 {
  padding: 0.491em 0.778em 0 0;
}

mjx-c.mjx-c7C {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c1D70C {
  padding: 0.442em 0.517em 0.216em 0;
}

mjx-c.mjx-c2265 {
  padding: 0.631em 0.778em 0.119em 0;
}

mjx-c.mjx-c1D443 {
  padding: 0.683em 0.754em 0 0;
}

mjx-c.mjx-c6D {
  padding: 0.442em 0.833em 0 0;
}

mjx-c.mjx-c61 {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c78 {
  padding: 0.431em 0.528em 0 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c3C {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c1D437 {
  padding: 0.683em 0.828em 0 0;
}

mjx-c.mjx-c1D440 {
  padding: 0.683em 1.044em 0 0;
}

mjx-c.mjx-c1D446 {
  padding: 0.705em 0.645em 0.022em 0;
}

mjx-c.mjx-c48 {
  padding: 0.683em 0.75em 0 0;
}

mjx-c.mjx-c52 {
  padding: 0.683em 0.736em 0.022em 0;
}

mjx-c.mjx-c4D {
  padding: 0.683em 0.917em 0 0;
}

mjx-c.mjx-c53 {
  padding: 0.705em 0.556em 0.022em 0;
}

mjx-c.mjx-c45 {
  padding: 0.68em 0.681em 0 0;
}

mjx-c.mjx-c2F {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c28.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c29.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c1D465 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c2248 {
  padding: 0.457em 0.773em 0 0;
}

mjx-c.mjx-c303 {
  padding: 0.746em 0 0 0;
  width: 0;
}

mjx-c.mjx-c223C {
  padding: 0.366em 0.773em 0 0;
}

mjx-c.mjx-c226A {
  padding: 0.547em 1em 0.047em 0;
}

mjx-c.mjx-c1D44F {
  padding: 0.694em 0.429em 0.011em 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c1D43A {
  padding: 0.705em 0.786em 0.022em 0;
}

mjx-c.mjx-c1D45A {
  padding: 0.442em 0.878em 0.011em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-cB1 {
  padding: 0.583em 0.778em 0.084em 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c1D447 {
  padding: 0.677em 0.704em 0 0;
}

mjx-c.mjx-c222B {
  padding: 0.805em 0.665em 0.306em 0;
}

mjx-c.mjx-c28.MM-S7 {
  padding: 1.745em 0.875em 1.245em 0;
}

mjx-c.mjx-c29.MM-S7 {
  padding: 1.745em 0.875em 1.245em 0;
}

mjx-c.mjx-c44 {
  padding: 0.683em 0.764em 0 0;
}

mjx-c.mjx-c226B {
  padding: 0.547em 1em 0.047em 0;
}

mjx-c.mjx-c7B {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c7D {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c1D466 {
  padding: 0.442em 0.49em 0.205em 0;
}

mjx-c.mjx-c2211 {
  padding: 0.75em 1.056em 0.25em 0;
}

mjx-c.mjx-c1D454 {
  padding: 0.442em 0.477em 0.205em 0;
}

mjx-c.mjx-c66 {
  padding: 0.705em 0.357em 0 0;
}

mjx-c.mjx-c6F {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c72 {
  padding: 0.442em 0.392em 0 0;
}

mjx-c.mjx-cA0 {
  padding: 0 0.332em 0 0;
}

mjx-c.mjx-c2026 {
  padding: 0.106em 0.837em 0 0;
}

mjx-c.mjx-c69 {
  padding: 0.657em 0.278em 0 0;
}

mjx-c.mjx-c6E {
  padding: 0.442em 0.556em 0 0;
}

mjx-c.mjx-c7B.MM-S7 {
  padding: 1.75em 0.902em 1.25em 0;
}

mjx-c.mjx-c28.MM-S6 {
  padding: 1.446em 0.736em 0.946em 0;
}

mjx-c.mjx-c29.MM-S6 {
  padding: 1.446em 0.736em 0.946em 0;
}

mjx-c.mjx-c1D706 {
  padding: 0.694em 0.583em 0.013em 0;
}

mjx-c.mjx-c2223.MM-S5 {
  padding: 1.501em 0.278em 1.001em 0;
}

mjx-c.mjx-c7D.MM-S7 {
  padding: 1.75em 0.902em 1.25em 0;
}

mjx-c.mjx-c302.MM-SO {
  padding: 0.746em 0 0 0;
  width: 0.644em;
}

mjx-c.mjx-c1D6EC {
  padding: 0.716em 0.694em 0 0;
}

mjx-c.mjx-c1D458 {
  padding: 0.694em 0.521em 0.011em 0;
}

mjx-c.mjx-c2208 {
  padding: 0.543em 0.667em 0.043em 0;
}

mjx-c.mjx-c2260 {
  padding: 0.73em 0.778em 0.23em 0;
}

mjx-c.mjx-c1D43B {
  padding: 0.683em 0.881em 0 0;
}

mjx-c.mjx-c5B {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c5D {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c3E {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c1D707 {
  padding: 0.442em 0.603em 0.216em 0;
}

mjx-c.mjx-c7B.MM-SO {
  padding: 0.8em 0.54em 0.3em 0;
}

mjx-c.mjx-c7D.MM-SO {
  padding: 0.8em 0.54em 0.3em 0;
}

mjx-c.mjx-c1D450 {
  padding: 0.442em 0.433em 0.011em 0;
}

mjx-c.mjx-c1D4AE.MM-S {
  padding: 0.741em 1.168em 0.015em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901542-7" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01542-7"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)01542-7"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901542-7" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-70596" aria-expanded="false" aria-controls="login-links-70596" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-70596" id="login-links--70596" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901542-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901542-7" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901542-7" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-70596" data-opens="#links-dialog-70596" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-70596" aria-modal="true"><button data-closes="#links-dialog-70596" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901542-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901542-7" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901542-7" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.12.036">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)01542-7" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0004-9"><span>Volume 185</span>, <span>Issue 3</span></a></span><span class="meta-panel__pages"><span>p530-546.e25</span></span><span class="meta-panel__onlineDate">February 03, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=3&amp;pii=S0092-8674%2821%29X0004-9" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Genomic structure predicts metabolite dynamics in microbial communities</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Karna</span> <span property="familyName">Gowda</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Karna</span> <span property="familyName">Gowda</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Ecology and Evolution, University of Chicago, Chicago, IL 60637, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for the Physics of Evolving Systems, University of Chicago, Chicago, IL 60637, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gowda/Karna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Derek</span> <span property="familyName">Ping</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Derek</span> <span property="familyName">Ping</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Physics, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ping/Derek" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Madhav</span> <span property="familyName">Mani</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Madhav</span> <span property="familyName">Mani</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:madhav.mani@gmail.com" property="email" title="Send email to Madhav Mani">madhav.mani@gmail.com</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Engineering Sciences and Applied Mathematics, Northwestern University, Evanston, IL 60208, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Biosciences, Northwestern University, Evanston, IL 60208, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">NSF-Simons Center for Quantitative Biology, Northwestern University, Northwestern University, Evanston, IL 60208, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Mani/Madhav" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup> <a id="eadd0010" href="mailto:madhav.mani@gmail.com" property="email" title="Send email to Madhav Mani">madhav.mani@gmail.com</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Seppe</span> <span property="familyName">Kuehn</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Seppe</span> <span property="familyName">Kuehn</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:seppe.kuehn@gmail.com" property="email" title="Send email to Seppe Kuehn">seppe.kuehn@gmail.com</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Ecology and Evolution, University of Chicago, Chicago, IL 60637, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for the Physics of Evolving Systems, University of Chicago, Chicago, IL 60637, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kuehn/Seppe" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">7</a></sup> <a id="eadd0015" href="mailto:seppe.kuehn@gmail.com" property="email" title="Send email to Seppe Kuehn">seppe.kuehn@gmail.com</a></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Ecology and Evolution, University of Chicago, Chicago, IL 60637, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Center for the Physics of Evolving Systems, University of Chicago, Chicago, IL 60637, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Physics, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Engineering Sciences and Applied Mathematics, Northwestern University, Evanston, IL 60208, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Molecular Biosciences, Northwestern University, Evanston, IL 60208, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">NSF-Simons Center for Quantitative Biology, Northwestern University, Northwestern University, Evanston, IL 60208, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received January 22, 2021</span>; <span>Revised July 16, 2021</span>; <span>Accepted December 21, 2021</span>; <span>Published online January 26, 2022</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.12.036" property="sameAs" target="_blank">10.1016/j.cell.2021.12.036 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421015427" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.12.036" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0004-9"><img src="/cms/asset/32fff431-7987-4458-b89c-6b5f91c0139a/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 3"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2901542-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">Conclusion</a></li><li><a href="#sec-5">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (1)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901542-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.12.036">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2901542-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.12.036&amp;pii=S0092-8674%2821%2901542-7" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901542-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.12.036">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.12.036" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421015427&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421015427&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=3&amp;pii=S0092-8674%2821%29X0004-9" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(22)00002-2" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)01541-5" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">Conclusion</a></li><li><a href="#sec-5">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (1)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Metabolite fluxes in microbial communities are predictable from individual genotypes</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">A diverse collection of 79 bacterial isolates was sequenced and phenotyped</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Gene presence and absence predict metabolic phenotypes of isolates via regression</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">A consumer-resource model predicts community metabolite fluxes from phenotypes</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">The metabolic activities of microbial communities play a defining role in the evolution and persistence of life on Earth, driving redox reactions that give rise to global biogeochemical cycles. Community metabolism emerges from a hierarchy of processes, including gene expression, ecological interactions, and environmental factors. In wild communities, gene content is correlated with environmental context, but predicting metabolite dynamics from genomes remains elusive. Here, we show, for the process of denitrification, that metabolite dynamics of a community are predictable from the genes each member of the community possesses. A simple linear regression reveals a sparse and generalizable mapping from gene content to metabolite dynamics for genomically diverse bacteria. A consumer-resource model correctly predicts community metabolite dynamics from single-strain phenotypes. Our results demonstrate that the conserved impacts of metabolic genes can predict community metabolite dynamics, enabling the prediction of metabolite dynamics from metagenomes, designing denitrifying communities, and discovering how genome evolution impacts metabolism.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/5cd2d5e9-8ac5-43b1-a57b-bfd845e83cae/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.12.036/asset/d5296a79-1be1-4045-bded-e10f51ff5a60/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/5cd2d5e9-8ac5-43b1-a57b-bfd845e83cae/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;microbial+ecology&quot;" alt="Search on this keyword" rel="nofollow">microbial ecology</a></li><li><a href="/action/doSearch?AllField=&quot;microbial+communities&quot;" alt="Search on this keyword" rel="nofollow">microbial communities</a></li><li><a href="/action/doSearch?AllField=&quot;biological+physics&quot;" alt="Search on this keyword" rel="nofollow">biological physics</a></li><li><a href="/action/doSearch?AllField=&quot;statistical+genomics&quot;" alt="Search on this keyword" rel="nofollow">statistical genomics</a></li><li><a href="/action/doSearch?AllField=&quot;microbial+community+metabolism&quot;" alt="Search on this keyword" rel="nofollow">microbial community metabolism</a></li><li><a href="/action/doSearch?AllField=&quot;denitrification&quot;" alt="Search on this keyword" rel="nofollow">denitrification</a></li><li><a href="/action/doSearch?AllField=&quot;population+dynamics&quot;" alt="Search on this keyword" rel="nofollow">population dynamics</a></li><li><a href="/action/doSearch?AllField=&quot;microbial+interactions&quot;" alt="Search on this keyword" rel="nofollow">microbial interactions</a></li><li><a href="/action/doSearch?AllField=&quot;community+assembly&quot;" alt="Search on this keyword" rel="nofollow">community assembly</a></li><li><a href="/action/doSearch?AllField=&quot;theoretical+ecology&quot;" alt="Search on this keyword" rel="nofollow">theoretical ecology</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0050" role="paragraph">The metabolism of microbial communities plays an essential role in sustaining life on Earth, impacting global nutrient cycles (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Falkowski et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Falkowski, P.G. ∙ Fenchel, T. ∙ Delong, E.F.</div><div><strong>The microbial engines that drive earth's biogeochemical cycles</strong></div><div><em>Science.</em> 2008; <strong>320</strong>:1034-1039</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1153213" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-44449137093" target="_blank" aria-label="Open Scopus in new tab">Scopus (1460)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18497287/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1153213&amp;pmid=18497287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Canfield et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Canfield, D.E. ∙ Glazer, A.N. ∙ Falkowski, P.G.</div><div><strong>The evolution and future of earth's nitrogen cycle</strong></div><div><em>Science.</em> 2010; <strong>330</strong>:192-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1186120" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-77957744758" target="_blank" aria-label="Open Scopus in new tab">Scopus (1276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20929768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1186120&amp;pmid=20929768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Stein and Klotz, 2016</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Stein, L.Y. ∙ Klotz, M.G.</div><div><strong>The nitrogen cycle</strong></div><div><em>Curr. Biol.</em> 2016; <strong>26</strong>:R94-R98</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cub.2015.12.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cub.2015.12.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84958755229" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26859274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2015.12.021&amp;pmid=26859274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), wastewater treatment (<span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Lu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Lu, H. ∙ Chandran, K. ∙ Stensel, D.</div><div><strong>Microbial ecology of denitrification in biological wastewater treatment</strong></div><div><em>Water Res.</em> 2014; <strong>64</strong>:237-254</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.watres.2014.06.042" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84904891891" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25078442/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.watres.2014.06.042&amp;pmid=25078442" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and human health (<span class="dropBlock reference-citations"><a id="body-ref-sref106" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106" aria-controls="sref106">Subramanian et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref106"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Subramanian, S. ∙ Huq, S. ∙ Yatsunenko, T. ...</div><div><strong>Persistent gut microbiota immaturity in malnourished Bangladeshi children</strong></div><div><em>Nature.</em> 2014; <strong>510</strong>:417-421</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13421" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84903134830" target="_blank" aria-label="Open Scopus in new tab">Scopus (625)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24896187/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13421&amp;pmid=24896187" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A challenge in microbial ecology is understanding how community metabolism is determined by the taxa present, their metabolic traits, and the genes they possess (<span class="dropBlock reference-citations"><a id="body-ref-sref116" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116" aria-controls="sref116">Widder et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref116"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Widder, S. ∙ Allen, R.J. ∙ Pfeiffer, T. ...</div><div><strong>Challenges in microbial ecology: building predictive understanding of community function and dynamics</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:2557-2568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2016.45" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84961904331" target="_blank" aria-label="Open Scopus in new tab">Scopus (321)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27022995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2016.45&amp;pmid=27022995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Louca et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Louca, S. ∙ Polz, M.F. ∙ Mazel, F. ...</div><div><strong>Function and functional redundancy in microbial systems</strong></div><div><em>Nat. Ecol. Evol.</em> 2018; <strong>2</strong>:936-943</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-018-0519-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85045435762" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29662222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-018-0519-1&amp;pmid=29662222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Addressing this challenge requires mapping the genotypes of each community member to its metabolic traits and then deciphering how complex interactions between each member impact the flux of metabolites through the community. Complicating the prediction of metabolite fluxes from community composition, interactions can depend on extracellular metabolites (<span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Lilja and Johnson, 2016</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Lilja, E.E. ∙ Johnson, D.R.</div><div><strong>Segregating metabolic processes into different microbial cells accelerates the consumption of inhibitory substrates</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:1568-1578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2015.243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84954437600" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2015.243&amp;pmid=26771930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), abiotic factors (<span class="dropBlock reference-citations"><a id="body-ref-sref113" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113" aria-controls="sref113">Ward et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref113"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Ward, D.M. ∙ Bateson, M.M. ∙ Ferris, M.J. ...</div><div><strong>Cyanobacterial ecotypes in the microbial mat community of Mushroom Spring (Yellowstone National Park, Wyoming) as species-like units linking microbial community composition, structure and function</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2006; <strong>361</strong>:1997-2008</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2006.1919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33751210106" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17028085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2006.1919&amp;pmid=17028085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), cooperation (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Cordero et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cordero, O.X. ∙ Ventouras, L.A. ∙ DeLong, E.F. ...</div><div><strong>Public good dynamics drive evolution of iron acquisition strategies in natural bacterioplankton populations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:20059-20064</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1213344109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84870578736" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23169633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1213344109&amp;pmid=23169633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and higher-order effects (<span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">Sanchez-Gorostiaga et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Sanchez-Gorostiaga, A. ∙ Bajić, D. ∙ Osborne, M.L. ...</div><div><strong>High-order interactions distort the functional landscape of microbial consortia</strong></div><div><em>PLoS Biol.</em> 2019; <strong>17</strong>:e3000550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3000550" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85077298064" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31830028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3000550&amp;pmid=31830028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Mickalide and Kuehn, 2019</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Mickalide, H. ∙ Kuehn, S.</div><div><strong>Higher-order interaction between species inhibits bacterial invasion of a phototroph-predator microbial community</strong></div><div><em>Cell Syst.</em> 2019; <strong>9</strong>:521-533.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2019.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2019.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85076261763" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31838145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2019.11.004&amp;pmid=31838145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Despite these challenges, connecting genomic structure to the collective metabolism of a community is important for functionally interpreting community gene content (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Anantharaman et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Anantharaman, K. ∙ Brown, C.T. ∙ Hug, L.A. ...</div><div><strong>Thousands of microbial genomes shed light on interconnected biogeochemical processes in an aquifer system</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:13219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms13219" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84992418589" target="_blank" aria-label="Open Scopus in new tab">Scopus (451)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27774985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms13219&amp;pmid=27774985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), designing synthetic communities (<span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Shou et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Shou, W. ∙ Ram, S. ∙ Vilar, J.M.G.</div><div><strong>Synthetic cooperation in engineered yeast populations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:1877-1882</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0610575104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33846936445" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17267602/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0610575104&amp;pmid=17267602" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and understanding how gene gain and loss (<span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Molina and Nimwegen, 2009</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Molina, N. ∙ Nimwegen, E. van.</div><div><strong>Scaling laws in functional genome content across prokaryotic clades and lifestyles</strong></div><div><em>Trends Genet.</em> 2009; <strong>25</strong>:243-247</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.tig.2009.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.tig.2009.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-66449114096" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19457568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tig.2009.04.004&amp;pmid=19457568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Sela et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Sela, I. ∙ Wolf, Y.I. ∙ Koonin, E.V.</div><div><strong>Selection and genome plasticity as the key factors in the evolution of bacteria</strong></div><div><em>Phys. Rev. X.</em> 2019; <strong>9</strong>, 031018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.SelaY.I.WolfE.V.KooninSelection+and+genome+plasticity+as+the+key+factors+in+the+evolution+of+bacteriaPhys.+Rev.+X92019031018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) impact community metabolism.</div><div id="p0055" role="paragraph">Recent work suggests that the genes present in a community may be more informative about metabolic activity than the identity of strains or species making up the community. Sequencing studies of environmental and host-associated communities show that, while the individual strains or species present are often highly variable (<span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Louca et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Louca, S. ∙ Polz, M.F. ∙ Mazel, F. ...</div><div><strong>Function and functional redundancy in microbial systems</strong></div><div><em>Nat. Ecol. Evol.</em> 2018; <strong>2</strong>:936-943</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-018-0519-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85045435762" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29662222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-018-0519-1&amp;pmid=29662222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the genes or pathways present are often observed to be stable across communities in similar environments (<span class="dropBlock reference-citations"><a id="body-ref-sref65-3" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-3" aria-controls="sref65-3">Louca et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref65-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Louca, S. ∙ Polz, M.F. ∙ Mazel, F. ...</div><div><strong>Function and functional redundancy in microbial systems</strong></div><div><em>Nat. Ecol. Evol.</em> 2018; <strong>2</strong>:936-943</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-018-0519-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85045435762" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29662222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-018-0519-1&amp;pmid=29662222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Human Microbiome Project Consortium, 2012</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Human Microbiome Project Consortium</div><div><strong>Structure, function and diversity of the healthy human microbiome</strong></div><div><em>Nature.</em> 2012; <strong>486</strong>:207-214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11234" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84862276328" target="_blank" aria-label="Open Scopus in new tab">Scopus (6048)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22699609/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11234&amp;pmid=22699609" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example, aquatic communities native to bromeliads contain prokaryotes from several functional groups (e.g., methanogens, fermenters, and photoautotrophs). The strain or species representing each functional group varies widely from one plant to the next, but the relative abundance of each functional group is remarkably stable across plants (<span class="dropBlock reference-citations"><a id="body-ref-sref63-1" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-1" aria-controls="sref63-1">Louca et&nbsp;al., 2016a</a><div class="dropBlock__holder" data-db-target-of="sref63-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Louca, S. ∙ Jacques, S.M.S. ∙ Pires, A.P.F. ...</div><div><strong>High taxonomic variability despite stable functional structure across microbial communities</strong></div><div><em>Nat. Ecol. Evol.</em> 2016; <strong>1</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-016-0015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28812567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-016-0015&amp;pmid=28812567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similarly, studies in oceans and soils that measure both gene content and nutrient levels have found that the relative abundances of specific metabolic genes are better predictors of nutrient levels than the abundances of specific taxa (<span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Jones and Hallin, 2010</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Jones, C.M. ∙ Hallin, S.</div><div><strong>Ecological and evolutionary factors underlying global and local assembly of denitrifier communities</strong></div><div><em>ISME J.</em> 2010; <strong>4</strong>:633-641</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2009.152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-77951498193" target="_blank" aria-label="Open Scopus in new tab">Scopus (174)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20090785/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2009.152&amp;pmid=20090785" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Fierer et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Fierer, N. ∙ Lauber, C.L. ∙ Ramirez, K.S. ...</div><div><strong>Comparative metagenomic, phylogenetic and physiological analyses of soil microbial communities across nitrogen gradients</strong></div><div><em>ISME J.</em> 2012; <strong>6</strong>:1007-1017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2011.159" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84859911168" target="_blank" aria-label="Open Scopus in new tab">Scopus (909)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22134642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2011.159&amp;pmid=22134642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Louca et&nbsp;al., 2016b</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Louca, S. ∙ Parfrey, L.W. ∙ Doebeli, M.</div><div><strong>Decoupling function and taxonomy in the global ocean microbiome</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:1272-1277</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf4507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84988369671" target="_blank" aria-label="Open Scopus in new tab">Scopus (723)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27634532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf4507&amp;pmid=27634532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These results suggest that the availability of nutrients, such as organic carbon, oxygen, nitrate, carbon dioxide, and light, constrain the composition of the community in terms of the abundances of specific metabolic capabilities more so than they constrain the taxa possessing those capabilities. One implication of this finding is that communities with similar genomic composition, in terms of the metabolic pathways they possess, might exhibit similar rates and productivity of the associated metabolic process, but any such correspondence is yet to be demonstrated.</div><div id="p0060" role="paragraph">Corroborating the idea that nutrient availability strongly determines community composition, experiments in fixed nutrient conditions have shown that the metabolic traits of bacterial strains in assembled communities can be highly reproducible. To show this, several groups have sampled complex communities from natural environments and grown them under defined nutrient conditions in the laboratory (<span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Datta et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Datta, M.S. ∙ Sliwerska, E. ∙ Gore, J. ...</div><div><strong>Microbial interactions lead to rapid micro-scale successions on model marine particles</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:11965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms11965" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84975511062" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27311813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms11965&amp;pmid=27311813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Goldford et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Goldford, J.E. ∙ Lu, N. ∙ Bajić, D. ...</div><div><strong>Emergent simplicity in microbial community assembly</strong></div><div><em>Science.</em> 2018; <strong>361</strong>:469-474</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat1168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85051138998" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30072533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat1168&amp;pmid=30072533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Using this approach, Datta et&nbsp;al. showed that marine microbial communities degrading polysaccharide particles exhibit a succession of bacterial taxa (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Datta et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Datta, M.S. ∙ Sliwerska, E. ∙ Gore, J. ...</div><div><strong>Microbial interactions lead to rapid micro-scale successions on model marine particles</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:11965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms11965" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84975511062" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27311813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms11965&amp;pmid=27311813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Succession on these particles arises from initial colonizers that cleave polysaccharides, followed by strains that compete for the resulting oligosaccharides or consume byproducts of sugar metabolism. Similarly, bacterial communities sampled from leaf surfaces and enriched in&nbsp;glucose minimal medium reproducibly yield communities comprising both consumers of glucose and consumers of glucose metabolic byproducts (<span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Goldford et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Goldford, J.E. ∙ Lu, N. ∙ Bajić, D. ...</div><div><strong>Emergent simplicity in microbial community assembly</strong></div><div><em>Science.</em> 2018; <strong>361</strong>:469-474</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat1168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85051138998" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30072533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat1168&amp;pmid=30072533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0065" role="paragraph">Although the traits of organisms in an assembled community appear to be strongly influenced by the available nutrients, quantitatively relating genomic structure to the rates and productivity of metabolic processes at the community level remains a challenge. Phenotyping mutants of model organisms has done much to uncover the molecular basis of specific metabolic processes such as denitrification (<span class="dropBlock reference-citations"><a id="body-ref-sref117-1" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-1" aria-controls="sref117-1">Zumft, 1997</a><div class="dropBlock__holder" data-db-target-of="sref117-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Zumft, W.G.</div><div><strong>Cell biology and molecular basis of denitrification</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 1997; <strong>61</strong>:533-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/.61.4.533-616.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2F.61.4.533-616.1997&amp;pmid=9409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, it is unclear how to use a detailed molecular characterization of metabolism in a few model organisms to understand the metabolic traits of organisms in natural communities that possess high levels of genomic diversity (<span class="dropBlock reference-citations"><a id="body-ref-sref92-1" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-1" aria-controls="sref92-1">Sakoparnig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref92-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Sakoparnig, T. ∙ Field, C. ∙ Nimwegen, E. van.</div><div><strong>Whole genome phylogenies reflect the distributions of recombination rates for many bacterial species</strong></div><div><em>eLife.</em> 2021; <strong>10</strong>:e65366</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.65366" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85100215005" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33416498/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.65366&amp;pmid=33416498" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally, we lack an understanding of how this genomic diversity drives variation in metabolic traits and how phenotypes of individual taxa combine to give rise to metabolite fluxes through a community.</div><div id="p0070" role="paragraph">Here, we address the challenge of mapping gene content to metabolite dynamics by quantifying the flux of metabolites in an ensemble of genomically diverse communities composed of non-model organisms (see <a id="crosref0210" href="#fig1">Figure&nbsp;1</a> for a summary of the approach). We used bacterial denitrification, an essential metabolic process in the global nitrogen cycle that is performed by diverse and culturable bacterial taxa (<span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Lycus et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Lycus, P. ∙ Lovise Bøthun, K. ∙ Bergaust, L. ...</div><div><strong>Phenotypic and genotypic richness of denitrifiers revealed by a novel isolation strategy</strong></div><div><em>ISME J.</em> 2017; <strong>11</strong>:2219-2232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2017.82" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85029618940" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28696424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2017.82&amp;pmid=28696424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as a model metabolic process. We isolated an ensemble of denitrifiers and measured the dynamics of metabolite consumption and production for each isolate under controlled conditions. We then parameterized metabolite dynamics using a consumer-resource model. The genomic diversity of the ensemble of isolates enabled a simple linear regression approach to mapping gene content to consumer-resource model parameters, which resulted in a sparse and generalizable mapping of gene presence and absence to metabolic phenotypes. Finally, the consumer-resource model captured interactions between strains mediated by resource competition, yielding predictions for community-level metabolite dynamics that we verified experimentally.</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/fb5ee9c5-d1dd-4e9b-afac-7d61131a9008/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.12.036/asset/9000c6fd-78a9-45fa-aa4b-2cdc4ff6cf0e/main.assets/gr1.jpg" height="631" width="499" aria-labelledby="fig1-title" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/fb5ee9c5-d1dd-4e9b-afac-7d61131a9008/main.assets/gr1_lrg.jpg" loading="lazy"></a><div><button class="figure__open__ctrl">Figure viewer</button></div><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Workflow for predicting community metabolite dynamics from genomic structure</span></div></figcaption></figure></div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Denitrification as a model metabolic process</h3><div id="p0075" role="paragraph">We used denitrification as a model metabolic process because it is performed by diverse bacterial taxa, it is well characterized at the molecular level, and the relevant metabolites are readily quantifiable (<span class="dropBlock reference-citations"><a id="body-ref-sref117-2" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-2" aria-controls="sref117-2">Zumft, 1997</a><div class="dropBlock__holder" data-db-target-of="sref117-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Zumft, W.G.</div><div><strong>Cell biology and molecular basis of denitrification</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 1997; <strong>61</strong>:533-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/.61.4.533-616.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2F.61.4.533-616.1997&amp;pmid=9409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because denitrifiers are easily isolated and cultured (<span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Lycus et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Lycus, P. ∙ Lovise Bøthun, K. ∙ Bergaust, L. ...</div><div><strong>Phenotypic and genotypic richness of denitrifiers revealed by a novel isolation strategy</strong></div><div><em>ISME J.</em> 2017; <strong>11</strong>:2219-2232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2017.82" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85029618940" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28696424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2017.82&amp;pmid=28696424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we can capture substantial genomic diversity in an ensemble of natural isolates.</div><div id="p0080" role="paragraph">Denitrification is a form of anaerobic respiration whereby microbes use oxidized nitrogen compounds as electron acceptors, driving a cascade of four successive reduction reactions, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="right arrow" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="0" data-semantic-speech="right arrow"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="right arrow" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="0" data-semantic-speech="right arrow"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> NO <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="right arrow" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="0" data-semantic-speech="right arrow"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N 2" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal upper N 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>O <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="right arrow" tabindex="0" ctxtmenu_counter="6" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="0" data-semantic-speech="right arrow"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N 2" tabindex="0" ctxtmenu_counter="7" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal upper N 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<span class="dropBlock reference-citations"><a id="body-ref-sref117-3" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-3" aria-controls="sref117-3">Zumft, 1997</a><div class="dropBlock__holder" data-db-target-of="sref117-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Zumft, W.G.</div><div><strong>Cell biology and molecular basis of denitrification</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 1997; <strong>61</strong>:533-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/.61.4.533-616.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2F.61.4.533-616.1997&amp;pmid=9409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0235" href="#fig2">Figure&nbsp;2</a>A). As a biogeochemical process, denitrification is essential to nitrogen cycling at a global scale through activity in soils, freshwater systems, and marine environments (<span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Seitzinger et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Seitzinger, S. ∙ Harrison, J.A. ∙ Böhlke, J.K. ...</div><div><strong>Denitrification across landscapes and waterscapes: a synthesis</strong></div><div><em>Ecol. Appl.</em> 2006; <strong>16</strong>:2064-2090</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1890/1051-0761(2006)016%5B2064:DALAWA%5D2.0.CO;2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33845509937" target="_blank" aria-label="Open Scopus in new tab">Scopus (1092)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17205890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1890%2F1051-0761%282006%29016%255B2064%3ADALAWA%255D2.0.CO%3B2&amp;pmid=17205890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, denitrification impacts human health through activity in wastewater treatment plants (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Lu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Lu, H. ∙ Chandran, K. ∙ Stensel, D.</div><div><strong>Microbial ecology of denitrification in biological wastewater treatment</strong></div><div><em>Water Res.</em> 2014; <strong>64</strong>:237-254</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.watres.2014.06.042" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84904891891" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25078442/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.watres.2014.06.042&amp;pmid=25078442" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and in the human gut (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Irrazábal et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Irrazábal, T. ∙ Belcheva, A. ∙ Girardin, S.E. ...</div><div><strong>The multifaceted role of the intestinal microbiota in colon cancer</strong></div><div><em>Mol. Cell.</em> 2014; <strong>54</strong>:309-320</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.molcel.2014.03.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.molcel.2014.03.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84899156759" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24766895/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2014.03.039&amp;pmid=24766895" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The process is performed by taxonomically diverse bacteria (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Graf et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Graf, D.R.H. ∙ Jones, C.M. ∙ Hallin, S.</div><div><strong>Intergenomic comparisons highlight modularity of the denitrification pathway and underpin the importance of community structure for N2O emissions</strong></div><div><em>PLoS ONE.</em> 2014; <strong>9</strong>:e114118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0114118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84914680332" target="_blank" aria-label="Open Scopus in new tab">Scopus (234)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25436772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0114118&amp;pmid=25436772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) that are typically facultative anaerobes. The denitrification pathway is known to be modular, with some strains performing all four steps in the cascade and others performing one or a nearly arbitrary subset of reduction reactions (<span class="dropBlock reference-citations"><a id="body-ref-sref67-3" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-3" aria-controls="sref67-3">Lycus et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref67-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Lycus, P. ∙ Lovise Bøthun, K. ∙ Bergaust, L. ...</div><div><strong>Phenotypic and genotypic richness of denitrifiers revealed by a novel isolation strategy</strong></div><div><em>ISME J.</em> 2017; <strong>11</strong>:2219-2232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2017.82" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85029618940" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28696424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2017.82&amp;pmid=28696424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0265" href="#fig2">Figure&nbsp;2</a>A). Denitrification in nature is, therefore, a collective process, wherein a given strain can produce electron acceptors that can be utilized by other strains (<span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Lilja and Johnson, 2016</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Lilja, E.E. ∙ Johnson, D.R.</div><div><strong>Segregating metabolic processes into different microbial cells accelerates the consumption of inhibitory substrates</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:1568-1578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2015.243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84954437600" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2015.243&amp;pmid=26771930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/879e5ac8-d801-4305-8444-b297c2a22da1/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.12.036/asset/80341704-4b42-41b9-bc1e-a21b331097cf/main.assets/gr2.jpg" height="177" width="508" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/879e5ac8-d801-4305-8444-b297c2a22da1/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Denitrification as a model metabolic process</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0020" role="paragraph">(A) Denitrification is a form of anaerobic respiration whereby oxidized nitrogen compounds are used as electron acceptors. The process results in a cascade of reactions from nitrate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis normal upper N normal upper O 3 Superscript minus Baseline right parenthesis" tabindex="0" ctxtmenu_counter="8" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 (8 1 7 (6 (5 2 3) 4)) 9)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="0,9" data-semantic-owns="0 8 9" data-semantic-speech="left parenthesis normal upper N normal upper O 3 Superscript minus Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,6" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="1 7 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-parent="8" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> to di-nitrogen <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis normal upper N 2 right parenthesis" tabindex="0" ctxtmenu_counter="9" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (3 1 2) 4)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-owns="0 3 4" data-semantic-speech="left parenthesis normal upper N 2 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Some bacteria perform all four steps in the cascade (purple, “Nar/Nir”), whereas others perform only a subset of reactions. Two examples of the latter are shown here: “Nar” strains (blue) perform only nitrate reduction, and “Nir” strains (red) perform nitrite <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis normal upper N normal upper O 2 Superscript minus Baseline right parenthesis" tabindex="0" ctxtmenu_counter="10" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 (8 1 7 (6 (5 2 3) 4)) 9)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="0,9" data-semantic-owns="0 8 9" data-semantic-speech="left parenthesis normal upper N normal upper O 2 Superscript minus Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,6" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="1 7 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-parent="8" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> reduction and potentially also subsequent steps (dashed lines).</div><div id="fspara0025" role="paragraph">(B) A schematic representation of the molecular steps in the denitrification process. Denitrification serves as the terminal step in the electron transport chain (not shown) and, thereby, contributes to ATP generation. Reduction of nitrate to nitrite takes place either in the cytoplasm (via the enzyme Nar) or in the periplasm (Nap). Nitrate reduction in the cytoplasm via Nar requires nitrate and nitrite to be transported across the inner membrane (NarK1, NarK2, and NarK1K2). The subsequent three steps all occur in the periplasm and are encoded by the reductases Nir, Nor, and Nos as shown. There are two functionally equivalent types of Nir and Nor reductases: NirK/NirS and qNor/cNor, respectively.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Denitrification is well understood at the molecular level. The process couples the reduction of oxidized nitrogen compounds to the electron transport chain and, therefore, ATP production. The enzymes (reductases) that perform each step in the cascade are shown in <a id="crosref0275" href="#fig2">Figure&nbsp;2</a>B. Reduction of nitrate to nitrite can occur either in the cytoplasm, by the Nar reductase, or the periplasm, using Nap. Inner membrane NarK transporters (NarK1, NarK2, and NarK1K2) facilitate the exchange of nitrate and nitrite between the cytoplasm and the periplasm. The remaining three reactions all occur exclusively in the periplasmic space (<a id="crosref0280" href="#fig2">Figure&nbsp;2</a>B). The regulatory elements that control the expression of denitrification genes are also well characterized and include two-component systems that sense the oxidized nitrogen compounds and regulators that detect the loss of oxygen from the environment (<span class="dropBlock reference-citations"><a id="body-ref-sref117-4" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-4" aria-controls="sref117-4">Zumft, 1997</a><div class="dropBlock__holder" data-db-target-of="sref117-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Zumft, W.G.</div><div><strong>Cell biology and molecular basis of denitrification</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 1997; <strong>61</strong>:533-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/.61.4.533-616.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2F.61.4.533-616.1997&amp;pmid=9409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-1" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-1" aria-controls="sref89-1">Rodionov et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref89-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Rodionov, D.A. ∙ Dubchak, I.L. ∙ Arkin, A.P. ...</div><div><strong>Dissimilatory metabolism of nitrogen oxides in bacteria: comparative reconstruction of transcriptional networks</strong></div><div><em>PLoS Comput. Biol.</em> 2005; <strong>1</strong></div><div>e55, 17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.0010055" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85044710067" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16261196/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.0010055&amp;pmid=16261196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because most of these reactions occur in the periplasm, substrates can readily leak into the surrounding environment, enabling cross-feeding between denitrifiers (<span class="dropBlock reference-citations"><a id="body-ref-sref60-3" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-3" aria-controls="sref60-3">Lilja and Johnson, 2016</a><div class="dropBlock__holder" data-db-target-of="sref60-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Lilja, E.E. ∙ Johnson, D.R.</div><div><strong>Segregating metabolic processes into different microbial cells accelerates the consumption of inhibitory substrates</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:1568-1578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2015.243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84954437600" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2015.243&amp;pmid=26771930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0090" role="paragraph">We focused experimentally on the first two steps of denitrification: the conversion of nitrate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis normal upper N normal upper O 3 Superscript minus Baseline right parenthesis" tabindex="0" ctxtmenu_counter="11" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 (8 1 7 (6 (5 2 3) 4)) 9)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="0,9" data-semantic-owns="0 8 9" data-semantic-speech="left parenthesis normal upper N normal upper O 3 Superscript minus Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,6" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="1 7 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-parent="8" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> to nitrite <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis normal upper N normal upper O 2 Superscript minus Baseline right parenthesis" tabindex="0" ctxtmenu_counter="12" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 (8 1 7 (6 (5 2 3) 4)) 9)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="0,9" data-semantic-owns="0 8 9" data-semantic-speech="left parenthesis normal upper N normal upper O 2 Superscript minus Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,6" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="1 7 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-parent="8" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and subsequently nitric oxide (NO) (<a id="crosref0300" href="#fig2">Figure&nbsp;2</a>A). Nitrate and nitrite are soluble, enabling high-throughput measurements of metabolite dynamics (<span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Miranda et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Miranda, K.M. ∙ Espey, M.G. ∙ Wink, D.A.</div><div><strong>A rapid, simple spectrophotometric method for simultaneous detection of nitrate and nitrite</strong></div><div><em>Nitric Oxide.</em> 2001; <strong>5</strong>:62-71</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/niox.2000.0319" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0035709008" target="_blank" aria-label="Open Scopus in new tab">Scopus (2479)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11178938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fniox.2000.0319&amp;pmid=11178938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To obtain a genomically diverse ensemble of non-model organisms, we isolated 78 bacterial strains spanning <i>α</i>-, <i>β</i>-, and <i>γ</i>-proteobacteria from local soils using established techniques (<a id="crosref0305" href="#mmc1">Tables S1–S3</a>; <a id="crosref0310" href="#sec-5">STAR Methods</a>). Each strain was obtained in axenic culture and was characterized as performing one or both of the first two steps of denitrification. Therefore, strains were classified into one of three possible phenotypes (<a id="crosref0315" href="#fig2">Figures 2</a>A and <a id="crosref0320" href="#fig3">3</a>A): (1) Nar/Nir strains that perform both nitrate and nitrite reduction (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="13" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="right arrow" tabindex="0" ctxtmenu_counter="14" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="0" data-semantic-speech="right arrow"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="15" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="right arrow" tabindex="0" ctxtmenu_counter="16" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="0" data-semantic-speech="right arrow"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> NO), (2) Nar strains that perform only nitrate reduction (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="17" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="right arrow" tabindex="0" ctxtmenu_counter="18" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="0" data-semantic-speech="right arrow"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="19" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>), and (3) Nir strains that perform only nitrite reduction (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="20" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="right arrow" tabindex="0" ctxtmenu_counter="21" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="0" data-semantic-speech="right arrow"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> NO). In addition to these 78 isolates, our strain library also included the model denitrifier <i>Paracoccus denitrificans</i> (ATCC 19367).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/db54d1c3-11ac-4858-a8ef-10d0e4556678/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.12.036/asset/5e9d6e6b-7b74-4800-8080-11884d3c0800/main.assets/gr3.jpg" height="597" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/db54d1c3-11ac-4858-a8ef-10d0e4556678/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Quantifying nitrate and nitrite dynamics in an ensemble of denitrifiers to map genomic structure to community metabolism</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0035" role="paragraph">(A) Example batch culture metabolite dynamics for Nar/Nir (purple), Nar (blue), and Nir (red) isolates. Nitrate (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="22" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, blue points) and nitrite (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="23" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, red points) dynamics are measured at logarithmically spaced intervals (circles) via sampling and colorimetric assay (<a id="crosref2040" href="#sec-5">STAR Methods</a>), with ± 5% error bars shown. Biomass densities are only measured at the final time point. Curves show fits to a consumer-resource model shown in (B).</div><div id="fspara0040" role="paragraph">(B) A consumer-resource model of nitrate and nitrite reduction by each strain describes the evolution of biomass density (<i>x</i>, OD), nitrate concentration (<i>A</i>, mM), and nitrite concentration (<i>I</i>, mM) with time. The model is parameterized by reduction rates <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="24" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="25" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (mM/OD/h), and yields <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="26" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="27" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (OD/mM), for growth on nitrate and nitrite, respectively. The affinity parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="28" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="29" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (mM) were not well constrained by the data and were fixed for all strains in the library (<a id="crosref2045" href="#sec-5">STAR Methods</a>).</div><div id="fspara0045" role="paragraph">(C) Phylogenetic tree and normalized consumer-resource parameters for 79 denitrifying strains (78 isolates and the model denitrifier <i>Paracoccus denitrificans</i>). The strain library comprised 51 Nar/Nir, 24 Nar, and 4 Nir strains. Consumer-resource parameters were measured in a succinate-defined medium (SDM). Phylogenetic tree constructed using the 16S rRNA gene, and scale bar represents the estimated number of substitutions per site. Darker colors indicate larger values of the normalized parameters. Nitrate and nitrite reduction parameters were not measured for Nir and Nar strains, respectively. Consumer-resource parameters measured across diverse isolates constituted a dataset for relating genomic diversity to metabolite dynamics. See also <a id="crosref2050" href="#figs1">Figure&nbsp;S1</a> and <a id="crosref2055" href="#mmc1">Tables S1–S5</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Parameterizing metabolite dynamics</h3><div id="p0095" role="paragraph">We first set out to quantify the metabolic phenotypes of each isolate in our diverse strain library (Step 1, <a id="crosref0325" href="#fig1">Figure&nbsp;1</a>). We focused our efforts on quantifying the dynamics of the relevant metabolites, nitrate and nitrite. To accomplish this, strains were inoculated at low starting densities into 96-well plates containing a chemically defined, electron-acceptor-limited medium containing succinate as the sole non-fermentable carbon source (succinate-defined medium, SDM; <a id="crosref0330" href="#mmc1">Table S4</a>; <a id="crosref0335" href="#sec-5">STAR Methods</a>), with either nitrate or nitrite provided as the sole electron acceptor. Cultures were then incubated under anaerobic conditions (<a id="crosref0340" href="#sec-5">STAR Methods</a>). Small samples (10&nbsp;μL) were taken at logarithmically spaced time intervals over a period of 64&nbsp;h and assayed for nitrate and nitrite concentrations (<a id="crosref0345" href="#sec-5">STAR Methods</a>). At the end of the time course, optical density was assayed. The measurement resulted in a time series of nitrate and nitrite production/consumption dynamics in batch culture (points, <a id="crosref0350" href="#fig3">Figure&nbsp;3</a>A). Contamination between wells using this culturing and sampling approach was assessed to be low (<a id="crosref0355" href="#sec-5">STAR Methods</a>).</div><div id="p0100" role="paragraph">To parameterize the metabolite dynamics of each strain within a common framework, we utilized a consumer-resource model, which explicitly relates the growth of each strain to the dynamics of metabolite production and consumption (<a id="crosref0360" href="#fig3">Figure&nbsp;3</a>B; <a id="crosref0365" href="#fd3">Equation 3</a>; <a id="crosref0370" href="#sec-5">STAR Methods</a>). The model contains up to six parameters: rates (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript italic asterisk" tabindex="0" ctxtmenu_counter="30" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript italic asterisk"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, mM/OD/h), biomass yields (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript italic asterisk" tabindex="0" ctxtmenu_counter="31" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript italic asterisk"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, OD/mM), and affinities (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript italic asterisk" tabindex="0" ctxtmenu_counter="32" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript italic asterisk"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, mM), for the substrates nitrate (<i>A</i>) and nitrite (<i>I</i>). For each strain in monoculture, we parameterized the consumer-resource model using measured denitrification dynamics across a range of initial biomass densities and nitrate/nitrite concentrations (<a id="crosref0375" href="#fig3">Figures 3</a>A and <a id="crosref0380" href="#figs1">S1</a>A–S1E; <a id="crosref0385" href="#sec-5">STAR Methods</a>). These data allowed us to quantify rates <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis r Subscript italic asterisk Baseline right parenthesis" tabindex="0" ctxtmenu_counter="33" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (3 1 2) 4)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-owns="0 3 4" data-semantic-speech="left parenthesis r Subscript italic asterisk Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and biomass yields <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis gamma Subscript italic asterisk Baseline right parenthesis" tabindex="0" ctxtmenu_counter="34" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (3 1 2) 4)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-owns="0 3 4" data-semantic-speech="left parenthesis gamma Subscript italic asterisk Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> but not the affinity parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper K Subscript italic asterisk Baseline right parenthesis" tabindex="0" ctxtmenu_counter="35" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (3 1 2) 4)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-owns="0 3 4" data-semantic-speech="left parenthesis upper K Subscript italic asterisk Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, which require measuring growth rates at very low substrate concentrations. Because the results of parameter fits were not sensitive to the values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript italic asterisk" tabindex="0" ctxtmenu_counter="36" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript italic asterisk"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> across a broad range (<a id="crosref0390" href="#figs1">Figures S1</a>F–S1I; <a id="crosref0395" href="#sec-5">STAR Methods</a>), we fixed the affinity parameter to a small constant value. Therefore, we captured the phenotype of each strain in the library using at most four parameters: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="37" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="38" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="39" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="40" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (the models for Nar and Nir strains correspond to setting <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Baseline comma gamma Subscript upper I Baseline equals 0" tabindex="0" ctxtmenu_counter="41" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (2 0 1) 3 (9 (6 4 5) 7 8))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="2,3,9" data-semantic-content="3" data-semantic-owns="2 3 9" data-semantic-speech="r Subscript upper I Baseline comma gamma Subscript upper I Baseline equals 0"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="10" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="6 7 8"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="9" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container> or <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Baseline comma gamma Subscript upper A Baseline equals 0" tabindex="0" ctxtmenu_counter="42" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (2 0 1) 3 (9 (6 4 5) 7 8))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="2,3,9" data-semantic-content="3" data-semantic-owns="2 3 9" data-semantic-speech="r Subscript upper A Baseline comma gamma Subscript upper A Baseline equals 0"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="10" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="6 7 8"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="9" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, respectively). Yields <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis gamma Subscript italic asterisk Baseline right parenthesis" tabindex="0" ctxtmenu_counter="43" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (3 1 2) 4)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-owns="0 3 4" data-semantic-speech="left parenthesis gamma Subscript italic asterisk Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were inferred using optical density measurements at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t equals" tabindex="0" ctxtmenu_counter="44" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 64 h, and rates <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis r Subscript italic asterisk Baseline right parenthesis" tabindex="0" ctxtmenu_counter="45" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (3 1 2) 4)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-owns="0 3 4" data-semantic-speech="left parenthesis r Subscript italic asterisk Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were inferred by fitting the observed nitrate and nitrite dynamics to the consumer-resource model (<a id="crosref0400" href="#fig3">Figure&nbsp;3</a>B). For the majority of strains in our library (62 out of 79), a single set of parameters quantitatively described metabolite dynamics across a range of initial biomass densities and nitrate/nitrite concentrations. The consumer-resource model captured metabolite dynamics over a restricted set of initial conditions for the remaining 17 strains (<a id="crosref0405" href="#figs1">Figures S1</a>J–S1L; <a id="crosref0410" href="#mmc1">Table S5</a>; <a id="crosref0415" href="#sec-5">STAR Methods</a>). Using a representative subset of four strains, we confirmed that biomass density dynamics were well predicted by the consumer-resource parameters, despite the fact that biomass density was not directly measured over time (<a id="crosref0420" href="#figs1">Figure&nbsp;S1</a>M; <a id="crosref0425" href="#sec-5">STAR Methods</a>).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/363e7a12-d4f0-4991-a69f-16d53cf7518d/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.12.036/asset/69baad6f-d7b0-4c5d-81a6-4b5f351317ab/main.assets/figs1.jpg" height="923" width="686" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/363e7a12-d4f0-4991-a69f-16d53cf7518d/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Fitting consumer-resource parameters, related to <a id="crosref2135" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0150" role="paragraph">(A) Histogram showing the distribution of yield intercept <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis gamma 0 right parenthesis" tabindex="0" ctxtmenu_counter="46" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (3 1 2) 4)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-owns="0 3 4" data-semantic-speech="left parenthesis gamma 0 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> values for 79 strains cultured in SDM. Units are dimensionless absorbances at 600&nbsp;nm, path length normalized to 1&nbsp;cm.</div><div id="fspara0155" role="paragraph">(B–D) Examples of yield parameter fits using data obtained in different growth conditions. Points show observed values of ΔOD600 <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals" tabindex="0" ctxtmenu_counter="47" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-speech="equals"><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> OD600<sub>64</sub> – OD600<sub>0</sub> from different conditions where different amounts of nitrate and nitrite are reduced (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Delta upper A equals upper A 0 minus upper A 64" tabindex="0" ctxtmenu_counter="48" style="font-size: 130%;"><mjx-math data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="13" data-semantic-children="11,12" data-semantic-content="2" data-semantic-owns="11 2 12" data-semantic-structure="(13 (11 0 10 1) 2 (12 (5 3 4) 6 (9 7 8)))" data-semantic-speech="upper Delta upper A equals upper A 0 minus upper A 64" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="11" data-semantic-children="0,1" data-semantic-content="10" data-semantic-parent="13" data-semantic-owns="0 10 1"><mjx-mtext data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="0" data-semantic-parent="11"><mjx-c class="mjx-c394">Δ</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="11"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="13" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="12" data-semantic-children="5,9" data-semantic-content="6" data-semantic-parent="13" data-semantic-owns="5 6 9" inline-breaks="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="12" data-semantic-owns="3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-break size="3"></mjx-break><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="12" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="12" data-semantic-owns="7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.677em;">6</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Delta upper I equals upper A 0 minus upper A 64 plus upper I 0 minus upper I 64" tabindex="0" ctxtmenu_counter="49" style="font-size: 130%;"><mjx-math data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="23" data-semantic-children="19,22" data-semantic-content="2" data-semantic-owns="19 2 22" data-semantic-structure="(23 (19 0 18 1) 2 (22 (21 (20 (5 3 4) 6 (9 7 8)) 10 (13 11 12)) 14 (17 15 16)))" data-semantic-speech="upper Delta upper I equals upper A 0 minus upper A 64 plus upper I 0 minus upper I 64" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="19" data-semantic-children="0,1" data-semantic-content="18" data-semantic-parent="23" data-semantic-owns="0 18 1"><mjx-mtext data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="0" data-semantic-parent="19"><mjx-c class="mjx-c394">Δ</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="19"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="23" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="22" data-semantic-children="21,17" data-semantic-content="14" data-semantic-parent="23" data-semantic-owns="21 14 17" inline-breaks="true"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="21" data-semantic-children="20,13" data-semantic-content="10" data-semantic-parent="22" data-semantic-owns="20 10 13" inline-breaks="true"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="20" data-semantic-children="5,9" data-semantic-content="6" data-semantic-parent="21" data-semantic-owns="5 6 9" inline-breaks="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="20" data-semantic-owns="3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-break size="3"></mjx-break><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="20" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="20" data-semantic-owns="7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.677em;">6</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-break size="3"></mjx-break><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="10" data-semantic-parent="21" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="21" data-semantic-owns="11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-break size="3"></mjx-break><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="14" data-semantic-parent="22" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="22" data-semantic-owns="15 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.677em;">6</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, respectively) with 4 replicates used in each condition. The plane/lines show the least-squares fits of the data to <a id="crosref2140" href="#fd6">Equation 6</a> (<a id="crosref2145" href="#sec-5">STAR Methods</a>). (B) shows the yield fit for the Nar/Nir strain <i>P.&nbsp;denitrificans</i> ATCC 19367 in SDM. (C) shows the yield fit for the Nar strain <i>Raoultella</i> sp. RLT01 in SDM. (D) shows the yield fit for the Nir strain <i>Pseudomonas</i> sp. PDM13 in SDM.</div><div id="fspara0160" role="paragraph">(E) Example global fits of the consumer-resource model (<a id="crosref2150" href="#fd3">Equation 3</a>; <a id="crosref2155" href="#sec-5">STAR Methods</a>) to nitrate and nitrite dynamics data for the Nar/Nir strain <i>P.&nbsp;denitrificans</i> ATCC 19367 cultured in SDM. Points show measured concentrations of nitrate and nitrite, and curves show optimal model fits. Four replicates were used in each experimental condition.</div><div id="fspara0165" role="paragraph">(F and G) Distributions of fractional errors (%) for (F) the affinity parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="50" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="51" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and (G) the rate parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="52" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="53" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, fit using SDM monocultures. Distributions were computed via nonparametric bootstrap with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A Baseline comma upper K Subscript upper I Baseline" tabindex="0" ctxtmenu_counter="54" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 (2 0 1) 3 (6 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="2,3,6" data-semantic-content="3" data-semantic-owns="2 3 6" data-semantic-speech="upper K Subscript upper A Baseline comma upper K Subscript upper I Baseline"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="7" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="7" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> constrained during fitting between 0.001 and 10. Fractional error is defined as the ratio of the interquartile range obtained via bootstrapping to the value of the parameter obtained using a standard fit to all experimental data.</div><div id="fspara0170" role="paragraph">(H) Example nitrate and nitrite dynamics for the Nar strain <i>Raoultella</i> sp. RLT01 cultured in SDM. Solid lines show the fit to <a id="crosref2160" href="#fd3">Equation 3</a> (<a id="crosref2165" href="#sec-5">STAR Methods</a>) holding <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="55" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> fixed at 0.001, while dashed lines show fits holding <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A Baseline equals 0 period 1" tabindex="0" ctxtmenu_counter="56" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 (7 (2 0 1) 3 4) 5 6)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="8" data-semantic-children="7,5,6" data-semantic-content="5" data-semantic-owns="7 5 6" data-semantic-speech="upper K Subscript upper A Baseline equals 0 period 1"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="7" data-semantic-children="2,4" data-semantic-content="3" data-semantic-parent="8" data-semantic-owns="2 3 4"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="7" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="7" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="5" data-semantic-parent="8" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>.</div><div id="fspara0175" role="paragraph">(I) Comparison of model fit errors (RMSE) for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N equals 79" tabindex="0" ctxtmenu_counter="57" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper N equals 79"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c37" noic="true" style="padding-top: 0.676em;">7</mjx-c><mjx-c class="mjx-c39" style="padding-top: 0.676em;">9</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> denitrifying strains cultured in SDM. Fits that hold <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A Baseline equals upper K Subscript upper I Baseline equals 0 period 01" tabindex="0" ctxtmenu_counter="58" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 (11 (2 0 1) 3 (6 4 5) 7 8) 9 10)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="12" data-semantic-children="11,9,10" data-semantic-content="9" data-semantic-owns="11 9 10" data-semantic-speech="upper K Subscript upper A Baseline equals upper K Subscript upper I Baseline equals 0 period 01"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="2,6,8" data-semantic-content="3,7" data-semantic-parent="12" data-semantic-owns="2 3 6 7 8"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="11" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="11" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="9" data-semantic-parent="12" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and fits that take <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="59" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="60" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> as free fitting parameters are compared. A two-sample Kolomogorov-Smirnov test accepts the null hypothesis that underlying distributions for the two samples are the same <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis p equals 0 period 97 right parenthesis" tabindex="0" ctxtmenu_counter="61" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (7 (6 1 2 3) 4 5) 8)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="9" data-semantic-children="7" data-semantic-content="0,8" data-semantic-owns="0 7 8" data-semantic-speech="left parenthesis p equals 0 period 97 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="6,4,5" data-semantic-content="4" data-semantic-parent="9" data-semantic-owns="6 4 5"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="6" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="7" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="4" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c39" noic="true" style="padding-top: 0.676em;">9</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="8" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Boxplots indicating quartiles of each distribution are shown.</div><div id="fspara0180" role="paragraph">(J) Example Nar/Nir strain PDM25 cultured in SDM for which reduction rates were fit using all experimental conditions (dashed lines, RMSE <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 177" tabindex="0" ctxtmenu_counter="62" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 177"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.676em;">1</mjx-c><mjx-c class="mjx-c37" noic="true" style="padding-top: 0.676em;">7</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>) and using only conditions where nitrate was initially supplied (solid lines, RMSE <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 138" tabindex="0" ctxtmenu_counter="63" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 138"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c33" noic="true" style="padding-top: 0.666em;">3</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>).</div><div id="fspara0185" role="paragraph">(K) Example Nar strain ENT03 cultured in SDM for which the reduction rate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="64" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> was fit using all experimental conditions (dashed lines, RMSE <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 254" tabindex="0" ctxtmenu_counter="65" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 254"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.677em;">2</mjx-c><mjx-c class="mjx-c35" noic="true" style="padding-top: 0.677em;">5</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>) and using only conditions where OD600<sub>0</sub><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 01" tabindex="0" ctxtmenu_counter="66" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 01"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> (solid lines, RMSE <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 095" tabindex="0" ctxtmenu_counter="67" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 095"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c39" noic="true" style="padding-top: 0.666em;">9</mjx-c><mjx-c class="mjx-c35" style="padding-top: 0.666em;">5</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>).</div><div id="fspara0190" role="paragraph">(L) Example Nar strain PDM27 cultured in SDM for which nitrate concentrations appear to asymptotically approach a nonzero value.</div><div id="fspara0195" role="paragraph">(M) The biomass densities of four Nar/Nir strains (<i>Achromobacter</i> sp. ACM01, <i>Ensifer</i> sp. ENS09, <i>Paracoccus denitrificans</i> ATCC 19367, and <i>Pseudomonas</i> sp. PDM21) grown in SDM were measured in denitrifying conditions over 64&nbsp;h (points) to validate the predictions (curves) of the consumer-resource model (<a id="crosref2170" href="#fd3">Equation 3</a>; <a id="crosref2175" href="#sec-5">STAR Methods</a>). The parameters for each strain were inferred in previous SDM monocultures (<a id="crosref2180" href="#sec-5">STAR Methods</a>). 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="68" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> was initially supplied to each culture, and three experimental replicates were used for each time point. The median relative errors in biomass density predictions at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t equals" tabindex="0" ctxtmenu_counter="69" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 64&nbsp;h were −5.2%, −7.5%, 9.4%, and 6.4% for ACM01, ENS09, PAR19367, and PDM21, respectively.</div></div></div></div></figcaption></figure></div><div id="p0105" role="paragraph">Fitting our consumer-resource model to data for each strain yielded a quantitative description of the metabolic traits (i.e., denitrification rates and yields) of each strain in the library (<a id="crosref0430" href="#fig3">Figure 3</a>C). We observed large variability between taxa, with coefficients of variation for rate constants (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="70" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="71" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>) around 70% and yields (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="72" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="73" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>) around 100%. We also observed some patterns of phylogenetic conservation, for example, <i>α</i>-proteobacteria produced generally higher yields than <i>β</i>- or <i>γ</i>-proteobacteria did, and a clade of <i>Pseudomonas</i> sp. isolates showed consistently higher rates of nitrite reduction than most other strains (PDM17-23, <a id="crosref0435" href="#fig3">Figure&nbsp;3</a>C). Despite these patterns, the prevalence of each of the three qualitative phenotypes is not strongly dependent on phylogeny, with each present across the tree (<a id="crosref0440" href="#fig3">Figure&nbsp;3</a>C). The latter observation is consistent with pervasive horizontal gene transfer of denitrifying enzymes (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Heylen et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Heylen, K. ∙ Gevers, D. ∙ Vanparys, B. ...</div><div><strong>The incidence of nirS and nirK and their genetic heterogeneity in cultivated denitrifiers</strong></div><div><em>Environ. Microbiol.</em> 2006; <strong>8</strong>:2012-2021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1462-2920.2006.01081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33749411590" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17014499/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1462-2920.2006.01081.x&amp;pmid=17014499" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Jones et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Jones, C.M. ∙ Stres, B. ∙ Rosenquist, M. ...</div><div><strong>Phylogenetic analysis of nitrite, nitric oxide, and nitrous oxide respiratory enzymes reveal a complex evolutionary history for denitrification</strong></div><div><em>Mol. Biol. Evol.</em> 2008; <strong>25</strong>:1955-1966</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msn146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-49749112308" target="_blank" aria-label="Open Scopus in new tab">Scopus (335)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18614527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsn146&amp;pmid=18614527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, neither did we observe a correlation between rates and yields, nor was there an obvious bound on these parameters, suggesting that they are not subjected to a trade-off.</div></section><section id="sec-2-3"><h3>Predicting metabolite dynamics from genomes</h3><div id="p0110" role="paragraph">Understanding how genomic variation impacts metabolite dynamics at the community level requires first learning how genomic variation impacts the metabolic traits of individual strains. Therefore, we sought to determine how genomic variation across the strains in our library is related to variation in denitrification rates and yields (<a id="crosref0455" href="#fig3">Figure&nbsp;3</a>C). One common approach to the problem of relating genomes to metabolite dynamics is constraint-based modeling. Constraint-based models infer the set of all metabolic reactions performed by an organism from an annotated genome, and then predict growth rates and metabolite fluxes, assuming the metabolic network is in steady state and is subject to biologically motivated constraints (<span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Orth et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Orth, J.D. ∙ Thiele, I. ∙ Palsson, B.Ø.</div><div><strong>What is flux balance analysis?</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:245-248</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1614" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-77749320898" target="_blank" aria-label="Open Scopus in new tab">Scopus (1994)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20212490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1614&amp;pmid=20212490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Constraint-based methods have found some success in predicting collective metabolism from genomes (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Klitgord and Segrè, 2010</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Klitgord, N. ∙ Segrè, D.</div><div><strong>Environments that induce synthetic microbial ecosystems</strong></div><div><em>PLoS Comput. Biol.</em> 2010; <strong>6</strong>:e1001002</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1001002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-78649677586" target="_blank" aria-label="Open Scopus in new tab">Scopus (201)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21124952/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1001002&amp;pmid=21124952" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Mori et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Mori, M. ∙ Hwa, T. ∙ Martin, O.C. ...</div><div><strong>Constrained allocation flux balance analysis</strong></div><div><em>PLoS Comput. Biol.</em> 2016; <strong>12</strong>:e1004913</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1004913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84978791333" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27355325/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1004913&amp;pmid=27355325" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Harcombe et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Harcombe, W.R. ∙ Riehl, W.J. ∙ Dukovski, I. ...</div><div><strong>Metabolic resource allocation in individual microbes determines ecosystem interactions and spatial dynamics</strong></div><div><em>Cell Rep.</em> 2014; <strong>7</strong>:1104-1115</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.celrep.2014.03.070&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.celrep.2014.03.070&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84901273253" target="_blank" aria-label="Open Scopus in new tab">Scopus (280)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24794435/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2014.03.070&amp;pmid=24794435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but these methods require significant manual refinement (<span class="dropBlock reference-citations"><a id="body-ref-sref82-1" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-1" aria-controls="sref82-1">Norsigian et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref82-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Norsigian, C.J. ∙ Fang, X. ∙ Seif, Y. ...</div><div><strong>A workflow for generating multi-strain genome-scale metabolic models of prokaryotes</strong></div><div><em>Nat. Protoc.</em> 2020; <strong>15</strong>:1-14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0254-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85077092545" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31863076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0254-3&amp;pmid=31863076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), complicating the prospect of making predictions from the genomes of non-model organisms. As a result, successfully constructing constraint-based models of denitrification for all strains in our library is a daunting task.</div><div id="p0115" role="paragraph">We took an alternative approach to the problem of mapping genomes to metabolite dynamics. We asked whether the variation in metabolic phenotypes across strains in our library can be quantitatively predicted simply from knowledge of the genes possessed by each strain. Our conjecture was motivated by two observations. First the metabolic traits of bacteria correlate strongly with environmental variables in marine microbial communities (<span class="dropBlock reference-citations"><a id="body-ref-sref63-2" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-2" aria-controls="sref63-2">Louca et&nbsp;al., 2016a</a><div class="dropBlock__holder" data-db-target-of="sref63-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Louca, S. ∙ Jacques, S.M.S. ∙ Pires, A.P.F. ...</div><div><strong>High taxonomic variability despite stable functional structure across microbial communities</strong></div><div><em>Nat. Ecol. Evol.</em> 2016; <strong>1</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-016-0015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28812567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-016-0015&amp;pmid=28812567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example, the relative abundance of taxa capable of nitrate reduction are strongly correlated with local temperature, phosphate, and nitrate levels, suggesting that the presence of genes responsible for those traits might also be predictable from nutrient levels and temperature. Second, the statistics of gene presence and absence across large numbers of sequenced genomes provides insights into the functional roles that genes play in pathways, such as the coupling between dihydrofolate reductase and thymidylate synthase activity in the folate metabolism pathway (<span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Schober et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Schober, A.F. ∙ Mathis, A.D. ∙ Ingle, C. ...</div><div><strong>A two-enzyme adaptive unit within bacterial folate metabolism</strong></div><div><em>Cell Rep.</em> 2019; <strong>27</strong>:3359-3370.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.celrep.2019.05.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.celrep.2019.05.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85066804656" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31189117/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.05.030&amp;pmid=31189117" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Together, these observations suggest that the genes a strain possesses could allow for predictions of metabolic traits. Therefore, rather than building constraint-based metabolic models for all of our strains, each of which would require significant manual refinement, we took a simple regression approach.</div><div id="p0120" role="paragraph">We used linear regression to predict the consumer-resource model parameters (<a id="crosref0495" href="#fig3">Figure&nbsp;3</a>C) of each strain from gene presence and absence (Step 2, <a id="crosref0500" href="#fig1">Figure&nbsp;1</a>). To accomplish this, we performed whole genome sequencing on all 79 strains in the library. Then, we assembled and annotated each genome (<a id="crosref0505" href="#sec-5">STAR Methods</a>) and determined the complement of 17 denitrification-related genes possessed by each strain (<a id="crosref0510" href="#mmc1">Table S6</a>), exploiting the fact that the molecular and genetic basis of denitrification is well understood (<span class="dropBlock reference-citations"><a id="body-ref-sref117-5" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-5" aria-controls="sref117-5">Zumft, 1997</a><div class="dropBlock__holder" data-db-target-of="sref117-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Zumft, W.G.</div><div><strong>Cell biology and molecular basis of denitrification</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 1997; <strong>61</strong>:533-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/.61.4.533-616.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2F.61.4.533-616.1997&amp;pmid=9409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We identified not only the reductases that perform the reduction of the oxidized nitrogen compounds but also the sensors/regulators (<span class="dropBlock reference-citations"><a id="body-ref-sref89-2" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-2" aria-controls="sref89-2">Rodionov et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref89-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Rodionov, D.A. ∙ Dubchak, I.L. ∙ Arkin, A.P. ...</div><div><strong>Dissimilatory metabolism of nitrogen oxides in bacteria: comparative reconstruction of transcriptional networks</strong></div><div><em>PLoS Comput. Biol.</em> 2005; <strong>1</strong></div><div>e55, 17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.0010055" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85044710067" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16261196/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.0010055&amp;pmid=16261196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and transporters (<span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Moir and Wood, 2001</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Moir, J.W.B. ∙ Wood, N.J.</div><div><strong>Nitrate and nitrite transport in bacteria</strong></div><div><em>Cell. Mol. Life Sci.</em> 2001; <strong>58</strong>:215-224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/PL00000849" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11289303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FPL00000849&amp;pmid=11289303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) known to be involved in denitrification (<a id="crosref0530" href="#sec-5">STAR Methods</a>). We intentionally excluded genes encoding structural subunits and chaperones required for the functioning of any reductase (<a id="crosref0535" href="#mmc1">Table S7</a>) because such genes have the same presence/absence pattern as the corresponding reductases and, therefore, would have identical predictive power. The presence and absence the denitrification-related genes in each genome are presented in <a id="crosref0540" href="#fig4">Figure&nbsp;4</a>A. Patterns of gene presence and absence agree well with known features of the denitrification pathway, including the mutual exclusion (Pearson correlation −1.0 among nitrite reducers) of the two reductases performing nitrite reduction, NirS and NirK (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Jones et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Jones, C.M. ∙ Stres, B. ∙ Rosenquist, M. ...</div><div><strong>Phylogenetic analysis of nitrite, nitric oxide, and nitrous oxide respiratory enzymes reveal a complex evolutionary history for denitrification</strong></div><div><em>Mol. Biol. Evol.</em> 2008; <strong>25</strong>:1955-1966</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msn146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-49749112308" target="_blank" aria-label="Open Scopus in new tab">Scopus (335)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18614527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsn146&amp;pmid=18614527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Jones and Hallin, 2010</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Jones, C.M. ∙ Hallin, S.</div><div><strong>Ecological and evolutionary factors underlying global and local assembly of denitrifier communities</strong></div><div><em>ISME J.</em> 2010; <strong>4</strong>:633-641</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2009.152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-77951498193" target="_blank" aria-label="Open Scopus in new tab">Scopus (174)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20090785/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2009.152&amp;pmid=20090785" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/f26d0a0b-4e66-41db-9a07-64c9aefb4b4c/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.12.036/asset/cde99afa-7b18-4f0e-8d79-cd4e4e1dab0c/main.assets/gr4.jpg" height="670" width="656" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/f26d0a0b-4e66-41db-9a07-64c9aefb4b4c/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">A statistical mapping from gene presence and absence to metabolite dynamics of individual strains</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0055" role="paragraph">(A) The presence and absence of genes in the denitrification pathway for the 79 denitrifying strains in our library. The color of each circle corresponds to the gene function as indicated in the legend further on.</div><div id="fspara0060" role="paragraph">(B) Observed consumer-resource phenotypic parameters for each strain in SDM (e.g., nitrate reduction rate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="74" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <a id="crosref2060" href="#fig3">Figure&nbsp;3</a>C) were linearly regressed against gene presence and absence via <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L 1" tabindex="0" ctxtmenu_counter="75" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper L 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>-regularized regression, resulting in regression coefficients <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript j" tabindex="0" ctxtmenu_counter="76" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta Subscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> for each gene <i>j</i>, an intercept <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta 0" tabindex="0" ctxtmenu_counter="77" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and a noise term <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="epsilon Subscript i" tabindex="0" ctxtmenu_counter="78" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="epsilon Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D700">𝜀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> for each observation <i>i</i>. Coefficient <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript j" tabindex="0" ctxtmenu_counter="79" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta Subscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> captures the impact of possessing gene <i>j</i> on the corresponding phenotypic parameter. Independent regressions were performed for each phenotypic parameter.</div><div id="fspara0065" role="paragraph">(C–F) Predicted values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="80" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="81" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="82" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="83" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, respectively, plotted against measured values. The dashed line indicates perfect agreement between observations and predictions. The in-sample coefficients of determination for these data <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper R Subscript f i t Superscript 2 Baseline right parenthesis" tabindex="0" ctxtmenu_counter="84" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 0 (10 (9 1 (7 2 5 3 6 4)) 8) 11)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="0,11" data-semantic-owns="0 10 11" data-semantic-speech="left parenthesis upper R Subscript f i t Superscript 2 Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="1,7,8" data-semantic-parent="12" data-semantic-collapsed="(10 (9 1 7) 8)" data-semantic-owns="1 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="10"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="2,3,4" data-semantic-content="5,6" data-semantic-parent="10" data-semantic-owns="2 5 3 6 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and the out-of-sample coefficients of determination estimated via iterated 4-fold cross-validation <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper R overbar Subscript italic CV Superscript 2 Baseline right parenthesis" tabindex="0" ctxtmenu_counter="85" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 7 (6 (5 (2 0 1) 3) 4) 8)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="9" data-semantic-children="6" data-semantic-content="7,8" data-semantic-owns="7 6 8" data-semantic-speech="left parenthesis upper R overbar Subscript italic CV Superscript 2 Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="7" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-parent="9" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="6" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mtext size="s" data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="italic" data-semantic-annotation="clearspeak:unit" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436" noic="true" style="padding-top: 0.705em;">𝐶</mjx-c><mjx-c class="mjx-c1D449" style="padding-top: 0.705em;">𝑉</mjx-c></mjx-mtext></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="8" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> are shown. <i>N</i> indicates the number strains in each regression. Strains that do not perform a particular reaction were omitted from the corresponding regression (e.g., Nir strains were excluded from the regression for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="86" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>).</div><div id="fspara0070" role="paragraph">(G–J) Estimates of <i>β</i> for each gene and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta 0" tabindex="0" ctxtmenu_counter="87" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="88" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="89" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="90" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="91" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, respectively. Asterisks indicate significance level for each <i>β</i> (<sup>∗</sup>: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than or equals 0 period 05" tabindex="0" ctxtmenu_counter="92" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="p less than or equals 0 period 05"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,≤"><mjx-c class="mjx-c2264">≤</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c35" style="padding-top: 0.666em;">5</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, <sup>∗∗</sup>: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than or equals 10 Superscript negative 2" tabindex="0" ctxtmenu_counter="93" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 1 (6 2 (5 3 4)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="0,6" data-semantic-content="1" data-semantic-owns="0 1 6" data-semantic-speech="p less than or equals 10 Superscript negative 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="relseq,≤"><mjx-c class="mjx-c2264">≤</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="6" data-semantic-children="2,5" data-semantic-parent="7" data-semantic-owns="2 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-children="4" data-semantic-content="3" data-semantic-parent="6" data-semantic-owns="3 4"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>, <sup>∗∗∗</sup>: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than or equals 10 Superscript negative 3" tabindex="0" ctxtmenu_counter="94" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 1 (6 2 (5 3 4)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="0,6" data-semantic-content="1" data-semantic-owns="0 1 6" data-semantic-speech="p less than or equals 10 Superscript negative 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="relseq,≤"><mjx-c class="mjx-c2264">≤</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="6" data-semantic-children="2,5" data-semantic-parent="7" data-semantic-owns="2 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-children="4" data-semantic-content="3" data-semantic-parent="6" data-semantic-owns="3 4"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>, and <sup>∗∗∗∗</sup>: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than or equals 10 Superscript negative 4" tabindex="0" ctxtmenu_counter="95" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 1 (6 2 (5 3 4)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="0,6" data-semantic-content="1" data-semantic-owns="0 1 6" data-semantic-speech="p less than or equals 10 Superscript negative 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="relseq,≤"><mjx-c class="mjx-c2264">≤</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="6" data-semantic-children="2,5" data-semantic-parent="7" data-semantic-owns="2 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-children="4" data-semantic-content="3" data-semantic-parent="6" data-semantic-owns="3 4"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>; <a id="crosref2065" href="#sec-5">STAR Methods</a>). See also <a id="crosref2070" href="#figs2">Figure&nbsp;S2</a> and <a id="crosref2075" href="#mmc1">Tables S6</a> and <a id="crosref2080" href="#mmc1">S7</a>.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">Next, we showed that the presence and absence of denitrification genes in each strain were sufficient to quantitatively predict metabolite dynamics in monoculture. Specifically, we constructed a linear regression where the measured phenotypic parameters of our consumer-resource model were predicted on the basis of gene presence and absence (<a id="crosref0555" href="#fig4">Figure&nbsp;4</a>B). Consistent with the observation that bacterial genomes are streamlined (<span class="dropBlock reference-citations"><a id="body-ref-sref68-1" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-1" aria-controls="sref68-1">Lynch, 2006</a><div class="dropBlock__holder" data-db-target-of="sref68-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Lynch, M.</div><div><strong>Streamlining and simplification of microbial genome architecture</strong></div><div><em>Annu. Rev. Microbiol.</em> 2006; <strong>60</strong>:327-349</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.micro.60.080805.142300" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33750344923" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16824010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.micro.60.080805.142300&amp;pmid=16824010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), almost all strains possessing nitrate and/or nitrite reductase performed the associated reactions in culture (the only exception being the Nar strain <i>Acidovorax</i> sp. ACV01, which possesses both nitrate and nitrite reductase, <a id="crosref0565" href="#fig4">Figure&nbsp;4</a>A). Therefore, we carried out independent regressions for each consumer-resource model parameter using only strains that performed the associated reaction (i.e., Nar and Nar/Nir strains for the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="96" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="97" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> regressions, and Nir and Nar/Nir strains for the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="98" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="99" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> regressions). The regression coefficients for each gene quantify the impact of the presence of the gene on a given phenotypic parameter. We used <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L 1" tabindex="0" ctxtmenu_counter="100" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper L 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>-regularized regression (least absolute shrinkage and selection operator, LASSO) to avoid overfitting, performing independent regressions for each of the phenotypic parameters in our consumer-resource model (<a id="crosref0570" href="#fig4">Figures 4</a>C–4J; <a id="crosref0575" href="#sec-5">STAR Methods</a>). By design, LASSO searches for a level of sparsity that optimizes predictive power, often selecting a few variables to make predictions while forcing other coefficients (<i>β<sub>j</sub></i>) to zero. The result can then be a sparse model that makes predictions using a handful of variables. It is important to note that LASSO does not first presume that a few variables are sufficient to make a prediction (in contrast to forward stepwise and best subset regression approaches). In the situation where strong predictive power does not exist, e.g., a phenotypic parameter cannot be predicted well from gene presence and absence, LASSO would effectively fail to identify a predictive model by returning <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript j Baseline equals 0" tabindex="0" ctxtmenu_counter="101" style="font-size: 130%;"><mjx-math data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="2,4" data-semantic-content="3" data-semantic-owns="2 3 4" data-semantic-structure="(5 (2 0 1) 3 4)" data-semantic-speech="beta Subscript j Baseline equals 0" breakable="true" class="MM-N" aria-hidden="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="5" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-math></mjx-container> for all genes (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Fraebel et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Fraebel, D.T. ∙ Gowda, K. ∙ Mani, M. ...</div><div><strong>Evolution of generalists by phenotypic plasticity</strong></div><div><em>iScience.</em> 2020; <strong>23</strong>:101678</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.101678&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.101678&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85094604886" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33163936/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2020.101678&amp;pmid=33163936" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0130" role="paragraph">Performing LASSO regressions on our dataset revealed that the presence and absence of a small set of genes is highly predictive of the consumer-resource parameters for all strains in our library (<a id="crosref0585" href="#fig4">Figures 4</a>C–4J). The in-sample coefficients of determination <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper R Subscript f i t Superscript 2 Baseline right parenthesis" tabindex="0" ctxtmenu_counter="102" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 0 (10 (9 1 (7 2 5 3 6 4)) 8) 11)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="0,11" data-semantic-owns="0 10 11" data-semantic-speech="left parenthesis upper R Subscript f i t Superscript 2 Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="1,7,8" data-semantic-parent="12" data-semantic-collapsed="(10 (9 1 7) 8)" data-semantic-owns="1 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="10"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="2,3,4" data-semantic-content="5,6" data-semantic-parent="10" data-semantic-owns="2 5 3 6 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> of our regressions were between 0.55 and 0.74 depending on the phenotypic parameter. Crucially, our regression approach generalized out-of-sample, as determined by iterated 4-fold cross-validation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 times 10 Superscript 4" tabindex="0" ctxtmenu_counter="103" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="1 times 10 Superscript 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> iterations; <a id="crosref0590" href="#sec-5">STAR Methods</a>), albeit with a slightly lower predictive power (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="104" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> between 0.36 and 0.56). Therefore, across a diverse set of natural isolates, knowledge of the full complement of genes a denitrifying strain possesses is sufficient to accurately predict the rates and biomass yields of that strain on nitrate and/or nitrite.</div><section id="sec-2-3-1"><h4>Validating regression approach to predicting traits from gene presence and absence</h4><div id="p0135" role="paragraph">Our regression approach leveraged biological knowledge of the denitrification pathway to predict metabolite dynamics, in effect presuming that denitrification gene content is the only significant genomic feature for prediction. To investigate whether this assumption is correct, we asked whether other genomic properties could better predict metabolite dynamics and also examined the role that phylogenetic correlations played in our predictions.</div><div id="p0140" role="paragraph">First, we tested the predictive capability of sets of randomly selected genes. To do this, we chose sets of 17 random genes that were not strongly correlated with any denitrification genes but retained the same marginal frequency distribution in the population as the denitrification genes. We found that regressions using these randomly selected genes had, on average, much less predictive power than regressions using the denitrification genes (<a id="crosref0595" href="#figs2">Figures S2</a>A–S2C; <a id="crosref0600" href="#sec-5">STAR Methods</a>). We also tested augmented sets of up to 2,048 predictors that were generated by adding varying numbers of randomly selected genes to the 17 denitrification genes. We found that the prediction quality changed remarkably little as more genes were added and that even sets of 2,048 predictors (representing approximately 30%–50% of genes in each genome) contained about as much predictive power as the regressions using the 17 denitrification genes alone (<a id="crosref0605" href="#figs2">Figure&nbsp;S2</a>D; <a id="crosref0610" href="#sec-5">STAR Methods</a>). This result indicates that the 17 denitrification genes harbor the majority of gene presence and absence predictive power.</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/d1d5bc29-3ecc-4d9c-b521-6050c2236522/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.12.036/asset/386063b3-daa9-4256-a890-962a1e22dafb/main.assets/figs2.jpg" height="515" width="506" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/d1d5bc29-3ecc-4d9c-b521-6050c2236522/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Randomly selected genes as alternative predictors for consumer-resource parameters, related to <a id="crosref2185" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0205" role="paragraph">(A) Distributions of gene presence frequency (fraction of strains that possess a given gene) for denitrification-related genes (red) and the distribution of gene presence frequency for all other annotated genes (black), both in the ensemble of the 75 nitrate-reducing strains. Solid lines show empirical cumulative distribution functions (CDFs) of gene frequencies, and dashed lines show kernel density estimates of these distributions (bandwidth&nbsp;= 0.4).</div><div id="fspara0210" role="paragraph">(B and C) Distributions of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="105" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> values obtained by regressing each of the SDM consumer-resource parameters onto the presence and absence of sets of randomly selected genes (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 times 10 cubed" tabindex="0" ctxtmenu_counter="106" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="1 times 10 cubed"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> sets of random genes per consumer-resource parameter). (B) shows results for genes randomly selected from the set of all annotated genes across strains in our library, while (C) shows results for genes randomly selected from the set of all annotated genes excluding those that have large and significant correlation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartAbsoluteValue rho EndAbsoluteValue greater than or equals" tabindex="0" ctxtmenu_counter="107" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (4 0 1 2) 3 5)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="6" data-semantic-children="4,5" data-semantic-content="3" data-semantic-owns="4 3 5" data-semantic-speech="StartAbsoluteValue rho EndAbsoluteValue greater than or equals"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="4" data-semantic-children="1" data-semantic-content="0,2" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="0" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="relseq,≥"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="5" data-semantic-parent="6"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 0.5) with any denitrification genes. Dashed lines indicate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="108" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> values obtained in regressions onto the presence and absence of denitrification-related genes (the same values are shown in both panels) with the corresponding quantile values (<i>q</i>).</div><div id="fspara0215" role="paragraph">(D) Points show distributions of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="109" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> obtained by regressing the SDM consumer-resource parameters onto sets of the 17 denitrification genes plus <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P minus 17" tabindex="0" ctxtmenu_counter="110" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper P minus 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.676em;">1</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> additional randomly selected genes (10 predictor sets per consumer-resource parameter), and the solid lines pass through the median values of these distributions as a function of <i>P</i>. The values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="111" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> shown at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P equals 17" tabindex="0" ctxtmenu_counter="112" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper P equals 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.676em;">1</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> are the same as the values indicated by dashed lines in (B) and (C).</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">Second, we tested whether 16S rRNA copy number, genome size, or GC-content improves the predictive ability of denitrification gene presence/absence regressions. We tested these genomic features because: (1) 16S rRNA copy number has been observed to correlate positively with maximal growth rate in nutrient-rich conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref90-1" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-1" aria-controls="sref90-1">Roller et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref90-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Roller, B.R.K. ∙ Stoddard, S.F. ∙ Schmidt, T.M.</div><div><strong>Exploiting rRNA operon copy number to investigate bacterial reproductive strategies</strong></div><div><em>Nat. Microbiol.</em> 2016; <strong>1</strong>:16160</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmicrobiol.2016.160" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84987667484" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27617693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmicrobiol.2016.160&amp;pmid=27617693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Li, J. ∙ Mau, R.L. ∙ Dijkstra, P. ...</div><div><strong>Predictive genomic traits for bacterial growth in culture versus actual growth in soil</strong></div><div><em>ISME J.</em> 2019; <strong>13</strong>:2162-2172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41396-019-0422-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85065340904" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31053828/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41396-019-0422-z&amp;pmid=31053828" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), (2) smaller genomes are associated with faster growth (<span class="dropBlock reference-citations"><a id="body-ref-sref68-2" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-2" aria-controls="sref68-2">Lynch, 2006</a><div class="dropBlock__holder" data-db-target-of="sref68-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Lynch, M.</div><div><strong>Streamlining and simplification of microbial genome architecture</strong></div><div><em>Annu. Rev. Microbiol.</em> 2006; <strong>60</strong>:327-349</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.micro.60.080805.142300" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33750344923" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16824010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.micro.60.080805.142300&amp;pmid=16824010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Li, J. ∙ Mau, R.L. ∙ Dijkstra, P. ...</div><div><strong>Predictive genomic traits for bacterial growth in culture versus actual growth in soil</strong></div><div><em>ISME J.</em> 2019; <strong>13</strong>:2162-2172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41396-019-0422-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85065340904" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31053828/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41396-019-0422-z&amp;pmid=31053828" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and (3) GC-content has been investigated as a correlate for numerous bacterial phenotypes, such as optimal growth temperature (<span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">Galtier and Lobry, 1997</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Galtier, N. ∙ Lobry, J.R.</div><div><strong>Relationships between genomic G+C content, RNA secondary structures, and optimal growth temperature in prokaryotes</strong></div><div><em>J.&nbsp;Mol. Evol.</em> 1997; <strong>44</strong>:632-636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/PL00006186" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0030900384" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9169555/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FPL00006186&amp;pmid=9169555" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and can serve as a baseline for spurious phylogenetic correlations because it is a slowly evolving genomic property that exhibits a high degree of phylogenetic correlation (<span class="dropBlock reference-citations"><a id="body-ref-sref39-1" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-1" aria-controls="sref39-1">Haywood-Farmer and Otto, 2003</a><div class="dropBlock__holder" data-db-target-of="sref39-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Haywood-Farmer, E. ∙ Otto, S.P.</div><div><strong>The evolution of genomic base composition in bacteria</strong></div><div><em>Evolution.</em> 2003; <strong>57</strong>:1783-1792</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.0014-3820.2003.tb00586.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0042287719" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14503620/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.0014-3820.2003.tb00586.x&amp;pmid=14503620" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found that including these additional predictors in our regressions alongside the 17 denitrification genes did not meaningfully improve predictive ability or alter the inferred coefficients (<a id="crosref0645" href="#sec-5">STAR Methods</a>). Thus, denitrification gene presence and absence outperformed these coarse genomic features.</div><div id="p0150" role="paragraph">Third, we examined the role of correlations in consumer-resource parameters between closely related strains in the success of our regressions. We quantified the extent of phylogenetic correlation in our 79-strain library by computing the autocorrelation (Moran’s <i>I</i>) for each consumer-resource parameter as a function of phylogenetic distance (<a id="crosref0650" href="#sec-5">STAR Methods</a>). We observed that the rate parameter <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="113" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> was correlated to a small degree <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis max left parenthesis upper I right parenthesis equals 0 period 16 right parenthesis" tabindex="0" ctxtmenu_counter="114" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 13 (12 (11 (10 0 9 (4 2 1 3)) 5 6) 7 8) 14)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="12" data-semantic-content="13,14" data-semantic-owns="13 12 14" data-semantic-speech="left parenthesis max left parenthesis upper I right parenthesis equals 0 period 16 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="13" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="12" data-semantic-children="11,7,8" data-semantic-content="7" data-semantic-parent="15" data-semantic-owns="11 7 8"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="10,6" data-semantic-content="5" data-semantic-parent="12" data-semantic-owns="10 5 6"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="limit function" data-semantic-id="10" data-semantic-children="0,4" data-semantic-content="9,0" data-semantic-parent="11" data-semantic-owns="0 9 4"><mjx-mtext data-semantic-type="function" data-semantic-role="limit function" data-semantic-annotation="general:text" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="appl"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.448em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.448em;">a</mjx-c><mjx-c class="mjx-c78" style="padding-top: 0.448em;">x</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="1" data-semantic-content="2,3" data-semantic-parent="10" data-semantic-owns="2 1 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="7" data-semantic-parent="12" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c36" style="padding-top: 0.666em;">6</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> over short a phylogenetic distance (16S distance 0.01), whereas the parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="115" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="116" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="117" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> showed a modest degree of correlation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="max left parenthesis upper I right parenthesis equals 0 period 33" tabindex="0" ctxtmenu_counter="118" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="12" data-semantic-children="11,7,8" data-semantic-content="7" data-semantic-owns="11 7 8" data-semantic-structure="(12 (11 (10 0 9 (4 2 1 3)) 5 6) 7 8)" data-semantic-speech="max left parenthesis upper I right parenthesis equals 0 period 33" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="10,6" data-semantic-content="5" data-semantic-parent="12" data-semantic-owns="10 5 6" inline-breaks="true"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="limit function" data-semantic-id="10" data-semantic-children="0,4" data-semantic-content="9,0" data-semantic-parent="11" data-semantic-owns="0 9 4"><mjx-mtext data-semantic-type="function" data-semantic-role="limit function" data-semantic-annotation="general:text" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="appl"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.448em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.448em;">a</mjx-c><mjx-c class="mjx-c78" style="padding-top: 0.448em;">x</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="1" data-semantic-content="2,3" data-semantic-parent="10" data-semantic-owns="2 1 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="7" data-semantic-parent="12" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c33" noic="true" style="padding-top: 0.666em;">3</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.666em;">3</mjx-c></mjx-mn></mjx-math></mjx-container>, 0.27, and 0.48, respectively) over relatively longer distances (16S distance 0.16, 0.06, and 0.12, respectively). Pruning clades of closely related strains (e.g., ENS01–08, PDM20–23, <a id="crosref0655" href="#fig3">Figure&nbsp;3</a>C) from the dataset decreased the correlation of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="119" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="120" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="121" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="max left parenthesis upper I right parenthesis equals 0 period 30" tabindex="0" ctxtmenu_counter="122" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="12" data-semantic-children="11,7,8" data-semantic-content="7" data-semantic-owns="11 7 8" data-semantic-structure="(12 (11 (10 0 9 (4 2 1 3)) 5 6) 7 8)" data-semantic-speech="max left parenthesis upper I right parenthesis equals 0 period 30" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="10,6" data-semantic-content="5" data-semantic-parent="12" data-semantic-owns="10 5 6" inline-breaks="true"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="limit function" data-semantic-id="10" data-semantic-children="0,4" data-semantic-content="9,0" data-semantic-parent="11" data-semantic-owns="0 9 4"><mjx-mtext data-semantic-type="function" data-semantic-role="limit function" data-semantic-annotation="general:text" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="appl"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.448em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.448em;">a</mjx-c><mjx-c class="mjx-c78" style="padding-top: 0.448em;">x</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="1" data-semantic-content="2,3" data-semantic-parent="10" data-semantic-owns="2 1 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="7" data-semantic-parent="12" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c33" noic="true" style="padding-top: 0.666em;">3</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn></mjx-math></mjx-container>, 0.21, and 0.39, respectively; 16S distance 0.05, 0.06, and 0.09, respectively) but had little impact on the correlation of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="123" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. Thus, some of the phylogenetic correlation is attributable to the over-representation of close relatives. Finally, we showed that the presence of these close relatives in our dataset did not skew the results of our regressions. We performed regressions on the pruned dataset (comprising 64 strains) and found that the predictive power and regression coefficients were similar to those for the full dataset (<a id="crosref0660" href="#sec-5">STAR Methods</a>). From this, we concluded that the over-representation of close relatives did not have a large impact on the results of our regressions on the consumer-resource parameters.</div></section><section id="sec-2-3-2"><h4>Generalizing the regression approach to an alternative medium condition</h4><div id="p0155" role="paragraph">Having mapped gene content to metabolite dynamics in a medium with succinate supplied as the carbon source, we next asked whether our regression approach would generalize to other media conditions. Of the 79 strains in our library, 64 grew on a defined medium with acetate supplied as the sole (non-fermentable) carbon source (acetate-defined medium, ADM; <a id="crosref0665" href="#mmc1">Table S1</a>; <a id="crosref0670" href="#sec-5">STAR Methods</a>). We assayed nitrate and nitrite dynamics for the 64 strains in this medium and inferred consumer-resource parameters. We observed that the consumer-resource parameters in the SDM and ADM conditions were strongly correlated (Pearson correlations 0.52–0.93, <a id="crosref0675" href="#fig5">Figure&nbsp;5</a>A). Furthermore, LASSO regressions to predict consumer-resource model parameters measured in ADM from gene presence and absence achieved predictive power similar to what we observed in SDM (<a id="crosref0680" href="#sec-5">STAR Methods</a>). The regression coefficients were correlated between nutrient conditions (<a id="crosref0685" href="#fig5">Figure&nbsp;5</a>B), suggesting that the impacts of genes on phenotypes were conserved between conditions. We note, however, that rates and yields in ADM were systematically lower relative to SDM (<a id="crosref0690" href="#fig5">Figure&nbsp;5</a>A), consistent with what has been observed previously for relative growth rates on these carbon sources (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Forchhammer and Lindahl, 1971</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Forchhammer, J. ∙ Lindahl, L.</div><div><strong>Growth rate of polypeptide chains as a function of the cell growth rate in a mutant of <i>Escherichia coli</i> 15</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1971; <strong>55</strong>:563-568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(71)90337-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0015222649" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4927947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2871%2990337-8&amp;pmid=4927947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Hempfling and Mainzer, 1975</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Hempfling, W.P. ∙ Mainzer, S.E.</div><div><strong>Effects of varying the carbon source limiting growth on yield and maintenance characteristics of <i>Escherichia coli</i> in continuous culture</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1975; <strong>123</strong>:1076-1087</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.123.3.1076-1087.1975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/169226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.123.3.1076-1087.1975&amp;pmid=169226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consequently, the magnitudes of regression coefficients were generally smaller in ADM than in SDM (<a id="crosref0705" href="#fig5">Figure&nbsp;5</a>B). This indicates that, while conserved genotype to phenotype relationships may generally underlie predictive power across different environments and media conditions, predictions for a particular environment will be more accurate when trained using data measured in that environment.</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/9f884e43-394e-4ec4-b659-2d3d2671f6a3/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.12.036/asset/b400299d-506f-4f79-b6ad-cd5445c36dfa/main.assets/gr5.jpg" height="350" width="656" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/9f884e43-394e-4ec4-b659-2d3d2671f6a3/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Metabolite dynamics of individual strains are predictable from gene presence and absence in an alternate carbon source</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0080" role="paragraph">All strains in the 79-strain library were screened for growth on an acetate-defined medium (ADM), and consumer-resource parameters were measured for the 64 strains that grew in this medium.</div><div id="fspara0085" role="paragraph">(A) Observed consumer-resource parameters on succinate-defined medium (SDM) are plotted against observed parameters on ADM. The dashed line indicates perfect agreement between the values observed on SDM and ADM. The Pearson correlations between the observed values are shown, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than 10 Superscript negative 4" tabindex="0" ctxtmenu_counter="124" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 1 (6 2 (5 3 4)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="0,6" data-semantic-content="1" data-semantic-owns="0 1 6" data-semantic-speech="p less than 10 Superscript negative 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="6" data-semantic-children="2,5" data-semantic-parent="7" data-semantic-owns="2 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-children="4" data-semantic-content="3" data-semantic-parent="6" data-semantic-owns="3 4"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> for all correlations (permutation test).</div><div id="fspara0090" role="paragraph">(B) The consumer-resource parameters on ADM were regressed against gene presence and absence via <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L 1" tabindex="0" ctxtmenu_counter="125" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper L 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>-regularized linear regression. The resulting regression coefficients, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript upper A upper D upper M" tabindex="0" ctxtmenu_counter="126" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 (6 1 4 2 5 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="0,6" data-semantic-owns="0 6" data-semantic-speech="beta Subscript upper A upper D upper M"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="7" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, are plotted against the coefficients for regressions on parameters measured in SDM, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript upper S upper D upper M" tabindex="0" ctxtmenu_counter="127" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 (6 1 4 2 5 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="0,6" data-semantic-owns="0 6" data-semantic-speech="beta Subscript upper S upper D upper M"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="7" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D446">𝑆</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (shown also in <a id="crosref2085" href="#fig4">Figures 4</a>G–4J). The dashed line indicates perfect agreement between each pair of regression coefficients. Pearson correlations are shown, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p equals 0 period 008" tabindex="0" ctxtmenu_counter="128" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="p equals 0 period 008"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, 0.01, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than 10 Superscript negative 4" tabindex="0" ctxtmenu_counter="129" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 6 0 (5 1 (4 2 3)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="6,5" data-semantic-content="0" data-semantic-owns="6 0 5" data-semantic-speech="less than 10 Superscript negative 4"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="6" data-semantic-parent="7"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-parent="7" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="5" data-semantic-children="1,4" data-semantic-parent="7" data-semantic-owns="1 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-children="3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than 10 Superscript negative 4" tabindex="0" ctxtmenu_counter="130" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 6 0 (5 1 (4 2 3)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="6,5" data-semantic-content="0" data-semantic-owns="6 0 5" data-semantic-speech="less than 10 Superscript negative 4"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="6" data-semantic-parent="7"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-parent="7" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="5" data-semantic-children="1,4" data-semantic-parent="7" data-semantic-owns="1 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-children="3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="131" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="132" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="133" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="134" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, respectively (permutation test). The color of each point corresponds to the gene function, as indicated in the legend further on. See also <a id="crosref2090" href="#mmc1">Table S1</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-3-3"><h4>Mechanistic interpretation of regression coefficients</h4><div id="p0160" role="paragraph">Why did gene presence and absence alone hold such strong predictive power for metabolite dynamics, and why did the regressions select specific genes in the denitrification pathway as informative predictors? We propose that by characterizing metabolic phenotypes in terms of rates and yields, we captured the salient features of the metabolic process for each strain and that this enabled the regressions to succeed by exploiting the conserved correlations between the presence of specific genes and metabolic phenotypes. In some cases, these correlations appear to be related to the functional roles of specific genes in the pathway. We found that, for some genes, the sign and magnitude of the regression coefficients agree qualitatively with known properties of the associated enzymes. For example, previous comparisons between membrane-bound and periplasmic nitrate reductases (encoded by <i>narG</i> and <i>napA</i>, respectively; <a id="crosref0710" href="#fig2">Figure&nbsp;2</a>B) in multiple bacterial strains showed that the membrane-bound enzyme exhibits higher nitrate reduction activity <i>in&nbsp;vitro</i> than the periplasmic enzyme (<span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Stewart et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Stewart, V. ∙ Lu, Y. ∙ Darwin, A.J.</div><div><strong>Periplasmic nitrate reductase (NapABC enzyme) supports anaerobic respiration by <i>Escherichia coli</i> K-12</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2002; <strong>184</strong>:1314-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.184.5.1314-1323.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0036182372" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11844760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.184.5.1314-1323.2002&amp;pmid=11844760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref111" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111" aria-controls="sref111">Van Alst et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref111"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Van Alst, N.E. ∙ Sherrill, L.A. ∙ Iglewski, B.H. ...</div><div><strong>Compensatory periplasmic nitrate reductase activity supports anaerobic growth of <i>Pseudomonas aeruginosa</i> PAO1 in the absence of membrane nitrate reductase</strong></div><div><em>Can. J. Microbiol.</em> 2009; <strong>55</strong>:1133-1144</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1139/W09-065" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-70350228801" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19935885/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1139%2FW09-065&amp;pmid=19935885" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Ikeda et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Ikeda, E. ∙ Andou, S. ∙ Iwama, U. ...</div><div><strong>Physiological roles of two dissimilatory nitrate reductases in the deep-sea denitrifier <i>Pseudomonas</i> sp. strain MT-1</strong></div><div><em>Biosci. Biotechnol. Biochem.</em> 2009; <strong>73</strong>:896-900</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1271/bbb.80833" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-67149086712" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19352028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1271%2Fbbb.80833&amp;pmid=19352028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Bell et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bell, L.C. ∙ Richardson, D.J. ∙ Ferguson, S.J.</div><div><strong>Periplasmic and membrane-bound respiratory nitrate reductases in <i>Thiosphaera pantotropha</i>: the periplasmic enzyme catalyzes the first step in aerobic denitrification</strong></div><div><em>FEBS Lett.</em> 1990; <strong>265</strong>:85-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0014-5793(90)80889-Q" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0025333241" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2365057/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0014-5793%2890%2980889-Q&amp;pmid=2365057" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref114" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114" aria-controls="sref114">Warnecke-Eberz and Friedrich, 1993</a><div class="dropBlock__holder" data-db-target-of="sref114"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Warnecke-Eberz, U. ∙ Friedrich, B.</div><div><strong>Three nitrate reductase activities in <i>Alcaligenes eutrophus</i></strong></div><div><em>Arch. Microbiol.</em> 1993; <strong>159</strong>:405-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00288585" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0027450107" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00288585" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This accords with the large positive coefficient for <i>narG</i> in the nitrate reduction rate regression (<a id="crosref0740" href="#fig4">Figure&nbsp;4</a>G). Similarly, in the nitrite reduction rate regression, we observed a large positive coefficient for the gene encoding the copper-based nitrite reductase (<i>nirK</i>) (<a id="crosref0745" href="#fig4">Figure&nbsp;4</a>I), which in previous studies, showed markedly higher activity <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Abraham et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Abraham, Z.H.L. ∙ Lowe, D.J. ∙ Smith, B.E.</div><div><strong>Purification and characterization of the dissimilatory nitrite reductase from <i>Alcaligenes xylosoxidans subsp</i>. xylosoxidans (N.C.I.M.B. 11015): evidence for the presence of both type 1 and type 2 copper centres</strong></div><div><em>Biochem. J.</em> 1993; <strong>295</strong>:587-593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1042/bj2950587" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0027437592" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8240262/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1042%2Fbj2950587&amp;pmid=8240262" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Masuko et&nbsp;al., 1984</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Masuko, M. ∙ Iwasaki, H. ∙ Sakurai, T. ...</div><div><strong>Characterization of nitrite reductase from a denitrifier, <i>Alcaligenes</i> sp. NCIB 11015. A novel copper protein</strong></div><div><em>J.&nbsp;Biochem.</em> 1984; <strong>96</strong>:447-454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oxfordjournals.jbchem.a134856" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0021478113" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6501252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foxfordjournals.jbchem.a134856&amp;pmid=6501252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Iwasaki and Matsubara, 1972</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Iwasaki, H. ∙ Matsubara, T.</div><div><strong>A nitrite reductase from <i>Achromobacter cyclastes</i></strong></div><div><em>J.&nbsp;Biochem.</em> 1972; <strong>71</strong>:645-652</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5042456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=5042456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">Liu et&nbsp;al., 1986</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Liu, M.Y. ∙ Liu, M.C. ∙ Payne, W.J. ...</div><div><strong>Properties and electron transfer specificity of copper proteins from the denitrifier <i>“Achromobacter cycloclastes”</i></strong></div><div><em>J.&nbsp;Bacteriol.</em> 1986; <strong>166</strong>:604-608</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.166.2.604-608.1986" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3700338/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.166.2.604-608.1986&amp;pmid=3700338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Kakutani et&nbsp;al., 1981</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kakutani, T. ∙ Watanabe, H. ∙ Arima, K. ...</div><div><strong>Purification and properties of a copper-containing nitrite reductase from a denitrifying bacterium, <i>Alcaligenes faecalis</i> strain S-6</strong></div><div><em>J.&nbsp;Biochem.</em> 1981; <strong>89</strong>:453-461</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oxfordjournals.jbchem.a133220" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0019457194" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7240122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foxfordjournals.jbchem.a133220&amp;pmid=7240122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Kukimoto et&nbsp;al., 1994</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Kukimoto, M. ∙ Nishiyama, M. ∙ Murphy, M.E. ...</div><div><strong>X-ray structure and site-directed mutagenesis of a nitrite reductase from <i>Alcaligenes faecalis</i> S-6: roles of two copper atoms in nitrite reduction</strong></div><div><em>Biochemistry.</em> 1994; <strong>33</strong>:5246-5252</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00183a030" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8172899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00183a030&amp;pmid=8172899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Michalski and Nicholas, 1985</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Michalski, W.P. ∙ Nicholas, D.</div><div><strong>Molecular characterization of a copper-containing nitrite reductase from <i>Rhodopseudomonas sphaeroides</i> forma sp. <i>denitrificans</i></strong></div><div><em>Biochim. Biophys. Acta Protein Struct. Mol. Enzymol.</em> 1985; <strong>828</strong>:130-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0167-4838(85)90048-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0021989793" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0167-4838%2885%2990048-2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Sawada et&nbsp;al., 1978</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Sawada, E. ∙ Toshio, S. ∙ Kitamura, H.</div><div><strong>Purification and properties of a dissimilatory nitrite reductase of a denitrifying phototrophic bacterium</strong></div><div><em>Plant Cell Physiol.</em> 1978; <strong>19</strong>:1339-1351</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.SawadaS.ToshioH.KitamuraPurification+and+properties+of+a+dissimilatory+nitrite+reductase+of+a+denitrifying+phototrophic+bacteriumPlant+Cell+Physiol.19197813391351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Denariaz et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Denariaz, G. ∙ Payne, W.J. ∙ LeGall, J.</div><div><strong>The denitrifying nitrite reductase of <i>Bacillus halodenitrificans</i></strong></div><div><em>Biochim. Biophys. Acta Bioenerg.</em> 1991; <strong>1056</strong>:225-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0005-2728(05)80053-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0026089232" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0005-2728%2805%2980053-2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Glockner et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Glockner, A.B. ∙ Jüngst, A. ∙ Zumft, W.G.</div><div><strong>Copper-containing nitrite reductase from <i>Pseudomonas aureofaciens</i> is functional in a mutationally cytochrome cd1-free background (NirS-) of <i>Pseudomonas stutzeri</i></strong></div><div><em>Arch. Microbiol.</em> 1993; <strong>160</strong>:18-26</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8352648/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=8352648" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) compared with the alternate nitrite reductase enzyme encoded by <i>nirS</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref117-6" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-6" aria-controls="sref117-6">Zumft, 1997</a><div class="dropBlock__holder" data-db-target-of="sref117-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Zumft, W.G.</div><div><strong>Cell biology and molecular basis of denitrification</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 1997; <strong>61</strong>:533-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/.61.4.533-616.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2F.61.4.533-616.1997&amp;pmid=9409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref109" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109" aria-controls="sref109">Timkovich et&nbsp;al., 1982</a><div class="dropBlock__holder" data-db-target-of="sref109"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Timkovich, R. ∙ Dhesi, R. ∙ Martinkus, K.J. ...</div><div><strong>Isolation of <i>Paracoccus denitrificans</i> cytochrome cd1: comparative kinetics with other nitrite reductases</strong></div><div><em>Arch. Biochem. Biophys.</em> 1982; <strong>215</strong>:47-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0003-9861(82)90277-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0020396614" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6284044/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0003-9861%2882%2990277-6&amp;pmid=6284044" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Gordon et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Gordon, E.H.J. ∙ Sjögren, T. ∙ Löfqvist, M. ...</div><div><strong>Structure and kinetic properties of <i>Paracoccus pantotrophus</i> cytochrome cd1 nitrite reductase with the d1 heme active site ligand tyrosine 25 replaced by serine</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2003; <strong>278</strong>:11773-11781</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1074%2Fjbc.M211886200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1074%2Fjbc.M211886200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0038146890" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12556530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M211886200&amp;pmid=12556530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Besson et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Besson, S. ∙ Carneiro, C. ∙ Moura, J.J. ...</div><div><strong>A cytochrome cd1-type nitrite reductase isolated from the marine denitrifier <i>Pseudomonas nautica</i> 617: purification and characterization</strong></div><div><em>Anaerobe.</em> 1995; <strong>1</strong>:219-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/anae.1995.1021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0001191264" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16887530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fanae.1995.1021&amp;pmid=16887530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">Sawhney and Nicholas, 1978</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Sawhney, V. ∙ Nicholas, D.J.D.</div><div><strong>Sulphide-linked nitrite reductase from <i>Thiobacillus denitrificans</i> with cytochrome oxidase activity: purification and properties</strong></div><div><em>J.&nbsp;Gen. Microbiol.</em> 1978; <strong>106</strong>:119-128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1099/00221287-106-1-119" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1099%2F00221287-106-1-119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Further, our regression coefficients showed larger contributions of <i>narG</i> versus <i>napA</i> to yield on nitrate (<a id="crosref0825" href="#fig4">Figure&nbsp;4</a>H) and, similarly, <i>cnor</i> versus <i>qnor</i> to yield on nitrite (<a id="crosref0830" href="#fig4">Figure&nbsp;4</a>J). Both of these observations are consistent with the fact that the genes encoded by <i>narG</i> and <i>cnor</i> contribute more to the proton motive force (and, therefore, to ATP generation) than their alternatives (<i>napA</i> and <i>qnor</i>, respectively) do (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Ferguson and Richardson, 2004</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Ferguson, S.J. ∙ Richardson, D.J.</div><div><strong>The enzymes and bioenergetics of bacterial nitrate, nitrite, nitric oxide and nitrous oxide respiration</strong></div><div><div>Zannoni, D. (Editor)</div></div><div><strong>Respiration in Archaea and Bacteria: Diversity of Prokaryotic Respiratory Systems</strong></div><div>Springer, 2004; 169-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4020-3163-2_8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4020-3163-2_8" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, the transporter encoded by the gene <i>narK1K2</i> (<a id="crosref0840" href="#fig2">Figure&nbsp;2</a>B) is a fusion of the nitrate/<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper H Superscript plus" tabindex="0" ctxtmenu_counter="135" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal upper H Superscript plus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c48">H</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> symporter NarK1 and the nitrate/nitrite antiporter NarK2, the latter of which is crucial for exchanging nitrate and nitrite between the cytoplasm and periplasm during denitrification when the membrane-bound nitrate reductase is utilized. In <i>Paracoccus denitrificans</i>, this fusion has been shown to have substantially higher affinity for nitrate than NarK2 alone, resulting in higher growth rates under denitrifying conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Goddard et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Goddard, A.D. ∙ Moir, J.W.B. ∙ Richardson, D.J. ...</div><div><strong>Interdependence of two NarK domains in a fused nitrate/nitrite transporter</strong></div><div><em>Mol. Microbiol.</em> 2008; <strong>70</strong>:667-681</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2008.06436.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-53849131672" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18823285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2008.06436.x&amp;pmid=18823285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This agrees with what we found in the nitrate and nitrite reduction rate regressions, in which we observed large positive contributions of <i>narK1K2</i> (<a id="crosref0850" href="#fig4">Figures 4</a>G and 4I).</div><div id="p0165" role="paragraph">Taken together, these observations suggest that the regressions exploited conserved correlations between gene presence and metabolic traits that reflect known mechanistic properties of the denitrification pathway. It is important to note, however, that for many nonzero coefficients in our regressions, notably those corresponding to regulators, there is no clear mechanistic interpretation. Further, given that our regressions were trained on genomes of wild isolates and not on phenotypes of deletion mutants, we do not expect that the regression can be reliably used to predict mutant phenotypes. Instead, we expect that the regressions exploited the tendency for strains possessing specific genes to have specific traits on average (e.g., strains with NarG tend to have high <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Baseline comma gamma Subscript upper A Baseline" tabindex="0" ctxtmenu_counter="136" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 (2 0 1) 3 (6 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="2,3,6" data-semantic-content="3" data-semantic-owns="2 3 6" data-semantic-speech="r Subscript upper A Baseline comma gamma Subscript upper A Baseline"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="7" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="7" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>). These correlations between the presence of specific genes and metabolic traits qualitatively agree with the mechanistic details of some genes in the pathway, but we do not expect the regression coefficients to make causal predictions about the loss of a single gene.</div></section><section id="sec-2-3-4"><h4>Implications of a statistical approach to mapping genomic structure to metabolic traits</h4><div id="p0170" role="paragraph">Our statistical approach took two important steps toward mapping genomic structure to metabolic dynamics at the single-strain level. First, by making quantitative measurements in the laboratory, we removed the confounding environmental factors present in sequencing and metabolomic studies of natural communities to reveal that gene content has a conserved impact on dynamic metabolic phenotypes. Second, our results suggest that a statistical approach could be used to discover the key genomic features of pathways that determine other metabolic phenotypes, complementing direct genetic investigation of model organisms (<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Nichols et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Nichols, R.J. ∙ Sen, S. ∙ Choo, Y.J. ...</div><div><strong>Phenotypic landscape of a bacterial cell</strong></div><div><em>Cell.</em> 2011; <strong>144</strong>:143-156</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cell.2010.11.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cell.2010.11.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-78650475398" target="_blank" aria-label="Open Scopus in new tab">Scopus (436)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21185072/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2010.11.052&amp;pmid=21185072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, our predictions of metabolic phenotypes from genomes apply across a range of conditions and generalized well out-of-sample, suggesting that this approach can predict metabolite dynamics in settings for strains where only genome sequence data are available. These insights were made possible by parameterizing metabolic phenotypes across a genomically diverse strain library of non-model organisms, thereby exploiting genomic variation to learn the mapping from genotype to metabolic phenotypes.</div></section></section><section id="sec-2-4"><h3>Predicting metabolite dynamics in communities</h3><div id="p0175" role="paragraph">Predicting community metabolite dynamics from genomic structure requires mapping single-strain phenotypes to collective behavior. Previous studies have found some success in predicting metabolite dynamics in consortia from knowledge of the monoculture metabolite consumption dynamics (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Erbilgin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Erbilgin, O. ∙ Bowen, B.P. ∙ Kosina, S.M. ...</div><div><strong>Dynamic substrate preferences predict metabolic properties of a simple microbial consortium</strong></div><div><em>BMC Bioinformatics.</em> 2017; <strong>18</strong>:57</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-017-1478-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85011072524" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28114881/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-017-1478-2&amp;pmid=28114881" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Medlock et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Medlock, G.L. ∙ Carey, M.A. ∙ McDuffie, D.G. ...</div><div><strong>Inferring metabolic mechanisms of interaction within a defined gut microbiota</strong></div><div><em>Cell Syst.</em> 2018; <strong>7</strong>:245-257.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2018.08.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2018.08.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85054064942" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30195437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2018.08.003&amp;pmid=30195437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These approaches used simple assumptions, such as a fixed rate of metabolite production or consumption for each strain (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Medlock et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Medlock, G.L. ∙ Carey, M.A. ∙ McDuffie, D.G. ...</div><div><strong>Inferring metabolic mechanisms of interaction within a defined gut microbiota</strong></div><div><em>Cell Syst.</em> 2018; <strong>7</strong>:245-257.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2018.08.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2018.08.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85054064942" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30195437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2018.08.003&amp;pmid=30195437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), rather than a dynamic model of metabolites. To predict community metabolite dynamics, we used the consumer-resource modeling formalism that describes metabolite dynamics for each strain to make quantitative predictions for metabolite dynamics in communities of multiple strains (Step 3, <a id="crosref0875" href="#fig1">Figure&nbsp;1</a>). Since the consumer-resource parameters were sparsely encoded by the genomes of each strain (<a id="crosref0880" href="#fig4">Figure&nbsp;4</a>), predicting community metabolite dynamics from the consumer-resource model would provide a mapping from gene content to community metabolism.</div><div id="p0180" role="paragraph">Therefore, we extended to our modeling formalism to <i>N</i>-strain communities by adding the rate contributions of each strain to the dynamics of nitrate and nitrite (<a id="crosref0885" href="#fig6">Figure&nbsp;6</a>B; <a id="crosref0890" href="#fd10">Equation 10</a>; <a id="crosref0895" href="#sec-5">STAR Methods</a>). This “additive” model assumes that strains interact only via cross-feeding and resource competition for electron acceptors. This model also assumes that the rates and yields on nitrate and nitrite for strains in pair culture are the same as in monoculture. As a result, the model provides predictions for <i>N</i>-strain community metabolite dynamics given the consumer-resource model parameters for individual strains without any free parameters.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/a35486da-0a48-4d55-adb7-1055a38dcb64/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.12.036/asset/a5d2fdd7-3b7f-4ed0-8030-4a4bcd0c5b0f/main.assets/gr6.jpg" height="538" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/a35486da-0a48-4d55-adb7-1055a38dcb64/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Metabolite dynamics in two-strain communities are predictable from monocultures</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0100" role="paragraph">(A) Examples of pair culture dynamics for all combinations of the three denitrification phenotypes (Nar/Nir, purple; Nar, blue; Nir; red). The first two columns show metabolite dynamics for each of two strains cultured individually. The third column shows the metabolite dynamics for pair cultures of the two strains (points) with zero-free-parameter predictions using the consumer-resource model (curves, see model in B). All cultures were performed in SDM, and predictions were based on measured monoculture consumer-resource parameters in SDM, not those inferred from genomes. Errors in pair culture predictions are shown in each panel in the third column as quantified by the normalized root-mean-square error (NRMSE). For pair cultures, we defined <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="NRMSE Subscript i j Baseline equals RMSE Subscript i j Baseline divided by left parenthesis left parenthesis RMSE Subscript i Superscript 2 Baseline plus RMSE Subscript j Superscript 2 Baseline right parenthesis divided by 2 right parenthesis Superscript 1 divided by 2" tabindex="0" ctxtmenu_counter="137" style="font-size: 130%;"><mjx-math data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="41" data-semantic-children="5,40" data-semantic-content="6" data-semantic-owns="5 6 40" data-semantic-structure="(41 (5 0 (4 1 3 2)) 6 (40 (12 7 (11 8 10 9)) 13 (39 (34 14 (33 (29 15 (27 (20 (19 16 17) 18) 21 (26 (25 22 23) 24)) 28) 30 31) 32) (38 35 36 37))))" data-semantic-speech="NRMSE Subscript i j Baseline equals RMSE Subscript i j Baseline divided by left parenthesis left parenthesis RMSE Subscript i Superscript 2 Baseline plus RMSE Subscript j Superscript 2 Baseline right parenthesis divided by 2 right parenthesis Superscript 1 divided by 2" breakable="true" class="MM-N" aria-hidden="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="41" data-semantic-owns="0 4"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c4E" noic="true" style="padding-top: 0.705em;">N</mjx-c><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.705em;">R</mjx-c><mjx-c class="mjx-c4D" noic="true" style="padding-top: 0.705em;">M</mjx-c><mjx-c class="mjx-c53" noic="true" style="padding-top: 0.705em;">S</mjx-c><mjx-c class="mjx-c45" style="padding-top: 0.705em;">E</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="6" data-semantic-parent="41" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="40" data-semantic-children="12,39" data-semantic-content="13" data-semantic-parent="41" data-semantic-owns="12 13 39"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="12" data-semantic-children="7,11" data-semantic-parent="40" data-semantic-owns="7 11"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-parent="12"><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.705em;">R</mjx-c><mjx-c class="mjx-c4D" noic="true" style="padding-top: 0.705em;">M</mjx-c><mjx-c class="mjx-c53" noic="true" style="padding-top: 0.705em;">S</mjx-c><mjx-c class="mjx-c45" style="padding-top: 0.705em;">E</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="11" data-semantic-children="8,9" data-semantic-content="10" data-semantic-parent="12" data-semantic-owns="8 10 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="13" data-semantic-parent="40" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="39" data-semantic-children="34,38" data-semantic-parent="40" data-semantic-owns="34 38"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="34" data-semantic-children="33" data-semantic-content="14,32" data-semantic-parent="39" data-semantic-owns="14 33 32"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="14" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="33" data-semantic-children="29,31" data-semantic-content="30" data-semantic-parent="34" data-semantic-owns="29 30 31"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="29" data-semantic-children="27" data-semantic-content="15,28" data-semantic-parent="33" data-semantic-owns="15 27 28"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="15" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="27" data-semantic-children="20,26" data-semantic-content="21" data-semantic-parent="29" data-semantic-owns="20 21 26"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="unknown" data-semantic-id="20" data-semantic-children="16,17,18" data-semantic-parent="27" data-semantic-collapsed="(20 (19 16 17) 18)" data-semantic-owns="16 17 18"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="16" data-semantic-parent="20"><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.705em;">R</mjx-c><mjx-c class="mjx-c4D" noic="true" style="padding-top: 0.705em;">M</mjx-c><mjx-c class="mjx-c53" noic="true" style="padding-top: 0.705em;">S</mjx-c><mjx-c class="mjx-c45" style="padding-top: 0.705em;">E</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.212em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="20"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="21" data-semantic-parent="27" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msubsup space="3" data-semantic-type="subsup" data-semantic-role="unknown" data-semantic-id="26" data-semantic-children="22,23,24" data-semantic-parent="27" data-semantic-collapsed="(26 (25 22 23) 24)" data-semantic-owns="22 23 24"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-parent="26"><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.705em;">R</mjx-c><mjx-c class="mjx-c4D" noic="true" style="padding-top: 0.705em;">M</mjx-c><mjx-c class="mjx-c53" noic="true" style="padding-top: 0.705em;">S</mjx-c><mjx-c class="mjx-c45" style="padding-top: 0.705em;">E</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="26"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.212em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="26"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="28" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="30" data-semantic-parent="33" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="33"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="32" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.63em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="38" data-semantic-children="35,37" data-semantic-content="36" data-semantic-parent="39" data-semantic-owns="35 36 37"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="38"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="36" data-semantic-parent="38" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="38"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper R normal upper M normal upper S normal upper E Subscript i j" tabindex="0" ctxtmenu_counter="138" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 0 9 1 10 2 11 (8 3 (7 4 6 5)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="0,1,2,8" data-semantic-content="9,10,11" data-semantic-owns="0 9 1 10 2 11 8" data-semantic-speech="normal upper R normal upper M normal upper S normal upper E Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="12"><mjx-c class="mjx-c52">R</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="12"><mjx-c class="mjx-c4D">M</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="12"><mjx-c class="mjx-c53">S</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="12" data-semantic-owns="3 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c45">E</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the root-mean-square error between model predictions and observed metabolite concentrations of strains <i>i</i> and <i>j</i> in pair culture, and RMSE<sub><i>i</i></sub> and RMSE<sub><i>j</i></sub> are the RMSEs of strains <i>i</i> and <i>j</i> in monoculture. NRMSE in the range 0–2 indicates errors in 2-strain communities that are within 2-fold of fits associated with their constituent monocultures.</div><div id="fspara0105" role="paragraph">(B) An <i>N</i>-strain consumer-resource model (based on the model in <a id="crosref2095" href="#fig3">Figure&nbsp;3</a>B) was used to predict pair culture metabolite dynamics <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper N equals 2 right parenthesis" tabindex="0" ctxtmenu_counter="139" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 (4 1 2 3) 5)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="0,5" data-semantic-owns="0 4 5" data-semantic-speech="left parenthesis upper N equals 2 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. <i>A</i> and <i>I</i> are nitrate and nitrite concentrations, respectively. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="x Subscript i" tabindex="0" ctxtmenu_counter="140" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="x Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> denotes the biomass density of strain <i>i</i> with parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Superscript i" tabindex="0" ctxtmenu_counter="141" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="r Subscript upper A Superscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A Superscript i" tabindex="0" ctxtmenu_counter="142" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="gamma Subscript upper A Superscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.108em;"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Superscript i" tabindex="0" ctxtmenu_counter="143" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="r Subscript upper I Superscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I Superscript i" tabindex="0" ctxtmenu_counter="144" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="gamma Subscript upper I Superscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.108em;"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, which were determined from monoculture experiments (<a id="crosref2100" href="#fig3">Figure&nbsp;3</a>C). The <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript italic asterisk" tabindex="0" ctxtmenu_counter="145" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript italic asterisk"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> values were fixed at 0.01 mM for all strains.</div><div id="fspara0110" role="paragraph">(C) A matrix of NRMSE values quantifying the quality of model predictions for all pairs of 12 strains: 4 Nar/Nir, 4 Nar, and 4 Nir. NRMSE values are shown for communities cultured in SDM with nitrate initially supplied, with the exception of Nir&nbsp;+ Nir pairs for which nitrite was initially supplied. Only Nar&nbsp;+ Nir communities are poorly predicted by the consumer-resource model (permutation test, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than 1 times 10 Superscript negative 5" tabindex="0" ctxtmenu_counter="146" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (9 2 3 (8 4 (7 5 6))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="10" data-semantic-children="0,9" data-semantic-content="1" data-semantic-owns="0 1 9" data-semantic-speech="p less than 1 times 10 Superscript negative 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-children="2,8" data-semantic-content="3" data-semantic-parent="10" data-semantic-owns="2 3 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="8" data-semantic-children="4,7" data-semantic-parent="9" data-semantic-owns="4 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-children="6" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="5 6"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, <a id="crosref2105" href="#figs3">Figures S3</a>C and S3D). See also <a id="crosrefs0010" href="#figs3">Figures S3–S6</a>.</div></div></div></div></figcaption></figure></div><div id="p0185" role="paragraph">To evaluate the ability of our consumer-resource model to make predictions of metabolite dynamics in communities, we used measured consumer-resource parameter values (<a id="crosref0900" href="#fig3">Figure&nbsp;3</a>C) and not the values predicted by gene presence and absence (<a id="crosref0905" href="#fig4">Figures 4</a>C–4F). This allowed us to disambiguate the errors associated with the failure of the model to predict metabolite dynamics from the errors associated with predicting phenotypic parameters from genomes. However, as we subsequently discuss, using consumer-resource model parameters predicted from genomes has, at most, a modest impact on errors in our predictions of community metabolite dynamics.</div><section id="sec-2-4-1"><h4>Predicting metabolite dynamics in two-strain communities</h4><div id="p0190" role="paragraph">We tested the ability of this approach to predict metabolite dynamics in all pair combinations of 12 strains from our library (4 Nar/Nir, 4 Nar, and 4 Nir). We assembled communities in 96-well plates containing SDM, supplying either nitrate or nitrite initially in two separate experimental conditions and then sampled over a 64-h period to measure concentrations of nitrate and nitrite (<a id="crosref0910" href="#sec-5">STAR Methods</a>). Remarkably, we found that the additive model accurately predicted the metabolic dynamics for most 2-strain communities (<a id="crosref0915" href="#fig6">Figures 6</a>, <a id="crosref0920" href="#figs3">S3</a>A, and S3B) using only the measured consumer-resource parameters for individual strains. Specifically, the third column of <a id="crosref0925" href="#fig6">Figure&nbsp;6</a>A shows the zero-free-parameter predictions (curves) of denitrification dynamics in 2-strain communities, which agreed well with measurements (points). The 2-strain community predictions include non-trivial dynamics, such as a transient increase in nitrite for a Nar/Nir&nbsp;+ Nar community. In addition, we observed that the additive model accurately predicted total endpoint optical densities and community compositions (<a id="crosref0930" href="#figs4">Figure&nbsp;S4</a>; <a id="crosref0935" href="#sec-5">STAR Methods</a>) in most cases, indicating that the model generally captures strain abundance dynamics in communities.</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/25ab0bbd-072a-4426-8d66-df8477c97b9b/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.12.036/asset/44217050-f8dd-41b8-b2bc-63501fa727e6/main.assets/figs3.jpg" height="733" width="506" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/25ab0bbd-072a-4426-8d66-df8477c97b9b/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Accuracy of consumer-resource nitrate/nitrite dynamics predictions for simple communities, related to <a id="crosref2190" href="#fig6">Figures 6</a> and <a id="crosref2195" href="#fig7">7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0225" role="paragraph">(A and B) Normalized RMSE (NRMSE) values comparing measurements and predictions for nitrate and nitrite dynamics in all pair communities of 12 strains. Values are grouped according to the constituent phenotypes in a community, with Nar/Nir strains labeled in purple, Nar strains in blue, and Nir strains in orange. (A) shows NRMSEs for communities initialized with 2&nbsp;mM nitrate (with “Nir&nbsp;+ Nir” communities omitted, since Nir strains do not utilize nitrate as an electron acceptor). (B) shows NRMSEs for communities initialized with 2&nbsp;mM nitrite (with Nar&nbsp;+ Nar communities omitted, since Nar strains do not utilize nitrite as an electron acceptor). All cultures were performed in SDM.</div><div id="fspara0230" role="paragraph">(C and D) Hypothesis testing on means of NRMSEs grouped by constituent phenotypes (e.g., Nar/Nir&nbsp;+ Nar/Nir communities, etc.). Null distributions for group means are generated via permutation of group labels for each pair community (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 times 10 Superscript 5" tabindex="0" ctxtmenu_counter="147" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="1 times 10 Superscript 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> permutations) and are compared with the observed mean NRMSEs in each group (dashed lines). A Bonferroni-corrected threshold for 5% significance over 10 hypothesis tests is 0.005. Note that some groups contain only 6 NRMSE values (e.g., Nar/Nir&nbsp;+ Nar/Nir communities), while others contain 16 NRMSE values (e.g., Nar/Nir&nbsp;+ Nar communities). Panel C shows hypothesis testing on the NRMSE values in (A) (communities initialized with nitrate), while (D) shows inference on the values in (B) (communities initialized with nitrite).</div><div id="fspara0235" role="paragraph">(E) Distributions of NRMSE values comparing measurements and predictions for nitrate and nitrite dynamics in 3–5-strain communities (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N equals 108" tabindex="0" ctxtmenu_counter="148" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper N equals 108"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>). Communities were cultured in SDM and separately initialized with 2&nbsp;mM nitrate or 2&nbsp;mM nitrite. Points in yellow indicate values for communities that contain both a Nar strain and a Nir strain.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/d78a9d0e-a047-4c2c-a5b6-9d0f2ef50ade/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.12.036/asset/89f28627-edfa-4c11-8721-38a89af9a351/main.assets/figs4.jpg" height="894" width="658" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/d78a9d0e-a047-4c2c-a5b6-9d0f2ef50ade/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Accuracy of consumer-resource endpoint biomass density and composition predictions, related to <a id="crosref2200" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0245" role="paragraph">(A) Comparisons between observed endpoint optical densities and values predicted by the additive consumer-resource model (<a id="crosref2205" href="#fd10">Equation 10</a>; <a id="crosref2210" href="#sec-5">STAR Methods</a>), where the latter was obtained by summing the endpoint biomass densities of each strain in the community. Comparisons are shown for monoculture controls, pair cultures, and 3–5-strain communities, in 2&nbsp;mM nitrate and nitrite media conditions separately. All cultures were performed in SDM. Initial optical densities are indicated (red cross), and coefficients of determination <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R squared" tabindex="0" ctxtmenu_counter="149" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> and root-mean-square errors (RMSE) for observed versus predicted values are shown.</div><div id="fspara0250" role="paragraph">(B) Comparisons between observed relative biomass densities and relative biomass densities predicted by the additive consumer-resource model (<a id="crosref2215" href="#fd10">Equation 10</a>; <a id="crosref2220" href="#sec-5">STAR Methods</a>). Root-mean-square error for observed versus predicted relative abundances across all pairs is 0.16. Endpoint community relative abundances were measured for all pair cultures of the six strains <i>Achromobacter</i> sp. ACM01 (Nar/Nir), <i>Ensifer</i> sp. ENS09 (Nar/Nir), <i>Paracoccus</i> sp. PAR01 (Nar), <i>Pantoea</i> sp. PNT03 (Nar), <i>Agrobacterium</i> sp. AGB01 (Nir), and <i>Pseudomonas</i> sp. PDM13 (Nir). All cultures were performed in SDM initiated with 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="150" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, with the exception of the pair culture of the two Nir strains AGB01 and PDM13, which was initiated with 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="151" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>. Relative abundances of 16S rRNA amplicons were measured using CASEU (<span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Cermak et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cermak, N. ∙ Datta, M.S. ∙ Conwill, A.</div><div><strong>Rapid, inexpensive measurement of synthetic bacterial community composition by Sanger sequencing of amplicon mixtures</strong></div><div><em>iScience.</em> 2020; <strong>23</strong>:100915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85080040522" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32114382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2020.100915&amp;pmid=32114382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref2230" href="#sec-5">STAR Methods</a>). To make comparisons in terms of relative biomass densities, conversion factors from 16S amplicon relative abundance to relative biomass density were computed using mixtures of monocultures (<a id="crosref2235" href="#fd11">Equation 11</a>; <a id="crosref2240" href="#sec-5">STAR Methods</a>).</div></div></div></div></figcaption></figure></div><div id="p0195" role="paragraph">We quantified the quality of the additive model predictions for metabolite dynamics by computing a normalized root-mean-square error (NRMSE; see caption of <a id="crosref0940" href="#fig6">Figure&nbsp;6</a>; <a id="crosref0945" href="#fd12">Equation 12</a>; <a id="crosref0950" href="#sec-5">STAR Methods</a>). We found that most 2-strain communities have NRMSE between 0 and 2, indicating that our model successfully predicted metabolite dynamics given only the measured consumer-resource parameters for each strain. Predictions of metabolite dynamics in pair cultures were also accurate when using consumer-resource parameters predicted from genomes via regression (<a id="crosref0955" href="#figs5">Figures S5</a>A and S5B; <a id="crosref0960" href="#sec-5">STAR Methods</a>). Further, the success or failure of the model predictions depended on the phenotypes of the strains present. The model successfully predicted 2-strain metabolite dynamics for most combinations of phenotypes (e.g., Nar/Nir&nbsp;+ Nar or Nar&nbsp;+ Nar) but failed only in the case where Nar strains were cultured with Nir strains (<a id="crosref0965" href="#fig6">Figures 6</a>A, 6C, <a id="crosref0970" href="#figs3">S3</a>C, and S3D). The failure of our model predictions in Nar&nbsp;+ Nir communities followed the common pattern that the rate of nitrate reduction was slower than expected (bottom row, <a id="crosref0975" href="#fig6">Figures 6</a>A and <a id="crosref0980" href="#figs6">S6</a>). We speculate that this failure of the model to predict metabolite dynamics in Nar&nbsp;+ Nir communities was caused by excretion of nitric oxide by the Nir strain. Nitric oxide can be cytotoxic (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Braun and Zumft, 1991</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Braun, C. ∙ Zumft, W.G.</div><div><strong>Marker exchange of the structural genes for nitric oxide reductase blocks the denitrification pathway of <i>Pseudomonas stutzeri</i> at nitric oxide</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1991; <strong>266</strong>:22785-22788</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2FS0021-9258%2818%2954420-3&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2FS0021-9258%2818%2954420-3&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1744072/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2954420-3&amp;pmid=1744072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which may explain slower rates of nitrate reduction for Nar strains. For further exploration of this phenomenon, see the <a id="crosref0990" href="#sec-3">discussion</a> section.</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/821d9bed-d80d-4499-b338-fd18fa818f73/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.12.036/asset/472f6efa-b906-4589-98b1-25bf23994ec8/main.assets/figs5.jpg" height="508" width="506" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/821d9bed-d80d-4499-b338-fd18fa818f73/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Accuracy of consumer-resource nitrate/nitrite dynamics predictions for 2–5-strain communities using genomic predictions of phenotypes, related to <a id="crosref2245" href="#fig6">Figures 6</a> and <a id="crosref2250" href="#fig7">7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0260" role="paragraph">(A–C) NRMSE values comparing measurements and predictions for nitrate and nitrite dynamics in (A and B) pair cultures and (C) 3–5-strain communities <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper N equals 108 right parenthesis" tabindex="0" ctxtmenu_counter="152" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 (4 1 2 3) 5)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="0,5" data-semantic-owns="0 4 5" data-semantic-speech="left parenthesis upper N equals 108 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Predictions were made using regressions in <a id="crosref2255" href="#fig4">Figure&nbsp;4</a> to predict single-strain consumer-resource parameters from gene presence/absence, which were, in turn, used by the consumer-resource model to predict metabolite dynamics. This is in contrast to <a id="crosref2260" href="#fig6">Figures 6</a>, <a id="crosref2265" href="#fig7">7</a>, and <a id="crosref2270" href="#figs3">S3</a>, where predictions for metabolite dynamics were made using directly measured values of consumer-resource parameters. Communities were cultured in SDM and were separately initialized with 2&nbsp;mM nitrate and 2&nbsp;mM nitrite. Points in yellow indicate values for communities that contain both a Nar strain and a Nir strain.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/cbf9ff33-a523-4feb-8001-01fd9c071471/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.12.036/asset/fee1c05c-cad1-418a-ba56-62f9d36a9daa/main.assets/figs6.jpg" height="614" width="761" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/cbf9ff33-a523-4feb-8001-01fd9c071471/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Nitrate/nitrite dynamics for Nar&nbsp;+ Nir pair cultures, related to <a id="crosref2275" href="#fig6">Figures 6</a> and <a id="crosref2280" href="#fig7">7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0270" role="paragraph">Monoculture controls are shown for Nar strains (left column) and Nir strains (bottom row). Points show measured concentrations of nitrate and nitrite, and curves show predictions of <a id="crosref2285" href="#fd10">Equation 10</a> (<a id="crosref2290" href="#sec-5">STAR Methods</a>). NRMSE values for pair cultures are shown. 2–3 experimental replicates are used for each combination of strains. All cultures were performed in SDM.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-4-2"><h4>Predicting metabolite dynamics in larger communities</h4><div id="p0200" role="paragraph">Next, we asked whether dynamical metabolic phenotypes measured from monocultures could be used to predict metabolite dynamics in 3–5-strain communities. We applied the additive model to predicting the nitrate and nitrite dynamics in 81 combinations of 3 strains, 21 combinations of 4 strains, and 6 combinations of 5 strains from the 12-strain subset (<a id="crosref0995" href="#sec-5">STAR Methods</a>). As with pair cultures, 3–5-strain communities were cultured in SDM with either nitrate or nitrite supplied initially in two separate experimental conditions. In communities that did not contain a Nar&nbsp;+ Nir pair (e.g., <a id="crosref1000" href="#fig7">Figure&nbsp;7</a>A), we found that prediction accuracy was high (gray symbols, <a id="crosref1005" href="#fig7">Figures 7</a>B and <a id="crosref1010" href="#figs3">S3</a>E). This again indicated that in most combinations of phenotypes, community dynamics were predictable from the consumer-resource parameters of each strain in the community. However, in communities that contained a Nar&nbsp;+ Nir pair, predictions were relatively poor (yellow symbols, <a id="crosref1015" href="#fig7">Figures 7</a>B and <a id="crosref1020" href="#figs3">S3</a>E), suggesting that interactions between Nar and Nir phenotypes that were not captured in the additive model were again driving low prediction accuracy. Finally, we note that the additional error in community metabolite dynamics predictions associated with predicting phenotypes from genomes was typically modest (median increase in NRMSE <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="almost equals 0 period 5" tabindex="0" ctxtmenu_counter="153" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="almost equals 0 period 5"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,≈"><mjx-c class="mjx-c2248">≈</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>–1.4) for 3–5-strain communities (<a id="crosref1025" href="#figs5">Figure&nbsp;S5</a>C; <a id="crosref1030" href="#sec-5">STAR Methods</a>).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/f9e6a02e-b722-4572-a1ac-e7cf73e13f96/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.12.036/asset/759465c8-aeca-42db-8130-c2890b0b19d9/main.assets/gr7.jpg" height="540" width="505" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/f9e6a02e-b722-4572-a1ac-e7cf73e13f96/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Metabolite dynamics are predictable in 3–5-strain communities</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0120" role="paragraph">The additive consumer-resource model provides predictions for metabolite dynamics in communities of more than two strains, and these predictions were verified experimentally.</div><div id="fspara0125" role="paragraph">(A) Metabolite dynamics for an example 3-strain (Nar/Nir&nbsp;+ Nar/Nir&nbsp;+ Nar) community cultured in SDM. The first three panels show metabolite dynamics for each strain cultured individually, and the fourth panel shows the metabolite dynamics of the 3-strain community. Curves show the prediction of the consumer-resource model (<a id="crosref2110" href="#fig6">Figure&nbsp;6</a>B) using measured values of consumer-resource model parameters in SDM (not values inferred from genomes).</div><div id="fspara0130" role="paragraph">(B) NRMSE (<a id="opteNF9Jc8TcB" href="#fd12">Equation 12</a>; <a id="crosref2300" href="#sec-5">STAR Methods</a>) values quantifying quality of consumer-resource model predictions for 3–5-strain communities cultured in SDM with nitrate. Yellow symbols denote communities that contain a Nar&nbsp;+ Nir pair. Nar&nbsp;+ Nir pair culture dynamics were poorly predicted by the model (<a id="crosref2120" href="#fig6">Figure&nbsp;6</a>C) and resulted in a high NRMSE for communities containing Nar&nbsp;+ Nir pairs (compare yellow and gray symbols). Left and right scatterplots compare predictions from a consumer-resource model using only monoculture data to a coarse-graining approach that describes Nar&nbsp;+ Nir pairs as modules within the community (described in C and D). The coarse-graining approach improves the 3–5-strain community predictions; mean NRMSE (black lines) decreases when Nar&nbsp;+ Nir pair information is used for prediction (<i>t</i>-test, <sup>∗∗∗∗</sup> denotes <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than 10 Superscript negative 4" tabindex="0" ctxtmenu_counter="154" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 1 (6 2 (5 3 4)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="0,6" data-semantic-content="1" data-semantic-owns="0 1 6" data-semantic-speech="p less than 10 Superscript negative 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="6" data-semantic-children="2,5" data-semantic-parent="7" data-semantic-owns="2 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-children="4" data-semantic-content="3" data-semantic-parent="6" data-semantic-owns="3 4"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>).</div><div id="fspara0135" role="paragraph">(C) Metabolite dynamics for an example Nar&nbsp;+ Nir pair cultured in SDM, where curves in the left panel show the prediction of the consumer-resource model using only parameters fit to monocultures, and curves in the right panel show the results of refitting the reduction rates (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="155" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="156" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>) to Nar&nbsp;+ Nir pair culture data but leaving yields (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="157" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="158" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>) fixed to monoculture values.</div><div id="fspara0140" role="paragraph">(D) Metabolite dynamics for a 3-strain community cultured in SDM containing a Nar/Nir strain and the Nar&nbsp;+ Nir pair shown in (C). Curves in the left panel show the prediction of the consumer-resource model using parameters inferred from monoculture experiments for each strain, and curves in the right panel show the prediction when the Nar&nbsp;+ Nir pair is treated as a module, with rate parameters refit from pair culture data (right panel in C). NRMSE decreased due to the coarse-graining of the Nar&nbsp;+ Nir pair. Panels highlighted in beige denote zero-free-parameter predictions. See also <a id="crosref2125" href="#figs3">Figures S3</a> and <a id="crosref2130" href="#mmc1">S5–S7</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-4-3"><h4>Correcting for interactions between Nar and Nir strains</h4><div id="p0205" role="paragraph">To address the impact of interactions between Nar and Nir strains not accounted for by our additive model in 3–5-strain communities, we took a coarse-graining approach. We asked whether the metabolic contributions of Nar&nbsp;+ Nir pairs could be treated as modules within larger communities. To do this, we re-fitted nitrate and nitrite reduction rates (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="159" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="160" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>) to pair culture data (cultured in SDM with nitrate) for each Nar&nbsp;+ Nir pair, leaving yields fixed (<a id="crosref1035" href="#fig7">Figures 7</a>C and <a id="crosref1040" href="#figs7">S7</a>A; <a id="crosref1045" href="#sec-5">STAR Methods</a>). This resulted in effective nitrate and nitrite reduction rates (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper A" tabindex="0" ctxtmenu_counter="161" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (2 0 1) 3)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-owns="2 3" data-semantic-speech="r overtilde Subscript upper A"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="4" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper I" tabindex="0" ctxtmenu_counter="162" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (2 0 1) 3)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-owns="2 3" data-semantic-speech="r overtilde Subscript upper I"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="4" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>) for each Nar&nbsp;+ Nir pair. In every case, we observed that the re-fitted nitrate reduction rates <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper A" tabindex="0" ctxtmenu_counter="163" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (2 0 1) 3)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-owns="2 3" data-semantic-speech="r overtilde Subscript upper A"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="4" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were lower than the monoculture nitrate reduction rates (<a id="crosref1050" href="#figs7">Figure&nbsp;S7</a>B), demonstrating quantitatively that Nar strains were consistently slowed by the presence of Nir strains. This observation is consistent with the hypothesis of excretion of cytotoxic nitric oxide by the Nir strain.</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.036/asset/7c929b40-c7ed-4dc1-96fb-93572f111a7f/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.12.036/asset/2513467f-f526-40d9-a2ab-29e6accf10d4/main.assets/figs7.jpg" height="370" width="658" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/7c929b40-c7ed-4dc1-96fb-93572f111a7f/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Refitting model parameters for Nar&nbsp;+ Nir pair cultures, related to <a id="crosref2295" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0280" role="paragraph">(A) The <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="164" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> parameter for the Nar strain and/or the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="165" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> parameter for the Nir strain were refit to the Nar&nbsp;+ Nir pair culture measurements to obtain new parameter values <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper A" tabindex="0" ctxtmenu_counter="166" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (2 0 1) 3)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-owns="2 3" data-semantic-speech="r overtilde Subscript upper A"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="4" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper I" tabindex="0" ctxtmenu_counter="167" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (2 0 1) 3)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-owns="2 3" data-semantic-speech="r overtilde Subscript upper I"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="4" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, respectively. In some cases (panels with only solid lines), it was only necessary to refit the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="168" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> parameter to obtain a good model fit. In other cases (panels with both solid and dashed lines), better fits (more than 10% smaller NRMSE) could be obtained by refitting both <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="169" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="170" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (solid lines). NRMSE values for refit models are indicated. 2–3 experimental replicates are used for each combination of strains.</div><div id="fspara0285" role="paragraph">(B) Relative changes in <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="171" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="172" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> parameters. Nar strains are coded by color: ACV02 (yellow), PAR01 (green), PDM12 (blue), and PNT03 (red). Nir strains are coded by symbol: AGB01 (circle), PDM13 (square), PDM14 (triangle), and PXM03 (diamond). NRMSE values for pair cultures are shown. All cultures were performed in SDM.</div></div></div></div></figcaption></figure></div><div id="p0210" role="paragraph">We then used the re-fitted rates for Nar&nbsp;+ Nir pairs to make predictions for communities (cultured in SDM with nitrate) that included such pairs (e.g., <a id="crosref1055" href="#fig7">Figure&nbsp;7</a>D). For communities that included multiple Nar&nbsp;+ Nir pairs, we developed a simple averaging rule for determining the effective rates from the rates for each Nar&nbsp;+ Nir pair present (<a id="crosref1060" href="#sec-5">STAR Methods</a>). For example, in a Nar&nbsp;+ Nar&nbsp;+ Nir community, there are two sets of Nar&nbsp;+ Nir pair interactions, with a different effective nitrite reduction rate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper I" tabindex="0" ctxtmenu_counter="173" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (2 0 1) 3)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-owns="2 3" data-semantic-speech="r overtilde Subscript upper I"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="4" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> measured for the Nir strain in its interactions with the two Nar strains. In this example, we would take the mean of these two effective reduction rates as the value used for prediction. We found that the metabolite dynamics in 3–5-strain communities containing Nar&nbsp;+ Nir pairs were quantitatively well predicted by this coarse-graining approach (yellow symbols, <a id="crosref1065" href="#fig7">Figure&nbsp;7</a>B). We concluded that treating Nar&nbsp;+ Nir pairs as effective modules within larger communities recovered the predictive power of the additive consumer-resource model.</div></section></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0215" role="paragraph">Quantifying the metabolic phenotypes of a diverse library of natural isolates using a consumer-resource model allowed us to take a statistical approach to connecting genotypes to dynamical metabolic phenotypes. The outcome was a sparse mapping from gene content to single-strain metabolite dynamics that exploited conserved correlations between metabolic traits and gene presence, some of which reflect the known mechanistic properties of enzymes in the denitrification pathway. The resource-based modeling formalism then permitted quantitative predictions of community-level metabolite dynamics. As a result, the approach yielded a mapping from genomic structure to metabolite dynamics at the community level for denitrifying bacterial communities.</div><div id="p0220" role="paragraph">A key contribution of this study is the demonstration of a quantitative mapping between gene content and metabolic traits for a model metabolic process. One might expect that gene presence and absence is too coarse a genomic feature to predict dynamic metabolic traits and that other genomic features, such as promoter sequences, synteny, or allelic variation, would be necessary to make predictions. We instead found that that the association between gene presence/absence and metabolic traits is strong. This result suggests that selection for specific metabolic traits in bacteria may primarily favor genomes with specific complements of genes (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Cordero et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cordero, O.X. ∙ Ventouras, L.A. ∙ DeLong, E.F. ...</div><div><strong>Public good dynamics drive evolution of iron acquisition strategies in natural bacterioplankton populations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:20059-20064</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1213344109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84870578736" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23169633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1213344109&amp;pmid=23169633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-2" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-2" aria-controls="sref92-2">Sakoparnig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref92-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Sakoparnig, T. ∙ Field, C. ∙ Nimwegen, E. van.</div><div><strong>Whole genome phylogenies reflect the distributions of recombination rates for many bacterial species</strong></div><div><em>eLife.</em> 2021; <strong>10</strong>:e65366</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.65366" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85100215005" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33416498/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.65366&amp;pmid=33416498" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and that more granular details of the genome, such as promoter sequences or allelic variation, are less important.</div><div id="p0225" role="paragraph">At the community level, we found that interactions beyond those described by the additive consumer-resource model are not idiosyncratic but instead exhibit a general pattern (i.e., they occur only when Nar and Nir strains are both present). This suggests that interactions beyond resource competition may exhibit patterns that can be discovered in the laboratory. The fact that community-level metabolite dynamics departed from the additive model in Nar&nbsp;+ Nir communities suggests that such interactions may be more likely to occur when specific metabolic processes, such as facilitation via the exchange of a metabolite, are at work.</div><section id="sec-3-1"><h3>Improving predictions of community metabolism from genomes</h3><div id="p0230" role="paragraph">There are some important caveats that apply to our prediction of single-strain metabolic traits from genomes and community-level metabolism from monocultures. For one, by parameterizing metabolite dynamics using a consumer-resource model, we assumed that the model could approximate the metabolic phenotypes of wild isolates. For most of our library (62/79 strains), this approximation worked well, but in some cases (17/79 strains), the model failed for at least some initial conditions (<a id="crosref1080" href="#figs1">Figures S1</a>J–S1L; <a id="crosref1085" href="#mmc1">Table S5</a>; <a id="crosref1090" href="#sec-5">STAR Methods</a>). These failures may have occurred because the model does not capture phenomena such as the inhibition of reduction rates by reaction products. Going forward, the assumptions of the model could be relaxed by applying methods to learn the appropriate phenotypic parameters directly from the data (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Berman et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Berman, G.J. ∙ Choi, D.M. ∙ Bialek, W. ...</div><div><strong>Mapping the stereotyped behaviour of freely moving fruit flies</strong></div><div><em>J.&nbsp;R. Soc. Interface.</em> 2014; <strong>11</strong>:20140672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rsif.2014.0672" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84906994989" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25142523/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frsif.2014.0672&amp;pmid=25142523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Daniels and Nemenman, 2015</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Daniels, B.C. ∙ Nemenman, I.</div><div><strong>Automated adaptive inference of phenomenological dynamical models</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:8133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms9133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84939832859" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26293508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms9133&amp;pmid=26293508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0235" role="paragraph">Although we set out to obtain a diverse strain library for the purpose of mapping genomic variation to dynamic metabolic phenotypes, it is important to note that our library is composed solely of Proteobacteria and does not contain representatives from other phyla. This limitation means that it is unclear whether our regression approach can predict phenotypes of distantly related strains (e.g., gram-positive bacteria). In addition to the 79 strains described in this study, we attempted to assay the denitrification dynamics for three gram-positive Nar strains from the phylum Actinobacteria. We found their reduction rates to be slower than any strain in our library (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="tilde" tabindex="0" ctxtmenu_counter="174" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-speech="tilde"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 0.1&nbsp;mM/OD/h), resulting in almost negligible nitrate reduction over 64 h. This observation suggests that denitrification phenotypes in clades distant from Proteobacteria may be distinct, with rates that are potentially much slower than what we observed for Proteobacteria. Supporting this idea, denitrification in gram-positive bacteria is poorly understood (<span class="dropBlock reference-citations"><a id="body-ref-sref112" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112" aria-controls="sref112">Verbaendert et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref112"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Verbaendert, I. ∙ De Vos, P. ∙ Boon, N. ...</div><div><strong>Denitrification in Gram-positive bacteria: an underexplored trait</strong></div><div><em>Biochem. Soc. Trans.</em> 2011; <strong>39</strong>:254-258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1042/BST0390254" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-79551673854" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21265783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1042%2FBST0390254&amp;pmid=21265783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and previous studies that collected phenotypic data similar to ours characterized only Proteobacteria (<span class="dropBlock reference-citations"><a id="body-ref-sref67-4" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-4" aria-controls="sref67-4">Lycus et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref67-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Lycus, P. ∙ Lovise Bøthun, K. ∙ Bergaust, L. ...</div><div><strong>Phenotypic and genotypic richness of denitrifiers revealed by a novel isolation strategy</strong></div><div><em>ISME J.</em> 2017; <strong>11</strong>:2219-2232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2017.82" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85029618940" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28696424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2017.82&amp;pmid=28696424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Liu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Liu, B. ∙ Mao, Y. ∙ Bergaust, L. ...</div><div><strong>Strains in the genus Thauera exhibit remarkably different denitrification regulatory phenotypes</strong></div><div><em>Environ. Microbiol.</em> 2013; <strong>15</strong>:2816-2828</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23663391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=23663391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Therefore, extending our results to more diverse strains would require phenotyping a phylogenetically expanded library.</div><div id="p0240" role="paragraph">Considering the broader applicability of our statistical approach, there are some limitations to the types of metabolic processes and interactions that can be readily studied. Denitrification is a well-studied metabolic process, with the relevant enzymes known and easily annotated. Extending our method to less well-studied metabolic traits would require new approaches to learn the appropriate genomic features from data, since it may be challenging in those contexts to choose genes based on mechanistic knowledge. High-throughput mutant screens on wild isolates, for instance, via barcoded transposon mutant libraries (<span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Price et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Price, M.N. ∙ Wetmore, K.M. ∙ Waters, R.J. ...</div><div><strong>Mutant phenotypes for thousands of bacterial genes of unknown function</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:503-509</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0124-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85047501751" target="_blank" aria-label="Open Scopus in new tab">Scopus (173)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29769716/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0124-0&amp;pmid=29769716" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), could be used to discover unannotated or poorly annotated genes that are important for metabolic traits and potentially useful as predictors for metabolic phenotypes (<span class="dropBlock reference-citations"><a id="body-ref-sref110" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110" aria-controls="sref110">Vaccaro et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref110"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Vaccaro, B.J. ∙ Thorgersen, M.P. ∙ Lancaster, W.A. ...</div><div><strong>Determining roles of accessory genes in denitrification by mutant fitness analyses</strong></div><div><em>Appl. Environ. Microbiol.</em> 2016; <strong>82</strong>:51-61</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/AEM.02602-15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84953923654" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26452555/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FAEM.02602-15&amp;pmid=26452555" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0245" role="paragraph">Bridging the gap between the synthetic communities studied here and communities in the wild will require engaging with the chemical and spatial complexity of natural denitrifying communities. First, it is unclear whether the additive and non-additive interactions described here are relevant to wild communities. One way to determine the relevance of these interactions would be to measure co-occurrence between genotypes in natural contexts. Second, it remains to be seen how our approach generalizes to the complex nutrient environments, such as mixtures of organic carbon sources (<span class="dropBlock reference-citations"><a id="body-ref-sref108-1" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-1" aria-controls="sref108-1">Tiedje et&nbsp;al., 1982</a><div class="dropBlock__holder" data-db-target-of="sref108-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Tiedje, J.M. ∙ Sexstone, A.J. ∙ Myrold, D.D. ...</div><div><strong>Denitrification: ecological niches, competition and survival</strong></div><div><em>Antonie van Leeuwenhoek.</em> 1982; <strong>48</strong>:569-583</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00399542" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-34250229684" target="_blank" aria-label="Open Scopus in new tab">Scopus (423)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6762848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00399542&amp;pmid=6762848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), that are characteristic of natural communities. One approach to this problem would be to quantify nitrate and nitrite dynamics directly in soils and ask whether gene content can predict metabolite dynamics in this context. Finally, denitrification in nature occurs in the presence of other metabolic processes, where it often depends on nitrate from nitrifiers and competes with dissimilatory nitrate reduction to ammonia for electron acceptors (<span class="dropBlock reference-citations"><a id="body-ref-sref108-2" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-2" aria-controls="sref108-2">Tiedje et&nbsp;al., 1982</a><div class="dropBlock__holder" data-db-target-of="sref108-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Tiedje, J.M. ∙ Sexstone, A.J. ∙ Myrold, D.D. ...</div><div><strong>Denitrification: ecological niches, competition and survival</strong></div><div><em>Antonie van Leeuwenhoek.</em> 1982; <strong>48</strong>:569-583</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00399542" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-34250229684" target="_blank" aria-label="Open Scopus in new tab">Scopus (423)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6762848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00399542&amp;pmid=6762848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Extending the approach taken here to a broader ecological context that includes other metabolic fluxes is an important avenue to pursue.</div></section><section id="sec-3-2"><h3>Applying predictions of community metabolism from genomes</h3><div id="p0250" role="paragraph">At the single-strain level, the apparent mechanistic relevance of&nbsp;the regression coefficients in this study suggests that a statistical approach, coupled with large-scale culturing and phenotyping on libraries of isolates (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Connon and Giovannoni, 2002</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Connon, S.A. ∙ Giovannoni, S.J.</div><div><strong>High-throughput methods for culturing microorganisms in very-low-nutrient media yield diverse new marine isolates</strong></div><div><em>Appl. Environ. Microbiol.</em> 2002; <strong>68</strong>:3878-3885</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/AEM.68.8.3878-3885.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0036324056" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12147485/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FAEM.68.8.3878-3885.2002&amp;pmid=12147485" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Kehe et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Kehe, J. ∙ Kulesa, A. ∙ Ortiz, A. ...</div><div><strong>Massively parallel screening of synthetic microbial communities</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:12804-12809</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1900102116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85068117260" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31186361/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1900102116&amp;pmid=31186361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), can be exploited to discover the salient features of genomes that determine community metabolism. Higher-throughput measurements will enable a more detailed investigation of genomic features, allowing us to extend our statistical approach to variation in gene sequences and synteny.</div><div id="p0255" role="paragraph">Further, statistical predictions similar to those employed here could be used to help specify constraint-based metabolic models. Constraint-based metabolic models are refined using experimental measurements of metabolic traits (<span class="dropBlock reference-citations"><a id="body-ref-sref82-2" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-2" aria-controls="sref82-2">Norsigian et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref82-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Norsigian, C.J. ∙ Fang, X. ∙ Seif, Y. ...</div><div><strong>A workflow for generating multi-strain genome-scale metabolic models of prokaryotes</strong></div><div><em>Nat. Protoc.</em> 2020; <strong>15</strong>:1-14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0254-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85077092545" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31863076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0254-3&amp;pmid=31863076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but measuring these traits is challenging, especially for unculturable taxa or strains that are difficult to isolate from complex communities. Since our approach enables the prediction of metabolic phenotypes from genomes, these predictions could be used to refine constraint-based models of metabolic networks using genomic data alone and, thus, circumventing the need to experimentally measure metabolic phenotypes.</div><div id="p0260" role="paragraph">At the community level, our approach could eventually enable the prediction of metabolite dynamics in communities where gene presence and absence for individual genomes is known (<span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Sieber et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Sieber, C.M.K. ∙ Probst, A.J. ∙ Sharrar, A. ...</div><div><strong>Recovery of genomes from metagenomes via a dereplication, aggregation and scoring strategy</strong></div><div><em>Nat. Microbiol.</em> 2018; <strong>3</strong>:836-843</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-018-0171-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85047827857" target="_blank" aria-label="Open Scopus in new tab">Scopus (238)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29807988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-018-0171-1&amp;pmid=29807988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Soils and host-associated communities typically contain hundreds of bacterial taxa; therefore, it may be necessary to test the predictive power of the consumer-resource formalism in communities of many taxa. However, data from soils suggest that denitrification may occur locally, on 10–20&nbsp;μm grains (<span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Lensi et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Lensi, R. ∙ Clays-Josserand, A. ∙ Jocteur Monrozier, L.</div><div><strong>Denitrifiers and denitrifying activity in size fractions of a mollisol under permanent pasture and continuous cultivation</strong></div><div><em>Soil Biol. Biochem.</em> 1995; <strong>27</strong>:61-69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0038-0717(94)00132-K" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0028976873" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0038-0717%2894%2900132-K" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At this small scale, it is possible that communities are composed of just a few strains. If this is indeed the case, our results for communities of 2–5 strains (<a id="crosref1165" href="#fig6">Figures 6</a> and <a id="crosref1170" href="#fig7">7</a>) might apply to denitrifying communities in soil.</div></section><section id="sec-3-3"><h3>Departures from model predictions in Nar&nbsp;+ Nir communities</h3><div id="p0265" role="paragraph">It is striking that communities containing both Nar and Nir phenotypes departed from the expectation of an additive consumer-resource model (<a id="crosref1175" href="#fig6">Figures 6</a>C and <a id="opt11mKGXWzoI" href="#fig7">7</a>B). We proposed that the inhibition of nitrate reduction in Nar&nbsp;+ Nir communities may be caused by nitric oxide produced by the Nir strains. Consistent with this hypothesis, the most strongly inhibited Nar strains (PDM12 and PNT03, <a id="crosref1180" href="#figs7">Figure&nbsp;S7</a>) lack nitric oxide reductase (<a id="crosref1185" href="#fig4">Figure&nbsp;4</a>A); therefore, they likely cannot alleviate this toxicity. In addition, the strongly inhibited Nar strains possess the periplasmic nitrate reductase (<a id="optgSjbWlQapn" href="#fig2">Figure 2</a>B), which is exposed to the toxic effects of extracellular nitric oxide, whereas the weakly inhibited Nar strain ACV02 possesses the membrane-bound nitrate reductase, which is shielded from nitric oxide in the cytoplasm. Although Nir strains possess nitric oxide reductase and, therefore, could alleviate toxicity by reducing nitric oxide to nitrous oxide, Nir strains often transiently accumulate nitric oxide transiently (<span class="dropBlock reference-citations"><a id="body-ref-sref67-5" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-5" aria-controls="sref67-5">Lycus et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref67-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Lycus, P. ∙ Lovise Bøthun, K. ∙ Bergaust, L. ...</div><div><strong>Phenotypic and genotypic richness of denitrifiers revealed by a novel isolation strategy</strong></div><div><em>ISME J.</em> 2017; <strong>11</strong>:2219-2232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2017.82" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85029618940" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28696424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2017.82&amp;pmid=28696424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with this idea, when we measured relative abundances of Nar and Nir strains in co-culture, we observed smaller fractions of Nar strains relative to our model predictions in most cases (<a id="crosref1200" href="#figs4">Figure&nbsp;S4</a>B).</div><div id="p0270" role="paragraph">To describe metabolite dynamics in communities where both Nar and Nir strains were present, we chose not to expand our modeling formalism to include our hypothesized mechanism of Nar strain inhibition. Instead, we used measurements from Nar&nbsp;+ Nir pair cultures to describe community-level metabolite dynamics (<a id="crosref1205" href="#fig7">Figure&nbsp;7</a>). The advantage of this approach was to maintain a small number of model parameters, but it came at the expense of mechanistic interpretation. Another possible disadvantage of our approach was the challenge of modeling communities with multiple Nar and Nir pairs. However, we found that a simple averaging method (<a id="crosref1210" href="#sec-5">STAR Methods</a>) succeeded in describing community metabolite dynamics, even when multiple Nar&nbsp;+ Nir pairs were present in communities of 3–5 strains (<a id="crosref1215" href="#fig7">Figure&nbsp;7</a>B).</div><div id="p0275" role="paragraph">We note that Nar&nbsp;+ Nir pair cultures are metabolically distinct from Nar/Nir monocultures, in that the former splits the denitrification pathway across two genomes resulting in obligate cross-feeding. It is notable that our model fails only in the case where cross-feeding is required, suggesting that our formalism is most relevant for competitive interactions and that accurately predicting obligate cross-feeding from monoculture information alone may require additional parameters. The ecological context of denitrification pathway splitting at nitrite reduction is believed to be associated with environmental pH, with low pH favoring a split pathway. This hypothesis comes from a previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref60-4" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-4" aria-controls="sref60-4">Lilja and Johnson, 2016</a><div class="dropBlock__holder" data-db-target-of="sref60-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Lilja, E.E. ∙ Johnson, D.R.</div><div><strong>Segregating metabolic processes into different microbial cells accelerates the consumption of inhibitory substrates</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:1568-1578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2015.243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84954437600" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2015.243&amp;pmid=26771930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) showing that the transient accumulation of nitrite during denitrification can be reduced by segregating the processes of nitrate and nitrite reduction across genomes. Reducing transient nitrite accumulation is advantageous in low pH environments, where nitrite forms toxic intermediates (<span class="dropBlock reference-citations"><a id="body-ref-sref60-5" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-5" aria-controls="sref60-5">Lilja and Johnson, 2016</a><div class="dropBlock__holder" data-db-target-of="sref60-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Lilja, E.E. ∙ Johnson, D.R.</div><div><strong>Segregating metabolic processes into different microbial cells accelerates the consumption of inhibitory substrates</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:1568-1578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2015.243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84954437600" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2015.243&amp;pmid=26771930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because we observe Nar&nbsp;+ Nir communities escaping the transient accumulation of nitrite (<a id="crosref1230" href="#fig6">Figures 6</a>A and <a id="optgKMs5BQJaE" href="#figs6">S6</a>), our results are consistent with splitting of the denitrification pathway at nitrite reduction as an adaptation to acidic environments.</div></section></section><section id="sec-4"><h2>Conclusion</h2><div id="p0280" role="paragraph">We find it striking that a statistical approach can uncover a simple relationship between gene content and metabolite dynamics in communities of diverse wild isolates. It is our hope that future work can leverage this approach to understand and predict the metabolic activity of microbial communities in natural settings.</div><section id="sec-4-1"><h3>Limitations of the study</h3><div id="p0285" role="paragraph">We assumed that metabolic phenotypes can be captured by a consumer-resource model, an assumption that breaks down for a fraction of our isolates and limits the direct applicability of our approach to strains and processes that can be modeled using a simple phenomenology. For example, our modeling formalism works well when the electron acceptor is limiting but may fail when the donor (organic carbon) is limiting.</div><div id="p0290" role="paragraph">Our regression approach exploits correlations between genotype and phenotype to make predictions. To some extent, these correlations reflect conserved phenotypic impact of certain genes, but phylogenetic correlation also plays a role. Therefore, we do not expect the regression to make causal predictions of the impact of single-gene knockout mutations on phenotypes.</div><div id="p0295" role="paragraph">Our library of isolates comprises strains from the phylum Proteobacteria. We do not expect our results to generalize to distantly related denitrifiers in other phyla, such as gram-positive bacteria. Expanding the library is likely necessary to predict phenotypes of distantly related strains.</div><div id="p0300" role="paragraph">Our approach has been demonstrated for comparatively simple nutrient conditions in well-mixed conditions. It remains to be seen how well this statistical approach will work in natural contexts, where spatial structure and complex chemical environments are present.</div></section></section><section id="sec-5"><h2>STAR★Methods</h2><section id="sec-5-1"><h3>Key resources table</h3><div id="p0305" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td>Bacterial isolates</td><td>This paper</td><td><a id="crosref1235" href="#mmc1">Table S2</a></td></tr><tr class="rowsep"><td><i>Paracoccus denitrificans</i></td><td>ATCC</td><td>Cat#19367</td></tr><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr class="rowsep"><td>Soils for bacterial isolation</td><td>This paper</td><td><a id="crosref1240" href="#mmc1">Table S1</a></td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td><i>N</i>-(1-Naphthyl)ethylenediamine dihydrochloride</td><td>Sigma-Aldrich</td><td>Cat#222488-10G</td></tr><tr><td>Sulfanilamide</td><td>Sigma-Aldrich</td><td>Cat#S9251-100G</td></tr><tr class="rowsep"><td>Vanadium(III) chloride</td><td>Fisher Scientific</td><td>Cat#AC197000050</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>DNeasy UltraClean Microbial Kit</td><td>Qiagen</td><td>Cat#12224-250</td></tr><tr><td>MiSeq Reagent Kit v3 (600 cycle)</td><td>Illumina</td><td>Cat#MS-102-3003</td></tr><tr><td>Nextera DNA CD Indexes</td><td>Illumina</td><td>Cat#20018707</td></tr><tr><td>Nextera DNA Flex Library Prep Kit</td><td>Illumina</td><td>Cat#20018704</td></tr><tr><td>PhiX Control v3</td><td>Illumina</td><td>Cat#FC-110-3001</td></tr><tr><td>Platinum Hot Start PCR Master Mix</td><td>Invitrogen</td><td>Cat#13000013</td></tr><tr><td>QIAquick PCR Purification Kit</td><td>Qiagen</td><td>Cat#28106</td></tr><tr class="rowsep"><td>Qubit dsDNA BR Assay</td><td>Invitrogen</td><td>Cat#Q32853</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Metabolite dynamics data, consumer-resource parameters, genome annotations</td><td>This paper</td><td><a id="intref0010" href="https://doi.org/10.17605/osf.io/t3prd" target="_blank">https://doi.org/10.17605/osf.io/t3prd</a></td></tr><tr><td><i>Paracoccus denitrificans</i> complete genome</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref101-1" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-1" aria-controls="sref101-1">Si et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref101-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Si, Y.-Y. ∙ Xu, K.-H. ∙ Yu, X.-Y. ...</div><div><strong>Complete genome sequence of <i>Paracoccus denitrificans</i> ATCC 19367 and its denitrification characteristics</strong></div><div><em>Can. J. Microbiol.</em> 2019; <strong>65</strong>:486-495</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1139/cjm-2019-0037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85068138644" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30897350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1139%2Fcjm-2019-0037&amp;pmid=30897350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>BioProject ID PRJNA513156</td></tr><tr class="rowsep"><td>Raw sequencing data and draft genome assemblies</td><td>This paper</td><td>BioProject ID PRJNA660495</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>16S rRNA universal primer (27F)</td><td>Integrated DNA Technologies</td><td>AGAGTTTGATCMTGGCTCAG</td></tr><tr class="rowsep"><td>16S rRNA universal primer (806R)</td><td>Integrated DNA Technologies</td><td>GGACTACNVGGGTWTCTAAT</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>MATLAB R2017B</td><td>Mathworks</td><td><a id="intref0015" href="https://mathworks.com/products/matlab.html" target="_blank">https://www.mathworks.com/products/matlab.html</a></td></tr><tr><td>R 3.6.1</td><td>R Core Team</td><td><a id="intref0020" href="https://r-project.org/" target="_blank">https://www.r-project.org/</a></td></tr><tr><td>Original code</td><td>This paper</td><td><a id="intref0025" href="https://doi.org/10.17605/osf.io/t3prd" target="_blank">https://doi.org/10.17605/osf.io/t3prd</a></td></tr><tr><td>CASEU 0.1.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Cermak et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cermak, N. ∙ Datta, M.S. ∙ Conwill, A.</div><div><strong>Rapid, inexpensive measurement of synthetic bacterial community composition by Sanger sequencing of amplicon mixtures</strong></div><div><em>iScience.</em> 2020; <strong>23</strong>:100915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85080040522" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32114382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2020.100915&amp;pmid=32114382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0030" href="https://bitbucket.org/DattaManoshi/caseu" target="_blank">https://bitbucket.org/DattaManoshi/caseu</a></td></tr><tr><td>MEGA X 10.1.8</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Kumar et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kumar, S. ∙ Stecher, G. ∙ Li, M. ...</div><div><strong>MEGA X: molecular evolutionary genetics analysis across computing platforms</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:1547-1549</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msy096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85050137432" target="_blank" aria-label="Open Scopus in new tab">Scopus (11440)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29722887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsy096&amp;pmid=29722887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0035" href="https://megasoftware.net/" target="_blank">https://www.megasoftware.net/</a></td></tr><tr><td>phylosignal 1.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Keck et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Keck, F. ∙ Rimet, F. ∙ Bouchez, A. ...</div><div><strong>phylosignal: an R package to measure, test, and explore the phylogenetic signal</strong></div><div><em>Ecol. Evol.</em> 2016; <strong>6</strong>:2774-2780</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ece3.2051" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84962504104" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27066252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fece3.2051&amp;pmid=27066252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0040" href="https://cran.r-project.org/web/packages/phylosignal" target="_blank">https://cran.r-project.org/web/packages/phylosignal</a></td></tr><tr><td>QUAST 5.02</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Gurevich et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Gurevich, A. ∙ Saveliev, V. ∙ Vyahhi, N. ...</div><div><strong>QUAST: quality assessment tool for genome assemblies</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1072-1075</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt086" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84876266928" target="_blank" aria-label="Open Scopus in new tab">Scopus (2977)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt086&amp;pmid=23422339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="https://github.com/ablab/quast" target="_blank">https://github.com/ablab/quast</a></td></tr><tr><td>RAST</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Brettin et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Brettin, T. ∙ Davis, J.J. ∙ Disz, T. ...</div><div><strong>RASTtk: a modular and extensible implementation of the RAST algorithm for building custom annotation pipelines and annotating batches of genomes</strong></div><div><em>Sci. Rep.</em> 2015; <strong>5</strong>:8365</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep08365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84938841197" target="_blank" aria-label="Open Scopus in new tab">Scopus (988)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25666585/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep08365&amp;pmid=25666585" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="http://rast.theseed.org/" target="_blank">http://rast.theseed.org</a></td></tr><tr><td>selectiveInference 1.2.5</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref107-1" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-1" aria-controls="sref107-1">Taylor and Tibshirani, 2015</a><div class="dropBlock__holder" data-db-target-of="sref107-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Taylor, J. ∙ Tibshirani, R.J.</div><div><strong>Statistical learning and selective inference</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:7629-7634</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507583112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84934993654" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26100887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507583112&amp;pmid=26100887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0055" href="https://cran.r-project.org/web/packages/selectiveInference/" target="_blank">https://cran.r-project.org/web/packages/selectiveInference/</a></td></tr><tr><td>SILVA ACT</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Pruesse et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Pruesse, E. ∙ Peplies, J. ∙ Glöckner, F.O.</div><div><strong>SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:1823-1829</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84864134721" target="_blank" aria-label="Open Scopus in new tab">Scopus (1834)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22556368/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts252&amp;pmid=22556368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="http://arb-silva.de/" target="_blank">http://www.arb-silva.de/</a></td></tr><tr><td>SPAdes 3.13.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Bankevich et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Bankevich, A. ∙ Nurk, S. ∙ Antipov, D. ...</div><div><strong>SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</strong></div><div><em>J.&nbsp;Comput. Biol.</em> 2012; <strong>19</strong>:455-477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/cmb.2012.0021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84860771820" target="_blank" aria-label="Open Scopus in new tab">Scopus (10972)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22506599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Fcmb.2012.0021&amp;pmid=22506599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0065" href="https://github.com/ablab/spades" target="_blank">https://github.com/ablab/spades</a></td></tr><tr class="rowsep"><td>Trimmomatic 0.39</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Bolger et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Bolger, A.M. ∙ Lohse, M. ∙ Usadel, B.</div><div><strong>Trimmomatic: a flexible trimmer for Illumina sequence data</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:2114-2120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84905049901" target="_blank" aria-label="Open Scopus in new tab">Scopus (21176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu170&amp;pmid=24695404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0070" href="https://github.com/usadellab/Trimmomatic" target="_blank">https://github.com/usadellab/Trimmomatic</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Anaerobic glove box</td><td>Coy Laboratory Products</td><td>Cat#7601-110/220</td></tr><tr><td>Breathe-Easier membranes</td><td>Diversified Biotech</td><td>Cat#BERM-2000</td></tr><tr><td>Deepwell plates</td><td>Axygen</td><td>Cat#PDW20C</td></tr><tr><td>Digital mass flow controllers</td><td>Sierra Instruments</td><td>SmartTrak 50</td></tr><tr><td>High-throughput DNA sequencing system</td><td>Illumina</td><td>MiSeq</td></tr><tr><td>Liquid handling robot</td><td>Formulatrix</td><td>Mantis</td></tr><tr><td>Microplate reader</td><td>BMG</td><td>CLARIOstar</td></tr><tr><td>Plate shaker</td><td>Troemner</td><td>Talboys Professional 1000MP</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2901542-7" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-5-2"><h3>Resource availability</h3><section id="sec-5-2-1"><h4>Lead contact</h4><div id="p0310" role="paragraph">Further information and requests for resources should be directed to and will be fulfilled by the lead contact, Seppe Kuehn (<a id="intref0075" href="mailto:seppe.kuehn@gmail.com">seppe.kuehn@gmail.com</a>).</div></section><section id="sec-5-2-2"><h4>Materials availability</h4><div id="p0315" role="paragraph">Strains isolated in this study will be made available on request. No new or unique reagents were generated in this study.</div></section></section><section id="sec-5-3"><h3>Experimental model and subject details</h3><section id="sec-5-3-1"><h4>Strains</h4><div id="p0320" role="paragraph">The bacterial strains and isolates used in this study are listed in <a id="crosref1285" href="#mmc1">Table S1</a>.</div></section><section id="sec-5-3-2"><h4>Isolation of denitrifying bacteria from soils</h4><div id="p0325" role="paragraph">Denitrifying bacteria were isolated from soil samples following a modified version of the protocol developed by Lycus et&nbsp;al. (<span class="dropBlock reference-citations"><a id="body-ref-sref67-6" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-6" aria-controls="sref67-6">Lycus et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref67-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Lycus, P. ∙ Lovise Bøthun, K. ∙ Bergaust, L. ...</div><div><strong>Phenotypic and genotypic richness of denitrifiers revealed by a novel isolation strategy</strong></div><div><em>ISME J.</em> 2017; <strong>11</strong>:2219-2232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2017.82" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85029618940" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28696424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2017.82&amp;pmid=28696424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). 5–10 g soil samples were collected from local prairie and forest environments, agricultural land, and manicured lawns (<a id="crosref1290" href="#mmc1">Table S2</a>). Samples were stored separately in 50&nbsp;mL centrifuge tubes at 4°C for no longer than three months before use.</div><div id="p0330" role="paragraph">Each soil sample was prepared for isolation by combining with 25&nbsp;mL PBS (pH 7.4) and 5–10 g sterile 4&nbsp;mm glass beads. Additionally 0.5&nbsp;mL cycloheximide solution (10&nbsp;mg/mL) and 20&nbsp;μL nystatin solution (25&nbsp;mg/mL) were added to prevent fungal growth. In certain rounds of isolation (<a id="crosref1295" href="#mmc1">Table S2</a>), sterile NaNO<sub>2</sub> solution (1 M) was added to enrich for nitrite-reducing bacteria. Tubes were then vortexed (Vortex-Genie 2) at high speed for 1&nbsp;min to homogenize the samples. In certain rounds of isolation (<a id="crosref1300" href="#mmc1">Table S2</a>), homogenized soils were incubated at either room temperature or 30°C for up to two weeks, and then briefly re-vortexed before further processing.</div><div id="p0335" role="paragraph">Supernatants from homogenized soils were then diluted and plated. After vortexing, large particles in homogenized soil samples were allowed to settle for 20–30 min before transferring 1&nbsp;mL of supernatant to sterile 1.7&nbsp;mL microcentrifuge tubes. Soil supernatants were then serially-diluted in PBS to obtain <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="10 Superscript negative 4" tabindex="0" ctxtmenu_counter="175" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="10 Superscript negative 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-children="2" data-semantic-content="1" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>-fold and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="10 Superscript negative 5" tabindex="0" ctxtmenu_counter="176" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="10 Superscript negative 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-children="2" data-semantic-content="1" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>-fold dilutions. 100&nbsp;μL of the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="10 Superscript negative 4" tabindex="0" ctxtmenu_counter="177" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="10 Superscript negative 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-children="2" data-semantic-content="1" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="10 Superscript negative 5" tabindex="0" ctxtmenu_counter="178" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="10 Superscript negative 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-children="2" data-semantic-content="1" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> soil supernatant dilutions were plated on 1/10X tryptic soy agar (1/10X TSA, 1.5 g/L tryptone, 0.5 g/L soytone, 0.5 g/L NaCl, 15 g/L agar), with two replicates for each dilution. Plates were then incubated under aerobic conditions at 30°C for 48 h.</div><div id="p0340" role="paragraph">Colonies from plated soil supernatants were picked and streaked to purity. Plated soil dilutions were examined for growth after incubation. Plates showing little or no growth were incubated for an additional 24–48&nbsp;h until colonies appeared. Plates showing likely fungal growth were discarded. For each set of plates derived from a soil sample, 5–15 well-separated colonies were picked and streaked to purity on 1/10X TSA plates, again incubating at 30°C. Whenever possible, colonies were selected which varied in morphology, size, and color, in order to enhance the diversity of the isolate collection.</div><div id="p0345" role="paragraph">Bacterial isolates were assayed for nitrate and/or nitrite-reduction capability. Isolates were first cultured aerobically in sterile 96-deepwell plates (Axygen PDW20C) containing 1 mL per well 1/10X tryptic soy broth liquid medium (1/10X TSB), inoculated directly from streak plate colonies. Deepwell plates were sealed with a gas-permeable membrane (Diversified Biotech BERM-2000) and incubated/shaken at 30°C/950RPM (Talboys Professional 1000MP) for 48 h. 1&nbsp;μL of each dense aerobic culture was then passaged to fresh 96-deepwell plates containing 1 mL per well 1/10X TSB supplemented with either sterile NaNO<sub>3</sub> or NaNO<sub>2</sub> solution (1 M) to yield a final concentration of 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="179" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> or <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="180" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, with two replicates of each condition per isolate. Plates were sealed with gas-permeable membranes (Diversified Biotech BEM-1) and transferred to an <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N 2" tabindex="0" ctxtmenu_counter="181" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal upper N 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>-purged anaerobic glove box (Coy Laboratory Products 7601-110/220). Plates were incubated/shaken at 30°C/950RPM for 48&nbsp;h under anaerobic conditions. After incubation, plates were sampled to determine endpoint concentrations of nitrate and nitrite.</div><div id="p0350" role="paragraph">Nitrate and nitrite-reduction capability for each isolate was determined by comparing endpoint concentrations of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="182" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="183" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> in each culture with the corresponding concentrations in uninoculated controls. Isolates showing lower endpoint levels of nitrate and/or nitrite than the uninoculated controls were identified as nitrate and/or nitrite reducers, respectively. Strains that performed both nitrate and nitrite reduction were classified as “Nar/Nir” strains, while strains that performed nitrate or nitrite reduction only were classified as “Nar” or “Nir” strains, respectively. These isolates were cryopreserved by mixing 500&nbsp;μL of saturated culture grown in 1/10X TSB under aerobic conditions with 500&nbsp;μL filter-sterilized 50% glycerol solution in 2&nbsp;mL cryotubes, and freezing/storing at -80°C.</div></section><section id="sec-5-3-3"><h4>Defined growth medium</h4><div id="p0355" role="paragraph">In order to eliminate the possibility of fermentative metabolism in subsequent (post-isolation) denitrification experiments, defined growth media based on the medium designed by <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Heylen et al. (2006)</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Heylen, K. ∙ Vanparys, B. ∙ Wittebolle, L. ...</div><div><strong>Cultivation of denitrifying bacteria: optimization of isolation conditions and diversity study</strong></div><div><em>Appl. Environ. Microbiol.</em> 2006; <strong>72</strong>:2637-2643</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/AEM.72.4.2637-2643.2006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33646102691" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16597968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FAEM.72.4.2637-2643.2006&amp;pmid=16597968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> for the cultivation of diverse denitrifying bacteria were used (<a id="crosref1315" href="#mmc1">Table S4</a>). A succinate-defined medium (SDM) that contained 25 mM succinate (100 mM C) as the sole (non-fermentable) carbon source was used for most denitrification experiments. The medium also contained trace metals and vitamins, 15 mM ammonium as the assimilatory nitrogen source, and a 40 mM phosphate buffer with the final medium pH adjusted to 7.3. It was expected that nitrate and nitrite assimilation were inhibited by high concentrations of ammonia (<span class="dropBlock reference-citations"><a id="body-ref-sref117-7" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-7" aria-controls="sref117-7">Zumft, 1997</a><div class="dropBlock__holder" data-db-target-of="sref117-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Zumft, W.G.</div><div><strong>Cell biology and molecular basis of denitrification</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 1997; <strong>61</strong>:533-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/.61.4.533-616.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2F.61.4.533-616.1997&amp;pmid=9409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally an acetate defined medium (ADM) containing 50 mM acetate (100 mM C) in place of succinate was used for some experiments, with all other medium components unchanged. In ADM, acetate served as the sole non-fermentable carbon source.</div><div id="p0360" role="paragraph">In denitrification experiments using defined media, nitrate or nitrite was provided in at most 2 mM concentrations, and it was expected that growth was nitrate and/or nitrite limited. This expectation was corroborated by measurements of endpoint biomass densities for strains in axenic culture, in which typically higher biomass densities were observed in conditions where 2 mM nitrate or nitrite was supplied (and completely reduced) than when 1 mM was supplied. Moreover, in a previous study, growth experiments using a mineral medium containing 5 mM acetate (10 mM C), 3.7 mM ammonia, and 1.5 mM phosphate, demonstrated that nitrate and nitrite were limiting at up to a 5 mM concentration for the strains <i>Paracoccus denitrificans</i> and <i>Pseudomonas stutzeri</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref105-1" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-1" aria-controls="sref105-1">Strohm et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref105-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Strohm, T.O. ∙ Griffin, B. ∙ Zumft, W.G. ...</div><div><strong>Growth yields in bacterial denitrification and nitrate ammonification</strong></div><div><em>Appl. Environ. Microbiol.</em> 2007; <strong>73</strong>:1420-1424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/AEM.02508-06" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33947193902" target="_blank" aria-label="Open Scopus in new tab">Scopus (174)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17209072/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FAEM.02508-06&amp;pmid=17209072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For comparison, SDM and ADM supplied carbon, ammonia, and phosphate in significantly greater concentrations, and nitrate and nitrite in lower concentrations, further supporting the claim that nitrate and/or nitrite were growth limiting under denitrifying conditions.</div><div id="p0365" role="paragraph">Denitrifying isolates were screened for growth on SDM under aerobic conditions by first growing axenically from freezer stocks in 1/10X TSB, passaging 1:300 into 300&nbsp;μL SDM in 96-well plates (with two replicates per isolate), and culturing at 30°C for 48 h. Endpoint optical density was measured to assess growth on SDM using a microplate reader (BMG CLARIOstar). Denitrifying isolates unable to grow on SDM (approximately 36% of isolates) were excluded from further experimentation and analysis. Strains that grew on SDM were also further screened for growth on ADM (<a id="crosref1330" href="#mmc1">Table S1</a>).</div></section><section id="sec-5-3-4"><h4>Denitrifying conditions</h4><div id="p0370" role="paragraph">Post-isolation denitrification experiments were performed in a vinyl glove box (Coy Laboratory Products 7601-110/220) purged of oxygen with a 99%/1% N<sub>2</sub>/CO<sub>2</sub> gas mixture. Provision of CO<sub>2</sub> was necessary to support the growth of cultures from low initial biomass densities (OD600<sub>0</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="much less than" tabindex="0" ctxtmenu_counter="184" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="much less than"><mjx-c class="mjx-c226A">≪</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 0.01), likely due to the CO<sub>2</sub>-fixation requirements of core anaplerotic metabolism (<span class="dropBlock reference-citations"><a id="body-ref-sref115" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115" aria-controls="sref115">White et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref115"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>White, D. ∙ Drummond, J. ∙ Fuqua, C.</div><div><strong>The Physiology and Biochemistry of Prokaryotes</strong></div><div>Oxford University Press, 2012</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.WhiteJ.DrummondC.FuquaThe+Physiology+and+Biochemistry+of+Prokaryotes4th+Edition2012Oxford+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Gas mixing and a purge rate of 20 SLPM were controlled using digital mass flow controllers (Sierra Instruments SmartTrak 50). Oxygen and CO<sub>2</sub> concentrations inside the glove box were continually monitored using Arduino-attached (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Mathupala et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Mathupala, S.P. ∙ Kiousis, S. ∙ Szerlip, N.J.</div><div><strong>A lab assembled microcontroller-based sensor module for continuous oxygen measurement in portable hypoxia chambers</strong></div><div><em>PLoS ONE.</em> 2016; <strong>11</strong>:e0148923</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0148923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26862760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0148923&amp;pmid=26862760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) optical sensors (SST Sensing LOX-02, Gas Sensing Solutions EXPLORIR-M-20). Gaseous oxygen concentration was maintained around 200 ppm, which was sufficient to prevent growth via aerobic respiration. This was verified using two denitrifying isolates, the Nar strain <i>Paracoccus</i> sp. PAR01 and the Nir strain <i>Pseudomonas</i> sp. PDM13, which were grown under denitrifying conditions for 64h with and without a suitable electron acceptor (i.e., nitrate for the Nar strain and nitrite for the Nir strain). The Nar strain PAR01 was inoculated at biomass density OD600<sub>0</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 011" tabindex="0" ctxtmenu_counter="185" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 011"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and grew to OD600<sub>64</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 072" tabindex="0" ctxtmenu_counter="186" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 072"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.676em;">0</mjx-c><mjx-c class="mjx-c37" noic="true" style="padding-top: 0.676em;">7</mjx-c><mjx-c class="mjx-c32" style="padding-top: 0.676em;">2</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> with nitrate and remained at OD600<sub>64</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 013" tabindex="0" ctxtmenu_counter="187" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 013"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.666em;">3</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> without nitrate. Similarly, the Nir strain PDM13 was inoculated at biomass density OD600<sub>0</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 010" tabindex="0" ctxtmenu_counter="188" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 010"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and grew to OD600<sub>64</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 026" tabindex="0" ctxtmenu_counter="189" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 026"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.666em;">2</mjx-c><mjx-c class="mjx-c36" style="padding-top: 0.666em;">6</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> with nitrite and remained at OD600<sub>64</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 010" tabindex="0" ctxtmenu_counter="190" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 010"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> without nitrite.</div></section></section><section id="sec-5-4"><h3>Method details</h3><section id="sec-5-4-1"><h4>Assay of nitrate and nitrite</h4><div id="p0375" role="paragraph">Concentrations of nitrate and nitrite were measured using a microplate reader (BMG CLARIOstar) following a modified version of the protocol by <span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Miranda et al. (2001)</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Miranda, K.M. ∙ Espey, M.G. ∙ Wink, D.A.</div><div><strong>A rapid, simple spectrophotometric method for simultaneous detection of nitrate and nitrite</strong></div><div><em>Nitric Oxide.</em> 2001; <strong>5</strong>:62-71</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/niox.2000.0319" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0035709008" target="_blank" aria-label="Open Scopus in new tab">Scopus (2479)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11178938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fniox.2000.0319&amp;pmid=11178938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, where the conventional Griess assay for detection of nitrite is coupled with the chemical reduction of nitrate to nitrite using vanadium (III) chloride. Stock solutions of <i>N</i>-(1-Napthyl)ethylenediamine dihydrochloride (NEDD), sulfanilamide (SULF, in 5% HCl), and VCl<sub>3</sub> (in 1 M HCl) were prepared and stored as described in <span class="dropBlock reference-citations"><a id="body-ref-sref75-3" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-3" aria-controls="sref75-3">Miranda et al. (2001)</a><div class="dropBlock__holder" data-db-target-of="sref75-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Miranda, K.M. ∙ Espey, M.G. ∙ Wink, D.A.</div><div><strong>A rapid, simple spectrophotometric method for simultaneous detection of nitrate and nitrite</strong></div><div><em>Nitric Oxide.</em> 2001; <strong>5</strong>:62-71</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/niox.2000.0319" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0035709008" target="_blank" aria-label="Open Scopus in new tab">Scopus (2479)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11178938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fniox.2000.0319&amp;pmid=11178938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Griess reagent was freshly prepared on the day of each assay by mixing NEDD and SULF stock solutions with ultrapure H<sub>2</sub>O in a 5:5:9 ratio.</div><div id="p0380" role="paragraph">Nitrate and nitrite were measured using 10&nbsp;μL samples of analyte in a 96-well microplate. Nitrite was first quantified by adding 190&nbsp;μL of Griess reagent to each sample and recording blank-corrected maximum absorbance within the interval 450 to 650&nbsp;nm, denoted <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A b s Subscript m a x Superscript upper G" tabindex="0" ctxtmenu_counter="191" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 0 12 1 13 (11 (10 2 (8 3 6 4 7 5)) 9))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="0,1,11" data-semantic-content="12,13" data-semantic-owns="0 12 1 13 11" data-semantic-speech="upper A b s Subscript m a x Superscript upper G"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="14"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="2,8,9" data-semantic-parent="14" data-semantic-collapsed="(11 (10 2 8) 9)" data-semantic-owns="2 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="11" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>. Then nitrate was chemically reduced to nitrite by adding 50&nbsp;μL of VCl<sub>3</sub> solution to each sample and incubating at 30°C for 6–7 h, which was sufficient time for complete reduction of nitrate to nitrite. After incubation, the maximum blank-corrected absorbance within the interval 450 to 650&nbsp;nm, denoted <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A b s Subscript m a x Superscript upper V" tabindex="0" ctxtmenu_counter="192" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 0 12 1 13 (11 (10 2 (8 3 6 4 7 5)) 9))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="0,1,11" data-semantic-content="12,13" data-semantic-owns="0 12 1 13 11" data-semantic-speech="upper A b s Subscript m a x Superscript upper V"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="14"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="2,8,9" data-semantic-parent="14" data-semantic-collapsed="(11 (10 2 8) 9)" data-semantic-owns="2 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="11" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, was recorded to quantify the sum total of nitrate and nitrite in the sample.</div><div id="p0385" role="paragraph">Concentrations of nitrate and nitrite were determined using 8-point standard curves. Separate two-fold dilutions of NaNO<sub>2</sub> and NaNO<sub>3</sub> standards spanning 31&nbsp;μM–2&nbsp;mM were prepared, and 10&nbsp;μL samples of these standards were dispensed in triplicate into a 96-well plate, along with 10&nbsp;μL samples of ultrapure water that served as blanks. Griess reagents were used as described above to measure nitrite in the NaNO<sub>2</sub> standards, and these data were parameterized by fitting the equation:<div id="fd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A b s Subscript m a x Superscript upper G Baseline left parenthesis upper I right parenthesis equals a 0 plus a 1 upper I plus a 2 upper I squared comma" tabindex="0" ctxtmenu_counter="193" style="font-size: 130%;"><mjx-math data-semantic-structure="(44 (43 (37 0 35 1 36 (34 (11 (10 2 (8 3 6 4 7 5)) 9) 33 (15 12 13 14))) 16 (40 (19 17 18) 20 (39 (23 21 22) 38 24) 25 (42 (28 26 27) 41 (31 29 30)))) 32)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="44" data-semantic-children="43,32" data-semantic-content="32" data-semantic-owns="43 32" data-semantic-speech="upper A b s Subscript m a x Superscript upper G Baseline left parenthesis upper I right parenthesis equals a 0 plus a 1 upper I plus a 2 upper I squared comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="43" data-semantic-children="37,40" data-semantic-content="16" data-semantic-parent="44" data-semantic-owns="37 16 40"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="37" data-semantic-children="0,1,34" data-semantic-content="35,36" data-semantic-parent="43" data-semantic-owns="0 35 1 36 34"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="37"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="35" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="37"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="36" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-children="11,15" data-semantic-content="33,2" data-semantic-parent="37" data-semantic-owns="11 33 15"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="simple function" data-semantic-id="11" data-semantic-children="2,8,9" data-semantic-parent="34" data-semantic-collapsed="(11 (10 2 8) 9)" data-semantic-owns="2 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11" data-semantic-operator="appl"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="11" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="13" data-semantic-content="12,14" data-semantic-parent="34" data-semantic-owns="12 13 14"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="12" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="16" data-semantic-parent="43" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="40" data-semantic-children="19,39,42" data-semantic-content="20,25" data-semantic-parent="43" data-semantic-owns="19 20 39 25 42"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="19" data-semantic-children="17,18" data-semantic-parent="40" data-semantic-owns="17 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="19"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="20" data-semantic-parent="40" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="39" data-semantic-children="23,24" data-semantic-content="38" data-semantic-parent="40" data-semantic-owns="23 38 24"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="23" data-semantic-children="21,22" data-semantic-parent="39" data-semantic-owns="21 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="23"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="23"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="39"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="25" data-semantic-parent="40" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="42" data-semantic-children="28,31" data-semantic-content="41" data-semantic-parent="40" data-semantic-owns="28 41 31"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="26,27" data-semantic-parent="42" data-semantic-owns="26 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="41" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="31" data-semantic-children="29,30" data-semantic-parent="42" data-semantic-owns="29 30"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em; margin-left: 0.053em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="32" data-semantic-parent="44" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">(Equation 1)</div></div>where <i>I</i> is the concentration of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="194" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>. A quadratic term was included to account for the slight nonlinearity of the absorbance at high values of concentration. Then VCl<sub>3</sub> solution was used as described above to measure nitrate in the NaNO<sub>3</sub> standards, and these data were parameterized by fitting the equation:<div id="fd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A b s Subscript m a x Superscript upper V Baseline left parenthesis upper A right parenthesis equals b 0 plus b 1 upper A plus b 2 upper A squared comma" tabindex="0" ctxtmenu_counter="195" style="font-size: 130%;"><mjx-math data-semantic-structure="(44 (43 (37 0 35 1 36 (34 (11 (10 2 (8 3 6 4 7 5)) 9) 33 (15 12 13 14))) 16 (40 (19 17 18) 20 (39 (23 21 22) 38 24) 25 (42 (28 26 27) 41 (31 29 30)))) 32)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="44" data-semantic-children="43,32" data-semantic-content="32" data-semantic-owns="43 32" data-semantic-speech="upper A b s Subscript m a x Superscript upper V Baseline left parenthesis upper A right parenthesis equals b 0 plus b 1 upper A plus b 2 upper A squared comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="43" data-semantic-children="37,40" data-semantic-content="16" data-semantic-parent="44" data-semantic-owns="37 16 40"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="37" data-semantic-children="0,1,34" data-semantic-content="35,36" data-semantic-parent="43" data-semantic-owns="0 35 1 36 34"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="37"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="35" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="37"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="36" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-children="11,15" data-semantic-content="33,2" data-semantic-parent="37" data-semantic-owns="11 33 15"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="simple function" data-semantic-id="11" data-semantic-children="2,8,9" data-semantic-parent="34" data-semantic-collapsed="(11 (10 2 8) 9)" data-semantic-owns="2 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11" data-semantic-operator="appl"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="11" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="13" data-semantic-content="12,14" data-semantic-parent="34" data-semantic-owns="12 13 14"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="12" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="16" data-semantic-parent="43" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="40" data-semantic-children="19,39,42" data-semantic-content="20,25" data-semantic-parent="43" data-semantic-owns="19 20 39 25 42"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="19" data-semantic-children="17,18" data-semantic-parent="40" data-semantic-owns="17 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="19"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="20" data-semantic-parent="40" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="39" data-semantic-children="23,24" data-semantic-content="38" data-semantic-parent="40" data-semantic-owns="23 38 24"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="23" data-semantic-children="21,22" data-semantic-parent="39" data-semantic-owns="21 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="23"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="23"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="39"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="25" data-semantic-parent="40" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="42" data-semantic-children="28,31" data-semantic-content="41" data-semantic-parent="40" data-semantic-owns="28 41 31"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="26,27" data-semantic-parent="42" data-semantic-owns="26 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="41" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="31" data-semantic-children="29,30" data-semantic-parent="42" data-semantic-owns="29 30"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="32" data-semantic-parent="44" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">(Equation 2)</div></div>where <i>A</i> is the concentration of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="196" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>. Standard curves were run in triplicate on each day that an assay was performed.</div><div id="p0390" role="paragraph">The nitrate and nitrite concentrations in a new sample, for which <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A b s Subscript m a x Superscript upper G" tabindex="0" ctxtmenu_counter="197" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 0 12 1 13 (11 (10 2 (8 3 6 4 7 5)) 9))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="0,1,11" data-semantic-content="12,13" data-semantic-owns="0 12 1 13 11" data-semantic-speech="upper A b s Subscript m a x Superscript upper G"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="14"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="2,8,9" data-semantic-parent="14" data-semantic-collapsed="(11 (10 2 8) 9)" data-semantic-owns="2 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="11" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A b s Subscript m a x Superscript upper V" tabindex="0" ctxtmenu_counter="198" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 0 12 1 13 (11 (10 2 (8 3 6 4 7 5)) 9))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="0,1,11" data-semantic-content="12,13" data-semantic-owns="0 12 1 13 11" data-semantic-speech="upper A b s Subscript m a x Superscript upper V"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="14"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="2,8,9" data-semantic-parent="14" data-semantic-collapsed="(11 (10 2 8) 9)" data-semantic-owns="2 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="11" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> were recorded, were then determined by:<div id="olist0010" role="list"><div id="o0010" role="listitem"><div class="label">1.</div><div class="content"><div id="p0395" role="paragraph">Solving <a id="crosref1355" href="#fd1">Equation 1</a> to determine <i>I</i> from <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A b s Subscript m a x Superscript upper G Baseline left parenthesis upper I right parenthesis" tabindex="0" ctxtmenu_counter="199" style="font-size: 130%;"><mjx-math data-semantic-structure="(20 0 18 1 19 (17 (11 (10 2 (8 3 6 4 7 5)) 9) 16 (15 12 13 14)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="20" data-semantic-children="0,1,17" data-semantic-content="18,19" data-semantic-owns="0 18 1 19 17" data-semantic-speech="upper A b s Subscript m a x Superscript upper G Baseline left parenthesis upper I right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="20"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="20"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-children="11,15" data-semantic-content="16,2" data-semantic-parent="20" data-semantic-owns="11 16 15"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="simple function" data-semantic-id="11" data-semantic-children="2,8,9" data-semantic-parent="17" data-semantic-collapsed="(11 (10 2 8) 9)" data-semantic-owns="2 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11" data-semantic-operator="appl"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="11" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="13" data-semantic-content="12,14" data-semantic-parent="17" data-semantic-owns="12 13 14"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="12" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>,</div></div></div><div id="o0015" role="listitem"><div class="label">2.</div><div class="content"><div id="p0400" role="paragraph">Solving <a id="crosref1360" href="#fd2">Equation 2</a> to determine the quantity <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A plus upper I" tabindex="0" ctxtmenu_counter="200" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper A plus upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> from <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A b s Subscript m a x Superscript upper V Baseline left parenthesis upper A plus upper I right parenthesis" tabindex="0" ctxtmenu_counter="201" style="font-size: 130%;"><mjx-math data-semantic-structure="(22 0 19 1 20 (11 (10 2 (8 3 6 4 7 5)) 9) 21 (18 12 (16 13 14 15) 17))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-children="0,1,11,18" data-semantic-content="19,20,21" data-semantic-owns="0 19 1 20 11 21 18" data-semantic-speech="upper A b s Subscript m a x Superscript upper V Baseline left parenthesis upper A plus upper I right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="22"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="22"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="2,8,9" data-semantic-parent="22" data-semantic-collapsed="(11 (10 2 8) 9)" data-semantic-owns="2 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="11" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="18" data-semantic-children="16" data-semantic-content="12,17" data-semantic-parent="22" data-semantic-owns="12 16 17"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="12" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="16" data-semantic-children="13,15" data-semantic-content="14" data-semantic-parent="18" data-semantic-owns="13 14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="16"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="14" data-semantic-parent="16" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> (since this step of the assay measures the sum total of nitrate and nitrite in a sample),</div></div></div><div id="o0020" role="listitem"><div class="label">3.</div><div class="content"><div id="p0405" role="paragraph">Subtracting the result of step 1 from the result of step 2 to obtain <i>A.</i></div></div></div></div></div><div id="p0410" role="paragraph">This approach was validated using mixed standards containing both nitrate and nitrite in a 1:1 ratio, and relative error was found to be typically less than <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="plus or minus" tabindex="0" ctxtmenu_counter="202" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="0" data-semantic-speech="plus or minus"><mjx-c class="mjx-cB1">±</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 5% for concentrations <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than or equals" tabindex="0" ctxtmenu_counter="203" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than or equals"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 62.5&nbsp;μM.</div><div id="p0415" role="paragraph">In this study, nitrate and nitrite assays were performed directly on samples taken from denitrifying cultures (i.e., without first removing cells via filtration or centrifugation). To verify that neither the presence of cells nor molecules secreted by strains during denitrification interfere with the assay, measurement error was evaluated in standards combined with samples taken from denitrifying cultures of four representative strains. The strains <i>Achromobacter</i> sp. ACM01, <i>Ensifer</i> sp. ENS09, <i>Paracoccus denitrificans</i> ATCC 19367 (PAR19367), and <i>Pseudomonas</i> sp. PDM21 were grown in a succinate defined medium under denitrifying conditions from low initial abundance (OD600<sub>0</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="almost equals" tabindex="0" ctxtmenu_counter="204" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-speech="almost equals"><mjx-c class="mjx-c2248">≈</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 0.01) for 64 h, by which time cells were in stationary phase (OD600<sub>64</sub>&nbsp;= 0.052–0.148). 10&nbsp;μL samples of these cultures were then then combined with 10&nbsp;μL samples of nitrate and nitrite standards and then assayed as described above.&nbsp;In standards containing nitrate or nitrite separately, relative error of the assay was typically less than <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="plus or minus" tabindex="0" ctxtmenu_counter="205" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="0" data-semantic-speech="plus or minus"><mjx-c class="mjx-cB1">±</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 5% for concentrations <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than or equals" tabindex="0" ctxtmenu_counter="206" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than or equals"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 125&nbsp;μM, with relative error <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="plus or minus" tabindex="0" ctxtmenu_counter="207" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="0" data-semantic-speech="plus or minus"><mjx-c class="mjx-cB1">±</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 10% at 62.5&nbsp;μM. In standards containing mixtures of both nitrate and nitrite in a 1:1 ratio, relative error was again found to be typically less than <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="plus or minus" tabindex="0" ctxtmenu_counter="208" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="0" data-semantic-speech="plus or minus"><mjx-c class="mjx-cB1">±</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 5% for concentrations <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than or equals" tabindex="0" ctxtmenu_counter="209" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than or equals"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 125&nbsp;μM, with relative error <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="plus or minus" tabindex="0" ctxtmenu_counter="210" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="0" data-semantic-speech="plus or minus"><mjx-c class="mjx-cB1">±</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 10% at 62.5&nbsp;μM. This indicates that presence of cells and/or molecules secreted by strains during denitrification do not appreciably increase measurement error above concentrations of 125&nbsp;μM.</div></section><section id="sec-5-4-2"><h4>Denitrification dynamics experiments</h4><div id="p0420" role="paragraph">Strains were pre-cultured under aerobic conditions (using first broth then defined medium) prior to growth under denitrifying (anaerobic) conditions in defined medium. Strains were first grown axenically to saturation from freezer stocks in 1/5X TSB, then passaged 1:100 into defined medium and again grown to saturation. All pre-cultures were grown in 24-well plates containing 1.7 mL medium per well and incubated/shaken at 30°C/400RPM under aerobic conditions. Since strains in the library vary widely in the time required to grow to saturation in 1/5X TSB and defined medium (12–96 h), inoculation and passaging of strains was timed so that defined medium aerobic cultures would enter stationary phase within 12 h of the start of an experiment in denitrifying conditions.</div><div id="p0425" role="paragraph">Experiments to measure the nitrate/nitrite-reduction dynamics of strains in monoculture were performed with multiple initial media and biomass density conditions: (a) 2 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O Subscript italic asterisk Superscript minus" tabindex="0" ctxtmenu_counter="211" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O Subscript italic asterisk Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.225em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, OD600<sub>0</sub>&nbsp;= 0.01, (b) 1 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O Subscript italic asterisk Superscript minus" tabindex="0" ctxtmenu_counter="212" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O Subscript italic asterisk Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.225em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, OD600<sub>0</sub>&nbsp;= 0.01, (c) 2 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O Subscript italic asterisk Superscript minus" tabindex="0" ctxtmenu_counter="213" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O Subscript italic asterisk Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.225em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, OD600<sub>0</sub>&nbsp;= 0.001, where Nar/Nir strains were cultured with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="214" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="215" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> separately (6 conditions total), Nar strains were cultured with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="216" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> (3 conditions total), and Nir strains were cultured with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="217" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> (3 conditions total). Medium containing (separately) 2 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="218" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, 1 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="219" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, 2 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="220" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, and 1 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="221" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> were prepared by supplementing fresh defined medium with sterile NaNO<sub>3</sub> or NaNO<sub>2</sub> (1 M) solutions. Aerobic defined medium pre-cultures were normalized to two levels of optical density (OD600 <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="almost equals" tabindex="0" ctxtmenu_counter="222" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-speech="almost equals"><mjx-c class="mjx-c2248">≈</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 1.2 and 0.12) by diluting with PBS, and these normalized densities were recorded. 96-deepwell plates containing 1.2&nbsp;mL of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="223" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> or <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="224" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>-supplemented defined medium were then inoculated with 10&nbsp;μL of density-normalized culture, resulting in OD600 <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="almost equals" tabindex="0" ctxtmenu_counter="225" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-speech="almost equals"><mjx-c class="mjx-c2248">≈</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 0.01 and 0.001 cell density conditions. 2–4 replicates were used for each combination of media and cell density conditions.</div><div id="p0430" role="paragraph">Inoculated plates were sealed with gas-permeable membranes (Diversified Biotech BERM-2000) and transferred to an anaerobic glove box for incubation/shaking at 30°C/950RPM. Cultures were manually sampled with a multichannel pipette (10&nbsp;μL per well) at 2, 4, 6, 8, 16, 32, and 64&nbsp;h from the start of anaerobic culture. Samples were stored in clean 96-well plates, and immediately sealed (Bio-Rad MSB1001) and frozen at -20°C for later assay of nitrate and nitrite. Additionally, at 64 h, 300&nbsp;μL per well of culture was sampled into clean 96-well plates and the optical density (pathlength normalized to 1&nbsp;cm) immediately measured to quantify endpoint biomass density. Strains observed to form excessive cell aggregation (to the extent that quantification of endpoint biomass by optical density was not possible) were at this point excluded from further experimentation and analysis (<a id="crosref1365" href="#mmc1">Table S3</a>).</div><div id="p0435" role="paragraph">Additional experiments to measure nitrate/nitrite-reduction dynamics on communities of 2–5 strains (including monoculture controls) were performed in separate 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="226" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and 2 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="227" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> conditions. Aerobic pre-cultures were normalized to OD600 <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="almost equals" tabindex="0" ctxtmenu_counter="228" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-speech="almost equals"><mjx-c class="mjx-c2248">≈</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 1.2 with PBS. Using liquid handling robotics (Formulatrix Mantis), 96-deepwell plates containing 1.2&nbsp;mL of 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="229" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> or <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="230" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> SDM was inoculated with 10&nbsp;μL droplets of density-normalized culture in defined combinations (e.g., a pair community containing strains A and B would be inoculated with separate 10&nbsp;μL droplets of strain A and strain B). Following inoculation, all culture and sampling details were the same as described above.</div><div id="p0440" role="paragraph">Measurements were post-processed to correct for gradual evaporation of H<sub>2</sub>O that increased apparent concentrations with time. 3–8 uninoculated controls containing nitrate or nitrite-supplemented medium were used on each culture plate to quantify the effect of evaporation. Raw nitrate and nitrite measurements for cultures were scaled at each time point by the concentration of initially supplied nitrate or nitrite divided by the median concentration across blank measurements at that time point.</div></section><section id="sec-5-4-3"><h4>Whole genome sequencing and annotation</h4><div id="p0445" role="paragraph">Denitrifying strains were grown axenically from freezer stocks or plated colonies in 1/10X TSB incubated at 30°C with shaking. Saturated cell cultures were harvested and DNA extracted using the DNeasy UltraClean Microbial Kit (Qiagen). DNA concentrations were quantified using the Qubit dsDNA BR Assay Kit (Invitrogen).</div><div id="p0450" role="paragraph">Library preparation for sequencing of DNA extracts was performed using the Nextera DNA Flex Library Prep Kit (Illumina) and the Nextera DNA CD Indexes (Illumina). Barcoded libraries for each strain were pooled in groups of 21–24 strains and quantified using the Qubit dsDNA BR kit and an Agilent 2100 Bioanalyzer (Carver Biotechnology Center, University of Illinois at Urbana-Champaign). Each pooled library was then separately sequenced using a MiSeq Reagent Kit v3 (Illumina, 2 <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="times" tabindex="0" ctxtmenu_counter="231" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="0" data-semantic-speech="times"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 300 bp paired-end), with a 12 pM library loading concentration and a 1% spike-in of PhiX Control v3 (Illumina). Sequencing was performed on a locally maintained and operated Illumina MiSeq system.</div><div id="p0455" role="paragraph">Raw paired-end reads were trimmed of low-quality regions and Illumina adapters using Trimmomatic 0.39 (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Bolger et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Bolger, A.M. ∙ Lohse, M. ∙ Usadel, B.</div><div><strong>Trimmomatic: a flexible trimmer for Illumina sequence data</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:2114-2120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84905049901" target="_blank" aria-label="Open Scopus in new tab">Scopus (21176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu170&amp;pmid=24695404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Trimmed reads were assembled into contigs <i>de novo</i> using SPAdes 3.13.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Bankevich et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Bankevich, A. ∙ Nurk, S. ∙ Antipov, D. ...</div><div><strong>SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</strong></div><div><em>J.&nbsp;Comput. Biol.</em> 2012; <strong>19</strong>:455-477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/cmb.2012.0021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84860771820" target="_blank" aria-label="Open Scopus in new tab">Scopus (10972)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22506599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Fcmb.2012.0021&amp;pmid=22506599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with k-mer lengths of 21, 33, 55, 77, 99, and 127 and the error-correction option “careful” enabled. Assembly quality was assessed using QUAST 5.02 (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Gurevich et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Gurevich, A. ∙ Saveliev, V. ∙ Vyahhi, N. ...</div><div><strong>QUAST: quality assessment tool for genome assemblies</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1072-1075</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt086" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84876266928" target="_blank" aria-label="Open Scopus in new tab">Scopus (2977)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt086&amp;pmid=23422339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The draft assembly of the laboratory strain <i>Paracoccus denitrificans</i> ATCC 19367 was compared to a complete assembly of the same strain by Si et&nbsp;al. (<span class="dropBlock reference-citations"><a id="body-ref-sref101-2" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-2" aria-controls="sref101-2">Si et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref101-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Si, Y.-Y. ∙ Xu, K.-H. ∙ Yu, X.-Y. ...</div><div><strong>Complete genome sequence of <i>Paracoccus denitrificans</i> ATCC 19367 and its denitrification characteristics</strong></div><div><em>Can. J. Microbiol.</em> 2019; <strong>65</strong>:486-495</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1139/cjm-2019-0037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85068138644" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30897350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1139%2Fcjm-2019-0037&amp;pmid=30897350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which served as a reference. The comparison indicated a 97.8% genome fraction (percentage of bases in the reference that align to the draft assembly), with one 30.8 kbp relocation misassembly, 7.02 mismatches (sequencing errors or single nucleotide polymorphisms) per 100 kbp, and 0.43 indels per 100 kbp. The draft assembly produced 4635 predicted genes relative to 4644 genes predicted in the reference. The coverage depth of the <i>P.&nbsp;denitrificans</i> draft assembly was 32X, compared to a median of 35X for the other strains in the library. Together these analyses suggest that the draft assemblies are likely to cover the vast majority of the protein-coding sequences, with relatively few errors that would affect the inference of gene presence or absence.</div><div id="p0460" role="paragraph">Contigs were uploaded for gene annotation on the RAST Server (<a id="intref0080" href="http://rast.theseed.org/" target="_blank">http://rast.theseed.org</a>) using the RASTtk pipeline (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Brettin et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Brettin, T. ∙ Davis, J.J. ∙ Disz, T. ...</div><div><strong>RASTtk: a modular and extensible implementation of the RAST algorithm for building custom annotation pipelines and annotating batches of genomes</strong></div><div><em>Sci. Rep.</em> 2015; <strong>5</strong>:8365</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep08365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84938841197" target="_blank" aria-label="Open Scopus in new tab">Scopus (988)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25666585/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep08365&amp;pmid=25666585" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Denitrification gene presence and absence information was obtained from annotation files by a text search of gene function labels (<a id="crosref1395" href="#mmc1">Table S6</a>). The NarXL two-component nitrate/nitrite sensing system was considered “present” if a gene encoding the sensor (<i>narX</i>) and/or the response regulator (<i>narL</i>) were identified in the annotation. Each gene identified as a NarK-type nitrate transporter was classified post-annotation as encoding either a nitrate/<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper H Superscript plus" tabindex="0" ctxtmenu_counter="232" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal upper H Superscript plus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c48">H</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> symporter (<i>narK1</i>), a nitrate/nitrite antiporter (<i>narK2</i>), or a fusion of both transporters (<i>narK1K2</i>). This classification was performed by locally aligning each transporter sequence with the <i>narK1K2</i> gene previously identified in <i>Paracoccus denitrificans</i> PD1222 (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Goddard et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Goddard, A.D. ∙ Bali, S. ∙ Mavridou, D.A. ...</div><div><strong>The <i>Paracoccus denitrificans</i> NarK-like nitrate and nitrite transporters-probing nitrate uptake and nitrate/nitrite exchange mechanisms</strong></div><div><em>Mol. Microbiol.</em> 2017; <strong>103</strong>:117-133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13546" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84999767719" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27696579/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13546&amp;pmid=27696579" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), for which the N-terminal domain has been identified to be NarK1 and the C-terminal domain NarK2. From this it was determined which RAST gene function labels correspond to <i>narK1</i>, <i>narK2</i>, and <i>narK1K2</i>.</div><div id="p0465" role="paragraph">Additionally annotation files were searched for the cytochrome c nitrite reductase gene (<i>nrfA</i>) associated with dissimilatory nitrate reduction to ammonia (DNRA). Strains possessing this gene were excluded from further experimentation and analysis (<a id="crosref1405" href="#mmc1">Table S3</a>).</div></section><section id="sec-5-4-4"><h4>Phylogenetic classification of strains</h4><div id="p0470" role="paragraph">Phylogenetic analysis of denitrifying strains was performed using full 16S rRNA sequences identified in annotated draft genome assemblies. 16S sequences were uploaded for classification up to the genus level using the SILVA ACT service (<a id="intref0085" href="http://arb-silva.de/" target="_blank">http://www.arb-silva.de/</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Pruesse et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Pruesse, E. ∙ Peplies, J. ∙ Glöckner, F.O.</div><div><strong>SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:1823-1829</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84864134721" target="_blank" aria-label="Open Scopus in new tab">Scopus (1834)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22556368/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts252&amp;pmid=22556368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A maximum-likelihood phylogenetic tree was computed using MEGA X 10.1.8 (<span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Kumar et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kumar, S. ∙ Stecher, G. ∙ Li, M. ...</div><div><strong>MEGA X: molecular evolutionary genetics analysis across computing platforms</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:1547-1549</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msy096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85050137432" target="_blank" aria-label="Open Scopus in new tab">Scopus (11440)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29722887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsy096&amp;pmid=29722887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-5-4-5"><h4>Measuring relative abundances and contamination</h4><div id="p0475" role="paragraph">The <i>N</i>-strain consumer-resource model provides predictions of the relative abundances of each strain in a community. These predictions were tested by measuring relative abundances for a set of 2-strain communities using the package CASEU 0.1.2 in R 3.6.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref12-3" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-3" aria-controls="sref12-3">Cermak et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref12-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cermak, N. ∙ Datta, M.S. ∙ Conwill, A.</div><div><strong>Rapid, inexpensive measurement of synthetic bacterial community composition by Sanger sequencing of amplicon mixtures</strong></div><div><em>iScience.</em> 2020; <strong>23</strong>:100915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85080040522" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32114382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2020.100915&amp;pmid=32114382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Compositional Analysis by Sanger Electropherogram Unmixing (CASEU) infers relative abundances in communities via Sanger sequencing amplicons of a marker gene that distinguishes the different strains (e.g., the 16S rRNA gene). The approach works by deconvolving mixed Sanger electropherograms of amplicons from a community using the pure electropherograms obtained by sequencing amplicons from axenic cultures. Therefore the approach requires that (a) the constituents of a community can be cultured axenically, and (b) the electropherograms of a marker gene for each strain individually be sufficiently distinct (i.e., having pairwise electropherogram correlations less or equal to than approximately 0.8; <span class="dropBlock reference-citations"><a id="body-ref-sref12-4" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-4" aria-controls="sref12-4">Cermak et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref12-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cermak, N. ∙ Datta, M.S. ∙ Conwill, A.</div><div><strong>Rapid, inexpensive measurement of synthetic bacterial community composition by Sanger sequencing of amplicon mixtures</strong></div><div><em>iScience.</em> 2020; <strong>23</strong>:100915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85080040522" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32114382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2020.100915&amp;pmid=32114382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0480" role="paragraph">All pair combinations and monocultures of six taxonomically and phenotypically representative strains, <i>Achromobacter</i> sp. ACM01 (Nar/Nir), <i>Ensifer</i> sp. ENS09 (Nar/Nir), <i>Paracoccus</i> sp. PAR01 (Nar), <i>Pantoea</i> sp. PNT03 (Nar), <i>Agrobacterium</i> sp. AGB01 (Nir), and <i>Pseudomonas</i> sp. PDM13 (Nir), were inoculated, grown, and periodically sampled in denitrifying conditions. All cultures were initiated with 2 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="233" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, with the exception of monocultures of Nir strains AGB01 and PDM13 and the pair culture of these two strains, which were initiated with 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="234" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>. Cultures were harvested at 64 h, at which time optical densities were measured. In order to calibrate a conversion from relative biomass density (as measured by OD600) and relative abundance of 16S amplicons, equal volume (450&nbsp;μL) synthetic mixtures of all pairs of monocultures were also prepared with known relative abundances of each strain.</div><div id="p0485" role="paragraph">DNA was extracted from monocultures, pair cultures, and synthetic pair mixtures using the DNeasy UltraClean Microbial Kit. A fragment of the 16S rRNA gene was amplified using the 27F (AGAGTTTGATCMTGGCTCAG) and 806R (GGACTACNVGGGTWTCTAAT) universal primers. The following reagents were used for each reaction: 22&nbsp;μL nuclease-free H<sub>2</sub>O, 1&nbsp;μL 27F primer (10&nbsp;μM), 1&nbsp;μL 807R primer (10&nbsp;μM), 1&nbsp;μL DNA extract, 25&nbsp;μL Platinum Hot Start PCR Master Mix (Invitrogen). The following thermocycler settings were used: initial denaturation, 2&nbsp;min at 94°C; amplification (35 cycles), 30 sec at 94°C, 30 sec at 53°C, 60 sec at 72°C; final extension, 10 min at 72°C. PCR products were cleaned using the QIAquick PCR Purification Kit (Qiagen). Sanger sequencing of cleaned PCR products was performed at the University of Chicago Comprehensive Cancer Center DNA Sequencing and Genotyping Facility.</div><div id="p0490" role="paragraph">Resulting AB1 files containing electropherogram signals were analyzed using CASEU to infer relative abundances in each sample. First, it was verified that the six pure electropherograms from monocultures of the different strains were sufficiently distinct to distinguish signals in mixed electropherograms. Pairwise correlations between pure electropherograms were computed using CASEU, and were generally found to be much smaller than 0.8, with the exception of the Alphaproteobacteria ENS09, PAR01, and AGB01, pairs of which had electropherogram correlations of approximately 0.8. It was therefore concluded that these strains could be distinguished in a community using the CASEU approach.</div><div id="p0495" role="paragraph">Next, the six pure electropherograms were used to infer relative abundances in the pair cultures. In order to assess the extent of cross-contamination in these pair cultures, CASEU was allowed infer relative abundance using all six pure electropherograms for each pair culture, in essence assuming that any of the six strains could be present in any given pair culture. In every community, no evidence of cross-contamination was observed (i.e., in each pair community, only the intended pair of strains was measured at non-zero relative abundance. Though the CASEU approach cannot reliably resolve relative abundances below 1–2% (<span class="dropBlock reference-citations"><a id="body-ref-sref12-5" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-5" aria-controls="sref12-5">Cermak et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref12-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cermak, N. ∙ Datta, M.S. ∙ Conwill, A.</div><div><strong>Rapid, inexpensive measurement of synthetic bacterial community composition by Sanger sequencing of amplicon mixtures</strong></div><div><em>iScience.</em> 2020; <strong>23</strong>:100915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85080040522" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32114382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2020.100915&amp;pmid=32114382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), if cross-contamination was present at these levels, it would have little to no impact on community metabolite dynamics. Therefore it was concluded that cross-contamination was generally negligible in denitrification experiments.</div></section></section><section id="sec-5-5"><h3>Quantification and statistical analysis</h3><section id="sec-5-5-1"><h4>Consumer-resource model for metabolite dynamics</h4><div id="p0500" role="paragraph">A consumer-resource model was used to parameterize the dynamics of nitrate and nitrite-reduction. For a single strain that performs both nitrate and nitrite reduction (Nar/Nir strain), the model is as follows:<div id="fd3" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartLayout 1st Row  StartFraction d x Over d t EndFraction equals left parenthesis gamma Subscript upper A Baseline r Subscript upper A Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction plus gamma Subscript upper I Baseline r Subscript upper I Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x comma 2nd Row  StartFraction d upper A Over d t EndFraction equals minus r Subscript upper A Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction x comma 3rd Row  StartFraction d upper I Over d t EndFraction equals left parenthesis r Subscript upper A Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction minus r Subscript upper I Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x period EndLayout" tabindex="0" ctxtmenu_counter="235" style="font-size: 130%;"><mjx-math data-semantic-structure="(138 (56 (54 (53 (8 (3 0 2 1) (7 4 6 5)) 9 (52 (48 46 (45 (41 (12 10 11) 39 (15 13 14) 40 (23 16 (22 (19 17 18) 20 21))) 24 (44 (27 25 26) 42 (30 28 29) 43 (38 31 (37 (34 32 33) 35 36)))) 47) 51 49)) 50)) (88 (86 (85 (65 (60 57 59 58) (64 61 63 62)) 66 (84 67 (83 (70 68 69) 81 (78 71 (77 (74 72 73) 75 76)) 82 79))) 80)) (137 (135 (134 (97 (92 89 91 90) (96 93 95 94)) 98 (133 (129 127 (126 (123 (101 99 100) 122 (109 102 (108 (105 103 104) 106 107))) 110 (125 (113 111 112) 124 (121 114 (120 (117 115 116) 118 119)))) 128) 132 130)) 131)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mtable data-semantic-type="multiline" data-semantic-role="unknown" data-semantic-id="138" data-semantic-children="56,88,137" data-semantic-owns="56 88 137" data-semantic-speech="StartLayout 1st Row  StartFraction d x Over d t EndFraction equals left parenthesis gamma Subscript upper A Baseline r Subscript upper A Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction plus gamma Subscript upper I Baseline r Subscript upper I Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x comma 2nd Row  StartFraction d upper A Over d t EndFraction equals minus r Subscript upper A Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction x comma 3rd Row  StartFraction d upper I Over d t EndFraction equals left parenthesis r Subscript upper A Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction minus r Subscript upper I Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x period EndLayout" justify="left" style="min-width: 14.668em;"><mjx-table><mjx-itable><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="56" data-semantic-children="54" data-semantic-parent="138" data-semantic-owns="54"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="54" data-semantic-children="53,50" data-semantic-content="50" data-semantic-parent="56" data-semantic-owns="53 50"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="53" data-semantic-children="8,52" data-semantic-content="9" data-semantic-parent="54" data-semantic-owns="8 9 52"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="53" data-semantic-owns="3 7"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="3" data-semantic-children="0,1" data-semantic-content="2" data-semantic-parent="8" data-semantic-owns="0 2 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="2" data-semantic-parent="3" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="53" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="52" data-semantic-children="48,49" data-semantic-content="51" data-semantic-parent="53" data-semantic-owns="48 51 49"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="48" data-semantic-children="45" data-semantic-content="46,47" data-semantic-parent="52" data-semantic-owns="46 45 47"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="46" data-semantic-parent="48" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="45" data-semantic-children="41,44" data-semantic-content="24" data-semantic-parent="48" data-semantic-owns="41 24 44"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="41" data-semantic-children="12,15,23" data-semantic-content="39,40" data-semantic-parent="45" data-semantic-owns="12 39 15 40 23"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="41" data-semantic-owns="10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="39" data-semantic-parent="41" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="41" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="40" data-semantic-parent="41" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="23" data-semantic-children="16,22" data-semantic-parent="41" data-semantic-owns="16 22"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="23"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="22" data-semantic-children="19,21" data-semantic-content="20" data-semantic-parent="23" data-semantic-owns="19 20 21"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="19" data-semantic-children="17,18" data-semantic-parent="22" data-semantic-owns="17 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="19"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="20" data-semantic-parent="22" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="22"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="24" data-semantic-parent="45" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="44" data-semantic-children="27,30,38" data-semantic-content="42,43" data-semantic-parent="45" data-semantic-owns="27 42 30 43 38"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="27" data-semantic-children="25,26" data-semantic-parent="44" data-semantic-owns="25 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="27"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="27"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="42" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="30" data-semantic-children="28,29" data-semantic-parent="44" data-semantic-owns="28 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="30"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="30"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="38" data-semantic-children="31,37" data-semantic-parent="44" data-semantic-owns="31 37"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="38"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="37" data-semantic-children="34,36" data-semantic-content="35" data-semantic-parent="38" data-semantic-owns="34 35 36"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="34" data-semantic-children="32,33" data-semantic-parent="37" data-semantic-owns="32 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="34"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="34"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="35" data-semantic-parent="37" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="37"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="47" data-semantic-parent="48" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="51" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="52"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="50" data-semantic-parent="54" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="88" data-semantic-children="86" data-semantic-parent="138" data-semantic-owns="86"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="86" data-semantic-children="85,80" data-semantic-content="80" data-semantic-parent="88" data-semantic-owns="85 80"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="85" data-semantic-children="65,84" data-semantic-content="66" data-semantic-parent="86" data-semantic-owns="65 66 84"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="65" data-semantic-children="60,64" data-semantic-parent="85" data-semantic-owns="60 64"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="60" data-semantic-children="57,58" data-semantic-content="59" data-semantic-parent="65" data-semantic-owns="57 59 58"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="57" data-semantic-parent="60"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="59" data-semantic-parent="60" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="58" data-semantic-parent="60"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="64" data-semantic-children="61,62" data-semantic-content="63" data-semantic-parent="65" data-semantic-owns="61 63 62"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="64"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="63" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="62" data-semantic-parent="64"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="66" data-semantic-parent="85" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="84" data-semantic-children="83" data-semantic-content="67" data-semantic-parent="85" data-semantic-owns="67 83"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="67" data-semantic-parent="84" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="83" data-semantic-children="70,78,79" data-semantic-content="81,82" data-semantic-parent="84" data-semantic-owns="70 81 78 82 79"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="70" data-semantic-children="68,69" data-semantic-parent="83" data-semantic-owns="68 69"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="68" data-semantic-parent="70"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="69" data-semantic-parent="70"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="81" data-semantic-parent="83" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="78" data-semantic-children="71,77" data-semantic-parent="83" data-semantic-owns="71 77"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="71" data-semantic-parent="78"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="77" data-semantic-children="74,76" data-semantic-content="75" data-semantic-parent="78" data-semantic-owns="74 75 76"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="74" data-semantic-children="72,73" data-semantic-parent="77" data-semantic-owns="72 73"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="74"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="73" data-semantic-parent="74"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="75" data-semantic-parent="77" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="76" data-semantic-parent="77"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="82" data-semantic-parent="83" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="79" data-semantic-parent="83"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="80" data-semantic-parent="86" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="137" data-semantic-children="135" data-semantic-parent="138" data-semantic-owns="135"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="135" data-semantic-children="134,131" data-semantic-content="131" data-semantic-parent="137" data-semantic-owns="134 131"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="134" data-semantic-children="97,133" data-semantic-content="98" data-semantic-parent="135" data-semantic-owns="97 98 133"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="97" data-semantic-children="92,96" data-semantic-parent="134" data-semantic-owns="92 96"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="92" data-semantic-children="89,90" data-semantic-content="91" data-semantic-parent="97" data-semantic-owns="89 91 90"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="89" data-semantic-parent="92"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="91" data-semantic-parent="92" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="90" data-semantic-parent="92"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="96" data-semantic-children="93,94" data-semantic-content="95" data-semantic-parent="97" data-semantic-owns="93 95 94"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="93" data-semantic-parent="96"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="95" data-semantic-parent="96" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="94" data-semantic-parent="96"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="98" data-semantic-parent="134" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="133" data-semantic-children="129,130" data-semantic-content="132" data-semantic-parent="134" data-semantic-owns="129 132 130"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="129" data-semantic-children="126" data-semantic-content="127,128" data-semantic-parent="133" data-semantic-owns="127 126 128"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="127" data-semantic-parent="129" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="126" data-semantic-children="123,125" data-semantic-content="110" data-semantic-parent="129" data-semantic-owns="123 110 125"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="123" data-semantic-children="101,109" data-semantic-content="122" data-semantic-parent="126" data-semantic-owns="101 122 109"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="101" data-semantic-children="99,100" data-semantic-parent="123" data-semantic-owns="99 100"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="99" data-semantic-parent="101"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="100" data-semantic-parent="101"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="122" data-semantic-parent="123" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="109" data-semantic-children="102,108" data-semantic-parent="123" data-semantic-owns="102 108"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="102" data-semantic-parent="109"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="108" data-semantic-children="105,107" data-semantic-content="106" data-semantic-parent="109" data-semantic-owns="105 106 107"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="105" data-semantic-children="103,104" data-semantic-parent="108" data-semantic-owns="103 104"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="103" data-semantic-parent="105"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="104" data-semantic-parent="105"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="106" data-semantic-parent="108" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="107" data-semantic-parent="108"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="110" data-semantic-parent="126" data-semantic-operator="infixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="125" data-semantic-children="113,121" data-semantic-content="124" data-semantic-parent="126" data-semantic-owns="113 124 121"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="113" data-semantic-children="111,112" data-semantic-parent="125" data-semantic-owns="111 112"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="111" data-semantic-parent="113"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="112" data-semantic-parent="113"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="124" data-semantic-parent="125" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="121" data-semantic-children="114,120" data-semantic-parent="125" data-semantic-owns="114 120"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="114" data-semantic-parent="121"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="120" data-semantic-children="117,119" data-semantic-content="118" data-semantic-parent="121" data-semantic-owns="117 118 119"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="117" data-semantic-children="115,116" data-semantic-parent="120" data-semantic-owns="115 116"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="115" data-semantic-parent="117"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="116" data-semantic-parent="117"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="118" data-semantic-parent="120" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="119" data-semantic-parent="120"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="128" data-semantic-parent="129" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="132" data-semantic-parent="133" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="130" data-semantic-parent="133"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="131" data-semantic-parent="135" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr></mjx-itable></mjx-table></mjx-mtable></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">(Equation 3)</div></div></div><div id="p0505" role="paragraph">The variable <i>x</i> (in units of OD at 600 nm) is the biomass density of a single population, and <i>A</i> and <i>I</i> (mM) are the concentrations of nitrate and nitrite respectively. The model is parameterized by reduction rates <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="236" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="237" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (mM/OD/h), yields <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="238" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="239" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (OD/mM), and substrate affinities and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="240" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="241" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (mM), for growth on nitrate and nitrite, respectively.</div><div id="p0510" role="paragraph">The model, which resembles the Monod model for bacterial growth (<span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Monod, 1949</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Monod, J.</div><div><strong>The growth of bacterial cultures</strong></div><div><em>Annu. Rev. Microbiol.</em> 1949; <strong>3</strong>:371-394</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.mi.03.100149.002103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.mi.03.100149.002103" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), assumes that growth of the population occurs at a rate proportional to the reduction rates of nitrate and nitrite, with the two resources treated as substitutable. Nitrite is the direct product of nitrate reduction (in a 1:1 stoichiometry). Nitric oxide, the product of nitrite reduction, is not explicitly modeled, and thus mass leaves the system when nitrite is reduced. For a strain that performs only nitrate reduction (Nar strain), the model simplifies by setting <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Baseline comma gamma Subscript upper I Baseline equals 0" tabindex="0" ctxtmenu_counter="242" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (2 0 1) 3 (9 (6 4 5) 7 8))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="2,3,9" data-semantic-content="3" data-semantic-owns="2 3 9" data-semantic-speech="r Subscript upper I Baseline comma gamma Subscript upper I Baseline equals 0"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="10" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="6 7 8"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="9" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Likewise for a strain that performs only nitrite reduction (Nir strain), <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Baseline comma gamma Subscript upper A Baseline equals 0" tabindex="0" ctxtmenu_counter="243" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (2 0 1) 3 (9 (6 4 5) 7 8))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="2,3,9" data-semantic-content="3" data-semantic-owns="2 3 9" data-semantic-speech="r Subscript upper A Baseline comma gamma Subscript upper A Baseline equals 0"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="10" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="6 7 8"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="9" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. This model assumes that nitrate and/or nitrite are the growth-limiting nutrients and therefore the concentration of electron donors (e.g., succinate) is not explicitly modeled.</div></section><section id="sec-5-5-2"><h4>Inferring phenotypic parameters from data</h4><div id="p0515" role="paragraph">Monoculture experiments were performed for 78 soil isolates and one reference strain <i>Paracoccus denitrificans</i> ATCC 19367 under denitrifying conditions in succinate defined medium (SDM) in order to estimate parameters of the consumer-resource model (<a id="crosref1440" href="#fd3">Equation 3</a>). Monoculture experiments were also performed in acetate defined medium (ADM) for the 64 strains that grew aerobically on ADM (<a id="crosref1445" href="#mmc1">Table S1</a>).</div><section id="sec-5-5-2-1"><h5>Fitting yield parameters</h5><div id="p0520" role="paragraph">The yield parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="244" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and/or <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="245" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were directly measured for each strain from endpoint measurements of optical density and nitrate/nitrite concentration.</div><div id="p0525" role="paragraph">To demonstrate how this is possible, <a id="crosref1450" href="#fd3">Equation 3</a> gives:<div id="fd4" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartFraction d x Over d t EndFraction equals minus gamma Subscript upper A Baseline StartFraction d upper A Over d t EndFraction minus gamma Subscript upper I Baseline left parenthesis StartFraction d upper A Over d t EndFraction plus StartFraction d upper I Over d t EndFraction right parenthesis period" tabindex="0" ctxtmenu_counter="246" style="font-size: 130%;"><mjx-math data-semantic-structure="(58 (57 (8 (3 0 2 1) (7 4 6 5)) 9 (56 (53 10 (52 (13 11 12) 51 (22 (17 14 16 15) (21 18 20 19)))) 23 (55 (26 24 25) 54 (49 27 (47 (36 (31 28 30 29) (35 32 34 33)) 37 (46 (41 38 40 39) (45 42 44 43))) 48)))) 50)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="58" data-semantic-children="57,50" data-semantic-content="50" data-semantic-owns="57 50" data-semantic-speech="StartFraction d x Over d t EndFraction equals minus gamma Subscript upper A Baseline StartFraction d upper A Over d t EndFraction minus gamma Subscript upper I Baseline left parenthesis StartFraction d upper A Over d t EndFraction plus StartFraction d upper I Over d t EndFraction right parenthesis period"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="57" data-semantic-children="8,56" data-semantic-content="9" data-semantic-parent="58" data-semantic-owns="8 9 56"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="57" data-semantic-owns="3 7"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="3" data-semantic-children="0,1" data-semantic-content="2" data-semantic-parent="8" data-semantic-owns="0 2 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="2" data-semantic-parent="3" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="57" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="56" data-semantic-children="53,55" data-semantic-content="23" data-semantic-parent="57" data-semantic-owns="53 23 55"><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="53" data-semantic-children="52" data-semantic-content="10" data-semantic-parent="56" data-semantic-owns="10 52"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="10" data-semantic-parent="53" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="52" data-semantic-children="13,22" data-semantic-content="51" data-semantic-parent="53" data-semantic-owns="13 51 22"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="52" data-semantic-owns="11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="51" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="22" data-semantic-children="17,21" data-semantic-parent="52" data-semantic-owns="17 21"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="17" data-semantic-children="14,15" data-semantic-content="16" data-semantic-parent="22" data-semantic-owns="14 16 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="17"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="21" data-semantic-children="18,19" data-semantic-content="20" data-semantic-parent="22" data-semantic-owns="18 20 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="21"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="23" data-semantic-parent="56" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="55" data-semantic-children="26,49" data-semantic-content="54" data-semantic-parent="56" data-semantic-owns="26 54 49"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="26" data-semantic-children="24,25" data-semantic-parent="55" data-semantic-owns="24 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="26"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="54" data-semantic-parent="55" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="49" data-semantic-children="47" data-semantic-content="27,48" data-semantic-parent="55" data-semantic-owns="27 47 48"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="27" data-semantic-parent="49" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="47" data-semantic-children="36,46" data-semantic-content="37" data-semantic-parent="49" data-semantic-owns="36 37 46"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="36" data-semantic-children="31,35" data-semantic-parent="47" data-semantic-owns="31 35"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="31" data-semantic-children="28,29" data-semantic-content="30" data-semantic-parent="36" data-semantic-owns="28 30 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="31"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="31" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="35" data-semantic-children="32,33" data-semantic-content="34" data-semantic-parent="36" data-semantic-owns="32 34 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="35"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="34" data-semantic-parent="35" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="35"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="37" data-semantic-parent="47" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mfrac space="3" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="46" data-semantic-children="41,45" data-semantic-parent="47" data-semantic-owns="41 45"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="41" data-semantic-children="38,39" data-semantic-content="40" data-semantic-parent="46" data-semantic-owns="38 40 39"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="41"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="40" data-semantic-parent="41" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="41"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="45" data-semantic-children="42,43" data-semantic-content="44" data-semantic-parent="46" data-semantic-owns="42 44 43"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="45"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="44" data-semantic-parent="45" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="45"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="48" data-semantic-parent="49" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="50" data-semantic-parent="58" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">(Equation 4)</div></div></div><div id="p0530" role="paragraph">Therefore it follows from the fundamental theorem of calculus that:<div id="fd5" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartLayout 1st Row  integral Subscript 0 Superscript upper T Baseline StartFraction d x Over d t EndFraction d t equals x Subscript upper T Baseline minus x 0 equals minus gamma Subscript upper A Baseline integral Subscript 0 Superscript upper T Baseline StartFraction d upper A Over d t EndFraction d t minus gamma Subscript upper I Baseline left parenthesis integral Subscript 0 Superscript upper T Baseline StartFraction d upper A Over d t EndFraction d t plus integral Subscript 0 Superscript upper T Baseline StartFraction d upper I Over d t EndFraction d t right parenthesis 2nd Row  equals gamma Subscript upper A Baseline left parenthesis upper A 0 minus upper A Subscript upper T Baseline right parenthesis plus gamma Subscript upper I Baseline left parenthesis upper A 0 minus upper A Subscript upper T Baseline plus upper I 0 minus upper I Subscript upper T Baseline right parenthesis comma EndLayout" tabindex="0" ctxtmenu_counter="247" style="font-size: 130%;"><mjx-math data-semantic-structure="(151 (99 (97 (89 (3 0 1 2) (12 (7 4 6 5) (11 8 10 9)) (88 13 14)) 15 (90 (18 16 17) 19 (22 20 21)) 23 (96 (93 24 (92 (27 25 26) 91 (87 (31 28 29 30) (40 (35 32 34 33) (39 36 38 37)) (86 41 42)))) 43 (95 (46 44 45) 94 (85 83 (82 (81 (50 47 48 49) (59 (54 51 53 52) (58 55 57 56)) (80 60 61)) 62 (79 (66 63 64 65) (75 (70 67 69 68) (74 71 73 72)) (78 76 77))) 84))))) (150 (148 (147 141 100 (146 (143 (103 101 102) 142 (114 112 (111 (106 104 105) 107 (110 108 109)) 113)) 115 (145 (118 116 117) 144 (139 137 (136 (135 (134 (121 119 120) 122 (125 123 124)) 126 (129 127 128)) 130 (133 131 132)) 138)))) 140)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mtable data-semantic-type="multiline" data-semantic-role="binomial" data-semantic-id="151" data-semantic-children="99,150" data-semantic-owns="99 150" data-semantic-speech="StartLayout 1st Row  integral Subscript 0 Superscript upper T Baseline StartFraction d x Over d t EndFraction d t equals x Subscript upper T Baseline minus x 0 equals minus gamma Subscript upper A Baseline integral Subscript 0 Superscript upper T Baseline StartFraction d upper A Over d t EndFraction d t minus gamma Subscript upper I Baseline left parenthesis integral Subscript 0 Superscript upper T Baseline StartFraction d upper A Over d t EndFraction d t plus integral Subscript 0 Superscript upper T Baseline StartFraction d upper I Over d t EndFraction d t right parenthesis 2nd Row  equals gamma Subscript upper A Baseline left parenthesis upper A 0 minus upper A Subscript upper T Baseline right parenthesis plus gamma Subscript upper I Baseline left parenthesis upper A 0 minus upper A Subscript upper T Baseline plus upper I 0 minus upper I Subscript upper T Baseline right parenthesis comma EndLayout" justify="left" style="min-width: 25.365em;"><mjx-table><mjx-itable><mjx-mtr data-semantic-type="line" data-semantic-role="binomial" data-semantic-id="99" data-semantic-children="97" data-semantic-parent="151" data-semantic-owns="97"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="97" data-semantic-children="89,90,96" data-semantic-content="15,23" data-semantic-parent="99" data-semantic-owns="89 15 90 23 96"><mjx-mrow data-semantic-added="true" data-semantic-type="integral" data-semantic-role="integral" data-semantic-id="89" data-semantic-children="3,12,88" data-semantic-content="0" data-semantic-parent="97" data-semantic-owns="3 12 88"><mjx-munderover data-semantic-type="limboth" data-semantic-role="integral" data-semantic-id="3" data-semantic-children="0,1,2" data-semantic-parent="89" data-semantic-owns="0 1 2"><mjx-over style="padding-bottom: 0.2em; padding-left: 0.174em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-over><mjx-box><mjx-munder><mjx-row><mjx-base><mjx-mo data-semantic-type="largeop" data-semantic-role="integral" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="integral"><mjx-c class="mjx-c222B">∫</mjx-c></mjx-mo></mjx-base></mjx-row><mjx-row><mjx-under style="padding-top: 0.167em; padding-left: 0.066em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-under></mjx-row></mjx-munder></mjx-box></mjx-munderover><mjx-mfrac space="2" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="12" data-semantic-children="7,11" data-semantic-parent="89" data-semantic-owns="7 11"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="11" data-semantic-children="8,9" data-semantic-content="10" data-semantic-parent="12" data-semantic-owns="8 10 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="integral" data-semantic-font="normal" data-semantic-id="88" data-semantic-children="14" data-semantic-content="13" data-semantic-parent="89" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-id="13" data-semantic-parent="88" data-semantic-operator="prefixop,d"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="88"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="15" data-semantic-parent="97" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="90" data-semantic-children="18,22" data-semantic-content="19" data-semantic-parent="97" data-semantic-owns="18 19 22"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="18" data-semantic-children="16,17" data-semantic-parent="90" data-semantic-owns="16 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="18"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="18"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="19" data-semantic-parent="90" data-semantic-operator="infixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="22" data-semantic-children="20,21" data-semantic-parent="90" data-semantic-owns="20 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="22"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="22"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="23" data-semantic-parent="97" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="96" data-semantic-children="93,95" data-semantic-content="43" data-semantic-parent="97" data-semantic-owns="93 43 95"><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="93" data-semantic-children="92" data-semantic-content="24" data-semantic-parent="96" data-semantic-owns="24 92"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="24" data-semantic-parent="93" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="92" data-semantic-children="27,87" data-semantic-content="91" data-semantic-parent="93" data-semantic-owns="27 91 87"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="27" data-semantic-children="25,26" data-semantic-parent="92" data-semantic-owns="25 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="27"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="27"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="91" data-semantic-parent="92" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="integral" data-semantic-role="integral" data-semantic-id="87" data-semantic-children="31,40,86" data-semantic-content="28" data-semantic-parent="92" data-semantic-owns="31 40 86"><mjx-munderover data-semantic-type="limboth" data-semantic-role="integral" data-semantic-id="31" data-semantic-children="28,29,30" data-semantic-parent="87" data-semantic-owns="28 29 30"><mjx-over style="padding-bottom: 0.2em; padding-left: 0.174em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-over><mjx-box><mjx-munder><mjx-row><mjx-base><mjx-mo data-semantic-type="largeop" data-semantic-role="integral" data-semantic-id="28" data-semantic-parent="31" data-semantic-operator="integral"><mjx-c class="mjx-c222B">∫</mjx-c></mjx-mo></mjx-base></mjx-row><mjx-row><mjx-under style="padding-top: 0.167em; padding-left: 0.066em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-under></mjx-row></mjx-munder></mjx-box></mjx-munderover><mjx-mfrac space="2" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="40" data-semantic-children="35,39" data-semantic-parent="87" data-semantic-owns="35 39"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="35" data-semantic-children="32,33" data-semantic-content="34" data-semantic-parent="40" data-semantic-owns="32 34 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="35"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="34" data-semantic-parent="35" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="35"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="39" data-semantic-children="36,37" data-semantic-content="38" data-semantic-parent="40" data-semantic-owns="36 38 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="39"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="39"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="integral" data-semantic-font="normal" data-semantic-id="86" data-semantic-children="42" data-semantic-content="41" data-semantic-parent="87" data-semantic-owns="41 42"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-id="41" data-semantic-parent="86" data-semantic-operator="prefixop,d"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="86"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="43" data-semantic-parent="96" data-semantic-operator="infixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="95" data-semantic-children="46,85" data-semantic-content="94" data-semantic-parent="96" data-semantic-owns="46 94 85"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="46" data-semantic-children="44,45" data-semantic-parent="95" data-semantic-owns="44 45"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="46"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="46"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="94" data-semantic-parent="95" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="85" data-semantic-children="82" data-semantic-content="83,84" data-semantic-parent="95" data-semantic-owns="83 82 84"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="83" data-semantic-parent="85" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S7">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="82" data-semantic-children="81,79" data-semantic-content="62" data-semantic-parent="85" data-semantic-owns="81 62 79"><mjx-mrow data-semantic-added="true" data-semantic-type="integral" data-semantic-role="integral" data-semantic-id="81" data-semantic-children="50,59,80" data-semantic-content="47" data-semantic-parent="82" data-semantic-owns="50 59 80"><mjx-munderover data-semantic-type="limboth" data-semantic-role="integral" data-semantic-id="50" data-semantic-children="47,48,49" data-semantic-parent="81" data-semantic-owns="47 48 49"><mjx-over style="padding-bottom: 0.2em; padding-left: 0.174em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="50"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-over><mjx-box><mjx-munder><mjx-row><mjx-base><mjx-mo data-semantic-type="largeop" data-semantic-role="integral" data-semantic-id="47" data-semantic-parent="50" data-semantic-operator="integral"><mjx-c class="mjx-c222B">∫</mjx-c></mjx-mo></mjx-base></mjx-row><mjx-row><mjx-under style="padding-top: 0.167em; padding-left: 0.066em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-parent="50"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-under></mjx-row></mjx-munder></mjx-box></mjx-munderover><mjx-mfrac space="2" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="59" data-semantic-children="54,58" data-semantic-parent="81" data-semantic-owns="54 58"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="54" data-semantic-children="51,52" data-semantic-content="53" data-semantic-parent="59" data-semantic-owns="51 53 52"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="54"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="53" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="54"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="58" data-semantic-children="55,56" data-semantic-content="57" data-semantic-parent="59" data-semantic-owns="55 57 56"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="55" data-semantic-parent="58"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="57" data-semantic-parent="58" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="56" data-semantic-parent="58"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="integral" data-semantic-font="normal" data-semantic-id="80" data-semantic-children="61" data-semantic-content="60" data-semantic-parent="81" data-semantic-owns="60 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-id="60" data-semantic-parent="80" data-semantic-operator="prefixop,d"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="80"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="62" data-semantic-parent="82" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="integral" data-semantic-role="integral" data-semantic-id="79" data-semantic-children="66,75,78" data-semantic-content="63" data-semantic-parent="82" data-semantic-owns="66 75 78"><mjx-munderover data-semantic-type="limboth" data-semantic-role="integral" data-semantic-id="66" data-semantic-children="63,64,65" data-semantic-parent="79" data-semantic-owns="63 64 65"><mjx-over style="padding-bottom: 0.2em; padding-left: 0.174em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="65" data-semantic-parent="66"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-over><mjx-box><mjx-munder><mjx-row><mjx-base><mjx-mo data-semantic-type="largeop" data-semantic-role="integral" data-semantic-id="63" data-semantic-parent="66" data-semantic-operator="integral"><mjx-c class="mjx-c222B">∫</mjx-c></mjx-mo></mjx-base></mjx-row><mjx-row><mjx-under style="padding-top: 0.167em; padding-left: 0.066em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="66"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-under></mjx-row></mjx-munder></mjx-box></mjx-munderover><mjx-mfrac space="2" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="75" data-semantic-children="70,74" data-semantic-parent="79" data-semantic-owns="70 74"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="70" data-semantic-children="67,68" data-semantic-content="69" data-semantic-parent="75" data-semantic-owns="67 69 68"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="67" data-semantic-parent="70"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="69" data-semantic-parent="70" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="68" data-semantic-parent="70"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="74" data-semantic-children="71,72" data-semantic-content="73" data-semantic-parent="75" data-semantic-owns="71 73 72"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="71" data-semantic-parent="74"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="73" data-semantic-parent="74" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="74"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="integral" data-semantic-font="normal" data-semantic-id="78" data-semantic-children="77" data-semantic-content="76" data-semantic-parent="79" data-semantic-owns="76 77"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-id="76" data-semantic-parent="78" data-semantic-operator="prefixop,d"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="77" data-semantic-parent="78"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="84" data-semantic-parent="85" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S7">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="binomial" data-semantic-id="150" data-semantic-children="148" data-semantic-parent="151" data-semantic-owns="148"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="148" data-semantic-children="147,140" data-semantic-content="140" data-semantic-parent="150" data-semantic-owns="147 140"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="147" data-semantic-children="141,146" data-semantic-content="100" data-semantic-parent="148" data-semantic-owns="141 100 146"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="141" data-semantic-parent="147"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="100" data-semantic-parent="147" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="146" data-semantic-children="143,145" data-semantic-content="115" data-semantic-parent="147" data-semantic-owns="143 115 145"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="143" data-semantic-children="103,114" data-semantic-content="142" data-semantic-parent="146" data-semantic-owns="103 142 114"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="103" data-semantic-children="101,102" data-semantic-parent="143" data-semantic-owns="101 102"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="101" data-semantic-parent="103"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="102" data-semantic-parent="103"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="142" data-semantic-parent="143" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="114" data-semantic-children="111" data-semantic-content="112,113" data-semantic-parent="143" data-semantic-owns="112 111 113"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="112" data-semantic-parent="114" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="111" data-semantic-children="106,110" data-semantic-content="107" data-semantic-parent="114" data-semantic-owns="106 107 110"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="106" data-semantic-children="104,105" data-semantic-parent="111" data-semantic-owns="104 105"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="104" data-semantic-parent="106"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="105" data-semantic-parent="106"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="107" data-semantic-parent="111" data-semantic-operator="infixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="110" data-semantic-children="108,109" data-semantic-parent="111" data-semantic-owns="108 109"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="108" data-semantic-parent="110"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="109" data-semantic-parent="110"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="113" data-semantic-parent="114" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="115" data-semantic-parent="146" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="145" data-semantic-children="118,139" data-semantic-content="144" data-semantic-parent="146" data-semantic-owns="118 144 139"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="118" data-semantic-children="116,117" data-semantic-parent="145" data-semantic-owns="116 117"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="116" data-semantic-parent="118"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="117" data-semantic-parent="118"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="144" data-semantic-parent="145" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="139" data-semantic-children="136" data-semantic-content="137,138" data-semantic-parent="145" data-semantic-owns="137 136 138"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="137" data-semantic-parent="139" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="136" data-semantic-children="135,133" data-semantic-content="130" data-semantic-parent="139" data-semantic-owns="135 130 133"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="135" data-semantic-children="134,129" data-semantic-content="126" data-semantic-parent="136" data-semantic-owns="134 126 129"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="134" data-semantic-children="121,125" data-semantic-content="122" data-semantic-parent="135" data-semantic-owns="121 122 125"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="121" data-semantic-children="119,120" data-semantic-parent="134" data-semantic-owns="119 120"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="119" data-semantic-parent="121"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="120" data-semantic-parent="121"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="122" data-semantic-parent="134" data-semantic-operator="infixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="125" data-semantic-children="123,124" data-semantic-parent="134" data-semantic-owns="123 124"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="123" data-semantic-parent="125"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="124" data-semantic-parent="125"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="126" data-semantic-parent="135" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="129" data-semantic-children="127,128" data-semantic-parent="135" data-semantic-owns="127 128"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="127" data-semantic-parent="129"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="128" data-semantic-parent="129"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="130" data-semantic-parent="136" data-semantic-operator="infixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="133" data-semantic-children="131,132" data-semantic-parent="136" data-semantic-owns="131 132"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="131" data-semantic-parent="133"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="132" data-semantic-parent="133"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="138" data-semantic-parent="139" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="140" data-semantic-parent="148" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr></mjx-itable></mjx-table></mjx-mtable></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">(Equation 5)</div></div>where the subscripts 0 and <i>T</i> denote the values of the variables at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t equals 0" tabindex="0" ctxtmenu_counter="248" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t equals 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and <i>T</i>, respectively.</div><div id="p0535" role="paragraph"><a id="crosref1455" href="#fd5">Equation 5</a> indicates that <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="249" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="250" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> can be inferred using measurements of biomass density and nitrate/nitrite concentrations at some time <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t equals upper T" tabindex="0" ctxtmenu_counter="251" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t equals upper T"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, given knowledge of the experimentally-specified initial values of these variables. Because this equation is linear with up to two unknowns (recall that <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I Baseline equals 0" tabindex="0" ctxtmenu_counter="252" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 (2 0 1) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="2,4" data-semantic-content="3" data-semantic-owns="2 3 4" data-semantic-speech="gamma Subscript upper I Baseline equals 0"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="5" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> for Nar strains and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A Baseline equals 0" tabindex="0" ctxtmenu_counter="253" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 (2 0 1) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="2,4" data-semantic-content="3" data-semantic-owns="2 3 4" data-semantic-speech="gamma Subscript upper A Baseline equals 0"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="5" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> for Nir strains), fitting this equation could in principle be accomplished using data from a single experimental condition for Nar and Nir strains or two experimental conditions for Nar/Nir strains. Instead <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="254" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="255" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were fit to data from many conditions in order to obtain more robust estimates. Specifically, monocultures were grown in the following conditions: (a) 2 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O Subscript italic asterisk Superscript minus" tabindex="0" ctxtmenu_counter="256" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O Subscript italic asterisk Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.225em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, OD600<sub>0</sub>&nbsp;= 0.01, (b) 1 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O Subscript italic asterisk Superscript minus" tabindex="0" ctxtmenu_counter="257" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O Subscript italic asterisk Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.225em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, OD600<sub>0</sub>&nbsp;= 0.01, (c) 2 mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O Subscript italic asterisk Superscript minus" tabindex="0" ctxtmenu_counter="258" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O Subscript italic asterisk Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.225em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, OD600<sub>0</sub>&nbsp;= 0.001, where Nar/Nir strains were cultured with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="259" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="260" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> separately (6 conditions total), Nar strains were cultured with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="261" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> (3 conditions total), and Nir strains were cultured with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="262" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> (3 conditions total). Then the equation<div id="fd6" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="OD 600 Subscript 64 Baseline minus OD 600 Subscript 0 Baseline equals gamma 0 plus gamma Subscript upper A Baseline left parenthesis upper A 0 minus upper A 64 right parenthesis plus gamma Subscript upper I Baseline left parenthesis upper A 0 minus upper A 64 plus upper I 0 minus upper I 64 right parenthesis comma" tabindex="0" ctxtmenu_counter="263" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="65" data-semantic-children="64,53" data-semantic-content="53" data-semantic-owns="64 53" data-semantic-structure="(65 (64 0 (54 (3 1 2) 4) 5 (60 (8 6 7) 9 (57 (12 10 11) 13 (56 (16 14 15) 55 (27 25 (24 (19 17 18) 20 (23 21 22)) 26)) 28 (59 (31 29 30) 58 (52 50 (49 (48 (47 (34 32 33) 35 (38 36 37)) 39 (42 40 41)) 43 (46 44 45)) 51))))) 53)" data-semantic-speech="OD 600 Subscript 64 Baseline minus OD 600 Subscript 0 Baseline equals gamma 0 plus gamma Subscript upper A Baseline left parenthesis upper A 0 minus upper A 64 right parenthesis plus gamma Subscript upper I Baseline left parenthesis upper A 0 minus upper A 64 plus upper I 0 minus upper I 64 right parenthesis comma" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="64" data-semantic-children="0,54,5,60" data-semantic-parent="65" data-semantic-collapsed="(64 (c 61 62 63) 0 54 5 60)" data-semantic-owns="0 54 5 60" inline-breaks="true"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="64"><mjx-c class="mjx-c4F" noic="true" style="padding-top: 0.705em;">O</mjx-c><mjx-c class="mjx-c44" style="padding-top: 0.705em;">D</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="postfixop" data-semantic-role="subtraction" data-semantic-id="54" data-semantic-children="3" data-semantic-content="4" data-semantic-parent="64" data-semantic-owns="3 4" inline-breaks="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="integer" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="54" data-semantic-owns="1 2"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.666em;">6</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.677em;">6</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-break size="0"></mjx-break><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-parent="54" data-semantic-operator="postfixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="5" data-semantic-parent="64"><mjx-c class="mjx-c4F" noic="true" style="padding-top: 0.705em;">O</mjx-c><mjx-c class="mjx-c44" style="padding-top: 0.705em;">D</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="60" data-semantic-children="8,57" data-semantic-content="9" data-semantic-parent="64" data-semantic-owns="8 9 57" inline-breaks="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="integer" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="60" data-semantic-owns="6 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.666em;">6</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="60" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="57" data-semantic-children="12,56,59" data-semantic-content="13,28" data-semantic-parent="60" data-semantic-owns="12 13 56 28 59" inline-breaks="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="57" data-semantic-owns="10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-break size="3"></mjx-break><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="13" data-semantic-parent="57" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="56" data-semantic-children="16,27" data-semantic-content="55" data-semantic-parent="57" data-semantic-owns="16 55 27"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="56" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="55" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="27" data-semantic-children="24" data-semantic-content="25,26" data-semantic-parent="56" data-semantic-owns="25 24 26"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="25" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="24" data-semantic-children="19,23" data-semantic-content="20" data-semantic-parent="27" data-semantic-owns="19 20 23"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="19" data-semantic-children="17,18" data-semantic-parent="24" data-semantic-owns="17 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="19"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="20" data-semantic-parent="24" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="23" data-semantic-children="21,22" data-semantic-parent="24" data-semantic-owns="21 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="23"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="23"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.677em;">6</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-break size="3"></mjx-break><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="28" data-semantic-parent="57" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="59" data-semantic-children="31,52" data-semantic-content="58" data-semantic-parent="57" data-semantic-owns="31 58 52"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="31" data-semantic-children="29,30" data-semantic-parent="59" data-semantic-owns="29 30"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="58" data-semantic-parent="59" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="52" data-semantic-children="49" data-semantic-content="50,51" data-semantic-parent="59" data-semantic-owns="50 49 51"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="50" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="49" data-semantic-children="48,46" data-semantic-content="43" data-semantic-parent="52" data-semantic-owns="48 43 46"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="48" data-semantic-children="47,42" data-semantic-content="39" data-semantic-parent="49" data-semantic-owns="47 39 42"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="47" data-semantic-children="34,38" data-semantic-content="35" data-semantic-parent="48" data-semantic-owns="34 35 38"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="34" data-semantic-children="32,33" data-semantic-parent="47" data-semantic-owns="32 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="34"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="34"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="35" data-semantic-parent="47" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="38" data-semantic-children="36,37" data-semantic-parent="47" data-semantic-owns="36 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="38"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="38"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.677em;">6</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="39" data-semantic-parent="48" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="42" data-semantic-children="40,41" data-semantic-parent="48" data-semantic-owns="40 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="42"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="42"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="43" data-semantic-parent="49" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="46" data-semantic-children="44,45" data-semantic-parent="49" data-semantic-owns="44 45"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="46"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="46"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.677em;">6</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="51" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="53" data-semantic-parent="65" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-math></mjx-container></div></div><div class="label">(Equation 6)</div></div>was fit via ordinary least squares regression using endpoint measurements from all conditions taken at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t equals" tabindex="0" ctxtmenu_counter="264" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 64 h. The intercept term <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma 0" tabindex="0" ctxtmenu_counter="265" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> was included in <a id="crosref1460" href="#fd6">Equation 6</a> to account for small systematic measurement errors, and was typically close to zero (<a id="crosref1465" href="#figs1">Figure&nbsp;S1</a>A). Examples of fits to <a id="crosref1470" href="#fd6">Equation 6</a> are shown in <a id="crosref1475" href="#figs1">Figures S1</a>B–S1D.</div><div id="p0540" role="paragraph">Parameterizing <a id="crosref1480" href="#fd3">Equation 3</a> in terms of OD assumed that biomass (g dry weight/L) is proportional to OD600, and that the constant of proportionality is the same for all strains in the library. Previous literature reports that the ratio g dry weight/L/OD is typically 0.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Coultate and Sundaram, 1975</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Coultate, T.P. ∙ Sundaram, T.K.</div><div><strong>Energetics of <i>Bacillus stearothermophilus</i> growth: molar growth yield and temperature effects on growth efficiency</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1975; <strong>121</strong>:55-64</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.121.1.55-64.1975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1116992/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.121.1.55-64.1975&amp;pmid=1116992" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Rosenberger and Elsden, 1960</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Rosenberger, R.F. ∙ Elsden, S.R.</div><div><strong>The yields of Streptococcus faecalis grown in continuous culture</strong></div><div><em>J.&nbsp;Gen. Microbiol.</em> 1960; <strong>22</strong>:726-739</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1099/00221287-22-3-726" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14438964/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1099%2F00221287-22-3-726&amp;pmid=14438964" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Hassen et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Hassen, A. ∙ Saidi, N. ∙ Cherif, M. ...</div><div><strong>Effects of heavy metals on <i>Pseudomonas aeruginosa</i> and <i>Bacillus thuringiensis</i></strong></div><div><em>Bioresour. Technol.</em> 1998; <strong>65</strong>:73-82</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0960-8524(98)00011-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0032126284" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0960-8524%2898%2900011-X" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Péquignot et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Péquignot, C. ∙ Larroche, C. ∙ Gros, J.B.</div><div><strong>A spectrophotometric method for determination of bacterial biomass in the presence of a polymer</strong></div><div><em>Biotechnol. Tech.</em> 1998; <strong>12</strong>:899-903</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1008865512272" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0032436997" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1008865512272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Nikel et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Nikel, P.I. ∙ Fuhrer, T. ∙ Chavarría, M. ...</div><div><strong>Reconfiguration of metabolic fluxes in <i>Pseudomonas putida</i> as a response to sub-lethal oxidative stress</strong></div><div><em>ISME J.</em> 2021; <strong>15</strong>:1751-1766</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41396-020-00884-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85100198995" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33432138/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41396-020-00884-9&amp;pmid=33432138" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Lee et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Lee, K.H. ∙ Park, J.H. ∙ Kim, T.Y. ...</div><div><strong>Systems metabolic engineering of <i>Escherichia coli</i> for L-threonine production</strong></div><div><em>Mol. Syst. Biol.</em> 2007; <strong>3</strong>:149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/msb4100196" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-36849002434" target="_blank" aria-label="Open Scopus in new tab">Scopus (345)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18059444/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmsb4100196&amp;pmid=18059444" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In order to test whether some strains might secrete optically active compounds that absorb strongly at 600 nm, and thereby corrupt the biomass to OD conversion, four representative strains (<i>Achromobacter</i> sp. ACM01, <i>Ensifer</i> sp. ENS09, <i>Paracoccus denitrificans</i> ATCC 19367, and <i>Pseudomonas</i> sp. PDM21) were cultured in SDM under denitrifying conditions for 64 h. Cells were then removed from the endpoint cultures by filtration and OD600 of the filtered medium was measured. It was observed that absorbance levels were at or below that of fresh medium. It was therefore concluded that secretion of optically active molecules likely does not impact the inference of biomass from OD, and that using OD600 as a proxy for biomass incurs only a modest error. However, it is important to recognize that directly measuring the biomass of each strain in the library may improve the quality of model fits to data and alter regression results.</div></section><section id="sec-5-5-2-2"><h5>Fitting rate parameters</h5><div id="p0545" role="paragraph">For each strain, having determined the yield parameters, the rate parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="266" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="267" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were then globally (simultaneously) fit to nitrate/nitrite dynamics data across all experimental conditions by minimizing the sum of squared residuals between the data and numerical solutions to <a id="crosref1515" href="#fd3">Equation 3</a> (see example in <a id="crosref1520" href="#figs1">Figure&nbsp;S1</a>E). <a id="crosref1525" href="#fd3">Equation 3</a> was solved numerically using the differential equation solver ode23s in MATLAB R2017b. The solver was initialized at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t equals t 1" tabindex="0" ctxtmenu_counter="268" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="t equals t 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t 1" tabindex="0" ctxtmenu_counter="269" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="t 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the time point of first nitrate/nitrite measurement, approximately 2 h), setting initial conditions <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N left parenthesis t 1 right parenthesis equals OD 600 Subscript 0 Baseline" tabindex="0" ctxtmenu_counter="270" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="18" data-semantic-children="15,8,11" data-semantic-collapsed="(18 (c 16 17) 15 8 11)" data-semantic-owns="15 8 11" data-semantic-structure="(18 (15 (13 0 12 (6 4 (3 1 2) 5)) 7 14) 8 (11 9 10))" data-semantic-speech="upper N left parenthesis t 1 right parenthesis equals OD 600 Subscript 0 Baseline" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="15" data-semantic-children="13,14" data-semantic-content="7" data-semantic-parent="18" data-semantic-owns="13 7 14" inline-breaks="true"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="13" data-semantic-children="0,6" data-semantic-content="12,0" data-semantic-parent="15" data-semantic-owns="0 12 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="13" data-semantic-operator="appl"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="3" data-semantic-content="4,5" data-semantic-parent="13" data-semantic-owns="4 3 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="6" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="15" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="14" data-semantic-parent="15"></mjx-mrow></mjx-mrow><mjx-mtext space="4" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-parent="18"><mjx-c class="mjx-c4F" noic="true" style="padding-top: 0.705em;">O</mjx-c><mjx-c class="mjx-c44" style="padding-top: 0.705em;">D</mjx-c></mjx-mtext><mjx-msub data-semantic-type="subscript" data-semantic-role="integer" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="18" data-semantic-owns="9 10"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.666em;">6</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A left parenthesis t 1 right parenthesis" tabindex="0" ctxtmenu_counter="271" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 7 (6 1 (4 2 3) 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="8" data-semantic-children="0,6" data-semantic-content="7,0" data-semantic-owns="0 7 6" data-semantic-speech="upper A left parenthesis t 1 right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="8" data-semantic-operator="appl"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="1,5" data-semantic-parent="8" data-semantic-owns="1 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="6" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I left parenthesis t 1 right parenthesis" tabindex="0" ctxtmenu_counter="272" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 7 (6 1 (4 2 3) 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="8" data-semantic-children="0,6" data-semantic-content="7,0" data-semantic-owns="0 7 6" data-semantic-speech="upper I left parenthesis t 1 right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="8" data-semantic-operator="appl"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="1,5" data-semantic-parent="8" data-semantic-owns="1 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="6" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> to the median measured values of nitrate and nitrite concentration at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t equals t 1" tabindex="0" ctxtmenu_counter="273" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="t equals t 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> over experimental replicates within a given condition. The constrained optimization function fmincon was used in conjunction with the global minimum search function GlobalSearch to minimize the sum of squared residuals and obtain optimal values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="274" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="275" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. Values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="276" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="277" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were constrained between 0 and 50&nbsp;mM/OD/h.</div><div id="p0550" role="paragraph">Initially, this approach was used to simultaneously fit both rates (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="278" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="279" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>) and substrate affinities (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="280" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="281" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>), but it was observed in nonparametric bootstrap estimates of parameter error that the affinities <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="282" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="283" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were not well constrained by experimental measurements (<a id="crosref1530" href="#figs1">Figure&nbsp;S1</a>F, median fractional errors of 30.0% and 26.5% for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="284" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="285" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> in SDM, respectively). This is likely because the true values are small relative to the typical scale of substrate concentrations in the experiments: values for nitrate/nitrite affinity constants between 0.003–0.055 mM have been identified for denitrifying cultures previously (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Beccari et&nbsp;al., 1983</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Beccari, M. ∙ Passino, R. ∙ Ramadori, R. ...</div><div><strong>Kinetics of dissimilatory nitrate and nitrite reduction in suspended growth culture</strong></div><div><em>J.&nbsp;Water Pollut. Control Fed.</em> 1983; <strong>55</strong>:58-64</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BeccariR.PassinoR.RamadoriV.TandoiKinetics+of+dissimilatory+nitrate+and+nitrite+reduction+in+suspended+growth+cultureJ.%C2%A0Water+Pollut.+Control+Fed.5519835864" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Claus and Kutzner, 1985</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Claus, G. ∙ Kutzner, H.</div><div><strong>Physiology and kinetics of autotrophic denitrification by <i>Thiobacillus denitrificans</i></strong></div><div><em>Appl. Microbiol. Biotechnol.</em> 1985; <strong>22</strong>:283-288</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.ClausH.KutznerPhysiology+and+kinetics+of+autotrophic+denitrification+by+Thiobacillus+denitrificansAppl.+Microbiol.+Biotechnol.221985283288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Kornaros et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Kornaros, M. ∙ Zafiri, C. ∙ Lyberatos, G.</div><div><strong>Kinetics of denitrification by <i>Pseudomonas denitrificans</i> under growth conditions limited by carbon and/or nitrate or nitrite</strong></div><div><em>Water Environ. Res.</em> 1996; <strong>68</strong>:934-945</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2175/106143096X127947" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0030200160" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2175%2F106143096X127947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Dinçer and Kargı, 2000</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Dinçer, A.R. ∙ Kargı, F.</div><div><strong>Kinetics of sequential nitrification and denitrification processes</strong></div><div><em>Enzyme Microb. Technol.</em> 2000; <strong>27</strong>:37-42</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0141-0229(00)00145-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0342264575" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10862900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0141-0229%2800%2900145-9&amp;pmid=10862900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while cultures were initialized with 1–2 mM nitrate/nitrite. In the dynamical regimes of <a id="crosref1555" href="#fd3">Equation 3</a> where substrate concentrations are much greater than affinity constants (i.e., <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A much greater than upper K Subscript upper A" tabindex="0" ctxtmenu_counter="286" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="upper A much greater than upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,≫"><mjx-c class="mjx-c226B">≫</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I much greater than upper K Subscript upper I" tabindex="0" ctxtmenu_counter="287" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="upper I much greater than upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,≫"><mjx-c class="mjx-c226B">≫</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>), the dependence of <a id="crosref1560" href="#fd3">Equation 3</a> on <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="288" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="289" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> vanishes. This restricts identifiability of the affinity parameters to the regime where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A comma upper I" tabindex="0" ctxtmenu_counter="290" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="3" data-semantic-children="0,1,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper A comma upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> are on the same scale as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A Baseline comma upper K Subscript upper I Baseline" tabindex="0" ctxtmenu_counter="291" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 (2 0 1) 3 (6 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="2,3,6" data-semantic-content="3" data-semantic-owns="2 3 6" data-semantic-speech="upper K Subscript upper A Baseline comma upper K Subscript upper I Baseline"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="7" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="7" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (likely in the micromolar range). This regime typically occurs only briefly during the experiments, just before the substrates are exhausted, and sampling this regime for all strains would require a much greater sampling frequency than what was attempted in this study. Alternatively, the affinity parameters could be measured by comparing differences in growth/reduction rates at very low substrate concentrations, so that growth occurs in a regime where these rates depend more strongly on the affinity parameters (i.e., <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A 0 much less than upper K Subscript upper A" tabindex="0" ctxtmenu_counter="292" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 (2 0 1) 3 (6 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="2,6" data-semantic-content="3" data-semantic-owns="2 3 6" data-semantic-speech="upper A 0 much less than upper K Subscript upper A"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="7" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="3" data-semantic-parent="7" data-semantic-operator="relseq,≪"><mjx-c class="mjx-c226A">≪</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="7" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I 0 much less than upper K Subscript upper I" tabindex="0" ctxtmenu_counter="293" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 (2 0 1) 3 (6 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="2,6" data-semantic-content="3" data-semantic-owns="2 3 6" data-semantic-speech="upper I 0 much less than upper K Subscript upper I"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="7" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="3" data-semantic-parent="7" data-semantic-operator="relseq,≪"><mjx-c class="mjx-c226A">≪</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="7" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>). Doing this would require much more sensitive measurement of biomass density or nitrate/nitrite concentration than what was attempted in this study. Regardless, it was observed in nonparametric bootstrap estimates of parameter error that fitted values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="294" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="295" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were insensitive to <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="296" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="297" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<a id="crosref1565" href="#figs1">Figure&nbsp;S1</a>G, median fractional errors of 3.8% and 4.2% for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="298" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="299" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> in SDM, respectively), as was the qualitative model behavior (<a id="crosref1570" href="#figs1">Figure&nbsp;S1</a>H) and the fit quality (<a id="crosref1575" href="#figs1">Figure&nbsp;S1</a>I). Therefore <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A" tabindex="0" ctxtmenu_counter="300" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper I" tabindex="0" ctxtmenu_counter="301" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were fixed to 0.01 mM.</div></section><section id="sec-5-5-2-3"><h5>Treatment of poorly-fitting strains</h5><div id="p0555" role="paragraph">By fitting metabolite dynamics across multiple initial media and biomass density conditions to the model <a id="crosref1580" href="#fd3">Equation 3</a>, it was assumed that this simple model is a good representation of the metabolic phenotypes of denitrifying strains, with a few parameters capturing the full extent of the metabolite dynamics. In the majority of cases it was observed that the model fit the metabolite dynamics well (see example in <a id="crosref1585" href="#figs1">Figure&nbsp;S1</a>E), indicating that the assumptions of model <a id="crosref1590" href="#fd3">Equation 3</a> were appropriate. However in some cases (e.g., <a id="crosref1595" href="#figs1">Figures S1</a>J and S1K) the observed metabolite dynamics departed significantly from the optimal model fits. Often it appeared that different parameterizations were necessary for different initial conditions, e.g., nitrite reduction rates appeared to differ depending on whether nitrite was initially supplied (for example <a id="crosref1600" href="#figs1">Figure&nbsp;S1</a>J). In these cases, while the model apparently failed to capture the metabolite dynamics phenotypes across all conditions simultaneously, it appeared that it was still possible to estimate consumer-resource parameters for a subset of initial conditions.</div><div id="p0560" role="paragraph">Therefore a pipeline was developed for parameterizing the model <a id="crosref1605" href="#fd3">Equation 3</a> on a subset of experimental conditions if poor fits were obtained when fitting to all conditions. First yields on nitrate and nitrite were fit using all experimental conditions. Then the pipeline proceeded as follows for each strain:<div id="olist0015" role="list"><div id="o0025" role="listitem"><div class="label">1.</div><div class="content"><div id="p0565" role="paragraph">Fit reduction rates to all experimental conditions. If the root-mean-square error (RMSE) evaluated between observed data and model solutions across all experimental conditions and replicates exceeds a threshold <i>T</i>, then proceed to step 2. If not, accept the parameter fit.</div></div></div><div id="o0030" role="listitem"><div class="label">2.</div><div class="content"><div id="p0570" role="paragraph">Fit reduction rates only to experimental conditions where OD600<sub>0</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 01" tabindex="0" ctxtmenu_counter="302" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 01"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>. If the RMSE evaluated across these high biomass-density conditions exceeds <i>T</i>, proceed to step 3. If not, accept the parameter fit.</div></div></div><div id="o0035" role="listitem"><div class="label">3.</div><div class="content"><div id="p0575" role="paragraph">(Nar/Nir strains only) Fit reduction rates only to experimental conditions where nitrate was initially supplied. If the RMSE evaluated across these conditions exceeds <i>T</i>, then proceed to step 4. If not, accept the parameter fit.</div></div></div><div id="o0040" role="listitem"><div class="label">4.</div><div class="content"><div id="p0580" role="paragraph">(Nar/Nir strains only) Fit reduction rates only to experimental conditions where OD600<sub>0</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 01" tabindex="0" ctxtmenu_counter="303" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 01"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and nitrate was initially supplied. If the RMSE evaluted across these conditions is less than or equal to <i>T</i>, accept the parameter fit.</div></div></div></div></div><div id="p0585" role="paragraph">This pipeline was used with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper T equals 0 period 17" tabindex="0" ctxtmenu_counter="304" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="upper T equals 0 period 17"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.676em;">1</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> to obtain fits for monocultures of all 79 strains in SDM and 64 strains in ADM. The RMSEs for strains at each step in the fitting pipeline are shown in <a id="crosref1610" href="#mmc1">Table S5</a>. An example of a fit obtained using the first three steps of the pipeline is shown in <a id="crosref1615" href="#figs1">Figure&nbsp;S1</a>J, and a fit obtained using the first two steps is shown in <a id="crosref1620" href="#figs1">Figure&nbsp;S1</a>K. Only the first three steps of the pipeline were necessary to produce parameter fits with error less than <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper T equals 0 period 17" tabindex="0" ctxtmenu_counter="305" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="upper T equals 0 period 17"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.676em;">1</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> for the monocultures in SDM, while step 4 was necessary for only 4 out of 64 ADM monocultures (<a id="crosref1625" href="#mmc1">Table S5</a>).</div><div id="p0590" role="paragraph">Finally, there were 7 Nar strains (PDM26–PDM32) for which nitrate concentrations appeared to asymptotically approach nonzero values in SDM (see example in <a id="crosref1630" href="#figs1">Figure&nbsp;S1</a>L). However, since it was possible to fit consumer-resource parameters to these cases with relatively low error (<a id="crosref1635" href="#mmc1">Table S5</a>), these parameter fits were accepted for use in subsequent analyses.</div></section><section id="sec-5-5-2-4"><h5>Validating predictions for biomass density dynamics</h5><div id="p0595" role="paragraph">Although the rate parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="306" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="307" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> for each strain were fit by directly measuring the dynamics of nitrate and nitrite reduction, only endpoint measurements were used to fit the yield parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="308" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="309" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and biomass density dynamics were not directly measured. Though the yield parameters are identifiable via endpoint measurements (<a id="crosref1640" href="#fd5">Equation 5</a>), the accuracy of predictions for biomass density dynamics depends on the consumer-resource model (<a id="crosref1645" href="#fd3">Equation 3</a>) being reasonably well-specified. For instance, if mortality is an important factor that causes biomass density to change significantly on the timescale of the experiment, then the measurement of yields would lead to poor biomass predictions because cell death is not accounted for in the model.</div><div id="p0600" role="paragraph">To validate the inference of yields and model predictions for biomass density dynamics, these dynamics were directly measured for a set of four taxonomically-representative Nar/Nir strains: <i>Achromobacter</i> sp. ACM01, <i>Ensifer</i> sp. ENS09, <i>Paracoccus denitrificans</i> ATCC 19367, and <i>Pseudomonas</i> sp. PDM21. These four strains were cultured in SDM under denitrifying conditions with 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="310" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, OD600<sub>0</sub> <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="equals 0 period 01" tabindex="0" ctxtmenu_counter="311" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="equals 0 period 01"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, and 21 experimental replicates. Cultures were manually sampled (300&nbsp;μL) from 3 replicates at 7 time points (2, 4, 6, 8, 16, 32, and 64&nbsp;h from the start of anaerobic culture), with different replicates sampled at each time point. Optical density was measured immediately after sampling, and measurements were post-processed to correct for evaporation.</div><div id="p0605" role="paragraph">The measured values of biomass density are shown in <a id="crosref1650" href="#figs1">Figure&nbsp;S1</a>M, alongside predictions of the model <a id="crosref1655" href="#fd3">Equation 3</a> using previous inferences of rate and yield parameters for these strains in SDM. Predicted biomass dynamics agree well with the measured optical densities, both in the exponential and stationary phases. The errors of predicted biomass density relative to measurements at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t equals" tabindex="0" ctxtmenu_counter="312" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 64&nbsp;h are all less than 10%, indicating that the yields were well-estimated by the fitting protocol. No appreciable evidence for mortality was observed, with little to no decline in biomass density observed after nitrate and nitrite were exhausted. This is consistent with previous measurements of low-density starving cultures of <i>Escherischia coli</i> K12, where negligible cell death was observed after approximately 50 h (<span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Phaiboun et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Phaiboun, A. ∙ Zhang, Y. ∙ Park, B. ...</div><div><strong>Survival kinetics of starving bacteria is biphasic and density-dependent</strong></div><div><em>PLoS Comput. Biol.</em> 2015; <strong>11</strong>:e1004198</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1004198" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84929485992" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25838110/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1004198&amp;pmid=25838110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It was concluded that the fitting procedure for rates and yields faithfully captures biomass density dynamics, despite the fact that biomass density is only measured at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t equals" tabindex="0" ctxtmenu_counter="313" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 64 h.</div></section><section id="sec-5-5-2-5"><h5>Estimating parameter error</h5><div id="p0610" role="paragraph">A nonparametric bootstrap was used to estimate the sampling distributions of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="314" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> for the 79-strain library in SDM. Bootstrap datasets were generated by resampling (with replacement) among replicates; that is, for an experimental condition (e.g., <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A 0" tabindex="0" ctxtmenu_counter="315" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper A 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>&nbsp;= 2&nbsp;mM, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I 0" tabindex="0" ctxtmenu_counter="316" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper I 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0&nbsp;mM, OD600<sub>0</sub>&nbsp;= 0.01) performed with four replicates, a new dataset was created by randomly selecting a set of four replicates with replacement, doing this for all conditions. Then <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper A Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="317" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper A Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were refit to these bootstrap resamples. This was repeated 100 times for each strain to obtain sampling distributions for these parameters. Fractional errors, defined as the ratio of the interquartile range to the value of the parameter, are less than 10% for the vast majority of inferred parameters. Since <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="318" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="319" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> inferences are error-prone when measured biomass densities (and therefore yields) are low, a yield parameter was set identically to zero if its estimate was negative or within one standard error of zero.</div></section></section><section id="sec-5-5-3"><h4>Regressing SDM phenotypes onto denitrification gene content</h4><section id="sec-5-5-3-1"><h5>Formulating the regression problem</h5><div id="p0615" role="paragraph">Linear regression was used to predict the measured consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="320" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> from the presence and absence of denitrification-related genes in the genomes of each strain. The regression problem was formulated as follows:<div id="fd7" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="y Subscript i Baseline equals beta 0 plus sigma summation Underscript j equals 1 Overscript upper P Endscripts beta Subscript j Baseline g Subscript i j Baseline plus epsilon Subscript i Baseline for i equals 1 comma ellipsis comma upper N period" tabindex="0" ctxtmenu_counter="321" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="48" data-semantic-children="47,33,34,35,36,37" data-semantic-content="33,35,37" data-semantic-owns="47 33 34 35 36 37" data-semantic-structure="(48 (47 (42 (2 0 1) 3 (41 (6 4 5) 7 (40 (14 8 (12 9 10 11) 13) (39 (17 15 16) 38 (23 18 (22 19 21 20)))) 24 (27 25 26))) 28 29 (43 30 31 32)) 33 34 35 36 37)" data-semantic-speech="y Subscript i Baseline equals beta 0 plus sigma summation Underscript j equals 1 Overscript upper P Endscripts beta Subscript j Baseline g Subscript i j Baseline plus epsilon Subscript i Baseline for i equals 1 comma ellipsis comma upper N period" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="47" data-semantic-children="42,28,29,43" data-semantic-parent="48" data-semantic-collapsed="(47 (c 44 45 46) 42 28 29 43)" data-semantic-owns="42 28 29 43" inline-breaks="true"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="42" data-semantic-children="2,41" data-semantic-content="3" data-semantic-parent="47" data-semantic-owns="2 3 41" inline-breaks="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="42" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="42" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="41" data-semantic-children="6,40,27" data-semantic-content="7,24" data-semantic-parent="42" data-semantic-owns="6 7 40 24 27" inline-breaks="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="41" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-break size="3"></mjx-break><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="7" data-semantic-parent="41" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="40" data-semantic-children="14,39" data-semantic-content="8" data-semantic-parent="41" data-semantic-owns="14 39"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="14" data-semantic-children="8,12,13" data-semantic-parent="40" data-semantic-owns="8 12 13" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="8" data-semantic-parent="14" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.291em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="12" data-semantic-children="9,11" data-semantic-content="10" data-semantic-parent="14" data-semantic-owns="9 10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="12"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="10" data-semantic-parent="12" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="39" data-semantic-children="17,23" data-semantic-content="38" data-semantic-parent="40" data-semantic-owns="17 38 23"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="39" data-semantic-owns="15 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="23" data-semantic-children="18,22" data-semantic-parent="39" data-semantic-owns="18 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="23"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="22" data-semantic-children="19,20" data-semantic-content="21" data-semantic-parent="23" data-semantic-owns="19 21 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="22"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="22"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-break size="3"></mjx-break><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="24" data-semantic-parent="41" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="27" data-semantic-children="25,26" data-semantic-parent="41" data-semantic-owns="25 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="27"><mjx-c class="mjx-c1D700">𝜀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="27"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-break size="0"></mjx-break><mjx-mspace data-semantic-type="text" data-semantic-role="space" data-semantic-annotation="clearspeak:unit" data-semantic-id="28" data-semantic-parent="47" style="width: 1em;"></mjx-mspace><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="29" data-semantic-parent="47"><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.705em;">o</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.705em;">&nbsp;</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="43" data-semantic-children="30,32" data-semantic-content="31" data-semantic-parent="47" data-semantic-owns="30 31 32" inline-breaks="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="43"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="31" data-semantic-parent="43" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="43"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="33" data-semantic-parent="48" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="34" data-semantic-parent="48"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="35" data-semantic-parent="48" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="48"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="37" data-semantic-parent="48" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-math></mjx-container></div></div><div class="label">(Equation 7)</div></div></div><div id="p0620" role="paragraph">The response variable <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="y Subscript i" tabindex="0" ctxtmenu_counter="322" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="y Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the observed value of a consumer-resource model parameter (e.g., <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="323" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>) for strain <i>i</i>, with <i>N</i> such observations in total. The predictors <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g Subscript i j" tabindex="0" ctxtmenu_counter="324" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="g Subscript i j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are indicator variables that take the value 1 if strain <i>I</i> has gene <i>j</i> and take 0 otherwise, with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P equals 17" tabindex="0" ctxtmenu_counter="325" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper P equals 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.676em;">1</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> predictors in total. The coefficients <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript j" tabindex="0" ctxtmenu_counter="326" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta Subscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and intercept <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta 0" tabindex="0" ctxtmenu_counter="327" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are fit by the regression, which determines the residual term <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="epsilon Subscript i" tabindex="0" ctxtmenu_counter="328" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="epsilon Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D700">𝜀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. This formulation assumes that genes contribute additively (and not quadratically, etc.) to the value of a phenotypic parameter. While in principle there may exist a formulation of this regression with greater predictive power that takes into account the nonlinear interactions between predictors (e.g., a model with quadratic terms), a linear formulation was chosen to keep the number of fitting parameters small relative to the number of observations.</div><div id="p0625" role="paragraph">The LASSO regression method (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Hastie et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Hastie, T. ∙ Tibshirani, R. ∙ Friedman, J.</div><div><strong>The Elements of Statistical Learning</strong></div><div>Springer, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.HastieR.TibshiraniJ.FriedmanThe+Elements+of+Statistical+Learning2nd+Edition2008Springer" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Hastie et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Hastie, T. ∙ Tibshirani, R.J. ∙ Wainwright, R.</div><div><strong>Statistical learning with sparsity</strong></div><div>CRC Press, 2016</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.HastieR.J.TibshiraniR.WainwrightStatistical+learning+with+sparsityMonographs+on+Statistics+and+Applied+Probability1432016CRC+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to solve <a id="crosref1675" href="#fd7">Equation 7</a>. For given gene presence/absence vectors <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove g With right arrow Subscript i" tabindex="0" ctxtmenu_counter="329" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (2 0 1) 3)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-owns="2 3" data-semantic-speech="ModifyingAbove g With right arrow Subscript i"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="4" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.316em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and response vector <i>y</i>, LASSO regression solves:<div id="fd8" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="min Underscript beta 0 comma ModifyingAbove beta With right arrow Endscripts left brace StartFraction 1 Over 2 upper N EndFraction sigma summation Underscript i equals 1 Overscript upper N Endscripts left parenthesis y Subscript i Baseline minus beta 0 minus ModifyingAbove g With right arrow Subscript i Superscript upper T Baseline ModifyingAbove beta With right arrow right parenthesis squared plus lamda StartAbsoluteValue StartAbsoluteValue ModifyingAbove beta With right arrow EndAbsoluteValue EndAbsoluteValue Subscript 1 Baseline right brace comma" tabindex="0" ctxtmenu_counter="330" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="74" data-semantic-children="73,71" data-semantic-content="71" data-semantic-owns="73 71" data-semantic-structure="(74 (73 (9 0 (8 (3 1 2) 4 (7 5 6))) 72 (70 68 (67 (64 (15 10 (14 11 13 12)) 63 (62 (22 16 (20 17 18 19) 21) (48 (46 44 (41 (25 23 24) 26 (29 27 28) 30 (43 (37 (36 (33 31 32) 34) 35) 42 (40 38 39))) 45) 47))) 49 (66 50 65 (61 (59 51 (57 52 (55 53 54) 56) 58) 60))) 69)) 71)" data-semantic-speech="min Underscript beta 0 comma ModifyingAbove beta With right arrow Endscripts left brace StartFraction 1 Over 2 upper N EndFraction sigma summation Underscript i equals 1 Overscript upper N Endscripts left parenthesis y Subscript i Baseline minus beta 0 minus ModifyingAbove g With right arrow Subscript i Superscript upper T Baseline ModifyingAbove beta With right arrow right parenthesis squared plus lamda StartAbsoluteValue StartAbsoluteValue ModifyingAbove beta With right arrow EndAbsoluteValue EndAbsoluteValue Subscript 1 Baseline right brace comma" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="73" data-semantic-children="9,70" data-semantic-content="72" data-semantic-parent="74" data-semantic-owns="9 72 70"><mjx-munder data-semantic-type="underscore" data-semantic-role="unknown" data-semantic-id="9" data-semantic-children="0,8" data-semantic-parent="73" data-semantic-owns="0 8"><mjx-row><mjx-base><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.657em;">m</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.657em;">n</mjx-c></mjx-mtext></mjx-base></mjx-row><mjx-row><mjx-under style="padding-top: 0.167em; padding-left: 0.178em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="8" data-semantic-children="3,4,7" data-semantic-content="4" data-semantic-parent="9" data-semantic-owns="3 4 7"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="8" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="8" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="8" data-semantic-owns="5 6"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.437em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="6" data-semantic-parent="7" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-under></mjx-row></mjx-munder><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="72" data-semantic-parent="73" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="70" data-semantic-children="67" data-semantic-content="68,69" data-semantic-parent="73" data-semantic-owns="68 67 69"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="68" data-semantic-parent="70" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c7B MM-S7">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="67" data-semantic-children="64,66" data-semantic-content="49" data-semantic-parent="70" data-semantic-owns="64 49 66"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="64" data-semantic-children="15,62" data-semantic-content="63" data-semantic-parent="67" data-semantic-owns="15 63 62"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="15" data-semantic-children="10,14" data-semantic-parent="64" data-semantic-owns="10 14"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="15"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="14" data-semantic-children="11,12" data-semantic-content="13" data-semantic-parent="15" data-semantic-owns="11 13 12"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="63" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="62" data-semantic-children="22,48" data-semantic-content="16" data-semantic-parent="64" data-semantic-owns="22 48"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="22" data-semantic-children="16,20,21" data-semantic-parent="62" data-semantic-owns="16 20 21" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="16" data-semantic-parent="22" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="22"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.291em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="20" data-semantic-children="17,19" data-semantic-content="18" data-semantic-parent="22" data-semantic-owns="17 18 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="20"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="18" data-semantic-parent="20" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="20"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-msup space="2" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="48" data-semantic-children="46,47" data-semantic-parent="62" data-semantic-owns="46 47"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="46" data-semantic-children="41" data-semantic-content="44,45" data-semantic-parent="48" data-semantic-owns="44 41 45"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="44" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S6">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="41" data-semantic-children="25,29,43" data-semantic-content="26,30" data-semantic-parent="46" data-semantic-owns="25 26 29 30 43"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="25" data-semantic-children="23,24" data-semantic-parent="41" data-semantic-owns="23 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="25"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="25"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="26" data-semantic-parent="41" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="29" data-semantic-children="27,28" data-semantic-parent="41" data-semantic-owns="27 28"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="29"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="29"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="30" data-semantic-parent="41" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="43" data-semantic-children="37,40" data-semantic-content="42" data-semantic-parent="41" data-semantic-owns="37 42 40"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="37" data-semantic-children="33,34,35" data-semantic-parent="43" data-semantic-collapsed="(37 (36 33 34) 35)" data-semantic-owns="33 34 35"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="33" data-semantic-children="31,32" data-semantic-parent="37" data-semantic-owns="31 32"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.316em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="32" data-semantic-parent="33" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="33"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.284em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="37"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="37"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="42" data-semantic-parent="43" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="40" data-semantic-children="38,39" data-semantic-parent="43" data-semantic-owns="38 39"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.437em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="39" data-semantic-parent="40" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="40"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S6">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 1.173em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="47" data-semantic-parent="48"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="49" data-semantic-parent="67" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="66" data-semantic-children="50,61" data-semantic-content="65" data-semantic-parent="67" data-semantic-owns="50 65 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="50" data-semantic-parent="66"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="65" data-semantic-parent="66" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="neutral" data-semantic-id="61" data-semantic-children="59,60" data-semantic-parent="66" data-semantic-owns="59 60"><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="59" data-semantic-children="57" data-semantic-content="51,58" data-semantic-parent="61" data-semantic-owns="51 57 58"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="51" data-semantic-parent="59" data-semantic-operator="fenced"><mjx-c class="mjx-c2223 MM-S5">∣</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="57" data-semantic-children="55" data-semantic-content="52,56" data-semantic-parent="59" data-semantic-owns="52 55 56"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="52" data-semantic-parent="57" data-semantic-operator="fenced"><mjx-c class="mjx-c2223 MM-S5">∣</mjx-c></mjx-mo><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="55" data-semantic-children="53,54" data-semantic-parent="57" data-semantic-owns="53 54"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.437em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="54" data-semantic-parent="55" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="53" data-semantic-parent="55"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="56" data-semantic-parent="57" data-semantic-operator="fenced"><mjx-c class="mjx-c2223 MM-S5">∣</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="58" data-semantic-parent="59" data-semantic-operator="fenced"><mjx-c class="mjx-c2223 MM-S5">∣</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -1.036em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="61"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="69" data-semantic-parent="70" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c7D MM-S7">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="71" data-semantic-parent="74" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-math></mjx-container></div></div><div class="label">(Equation 8)</div></div>performing both variable selection and regularization by penalizing the sum of squared residuals by the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L 1" tabindex="0" ctxtmenu_counter="331" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper L 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> norm of the coefficient vector <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove beta With right arrow" tabindex="0" ctxtmenu_counter="332" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="ModifyingAbove beta With right arrow"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.437em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container>. The strength of the penalty is controlled by the hyperparameter <i>λ</i>, which at moderate values sets the coefficients of poor predictors identically to zero, thus resulting in a sparse model. Typically the hyperparameter value <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda equals ModifyingAbove lamda With ˆ" tabindex="0" ctxtmenu_counter="333" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="lamda equals ModifyingAbove lamda With ˆ"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mover space="4" data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.238em; margin-bottom: -0.572em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="3" data-semantic-parent="4" style="width: 0px; margin-left: -0.322em;"><mjx-c class="mjx-c302 MM-SO">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container> is selected by minimizing prediction error in cross-validation, which optimizes the ability of the model to generalize out of sample, and makes the method suitable for datasets where overfitting via an approach such as ordinary least squares (OLS) is likely because the number of predictors and the number of observations are on the same order of magnitude.</div><div id="p0630" role="paragraph">LASSO regressions were performed in MATLAB R2017b for each consumer-resource parameter measured in SDM separately, obtaining different <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove beta With right arrow" tabindex="0" ctxtmenu_counter="334" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="ModifyingAbove beta With right arrow"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.437em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta 0" tabindex="0" ctxtmenu_counter="335" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> for each regression. The fits and coefficients for regressions on consumer-resource parameters measured in SDM are shown in <a id="crosref1680" href="#fig4">Figures 4</a>C–4J. Note that for nitrate-related parameters (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="336" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="337" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>), only strains capable of nitrate reduction (Nar and Nar/Nir phenotypes) were included in the predictor-response datasets. Similarly for the nitrite-related parameters, only strains with Nar/Nir and Nir phenotypes were included in the predictor-response datasets. Prior to fitting, all predictors were standardized to have zero mean and unit variance.</div></section><section id="sec-5-5-3-2"><h5>Hyperparameter selection via cross-validation</h5><div id="p0635" role="paragraph">For each regression of the form given in <a id="crosref1685" href="#fd7">Equation 7</a>, iterated <i>K</i>-fold cross-validation was used to determine the value <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda equals ModifyingAbove lamda With ˆ" tabindex="0" ctxtmenu_counter="338" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="lamda equals ModifyingAbove lamda With ˆ"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mover space="4" data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.238em; margin-bottom: -0.572em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="3" data-semantic-parent="4" style="width: 0px; margin-left: -0.322em;"><mjx-c class="mjx-c302 MM-SO">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container> that minimizes estimated prediction error. Optimal hyperparameter values were determined for each regression individually, rather determining than one hyperparameter value to be used for all regressions. This was done because the hyperparameter controls the level of sparsity in the resulting solution (i.e., the number of genes that are nonzero in the optimal set of regression coefficients), and <i>a priori</i> the optimal level of sparsity may differ among the different regressions on consumer-resource parameters. Choosing an optimal value of <i>λ</i> for each regression individually ensured that the best LASSO solution for each regression was identified.</div><div id="p0640" role="paragraph">Let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Lamda" tabindex="0" ctxtmenu_counter="339" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper Lamda"><mjx-c class="mjx-c1D6EC">𝛬</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> be a set of discrete values of <i>λ</i> for which <a id="crosref1690" href="#fd8">Equation 8</a> is solved. Each iteration of cross-validation proceeds in the following way:<div id="olist0020" role="list"><div id="o0045" role="listitem"><div class="label">1.</div><div class="content"><div id="p0645" role="paragraph">Randomly partition the data into <i>K</i> subsamples of roughly equal size.</div></div></div><div id="o0050" role="listitem"><div class="label">2.</div><div class="content"><div id="p0650" role="paragraph">For each <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="k equals 1 comma 2 comma ellipsis comma upper K" tabindex="0" ctxtmenu_counter="340" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (9 0 1 2) 3 4 5 6 7 8)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="9,3,4,5,6,7,8" data-semantic-content="3,5,7" data-semantic-owns="9 3 4 5 6 7 8" data-semantic-speech="k equals 1 comma 2 comma ellipsis comma upper K"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="10" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="10"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="6" data-semantic-parent="10"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="7" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>:<div id="olist0025" role="list"><div id="o0055" role="listitem"><div class="label">(a)</div><div class="content"><div id="p0655" role="paragraph">Holding out the <i>k</i>th fold (test set), solve <a id="crosref1695" href="#fd8">Equation 8</a> at each <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda element of upper Lamda" tabindex="0" ctxtmenu_counter="341" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="lamda element of upper Lamda"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D6EC">𝛬</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> using the union of the remaining remaining <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K minus 1" tabindex="0" ctxtmenu_counter="342" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper K minus 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> folds of data (training set).</div></div></div><div id="o0060" role="listitem"><div class="label">(b)</div><div class="content"><div id="p0660" role="paragraph">Use the resulting regression coefficients to predict the responses in the test set.</div></div></div><div id="o0065" role="listitem"><div class="label">(c)</div><div class="content"><div id="p0665" role="paragraph">Evaluate root-mean-square error of the test set prediction.</div></div></div></div></div></div></div></div></div><div id="p0670" role="paragraph">This process is iterated <i>M</i> times, each time choosing a new random partition of the data into <i>K</i> subsamples. The average over the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K upper M" tabindex="0" ctxtmenu_counter="343" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 2 1)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="3" data-semantic-children="0,1" data-semantic-content="2" data-semantic-owns="0 2 1" data-semantic-speech="upper K upper M"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="2" data-semantic-parent="3" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> estimates of prediction error was recorded at each value of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda element of upper Lamda" tabindex="0" ctxtmenu_counter="344" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="lamda element of upper Lamda"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D6EC">𝛬</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, and the optimal value <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda equals ModifyingAbove lamda With ˆ" tabindex="0" ctxtmenu_counter="345" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="lamda equals ModifyingAbove lamda With ˆ"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mover space="4" data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.238em; margin-bottom: -0.572em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="3" data-semantic-parent="4" style="width: 0px; margin-left: -0.322em;"><mjx-c class="mjx-c302 MM-SO">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container> was then selected as that which minimized this estimate of error. An iterated <i>K</i>-fold cross-validation approach was chosen over either basic <i>K</i>-fold <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper M equals 1 right parenthesis" tabindex="0" ctxtmenu_counter="346" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 (4 1 2 3) 5)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="0,5" data-semantic-owns="0 4 5" data-semantic-speech="left parenthesis upper M equals 1 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> or leave-one-out <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper K equals upper N right parenthesis" tabindex="0" ctxtmenu_counter="347" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 (4 1 2 3) 5)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="0,5" data-semantic-owns="0 4 5" data-semantic-speech="left parenthesis upper K equals upper N right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> cross-validation because iterated cross-validation averages over the sampling variability inherent in randomly partitioning the dataset into groups, and also because partitioning into <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K not equals upper N" tabindex="0" ctxtmenu_counter="348" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper K not equals upper N"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,≠"><mjx-c class="mjx-c2260">≠</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> folds allows the estimation the out-of-sample performance of the model through statistics such as the coefficient of determination between observed and predicted values on each test set.</div><div id="p0675" role="paragraph">A value of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K equals 4" tabindex="0" ctxtmenu_counter="349" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper K equals 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> for iterated cross-validation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M equals 10 Superscript 4" tabindex="0" ctxtmenu_counter="350" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="upper M equals 10 Superscript 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> iterations) was chosen to balance the relative sizes of training and test sets for the purpose of evaluating out-of-sample performance; as <i>K</i> increases, the size of the cross-validation training set increases, while the size of test set decreases. It was observed that <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove lamda With ˆ" tabindex="0" ctxtmenu_counter="351" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="ModifyingAbove lamda With ˆ"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.238em; margin-bottom: -0.572em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: -0.322em;"><mjx-c class="mjx-c302 MM-SO">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container> was insensitive to the choice of <i>K</i>.</div></section><section id="sec-5-5-3-3"><h5>Estimating out-of-sample performance</h5><div id="p0680" role="paragraph">The out-of-sample performance of regressions on each consumer-resource parameter was estimated during iterated <i>K</i>-fold cross-validation by computing the coefficient of determination <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper R squared right parenthesis" tabindex="0" ctxtmenu_counter="352" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (3 1 2) 4)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-owns="0 3 4" data-semantic-speech="left parenthesis upper R squared right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> between observed and predicted response values in each test set. The median values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R squared" tabindex="0" ctxtmenu_counter="353" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> obtained via iterated 4-fold cross-validation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M equals 10 Superscript 4" tabindex="0" ctxtmenu_counter="354" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="upper M equals 10 Superscript 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> iterations) at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda equals ModifyingAbove lamda With ˆ" tabindex="0" ctxtmenu_counter="355" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="lamda equals ModifyingAbove lamda With ˆ"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mover space="4" data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.238em; margin-bottom: -0.572em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="3" data-semantic-parent="4" style="width: 0px; margin-left: -0.322em;"><mjx-c class="mjx-c302 MM-SO">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container>, denoted <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="356" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, is positive for all regressions.</div><div id="p0685" role="paragraph">The significance of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="357" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> for regressions on each consumer-resource parameter was evaluated under the null hypothesis that there is no relationship between the predictor and the response via a permutation test. The response variable for each regression was repeatedly permuted (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 times 10 cubed" tabindex="0" ctxtmenu_counter="358" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="1 times 10 cubed"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> permutations) and regression coefficients were re-fitted, using iterated 4-fold cross-validation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M equals 10 squared" tabindex="0" ctxtmenu_counter="359" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="upper M equals 10 squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> iterations) to select the hyperparameter for each permutation. Under the null distributions of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="360" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> computed using this approach for regressions on SDM consumer-resource parameters, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than 0 period 001" tabindex="0" ctxtmenu_counter="361" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="p less than 0 period 001"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> in all cases.</div></section><section id="sec-5-5-3-4"><h5>Post-selection inference</h5><div id="p0690" role="paragraph">When the data used for model training via LASSO are also used for inference of significant regression coefficients, it is necessary to account for the fact that LASSO performs variable selection (i.e., selects some coefficients to be nonzero over others) (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Hastie et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Hastie, T. ∙ Tibshirani, R.J. ∙ Wainwright, R.</div><div><strong>Statistical learning with sparsity</strong></div><div>CRC Press, 2016</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.HastieR.J.TibshiraniR.WainwrightStatistical+learning+with+sparsityMonographs+on+Statistics+and+Applied+Probability1432016CRC+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref107-2" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-2" aria-controls="sref107-2">Taylor and Tibshirani, 2015</a><div class="dropBlock__holder" data-db-target-of="sref107-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Taylor, J. ∙ Tibshirani, R.J.</div><div><strong>Statistical learning and selective inference</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:7629-7634</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507583112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84934993654" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26100887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507583112&amp;pmid=26100887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because LASSO preferentially selects variables with high predictive power in the training set, neglecting this selection results in overoptimistic confidence intervals and <i>p</i>-values.</div><div id="p0695" role="paragraph">To illustrate how and why this was done, first consider the simpler case of inference on regression coefficients determined by OLS. If the residuals <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="epsilon Subscript i" tabindex="0" ctxtmenu_counter="362" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="epsilon Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D700">𝜀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are normally distributed, it can be shown that the sampling distribution of a coefficient <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript j" tabindex="0" ctxtmenu_counter="363" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta Subscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is also normal, with a sampling variance that can be estimated from the data. This sampling distribution can then be used to test the hypothesis that the true value of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript j" tabindex="0" ctxtmenu_counter="364" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta Subscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is zero. In contrast, in the case where LASSO regression is used and the coefficient <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript j" tabindex="0" ctxtmenu_counter="365" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta Subscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is selected to be nonzero, it can be shown that a normal distribution is not appropriate for the sampling distribution of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript j" tabindex="0" ctxtmenu_counter="366" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta Subscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<span class="dropBlock reference-citations"><a id="body-ref-sref107-3" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-3" aria-controls="sref107-3">Taylor and Tibshirani, 2015</a><div class="dropBlock__holder" data-db-target-of="sref107-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Taylor, J. ∙ Tibshirani, R.J.</div><div><strong>Statistical learning and selective inference</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:7629-7634</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507583112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84934993654" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26100887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507583112&amp;pmid=26100887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Instead the appropriate distribution is a truncated normal distribution, i.e., a normal distribution only defined along a finite interval. Intuitively, the distribution is truncated because the likely range of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta Subscript j" tabindex="0" ctxtmenu_counter="367" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="beta Subscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> must be conditioned upon the knowledge that LASSO selected it (along with a set of other elements of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove beta With right arrow" tabindex="0" ctxtmenu_counter="368" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="ModifyingAbove beta With right arrow"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.437em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container>) to be nonzero. The bounds of the truncated distribution are determined by both the data and by the set of other nonzero elements of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove beta With right arrow" tabindex="0" ctxtmenu_counter="369" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="ModifyingAbove beta With right arrow"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.437em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container> via analytical relations, and can be computed using the package selectiveInference 1.2.5 in R 3.6.1 to estimate the sampling distributions for each nonzero element of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove beta With right arrow" tabindex="0" ctxtmenu_counter="370" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="ModifyingAbove beta With right arrow"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.437em;"><mjx-mo data-semantic-type="relation" data-semantic-role="overaccent" data-semantic-annotation="accent:arrow" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: -0.5em;"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container>. An intuitive description of this post-selection inference approach applied to the simpler context of forward stepwise regression can be found in <span class="dropBlock reference-citations"><a id="body-ref-sref107-4" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-4" aria-controls="sref107-4">Taylor and Tibshirani (2015)</a><div class="dropBlock__holder" data-db-target-of="sref107-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Taylor, J. ∙ Tibshirani, R.J.</div><div><strong>Statistical learning and selective inference</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:7629-7634</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507583112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84934993654" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26100887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507583112&amp;pmid=26100887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, and a technical description of the approach applied to LASSO can be found in <span class="dropBlock reference-citations"><a id="body-ref-sref38-3" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-3" aria-controls="sref38-3">Hastie et al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref38-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Hastie, T. ∙ Tibshirani, R.J. ∙ Wainwright, R.</div><div><strong>Statistical learning with sparsity</strong></div><div>CRC Press, 2016</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.HastieR.J.TibshiraniR.WainwrightStatistical+learning+with+sparsityMonographs+on+Statistics+and+Applied+Probability1432016CRC+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div><div id="p0700" role="paragraph">The function fixedLassoInf in the selectiveInference package was used to compute 90% confidence intervals and to obtain <i>p</i>-values under the null hypothesis the true value is zero for each nonzero coefficient for regressions on SDM consumer-resource parameters, evaluated at the hyperparameter value <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda equals ModifyingAbove lamda With ˆ" tabindex="0" ctxtmenu_counter="371" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="lamda equals ModifyingAbove lamda With ˆ"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mover space="4" data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.238em; margin-bottom: -0.572em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="3" data-semantic-parent="4" style="width: 0px; margin-left: -0.322em;"><mjx-c class="mjx-c302 MM-SO">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container> selected by cross-validation.</div></section></section><section id="sec-5-5-4"><h4>Characterizing phylogenetic correlation</h4><div id="p0705" role="paragraph">Microbial phenotypic traits exhibit varying degrees of phylogenetic correlation as a result of shared evolutionary history (<span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Martiny et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Martiny, J.B.H. ∙ Jones, S.E. ∙ Lennon, J.T. ...</div><div><strong>Microbiomes in light of traits: a phylogenetic perspective</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:aac9323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aac9323" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84947460926" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26542581/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aac9323&amp;pmid=26542581" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To characterize the extent of phylogenetic correlation in the consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="372" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, the function phyloCorrelogram in the package phylosignal 1.3 for R 3.6.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Keck et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Keck, F. ∙ Rimet, F. ∙ Bouchez, A. ...</div><div><strong>phylosignal: an R package to measure, test, and explore the phylogenetic signal</strong></div><div><em>Ecol. Evol.</em> 2016; <strong>6</strong>:2774-2780</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ece3.2051" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84962504104" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27066252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fece3.2051&amp;pmid=27066252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used. This function estimates autocorrelation in trait values by computing Moran’s <i>I</i> index as a function of phylogenetic distance, measured here using the 16S rRNA phylogenetic tree. Statistically significant correlation (95% confidence intervals estimated using <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 times 10 cubed" tabindex="0" ctxtmenu_counter="373" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="1 times 10 cubed"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> bootstrap replicates) was observed over varying levels of phylogenetic distance for the different parameters. The rate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="374" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> was correlated to a small degree <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis max left parenthesis upper I right parenthesis equals 0 period 16 right parenthesis" tabindex="0" ctxtmenu_counter="375" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 13 (12 (11 (10 0 9 (4 2 1 3)) 5 6) 7 8) 14)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="12" data-semantic-content="13,14" data-semantic-owns="13 12 14" data-semantic-speech="left parenthesis max left parenthesis upper I right parenthesis equals 0 period 16 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="13" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="12" data-semantic-children="11,7,8" data-semantic-content="7" data-semantic-parent="15" data-semantic-owns="11 7 8"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="10,6" data-semantic-content="5" data-semantic-parent="12" data-semantic-owns="10 5 6"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="limit function" data-semantic-id="10" data-semantic-children="0,4" data-semantic-content="9,0" data-semantic-parent="11" data-semantic-owns="0 9 4"><mjx-mtext data-semantic-type="function" data-semantic-role="limit function" data-semantic-annotation="general:text" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="appl"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.448em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.448em;">a</mjx-c><mjx-c class="mjx-c78" style="padding-top: 0.448em;">x</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="1" data-semantic-content="2,3" data-semantic-parent="10" data-semantic-owns="2 1 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="7" data-semantic-parent="12" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c36" style="padding-top: 0.666em;">6</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> over a relatively short interval of phylogenetic distance (16S tree distance 0.01), while <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="376" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="377" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="378" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were correlated to a modest degree (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="max left parenthesis upper I right parenthesis equals 0 period 33" tabindex="0" ctxtmenu_counter="379" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="12" data-semantic-children="11,7,8" data-semantic-content="7" data-semantic-owns="11 7 8" data-semantic-structure="(12 (11 (10 0 9 (4 2 1 3)) 5 6) 7 8)" data-semantic-speech="max left parenthesis upper I right parenthesis equals 0 period 33" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="10,6" data-semantic-content="5" data-semantic-parent="12" data-semantic-owns="10 5 6" inline-breaks="true"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="limit function" data-semantic-id="10" data-semantic-children="0,4" data-semantic-content="9,0" data-semantic-parent="11" data-semantic-owns="0 9 4"><mjx-mtext data-semantic-type="function" data-semantic-role="limit function" data-semantic-annotation="general:text" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="appl"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.448em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.448em;">a</mjx-c><mjx-c class="mjx-c78" style="padding-top: 0.448em;">x</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="1" data-semantic-content="2,3" data-semantic-parent="10" data-semantic-owns="2 1 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="7" data-semantic-parent="12" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c33" noic="true" style="padding-top: 0.666em;">3</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.666em;">3</mjx-c></mjx-mn></mjx-math></mjx-container>, 0.27 and 0.48, respectively) over longer intervals (0.16, 0.06 and 0.12, respectively).</div><div id="p0710" role="paragraph">Because the strain library contains a few clades of very closely related strains (e.g., ENS01–08, PDM20–23, <a id="crosref1735" href="#fig3">Figure&nbsp;3</a>C), it is possible that some of the phylogenetic correlation measured in the full 79-strain library is attributable to the over-representation of these close relatives. In addition, the over-representation of these close relatives could in principle skew the results of the regressions. To estimate the influence of over-represented close relatives in the strain library on phylogenetic correlations and regression results, clades comprising strains with identical 16S rRNA sequences were pruned by randomly selecting one representative and removing the remaining strains from the dataset. This removed 15 strains from the library, resulting in a subsampled dataset of 64 strains. Phylogenetic correlograms were then computed for the SDM consumer-resource parameters on this pruned dataset. Pruning the dataset reduced correlations for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="380" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="381" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="382" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, with maximum correlations decreasing for all three parameters (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="max left parenthesis upper I right parenthesis equals 0 period 30" tabindex="0" ctxtmenu_counter="383" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="12" data-semantic-children="11,7,8" data-semantic-content="7" data-semantic-owns="11 7 8" data-semantic-structure="(12 (11 (10 0 9 (4 2 1 3)) 5 6) 7 8)" data-semantic-speech="max left parenthesis upper I right parenthesis equals 0 period 30" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="10,6" data-semantic-content="5" data-semantic-parent="12" data-semantic-owns="10 5 6" inline-breaks="true"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="limit function" data-semantic-id="10" data-semantic-children="0,4" data-semantic-content="9,0" data-semantic-parent="11" data-semantic-owns="0 9 4"><mjx-mtext data-semantic-type="function" data-semantic-role="limit function" data-semantic-annotation="general:text" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="appl"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.448em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.448em;">a</mjx-c><mjx-c class="mjx-c78" style="padding-top: 0.448em;">x</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="1" data-semantic-content="2,3" data-semantic-parent="10" data-semantic-owns="2 1 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="7" data-semantic-parent="12" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="12"><mjx-c class="mjx-c33" noic="true" style="padding-top: 0.666em;">3</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn></mjx-math></mjx-container>, 0.21 and 0.39, respectively), and significant correlation lengths decreasing for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="384" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="385" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (0.05 and 0.09, respectively). The correlogram for the rate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="386" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> in the pruned dataset, which showed a small degree of correlation in the full dataset, changed little. It was concluded that some of the phylogenetic correlation estimated for the full dataset was attributable to the over-representation of close relatives, but it is still the case that there exists statistically significant phylogenetic correlations at short distances. Next LASSO regressions were performed on the pruned dataset. Both in-sample and out-of-sample metrics of model performance (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript f i t Superscript 2" tabindex="0" ctxtmenu_counter="387" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 0 (6 1 4 2 5 3)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,6,7" data-semantic-collapsed="(9 (8 0 6) 7)" data-semantic-owns="0 6 7" data-semantic-speech="upper R Subscript f i t Superscript 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="9" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="388" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, respectively) changed little relative to the corresponding values for regressions on the full dataset. Similarly, the regression coefficients did not change substantially. From this it was concluded that the over-representation of close relatives did not have a large impact on the results of regressions on the consumer-resource parameters.</div></section><section id="sec-5-5-5"><h4>Evaluating randomly-selected genes as predictors</h4><div id="p0715" role="paragraph">In order to assess whether denitrification genes make better predictor variables than other possible sets of genes, the SDM consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="389" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were regressed onto the presence and absence of randomly-selected genes from the set of all annotated genes in strains in the library.</div><div id="p0720" role="paragraph">First the set of all uniquely-labelled protein-encoding genes present in the RAST annotations were identified for the 79-strain library. From this set of 13415 unique genes, 51 genes associated with denitrification were removed; this included not only terminal reductases, sensors/regulators, and transporters that were used as variables in the regression (<a id="crosref1740" href="#mmc1">Table S6</a>), but also related chaperones, structural genes, and biosynthesis genes that frequently occur in clusters with the denitrification genes (<a id="crosref1745" href="#mmc1">Table S7</a>).</div><div id="p0725" role="paragraph">New sets of predictor variables were then generated by randomly selecting genes from the set of 13364 non-denitrification genes and constructing binary presence/absence matrices as for denitrification-related genes. In order to create a fair comparison between these randomly-selected predictors and the denitrification gene predictors, (a) sets of only 17 genes were selected, matching the dimensionality of the denitrification gene presence/absence matrix, and (b) rejection sampling was performed to match the gene presence frequency (fraction of strains that possess a given gene) distribution of the denitrification genes. The latter was a necessary consideration because the distribution of presence frequencies of non-denitrification genes is heavily skewed toward zero relative to the frequencies of denitrification genes (<a id="crosref1750" href="#figs2">Figure&nbsp;S2</a>A), indicating that a large portion of genes occur only in a small number of strains.</div><div id="p0730" role="paragraph">To perform rejection sampling, densities for the gene frequency distributions were estimated, and these densities were used to define acceptance probabilities for random samples. Let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f left parenthesis x right parenthesis" tabindex="0" ctxtmenu_counter="390" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 5 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-children="0,4" data-semantic-content="5,0" data-semantic-owns="0 5 4" data-semantic-speech="f left parenthesis x right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> denote the probability density of gene frequencies for non-denitrification genes and similarly <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g left parenthesis x right parenthesis" tabindex="0" ctxtmenu_counter="391" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 5 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-children="0,4" data-semantic-content="5,0" data-semantic-owns="0 5 4" data-semantic-speech="g left parenthesis x right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="appl"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> for denitrification genes, where <i>x</i> here denotes gene presence frequency, and set the constant <i>H</i> so that <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g left parenthesis x right parenthesis divided by f left parenthesis x right parenthesis divided by upper H less than or equals 1" tabindex="0" ctxtmenu_counter="392" style="font-size: 130%;"><mjx-math data-semantic-structure="(21 (19 (18 0 17 (4 1 2 3)) 5 (20 (16 6 15 (10 7 8 9)) 11 12)) 13 14)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="21" data-semantic-children="19,14" data-semantic-content="13" data-semantic-owns="19 13 14" data-semantic-speech="g left parenthesis x right parenthesis divided by f left parenthesis x right parenthesis divided by upper H less than or equals 1"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="19" data-semantic-children="18,20" data-semantic-content="5" data-semantic-parent="21" data-semantic-owns="18 5 20"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-children="0,4" data-semantic-content="17,0" data-semantic-parent="19" data-semantic-owns="0 17 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="appl"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="18" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="5" data-semantic-parent="19" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="20" data-semantic-children="16,12" data-semantic-content="11" data-semantic-parent="19" data-semantic-owns="16 11 12"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-children="6,10" data-semantic-content="15,6" data-semantic-parent="20" data-semantic-owns="6 15 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="16" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="7,9" data-semantic-parent="16" data-semantic-owns="7 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="7" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="11" data-semantic-parent="20" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="20"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="13" data-semantic-parent="21" data-semantic-operator="relseq,≤"><mjx-c class="mjx-c2264">≤</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="21"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> for all <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="x element of left bracket 0 comma 1 right bracket" tabindex="0" ctxtmenu_counter="393" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 1 (8 2 (6 3 4 5) 7))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="9" data-semantic-children="0,8" data-semantic-content="1" data-semantic-owns="0 1 8" data-semantic-speech="x element of left bracket 0 comma 1 right bracket"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="9" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="2,7" data-semantic-parent="9" data-semantic-owns="2 6 7"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c5B">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="3,4,5" data-semantic-content="4" data-semantic-parent="8" data-semantic-owns="3 4 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c5D">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Rejection sampling was then performed in the following way:<div id="olist0030" role="list"><div id="o0070" role="listitem"><div class="label">1.</div><div class="content"><div id="p0735" role="paragraph">With uniform probability, randomly sample a candidate gene from the set of non-denitrification genes. Denote the frequency of this gene as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="x 0" tabindex="0" ctxtmenu_counter="394" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="x 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>.</div></div></div><div id="o0075" role="listitem"><div class="label">2.</div><div class="content"><div id="p0740" role="paragraph">Accept the candidate gene as a predictor with probability <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g left parenthesis x 0 right parenthesis divided by f left parenthesis x 0 right parenthesis divided by upper H" tabindex="0" ctxtmenu_counter="395" style="font-size: 130%;"><mjx-math data-semantic-structure="(21 (20 0 19 (6 1 (4 2 3) 5)) 7 (22 (18 8 17 (14 9 (12 10 11) 13)) 15 16))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="21" data-semantic-children="20,22" data-semantic-content="7" data-semantic-owns="20 7 22" data-semantic-speech="g left parenthesis x 0 right parenthesis divided by f left parenthesis x 0 right parenthesis divided by upper H"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="20" data-semantic-children="0,6" data-semantic-content="19,0" data-semantic-parent="21" data-semantic-owns="0 19 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="20" data-semantic-operator="appl"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="1,5" data-semantic-parent="20" data-semantic-owns="1 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="6" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="7" data-semantic-parent="21" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="22" data-semantic-children="18,16" data-semantic-content="15" data-semantic-parent="21" data-semantic-owns="18 15 16"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="18" data-semantic-children="8,14" data-semantic-content="17,8" data-semantic-parent="22" data-semantic-owns="8 17 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="18" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="12" data-semantic-content="9,13" data-semantic-parent="18" data-semantic-owns="9 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="9" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="14" data-semantic-owns="10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="15" data-semantic-parent="22" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="22"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Otherwise reject the candidate and return to step 1.</div></div></div><div id="o0080" role="listitem"><div class="label">3.</div><div class="content"><div id="p0745" role="paragraph">Return to step 1 until 17 genes are accepted.</div></div></div></div></div><div id="p0750" role="paragraph">This process results in a set of 17 predictors that have a gene presence frequency distribution approximately equal to <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g left parenthesis x right parenthesis" tabindex="0" ctxtmenu_counter="396" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 5 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-children="0,4" data-semantic-content="5,0" data-semantic-owns="0 5 4" data-semantic-speech="g left parenthesis x right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="appl"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. The ksdensity function in MATLAB R2017b was used to estimate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f left parenthesis x right parenthesis" tabindex="0" ctxtmenu_counter="397" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 5 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-children="0,4" data-semantic-content="5,0" data-semantic-owns="0 5 4" data-semantic-speech="f left parenthesis x right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="appl"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g left parenthesis x right parenthesis" tabindex="0" ctxtmenu_counter="398" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 5 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-children="0,4" data-semantic-content="5,0" data-semantic-owns="0 5 4" data-semantic-speech="g left parenthesis x right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="appl"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> using a bandwidth parameter value of 0.4 (<a id="crosref1755" href="#figs2">Figure&nbsp;S2</a>A). This rejection sampling approach was used to generate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 times 10 cubed" tabindex="0" ctxtmenu_counter="399" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="1 times 10 cubed"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> different sets of 17 predictors for regressing against each of the SDM consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="400" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0755" role="paragraph">LASSO regressions were performed on each of the SDM consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="401" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> using the rejection-sampled sets of gene predictors, where the optimal hyperparameter value <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda equals ModifyingAbove lamda With ˆ" tabindex="0" ctxtmenu_counter="402" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="lamda equals ModifyingAbove lamda With ˆ"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mover space="4" data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.238em; margin-bottom: -0.572em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="3" data-semantic-parent="4" style="width: 0px; margin-left: -0.322em;"><mjx-c class="mjx-c302 MM-SO">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow></mjx-math></mjx-container> was selected via iterated 4-fold cross-validation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M equals 10 cubed" tabindex="0" ctxtmenu_counter="403" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="upper M equals 10 cubed"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> iterations). Out-of-sample performance of each regression was also evaluated in cross-validation by computing <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="404" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, the median coefficient of determination value computed across cross-validation test sets. The distributions of these <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="405" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> values across different sets of random genes are shown in (<a id="crosref1760" href="#figs2">Figure&nbsp;S2</a>B), alongside the values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="406" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> obtained via the regressions onto denitrification-related genes. The denitrification genes performed better than the typical set of random genes in regressions on two out of four consumer-resource parameters (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="407" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="408" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>), and as well as random genes for one parameter (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="409" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>). On the whole, this analysis suggested that denitrification genes were better predictors than arbitrary genes, but it remains unclear why arbitrary genes tend have modest predictive power <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper R overbar Subscript upper C upper V Superscript 2 Baseline greater than 0 right parenthesis" tabindex="0" ctxtmenu_counter="410" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 0 (13 (10 (9 (3 1 2) (7 4 6 5)) 8) 11 12) 14)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="13" data-semantic-content="0,14" data-semantic-owns="0 13 14" data-semantic-speech="left parenthesis upper R overbar Subscript upper C upper V Superscript 2 Baseline greater than 0 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="13" data-semantic-children="10,12" data-semantic-content="11" data-semantic-parent="15" data-semantic-owns="10 11 12"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="3,7,8" data-semantic-parent="13" data-semantic-collapsed="(10 (9 3 7) 8)" data-semantic-owns="3 7 8"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="10" data-semantic-owns="1 2"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="2" data-semantic-parent="3"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="10" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, and why the denitrification genes performed worse than the typical set of random genes for the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="411" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> parameter. One possible explanation is that some random genes served as good predictors only because they resemble the denitrification genes in terms of presence and absence.</div><div id="p0760" role="paragraph">To investigate whether the predictive power of random genes arises because of correlations (in terms of presence and absence) with the denitrification genes, new sets of random gene predictors were constructed via rejection sampling as described above, but this time sampling only from the set of genes that have low or non-significant correlation with denitrification genes. Specifically, for each of the non-denitrification gene presence/absence vectors, the Pearson correlation <i>ρ</i> was computed with each of the denitrification gene presence/absence vectors. This identified the set of non-denitrification genes that have large correlations (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartAbsoluteValue rho EndAbsoluteValue greater than or equals 0 period 5" tabindex="0" ctxtmenu_counter="412" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (7 0 1 2) 3 4) 5 6)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="9" data-semantic-children="8,5,6" data-semantic-content="5" data-semantic-owns="8 5 6" data-semantic-speech="StartAbsoluteValue rho EndAbsoluteValue greater than or equals 0 period 5"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="8" data-semantic-children="7,4" data-semantic-content="3" data-semantic-parent="9" data-semantic-owns="7 3 4"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="7" data-semantic-children="1" data-semantic-content="0,2" data-semantic-parent="8" data-semantic-owns="0 1 2"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="0" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="7"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="2" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="3" data-semantic-parent="8" data-semantic-operator="relseq,≥"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="5" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>) with any of the denitrification genes that are significant at the 1% level, where significance was evaluated by generating a zero-correlation null distribution via a permutation test. The latter consideration is important for evaluating correlations between very high or very low-frequency genes, for which the probability of spurious large correlations can be high. These denitrification gene-correlates were excluded from sampling when generating the new sets of random gene predictors. As before, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 times 10 cubed" tabindex="0" ctxtmenu_counter="413" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="1 times 10 cubed"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> different sets of 17 predictors for regressing against each of the consumer-resource parameters were generated.</div><div id="p0765" role="paragraph">LASSO regressions on each of the SDM consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="414" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were again performed using the rejection-sampled genes that exclude denitrification gene correlates. The distributions of the resulting <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="415" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> values are shown in <a id="crosref1765" href="#figs2">Figure&nbsp;S2</a>C. The following changes in these distributions relative to <a id="crosref1770" href="#figs2">Figure&nbsp;S2</a>B were observed: (a) the median predictive power for all random gene regressions decreased substantially (i.e., <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="416" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> decreased), indicating that indeed a substantial part of the predictive power of random genes arises due to large correlations with denitrification genes, and (b) denitrification genes now outperformed the typical set of random genes as predictors for even the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="417" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> regression, demonstrating the superiority of denitrification genes as predictors for regressions on all consumer-resource parameters.</div><div id="p0770" role="paragraph">While the superior predictive power of the 17 denitrification genes was demonstrated in direct comparisons with sets of 17 randomly-selected genes, there may exist genes outside of the denitrification pathway with complementary predictive power when added to the denitrification genes. Therefore it was investigated whether adding additional randomly-selected genes to the denitrification genes further improves the predictive power of regressions on the consumer-resource parameters. To address this question, sets of predictor variables were generated by randomly sampling (without replacement) <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P minus 17" tabindex="0" ctxtmenu_counter="418" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper P minus 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.676em;">1</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> genes from the set of 13364 non-denitrification-related genes and constructing binary presence/absence matrices using the denitrification genes as the first 17 predictors and the randomly-selected genes as the subsequent <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P minus 17" tabindex="0" ctxtmenu_counter="419" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper P minus 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.676em;">1</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> predictors. <i>P</i> was varied between 32 and 2048 to investigate the dependence of prediction quality on the number of additional genes added. 10 sets of predictor variables were randomly generated for each value of P. Then LASSO regressions on each of the SDM consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="420" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were performed using these predictors. The distributions of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="421" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> as a function of <i>P</i> are shown in <a id="crosref1775" href="#figs2">Figure&nbsp;S2</a>D. Adding additional predictors beyond the 17 denitrification genes did not substantially improve the predictive power of the models, regardless of how many additional predictors were added. This indicates that the 17 denitrification genes harbor the majority of gene presence/absence predictive power.</div></section><section id="sec-5-5-6"><h4>Evaluating alternative genomic predictors</h4><div id="p0775" role="paragraph">It was investigated whether either 16S rRNA copy number, genome size, or GC-content could serve as better predictor variables for the consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="422" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> than denitrification genes. Previous work demonstrates a positive relationship between 16S copy number and maximum potential growth rate across diverse taxa in nutrient-replete conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref90-2" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-2" aria-controls="sref90-2">Roller et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref90-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Roller, B.R.K. ∙ Stoddard, S.F. ∙ Schmidt, T.M.</div><div><strong>Exploiting rRNA operon copy number to investigate bacterial reproductive strategies</strong></div><div><em>Nat. Microbiol.</em> 2016; <strong>1</strong>:16160</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmicrobiol.2016.160" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84987667484" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27617693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmicrobiol.2016.160&amp;pmid=27617693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-3" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-3" aria-controls="sref59-3">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Li, J. ∙ Mau, R.L. ∙ Dijkstra, P. ...</div><div><strong>Predictive genomic traits for bacterial growth in culture versus actual growth in soil</strong></div><div><em>ISME J.</em> 2019; <strong>13</strong>:2162-2172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41396-019-0422-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85065340904" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31053828/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41396-019-0422-z&amp;pmid=31053828" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), likely because increased ribosome production allows a higher rate of protein synthesis, thereby increasing growth rate (<span class="dropBlock reference-citations"><a id="body-ref-sref97-1" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-1" aria-controls="sref97-1">Scott et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref97-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Scott, M. ∙ Gunderson, C.W. ∙ Mateescu, E.M. ...</div><div><strong>Interdependence of cell growth and gene expression: origins and consequences</strong></div><div><em>Science.</em> 2010; <strong>330</strong>:1099-1102</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1192588" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-78449268845" target="_blank" aria-label="Open Scopus in new tab">Scopus (698)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21097934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1192588&amp;pmid=21097934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A negative relationship between growth rate and genome size has also been observed (<span class="dropBlock reference-citations"><a id="body-ref-sref59-4" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-4" aria-controls="sref59-4">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Li, J. ∙ Mau, R.L. ∙ Dijkstra, P. ...</div><div><strong>Predictive genomic traits for bacterial growth in culture versus actual growth in soil</strong></div><div><em>ISME J.</em> 2019; <strong>13</strong>:2162-2172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41396-019-0422-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85065340904" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31053828/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41396-019-0422-z&amp;pmid=31053828" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), possibly due to a reduced nutrient burden required by smaller genomes (<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Hessen et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Hessen, D.O. ∙ Jeyasingh, P.D. ∙ Neiman, M. ...</div><div><strong>Genome streamlining and the elemental costs of growth</strong></div><div><em>Trends Ecol. Evol.</em> 2010; <strong>25</strong>:75-80</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.tree.2009.08.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.tree.2009.08.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-74049085522" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19796842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tree.2009.08.004&amp;pmid=19796842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GC-content has been investigated as a genomic predictor for various bacterial phenotypes such as optimal growth temperature (<span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">Galtier and Lobry, 1997</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Galtier, N. ∙ Lobry, J.R.</div><div><strong>Relationships between genomic G+C content, RNA secondary structures, and optimal growth temperature in prokaryotes</strong></div><div><em>J.&nbsp;Mol. Evol.</em> 1997; <strong>44</strong>:632-636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/PL00006186" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0030900384" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9169555/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FPL00006186&amp;pmid=9169555" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and can serve as a baseline for spurious phylogenetic correlations because it is a slowly-evolving genomic property (<span class="dropBlock reference-citations"><a id="body-ref-sref39-2" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-2" aria-controls="sref39-2">Haywood- Farmer and Otto, 2003</a><div class="dropBlock__holder" data-db-target-of="sref39-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Haywood-Farmer, E. ∙ Otto, S.P.</div><div><strong>The evolution of genomic base composition in bacteria</strong></div><div><em>Evolution.</em> 2003; <strong>57</strong>:1783-1792</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.0014-3820.2003.tb00586.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0042287719" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14503620/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.0014-3820.2003.tb00586.x&amp;pmid=14503620" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0780" role="paragraph">16S copy number was estimated for all 79 strains in the library using the 16Stimator pipeline (<span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Perisin et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Perisin, M. ∙ Vetter, M. ∙ Gilbert, J.A. ...</div><div><strong>16Stimator: statistical estimation of ribosomal gene copy numbers from draft genome assemblies</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:1020-1024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2015.161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84941253528" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26359911/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2015.161&amp;pmid=26359911" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This approach uses Illumina sequencing reads and annotated draft assemblies to compute the coverage ratio of the 16S gene relative to a curated set of single-copy genes. The genome size was estimated for all strains in the library by summing the lengths of all assembled contigs for each strain. Genomic GC-content was computed from draft assemblies using QUAST 5.02 (<span class="dropBlock reference-citations"><a id="body-ref-sref34-3" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-3" aria-controls="sref34-3">Gurevich et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref34-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Gurevich, A. ∙ Saveliev, V. ∙ Vyahhi, N. ...</div><div><strong>QUAST: quality assessment tool for genome assemblies</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1072-1075</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt086" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84876266928" target="_blank" aria-label="Open Scopus in new tab">Scopus (2977)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt086&amp;pmid=23422339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For the reference strain <i>Paracoccus denitrificans</i> ATCC 19367, a complete genome assembly (<span class="dropBlock reference-citations"><a id="body-ref-sref101-3" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-3" aria-controls="sref101-3">Si et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref101-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Si, Y.-Y. ∙ Xu, K.-H. ∙ Yu, X.-Y. ...</div><div><strong>Complete genome sequence of <i>Paracoccus denitrificans</i> ATCC 19367 and its denitrification characteristics</strong></div><div><em>Can. J. Microbiol.</em> 2019; <strong>65</strong>:486-495</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1139/cjm-2019-0037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85068138644" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30897350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1139%2Fcjm-2019-0037&amp;pmid=30897350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) showed 3 copies of the 16S gene, a genome size of 5.24 mb, and a GC-content of 66.80%, and 3.36 16S copies; a genome size of 5.15 mb, and a GC-content of 66.81% were estimated using the draft assembly.</div><div id="p0785" role="paragraph">First the relationships between 16S copy number, genome size, GC-content, and the SDM consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="423" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were considered. No significant (permutation test) correlations were observed between 16S copy number and the consumer-resource parameters, significant positive correlations were observed between genome size and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="424" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho equals 0 period 40" tabindex="0" ctxtmenu_counter="425" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="rho equals 0 period 40"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c34" noic="true" style="padding-top: 0.677em;">4</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.677em;">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than 1 times 10 Superscript negative 4" tabindex="0" ctxtmenu_counter="426" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (9 2 3 (8 4 (7 5 6))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="10" data-semantic-children="0,9" data-semantic-content="1" data-semantic-owns="0 1 9" data-semantic-speech="p less than 1 times 10 Superscript negative 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-children="2,8" data-semantic-content="3" data-semantic-parent="10" data-semantic-owns="2 3 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="8" data-semantic-children="4,7" data-semantic-parent="9" data-semantic-owns="4 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-children="6" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="5 6"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>) and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="427" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho equals 0 period 63" tabindex="0" ctxtmenu_counter="428" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-structure="(6 (5 0 1 2) 3 4)" data-semantic-speech="rho equals 0 period 63" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2" inline-breaks="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.666em;">6</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.666em;">3</mjx-c></mjx-mn></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than 1 times 10 Superscript negative 4" tabindex="0" ctxtmenu_counter="429" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (9 2 3 (8 4 (7 5 6))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="10" data-semantic-children="0,9" data-semantic-content="1" data-semantic-owns="0 1 9" data-semantic-speech="p less than 1 times 10 Superscript negative 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-children="2,8" data-semantic-content="3" data-semantic-parent="10" data-semantic-owns="2 3 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="8" data-semantic-children="4,7" data-semantic-parent="9" data-semantic-owns="4 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-children="6" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="5 6"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>), and significant but weak positive correlations were observed between GC-content and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="430" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho equals 0 period 24" tabindex="0" ctxtmenu_counter="431" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="rho equals 0 period 24"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.677em;">2</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p equals 0 period 02" tabindex="0" ctxtmenu_counter="432" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="p equals 0 period 02"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c32" style="padding-top: 0.666em;">2</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>) and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="433" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho equals 0 period 23" tabindex="0" ctxtmenu_counter="434" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="rho equals 0 period 23"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.666em;">2</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.666em;">3</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p equals 0 period 04" tabindex="0" ctxtmenu_counter="435" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="p equals 0 period 04"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.677em;">0</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>). These data suggest that genome size may be a good predictor for the yields <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A" tabindex="0" ctxtmenu_counter="436" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I" tabindex="0" ctxtmenu_counter="437" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0790" role="paragraph">In order to compare consumer-resource parameter measurements to previous findings relating growth rate to 16S copy number and genome size, growth rates on nitrate and nitrite were computed for each strain. <a id="crosref1835" href="#fd3">Equation 3</a> can be recast in terms of growth rates <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu Subscript italic asterisk" tabindex="0" ctxtmenu_counter="438" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="mu Subscript italic asterisk"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (1/h) by setting <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript italic asterisk Baseline equals mu Subscript italic asterisk Baseline divided by gamma Subscript italic asterisk" tabindex="0" ctxtmenu_counter="439" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 (2 0 1) 3 (11 (6 4 5) 7 (10 8 9)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="12" data-semantic-children="2,11" data-semantic-content="3" data-semantic-owns="2 3 11" data-semantic-speech="r Subscript italic asterisk Baseline equals mu Subscript italic asterisk Baseline divided by gamma Subscript italic asterisk"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="12" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="12" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="11" data-semantic-children="6,10" data-semantic-content="7" data-semantic-parent="12" data-semantic-owns="6 7 10"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="11" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="7" data-semantic-parent="11" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="11" data-semantic-owns="8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-math></mjx-container>:<div id="fd9" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartLayout 1st Row  StartFraction d x Over d t EndFraction equals left parenthesis mu Subscript upper A Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction plus mu Subscript upper I Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x comma 2nd Row  StartFraction d upper A Over d t EndFraction equals minus StartFraction mu Subscript upper A Baseline Over gamma Subscript upper A Baseline EndFraction StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction x comma 3rd Row  StartFraction d upper I Over d t EndFraction equals left parenthesis StartFraction mu Subscript upper A Baseline Over gamma Subscript upper A Baseline EndFraction StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction minus StartFraction mu Subscript upper I Baseline Over gamma Subscript upper I Baseline EndFraction StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x period EndLayout" tabindex="0" ctxtmenu_counter="440" style="font-size: 130%;"><mjx-math data-semantic-structure="(142 (48 (46 (45 (8 (3 0 2 1) (7 4 6 5)) 9 (44 (40 38 (37 (34 (12 10 11) 33 (20 13 (19 (16 14 15) 17 18))) 21 (36 (24 22 23) 35 (32 25 (31 (28 26 27) 29 30)))) 39) 43 41)) 42)) (84 (82 (81 (57 (52 49 51 50) (56 53 55 54)) 58 (80 59 (79 (66 (62 60 61) (65 63 64)) 77 (74 67 (73 (70 68 69) 71 72)) 78 75))) 76)) (141 (139 (138 (93 (88 85 87 86) (92 89 91 90)) 94 (137 (133 131 (130 (127 (101 (97 95 96) (100 98 99)) 126 (109 102 (108 (105 103 104) 106 107))) 110 (129 (117 (113 111 112) (116 114 115)) 128 (125 118 (124 (121 119 120) 122 123)))) 132) 136 134)) 135)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mtable data-semantic-type="multiline" data-semantic-role="unknown" data-semantic-id="142" data-semantic-children="48,84,141" data-semantic-owns="48 84 141" data-semantic-speech="StartLayout 1st Row  StartFraction d x Over d t EndFraction equals left parenthesis mu Subscript upper A Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction plus mu Subscript upper I Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x comma 2nd Row  StartFraction d upper A Over d t EndFraction equals minus StartFraction mu Subscript upper A Baseline Over gamma Subscript upper A Baseline EndFraction StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction x comma 3rd Row  StartFraction d upper I Over d t EndFraction equals left parenthesis StartFraction mu Subscript upper A Baseline Over gamma Subscript upper A Baseline EndFraction StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction minus StartFraction mu Subscript upper I Baseline Over gamma Subscript upper I Baseline EndFraction StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x period EndLayout" justify="left" style="min-width: 13.055em;"><mjx-table><mjx-itable><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="48" data-semantic-children="46" data-semantic-parent="142" data-semantic-owns="46"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="46" data-semantic-children="45,42" data-semantic-content="42" data-semantic-parent="48" data-semantic-owns="45 42"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="45" data-semantic-children="8,44" data-semantic-content="9" data-semantic-parent="46" data-semantic-owns="8 9 44"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="45" data-semantic-owns="3 7"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="3" data-semantic-children="0,1" data-semantic-content="2" data-semantic-parent="8" data-semantic-owns="0 2 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="2" data-semantic-parent="3" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="45" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="44" data-semantic-children="40,41" data-semantic-content="43" data-semantic-parent="45" data-semantic-owns="40 43 41"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="40" data-semantic-children="37" data-semantic-content="38,39" data-semantic-parent="44" data-semantic-owns="38 37 39"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="38" data-semantic-parent="40" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="37" data-semantic-children="34,36" data-semantic-content="21" data-semantic-parent="40" data-semantic-owns="34 21 36"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="34" data-semantic-children="12,20" data-semantic-content="33" data-semantic-parent="37" data-semantic-owns="12 33 20"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="34" data-semantic-owns="10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="20" data-semantic-children="13,19" data-semantic-parent="34" data-semantic-owns="13 19"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="20"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="19" data-semantic-children="16,18" data-semantic-content="17" data-semantic-parent="20" data-semantic-owns="16 17 18"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="19" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="17" data-semantic-parent="19" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="21" data-semantic-parent="37" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="36" data-semantic-children="24,32" data-semantic-content="35" data-semantic-parent="37" data-semantic-owns="24 35 32"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="24" data-semantic-children="22,23" data-semantic-parent="36" data-semantic-owns="22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="24"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="24"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="35" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="32" data-semantic-children="25,31" data-semantic-parent="36" data-semantic-owns="25 31"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="32"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="31" data-semantic-children="28,30" data-semantic-content="29" data-semantic-parent="32" data-semantic-owns="28 29 30"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="26,27" data-semantic-parent="31" data-semantic-owns="26 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="29" data-semantic-parent="31" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="39" data-semantic-parent="40" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="44"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="42" data-semantic-parent="46" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="84" data-semantic-children="82" data-semantic-parent="142" data-semantic-owns="82"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="82" data-semantic-children="81,76" data-semantic-content="76" data-semantic-parent="84" data-semantic-owns="81 76"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="81" data-semantic-children="57,80" data-semantic-content="58" data-semantic-parent="82" data-semantic-owns="57 58 80"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="57" data-semantic-children="52,56" data-semantic-parent="81" data-semantic-owns="52 56"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="52" data-semantic-children="49,50" data-semantic-content="51" data-semantic-parent="57" data-semantic-owns="49 51 50"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="52"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="51" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="50" data-semantic-parent="52"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="56" data-semantic-children="53,54" data-semantic-content="55" data-semantic-parent="57" data-semantic-owns="53 55 54"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="53" data-semantic-parent="56"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="55" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="54" data-semantic-parent="56"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="58" data-semantic-parent="81" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="80" data-semantic-children="79" data-semantic-content="59" data-semantic-parent="81" data-semantic-owns="59 79"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="59" data-semantic-parent="80" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="79" data-semantic-children="66,74,75" data-semantic-content="77,78" data-semantic-parent="80" data-semantic-owns="66 77 74 78 75"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="66" data-semantic-children="62,65" data-semantic-parent="79" data-semantic-owns="62 65"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="62" data-semantic-children="60,61" data-semantic-parent="66" data-semantic-owns="60 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="62"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="62"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="65" data-semantic-children="63,64" data-semantic-parent="66" data-semantic-owns="63 64"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="63" data-semantic-parent="65"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="65"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="77" data-semantic-parent="79" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="74" data-semantic-children="67,73" data-semantic-parent="79" data-semantic-owns="67 73"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="67" data-semantic-parent="74"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="73" data-semantic-children="70,72" data-semantic-content="71" data-semantic-parent="74" data-semantic-owns="70 71 72"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="70" data-semantic-children="68,69" data-semantic-parent="73" data-semantic-owns="68 69"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="68" data-semantic-parent="70"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="69" data-semantic-parent="70"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="71" data-semantic-parent="73" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="73"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="78" data-semantic-parent="79" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="75" data-semantic-parent="79"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="76" data-semantic-parent="82" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="141" data-semantic-children="139" data-semantic-parent="142" data-semantic-owns="139"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="139" data-semantic-children="138,135" data-semantic-content="135" data-semantic-parent="141" data-semantic-owns="138 135"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="138" data-semantic-children="93,137" data-semantic-content="94" data-semantic-parent="139" data-semantic-owns="93 94 137"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="93" data-semantic-children="88,92" data-semantic-parent="138" data-semantic-owns="88 92"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="88" data-semantic-children="85,86" data-semantic-content="87" data-semantic-parent="93" data-semantic-owns="85 87 86"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="85" data-semantic-parent="88"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="87" data-semantic-parent="88" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="86" data-semantic-parent="88"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="92" data-semantic-children="89,90" data-semantic-content="91" data-semantic-parent="93" data-semantic-owns="89 91 90"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="89" data-semantic-parent="92"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="91" data-semantic-parent="92" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="90" data-semantic-parent="92"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="94" data-semantic-parent="138" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="137" data-semantic-children="133,134" data-semantic-content="136" data-semantic-parent="138" data-semantic-owns="133 136 134"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="133" data-semantic-children="130" data-semantic-content="131,132" data-semantic-parent="137" data-semantic-owns="131 130 132"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="131" data-semantic-parent="133" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="130" data-semantic-children="127,129" data-semantic-content="110" data-semantic-parent="133" data-semantic-owns="127 110 129"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="127" data-semantic-children="101,109" data-semantic-content="126" data-semantic-parent="130" data-semantic-owns="101 126 109"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="101" data-semantic-children="97,100" data-semantic-parent="127" data-semantic-owns="97 100"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="97" data-semantic-children="95,96" data-semantic-parent="101" data-semantic-owns="95 96"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="95" data-semantic-parent="97"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="96" data-semantic-parent="97"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="100" data-semantic-children="98,99" data-semantic-parent="101" data-semantic-owns="98 99"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="98" data-semantic-parent="100"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="99" data-semantic-parent="100"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="126" data-semantic-parent="127" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="109" data-semantic-children="102,108" data-semantic-parent="127" data-semantic-owns="102 108"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="102" data-semantic-parent="109"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="108" data-semantic-children="105,107" data-semantic-content="106" data-semantic-parent="109" data-semantic-owns="105 106 107"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="105" data-semantic-children="103,104" data-semantic-parent="108" data-semantic-owns="103 104"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="103" data-semantic-parent="105"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="104" data-semantic-parent="105"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="106" data-semantic-parent="108" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="107" data-semantic-parent="108"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="110" data-semantic-parent="130" data-semantic-operator="infixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="129" data-semantic-children="117,125" data-semantic-content="128" data-semantic-parent="130" data-semantic-owns="117 128 125"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="117" data-semantic-children="113,116" data-semantic-parent="129" data-semantic-owns="113 116"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="113" data-semantic-children="111,112" data-semantic-parent="117" data-semantic-owns="111 112"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="111" data-semantic-parent="113"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="112" data-semantic-parent="113"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="116" data-semantic-children="114,115" data-semantic-parent="117" data-semantic-owns="114 115"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="114" data-semantic-parent="116"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="115" data-semantic-parent="116"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="128" data-semantic-parent="129" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="125" data-semantic-children="118,124" data-semantic-parent="129" data-semantic-owns="118 124"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="118" data-semantic-parent="125"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="124" data-semantic-children="121,123" data-semantic-content="122" data-semantic-parent="125" data-semantic-owns="121 122 123"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="121" data-semantic-children="119,120" data-semantic-parent="124" data-semantic-owns="119 120"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="119" data-semantic-parent="121"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="120" data-semantic-parent="121"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="122" data-semantic-parent="124" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="123" data-semantic-parent="124"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="132" data-semantic-parent="133" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="136" data-semantic-parent="137" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="134" data-semantic-parent="137"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="135" data-semantic-parent="139" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr></mjx-itable></mjx-table></mjx-mtable></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">(Equation 9)</div></div></div><div id="p0795" role="paragraph">In this form the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu Subscript italic asterisk" tabindex="0" ctxtmenu_counter="441" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="mu Subscript italic asterisk"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> represent the exponential growth rates when substrates are not limiting (e.g., when <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A much greater than upper K Subscript upper A" tabindex="0" ctxtmenu_counter="442" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="upper A much greater than upper K Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,≫"><mjx-c class="mjx-c226B">≫</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> or <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I much greater than upper K Subscript upper I" tabindex="0" ctxtmenu_counter="443" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="upper I much greater than upper K Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,≫"><mjx-c class="mjx-c226B">≫</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>). Therefore <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu Subscript italic asterisk Baseline equals r Subscript italic asterisk Baseline gamma Subscript italic asterisk" tabindex="0" ctxtmenu_counter="444" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 (2 0 1) 3 (11 (6 4 5) 10 (9 7 8)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="12" data-semantic-children="2,11" data-semantic-content="3" data-semantic-owns="2 3 11" data-semantic-speech="mu Subscript italic asterisk Baseline equals r Subscript italic asterisk Baseline gamma Subscript italic asterisk"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="12" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="12" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="6,9" data-semantic-content="10" data-semantic-parent="12" data-semantic-owns="6 10 9"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="11" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="11" data-semantic-owns="7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-font="italic" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were computed using the SDM consumer-resource parameters.</div><div id="p0800" role="paragraph">No significant positive correlations were observed between either of the growth rates with 16S copy number. This differs from what has been previously observed under aerobic, nutrient-replete conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref90-3" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-3" aria-controls="sref90-3">Roller et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref90-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Roller, B.R.K. ∙ Stoddard, S.F. ∙ Schmidt, T.M.</div><div><strong>Exploiting rRNA operon copy number to investigate bacterial reproductive strategies</strong></div><div><em>Nat. Microbiol.</em> 2016; <strong>1</strong>:16160</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmicrobiol.2016.160" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84987667484" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27617693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmicrobiol.2016.160&amp;pmid=27617693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-5" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-5" aria-controls="sref59-5">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Li, J. ∙ Mau, R.L. ∙ Dijkstra, P. ...</div><div><strong>Predictive genomic traits for bacterial growth in culture versus actual growth in soil</strong></div><div><em>ISME J.</em> 2019; <strong>13</strong>:2162-2172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41396-019-0422-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85065340904" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31053828/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41396-019-0422-z&amp;pmid=31053828" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), where ribosomal production is likely to be growth limiting. This may be because these previous studies were based on maximum potential growth rates (measured across a variety of medium and culturing conditions), whereas maximum potential growth rates were not systematically identified for strains in the library. Moreover it is likely that most of the measured growth rates in denitrifying conditions with a succinate defined medium are much smaller than maximum potential growth rates, because denitrification is generally a significantly lower ATP-yielding process than aerobic respiration (<span class="dropBlock reference-citations"><a id="body-ref-sref105-2" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-2" aria-controls="sref105-2">Strohm et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref105-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Strohm, T.O. ∙ Griffin, B. ∙ Zumft, W.G. ...</div><div><strong>Growth yields in bacterial denitrification and nitrate ammonification</strong></div><div><em>Appl. Environ. Microbiol.</em> 2007; <strong>73</strong>:1420-1424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/AEM.02508-06" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33947193902" target="_blank" aria-label="Open Scopus in new tab">Scopus (174)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17209072/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FAEM.02508-06&amp;pmid=17209072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and also because succinate defined medium likely imposes more stringent biosynthesis demands than a complex, rich medium (<span class="dropBlock reference-citations"><a id="body-ref-sref97-2" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-2" aria-controls="sref97-2">Scott et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref97-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Scott, M. ∙ Gunderson, C.W. ∙ Mateescu, E.M. ...</div><div><strong>Interdependence of cell growth and gene expression: origins and consequences</strong></div><div><em>Science.</em> 2010; <strong>330</strong>:1099-1102</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1192588" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-78449268845" target="_blank" aria-label="Open Scopus in new tab">Scopus (698)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21097934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1192588&amp;pmid=21097934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Therefore the relative benefit of high gene copy number of rRNA for many strains under denitrifying growth conditions may be small. However significant (permutation test) positive correlations were observed between genome size and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu Subscript upper A" tabindex="0" ctxtmenu_counter="445" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="mu Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho equals 0 period 21" tabindex="0" ctxtmenu_counter="446" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="rho equals 0 period 21"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.666em;">2</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p equals 0 period 04" tabindex="0" ctxtmenu_counter="447" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 1 2) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-speech="p equals 0 period 04"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.677em;">0</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>), and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu Subscript upper I" tabindex="0" ctxtmenu_counter="448" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="mu Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho equals 0 period 63" tabindex="0" ctxtmenu_counter="449" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-owns="5 3 4" data-semantic-structure="(6 (5 0 1 2) 3 4)" data-semantic-speech="rho equals 0 period 63" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2" inline-breaks="true"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.666em;">6</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.666em;">3</mjx-c></mjx-mn></mjx-math></mjx-container> 0.52, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p less than 1 times 10 Superscript negative 4" tabindex="0" ctxtmenu_counter="450" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (9 2 3 (8 4 (7 5 6))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="10" data-semantic-children="0,9" data-semantic-content="1" data-semantic-owns="0 1 9" data-semantic-speech="p less than 1 times 10 Superscript negative 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-children="2,8" data-semantic-content="3" data-semantic-parent="10" data-semantic-owns="2 3 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="8" data-semantic-children="4,7" data-semantic-parent="9" data-semantic-owns="4 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-children="6" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="5 6"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>). This finding also differs the previous observation of a negative relationship between genome size and growth rate (<span class="dropBlock reference-citations"><a id="body-ref-sref59-6" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-6" aria-controls="sref59-6">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Li, J. ∙ Mau, R.L. ∙ Dijkstra, P. ...</div><div><strong>Predictive genomic traits for bacterial growth in culture versus actual growth in soil</strong></div><div><em>ISME J.</em> 2019; <strong>13</strong>:2162-2172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41396-019-0422-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85065340904" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31053828/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41396-019-0422-z&amp;pmid=31053828" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0805" role="paragraph">Finally, the potential of 16S copy number, genome size, and GC-content as predictor variables was evaluated in a head-to-head comparison with denitrification genes by using all these predictors simultaneously in LASSO regressions on the SDM consumer-resource parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet" tabindex="0" ctxtmenu_counter="451" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 (16 (3 1 2) 4 (7 5 6) 8 (11 9 10) 12 (15 13 14)) 17)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="18" data-semantic-children="16" data-semantic-content="0,17" data-semantic-owns="0 16 17" data-semantic-speech="StartSet r Subscript upper A Baseline comma gamma Subscript upper A Baseline comma r Subscript upper I Baseline comma gamma Subscript upper I Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="3,4,7,8,11,12,15" data-semantic-content="4,8,12" data-semantic-parent="18" data-semantic-owns="3 4 7 8 11 12 15"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="16" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="16" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="16" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="16" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> (4-fold cross-validation, iterated <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 times 10 Superscript 4" tabindex="0" ctxtmenu_counter="452" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="1 times 10 Superscript 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> times). As before, all predictors were standardized to have zero mean and unit variance before fitting. The following was observed: (a) 16S copy number, genome size, and GC-content were not assigned large coefficients in any regression, (b) the statistics of fitting and generalization quality (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript f i t Superscript 2" tabindex="0" ctxtmenu_counter="453" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 0 (6 1 4 2 5 3)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,6,7" data-semantic-collapsed="(9 (8 0 6) 7)" data-semantic-owns="0 6 7" data-semantic-speech="upper R Subscript f i t Superscript 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="9" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="454" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>) were essentially the same as those obtained in the original regressions, and (c) the coefficients for denitrification genes were very similar to those obtained in the original regressions. It was concluded that the denitrification genes hold greater predictive power than 16S copy number, genome size, and GC-content, since the latter predictors are not selected as important variables by LASSO regression.</div></section><section id="sec-5-5-7"><h4>Regressing ADM phenotypes onto denitrification gene content</h4><div id="p0810" role="paragraph">Regressions for the consumer-resource parameters measured in ADM were performed as for those parameters measured in SDM. The median values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R squared" tabindex="0" ctxtmenu_counter="455" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> obtained via iterated 4-fold cross-validation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M equals 10 Superscript 4" tabindex="0" ctxtmenu_counter="456" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="upper M equals 10 Superscript 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> iterations) (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="457" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>) were positive for all regressions. The <i>p</i>-values of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R overbar Subscript upper C upper V Superscript 2" tabindex="0" ctxtmenu_counter="458" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) (6 3 5 4)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,6,7" data-semantic-collapsed="(9 (8 2 6) 7)" data-semantic-owns="2 6 7" data-semantic-speech="upper R overbar Subscript upper C upper V Superscript 2"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.018em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.759em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="3 5 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> (permutation test) were all <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than 0 period 01" tabindex="0" ctxtmenu_counter="459" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 4 0 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,2,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="less than 0 period 01"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="5" data-semantic-children="4,1" data-semantic-content="0" data-semantic-parent="6" data-semantic-owns="4 0 1"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>.</div></section><section id="sec-5-5-8"><h4>Predicting community metabolic dynamics</h4><div id="p0815" role="paragraph">The consumer-resource model (<a id="crosref1865" href="#fd3">Equation 3</a>) was extended to generate predictions for community (i.e., multi-strain) metabolite dynamics. For an <i>N</i>-strain community, the extended model is as follows:<div id="fd10" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartLayout 1st Row  StartFraction d x Subscript i Baseline Over d t EndFraction equals left parenthesis gamma Subscript upper A Baseline r Subscript upper A Superscript i Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction plus gamma Subscript upper I Superscript i Baseline r Subscript upper I Superscript i Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x Subscript i Baseline comma for i equals 1 comma ellipsis comma upper N 2nd Row  StartFraction d upper A Over d t EndFraction equals minus sigma summation Underscript i equals 1 Overscript upper N Endscripts r Subscript upper A Superscript i Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction x Subscript i Baseline comma 3rd Row  StartFraction d upper I Over d t EndFraction equals sigma summation Underscript i equals 1 Overscript upper N Endscripts left parenthesis r Subscript upper A Superscript i Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction minus r Subscript upper I Superscript i Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x Subscript i Baseline period EndLayout" tabindex="0" ctxtmenu_counter="460" style="font-size: 130%;"><mjx-math data-semantic-structure="(187 (79 (77 (72 (10 (5 0 4 (3 1 2)) (9 6 8 7)) 11 (71 (56 54 (53 (49 (14 12 13) 47 (19 (18 15 16) 17) 48 (27 20 (26 (23 21 22) 24 25))) 28 (52 (33 (32 29 30) 31) 50 (38 (37 34 35) 36) 51 (46 39 (45 (42 40 41) 43 44)))) 55) 70 (59 57 58))) 60 (76 61 62 (73 63 64 65)) 66 67 68 69)) (123 (121 (120 (88 (83 80 82 81) (87 84 86 85)) 89 (119 90 (118 (96 91 (95 92 93 94) 97) (117 (102 (101 98 99) 100) 115 (110 103 (109 (106 104 105) 107 108)) 116 (113 111 112))))) 114)) (186 (184 (183 (132 (127 124 126 125) (131 128 130 129)) 133 (182 (140 134 (138 135 136 137) 139) (181 (175 173 (172 (169 (145 (144 141 142) 143) 168 (153 146 (152 (149 147 148) 150 151))) 154 (171 (159 (158 155 156) 157) 170 (167 160 (166 (163 161 162) 164 165)))) 174) 180 (178 176 177)))) 179)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mtable data-semantic-type="multiline" data-semantic-role="unknown" data-semantic-id="187" data-semantic-children="79,123,186" data-semantic-owns="79 123 186" data-semantic-speech="StartLayout 1st Row  StartFraction d x Subscript i Baseline Over d t EndFraction equals left parenthesis gamma Subscript upper A Baseline r Subscript upper A Superscript i Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction plus gamma Subscript upper I Superscript i Baseline r Subscript upper I Superscript i Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x Subscript i Baseline comma for i equals 1 comma ellipsis comma upper N 2nd Row  StartFraction d upper A Over d t EndFraction equals minus sigma summation Underscript i equals 1 Overscript upper N Endscripts r Subscript upper A Superscript i Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction x Subscript i Baseline comma 3rd Row  StartFraction d upper I Over d t EndFraction equals sigma summation Underscript i equals 1 Overscript upper N Endscripts left parenthesis r Subscript upper A Superscript i Baseline StartFraction upper A Over upper K Subscript upper A Baseline plus upper A EndFraction minus r Subscript upper I Superscript i Baseline StartFraction upper I Over upper K Subscript upper I Baseline plus upper I EndFraction right parenthesis x Subscript i Baseline period EndLayout" justify="left" style="min-width: 22.875em;"><mjx-table><mjx-itable><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="79" data-semantic-children="77" data-semantic-parent="187" data-semantic-owns="77"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="77" data-semantic-children="72,60,76,66,67,68,69" data-semantic-content="60,66,68" data-semantic-parent="79" data-semantic-owns="72 60 76 66 67 68 69"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="72" data-semantic-children="10,71" data-semantic-content="11" data-semantic-parent="77" data-semantic-owns="10 11 71"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="10" data-semantic-children="5,9" data-semantic-parent="72" data-semantic-owns="5 9"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="5" data-semantic-children="0,3" data-semantic-content="4" data-semantic-parent="10" data-semantic-owns="0 4 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="9" data-semantic-children="6,7" data-semantic-content="8" data-semantic-parent="10" data-semantic-owns="6 8 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="11" data-semantic-parent="72" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="71" data-semantic-children="56,59" data-semantic-content="70" data-semantic-parent="72" data-semantic-owns="56 70 59"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="56" data-semantic-children="53" data-semantic-content="54,55" data-semantic-parent="71" data-semantic-owns="54 53 55"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="54" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="53" data-semantic-children="49,52" data-semantic-content="28" data-semantic-parent="56" data-semantic-owns="49 28 52"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="49" data-semantic-children="14,19,27" data-semantic-content="47,48" data-semantic-parent="53" data-semantic-owns="14 47 19 48 27"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="14" data-semantic-children="12,13" data-semantic-parent="49" data-semantic-owns="12 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="47" data-semantic-parent="49" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="19" data-semantic-children="15,16,17" data-semantic-parent="49" data-semantic-collapsed="(19 (18 15 16) 17)" data-semantic-owns="15 16 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="19"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="19"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="19"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="48" data-semantic-parent="49" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="27" data-semantic-children="20,26" data-semantic-parent="49" data-semantic-owns="20 26"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="27"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="26" data-semantic-children="23,25" data-semantic-content="24" data-semantic-parent="27" data-semantic-owns="23 24 25"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="23" data-semantic-children="21,22" data-semantic-parent="26" data-semantic-owns="21 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="23"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="23"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="24" data-semantic-parent="26" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="28" data-semantic-parent="53" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="52" data-semantic-children="33,38,46" data-semantic-content="50,51" data-semantic-parent="53" data-semantic-owns="33 50 38 51 46"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="33" data-semantic-children="29,30,31" data-semantic-parent="52" data-semantic-collapsed="(33 (32 29 30) 31)" data-semantic-owns="29 30 31"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="33"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="33" style="margin-left: 0.108em;"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="33"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="50" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="38" data-semantic-children="34,35,36" data-semantic-parent="52" data-semantic-collapsed="(38 (37 34 35) 36)" data-semantic-owns="34 35 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="38"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="38"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="38"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="51" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="46" data-semantic-children="39,45" data-semantic-parent="52" data-semantic-owns="39 45"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="46"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="45" data-semantic-children="42,44" data-semantic-content="43" data-semantic-parent="46" data-semantic-owns="42 43 44"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="42" data-semantic-children="40,41" data-semantic-parent="45" data-semantic-owns="40 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="42"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="42"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="43" data-semantic-parent="45" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="45"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="55" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="70" data-semantic-parent="71" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="59" data-semantic-children="57,58" data-semantic-parent="71" data-semantic-owns="57 58"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="57" data-semantic-parent="59"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="58" data-semantic-parent="59"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="60" data-semantic-parent="77" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="76" data-semantic-children="61,62,73" data-semantic-parent="77" data-semantic-collapsed="(76 (c 74 75) 61 62 73)" data-semantic-owns="61 62 73"><mjx-mspace data-semantic-type="text" data-semantic-role="space" data-semantic-annotation="clearspeak:unit" data-semantic-id="61" data-semantic-parent="76" style="width: 1em;"></mjx-mspace><mjx-mtext space="2" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="62" data-semantic-parent="76"><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.705em;">o</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.705em;">&nbsp;</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="73" data-semantic-children="63,65" data-semantic-content="64" data-semantic-parent="76" data-semantic-owns="63 64 65"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="63" data-semantic-parent="73"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="64" data-semantic-parent="73" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="65" data-semantic-parent="73"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="66" data-semantic-parent="77" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="67" data-semantic-parent="77"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="68" data-semantic-parent="77" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="69" data-semantic-parent="77"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="123" data-semantic-children="121" data-semantic-parent="187" data-semantic-owns="121"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="121" data-semantic-children="120,114" data-semantic-content="114" data-semantic-parent="123" data-semantic-owns="120 114"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="120" data-semantic-children="88,119" data-semantic-content="89" data-semantic-parent="121" data-semantic-owns="88 89 119"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="88" data-semantic-children="83,87" data-semantic-parent="120" data-semantic-owns="83 87"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="83" data-semantic-children="80,81" data-semantic-content="82" data-semantic-parent="88" data-semantic-owns="80 82 81"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="80" data-semantic-parent="83"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="82" data-semantic-parent="83" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="81" data-semantic-parent="83"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="87" data-semantic-children="84,85" data-semantic-content="86" data-semantic-parent="88" data-semantic-owns="84 86 85"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="84" data-semantic-parent="87"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="86" data-semantic-parent="87" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="85" data-semantic-parent="87"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="89" data-semantic-parent="120" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="119" data-semantic-children="118" data-semantic-content="90" data-semantic-parent="120" data-semantic-owns="90 118"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="90" data-semantic-parent="119" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="118" data-semantic-children="96,117" data-semantic-content="91" data-semantic-parent="119" data-semantic-owns="96 117"><mjx-mover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="96" data-semantic-children="91,95,97" data-semantic-parent="118" data-semantic-owns="91 95 97" limits="false"><mjx-munder limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="91" data-semantic-parent="96" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="95" data-semantic-children="92,94" data-semantic-content="93" data-semantic-parent="96" data-semantic-owns="92 93 94"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="92" data-semantic-parent="95"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="93" data-semantic-parent="95" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="94" data-semantic-parent="95"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munder><mjx-script style="vertical-align: 0.477em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="97" data-semantic-parent="96"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-script></mjx-mover><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="117" data-semantic-children="102,110,113" data-semantic-content="115,116" data-semantic-parent="118" data-semantic-owns="102 115 110 116 113"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="102" data-semantic-children="98,99,100" data-semantic-parent="117" data-semantic-collapsed="(102 (101 98 99) 100)" data-semantic-owns="98 99 100"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="98" data-semantic-parent="102"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="100" data-semantic-parent="102"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="99" data-semantic-parent="102"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="115" data-semantic-parent="117" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="110" data-semantic-children="103,109" data-semantic-parent="117" data-semantic-owns="103 109"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="103" data-semantic-parent="110"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="109" data-semantic-children="106,108" data-semantic-content="107" data-semantic-parent="110" data-semantic-owns="106 107 108"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="106" data-semantic-children="104,105" data-semantic-parent="109" data-semantic-owns="104 105"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="104" data-semantic-parent="106"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="105" data-semantic-parent="106"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="107" data-semantic-parent="109" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="108" data-semantic-parent="109"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="116" data-semantic-parent="117" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="113" data-semantic-children="111,112" data-semantic-parent="117" data-semantic-owns="111 112"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="111" data-semantic-parent="113"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="112" data-semantic-parent="113"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="114" data-semantic-parent="121" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="186" data-semantic-children="184" data-semantic-parent="187" data-semantic-owns="184"><mjx-mtd><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="184" data-semantic-children="183,179" data-semantic-content="179" data-semantic-parent="186" data-semantic-owns="183 179"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="183" data-semantic-children="132,182" data-semantic-content="133" data-semantic-parent="184" data-semantic-owns="132 133 182"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="132" data-semantic-children="127,131" data-semantic-parent="183" data-semantic-owns="127 131"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="127" data-semantic-children="124,125" data-semantic-content="126" data-semantic-parent="132" data-semantic-owns="124 126 125"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="124" data-semantic-parent="127"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="126" data-semantic-parent="127" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="125" data-semantic-parent="127"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="131" data-semantic-children="128,129" data-semantic-content="130" data-semantic-parent="132" data-semantic-owns="128 130 129"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="128" data-semantic-parent="131"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="130" data-semantic-parent="131" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="129" data-semantic-parent="131"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="133" data-semantic-parent="183" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="182" data-semantic-children="140,181" data-semantic-content="134" data-semantic-parent="183" data-semantic-owns="140 181"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="140" data-semantic-children="134,138,139" data-semantic-parent="182" data-semantic-owns="134 138 139" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="134" data-semantic-parent="140" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="139" data-semantic-parent="140"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.291em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="138" data-semantic-children="135,137" data-semantic-content="136" data-semantic-parent="140" data-semantic-owns="135 136 137"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="135" data-semantic-parent="138"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="136" data-semantic-parent="138" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="137" data-semantic-parent="138"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="181" data-semantic-children="175,178" data-semantic-content="180" data-semantic-parent="182" data-semantic-owns="175 180 178"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="175" data-semantic-children="172" data-semantic-content="173,174" data-semantic-parent="181" data-semantic-owns="173 172 174"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="173" data-semantic-parent="175" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="172" data-semantic-children="169,171" data-semantic-content="154" data-semantic-parent="175" data-semantic-owns="169 154 171"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="169" data-semantic-children="145,153" data-semantic-content="168" data-semantic-parent="172" data-semantic-owns="145 168 153"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="145" data-semantic-children="141,142,143" data-semantic-parent="169" data-semantic-collapsed="(145 (144 141 142) 143)" data-semantic-owns="141 142 143"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="141" data-semantic-parent="145"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="143" data-semantic-parent="145"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="142" data-semantic-parent="145"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="168" data-semantic-parent="169" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="153" data-semantic-children="146,152" data-semantic-parent="169" data-semantic-owns="146 152"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="146" data-semantic-parent="153"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="152" data-semantic-children="149,151" data-semantic-content="150" data-semantic-parent="153" data-semantic-owns="149 150 151"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="149" data-semantic-children="147,148" data-semantic-parent="152" data-semantic-owns="147 148"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="147" data-semantic-parent="149"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="148" data-semantic-parent="149"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="150" data-semantic-parent="152" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="151" data-semantic-parent="152"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="154" data-semantic-parent="172" data-semantic-operator="infixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="171" data-semantic-children="159,167" data-semantic-content="170" data-semantic-parent="172" data-semantic-owns="159 170 167"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="159" data-semantic-children="155,156,157" data-semantic-parent="171" data-semantic-collapsed="(159 (158 155 156) 157)" data-semantic-owns="155 156 157"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="155" data-semantic-parent="159"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="157" data-semantic-parent="159"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="156" data-semantic-parent="159"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="170" data-semantic-parent="171" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="167" data-semantic-children="160,166" data-semantic-parent="171" data-semantic-owns="160 166"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="160" data-semantic-parent="167"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="166" data-semantic-children="163,165" data-semantic-content="164" data-semantic-parent="167" data-semantic-owns="163 164 165"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="163" data-semantic-children="161,162" data-semantic-parent="166" data-semantic-owns="161 162"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="161" data-semantic-parent="163"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="162" data-semantic-parent="163"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="164" data-semantic-parent="166" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="165" data-semantic-parent="166"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="174" data-semantic-parent="175" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="180" data-semantic-parent="181" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="178" data-semantic-children="176,177" data-semantic-parent="181" data-semantic-owns="176 177"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="176" data-semantic-parent="178"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="177" data-semantic-parent="178"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="179" data-semantic-parent="184" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr></mjx-itable></mjx-table></mjx-mtable></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">(Equation 10)</div></div></div><div id="p0820" role="paragraph">For each strain <i>i</i>, which has biomass density <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="x Subscript i" tabindex="0" ctxtmenu_counter="461" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="x Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, the parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Superscript i Baseline comma gamma Subscript upper A Superscript i Baseline comma r Subscript upper I Superscript i Baseline comma gamma Subscript upper I Superscript i Baseline EndSet" tabindex="0" ctxtmenu_counter="462" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 0 (24 (5 (4 1 2) 3) 6 (11 (10 7 8) 9) 12 (17 (16 13 14) 15) 18 (23 (22 19 20) 21)) 25)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="26" data-semantic-children="24" data-semantic-content="0,25" data-semantic-owns="0 24 25" data-semantic-speech="StartSet r Subscript upper A Superscript i Baseline comma gamma Subscript upper A Superscript i Baseline comma r Subscript upper I Superscript i Baseline comma gamma Subscript upper I Superscript i Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="26" data-semantic-operator="fenced"><mjx-c class="mjx-c7B MM-SO">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="24" data-semantic-children="5,6,11,12,17,18,23" data-semantic-content="6,12,18" data-semantic-parent="26" data-semantic-owns="5 6 11 12 17 18 23"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="24" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="24" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="11" data-semantic-children="7,8,9" data-semantic-parent="24" data-semantic-collapsed="(11 (10 7 8) 9)" data-semantic-owns="7 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="11"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11" style="margin-left: 0.108em;"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="24" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="13,14,15" data-semantic-parent="24" data-semantic-collapsed="(17 (16 13 14) 15)" data-semantic-owns="13 14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="17"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="17"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="18" data-semantic-parent="24" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="23" data-semantic-children="19,20,21" data-semantic-parent="24" data-semantic-collapsed="(23 (22 19 20) 21)" data-semantic-owns="19 20 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="23"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="23" style="margin-left: 0.108em;"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="23"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="25" data-semantic-parent="26" data-semantic-operator="fenced"><mjx-c class="mjx-c7D MM-SO">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were measured in monoculture. Note that, as before, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript upper A Baseline equals upper K Subscript upper I Baseline equals" tabindex="0" ctxtmenu_counter="463" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (2 0 1) 3 (6 4 5) 7 8)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="2,6,8" data-semantic-content="3,7" data-semantic-owns="2 3 6 7 8" data-semantic-speech="upper K Subscript upper A Baseline equals upper K Subscript upper I Baseline equals"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="9" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="8" data-semantic-parent="9"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 0.01&nbsp;mM for all strains. This “additive” model sums the independent rate contributions of each strain to the nitrate and nitrite differential equations, in effect assuming that strains only interact via cross-feeding and competition for extracellular nitrate and nitrite. The model <i>does not</i> assume that strains interact through Lotka-Volterra-type (quadratic) terms in the biomass density equations, nor does it assume that the parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet r Subscript upper A Superscript i Baseline comma gamma Subscript upper A Superscript i Baseline comma r Subscript upper I Superscript i Baseline comma gamma Subscript upper I Superscript i Baseline EndSet" tabindex="0" ctxtmenu_counter="464" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 0 (24 (5 (4 1 2) 3) 6 (11 (10 7 8) 9) 12 (17 (16 13 14) 15) 18 (23 (22 19 20) 21)) 25)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set collection" data-semantic-id="26" data-semantic-children="24" data-semantic-content="0,25" data-semantic-owns="0 24 25" data-semantic-speech="StartSet r Subscript upper A Superscript i Baseline comma gamma Subscript upper A Superscript i Baseline comma r Subscript upper I Superscript i Baseline comma gamma Subscript upper I Superscript i Baseline EndSet"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="26" data-semantic-operator="fenced"><mjx-c class="mjx-c7B MM-SO">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="24" data-semantic-children="5,6,11,12,17,18,23" data-semantic-content="6,12,18" data-semantic-parent="26" data-semantic-owns="5 6 11 12 17 18 23"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="24" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="24" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="11" data-semantic-children="7,8,9" data-semantic-parent="24" data-semantic-collapsed="(11 (10 7 8) 9)" data-semantic-owns="7 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="11"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11" style="margin-left: 0.108em;"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="24" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="13,14,15" data-semantic-parent="24" data-semantic-collapsed="(17 (16 13 14) 15)" data-semantic-owns="13 14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="17"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="17"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="18" data-semantic-parent="24" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="23" data-semantic-children="19,20,21" data-semantic-parent="24" data-semantic-collapsed="(23 (22 19 20) 21)" data-semantic-owns="19 20 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="23"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="23" style="margin-left: 0.108em;"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="23"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="25" data-semantic-parent="26" data-semantic-operator="fenced"><mjx-c class="mjx-c7D MM-SO">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, measured in monoculture, are modulated by the presence of any other strain. Thus <a id="crosref1870" href="#fd10">Equation 10</a> represents a null model for community metabolite dynamics where each strain in a community behaves as it does in monoculture, and as a result provides a prediction requiring no additional free parameters.</div><div id="p0825" role="paragraph">Simple communities were assembled and cultured to test the predictions of <a id="crosref1875" href="#fd10">Equation 10</a>, drawing from a representative 12-strain subset of the 79-strain library: Nar/Nir strains <i>Achromobacter</i> sp. ACM01, <i>Ensifer</i> sp. ENS09, <i>Paracoccus denitrificans</i>, <i>Pseudomonas</i> sp. PDM21; Nar strains <i>Acidovorax</i> sp. ACV02, <i>Paracoccus</i> sp. PAR01, <i>Pseudomonas</i> sp. PDM12, <i>Pantoea</i> sp. PNT03; Nir strains <i>Agrobacterium</i> sp. AGB01, <i>Pseudomonas</i> sp. PDM13, <i>Pseudomonas</i> sp. PDM14, and <i>Pseudoxanthomonas</i> sp. PXM03. Nitrate/nitrite dynamics and endpoint optical density were measured for (a) all 66 2-strain combinations, (b) a random set of 81 three-strain combinations, (c) 21 four-strain combinations, (d) 6 five-strain combinations, and (e) controls of each strain in monoculture. 6 of the four-strain combinations were chosen randomly from the set of 12 strains, while the remaining 15 four-strain combinations and all five-strain combinations were exhaustive combinations of the six strains ACM01, ENS09, PAR01, PNT03, AGB01, and PDM13. All communities/controls were cultured in SDM in each of two media conditions, 2&nbsp;mM nitrate and 2&nbsp;mM nitrite, with 2–3 experimental replicates per condition. Initial biomass densities were OD600<sub>0</sub>&nbsp;= 0.01 for each strain in a given community.</div><div id="p0830" role="paragraph">First endpoint biomass densities were compared with the values predicted by the additive model, <a id="crosref1880" href="#fd10">Equation 10</a>. Since the optical density measurement cannot discern between strains in a mixed population, a model prediction for total endpoint biomass density was computed by summing the endpoint biomass densities of each strain. Endpoint biomass densities were well-predicted by <a id="crosref1885" href="#fd10">Equation 10</a> (<a id="crosref1890" href="#figs4">Figure&nbsp;S4</a>A). This gives one indication that the additive consumer-resource model in <a id="crosref1895" href="#fd10">Equation 10</a> accurately predicts community behavior using only information from monocultures.</div><div id="p0835" role="paragraph">The ability of the additive model to predict relative abundances for a subset of pair cultures was evaluated. All pair combinations and monocultures of six taxonomically and phenotypically representative strains (Nar/Nir strains <i>Achromobacter</i> sp. ACM01 and <i>Ensifer</i> sp. ENS09; Nar strains <i>Paracoccus</i> sp. PAR01 and <i>Pantoea</i> sp. PNT03; Nir strains <i>Agrobacterium</i> sp. AGB01 and <i>Pseudomonas</i> sp. PDM13) were cultured and periodically sampled. All cultures were initiated with 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 3 Superscript minus" tabindex="0" ctxtmenu_counter="465" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 3 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, with the exception of monocultures of Nir strains AGB01 and PDM13 and the pair culture of these two strains, which were initiated with 2&nbsp;mM <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper N normal upper O 2 Superscript minus" tabindex="0" ctxtmenu_counter="466" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 6 (5 (4 1 2) 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="0,5" data-semantic-content="6" data-semantic-owns="0 6 5" data-semantic-speech="normal upper N normal upper O 2 Superscript minus"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="7" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c4F">O</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>. Then the relative abundance of 16S rRNA amplicons in the pair cultures was measured using CASEU (<span class="dropBlock reference-citations"><a id="body-ref-sref12-6" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-6" aria-controls="sref12-6">Cermak et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref12-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cermak, N. ∙ Datta, M.S. ∙ Conwill, A.</div><div><strong>Rapid, inexpensive measurement of synthetic bacterial community composition by Sanger sequencing of amplicon mixtures</strong></div><div><em>iScience.</em> 2020; <strong>23</strong>:100915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85080040522" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32114382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2020.100915&amp;pmid=32114382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In order to enable a comparison with the predictions of the model, <a id="crosref1905" href="#fd10">Equation 10</a>, which makes predictions in terms of biomass density (in units of OD), conversion constants were measured to transform 16S amplicon relative abundances to OD600 relative abundances. Doing so is necessary because (a) strains generally differ in 16S rRNA gene copy number per cell (ranging from 4–7 copies per cell for these strains), and (b) strains will have generally different conversion factors from OD600 to cells/mL, arising for instance due to differences in cell size. Therefore, the ratios of 16S amplicon abundances will be related to the ratio of biomass densities by the equation:<div id="fd11" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartFraction a Subscript i Baseline Over a Subscript j Baseline EndFraction equals StartFraction b Subscript i Baseline c Subscript i Baseline x Subscript i Baseline Over b Subscript j Baseline c Subscript j Baseline x Subscript j Baseline EndFraction equals upper K Subscript i comma j Baseline StartFraction x Subscript i Baseline Over x Subscript j Baseline EndFraction comma" tabindex="0" ctxtmenu_counter="467" style="font-size: 130%;"><mjx-math data-semantic-structure="(51 (50 (6 (2 0 1) (5 3 4)) 7 (32 (19 (10 8 9) 17 (13 11 12) 18 (16 14 15)) (31 (22 20 21) 29 (25 23 24) 30 (28 26 27))) 33 (49 (39 34 (38 35 36 37)) 48 (46 (42 40 41) (45 43 44)))) 47)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="51" data-semantic-children="50,47" data-semantic-content="47" data-semantic-owns="50 47" data-semantic-speech="StartFraction a Subscript i Baseline Over a Subscript j Baseline EndFraction equals StartFraction b Subscript i Baseline c Subscript i Baseline x Subscript i Baseline Over b Subscript j Baseline c Subscript j Baseline x Subscript j Baseline EndFraction equals upper K Subscript i comma j Baseline StartFraction x Subscript i Baseline Over x Subscript j Baseline EndFraction comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="50" data-semantic-children="6,32,49" data-semantic-content="7,33" data-semantic-parent="51" data-semantic-owns="6 7 32 33 49"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="6" data-semantic-children="2,5" data-semantic-parent="50" data-semantic-owns="2 5"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="6" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="6" data-semantic-owns="3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="50" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mfrac space="4" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="32" data-semantic-children="19,31" data-semantic-parent="50" data-semantic-owns="19 31"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="19" data-semantic-children="10,13,16" data-semantic-content="17,18" data-semantic-parent="32" data-semantic-owns="10 17 13 18 16"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="19" data-semantic-owns="8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="19" data-semantic-owns="11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="19" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="31" data-semantic-children="22,25,28" data-semantic-content="29,30" data-semantic-parent="32" data-semantic-owns="22 29 25 30 28"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="22" data-semantic-children="20,21" data-semantic-parent="31" data-semantic-owns="20 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="22"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="22"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="29" data-semantic-parent="31" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="25" data-semantic-children="23,24" data-semantic-parent="31" data-semantic-owns="23 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="25"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="25"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="31" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="26,27" data-semantic-parent="31" data-semantic-owns="26 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="33" data-semantic-parent="50" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="49" data-semantic-children="39,46" data-semantic-content="48" data-semantic-parent="50" data-semantic-owns="39 48 46"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="39" data-semantic-children="34,38" data-semantic-parent="49" data-semantic-owns="34 38"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="39"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="38" data-semantic-children="35,36,37" data-semantic-content="36" data-semantic-parent="39" data-semantic-owns="35 36 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="38"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="36" data-semantic-parent="38" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="38"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="48" data-semantic-parent="49" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="46" data-semantic-children="42,45" data-semantic-parent="49" data-semantic-owns="42 45"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="42" data-semantic-children="40,41" data-semantic-parent="46" data-semantic-owns="40 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="42"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="42"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="45" data-semantic-children="43,44" data-semantic-parent="46" data-semantic-owns="43 44"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="45"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="45"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="47" data-semantic-parent="51" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">(Equation 11)</div></div>where for strain <i>i</i>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="a Subscript i" tabindex="0" ctxtmenu_counter="468" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="a Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the 16S amplicon relative abundance, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="x Subscript i" tabindex="0" ctxtmenu_counter="469" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="x Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> the biomass density relative abundance, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b Subscript i" tabindex="0" ctxtmenu_counter="470" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="b Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c Subscript i" tabindex="0" ctxtmenu_counter="471" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="c Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are respectively the 16S rRNA gene copy number (16S rRNA copies/cell) and cell density to OD600 conversion factor (cells/mL/OD). This means that the ratio of 16S amplicon abundances for strains <i>i</i> and <i>j</i> will be related to the ratio of OD600 abundances by the conversion factor <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript i comma j" tabindex="0" ctxtmenu_counter="472" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="upper K Subscript i comma j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="4" data-semantic-children="1,2,3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. Therefore <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript i comma j" tabindex="0" ctxtmenu_counter="473" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="upper K Subscript i comma j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="4" data-semantic-children="1,2,3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> was directly measured for each pair of strains by growing monocultures of each strain in denitrifying conditions, harvesting cells after 64 h, measuring OD600, combining monocultures in equal-volume (450&nbsp;μL) mixtures, and then processing, sequencing, and analyzing mixtures. Relative amplicon abundances of these mixtures along with OD600 measurements of the constituent monocultures were used to solve <a id="crosref1910" href="#fd11">Equation 11</a> for the parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper K Subscript i comma j" tabindex="0" ctxtmenu_counter="474" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="upper K Subscript i comma j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="4" data-semantic-children="1,2,3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> for each pair. Then these conversion factors were used to transform the 16S amplicon relative abundances for pair cultures to OD600 relative abundances. These measurements were compared with the OD600 relative abundance predictions of <a id="crosref1915" href="#fd10">Equation 10</a> (<a id="crosref1920" href="#figs4">Figure&nbsp;S4</a>B). In general, the model predictions for strain relative abundances closely matched the observed values, indicating that the model accurately predicts community compositions in the majority of cases using only information from monocultures. There were a small number of pair cultures with modest differences between predicted and observed relative abundances, mainly involving combinations of the Nar strains PAR01 and PNT03 and the Nir strains AGB01 and PDM13. These lower quality predictions may reflect interactions between these phenotypes not captured in the model (<a id="crosref1925" href="#fd10">Equation 10</a>).</div><div id="p0840" role="paragraph">Finally the ability of additive model to predict community nitrate and nitrite dynamics from individual strain consumer-resource parameters was evaluated. To do this, a normalized root-mean-square error (NRMSE) was computed for a given <i>N</i>-strain community as:<div id="fd12" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="NRMSE Subscript 1 comma ellipsis comma upper N Baseline equals StartFraction RMSE Subscript 1 comma ellipsis comma upper N Baseline Over left parenthesis StartFraction 1 Over upper N EndFraction sigma summation Underscript i equals 1 Overscript upper N Endscripts RMSE Subscript i Superscript 2 Baseline right parenthesis Superscript 1 divided by 2 Baseline EndFraction comma" tabindex="0" ctxtmenu_counter="475" style="font-size: 130%;"><mjx-math data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="46" data-semantic-children="45,44" data-semantic-content="44" data-semantic-owns="45 44" data-semantic-structure="(46 (45 (7 0 (6 1 2 3 4 5)) 8 (43 (16 9 (15 10 11 12 13 14)) (42 (37 35 (34 (19 17 18) 33 (32 (26 20 (24 21 22 23) 25) (31 (30 27 28) 29))) 36) (41 38 39 40)))) 44)" data-semantic-speech="NRMSE Subscript 1 comma ellipsis comma upper N Baseline equals StartFraction RMSE Subscript 1 comma ellipsis comma upper N Baseline Over left parenthesis StartFraction 1 Over upper N EndFraction sigma summation Underscript i equals 1 Overscript upper N Endscripts RMSE Subscript i Superscript 2 Baseline right parenthesis Superscript 1 divided by 2 Baseline EndFraction comma" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="45" data-semantic-children="7,43" data-semantic-content="8" data-semantic-parent="46" data-semantic-owns="7 8 43" inline-breaks="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="7" data-semantic-children="0,6" data-semantic-parent="45" data-semantic-owns="0 6"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c4E" noic="true" style="padding-top: 0.705em;">N</mjx-c><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.705em;">R</mjx-c><mjx-c class="mjx-c4D" noic="true" style="padding-top: 0.705em;">M</mjx-c><mjx-c class="mjx-c53" noic="true" style="padding-top: 0.705em;">S</mjx-c><mjx-c class="mjx-c45" style="padding-top: 0.705em;">E</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="1,2,3,4,5" data-semantic-content="2,4" data-semantic-parent="7" data-semantic-owns="1 2 3 4 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="8" data-semantic-parent="45" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mfrac space="4" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="43" data-semantic-children="16,42" data-semantic-parent="45" data-semantic-owns="16 42"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="16" data-semantic-children="9,15" data-semantic-parent="43" data-semantic-owns="9 15"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-parent="16"><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.705em;">R</mjx-c><mjx-c class="mjx-c4D" noic="true" style="padding-top: 0.705em;">M</mjx-c><mjx-c class="mjx-c53" noic="true" style="padding-top: 0.705em;">S</mjx-c><mjx-c class="mjx-c45" style="padding-top: 0.705em;">E</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="15" data-semantic-children="10,11,12,13,14" data-semantic-content="11,13" data-semantic-parent="16" data-semantic-owns="10 11 12 13 14"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="15"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="15" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="12" data-semantic-parent="15"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="13" data-semantic-parent="15" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msup size="s" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="42" data-semantic-children="37,41" data-semantic-parent="43" data-semantic-owns="37 41"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="37" data-semantic-children="34" data-semantic-content="35,36" data-semantic-parent="42" data-semantic-owns="35 34 36"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="35" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="34" data-semantic-children="19,32" data-semantic-content="33" data-semantic-parent="37" data-semantic-owns="19 33 32"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="19" data-semantic-children="17,18" data-semantic-parent="34" data-semantic-owns="17 18"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="19"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="32" data-semantic-children="26,31" data-semantic-content="20" data-semantic-parent="34" data-semantic-owns="26 31"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="26" data-semantic-children="20,24,25" data-semantic-parent="32" data-semantic-owns="20 24 25"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="20" data-semantic-parent="26" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.291em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="24" data-semantic-children="21,23" data-semantic-content="22" data-semantic-parent="26" data-semantic-owns="21 22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="24"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="22" data-semantic-parent="24" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="24"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="unknown" data-semantic-id="31" data-semantic-children="27,28,29" data-semantic-parent="32" data-semantic-collapsed="(31 (30 27 28) 29)" data-semantic-owns="27 28 29"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="27" data-semantic-parent="31"><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.705em;">R</mjx-c><mjx-c class="mjx-c4D" noic="true" style="padding-top: 0.705em;">M</mjx-c><mjx-c class="mjx-c53" noic="true" style="padding-top: 0.705em;">S</mjx-c><mjx-c class="mjx-c45" style="padding-top: 0.705em;">E</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.212em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="31"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="36" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.699em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="41" data-semantic-children="38,40" data-semantic-content="39" data-semantic-parent="42" data-semantic-owns="38 39 40"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="41"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="39" data-semantic-parent="41" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="41"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="44" data-semantic-parent="46" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-math></mjx-container></div></div><div class="label">(Equation 12)</div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper R normal upper M normal upper S normal upper E Subscript 1 comma ellipsis comma upper N" tabindex="0" ctxtmenu_counter="476" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 0 11 1 12 2 13 (10 3 (9 4 5 6 7 8)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="0,1,2,10" data-semantic-content="11,12,13" data-semantic-owns="0 11 1 12 2 13 10" data-semantic-speech="normal upper R normal upper M normal upper S normal upper E Subscript 1 comma ellipsis comma upper N"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c52">R</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="14"><mjx-c class="mjx-c4D">M</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="14"><mjx-c class="mjx-c53">S</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="3,9" data-semantic-parent="14" data-semantic-owns="3 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="10"><mjx-c class="mjx-c45">E</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="9" data-semantic-children="4,5,6,7,8" data-semantic-content="5,7" data-semantic-parent="10" data-semantic-owns="4 5 6 7 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the root-mean-square error between measurements and predictions for the community, and RMSE<sub><i>i</i></sub> is the error for each constituent strain in monoculture, with errors averaged over experimental replicates. The monoculture RMSEs of each strain were used in this normalization in order to correct for variations in monoculture fit quality. NRMSE in the range 0–2 indicates errors in 2-strain communities that are within twofold of fits associated with their constituent monocultures, while values of NRMSE much greater than this indicate low quality predictions. It should be noted that, because the denominator of <a id="crosref1930" href="#fd12">Equation 12</a> depends on the RMSE of the constituent monocultures in a community, a potential pathology is that NRMSEs can be modestly inflated when the RMSEs for constituent monocultures are all small (indicating a strong correspondence between the data and the model). However, this pathology is expected to be rare because it requires all monoculture RMSEs to be small.</div><div id="p0845" role="paragraph">The NRMSEs for all pairs of 12 strains cultured in separate nitrate and nitrite medium conditions are shown in <a id="crosref1935" href="#figs3">Figures S3</a>A and S3B. For most pair communities, typically low values of NRMSE indicated that <a id="crosref1940" href="#fd10">Equation 10</a> made high quality predictions in most cases. The apparent exception to this was Nar&nbsp;+ Nir communities (comprising any combination of Nar strain and Nir strain) initialized with nitrate (<a id="crosref1945" href="#figs6">Figure&nbsp;S6</a>), for which NRMSE values appeared much higher than for any other group. Permutation tests were then performed on the mean NRMSE values within phenotypic groups to determine whether the Nar&nbsp;+ Nir group had significantly higher NRMSE than other groups (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 times 10 Superscript 5" tabindex="0" ctxtmenu_counter="477" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="1 times 10 Superscript 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> permutations). Since 10 hypothesis tests were performed in this manner, a Bonferroni-corrected threshold of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p equals 0 period 05 divided by 10 equals 0 period 005" tabindex="0" ctxtmenu_counter="478" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 (6 (5 0 1 2) 3 4) 7 (14 (13 8 9 10) 11 12))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="15" data-semantic-children="6,14" data-semantic-content="7" data-semantic-owns="6 7 14" data-semantic-speech="p equals 0 period 05 divided by 10 equals 0 period 005"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="5,3,4" data-semantic-content="3" data-semantic-parent="15" data-semantic-owns="5 3 4"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="6" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c35" style="padding-top: 0.666em;">5</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="7" data-semantic-parent="15" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="14" data-semantic-children="13,11,12" data-semantic-content="11" data-semantic-parent="15" data-semantic-owns="13 11 12"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="13" data-semantic-children="8,10" data-semantic-content="9" data-semantic-parent="14" data-semantic-owns="8 9 10"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="13"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="13" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="11" data-semantic-parent="14" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c35" style="padding-top: 0.666em;">5</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> was used for testing at the 5% significance level. These tests indicated that only the Nar&nbsp;+ Nir group in nitrate medium conditions had a significantly large mean NRMSE value <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis p less than 1 times 10 Superscript negative 5 Baseline right parenthesis" tabindex="0" ctxtmenu_counter="479" style="font-size: 130%;"><mjx-math data-semantic-structure="(13 0 (11 1 2 (10 3 4 (9 5 (8 6 7)))) 12)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="13" data-semantic-children="11" data-semantic-content="0,12" data-semantic-owns="0 11 12" data-semantic-speech="left parenthesis p less than 1 times 10 Superscript negative 5 Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="11" data-semantic-children="1,10" data-semantic-content="2" data-semantic-parent="13" data-semantic-owns="1 2 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="11"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="2" data-semantic-parent="11" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-children="3,9" data-semantic-content="4" data-semantic-parent="11" data-semantic-owns="3 4 9"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="10"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="10" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="9" data-semantic-children="5,8" data-semantic-parent="10" data-semantic-owns="5 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="9"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-children="7" data-semantic-content="6" data-semantic-parent="9" data-semantic-owns="6 7"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="12" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0850" role="paragraph">The NRMSEs for 108 3–5-strain communities cultured in separate nitrate and nitrite medium conditions are shown in <a id="crosref1950" href="#figs3">Figure&nbsp;S3</a>E. As with pair cultures, many 3–5-strain communities communities have low values of NRMSE indicating that <a id="crosref1955" href="#fd10">Equation 10</a> made high quality predictions of nitrate and nitrite dynamics. Also, most apparent failures of the model predictions occurred when a community contains both a Nar strain and a Nir strain.</div><div id="p0855" role="paragraph">Up to this point, to test the ability of the consumer-resource model to predict community metabolic dynamics, directly measured values of the consumer-resource parameters were used to make community predictions. Using regressions instead to make predictions of single-strain phenotypes from gene presence/absence would be expected to add some degree of error to community metabolite dynamics predictions. To quantify this additional error, regressions on each of the SDM consumer-resource parameters (<a id="crosref1960" href="#fig4">Figures 4</a>C–4F) were used to predict these parameters for the each of the 12 strains, and then these parameters were used to predict community dynamics in the pair cultures and 3–5-strain communities. The NRMSEs evaluating the accuracy of these predictions relative to observed community metabolite dynamics are shown in <a id="crosref1965" href="#figs5">Figure&nbsp;S5</a>. For pair cultures (<a id="crosref1970" href="#figs5">Figures S5</a>A and S5B), NRMSEs changed little relative to using the directly measured parameters to make predictions; the median difference between NRMSEs using regression predicted parameters versus measured parameters for pair cultures in nitrate was -0.1, and for pair cultures in nitrite the median difference was approximately zero. NRMSEs increased modestly for 3–5-strain communities (<a id="crosref1975" href="#figs5">Figure&nbsp;S5</a>C); the median difference was 0.5 for communities in nitrate, and 1.4 for communities in nitrite.</div></section><section id="sec-5-5-9"><h4>Correcting for Nar&nbsp;+ Nir interactions</h4><div id="p0860" role="paragraph">The additive <i>N</i>-strain consumer-resource model, <a id="crosref1980" href="#fd10">Equation 10</a>, generally failed to predict the dynamics of communities initialized with nitrate that included both a Nar and a Nir strain (see plots of all Nar&nbsp;+ Nir pairs in <a id="crosref1985" href="#figs6">Figure&nbsp;S6</a>). In every instance, the nitrate reduction rate of the Nar strain appears to be diminished relative to the rate in monoculture, and in some instances the nitrite reduction rate for the Nir strain appears to be increased relative to monoculture. Since Nar and Nir strains utilize different electron acceptors and therefore do not plausibly compete for any resource under these experimental conditions, it was concluded that interactions aside from resource competition must be taking place.</div><div id="p0865" role="paragraph">It was therefore investigated whether the predictions of the additive model in <a id="crosref1990" href="#fd10">Equation 10</a> could be improved by correcting for interactions between Nar and Nir strains. To do this, the nitrate and nitrite reduction parameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Superscript i" tabindex="0" ctxtmenu_counter="480" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="r Subscript upper A Superscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Superscript j" tabindex="0" ctxtmenu_counter="481" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="r Subscript upper I Superscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> of the Nar strain <i>i</i> and the Nir strain <i>j</i>, respectively, were refit using the measured metabolite dynamics data from the pair cultures of these strains. Changes to the reduction rate parameters rather than the yields (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper A Superscript i" tabindex="0" ctxtmenu_counter="482" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="gamma Subscript upper A Superscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.108em;"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper I Superscript j" tabindex="0" ctxtmenu_counter="483" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="gamma Subscript upper I Superscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.309em; margin-left: -0.025em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.108em;"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>) were investigated because the additive model accurately predicted endpoint optical densities in all pair communities (<a id="crosref1995" href="#figs4">Figure&nbsp;S4</a>A), and therefore there was no evidence that interactions between Nar and Nir strains involved a change in yields. However, given the hypothesis that Nir strains inhibit Nar strains via excretion of cytotoxic nitric oxide (see <a id="crosref2000" href="#sec-3">discussion</a>), it is plausible that interactions may involve a change in yields that are not detectable given the precision of the optical density measurement.</div><div id="p0870" role="paragraph">Nitrate and nitrite reduction rate parameters for each Nar&nbsp;+ Nir pair culture were refit in the following way:<div id="olist0035" role="list"><div id="o0085" role="listitem"><div class="label">1.</div><div class="content"><div id="p0875" role="paragraph">Refit <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Superscript i" tabindex="0" ctxtmenu_counter="484" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="r Subscript upper A Superscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> for the Nar strain <i>i</i>, holding all other parameters fixed. Record the NRMSE for the resulting prediction.</div></div></div><div id="o0090" role="listitem"><div class="label">2.</div><div class="content"><div id="p0880" role="paragraph">Refit both <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Superscript i" tabindex="0" ctxtmenu_counter="485" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="r Subscript upper A Superscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> for the Nar strain <i>i</i> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Superscript j" tabindex="0" ctxtmenu_counter="486" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="r Subscript upper I Superscript j"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> for the Nir strain <i>j</i>, holding all other parameters fixed. Record the NRMSE for the resulting prediction.</div></div></div><div id="o0095" role="listitem"><div class="label">3.</div><div class="content"><div id="p0885" role="paragraph">If the NRMSE obtained in step 2 was more than 10% smaller than the NRMSE in step 1, then accept the refit parameters obtained in step 2, denoting these values as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper A Superscript i comma j" tabindex="0" ctxtmenu_counter="487" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) 3) (7 4 5 6))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,3,7" data-semantic-collapsed="(9 (8 2 3) 7)" data-semantic-owns="2 3 7" data-semantic-speech="r overtilde Subscript upper A Superscript i comma j"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.333em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="4,5,6" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="9"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper I Superscript j comma i" tabindex="0" ctxtmenu_counter="488" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) 3) (7 4 5 6))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,3,7" data-semantic-collapsed="(9 (8 2 3) 7)" data-semantic-owns="2 3 7" data-semantic-speech="r overtilde Subscript upper I Superscript j comma i"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="4,5,6" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="9"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>. Otherwise, accepted the refit parameter obtained in step 1 as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper A Superscript i comma j" tabindex="0" ctxtmenu_counter="489" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (8 (2 0 1) 3) (7 4 5 6))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,3,7" data-semantic-collapsed="(9 (8 2 3) 7)" data-semantic-owns="2 3 7" data-semantic-speech="r overtilde Subscript upper A Superscript i comma j"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.333em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="4,5,6" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="9"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, and let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r overtilde Subscript upper I Superscript j comma i Baseline equals r Subscript upper I Superscript j" tabindex="0" ctxtmenu_counter="490" style="font-size: 130%;"><mjx-math data-semantic-structure="(16 (9 (8 (2 0 1) 3) (7 4 5 6)) 10 (15 (14 11 12) 13))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="16" data-semantic-children="9,15" data-semantic-content="10" data-semantic-owns="9 10 15" data-semantic-speech="r overtilde Subscript upper I Superscript j comma i Baseline equals r Subscript upper I Superscript j"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,3,7" data-semantic-parent="16" data-semantic-collapsed="(9 (8 2 3) 7)" data-semantic-owns="2 3 7"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="4,5,6" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="9"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="10" data-semantic-parent="16" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="11,12,13" data-semantic-parent="16" data-semantic-collapsed="(15 (14 11 12) 13)" data-semantic-owns="11 12 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="15"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>.</div></div></div></div></div><div id="p0890" role="paragraph">Refitting was performed as described above for <a id="crosref2005" href="#fd3">Equation 3</a>. In step 3, improvements in fit quality obtained by refitting only <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="491" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> versus refitting both <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="492" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="493" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> were compared in order to identify cases where increases in nitrite reduction rate were necessary to improve fits of <a id="crosref2010" href="#fd10">Equation 10</a> to the data.</div><div id="p0895" role="paragraph">The results of this refitting procedure are shown in <a id="crosref2015" href="#figs7">Figure&nbsp;S7</a>. These results demonstrate that, in all cases, the nitrate reduction rate of the Nar strain was slowed <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis r overtilde Subscript upper A Baseline less than r Subscript upper A Baseline right parenthesis" tabindex="0" ctxtmenu_counter="494" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 0 (10 (5 (3 1 2) 4) 6 (9 7 8)) 11)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="0,11" data-semantic-owns="0 10 11" data-semantic-speech="left parenthesis r overtilde Subscript upper A Baseline less than r Subscript upper A Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="10" data-semantic-children="5,9" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="5 6 9"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="10" data-semantic-owns="3 4"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="2" data-semantic-parent="3" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="6" data-semantic-parent="10" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="10" data-semantic-owns="7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> in Nar&nbsp;+ Nir pair culture. Additionally, in several cases the nitrite reduction rate of the Nir strain was increased <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis r overtilde Subscript upper I Baseline greater than r Subscript upper I Baseline right parenthesis" tabindex="0" ctxtmenu_counter="495" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 0 (10 (5 (3 1 2) 4) 6 (9 7 8)) 11)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="0,11" data-semantic-owns="0 10 11" data-semantic-speech="left parenthesis r overtilde Subscript upper I Baseline greater than r Subscript upper I Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="10" data-semantic-children="5,9" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="5 6 9"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="10" data-semantic-owns="3 4"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="2" data-semantic-parent="3" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="6" data-semantic-parent="10" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="10" data-semantic-owns="7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> in Nar&nbsp;+ Nir pair culture. Each Nar strain is affected in essentially the same way by every Nir strain (e.g., the nitrate reduction rate of Nar strain PAR01 diminished by approximately 50% in every pair culture with a Nir strain). This observation is consistent with the proposal that nitric oxide toxicity impacted nitrate reduction by the Nar-strain in a manner that is specific to each Nar strain (see <a id="crosref2020" href="#sec-3">discussion</a>).</div><div id="p0900" role="paragraph">These refit parameter values were then used in <a id="crosref2025" href="#fd10">Equation 10</a> to generate new predictions for 3–5-strain communities containing at least one Nar&nbsp;+ Nir pair. To do so, the following rules were employed for replacing <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="496" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="497" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> in <a id="crosref2030" href="#fd10">Equation 10</a>, which depend on the number of Nar and Nir strains in the community:<div id="olist0040" role="list"><div id="o0100" role="listitem"><div class="label">1.</div><div class="content"><div id="p0905" role="paragraph">One Nar strain <i>i</i> and one Nir strain <i>j</i>: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Superscript i Baseline equals r overtilde Subscript upper A Superscript i comma j" tabindex="0" ctxtmenu_counter="498" style="font-size: 130%;"><mjx-math data-semantic-structure="(16 (4 (3 0 1) 2) 5 (15 (14 (8 6 7) 9) (13 10 11 12)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="16" data-semantic-children="4,15" data-semantic-content="5" data-semantic-owns="4 5 15" data-semantic-speech="r Subscript upper A Superscript i Baseline equals r overtilde Subscript upper A Superscript i comma j"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="16" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="16" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="8,9,13" data-semantic-parent="16" data-semantic-collapsed="(15 (14 8 9) 13)" data-semantic-owns="8 9 13"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="15" data-semantic-owns="6 7"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="7" data-semantic-parent="8" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.333em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="13" data-semantic-children="10,11,12" data-semantic-content="11" data-semantic-parent="15" data-semantic-owns="10 11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="15"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Superscript j Baseline equals r overtilde Subscript upper I Superscript j comma i" tabindex="0" ctxtmenu_counter="499" style="font-size: 130%;"><mjx-math data-semantic-structure="(16 (4 (3 0 1) 2) 5 (15 (14 (8 6 7) 9) (13 10 11 12)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="16" data-semantic-children="4,15" data-semantic-content="5" data-semantic-owns="4 5 15" data-semantic-speech="r Subscript upper I Superscript j Baseline equals r overtilde Subscript upper I Superscript j comma i"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="16" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="16" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="8,9,13" data-semantic-parent="16" data-semantic-collapsed="(15 (14 8 9) 13)" data-semantic-owns="8 9 13"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="15" data-semantic-owns="6 7"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="7" data-semantic-parent="8" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="13" data-semantic-children="10,11,12" data-semantic-content="11" data-semantic-parent="15" data-semantic-owns="10 11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>.</div></div></div><div id="o0105" role="listitem"><div class="label">2.</div><div class="content"><div id="p0910" role="paragraph">Two Nar strains <i>i</i> and <i>j</i> and one Nir strain <i>k</i>: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Superscript i Baseline equals r overtilde Subscript upper A Superscript i comma k" tabindex="0" ctxtmenu_counter="500" style="font-size: 130%;"><mjx-math data-semantic-structure="(16 (4 (3 0 1) 2) 5 (15 (14 (8 6 7) 9) (13 10 11 12)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="16" data-semantic-children="4,15" data-semantic-content="5" data-semantic-owns="4 5 15" data-semantic-speech="r Subscript upper A Superscript i Baseline equals r overtilde Subscript upper A Superscript i comma k"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="16" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="16" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="8,9,13" data-semantic-parent="16" data-semantic-collapsed="(15 (14 8 9) 13)" data-semantic-owns="8 9 13"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="15" data-semantic-owns="6 7"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="7" data-semantic-parent="8" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.333em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="13" data-semantic-children="10,11,12" data-semantic-content="11" data-semantic-parent="15" data-semantic-owns="10 11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="15"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Superscript j Baseline equals r overtilde Subscript upper A Superscript j comma k" tabindex="0" ctxtmenu_counter="501" style="font-size: 130%;"><mjx-math data-semantic-structure="(16 (4 (3 0 1) 2) 5 (15 (14 (8 6 7) 9) (13 10 11 12)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="16" data-semantic-children="4,15" data-semantic-content="5" data-semantic-owns="4 5 15" data-semantic-speech="r Subscript upper A Superscript j Baseline equals r overtilde Subscript upper A Superscript j comma k"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="16" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.333em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="16" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="8,9,13" data-semantic-parent="16" data-semantic-collapsed="(15 (14 8 9) 13)" data-semantic-owns="8 9 13"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="15" data-semantic-owns="6 7"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="7" data-semantic-parent="8" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.333em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="13" data-semantic-children="10,11,12" data-semantic-content="11" data-semantic-parent="15" data-semantic-owns="10 11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="15"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Superscript k Baseline equals left parenthesis r overtilde Subscript upper I Superscript k comma i Baseline plus r overtilde Subscript upper I Superscript k comma j Baseline right parenthesis divided by 2" tabindex="0" ctxtmenu_counter="502" style="font-size: 130%;"><mjx-math data-semantic-structure="(34 (4 (3 0 1) 2) 5 (33 (30 6 (28 (16 (15 (9 7 8) 10) (14 11 12 13)) 17 (27 (26 (20 18 19) 21) (25 22 23 24))) 29) 31 32))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="34" data-semantic-children="4,33" data-semantic-content="5" data-semantic-owns="4 5 33" data-semantic-speech="r Subscript upper I Superscript k Baseline equals left parenthesis r overtilde Subscript upper I Superscript k comma i Baseline plus r overtilde Subscript upper I Superscript k comma j Baseline right parenthesis divided by 2"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="34" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="34" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="33" data-semantic-children="30,32" data-semantic-content="31" data-semantic-parent="34" data-semantic-owns="30 31 32"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="30" data-semantic-children="28" data-semantic-content="6,29" data-semantic-parent="33" data-semantic-owns="6 28 29"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="6" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="28" data-semantic-children="16,27" data-semantic-content="17" data-semantic-parent="30" data-semantic-owns="16 17 27"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="9,10,14" data-semantic-parent="28" data-semantic-collapsed="(16 (15 9 10) 14)" data-semantic-owns="9 10 14"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="16" data-semantic-owns="7 8"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="8" data-semantic-parent="9" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="14" data-semantic-children="11,12,13" data-semantic-content="12" data-semantic-parent="16" data-semantic-owns="11 12 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="14" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="16"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="17" data-semantic-parent="28" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msubsup space="3" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="27" data-semantic-children="20,21,25" data-semantic-parent="28" data-semantic-collapsed="(27 (26 20 21) 25)" data-semantic-owns="20 21 25"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="27" data-semantic-owns="18 19"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="19" data-semantic-parent="20" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="25" data-semantic-children="22,23,24" data-semantic-content="23" data-semantic-parent="27" data-semantic-owns="22 23 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="25"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="23" data-semantic-parent="25" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="25"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="27"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="29" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="31" data-semantic-parent="33" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="33"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container>.</div></div></div><div id="o0110" role="listitem"><div class="label">3.</div><div class="content"><div id="p0915" role="paragraph">One Nar strain <i>i</i> and two Nir strains <i>j</i> and <i>k</i>: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Superscript i Baseline equals left parenthesis r overtilde Subscript upper A Superscript i comma j Baseline plus r overtilde Subscript upper A Superscript i comma k Baseline right parenthesis divided by 2" tabindex="0" ctxtmenu_counter="503" style="font-size: 130%;"><mjx-math data-semantic-structure="(34 (4 (3 0 1) 2) 5 (33 (30 6 (28 (16 (15 (9 7 8) 10) (14 11 12 13)) 17 (27 (26 (20 18 19) 21) (25 22 23 24))) 29) 31 32))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="34" data-semantic-children="4,33" data-semantic-content="5" data-semantic-owns="4 5 33" data-semantic-speech="r Subscript upper A Superscript i Baseline equals left parenthesis r overtilde Subscript upper A Superscript i comma j Baseline plus r overtilde Subscript upper A Superscript i comma k Baseline right parenthesis divided by 2"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="34" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="34" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="33" data-semantic-children="30,32" data-semantic-content="31" data-semantic-parent="34" data-semantic-owns="30 31 32"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="30" data-semantic-children="28" data-semantic-content="6,29" data-semantic-parent="33" data-semantic-owns="6 28 29"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="6" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="28" data-semantic-children="16,27" data-semantic-content="17" data-semantic-parent="30" data-semantic-owns="16 17 27"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="9,10,14" data-semantic-parent="28" data-semantic-collapsed="(16 (15 9 10) 14)" data-semantic-owns="9 10 14"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="16" data-semantic-owns="7 8"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="8" data-semantic-parent="9" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.333em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="14" data-semantic-children="11,12,13" data-semantic-content="12" data-semantic-parent="16" data-semantic-owns="11 12 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="12" data-semantic-parent="14" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="16"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="17" data-semantic-parent="28" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msubsup space="3" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="27" data-semantic-children="20,21,25" data-semantic-parent="28" data-semantic-collapsed="(27 (26 20 21) 25)" data-semantic-owns="20 21 25"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="27" data-semantic-owns="18 19"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="19" data-semantic-parent="20" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.333em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="25" data-semantic-children="22,23,24" data-semantic-content="23" data-semantic-parent="27" data-semantic-owns="22 23 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="25"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="23" data-semantic-parent="25" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="25"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="27"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="29" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="31" data-semantic-parent="33" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="33"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Superscript j Baseline equals r overtilde Subscript upper I Superscript j comma i" tabindex="0" ctxtmenu_counter="504" style="font-size: 130%;"><mjx-math data-semantic-structure="(16 (4 (3 0 1) 2) 5 (15 (14 (8 6 7) 9) (13 10 11 12)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="16" data-semantic-children="4,15" data-semantic-content="5" data-semantic-owns="4 5 15" data-semantic-speech="r Subscript upper I Superscript j Baseline equals r overtilde Subscript upper I Superscript j comma i"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="16" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="16" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="8,9,13" data-semantic-parent="16" data-semantic-collapsed="(15 (14 8 9) 13)" data-semantic-owns="8 9 13"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="15" data-semantic-owns="6 7"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="7" data-semantic-parent="8" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="13" data-semantic-children="10,11,12" data-semantic-content="11" data-semantic-parent="15" data-semantic-owns="10 11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Superscript k Baseline equals r overtilde Subscript upper I Superscript k comma i" tabindex="0" ctxtmenu_counter="505" style="font-size: 130%;"><mjx-math data-semantic-structure="(16 (4 (3 0 1) 2) 5 (15 (14 (8 6 7) 9) (13 10 11 12)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="16" data-semantic-children="4,15" data-semantic-content="5" data-semantic-owns="4 5 15" data-semantic-speech="r Subscript upper I Superscript k Baseline equals r overtilde Subscript upper I Superscript k comma i"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="16" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.308em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="16" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="8,9,13" data-semantic-parent="16" data-semantic-collapsed="(15 (14 8 9) 13)" data-semantic-owns="8 9 13"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="15" data-semantic-owns="6 7"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="7" data-semantic-parent="8" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="13" data-semantic-children="10,11,12" data-semantic-content="11" data-semantic-parent="15" data-semantic-owns="10 11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="15"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>.</div></div></div><div id="o0115" role="listitem"><div class="label">4.</div><div class="content"><div id="p0920" role="paragraph">General case with <i>N</i> Nar strains in the set <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="script upper S Subscript upper N a r" tabindex="0" ctxtmenu_counter="506" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 (6 1 4 2 5 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="0,6" data-semantic-owns="0 6" data-semantic-speech="script upper S Subscript upper N a r"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="script" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D4AE MM-S">𝒮</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.229em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="7" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <i>M</i> Nir strains in the set <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="script upper S Subscript upper N i r" tabindex="0" ctxtmenu_counter="507" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 (6 1 4 2 5 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="0,6" data-semantic-owns="0 6" data-semantic-speech="script upper S Subscript upper N i r"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="script" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D4AE MM-S">𝒮</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.229em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="7" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>:</div></div></div></div></div><div id="p0925" role="paragraph"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A Superscript i Baseline equals 1 divided by upper M sigma summation Underscript j element of script upper S Subscript upper N i r Baseline Endscripts r overtilde Subscript upper A Superscript i comma j" tabindex="0" ctxtmenu_counter="508" style="font-size: 130%;"><mjx-math data-semantic-structure="(36 (4 (3 0 1) 2) 5 (35 (9 6 7 8) 34 (33 (22 10 (21 11 12 (20 13 (19 14 17 15 18 16)))) (32 (31 (25 23 24) 26) (30 27 28 29)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="36" data-semantic-children="4,35" data-semantic-content="5" data-semantic-owns="4 5 35" data-semantic-speech="r Subscript upper A Superscript i Baseline equals 1 divided by upper M sigma summation Underscript j element of script upper S Subscript upper N i r Baseline Endscripts r overtilde Subscript upper A Superscript i comma j"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="36" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.331em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="36" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="35" data-semantic-children="9,33" data-semantic-content="34" data-semantic-parent="36" data-semantic-owns="9 34 33"><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="35" data-semantic-owns="6 7 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="34" data-semantic-parent="35" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="33" data-semantic-children="22,32" data-semantic-content="10" data-semantic-parent="35" data-semantic-owns="22 32"><mjx-msub data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="22" data-semantic-children="10,21" data-semantic-parent="33" data-semantic-owns="10 21"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="10" data-semantic-parent="22" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="21" data-semantic-children="11,20" data-semantic-content="12" data-semantic-parent="22" data-semantic-owns="11 12 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="21"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="12" data-semantic-parent="21" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="20" data-semantic-children="13,19" data-semantic-parent="21" data-semantic-owns="13 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="script" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="20"><mjx-c class="mjx-c1D4AE MM-S">𝒮</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.229em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="19" data-semantic-children="14,15,16" data-semantic-content="17,18" data-semantic-parent="20" data-semantic-owns="14 17 15 18 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="19"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="19"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="19"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="32" data-semantic-children="25,26,30" data-semantic-parent="33" data-semantic-collapsed="(32 (31 25 26) 30)" data-semantic-owns="25 26 30"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="25" data-semantic-children="23,24" data-semantic-parent="32" data-semantic-owns="23 24"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="24" data-semantic-parent="25" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="25"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.333em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="30" data-semantic-children="27,28,29" data-semantic-content="28" data-semantic-parent="32" data-semantic-owns="27 28 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="30"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="28" data-semantic-parent="30" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="30"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="32"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> for all <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i element of script upper S Subscript upper N a r" tabindex="0" ctxtmenu_counter="509" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (9 2 (8 3 6 4 7 5)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="10" data-semantic-children="0,9" data-semantic-content="1" data-semantic-owns="0 1 9" data-semantic-speech="i element of script upper S Subscript upper N a r"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,8" data-semantic-parent="10" data-semantic-owns="2 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="script" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c1D4AE MM-S">𝒮</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.229em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="9" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I Superscript j Baseline equals 1 divided by upper N sigma summation Underscript i element of script upper S Subscript upper N a r Baseline Endscripts r overtilde Subscript upper I Superscript j comma i" tabindex="0" ctxtmenu_counter="510" style="font-size: 130%;"><mjx-math data-semantic-structure="(36 (4 (3 0 1) 2) 5 (35 (9 6 7 8) 34 (33 (22 10 (21 11 12 (20 13 (19 14 17 15 18 16)))) (32 (31 (25 23 24) 26) (30 27 28 29)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="36" data-semantic-children="4,35" data-semantic-content="5" data-semantic-owns="4 5 35" data-semantic-speech="r Subscript upper I Superscript j Baseline equals 1 divided by upper N sigma summation Underscript i element of script upper S Subscript upper N a r Baseline Endscripts r overtilde Subscript upper I Superscript j comma i"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="36" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="36" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="35" data-semantic-children="9,33" data-semantic-content="34" data-semantic-parent="36" data-semantic-owns="9 34 33"><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="35" data-semantic-owns="6 7 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="34" data-semantic-parent="35" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="33" data-semantic-children="22,32" data-semantic-content="10" data-semantic-parent="35" data-semantic-owns="22 32"><mjx-msub data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="22" data-semantic-children="10,21" data-semantic-parent="33" data-semantic-owns="10 21"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="10" data-semantic-parent="22" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="21" data-semantic-children="11,20" data-semantic-content="12" data-semantic-parent="22" data-semantic-owns="11 12 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="21"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="12" data-semantic-parent="21" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="20" data-semantic-children="13,19" data-semantic-parent="21" data-semantic-owns="13 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="script" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="20"><mjx-c class="mjx-c1D4AE MM-S">𝒮</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.229em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="19" data-semantic-children="14,15,16" data-semantic-content="17,18" data-semantic-parent="20" data-semantic-owns="14 17 15 18 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="19"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="19"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="19"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub><mjx-msubsup space="2" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="32" data-semantic-children="25,26,30" data-semantic-parent="33" data-semantic-collapsed="(32 (31 25 26) 30)" data-semantic-owns="25 26 30"><mjx-mover data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="25" data-semantic-children="23,24" data-semantic-parent="32" data-semantic-owns="23 24"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.51em; margin-bottom: -0.554em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:tilde" data-semantic-id="24" data-semantic-parent="25" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c303">̃</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="25"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.309em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="30" data-semantic-children="27,28,29" data-semantic-content="28" data-semantic-parent="32" data-semantic-owns="27 28 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="30"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="28" data-semantic-parent="30" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="30"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="32"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> for all <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="j element of script upper S Subscript upper N i r" tabindex="0" ctxtmenu_counter="511" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (9 2 (8 3 6 4 7 5)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="10" data-semantic-children="0,9" data-semantic-content="1" data-semantic-owns="0 1 9" data-semantic-speech="j element of script upper S Subscript upper N i r"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,8" data-semantic-parent="10" data-semantic-owns="2 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="script" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c1D4AE MM-S">𝒮</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.229em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="9" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0930" role="paragraph">The results of using refit Nar and Nir parameters to predict 3–5-strain cultures are shown in <a id="crosref2035" href="#fig7">Figure&nbsp;7</a>B (yellow points). Correcting predictions using refit <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper A" tabindex="0" ctxtmenu_counter="512" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript upper I" tabindex="0" ctxtmenu_counter="513" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript upper I"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> parameters substantially improved prediction quality, reducing median NRMSE values from 1.68 to 0.92. These results did not appreciably change if a min or max function is used instead of a mean.</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0935" role="paragraph">We thank Laura Troyer for assistance with isolating bacterial strains and Elizabeth Ujhelyi and Annette Wells for assistance with sequencing. We acknowledge Cameron Pittelkow for access to corn and soybean fields in Savoy, Illinois, and the laboratory of Julie Zilles for providing the bacterial strain <i>Paracoccus denitrificans</i> ATCC 19367. We also thank Rama Ranganathan, David Pincus, James Sethna, William Metcalf, Jun Song, Sara Clifton, and members of the Kuehn laboratory and Mani group for helpful discussions. This work was supported by the National Science Foundation Division of Emerging Frontiers EF 2025293 (S.K.) and EF 2025521 (M.M.), the National Science Foundation Physics Frontiers Center Program PHY 0822613 and PHY 1430124 (S.K.), James S. McDonnell Foundation Postdoctoral Fellowship Award 220020499 (K.G.), and the Simons Foundation Investigator Award 597491 (M.M.).</div><section id="sec-6"><h3>Author contributions</h3><div id="p0940" role="paragraph">K.G.: conceptualization, experimental design, data collection, formal analysis, coding, writing–original draft, and writing–review &amp; editing. D.P.: data collection. M.M.: conceptualization, formal analysis, writing – review &amp; editing, supervision, and funding acquisition. S.K.: conceptualization, experimental design, formal analysis, writing–original draft, writing–review &amp; editing, supervision, and funding acquisition.</div></section><section id="sec-7"><h3>Declaration of interests</h3><div id="p0945" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (1)</h2><div id="p0955" role="paragraph"><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.12.036/attachment/70e71cdc-e768-4d16-b4f4-d7d7cceab83b/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (115.66 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Document S1. Tables S1–S7</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">Raw sequencing data and draft genome assemblies have been deposited to NCBI Sequence Read Archive and NCBI GenBank, respectively. The BioProject ID is listed in the key resources table. Metabolite dynamics data, phenotype data, and RAST annotations of draft assemblies used to infer gene presence and absence have been deposited on Open Science Framework and are publicly available as of the date of publication. The DOI is listed in the key resources table.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">All original code has been deposited at Open Science Framework and is publicly available as of the date of publication. The DOI is listed in the key resources table.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Abraham, Z.H.L. ∙ Lowe, D.J. ∙ Smith, B.E.</div><div><strong>Purification and characterization of the dissimilatory nitrite reductase from <i>Alcaligenes xylosoxidans subsp</i>. xylosoxidans (N.C.I.M.B. 11015): evidence for the presence of both type 1 and type 2 copper centres</strong></div><div><em>Biochem. J.</em> 1993; <strong>295</strong>:587-593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1042/bj2950587" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0027437592" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8240262/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1042%2Fbj2950587&amp;pmid=8240262" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Anantharaman, K. ∙ Brown, C.T. ∙ Hug, L.A. ...</div><div><strong>Thousands of microbial genomes shed light on interconnected biogeochemical processes in an aquifer system</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:13219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms13219" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84992418589" target="_blank" aria-label="Open Scopus in new tab">Scopus (451)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27774985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms13219&amp;pmid=27774985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Bankevich, A. ∙ Nurk, S. ∙ Antipov, D. ...</div><div><strong>SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</strong></div><div><em>J.&nbsp;Comput. Biol.</em> 2012; <strong>19</strong>:455-477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/cmb.2012.0021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84860771820" target="_blank" aria-label="Open Scopus in new tab">Scopus (10972)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22506599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Fcmb.2012.0021&amp;pmid=22506599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Beccari, M. ∙ Passino, R. ∙ Ramadori, R. ...</div><div><strong>Kinetics of dissimilatory nitrate and nitrite reduction in suspended growth culture</strong></div><div><em>J.&nbsp;Water Pollut. Control Fed.</em> 1983; <strong>55</strong>:58-64</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BeccariR.PassinoR.RamadoriV.TandoiKinetics+of+dissimilatory+nitrate+and+nitrite+reduction+in+suspended+growth+cultureJ.%C2%A0Water+Pollut.+Control+Fed.5519835864" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Bell, L.C. ∙ Richardson, D.J. ∙ Ferguson, S.J.</div><div><strong>Periplasmic and membrane-bound respiratory nitrate reductases in <i>Thiosphaera pantotropha</i>: the periplasmic enzyme catalyzes the first step in aerobic denitrification</strong></div><div><em>FEBS Lett.</em> 1990; <strong>265</strong>:85-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0014-5793(90)80889-Q" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0025333241" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2365057/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0014-5793%2890%2980889-Q&amp;pmid=2365057" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Berman, G.J. ∙ Choi, D.M. ∙ Bialek, W. ...</div><div><strong>Mapping the stereotyped behaviour of freely moving fruit flies</strong></div><div><em>J.&nbsp;R. Soc. Interface.</em> 2014; <strong>11</strong>:20140672</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rsif.2014.0672" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84906994989" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25142523/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frsif.2014.0672&amp;pmid=25142523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Besson, S. ∙ Carneiro, C. ∙ Moura, J.J. ...</div><div><strong>A cytochrome cd1-type nitrite reductase isolated from the marine denitrifier <i>Pseudomonas nautica</i> 617: purification and characterization</strong></div><div><em>Anaerobe.</em> 1995; <strong>1</strong>:219-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/anae.1995.1021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0001191264" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16887530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fanae.1995.1021&amp;pmid=16887530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Bolger, A.M. ∙ Lohse, M. ∙ Usadel, B.</div><div><strong>Trimmomatic: a flexible trimmer for Illumina sequence data</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:2114-2120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84905049901" target="_blank" aria-label="Open Scopus in new tab">Scopus (21176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu170&amp;pmid=24695404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Braun, C. ∙ Zumft, W.G.</div><div><strong>Marker exchange of the structural genes for nitric oxide reductase blocks the denitrification pathway of <i>Pseudomonas stutzeri</i> at nitric oxide</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1991; <strong>266</strong>:22785-22788</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2FS0021-9258%2818%2954420-3&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2FS0021-9258%2818%2954420-3&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1744072/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2954420-3&amp;pmid=1744072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Brettin, T. ∙ Davis, J.J. ∙ Disz, T. ...</div><div><strong>RASTtk: a modular and extensible implementation of the RAST algorithm for building custom annotation pipelines and annotating batches of genomes</strong></div><div><em>Sci. Rep.</em> 2015; <strong>5</strong>:8365</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep08365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84938841197" target="_blank" aria-label="Open Scopus in new tab">Scopus (988)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25666585/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep08365&amp;pmid=25666585" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Canfield, D.E. ∙ Glazer, A.N. ∙ Falkowski, P.G.</div><div><strong>The evolution and future of earth's nitrogen cycle</strong></div><div><em>Science.</em> 2010; <strong>330</strong>:192-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1186120" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-77957744758" target="_blank" aria-label="Open Scopus in new tab">Scopus (1276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20929768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1186120&amp;pmid=20929768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#figs4" title="View in article">12.</a></div><div>Cermak, N. ∙ Datta, M.S. ∙ Conwill, A.</div><div><strong>Rapid, inexpensive measurement of synthetic bacterial community composition by Sanger sequencing of amplicon mixtures</strong></div><div><em>iScience.</em> 2020; <strong>23</strong>:100915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.100915&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85080040522" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32114382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2020.100915&amp;pmid=32114382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Claus, G. ∙ Kutzner, H.</div><div><strong>Physiology and kinetics of autotrophic denitrification by <i>Thiobacillus denitrificans</i></strong></div><div><em>Appl. Microbiol. Biotechnol.</em> 1985; <strong>22</strong>:283-288</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.ClausH.KutznerPhysiology+and+kinetics+of+autotrophic+denitrification+by+Thiobacillus+denitrificansAppl.+Microbiol.+Biotechnol.221985283288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Connon, S.A. ∙ Giovannoni, S.J.</div><div><strong>High-throughput methods for culturing microorganisms in very-low-nutrient media yield diverse new marine isolates</strong></div><div><em>Appl. Environ. Microbiol.</em> 2002; <strong>68</strong>:3878-3885</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/AEM.68.8.3878-3885.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0036324056" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12147485/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FAEM.68.8.3878-3885.2002&amp;pmid=12147485" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Cordero, O.X. ∙ Ventouras, L.A. ∙ DeLong, E.F. ...</div><div><strong>Public good dynamics drive evolution of iron acquisition strategies in natural bacterioplankton populations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:20059-20064</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1213344109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84870578736" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23169633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1213344109&amp;pmid=23169633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Coultate, T.P. ∙ Sundaram, T.K.</div><div><strong>Energetics of <i>Bacillus stearothermophilus</i> growth: molar growth yield and temperature effects on growth efficiency</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1975; <strong>121</strong>:55-64</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.121.1.55-64.1975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1116992/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.121.1.55-64.1975&amp;pmid=1116992" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Daniels, B.C. ∙ Nemenman, I.</div><div><strong>Automated adaptive inference of phenomenological dynamical models</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:8133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms9133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84939832859" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26293508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms9133&amp;pmid=26293508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Datta, M.S. ∙ Sliwerska, E. ∙ Gore, J. ...</div><div><strong>Microbial interactions lead to rapid micro-scale successions on model marine particles</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:11965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms11965" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84975511062" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27311813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms11965&amp;pmid=27311813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Denariaz, G. ∙ Payne, W.J. ∙ LeGall, J.</div><div><strong>The denitrifying nitrite reductase of <i>Bacillus halodenitrificans</i></strong></div><div><em>Biochim. Biophys. Acta Bioenerg.</em> 1991; <strong>1056</strong>:225-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0005-2728(05)80053-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0026089232" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0005-2728%2805%2980053-2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Dinçer, A.R. ∙ Kargı, F.</div><div><strong>Kinetics of sequential nitrification and denitrification processes</strong></div><div><em>Enzyme Microb. Technol.</em> 2000; <strong>27</strong>:37-42</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0141-0229(00)00145-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0342264575" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10862900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0141-0229%2800%2900145-9&amp;pmid=10862900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Erbilgin, O. ∙ Bowen, B.P. ∙ Kosina, S.M. ...</div><div><strong>Dynamic substrate preferences predict metabolic properties of a simple microbial consortium</strong></div><div><em>BMC Bioinformatics.</em> 2017; <strong>18</strong>:57</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-017-1478-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85011072524" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28114881/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-017-1478-2&amp;pmid=28114881" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Falkowski, P.G. ∙ Fenchel, T. ∙ Delong, E.F.</div><div><strong>The microbial engines that drive earth's biogeochemical cycles</strong></div><div><em>Science.</em> 2008; <strong>320</strong>:1034-1039</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1153213" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-44449137093" target="_blank" aria-label="Open Scopus in new tab">Scopus (1460)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18497287/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1153213&amp;pmid=18497287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Ferguson, S.J. ∙ Richardson, D.J.</div><div><strong>The enzymes and bioenergetics of bacterial nitrate, nitrite, nitric oxide and nitrous oxide respiration</strong></div><div><div>Zannoni, D. (Editor)</div></div><div><strong>Respiration in Archaea and Bacteria: Diversity of Prokaryotic Respiratory Systems</strong></div><div>Springer, 2004; 169-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4020-3163-2_8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4020-3163-2_8" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Fierer, N. ∙ Lauber, C.L. ∙ Ramirez, K.S. ...</div><div><strong>Comparative metagenomic, phylogenetic and physiological analyses of soil microbial communities across nitrogen gradients</strong></div><div><em>ISME J.</em> 2012; <strong>6</strong>:1007-1017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2011.159" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84859911168" target="_blank" aria-label="Open Scopus in new tab">Scopus (909)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22134642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2011.159&amp;pmid=22134642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Forchhammer, J. ∙ Lindahl, L.</div><div><strong>Growth rate of polypeptide chains as a function of the cell growth rate in a mutant of <i>Escherichia coli</i> 15</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1971; <strong>55</strong>:563-568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(71)90337-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0015222649" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4927947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2871%2990337-8&amp;pmid=4927947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Fraebel, D.T. ∙ Gowda, K. ∙ Mani, M. ...</div><div><strong>Evolution of generalists by phenotypic plasticity</strong></div><div><em>iScience.</em> 2020; <strong>23</strong>:101678</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.101678&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.isci.2020.101678&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85094604886" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33163936/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.isci.2020.101678&amp;pmid=33163936" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">27.</a></div><div>Galtier, N. ∙ Lobry, J.R.</div><div><strong>Relationships between genomic G+C content, RNA secondary structures, and optimal growth temperature in prokaryotes</strong></div><div><em>J.&nbsp;Mol. Evol.</em> 1997; <strong>44</strong>:632-636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/PL00006186" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0030900384" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9169555/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FPL00006186&amp;pmid=9169555" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">28.</a></div><div>Glockner, A.B. ∙ Jüngst, A. ∙ Zumft, W.G.</div><div><strong>Copper-containing nitrite reductase from <i>Pseudomonas aureofaciens</i> is functional in a mutationally cytochrome cd1-free background (NirS-) of <i>Pseudomonas stutzeri</i></strong></div><div><em>Arch. Microbiol.</em> 1993; <strong>160</strong>:18-26</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8352648/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=8352648" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Goddard, A.D. ∙ Bali, S. ∙ Mavridou, D.A. ...</div><div><strong>The <i>Paracoccus denitrificans</i> NarK-like nitrate and nitrite transporters-probing nitrate uptake and nitrate/nitrite exchange mechanisms</strong></div><div><em>Mol. Microbiol.</em> 2017; <strong>103</strong>:117-133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13546" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84999767719" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27696579/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13546&amp;pmid=27696579" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Goddard, A.D. ∙ Moir, J.W.B. ∙ Richardson, D.J. ...</div><div><strong>Interdependence of two NarK domains in a fused nitrate/nitrite transporter</strong></div><div><em>Mol. Microbiol.</em> 2008; <strong>70</strong>:667-681</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2008.06436.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-53849131672" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18823285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2008.06436.x&amp;pmid=18823285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">31.</a></div><div>Goldford, J.E. ∙ Lu, N. ∙ Bajić, D. ...</div><div><strong>Emergent simplicity in microbial community assembly</strong></div><div><em>Science.</em> 2018; <strong>361</strong>:469-474</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat1168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85051138998" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30072533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat1168&amp;pmid=30072533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Gordon, E.H.J. ∙ Sjögren, T. ∙ Löfqvist, M. ...</div><div><strong>Structure and kinetic properties of <i>Paracoccus pantotrophus</i> cytochrome cd1 nitrite reductase with the d1 heme active site ligand tyrosine 25 replaced by serine</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2003; <strong>278</strong>:11773-11781</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1074%2Fjbc.M211886200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1074%2Fjbc.M211886200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0038146890" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12556530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M211886200&amp;pmid=12556530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Graf, D.R.H. ∙ Jones, C.M. ∙ Hallin, S.</div><div><strong>Intergenomic comparisons highlight modularity of the denitrification pathway and underpin the importance of community structure for N2O emissions</strong></div><div><em>PLoS ONE.</em> 2014; <strong>9</strong>:e114118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0114118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84914680332" target="_blank" aria-label="Open Scopus in new tab">Scopus (234)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25436772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0114118&amp;pmid=25436772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Gurevich, A. ∙ Saveliev, V. ∙ Vyahhi, N. ...</div><div><strong>QUAST: quality assessment tool for genome assemblies</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:1072-1075</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt086" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84876266928" target="_blank" aria-label="Open Scopus in new tab">Scopus (2977)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt086&amp;pmid=23422339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">35.</a></div><div>Harcombe, W.R. ∙ Riehl, W.J. ∙ Dukovski, I. ...</div><div><strong>Metabolic resource allocation in individual microbes determines ecosystem interactions and spatial dynamics</strong></div><div><em>Cell Rep.</em> 2014; <strong>7</strong>:1104-1115</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.celrep.2014.03.070&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.celrep.2014.03.070&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84901273253" target="_blank" aria-label="Open Scopus in new tab">Scopus (280)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24794435/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2014.03.070&amp;pmid=24794435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Hassen, A. ∙ Saidi, N. ∙ Cherif, M. ...</div><div><strong>Effects of heavy metals on <i>Pseudomonas aeruginosa</i> and <i>Bacillus thuringiensis</i></strong></div><div><em>Bioresour. Technol.</em> 1998; <strong>65</strong>:73-82</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0960-8524(98)00011-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0032126284" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0960-8524%2898%2900011-X" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Hastie, T. ∙ Tibshirani, R. ∙ Friedman, J.</div><div><strong>The Elements of Statistical Learning</strong></div><div>Springer, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.HastieR.TibshiraniJ.FriedmanThe+Elements+of+Statistical+Learning2nd+Edition2008Springer" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">38.</a></div><div>Hastie, T. ∙ Tibshirani, R.J. ∙ Wainwright, R.</div><div><strong>Statistical learning with sparsity</strong></div><div>CRC Press, 2016</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.HastieR.J.TibshiraniR.WainwrightStatistical+learning+with+sparsityMonographs+on+Statistics+and+Applied+Probability1432016CRC+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39-1" title="View in article">39.</a></div><div>Haywood-Farmer, E. ∙ Otto, S.P.</div><div><strong>The evolution of genomic base composition in bacteria</strong></div><div><em>Evolution.</em> 2003; <strong>57</strong>:1783-1792</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.0014-3820.2003.tb00586.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0042287719" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14503620/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.0014-3820.2003.tb00586.x&amp;pmid=14503620" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Hempfling, W.P. ∙ Mainzer, S.E.</div><div><strong>Effects of varying the carbon source limiting growth on yield and maintenance characteristics of <i>Escherichia coli</i> in continuous culture</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1975; <strong>123</strong>:1076-1087</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.123.3.1076-1087.1975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/169226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.123.3.1076-1087.1975&amp;pmid=169226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">41.</a></div><div>Hessen, D.O. ∙ Jeyasingh, P.D. ∙ Neiman, M. ...</div><div><strong>Genome streamlining and the elemental costs of growth</strong></div><div><em>Trends Ecol. Evol.</em> 2010; <strong>25</strong>:75-80</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.tree.2009.08.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.tree.2009.08.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-74049085522" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19796842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tree.2009.08.004&amp;pmid=19796842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Heylen, K. ∙ Gevers, D. ∙ Vanparys, B. ...</div><div><strong>The incidence of nirS and nirK and their genetic heterogeneity in cultivated denitrifiers</strong></div><div><em>Environ. Microbiol.</em> 2006; <strong>8</strong>:2012-2021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1462-2920.2006.01081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33749411590" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17014499/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1462-2920.2006.01081.x&amp;pmid=17014499" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Heylen, K. ∙ Vanparys, B. ∙ Wittebolle, L. ...</div><div><strong>Cultivation of denitrifying bacteria: optimization of isolation conditions and diversity study</strong></div><div><em>Appl. Environ. Microbiol.</em> 2006; <strong>72</strong>:2637-2643</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/AEM.72.4.2637-2643.2006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33646102691" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16597968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FAEM.72.4.2637-2643.2006&amp;pmid=16597968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>Human Microbiome Project Consortium</div><div><strong>Structure, function and diversity of the healthy human microbiome</strong></div><div><em>Nature.</em> 2012; <strong>486</strong>:207-214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11234" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84862276328" target="_blank" aria-label="Open Scopus in new tab">Scopus (6048)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22699609/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11234&amp;pmid=22699609" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Ikeda, E. ∙ Andou, S. ∙ Iwama, U. ...</div><div><strong>Physiological roles of two dissimilatory nitrate reductases in the deep-sea denitrifier <i>Pseudomonas</i> sp. strain MT-1</strong></div><div><em>Biosci. Biotechnol. Biochem.</em> 2009; <strong>73</strong>:896-900</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1271/bbb.80833" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-67149086712" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19352028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1271%2Fbbb.80833&amp;pmid=19352028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">46.</a></div><div>Irrazábal, T. ∙ Belcheva, A. ∙ Girardin, S.E. ...</div><div><strong>The multifaceted role of the intestinal microbiota in colon cancer</strong></div><div><em>Mol. Cell.</em> 2014; <strong>54</strong>:309-320</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.molcel.2014.03.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.molcel.2014.03.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84899156759" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24766895/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2014.03.039&amp;pmid=24766895" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Iwasaki, H. ∙ Matsubara, T.</div><div><strong>A nitrite reductase from <i>Achromobacter cyclastes</i></strong></div><div><em>J.&nbsp;Biochem.</em> 1972; <strong>71</strong>:645-652</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5042456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=5042456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">48.</a></div><div>Jones, C.M. ∙ Hallin, S.</div><div><strong>Ecological and evolutionary factors underlying global and local assembly of denitrifier communities</strong></div><div><em>ISME J.</em> 2010; <strong>4</strong>:633-641</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2009.152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-77951498193" target="_blank" aria-label="Open Scopus in new tab">Scopus (174)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20090785/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2009.152&amp;pmid=20090785" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">49.</a></div><div>Jones, C.M. ∙ Stres, B. ∙ Rosenquist, M. ...</div><div><strong>Phylogenetic analysis of nitrite, nitric oxide, and nitrous oxide respiratory enzymes reveal a complex evolutionary history for denitrification</strong></div><div><em>Mol. Biol. Evol.</em> 2008; <strong>25</strong>:1955-1966</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msn146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-49749112308" target="_blank" aria-label="Open Scopus in new tab">Scopus (335)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18614527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsn146&amp;pmid=18614527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">50.</a></div><div>Kakutani, T. ∙ Watanabe, H. ∙ Arima, K. ...</div><div><strong>Purification and properties of a copper-containing nitrite reductase from a denitrifying bacterium, <i>Alcaligenes faecalis</i> strain S-6</strong></div><div><em>J.&nbsp;Biochem.</em> 1981; <strong>89</strong>:453-461</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oxfordjournals.jbchem.a133220" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0019457194" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7240122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foxfordjournals.jbchem.a133220&amp;pmid=7240122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">51.</a></div><div>Keck, F. ∙ Rimet, F. ∙ Bouchez, A. ...</div><div><strong>phylosignal: an R package to measure, test, and explore the phylogenetic signal</strong></div><div><em>Ecol. Evol.</em> 2016; <strong>6</strong>:2774-2780</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ece3.2051" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84962504104" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27066252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fece3.2051&amp;pmid=27066252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">52.</a></div><div>Kehe, J. ∙ Kulesa, A. ∙ Ortiz, A. ...</div><div><strong>Massively parallel screening of synthetic microbial communities</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:12804-12809</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1900102116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85068117260" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31186361/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1900102116&amp;pmid=31186361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">53.</a></div><div>Klitgord, N. ∙ Segrè, D.</div><div><strong>Environments that induce synthetic microbial ecosystems</strong></div><div><em>PLoS Comput. Biol.</em> 2010; <strong>6</strong>:e1001002</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1001002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-78649677586" target="_blank" aria-label="Open Scopus in new tab">Scopus (201)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21124952/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1001002&amp;pmid=21124952" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">54.</a></div><div>Kornaros, M. ∙ Zafiri, C. ∙ Lyberatos, G.</div><div><strong>Kinetics of denitrification by <i>Pseudomonas denitrificans</i> under growth conditions limited by carbon and/or nitrate or nitrite</strong></div><div><em>Water Environ. Res.</em> 1996; <strong>68</strong>:934-945</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2175/106143096X127947" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0030200160" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2175%2F106143096X127947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Kukimoto, M. ∙ Nishiyama, M. ∙ Murphy, M.E. ...</div><div><strong>X-ray structure and site-directed mutagenesis of a nitrite reductase from <i>Alcaligenes faecalis</i> S-6: roles of two copper atoms in nitrite reduction</strong></div><div><em>Biochemistry.</em> 1994; <strong>33</strong>:5246-5252</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00183a030" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8172899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00183a030&amp;pmid=8172899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">56.</a></div><div>Kumar, S. ∙ Stecher, G. ∙ Li, M. ...</div><div><strong>MEGA X: molecular evolutionary genetics analysis across computing platforms</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:1547-1549</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msy096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85050137432" target="_blank" aria-label="Open Scopus in new tab">Scopus (11440)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29722887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsy096&amp;pmid=29722887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">57.</a></div><div>Lee, K.H. ∙ Park, J.H. ∙ Kim, T.Y. ...</div><div><strong>Systems metabolic engineering of <i>Escherichia coli</i> for L-threonine production</strong></div><div><em>Mol. Syst. Biol.</em> 2007; <strong>3</strong>:149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/msb4100196" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-36849002434" target="_blank" aria-label="Open Scopus in new tab">Scopus (345)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18059444/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmsb4100196&amp;pmid=18059444" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">58.</a></div><div>Lensi, R. ∙ Clays-Josserand, A. ∙ Jocteur Monrozier, L.</div><div><strong>Denitrifiers and denitrifying activity in size fractions of a mollisol under permanent pasture and continuous cultivation</strong></div><div><em>Soil Biol. Biochem.</em> 1995; <strong>27</strong>:61-69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0038-0717(94)00132-K" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0028976873" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0038-0717%2894%2900132-K" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">59.</a></div><div>Li, J. ∙ Mau, R.L. ∙ Dijkstra, P. ...</div><div><strong>Predictive genomic traits for bacterial growth in culture versus actual growth in soil</strong></div><div><em>ISME J.</em> 2019; <strong>13</strong>:2162-2172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41396-019-0422-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85065340904" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31053828/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41396-019-0422-z&amp;pmid=31053828" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">60.</a></div><div>Lilja, E.E. ∙ Johnson, D.R.</div><div><strong>Segregating metabolic processes into different microbial cells accelerates the consumption of inhibitory substrates</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:1568-1578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2015.243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84954437600" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2015.243&amp;pmid=26771930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">61.</a></div><div>Liu, B. ∙ Mao, Y. ∙ Bergaust, L. ...</div><div><strong>Strains in the genus Thauera exhibit remarkably different denitrification regulatory phenotypes</strong></div><div><em>Environ. Microbiol.</em> 2013; <strong>15</strong>:2816-2828</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23663391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=23663391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">62.</a></div><div>Liu, M.Y. ∙ Liu, M.C. ∙ Payne, W.J. ...</div><div><strong>Properties and electron transfer specificity of copper proteins from the denitrifier <i>“Achromobacter cycloclastes”</i></strong></div><div><em>J.&nbsp;Bacteriol.</em> 1986; <strong>166</strong>:604-608</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.166.2.604-608.1986" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3700338/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.166.2.604-608.1986&amp;pmid=3700338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63-1" title="View in article">63.</a></div><div>Louca, S. ∙ Jacques, S.M.S. ∙ Pires, A.P.F. ...</div><div><strong>High taxonomic variability despite stable functional structure across microbial communities</strong></div><div><em>Nat. Ecol. Evol.</em> 2016; <strong>1</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-016-0015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28812567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-016-0015&amp;pmid=28812567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">64.</a></div><div>Louca, S. ∙ Parfrey, L.W. ∙ Doebeli, M.</div><div><strong>Decoupling function and taxonomy in the global ocean microbiome</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:1272-1277</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf4507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84988369671" target="_blank" aria-label="Open Scopus in new tab">Scopus (723)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27634532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf4507&amp;pmid=27634532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">65.</a></div><div>Louca, S. ∙ Polz, M.F. ∙ Mazel, F. ...</div><div><strong>Function and functional redundancy in microbial systems</strong></div><div><em>Nat. Ecol. Evol.</em> 2018; <strong>2</strong>:936-943</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-018-0519-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85045435762" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29662222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-018-0519-1&amp;pmid=29662222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">66.</a></div><div>Lu, H. ∙ Chandran, K. ∙ Stensel, D.</div><div><strong>Microbial ecology of denitrification in biological wastewater treatment</strong></div><div><em>Water Res.</em> 2014; <strong>64</strong>:237-254</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.watres.2014.06.042" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84904891891" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25078442/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.watres.2014.06.042&amp;pmid=25078442" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">67.</a></div><div>Lycus, P. ∙ Lovise Bøthun, K. ∙ Bergaust, L. ...</div><div><strong>Phenotypic and genotypic richness of denitrifiers revealed by a novel isolation strategy</strong></div><div><em>ISME J.</em> 2017; <strong>11</strong>:2219-2232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2017.82" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85029618940" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28696424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2017.82&amp;pmid=28696424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68-1" title="View in article">68.</a></div><div>Lynch, M.</div><div><strong>Streamlining and simplification of microbial genome architecture</strong></div><div><em>Annu. Rev. Microbiol.</em> 2006; <strong>60</strong>:327-349</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.micro.60.080805.142300" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33750344923" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16824010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.micro.60.080805.142300&amp;pmid=16824010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">69.</a></div><div>Martiny, J.B.H. ∙ Jones, S.E. ∙ Lennon, J.T. ...</div><div><strong>Microbiomes in light of traits: a phylogenetic perspective</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:aac9323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aac9323" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84947460926" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26542581/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aac9323&amp;pmid=26542581" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">70.</a></div><div>Masuko, M. ∙ Iwasaki, H. ∙ Sakurai, T. ...</div><div><strong>Characterization of nitrite reductase from a denitrifier, <i>Alcaligenes</i> sp. NCIB 11015. A novel copper protein</strong></div><div><em>J.&nbsp;Biochem.</em> 1984; <strong>96</strong>:447-454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oxfordjournals.jbchem.a134856" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0021478113" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6501252/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foxfordjournals.jbchem.a134856&amp;pmid=6501252" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">71.</a></div><div>Mathupala, S.P. ∙ Kiousis, S. ∙ Szerlip, N.J.</div><div><strong>A lab assembled microcontroller-based sensor module for continuous oxygen measurement in portable hypoxia chambers</strong></div><div><em>PLoS ONE.</em> 2016; <strong>11</strong>:e0148923</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0148923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26862760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0148923&amp;pmid=26862760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">72.</a></div><div>Medlock, G.L. ∙ Carey, M.A. ∙ McDuffie, D.G. ...</div><div><strong>Inferring metabolic mechanisms of interaction within a defined gut microbiota</strong></div><div><em>Cell Syst.</em> 2018; <strong>7</strong>:245-257.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2018.08.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2018.08.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85054064942" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30195437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2018.08.003&amp;pmid=30195437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">73.</a></div><div>Michalski, W.P. ∙ Nicholas, D.</div><div><strong>Molecular characterization of a copper-containing nitrite reductase from <i>Rhodopseudomonas sphaeroides</i> forma sp. <i>denitrificans</i></strong></div><div><em>Biochim. Biophys. Acta Protein Struct. Mol. Enzymol.</em> 1985; <strong>828</strong>:130-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0167-4838(85)90048-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0021989793" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0167-4838%2885%2990048-2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">74.</a></div><div>Mickalide, H. ∙ Kuehn, S.</div><div><strong>Higher-order interaction between species inhibits bacterial invasion of a phototroph-predator microbial community</strong></div><div><em>Cell Syst.</em> 2019; <strong>9</strong>:521-533.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2019.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cels.2019.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85076261763" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31838145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2019.11.004&amp;pmid=31838145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">75.</a></div><div>Miranda, K.M. ∙ Espey, M.G. ∙ Wink, D.A.</div><div><strong>A rapid, simple spectrophotometric method for simultaneous detection of nitrate and nitrite</strong></div><div><em>Nitric Oxide.</em> 2001; <strong>5</strong>:62-71</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/niox.2000.0319" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0035709008" target="_blank" aria-label="Open Scopus in new tab">Scopus (2479)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11178938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fniox.2000.0319&amp;pmid=11178938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">76.</a></div><div>Moir, J.W.B. ∙ Wood, N.J.</div><div><strong>Nitrate and nitrite transport in bacteria</strong></div><div><em>Cell. Mol. Life Sci.</em> 2001; <strong>58</strong>:215-224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/PL00000849" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11289303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FPL00000849&amp;pmid=11289303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">77.</a></div><div>Molina, N. ∙ Nimwegen, E. van.</div><div><strong>Scaling laws in functional genome content across prokaryotic clades and lifestyles</strong></div><div><em>Trends Genet.</em> 2009; <strong>25</strong>:243-247</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.tig.2009.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.tig.2009.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-66449114096" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19457568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tig.2009.04.004&amp;pmid=19457568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">78.</a></div><div>Monod, J.</div><div><strong>The growth of bacterial cultures</strong></div><div><em>Annu. Rev. Microbiol.</em> 1949; <strong>3</strong>:371-394</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.mi.03.100149.002103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.mi.03.100149.002103" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">79.</a></div><div>Mori, M. ∙ Hwa, T. ∙ Martin, O.C. ...</div><div><strong>Constrained allocation flux balance analysis</strong></div><div><em>PLoS Comput. Biol.</em> 2016; <strong>12</strong>:e1004913</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1004913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84978791333" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27355325/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1004913&amp;pmid=27355325" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">80.</a></div><div>Nichols, R.J. ∙ Sen, S. ∙ Choo, Y.J. ...</div><div><strong>Phenotypic landscape of a bacterial cell</strong></div><div><em>Cell.</em> 2011; <strong>144</strong>:143-156</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cell.2010.11.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cell.2010.11.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-78650475398" target="_blank" aria-label="Open Scopus in new tab">Scopus (436)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21185072/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2010.11.052&amp;pmid=21185072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">81.</a></div><div>Nikel, P.I. ∙ Fuhrer, T. ∙ Chavarría, M. ...</div><div><strong>Reconfiguration of metabolic fluxes in <i>Pseudomonas putida</i> as a response to sub-lethal oxidative stress</strong></div><div><em>ISME J.</em> 2021; <strong>15</strong>:1751-1766</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41396-020-00884-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85100198995" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33432138/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41396-020-00884-9&amp;pmid=33432138" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82-1" title="View in article">82.</a></div><div>Norsigian, C.J. ∙ Fang, X. ∙ Seif, Y. ...</div><div><strong>A workflow for generating multi-strain genome-scale metabolic models of prokaryotes</strong></div><div><em>Nat. Protoc.</em> 2020; <strong>15</strong>:1-14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0254-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85077092545" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31863076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0254-3&amp;pmid=31863076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">83.</a></div><div>Orth, J.D. ∙ Thiele, I. ∙ Palsson, B.Ø.</div><div><strong>What is flux balance analysis?</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:245-248</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1614" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-77749320898" target="_blank" aria-label="Open Scopus in new tab">Scopus (1994)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20212490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1614&amp;pmid=20212490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">84.</a></div><div>Péquignot, C. ∙ Larroche, C. ∙ Gros, J.B.</div><div><strong>A spectrophotometric method for determination of bacterial biomass in the presence of a polymer</strong></div><div><em>Biotechnol. Tech.</em> 1998; <strong>12</strong>:899-903</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1008865512272" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0032436997" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1008865512272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">85.</a></div><div>Perisin, M. ∙ Vetter, M. ∙ Gilbert, J.A. ...</div><div><strong>16Stimator: statistical estimation of ribosomal gene copy numbers from draft genome assemblies</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:1020-1024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2015.161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84941253528" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26359911/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2015.161&amp;pmid=26359911" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">86.</a></div><div>Phaiboun, A. ∙ Zhang, Y. ∙ Park, B. ...</div><div><strong>Survival kinetics of starving bacteria is biphasic and density-dependent</strong></div><div><em>PLoS Comput. Biol.</em> 2015; <strong>11</strong>:e1004198</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1004198" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84929485992" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25838110/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1004198&amp;pmid=25838110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">87.</a></div><div>Price, M.N. ∙ Wetmore, K.M. ∙ Waters, R.J. ...</div><div><strong>Mutant phenotypes for thousands of bacterial genes of unknown function</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:503-509</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0124-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85047501751" target="_blank" aria-label="Open Scopus in new tab">Scopus (173)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29769716/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0124-0&amp;pmid=29769716" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">88.</a></div><div>Pruesse, E. ∙ Peplies, J. ∙ Glöckner, F.O.</div><div><strong>SINA: accurate high-throughput multiple sequence alignment of ribosomal RNA genes</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:1823-1829</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84864134721" target="_blank" aria-label="Open Scopus in new tab">Scopus (1834)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22556368/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts252&amp;pmid=22556368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89-1" title="View in article">89.</a></div><div>Rodionov, D.A. ∙ Dubchak, I.L. ∙ Arkin, A.P. ...</div><div><strong>Dissimilatory metabolism of nitrogen oxides in bacteria: comparative reconstruction of transcriptional networks</strong></div><div><em>PLoS Comput. Biol.</em> 2005; <strong>1</strong></div><div>e55, 17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.0010055" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85044710067" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16261196/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.0010055&amp;pmid=16261196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90-1" title="View in article">90.</a></div><div>Roller, B.R.K. ∙ Stoddard, S.F. ∙ Schmidt, T.M.</div><div><strong>Exploiting rRNA operon copy number to investigate bacterial reproductive strategies</strong></div><div><em>Nat. Microbiol.</em> 2016; <strong>1</strong>:16160</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmicrobiol.2016.160" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84987667484" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27617693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmicrobiol.2016.160&amp;pmid=27617693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">91.</a></div><div>Rosenberger, R.F. ∙ Elsden, S.R.</div><div><strong>The yields of Streptococcus faecalis grown in continuous culture</strong></div><div><em>J.&nbsp;Gen. Microbiol.</em> 1960; <strong>22</strong>:726-739</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1099/00221287-22-3-726" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14438964/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1099%2F00221287-22-3-726&amp;pmid=14438964" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92-1" title="View in article">92.</a></div><div>Sakoparnig, T. ∙ Field, C. ∙ Nimwegen, E. van.</div><div><strong>Whole genome phylogenies reflect the distributions of recombination rates for many bacterial species</strong></div><div><em>eLife.</em> 2021; <strong>10</strong>:e65366</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.65366" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85100215005" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33416498/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.65366&amp;pmid=33416498" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">93.</a></div><div>Sanchez-Gorostiaga, A. ∙ Bajić, D. ∙ Osborne, M.L. ...</div><div><strong>High-order interactions distort the functional landscape of microbial consortia</strong></div><div><em>PLoS Biol.</em> 2019; <strong>17</strong>:e3000550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3000550" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85077298064" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31830028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3000550&amp;pmid=31830028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">94.</a></div><div>Sawada, E. ∙ Toshio, S. ∙ Kitamura, H.</div><div><strong>Purification and properties of a dissimilatory nitrite reductase of a denitrifying phototrophic bacterium</strong></div><div><em>Plant Cell Physiol.</em> 1978; <strong>19</strong>:1339-1351</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.SawadaS.ToshioH.KitamuraPurification+and+properties+of+a+dissimilatory+nitrite+reductase+of+a+denitrifying+phototrophic+bacteriumPlant+Cell+Physiol.19197813391351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">95.</a></div><div>Sawhney, V. ∙ Nicholas, D.J.D.</div><div><strong>Sulphide-linked nitrite reductase from <i>Thiobacillus denitrificans</i> with cytochrome oxidase activity: purification and properties</strong></div><div><em>J.&nbsp;Gen. Microbiol.</em> 1978; <strong>106</strong>:119-128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1099/00221287-106-1-119" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1099%2F00221287-106-1-119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">96.</a></div><div>Schober, A.F. ∙ Mathis, A.D. ∙ Ingle, C. ...</div><div><strong>A two-enzyme adaptive unit within bacterial folate metabolism</strong></div><div><em>Cell Rep.</em> 2019; <strong>27</strong>:3359-3370.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.celrep.2019.05.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.celrep.2019.05.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85066804656" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31189117/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.05.030&amp;pmid=31189117" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97-1" title="View in article">97.</a></div><div>Scott, M. ∙ Gunderson, C.W. ∙ Mateescu, E.M. ...</div><div><strong>Interdependence of cell growth and gene expression: origins and consequences</strong></div><div><em>Science.</em> 2010; <strong>330</strong>:1099-1102</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1192588" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-78449268845" target="_blank" aria-label="Open Scopus in new tab">Scopus (698)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21097934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1192588&amp;pmid=21097934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">98.</a></div><div>Seitzinger, S. ∙ Harrison, J.A. ∙ Böhlke, J.K. ...</div><div><strong>Denitrification across landscapes and waterscapes: a synthesis</strong></div><div><em>Ecol. Appl.</em> 2006; <strong>16</strong>:2064-2090</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1890/1051-0761(2006)016%5B2064:DALAWA%5D2.0.CO;2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33845509937" target="_blank" aria-label="Open Scopus in new tab">Scopus (1092)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17205890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1890%2F1051-0761%282006%29016%255B2064%3ADALAWA%255D2.0.CO%3B2&amp;pmid=17205890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">99.</a></div><div>Sela, I. ∙ Wolf, Y.I. ∙ Koonin, E.V.</div><div><strong>Selection and genome plasticity as the key factors in the evolution of bacteria</strong></div><div><em>Phys. Rev. X.</em> 2019; <strong>9</strong>, 031018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.SelaY.I.WolfE.V.KooninSelection+and+genome+plasticity+as+the+key+factors+in+the+evolution+of+bacteriaPhys.+Rev.+X92019031018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">100.</a></div><div>Shou, W. ∙ Ram, S. ∙ Vilar, J.M.G.</div><div><strong>Synthetic cooperation in engineered yeast populations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:1877-1882</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0610575104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33846936445" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17267602/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0610575104&amp;pmid=17267602" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101-1" title="View in article">101.</a></div><div>Si, Y.-Y. ∙ Xu, K.-H. ∙ Yu, X.-Y. ...</div><div><strong>Complete genome sequence of <i>Paracoccus denitrificans</i> ATCC 19367 and its denitrification characteristics</strong></div><div><em>Can. J. Microbiol.</em> 2019; <strong>65</strong>:486-495</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1139/cjm-2019-0037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85068138644" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30897350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1139%2Fcjm-2019-0037&amp;pmid=30897350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">102.</a></div><div>Sieber, C.M.K. ∙ Probst, A.J. ∙ Sharrar, A. ...</div><div><strong>Recovery of genomes from metagenomes via a dereplication, aggregation and scoring strategy</strong></div><div><em>Nat. Microbiol.</em> 2018; <strong>3</strong>:836-843</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-018-0171-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-85047827857" target="_blank" aria-label="Open Scopus in new tab">Scopus (238)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29807988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-018-0171-1&amp;pmid=29807988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">103.</a></div><div>Stein, L.Y. ∙ Klotz, M.G.</div><div><strong>The nitrogen cycle</strong></div><div><em>Curr. Biol.</em> 2016; <strong>26</strong>:R94-R98</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cub.2015.12.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=10.1016%2Fj.cub.2015.12.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84958755229" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26859274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2015.12.021&amp;pmid=26859274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">104.</a></div><div>Stewart, V. ∙ Lu, Y. ∙ Darwin, A.J.</div><div><strong>Periplasmic nitrate reductase (NapABC enzyme) supports anaerobic respiration by <i>Escherichia coli</i> K-12</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2002; <strong>184</strong>:1314-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.184.5.1314-1323.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0036182372" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11844760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.184.5.1314-1323.2002&amp;pmid=11844760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105-1" title="View in article">105.</a></div><div>Strohm, T.O. ∙ Griffin, B. ∙ Zumft, W.G. ...</div><div><strong>Growth yields in bacterial denitrification and nitrate ammonification</strong></div><div><em>Appl. Environ. Microbiol.</em> 2007; <strong>73</strong>:1420-1424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/AEM.02508-06" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33947193902" target="_blank" aria-label="Open Scopus in new tab">Scopus (174)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17209072/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FAEM.02508-06&amp;pmid=17209072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106" title="View in article">106.</a></div><div>Subramanian, S. ∙ Huq, S. ∙ Yatsunenko, T. ...</div><div><strong>Persistent gut microbiota immaturity in malnourished Bangladeshi children</strong></div><div><em>Nature.</em> 2014; <strong>510</strong>:417-421</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13421" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84903134830" target="_blank" aria-label="Open Scopus in new tab">Scopus (625)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24896187/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13421&amp;pmid=24896187" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107-1" title="View in article">107.</a></div><div>Taylor, J. ∙ Tibshirani, R.J.</div><div><strong>Statistical learning and selective inference</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:7629-7634</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507583112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84934993654" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26100887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507583112&amp;pmid=26100887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108-1" title="View in article">108.</a></div><div>Tiedje, J.M. ∙ Sexstone, A.J. ∙ Myrold, D.D. ...</div><div><strong>Denitrification: ecological niches, competition and survival</strong></div><div><em>Antonie van Leeuwenhoek.</em> 1982; <strong>48</strong>:569-583</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00399542" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-34250229684" target="_blank" aria-label="Open Scopus in new tab">Scopus (423)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6762848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00399542&amp;pmid=6762848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109" title="View in article">109.</a></div><div>Timkovich, R. ∙ Dhesi, R. ∙ Martinkus, K.J. ...</div><div><strong>Isolation of <i>Paracoccus denitrificans</i> cytochrome cd1: comparative kinetics with other nitrite reductases</strong></div><div><em>Arch. Biochem. Biophys.</em> 1982; <strong>215</strong>:47-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0003-9861(82)90277-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0020396614" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6284044/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0003-9861%2882%2990277-6&amp;pmid=6284044" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110" title="View in article">110.</a></div><div>Vaccaro, B.J. ∙ Thorgersen, M.P. ∙ Lancaster, W.A. ...</div><div><strong>Determining roles of accessory genes in denitrification by mutant fitness analyses</strong></div><div><em>Appl. Environ. Microbiol.</em> 2016; <strong>82</strong>:51-61</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/AEM.02602-15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84953923654" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26452555/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FAEM.02602-15&amp;pmid=26452555" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111" title="View in article">111.</a></div><div>Van Alst, N.E. ∙ Sherrill, L.A. ∙ Iglewski, B.H. ...</div><div><strong>Compensatory periplasmic nitrate reductase activity supports anaerobic growth of <i>Pseudomonas aeruginosa</i> PAO1 in the absence of membrane nitrate reductase</strong></div><div><em>Can. J. Microbiol.</em> 2009; <strong>55</strong>:1133-1144</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1139/W09-065" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-70350228801" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19935885/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1139%2FW09-065&amp;pmid=19935885" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">112.</a></div><div>Verbaendert, I. ∙ De Vos, P. ∙ Boon, N. ...</div><div><strong>Denitrification in Gram-positive bacteria: an underexplored trait</strong></div><div><em>Biochem. Soc. Trans.</em> 2011; <strong>39</strong>:254-258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1042/BST0390254" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-79551673854" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21265783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1042%2FBST0390254&amp;pmid=21265783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113" title="View in article">113.</a></div><div>Ward, D.M. ∙ Bateson, M.M. ∙ Ferris, M.J. ...</div><div><strong>Cyanobacterial ecotypes in the microbial mat community of Mushroom Spring (Yellowstone National Park, Wyoming) as species-like units linking microbial community composition, structure and function</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2006; <strong>361</strong>:1997-2008</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2006.1919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-33751210106" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17028085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2006.1919&amp;pmid=17028085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114" title="View in article">114.</a></div><div>Warnecke-Eberz, U. ∙ Friedrich, B.</div><div><strong>Three nitrate reductase activities in <i>Alcaligenes eutrophus</i></strong></div><div><em>Arch. Microbiol.</em> 1993; <strong>159</strong>:405-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00288585" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-0027450107" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00288585" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115" title="View in article">115.</a></div><div>White, D. ∙ Drummond, J. ∙ Fuqua, C.</div><div><strong>The Physiology and Biochemistry of Prokaryotes</strong></div><div>Oxford University Press, 2012</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.WhiteJ.DrummondC.FuquaThe+Physiology+and+Biochemistry+of+Prokaryotes4th+Edition2012Oxford+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">116.</a></div><div>Widder, S. ∙ Allen, R.J. ∙ Pfeiffer, T. ...</div><div><strong>Challenges in microbial ecology: building predictive understanding of community function and dynamics</strong></div><div><em>ISME J.</em> 2016; <strong>10</strong>:2557-2568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ismej.2016.45" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.036&amp;key=2-s2.0-84961904331" target="_blank" aria-label="Open Scopus in new tab">Scopus (321)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27022995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fismej.2016.45&amp;pmid=27022995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117-1" title="View in article">117.</a></div><div>Zumft, W.G.</div><div><strong>Cell biology and molecular basis of denitrification</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 1997; <strong>61</strong>:533-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/.61.4.533-616.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9409151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2F.61.4.533-616.1997&amp;pmid=9409151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none linkback-ad widget-" id="684c6365-474d-4ce7-932a-ddffa4f64d88">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <h2>CHORUS Manuscript</h2>
<p style="margin-bottom:5px;">
<a target="_blank" href="/action/clickThrough?id=2959&amp;url=https%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fam%2Fpii%2FS0092867421015427&amp;loc=%2Fcell%2Ffulltext%2FS0092-8674%2821%2901542-7&amp;pubId=60727892&amp;placeholderId=1346&amp;productId=1822"> View Open Manuscript</a>
</p>
        </div></div>
                </div>
                </div>

        
</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/9000c6fd-78a9-45fa-aa4b-2cdc4ff6cf0e/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/80341704-4b42-41b9-bc1e-a21b331097cf/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/5e9d6e6b-7b74-4800-8080-11884d3c0800/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/69baad6f-d7b0-4c5d-81a6-4b5f351317ab/main.assets/figs1.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/cde99afa-7b18-4f0e-8d79-cd4e4e1dab0c/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/386063b3-daa9-4256-a890-962a1e22dafb/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/b400299d-506f-4f79-b6ad-cd5445c36dfa/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/a5d2fdd7-3b7f-4ed0-8030-4a4bcd0c5b0f/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/44217050-f8dd-41b8-b2bc-63501fa727e6/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/89f28627-edfa-4c11-8721-38a89af9a351/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/472f6efa-b906-4589-98b1-25bf23994ec8/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/fee1c05c-cad1-418a-ba56-62f9d36a9daa/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/759465c8-aeca-42db-8130-c2890b0b19d9/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/asset/2513467f-f526-40d9-a2ab-29e6accf10d4/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Barticle%3Aarticle%3Apii%5C%3AS0092867421015427%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3Apii%5C%3AS0092867421X00049" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (1)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.12.036/attachment/70e71cdc-e768-4d16-b4f4-d7d7cceab83b/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (115.66 KB)</span></a><div class="core-description">Document S1. Tables S1–S7</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421015427" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)12764-8" target="_blank" title="Metagenomic and metabolite analysis reveals microbial community and metabolite dynamics in fermented Indigo naturalis">Metagenomic and metabolite analysis reveals microbial community and metabolite dynamics in fermented Indigo naturalis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yuan et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 22, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(25)00321-9" target="_blank" title="Role of Cyanobacteria in the assembly and dynamics of microbial communities on glacier surfaces">Role of Cyanobacteria in the assembly and dynamics of microbial communities on glacier surfaces</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Xu et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">February 18, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)01652-9" target="_blank" title="Physicochemical and microbial community dynamics of Kocho fermented from different enset varieties in South West Ethiopia">Physicochemical and microbial community dynamics of Kocho fermented from different enset varieties in South West Ethiopia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Tadesse et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">February 7, 2024</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-sustainability/fulltext/S2949-7906(25)00042-4" target="_blank" title="(Re-)engineering microbial community structure-function-metabolism toward sustainable wastewater treatment and sanitation">(Re-)engineering microbial community structure-function-metabolism toward sustainable wastewater treatment and sanitation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Kartik Chandran</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Sustainability</i></span><span id="item_date">March 25, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)07559-5" target="_blank" title="Organic management pattern improves microbial community diversity and alters microbial network structure in karst tea plantation">Organic management pattern improves microbial community diversity and alters microbial network structure in karst tea plantation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Pu et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">May 18, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)17148-4" target="_blank" title="Long-term effect of repeated application of pig slurry digestate on microbial communities in arable soils">Long-term effect of repeated application of pig slurry digestate on microbial communities in arable soils</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Mora-Salguero et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">December 10, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-metabolism/fulltext/S1550-4131(24)00495-9" target="_blank" title="Interplay between gut microbial communities and metabolites modulates pan-cancer immunotherapy responses">Interplay between gut microbial communities and metabolites modulates pan-cancer immunotherapy responses</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Metabolism</i></span><span id="item_date">February 4, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/microbiology/fulltext/S0966-842X(25)00006-X" target="_blank" title="Composing a microbial symphony: synthetic communities for promoting plant growth">Composing a microbial symphony: synthetic communities for promoting plant growth</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Xu et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Microbiology</i></span><span id="item_date">February 17, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(25)00777-1" target="_blank" title="CW-MFC for efficient degradation of ofloxacin-containing wastewater and dynamic analysis of microbial communities">CW-MFC for efficient degradation of ofloxacin-containing wastewater and dynamic analysis of microbial communities</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Li et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">April 23, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)01332-X" target="_blank" title="Purification effect and microbial community structure of bio filter on rural dispersed sewage">Purification effect and microbial community structure of bio filter on rural dispersed sewage</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Shen et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">January 25, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)09671-3" target="_blank" title="Sediment microbial communities of a technogenic saline-alkaline reservoir">Sediment microbial communities of a technogenic saline-alkaline reservoir</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Belkin et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">June 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/biotechnology/fulltext/S0167-7799(24)00305-6" target="_blank" title="Designing synthetic microbial communities with the capacity to upcycle fermentation byproducts to increase production yields">Designing synthetic microbial communities with the capacity to upcycle fermentation byproducts to increase production yields</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Rafieenia et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Biotechnology</i></span><span id="item_date">November 27, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-systems/fulltext/S2405-4712(25)00073-0" target="_blank" title="Assembly of stool-derived bacterial communities follows “early-bird” resource utilization dynamics">Assembly of stool-derived bacterial communities follows “early-bird” resource utilization dynamics</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Aranda-Díaz et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Systems</i></span><span id="item_date">March 28, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)08068-4" target="_blank" title="Invasion by Cenchrus spinifex changes the soil microbial community structure in a sandy grassland ecosystem">Invasion by Cenchrus spinifex changes the soil microbial community structure in a sandy grassland ecosystem</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ren et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">October 18, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)00950-2" target="_blank" title="Exploring the influence of garlic on microbial diversity and metabolite dynamics during kimchi fermentation">Exploring the influence of garlic on microbial diversity and metabolite dynamics during kimchi fermentation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">January 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-systems/fulltext/S2405-4712(25)00125-5" target="_blank" title="Decoding the role of the arginine dihydrolase pathway in shaping human gut community assembly and health-relevant metabolites">Decoding the role of the arginine dihydrolase pathway in shaping human gut community assembly and health-relevant metabolites</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Systems</i></span><span id="item_date">May 7, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(23)01753-0" target="_blank" title="Eco-evolutionary dynamics: The repeatability of diversification in an experimental microbial community">Eco-evolutionary dynamics: The repeatability of diversification in an experimental microbial community</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>William R. Shoemaker</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">February 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)00294-3" target="_blank" title="Inoculum and pH effects on ammonium removal and microbial community dynamics in aquaponics systems">Inoculum and pH effects on ammonium removal and microbial community dynamics in aquaponics systems</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Derikvand et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">January 31, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/endocrinology-metabolism/fulltext/S1043-2760(24)00062-6" target="_blank" title="Emerging methods for genome-scale metabolic modeling of microbial communities">Emerging methods for genome-scale metabolic modeling of microbial communities</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Tarzi et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Endocrinology &amp; Metabolism</i></span><span id="item_date">April 3, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(25)00131-9" target="_blank" title="Mass lysis of predatory bacteria drives the enrichment of antibiotic resistance in soil microbial communities">Mass lysis of predatory bacteria drives the enrichment of antibiotic resistance in soil microbial communities</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Saha et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">February 20, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-systems/fulltext/S2405-4712(23)00267-3" target="_blank" title="Promotion of plasmid maintenance by heterogeneous partitioning of microbial communities">Promotion of plasmid maintenance by heterogeneous partitioning of microbial communities</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Weiss et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Systems</i></span><span id="item_date">October 10, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(25)00755-2" target="_blank" title="Microbial community diversity and assembly processes in the aridification of wetlands on the Qinghai-Tibet Plateau">Microbial community diversity and assembly processes in the aridification of wetlands on the Qinghai-Tibet Plateau</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Di et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">April 22, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(24)00450-1" target="_blank" title="A simple rule for predicting function of microbial communities">A simple rule for predicting function of microbial communities</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sergey Kryazhimskiy</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">June 06, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-methods/fulltext/S2667-2375(24)00205-4" target="_blank" title="SkinCom, a synthetic skin microbial community, enables reproducible investigations of the human skin microbiome">SkinCom, a synthetic skin microbial community, enables reproducible investigations of the human skin microbiome</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lekbua et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Methods</i></span><span id="item_date">August 6, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)01973-X" target="_blank" title="Submersed macrophytes Vallisneria natans and Vallisneria spinulosa improve water quality and affect microbial communities in sediment and water columns">Submersed macrophytes Vallisneria natans and Vallisneria spinulosa improve water quality and affect microbial communities in sediment and water columns</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liao et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">February 7, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(22)02648-2" target="_blank" title="Effects of salt type on the metabolites and microbial community in kimchi fermentation">Effects of salt type on the metabolites and microbial community in kimchi fermentation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lee et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">November 2, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(22)00031-1" target="_blank" title="An ensemble approach to the structure-function problem in microbial communities">An ensemble approach to the structure-function problem in microbial communities</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gopalakrishnappa et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">January 11, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/microbiology/fulltext/S0966-842X(25)00116-7" target="_blank" title="Microbial electrotaxis: rewiring environmental microbiomes">Microbial electrotaxis: rewiring environmental microbiomes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Microbiology</i></span><span id="item_date">April 29, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-systems/fulltext/S2405-4712(24)00033-4" target="_blank" title="High immigration rates critical for establishing emigration-driven diversity in microbial communities">High immigration rates critical for establishing emigration-driven diversity in microbial communities</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chen et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Systems</i></span><span id="item_date">February 23, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/microbiology/fulltext/S0966-842X(23)00025-2" target="_blank" title="Conceptualizing microbe–plasmid communities as complex adaptive systems">Conceptualizing microbe–plasmid communities as complex adaptive systems</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Shai Pilosof</li></ul></div><div class="linked-article__details"><span><i>Trends in Microbiology</i></span><span id="item_date">February 21, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/microbiology/fulltext/S0966-842X(24)00057-X" target="_blank" title="Critically evaluating the relative importance of phage in shaping microbial community composition">Critically evaluating the relative importance of phage in shaping microbial community composition</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Castledine et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Microbiology</i></span><span id="item_date">April 10, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)06645-3" target="_blank" title="Spatial and temporal characteristics of microbial communities in the Seine river in the greater Paris area under anthropogenic perturbation">Spatial and temporal characteristics of microbial communities in the Seine river in the greater Paris area under anthropogenic perturbation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Bagagnan et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">May 2, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-systems/fulltext/S2405-4712(23)00358-7" target="_blank" title="The trade-off between individual metabolic specialization and versatility determines the metabolic efficiency of microbial communities">The trade-off between individual metabolic specialization and versatility determines the metabolic efficiency of microbial communities</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Systems</i></span><span id="item_date">January 17, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)07928-3" target="_blank" title="Estimating postmortem interval based on oral microbial community succession in rat cadavers">Estimating postmortem interval based on oral microbial community succession in rat cadavers</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">May 26, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/microbiology/fulltext/S0966-842X(25)00027-7" target="_blank" title="Disentangling microbial interaction networks">Disentangling microbial interaction networks</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Oña et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Microbiology</i></span><span id="item_date">March 5, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)10365-9" target="_blank" title="Macroinvertebrate community dynamics following wetland restoration: Insights from taxonomic structure and functional feeding groups in response to water quality">Macroinvertebrate community dynamics following wetland restoration: Insights from taxonomic structure and functional feeding groups in response to water quality</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Liu et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 8, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)00110-X" target="_blank" title="Invasive earthworms shift soil microbial community structure in northern North American forest ecosystems">Invasive earthworms shift soil microbial community structure in northern North American forest ecosystems</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ferlian et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">January 12, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)09927-4" target="_blank" title="Higher-order interactions and emergent properties of microbial communities: The power of synthetic ecology">Higher-order interactions and emergent properties of microbial communities: The power of synthetic ecology</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gallardo-Navarro et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 8, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-systems/fulltext/S2405-4712(22)00499-9" target="_blank" title="The community-function landscape of microbial consortia">The community-function landscape of microbial consortia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sanchez et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Systems</i></span><span id="item_date">February 15, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/biophysj/fulltext/S0006-3495(24)00320-5" target="_blank" title="DyMMM-LEAPS: An ML-based framework for modulating evenness and stability in synthetic microbial communities">DyMMM-LEAPS: An ML-based framework for modulating evenness and stability in synthetic microbial communities</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Choudhary et al.</li></ul></div><div class="linked-article__details"><span><i>Biophysical Journal</i></span><span id="item_date">May 9, 2024</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.12.036","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Genomic structure predicts metabolite dynamics in microbial communities</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.12.036/asset/9000c6fd-78a9-45fa-aa4b-2cdc4ff6cf0e/main.assets/gr1.jpg" height="631" width="499" aria-labelledby="fig1-title" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/fb5ee9c5-d1dd-4e9b-afac-7d61131a9008/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.12.036/asset/80341704-4b42-41b9-bc1e-a21b331097cf/main.assets/gr2.jpg" height="177" width="508" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/879e5ac8-d801-4305-8444-b297c2a22da1/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.12.036/asset/5e9d6e6b-7b74-4800-8080-11884d3c0800/main.assets/gr3.jpg" height="597" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/db54d1c3-11ac-4858-a8ef-10d0e4556678/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.12.036/asset/69baad6f-d7b0-4c5d-81a6-4b5f351317ab/main.assets/figs1.jpg" height="923" width="686" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/363e7a12-d4f0-4991-a69f-16d53cf7518d/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.12.036/asset/cde99afa-7b18-4f0e-8d79-cd4e4e1dab0c/main.assets/gr4.jpg" height="670" width="656" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/f26d0a0b-4e66-41db-9a07-64c9aefb4b4c/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.12.036/asset/386063b3-daa9-4256-a890-962a1e22dafb/main.assets/figs2.jpg" height="515" width="506" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/d1d5bc29-3ecc-4d9c-b521-6050c2236522/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.12.036/asset/b400299d-506f-4f79-b6ad-cd5445c36dfa/main.assets/gr5.jpg" height="350" width="656" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/9f884e43-394e-4ec4-b659-2d3d2671f6a3/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.12.036/asset/a5d2fdd7-3b7f-4ed0-8030-4a4bcd0c5b0f/main.assets/gr6.jpg" height="538" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/a35486da-0a48-4d55-adb7-1055a38dcb64/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.12.036/asset/44217050-f8dd-41b8-b2bc-63501fa727e6/main.assets/figs3.jpg" height="733" width="506" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/25ab0bbd-072a-4426-8d66-df8477c97b9b/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.12.036/asset/89f28627-edfa-4c11-8721-38a89af9a351/main.assets/figs4.jpg" height="894" width="658" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/d78a9d0e-a047-4c2c-a5b6-9d0f2ef50ade/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.12.036/asset/472f6efa-b906-4589-98b1-25bf23994ec8/main.assets/figs5.jpg" height="508" width="506" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/821d9bed-d80d-4499-b338-fd18fa818f73/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.12.036/asset/fee1c05c-cad1-418a-ba56-62f9d36a9daa/main.assets/figs6.jpg" height="614" width="761" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/cbf9ff33-a523-4feb-8001-01fd9c071471/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.12.036/asset/759465c8-aeca-42db-8130-c2890b0b19d9/main.assets/gr7.jpg" height="540" width="505" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/f9e6a02e-b722-4572-a1ac-e7cf73e13f96/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.12.036/asset/2513467f-f526-40d9-a2ab-29e6accf10d4/main.assets/figs7.jpg" height="370" width="658" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/7c929b40-c7ed-4dc1-96fb-93572f111a7f/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.12.036/asset/9000c6fd-78a9-45fa-aa4b-2cdc4ff6cf0e/main.assets/gr1.jpg" height="631" width="499" aria-labelledby="fig1-title" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/fb5ee9c5-d1dd-4e9b-afac-7d61131a9008/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.12.036/asset/80341704-4b42-41b9-bc1e-a21b331097cf/main.assets/gr2.jpg" height="177" width="508" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/879e5ac8-d801-4305-8444-b297c2a22da1/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.12.036/asset/5e9d6e6b-7b74-4800-8080-11884d3c0800/main.assets/gr3.jpg" height="597" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/db54d1c3-11ac-4858-a8ef-10d0e4556678/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.12.036/asset/69baad6f-d7b0-4c5d-81a6-4b5f351317ab/main.assets/figs1.jpg" height="923" width="686" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/363e7a12-d4f0-4991-a69f-16d53cf7518d/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.12.036/asset/cde99afa-7b18-4f0e-8d79-cd4e4e1dab0c/main.assets/gr4.jpg" height="670" width="656" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/f26d0a0b-4e66-41db-9a07-64c9aefb4b4c/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.12.036/asset/386063b3-daa9-4256-a890-962a1e22dafb/main.assets/figs2.jpg" height="515" width="506" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/d1d5bc29-3ecc-4d9c-b521-6050c2236522/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.12.036/asset/b400299d-506f-4f79-b6ad-cd5445c36dfa/main.assets/gr5.jpg" height="350" width="656" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/9f884e43-394e-4ec4-b659-2d3d2671f6a3/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.12.036/asset/a5d2fdd7-3b7f-4ed0-8030-4a4bcd0c5b0f/main.assets/gr6.jpg" height="538" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/a35486da-0a48-4d55-adb7-1055a38dcb64/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.12.036/asset/44217050-f8dd-41b8-b2bc-63501fa727e6/main.assets/figs3.jpg" height="733" width="506" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/25ab0bbd-072a-4426-8d66-df8477c97b9b/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.12.036/asset/89f28627-edfa-4c11-8721-38a89af9a351/main.assets/figs4.jpg" height="894" width="658" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/d78a9d0e-a047-4c2c-a5b6-9d0f2ef50ade/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.12.036/asset/472f6efa-b906-4589-98b1-25bf23994ec8/main.assets/figs5.jpg" height="508" width="506" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/821d9bed-d80d-4499-b338-fd18fa818f73/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.12.036/asset/fee1c05c-cad1-418a-ba56-62f9d36a9daa/main.assets/figs6.jpg" height="614" width="761" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/cbf9ff33-a523-4feb-8001-01fd9c071471/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.12.036/asset/759465c8-aeca-42db-8130-c2890b0b19d9/main.assets/gr7.jpg" height="540" width="505" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/f9e6a02e-b722-4572-a1ac-e7cf73e13f96/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.12.036/asset/2513467f-f526-40d9-a2ab-29e6accf10d4/main.assets/figs7.jpg" height="370" width="658" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.036/asset/7c929b40-c7ed-4dc1-96fb-93572f111a7f/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01542-7';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01542-7';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749592569271","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"185","issueNumber":"3","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421015427","coverDate":"202202","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Genomic structure predicts metabolite dynamics in microbial communities","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc1dedacfd4ee9',t:'MTc0OTU5MjU2OC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc1dedacfd4ee9&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.12.036&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>