<html lang="en" class="pb-page js" data-request-id="94d6a712622201a2-GRU"><head data-pb-dropzone="head"><style type="text/css" nonce="94d6a712622201a2-GRU">@font-face {font-family: "Source Sans Pro";src: url(https://consent.trustarc.com/get?name=SourceSansPro-Regular.ttf) format("truetype"),url(https://consent.trustarc.com/get?name=SourceSansPro-Regular.woff) format("woff"),url(https://consent.trustarc.com/get?name=SourceSansPro-Regular.otf) format("opentype"),url(https://consent.trustarc.com/get?name=SourceSansPro-Regular.eot) format("embedded-opentype");}.truste_cursor_pointer {cursor: pointer;}.truste_border_none {border: none;}.truste_accessible_link {font-family: "Source Sans Pro", sans-serif;color: #1D4ED8;font-size: 14px;font-weight: 600;text-decoration: underline;}.truste_accessible_link:hover {color: #1D4ED8;text-decoration: none !important;}.truste_accessible_link:focus-visible {outline: none;border-radius: 4px;box-shadow: 0 0 0 1px #FFFFFF, 0 0 0 4px #3699F1;}</style><script src="https://ssl-cdn.janraincapture.com/widget_data/flow.js:7h8bznq6j7bvyvzqcrwvyp8dfb:en-US:20240514181256512029:standard" type="text/javascript"></script><script src="https://cssjs.nejm.org/akamai/capture:login" type="text/javascript"></script><script src="https://ssl-widget-cdn.rpxnow.com/translations/login/en-US" type="text/javascript"></script><script type="text/javascript" id="janrainAuthWidget" src="https://cssjs.nejm.org/akamai/mms-prod.js"></script><script type="text/javascript" src="https://www.nejm.org/kbl_sPGMZ5s1r3SVIYo_sOWblHlsulg6jLLBNusUYGIrVvcjxj16EX6K_9sqtvN51DyG_h3nvBkbjBgxFWpffA=="></script>
            

<meta charset="UTF-8">






    
        
            
        
    
        
            
        
    
        
            
        
    
        
            
        
    
        
    
        
    
        
            
                
                
<title>100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care — Preliminary Report | New England Journal of Medicine</title>





                
            
        
    







    
        
            
        
    
        
            
        
    
        
            
                
                



                
            
        
    
        
            
        
    
        
    
        
    
        
            
        
    











    








    

    
    

    
    
    
    
        
            
                <link rel="stylesheet" type="text/css" href="/wro/1125813~article-metrics.css">
            
            
            
        
    

    
    
        
            <link rel="icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_nejm_1-994b0160d1eb33dc79517ef0295aab1b.ico" type="image/x-icon">
        
        
            <link rel="icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_16-4aa032aa775940f725817b7ff42f0e2c.png" type="image/png" sizes="16x16">
        
        
            <link rel="icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_32-0a94fa44823476ffa0bb15bdb320fb34.png" type="image/png" sizes="32x32">
        
        
            <link rel="apple-touch-icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_152-d6c3522864a8e483996b7e9c132c901a.png" sizes="152x152">
        
        
            <link rel="apple-touch-icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_180-f2538afc53a365d4cd3f10ab73c56aad.png" sizes="180x180">
        
        
            <link rel="apple-touch-icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_167-8e888012bbdd31a00c3e54bc5b8ca5b6.png" sizes="167x167">
        
    

    
    
            <link rel="stylesheet" href="/products/mms-nextgen/mms/releasedAssets/css/build-5b89b331018866a4a965.css" media="all">
        <link rel="stylesheet" href="/products/mms-nextgen/mms/releasedAssets/css/print-5b89b331018866a4a965.css" media="print">
           <link rel="stylesheet" href="/products/mms-nextgen/mms/releasedAssets/css/build-article-5b89b331018866a4a965.css" media="all" id="article-style-sheet">
    
        <script nonce="94d6a712622201a2-GRU">
        try {
            document.documentElement.classList.add('js');
        } catch (e) {
        }
    </script>
     
    

















    
        
            
                
                
                
                    
                    <!-- TrustArc Scripts -->
<script async="async" src="https://consent.trustarc.com/notice?domain=nejm.com&amp;c=teconsent&amp;js=nj&amp;noticeType=bb&amp;gtm=1&amp;text=true&amp;pcookie" crossorigin="" id="truste_0.9116024304611817"></script>
<script src="https://consent.trustarc.com/autoblockasset/core.min.js?domain=nejm.com"></script>
<script src="https://consent.trustarc.com/autoblockoptout?domain=nejm.com"></script>
<!-- /TrustArc Scripts -->
<!-- BlueConic Scripts -->
<script src="https://bc.nejm.org/script.js"></script>
    <!-- /BlueConic Scripts -->
<!-- Printing Meta Tags -->
    <meta name="articleType" content="Original Article">
<meta name="articleCategory" content="Research">
<meta name="topics" content="Health Care Delivery|Genetics General">
<meta name="Specialties" content="Genetics|Health Policy"><script type="text/javascript">var mmsData = {"access":{"isFreeCountry":"n"},"ad":{"category":"article"},"imageChallenge":{"id":null},"article":{"doi":"10.1056/NEJMoa2035790","title":"100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care — Preliminary Report","category":"Research","type":"Original Article","topics":"Health Care Delivery|Genetics General","specialties":"Genetics|Health Policy","seriesTitle":"","viewType":"Full","age":"2Years-1990","issueDate":"2021-11-11T00:00Z","isFree":"y","broadTopics":"","perspectiveTopics":"","granularTopics":"","methodologicalArea":"","isOnlineFirst":"n","isCurrentIssue":"n","metaTags":"\u003cmeta name\u003d\"articleType\" content\u003d\"Original Article\"/\u003e\n\u003cmeta name\u003d\"articleCategory\" content\u003d\"Research\"/\u003e\n\u003cmeta name\u003d\"topics\" content\u003d\"Health Care Delivery|Genetics General\"/\u003e\n\u003cmeta name\u003d\"Specialties\" content\u003d\"Genetics|Health Policy\"/\u003e"},"page":{"activity":"showFullText","url":"www.nejm.org/doi/full/10.1056/NEJMoa2035790","viewType":"desktop","journal":"NEJM"},"user":{"ucid":null,"audienceSegment":null,"specialtyGroupCode":null,"meterCountRemaining":null,"totalMeterCount":null},"object":{"type":"Visual Abstract","title":"The 100,000 Genomes Pilot on Rare-Disease Diagnosis","doi":"10.1056/NEJMdo006277","issueDate":"2021-11-11T00:00Z","age":"6Months-1990","isFree":"y","topics":"Health Care Delivery|Genetics General","viewType":"Full","specialties":"Genetics|Health Policy"}};</script>
    <script type="text/javascript">var mmsDataAdd = {"user":{"customerType":"null","accessInstitutionID":"null"}};</script>
    <!-- Prehiding snippet for Adobe Target with asynchronous Launch deployment -->
    <!-- placeholder id=null, description=Adobe-Target-Prehiding-Snippet --><!--
  Adobe Target - Prehiding Snippet
  v2.0.0
-->
<script nonce="94d6a712622201a2-GRU">
function tgtNoFlicker(e,n,t,o,i,c,l,r){function d(){return n.getElementsByTagName("head")[0]}function a(e,t){if(e){var o=n.getElementById(t);o&&e.removeChild(o)}}function f(e,o){!function(e,t,o){if(e){var i=n.createElement("style");i.id=t,i.innerHTML=o,e.appendChild(i)}}(d(),t,e),setTimeout(function(){a(d(),t)},o)}var m;n.location.href.indexOf(c)>-1?f(l,r):f(o,i),n.location.href.indexOf("mboxDisable")>-1?(a(d(),t),console.log("[TARGET] - Disabled, removing prehiding style")):(m=function(e){e&&(a(d(),t),console.log("[TARGET] - Blocked, removing prehiding style"))},fetch("//cdn.tt.omtrdc.net/cdn/target-vec.js",{method:"HEAD",mode:"no-cors"}).then(e=>m(!1)).catch(e=>m(!0)))}

tgtNoFlicker(window, document, 'at-body-style', 'div[data-location-name="Anonymous Module"], div[data-location-name="Anonymous Banner"], main > div.page-body > div > div > div > div.col-md-2-3.o-col.o-col--primary > div[class*="u-hide"], main > div.page-body > div > div > div > div.col-md-1-3.o-col.o-col--secondary > aside, header > h1#mainContent, a.m-article__link[href*="/10.1056/NEJMoa2116747"] > span.m-article__blurb > p, li.g-nejm-group__user-tool-secondary {opacity: 0 !important}', 3000);

tgtNoFlicker(window, document, 'at-mbox-style', '.N-ad-global-top-right, #N-ad-myNEJM-contextual-top, .N-ad-recentArticles, #N-ad-article-rightRail-1, #N-ad-siteWide-rightRail-1, #N-ad-imageChallenge-rightRail-2 {opacity: 0 !important}', 3000);
</script><!-- /Prehiding snippet for Adobe Target with asynchronous Launch deployment -->
    <!-- Launch Header Embed Code -->
    <script src="//assets.adobedtm.com/launch-ENcd6450fae3ae4870becbe6a2b01ab2e1.min.js" async="" type="ta-blocked" data-ta-blocked="1"></script>
    <!-- /Launch Header Embed Code -->

                    
                
            
        
    
        
            
                
                
                
                    
                    <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="New England Journal of Medicine"><meta name="dc.Title" content="100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care — Preliminary Report"><meta name="dc.Creator" content="The 100,000 Genomes Project Pilot Investigators"><meta name="dc.Description" content="The U.K. 100,000 Genomes Project is in the process of investigating the role of genome sequencing in patients with undiagnosed rare diseases after usual care and the alignment of this research with..."><meta name="Description" content="The U.K. 100,000 Genomes Project is in the process of investigating the role of genome sequencing in patients with undiagnosed rare diseases after usual care and the alignment of this research with..."><meta name="dc.Publisher" content="Massachusetts Medical Society"><meta name="dc.Date" scheme="WTN8601" content="2021-11-11"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1056/NEJMoa2035790"><meta name="dc.Identifier" scheme="publisher-id" content="NJ202111113852008"><meta name="dc.Language" content="EN"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="Copyright © 2021 Massachusetts Medical Society. All rights reserved.">




<link rel="meta" type="application/atom+xml" href="https://doi.org/10.1056%2FNEJMoa2035790">
<link rel="meta" type="application/rdf+json" href="https://doi.org/10.1056%2FNEJMoa2035790">
<link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1056%2FNEJMoa2035790">


                    
                
            
        
    
        
            
                
                
                
                
                
            
        
    
        
            
                
                
                
                    
                    





<meta name="robots" content="noarchive">





    





    
    
        
        <meta name="twitter:title" content="100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care — Preliminary Report | NEJM">
        <meta property="og:title" content="100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care — Preliminary Report | NEJM">
    


    <meta property="og:type" content="article">


    
    
        <meta property="og:url" content="https://www.nejm.org/doi/full/10.1056/NEJMoa2035790">
    
    


    
    
        <meta property="og:image" content="https://www.nejm.org/cms/asset/ee2525a0-274f-49b3-a2ee-6a621a475e47/nejmoa2035790_f1.jpg">
        <meta name="twitter:image" content="https://www.nejm.org/cms/asset/ee2525a0-274f-49b3-a2ee-6a621a475e47/nejmoa2035790_f1.jpg">
    



    <meta property="og:site_name" content="The New England Journal of Medicine">


    
    
        
        <meta property="og:description" content="The U.K. 100,000 Genomes Project is in the process of investigating the role of genome
sequencing in patients with undiagnosed rare diseases after usual care and the alignment
of this research with...">
        <meta name="twitter:description" content="The U.K. 100,000 Genomes Project is in the process of investigating the role of genome
sequencing in patients with undiagnosed rare diseases after usual care and the alignment
of this research with...">
    


    
    
        <meta property="og:image:width" content="2640">
    


    
    
        <meta property="og:image:height" content="2845">
    


    <meta name="twitter:card" content="summary">


    <meta name="twitter:site" content="@NEJM">


<meta name="viewport" content="width=device-width,initial-scale=1"><meta name="citation_fulltext_world_readable" content="">
    <meta name="publication_doi" content="10.1056/NEJMoa2035790">


    















                    
                
            
        
    
        
    
        
    
        
            
                
                
                
                
                
            
        
    




<meta name="format-detection" content="telephone=no">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
    




    
        
        
        <link rel="canonical" href="https://www.nejm.org/doi/full/10.1056/NEJMoa2035790">
    
    







<meta name="pbContext" content=";requestedJournal:journal:nejm;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:doi\:10.1056/NEJMoa2035790;wgroup:string:MMS NextGen Website Group;issue:issue:doi\:10.1056/nejm_2021.385.issue-20;pageGroup:string:Publication Pages;subPage:string:Full Text;website:website:mms-site;journal:journal:nejms">
        <script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-288192a491f0ad9cc434f05ea175659426f5ab0cfd6befb3f3592f96bb90b0d1.js"></script><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-933-8e181dff8b36ea01dc4c.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-34-6ea546d794fb7a1cea52.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-743-ddd329e1530654ec0846.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-305-9eee37798fdfe5196307.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-388-bb098c62445f518c0b46.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-556-8e58ca05fb5497e541ef.css"><script charset="utf-8" src="https://www.nejm.org/products/mms-nextgen/mms/releasedAssets/js/0-dfd243c732a07d3608f6.js"></script><script charset="utf-8" src="https://www.nejm.org/products/mms-nextgen/mms/releasedAssets/js/7-10d355ba95f7ec1fddf2.js"></script><script charset="utf-8" src="https://www.nejm.org/products/mms-nextgen/mms/releasedAssets/js/11-04bf2a0da30146b4a5d0.js"></script><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><link rel="preconnect" href="https://use.typekit.net"><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><meta http-equiv="P3P" content="CP=&quot;NOI DSP ADM OUR IND OTC&quot;"><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-59614f5c46b49b21eeef3bb28c4fb38d1e7069e8d014752fcb66e84942556802.css"><link rel="stylesheet" href="https://quilt-cdn.janrain.com/HEAD/providers.css" type="text/css"></head>
    

    



    
        
        
        
            <body class="pb-ui website-mms-site loaded">
            






    
    
        





    
    
        
        
    
    
    







<input type="hidden" id="isRoMode" name="isRoMode" value="false">

        <div id="pb-page-content" data-ng-non-bindable="">
                
            <div data-pb-dropzone="main" data-pb-dropzone-name="Main">
                
                    
                        



        
        <header data-sticky="false" class="ng-header js_stickyAd"><a href="#mainContent" class="ng-skipToMainContent">Skip to main content</a><div class="ng-header_topAd"><div class="ng-header_topAd-content d-flex justify-content-center align-items-center">



        
        <div id="DTM_Position_Topbanner" class="ad">
 <!--emptycomment-->
</div>
</div></div><div class="ng-header_journal-navbar"><div class="ng-header_journal-navbar-content"><div class="container-fluid"><div class="row"><div class="ng-header_journal-navbar_left col-lg-8 col-sm-12 pr-lg-0 d-flex align-items-center"><div data-location="nejm_group_product_nav" class="ng-header_journalNavMenu">



        
        <nav data-location="nejm_group_product_nav" role="navigation" class="ng-journalNavMenu">
<ul class="ng-journalNavMenu_list">
<li class="ng-journalNavMenu_list-item"><a href="/" class="ng-journalNavMenu_mainLink active" aria-current="page" aria-label="The New England Journal of Medicine selected item"><span class="ng-journalNavMenu_mainLink-text d-lg-block d-none">The New England Journal of Medicine</span><span class="ng-journalNavMenu_mainLink-text d-lg-none d-block">NEJM</span></a></li>
<li class="ng-journalNavMenu_list-item"><a href="https://evidence.nejm.org/" class="ng-journalNavMenu_mainLink"><span class="ng-journalNavMenu_mainLink-text"><span class="ng-journalNavMenu_mainLink-text-firstWord">NEJM</span> Evidence</span></a></li>
<li class="ng-journalNavMenu_list-item"><a href="https://ai.nejm.org/" class="ng-journalNavMenu_mainLink"><span class="ng-journalNavMenu_mainLink-text"><span class="ng-journalNavMenu_mainLink-text-firstWord">NEJM</span> AI</span></a></li>
<li class="ng-journalNavMenu_list-item"><a href="https://catalyst.nejm.org/" class="ng-journalNavMenu_mainLink"><span class="ng-journalNavMenu_mainLink-text"><span class="ng-journalNavMenu_mainLink-text-firstWord">NEJM</span> Catalyst</span></a></li>
<li class="ng-journalNavMenu_list-item"><a href="https://www.jwatch.org/" class="ng-journalNavMenu_mainLink"><span class="ng-journalNavMenu_mainLink-text"><span class="ng-journalNavMenu_mainLink-text-firstWord">NEJM</span> Journal Watch</span></a></li>
</ul>
</nav>

</div></div><div class="ng-header_journal-navbar_right col-lg-4 pl-lg-0 d-none d-xl-flex align-items-center justify-content-end"><div class="ng-header_loginBar d-xl-flex align-items-lg-center">



        
        <div data-location="authstring_nav" class="ng-header_loginBar-menu"><span class="ng-simple-menu_mainLink-icon signIn newIcon-2"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#user2"></use></svg></span></span><span class="ng-header_loginBar-signIn"><a href="/sign-in?uri=/doi/full/10.1056/NEJMoa2035790" class="ng-header_loginBar-signIn-link">Sign In</a></span><span class="ng-header_loginBar-separator">|</span><span class="ng-header_loginBar-createAcc"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC1" class="ng-header_loginBar-createAcc-link ro-mode_disabled">Create Account</a></span></div><div data-location="authstring_nav" class="ng-header_loginBar-subscribe pl-xl-16"><a href="/action/storeProxy?action=subscribe&amp;promo=ONF4NRS1" class="ng-header_loginBar-subscribe-btn ng-btn_primary ro-mode_disabled">Subscribe</a></div>
</div></div></div></div></div></div><div class="ng-header_instBanner d-none d-lg-block"></div><div class="ng-header_logoBar"><div class="container-fluid"><div class="row"><div data-location="logo" class="ng-header_logoBar_left col-lg-4 col-6 d-flex align-items-center justify-content-start"><a href="/" title="The New England Journal of Medicine homepage" class="ng-header_logo-link"><img src="/specs/products/mms-nextgen/mms/releasedAssets/images/other-images/nejm-logo-166667e91992a5212bc723b03e45d39f.svg" alt="The New England Journal of Medicine homepage" width="366" height="58" class="ng-header_logo-image"></a></div><div class="ng-header_logoBar_right ng-header_navBar col-lg-8 col-sm-12 p-lg-0 align-items-center justify-content-end"><div class="ng-header_navBar-content"><div class="container-fluid"><div class="row"><div class="ng-header_instBanner-md col-12 d-block d-lg-none"></div><div class="ng-header_loginBar-md col-12 d-block d-xl-none">



        
        <div data-location="authstring_nav" class="ng-header_loginBar-menu"><span class="ng-simple-menu_mainLink-icon signIn newIcon-2"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#user2"></use></svg></span></span><span class="ng-header_loginBar-signIn"><a href="/sign-in?uri=/doi/full/10.1056/NEJMoa2035790" class="ng-header_loginBar-signIn-link">Sign In</a></span><span class="ng-header_loginBar-separator">|</span><span class="ng-header_loginBar-createAcc"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC1" class="ng-header_loginBar-createAcc-link ro-mode_disabled">Create Account</a></span></div><div data-location="authstring_nav" class="ng-header_loginBar-subscribe pl-xl-16"><a href="/action/storeProxy?action=subscribe&amp;promo=ONF4NRS1" class="ng-header_loginBar-subscribe-btn ng-btn_primary ro-mode_disabled">Subscribe</a></div>
</div><div class="ng-header_navBar_left col-lg-8 col-sm-12 d-flex align-items-center"><div class="ng-header_stickyLogo mr-32"><a href="/" title="The New England Journal of Medicine homepage" class="ng-header_stickyLogo-link"><img src="/specs/products/mms-nextgen/mms/releasedAssets/images/other-images/nejm-small-logo-070df7a3f4d8cfa2e71c3c45bb4b172f.svg" alt="The New England Journal of Medicine homepage" class="ng-header_stickyLogo-image"></a></div><div class="ng-header_mainMenu">









    
    
        <div data-widget-def="menuWidget" data-widget-id="97fb3983-db63-4b0b-97dd-d9bb7929fce1" data-location="main_nav">
        



        
        <nav class="ng-menu"><ul class="ng-menu_list ng-simple-menu"><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/toc/nejm/current" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">current issue</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/toc/nejm/current" class="ng-simple-menu_list-link">VIEW CURRENT ISSUE</a></li><li class="ng-simple-menu_list-item"><a href="/toc/nejm/recently-published" class="ng-simple-menu_list-link">BROWSE RECENTLY PUBLISHED</a></li><li class="ng-simple-menu_list-item"><a href="/loi/nejm" class="ng-simple-menu_list-link">BROWSE ALL ISSUES</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/medical-specialties" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">SPECIALTIES</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/browse/specialty/cardiology" class="ng-simple-menu_list-link">Cardiology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/clinical-medicine" class="ng-simple-menu_list-link">Clinical Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/endocrinology" class="ng-simple-menu_list-link">Endocrinology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/gastroenterology" class="ng-simple-menu_list-link">Gastroenterology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/hematology-oncology" class="ng-simple-menu_list-link">Hematology/Oncology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/infectious-disease" class="ng-simple-menu_list-link">Infectious Disease</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/nephrology" class="ng-simple-menu_list-link">Nephrology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/neurology-neurosurgery" class="ng-simple-menu_list-link">Neurology/Neurosurgery</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/obstetrics-gynecology" class="ng-simple-menu_list-link">Obstetrics/Gynecology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/pediatrics" class="ng-simple-menu_list-link">Pediatrics</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/pulmonary-critical-care" class="ng-simple-menu_list-link">Pulmonary/Critical Care</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/surgery" class="ng-simple-menu_list-link">Surgery</a></li><li class="ng-simple-menu_list-item"><a href="/medical-specialties" class="ng-simple-menu_list-link">View All Specialties</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/topics" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">TOPICS</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/ai-in-medicine" class="ng-simple-menu_list-link">AI in Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/climate-change" class="ng-simple-menu_list-link">Climate Change</a></li><li class="ng-simple-menu_list-item"><a href="/coronavirus" class="ng-simple-menu_list-link">Coronavirus</a></li><li class="ng-simple-menu_list-item"><a href="/equity" class="ng-simple-menu_list-link">Efforts toward Equity</a></li><li class="ng-simple-menu_list-item"><a href="/firearm-injury-prevention" class="ng-simple-menu_list-link">Firearm Injury Prevention</a></li><li class="ng-simple-menu_list-item"><a href="/nam" class="ng-simple-menu_list-link">From the National Academy of Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/fundamentals-of-medical-ethics" class="ng-simple-menu_list-link">Fundamentals of Medical Ethics</a></li><li class="ng-simple-menu_list-item"><a href="/gray-matters" class="ng-simple-menu_list-link">Gray Matters</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/health-policy" class="ng-simple-menu_list-link">Health Policy</a></li><li class="ng-simple-menu_list-item"><a href="/medicine-and-society" class="ng-simple-menu_list-link">Medicine and Society</a></li><li class="ng-simple-menu_list-item"><a href="/nutrition-in-medicine" class="ng-simple-menu_list-link">Nutrition in Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/outbreaks" class="ng-simple-menu_list-link">Outbreaks Center</a></li><li class="ng-simple-menu_list-item"><a href="/race-and-medicine" class="ng-simple-menu_list-link">Race and Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/recognizing-historical-injustices" class="ng-simple-menu_list-link">Recognizing Historical Injustices and the Journal</a></li><li class="ng-simple-menu_list-item"><a href="/tobacco-use-reduction" class="ng-simple-menu_list-link">Tobacco Use Reduction</a></li><li class="ng-simple-menu_list-item"><a href="/topics" class="ng-simple-menu_list-link">View All Topics</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/multimedia" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">MULTIMEDIA</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/rss-feed" class="ng-simple-menu_list-link">Podcasts</a></li><li class="ng-simple-menu_list-item"><a href="/double-take" class="ng-simple-menu_list-link">Double Takes</a></li><li class="ng-simple-menu_list-item"><a href="https://illustrated-glossary.nejm.org/" class="ng-simple-menu_list-link">Illustrated Glossary</a></li><li class="ng-simple-menu_list-item"><a href="/image-challenge" class="ng-simple-menu_list-link">Image Challenge</a></li><li class="ng-simple-menu_list-item"><a href="/browse/nejm-media-type/images-in-clinical-medicine" class="ng-simple-menu_list-link">Images in Clinical Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/browse/nejm-media-type/interactive-medical-case" class="ng-simple-menu_list-link">Interactive Medical Cases</a></li><li class="ng-simple-menu_list-item"><a href="/plain-language-research-summaries" class="ng-simple-menu_list-link">Plain Language/Research Summaries</a></li><li class="ng-simple-menu_list-item"><a href="/multimedia/quick-take-video" class="ng-simple-menu_list-link">Quick Takes</a></li><li class="ng-simple-menu_list-item"><a href="/browse/nejm-media-type/videos-in-clinical-medicine" class="ng-simple-menu_list-link">Videos in Clinical Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/multimedia" class="ng-simple-menu_list-link">View All Multimedia</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/learning" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">LEARNING/CME</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/continuing-medical-education" class="ng-simple-menu_list-link">Weekly CME</a></li><li class="ng-simple-menu_list-item"><a href="/cme-ce/obesity" class="ng-simple-menu_list-link">Obesity CME/CE</a></li><li class="ng-simple-menu_list-item"><a href="/learning" class="ng-simple-menu_list-link">VIEW ALL LEARNING/CME</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item"><a href="/author-center/home" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">AUTHOR CENTER</span></a></li></ul></nav>

        </div>
    

</div><button aria-label="Search" class="ng-header_searchBtn searchBtnNav d-block ml-24"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#search"></use></svg></span></button></div><div class="ng-header_loginBar ng-header_loginBar-sticky col-4">



        
        <div data-location="authstring_nav" class="ng-header_loginBar-menu"><span class="ng-simple-menu_mainLink-icon signIn newIcon-2"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#user2"></use></svg></span></span><span class="ng-header_loginBar-signIn"><a href="/sign-in?uri=/doi/full/10.1056/NEJMoa2035790" class="ng-header_loginBar-signIn-link">Sign In</a></span><span class="ng-header_loginBar-separator">|</span><span class="ng-header_loginBar-createAcc"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC1" class="ng-header_loginBar-createAcc-link ro-mode_disabled">Create Account</a></span></div><div data-location="authstring_nav" class="ng-header_loginBar-subscribe pl-xl-16"><a href="/action/storeProxy?action=subscribe&amp;promo=ONF4NRS1" class="ng-header_loginBar-subscribe-btn ng-btn_primary ro-mode_disabled">Subscribe</a></div>
</div></div></div></div></div><div class="ng-header_logoBar_right-lg col-lg-8 col-6 d-flex align-items-center justify-content-end d-xl-none"><button aria-label="Search" class="ng-header_searchBtn d-flex justify-content-end"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#search"></use></svg></span></button><div class="ng-header_openCloseBtns d-flex align-items-center justify-content-end"><button aria-label="Open" class="ng-header_openBtn"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#burger"></use></svg></span></button><button aria-label="Close" class="ng-header_closeBtn"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#close"></use></svg></span></button></div></div><div class="ng-header_quickSearch col-12"><div class="ng-header_quickSearch-content">



        
        <div class="ng-quick-search js-ng-quick-search"><form action="/search" method="get" class="ng-quick-search_form js-form"><div class="ng-quick-search_inputArea js-inputArea"><input type="search" name="q" placeholder="Enter keyword, author, title or citation" autocomplete="off" value="" required="required" class="ng-quick-search_allField ng-large-text-input js-allField js-allField-def-tmpl ui-autocomplete-input"><button type="submit" aria-label="Search Button" class="ng-quick-search_iconBtn d-xl-none d-block js-submitBtn"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#search"></use></svg></span></button><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" style="display: none;"></ul></div><div class="ng-quick-search_btnsArea d-none d-xl-flex align-items-center justify-content-end"><a href="/search/advanced" class="ng-quick-search_advSearchLink ng-btn_link js-advancedSearchLink">Advanced Search</a><button type="submit" class="ng-quick-search_textBtn ng-btn_secondary ng-btn_iconRight js-submitBtn"><span class="ng-btn_text">SEARCH</span><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-right"></use></svg></span></button></div></form></div>
</div></div></div></div></div></header><div class="ng-header_overlay"></div><div class="ng-header_after">









    
    
        <div data-widget-def="literatumAd" data-widget-id="4931ddbd-d46e-4597-b4fd-92549e9847ca" id="ad-global-banner-FULLx64-1">
        



        
        



    

    

    
        
    



        </div>
    

</div>

                    
                        



        
        <div>









    
    
        <main data-widget-def="axelPublicationContent" data-widget-id="fcdb5261-d6a2-4a9c-81d5-3a3e2304e9c1" id="mainContent">
        



        
        <div class="article-tools__savePopup"><div class="ng-save"><input type="hidden" value="/action/getSavedTags" class="getKeywordsServiceUrl"><input type="hidden" value="/action/mmsSaveItemsService?itemType=article" class="saveServiceUrl"><input type="hidden" value="article" class="saveType"><input type="hidden" value="10.1056/NEJMoa2035790" class="inputDoi"><input type="hidden" value="The 100,000 Genomes Project Pilot Investigators" class="inputAuthor"><input type="hidden" value="N Engl J Med 2021;385:1868-1880" class="inputCitation"><input type="hidden" value="11-10-2021" class="inputEPubDate"><input type="hidden" value="November 2021" class="inputCoverDate"><input type="hidden" value="Original Article" class="inputContentType"><input type="hidden" value="100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care — Preliminary Report" class="inputArticleTitle"><input type="hidden" value="/browse/nejm-article-type/original-article" class="inputContentTypeUrl"><button type="button" id="saveArticleShowPopupBtn" aria-hidden="true" tabindex="-1" class="ng-save-btn ng-btn_secondary isLoggedOut">Save</button></div></div><div class="article-tools__articleAlertPopup"><div id="articleAlertModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true" class="ng-modal modal"><div role="document" class="ng-modal_dialog modal-dialog modal-dialog-centered"><div class="ng-modal_content modal-content"><div class="ng-modal_header modal-header"><h5 id="articleAlertModalTitle" class="ng-modal_title modal-title">Create an E-mail Alert for This Article</h5><button type="button" data-dismiss="modal" aria-label="Close" class="ng-modal_close close"><span aria-hidden="true"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#close"></use></svg></span></span></button></div><div class="ng-modal_body modal-body"></div></div></div></div></div><div class="ng-do-media_popup modal-lg"><div id="doPopup" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true" class="ng-modal modal"><div role="document" class="ng-modal_dialog modal-dialog modal-dialog-centered"><div class="ng-modal_content modal-content"><div class="ng-modal_header modal-header"><button type="button" data-dismiss="modal" aria-label="Close" class="ng-modal_close close"><span aria-hidden="true"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#close"></use></svg></span></span></button></div><div class="ng-modal_body modal-body"></div></div></div></div></div><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/mms-nextgen/mms/releasedAssets/css/build-article-5b89b331018866a4a965.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" data-type="research-article" vocab="http://schema.org/" typeof="ScholarlyArticle" lang="en" dir="ltr"><header data-extent="frontmatter" data-location="header_article"><div class="core-container"><div data-article-access="free" data-article-access-type="free" class="meta-panel"><div class="meta-panel__left-content"><div class="meta-panel__type"><a href="/browse/nejm-article-type/original-article">Original Article</a></div></div><div class="meta-panel__right-content"><div class="meta-panel__share">









    
    
        <div data-widget-def="UX3share" data-widget-id="eee5f139-16f2-48f8-97ea-1b114367da16" data-location="share_tools_article">
        



        
        <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js" nonce="94d6a712622201a2-GRU"></script><div class="share"><div class="share__block share__inline-links"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="d-flex list-unstyled a2a a2a_kit a2a_default_style mb-0 a2a_kit_size_32" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Facebook" data-interactiontype="social" class="share__link a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-facebook"></use></svg></span></a></li><li class="a2a_listitem_custom"><a role="link" title="X (formerly Twitter)" data-interactiontype="social" class="share__link a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-twitter"></use></svg></span></a></li><li class="a2a_listitem_custom"><a role="link" title="Linked In" data-interactiontype="social" class="share__link a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-linkedin"></use></svg></span></a></li><li class="a2a_listitem_custom"><a role="link" title="Email" data-interactiontype="content_email_click" class="share__link a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-email"></use></svg></span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" data-interactiontype="social" class="share__link a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-bluesky"></use></svg></span></a></li></ul></div></div>

        </div>
    

</div></div></div><h1 property="name">100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care — Preliminary Report</h1><div class="core-relations my-3"></div><div class="contributors"><span class="authors"><span class="heading">Author</span>: <span role="list"><span property="author" typeof="Person" role="listitem">The 100,000 Genomes Project Pilot Investigators<sup class="xref"><a href="#fn0" role="doc-noteref">*</a></sup></span></span></span><a href="#tab-contributors" class="to-authors-affiliations" data-id="article-authors-viewall">Author Info &amp; Affiliations</a></div><div class="core-self-citation"><div class="core-date-published">Published <span property="datePublished">November 10, 2021</span></div><div property="isPartOf" typeof="Periodical"><span property="name">N Engl J Med</span> 2021<span property="isPartOf" typeof="PublicationVolume">;<span property="volumeNumber">385</span></span>:<span property="pageStart">1868</span>-<span property="pageEnd">1880</span></div><div class="doi">DOI: 10.1056/NEJMoa2035790</div><div class="core-enumeration"><a href="/toc/nejm/385/20"><span property="isPartOf" typeof="PublicationVolume">VOL. <span property="volumeNumber">385</span></span> <span property="isPartOf" typeof="PublicationIssue">NO. <span property="issueNumber">20</span></span></a></div><div><a href="#tab-information">Copyright © 2021</a></div></div><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__metrics info-panel__item"></div></div><div data-location="tools_article" class="info-panel__right-content"><div class="info-panel__article_tools info-panel__item">



        
        <div data-permission="/servlet/linkout?type=rightslink&amp;url=publisherName%3Dmassmed%26author%3D%26publication%3Dnejm%26orderBeanReset%3Dtrue%26volumeNum%3D385%26issueNum%3D20%26contentID%3D10.1056%252FNEJMoa2035790%26title%3D100%252C000%2BGenomes%2BPilot%2Bon%2BRare-Disease%2BDiagnosis%2Bin%2BHealth%2BCare%2B%25E2%2580%2594%2BPreliminary%2BReport%26publicationDate%3D11%252F11%252F2021" class="article-tools"><button id="articleToolsAlertBtn" data-url="/action/addCitationAlert?doi=10.1056%2FNEJMoa2035790" data-toggle="" data-target="#articleAlertModal" aria-label="Sign in or create account to add an article alert" data-interactiontype="article_alert" class="article-tools__citation btn btn--slim isLoggedOut"><span title="" data-toggle="tooltip" class="d-inline-block" data-original-title="Sign in or create account to add an article alert"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#alerts"></use></svg></span></span></button><button id="articleToolsSaveBtn" data-toggle="modal" data-target="#saveArticle" aria-label="Sign in or create account to save this article" data-interactiontype="article_tools_save_click" class="article-tools__favorite btn btn--slim isLoggedOut"><span title="" data-toggle="tooltip" class="d-inline-block" data-original-title="Sign in or create account to save this article"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#bookmarkFilled"></use></svg></span></span></button><a href="/servlet/linkout?type=rightslink&amp;url=publisherName%3Dmassmed%26author%3D%26publication%3Dnejm%26orderBeanReset%3Dtrue%26volumeNum%3D385%26issueNum%3D20%26contentID%3D10.1056%252FNEJMoa2035790%26title%3D100%252C000%2BGenomes%2BPilot%2Bon%2BRare-Disease%2BDiagnosis%2Bin%2BHealth%2BCare%2B%25E2%2580%2594%2BPreliminary%2BReport%26publicationDate%3D11%252F11%252F2021" target="_blank" title="" data-toggle="tooltip" aria-label="Permissions" data-interactiontype="article_tools_permissions_click" class="article-tools__permissions btn btn--slim" data-original-title="Permissions"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#permissions"></use></svg></span></a><a href="/about-nejm/reprints" title="" data-toggle="tooltip" aria-label="Reprints" data-interactiontype="article_tools_reprints_click" class="article-tools__reprints btn btn--slim" data-original-title="Reprints"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#reprints"></use></svg></span></a><a href="/cms/asset/a49abbe0-f1b0-45d6-bf35-2089bf0eed44/nejmoa2035790.pptx" target="_blank" title="" data-toggle="tooltip" aria-label="Download Slides" data-interactiontype="article_tools_slide_set_download" data-multimedia-type="Article Slideset" data-multimedia-filename="nejmoa2035790.pptx" class="article-tools__downloadSlides btn btn--slim" data-original-title="Download Slides"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#download"></use></svg></span></a><div class="article-tools__permissionsPopup"><div id="permissionsModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true" class="ng-modal modal"><div role="document" class="ng-modal_dialog modal-dialog modal-dialog-centered"><div class="ng-modal_content modal-content"><div class="ng-modal_header modal-header"><h5 id="permissionsModalTitle" class="ng-modal_title modal-title">Permissions</h5><button type="button" data-dismiss="modal" aria-label="Close" class="ng-modal_close close"><span aria-hidden="true"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#close"></use></svg></span></span></button></div><div class="ng-modal_body modal-body">For permission requests, please contact NEJM Reprints at <a href="mailto:reprints@nejm.org">reprints@nejm.org</a></div></div></div></div></div></div>
</div><div class="info-panel__citations info-panel__item"><a href="#tab-citations" class="btn btn--slim" aria-label="View Citations" data-toggle="tooltip" data-original-title="View Citations" data-interactiontype="article_tools_citation_download" data-multimedia-type="Citation"><i aria-hidden="true" class="icon-format_quote"></i></a></div><div class="info-panel__formats info-panel__item"><a href="https://www.nejm.org/doi/pdf/10.1056/NEJMoa2035790" class="btn btn--pdf btn--slim" aria-label="View PDF" data-toggle="tooltip" data-original-title="View PDF" data-behavior="trackDownloadEvent" data-doi="10.1056/NEJMoa2035790" data-interactiontype="article_tools_download_pdf" data-multimedia-type="Article PDF" data-multimedia-filename="nejmoa2035790.pdf"><i aria-hidden="true" class="icon-PDF"></i></a></div></div></div></div></header><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Contents</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><ul><li><a href="#summary-abstract" aria-current="true">Abstract</a></li><li><a href="#sec-1" aria-current="false">Methods</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li class="bordered"><a href="#backnotes" aria-current="false">Notes</a></li><li><a href="#supplementary-materials" aria-current="false">Supplementary Material</a></li><li><a href="#appendix-1" aria-current="false">Appendix</a></li><li><a href="#bibliography" aria-current="false">References</a></li></ul></nav></div><nav id="article_collateral_menu"><ul data-core-nav="collateral" class="collateral-pill" style="margin-top: 0px;"><li><a href="#core-collateral-info" data-id="article-nav-menubar-info" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Information &amp; Authors"><i aria-hidden="true" class="icon-info"></i><span class="sr-only">Information &amp; Authors</span></a></li><li><a href="#core-collateral-metrics" data-id="article-nav-menubar-metrics" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Metrics &amp; Citations"><i aria-hidden="true" class="icon-timeline"></i><span class="sr-only">Metrics &amp; Citations</span></a></li><li><a href="#core-collateral-fulltext-options" data-id="article-nav-menubar-metrics" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="View Options"><i aria-hidden="true" class="icon-eye"></i><span class="sr-only">View Options</span></a></li><li><a href="#core-collateral-references" data-id="article-nav-menubar-references" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="References"><i aria-hidden="true" class="icon-references"></i><span class="sr-only">References</span></a></li><li><a href="#core-collateral-media" data-id="article-nav-menubar-media" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Media"><i aria-hidden="true" class="icon-photo"></i><span class="sr-only">Media</span></a></li><li><a href="#core-collateral-tables" data-id="article-nav-menubar-tables" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Tables"><i aria-hidden="true" class="icon-tables"></i><span class="sr-only">Tables</span></a></li><li><a href="#core-collateral-share" data-id="article-nav-menubar-share" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Share"><i aria-hidden="true" class="icon-share"></i><span class="sr-only">Share</span></a></li></ul></nav></div></div><div data-core-wrapper="content"><div id="abstracts" data-extent="frontmatter" data-location="articleTab_article"><div class="core-container"><section id="summary-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Abstract</h2><section id="abs-sec-1"><h3>Background</h3><div role="paragraph">The U.K. 100,000 Genomes Project is in the process of investigating the role of genome sequencing in patients with undiagnosed rare diseases after usual care and the alignment of this research with health care implementation in the U.K. National Health Service. Other parts of this project focus on patients with cancer and infection.</div></section><section id="abs-sec-2"><h3>Methods</h3><div role="paragraph">We conducted a pilot study involving 4660 participants from 2183 families, among whom 161 disorders covering a broad spectrum of rare diseases were present. We collected data on clinical features with the use of Human Phenotype Ontology terms, undertook genome sequencing, applied automated variant prioritization on the basis of applied virtual gene panels and phenotypes, and identified novel pathogenic variants through research analysis.</div></section><section id="abs-sec-3"><h3>Results</h3><div role="paragraph">Diagnostic yields varied among family structures and were highest in family trios (both parents and a proband) and families with larger pedigrees. Diagnostic yields were much higher for disorders likely to have a monogenic cause (35%) than for disorders likely to have a complex cause (11%). Diagnostic yields for intellectual disability, hearing disorders, and vision disorders ranged from 40 to 55%. We made genetic diagnoses in 25% of the probands. A total of 14% of the diagnoses were made by means of the combination of research and automated approaches, which was critical for cases in which we found etiologic noncoding, structural, and mitochondrial genome variants and coding variants poorly covered by exome sequencing. Cohortwide burden testing across 57,000 genomes enabled the discovery of three new disease genes and 19 new associations. Of the genetic diagnoses that we made, 25% had immediate ramifications for clinical decision making for the patients or their relatives.</div></section><section id="abs-sec-4"><h3>Conclusions</h3><div role="paragraph">Our pilot study of genome sequencing in a national health care system showed an increase in diagnostic yield across a range of rare diseases. (Funded by the National Institute for Health Research and others.)</div></section></section><div class="core-digital-object visual-abstract">
<div class="ng-page_separator pt-24"></div>
<div class="ng-do-media">
    <button data-toggle="modal" data-target="#doPopup" class="ng-do-media_btn" modal-size="lg"></button>
    <div data-ajaxurl="/do/10.1056/NEJMdo006277/full/?ajaxRequest=true" class="ng-do-media_item">
        <div class="ng-do-media_item-left">
            <a href="#" class="ng-do-media_item-img-link" data-interactiontype="multimedia_click" data-multimedia-type="Visual Abstract" data-multimedia-contentid="10.1056/NEJMdo006277" data-multimedia-filename="NEJMdo006277_600x400.jpg">
                <img src="/cms/asset/57b09620-f1b5-4b00-8b9b-bec62756f3c2/media/NEJMdo006277_300x200.jpg" class="ng-do-media_item-img">
                <span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#tooltip-visual"></use></svg></span>
            </a>
        </div>
        <div class="ng-do-media_item-right">
            <div class="ng-do-media_item-type">
                <a href="/browse/nejm-media-type/visual-abstract" class="ng-do-media_item-type-link">Visual Abstract</a>
            </div>
            <h6 class="ng-do-media_item-title">
                <a href="#" class="ng-do-media_item-title-link" data-interactiontype="multimedia_click" data-multimedia-type="Visual Abstract" data-multimedia-contentid="10.1056/NEJMdo006277" data-multimedia-filename="NEJMdo006277_600x400.jpg">The 100,000 Genomes Pilot on Rare-Disease Diagnosis</a>
            </h6>
        </div>
    </div>
</div>

</div></div></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text" data-location="articleTab_article"><div class="core-container"><div role="paragraph">Rare diseases are a worldwide health care challenge, with approximately 10,000 disorders affecting 6% of the population in Western societies.<sup><a href="#core-r1" role="doc-biblioref" data-xml-rid="r1 r2" id="body-ref-r2" href-manipulated="true">1,2</a></sup> More than 80% of rare diseases have a genetic component, and these conditions are disabling and expensive to manage. One third of children with a rare disease die before their fifth birthday.<sup><a href="#core-r1" role="doc-biblioref" data-xml-rid="r1" id="body-ref-r1-2" href-manipulated="true" aria-label="Reference 1">1</a></sup> The adoption of next-generation sequencing has improved rates of diagnosis of rare diseases over the past decade.<sup><a href="#core-r3" role="doc-biblioref" data-xml-rid="r3 r4 r5" id="body-ref-r5-1" href-manipulated="true">3-5</a></sup> However, the majority of patients with rare diseases remain without a molecular diagnosis after standard diagnostic testing.<sup><a href="#core-r3" role="doc-biblioref" data-xml-rid="r3 r4 r5" id="body-ref-r5-2" href-manipulated="true">3-5</a></sup> To address this lack of diagnosis, the U.K. government launched the 100,000 Genomes Project in 2013 to apply whole-genome sequencing to the study of rare diseases, cancers, and infections in a national health care setting.<sup><a href="#core-r6" role="doc-biblioref" data-xml-rid="r6" id="body-ref-r6" href-manipulated="true" aria-label="Reference 6">6</a></sup></div><div role="paragraph">To assess the effect of the whole-genome–sequencing approach on the genetic diagnosis of rare diseases in the National Health Service (NHS) in the United Kingdom, we carried out a pilot study in which we enrolled families and undertook detailed clinical phenotyping of the proband.<sup><a href="#core-r4" role="doc-biblioref" data-xml-rid="r4" id="body-ref-r4-3" href-manipulated="true" aria-label="Reference 4">4</a></sup> We collected electronic health records from all participants and stored these together with the genomic and clinical data in a computer environment with multi-petabytes of storage (the Genomics England research environment).<sup><a href="#core-r5" role="doc-biblioref" data-xml-rid="r5" id="body-ref-r5-3" href-manipulated="true" aria-label="Reference 5">5</a></sup> When necessary, we validated diagnostic variants in the laboratory and performed computational analyses.</div><section id="sec-1" data-type="methods"><h2>Methods</h2><section id="sec-1-1"><h3>Participants</h3><div role="paragraph">After approval from the national research ethics committee was obtained, we recruited participants who had been identified by health care professionals and researchers as having rare diseases (across a broad range of categories) that had not been diagnosed after receipt of usual care in the NHS, which included either no diagnostic tests (because none were available) or approved diagnostic tests that did not include genome sequencing. The participants were recruited at nine English hospitals, and written informed consent was obtained from the participants by the National Institute for Health Research (NIHR) BioResource for Rare Diseases.</div><div role="paragraph">To test the broad applicability of genome sequencing, we determined that participants were eligible if they had a rare disease (as defined in the United Kingdom as a disorder affecting ≤1 in 2000 persons), were likely to have a single-gene or oligogenic cause, and had not received a genomic diagnosis. Data on previous testing in probands were collected when possible; testing included single-gene tests, karyotyping, single-nucleotide polymorphism arrays, next-generation sequencing panels, and exome sequencing. Probands and, when feasible, parents or other family members were enrolled across multiple clinical specialties in the NHS. Standardized baseline clinical data were recorded with the use of Human Phenotype Ontology (HPO) terms<sup><a href="#core-r7" role="doc-biblioref" data-xml-rid="r7" id="body-ref-r7" href-manipulated="true" aria-label="Reference 7">7</a></sup> guided by disease-specific data models,<sup><a href="#core-r8" role="doc-biblioref" data-xml-rid="r8" id="body-ref-r8" href-manipulated="true" aria-label="Reference 8">8</a></sup> and whole blood samples were obtained for DNA extraction. In the 100,000 Genomes Project, participants are followed over their life course with the use of electronic health records (all hospital episodes, registry entries, and cause of death).</div><div role="paragraph">This pilot study was undertaken in partnership with the NIHR BioResource and is part of the portfolio of translational research at the NIHR Biomedical Research Centres at Barts, Cambridge University Hospitals NHS Foundation Trust, Great Ormond Street Hospital for Children NHS Foundation Trust, Manchester University NHS Foundation Trust, Moorfields Eye Hospital NHS Foundation Trust, Newcastle upon Tyne Hospitals NHS Foundation Trust, Oxford University Hospitals NHS Foundation Trust, and University College London Hospitals NHS Foundation Trust. Clinical data from the NHS and NHS Digital were used in this work.</div></section><section id="sec-1-2"><h3>Genome Sequencing</h3><div role="paragraph">Genome sequencing<sup><a href="#core-r9" role="doc-biblioref" data-xml-rid="r9" id="body-ref-r9" href-manipulated="true" aria-label="Reference 9">9</a></sup> was performed with the use of the TruSeq DNA polymerase-chain-reaction (PCR)–free sample preparation kit (Illumina) on a HiSeq 2500 sequencer, which generates a mean depth of 32× (range, 27 to 54) and a depth greater than 15× for at least 95% of the reference human genome. Whole-genome sequencing reads were aligned to the Genome Reference Consortium human genome build 37 (GRCh37) with the use of Isaac Genome Alignment Software. Family-based variant calling of single-nucleotide variants (SNVs) and insertion or deletions (indels) for chromosomes 1 to 22, the X chromosome, and the mitochondrial genome (mean coverage, 2814×; range, 142 to 16,581) was performed with the use of the Platypus variant caller.<sup><a href="#core-r10" role="doc-biblioref" data-xml-rid="r10" id="body-ref-r10" href-manipulated="true" aria-label="Reference 10">10</a></sup></div></section><section id="sec-1-3"><h3>Diagnostic Pipeline</h3><div role="paragraph">We constructed an automated analytic pipeline to filter the genome down to rare, segregating, and predicted damaging candidate variants in coding regions. To limit the possibility of overlooking or inefficiently prioritizing diagnoses, we focused initially on applied virtual gene panels (applied panels) that were based on both the recruited clinical indication or disease and the submitted HPO terms. To address the issue of which genes have sufficient evidence to show causation and be included in these applied panels, we used our PanelApp software to enable expert, crowd-sourced review and curation of genes with diagnostic-grade evidence for each of our disease categories (e.g., evidence in at least three unrelated families).<sup><a href="#core-r11" role="doc-biblioref" data-xml-rid="r11" id="body-ref-r11" href-manipulated="true" aria-label="Reference 11">11</a></sup> Loss-of-function or de novo protein-altering variants affecting genes in the applied panels were classified as tier 1, other variant types such as missense variants affecting these genes were classified as tier 2, and all other filtered variants were classified as tier 3 (Fig. S1 in the <a href="#ap1">Supplementary Appendix</a>, available with the full text of this article at NEJM.org). To further reduce the possibility of missing or inefficiently prioritized diagnoses, we used a phenotype-based approach with the Exomiser application<sup><a href="#core-r12" role="doc-biblioref" data-xml-rid="r12" id="body-ref-r12" href-manipulated="true" aria-label="Reference 12">12</a></sup> to search across all genes in the genome for a diagnosis. Exomiser prioritizes rare, segregating, and predicted pathogenic variants in genes in which the patient phenotypes match previously referenced knowledge from human disease or model organism databases. The ontology-driven phenotype matching can identify patients who have an atypical profile for a disease. Additional details regarding the Exomiser are provided in the Diagnostic Pipeline section in the <a href="#ap1">Supplementary Appendix</a>.</div><div role="paragraph">Prioritization of variants and return of candidate variants for presentation to the 13 NHS Genomic Medicine Centres (GMCs) were performed with the use of decision-support systems and with assistance from clinical genetics teams from Congenica and Fabric Genomics.<sup><a href="#core-r13" role="doc-biblioref" data-xml-rid="r13 r14" id="body-ref-r14" href-manipulated="true">13,14</a></sup> These variants were reviewed by NHS clinical scientists and clinicians using the guidelines of the American College of Medical Genetics and Genomics, and a diagnostic report was issued for each proband.<sup><a href="#core-r15" role="doc-biblioref" data-xml-rid="r15" id="body-ref-r15-1" href-manipulated="true" aria-label="Reference 15">15</a></sup> Final clinical outcomes included whether a genetic diagnosis was obtained, identification of the variant or variants involved, whether the variant or variants explained all or some of the phenotypes, and whether an intervention was used.</div><div role="paragraph">Recruitment of the participants in the pilot study and sequencing were performed during the period from January 2014 through December 2016, while the infrastructure to collect, quality check, process, and return data was being established. Results were returned to the GMCs from May 2016 through April 2019. Now that the information pipeline has been established (post-pilot phase), results are returned to the GMCs within 6 weeks after the sample is obtained.</div></section><section id="sec-1-4"><h3>Novel Pathogenic Variants</h3><div role="paragraph">Researchers investigated coding and noncoding regions to detect novel diagnostic variants in genes matching the patients’ phenotypes, including the presence of de novo variants in highly constrained coding regions<sup><a href="#core-r16" role="doc-biblioref" data-xml-rid="r16" id="body-ref-r16-1" href-manipulated="true" aria-label="Reference 16">16</a></sup> in the 95th percentile. We use the term novel to describe diagnostic variants we have detected that have not previously been described in the literature as causative. This is distinct from de novo variants, which are present for the first time in a family member due to either a new variant in an egg or sperm or a new mutation at conception. The variant may have been previously described. We used a new method described by Wei et al.<sup><a href="#core-r17" role="doc-biblioref" data-xml-rid="r17" id="body-ref-r17" href-manipulated="true" aria-label="Reference 17">17</a></sup> to analyze mitochondrial DNA that accounts for heteroplasmy, the Genomiser to detect noncoding pathogenic variants,<sup><a href="#core-r18" role="doc-biblioref" data-xml-rid="r18" id="body-ref-r18" href-manipulated="true" aria-label="Reference 18">18</a></sup> and the ExpansionHunter software tool to detect simple tandem repeat expansions.<sup><a href="#core-r19" role="doc-biblioref" data-xml-rid="r19" id="body-ref-r19" href-manipulated="true" aria-label="Reference 19">19</a></sup> Finally we used a new random forest method to analyze Canvas<sup><a href="#core-r20" role="doc-biblioref" data-xml-rid="r20" id="body-ref-r20" href-manipulated="true" aria-label="Reference 20">20</a></sup> and Manta<sup><a href="#core-r21" role="doc-biblioref" data-xml-rid="r21" id="body-ref-r21" href-manipulated="true" aria-label="Reference 21">21</a></sup> calls and to identify potentially pathogenic copy-number and structural variants.</div><div role="paragraph">Gene-based burden testing to detect enrichment of rare, predicted pathogenic, and segregating variants in novel genes in specific disease cohorts relative to controls was performed on the genomes in the pilot study as well as on additional genomes from the rest of the 100,000 Genomes Project to increase power (57,002 genomes; see the Supplementary Methods in the <a href="#ap1">Supplementary Appendix</a>). The genomic and clinical data from the pilot study are freely accessible to members of a Genomics England Clinical Interpretation Partnership domain (<a href="https://www.genomicsengland.co.uk/about-gecip/" target="_blank">https://www.genomicsengland.co.uk/about-gecip/</a>).</div></section><section id="sec-1-5"><h3>Statistical Analysis</h3><div role="paragraph">Testing was performed with the use of the R software, version 3.6.0 (R Foundation for Statistical Computing), and Stata software, version 16 (StataCorp). Further details on the individual methods used in the study are provided in the <a href="#ap1">Supplementary Appendix</a>.</div></section></section><section id="sec-2" data-type="results"><h2>Results</h2><section id="sec-2-1"><h3>Participants</h3><div role="paragraph">We enrolled 4660 participants (2183 probands and 2477 family members), among whom 161 disorders across a broad spectrum of rare diseases were present (<a href="#t1">Table 1</a>).<sup><a href="#core-r22" role="doc-biblioref" data-xml-rid="r22" id="body-ref-r22" href-manipulated="true" aria-label="Reference 22">22</a></sup> Neurologic, ophthalmologic, and tumor syndromes were commonly represented (<a href="#t2">Table 2</a>). Participants were recruited with varying numbers of affected and unaffected family members. We aimed to recruit family trios (both parents and a proband) or larger family structures to facilitate more effective variant prioritization, and our efforts were met with varying degrees of success. Among the recruited probands with multiple bowel polyps, 93% were singletons (i.e., probands for whom no other family member was recruited). In contrast, 12% of the probands with intellectual disability were singletons. Adult probands were more commonly enrolled than pediatric probands (age ≤18 years at recruitment) (74% vs. 26%), which is in line with the percentage of children and adults in the general population in England and Wales (79% vs. 21% [2011 census of England and Wales<sup><a href="#core-r23" role="doc-biblioref" data-xml-rid="r23" id="body-ref-r23" href-manipulated="true" aria-label="Reference 23">23</a></sup>]). The preponderance of adults was unusual as compared with previous sequencing projects and reflects the eligibility criterion that probands had to have undergone usual care; in many cases, usual care involved standard genetic testing (mostly single-gene or panel-based). A lower percentage of recruited probands were female than male owing to the difference among pediatric probands (232 girls and female adolescents [11%] vs. 339 boys and male adolescents [16%], P&lt;0.001); the expected percentage of female probands was 51% (on the basis of the 2011 census of England and Wales) across most disease categories. The greater susceptibility of males than of females to recessive X-linked conditions may account for this sex bias: more than 6% of all diagnoses involved variants on the X chromosome (which represents approximately 5% of the genome). The inferred ancestry of the probands (see the <a href="#ap1">Supplementary Appendix</a>) was in line with what was expected on the basis of the general population, in which 86% of children and adults were White, 8% Asian, 3% Black, 2% mixed, and 1% other (2011 census of England and Wales). However, South Asian ancestry was significantly more common among pediatric probands than among adult probands (16% vs. 4%, P&lt;0.001); our results indicated potential consanguinity in 43% of the 93 pediatric South Asian probands and in 1% of the other 478 pediatric probands (<a href="#t1">Table 1</a>).</div><div class="figure-wrap" data-presentation="diminished-in-flow"><header><div class="label">Table 1</div><nav><a href="#t1" title="Open in viewer" class="open-in-viewer" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t1.jpg"><i aria-hidden="true" class="icon-expand"></i></a></nav></header><figure id="t1" class="table"><div class="graphic-wrap" data-presentation="suppressed-in-viewer"><a aria-label="Open figure in viewer" class="open-in-viewer" data-index="0" href="#t1" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t1.jpg"><img src="/cms/10.1056/NEJMoa2035790/asset/21a9ed96-3de9-4dbb-9ceb-e2c4838810e6/assets/images/large/nejmoa2035790_t1.jpg" height="1235" width="1711" loading="lazy"></a></div><div class="table-wrap catalyst-standard" data-presentation="suppressed-in-flow"><table data-shading="custom"><thead><tr class="head1" data-type="row head1"><th class="txlx-borders" data-xml-align="left">Characteristic</th><th class="txxx-borders">All Probands<br>(N=2183)</th><th class="txxx-borders">Pediatric Probands<br>(N=571)</th><th class="txxr-borders">Adult Probands<br>(N=1612)</th></tr></thead><tbody><tr data-type="row"><td class="xxlx-borders hanging02 shading">Sex — no. (%)</td><td class="xxxx-borders shading">&nbsp;</td><td class="xxxx-borders shading">&nbsp;</td><td class="xxxr-borders shading">&nbsp;</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">Male</td><td class="xxxx-borders">1138 (52)</td><td class="xxxx-borders">339 (59)</td><td class="xxxr-borders">799 (50)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12 shading" data-xml-align="left">Female</td><td class="xxxx-borders shading">1045 (48)</td><td class="xxxx-borders shading">232 (41)</td><td class="xxxr-borders shading">813 (50)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">Total</td><td class="xxxx-borders">2183 (100)</td><td class="xxxx-borders">571 (100)</td><td class="xxxr-borders">1612 (100)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging02 shading" data-xml-align="left">Median age at recruitment (IQR) — yr</td><td class="xxxx-borders shading">35 (18–54)</td><td class="xxxx-borders shading">9 (5–14)</td><td class="xxxr-borders shading">45 (31–60)</td></tr><tr data-type="row"><td class="xxlx-borders hanging02">Inferred ancestry — no. (%) [% consanguinity suggested in record]<a href="#t1fn2" role="doc-noteref">†</a></td><td class="xxxx-borders">&nbsp;</td><td class="xxxx-borders">&nbsp;</td><td class="xxxr-borders">&nbsp;</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12 shading" data-xml-align="left">African</td><td class="xxxx-borders shading">50 (2) [0]</td><td class="xxxx-borders shading">25 (4) [0]</td><td class="xxxr-borders shading">25 (2) [0]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">American with mixed ancestry</td><td class="xxxx-borders">26 (1) [23]</td><td class="xxxx-borders">12 (2) [25]</td><td class="xxxr-borders">14 (1) [21]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12 shading" data-xml-align="left">East Asian</td><td class="xxxx-borders shading">8 (&lt;1) [0]</td><td class="xxxx-borders shading">2 (&lt;1) [0]</td><td class="xxxr-borders shading">6 (&lt;1) [0]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">European</td><td class="xxxx-borders">1931 (88) [&lt;1]</td><td class="xxxx-borders">438 (77) [&lt;1]</td><td class="xxxr-borders">1493 (93) [&lt;1]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12 shading" data-xml-align="left">South Asian</td><td class="xxxx-borders shading">163 (7) [36]</td><td class="xxxx-borders shading">93 (16) [43]</td><td class="xxxr-borders shading">70 (4) [25]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">Not determined</td><td class="xxxx-borders">5 (&lt;1) [0]</td><td class="xxxx-borders">1 (&lt;1) [0]</td><td class="xxxr-borders">4 (&lt;1) [0]</td></tr><tr data-xml-align="center" data-type="row"><td class="xblx-borders hanging12 shading" data-xml-align="left">Total</td><td class="xbxx-borders shading">2183 (100) [3]</td><td class="xbxx-borders shading">571 (26) [8]</td><td class="xbxr-borders shading">1612 (74) [2]</td></tr></tbody></table></div><figcaption><div class="caption">Demographic Characteristics of the Probands (Including Inferred Ancestry) in the 100,000 Genomes Project Pilot Study.<a href="#t1fn1" role="doc-noteref">*</a></div><div class="notes"><div role="doc-footnote" data-has="label"><div class="label">*</div><div id="t1fn1" role="paragraph">Pediatric probands were younger than 18 years of age at recruitment, and adult probands were 18 years of age or older at recruitment. IQR denotes interquartile range.</div></div><div role="doc-footnote" data-has="label"><div class="label">†</div><div id="t1fn2" role="paragraph">Ancestry was inferred on the basis of a random forest model trained on principal component analysis of the five super populations described for the 1000 Genomes Project.</div></div></div></figcaption></figure></div><div class="figure-wrap" data-presentation="diminished-in-flow"><header><div class="label">Table 2</div><nav><a href="#t2" title="Open in viewer" class="open-in-viewer" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t2.jpg"><i aria-hidden="true" class="icon-expand"></i></a></nav></header><figure id="t2" class="table"><div class="graphic-wrap" data-presentation="suppressed-in-viewer"><a aria-label="Open figure in viewer" class="open-in-viewer" data-index="0" href="#t2" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t2.jpg"><img src="/cms/10.1056/NEJMoa2035790/asset/9de2c9af-b71b-43a6-8296-12f7e7d69943/assets/images/large/nejmoa2035790_t2.jpg" height="1694" width="1999" loading="lazy"></a></div><div class="table-wrap catalyst-standard" data-presentation="suppressed-in-flow"><table data-shading="custom"><thead><tr class="head1" data-type="row head1"><th class="txlx-borders" data-xml-align="left">Disease Category</th><th class="txxx-borders">All Family Structures<br>(2183 Probands)</th><th class="txxx-borders">Singleton<br>(881 Probands)</th><th class="txxx-borders">Family Duo<br>(343 Probands)</th><th class="txxx-borders">Family Trio<br>(797 Probands)</th><th class="txxr-borders">Larger Family Structure<br>(162 Probands)</th></tr><tr class="head3" data-type="row head3"><td class="xxlx-borders">&nbsp;</td><th class="xxxr-borders" colspan="5"><span>number of probands (percent)</span></th></tr></thead><tbody><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Cardiovascular disorder</td><td class="xxxx-borders shading">147 (7)</td><td class="xxxx-borders shading">56 (6)</td><td class="xxxx-borders shading">24 (7)</td><td class="xxxx-borders shading">49 (6)</td><td class="xxxr-borders shading">18 (11)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Ciliopathy</td><td class="xxxx-borders">69 (3)</td><td class="xxxx-borders">34 (4)</td><td class="xxxx-borders">14 (4)</td><td class="xxxx-borders">16 (2)</td><td class="xxxr-borders">5 (3)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Dermatologic disorder</td><td class="xxxx-borders shading">38 (2)</td><td class="xxxx-borders shading">9 (1)</td><td class="xxxx-borders shading">5 (1)</td><td class="xxxx-borders shading">22 (3)</td><td class="xxxr-borders shading">2 (1)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Dysmorphic or congenital abnormality</td><td class="xxxx-borders">20 (1)</td><td class="xxxx-borders">10 (1)</td><td class="xxxx-borders">2 (1)</td><td class="xxxx-borders">7 (1)</td><td class="xxxr-borders">1 (1)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Endocrine disorder</td><td class="xxxx-borders shading">87 (4)</td><td class="xxxx-borders shading">57 (6)</td><td class="xxxx-borders shading">14 (4)</td><td class="xxxx-borders shading">12 (2)</td><td class="xxxr-borders shading">4 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Gastroenterologic disorder</td><td class="xxxx-borders">32 (1)</td><td class="xxxx-borders">0</td><td class="xxxx-borders">0</td><td class="xxxx-borders">18 (2)</td><td class="xxxr-borders">14 (9)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Growth disorder</td><td class="xxxx-borders shading">3 (&lt;1)</td><td class="xxxx-borders shading">3 (&lt;1)</td><td class="xxxx-borders shading">0</td><td class="xxxx-borders shading">0</td><td class="xxxr-borders shading">0</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Hematologic or immunologic disorder</td><td class="xxxx-borders">5 (&lt;1)</td><td class="xxxx-borders">2 (&lt;1)</td><td class="xxxx-borders">3 (1)</td><td class="xxxx-borders">0</td><td class="xxxr-borders">0</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Hematologic disorder</td><td class="xxxx-borders shading">7 (&lt;1)</td><td class="xxxx-borders shading">0</td><td class="xxxx-borders shading">3 (1)</td><td class="xxxx-borders shading">2 (&lt;1)</td><td class="xxxr-borders shading">2 (1)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Hearing or ear disorder</td><td class="xxxx-borders">35 (2)</td><td class="xxxx-borders">6 (1)</td><td class="xxxx-borders">5 (1)</td><td class="xxxx-borders">17 (2)</td><td class="xxxr-borders">7 (4)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Metabolic disorder</td><td class="xxxx-borders shading">93 (4)</td><td class="xxxx-borders shading">24 (3)</td><td class="xxxx-borders shading">12 (3)</td><td class="xxxx-borders shading">48 (6)</td><td class="xxxr-borders shading">9 (6)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Intellectual disability</td><td class="xxxx-borders">130 (6)</td><td class="xxxx-borders">10 (1)</td><td class="xxxx-borders">24 (7)</td><td class="xxxx-borders">78 (10)</td><td class="xxxr-borders">18 (11)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Neurologic or neurodevelopmental disorder<a href="#t2fn2" role="doc-noteref">†</a></td><td class="xxxx-borders shading">521 (24)</td><td class="xxxx-borders shading">193 (22)</td><td class="xxxx-borders shading">93 (27)</td><td class="xxxx-borders shading">194 (24)</td><td class="xxxr-borders shading">41 (25)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Ophthalmologic disorder</td><td class="xxxx-borders">348 (16)</td><td class="xxxx-borders">74 (8)</td><td class="xxxx-borders">62 (18)</td><td class="xxxx-borders">199 (25)</td><td class="xxxr-borders">13 (8)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Renal and urinary tract disorder</td><td class="xxxx-borders shading">176 (8)</td><td class="xxxx-borders shading">125 (14)</td><td class="xxxx-borders shading">21 (6)</td><td class="xxxx-borders shading">26 (3)</td><td class="xxxr-borders shading">4 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Respiratory disorder</td><td class="xxxx-borders">2 (&lt;1)</td><td class="xxxx-borders">1 (&lt;1)</td><td class="xxxx-borders">0</td><td class="xxxx-borders">1 (&lt;1)</td><td class="xxxr-borders">0</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Rheumatologic disorder</td><td class="xxxx-borders shading">48 (2)</td><td class="xxxx-borders shading">14 (2)</td><td class="xxxx-borders shading">6 (2)</td><td class="xxxx-borders shading">25 (3)</td><td class="xxxr-borders shading">3 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Skeletal disorder</td><td class="xxxx-borders">62 (3)</td><td class="xxxx-borders">15 (2)</td><td class="xxxx-borders">11 (3)</td><td class="xxxx-borders">23 (3)</td><td class="xxxr-borders">13 (8)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Tumor syndrome</td><td class="xxxx-borders shading">293 (13)</td><td class="xxxx-borders shading">231 (26)</td><td class="xxxx-borders shading">31 (9)</td><td class="xxxx-borders shading">27 (3)</td><td class="xxxr-borders shading">4 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Other</td><td class="xxxx-borders">67 (3)</td><td class="xxxx-borders">17 (2)</td><td class="xxxx-borders">12 (3)</td><td class="xxxx-borders">34 (4)</td><td class="xxxr-borders">4 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xblx-borders hanging01 shading" data-xml-align="left">Total</td><td class="xbxx-borders shading">2183 (100)</td><td class="xbxx-borders shading">881 (100)</td><td class="xbxx-borders shading">343 (100)</td><td class="xbxx-borders shading">797 (100)</td><td class="xbxr-borders shading">162 (100)</td></tr></tbody></table></div><figcaption><div class="caption">Disease Categories among the Probands in the 100,000 Genomes Project Pilot Study.<a href="#t2fn1" role="doc-noteref">*</a></div><div class="notes"><div role="doc-footnote" data-has="label"><div class="label">*</div><div id="t2fn1" role="paragraph">Singleton refers to a proband for whom no other family member was recruited, family duo to a parent–proband pair, family trio to both parents and a proband, and family quad to a proband, sibling, and parents.</div></div><div role="doc-footnote" data-has="label"><div class="label">†</div><div id="t2fn2" role="paragraph">Neurologic and neurodevelopmental symptoms excluded intellectual disability.</div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Clinical Data and Sequencing</h3><div role="paragraph">We collected clinical data with the use of HPO terms for each affected participant (a median of 4 [range, 1 to 61] present terms, and a median of 4 [range, 0 to 144] absent terms [phenotypes that were assessed and confirmed as definitely not observed in the proband]). We then performed genome sequencing, followed by quality assurance to check coverage, sequence quality, presence of repeat sample submissions or sample swaps, and consistency with reported family structures (see the <a href="#ap1">Supplementary Appendix</a>).</div></section><section id="sec-2-3"><h3>Diagnostic Yield</h3><div role="paragraph">We made genetic diagnoses in 25% of the probands and deposited the genotypes into the ClinVar repository (accession numbers, SCV001759972 to SCV001760540). Of these diagnoses, 60% were made on the basis of coding SNVs or indels in the applied panels; 26% were made on the basis of coding SNVs or indels affecting well-established disease genes not included in the applied panels (diagnoses were made through phenotype-based prioritization or expert review by the study clinicians or the clinical genetics teams from Congenica or Fabric Genomics); and 14% were made on the basis of genomewide, phenotype-agnostic research analysis that investigated beyond SNVs and indels, coding regions, and disease genes in the applied panels (<a href="#f1">Figure 1</a>). On the basis of international guidelines,<sup><a href="#core-r15" role="doc-biblioref" data-xml-rid="r15" id="body-ref-r15-2" href-manipulated="true" aria-label="Reference 15">15</a></sup> an additional 10% of the probands were classified as having variants of unknown significance in genes that were considered to be consistent with the phenotype on clinical review at the study site but that required further functional validation. Fewer candidate variants were returned to the GMCs after filtering (i.e., the removal of extremely unlikely candidates) in larger family structures (<a href="#t3">Table 3</a>), which made it easier to identify causative variants and in turn led to higher diagnostic yields for family trios and quads (proband, sibling, and parents) and more complex family structures (<a href="#f2">Figure 2A</a>), even within a disorder (e.g., the diagnostic yield for hereditary ataxia was 21% among singletons and 32% among persons in family trios) (Table S4).</div><div class="figure-wrap" data-presentation="diminished-in-flow"><header><div class="label">Figure 1</div><nav><a href="#f1" title="Open in viewer" class="open-in-viewer" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f1.jpg"><i aria-hidden="true" class="icon-expand"></i></a></nav></header><figure id="f1" class="graphic"><a aria-label="Open figure in viewer" class="open-in-viewer" data-index="0" href="#f1" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f1.jpg"><img src="/cms/10.1056/NEJMoa2035790/asset/ee2525a0-274f-49b3-a2ee-6a621a475e47/assets/images/large/nejmoa2035790_f1.jpg" height="2845" width="2640" aria-labelledby="f1" loading="lazy"></a><figcaption><div class="caption">Overview of the Diagnostic and Research Pipeline and Source of Diagnoses.</div><div class="notes"><div role="doc-footnote">Results from 2183 probands in the pilot study were returned for presentation to the Genomic Medicine Centres (GMCs) of the recruiting hospitals. A total of 25% of the probands received a positive diagnosis, and 10% had a variant or variants of unknown significance in genes that were determined by clinical geneticists at the recruiting site to be consistent with the phenotype but that required further functional validation. The remaining 65% of the probands received a negative report at the time but will be reassessed. The numbers and sources of these positive diagnoses are shown at each stage of the automated diagnostic pipeline, and the additional research is shown for diagnoses that were not immediately obvious. CCR denotes constrained coding region, indel insertion or deletion, mtDNA mitochondrial DNA, SNV single-nucleotide variant, and SV structural variant.</div></div></figcaption></figure></div><div class="figure-wrap" data-presentation="diminished-in-flow"><header><div class="label">Figure 2</div><nav><a href="#f2" title="Open in viewer" class="open-in-viewer" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f2.jpg"><i aria-hidden="true" class="icon-expand"></i></a></nav></header><figure id="f2" class="graphic"><a aria-label="Open figure in viewer" class="open-in-viewer" data-index="0" href="#f2" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f2.jpg"><img src="/cms/10.1056/NEJMoa2035790/asset/6ac3b65b-4124-4ff1-b695-0053a7cde6de/assets/images/large/nejmoa2035790_f2.jpg" height="3431" width="2640" aria-labelledby="f2" loading="lazy"></a><figcaption><div class="caption">Diagnoses in the Rare Disease Pilot Study.</div><div class="notes"><div role="doc-footnote">Panel A shows diagnostic yield for any disease and according to family structure and cause of disease. The diagnostic yield was 35% for diseases likely to have a monogenic cause and 11% for diseases likely to have a complex cause. The values above the bars are the numbers of probands. Singleton refers to a proband for whom no other family member was recruited, family duo to a parent–proband pair, family trio to both parents and a proband, and family quad to a proband, sibling, and parents. Panel B shows diagnostic yield according to disease category. The values above the bars are the numbers of probands. Panel C shows the diagnostic yield among probands according to previous genetic testing and most extensive testing type: chromosomal (karyotyping, array-based comparative genomic hybridization, single-nucleotide polymorphism arrays), targeted (targeted single-gene tests), next-generation sequencing (NGS) panels, or whole-exome sequencing (WES). The values above the bars are the numbers of probands. Panel D shows the performance of virtual panel-based and Exomiser-based prioritization for identifying the diagnoses. “Disease panel only” indicates the use of a single virtual panel for the recruited disease category. “Applied panels” indicates the use of all applied virtual gene panels used in the pipeline, including the recruited disease–associated panel as well as 0 or more additional panels selected on the basis of the patient’s phenotypes (Human Phenotype Ontology terms). “Exomiser top” indicates Exomiser use in the top-ranked candidate variants, “Exomiser top 3” use in the top three candidates, and “Exomiser top 5” use in the top five candidates. Sensitivity is the percentage of true positive diagnoses based on SNVs or indels that were identified, and the positive predictive value is the percentage of prioritized variants that led to a positive diagnosis. The values above the bars are percentages. In this analysis, the diagnosed variant or variants are true positives, and the other candidate variants that were returned are false positives.</div></div></figcaption></figure></div><div class="figure-wrap" data-presentation="diminished-in-flow"><header><div class="label">Table 3</div><nav><a href="#t3" title="Open in viewer" class="open-in-viewer" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t3.jpg"><i aria-hidden="true" class="icon-expand"></i></a></nav></header><figure id="t3" class="table"><div class="graphic-wrap" data-presentation="suppressed-in-viewer"><a aria-label="Open figure in viewer" class="open-in-viewer" data-index="0" href="#t3" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t3.jpg"><img src="/cms/10.1056/NEJMoa2035790/asset/4310832b-c338-4646-a091-5390e21fe9b0/assets/images/large/nejmoa2035790_t3.jpg" height="751" width="1711" loading="lazy"></a></div><div class="table-wrap catalyst-standard" data-presentation="suppressed-in-flow"><table data-shading="custom"><thead><tr class="head1" data-type="row head1"><th class="txlx-borders" data-xml-align="left">Variable</th><th class="txxx-borders">All Family Structures<br>(2183 Probands)</th><th class="txxx-borders">Singleton<br>(881 Probands)</th><th class="txxx-borders">Family Duo<br>(343 Probands)</th><th class="txxx-borders">Family Trio<br>(797 Probands)</th><th class="txxr-borders">Larger Family Structure<br>(162 Probands)</th></tr></thead><tbody><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Median no. of variants returned after filtering (IQR)<a href="#t3fn2" role="doc-noteref">†</a></td><td class="xxxx-borders shading">221 (49–288)</td><td class="xxxx-borders shading">292 (258–327)</td><td class="xxxx-borders shading">149 (117–213)</td><td class="xxxx-borders shading">29 (17–136)</td><td class="xxxr-borders shading">22 (9–71)</td></tr><tr data-xml-align="center" data-type="row"><td class="xblx-borders hanging01" data-xml-align="left">Median no. of variants in virtual panels returned after filtering (IQR)</td><td class="xbxx-borders">1 (0–2)</td><td class="xbxx-borders">1 (0–2)</td><td class="xbxx-borders">1 (0–3)</td><td class="xbxx-borders">1 (0–2)</td><td class="xbxr-borders">0 (0–1)</td></tr></tbody></table></div><figcaption><div class="caption">Candidate Variants Returned for Presentation to the NHS Genomic Medicine Centres per Proband with the Automated Virtual Panel–Based Analysis Pipeline.<a href="#t3fn1" role="doc-noteref">*</a></div><div class="notes"><div role="doc-footnote" data-has="label"><div class="label">*</div><div id="t3fn1" role="paragraph">Singleton refers to a proband for whom no other family member was recruited, family duo to a parent–proband pair, family trio to both parents and a proband, and family quad to a proband, sibling, and parents.</div></div><div role="doc-footnote" data-has="label"><div class="label">†</div><div id="t3fn2" role="paragraph">Filtering is the removal of extremely unlikely candidate variants.</div></div></div></figcaption></figure></div><div role="paragraph">We obtained a higher diagnostic yield for diseases that we considered likely to have a monogenic cause than those we considered likely to have a complex cause (35% vs. 11%) (<a href="#f2">Figure 2A</a>). Diseases were considered likely to have a monogenic cause if they were present in the Online Mendelian Inheritance in Man database, involved genetic testing as part of the standard diagnostic workup, and had a consensus of opinion among three clinical geneticists (who were unaware of each other’s assessments) that they had monogenic cause. Diagnostic yield was highly varied across diseases (<a href="#f2">Figure 2B</a> and Table S3); the diagnostic yield ranged from 40 to 55% for intellectual disability and various vision and hearing disorders and was 6% for tumor syndromes.</div><div role="paragraph">We obtained data on the presence or absence of previous genetic testing in 1177 participants. The number of tests per proband ranged from 0 to 16, with a median of 1 (interquartile range, 0 to 2), and approximately half the probands in this subgroup had been tested at least once. The overall diagnostic yield with the use of genome sequencing in this subgroup increased by 32%, and there was only a slight difference depending on whether previous testing had been performed (33%) or not (31%). However, many of these previous tests were not recent, dating back to the time of recruitment at the latest (2014 to 2016). The diagnostic yield provided by genome sequencing varied between 28% and 45%, depending on the type of previous testing (<a href="#f2">Figure 2C</a> and Table S5), which for the most part involved targeted single-gene and panel-based testing (Table S6).</div></section><section id="sec-2-4"><h3>Diagnostic Pipeline</h3><div role="paragraph">The aim of the automated diagnostic pipeline is to identify a few potentially causative candidate variants, among the millions in a whole genome, through the removal of extremely unlikely candidates (filtering) and the identification of the most likely candidates in the remainder (prioritization). This approach facilitates manual clinical interpretation and diagnostic reporting by clinicians at the GMCs.</div><div role="paragraph">A total of 322 (66%) of the 490 diagnoses that were based on SNVs or indels from the genomes were made with the virtual panel–based pipeline, and the positive predictive value was high given the millions of variants in the whole genomes — 291 of 1041 candidate variants (28%) returned to the GMCs proved to be diagnostic. We re-ran this analysis in December 2019 to assess the effects of updated versions of the applied panels with the latest disease gene discoveries, improved selection of the applied panel or panels on the basis of the patient’s phenotype, and advances in variant-filtering strategies (e.g., allowance for incomplete penetrance when suspected). With the use of these updated versions, the number of genetic diagnoses increased from 322 to 377 of the 490 diagnoses (77% sensitivity), and the positive predictive value was 15% (<a href="#f2">Figure 2D</a>). This result shows effective filtering and prioritization of the variants, with a median number of only 1 candidate variant (interquartile range, 0 to 2) included in the panels returned to the GMCs per proband (<a href="#t3">Table 3</a>). Ongoing evolution of the applied panels with new disease genes is expected to continue to increase the diagnostic yield with this approach.</div><div role="paragraph">With the use of phenotype-based prioritization with the Exomiser to score and rank the most likely causative variants, diagnoses were detected in 77% of the top-ranked candidate variants, in 86% of the top three candidates, and in 88% of the top five candidates (<a href="#f2">Figure 2D</a>). Use of the Exomiser and applied panels was complementary — 92% of the 490 diagnoses were made with the applied panels or the Exomizer top five candidates (last blue bar in <a href="#f2">Figure 2D</a>). Precision phenotyping in our participants was essential for both the Exomiser and the selection of additional applied panels; without such phenotyping, only 54% of these diagnoses would have been prioritized in the virtual panel for the recruited disease and presented to the GMCs as a likely candidate (first blue bar in <a href="#f2">Figure 2D</a>).</div></section><section id="sec-2-5"><h3>Research-Based Diagnoses</h3><div role="paragraph">A total of 14% of the genetic diagnoses required further research outside the diagnostic pipeline (<a href="#f1">Figure 1</a>). This research involved combined analysis of the genome sequences and clinical data in our research environment and validation with the use of wet-bench orthogonal tests and computational approaches (Table S7). Additional diagnoses were made by screening for the presence of de novo variants in highly constrained coding regions.<sup><a href="#core-r16" role="doc-biblioref" data-xml-rid="r16" id="body-ref-r16-2" href-manipulated="true" aria-label="Reference 16">16</a></sup> These diagnoses included a de novo <i>EBF3</i> missense variant in a patient with hereditary ataxia. A mitochondrial genome analysis that accounted for heteroplasmy led to four new diagnoses, as well as the nine that had already been made by means of the main pipeline. Twelve probands had intronic splicing variants that were prioritized by Exomiser owing to the known pathogenic status of these variants in the ClinVar database.<sup><a href="#core-r24" role="doc-biblioref" data-xml-rid="r24" id="body-ref-r24" href-manipulated="true" aria-label="Reference 24">24</a></sup> Nine diagnoses involving novel, previously undescribed noncoding variants required exploration of the whole genome and in vitro functional validation by means of reverse transcriptase–PCR, minigene, or luciferase assays.<sup><a href="#core-r25" role="doc-biblioref" data-xml-rid="r25 r26 r27" id="body-ref-r27-1" href-manipulated="true">25-27</a></sup> For these diagnoses, unsolved cases in probands had been queried for noncoding variants that affect genes, either alone or in compound heterozygosity with loss-of-function variants, included in the applied panels. These cases were identified with the use of Genomiser or, for probands with retinal disorders, systematic analysis of the untranslated regions, promoter, or introns. The cases in 43 additional probands were fully or partially explained by structural variants or simple tandem repeat expansions in the genes <i>HTT</i> or <i>FXN</i> in the probands with hereditary spastic paraplegia.</div></section><section id="sec-2-6"><h3>New Disease–Gene Associations</h3><div role="paragraph">We performed burden testing to identify new mendelian disease–gene associations and make potential genetic diagnoses in probands with unsolved cases; 828 significant disease–gene associations (Q value of &lt;0.1) were identified, including 249 known and 579 novel genes (novel with respect to their association with disease), with a mean (±SD) number of associations of only 0.03±0.2 (range, 0 to 3) from 10,000 permutations in which the cases and controls were assigned randomly. A total of 22 candidates represent the most probable new, fully penetrant, mendelian disease genes (Table S8; ClinVar accession numbers, SCV001759972 to SCV001760540) with three recently independently confirmed diagnoses: <i>UBAP1</i> in hereditary spastic paraplegia,<sup><a href="#core-r28" role="doc-biblioref" data-xml-rid="r28" id="body-ref-r28" href-manipulated="true" aria-label="Reference 28">28</a></sup> <i>FOXJ1</i> in non–cystic fibrosis bronchiectasis,<sup><a href="#core-r29" role="doc-biblioref" data-xml-rid="r29" id="body-ref-r29" href-manipulated="true" aria-label="Reference 29">29</a></sup> and <i>SORD</i> in Charcot–Marie–Tooth disease.<sup><a href="#core-r30" role="doc-biblioref" data-xml-rid="r30" id="body-ref-r30" href-manipulated="true" aria-label="Reference 30">30</a></sup> Diagnostic reports were issued for three probands with these genes (<a href="#f1">Figure 1</a>), and we are currently investigating others with the use of the online tool GeneMatcher and with functional validation studies in model organisms.</div></section><section id="sec-2-7"><h3>Health Care Outcomes after Diagnosis</h3><div role="paragraph">The findings from our approach ended long diagnostic odysseys for some participants and their families (the median duration of such an odyssey was 75 months, and the median number of hospital visits was 68) (Table S1), and we speculate that they will mitigate NHS resource costs (the combined cost for 183,273 episodes of hospital care among the affected participants was £87 million [$122 million]) (Table S3). In addition, 134 of the 533 genetic diagnoses (25%) were reported by clinicians to be of immediate clinical actionability — only 11 (0.2%) were described as having no benefit. As of now, the remainder of the diagnoses are of unknown usefulness. The benefits in terms of health care included 4 diagnoses that led to a suggested change in medication, 26 that led to suggested additional surveillance of the proband or relatives, 13 that allowed for clinical trial eligibility, 59 that informed future reproductive choices, and 32 that had other benefits (Table S9).</div><div role="paragraph">In several specific probands, diagnoses have had important clinical actionability. In a 36-year-old man with suspected choroideremia, we detected a novel <i>CHM</i> promoter variant causing loss of gene expression,<sup><a href="#core-r27" role="doc-biblioref" data-xml-rid="r27" id="body-ref-r27-2" href-manipulated="true" aria-label="Reference 27">27</a></sup> a diagnosis that enabled eligibility for a gene-replacement trial. A male neonate proband presented with severe infection and transient neurologic symptoms immediately after birth and died at 4 months of age with no diagnosis but with health care costs of approximately £80,000 ($112,000) (Table S10). A diagnosis of transcobalamin II deficiency due to a homozygous frameshift in <i>TCN2</i> was made from this study, which enabled predictive testing to be offered to the younger brother within 1 week after birth. The younger child, who received a positive result, received weekly hydroxocobalamin injections to prevent metabolic decompensation.</div><div role="paragraph">A 10-year-old girl was admitted to the intensive care unit with life-threatening chicken pox. She had undergone a diagnostic odyssey over a period of 7 years at a total cost of £356,571 ($499,199) across 307 secondary care episodes (Table S11). We were able to diagnose <i>CTPS1</i> deficiency due to a homozygous, known pathogenic splice acceptor variant. A diagnosis enabled a curative bone marrow transplantation (cost of £70,000 [$98,000]), and predictive testing in her siblings showed no additional family members to be at risk.</div><div role="paragraph">One proband had waited until his sixth decade of life for a genomic diagnosis of an <i>INF2</i> mutation causing focal segmental glomerulosclerosis. His father, brother, and uncle had all died from kidney failure. He had received two kidney transplants, had transmitted the condition to his daughter, and was concerned about whether his 15-year-old granddaughter, who was under surveillance, was at risk. After he received his genetic diagnosis, the granddaughter was tested, found to be negative, and discharged from regular medical surveillance.</div></section></section><section id="sec-3" data-type="discussion"><h2>Discussion</h2><div role="paragraph">Our findings show a substantial increase in yield of genomic diagnoses made in patients with the use of genome sequencing across a broad spectrum of rare disease. The enhanced diagnostic benefit was observed regardless of whether participants had undergone previous genetic testing (diagnostic yields were 31% among those who had undergone testing and 33% among those who had not). In 25% of those who received a genetic diagnosis, there was immediate clinical actionability. The standardization of procedures — from the enrollment of patients to the return of NHS-validated results to clinicians — was critical to our success. For example, the collection of clinical data with the use of disease-specific data models and HPO terms enabled diagnoses, which confirmed the value of standardization with the use of ontology terms and clinical annotation in precision medicine.<sup><a href="#core-r31" role="doc-biblioref" data-xml-rid="r31" id="body-ref-r31" href-manipulated="true" aria-label="Reference 31">31</a></sup> These additional diagnoses, beyond the 264 (49% of total diagnoses) observed with the use of the single-disease virtual panel, came from the use of Exomiser and additional applied panels. The diagnostic discoveries derived by combining research, decision support, and clinical validation and assessment leveraged an additional 72 diagnoses.</div><div role="paragraph">Diagnostic yield was influenced by family structure, and for disorders likely to have mendelian inheritance and a single-gene etiologic factor, our yield increased to 35%: ophthalmologic, metabolic, and neurologic disorders yielded the greatest percentage of diagnoses. The scale of our data set enabled cohortwide burden testing, which identified numerous novel disease–gene associations, including three that have now been confirmed and 19 with compelling evidence that are likely to be confirmed in independent data sets.</div><div role="paragraph">Of the diseases we diagnosed with the use of genome sequencing, 13% were caused by mutations in noncoding sequence or mitochondrial genomes, tandem repeat expansions in persons with Huntington’s disease, and a wide range of structural variants with nucleotide resolution of breakpoints (which were identified with the use of a new random forest method). An additional 2% of the diagnoses involved coding variants in regions of low coverage on exome sequencing. Our results provide new evidence of the value of genome sequencing and mirror the findings in a previous study in which 53% of the participants who received new diagnoses from genome sequencing had previously undergone exome sequencing.<sup><a href="#core-r5" role="doc-biblioref" data-xml-rid="r5" id="body-ref-r5-4" href-manipulated="true" aria-label="Reference 5">5</a></sup></div><div role="paragraph">Previous studies have shown how next-generation sequencing can lead to diagnoses, with yields of 25 to 29% with the use of exome sequencing in persons who had received no previous genetic testing.<sup><a href="#core-r32" role="doc-biblioref" data-xml-rid="r32 r33 r34" id="body-ref-r34" href-manipulated="true">32-34</a></sup> The Undiagnosed Disease Network reported a diagnostic yield of 26% with the use of a mixture of whole-exome and whole-genome sequence analysis in 382 patients,<sup><a href="#core-r5" role="doc-biblioref" data-xml-rid="r5" id="body-ref-r5-5" href-manipulated="true" aria-label="Reference 5">5</a></sup> and another study of genome sequencing showed a yield of 42% among 50 probands with intellectual disability who had previously undergone testing.<sup><a href="#core-r35" role="doc-biblioref" data-xml-rid="r35" id="body-ref-r35" href-manipulated="true" aria-label="Reference 35">35</a></sup> Among probands with a broad range of disorders (161 in total) with an unmet diagnostic need, we obtained results that were similar to those in the previous studies. Our approach is limited to diagnoses that are readily made by means of short-read genome sequencing. Fully phased, long-read sequencing better detects structural variation and delivers sequence information from parts of the genome that are poorly captured by short-read sequencing.<sup><a href="#core-r36" role="doc-biblioref" data-xml-rid="r36" id="body-ref-r36" href-manipulated="true" aria-label="Reference 36">36</a></sup></div><div role="paragraph">The findings from our pilot study support the case for genome sequencing in the diagnosis of certain specific rare diseases in the new NHS National Genomic Test Directory.<sup><a href="#core-r37" role="doc-biblioref" data-xml-rid="r37" id="body-ref-r37" href-manipulated="true" aria-label="Reference 37">37</a></sup> In patients with specific disorders, such as intellectual disability, genome sequencing is now the first-line test in the NHS (Table S12). With a new National Genomic Medicine Service, the NHS in England is in the process of sequencing 500,000 whole genomes in rare disease and cancer in health care. We hope that our findings will assist other health systems in considering the role of genome sequencing in the care of patients with rare diseases.</div></section>



        
        
</div></section><section id="backmatter" data-extent="backmatter" data-location="articleTab_article"><div class="core-container"><section id="backnotes" data-location="notes_article"><h2>Notes</h2><div data-type="disclaimer" role="paragraph">The views expressed are those of the authors and not necessarily those of the National Health Service (NHS), the National Institute for Health Research (NIHR), or the Department of Health and Social Care.</div><div role="paragraph">Supported by the <span class="named-content" data-type="funder">NIHR</span>, the <span class="named-content" data-type="funder">Wellcome</span> Trust, the Medical Research Council (<span class="named-content" data-type="funder">MRC</span>), <span class="named-content" data-type="funder">Cancer Research U.K.</span>, the <span class="named-content" data-type="funder">Department of Health and Social Care</span>, and <span class="named-content" data-type="funder">NHS England</span>. The <span class="named-content" data-type="funder">NIHR</span> BioResource is funded by the <span class="named-content" data-type="funder">NIHR</span>. Drs. Caulfield and Ouwehand are <span class="named-content" data-type="funder">NIHR</span> senior investigators. Dr. Chinnery is a <span class="named-content" data-type="funder">Wellcome</span> Trust Principal Research Fellow (212219/Z/18/Z) and an <span class="named-content" data-type="funder">NIHR</span> Senior Investigator who receives support from the <span class="named-content" data-type="funder">MRC</span> Mitochondrial Biology Unit (MC_UU_00015/9), the <span class="named-content" data-type="funder">MRC</span> International Centre for Genomic Medicine in Neuromuscular Disease (MR/S005021/1), and the <span class="named-content" data-type="funder">NIHR</span> Biomedical Research Centre (BRC). Dr. Wedderburn’s work is supported by grants from Versus Arthritis (21593), the <span class="named-content" data-type="funder">NIHR</span> BRC at Great Ormond Street Hospital, and the <span class="named-content" data-type="funder">MRC</span> (MR/R013926/1). Drs. Smedley, Cacheiro, and Cipriani receive support from the National Institutes of Health (NIH, grant 5-UM1-HG006370). Dr. Smedley’s team that performed much of the analysis was supported by grants from the NIH (1R24<span class="named-content" data-type="funder">OD</span>011883, U54 HG006370, and 1R01HD103805-01). Dr. Arno’s work is supported by a Fight for Sight (United Kingdom) Early Career Investigator Award (5045/46), <span class="named-content" data-type="funder">NIHR</span> BRC at Great Ormond Street Hospital Institute for Child Health, and Moorfields Eye Charity (Stephen and Elizabeth Archer in memory of Marion Woods). The Moorfields–University College London (UCL) Institute of Ophthalmology team is additionally funded by <span class="named-content" data-type="funder">NIHR</span> BRC at Moorfields Eye Hospital and UCL Institute of Ophthalmology.</div><div role="paragraph"><a href="#ap2">Disclosure forms</a> provided by the authors are available with the full text of this article at NEJM.org.</div><div role="paragraph">We thank the personnel at NIHR BioResource for their partnership in this study; all the health care teams at Addenbrooke’s Hospital in Cambridge, Great Ormond Street Hospital NHS Foundation Trust, University College London NHS Foundation Trust, Guy’s and St. Thomas’ Hospital, Barts Health, Oxford University Hospitals NHS Foundation Trust, Manchester University NHS Foundation Trust, and the Newcastle Hospitals NHS Foundation Trust; the NHS patients and their families who made this work possible; all those across the world who have contributed to the PanelApp knowledge base and to the validation and reporting working group (Dr. Dom McMullan, Dr. Helen Firth, Dr. Steve Abbs, and Dr. Sian Ellard) for their role in supporting the development of the bioinformatics pipeline and reporting process; Dr. David Bick and Dr. Gil McVean for providing feedback on our work; Dr. Dame Sue Hill and the team at NHS England for the work to fund and establish the 13 GMCs, which enabled the NHS contribution that included the clinical return of results within the NHS in a standardized and validated format that led to the confirmation of the diagnoses, provided additional information, and led to the patient benefit reported; the Illumina Laboratory Services team at Hinxton for genome sequencing and secondary analysis; and the developers of the Human Phenotype Ontology (Monarch Initiative) and Exomiser (funded by the NIH Office of the Director [1R24OD011883]) for the support that was provided through these resources.</div></section><section id="supplementary-materials" class="core-supplementary-materials"><h2>Supplementary Material</h2><div role="list"><div id="ap1" class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">Supplementary Appendix</span> <span class="core-filename">(nejmoa2035790_appendix.pdf)</span></div><div class="core-link"><ul><li><a href="/doi/suppl/10.1056/NEJMoa2035790/suppl_file/nejmoa2035790_appendix.pdf" download="nejmoa2035790_appendix.pdf" data-interactiontype="multimedia_download" data-multimedia-type="Supplementary Material" data-behavior="trackDownloadEvent" data-multimedia-filename="nejmoa2035790_appendix.pdf" data-doi="10.1056/NEJMoa2035790">Download</a></li><li>985.59 KB</li></ul></div></div><div id="ap2" class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">Disclosure Forms</span> <span class="core-filename">(nejmoa2035790_disclosures.pdf)</span></div><div class="core-link"><ul><li><a href="/doi/suppl/10.1056/NEJMoa2035790/suppl_file/nejmoa2035790_disclosures.pdf" download="nejmoa2035790_disclosures.pdf" data-interactiontype="multimedia_download" data-multimedia-type="Supplementary Material" data-behavior="trackDownloadEvent" data-multimedia-filename="nejmoa2035790_disclosures.pdf" data-doi="10.1056/NEJMoa2035790">Download</a></li><li>2.89 MB</li></ul></div></div></div></section><section id="appendix-1" role="doc-appendix"><h2>Appendix</h2><div data-type="alt-author-list" role="paragraph">The authors’ full names and academic degrees are as follows: Damian Smedley, Ph.D., Katherine R. Smith, Ph.D., Antonio Martin, M.Sc., Ellen A. Thomas, M.D., Ellen M. McDonagh, Ph.D., Valentina Cipriani, Ph.D., Jamie M. Ellingford, Ph.D., Gavin Arno, Ph.D., Arianna Tucci, M.D., Jana Vandrovcova, Ph.D., Georgia Chan, Ph.D., Hywel J. Williams, Ph.D., Thiloka Ratnaike, M.B., B.S., Ph.D., Wei Wei, Ph.D., Kathleen Stirrups, Ph.D., Kristina Ibanez, Ph.D., Loukas Moutsianas, Ph.D., Matthias Wielscher, Ph.D., Anna Need, Ph.D., Michael R. Barnes, Ph.D., Letizia Vestito, M.Sc., James Buchanan, D.Phil., Sarah Wordsworth, Ph.D., Sofie Ashford, B.Sc., Karola Rehmström, Ph.D., Emily Li, Ph.D., Gavin Fuller, M.Med.Sci., Philip Twiss, M.Sc., Olivera Spasic-Boskovic, M.Sc., Sally Halsall, Ph.D., R. Andres Floto, M.D., Ph.D., Kenneth Poole, M.D., Ph.D., Annette Wagner, M.D., Ph.D., Sarju G. Mehta, M.D., Mark Gurnell, M.D., Ph.D., Nigel Burrows, M.D., Roger James, Ph.D., Christopher Penkett, D.Phil., Eleanor Dewhurst, B.A., Stefan Gräf, Ph.D., Rutendo Mapeta, B.Sc., Mary Kasanicki, Ph.D., Andrea Haworth, M.Sc., F.R.C.Path., Helen Savage, M.Sc., Dip.R.C.Path., Melanie Babcock, Ph.D., Martin G. Reese, Ph.D., Mark Bale, Ph.D., Emma Baple, M.B., B.S., Ph.D., Christopher Boustred, Ph.D., Helen Brittain, M.D., Anna de Burca, M.B., B.S., Ph.D., Marta Bleda, Ph.D., Andrew Devereau, B.Sc., Dina Halai, M.Sc., Eik Haraldsdottir, M.Sc., Zerin Hyder, M.D., Dalia Kasperaviciute, Ph.D., Christine Patch, Ph.D., Dimitris Polychronopoulos, Ph.D., Angela Matchan, M.Sc., Razvan Sultana, Ph.D., Mina Ryten, M.D., Ph.D., Ana L.T. Tavares, M.B., B.S., Carolyn Tregidgo, Ph.D., Clare Turnbull, M.D., Ph.D., Matthew Welland, M.Sc., Suzanne Wood, M.Sc., Catherine Snow, Ph.D., Eleanor Williams, Ph.D., Sarah Leigh, Ph.D., Rebecca E. Foulger, Ph.D., Louise C. Daugherty, M.Sc., Olivia Niblock, M.Sc., Ivone U.S. Leong, Ph.D., Caroline F. Wright, Ph.D., Jim Davies, D.Phil., Charles Crichton, B.A., James Welch, B.A., Kerrie Woods, B.A., Lara Abulhoul, M.D., Paul Aurora, M.R.C.P., Ph.D., Detlef Bockenhauer, M.D., Alexander Broomfield, M.D., Maureen A. Cleary, M.D., Tanya Lam, M.B., B.S., M.P.H., Mehul Dattani, F.R.C.P., Emma Footitt, Ph.D., Vijeya Ganesan, M.D., Stephanie Grunewald, M.D., Ph.D., Sandrine Compeyrot-Lacassagne, M.D., Francesco Muntoni, M.D., Clarissa Pilkington, M.B., B.S., Rosaline Quinlivan, M.D., Nikhil Thapar, M.D., Ph.D., Colin Wallis, M.D., Lucy R. Wedderburn, F.R.C.P., Ph.D., Austen Worth, M.D., Teofila Bueser, M.Sc., Cecilia Compton, M.Sc., Charu Deshpande, M.R.C.P.C.H., Hiva Fassihi, F.R.C.P., Eshika Haque, M.Sc., Louise Izatt, Ph.D., Dragana Josifova, M.D., Shehla Mohammed, F.R.C.P., Leema Robert, M.R.C.P.C.H., Sarah Rose, M.Sc., Deborah Ruddy, Ph.D., Robert Sarkany, F.R.C.P., Genevieve Say, M.Sc., Adam C. Shaw, M.D., Agata Wolejko, M.Sc., Bishoy Habib, B.Sc., Gavin Burns, Ph.D., Sarah Hunter, M.Sc., Russell J. Grocock, Ph.D., Sean J. Humphray, B.Sc., Peter N. Robinson, M.D., Melissa Haendel, Ph.D., Michael A. Simpson, Ph.D., Siddharth Banka, M.D., Ph.D., Jill Clayton-Smith, F.R.C.P., Sofia Douzgou, F.R.C.P., Ph.D., Georgina Hall, M.Sc., Huw B. Thomas, Ph.D., Raymond T. O’Keefe, Ph.D., Michel Michaelides, F.R.C.Ophth., Anthony T. Moore, F.R.C.Ophth., Sam Malka, B.Sc., Nikolas Pontikos, Ph.D., Andrew C. Browning, M.D., Ph.D., Volker Straub, M.D., Ph.D., Gráinne S. Gorman, F.R.C.P., Ph.D., Rita Horvath, M.D., Ph.D., Richard Quinton, M.D., Andrew M. Schaefer, M.R.C.P., Patrick Yu-Wai-Man, F.R.C.Ophth., Ph.D., Doug M. Turnbull, F.Med.Sci., F.R.S., Robert McFarland, M.R.C.P.C.H., Ph.D., Robert W. Taylor, F.R.C.Path., Ph.D., Emer O’Connor, M.D., Janice Yip, M.Res., Katrina Newland, M.Sc., Huw R. Morris, F.R.C.P., Ph.D., James Polke, F.R.C.Path., Ph.D., Nicholas W. Wood, Ph.D., F.Med.Sci., Carolyn Campbell, F.R.C.Path., Carme Camps, Ph.D., Kate Gibson, B.Sc., Nils Koelling, Ph.D., Tracy Lester, Ph.D., F.R.C.Path., Andrea H. Németh, F.R.C.P., D.Phil., Claire Palles, Ph.D., Smita Patel, F.R.C.P., F.R.C.Path., Ph.D., Noemi B.A. Roy, F.R.C.Path., D.Phil., Arjune Sen, M.R.C.P., Ph.D., John Taylor, Ph.D., Pilar Cacheiro, Ph.D., Julius O. Jacobsen, Ph.D., Eleanor G. Seaby, M.D., Val Davison, F.R.C.Path., Lyn Chitty, Ph.D., M.R.C.O.G., Angela Douglas, Ph.D., F.R.C.Path., Kikkeri Naresh, F.R.C.Path., Dom McMullan, Ph.D., F.R.C.Path., Sian Ellard, Ph.D., F.R.C.Path., I. Karen Temple, Ph.D., F.R.C.Path., Andrew D. Mumford, Ph.D., F.R.C.Path., Gill Wilson, F.R.C.P., Phil Beales, F.Med.Sci., Maria Bitner-Glindzicz, M.B., B.S., Ph.D. (deceased), Graeme Black, M.D., D.Phil., John R. Bradley, D.M., Paul Brennan, F.R.C.P., John Burn, M.B., B.S., Ph.D., Patrick F. Chinnery, F.Med.Sci., Perry Elliott, M.D., Frances Flinter, M.D., Henry Houlden, M.D., Melita Irving, M.D., William Newman, M.D., Ph.D., Shamima Rahman, F.R.C.P., F.R.C.P.C.H., Ph.D., John A. Sayer, M.B., Ch.B., Ph.D., Jenny C. Taylor, Ph.D., Andrew R. Webster, F.R.C.Ophth., Andrew O.M. Wilkie, F.Med.Sci., F.R.S., Willem H. Ouwehand, F.Med.Sci., F. Lucy Raymond, M.D., Ph.D., John Chisholm, F.R.Eng., Sue Hill, Ph.D., David Bentley, D.Phil., Richard H. Scott, M.D., Ph.D., Tom Fowler, Ph.D., Augusto Rendon, Ph.D., and Mark Caulfield, F.R.C.P., F.Med.Sci.</div><div data-type="alt-author-affiliations" role="paragraph">The authors’ affiliations are as follows: Genomics England (D.S., K.R.S., A.M., E.A.T., E.M.M., A.T., G.C., K.I., L.M., M. Wielscher, A.N., M. Bale, E.B., C.B., H.B., M. Bleda, A. Devereau, D.H., E. Haraldsdottir, Z.H., D.K., C. Patch, D.P., A.M., R. Sultana, M.R., A.L.T.T., C. Tregidgo, C. Turnbull, M. Welland, S. Wood, C.S., E.W., S.L., R.E.F., L.C.D., O.N., I.U.S.L., C.F.W., J.C., R.H.S., T.F., A.R., M.C.), the William Harvey Research Institute, Queen Mary University of London (D.S., K.R.S., V.C., A.T., L.M., M.R.B., D.K., S. Wood, P.C., J.O.J., T.F., M.C.), University College London (UCL) Institute of Ophthalmology (V.C., G.A., M.M., A.T.M., S. Malka, N.P., P.Y.-W.-M., A.R.W.), UCL Genetics Institute (V.C., N.W.W.), GOSgene (H.J.W.), Genetics and Genomic Medicine Programme (L.V., M.R., M.D., L.C., P. Beales, M.B.-G.), National Institute for Health Research (NIHR) Great Ormond Street Hospital Biomedical Research Centre (BRC) (M.R., S. Grunewald, S.C.-L., F.M., C. Pilkington, L.R.W., L.C., P. Beales, M.B.-G.), Infection, Immunity, and Inflammation Research and Teaching Department (P.A., L.R.W.), Stem Cells and Regenerative Medicine (N.T.), and Mitochondrial Research Group (S. Rahman), UCL Great Ormond Street Institute of Child Health, UCL Ear Institute (L.V.), the Department of Renal Medicine (D. Bockenhauer), and Institute of Cardiovascular Science (P.E.), UCL, Moorfields Eye Hospital National Health Service (NHS) Foundation Trust (V.C., G.A., M.M., A.T.M., S. Malka, N.P., A.R.W.), the National Hospital for Neurology and Neurosurgery (J.V., E.O., J.Y., K. Newland, H.R.M., J.P., N.W.W., H.H.), the Metabolic Unit (L.A., S. Grunewald, S. Rahman), London Centre for Paediatric Endocrinology and Diabetes (M.D.), and the Department of Gastroenterology (N.T.), Great Ormond Street Hospital for Children NHS Foundation Trust (L.V., D. Bockenhauer, A. Broomfield, M.A.C., T. Lam, E.F., V.G., S.C.-L., F.M., C. Pilkington, R. Quinlivan, C.W., L.R.W., A. Worth, L.C., P. Beales, M.B.-G., R.H.S.), the Clinical Genetics Department (M.R., T.B., C. Compton, C.D., E. Haque, L.I., D.J., S. Mohammed, L.R., S. Rose, D.R., G.S., A.C.S., F.F., M.I.) and St. John’s Institute of Dermatology (H.F., R. Sarkany), Guy’s and St. Thomas’ NHS Foundation Trust, the Division of Genetics and Epidemiology, Institute of Cancer Research (C. Turnbull), Florence Nightingale Faculty of Nursing, Midwifery, and Palliative Care (T.B.), Division of Genetics and Molecular Medicine (M.A.S.), and Division of Medical and Molecular Genetics (M.I.), King’s College London, NIHR BRC at Moorfields Eye Hospital (P.Y.-W.-M.), NHS England and NHS Improvement, Skipton House (V.D., A. Douglas, S. Hill), and Imperial College Healthcare NHS Trust, Hammersmith Hospital (K. Naresh), London, Open Targets and European Molecular Biology Laboratory–European Bioinformatics Institute, Wellcome Genome Campus, Hinxton (E.M.M.), the Division of Evolution and Genomic Sciences, Faculty of Biology, Medicine, and Health, University of Manchester (J.M.E., S.B., J.C.-S., S.D., G.H., H.B.T., R.T.O., G. Black, W.N.), and the Manchester Centre for Genomic Medicine, St. Mary’s Hospital, Manchester University NHS Foundation Trust (J.M.E., Z.H., S.B., J.C.-S., S.D., G.H., G. Black, W.N.), Manchester, the Department of Genetic and Genomic Medicine, Institute of Medical Genetics, Cardiff University, Cardiff (H.J.W.), the Department of Clinical Neurosciences (T.R., W.W., R.H., P.F.C.), the Medical Research Council (MRC) Mitochondrial Biology Unit (T.R., W.W., P.Y.-W.-M., P.F.C.), the Department of Paediatrics (T.R.), the Department of Haematology (K.S., C. Penkett, S. Gräf, R.M., W.H.O., A.R.), the School of Clinical Medicine (K.R., E.L., R.A.F., K.P., F.L.R.), the Department of Medicine (S. Gräf), and Cambridge Centre for Brain Repair, Department of Clinical Neurosciences (P.Y.-W.-M.), University of Cambridge, NIHR BioResource, Cambridge University Hospitals (K.S., S.A., R.J., C. Penkett, E.D., S. Gräf, R.M., M.K., J.R.B., P.F.C., W.H.O., F.L.R.), and Addenbrooke’s Hospital, Cambridge University Hospitals NHS Foundation Trust (G.F., P.T., O.S.-B., S. Halsall, K.P., A. Wagner, S.G.M., N.B., M.K.), Cambridge Biomedical Campus, Wellcome–MRC Institute of Metabolic Science and NIHR Cambridge BRC (M.G.), Congenica (A.H., H.S.), Illumina Cambridge (A. Wolejko, B.H., G. Burns, S. Hunter, R.J.G., S.J.H., D. Bentley), NHS Blood and Transplant (W.H.O.), and Wellcome Sanger Institute (W.H.O.), Cambridge, the Health Economics Research Centre (J. Buchanan, S. Wordsworth) and the Wellcome Centre for Human Genetics (C. Camps, J.C.T.), University of Oxford, NIHR Oxford BRC (J. Buchanan, S. Wordsworth, J.D., C. Crichton, J.W., K.W., C. Camps, S.P., N.B.A.R., A.S., J.T., J.C.T.), the Oxford Centre for Genomic Medicine (A. de Burca, A.H.N.), and the Departments of Haematology (N.B.A.R.) and Neurology (A.S.), Oxford University Hospitals NHS Foundation Trust, Oxford Genetics Laboratories, Oxford University Hospitals NHS Foundation Trust, Churchill Hospital (C. Campbell, K.G., T. Lester, J.T.), the MRC Weatherall Institute of Molecular Medicine (N.K., N.B.A.R., A.O.M.W.) and the Oxford Epilepsy Research Group (A.S.), Nuffield Department of Clinical Neurosciences (A.H.N.), University of Oxford, and the Department of Clinical Immunology (S.P.), John Radcliffe Hospital, Oxford, Peninsula Clinical Genetics Service, Royal Devon and Exeter NHS Foundation Trust (E.B.), and the University of Exeter Medical School (E.B., C.F.W.), Royal Devon and Exeter Hospital (S.E.), Exeter, Newcastle Eye Centre, Royal Victoria Infirmary (A.C.B.), the Institute of Genetic Medicine, Newcastle University, International Centre for Life (V.S., P. Brennan), Wellcome Centre for Mitochondrial Research, Translational and Clinical Research Institute, Faculty of Medical Sciences, Newcastle University (G.S.G., R.H., A.M.S., D.M.T., R. Quinton, R.M., R.W.T., J.A.S.), Highly Specialised Mitochondrial Service (G.S.G., A.M.S., D.M.T., R.M., R.W.T.) and Northern Genetics Service (J. Burn), Newcastle upon Tyne Hospitals NHS Foundation Trust (J.A.S.), and NIHR Newcastle BRC (G.S.G., D.M.T., J.A.S.), Newcastle upon Tyne, the Institute of Cancer and Genomic Sciences, Institute of Biomedical Research, University of Birmingham (C. Palles), and Birmingham Women’s Hospital (D.M.), Birmingham, the Genomic Informatics Group (E.G.S.), University Hospital Southampton (I.K.T.), and the University of Southampton (I.K.T.), Southampton, Liverpool Women’s NHS Foundation Trust, Liverpool (A. Douglas), the School of Cellular and Molecular Medicine, University of Bristol, Bristol (A.D.M.), and Yorkshire and Humber, Sheffield Children’s Hospital, Sheffield (G.W.) — all in the United Kingdom; Fabric Genomics, Oakland (M. Babcock, M.G.R.), and the Ophthalmology Department, University of California, San Francisco School of Medicine, San Francisco (A.T.M.) — both in California; the Jackson Laboratory for Genomic Medicine, Farmington, CT (P.N.R.); and the Center for Genome Research and Biocomputing, Environmental and Molecular Toxicology, Oregon State University, Corvallis (M.H.).</div></section><section id="bibliography" class="core-reference-list" role="doc-bibliography" data-location="references_article"><h2>References</h2><div role="list" data-method="clamp" id="collapsible-text"><div role="listitem" data-has="label"><div class="label">1.</div><div id="r1" class="citations"><div class="citation"><div class="citation-content">Generation Genome. Annual report of the chief medical officer. 2016 (<a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/631043/CMO_annual_report_generation_genome.pdf">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/631043/CMO_annual_report_generation_genome.pdf</a>).</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r1" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Annual+report+of+the+chief+medical+officer&amp;publication_year=2016" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r1" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] 6% of the population in Western societies. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r1-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] disease die before their fifth birthday. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">2.</div><div id="r2" class="citations"><div class="citation"><div class="citation-content">Ferreira CR. The burden of rare diseases. <em>Am J Med Genet A</em> 2019;179:885-892.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r2"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajmg.a.61124" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30883013/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000468322800003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=The+burden+of+rare+diseases.&amp;publication_year=2019&amp;journal=Am+J+Med+Genet+A&amp;pages=885-892&amp;doi=10.1002%2Fajmg.a.61124&amp;pmid=30883013" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">3.</div><div id="r3" class="citations"><div class="citation"><div class="citation-content">Boycott KM, Rath A, Chong JX, et al. International cooperation to enable the diagnosis of all rare genetic diseases. <em>Am J Hum Genet</em> 2017;100:695-705.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r3" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2017.04.003" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28475856/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000400560000001" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=International+cooperation+to+enable+the+diagnosis+of+all+rare+genetic+diseases.&amp;publication_year=2017&amp;journal=Am+J+Hum+Genet&amp;pages=695-705&amp;doi=10.1016%2Fj.ajhg.2017.04.003&amp;pmid=28475856" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r3" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r5-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] of rare diseases over the past decade. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] after standard diagnostic testing. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">4.</div><div id="r4" class="citations"><div class="citation"><div class="citation-content">Taylor JC, Martin HC, Lise S, et al. Factors influencing success of clinical genome sequencing across a broad spectrum of disorders. <em>Nat Genet</em> 2015;47:717-726.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r4" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3304" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25985138/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000357090300008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Factors+influencing+success+of+clinical+genome+sequencing+across+a+broad+spectrum+of+disorders.&amp;publication_year=2015&amp;journal=Nat+Genet&amp;pages=717-726&amp;doi=10.1038%2Fng.3304&amp;pmid=25985138" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r4" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r5-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] of rare diseases over the past decade. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] after standard diagnostic testing. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r4-3" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] clinical phenotyping of the proband. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">5.</div><div id="r5" class="citations"><div class="citation"><div class="citation-content">Splinter K, Adams DR, Bacino CA, et al. Effect of genetic diagnosis on patients with previously undiagnosed disease. <em>N Engl J Med</em> 2018;379:2131-2139.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r5" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_6_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa2035790&amp;key=10.1056%2FNEJMoa1714458&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30304647/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000451402500008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Effect+of+genetic+diagnosis+on+patients+with+previously+undiagnosed+disease.&amp;publication_year=2018&amp;journal=N+Engl+J+Med&amp;pages=2131-2139&amp;doi=10.1056%2FNEJMoa1714458&amp;pmid=30304647" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r5" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r5-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] of rare diseases over the past decade. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] after standard diagnostic testing. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-3" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] Genomics England research environment). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-4" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>d [...] had previously undergone exome sequencing. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-5" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>e [...] sequence analysis in 382 patients, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">6.</div><div id="r6" class="citations"><div class="citation"><div class="citation-content">Genomics England. <em>The 100,000 Genomes Project protocol.</em> 2017 (<a href="https://figshare.com/articles/journal_contribution/GenomicEnglandProtocol_pdf/4530893/4">https://figshare.com/articles/journal_contribution/GenomicEnglandProtocol_pdf/4530893/4</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r6"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=The+100%2C000+Genomes+Project+protocol.&amp;publication_year=2017" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">7.</div><div id="r7" class="citations"><div class="citation"><div class="citation-content">Köhler S, Carmody L, Vasilevsky N, et al. Expansion of the Human Phenotype Ontology (HPO) knowledge base and resources. <em>Nucleic Acids Res</em> 2019;47:D1018-D1027.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r7"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1105" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30476213/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000462587400141" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Expansion+of+the+Human+Phenotype+Ontology+%28HPO%29+knowledge+base+and+resources.&amp;publication_year=2019&amp;journal=Nucleic+Acids+Res&amp;pages=D1018-D1027&amp;doi=10.1093%2Fnar%2Fgky1105&amp;pmid=30476213" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">8.</div><div id="r8" class="citations"><div class="citation"><div class="citation-content">Genomics England. <em>Rare disease conditions clinical data models</em>. 2018 (<a href="https://www.genomicsengland.co.uk/?wpdmdl=5500">https://www.genomicsengland.co.uk/?wpdmdl=5500</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r8"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Rare+disease+conditions+clinical+data+models&amp;publication_year=2018" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">9.</div><div id="r9" class="citations"><div class="citation"><div class="citation-content">Bentley DR, Balasubramanian S, Swerdlow HP, et al. Accurate whole human genome sequencing using reversible terminator chemistry. <em>Nature</em> 2008;456:53-59.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r9"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature07517" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18987734/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000260674000039" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Accurate+whole+human+genome+sequencing+using+reversible+terminator+chemistry.&amp;publication_year=2008&amp;journal=Nature&amp;pages=53-59&amp;doi=10.1038%2Fnature07517&amp;pmid=18987734" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">10.</div><div id="r10" class="citations"><div class="citation"><div class="citation-content">Rimmer A, Phan H, Mathieson I, et al. Integrating mapping-, assembly- and haplotype-based approaches for calling variants in clinical sequencing applications. <em>Nat Genet</em> 2014;46:912-918.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r10"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3036" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25017105/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000339704400024" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Integrating+mapping-%2C+assembly-+and+haplotype-based+approaches+for+calling+variants+in+clinical+sequencing+applications.&amp;publication_year=2014&amp;journal=Nat+Genet&amp;pages=912-918&amp;doi=10.1038%2Fng.3036&amp;pmid=25017105" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">11.</div><div id="r11" class="citations"><div class="citation"><div class="citation-content">Martin AR, Williams E, Foulger RE, et al. PanelApp crowdsources expert knowledge to establish consensus diagnostic gene panels. <em>Nat Genet</em> 2019;51:1560-1565.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r11"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0528-2" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31676867/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000494714300002" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=PanelApp+crowdsources+expert+knowledge+to+establish+consensus+diagnostic+gene+panels.&amp;publication_year=2019&amp;journal=Nat+Genet&amp;pages=1560-1565&amp;doi=10.1038%2Fs41588-019-0528-2&amp;pmid=31676867" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">12.</div><div id="r12" class="citations"><div class="citation"><div class="citation-content">Smedley D, Jacobsen JOB, Jäger M, et al. Next-generation diagnostics and disease-gene discovery with the Exomiser. <em>Nat Protoc</em> 2015;10:2004-2015.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2015.124" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26562621/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000366122600008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation+diagnostics+and+disease-gene+discovery+with+the+Exomiser.&amp;publication_year=2015&amp;journal=Nat+Protoc&amp;pages=2004-2015&amp;doi=10.1038%2Fnprot.2015.124&amp;pmid=26562621" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">13.</div><div id="r13" class="citations"><div class="citation"><div class="citation-content"><em>Congenica home page</em> (<a href="https://www.congenica.com/platform">https://www.congenica.com/platform</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r14"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Congenica+home+page" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">14.</div><div id="r14" class="citations"><div class="citation"><div class="citation-content"><em>Fabric Genomics home page</em> (<a href="https://fabricgenomics.com/">https://fabricgenomics.com/</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r14"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Fabric+Genomics+home+page" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">15.</div><div id="r15" class="citations"><div class="citation"><div class="citation-content">Richards S, Aziz N, Bale S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. <em>Genet Med</em> 2015;17:405-424.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r15" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2015.30" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25741868/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000354096900013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Standards+and+guidelines+for+the+interpretation+of+sequence+variants%3A+a+joint+consensus+recommendation+of+the+American+College+of+Medical+Genetics+and+Genomics+and+the+Association+for+Molecular+Pathology.&amp;publication_year=2015&amp;journal=Genet+Med&amp;pages=405-424&amp;doi=10.1038%2Fgim.2015.30&amp;pmid=25741868" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r15" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r15-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] report was issued for each proband. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r15-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] On the basis of international guidelines, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">16.</div><div id="r16" class="citations"><div class="citation"><div class="citation-content">Havrilla JM, Pedersen BS, Layer RM, Quinlan AR. A map of constrained coding regions in the human genome. <em>Nat Genet</em> 2019;51:88-95.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r16" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0294-6" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30531870/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000454108800016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+map+of+constrained+coding+regions+in+the+human+genome.&amp;publication_year=2019&amp;journal=Nat+Genet&amp;pages=88-95&amp;doi=10.1038%2Fs41588-018-0294-6&amp;pmid=30531870" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r16" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r16-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] in highly constrained coding regions </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r16-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] in highly constrained coding regions. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">17.</div><div id="r17" class="citations"><div class="citation"><div class="citation-content">Wei W, Tuna S, Keogh MJ, et al. Germline selection shapes human mitochondrial DNA diversity. <em>Science</em> 2019;364(6442):eaau6520-eaau6520.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r17"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau6520" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31123110/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000469296000033" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Germline+selection+shapes+human+mitochondrial+DNA+diversity.&amp;publication_year=2019&amp;journal=Science&amp;pages=eaau6520-eaau6520&amp;doi=10.1126%2Fscience.aau6520&amp;pmid=31123110" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">18.</div><div id="r18" class="citations"><div class="citation"><div class="citation-content">Smedley D, Schubach M, Jacobsen JOB, et al. A whole-genome analysis framework for effective identification of pathogenic regulatory variants in Mendelian disease. <em>Am J Hum Genet</em> 2016;99:595-606.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r18"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2016.07.005" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27569544/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000383114800006" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+whole-genome+analysis+framework+for+effective+identification+of+pathogenic+regulatory+variants+in+Mendelian+disease.&amp;publication_year=2016&amp;journal=Am+J+Hum+Genet&amp;pages=595-606&amp;doi=10.1016%2Fj.ajhg.2016.07.005&amp;pmid=27569544" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">19.</div><div id="r19" class="citations"><div class="citation"><div class="citation-content">Dolzhenko E, van Vugt JJFA, Shaw RJ, et al. Detection of long repeat expansions from PCR-free whole-genome sequence data. <em>Genome Res</em> 2017;27:1895-1903.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r19"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.225672.117" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28887402/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000414165900010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Detection+of+long+repeat+expansions+from+PCR-free+whole-genome+sequence+data.&amp;publication_year=2017&amp;journal=Genome+Res&amp;pages=1895-1903&amp;doi=10.1101%2Fgr.225672.117&amp;pmid=28887402" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">20.</div><div id="r20" class="citations"><div class="citation"><div class="citation-content">Zhang L, Bai W, Yuan N, Du Z. Comprehensively benchmarking applications for detecting copy number variation. <em>PLoS Comput Biol</em> 2019;15(5):e1007069-e1007069.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r20"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1007069" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31136576/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000471040500066" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensively+benchmarking+applications+for+detecting+copy+number+variation.&amp;publication_year=2019&amp;journal=PLoS+Comput+Biol&amp;pages=e1007069-e1007069&amp;doi=10.1371%2Fjournal.pcbi.1007069&amp;pmid=31136576" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">21.</div><div id="r21" class="citations"><div class="citation"><div class="citation-content">Kosugi S, Momozawa Y, Liu X, Terao C, Kubo M, Kamatani Y. Comprehensive evaluation of structural variation detection algorithms for whole genome sequencing. <em>Genome Biol</em> 2019;20:117-117.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r21"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1720-5" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31159850/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000470146500003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive+evaluation+of+structural+variation+detection+algorithms+for+whole+genome+sequencing.&amp;publication_year=2019&amp;journal=Genome+Biol&amp;pages=117-117&amp;doi=10.1186%2Fs13059-019-1720-5&amp;pmid=31159850" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">22.</div><div id="r22" class="citations"><div class="citation"><div class="citation-content">Genomics England. Rare disease conditions eligibility criteria. 2018 (<a href="https://www.genomicsengland.co.uk/wp-content/uploads/2018/06/Rare-Disease-Eligibility-Criteria-v1.9.0-PAR-GUI-058_approved-version-1.pdf">https://www.genomicsengland.co.uk/wp-content/uploads/2018/06/Rare-Disease-Eligibility-Criteria-v1.9.0-PAR-GUI-058_approved-version-1.pdf</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r22"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Rare+disease+conditions+eligibility+criteria&amp;publication_year=2018" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">23.</div><div id="r23" class="citations"><div class="citation"><div class="citation-content">Office for National Statistics. <em>2011 Census</em> (<a href="https://www.ons.gov.uk/census/2011census">https://www.ons.gov.uk/census/2011census</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r23"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=2011+Census" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">24.</div><div id="r24" class="citations"><div class="citation"><div class="citation-content">Landrum MJ, Lee JM, Benson M, et al. ClinVar: improving access to variant interpretations and supporting evidence. <em>Nucleic Acids Res</em> 2018;46:D1062-D1067.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r24"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1153" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29165669/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000419550700154" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=ClinVar%3A+improving+access+to+variant+interpretations+and+supporting+evidence.&amp;publication_year=2018&amp;journal=Nucleic+Acids+Res&amp;pages=D1062-D1067&amp;doi=10.1093%2Fnar%2Fgkx1153&amp;pmid=29165669" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">25.</div><div id="r25" class="citations"><div class="citation"><div class="citation-content">Carss KJ, Arno G, Erwood M, et al. Comprehensive rare variant analysis via whole-genome sequencing to determine the molecular pathology of inherited retinal disease. <em>Am J Hum Genet</em> 2017;100:75-90.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r27-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2016.12.003" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28041643/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000391910500007" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive+rare+variant+analysis+via+whole-genome+sequencing+to+determine+the+molecular+pathology+of+inherited+retinal+disease.&amp;publication_year=2017&amp;journal=Am+J+Hum+Genet&amp;pages=75-90&amp;doi=10.1016%2Fj.ajhg.2016.12.003&amp;pmid=28041643" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">26.</div><div id="r26" class="citations"><div class="citation"><div class="citation-content">Rowlands C, Thomas HB, Lord J, et al. Comparison of in silico strategies to prioritize rare genomic variants impacting RNA splicing for the diagnosis of genomic disorders. <em>Sci Rep</em> 2021;11:20607-20607.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r27-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-021-99747-2" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34663891/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000741362900004" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Comparison+of+in+silico+strategies+to+prioritize+rare+genomic+variants+impacting+RNA+splicing+for+the+diagnosis+of+genomic+disorders.&amp;publication_year=2021&amp;journal=Sci+Rep&amp;pages=20607-20607&amp;doi=10.1038%2Fs41598-021-99747-2&amp;pmid=34663891" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">27.</div><div id="r27" class="citations"><div class="citation"><div class="citation-content">Radziwon A, Arno G, Wheaton DK, et al. Single-base substitutions in the CHM promoter as a cause of choroideremia. <em>Hum Mutat</em> 2017;38:704-715.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r27" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/humu.23212" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28271586/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000401141900010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Single-base+substitutions+in+the+CHM+promoter+as+a+cause+of+choroideremia.&amp;publication_year=2017&amp;journal=Hum+Mutat&amp;pages=704-715&amp;doi=10.1002%2Fhumu.23212&amp;pmid=28271586" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r27" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r27-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] minigene, or luciferase assays. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r27-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] variant causing loss of gene expression, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">28.</div><div id="r28" class="citations"><div class="citation"><div class="citation-content">Farazi Fard MA, Rebelo AP, Buglo E, et al. Truncating mutations in UBAP1 cause hereditary spastic paraplegia. <em>Am J Hum Genet</em> 2019;104:767-773.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r28"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2019.03.001" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30929741/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000463474700017" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Truncating+mutations+in+UBAP1+cause+hereditary+spastic+paraplegia.&amp;publication_year=2019&amp;journal=Am+J+Hum+Genet&amp;pages=767-773&amp;doi=10.1016%2Fj.ajhg.2019.03.001&amp;pmid=30929741" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">29.</div><div id="r29" class="citations"><div class="citation"><div class="citation-content">Wallmeier J, Frank D, Shoemark A, et al. De novo mutations in FOXJ1 result in a motile ciliopathy with hydrocephalus and randomization of left/right body asymmetry. <em>Am J Hum Genet</em> 2019;105:1030-1039.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r29"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2019.09.022" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31630787/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000495878800014" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=De+novo+mutations+in+FOXJ1+result+in+a+motile+ciliopathy+with+hydrocephalus+and+randomization+of+left%2Fright+body+asymmetry.&amp;publication_year=2019&amp;journal=Am+J+Hum+Genet&amp;pages=1030-1039&amp;doi=10.1016%2Fj.ajhg.2019.09.022&amp;pmid=31630787" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">30.</div><div id="r30" class="citations"><div class="citation"><div class="citation-content">Cortese A, Zhu Y, Rebelo AP, et al. Biallelic mutations in <i>SORD</i> cause a common and potentially treatable hereditary neuropathy with implications for diabetes. <em>Nat Genet</em> 2020;52:473-481.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r30"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0615-4" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32367058/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000530227600002" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Biallelic+mutations+in+SORD+cause+a+common+and+potentially+treatable+hereditary+neuropathy+with+implications+for+diabetes.&amp;publication_year=2020&amp;journal=Nat+Genet&amp;pages=473-481&amp;doi=10.1038%2Fs41588-020-0615-4&amp;pmid=32367058" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">31.</div><div id="r31" class="citations"><div class="citation"><div class="citation-content">Haendel MA, Chute CG, Robinson PN. Classification, ontology, and precision medicine. <em>N Engl J Med</em> 2018;379:1452-1462.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r31"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_32_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa2035790&amp;key=10.1056%2FNEJMra1615014&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30304648/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000446923400009" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Classification%2C+ontology%2C+and+precision+medicine.&amp;publication_year=2018&amp;journal=N+Engl+J+Med&amp;pages=1452-1462&amp;doi=10.1056%2FNEJMra1615014&amp;pmid=30304648" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">32.</div><div id="r32" class="citations"><div class="citation"><div class="citation-content">Yang Y, Muzny DM, Xia F, et al. Molecular findings among patients referred for clinical whole-exome sequencing. <em>JAMA</em> 2014;312:1870-1879.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2014.14601" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25326635/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000344532600016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Molecular+findings+among+patients+referred+for+clinical+whole-exome+sequencing.&amp;publication_year=2014&amp;journal=JAMA&amp;pages=1870-1879&amp;doi=10.1001%2Fjama.2014.14601&amp;pmid=25326635" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">33.</div><div id="r33" class="citations"><div class="citation"><div class="citation-content">Hu X, Li N, Xu Y, et al. Proband-only medical exome sequencing as a cost-effective first-tier genetic diagnostic test for patients without prior molecular tests and clinical diagnosis in a developing country: the China experience. <em>Genet Med</em> 2018;20:1045-1053.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2017.195" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29095814/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000445531800018" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Proband-only+medical+exome+sequencing+as+a+cost-effective+first-tier+genetic+diagnostic+test+for+patients+without+prior+molecular+tests+and+clinical+diagnosis+in+a+developing+country%3A+the+China+experience.&amp;publication_year=2018&amp;journal=Genet+Med&amp;pages=1045-1053&amp;doi=10.1038%2Fgim.2017.195&amp;pmid=29095814" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">34.</div><div id="r34" class="citations"><div class="citation"><div class="citation-content">Vissers LELM, van Nimwegen KJM, Schieving JH, et al. A clinical utility study of exome sequencing versus conventional genetic testing in pediatric neurology. <em>Genet Med</em> 2017;19:1055-1063.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2017.1" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28333917/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000409525100014" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+clinical+utility+study+of+exome+sequencing+versus+conventional+genetic+testing+in+pediatric+neurology.&amp;publication_year=2017&amp;journal=Genet+Med&amp;pages=1055-1063&amp;doi=10.1038%2Fgim.2017.1&amp;pmid=28333917" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">35.</div><div id="r35" class="citations"><div class="citation"><div class="citation-content">Gilissen C, Hehir-Kwa JY, Thung DT, et al. Genome sequencing identifies major causes of severe intellectual disability. <em>Nature</em> 2014;511:344-347.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r35"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13394" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24896178/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000338992200035" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genome+sequencing+identifies+major+causes+of+severe+intellectual+disability.&amp;publication_year=2014&amp;journal=Nature&amp;pages=344-347&amp;doi=10.1038%2Fnature13394&amp;pmid=24896178" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">36.</div><div id="r36" class="citations"><div class="citation"><div class="citation-content">Eichler EE. Genetic variation, comparative genomics, and the diagnosis of disease. <em>N Engl J Med</em> 2019;381:64-74.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r36"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_37_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa2035790&amp;key=10.1056%2FNEJMra1809315&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31269367/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000474094900013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genetic+variation%2C+comparative+genomics%2C+and+the+diagnosis+of+disease.&amp;publication_year=2019&amp;journal=N+Engl+J+Med&amp;pages=64-74&amp;doi=10.1056%2FNEJMra1809315&amp;pmid=31269367" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">37.</div><div id="r37" class="citations"><div class="citation"><div class="citation-content">NHS. <em>National Genomic Test Directory</em>. 2020 (<a href="https://www.england.nhs.uk/publication/national-genomic-test-directories/">https://www.england.nhs.uk/publication/national-genomic-test-directories/</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r37"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=National+Genomic+Test+Directory&amp;publication_year=2020" target="_blank">Google Scholar</a></div></div></div></div></div></div><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--secondary" aria-expanded="false" aria-controls="collapsible-text"><span>Show all references</span></button></div></section></div></section></div><div class="core-collateral"><div id="core-collateral-info" role="tabpanel" data-core-tabs="core-collateral-info" tabindex="-1" aria-labelledby="pane-core-collateral-info"><header><h2 id="pane-core-collateral-info"><i class="icon-info" aria-hidden="true"></i>Information &amp; Authors</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><div role="tablist"><button aria-controls="tab-information" role="tab" id="tab-information-label" aria-selected="true" class="active">Information</button><button aria-controls="tab-contributors" role="tab" id="tab-contributors-label" tabindex="-1" aria-selected="false">Authors</button></div><section id="tab-information" aria-labelledby="tab-information-label" role="tabpanel" tabindex="0"><h3>Information</h3><section class="core-self-citation"><h4>Published In</h4><div class="core-journal-presentation"><div class="core-journal-description"><div property="isPartOf" typeof="Periodical"><span property="name">New England Journal of Medicine</span></div><div class="core-enumeration"><a href="/toc/nejm/385/20"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">385</span></span> • <span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">20</span></span> • <span property="datePublished">November 11, 2021</span></a></div><div class="core-pagination"><span class="heading">Pages</span>: <span class="content"><span property="pageStart">1868</span>-<span property="pageEnd">1880</span></span></div></div></div></section><section class="core-copyright"><h4>Copyright</h4><div role="paragraph">Copyright © 2021 Massachusetts Medical Society. All rights reserved.</div><div>For personal use only. Any commercial reuse of NEJM Group content requires <a href="mailto:permissions@nejm.org">permission</a>.</div></section><section class="core-history"><h4>History</h4><div><b class="core-label">Published online</b>: November 10, 2021</div><div><b class="core-label">Published in issue</b>: November 11, 2021</div></section><section data-location="recirc_topics_article" class="core-classifications"><h4>Topics</h4><div class="keywords"><ol><li><a href="/browse/topic/genetics-general" alt="View article keyword Genetics General" data-interactiontype="article_recirculation_click">Genetics General</a></li><li role="presentation" aria-hidden="true" class="list-line-break"></li><li><a href="/browse/topic/health-care-delivery" alt="View article keyword Health Care Delivery" data-interactiontype="article_recirculation_click">Health Care Delivery</a></li><li role="presentation" aria-hidden="true" class="list-line-break"></li></ol></div></section></section><section id="tab-contributors" aria-labelledby="tab-contributors-label" role="tabpanel" tabindex="-1"><h3>Authors</h3><section class="core-authors"><h4>Authors</h4><div role="paragraph"><span property="author" typeof="Person">The 100,000 Genomes Project Pilot Investigators<sup class="xref"><a href="#fn0" role="doc-noteref">*</a></sup></span></div></section><section class="core-authors-notes"><h4>Notes</h4><div role="doc-footnote">Dr. Caulfield can be contacted at <a href="mailto:m.j.caulfield@qmul.ac.uk">m.j.caulfield@qmul.ac.uk</a> or at Genomics England, William Harvey Research Institute, Queen Mary University of London, London EC1M 6BQ, United Kingdom.</div><div role="doc-footnote" data-has="label"><div class="label">*</div><div id="fn0" role="paragraph">The authors’ full names, academic degrees, and affiliations are listed in the <a href="#appendix-1">Appendix</a>.</div></div><div role="doc-footnote"><div id="fn2" role="paragraph">Drs. Smedley and Smith, Mr. Martin, and Drs. E.A. Thomas, McDonagh, Cipriani, Ellingford, Arno, Tucci, Vandrovcova, Chan, and H.J. Williams and Drs. Scott, Fowler, Rendon, and Caulfield contributed equally to this article.</div></div></section></section></div><div id="core-collateral-metrics" role="tabpanel" data-core-tabs="core-collateral-metrics" tabindex="-1" aria-labelledby="pane-core-collateral-metrics"><header><h2 id="pane-core-collateral-metrics"><i class="icon-timeline" aria-hidden="true"></i>Metrics &amp; Citations</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><div role="tablist"><button aria-controls="tab-metrics-inner" role="tab" id="tab-metrics-inner-label" aria-selected="true" class="active" data-interactiontype="article_tab" data-tabname="metricsWindow">Metrics</button><button aria-controls="tab-citations" role="tab" id="tab-citations-label" tabindex="-1" aria-selected="false" data-interactiontype="article_tab" data-tabname="metricsWindow">Citations<span class="citations-count">431</span></button></div><section id="tab-metrics-inner" aria-labelledby="tab-metrics-inner-label" role="tabpanel" tabindex="0"><h3>Metrics</h3><section>









    
    
        <div data-widget-def="UX3HTMLWidget" data-widget-id="95ea9f58-ec0e-4db2-b2ab-280fc876af53" data-location="article_metrics_article">
        



        
            <h4 class="">
                Altmetrics
            </h4>
        
        <div data-badge-details="right" data-badge-type="large-donut" data-doi="10.1056/NEJMoa2035790" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed" data-uuid="2234c7f4-0a95-b456-9c23-323e4b5bbddd"><div style="overflow:hidden;">
    <div class="altmetric-normal-legend">
        <a target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047" style="display:inline-block;">
                <img alt="Article has an altmetric score of 1275" src="https://badges.altmetric.com/?size=320&amp;score=1275&amp;types=mabrtfud" width="180" height="180" style="border:0; margin:0; max-width: none;">
        </a>
        <p class="altmetric-see-more-details" style="padding-top: 10px; text-align: center;"><a target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047">See more details</a></p>
    </div>
    <div id="_altmetric_popover_el_2234c7f4-0a95-b456-9c23-323e4b5bbddd" class="altmetric-embed right" style="margin:0; padding:0; display:inline-block; float:left; position:relative;">
        <div class="altmetric_container">
            <div class="altmetric-embed altmetric-popover-inner right">
                <div style="padding:0; margin: 0;" class="altmetric-embed altmetric-popover-content">
                        <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #FF0000;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=news">
          Picked up by <b>97</b> news outlets
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=blogs">
          Blogged by <b>9</b>
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #9f79f2;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=policy-documents">
          Referenced in <b>2</b> policy sources
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=twitter">
          Posted by <b>740</b> X users
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #f27700;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=patents">
          Referenced in <b>2</b> patents
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #2445bd;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=facebook">
          On <b>10</b> Facebook pages
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #958899;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=wikipedia">
          Referenced in <b>1</b> Wikipedia pages
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #D5E8F0;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=reddit">
          Reddited by <b>1</b>
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #F4006E;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=research-highlights">
          Highlighted by <b>1</b> platforms
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #70B1FF;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=116590047&amp;tab=bluesky">
          Referenced by <b>2</b> Bluesky users
      </a>
    </div>

    <div class="altmetric-embed readers" style="margin-top: 10px;">
          <div class="altmetric-embed tip_mendeley" style="padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;">
              <b>520</b> readers on Mendeley
          </div>
    </div>

                </div>
            </div>
        </div>
    </div>
</div></div>
<script async="" type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js" nonce="94d6a712622201a2-GRU"></script>

        </div>
    

</section></section><section id="tab-citations" aria-labelledby="tab-citations-label" role="tabpanel" tabindex="-1"><h3>Citations</h3><section>



        
            <h4 class="">
                Export citation
            </h4>
        
        <div class="citation-download cd-sec">
<p class="citation-download_msg">Select the format you want to export the citation of this publication.</p>
<form action="/action/downloadCitation" name="frmCitmgr" method="post" target="_self" class="citation-download_form"><input type="hidden" name="doi" value="10.1056/NEJMoa2035790"> <input type="hidden" name="downloadFileName" value="csp_385_"> <input type="hidden" name="include" value="abs">
<div class="dropdown dropdown-selectable dropdown-selectable--form-control"><input type="hidden" name="format" value=""> <label class="form-label">Format*</label> <button id="slct_format" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="btn dropdown-toggle d-flex align-items-center justify-content-between">Please Select </button>
<div aria-labelledby="slct_format" role="listbox" class="dropdown-menu"><a href="#" data-value="ris" class="dropdown-item">RIS (ProCite, Reference Manager)</a><a href="#" data-value="endnote" class="dropdown-item">EndNote</a><a href="#" data-value="bibtex" class="dropdown-item">BibTex</a><a href="#" data-value="medlars" class="dropdown-item">Medlars</a><a href="#" data-value="refworks" class="dropdown-item">RefWorks</a></div>
<div class="invalid-feedback">Please select an item in the list</div>
</div>
<div class="citation-download_checkbox"><label class="checkbox--primary"><input type="checkbox" name="direct" value="true"><span class="checkbox--primary-icons"><span class="checkbox--primary-unchecked"><span class="icon-component "><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#toggles-checboxUnchecked"></use></svg></span></span><span class="checkbox--primary-checked"><span class="icon-component "><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#toggles-checboxChecked"></use></svg></span></span></span><span class="checkbox--primary-label"><span class="checkbox--primary-label-text">Direct Import</span></span></label></div>
<div class="citation-download_btn"><button type="submit" name="submit" value="Download" class="ng-btn_default ng-btn_iconLeft" data-interactiontype="article_tools_citation_download" data-multimedia-type="Citation"><span class="icon-component "><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#citations"></use></svg></span><span class="ng-btn_text">Export citation</span></button></div>
</form></div>
</section><section>









    
    
        <div data-widget-def="UX3CitedByWidget" data-widget-id="efb4df9a-8e1a-47f9-bfb7-271959277a9f" data-location="citedby_article">
        



        
            <h4 class="">
                Cited by
            </h4>
        
        <section id="cited-by" class="cited-by"><div class="cited-by__wrapper"><div id="cited-by__content" class="cited-by__content"><ol data-source="/pb/widgets/citedBy?page=1&amp;widgetId=efb4df9a-8e1a-47f9-bfb7-271959277a9f&amp;pbContext=%3BrequestedJournal%3Ajournal%3Anejm%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Adoi%5C%3A10.1056%2FNEJMoa2035790%3Bwgroup%3Astring%3AMMS+NextGen+Website+Group%3Bissue%3Aissue%3Adoi%5C%3A10.1056%2Fnejm_2021.385.issue-20%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bwebsite%3Awebsite%3Amms-site%3Bjournal%3Ajournal%3Anejms" data-lazyload="false" data-total="431" class="cited-by__list list-unstyled"><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Sanjana M. Paye, </li><li class="list-inline-item cited-by__entry__author">Michael D. Edge, </li></ul><span class="cited-by__entry__title">
              Mathematical bounds on
              
              and the effect size in case-control genome-wide association studies
            , </span><span class="cited-by__entry__series-title">Theoretical Population Biology, </span><span class="cited-by__entry__volume"><strong>164</strong>, </span><span class="cited-by__entry__page-range">(1-11), </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1016/j.tpb.2025.04.003" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1016/j.tpb.2025.04.003</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1016/j.tpb.2025.04.003" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Chris Odhams, </li><li class="list-inline-item cited-by__entry__author">Hywel J. Williams, </li></ul><span class="cited-by__entry__title">Improved diagnosis of patients with rare diseases through the application of constrained coding region annotation and de novo status, </span><span class="cited-by__entry__series-title">Genetics in Medicine, </span><span class="cited-by__entry__volume"><strong>27</strong>, </span><span class="cited-by__entry__issue">7, </span><span class="cited-by__entry__page-range">(101447), </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1016/j.gim.2025.101447" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1016/j.gim.2025.101447</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1016/j.gim.2025.101447" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Miriam Ehrenberg, </li><li class="list-inline-item cited-by__entry__author">Maayan Avraham, </li><li class="list-inline-item cited-by__entry__author">Sandeep Sarma Asodu, </li><li class="list-inline-item cited-by__entry__author">Abigail R. Moye, </li><li class="list-inline-item cited-by__entry__author">Riccardo Sangermano, </li><li class="list-inline-item cited-by__entry__author">Leah Rizel, </li><li class="list-inline-item cited-by__entry__author">Tahleel Ali-Nasser, </li><li class="list-inline-item cited-by__entry__author">Ifat Sher, </li><li class="list-inline-item cited-by__entry__author">David Gurwitz, </li><li class="list-inline-item cited-by__entry__author">Katherine R. Chao, </li><li class="list-inline-item cited-by__entry__author">Antonio Rivera, </li><li class="list-inline-item cited-by__entry__author">Andrew R. Webster, </li><li class="list-inline-item cited-by__entry__author">Carlo Rivolta, </li><li class="list-inline-item cited-by__entry__author">Hadas Newman, </li><li class="list-inline-item cited-by__entry__author">Eran Pras, </li><li class="list-inline-item cited-by__entry__author">Ygal Rotenstreich, </li><li class="list-inline-item cited-by__entry__author">Eyal Banin, </li><li class="list-inline-item cited-by__entry__author">Eric A. Pierce, </li><li class="list-inline-item cited-by__entry__author">Dinah Zur, </li><li class="list-inline-item cited-by__entry__author">Gavin Arno, </li><li class="list-inline-item cited-by__entry__author">Kinga M. Bujakowska, </li><li class="list-inline-item cited-by__entry__author">Siying Lin, </li><li class="list-inline-item cited-by__entry__author">Dror Sharon, </li><li class="list-inline-item cited-by__entry__author">Tamar Ben-Yosef, </li></ul><span class="cited-by__entry__title">Biallelic null variants in C19orf44 cause a unique late-onset retinal dystrophy phenotype characterized by patchy perifoveal chorioretinal atrophy, </span><span class="cited-by__entry__series-title">Genetics in Medicine, </span><span class="cited-by__entry__volume"><strong>27</strong>, </span><span class="cited-by__entry__issue">6, </span><span class="cited-by__entry__page-range">(101401), </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1016/j.gim.2025.101401" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1016/j.gim.2025.101401</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1016/j.gim.2025.101401" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Isaac J. Siegel, </li><li class="list-inline-item cited-by__entry__author">Sarah L. Vaithilingam, </li><li class="list-inline-item cited-by__entry__author">Madeline M. Hartig, </li><li class="list-inline-item cited-by__entry__author">Ella C. Patty, </li><li class="list-inline-item cited-by__entry__author">Lily E. Mantsch, </li><li class="list-inline-item cited-by__entry__author">Sheldon R. Garrison, </li></ul><span class="cited-by__entry__title">Diagnostic delays in rare genetic disorders with neuropsychiatric manifestations: A systematic review, </span><span class="cited-by__entry__series-title">European Journal of Medical Genetics, </span><span class="cited-by__entry__volume"><strong>75</strong>, </span><span class="cited-by__entry__page-range">(105016), </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1016/j.ejmg.2025.105016" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1016/j.ejmg.2025.105016</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1016/j.ejmg.2025.105016" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Kishore Jaganathan, </li><li class="list-inline-item cited-by__entry__author">Nicole Ersaro, </li><li class="list-inline-item cited-by__entry__author">Gherman Novakovsky, </li><li class="list-inline-item cited-by__entry__author">Yuchuan Wang, </li><li class="list-inline-item cited-by__entry__author">Terena James, </li><li class="list-inline-item cited-by__entry__author">Jeremy Schwartzentruber, </li><li class="list-inline-item cited-by__entry__author">Petko Fiziev, </li><li class="list-inline-item cited-by__entry__author">Irfahan Kassam, </li><li class="list-inline-item cited-by__entry__author">Fan Cao, </li><li class="list-inline-item cited-by__entry__author">Johann Hawe, </li><li class="list-inline-item cited-by__entry__author">Henry Cavanagh, </li><li class="list-inline-item cited-by__entry__author">Ashley Lim, </li><li class="list-inline-item cited-by__entry__author">Grace Png, </li><li class="list-inline-item cited-by__entry__author">Jeremy McRae, </li><li class="list-inline-item cited-by__entry__author">Abhimanyu Banerjee, </li><li class="list-inline-item cited-by__entry__author">Arvind Kumar, </li><li class="list-inline-item cited-by__entry__author">Jacob Ulirsch, </li><li class="list-inline-item cited-by__entry__author">Yan Zhang, </li><li class="list-inline-item cited-by__entry__author">Francois Aguet, </li><li class="list-inline-item cited-by__entry__author">Pierrick Wainschtein, </li><li class="list-inline-item cited-by__entry__author">Laksshman Sundaram, </li><li class="list-inline-item cited-by__entry__author">Adriana Salcedo, </li><li class="list-inline-item cited-by__entry__author">Sofia Kyriazopoulou Panagiotopoulou, </li><li class="list-inline-item cited-by__entry__author">Delasa Aghamirzaie, </li><li class="list-inline-item cited-by__entry__author">Evin Padhi, </li><li class="list-inline-item cited-by__entry__author">Ziming Weng, </li><li class="list-inline-item cited-by__entry__author">Shan Dong, </li><li class="list-inline-item cited-by__entry__author">Damian Smedley, </li><li class="list-inline-item cited-by__entry__author">Mark Caulfield, </li><li class="list-inline-item cited-by__entry__author">Anne O’Donnell-Luria, </li><li class="list-inline-item cited-by__entry__author">Heidi L. Rehm, </li><li class="list-inline-item cited-by__entry__author">Stephan J. Sanders, </li><li class="list-inline-item cited-by__entry__author">Anshul Kundaje, </li><li class="list-inline-item cited-by__entry__author">Stephen B. Montgomery, </li><li class="list-inline-item cited-by__entry__author">Mark T. Ross, </li><li class="list-inline-item cited-by__entry__author">Kyle Kai-How Farh, </li></ul><span class="cited-by__entry__title">Predicting expression-altering promoter mutations with deep learning, </span><span class="cited-by__entry__series-title">Science, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1126/science.ads7373" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1126/science.ads7373</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1126/science.ads7373" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Adam Jackson, </li><li class="list-inline-item cited-by__entry__author">Nishi Thaker, </li><li class="list-inline-item cited-by__entry__author">Alexander Blakes, </li><li class="list-inline-item cited-by__entry__author">Gillian Rice, </li><li class="list-inline-item cited-by__entry__author">Sam Griffiths-Jones, </li><li class="list-inline-item cited-by__entry__author">Meena Balasubramanian, </li><li class="list-inline-item cited-by__entry__author">Jennifer Campbell, </li><li class="list-inline-item cited-by__entry__author">Nora Shannon, </li><li class="list-inline-item cited-by__entry__author">Jungmin Choi, </li><li class="list-inline-item cited-by__entry__author">Juhyeon Hong, </li><li class="list-inline-item cited-by__entry__author">David Hunt, </li><li class="list-inline-item cited-by__entry__author">Anna de Burca, </li><li class="list-inline-item cited-by__entry__author">Soo Yeon Kim, </li><li class="list-inline-item cited-by__entry__author">Taekeun Kim, </li><li class="list-inline-item cited-by__entry__author">Seungbok Lee, </li><li class="list-inline-item cited-by__entry__author">Melody Redman, </li><li class="list-inline-item cited-by__entry__author">Rocio Rius, </li><li class="list-inline-item cited-by__entry__author">Cas Simons, </li><li class="list-inline-item cited-by__entry__author">Tiong Yang Tan, </li><li class="list-inline-item cited-by__entry__author">Jamie Ellingford, </li><li class="list-inline-item cited-by__entry__author">Raymond T. O’Keefe, </li><li class="list-inline-item cited-by__entry__author">Jong Hee Chae, </li><li class="list-inline-item cited-by__entry__author">Siddharth Banka, </li></ul><span class="cited-by__entry__title">Analysis of R-loop forming regions identifies RNU2-2 and RNU5B-1 as neurodevelopmental disorder genes, </span><span class="cited-by__entry__series-title">Nature Genetics, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1038/s41588-025-02209-y" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1038/s41588-025-02209-y</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1038/s41588-025-02209-y" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Claudia Ching Yan Chung, </li><li class="list-inline-item cited-by__entry__author">Annie Tsz Wai Chu, </li><li class="list-inline-item cited-by__entry__author">Brian Hon Yin Chung, </li></ul><span class="cited-by__entry__title">A roadmap for genome projects to foster psychosocial and economic evidence to further policy and practice, </span><span class="cited-by__entry__series-title">Communications Medicine, </span><span class="cited-by__entry__volume"><strong>5</strong>, </span><span class="cited-by__entry__issue">1, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1038/s43856-025-00917-4" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1038/s43856-025-00917-4</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1038/s43856-025-00917-4" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Wai Yan Yau, </li><li class="list-inline-item cited-by__entry__author">Roisin Sullivan, </li><li class="list-inline-item cited-by__entry__author">Emer O’Connor, </li><li class="list-inline-item cited-by__entry__author">David Pellerin, </li><li class="list-inline-item cited-by__entry__author">Michael H Parkinson, </li><li class="list-inline-item cited-by__entry__author">Paola Giunti, </li><li class="list-inline-item cited-by__entry__author">Marie-Josée Dicaire, </li><li class="list-inline-item cited-by__entry__author">Matt C Danzi, </li><li class="list-inline-item cited-by__entry__author">Stephan Züchner, </li><li class="list-inline-item cited-by__entry__author">Bernard Brais, </li><li class="list-inline-item cited-by__entry__author">Nicholas W Wood, </li><li class="list-inline-item cited-by__entry__author">Henry Houlden, </li><li class="list-inline-item cited-by__entry__author">Jana Vandrovcova, </li></ul><span class="cited-by__entry__title">Diagnostic yield and limitations of whole-genome sequencing for hereditary cerebellar ataxia, </span><span class="cited-by__entry__series-title">Brain Communications, </span><span class="cited-by__entry__volume"><strong>7</strong>, </span><span class="cited-by__entry__issue">3, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1093/braincomms/fcaf188" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1093/braincomms/fcaf188</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1093/braincomms/fcaf188" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">O. Isaac Garcia-Salinas, </li><li class="list-inline-item cited-by__entry__author">Seongwon Hwang, </li><li class="list-inline-item cited-by__entry__author">Qin Qin Huang, </li><li class="list-inline-item cited-by__entry__author">Rashesh Sanghvi, </li><li class="list-inline-item cited-by__entry__author">Daniel S. Malawsky, </li><li class="list-inline-item cited-by__entry__author">Joanna Kaplanis, </li><li class="list-inline-item cited-by__entry__author">Matthew D. C. Neville, </li><li class="list-inline-item cited-by__entry__author">Felix R. Day, </li><li class="list-inline-item cited-by__entry__author">Raheleh Rahbari, </li><li class="list-inline-item cited-by__entry__author">Aylwyn Scally, </li><li class="list-inline-item cited-by__entry__author">Hilary C. Martin, </li></ul><span class="cited-by__entry__title">The impact of ancestral, genetic, and environmental influences on germline de novo mutation rates and spectra, </span><span class="cited-by__entry__series-title">Nature Communications, </span><span class="cited-by__entry__volume"><strong>16</strong>, </span><span class="cited-by__entry__issue">1, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1038/s41467-025-59750-x" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1038/s41467-025-59750-x</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1038/s41467-025-59750-x" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Paraskevi Christofidou, </li><li class="list-inline-item cited-by__entry__author">Christopher G. Bell, </li></ul><span class="cited-by__entry__title">The predictive power of profiling the DNA methylome in human health and disease, </span><span class="cited-by__entry__series-title">Epigenomics, </span><span class="cited-by__entry__volume"><strong>17</strong>, </span><span class="cited-by__entry__issue">9, </span><span class="cited-by__entry__page-range">(599-610), </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1080/17501911.2025.2500907" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1080/17501911.2025.2500907</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1080/17501911.2025.2500907" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry--see-more"><a href="javascript:void(0)" data-source="/pb/widgets/citedBy?page=1&amp;widgetId=efb4df9a-8e1a-47f9-bfb7-271959277a9f&amp;pbContext=%3BrequestedJournal%3Ajournal%3Anejm%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Adoi%5C%3A10.1056%2FNEJMoa2035790%3Bwgroup%3Astring%3AMMS+NextGen+Website+Group%3Bissue%3Aissue%3Adoi%5C%3A10.1056%2Fnejm_2021.385.issue-20%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bwebsite%3Awebsite%3Amms-site%3Bjournal%3Ajournal%3Anejms" class="cited-by__see-more">See more</a></li></ol></div><div class="cited-by__spinner justify-content-center align-items-center d-none"><div role="status" class="spinner-border text-gray"><span class="sr-only">Loading...</span></div></div></div></section>

        </div>
    

</section></section></div><div id="core-collateral-fulltext-options" role="tabpanel" data-core-tabs="core-collateral-fulltext-options" tabindex="-1" aria-labelledby="pane-core-collateral-fulltext-options"><header><h2 id="pane-core-collateral-fulltext-options"><i class="icon-eye" aria-hidden="true"></i>View Options</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><div class="section--wrapper"><h3>View options</h3><section class="format--pdf"><h4> <abbr title="Portable Document Format">PDF</abbr></h4><a href="https://www.nejm.org/doi/pdf/10.1056/NEJMoa2035790" class="btn btn--pdf" aria-label="View PDF" data-behavior="trackDownloadEvent" data-doi="10.1056/NEJMoa2035790" data-interactiontype="article_tools_download_pdf" data-multimedia-type="Article PDF" data-multimedia-filename="nejmoa2035790.pdf"><i aria-hidden="true" class="icon-pdf"></i><span>View PDF</span></a></section></div><!-- Its needed to duplicate this for PB check Collateral.js as well--></div><div id="core-collateral-media" role="tabpanel" data-core-tabs="core-collateral-media" tabindex="-1" aria-labelledby="pane-core-collateral-media"><header><h2 id="pane-core-collateral-media"><i class="icon-photo" aria-hidden="true"></i>Media</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><div role="tablist"><button aria-controls="tab-figures" role="tab" id="tab-figures-label" aria-selected="true" class="active">Figures</button><button aria-controls="tab-other" role="tab" id="tab-other-label" tabindex="-1" aria-selected="false" class="inactive" aria-disabled="true">Other</button></div><section id="tab-figures" aria-labelledby="tab-figures-label" role="tabpanel" tabindex="0"><h3>Figures</h3><div class="text-right"><a class="btn btn--inverse  open-in-viewer" href="#figures"><span>Open all in viewer</span><i class="icon-open_in_new" aria-hidden="true"></i></a></div><div id="core-f1"><figure class="graphic"><a class="open-in-viewer" href="#f1" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f1.jpg"><img src="/cms/10.1056/NEJMoa2035790/asset/ee2525a0-274f-49b3-a2ee-6a621a475e47/assets/images/large/nejmoa2035790_f1.jpg" height="2845" width="2640" aria-labelledby="f1" loading="lazy"></a><figcaption><div class="caption">Overview of the Diagnostic and Research Pipeline and Source of Diagnoses.</div><div class="notes"><div role="doc-footnote">Results from 2183 probands in the pilot study were returned for presentation to the Genomic Medicine Centres (GMCs) of the recruiting hospitals. A total of 25% of the probands received a positive diagnosis, and 10% had a variant or variants of unknown significance in genes that were determined by clinical geneticists at the recruiting site to be consistent with the phenotype but that required further functional validation. The remaining 65% of the probands received a negative report at the time but will be reassessed. The numbers and sources of these positive diagnoses are shown at each stage of the automated diagnostic pipeline, and the additional research is shown for diagnoses that were not immediately obvious. CCR denotes constrained coding region, indel insertion or deletion, mtDNA mitochondrial DNA, SNV single-nucleotide variant, and SV structural variant.</div></div></figcaption></figure><div class="external-links"><a class="pill__to-original" href="#f1"><i class="icon-return" aria-hidden="true"></i><span>Go to Figure</span></a><a class="open-in-viewer" href="#f1" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f1.jpg"><span>Open in Viewer</span><i class="icon-expand" aria-hidden="true"></i></a></div></div><div id="core-f2"><figure class="graphic"><a class="open-in-viewer" href="#f2" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f2.jpg"><img src="/cms/10.1056/NEJMoa2035790/asset/6ac3b65b-4124-4ff1-b695-0053a7cde6de/assets/images/large/nejmoa2035790_f2.jpg" height="3431" width="2640" aria-labelledby="f2" loading="lazy"></a><figcaption><div class="caption">Diagnoses in the Rare Disease Pilot Study.</div><div class="notes"><div role="doc-footnote">Panel A shows diagnostic yield for any disease and according to family structure and cause of disease. The diagnostic yield was 35% for diseases likely to have a monogenic cause and 11% for diseases likely to have a complex cause. The values above the bars are the numbers of probands. Singleton refers to a proband for whom no other family member was recruited, family duo to a parent–proband pair, family trio to both parents and a proband, and family quad to a proband, sibling, and parents. Panel B shows diagnostic yield according to disease category. The values above the bars are the numbers of probands. Panel C shows the diagnostic yield among probands according to previous genetic testing and most extensive testing type: chromosomal (karyotyping, array-based comparative genomic hybridization, single-nucleotide polymorphism arrays), targeted (targeted single-gene tests), next-generation sequencing (NGS) panels, or whole-exome sequencing (WES). The values above the bars are the numbers of probands. Panel D shows the performance of virtual panel-based and Exomiser-based prioritization for identifying the diagnoses. “Disease panel only” indicates the use of a single virtual panel for the recruited disease category. “Applied panels” indicates the use of all applied virtual gene panels used in the pipeline, including the recruited disease–associated panel as well as 0 or more additional panels selected on the basis of the patient’s phenotypes (Human Phenotype Ontology terms). “Exomiser top” indicates Exomiser use in the top-ranked candidate variants, “Exomiser top 3” use in the top three candidates, and “Exomiser top 5” use in the top five candidates. Sensitivity is the percentage of true positive diagnoses based on SNVs or indels that were identified, and the positive predictive value is the percentage of prioritized variants that led to a positive diagnosis. The values above the bars are percentages. In this analysis, the diagnosed variant or variants are true positives, and the other candidate variants that were returned are false positives.</div></div></figcaption></figure><div class="external-links"><a class="pill__to-original" href="#f2"><i class="icon-return" aria-hidden="true"></i><span>Go to Figure</span></a><a class="open-in-viewer" href="#f2" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f2.jpg"><span>Open in Viewer</span><i class="icon-expand" aria-hidden="true"></i></a></div></div></section><section id="tab-other" aria-labelledby="tab-other-label" role="tabpanel" tabindex="-1"><h3>Other</h3></section></div><div id="core-collateral-tables" role="tabpanel" data-core-tabs="core-collateral-tables" tabindex="-1" aria-labelledby="pane-core-collateral-tables"><header><h2 id="pane-core-collateral-tables"><i class="icon-tables" aria-hidden="true"></i>Tables</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><!-- There is no content. --><div class="section--wrapper"><div class="text-right"><a class="btn btn--inverse  open-in-viewer" href="#tables"><span>Open all in viewer</span><i class="icon-open_in_new" aria-hidden="true"></i></a></div><div id="core-t1"><figure class="table"><a class="open-in-viewer" href="#t1" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t1.jpg"><div class="graphic-wrap" data-presentation="suppressed-in-viewer"><img src="/cms/10.1056/NEJMoa2035790/asset/21a9ed96-3de9-4dbb-9ceb-e2c4838810e6/assets/images/large/nejmoa2035790_t1.jpg" height="1235" width="1711" loading="lazy"></div><div class="table-wrap catalyst-standard" data-presentation="suppressed-in-flow"><table data-shading="custom"><thead><tr class="head1" data-type="row head1"><th class="txlx-borders" data-xml-align="left">Characteristic</th><th class="txxx-borders">All Probands<br>(N=2183)</th><th class="txxx-borders">Pediatric Probands<br>(N=571)</th><th class="txxr-borders">Adult Probands<br>(N=1612)</th></tr></thead><tbody><tr data-type="row"><td class="xxlx-borders hanging02 shading">Sex — no. (%)</td><td class="xxxx-borders shading">&nbsp;</td><td class="xxxx-borders shading">&nbsp;</td><td class="xxxr-borders shading">&nbsp;</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">Male</td><td class="xxxx-borders">1138 (52)</td><td class="xxxx-borders">339 (59)</td><td class="xxxr-borders">799 (50)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12 shading" data-xml-align="left">Female</td><td class="xxxx-borders shading">1045 (48)</td><td class="xxxx-borders shading">232 (41)</td><td class="xxxr-borders shading">813 (50)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">Total</td><td class="xxxx-borders">2183 (100)</td><td class="xxxx-borders">571 (100)</td><td class="xxxr-borders">1612 (100)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging02 shading" data-xml-align="left">Median age at recruitment (IQR) — yr</td><td class="xxxx-borders shading">35 (18–54)</td><td class="xxxx-borders shading">9 (5–14)</td><td class="xxxr-borders shading">45 (31–60)</td></tr><tr data-type="row"><td class="xxlx-borders hanging02">Inferred ancestry — no. (%) [% consanguinity suggested in record]<a href="#core-t1fn2" role="doc-noteref">†</a></td><td class="xxxx-borders">&nbsp;</td><td class="xxxx-borders">&nbsp;</td><td class="xxxr-borders">&nbsp;</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12 shading" data-xml-align="left">African</td><td class="xxxx-borders shading">50 (2) [0]</td><td class="xxxx-borders shading">25 (4) [0]</td><td class="xxxr-borders shading">25 (2) [0]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">American with mixed ancestry</td><td class="xxxx-borders">26 (1) [23]</td><td class="xxxx-borders">12 (2) [25]</td><td class="xxxr-borders">14 (1) [21]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12 shading" data-xml-align="left">East Asian</td><td class="xxxx-borders shading">8 (&lt;1) [0]</td><td class="xxxx-borders shading">2 (&lt;1) [0]</td><td class="xxxr-borders shading">6 (&lt;1) [0]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">European</td><td class="xxxx-borders">1931 (88) [&lt;1]</td><td class="xxxx-borders">438 (77) [&lt;1]</td><td class="xxxr-borders">1493 (93) [&lt;1]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12 shading" data-xml-align="left">South Asian</td><td class="xxxx-borders shading">163 (7) [36]</td><td class="xxxx-borders shading">93 (16) [43]</td><td class="xxxr-borders shading">70 (4) [25]</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging12" data-xml-align="left">Not determined</td><td class="xxxx-borders">5 (&lt;1) [0]</td><td class="xxxx-borders">1 (&lt;1) [0]</td><td class="xxxr-borders">4 (&lt;1) [0]</td></tr><tr data-xml-align="center" data-type="row"><td class="xblx-borders hanging12 shading" data-xml-align="left">Total</td><td class="xbxx-borders shading">2183 (100) [3]</td><td class="xbxx-borders shading">571 (26) [8]</td><td class="xbxr-borders shading">1612 (74) [2]</td></tr></tbody></table></div><figcaption><div class="notes"><div role="doc-footnote" data-has="label"><div class="label">*</div><div id="core-t1fn1" role="paragraph" data-to-manipulate="true">Pediatric probands were younger than 18 years of age at recruitment, and adult probands were 18 years of age or older at recruitment. IQR denotes interquartile range.</div></div><div role="doc-footnote" data-has="label"><div class="label">†</div><div id="core-t1fn2" role="paragraph" data-to-manipulate="true">Ancestry was inferred on the basis of a random forest model trained on principal component analysis of the five super populations described for the 1000 Genomes Project.</div></div></div></figcaption></a><figcaption><div class="caption">Demographic Characteristics of the Probands (Including Inferred Ancestry) in the 100,000 Genomes Project Pilot Study.<a href="#core-t1fn1" role="doc-noteref">*</a></div></figcaption></figure><div class="external-links"><a class="pill__to-original" href="#t1"><i class="icon-return" aria-hidden="true"></i><span>Go to Table</span></a><a class="open-in-viewer" href="#t1" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t1.jpg"><span>Open in Viewer</span><i class="icon-expand" aria-hidden="true"></i></a></div></div><div id="core-t2"><figure class="table"><a class="open-in-viewer" href="#t2" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t2.jpg"><div class="graphic-wrap" data-presentation="suppressed-in-viewer"><img src="/cms/10.1056/NEJMoa2035790/asset/9de2c9af-b71b-43a6-8296-12f7e7d69943/assets/images/large/nejmoa2035790_t2.jpg" height="1694" width="1999" loading="lazy"></div><div class="table-wrap catalyst-standard" data-presentation="suppressed-in-flow"><table data-shading="custom"><thead><tr class="head1" data-type="row head1"><th class="txlx-borders" data-xml-align="left">Disease Category</th><th class="txxx-borders">All Family Structures<br>(2183 Probands)</th><th class="txxx-borders">Singleton<br>(881 Probands)</th><th class="txxx-borders">Family Duo<br>(343 Probands)</th><th class="txxx-borders">Family Trio<br>(797 Probands)</th><th class="txxr-borders">Larger Family Structure<br>(162 Probands)</th></tr><tr class="head3" data-type="row head3"><td class="xxlx-borders">&nbsp;</td><th class="xxxr-borders" colspan="5"><span>number of probands (percent)</span></th></tr></thead><tbody><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Cardiovascular disorder</td><td class="xxxx-borders shading">147 (7)</td><td class="xxxx-borders shading">56 (6)</td><td class="xxxx-borders shading">24 (7)</td><td class="xxxx-borders shading">49 (6)</td><td class="xxxr-borders shading">18 (11)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Ciliopathy</td><td class="xxxx-borders">69 (3)</td><td class="xxxx-borders">34 (4)</td><td class="xxxx-borders">14 (4)</td><td class="xxxx-borders">16 (2)</td><td class="xxxr-borders">5 (3)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Dermatologic disorder</td><td class="xxxx-borders shading">38 (2)</td><td class="xxxx-borders shading">9 (1)</td><td class="xxxx-borders shading">5 (1)</td><td class="xxxx-borders shading">22 (3)</td><td class="xxxr-borders shading">2 (1)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Dysmorphic or congenital abnormality</td><td class="xxxx-borders">20 (1)</td><td class="xxxx-borders">10 (1)</td><td class="xxxx-borders">2 (1)</td><td class="xxxx-borders">7 (1)</td><td class="xxxr-borders">1 (1)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Endocrine disorder</td><td class="xxxx-borders shading">87 (4)</td><td class="xxxx-borders shading">57 (6)</td><td class="xxxx-borders shading">14 (4)</td><td class="xxxx-borders shading">12 (2)</td><td class="xxxr-borders shading">4 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Gastroenterologic disorder</td><td class="xxxx-borders">32 (1)</td><td class="xxxx-borders">0</td><td class="xxxx-borders">0</td><td class="xxxx-borders">18 (2)</td><td class="xxxr-borders">14 (9)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Growth disorder</td><td class="xxxx-borders shading">3 (&lt;1)</td><td class="xxxx-borders shading">3 (&lt;1)</td><td class="xxxx-borders shading">0</td><td class="xxxx-borders shading">0</td><td class="xxxr-borders shading">0</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Hematologic or immunologic disorder</td><td class="xxxx-borders">5 (&lt;1)</td><td class="xxxx-borders">2 (&lt;1)</td><td class="xxxx-borders">3 (1)</td><td class="xxxx-borders">0</td><td class="xxxr-borders">0</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Hematologic disorder</td><td class="xxxx-borders shading">7 (&lt;1)</td><td class="xxxx-borders shading">0</td><td class="xxxx-borders shading">3 (1)</td><td class="xxxx-borders shading">2 (&lt;1)</td><td class="xxxr-borders shading">2 (1)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Hearing or ear disorder</td><td class="xxxx-borders">35 (2)</td><td class="xxxx-borders">6 (1)</td><td class="xxxx-borders">5 (1)</td><td class="xxxx-borders">17 (2)</td><td class="xxxr-borders">7 (4)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Metabolic disorder</td><td class="xxxx-borders shading">93 (4)</td><td class="xxxx-borders shading">24 (3)</td><td class="xxxx-borders shading">12 (3)</td><td class="xxxx-borders shading">48 (6)</td><td class="xxxr-borders shading">9 (6)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Intellectual disability</td><td class="xxxx-borders">130 (6)</td><td class="xxxx-borders">10 (1)</td><td class="xxxx-borders">24 (7)</td><td class="xxxx-borders">78 (10)</td><td class="xxxr-borders">18 (11)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Neurologic or neurodevelopmental disorder<a href="#core-t2fn2" role="doc-noteref">†</a></td><td class="xxxx-borders shading">521 (24)</td><td class="xxxx-borders shading">193 (22)</td><td class="xxxx-borders shading">93 (27)</td><td class="xxxx-borders shading">194 (24)</td><td class="xxxr-borders shading">41 (25)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Ophthalmologic disorder</td><td class="xxxx-borders">348 (16)</td><td class="xxxx-borders">74 (8)</td><td class="xxxx-borders">62 (18)</td><td class="xxxx-borders">199 (25)</td><td class="xxxr-borders">13 (8)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Renal and urinary tract disorder</td><td class="xxxx-borders shading">176 (8)</td><td class="xxxx-borders shading">125 (14)</td><td class="xxxx-borders shading">21 (6)</td><td class="xxxx-borders shading">26 (3)</td><td class="xxxr-borders shading">4 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Respiratory disorder</td><td class="xxxx-borders">2 (&lt;1)</td><td class="xxxx-borders">1 (&lt;1)</td><td class="xxxx-borders">0</td><td class="xxxx-borders">1 (&lt;1)</td><td class="xxxr-borders">0</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Rheumatologic disorder</td><td class="xxxx-borders shading">48 (2)</td><td class="xxxx-borders shading">14 (2)</td><td class="xxxx-borders shading">6 (2)</td><td class="xxxx-borders shading">25 (3)</td><td class="xxxr-borders shading">3 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Skeletal disorder</td><td class="xxxx-borders">62 (3)</td><td class="xxxx-borders">15 (2)</td><td class="xxxx-borders">11 (3)</td><td class="xxxx-borders">23 (3)</td><td class="xxxr-borders">13 (8)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Tumor syndrome</td><td class="xxxx-borders shading">293 (13)</td><td class="xxxx-borders shading">231 (26)</td><td class="xxxx-borders shading">31 (9)</td><td class="xxxx-borders shading">27 (3)</td><td class="xxxr-borders shading">4 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01" data-xml-align="left">Other</td><td class="xxxx-borders">67 (3)</td><td class="xxxx-borders">17 (2)</td><td class="xxxx-borders">12 (3)</td><td class="xxxx-borders">34 (4)</td><td class="xxxr-borders">4 (2)</td></tr><tr data-xml-align="center" data-type="row"><td class="xblx-borders hanging01 shading" data-xml-align="left">Total</td><td class="xbxx-borders shading">2183 (100)</td><td class="xbxx-borders shading">881 (100)</td><td class="xbxx-borders shading">343 (100)</td><td class="xbxx-borders shading">797 (100)</td><td class="xbxr-borders shading">162 (100)</td></tr></tbody></table></div><figcaption><div class="notes"><div role="doc-footnote" data-has="label"><div class="label">*</div><div id="core-t2fn1" role="paragraph" data-to-manipulate="true">Singleton refers to a proband for whom no other family member was recruited, family duo to a parent–proband pair, family trio to both parents and a proband, and family quad to a proband, sibling, and parents.</div></div><div role="doc-footnote" data-has="label"><div class="label">†</div><div id="core-t2fn2" role="paragraph" data-to-manipulate="true">Neurologic and neurodevelopmental symptoms excluded intellectual disability.</div></div></div></figcaption></a><figcaption><div class="caption">Disease Categories among the Probands in the 100,000 Genomes Project Pilot Study.<a href="#core-t2fn1" role="doc-noteref">*</a></div></figcaption></figure><div class="external-links"><a class="pill__to-original" href="#t2"><i class="icon-return" aria-hidden="true"></i><span>Go to Table</span></a><a class="open-in-viewer" href="#t2" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t2.jpg"><span>Open in Viewer</span><i class="icon-expand" aria-hidden="true"></i></a></div></div><div id="core-t3"><figure class="table"><a class="open-in-viewer" href="#t3" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t3.jpg"><div class="graphic-wrap" data-presentation="suppressed-in-viewer"><img src="/cms/10.1056/NEJMoa2035790/asset/4310832b-c338-4646-a091-5390e21fe9b0/assets/images/large/nejmoa2035790_t3.jpg" height="751" width="1711" loading="lazy"></div><div class="table-wrap catalyst-standard" data-presentation="suppressed-in-flow"><table data-shading="custom"><thead><tr class="head1" data-type="row head1"><th class="txlx-borders" data-xml-align="left">Variable</th><th class="txxx-borders">All Family Structures<br>(2183 Probands)</th><th class="txxx-borders">Singleton<br>(881 Probands)</th><th class="txxx-borders">Family Duo<br>(343 Probands)</th><th class="txxx-borders">Family Trio<br>(797 Probands)</th><th class="txxr-borders">Larger Family Structure<br>(162 Probands)</th></tr></thead><tbody><tr data-xml-align="center" data-type="row"><td class="xxlx-borders hanging01 shading" data-xml-align="left">Median no. of variants returned after filtering (IQR)<a href="#core-t3fn2" role="doc-noteref">†</a></td><td class="xxxx-borders shading">221 (49–288)</td><td class="xxxx-borders shading">292 (258–327)</td><td class="xxxx-borders shading">149 (117–213)</td><td class="xxxx-borders shading">29 (17–136)</td><td class="xxxr-borders shading">22 (9–71)</td></tr><tr data-xml-align="center" data-type="row"><td class="xblx-borders hanging01" data-xml-align="left">Median no. of variants in virtual panels returned after filtering (IQR)</td><td class="xbxx-borders">1 (0–2)</td><td class="xbxx-borders">1 (0–2)</td><td class="xbxx-borders">1 (0–3)</td><td class="xbxx-borders">1 (0–2)</td><td class="xbxr-borders">0 (0–1)</td></tr></tbody></table></div><figcaption><div class="notes"><div role="doc-footnote" data-has="label"><div class="label">*</div><div id="core-t3fn1" role="paragraph" data-to-manipulate="true">Singleton refers to a proband for whom no other family member was recruited, family duo to a parent–proband pair, family trio to both parents and a proband, and family quad to a proband, sibling, and parents.</div></div><div role="doc-footnote" data-has="label"><div class="label">†</div><div id="core-t3fn2" role="paragraph" data-to-manipulate="true">Filtering is the removal of extremely unlikely candidate variants.</div></div></div></figcaption></a><figcaption><div class="caption">Candidate Variants Returned for Presentation to the NHS Genomic Medicine Centres per Proband with the Automated Virtual Panel–Based Analysis Pipeline.<a href="#core-t3fn1" role="doc-noteref">*</a></div></figcaption></figure><div class="external-links"><a class="pill__to-original" href="#t3"><i class="icon-return" aria-hidden="true"></i><span>Go to Table</span></a><a class="open-in-viewer" href="#t3" data-interactiontype="multimedia_click" data-multimedia-type="Table" data-multimedia-filename="nejmoa2035790_t3.jpg"><span>Open in Viewer</span><i class="icon-expand" aria-hidden="true"></i></a></div></div></div></div><div id="core-collateral-share" role="tabpanel" data-core-tabs="core-collateral-share" tabindex="-1" aria-labelledby="pane-core-collateral-share"><header><h2 id="pane-core-collateral-share"><i class="icon-share" aria-hidden="true"></i>Share</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><h3>Share</h3><div class="section--wrapper"><section><h4>CONTENT LINK</h4><div id="share-self"><p data-id="article-share-self-link" class="share-self__source">https://www.nejm.org/doi/full/10.1056/NEJMoa2035790</p><button data-id="article-share-access" aria-label="CONTENT LINK" class="share-self__action btn btn--inverse"><i aria-hidden="true" class="icon-copy-o"></i><span>Copy Link</span></button><div aria-live="polite" class="share-self__status"><p class="share-self__success"><i aria-hidden="true" class="icon-check_circle"></i><span>Copied!</span></p><p class="share-self__failed"><i aria-hidden="true" class="icon-x_btnclose"></i><span>Copying failed.</span></p></div></div></section><section data-location="share_tools_article"><h4>Share</h4><div class="share-buttons a2a a2a_kit" style="line-height: 16px;"><a href="/#facebook" aria-label="Share on Facebook" data-id="article-share-facebook" rel="nofollow noopener" role="link" target="_blank" title="Share on Facebook" class="btn btn--facebook a2a_button_facebook" data-interactiontype="social"><i aria-hidden="true" class="icon-facebook"></i><span>Facebook</span></a><a href="/#twitter" aria-label="Share on X (formerly Twitter)" data-id="article-share-twitter" rel="nofollow noopener" role="link" target="_blank" title="Share on X (formerly Twitter)" class="btn btn--twitter a2a_button_twitter" data-interactiontype="social"><i aria-hidden="true" class="icon-twitter"></i><span>X (formerly Twitter)</span></a><a href="/#linkedin" aria-label="Share on LinkedIn" data-id="article-share-linkedin" rel="nofollow noopener" role="link" target="_blank" title="Share on LinkedIn" class="btn btn--linkedin a2a_button_linkedin" data-interactiontype="social"><i aria-hidden="true" class="icon-linkedin"></i><span>LinkedIn</span></a><a href="/#email" aria-label="Share on email" data-id="article-share-email" rel="nofollow noopener" role="link" target="_blank" title="Share on email" class="btn btn--email a2a_button_email" data-interactiontype="content_email_click"><i aria-hidden="true" class="icon-mail"></i><span>email</span></a><a href="/#bluesky" aria-label="Share on Bluesky" data-id="article-share-bluesky" rel="nofollow noopener" role="link" target="_blank" title="Share on Bluesky" class="btn btn--bluesky a2a_button_bluesky" data-interactiontype="social"><i aria-hidden="true" class="icon-bluesky"></i><span>Bluesky</span></a></div></section></div></div><div data-location="references_article" id="core-collateral-references" role="tabpanel" aria-labelledby="pane-core-collateral-references" data-core-tabs="core-collateral-references" tabindex="-1"><header><h2 id="pane-core-collateral-references"><i aria-hidden="true" class="icon-references"></i>References</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><h3>References</h3><div role="list" class="section--wrapper"><div role="listitem" data-has="label"><div class="label">1.</div><div id="core-r1" class="citations"><div class="citation"><div class="citation-content">Generation Genome. Annual report of the chief medical officer. 2016 (<a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/631043/CMO_annual_report_generation_genome.pdf">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/631043/CMO_annual_report_generation_genome.pdf</a>).</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r1-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Annual+report+of+the+chief+medical+officer&amp;publication_year=2016" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r1-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] 6% of the population in Western societies. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r1-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] disease die before their fifth birthday. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">2.</div><div id="core-r2" class="citations"><div class="citation"><div class="citation-content">Ferreira CR. The burden of rare diseases. <em>Am J Med Genet A</em> 2019;179:885-892.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r2"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajmg.a.61124" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30883013/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000468322800003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=The+burden+of+rare+diseases.&amp;publication_year=2019&amp;journal=Am+J+Med+Genet+A&amp;pages=885-892&amp;doi=10.1002%2Fajmg.a.61124&amp;pmid=30883013" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">3.</div><div id="core-r3" class="citations"><div class="citation"><div class="citation-content">Boycott KM, Rath A, Chong JX, et al. International cooperation to enable the diagnosis of all rare genetic diseases. <em>Am J Hum Genet</em> 2017;100:695-705.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r3-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2017.04.003" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28475856/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000400560000001" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=International+cooperation+to+enable+the+diagnosis+of+all+rare+genetic+diseases.&amp;publication_year=2017&amp;journal=Am+J+Hum+Genet&amp;pages=695-705&amp;doi=10.1016%2Fj.ajhg.2017.04.003&amp;pmid=28475856" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r3-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r5-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] of rare diseases over the past decade. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] after standard diagnostic testing. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">4.</div><div id="core-r4" class="citations"><div class="citation"><div class="citation-content">Taylor JC, Martin HC, Lise S, et al. Factors influencing success of clinical genome sequencing across a broad spectrum of disorders. <em>Nat Genet</em> 2015;47:717-726.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r4-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3304" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25985138/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000357090300008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Factors+influencing+success+of+clinical+genome+sequencing+across+a+broad+spectrum+of+disorders.&amp;publication_year=2015&amp;journal=Nat+Genet&amp;pages=717-726&amp;doi=10.1038%2Fng.3304&amp;pmid=25985138" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r4-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r5-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] of rare diseases over the past decade. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] after standard diagnostic testing. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r4-3" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] clinical phenotyping of the proband. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">5.</div><div id="core-r5" class="citations"><div class="citation"><div class="citation-content">Splinter K, Adams DR, Bacino CA, et al. Effect of genetic diagnosis on patients with previously undiagnosed disease. <em>N Engl J Med</em> 2018;379:2131-2139.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r5-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_6_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa2035790&amp;key=10.1056%2FNEJMoa1714458&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30304647/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000451402500008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Effect+of+genetic+diagnosis+on+patients+with+previously+undiagnosed+disease.&amp;publication_year=2018&amp;journal=N+Engl+J+Med&amp;pages=2131-2139&amp;doi=10.1056%2FNEJMoa1714458&amp;pmid=30304647" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r5-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r5-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] of rare diseases over the past decade. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] after standard diagnostic testing. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-3" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] Genomics England research environment). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-4" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>d [...] had previously undergone exome sequencing. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r5-5" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>e [...] sequence analysis in 382 patients, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">6.</div><div id="core-r6" class="citations"><div class="citation"><div class="citation-content">Genomics England. <em>The 100,000 Genomes Project protocol.</em> 2017 (<a href="https://figshare.com/articles/journal_contribution/GenomicEnglandProtocol_pdf/4530893/4">https://figshare.com/articles/journal_contribution/GenomicEnglandProtocol_pdf/4530893/4</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r6"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=The+100%2C000+Genomes+Project+protocol.&amp;publication_year=2017" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">7.</div><div id="core-r7" class="citations"><div class="citation"><div class="citation-content">Köhler S, Carmody L, Vasilevsky N, et al. Expansion of the Human Phenotype Ontology (HPO) knowledge base and resources. <em>Nucleic Acids Res</em> 2019;47:D1018-D1027.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r7"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1105" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30476213/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000462587400141" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Expansion+of+the+Human+Phenotype+Ontology+%28HPO%29+knowledge+base+and+resources.&amp;publication_year=2019&amp;journal=Nucleic+Acids+Res&amp;pages=D1018-D1027&amp;doi=10.1093%2Fnar%2Fgky1105&amp;pmid=30476213" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">8.</div><div id="core-r8" class="citations"><div class="citation"><div class="citation-content">Genomics England. <em>Rare disease conditions clinical data models</em>. 2018 (<a href="https://www.genomicsengland.co.uk/?wpdmdl=5500">https://www.genomicsengland.co.uk/?wpdmdl=5500</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r8"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Rare+disease+conditions+clinical+data+models&amp;publication_year=2018" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">9.</div><div id="core-r9" class="citations"><div class="citation"><div class="citation-content">Bentley DR, Balasubramanian S, Swerdlow HP, et al. Accurate whole human genome sequencing using reversible terminator chemistry. <em>Nature</em> 2008;456:53-59.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r9"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature07517" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18987734/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000260674000039" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Accurate+whole+human+genome+sequencing+using+reversible+terminator+chemistry.&amp;publication_year=2008&amp;journal=Nature&amp;pages=53-59&amp;doi=10.1038%2Fnature07517&amp;pmid=18987734" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">10.</div><div id="core-r10" class="citations"><div class="citation"><div class="citation-content">Rimmer A, Phan H, Mathieson I, et al. Integrating mapping-, assembly- and haplotype-based approaches for calling variants in clinical sequencing applications. <em>Nat Genet</em> 2014;46:912-918.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r10"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3036" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25017105/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000339704400024" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Integrating+mapping-%2C+assembly-+and+haplotype-based+approaches+for+calling+variants+in+clinical+sequencing+applications.&amp;publication_year=2014&amp;journal=Nat+Genet&amp;pages=912-918&amp;doi=10.1038%2Fng.3036&amp;pmid=25017105" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">11.</div><div id="core-r11" class="citations"><div class="citation"><div class="citation-content">Martin AR, Williams E, Foulger RE, et al. PanelApp crowdsources expert knowledge to establish consensus diagnostic gene panels. <em>Nat Genet</em> 2019;51:1560-1565.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r11"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0528-2" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31676867/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000494714300002" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=PanelApp+crowdsources+expert+knowledge+to+establish+consensus+diagnostic+gene+panels.&amp;publication_year=2019&amp;journal=Nat+Genet&amp;pages=1560-1565&amp;doi=10.1038%2Fs41588-019-0528-2&amp;pmid=31676867" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">12.</div><div id="core-r12" class="citations"><div class="citation"><div class="citation-content">Smedley D, Jacobsen JOB, Jäger M, et al. Next-generation diagnostics and disease-gene discovery with the Exomiser. <em>Nat Protoc</em> 2015;10:2004-2015.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2015.124" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26562621/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000366122600008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation+diagnostics+and+disease-gene+discovery+with+the+Exomiser.&amp;publication_year=2015&amp;journal=Nat+Protoc&amp;pages=2004-2015&amp;doi=10.1038%2Fnprot.2015.124&amp;pmid=26562621" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">13.</div><div id="core-r13" class="citations"><div class="citation"><div class="citation-content"><em>Congenica home page</em> (<a href="https://www.congenica.com/platform">https://www.congenica.com/platform</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r14"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Congenica+home+page" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">14.</div><div id="core-r14" class="citations"><div class="citation"><div class="citation-content"><em>Fabric Genomics home page</em> (<a href="https://fabricgenomics.com/">https://fabricgenomics.com/</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r14"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Fabric+Genomics+home+page" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">15.</div><div id="core-r15" class="citations"><div class="citation"><div class="citation-content">Richards S, Aziz N, Bale S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. <em>Genet Med</em> 2015;17:405-424.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r15-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2015.30" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25741868/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000354096900013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Standards+and+guidelines+for+the+interpretation+of+sequence+variants%3A+a+joint+consensus+recommendation+of+the+American+College+of+Medical+Genetics+and+Genomics+and+the+Association+for+Molecular+Pathology.&amp;publication_year=2015&amp;journal=Genet+Med&amp;pages=405-424&amp;doi=10.1038%2Fgim.2015.30&amp;pmid=25741868" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r15-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r15-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] report was issued for each proband. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r15-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] On the basis of international guidelines, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">16.</div><div id="core-r16" class="citations"><div class="citation"><div class="citation-content">Havrilla JM, Pedersen BS, Layer RM, Quinlan AR. A map of constrained coding regions in the human genome. <em>Nat Genet</em> 2019;51:88-95.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r16-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0294-6" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30531870/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000454108800016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+map+of+constrained+coding+regions+in+the+human+genome.&amp;publication_year=2019&amp;journal=Nat+Genet&amp;pages=88-95&amp;doi=10.1038%2Fs41588-018-0294-6&amp;pmid=30531870" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r16-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r16-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] in highly constrained coding regions </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r16-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] in highly constrained coding regions. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">17.</div><div id="core-r17" class="citations"><div class="citation"><div class="citation-content">Wei W, Tuna S, Keogh MJ, et al. Germline selection shapes human mitochondrial DNA diversity. <em>Science</em> 2019;364(6442):eaau6520-eaau6520.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r17"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau6520" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31123110/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000469296000033" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Germline+selection+shapes+human+mitochondrial+DNA+diversity.&amp;publication_year=2019&amp;journal=Science&amp;pages=eaau6520-eaau6520&amp;doi=10.1126%2Fscience.aau6520&amp;pmid=31123110" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">18.</div><div id="core-r18" class="citations"><div class="citation"><div class="citation-content">Smedley D, Schubach M, Jacobsen JOB, et al. A whole-genome analysis framework for effective identification of pathogenic regulatory variants in Mendelian disease. <em>Am J Hum Genet</em> 2016;99:595-606.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r18"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2016.07.005" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27569544/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000383114800006" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+whole-genome+analysis+framework+for+effective+identification+of+pathogenic+regulatory+variants+in+Mendelian+disease.&amp;publication_year=2016&amp;journal=Am+J+Hum+Genet&amp;pages=595-606&amp;doi=10.1016%2Fj.ajhg.2016.07.005&amp;pmid=27569544" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">19.</div><div id="core-r19" class="citations"><div class="citation"><div class="citation-content">Dolzhenko E, van Vugt JJFA, Shaw RJ, et al. Detection of long repeat expansions from PCR-free whole-genome sequence data. <em>Genome Res</em> 2017;27:1895-1903.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r19"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.225672.117" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28887402/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000414165900010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Detection+of+long+repeat+expansions+from+PCR-free+whole-genome+sequence+data.&amp;publication_year=2017&amp;journal=Genome+Res&amp;pages=1895-1903&amp;doi=10.1101%2Fgr.225672.117&amp;pmid=28887402" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">20.</div><div id="core-r20" class="citations"><div class="citation"><div class="citation-content">Zhang L, Bai W, Yuan N, Du Z. Comprehensively benchmarking applications for detecting copy number variation. <em>PLoS Comput Biol</em> 2019;15(5):e1007069-e1007069.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r20"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1007069" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31136576/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000471040500066" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensively+benchmarking+applications+for+detecting+copy+number+variation.&amp;publication_year=2019&amp;journal=PLoS+Comput+Biol&amp;pages=e1007069-e1007069&amp;doi=10.1371%2Fjournal.pcbi.1007069&amp;pmid=31136576" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">21.</div><div id="core-r21" class="citations"><div class="citation"><div class="citation-content">Kosugi S, Momozawa Y, Liu X, Terao C, Kubo M, Kamatani Y. Comprehensive evaluation of structural variation detection algorithms for whole genome sequencing. <em>Genome Biol</em> 2019;20:117-117.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r21"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1720-5" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31159850/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000470146500003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive+evaluation+of+structural+variation+detection+algorithms+for+whole+genome+sequencing.&amp;publication_year=2019&amp;journal=Genome+Biol&amp;pages=117-117&amp;doi=10.1186%2Fs13059-019-1720-5&amp;pmid=31159850" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">22.</div><div id="core-r22" class="citations"><div class="citation"><div class="citation-content">Genomics England. Rare disease conditions eligibility criteria. 2018 (<a href="https://www.genomicsengland.co.uk/wp-content/uploads/2018/06/Rare-Disease-Eligibility-Criteria-v1.9.0-PAR-GUI-058_approved-version-1.pdf">https://www.genomicsengland.co.uk/wp-content/uploads/2018/06/Rare-Disease-Eligibility-Criteria-v1.9.0-PAR-GUI-058_approved-version-1.pdf</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r22"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Rare+disease+conditions+eligibility+criteria&amp;publication_year=2018" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">23.</div><div id="core-r23" class="citations"><div class="citation"><div class="citation-content">Office for National Statistics. <em>2011 Census</em> (<a href="https://www.ons.gov.uk/census/2011census">https://www.ons.gov.uk/census/2011census</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r23"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=2011+Census" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">24.</div><div id="core-r24" class="citations"><div class="citation"><div class="citation-content">Landrum MJ, Lee JM, Benson M, et al. ClinVar: improving access to variant interpretations and supporting evidence. <em>Nucleic Acids Res</em> 2018;46:D1062-D1067.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r24"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1153" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29165669/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000419550700154" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=ClinVar%3A+improving+access+to+variant+interpretations+and+supporting+evidence.&amp;publication_year=2018&amp;journal=Nucleic+Acids+Res&amp;pages=D1062-D1067&amp;doi=10.1093%2Fnar%2Fgkx1153&amp;pmid=29165669" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">25.</div><div id="core-r25" class="citations"><div class="citation"><div class="citation-content">Carss KJ, Arno G, Erwood M, et al. Comprehensive rare variant analysis via whole-genome sequencing to determine the molecular pathology of inherited retinal disease. <em>Am J Hum Genet</em> 2017;100:75-90.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r27-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2016.12.003" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28041643/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000391910500007" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive+rare+variant+analysis+via+whole-genome+sequencing+to+determine+the+molecular+pathology+of+inherited+retinal+disease.&amp;publication_year=2017&amp;journal=Am+J+Hum+Genet&amp;pages=75-90&amp;doi=10.1016%2Fj.ajhg.2016.12.003&amp;pmid=28041643" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">26.</div><div id="core-r26" class="citations"><div class="citation"><div class="citation-content">Rowlands C, Thomas HB, Lord J, et al. Comparison of in silico strategies to prioritize rare genomic variants impacting RNA splicing for the diagnosis of genomic disorders. <em>Sci Rep</em> 2021;11:20607-20607.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r27-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-021-99747-2" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34663891/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000741362900004" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Comparison+of+in+silico+strategies+to+prioritize+rare+genomic+variants+impacting+RNA+splicing+for+the+diagnosis+of+genomic+disorders.&amp;publication_year=2021&amp;journal=Sci+Rep&amp;pages=20607-20607&amp;doi=10.1038%2Fs41598-021-99747-2&amp;pmid=34663891" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">27.</div><div id="core-r27" class="citations"><div class="citation"><div class="citation-content">Radziwon A, Arno G, Wheaton DK, et al. Single-base substitutions in the CHM promoter as a cause of choroideremia. <em>Hum Mutat</em> 2017;38:704-715.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r27-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/humu.23212" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28271586/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000401141900010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Single-base+substitutions+in+the+CHM+promoter+as+a+cause+of+choroideremia.&amp;publication_year=2017&amp;journal=Hum+Mutat&amp;pages=704-715&amp;doi=10.1002%2Fhumu.23212&amp;pmid=28271586" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r27-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r27-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] minigene, or luciferase assays. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r27-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] variant causing loss of gene expression, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">28.</div><div id="core-r28" class="citations"><div class="citation"><div class="citation-content">Farazi Fard MA, Rebelo AP, Buglo E, et al. Truncating mutations in UBAP1 cause hereditary spastic paraplegia. <em>Am J Hum Genet</em> 2019;104:767-773.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r28"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2019.03.001" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30929741/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000463474700017" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Truncating+mutations+in+UBAP1+cause+hereditary+spastic+paraplegia.&amp;publication_year=2019&amp;journal=Am+J+Hum+Genet&amp;pages=767-773&amp;doi=10.1016%2Fj.ajhg.2019.03.001&amp;pmid=30929741" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">29.</div><div id="core-r29" class="citations"><div class="citation"><div class="citation-content">Wallmeier J, Frank D, Shoemark A, et al. De novo mutations in FOXJ1 result in a motile ciliopathy with hydrocephalus and randomization of left/right body asymmetry. <em>Am J Hum Genet</em> 2019;105:1030-1039.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r29"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ajhg.2019.09.022" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31630787/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000495878800014" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=De+novo+mutations+in+FOXJ1+result+in+a+motile+ciliopathy+with+hydrocephalus+and+randomization+of+left%2Fright+body+asymmetry.&amp;publication_year=2019&amp;journal=Am+J+Hum+Genet&amp;pages=1030-1039&amp;doi=10.1016%2Fj.ajhg.2019.09.022&amp;pmid=31630787" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">30.</div><div id="core-r30" class="citations"><div class="citation"><div class="citation-content">Cortese A, Zhu Y, Rebelo AP, et al. Biallelic mutations in <i>SORD</i> cause a common and potentially treatable hereditary neuropathy with implications for diabetes. <em>Nat Genet</em> 2020;52:473-481.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r30"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0615-4" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32367058/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000530227600002" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Biallelic+mutations+in+SORD+cause+a+common+and+potentially+treatable+hereditary+neuropathy+with+implications+for+diabetes.&amp;publication_year=2020&amp;journal=Nat+Genet&amp;pages=473-481&amp;doi=10.1038%2Fs41588-020-0615-4&amp;pmid=32367058" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">31.</div><div id="core-r31" class="citations"><div class="citation"><div class="citation-content">Haendel MA, Chute CG, Robinson PN. Classification, ontology, and precision medicine. <em>N Engl J Med</em> 2018;379:1452-1462.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r31"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_32_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa2035790&amp;key=10.1056%2FNEJMra1615014&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30304648/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000446923400009" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Classification%2C+ontology%2C+and+precision+medicine.&amp;publication_year=2018&amp;journal=N+Engl+J+Med&amp;pages=1452-1462&amp;doi=10.1056%2FNEJMra1615014&amp;pmid=30304648" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">32.</div><div id="core-r32" class="citations"><div class="citation"><div class="citation-content">Yang Y, Muzny DM, Xia F, et al. Molecular findings among patients referred for clinical whole-exome sequencing. <em>JAMA</em> 2014;312:1870-1879.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2014.14601" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25326635/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000344532600016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Molecular+findings+among+patients+referred+for+clinical+whole-exome+sequencing.&amp;publication_year=2014&amp;journal=JAMA&amp;pages=1870-1879&amp;doi=10.1001%2Fjama.2014.14601&amp;pmid=25326635" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">33.</div><div id="core-r33" class="citations"><div class="citation"><div class="citation-content">Hu X, Li N, Xu Y, et al. Proband-only medical exome sequencing as a cost-effective first-tier genetic diagnostic test for patients without prior molecular tests and clinical diagnosis in a developing country: the China experience. <em>Genet Med</em> 2018;20:1045-1053.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2017.195" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29095814/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000445531800018" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Proband-only+medical+exome+sequencing+as+a+cost-effective+first-tier+genetic+diagnostic+test+for+patients+without+prior+molecular+tests+and+clinical+diagnosis+in+a+developing+country%3A+the+China+experience.&amp;publication_year=2018&amp;journal=Genet+Med&amp;pages=1045-1053&amp;doi=10.1038%2Fgim.2017.195&amp;pmid=29095814" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">34.</div><div id="core-r34" class="citations"><div class="citation"><div class="citation-content">Vissers LELM, van Nimwegen KJM, Schieving JH, et al. A clinical utility study of exome sequencing versus conventional genetic testing in pediatric neurology. <em>Genet Med</em> 2017;19:1055-1063.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2017.1" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28333917/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000409525100014" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+clinical+utility+study+of+exome+sequencing+versus+conventional+genetic+testing+in+pediatric+neurology.&amp;publication_year=2017&amp;journal=Genet+Med&amp;pages=1055-1063&amp;doi=10.1038%2Fgim.2017.1&amp;pmid=28333917" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">35.</div><div id="core-r35" class="citations"><div class="citation"><div class="citation-content">Gilissen C, Hehir-Kwa JY, Thung DT, et al. Genome sequencing identifies major causes of severe intellectual disability. <em>Nature</em> 2014;511:344-347.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r35"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13394" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24896178/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000338992200035" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genome+sequencing+identifies+major+causes+of+severe+intellectual+disability.&amp;publication_year=2014&amp;journal=Nature&amp;pages=344-347&amp;doi=10.1038%2Fnature13394&amp;pmid=24896178" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">36.</div><div id="core-r36" class="citations"><div class="citation"><div class="citation-content">Eichler EE. Genetic variation, comparative genomics, and the diagnosis of disease. <em>N Engl J Med</em> 2019;381:64-74.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r36"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_37_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa2035790&amp;key=10.1056%2FNEJMra1809315&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31269367/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000474094900013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genetic+variation%2C+comparative+genomics%2C+and+the+diagnosis+of+disease.&amp;publication_year=2019&amp;journal=N+Engl+J+Med&amp;pages=64-74&amp;doi=10.1056%2FNEJMra1809315&amp;pmid=31269367" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">37.</div><div id="core-r37" class="citations"><div class="citation"><div class="citation-content">NHS. <em>National Genomic Test Directory</em>. 2020 (<a href="https://www.england.nhs.uk/publication/national-genomic-test-directories/">https://www.england.nhs.uk/publication/national-genomic-test-directories/</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r37"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=National+Genomic+Test+Directory&amp;publication_year=2020" target="_blank">Google Scholar</a></div></div></div></div></div></div></div></div><aside data-core-aside="right-rail">



        
        <div id="DTM_Position_MedRectangle" class="ad">
 <!--emptycomment-->
</div>




        
        <div class="ng-page_layout-sidebarSpacing"></div>










    
    
        <div data-widget-def="relatedArticlesWidget" data-widget-id="8206d5cd-cb8c-4ff9-8c46-3d22e0dee215" data-location="recirc_curatedRelated_article">
        



        
        

        </div>
    





        
        <div class="ng-page_layout-sidebarSpacing"></div>




        
        <div id="nejm_jobs" class="nejm_jobs"> <!-- top red rectangle --><div id="nejm-widget" class="nejm-widget" style="display: block;"><div class="nejm-widget_logo">	<a target="nejm-win" href="https://www.nejmcareercenter.org/?query=fjw">		<img alt="CareerCenter" src="https://apps1.nejmcareercenter.org/images/nejmCCID-JobsWidget.gif" width="140" height="36">	</a></div><div class="nejm-widget_cont"> <!-- content --><div class="nejm-widget_title">PHYSICIAN JOBS</div><div class="nejm-widget_date">June 10, 2025</div><div class="nejm-widget_item"><div><span> New York</span></div><div><span>Neurology</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/883481/system-director-of-movement-disorders-northwell-health/?query=fjwp&amp;rid=3059">System Director of Movement Disorders - Northwell Health</a></div></div><div class="nejm-widget_item"><div><span> New York</span></div><div><span>Pathology</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/737820/renal-pathology-nyc-suburbs/?query=fjwp&amp;rid=3059">Renal Pathology - NYC Suburbs</a></div></div><div class="nejm-widget_item"><div><span> Florida</span></div><div><span>Family Medicine</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/880621/family-medicine-in-leesburg-fl-1-hour-from-orlando-earn-300k-/?query=fjwf&amp;rid=5127">Family Medicine in Leesburg, FL - 1 Hour from Orlando- Earn $300K+</a></div></div><div class="nejm-widget_item"><div><span> Santa Barbara, California</span></div><div><span>Maternal-Fetal Medicine</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/882911/maternal-fetal-medicine-physician/?query=fjwf&amp;rid=49812">Maternal Fetal Medicine Physician</a></div></div><div class="nejm-widget_item"><div><span> Billings, Montana</span></div><div><span>Critical Care Medicine</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/885110/physician-critical-care-icu-pulmonology/?query=fjwf&amp;rid=751">Physician Critical Care ICU Pulmonology</a></div></div><div class="nejm-widget_item"><div><span> Auburn, Massachusetts</span></div><div><span>Pediatrics, General</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/880853/pediatrician-reliant-medical-group/?query=fjwf&amp;rid=220874">Pediatrician- Reliant Medical Group</a></div></div></div> <!-- content --> <!-- footer --></div> <!-- nejm-widget --></div><script src="https://apps1.nejmcareercenter.org/?width=300&amp;sp=all"></script>




        
        <div class="ng-page_layout-sidebarSpacing"></div>










    
    
        <div data-widget-def="literatumAd" data-widget-id="001e4aa7-b6fe-4b21-9d5b-5bde81227c67" id="ad-article-right-rail-300x250-1">
        



        
        



    
        <div class="pb-house-message adplaceholder exists">
            <a href="/action/clickThrough?id=128720&amp;url=https%3A%2F%2Fstore.nejm.org%2Fsignup%2Fnejmai%2Fregister%2Faifallpdf24%3Fpromo%3DOAFQF4NA%26query%3Dcm-rr%26utm_source%3Dnejm%26utm_medium%3Dcm%26utm_campaign%3Daifallpdf24&amp;loc=%2Fdoi%2Ffull%2F10.1056%2FNEJMoa2035790&amp;pubId=41291304&amp;placeholderId=101337&amp;productId=1007" target="_blank"><img src="/sda/128720/NEJ-AI-Fall24-300x250---002.jpg" width="300" height="250"></a>
            
        </div>
    

    

    



        </div>
    





        
        <div class="ng-page_layout-sidebarSpacing"></div>




        
        <script src="https://widgets.nejm.org/onepub-widgets/loader/webcomponents-loader.js" async="" nonce="94d6a712622201a2-GRU">
<script src="https://widgets.nejm.org/onepub-widgets/loader/polyfill-support.js"  nonce="94d6a712622201a2-GRU"></script>
<script src="https://widgets.nejm.org/onepub-widgets/more-like-this.js" type="module" nonce="94d6a712622201a2-GRU"></script>

<more-like-this class="onepub-widget" dataid="onepub-more-like-this" collections="catalyst-article, clinician-article, evidence-article, nejm-ai-article, nejm-article" headertitle="More Like This" maxitems="5"><!----><div class="more-like-this onepub-more-like-this_"><!--?lit$348976082$--><onepub-fonts><wc-fetch></wc-fetch><!--?lit$348976082$--><div class="mlt-wrapper"><div class="mlt-header-container"><div class="mlt-header-content-left"><div class="mlt-header-title-container"><span class="mlt-header-title">More Like This</span></div><!--?lit$348976082$--></div></div><div class="mlt-body"><!--?lit$348976082$--><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm" href="https://www.nejm.org/browse/nejm-article-type/original-article" target="_self"><!--?lit$348976082$-->Original Article</a> <span class="mlt-article-site-label"><!--?lit$348976082$-->NEJM</span> <span class="mlt-article-pubdate"><!--?lit$348976082$-->Jun 06, 2024</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://www.nejm.org/doi/full/10.1056/NEJMoa2314761?query=recirc_Semantic" target="_self">Genome Sequencing for Diagnosing Rare Diseases</a></div><div class="mlt-article-authors"><!--?lit$348976082$-->M.H. Wojcik and Others</div></div><!----><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm" href="https://www.nejm.org/browse/nejm-article-type/perspective" target="_self"><!--?lit$348976082$-->Perspective</a> <span class="mlt-article-site-label"><!--?lit$348976082$-->NEJM</span> <span class="mlt-article-pubdate"><!--?lit$348976082$-->Mar 06, 2025</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://www.nejm.org/doi/full/10.1056/NEJMp2413309?query=recirc_Semantic" target="_self">Supporting the Development of Drugs for Rare Diseases — The Importance of Regulatory Transparency</a></div><div class="mlt-article-authors"><!--?lit$348976082$-->S.S. Dhruva, H.-G. Eichler, and S.K. Galson</div></div><!----><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm" href="https://www.nejm.org/browse/nejm-article-type/sounding-board" target="_self"><!--?lit$348976082$-->Sounding Board</a> <span class="mlt-article-site-label"><!--?lit$348976082$-->NEJM</span> <span class="mlt-article-pubdate"><!--?lit$348976082$-->Apr 10, 2025</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://www.nejm.org/doi/full/10.1056/NEJMsb2503956?query=recirc_Semantic" target="_self">From the National Academy of Medicine: Generative AI in Medicine — Evaluating Progress and Challenges</a></div><div class="mlt-article-authors"><!--?lit$348976082$-->T.M. Maddox and Others</div></div><!----><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm-catalyst" href="https://catalyst.nejm.org/browse/nejm-article-type/supplemental-issue?query=recirc_Semantic" target="_self"><!--?lit$348976082$-->Supplemental Issue</a> <span class="mlt-article-site-label"><!--?lit$348976082$-->NEJM Catalyst</span> <span class="mlt-article-pubdate"><!--?lit$348976082$-->Dec 22, 2023</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://catalyst.nejm.org/doi/full/10.1056/CAT.23.0297?query=recirc_Semantic" target="_self">A Patient-Centric, Coordinated Care Model for Rare Diseases: The Multidisciplinary Consultation Program at Peking Union Medical College Hospital</a></div><div class="mlt-article-authors"><!--?lit$348976082$-->J. Xie and Others</div></div><!----><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm-ai" href="https://ai.nejm.org/browse/nejm-article-type/review-article?query=recirc_Semantic" target="_self"><!--?lit$348976082$-->Review Article</a> <span class="mlt-article-site-label"><!--?lit$348976082$-->NEJM AI</span> <span class="mlt-article-pubdate"><!--?lit$348976082$-->Apr 24, 2025</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://ai.nejm.org/doi/full/10.1056/AIra2401164?query=recirc_Semantic" target="_self">The Use of Artificial Intelligence for Cancer Therapeutic Decision-Making</a></div><div class="mlt-article-authors"><!--?lit$348976082$-->O. Elemento, S. Khozin, and C.N. Sternberg</div></div><!----></div><div class="mlt-footer"></div></div></onepub-fonts><style>:host{display:block}@keyframes fadeIn{0%{opacity:0}100%{opacity:1}}.more-like-this{max-width:25rem}.more-like-this *{font-variant-numeric:lining-nums}.mlt-wrapper{animation:fadeIn 1s}.mlt-header-container{border-bottom:#4d4d4d solid .0625rem;border-top:#4d4d4d solid .125rem}.mlt-header-title{color:#1a1a1a;display:block;font-family:var(--sans-serif);font-size:1rem;font-weight:700;letter-spacing:.02rem;line-height:1.5rem;padding-bottom:.94rem;padding-top:.94rem;text-transform:uppercase}.mlt-header-description{color:#333;display:block;font-family:ff-scala-sans-pro;font-size:.75rem;letter-spacing:.05rem;margin-top:-.5rem;padding-bottom:1rem;text-transform:uppercase}.mlt-article-container{border-bottom:#e5e5e5 solid .0625rem;margin-top:1.38rem}.mlt-article-container-header{font-family:var(--sans-serif);font-size:.6875rem;letter-spacing:.04em;line-height:1.125rem;margin:0 0 1rem;text-transform:uppercase}.mlt-article-type{font-weight:700;line-height:1.25rem;letter-spacing:.04rem;text-decoration:none}.mlt-article-type:hover{text-decoration:underline}.nejm{color:#f30}.nejm-ai{color:#006197}.nejm-catalyst{color:#5708a6}.nejm-evidence{color:#69a323}.mlt-article-site-label::after,.mlt-article-type::after{border-right:1px solid #e5e5e5;content:'';display:inline;padding:.1875rem .375rem .125rem 0;margin:0 .375rem 0 0}.mlt-article-pubdate{color:#666;font-style:normal;font-weight:700;line-height:1.25rem;letter-spacing:.04em}.mlt-article-authors{color:#666;font-family:var(--sans-serif);font-size:1rem;font-style:normal;font-weight:400;letter-spacing:normal;line-height:1.5rem;margin-bottom:1.38rem}.mlt-article-site-label{color:#666;font-style:normal;font-weight:700;letter-spacing:.04em}.mlt-article-title{margin-bottom:.5rem}.mlt-article-title>a{background-image:linear-gradient(#f30,#f30);background-position:0 90%;background-repeat:no-repeat;background-size:0 0;color:#1a1a1a;font-family:var(--quadraat-web);font-size:1.25rem;font-style:normal;font-weight:700;letter-spacing:normal;line-height:1.8rem;margin:1.5rem 0 .35rem 0;text-decoration:none;transition:background .25s ease-in-out}.mlt-article-title>a:hover{background-position:0 98%;background-size:100% .125rem;text-decoration:none}.skeleton-card{box-shadow:0 0 4px 1px rgba(0,0,0,.1);margin:1rem 0;padding:1rem}</style></div></more-like-this>




        
        <div><div class="pb-dropzone" data-pb-dropzone="col-0" title="col-0"></div></div>










    
    
        <div data-widget-def="UX3HTMLWidget" data-widget-id="ad06db99-f228-4c92-9ad7-c4ae0fdbdbf1" id="bc-web-rec">
        



        
        <!-- BlueConic TEST empty HTML widget with DIV wrapper with id=bc-web-rec -->

        </div>
    

</aside></article><div data-extent="article-wrapper"><div class="core-container"></div><div class="after-credits"><div class="core-container"><a href="/doi/pdf/10.1056/NEJMoa2035790?download=true" id="downloadPdfUrl" data-doi="10.1056/NEJMoa2035790" data-behavior="trackDownloadEvent" data-interactiontype="article_tools_download_pdf" data-multimedia-type="Article PDF" data-multimedia-filename="nejmoa2035790.pdf"><span>Download PDF</span></a></div></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1056/NEJMoa2035790"}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><div class="articles-nav"></div><template id="figure_nav_template"><nav><a href="#" title="Open in viewer" class="open-in-viewer"><i aria-hidden="true" class="icon-expand"></i></a></nav></template><template id="fv_panel_template" data-sharesocial="Share"><div class="fv__panel js--hidden"><div class="fv__panel__text"><div class="fv__panel__contentTitle"></div><div class="fv__panel__contentText"></div></div></div></template>
<template id="fv_directory_template" data-overlay="View figure"><div class="fv__directory"><div class="fv__header"><nav class="tab__nav"></nav><a href="#" title="Close figure viewer" class="fv__close"><i aria-hidden="true" class="icon-close"></i></a></div><div class="fv__content tab__content"></div></div></template><template id="fv_directory_tabItem_template" data-figures="Figures" data-others="Others" data-tables="Tables"><button role="tab" data-toggle="tab" class="tab__nav__item"></button></template>
<template id="fv_lightbox_template" data-sr-back="Go to figure location within the article" data-sr-closealt="Close" data-sr-download="Toggle download panel" data-sr-download-pptx="Download PPT" data-sr-info="Toggle information panel" data-sr-panel="Close panel" data-sr-share="Toggle share panel" data-sr-zoom="Zoom" data-text-close="Back to article" data-text-nav-figures="All figures" data-text-nav-others="All others" data-text-nav-tables="All tables" data-text-viewall="View all material" data-title-back="Back to article" data-title-closealt="Close" data-title-download="Download" data-title-download-pptx="Download PPT" data-title-info="Info" data-title-share="Share" data-title-zoom="Zoom"></template>
<template id="fv_toolbar_template"><div class="fv__toolbar__info"><strong class="fv__toolbar__contentTitle uppercase"></strong><p class="fv__toolbar__contentText"></p></div></template><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-return"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-return"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-return"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--secondary"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-references="References" data-figure="Go to Figure" data-media="Go to Media" data-more="More" data-original="Go to original" data-table="Go to Table" data-inviewer="Open in Viewer" data-allinviewer="Open all in viewer"><svg data-tags="copyright" viewBox="0 0 24 24"><path d="M12 20.016q3.281 0 5.648-2.368T20.016 12t-2.368-5.648T12 3.984 6.352 6.352 3.984 12t2.368 5.648T12 20.016zm0-18q4.125 0 7.055 2.93T21.985 12t-2.93 7.055T12 21.985t-7.055-2.93T2.015 12t2.93-7.055T12 2.015zm-.14 7.125q-1.876 0-1.876 2.718v.282q0 2.718 1.875 2.718.704 0 1.172-.398t.469-1.008h1.781q0 1.172-1.031 2.063-.984.843-2.39.843-1.876 0-2.86-1.125t-.984-3.093v-.282q0-1.921.937-3 1.125-1.265 2.906-1.265 1.547 0 2.438.89.984.985.984 2.297H13.5q0-.328-.14-.61-.235-.468-.329-.562-.469-.468-1.172-.468z"></path></svg><span>Request permissions</span><i aria-hidden="true" class="icon-open_in_new"></i></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template>

        <div role="navigation" aria-label="Sticky Navigation" class="st-header"><div class="st-header__content"><div class="st-header__item st-header__menu"></div><div class="st-header__current st-header__item"><div class="st-header__label">Now Reading:</div><div class="st-header__title">100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care — Preliminary Report</div></div><div class="st-header__share st-header__item"><div class="share-dropblock dropdown"><a id="sticky-header-dropBlock" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="dropdown-toggle"><span title="" data-toggle="tooltip" data-trigger="hover" data-original-title="Share options"><span class="sr-only">Share</span><i aria-hidden="true" class="icon-share"></i></span></a><div aria-labelledby="sticky-header-dropBlock" data-location="share_tools_article" class="dropdown-menu"><ul class="rlist--inline a2a a2a_kit" style="line-height: 16px;"><li><a href="/#facebook" aria-label="Share on Facebook" data-id="article-share-facebook" rel="nofollow noopener" role="link" target="_blank" title="Share on Facebook" class="btn btn--facebook a2a_button_facebook" data-interactiontype="social"><i aria-hidden="true" class="icon-facebook"></i></a></li><li><a href="/#twitter" aria-label="Share on X (formerly Twitter)" data-id="article-share-twitter" rel="nofollow noopener" role="link" target="_blank" title="Share on X (formerly Twitter)" class="btn btn--twitter a2a_button_twitter" data-interactiontype="social"><i aria-hidden="true" class="icon-twitter"></i></a></li><li><a href="/#linkedin" aria-label="Share on LinkedIn" data-id="article-share-linkedin" rel="nofollow noopener" role="link" target="_blank" title="Share on LinkedIn" class="btn btn--linkedin a2a_button_linkedin" data-interactiontype="social"><i aria-hidden="true" class="icon-linkedin"></i></a></li><li><a href="/#email" aria-label="Share on email" data-id="article-share-email" rel="nofollow noopener" role="link" target="_blank" title="Share on email" class="btn btn--email a2a_button_email" data-interactiontype="content_email_click"><i aria-hidden="true" class="icon-mail"></i></a></li><li><a href="/#bluesky" aria-label="Share on Bluesky" data-id="article-share-bluesky" rel="nofollow noopener" role="link" target="_blank" title="Share on Bluesky" class="btn btn--bluesky a2a_button_bluesky" data-interactiontype="social"><i aria-hidden="true" class="icon-bluesky"></i></a></li></ul></div></div></div><div class="st-header__nav st-header__item"><div class="content-navigation"><a href="/doi/full/10.1056/NEJMoa2112435" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><div aria-hidden="true" class="content-navigation__hint"><div class="content-navigation__hint__content"><h6>PREVIOUS ARTICLE</h6><div>Pembrolizumab for Persistent, Recurrent, or Metastatic Cervical Cancer</div></div></div><i aria-hidden="true" class="icon-arrow-left"></i><span>Previous</span></a><a href="/doi/full/10.1056/NEJMra2030281" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><div aria-hidden="true" class="content-navigation__hint"><div class="content-navigation__hint__content"><h6>NEXT ARTICLE</h6><div>Pollution and the Heart</div></div></div><span>Next</span><i aria-hidden="true" class="icon-arrow-right"></i></a></div></div></div><div class="st-header__track"><div id="progress-tracker" class="st-header__tracker" style="width: 0px;"></div></div></div><div class="fv figureViewer js--has-focus-mode"><div class="fv__directory" data-core-tabs="default" style="display: flex;"><div class="fv__header"><nav class="tab__nav" role="tablist"><button role="tab" data-toggle="tab" class="tab__nav__item active" aria-controls="#tab-pane-figures" aria-selected="true">Figures</button><button role="tab" data-toggle="tab" class="tab__nav__item" aria-controls="#tab-pane-tables" tabindex="-1" aria-selected="false">Tables</button></nav><a href="#" title="Close figure viewer" class="fv__close"><i aria-hidden="true" class="icon-close"></i></a></div><div class="fv__content tab__content"><div class="tab__pane fv__list fv__list--figures" id="tab-pane-figures" aria-label="All figures" tabindex="0" role="tabpanel"><div class="fv__item"><a class="fv__open-item fv__item__view fv__fit" href="#fv-f1" data-type="figures" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f1.jpg"><img src="/cms/10.1056/NEJMoa2035790/asset/ee2525a0-274f-49b3-a2ee-6a621a475e47/assets/images/large/nejmoa2035790_f1.jpg" height="2845" width="2640" aria-labelledby="f1" loading="lazy"><span class="fv__fit--overlay"><span>View figure</span></span></a><div class="fv__item__title">Figure 1</div><div class="fv__item__description"><figcaption><div class="caption">Overview of the Diagnostic and Research Pipeline and Source of Diagnoses.</div><div class="notes"><div role="doc-footnote">Results from 2183 probands in the pilot study were returned for presentation to the Genomic Medicine Centres (GMCs) of the recruiting hospitals. A total of 25% of the probands received a positive diagnosis, and 10% had a variant or variants of unknown significance in genes that were determined by clinical geneticists at the recruiting site to be consistent with the phenotype but that required further functional validation. The remaining 65% of the probands received a negative report at the time but will be reassessed. The numbers and sources of these positive diagnoses are shown at each stage of the automated diagnostic pipeline, and the additional research is shown for diagnoses that were not immediately obvious. CCR denotes constrained coding region, indel insertion or deletion, mtDNA mitochondrial DNA, SNV single-nucleotide variant, and SV structural variant.</div></div></figcaption></div></div><div class="fv__item"><a class="fv__open-item fv__item__view fv__fit" href="#fv-f2" data-type="figures" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa2035790_f2.jpg"><img src="/cms/10.1056/NEJMoa2035790/asset/6ac3b65b-4124-4ff1-b695-0053a7cde6de/assets/images/large/nejmoa2035790_f2.jpg" height="3431" width="2640" aria-labelledby="f2" loading="lazy"><span class="fv__fit--overlay"><span>View figure</span></span></a><div class="fv__item__title">Figure 2</div><div class="fv__item__description"><figcaption><div class="caption">Diagnoses in the Rare Disease Pilot Study.</div><div class="notes"><div role="doc-footnote">Panel A shows diagnostic yield for any disease and according to family structure and cause of disease. The diagnostic yield was 35% for diseases likely to have a monogenic cause and 11% for diseases likely to have a complex cause. The values above the bars are the numbers of probands. Singleton refers to a proband for whom no other family member was recruited, family duo to a parent–proband pair, family trio to both parents and a proband, and family quad to a proband, sibling, and parents. Panel B shows diagnostic yield according to disease category. The values above the bars are the numbers of probands. Panel C shows the diagnostic yield among probands according to previous genetic testing and most extensive testing type: chromosomal (karyotyping, array-based comparative genomic hybridization, single-nucleotide polymorphism arrays), targeted (targeted single-gene tests), next-generation sequencing (NGS) panels, or whole-exome sequencing (WES). The values above the bars are the numbers of probands. Panel D shows the performance of virtual panel-based and Exomiser-based prioritization for identifying the diagnoses. “Disease panel only” indicates the use of a single virtual panel for the recruited disease category. “Applied panels” indicates the use of all applied virtual gene panels used in the pipeline, including the recruited disease–associated panel as well as 0 or more additional panels selected on the basis of the patient’s phenotypes (Human Phenotype Ontology terms). “Exomiser top” indicates Exomiser use in the top-ranked candidate variants, “Exomiser top 3” use in the top three candidates, and “Exomiser top 5” use in the top five candidates. Sensitivity is the percentage of true positive diagnoses based on SNVs or indels that were identified, and the positive predictive value is the percentage of prioritized variants that led to a positive diagnosis. The values above the bars are percentages. In this analysis, the diagnosed variant or variants are true positives, and the other candidate variants that were returned are false positives.</div></div></figcaption></div></div></div><div class="tab__pane fv__list fv__list--tables" id="tab-pane-tables" aria-label="All tables" tabindex="-1" role="tabpanel"><div class="fv__item"><a class="fv__open-item fv__item__view" href="#fv-t1" data-type="tables" data-interactiontype="multimedia_click" data-multimedia-type="Table"><i class="icon-tables"></i></a><div class="fv__item__title">Table 1</div><div class="fv__item__description"><figcaption><div class="caption">Demographic Characteristics of the Probands (Including Inferred Ancestry) in the 100,000 Genomes Project Pilot Study.<a href="#fv-t1fn1" role="doc-noteref">*</a></div></figcaption></div></div><div class="fv__item"><a class="fv__open-item fv__item__view" href="#fv-t2" data-type="tables" data-interactiontype="multimedia_click" data-multimedia-type="Table"><i class="icon-tables"></i></a><div class="fv__item__title">Table 2</div><div class="fv__item__description"><figcaption><div class="caption">Disease Categories among the Probands in the 100,000 Genomes Project Pilot Study.<a href="#fv-t2fn1" role="doc-noteref">*</a></div></figcaption></div></div><div class="fv__item"><a class="fv__open-item fv__item__view" href="#fv-t3" data-type="tables" data-interactiontype="multimedia_click" data-multimedia-type="Table"><i class="icon-tables"></i></a><div class="fv__item__title">Table 3</div><div class="fv__item__description"><figcaption><div class="caption">Candidate Variants Returned for Presentation to the NHS Genomic Medicine Centres per Proband with the Automated Virtual Panel–Based Analysis Pipeline.<a href="#fv-t3fn1" role="doc-noteref">*</a></div></figcaption></div></div></div></div></div></div><div class="references-pop-up fade" aria-hidden="true"><div class="references-pop-up__heading text-dark d-flex align-items-center"><div class="references-pop-up__title flex-grow-1">Reference <span class="references-pop-up__heading-number">1</span></div><a class="references-pop-up__close text-dark" href="#" title="close pop-up" tabindex="-1" aria-hidden="true"><i class="icon-close" aria-hidden="true"></i></a></div><div class="references-pop-up__items citations to-citation__accordion external-links"></div></div></main>
    

</div>

                    
                        



        
        <div class="ng-page_layout ng-page_layout-fullWidth"><div class="container-fluid"><div class="row"><div class="col-12">



        
        




        
        <div class="ng-page_layout-contentSpacing"></div>










    
    
        <div data-widget-def="oneSearchMultiSearch" data-widget-id="778b3ad8-1a8d-43d8-8196-63b03df4a138" data-location="recirc_inIssue_bottom_article">
        



        
        <div class="os-ms os-ms-more-from-issue"><h3 class="ng-subsection-title"><a href="/toc/nejm/385/20" class="ng-subsection-title_link animation-icon-shift"><span class="ng-subsection-title_text">More from Vol. 385 No. 20</span><span class="ng-subsection-title_icon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-longArrowRight"></use></svg></span></span></a></h3><ul class="os-ms-more-from-issue_list"><li class="os-ms-more-from-issue_item"><div class="issue-item issue-item_sm"><div class="issue-item_meta"><ul class="issue-item_meta-list clearfix"><li class="issue-item_meta-item"><a href="/browse/nejm-article-type/original-article" class="issue-item_type">Original Article</a></li><li class="issue-item_meta-item"><span class="issue-item_date">Nov 11, 2021</span></li><li data-toggle="tooltip" data-original-title="Video" class="issue-item_meta-item"><span role="img" aria-label="Video" tabindex="0" class="issue-item_video"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#tooltip-video"></use></svg></span></span></li><li class="issue-item_meta-item"><span class="issue-item_free">FREE</span></li></ul></div><div class="issue-item_row"><div class="issue-item_left"><div class="issue-item_content clearfix"><h4 class="issue-item_title"><a href="/doi/full/10.1056/NEJMoa2107727" data-interactiontype="article_recirculation_click" class="issue-item_title-link animation-underline">A Randomized Trial of Intravenous Alteplase before Endovascular Treatment for Stroke</a></h4><div class="issue-item_authors-and-text"><span class="issue-item_authors">N.E. LeCouffe and Others</span></div></div></div></div></div></li><li class="os-ms-more-from-issue_item"><div class="issue-item issue-item_sm"><div class="issue-item_meta"><ul class="issue-item_meta-list clearfix"><li class="issue-item_meta-item"><a href="/browse/nejm-article-type/original-article" class="issue-item_type">Original Article</a></li><li class="issue-item_meta-item"><span class="issue-item_date">Nov 11, 2021</span></li><li data-toggle="tooltip" data-original-title="Video" class="issue-item_meta-item"><span role="img" aria-label="Video" tabindex="0" class="issue-item_video"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#tooltip-video"></use></svg></span></span></li><li class="issue-item_meta-item"><span class="issue-item_free">FREE</span></li></ul></div><div class="issue-item_row"><div class="issue-item_left"><div class="issue-item_content clearfix"><h4 class="issue-item_title"><a href="/doi/full/10.1056/NEJMoa2112435" data-interactiontype="article_recirculation_click" class="issue-item_title-link animation-underline">Pembrolizumab for Persistent, Recurrent, or Metastatic Cervical Cancer</a></h4><div class="issue-item_authors-and-text"><span class="issue-item_authors">N. Colombo and Others</span></div></div></div></div></div></li><li class="os-ms-more-from-issue_item"><div class="issue-item issue-item_sm"><div class="issue-item_meta"><ul class="issue-item_meta-list clearfix"><li class="issue-item_meta-item"><a href="/browse/nejm-article-type/original-article" class="issue-item_type">Original Article</a></li><li class="issue-item_meta-item"><span class="issue-item_date">Nov 11, 2021</span></li><li data-toggle="tooltip" data-original-title="Video" class="issue-item_meta-item"><span role="img" aria-label="Video" tabindex="0" class="issue-item_video"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#tooltip-video"></use></svg></span></span></li><li class="issue-item_meta-item"><span class="issue-item_free">FREE</span></li></ul></div><div class="issue-item_row"><div class="issue-item_left"><div class="issue-item_content clearfix"><h4 class="issue-item_title"><a href="/doi/full/10.1056/NEJMoa2104508" data-interactiontype="article_recirculation_click" class="issue-item_title-link animation-underline">Angiotensin Receptor–Neprilysin Inhibition in Acute Myocardial Infarction</a></h4><div class="issue-item_authors-and-text"><span class="issue-item_authors">M.A. Pfeffer and Others</span></div></div></div></div></div></li></ul></div>

        </div>
    

</div></div></div></div>

                    
                        



        
        <div class="ng-page_layout-bottomAd"><div class="container-fluid"><div class="row"><div class="col-12">









    
    
        <div data-widget-def="literatumAd" data-widget-id="7e022d0d-4a48-44a7-af66-033cf5c13eb8" id="ad-article-bottom-FULLx320-1">
        



        
        



    
        <div class="pb-house-message adplaceholder exists">
            <!-- START: Circulation Footer 3 -->
<aside class="g-cta-subscribe">
	<style>
.bads_footer-banner24-ONJQNRF1{background:url(/pb-assets/contextual_messaging/footer-yel-background-2x-1513629627453.jpg) center center no-repeat;padding:20px;display:block;margin:0;min-height:inherit;background-size:cover;text-decoration:none!important;color:#1a1a1a;text-align:left}.bads_footer-banner24-ONJQNRF1 .g-cta-subscribe__title{font-size:42px;line-height:44px;letter-spacing:.2px;margin:0 0 20px;font-family:ff-quadraat-web-pro,sans-serif;font-weight:600}.bads_footer-banner24-ONJQNRF1-inner{max-width:1300px;margin:0 auto}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__actions .a-btn{width:130px;padding-left:12px}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__login{position:static;color:#000;font-size:18px;line-height:27px;border:0}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__actions{margin:20px 0 20px 0;list-style-type:none;padding:0}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__login a{color:#0a68ad}.bads_clearfix:after{visibility:hidden;display:block;font-size:0;content:" ";clear:both;height:0}.bads_clearfix{display:block}.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{float:left;width:50%;padding:30px 0}.bads_footer-banner24-ONJQNRF1-inner .bads_device-img{float:left;padding-left:30px;width:50%}.bads_footer-banner24-ONJQNRF1 .bads_device-img>img{display:block;max-width:100%;height:auto}.bads_footer-banner24-ONJQNRF1 .bads_h5{font-size:18px;text-transform:uppercase;line-height:24px;font-weight:600;margin:-15px 0 0;letter-spacing:.2px}@media only screen and (max-width:480px){.bads_footer-banner24-ONJQNRF1-inner .bads_device-img,.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{width:100%;float:none}.bads_footer-banner24-ONJQNRF1-inner .bads_device-img{padding-left:0}.bads_footer-banner24-ONJQNRF1 .g-cta-subscribe__title{font-size:20px;line-height:27px;margin-bottom:10px}.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{padding-top:0;padding-bottom:0}.bads_footer-banner24-ONJQNRF1{height:auto}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__actions{margin:0 0 12px;list-style-type:none}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__login{line-height:25px;font-size:15px;margin-bottom:10px}.bads_footer-banner24-ONJQNRF1 .bads_device-img{width:280px!important;margin:0 auto}.bads_footer-banner24-ONJQNRF1 .bads_h5{margin:0 0 3px;font-size:14px}}@media only screen and (min-width:480px) and (max-width:1024px){.bads_footer-banner24-ONJQNRF1 .g-cta-subscribe__title{font-size:20px;line-height:30px;margin-bottom:15px}.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{padding:0}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__login{font-size:14px}.bads_footer-banner24-ONJQNRF1 .bads_h5{font-size:13px;margin-top:0}}@media only screen and (min-width:1025px) and (max-width:1200px){.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{padding:10px 0}.bads_footer-banner24-ONJQNRF1 .bads_h5{margin:0 0 5px}}@media only screen and (min-width:1401px){.bads_footer-banner24-ONJQNRF1 .bads_device-img>img{float:right}}
	</style>
	<div class="bads_footer-banner24-ONJQNRF1">
	  <div class="bads_footer-banner24-ONJQNRF1-inner bads_clearfix">
		 <div class="bads_subs-btn">
			<p class="g-cta-subscribe__title">Tap into groundbreaking research and clinically relevant insights</p>
			<ul class="g-cta-subscribe__actions">
				<li class="g-cta-subscribe__actions-primary">
				  <a href="/action/clickThrough?id=127716&amp;url=%2Faction%2FstoreProxy%3Faction%3Dsubscribe%26url%3Dhttps%253A%252F%252Fwww.nejm.org%252Fdoi%252Ffull%252F10.1056%252FNEJMoa2035790%26promo%3DONFQNRF1&amp;loc=%2Fdoi%2Ffull%2F10.1056%2FNEJMoa2035790&amp;pubId=41291304&amp;placeholderId=101336&amp;productId=1035" class="ng-btn_primary ng-btn_iconRight"><span class="ng-btn_text">Subscribe </span><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#arrows-longArrowRight"></use></svg></span></a>
				</li>
			</ul>
			<p class="g-cta-subscribe__login">Already a subscriber? <a href="/action/clickThrough?id=127716&amp;url=%2Faction%2FstoreProxy%3Faction%3Dactivate%26url%3Dhttps%253A%252F%252Fwww.nejm.org%252Fdoi%252Ffull%252F10.1056%252FNEJMoa2035790%26promo%3DONFQANR4&amp;loc=%2Fdoi%2Ffull%2F10.1056%2FNEJMoa2035790&amp;pubId=41291304&amp;placeholderId=101336&amp;productId=1035">Activate</a> your online access.</p>
		 </div>

		<div class="bads_device-img">
		   <img src="/sda/127716/footer-devices-2x.png" alt="Devices">
		</div>
	   </div>
	</div>
</aside>
<!--/END: Circulation Footer 3 -->
            
        </div>
    

    

    



        </div>
    

</div></div></div></div>

                    
                        



        
        <footer class="ng-footer footer__top" style="position: relative;"><div data-location="footer" class="ng-footer_columns"><div class="container-fluid"><div class="row"><div class="col-lg-4 col-md-6 col-sm-12 ng-footer_column"><div class="row"><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title">ARTICLE CATEGORIES</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item">
        <a href="/browse/nejm-article-category/research" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Research</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/review" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Reviews</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/clinical-cases" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Clinical Cases</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/perspective" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Perspective</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/commentary" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Commentary</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/other" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Other</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/medical-article-index" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Browse all Articles</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/toc/nejm/current" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Current Issue</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/loi/nejm" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Issue Index</span></a></li>
</ul>
</div><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title">RESOURCES</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item"><a href="/author-center/home" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Authors &amp; Reviewers</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/author-center/home" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Submit a Manuscript</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/products-and-services" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Subscribers</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://tools.ovid.com/ovidtools/nejm.html" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Institutional Administrators</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/media" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Media</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/how-to-advertise" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Advertisers</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/information-for-subscription-agents" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Agents</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/permissions" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Permissions &amp; Licensing</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/reprints" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Reprints</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://www.nejmcareercenter.org" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">NEJM CareerCenter</span></a>
    </li>
</ul>
</div></div></div><div class="col-lg-4 col-md-6 col-sm-12 ng-footer_column"><div class="row"><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title" aria-label="About us">ABOUT US</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item"><a href="/about-nejm/about-nejm" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">About NEJM</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://www.nejmgroup.org/" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">NEJM Group</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/products-and-services" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Products &amp; Services</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/editors-and-publishers" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Editors &amp; Publishers</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/advertising-policies" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Advertising Policies</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/contact-nejm" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text" aria-label="Contact us">Contact Us</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/frequently-asked-questions?#Accessibility" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Accessibility</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/frequently-asked-questions" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">FAQs</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/help" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Help</span></a></li>
    <li class="ng-footer_column-list-item"><a href="mailto:sitefeedback@nejm.org" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Site Feedback</span></a></li>
</ul>
</div><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title">SUBSCRIPTIONS</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=subscribe&amp;promo=ONF4NRS3" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Subscribe</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=renew&amp;product=nejm&amp;promo=ONFLNRR3&amp;prc=ONFARN52" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Renew</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=activate&amp;promo=ONFLNAA1" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Activate Subscription</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC4" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Create Account</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://myaccount.nejm.org" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Manage Account</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=paybill&amp;product=nejm&amp;promo=BNFLPBA1" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Pay Bill</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://www.wolterskluwer.com/en/solutions/ovid/nejm-complete-collection" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Institutional Sales</span></a></li>    
    <li class="ng-footer_column-list-item"><a href="https://myaccount.nejm.org/special-content" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Special Content</span></a></li>
</ul>
</div></div></div><div class="col-lg-4 col-md-12 ng-footer_column"><div class="row"><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title">STAY CONNECTED</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item"><a href="https://myaccount.nejm.org/emails?section=NEJM" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Email Alerts</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC3" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Create Account</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/products-and-services" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Apps</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://www.nejmcareercenter.org" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">NEJM CareerCenter</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/rss-feed/" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Podcasts</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/rss-feed/" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">RSS Feed</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/institutional-access" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Remote Access</span></a></li>
</ul>
</div><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title" aria-label="Follow us">FOLLOW US</h4>
<ul class="ng-footer_column-list">
<li class="ng-footer_column-list-item"><a href="https://www.facebook.com/TheNewEnglandJournalofMedicine" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-facebook"></use></svg></span></span><span class="ng-footer_column-link-text">Facebook</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://twitter.com/nejm" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-twitter"></use></svg></span></span><span class="ng-footer_column-link-text">X (formerly Twitter)</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://www.instagram.com/nejm/" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-instagram"></use></svg></span></span><span class="ng-footer_column-link-text">Instagram</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://www.youtube.com/user/NEJMvideo" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-youtube"></use></svg></span></span><span class="ng-footer_column-link-text">Youtube</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://www.linkedin.com/company/nejm-group" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-linkedin"></use></svg></span></span><span class="ng-footer_column-link-text">LinkedIn</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://bsky.app/profile/nejm.org" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-bluesky"></use></svg></span></span><span class="ng-footer_column-link-text">Bluesky</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://www.tiktok.com/@nejm.org" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-tiktok"></use></svg></span></span><span class="ng-footer_column-link-text">TikTok</span></a></li>
</ul>
</div></div></div></div></div></div><div data-location="footer" class="ng-footer_journals"><div class="container-fluid"><div class="row"><div class="col-12">



        
        <h4 class="ng-footer_journals-label">JOURNALS</h4>
<ul class="ng-footer_journals-list">
<li class="ng-footer_journals-item"><a href="https://www.nejm.org/" target="_blank" rel="noopener" class="ng-footer_journals-link">The New England Journal of Medicine</a></li>
<li class="ng-footer_journals-item"><a href="https://catalyst.nejm.org/" target="_blank" rel="noopener" class="ng-footer_journals-link">NEJM Catalyst Innovations in Care Delivery</a></li>
<li class="ng-footer_journals-item"><a href="https://evidence.nejm.org/" target="_blank" rel="noopener" class="ng-footer_journals-link">NEJM Evidence</a></li>
<li class="ng-footer_journals-item"><a href="https://ai.nejm.org/" target="_blank" rel="noopener" class="ng-footer_journals-link">NEJM AI</a></li>
</ul>
</div></div></div></div><div data-location="legal" class="ng-footer_bottom"><div class="container-fluid"><div class="row"><div class="col-lg-7 col-md-6 ng-footer_bottom-left">



        
        <p>Copyright © 2025 <a href="https://www.massmed.org/" target="_blank" rel="noopener" class="ng-footer_bottom-link"><b>Massachusetts Medical Society</b></a>. All rights reserved, including those for text and data mining, AI training, and similar technologies. Electronic ISSN 1533-4406. Print ISSN 0028-4793.The content of this site is intended for health care professionals.</p>
</div><div class="col-lg-5 col-md-6 ng-footer_bottom-right">



        
        <style>
  #teconsent a {
    color: #666;
    text-decoration: none;
  }
  #teconsent a:hover {
    color: #666;
    text-decoration: underline;
  }
</style>
<ul class="ng-footer_bottom-list">
<li class="ng-footer_bottom-list-item"><a href="https://www.nejmgroup.org/legal/copyright-information.htm" target="_blank" rel="noopener" class="ng-footer_bottom-link">Copyright</a></li>
<li class="ng-footer_bottom-list-item"><a href="https://www.nejmgroup.org/legal/terms-of-use.htm" target="_blank" rel="noopener" class="ng-footer_bottom-link">Terms</a></li>
<li class="ng-footer_bottom-list-item"><a href="https://www.nejmgroup.org/legal/privacy-policy.htm" target="_blank" rel="noopener" class="ng-footer_bottom-link">Privacy Policy</a></li>
<li class="ng-footer_bottom-list-item"><span id="teconsent" consent="undefined" aria-label="Open Cookie Preferences Modal" role="complementary"><a role="link" id="icon-id007833670792265246" tabindex="0" lang="en" aria-haspopup="dialog" aria-label="Cookie Preferences, opens a dedicated popup modal window" class="truste_cursor_pointer">Cookie Preferences</a></span></li>
</ul>
<a href="https://www.nejmgroup.org/" title="NEJM Group" target="_blank" rel="noopener" class="ng-footer_logo-link"><img src="/specs/products/mms-nextgen/mms/releasedAssets/images/other-images/logo_group.svg" alt="NEJM Group logo" class="ng-footer_logo" loading="lazy" width="103" height="40"></a>
</div></div></div></div><div data-show-after="400" class="back-to-top back-to-top--align-right  "><button data-snap="footer__top" class="back-to-top__action position-relative p-0 m-0 border-0 d-flex flex-column align-items-center justify-content-center"><svg viewBox="0 0 36 36" class="position-relative"><path d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831" class="back-to-top__action__ring-bg"></path><path d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831" stroke-dasharray="0, 100" class="back-to-top__action__ring"></path></svg><span class="back-to-top__action-icon position-absolute d-flex"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-up"></use></svg></span></span><span class="sr-only">Back to top</span></button></div></footer>

                    
                        



        
        <!-- TrustArc: Banner -->
<div id="consent_blackbar"><style>  #consent_blackbar{    position:relative;    width:100%;    z-index: 999999;  }    #trustarc-banner-overlay {  background-color: rgb(0, 0, 0);   opacity: 0.3;   position: fixed;   z-index: 1000000;   width: 100%;  height: 100%;   top: 0px;   left: 0px;   overflow: hidden;}.truste-title {  /*font-family: "Source Sans Pro", sans-serif;  font-size: 23px;  font-style: normal;  font-stretch: normal;*/  color: #222;  margin-bottom: 20px;  text-align: center;/*  font-weight:bold;*/}.truste-footer {  margin: 0 auto;      }.truste-left {  margin: 0px 0px 15px 15px;/*font-family: "Source Sans Pro", sans-serif;  font-size: 14px;    */}.truste-banner { margin: 0 auto; }.truste-messageColumn {/*font-family: "Source Sans Pro", sans-serif;  font-size: 14px;*/  color: #666666;  margin: 0px;}#truste-consent-button, #truste-consent-required {    background-color: #FF1E00;    color: #F2F1F1;    padding: 5px 10px;    border: 1px solid #FF1E00;    border-radius: 0px;    cursor: pointer;    text-transform:uppercase;/*    font-family: "Source Sans Pro",sans-serif;    font-size: 15px;*/  }      #truste-show-consent {    background-color: #FF1E00;    color: #F2F1F1;    border: 1px solid #FF1E00;    padding: 5px 10px;    border-radius: 0px;    cursor: pointer;    text-transform:uppercase;/*    font-family: "Source Sans Pro",sans-serif;    font-size: 15px;*/  }div.truste-consent-track-class {  position: fixed;  z-index:9999999;   background: #666;   border: 1px solid #333;    /*padding: 10px 60px;*/    bottom:0px;    right:0px;    left:0px;    margin: 10px;}.truste-left {  margin: 0;}div.text-content {  display: flex;  flex-direction: column;  align-self: baseline;}#truste-consent-text {    /*font-family: "Source Sans Pro", sans-serif;    font-size: 16px;*/    color: #fff;    margin: 15px 0px;}.truste-footer {  padding-top: 15px;}#truste-consent-content {  margin: 0 auto;    padding: 0 10px;    max-width: 1366px;}.truste-consent-footer {  display: none;}.flex-container {  display: block;  align-items: center;  justify-content: space-between;}#truste-consent-buttons {  float: right;  margin: 10px 5px 10px 0px;}#truste-show-consent { background: #F30 !important;}#truste-consent-button, #truste-consent-required { background: #F30 !important; } /* MEDIA QUERIES */  @media screen and (max-width: 500px) {  #truste-consent-track {    padding: 5px 20px;  }  }@media screen and (max-width: 375px) {  .truste-buttonsColumn {    float: none;    margin: 0;  }  #truste-consent-button, #truste-consent-required, #truste-show-consent {    width: 100%;    margin-bottom: 5px;  }          #truste-consent-buttons {  float: none;  margin: 10px 0px 10px 0px;	}}</style><div id="truste-consent-track" style="display: block; opacity: 1;">  <div id="trustarc-banner-overlay"></div>  <div class="truste-consent-track-class">    <div id="truste-consent-content" class="truste-banner">        <div class="flex-container">          <div class="text-content">            <div id="truste-consent-text" class="truste-messageColumn">This site uses cookies and related technologies, as described in our privacy policy, for purposes that may include site operation, analytics, enhanced user experience, or advertising. You may choose to consent to our use of these technologies, or manage your own preferences.                       </div>          </div>          <div id="truste-consent-buttons" class="truste-buttonsColumn">            <button id="truste-show-consent" aria-haspopup="dialog">Manage Settings</button>      <button id="truste-consent-button">Accept</button>      <!-- <button id="truste-consent-required">Decline All</button> -->          </div>        </div>                <div style="clear:both;"></div>    </div>  </div></div><!--emptycomment--></div>
<!-- /TrustArc: Banner -->

                    
                        



        
        <link rel="stylesheet" href="https://cssjs.nejm.org/mmsWidgets.css"><script nonce="94d6a712622201a2-GRU" type="text/javascript" src="https://cssjs.nejm.org/mmsWidgets.js"></script><script nonce="94d6a712622201a2-GRU" type="text/javascript">var litSSO = {
    "ucid": null,
    "accessToken": null,
    "authState": null
};

var loginUri = '/action/doSsoLogin';
var logoutUri = '/action/doLogout';

displayAuthenticatedMessage();

mmsWidgets.init({
    clientId: 'qsra7g4d6jwdsgn3zbe6udp6r6278mr9',
    origin: 'NEJM',
    xdReceiver: 'http://www.nejm.org/pb-assets/sso/xd.html',
    isFullPageSignIn: isSignInPage(),
    debug: true,

    onReady: function (e) {
        console.log('<<<<<<<<<< client: received onReady event.');
        $("A.litSsoLogin").click(function (e) {
            if (!$(e.currentTarget).hasClass("noMd") || $(e.currentTarget).parents(".header_sm-login").length === 0) {
                mmsWidgets.signIn(e);
                return false;
            }
        });
        $("A.litSsoLogout").click(function (e) {
            mmsWidgets.signOut(e);
            return false;
        });
        $("A.litSsoCreate").click(function (e) {
            mmsWidgets.showRegModal(e);
            return false;
        });
        $("A.litSsoSubscribe").click(function (e) {
            if (!$(e.currentTarget).hasClass("noMd") || $(e.currentTarget).parents(".header_sm-login").length === 0)
                location.assign(e.currentTarget.href);
            return false;
        });
        if (window.doSSOLogin) {
            window.setTimeout(function () {
                mmsWidgets.signIn();
            }, 500);
        }
        ;
    },
    onLoginSuccess: function (e) {
        console.log('<<<<<<<<<< client: received onLoginSuccess event.' + JSON.stringify(e));

        const redirectUrl = getRedirectUri(window.location.search, location.href.replace(window.location.origin, ""));
        document.cookie = "skipSso=; expires=; path=/";
        if (!litSSO.ucid || e.ucid != litSSO.ucid || !litSSO.accessToken || e.accessToken != litSSO.accessToken) {
            location.assign(loginUri + "?ucid=" + e.profile.ucid
                + "&accessToken=" + e.accessToken
                + "&authState=" + e.authState
                + "&email=" + encodeURIComponent(e.profile.email)
                + "&givenNames=" + e.profile.givenName
                + "&surname=" + e.profile.familyName
                + "&redirectUri=" + redirectUrl
                + "&uccLastUpdatedDate=" + encodeURIComponent(e.profile.uccLastUpdatedDate)
                + "&rememberMe=" + e.rememberMe);
        }
    },
    onAuthStateReady: function (e) {
        console.log('<<<<<<<<<< client: received onAuthStateReady event.' + JSON.stringify(e))

        var isMmsSsoAuthenticated = (e.authState && e.authState !== "anonymous" && e.authState !== "lead" && e.ucid && e.accessToken) ? true : false;
        var isLitSsoAuthenticated = (litSSO.ucid && litSSO.accessToken && litSSO.authState) ? true : false;
        console.log('           isMmsSsoAuthenticated=' + isMmsSsoAuthenticated + ', isLitSsoAuthenticated=' + isLitSsoAuthenticated);

        if (isMmsSsoAuthenticated) {
            if (!isLitSsoAuthenticated || litSSO.ucid != e.ucid) {
                if (window.skipSso)
                    document.cookie = "skipSso=" + encodeURIComponent(e.ucid) + "; expires=; path=/";

                if (!window.skipSso && document.cookie.indexOf("skipSso=" + encodeURIComponent(e.ucid)) == -1) {
                    location.assign(loginUri + "?ucid=" + e.profile.ucid
                        + "&accessToken=" + e.accessToken
                        + "&authState=" + e.authState
                        + "&email=" + encodeURIComponent(e.profile.email)
                        + "&givenNames=" + e.profile.givenName
                        + "&surname=" + e.profile.familyName
                        + "&redirectUri=" + location.href.replace(window.location.origin, "")
                        + "&uccLastUpdatedDate=" + encodeURIComponent(e.profile.uccLastUpdatedDate)
                        + "&rememberMe=" + e.rememberMe);
                } else
                    console.log('**** client: Skip SSO ****');
            } else if (e.accessToken !== litSSO.accessToken)
                updateLitSession(e.ucid, e.accessToken);
        } else if (isLitSsoAuthenticated)
            location.assign(logoutUri + "?redirectUri=" + encodeURIComponent(location.href.replace(window.location.origin, "")));

        if (document.getElementById("ssoerr-ucid"))
            document.getElementById("ssoerr-ucid").innerHTML = e.ucid;

    },
    onRegistrationSuccess: function (e) {
        console.log('<<<<<<<<<< client: received onRegistrationSuccess event.' + JSON.stringify(e))
        document.cookie = "skipSso=; expires=; path=/";
        var uccLastUpdatedDate = '1812-01-01 00:00:00.1 +0000';
        location.assign(loginUri + "?ucid=" + e.profile.ucid
            + "&accessToken=" + e.accessToken
            + "&authState=" + e.authState
            + "&email=" + encodeURIComponent(e.profile.email)
            + "&givenNames=" + e.profile.givenName
            + "&surname=" + e.profile.familyName
            + "&redirectUri=" + location.href.replace(window.location.origin, "")
            + "&uccLastUpdatedDate=" + encodeURIComponent(uccLastUpdatedDate)
            + "&rememberMe=" + e.rememberMe);
    },
    onLogoutSuccess: function (e) {
        console.log('<<<<<<<<<< client: received onLogoutSuccess event.' + JSON.stringify(e))

        document.cookie = "skipSso=; expires=; path=/";
        location.assign(logoutUri);
    },
});

function updateLitSession(ucid, accessToken) {
    $.ajax({
        method: "GET",
        url: "/action/doSsoSessionUpdate",
        data: {ucid: ucid, accessToken: accessToken},
        success: function (callData, textStatus) {
            console.log("Session accessToken updated")
            litSSO.accessToken = accessToken
        },
        error: function (jqXHR, textStatus, errorThrown) {
            alert("Your website session has expired and will now be reset.")
            console.log("Session update failed: " + jqXHR.responseText)
            location.assign(logoutUri + "?redirectUri=" + encodeURIComponent(location.href.replace(window.location.origin, "")));
        }
    });
}

function getRedirectUri(location, fallback) {
    const urlParams = new URLSearchParams(location);
    const uriUrlParam = urlParams.get('uri');
    if (uriUrlParam)
        return encodeURIComponent(uriUrlParam);
    else
        return encodeURIComponent(fallback);
}

function isSignInPage() {
    return document.querySelector('meta[name="pbContext"]').content.indexOf("customPage:string:/sign-in") > -1;
}

function displayAuthenticatedMessage() {
    if (isSignInPage()) {
        var div = document.getElementById('authMsg');
        if ("" === "PersonUser") {
            div.innerHTML += "You are already signed in to " + window.location.origin.replace("https://", "").replace("www.", "") + ". If you would like to sign in with a different email address, please use the form below.";
        } else {
            div.innerHTML = '';
        }
    }
}</script>

                    
                
            </div>
        </div><iframe name="trustarc_notice" id="trustarcNoticeFrame" title="Trustarc Cross-Domain Consent Frame" src="https://consent.trustarc.com/get?name=crossdomain.html&amp;domain=nejm.com" style="display: none;"></iframe><script src="https://consent.trustarc.com/asset/notice.js/v/v1.7-1576" async="async" crossorigin="" importance="high" nonce="94d6a712622201a2-GRU"></script>
        


        
        

            <script src="/products/mms-nextgen/mms/releasedAssets/js/build.lazyload.bundle-c8c4e7d20884f81951fb.js" nonce="94d6a712622201a2-GRU"></script>

        <script src="/products/mms-nextgen/mms/releasedAssets/js/main.bundle-3819a02ec7bc2aca01ec.js" nonce="94d6a712622201a2-GRU"></script>
        


<div id="adModal" tabindex="-1" aria-labelledby="exampleModalCenterTitle" class="ng-modal modal" style="display: none;" aria-hidden="true">
    <div role="document" class="ng-modal_dialog modal-dialog modal-dialog-centered ad-promoLayer">
        <div class="ng-modal_content modal-content p-0">
            <div class="ng-modal_body modal-body m-0">
                <div class="ng-modal_header modal-header">
                    <button type="button" data-dismiss="modal" aria-label="Close" class="ng-modal_close close"><span aria-hidden="true"><span class="icon-component "><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#close"></use></svg></span></span>
                    </button>
                </div>

            </div>
        </div>
    </div>
</div>
<script type="text/javascript" src="/wro/1125813~product.js" nonce="94d6a712622201a2-GRU"></script>








<input id="showLayers" type="hidden" value="true">

    <!-- placeholder id=null, description=ad-layer-config -->
    <!-- placeholder id=null, description=ad-lauchLayer-1 -->
    <!-- placeholder id=null, description=ad-layer-promo-1 -->

    <input id="firstAd" type="hidden" value="">
    <input id="secondAd" type="hidden" value="">

    
    
    

    
    
    <input id="IsInstitution" type="hidden" value="false">
    <input id="IsSubscriber" type="hidden" value="true">
    <input id="CustomerId" type="hidden" value="">


<script type="text/javascript" nonce="94d6a712622201a2-GRU">
  jQuery.extend(jQuery.mmsLayers.config, {
    rules: [{
      "name": "Launch Layer Ad",
      "frequency": 30,
      "showOnPage": 5,
      "enabled": false,
      "cookieName": "MarketingLaunchLayer",
      "styleClass": "launchLayer"
    },
    {
      "name": "Promo Layer Ad",
      "frequency": 14,
      "enabled": true,
      "cookieName": "PromoLayer",
      "styleClass": "promoLayer"
    },
    {
      "name": "Interstitial Ad",
      "frequency": 7,
      "enabled": false,
      "cookieName": "InterstitialAd"
    },
    {
      "name": "Iperceptions - Ion Layer Ad",
      "showOnPage": 1,
      "enabled": false,
      "test": false,
      "cookieName":"Iperceptions"
    }]
 });
</script>

















    <script type="text/javascript" nonce="94d6a712622201a2-GRU">
        $(document).ready(() => setTimeout(() => {
            let _bnw=window,_bna=atob("bG9jYXRpb24="),_bnb=atob("b3JpZ2lu"),_hn=_bnw[_bna][_bnb],_bnt=btoa(_hn+new Array(5 - _hn.length % 4).join(" "));
            $.get("/resource/lodash?t="+_bnt);
        },4000));
    </script>












    

    
    

    
    
    
    
        
            
                <script type="text/javascript" src="/wro/1125813~article-metrics.js" nonce="94d6a712622201a2-GRU"></script>
            
            
            
        
    




<script type="text/javascript" src="/wro/1125813~full-text-analytics.js" nonce="94d6a712622201a2-GRU"></script>


    


            <script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94d6a7125d0a01a2',t:'MTc0OTUzNTI3MS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94d6a7125d0a01a2&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;f55e150c7a5f4a4ab9ef82f4a1c9c43c&quot;}" crossorigin="anonymous"></script><div style="display:none;" id="signInEmbedded" class="ucc-fullpage-signin ucc-widgets-nejm-next">
  <div class="capture_header">
    <h1>Sign In</h1>
  </div>    
  <div class="capture_signin">
    {* #signInForm *}
    <div class="ucc-form-inner-wrapper">
      {* signInEmailAddress *}
      {* currentPassword *}
      <div style="padding-top:10px;">
          <div class="capture_form_item capture_form_rememberme">
          <label class="ucc-form-checkbox">                
            <input checked="checked" id="ucc-remember-me-cb-embedded" name="ucc-remember-me-cb-embedded" type="checkbox" value="true">
            <span>Remember Me</span>
          </label>                           
          </div>
          <div class="capture_form_item capture_form_forgotpwd">
            <a role="button" class="ucc-anchor-button" data-capturescreen="forgotPassword">Forgot your password?</a>
          </div>
      </div>
      <div class="capture_form_item">      
        <button id="ucc-sign-in-submit-embedded" class="capture_primary capture-full-width-btn" type="submit">Sign In</button>
      </div>
      <div class="capture_form_item capture_form_footer">
        <div class="ucc-no-acct-msg">Don't have an account?</div>
        <div class="ucc-no-acct-msg-2">
          <a role="button" class="ucc-anchor-button" onclick="mmsWidgets.closeSignIn(event); mmsWidgets.showRegAuto(event);">Create Account</a>
          <span></span><a href="https://store.nejm.org/nejm?promo=ONF4NRS1">Subscribe</a>
        </div>
      </div>
      <div class="capture_form_item capture_form_submit">
        
      </div>      
    </div>
    {* /signInForm *}
  </div>
</div><div style="display:none;" id="forgotPassword" class="ucc-widgets-nejm-next">
  <div class="capture_header">
    <h1>Forgot Password</h1>
  </div>   
  <div class="ucc-form-inner-wrapper">
  <p style="color:black">Enter the email address associated with your account then click Continue. We will email you a link to a page where you can easily create a new password.</p>
  </div> 
  <div class="capture_forgotpassword">
    {* #forgotPasswordForm *}      
      <div class="ucc-form-inner-wrapper">
        {* forgotPasswordEmailAddress *}
        {* captcha *}
        <div class="capture_form_item" style="padding-top: 20px;">        
          <input id="ucc-widget-forgotpwd-submit" value="Continue" type="submit" class="capture_btn capture_primary capture-full-width-btn">
        </div>
      </div>
    {* /forgotPasswordForm *}
  </div>
  </div><div style="display:none;" id="forgotPasswordSuccess" class="ucc-widgets-nejm-next">
  <div class="capture_header">
    <h1>Create New Password</h1>
  </div>    
  <div class="capture_forgotpassword">
      <div class="ucc-form-inner-wrapper">
        <p style="padding-top: 30px; padding-bottom: 30px;">We've sent an email with instructions to create a new password. Your existing password has not been changed.</p>        
        <div class="capture_form_item">      
          <button class="capture_btn capture_primary capture-full-width-btn" onclick="janrain.capture.ui.modal.close()">CLOSE</button>
        </div>
      </div>
  </div>
  </div><div style="display:none;" id="resetPassword" class="ucc-widgets-nejm-next">
      <div class="row">      
        <div class="col-md-8 col-md-offset-2 reset-pwd-info"><p>To reset your password, enter a new password twice and click the 'Reset Password' button.</p></div>
      </div>
      <div class="col-md-4 col-md-offset-4">
        {* #changePasswordFormNoAuth *}
          {* newPassword *}
          {* newPasswordConfirm *}     
          <div class="capture_form_item">
            <input value="RESET PASSWORD" type="submit" class="capture_btn capture_primary capture-full-width-btn">
          </div>
          {* /changePasswordFormNoAuth *}
      </div>
    </div><div style="display:none;" id="resetPasswordSuccess">
        <div class="reset-pwd-info">
          <div class="reset-pwd-success-welcome"></div>
          <p class="reset-pwd-success-msg">Your password has been reset. You will need to <a href="/signin" onclick="mmsWidgets.signIn(event)">sign in</a> again using your new password to access site content and features.</p>
          <p class="reset-pwd-success-nav"></p>
        </div>      
    </div><div style="display:none;" id="resetPasswordRequestCode" class="ucc-widgets-nejm-next">
        {* #resetPasswordForm *}
        <div class="capture_form_error">The link that you followed to reset your password has expired.</div>
          <div class="col-md-8 col-md-offset-2">
            <p class="reset-pwd-info">Enter the email address associated with your account then click Continue. We will email you a link to a page where you can easily create a new password.</p>
          </div>
          <div class="col-md-4 col-md-offset-4">
            {* signInEmailAddress *}
            <div class="capture_form_item">
              <input value="continue" type="submit" class="capture_btn capture_primary capture-full-width-btn">
            </div>
          </div>
        {* /resetPasswordForm *}
    </div><div style="display:none;" id="resetPasswordRequestCodeSuccess">
      <div class="reset-pwd-info">
        <p>If the address matches an existing account, you will receive an email with instructions to reset your password.</p>
        <p class="reset-pwd-success-nav"></p>
      </div>
  </div>

        
        
    

<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><omi-ualstbaggehd></omi-ualstbaggehd><script type="text/javascript" src="https://api.altmetric.com/v1/internal-556fdf0f/doi/10.1056/nejmoa2035790?callback=_altmetric.embed_callback&amp;domain=www.nejm.org&amp;cache_until=3-10"></script><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="janrainEngageEmbed" style="display: none;"><div class="janrainContent" style="width: 380px !important; height: 156px !important; padding-left: 5px !important; padding-right: 5px !important; box-sizing: content-box !important; background-color: rgb(255, 255, 255) !important; border: 1px solid rgb(192, 192, 192) !important; border-radius: 10px !important; overflow: hidden !important; position: relative !important;"><div id="janrainView"><div class="janrainHeader" style="background-color: rgb(255, 255, 255) !important; font-size: 14px !important; color: rgb(102, 102, 102) !important; position: relative !important; height: 20px !important; left: -5px !important; padding: 5px 5px 0px !important; white-space: nowrap !important; width: 380px !important; z-index: 100 !important; border-radius: 10px 10px 0px 0px !important; text-align: left !important;"><div style="padding-left: 1px !important; font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important;">Sign in using your account with</div></div><div id="janrainProviderPages" style="padding-top: 5px; left: 5px; position: absolute;"><div class="janrainPage" pageindex="0"><ul class="providers" id="janrainProviders_0" style="float: left !important; list-style-type: none !important; margin: 0px !important; padding: 0px !important;"><li id="janrain-aol" role="button" style="list-style: none !important; height: 30px !important; width: 185px !important; margin-top: 0px !important; margin-right: 5px !important; margin-bottom: 5px !important; position: relative !important; border: 1px solid rgb(204, 204, 204) !important; border-radius: 5px !important; cursor: pointer !important; white-space: nowrap !important; overflow: hidden !important; background-image: -webkit-linear-gradient(bottom, rgb(238, 238, 238), rgb(255, 255, 255)) !important;"><a tabindex="1" href="javascript:void(0);" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; font-size: 12px !important; line-height: 14px !important; margin-left: auto !important; margin-right: auto !important; text-decoration: none !important; display: block !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; width: auto !important;"><span class="janrain-provider-icon-24 janrain-provider-icon-aol" style="margin-top: 3px !important; background-color: transparent !important;"></span><span class="janrain-provider-text-color-aol" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; cursor: pointer !important; margin-left: 7px !important; text-align: left !important; margin-top: 9px !important; vertical-align: top !important; display: inline-block !important;">AOL</span></a></li><li id="janrain-openid" role="button" style="list-style: none !important; height: 30px !important; width: 185px !important; margin-top: 0px !important; margin-right: 5px !important; margin-bottom: 5px !important; position: relative !important; border: 1px solid rgb(204, 204, 204) !important; border-radius: 5px !important; cursor: pointer !important; white-space: nowrap !important; overflow: hidden !important; background-image: -webkit-linear-gradient(bottom, rgb(238, 238, 238), rgb(255, 255, 255)) !important;"><a tabindex="3" href="javascript:void(0);" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; font-size: 12px !important; line-height: 14px !important; margin-left: auto !important; margin-right: auto !important; text-decoration: none !important; display: block !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; width: auto !important;"><span class="janrain-provider-icon-24 janrain-provider-icon-openid" style="margin-top: 3px !important; background-color: transparent !important;"></span><span class="janrain-provider-text-color-openid" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; cursor: pointer !important; margin-left: 7px !important; text-align: left !important; margin-top: 9px !important; vertical-align: top !important; display: inline-block !important;">OpenID</span></a></li></ul><ul class="providers" id="janrainProviders_1" style="float: left !important; list-style-type: none !important; margin: 0px !important; padding: 0px !important;"><li id="janrain-yahoo" role="button" style="list-style: none !important; height: 30px !important; width: 185px !important; margin-top: 0px !important; margin-right: 5px !important; margin-bottom: 5px !important; position: relative !important; border: 1px solid rgb(204, 204, 204) !important; border-radius: 5px !important; cursor: pointer !important; white-space: nowrap !important; overflow: hidden !important; background-image: -webkit-linear-gradient(bottom, rgb(238, 238, 238), rgb(255, 255, 255)) !important;"><a tabindex="2" href="javascript:void(0);" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; font-size: 12px !important; line-height: 14px !important; margin-left: auto !important; margin-right: auto !important; text-decoration: none !important; display: block !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; width: auto !important;"><span class="janrain-provider-icon-24 janrain-provider-icon-yahoo" style="margin-top: 3px !important; background-color: transparent !important;"></span><span class="janrain-provider-text-color-yahoo" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; cursor: pointer !important; margin-left: 7px !important; text-align: left !important; margin-top: 9px !important; vertical-align: top !important; display: inline-block !important;">Yahoo!</span></a></li></ul></div></div><div style="background-color: rgb(255, 255, 255) !important; width: 380px !important; height: 10px !important; position: absolute !important; left: 0px !important; padding-left: 5px !important; padding-right: 5px !important; padding-bottom: 5px !important; bottom: 0px !important; font-size: 10px !important; text-align: left !important; color: rgb(102, 102, 102) !important; font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; border-radius: 0px 0px 10px 10px !important;"><div style="padding-left: 1px !important;"><a href="http://janrain.com/products/engage/social-login?utm_source=mms.rpxnow.com&amp;utm_medium=Partner&amp;utm_campaign=attribution" target="_blank">Social Login by Janrain</a></div></div></div></div></div><iframe name="captureIFrame_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-transactionid="w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" id="captureIFrame_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" src="about:blank" class="" data-captureiframeloadeventbound="yes" style="display: none;"></iframe><div id="janrainModalOverlay" style="inset: 0px; position: fixed; background-color: rgb(0, 0, 0); opacity: 0.4; display: none; z-index: 1000;"></div><div id="janrainModal" style="width: 380px; height: 131px; position: absolute; z-index: 1000; display: none;"><a href="#" class="janrain_modal_closebutton" style="position: absolute; cursor: pointer; z-index: 1000;">X</a><div style="display: block;" id="signIn" class="ucc-widgets-nejm-next janrain-capture-ui capture-ui-content capture_screen_container" data-capturescreenname="signIn" data-captureventadded="true">
    <div class="capture_header">
      <h1>Sign In</h1>
    </div>    
    <div class="capture_signin">
      <form id="capture_signIn_signInForm" name="signInForm" data-capturefield="signInForm" action="https://mms.us.janraincapture.com/widget/traditional_signin.jsonp" class="capture_form capture_signInForm" method="POST" novalidate="novalidate" data-transactionid="w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" target="captureIFrame_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" accept-charset="UTF-8" next="{&quot;noop&quot;:&quot;&quot;}"><div id="capture_signIn_signInForm_defaultSavedProfileMessage"></div><div id="capture_signIn_signInForm_errorMessages"></div><input id="capture_signIn_utf8" data-capturefield="undefined" value="✓" type="hidden" class="capture_utf8" name="utf8"><input id="capture_signIn_screen_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="signIn" type="hidden" class="capture_screen_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="capture_screen"><input id="capture_signIn_js_version_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="d445bf4" type="hidden" class="capture_js_version_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="js_version"><input id="capture_signIn_transactionId_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" type="hidden" class="capture_transactionId_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="capture_transactionId"><input id="capture_signIn_form_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="signInForm" type="hidden" class="capture_form_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="form"><input id="capture_signIn_flow_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="standard" type="hidden" class="capture_flow_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="flow"><input id="capture_signIn_client_id_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="qsra7g4d6jwdsgn3zbe6udp6r6278mr9" type="hidden" class="capture_client_id_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="client_id"><input id="capture_signIn_redirect_uri_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="https://www.nejm.org" type="hidden" class="capture_redirect_uri_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="redirect_uri"><input id="capture_signIn_response_type_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="code" type="hidden" class="capture_response_type_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="response_type"><input id="capture_signIn_flow_version_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="20240514181256512029" type="hidden" class="capture_flow_version_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="flow_version"><input id="capture_signIn_settings_version_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="" type="hidden" class="capture_settings_version_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="settings_version"><input id="capture_signIn_locale_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="en-US" type="hidden" class="capture_locale_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="locale"><input id="capture_signIn_recaptcha_version_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" data-capturefield="undefined" value="2" type="hidden" class="capture_recaptcha_version_w54odb0xyl46uz7artzsp3vxct8g4bsnm22qrxc2" name="recaptchaVersion">
      <div class="ucc-form-inner-wrapper">
        <div id="capture_signIn_form_item_signInEmailAddress" class="capture_form_item capture_email capture_form_item_signInEmailAddress" data-capturefield="undefined"><label for="capture_signIn_signInEmailAddress">Email Address</label><input id="capture_signIn_signInEmailAddress" data-capturefield="signInEmailAddress" value="" type="email" class="capture_signInEmailAddress capture_required capture_text_input" placeholder="Email Address" name="signInEmailAddress"><div class="capture_tip" style="display:none;"></div><div class="capture_tip_validating" data-elementname="signInEmailAddress">Validating</div><div class="capture_tip_error" data-elementname="signInEmailAddress"></div></div>
        <div id="capture_signIn_form_item_currentPassword" class="capture_form_item capture_password capture_form_item_currentPassword" data-capturefield="undefined"><label for="capture_signIn_currentPassword">Password</label><input id="capture_signIn_currentPassword" data-capturefield="currentPassword" value="" type="password" class="capture_currentPassword capture_required capture_text_input" placeholder="Password" name="currentPassword"><div class="capture_tip" style="display:none;"></div><div class="capture_tip_validating" data-elementname="currentPassword">Validating</div><div class="capture_tip_error" data-elementname="currentPassword"></div></div>
        <div style="padding-top:10px;">
            <div class="capture_form_item capture_form_rememberme">
                <label class="ucc-form-checkbox">                
                    <input checked="checked" id="ucc-remember-me-cb" name="ucc-remember-me-cb" type="checkbox" value="true">
                    <span>Remember Me</span>
                </label>          
            </div>
            <div class="capture_form_item capture_form_forgotpwd">
              <a role="button" class="ucc-anchor-button" data-capturescreen="forgotPassword">Forgot your password?</a>
            </div>
        </div>
        <div class="capture_form_item">
          <button id="ucc-sign-in-submit" class="capture_primary capture-full-width-btn" type="submit">Sign In</button>
        </div>
       
        <div class="capture_form_item">
          <div class="ucc-no-acct-msg">Don't have an account?</div>
          <div class="ucc-no-acct-msg-2" id="signInModalCreateAccountLink">
          <a role="button" class="ucc-anchor-button" onclick="mmsWidgets.closeSignIn(event); mmsWidgets.showRegAuto(event);">Create Account</a>
            <span></span><a href="https://store.nejm.org/nejm?promo=ONF4NRS1">Subscribe</a>
          </div>

          <div class="ucc-no-acct-msg-2" id="signInModalCloseLink" style="display:none">
          <a role="button" class="ucc-anchor-button" onclick="mmsWidgets.closeSignIn(event);">Create Account</a>
            <span></span><a href="https://store.nejm.org/nejm?promo=ONF4NRS1">Subscribe</a>
          </div>

        </div>
     
        <div class="capture_form_item capture_form_submit">
          
        </div>      
      </div>
      </form>
    </div>
  </div></div></body></html>