<html lang="en" class="pb-page js" data-request-id="94d64844d6a5f20e-SJC"><head data-pb-dropzone="head"><style type="text/css" nonce="94d64844d6a5f20e-SJC">@font-face {font-family: "Source Sans Pro";src: url(https://consent.trustarc.com/get?name=SourceSansPro-Regular.ttf) format("truetype"),url(https://consent.trustarc.com/get?name=SourceSansPro-Regular.woff) format("woff"),url(https://consent.trustarc.com/get?name=SourceSansPro-Regular.otf) format("opentype"),url(https://consent.trustarc.com/get?name=SourceSansPro-Regular.eot) format("embedded-opentype");}.truste_cursor_pointer {cursor: pointer;}.truste_border_none {border: none;}.truste_accessible_link {font-family: "Source Sans Pro", sans-serif;color: #1D4ED8;font-size: 14px;font-weight: 600;text-decoration: underline;}.truste_accessible_link:hover {color: #1D4ED8;text-decoration: none !important;}.truste_accessible_link:focus-visible {outline: none;border-radius: 4px;box-shadow: 0 0 0 1px #FFFFFF, 0 0 0 4px #3699F1;}</style><script src="https://ssl-cdn.janraincapture.com/widget_data/flow.js:7h8bznq6j7bvyvzqcrwvyp8dfb:en-US:20240514181256512029:standard" type="text/javascript"></script><script src="https://cssjs.nejm.org/akamai/capture:login" type="text/javascript"></script><script src="https://ssl-widget-cdn.rpxnow.com/translations/login/en-US" type="text/javascript"></script><script type="text/javascript" id="janrainAuthWidget" src="https://cssjs.nejm.org/akamai/mms-prod.js"></script><script type="text/javascript" src="https://www.nejm.org/kW2UqAVF-Zg0fgfapvnvD3jBq1428bXfRBUABwjSFMl_cf_OIumIftPM7KOu4QonETQnHtWOd38E_k8pR0uaxg=="></script>
            

<meta charset="UTF-8">






    
        
            
        
    
        
            
        
    
        
            
        
    
        
            
        
    
        
    
        
    
        
            
                
                
<title>Causal Genetic Variants in Stillbirth | New England Journal of Medicine</title>





                
            
        
    







    
        
            
        
    
        
            
        
    
        
            
                
                



                
            
        
    
        
            
        
    
        
    
        
    
        
            
        
    











    








    

    
    

    
    
    
    
        
            
                <link rel="stylesheet" type="text/css" href="/wro/1125813~article-metrics.css">
            
            
            
        
    

    
    
        
            <link rel="icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_nejm_1-994b0160d1eb33dc79517ef0295aab1b.ico" type="image/x-icon">
        
        
            <link rel="icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_16-4aa032aa775940f725817b7ff42f0e2c.png" type="image/png" sizes="16x16">
        
        
            <link rel="icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_32-0a94fa44823476ffa0bb15bdb320fb34.png" type="image/png" sizes="32x32">
        
        
            <link rel="apple-touch-icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_152-d6c3522864a8e483996b7e9c132c901a.png" sizes="152x152">
        
        
            <link rel="apple-touch-icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_180-f2538afc53a365d4cd3f10ab73c56aad.png" sizes="180x180">
        
        
            <link rel="apple-touch-icon" href="/products/mms-nextgen/mms/releasedAssets/images/favicon/fav_167-8e888012bbdd31a00c3e54bc5b8ca5b6.png" sizes="167x167">
        
    

    
    
            <link rel="stylesheet" href="/products/mms-nextgen/mms/releasedAssets/css/build-5b89b331018866a4a965.css" media="all">
        <link rel="stylesheet" href="/products/mms-nextgen/mms/releasedAssets/css/print-5b89b331018866a4a965.css" media="print">
           <link rel="stylesheet" href="/products/mms-nextgen/mms/releasedAssets/css/build-article-5b89b331018866a4a965.css" media="all" id="article-style-sheet">
    
        <script nonce="94d64844d6a5f20e-SJC">
        try {
            document.documentElement.classList.add('js');
        } catch (e) {
        }
    </script>
     
    

















    
        
            
                
                
                
                    
                    <!-- TrustArc Scripts -->
<script async="async" src="https://consent.trustarc.com/notice?domain=nejm.com&amp;c=teconsent&amp;js=nj&amp;noticeType=bb&amp;gtm=1&amp;text=true&amp;pcookie" crossorigin="" id="truste_0.562187936408602"></script>
<script src="https://consent.trustarc.com/autoblockasset/core.min.js?domain=nejm.com"></script>
<script src="https://consent.trustarc.com/autoblockoptout?domain=nejm.com"></script>
<!-- /TrustArc Scripts -->
<!-- BlueConic Scripts -->
<script src="https://bc.nejm.org/script.js"></script>
    <!-- /BlueConic Scripts -->
<!-- Printing Meta Tags -->
    <meta name="topics" content="Reproductive Medicine|Obstetrics/Gynecology General">
<meta name="articleCategory" content="Research">
<meta name="articleType" content="Original Article">
<meta name="Specialties" content="Obstetrics/Gynecology|Genetics"><script type="text/javascript">var mmsData = {"access":{"isFreeCountry":"n"},"ad":{"category":"article"},"imageChallenge":{"id":null},"article":{"doi":"10.1056/NEJMoa1908753","title":"Causal Genetic Variants in Stillbirth","category":"Research","type":"Original Article","topics":"Reproductive Medicine|Obstetrics/Gynecology General","specialties":"Obstetrics/Gynecology|Genetics","seriesTitle":"","viewType":"Full","age":"2Years-1990","issueDate":"2020-09-17T00:00Z","isFree":"y","broadTopics":"","perspectiveTopics":"","granularTopics":"","methodologicalArea":"","isOnlineFirst":"n","isCurrentIssue":"n","metaTags":"\u003cmeta name\u003d\"topics\" content\u003d\"Reproductive Medicine|Obstetrics/Gynecology General\"/\u003e\n\u003cmeta name\u003d\"articleCategory\" content\u003d\"Research\"/\u003e\n\u003cmeta name\u003d\"articleType\" content\u003d\"Original Article\"/\u003e\n\u003cmeta name\u003d\"Specialties\" content\u003d\"Obstetrics/Gynecology|Genetics\"/\u003e"},"page":{"activity":"showFullText","url":"www.nejm.org/doi/full/10.1056/NEJMoa1908753","viewType":"desktop","journal":"NEJM"},"user":{"ucid":null,"audienceSegment":null,"specialtyGroupCode":null,"meterCountRemaining":null,"totalMeterCount":null},"object":[{"type":"Interactive Graphic","title":"Table 1. Molecular Diagnoses in Genes Previously Associated with Stillbirth.","doi":"10.1056/NEJMdo005825","issueDate":"2020-08-12T00:00Z","age":"6Months-1990","isFree":"y","topics":"Reproductive Medicine|Obstetrics/Gynecology General","viewType":"Full","specialties":"Obstetrics/Gynecology|Genetics"},{"type":"Interactive Graphic","title":"Table 2. Molecular Diagnoses in Genes Previously Associated with Stillbirth but Strong Candidates for Phenotype Expansion.","doi":"10.1056/NEJMdo005826","issueDate":"2020-08-12T00:00Z","age":"6Months-1990","isFree":"y","topics":"Reproductive Medicine|Obstetrics/Gynecology General","viewType":"Full","specialties":"Obstetrics/Gynecology|Genetics"},{"type":"Interactive Graphic","title":"Table 3. Candidate Stillbirth Genes.","doi":"10.1056/NEJMdo005827","issueDate":"2020-08-12T00:00Z","age":"6Months-1990","isFree":"y","topics":"Reproductive Medicine|Obstetrics/Gynecology General","viewType":"Full","specialties":"Obstetrics/Gynecology|Genetics"}]};</script>
    <script type="text/javascript">var mmsDataAdd = {"user":{"customerType":"null","accessInstitutionID":"null"}};</script>
    <!-- Prehiding snippet for Adobe Target with asynchronous Launch deployment -->
    <!-- placeholder id=null, description=Adobe-Target-Prehiding-Snippet --><!--
  Adobe Target - Prehiding Snippet
  v2.0.0
-->
<script nonce="94d64844d6a5f20e-SJC">
function tgtNoFlicker(e,n,t,o,i,c,l,r){function d(){return n.getElementsByTagName("head")[0]}function a(e,t){if(e){var o=n.getElementById(t);o&&e.removeChild(o)}}function f(e,o){!function(e,t,o){if(e){var i=n.createElement("style");i.id=t,i.innerHTML=o,e.appendChild(i)}}(d(),t,e),setTimeout(function(){a(d(),t)},o)}var m;n.location.href.indexOf(c)>-1?f(l,r):f(o,i),n.location.href.indexOf("mboxDisable")>-1?(a(d(),t),console.log("[TARGET] - Disabled, removing prehiding style")):(m=function(e){e&&(a(d(),t),console.log("[TARGET] - Blocked, removing prehiding style"))},fetch("//cdn.tt.omtrdc.net/cdn/target-vec.js",{method:"HEAD",mode:"no-cors"}).then(e=>m(!1)).catch(e=>m(!0)))}

tgtNoFlicker(window, document, 'at-body-style', 'div[data-location-name="Anonymous Module"], div[data-location-name="Anonymous Banner"], main > div.page-body > div > div > div > div.col-md-2-3.o-col.o-col--primary > div[class*="u-hide"], main > div.page-body > div > div > div > div.col-md-1-3.o-col.o-col--secondary > aside, header > h1#mainContent, a.m-article__link[href*="/10.1056/NEJMoa2116747"] > span.m-article__blurb > p, li.g-nejm-group__user-tool-secondary {opacity: 0 !important}', 3000);

tgtNoFlicker(window, document, 'at-mbox-style', '.N-ad-global-top-right, #N-ad-myNEJM-contextual-top, .N-ad-recentArticles, #N-ad-article-rightRail-1, #N-ad-siteWide-rightRail-1, #N-ad-imageChallenge-rightRail-2 {opacity: 0 !important}', 3000);
</script><!-- /Prehiding snippet for Adobe Target with asynchronous Launch deployment -->
    <!-- Launch Header Embed Code -->
    <script src="//assets.adobedtm.com/launch-ENcd6450fae3ae4870becbe6a2b01ab2e1.min.js" async="" type="ta-blocked" data-ta-blocked="1"></script>
    <!-- /Launch Header Embed Code -->

                    
                
            
        
    
        
            
                
                
                
                    
                    <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="New England Journal of Medicine"><meta name="dc.Title" content="Causal Genetic Variants in Stillbirth"><meta name="dc.Creator" content="Kate E. Stanley"><meta name="dc.Creator" content="Jessica Giordano"><meta name="dc.Creator" content="Vanessa Thorsten"><meta name="dc.Creator" content="Christie Buchovecky"><meta name="dc.Creator" content="Amanda Thomas"><meta name="dc.Creator" content="Mythily Ganapathi"><meta name="dc.Creator" content="Jun Liao"><meta name="dc.Creator" content="Avinash V. Dharmadhikari"><meta name="dc.Creator" content="Anya Revah-Politi"><meta name="dc.Creator" content="Michelle Ernst"><meta name="dc.Creator" content="Natalie Lippa"><meta name="dc.Creator" content="Halie Holmes"><meta name="dc.Creator" content="Gundula Povysil"><meta name="dc.Creator" content="Joseph Hostyk"><meta name="dc.Creator" content="Corette B. Parker"><meta name="dc.Creator" content="Robert Goldenberg"><meta name="dc.Creator" content="George R. Saade"><meta name="dc.Creator" content="Donald J. Dudley"><meta name="dc.Creator" content="Halit Pinar"><meta name="dc.Creator" content="Carol Hogue"><meta name="dc.Creator" content="Uma M. Reddy"><meta name="dc.Creator" content="Robert M. Silver"><meta name="dc.Creator" content="Vimla Aggarwal"><meta name="dc.Creator" content="Andrew S. Allen"><meta name="dc.Creator" content="Ronald J. Wapner"><meta name="dc.Creator" content="David B. Goldstein"><meta name="dc.Description" content="In the majority of cases, the cause of stillbirth remains unknown despite detailed clinical and laboratory evaluation. Approximately 10 to 20% of stillbirths are attributed to chromosomal abnormali..."><meta name="Description" content="In the majority of cases, the cause of stillbirth remains unknown despite detailed clinical and laboratory evaluation. Approximately 10 to 20% of stillbirths are attributed to chromosomal abnormali..."><meta name="dc.Publisher" content="Massachusetts Medical Society"><meta name="dc.Date" scheme="WTN8601" content="2020-09-17"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1056/NEJMoa1908753"><meta name="dc.Identifier" scheme="publisher-id" content="NJ202009173831208"><meta name="dc.Language" content="EN"><meta name="dc.Relation" content="10.1056/NEJMe2016410"><meta name="dc.Relation" content="YXQYoa1908753"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="Copyright Â© 2020 Massachusetts Medical Society. All rights reserved.">




<link rel="meta" type="application/atom+xml" href="https://doi.org/10.1056%2FNEJMoa1908753">
<link rel="meta" type="application/rdf+json" href="https://doi.org/10.1056%2FNEJMoa1908753">
<link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1056%2FNEJMoa1908753">


                    
                
            
        
    
        
            
                
                
                
                
                
            
        
    
        
            
                
                
                
                    
                    





<meta name="robots" content="noarchive">





    





    
    
        
        <meta name="twitter:title" content="Causal Genetic Variants in Stillbirth | NEJM">
        <meta property="og:title" content="Causal Genetic Variants in Stillbirth | NEJM">
    


    <meta property="og:type" content="article">


    
    
        <meta property="og:url" content="https://www.nejm.org/doi/full/10.1056/NEJMoa1908753">
    
    


    
    
        <meta property="og:image" content="https://www.nejm.org/cms/asset/fc1d9fab-8597-4cb8-b3b1-b9c2a5e17ea8/nejmoa1908753_f1.jpg">
        <meta name="twitter:image" content="https://www.nejm.org/cms/asset/fc1d9fab-8597-4cb8-b3b1-b9c2a5e17ea8/nejmoa1908753_f1.jpg">
    



    <meta property="og:site_name" content="The New England Journal of Medicine">


    
    
        
        <meta property="og:description" content="In the majority of cases, the cause of stillbirth remains unknown despite detailed
clinical and laboratory evaluation. Approximately 10 to 20% of stillbirths are attributed
to chromosomal abnormali...">
        <meta name="twitter:description" content="In the majority of cases, the cause of stillbirth remains unknown despite detailed
clinical and laboratory evaluation. Approximately 10 to 20% of stillbirths are attributed
to chromosomal abnormali...">
    


    
    
        <meta property="og:image:width" content="2640">
    


    
    
        <meta property="og:image:height" content="2347">
    


    <meta name="twitter:card" content="summary">


    <meta name="twitter:site" content="@NEJM">


<meta name="viewport" content="width=device-width,initial-scale=1"><meta name="citation_fulltext_world_readable" content="">
    <meta name="publication_doi" content="10.1056/NEJMoa1908753">


    















                    
                
            
        
    
        
    
        
    
        
            
                
                
                
                
                
            
        
    




<meta name="format-detection" content="telephone=no">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
    




    
        
        
        <link rel="canonical" href="https://www.nejm.org/doi/full/10.1056/NEJMoa1908753">
    
    







<meta name="pbContext" content=";requestedJournal:journal:nejm;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:doi\:10.1056/nejm_2020.383.issue-12;article:article:doi\:10.1056/NEJMoa1908753;wgroup:string:MMS NextGen Website Group;pageGroup:string:Publication Pages;subPage:string:Full Text;website:website:mms-site;journal:journal:nejms">
        <link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-933-8e181dff8b36ea01dc4c.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-34-6ea546d794fb7a1cea52.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-743-ddd329e1530654ec0846.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-305-9eee37798fdfe5196307.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-388-bb098c62445f518c0b46.css"><link rel="stylesheet" type="text/css" href="https://cssjs.nejm.org/custreg-556-8e58ca05fb5497e541ef.css"><script charset="utf-8" src="https://www.nejm.org/products/mms-nextgen/mms/releasedAssets/js/0-dfd243c732a07d3608f6.js"></script><script charset="utf-8" src="https://www.nejm.org/products/mms-nextgen/mms/releasedAssets/js/7-10d355ba95f7ec1fddf2.js"></script><script charset="utf-8" src="https://www.nejm.org/products/mms-nextgen/mms/releasedAssets/js/11-04bf2a0da30146b4a5d0.js"></script><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><link rel="preconnect" href="https://use.typekit.net"><meta http-equiv="P3P" content="CP=&quot;NOI DSP ADM OUR IND OTC&quot;"><script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-288192a491f0ad9cc434f05ea175659426f5ab0cfd6befb3f3592f96bb90b0d1.js"></script><link rel="stylesheet" href="https://quilt-cdn.janrain.com/HEAD/providers.css" type="text/css"><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-59614f5c46b49b21eeef3bb28c4fb38d1e7069e8d014752fcb66e84942556802.css"></head>
    

    



    
        
        
        
            <body class="pb-ui website-mms-site loaded">
            






    
    
        





    
    
        
        
    
    
    







<input type="hidden" id="isRoMode" name="isRoMode" value="false">

        <div id="pb-page-content" data-ng-non-bindable="">
                
            <div data-pb-dropzone="main" data-pb-dropzone-name="Main">
                
                    
                        



        
        <header data-sticky="false" class="ng-header js_stickyAd"><a href="#mainContent" class="ng-skipToMainContent">Skip to main content</a><div class="ng-header_topAd"><div class="ng-header_topAd-content d-flex justify-content-center align-items-center">



        
        <div id="DTM_Position_Topbanner" class="ad">
 <!--emptycomment-->
</div>
</div></div><div class="ng-header_journal-navbar"><div class="ng-header_journal-navbar-content"><div class="container-fluid"><div class="row"><div class="ng-header_journal-navbar_left col-lg-8 col-sm-12 pr-lg-0 d-flex align-items-center"><div data-location="nejm_group_product_nav" class="ng-header_journalNavMenu">



        
        <nav data-location="nejm_group_product_nav" role="navigation" class="ng-journalNavMenu">
<ul class="ng-journalNavMenu_list">
<li class="ng-journalNavMenu_list-item"><a href="/" class="ng-journalNavMenu_mainLink active" aria-current="page" aria-label="The New England Journal of Medicine selected item"><span class="ng-journalNavMenu_mainLink-text d-lg-block d-none">The New England Journal of Medicine</span><span class="ng-journalNavMenu_mainLink-text d-lg-none d-block">NEJM</span></a></li>
<li class="ng-journalNavMenu_list-item"><a href="https://evidence.nejm.org/" class="ng-journalNavMenu_mainLink"><span class="ng-journalNavMenu_mainLink-text"><span class="ng-journalNavMenu_mainLink-text-firstWord">NEJM</span> Evidence</span></a></li>
<li class="ng-journalNavMenu_list-item"><a href="https://ai.nejm.org/" class="ng-journalNavMenu_mainLink"><span class="ng-journalNavMenu_mainLink-text"><span class="ng-journalNavMenu_mainLink-text-firstWord">NEJM</span> AI</span></a></li>
<li class="ng-journalNavMenu_list-item"><a href="https://catalyst.nejm.org/" class="ng-journalNavMenu_mainLink"><span class="ng-journalNavMenu_mainLink-text"><span class="ng-journalNavMenu_mainLink-text-firstWord">NEJM</span> Catalyst</span></a></li>
<li class="ng-journalNavMenu_list-item"><a href="https://www.jwatch.org/" class="ng-journalNavMenu_mainLink"><span class="ng-journalNavMenu_mainLink-text"><span class="ng-journalNavMenu_mainLink-text-firstWord">NEJM</span> Journal Watch</span></a></li>
</ul>
</nav>

</div></div><div class="ng-header_journal-navbar_right col-lg-4 pl-lg-0 d-none d-xl-flex align-items-center justify-content-end"><div class="ng-header_loginBar d-xl-flex align-items-lg-center">



        
        <div data-location="authstring_nav" class="ng-header_loginBar-menu"><span class="ng-simple-menu_mainLink-icon signIn newIcon-2"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#user2"></use></svg></span></span><span class="ng-header_loginBar-signIn"><a href="/sign-in?uri=/doi/full/10.1056/NEJMoa1908753" class="ng-header_loginBar-signIn-link">Sign In</a></span><span class="ng-header_loginBar-separator">|</span><span class="ng-header_loginBar-createAcc"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC1" class="ng-header_loginBar-createAcc-link ro-mode_disabled">Create Account</a></span></div><div data-location="authstring_nav" class="ng-header_loginBar-subscribe pl-xl-16"><a href="/action/storeProxy?action=subscribe&amp;promo=ONF4NRS1" class="ng-header_loginBar-subscribe-btn ng-btn_primary ro-mode_disabled">Subscribe</a></div>
</div></div></div></div></div></div><div class="ng-header_instBanner d-none d-lg-block"></div><div class="ng-header_logoBar"><div class="container-fluid"><div class="row"><div data-location="logo" class="ng-header_logoBar_left col-lg-4 col-6 d-flex align-items-center justify-content-start"><a href="/" title="The New England Journal of Medicine homepage" class="ng-header_logo-link"><img src="/specs/products/mms-nextgen/mms/releasedAssets/images/other-images/nejm-logo-166667e91992a5212bc723b03e45d39f.svg" alt="The New England Journal of Medicine homepage" width="366" height="58" class="ng-header_logo-image"></a></div><div class="ng-header_logoBar_right ng-header_navBar col-lg-8 col-sm-12 p-lg-0 align-items-center justify-content-end"><div class="ng-header_navBar-content"><div class="container-fluid"><div class="row"><div class="ng-header_instBanner-md col-12 d-block d-lg-none"></div><div class="ng-header_loginBar-md col-12 d-block d-xl-none">



        
        <div data-location="authstring_nav" class="ng-header_loginBar-menu"><span class="ng-simple-menu_mainLink-icon signIn newIcon-2"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#user2"></use></svg></span></span><span class="ng-header_loginBar-signIn"><a href="/sign-in?uri=/doi/full/10.1056/NEJMoa1908753" class="ng-header_loginBar-signIn-link">Sign In</a></span><span class="ng-header_loginBar-separator">|</span><span class="ng-header_loginBar-createAcc"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC1" class="ng-header_loginBar-createAcc-link ro-mode_disabled">Create Account</a></span></div><div data-location="authstring_nav" class="ng-header_loginBar-subscribe pl-xl-16"><a href="/action/storeProxy?action=subscribe&amp;promo=ONF4NRS1" class="ng-header_loginBar-subscribe-btn ng-btn_primary ro-mode_disabled">Subscribe</a></div>
</div><div class="ng-header_navBar_left col-lg-8 col-sm-12 d-flex align-items-center"><div class="ng-header_stickyLogo mr-32"><a href="/" title="The New England Journal of Medicine homepage" class="ng-header_stickyLogo-link"><img src="/specs/products/mms-nextgen/mms/releasedAssets/images/other-images/nejm-small-logo-070df7a3f4d8cfa2e71c3c45bb4b172f.svg" alt="The New England Journal of Medicine homepage" class="ng-header_stickyLogo-image"></a></div><div class="ng-header_mainMenu">









    
    
        <div data-widget-def="menuWidget" data-widget-id="97fb3983-db63-4b0b-97dd-d9bb7929fce1" data-location="main_nav">
        



        
        <nav class="ng-menu"><ul class="ng-menu_list ng-simple-menu"><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/toc/nejm/current" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">current issue</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/toc/nejm/current" class="ng-simple-menu_list-link">VIEW CURRENT ISSUE</a></li><li class="ng-simple-menu_list-item"><a href="/toc/nejm/recently-published" class="ng-simple-menu_list-link">BROWSE RECENTLY PUBLISHED</a></li><li class="ng-simple-menu_list-item"><a href="/loi/nejm" class="ng-simple-menu_list-link">BROWSE ALL ISSUES</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/medical-specialties" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">SPECIALTIES</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/browse/specialty/cardiology" class="ng-simple-menu_list-link">Cardiology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/clinical-medicine" class="ng-simple-menu_list-link">Clinical Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/endocrinology" class="ng-simple-menu_list-link">Endocrinology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/gastroenterology" class="ng-simple-menu_list-link">Gastroenterology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/hematology-oncology" class="ng-simple-menu_list-link">Hematology/Oncology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/infectious-disease" class="ng-simple-menu_list-link">Infectious Disease</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/nephrology" class="ng-simple-menu_list-link">Nephrology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/neurology-neurosurgery" class="ng-simple-menu_list-link">Neurology/Neurosurgery</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/obstetrics-gynecology" class="ng-simple-menu_list-link">Obstetrics/Gynecology</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/pediatrics" class="ng-simple-menu_list-link">Pediatrics</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/pulmonary-critical-care" class="ng-simple-menu_list-link">Pulmonary/Critical Care</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/surgery" class="ng-simple-menu_list-link">Surgery</a></li><li class="ng-simple-menu_list-item"><a href="/medical-specialties" class="ng-simple-menu_list-link">View All Specialties</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/topics" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">TOPICS</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/ai-in-medicine" class="ng-simple-menu_list-link">AI in Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/climate-change" class="ng-simple-menu_list-link">Climate Change</a></li><li class="ng-simple-menu_list-item"><a href="/coronavirus" class="ng-simple-menu_list-link">Coronavirus</a></li><li class="ng-simple-menu_list-item"><a href="/equity" class="ng-simple-menu_list-link">Efforts toward Equity</a></li><li class="ng-simple-menu_list-item"><a href="/firearm-injury-prevention" class="ng-simple-menu_list-link">Firearm Injury Prevention</a></li><li class="ng-simple-menu_list-item"><a href="/nam" class="ng-simple-menu_list-link">From the National Academy of Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/fundamentals-of-medical-ethics" class="ng-simple-menu_list-link">Fundamentals of Medical Ethics</a></li><li class="ng-simple-menu_list-item"><a href="/gray-matters" class="ng-simple-menu_list-link">Gray Matters</a></li><li class="ng-simple-menu_list-item"><a href="/browse/specialty/health-policy" class="ng-simple-menu_list-link">Health Policy</a></li><li class="ng-simple-menu_list-item"><a href="/medicine-and-society" class="ng-simple-menu_list-link">Medicine and Society</a></li><li class="ng-simple-menu_list-item"><a href="/nutrition-in-medicine" class="ng-simple-menu_list-link">Nutrition in Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/outbreaks" class="ng-simple-menu_list-link">Outbreaks Center</a></li><li class="ng-simple-menu_list-item"><a href="/race-and-medicine" class="ng-simple-menu_list-link">Race and Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/recognizing-historical-injustices" class="ng-simple-menu_list-link">Recognizing Historical Injustices and the Journal</a></li><li class="ng-simple-menu_list-item"><a href="/tobacco-use-reduction" class="ng-simple-menu_list-link">Tobacco Use Reduction</a></li><li class="ng-simple-menu_list-item"><a href="/topics" class="ng-simple-menu_list-link">View All Topics</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/multimedia" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">MULTIMEDIA</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/rss-feed" class="ng-simple-menu_list-link">Podcasts</a></li><li class="ng-simple-menu_list-item"><a href="/double-take" class="ng-simple-menu_list-link">Double Takes</a></li><li class="ng-simple-menu_list-item"><a href="https://illustrated-glossary.nejm.org/" class="ng-simple-menu_list-link">Illustrated Glossary</a></li><li class="ng-simple-menu_list-item"><a href="/image-challenge" class="ng-simple-menu_list-link">Image Challenge</a></li><li class="ng-simple-menu_list-item"><a href="/browse/nejm-media-type/images-in-clinical-medicine" class="ng-simple-menu_list-link">Images in Clinical Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/browse/nejm-media-type/interactive-medical-case" class="ng-simple-menu_list-link">Interactive Medical Cases</a></li><li class="ng-simple-menu_list-item"><a href="/plain-language-research-summaries" class="ng-simple-menu_list-link">Plain Language/Research Summaries</a></li><li class="ng-simple-menu_list-item"><a href="/multimedia/quick-take-video" class="ng-simple-menu_list-link">Quick Takes</a></li><li class="ng-simple-menu_list-item"><a href="/browse/nejm-media-type/videos-in-clinical-medicine" class="ng-simple-menu_list-link">Videos in Clinical Medicine</a></li><li class="ng-simple-menu_list-item"><a href="/multimedia" class="ng-simple-menu_list-link">View All Multimedia</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item hasChildren"><a href="/learning" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">LEARNING/CME</span><span class="ng-simple-menu_mainLink-arrowIcon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-down"></use></svg></span></span></a><div class="ng-simple-menu_dropblock"><div class="ng-simple-menu_dropblock-cont"><div class="ng-simple-menu_list"><ul class="ng-simple-menu_list-items"><li class="ng-simple-menu_list-item"><a href="/continuing-medical-education" class="ng-simple-menu_list-link">Weekly CME</a></li><li class="ng-simple-menu_list-item"><a href="/cme-ce/obesity" class="ng-simple-menu_list-link">Obesity CME/CE</a></li><li class="ng-simple-menu_list-item"><a href="/learning" class="ng-simple-menu_list-link">VIEW ALL LEARNING/CME</a></li></ul></div></div></div></li><li class="ng-menu_list-item ng-simple-menu_item"><a href="/author-center/home" class="ng-simple-menu_mainLink"><span class="ng-simple-menu_mainLink-text">AUTHOR CENTER</span></a></li></ul></nav>

        </div>
    

</div><button aria-label="Search" class="ng-header_searchBtn searchBtnNav d-block ml-24"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#search"></use></svg></span></button></div><div class="ng-header_loginBar ng-header_loginBar-sticky col-4">



        
        <div data-location="authstring_nav" class="ng-header_loginBar-menu"><span class="ng-simple-menu_mainLink-icon signIn newIcon-2"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#user2"></use></svg></span></span><span class="ng-header_loginBar-signIn"><a href="/sign-in?uri=/doi/full/10.1056/NEJMoa1908753" class="ng-header_loginBar-signIn-link">Sign In</a></span><span class="ng-header_loginBar-separator">|</span><span class="ng-header_loginBar-createAcc"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC1" class="ng-header_loginBar-createAcc-link ro-mode_disabled">Create Account</a></span></div><div data-location="authstring_nav" class="ng-header_loginBar-subscribe pl-xl-16"><a href="/action/storeProxy?action=subscribe&amp;promo=ONF4NRS1" class="ng-header_loginBar-subscribe-btn ng-btn_primary ro-mode_disabled">Subscribe</a></div>
</div></div></div></div></div><div class="ng-header_logoBar_right-lg col-lg-8 col-6 d-flex align-items-center justify-content-end d-xl-none"><button aria-label="Search" class="ng-header_searchBtn d-flex justify-content-end"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#search"></use></svg></span></button><div class="ng-header_openCloseBtns d-flex align-items-center justify-content-end"><button aria-label="Open" class="ng-header_openBtn"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#burger"></use></svg></span></button><button aria-label="Close" class="ng-header_closeBtn"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#close"></use></svg></span></button></div></div><div class="ng-header_quickSearch col-12"><div class="ng-header_quickSearch-content">



        
        <div class="ng-quick-search js-ng-quick-search"><form action="/search" method="get" class="ng-quick-search_form js-form"><div class="ng-quick-search_inputArea js-inputArea"><input type="search" name="q" placeholder="Enter keyword, author, title or citation" autocomplete="off" value="" required="required" class="ng-quick-search_allField ng-large-text-input js-allField js-allField-def-tmpl ui-autocomplete-input"><button type="submit" aria-label="Search Button" class="ng-quick-search_iconBtn d-xl-none d-block js-submitBtn"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#search"></use></svg></span></button><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" style="display: none;"></ul></div><div class="ng-quick-search_btnsArea d-none d-xl-flex align-items-center justify-content-end"><a href="/search/advanced" class="ng-quick-search_advSearchLink ng-btn_link js-advancedSearchLink">Advanced Search</a><button type="submit" class="ng-quick-search_textBtn ng-btn_secondary ng-btn_iconRight js-submitBtn"><span class="ng-btn_text">SEARCH</span><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-right"></use></svg></span></button></div></form></div>
</div></div></div></div></div></header><div class="ng-header_overlay"></div><div class="ng-header_after">









    
    
        <div data-widget-def="literatumAd" data-widget-id="4931ddbd-d46e-4597-b4fd-92549e9847ca" id="ad-global-banner-FULLx64-1">
        



        
        



    

    

    
        
    



        </div>
    

</div>

                    
                        



        
        <div>









    
    
        <main data-widget-def="axelPublicationContent" data-widget-id="fcdb5261-d6a2-4a9c-81d5-3a3e2304e9c1" id="mainContent">
        



        
        <div class="article-tools__savePopup"><div class="ng-save"><input type="hidden" value="/action/getSavedTags" class="getKeywordsServiceUrl"><input type="hidden" value="/action/mmsSaveItemsService?itemType=article" class="saveServiceUrl"><input type="hidden" value="article" class="saveType"><input type="hidden" value="10.1056/NEJMoa1908753" class="inputDoi"><input type="hidden" value="K.E. Stanley and Others" class="inputAuthor"><input type="hidden" value="N Engl J Med 2020;383:1107-1116" class="inputCitation"><input type="hidden" value="08-12-2020" class="inputEPubDate"><input type="hidden" value="September 2020" class="inputCoverDate"><input type="hidden" value="Original Article" class="inputContentType"><input type="hidden" value="Causal Genetic Variants in Stillbirth" class="inputArticleTitle"><input type="hidden" value="/browse/nejm-article-type/original-article" class="inputContentTypeUrl"><button type="button" id="saveArticleShowPopupBtn" aria-hidden="true" tabindex="-1" class="ng-save-btn ng-btn_secondary isLoggedOut">Save</button></div></div><div class="article-tools__articleAlertPopup"><div id="articleAlertModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true" class="ng-modal modal"><div role="document" class="ng-modal_dialog modal-dialog modal-dialog-centered"><div class="ng-modal_content modal-content"><div class="ng-modal_header modal-header"><h5 id="articleAlertModalTitle" class="ng-modal_title modal-title">Create an E-mail Alert for This Article</h5><button type="button" data-dismiss="modal" aria-label="Close" class="ng-modal_close close"><span aria-hidden="true"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#close"></use></svg></span></span></button></div><div class="ng-modal_body modal-body"></div></div></div></div></div><div class="ng-do-media_popup"><div id="doPopup" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true" class="ng-modal modal"><div role="document" class="ng-modal_dialog modal-dialog modal-dialog-centered"><div class="ng-modal_content modal-content"><div class="ng-modal_header modal-header"><button type="button" data-dismiss="modal" aria-label="Close" class="ng-modal_close close"><span aria-hidden="true"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#close"></use></svg></span></span></button></div><div class="ng-modal_body modal-body"></div></div></div></div></div><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/mms-nextgen/mms/releasedAssets/css/build-article-5b89b331018866a4a965.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" data-type="research-article" vocab="http://schema.org/" typeof="ScholarlyArticle" lang="en" dir="ltr"><header data-extent="frontmatter" data-location="header_article"><div class="core-container"><div data-article-access="free" data-article-access-type="free" class="meta-panel"><div class="meta-panel__left-content"><div class="meta-panel__type"><a href="/browse/nejm-article-type/original-article">Original Article</a></div></div><div class="meta-panel__right-content"><div class="meta-panel__share">









    
    
        <div data-widget-def="UX3share" data-widget-id="eee5f139-16f2-48f8-97ea-1b114367da16" data-location="share_tools_article">
        



        
        <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js" nonce="94d64844d6a5f20e-SJC"></script><div class="share"><div class="share__block share__inline-links"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="d-flex list-unstyled a2a a2a_kit a2a_default_style mb-0 a2a_kit_size_32" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Facebook" data-interactiontype="social" class="share__link a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-facebook"></use></svg></span></a></li><li class="a2a_listitem_custom"><a role="link" title="X (formerly Twitter)" data-interactiontype="social" class="share__link a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-twitter"></use></svg></span></a></li><li class="a2a_listitem_custom"><a role="link" title="Linked In" data-interactiontype="social" class="share__link a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-linkedin"></use></svg></span></a></li><li class="a2a_listitem_custom"><a role="link" title="Email" data-interactiontype="content_email_click" class="share__link a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-email"></use></svg></span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" data-interactiontype="social" class="share__link a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><span class="icon-component " style="height: 24px; width: 24px"><svg aria-hidden="true" style="height: 24px; width: 24px"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#social-bluesky"></use></svg></span></a></li><div style="clear: both;"></div></ul></div></div>

        </div>
    

</div></div></div><h1 property="name">Causal Genetic Variants in Stillbirth</h1><div class="core-relations my-3"></div><div class="contributors"><span class="authors"><span class="heading">Authors</span>: <span role="list"><span property="author" typeof="Person" role="listitem"><span property="givenName">Kate E.</span> <span property="familyName">Stanley</span>, <span property="honorificSuffix">B.A.</span></span>, <span property="author" typeof="Person" role="listitem"><span property="givenName">Jessica</span> <span property="familyName">Giordano</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span>, <span property="author" typeof="Person" role="listitem"><span property="givenName">Vanessa</span> <span property="familyName">Thorsten</span>, <span property="honorificSuffix">M.P.H.</span></span>, <span property="author" typeof="Person" role="listitem"><span property="givenName">Christie</span> <span property="familyName">Buchovecky</span>, <span property="honorificSuffix">Ph.D.</span></span>, <span property="author" typeof="Person" role="listitem"><span property="givenName">Amanda</span> <span property="familyName">Thomas</span>, <span property="honorificSuffix">Ph.D.</span></span>, <span property="author" typeof="Person" role="listitem"><span property="givenName">Mythily</span> <span property="familyName">Ganapathi</span>, <span property="honorificSuffix">Ph.D.</span></span>, <span property="author" typeof="Person" role="listitem"><span property="givenName">Jun</span> <span property="familyName">Liao</span>, <span property="honorificSuffix">Ph.D.</span></span><span data-displayed-on="all">, <span data-action="reveal" tabindex="0" role="listitem">+18</span> </span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Avinash V.</span> <span property="familyName">Dharmadhikari</span>, <span property="honorificSuffix">Ph.D.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Anya</span> <span property="familyName">Revah-Politi</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Michelle</span> <span property="familyName">Ernst</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Natalie</span> <span property="familyName">Lippa</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Halie</span> <span property="familyName">Holmes</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Gundula</span> <span property="familyName">Povysil</span>, <span property="honorificSuffix">M.D., Ph.D.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Joseph</span> <span property="familyName">Hostyk</span>, <span property="honorificSuffix">B.S.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Corette B.</span> <span property="familyName">Parker</span>, <span property="honorificSuffix">Dr.P.H.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Robert</span> <span property="familyName">Goldenberg</span>, <span property="honorificSuffix">M.D.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">George R.</span> <span property="familyName">Saade</span>, <span property="honorificSuffix">M.D.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Donald J.</span> <span property="familyName">Dudley</span>, <span property="honorificSuffix">M.D.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Halit</span> <span property="familyName">Pinar</span>, <span property="honorificSuffix">M.D.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Carol</span> <span property="familyName">Hogue</span>, <span property="honorificSuffix">Ph.D., M.P.H.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Uma M.</span> <span property="familyName">Reddy</span>, <span property="honorificSuffix">M.D., M.P.H.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Robert M.</span> <span property="familyName">Silver</span>, <span property="honorificSuffix">M.D.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Vimla</span> <span property="familyName">Aggarwal</span>, <span property="honorificSuffix">M.B., B.S.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Andrew S.</span> <span property="familyName">Allen</span>, <span property="honorificSuffix">Ph.D.</span></span></span><span data-hidden-on="all">, <span property="author" typeof="Person" role="listitem"><span property="givenName">Ronald J.</span> <span property="familyName">Wapner</span>, <span property="honorificSuffix">M.D.</span></span></span>, and <span property="author" typeof="Person" role="listitem"><span property="givenName">David B.</span> <span property="familyName">Goldstein</span>, <span property="honorificSuffix">Ph.D.</span></span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">-18</span></span></span></span><a href="#tab-contributors" class="to-authors-affiliations" data-id="article-authors-viewall">Author Info &amp; Affiliations</a></div><div class="core-self-citation"><div class="core-date-published">Published <span property="datePublished">August 12, 2020</span></div><div property="isPartOf" typeof="Periodical"><span property="name">N Engl J Med</span> 2020<span property="isPartOf" typeof="PublicationVolume">;<span property="volumeNumber">383</span></span>:<span property="pageStart">1107</span>-<span property="pageEnd">1116</span></div><div class="doi">DOI: 10.1056/NEJMoa1908753</div><div class="core-enumeration"><a href="/toc/nejm/383/12"><span property="isPartOf" typeof="PublicationVolume">VOL. <span property="volumeNumber">383</span></span> <span property="isPartOf" typeof="PublicationIssue">NO. <span property="issueNumber">12</span></span></a></div><div><a href="#tab-information">Copyright Â© 2020</a></div></div><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__metrics info-panel__item"></div></div><div data-location="tools_article" class="info-panel__right-content"><div class="info-panel__article_tools info-panel__item">



        
        <div data-permission="/servlet/linkout?type=rightslink&amp;url=publisherName%3Dmassmed%26author%3DKate%2BE.%2BStanley%252C%2BJessica%2BGiordano%252C%2BVanessa%2BThorsten%252C%2Bet%2Bal%26publication%3Dnejm%26orderBeanReset%3Dtrue%26volumeNum%3D383%26issueNum%3D12%26contentID%3D10.1056%252FNEJMoa1908753%26title%3DCausal%2BGenetic%2BVariants%2Bin%2BStillbirth%26publicationDate%3D09%252F17%252F2020" class="article-tools"><button id="articleToolsAlertBtn" data-url="/action/addCitationAlert?doi=10.1056%2FNEJMoa1908753" data-toggle="" data-target="#articleAlertModal" aria-label="Sign in or create account to add an article alert" data-interactiontype="article_alert" class="article-tools__citation btn btn--slim isLoggedOut"><span title="" data-toggle="tooltip" class="d-inline-block" data-original-title="Sign in or create account to add an article alert"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#alerts"></use></svg></span></span></button><button id="articleToolsSaveBtn" data-toggle="modal" data-target="#saveArticle" aria-label="Sign in or create account to save this article" data-interactiontype="article_tools_save_click" class="article-tools__favorite btn btn--slim isLoggedOut"><span title="" data-toggle="tooltip" class="d-inline-block" data-original-title="Sign in or create account to save this article"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#bookmarkFilled"></use></svg></span></span></button><a href="/servlet/linkout?type=rightslink&amp;url=publisherName%3Dmassmed%26author%3DKate%2BE.%2BStanley%252C%2BJessica%2BGiordano%252C%2BVanessa%2BThorsten%252C%2Bet%2Bal%26publication%3Dnejm%26orderBeanReset%3Dtrue%26volumeNum%3D383%26issueNum%3D12%26contentID%3D10.1056%252FNEJMoa1908753%26title%3DCausal%2BGenetic%2BVariants%2Bin%2BStillbirth%26publicationDate%3D09%252F17%252F2020" target="_blank" title="" data-toggle="tooltip" aria-label="Permissions" data-interactiontype="article_tools_permissions_click" class="article-tools__permissions btn btn--slim" data-original-title="Permissions"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#permissions"></use></svg></span></a><a href="/about-nejm/reprints" title="" data-toggle="tooltip" aria-label="Reprints" data-interactiontype="article_tools_reprints_click" class="article-tools__reprints btn btn--slim" data-original-title="Reprints"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#reprints"></use></svg></span></a><a href="/cms/asset/155dc3ab-f043-4eb6-8d21-9eacec802f81/nejmoa1908753.pptx" target="_blank" title="" data-toggle="tooltip" aria-label="Download Slides" data-interactiontype="article_tools_slide_set_download" data-multimedia-type="Article Slideset" data-multimedia-filename="nejmoa1908753.pptx" class="article-tools__downloadSlides btn btn--slim" data-original-title="Download Slides"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#download"></use></svg></span></a><div class="article-tools__permissionsPopup"><div id="permissionsModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true" class="ng-modal modal"><div role="document" class="ng-modal_dialog modal-dialog modal-dialog-centered"><div class="ng-modal_content modal-content"><div class="ng-modal_header modal-header"><h5 id="permissionsModalTitle" class="ng-modal_title modal-title">Permissions</h5><button type="button" data-dismiss="modal" aria-label="Close" class="ng-modal_close close"><span aria-hidden="true"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#close"></use></svg></span></span></button></div><div class="ng-modal_body modal-body">For permission requests, please contact NEJM Reprints at <a href="mailto:reprints@nejm.org">reprints@nejm.org</a></div></div></div></div></div></div>
</div><div class="info-panel__citations info-panel__item"><a href="#tab-citations" class="btn btn--slim" aria-label="View Citations" data-toggle="tooltip" data-original-title="View Citations" data-interactiontype="article_tools_citation_download" data-multimedia-type="Citation"><i aria-hidden="true" class="icon-format_quote"></i></a></div><div class="info-panel__formats info-panel__item"><a href="https://www.nejm.org/doi/pdf/10.1056/NEJMoa1908753" class="btn btn--pdf btn--slim" aria-label="View PDF" data-toggle="tooltip" data-original-title="View PDF" data-behavior="trackDownloadEvent" data-doi="10.1056/NEJMoa1908753" data-interactiontype="article_tools_download_pdf" data-multimedia-type="Article PDF" data-multimedia-filename="nejmoa1908753.pdf"><i aria-hidden="true" class="icon-PDF"></i></a></div></div></div></div></header><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Contents</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><ul><li><a href="#summary-abstract" aria-current="true">Abstract</a></li><li><a href="#sec-1" aria-current="false">Methods</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li class="bordered"><a href="#backnotes" aria-current="false">Notes</a></li><li><a href="#supplementary-materials" aria-current="false">Supplementary Material</a></li><li><a href="#bibliography" aria-current="false">References</a></li></ul></nav></div><nav id="article_collateral_menu"><ul data-core-nav="collateral" class="collateral-pill" style="margin-top: 0px;"><li><a href="#core-collateral-info" data-id="article-nav-menubar-info" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Information &amp; Authors"><i aria-hidden="true" class="icon-info"></i><span class="sr-only">Information &amp; Authors</span></a></li><li><a href="#core-collateral-metrics" data-id="article-nav-menubar-metrics" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Metrics &amp; Citations"><i aria-hidden="true" class="icon-timeline"></i><span class="sr-only">Metrics &amp; Citations</span></a></li><li><a href="#core-collateral-fulltext-options" data-id="article-nav-menubar-metrics" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="View Options"><i aria-hidden="true" class="icon-eye"></i><span class="sr-only">View Options</span></a></li><li><a href="#core-collateral-references" data-id="article-nav-menubar-references" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="References"><i aria-hidden="true" class="icon-references"></i><span class="sr-only">References</span></a></li><li><a href="#core-collateral-media" data-id="article-nav-menubar-media" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Media"><i aria-hidden="true" class="icon-photo"></i><span class="sr-only">Media</span></a></li><li title="No tables available"><a href="#core-collateral-tables" data-id="article-nav-menubar-tables" disabled="true" aria-disabled="true" title="" tabindex="-1" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Tables"><i aria-hidden="true" class="icon-tables"></i><span class="sr-only">Tables</span></a></li><li><a href="#core-collateral-share" data-id="article-nav-menubar-share" title="" tabindex="0" data-toggle="tooltip" data-placement="left" data-offset="4px, 16px" data-original-title="Share"><i aria-hidden="true" class="icon-share"></i><span class="sr-only">Share</span></a></li></ul></nav></div></div><div data-core-wrapper="content"><div id="abstracts" data-extent="frontmatter" data-location="articleTab_article"><div class="core-container"><section id="summary-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Abstract</h2><section id="abs-sec-1"><h3>Background</h3><div role="paragraph">In the majority of cases, the cause of stillbirth remains unknown despite detailed clinical and laboratory evaluation. Approximately 10 to 20% of stillbirths are attributed to chromosomal abnormalities. However, the causal nature of single-nucleotide variants and small insertions and deletions in exomes has been understudied.</div></section><section id="abs-sec-2"><h3>Methods</h3><div role="paragraph">We generated exome sequencing data for 246 stillborn cases and followed established guidelines to identify causal variants in disease-associated genes. These genes included those that have been associated with stillbirth and strong candidate genes. We also evaluated the contribution of 18,653 genes in caseâcontrol analyses stratified according to the degree of depletion of functional variation (described here as âintoleranceâ to variation).</div></section><section id="abs-sec-3"><h3>Results</h3><div role="paragraph">We identified molecular diagnoses in 15 of 246 cases of stillbirth (6.1%) involving seven genes that have been implicated in stillbirth and six disease genes that are good candidates for phenotypic expansion. Among the cases we evaluated, we also found an enrichment of loss-of-function variants in genes that are intolerant to such variation in the human population (odds ratio, 2.15; 95% confidence interval [CI], 1.46 to 3.06). Loss-of-function variants in intolerant genes were concentrated in genes that have not been associated with human disease (odds ratio, 2.22; 95% CI, 1.41 to 3.34), findings that differ from those in two postnatal clinical populations that were also evaluated in this study.</div></section><section id="abs-sec-4"><h3>Conclusions</h3><div role="paragraph">Our findings establish the diagnostic utility of clinical exome sequencing to evaluate the role of small genomic changes in stillbirth. The strength of the novel risk signal (as generated through the stratified analysis) was similar to that in known disease genes, which indicates that the genetic cause of stillbirth remains largely unknown. (Funded by the Institute for Genomic Medicine.)</div></section></section></div></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text" data-location="articleTab_article"><div class="core-container"><div id="i1" class="core-digital-object ig">
<div class="ng-do-interactive">
    <div class="ng-do-interactive_label">Interactive Graphic</div>
    <div data-ajaxurl="/do/10.1056/NEJMdo005825/full/?ajaxRequest=true" class="ng-do-interactive_item">
        <div class="ng-do-interactive_item-left">
            <a href="#" class="ng-do-interactive_img-link" data-interactiontype="multimedia_click" data-multimedia-type="Interactive Graphic" data-multimedia-contentid="10.1056/NEJMdo005825">
                
                    <img src="/cms/asset/8f491259-d9e0-45f2-9a03-472dfe290fce/media/NEJMdo005825_300x200.jpg" class="ng-do-interactive_img">
                
                <span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#tooltip-interactive"></use></svg></span>
            </a>
        </div>
        <div class="ng-do-interactive_item-right">
            <a href="#" class="ng-do-interactive_link" data-interactiontype="multimedia_click" data-multimedia-type="Interactive Graphic" data-multimedia-contentid="10.1056/NEJMdo005825">Table 1. Molecular Diagnoses in Genes Previously Associated with Stillbirth.</a>
        </div>
    </div>
</div>

</div><div id="i2" class="core-digital-object ig">
<div class="ng-do-interactive">
    <div class="ng-do-interactive_label">Interactive Graphic</div>
    <div data-ajaxurl="/do/10.1056/NEJMdo005826/full/?ajaxRequest=true" class="ng-do-interactive_item">
        <div class="ng-do-interactive_item-left">
            <a href="#" class="ng-do-interactive_img-link" data-interactiontype="multimedia_click" data-multimedia-type="Interactive Graphic" data-multimedia-contentid="10.1056/NEJMdo005826">
                
                    <img src="/cms/asset/6f05d380-8b97-4973-8471-885a5d431aae/media/NEJMdo005826_300x200.jpg" class="ng-do-interactive_img">
                
                <span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#tooltip-interactive"></use></svg></span>
            </a>
        </div>
        <div class="ng-do-interactive_item-right">
            <a href="#" class="ng-do-interactive_link" data-interactiontype="multimedia_click" data-multimedia-type="Interactive Graphic" data-multimedia-contentid="10.1056/NEJMdo005826">Table 2. Molecular Diagnoses in Genes Previously Associated with Stillbirth but Strong Candidates for Phenotype Expansion.</a>
        </div>
    </div>
</div>

</div><div id="i3" class="core-digital-object ig">
<div class="ng-do-interactive">
    <div class="ng-do-interactive_label">Interactive Graphic</div>
    <div data-ajaxurl="/do/10.1056/NEJMdo005827/full/?ajaxRequest=true" class="ng-do-interactive_item">
        <div class="ng-do-interactive_item-left">
            <a href="#" class="ng-do-interactive_img-link" data-interactiontype="multimedia_click" data-multimedia-type="Interactive Graphic" data-multimedia-contentid="10.1056/NEJMdo005827">
                
                    <img src="/cms/asset/c298baeb-9c42-4a95-b07f-ea7daeb63a28/media/NEJMdo005827_300x200.jpg" class="ng-do-interactive_img">
                
                <span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#tooltip-interactive"></use></svg></span>
            </a>
        </div>
        <div class="ng-do-interactive_item-right">
            <a href="#" class="ng-do-interactive_link" data-interactiontype="multimedia_click" data-multimedia-type="Interactive Graphic" data-multimedia-contentid="10.1056/NEJMdo005827">Table 3. Candidate Stillbirth Genes.</a>
        </div>
    </div>
</div>

</div><div role="paragraph">Stillbirth (defined as fetal death in utero at â¥20 weeks of gestation) accounts for 60% of all perinatal deaths and is unexplained in 25 to 60% of cases.<sup><a href="#core-r1" role="doc-biblioref" data-xml-rid="r1 r2 r3" id="body-ref-r3-1" href-manipulated="true">1-3</a></sup> Data are limited regarding the contribution of known <a href="https://illustrated-glossary.nejm.org/index.html?term=mendelian_disease" target="_blank">mendelian diseases</a> to stillbirth. In many clinical disciplines, it is now standard practice to perform exome sequencing to test for pathogenic single-nucleotide variants and small insertions and deletions to determine the cause of disease.<sup><a href="#core-r4" role="doc-biblioref" data-xml-rid="r4 r5 r6 r7 r8 r9 r10 r11 r12" id="body-ref-r12-1" href-manipulated="true">4-12</a></sup> Clinical exome sequencing has been particularly useful in diagnosing otherwise unexplained childhood disorders (in 20 to 30% of such cases) and fetal structural anomalies (in 10 to 20%).<sup><a href="#core-r11" role="doc-biblioref" data-xml-rid="r11 r13 r14" id="body-ref-r14-1" href-manipulated="true">11,13,14</a></sup></div><div role="paragraph">Given the incidence of stillbirth and the presumption of a strong genetic contribution, it is unfortunate that clinical exome sequencing has not been applied more rigorously in this context. Previous studies have been small and have concentrated on predetermined causes, included data regarding early miscarriage, or had a high prevalence of recurrent cases or cases with structural anomalies.<sup><a href="#core-r15" role="doc-biblioref" data-xml-rid="r15 r16" id="body-ref-r16-1" href-manipulated="true">15,16</a></sup> The identification of monogenic disorders that are responsible for stillbirth may facilitate closure and bereavement for families, inform recurrence risk and management in subsequent pregnancies, allow for the avoidance of ineffective and potentially harmful therapies and interventions, and identify novel targets for risk stratification and therapy. Thus, we evaluated the diagnostic utility of clinical exome sequencing in a cohort of 246 stillbirth cases in a multicenter caseâcontrol study involving a geographically and demographically diverse population in the United States.</div><section id="sec-1" data-type="methods"><h2>Methods</h2><section id="sec-1-1"><h3>Study Design and Participants</h3><div role="paragraph">The Stillbirth Collaborative Research Network (SCRN) of the Eunice Kennedy Shriver National Institute of Child Health and Human Development conducted the study from March 2006 through September 2008.<sup><a href="#core-r2" role="doc-biblioref" data-xml-rid="r2" id="body-ref-r2-2" href-manipulated="true" aria-label="Reference 2">2</a></sup> The initial cytogenetic studies of this cohort were completed in 2012, and remaining tissue samples became available from the SCRN for exome sequencing in 2017; the complete data set became available in May 2018. The study was approved by the institutional review board of the data coordinating center at each clinical site. All the participants provided written informed consent.</div><div role="paragraph">The study included fetuses with a gestation of 20 weeks or more; in cases in which the dating criteria were questionable, that window was expanded to include fetuses with a gestation of 18 weeks 0 days to 19 weeks 6 days. All case reviews included a standardized maternal interview and a detailed medical-chart abstraction.</div><div role="paragraph">The study series, which has been described previously,<sup><a href="#core-r2" role="doc-biblioref" data-xml-rid="r2 r3" id="body-ref-r3-2" href-manipulated="true">2,3</a></sup> consisted of 953 potential participants; of these women, 663 enrolled, and 639 consented to participate in the genetic study. Of these women, 560 provided consent for fetuses to undergo partial or complete postmortem examination (Fig. S1 in the <a href="#ap1">Supplementary Appendix</a>, available with the full text of this article at NEJM.org).</div><div role="paragraph">In 512 cases for which data were available, the investigators used the Initial Causes of Fetal Death (INCODE) algorithmic classification tool<sup><a href="#core-r17" role="doc-biblioref" data-xml-rid="r17" id="body-ref-r17" href-manipulated="true" aria-label="Reference 17">17</a></sup> to determine the probable cause of death in 312 fetuses (60.9%; 95% confidence interval [CI], 56.5 to 65.2) and the possible or probable cause of death in 390 fetuses (76.2%; 95% CI, 72.2 to 79.8).<sup><a href="#core-r2" role="doc-biblioref" data-xml-rid="r2" id="body-ref-r2-4" href-manipulated="true" aria-label="Reference 2">2</a></sup> The most common causes were obstetrical conditions (e.g., preterm labor, placental abruption, cervical insufficiency, and preterm premature rupture of membranes) in 150 (29.3%; 95% CI, 25.4 to 33.5), placental abnormalities in 121 (23.6%; 95% CI, 20.1 to 27.6), fetal structural abnormalities in 70 (13.7%; 95% CI, 10.9 to 17.0), infection in 66 (12.9%; 95% CI, 10.2 to 16.2), umbilical cord abnormalities in 53 (10.4%; 95% CI, 7.9 to 13.4), hypertensive disorders in 47 (9.2%; 95% CI, 6.9 to 12.1), and other maternal medical conditions in 40 (7.8%; 95% CI, 5.7 to 10.6), with more than one cause identified in some cases.<sup><a href="#core-r2" role="doc-biblioref" data-xml-rid="r2" id="body-ref-r2-5" href-manipulated="true" aria-label="Reference 2">2</a></sup></div></section><section id="sec-1-2"><h3>DNA Preparation and Sequencing</h3><div role="paragraph">DNA was extracted with the use of established methods (Puregene, Qiagen Systems). (Details are provided in the Methods section in the <a href="#ap1">Supplementary Appendix</a>.) Exome sequencing of all samples was performed at the Institute for Genomic Medicine at Columbia University, which also provided control samples that had been obtained from healthy relatives of probands of mixed ancestry in a collection of studies. Exomes were captured and sequenced according to standard protocols and processed with the use of an in-house bioinformatics pipeline.<sup><a href="#core-r18" role="doc-biblioref" data-xml-rid="r18" id="body-ref-r18" href-manipulated="true" aria-label="Reference 18">18</a></sup> The analysis was limited to 18,653 protein-coding genes that had a consensus coding sequence (release 20), which included two-base intronic extensions to accommodate canonical splice variants. To limit confounding caused by differential coverage, we used a previously described site-based pruning strategy.<sup><a href="#core-r19" role="doc-biblioref" data-xml-rid="r19" id="body-ref-r19-1" href-manipulated="true" aria-label="Reference 19">19</a></sup> Sites of consensus coding sequences were excluded from the analysis if the absolute difference in the percentage of cases as compared with controls with adequate (10Ã) coverage of the site differed by more than 10 percentage points.<sup><a href="#core-r19" role="doc-biblioref" data-xml-rid="r19" id="body-ref-r19-2" href-manipulated="true" aria-label="Reference 19">19</a></sup> This process resulted in the pruning of 7% of bases of consensus coding sequence. All testing of caseâcontrol genes to determine the presence of enrichment of genetic variants in cases was performed on the pruned consensus coding sequence.</div></section><section id="sec-1-3"><h3>Molecular Diagnostic Evaluation</h3><div role="paragraph">We screened 246 stillbirth cases for molecular diagnoses using the guidelines of the American College of Medical Genetics and Genomics (ACMG).<sup><a href="#core-r20" role="doc-biblioref" data-xml-rid="r20" id="body-ref-r20" href-manipulated="true" aria-label="Reference 20">20</a></sup> We prioritized variants with characteristics designed to enrich for pathogenicity in mendelian disease genes from the Online Mendelian Inheritance in Man (OMIM) database (Fig. S2).<sup><a href="#core-r21" role="doc-biblioref" data-xml-rid="r21" id="body-ref-r21" href-manipulated="true" aria-label="Reference 21">21</a></sup> Given the lack of data regarding parental genotypes, we considered compound heterozygous variants (<a href="https://illustrated-glossary.nejm.org/index.html?term=minor-allele_frequency" target="_blank">allele frequency</a>, &lt;1%) as possible causes of stillbirth only if the variants were confirmed through read-based phasing in the Integrative Genomics Viewer.<sup><a href="#core-r22" role="doc-biblioref" data-xml-rid="r22" id="body-ref-r22" href-manipulated="true" aria-label="Reference 22">22</a></sup></div><div role="paragraph">We identified two sets of molecular diagnoses in 221 genes that have been previously described in stillbirth (Table S1) and in genes that have not been previously described but that are strong candidates for phenotype expansion. Laboratory-based molecular diagnoses do not always match precisely with the expected phenotype and may indicate phenotypic expansion, in which the spectrum of phenotypes that are known to be caused by pathogenic variants in a specific gene is broadened. In this study, phenotype expansions were determined through case-based literature searches and considered for genes associated with infant and adult disorders with a plausible lethal disease mechanism. Molecular diagnoses were classified as either âpathogenicâ or âlikely pathogenicâ according to the ACMG criteria or âsuggestiveâ by a multidisciplinary clinical and genetics team. (Details regarding this classification are provided in the Methods section in the <a href="#ap1">Supplementary Appendix</a>.)</div></section><section id="sec-1-4"><h3>Statistical Analysis</h3><div role="paragraph">We performed a standard gene-based collapsing analysis in which variants in a specific gene in the cases were compared with variants in the same gene in the controls. A two-tailed Fisherâs exact test was performed across 18,653 protein-coding genes with a consensus coding sequence for three models (Fig. S5). (Such genes are consistently annotated and are of high quality in human and mouse genomes.) We implemented standard procedures to reduce bias caused by relatedness and population stratification (Figs. S3 and S4).<sup><a href="#core-r19" role="doc-biblioref" data-xml-rid="r19 r23 r24" id="body-ref-r24-1" href-manipulated="true">19,23,24</a></sup></div><div role="paragraph">We also performed the same collapsing analysis to compare variants in the cases with those in controls across genes in specific categories of â<a href="https://illustrated-glossary.nejm.org/index.html?term=genic_intolerance" target="_blank">intolerance</a> to variationâ (<a href="#f1">Figure 1</a>). (Genes that harbor few functional variants in healthy persons are known to be more likely to cause disease than genes that carry many functional variants. A gene is said to be âintolerantâ if it has relatively less functional genetic variation in the general population than is expected. Expected values can be calculated differently on the basis of the size of the gene and frequency of mutation, the amount of putatively neutral variation, or the overall number of observed variants.) Specifically, we sought to evaluate the contribution of genes intolerant to <a href="https://illustrated-glossary.nejm.org/index.html?term=loss-of-function_variant" target="_blank">loss-of-function variants</a> by testing whether stillborn fetuses were more likely than unaffected persons to carry loss-of-function variants in such genes. Genic intolerance to variation was stratified on the basis of the loss-of-function observed-to-expected upper boundary fraction (LOEUF) value (i.e., the upper boundary of a Poisson-derived confidence interval of the observed-to-expected ratio) based on the Genome Aggregation Database, version 2.1.1.<sup><a href="#core-r29" role="doc-biblioref" data-xml-rid="r29" id="body-ref-r29-1" href-manipulated="true" aria-label="Reference 29">29</a></sup> We used Loss-of-Function Transcript Effect Estimator (LOFTEE) annotations,<sup><a href="#core-r29" role="doc-biblioref" data-xml-rid="r29" id="body-ref-r29-2" href-manipulated="true" aria-label="Reference 29">29</a></sup> along with low-complexity regions, common <a href="https://illustrated-glossary.nejm.org/index.html?term=structural_variant" target="_blank">structural variants</a>, and segmental duplications,<sup><a href="#core-r35" role="doc-biblioref" data-xml-rid="r35" id="body-ref-r35" href-manipulated="true" aria-label="Reference 35">35</a></sup> to filter for âhigh-confidenceâ loss-of-function variants in cases and controls. We then tested for a case or control enrichment of these loss-of-function variants at each LOEUF threshold in the data set. We used a logistic-regression model to test for an association between the caseâcontrol status and the presence or absence of a loss-of-function variant in each gene grouping. The logistic model accounted for background genetic variation in the gene grouping by controlling for presumed neutral ultra-rare synonymous variants.<sup><a href="#core-r19" role="doc-biblioref" data-xml-rid="r19" id="body-ref-r19-4" href-manipulated="true" aria-label="Reference 19">19</a></sup> We then assessed the significance of the observed minimum P value across all thresholds by means of permutation (number of permutations, 10,000). (Details are provided in the Methods section in the <a href="#ap1">Supplementary Appendix</a>.)</div><div class="figure-wrap" data-presentation="diminished-in-flow"><header><div class="label">Figure 1</div><nav><a href="#f1" title="Open in viewer" class="open-in-viewer" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f1.jpg"><i aria-hidden="true" class="icon-expand"></i></a></nav></header><figure id="f1" class="graphic"><a aria-label="Open figure in viewer" class="open-in-viewer" data-index="0" href="#f1" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f1.jpg"><img src="/cms/10.1056/NEJMoa1908753/asset/fc1d9fab-8597-4cb8-b3b1-b9c2a5e17ea8/assets/images/large/nejmoa1908753_f1.jpg" height="2347" width="2640" aria-labelledby="f1" loading="lazy"></a><figcaption><div class="caption">Intolerance to Genetic Variation.</div><div class="notes"><div role="doc-footnote">Regions of the human genome that are under the strongest natural selection are the most likely to cause disease when they are variant. Until 2013, the primary approach that was used to identify such regions relied on the genetic similarities of different species.<sup><a href="#core-r25" role="doc-biblioref" data-xml-rid="r25" id="body-ref-r25" href-manipulated="true" aria-label="Reference 25">25</a></sup> Genomic regions under selection show fewer DNA changes (e.g., nucleotide substitutions, deletions, or insertions) across species and are likely to be functionally important (Panel A). Pathogenic variants that cause human diseases have long been shown to fall preferentially within these âconstrainedâ regions. Although this approach is useful, it cannot identify genomic regions of particular importance in humans, as might happen because of the evolution of a novel function. In 2013, a new framework was developed to address this limitation: variation, solely within the human population, was used to identify genes with less functional variation than expected according to genomewide averages. Genes with a depletion of human variation are termed âintolerantâ<sup><a href="#core-r26" role="doc-biblioref" data-xml-rid="r26" id="body-ref-r26" href-manipulated="true" aria-label="Reference 26">26</a></sup> and reflect parts of the genome under strong selection specifically in humans. Since the introduction of intolerance scoring, there have been a number of important elaborations focused on regions of genes, specific types of variants, and regulatory regions.<sup><a href="#core-r27" role="doc-biblioref" data-xml-rid="r27 r28 r29 r30" id="body-ref-r30" href-manipulated="true">27-30</a></sup> Intolerance scores have now been shown to provide independent information about where in the human genome pathogenic variants are found (Panel B).<sup><a href="#core-r24" role="doc-biblioref" data-xml-rid="r24 r29 r31 r32 r33 r34" id="body-ref-r34" href-manipulated="true">24,29,31â34</a></sup> For example, the gene encoding olfactory receptor 51E1 (<i>OR51E1</i>) is tolerant to variation and does not cause disease, whereas <i>MYBPC3</i> and <i>HNF1B</i> are intolerant and are known to cause cardiac and kidney disease, respectively. <i>CASZ1</i> is highly intolerant to variation in the healthy population, but no pathogenic variants have been reported in the literature regarding postnatal disease, which indicates that it may result in lethality when variant in utero.</div></div></figcaption></figure></div><div role="paragraph">We then compared enrichment of loss-of-function variation in the stillbirth cohort with that in two additional clinical cohorts: live-born infants with fetal structural anomalies and a large cohort of patients with postnatal diseases. The patients with postnatal disease were ascertained for a mixture of phenotypes with variable ages of onset and severity. All additional cases were sequenced and processed at the Institute for Genomic Medicine. All the sequencing data underwent the same quality-control filtering and pruning procedures to ensure parity between cases and controls. We then partitioned a loss-of-function risk signal in intolerant genes according to OMIM disease-association status in a binary fashion according to whether the genes had previously been implicated in disease.</div></section></section><section id="sec-2" data-type="results"><h2>Results</h2><section id="sec-2-1"><h3>Study Population</h3><div role="paragraph">Among the 560 women who provided consent to participate in the study and for partial or complete postmortem fetal examination, DNA was available for exome sequencing in 392 stillbirths. Of these samples, sequencing provided data in 337 cases, and the quality was adequate<sup><a href="#core-r23" role="doc-biblioref" data-xml-rid="r23 r33" id="body-ref-r33-2" href-manipulated="true">23,33</a></sup> in 296 cases (Fig. S1). A total of 50 cases were excluded from downstream analyses because of a well-defined cause of stillbirth (i.e., multiple gestation, infection, maternal hypertension, maternal medical complications, or previously identified pathological karyotype or microarray).<sup><a href="#core-r3" role="doc-biblioref" data-xml-rid="r3 r4" id="body-ref-r4-2" href-manipulated="true">3,4</a></sup> Among the remaining 246 cases, we included those with probable placental disease, umbilical cord abnormalities, obstetrical complications, or fetal structural abnormalities because the genetic underpinnings of such cases are uncertain. In the majority of cases, both ultrasonography and autopsy (full or limited) were performed (Table S2). Of the 246 women in this study, 105 (42.7%) were non-Hispanic and of European ancestry. The percentage of women in this ancestry group was larger than the percentage among those who enrolled but were not included in the final analysis (129 of 417 [30.9%]) (Table S3). There was no significant difference in the reported characteristics between the 639 women who consented for genetic evaluation and those who were included in this study (Table S4).</div><div role="paragraph">The sources of the control samples that were used in studies at the Institute for Genomic Medicine are provided in Table S5. The average coverage rate of pruned bases of consensus coding sequence (at 10Ã) was 98.8% for cases and 98.6% for controls. The average coverage rate of pruned sites (at 30Ã) was 95.6% for cases and 95.6% for controls.</div></section><section id="sec-2-2"><h3>Diagnostic Yield</h3><div role="paragraph">We identified molecular diagnoses in 9 of 246 stillborn cases (3.7%) across seven genes that have previously been described in stillbirth (<a href="#i1">interactive Table 1</a>, available at NEJM.org). We further identified 6 cases (2.4%) with molecular diagnoses in six disease genes that are strong candidates for phenotype expansion (<a href="#i2">interactive Table 2</a>). We report a cumulative diagnostic yield of 6.1%, with 15 of 246 stillborn cases receiving a molecular diagnosis (according to ACMG criteria) in a known disease gene. An additional 6 cases (2.4%) had a âsuggestiveâ genotype in either a known stillbirth gene or a gene candidate for phenotype expansion (Table S6). With the inclusion of suggestive variants in the diagnostic yield, 21 of 246 of cases (8.5%) received a probable molecular diagnosis.</div><div role="paragraph">Of the 15 cases with a secure molecular diagnosis, 6 (40.0%) had a multisystem developmental disorder and 5 (33.3%) had an isolated cardiac disorder. The disease genes that were identified were enriched for several cardiac-related gene-ontology biologic processes (Table S7). The remaining cases had a disorder primarily affecting the kidney (in 2 [13.3%]) and either the brain or bone (in 1 each [6.7%]).</div><div role="paragraph">The 13 cases with a structural anomaly were more likely to receive a molecular diagnosis than cases without a structural anomaly (odds ratio, 8.80; 95% CI, 1.7 to 38.4). However, we found no significant difference in the frequency of diagnosis between cases with a probable INCODE cause of death and cases with an unexplained cause (odds ratio, 1.38; 95% CI, 0.39 to 4.54).</div></section><section id="sec-2-3"><h3>Case-Level Diagnostic Findings</h3><div role="paragraph">A recurrent molecular diagnosis occurred in <i>PTPN11</i>, which is associated with Noonan syndrome. We identified three previously reported ultra-rare <a href="https://illustrated-glossary.nejm.org/index.html?term=missense_mutation" target="_blank">missense variants</a> in the region of <i>PTPN11</i> encoding the interacting surfaces of the N-terminal src-homology 2 (N-SH2) and protein tyrosine phosphatase (PTP) domains. Most pathogenic missense variants that cause Noonan syndrome are known to cluster in these regions.<sup><a href="#core-r36" role="doc-biblioref" data-xml-rid="r36" id="body-ref-r36" href-manipulated="true" aria-label="Reference 36">36</a></sup> One of the identified missense variants in <i>PTPN11</i>, which predicted the substitution of alanine with threonine at position 72 (Ala72Thr), is a hot spot for isolated juvenile myelomonocytic leukemia (JMML). With few exceptions, germline variants causing Noonan syndrome do not occur as causative <a href="https://illustrated-glossary.nejm.org/index.html?term=somatic_variant" target="_blank">somatic variants</a> in JMML.<sup><a href="#core-r37" role="doc-biblioref" data-xml-rid="r37" id="body-ref-r37" href-manipulated="true" aria-label="Reference 37">37</a></sup> Somatic variants in <i>PTPN11</i> that cause JMML may have stronger <a href="https://illustrated-glossary.nejm.org/index.html?term=gain-of-function_variant" target="_blank">gain-of-function</a> activity than those associated with Noonan syndrome and have been speculated to result in embryonic death if the variants are germline.</div><div role="paragraph">We found some cases with loss-of-function variants in genes in which missense alleles are known to cause postnatal disease in infants. We identified an ultra-rare <a href="https://illustrated-glossary.nejm.org/index.html?term=frameshift_mutation" target="_blank">frameshift</a> variant in <i>SMC3</i>, a gene in which missense variants have been reported to cause a mild form of Cornelia de Lange syndrome,<sup><a href="#core-r31" role="doc-biblioref" data-xml-rid="r31 r38" id="body-ref-r38" href-manipulated="true">31,38</a></sup> in a stillbirth case with severe fetal growth restriction (birth weight, 0.06 percentile). Similarly, we found a novel frameshift variant in <i>FBN2</i>, in which missense variants are known to cause congenital contractural arachnodactyly.<sup><a href="#core-r39" role="doc-biblioref" data-xml-rid="r39" id="body-ref-r39" href-manipulated="true" aria-label="Reference 39">39</a></sup> Given the strong selection against loss-of-function variants in these genes and the lack of loss-of-function variation in the postnatal disease population, we suspect that loss-of-function variants in these genes caused the stillbirth.</div></section><section id="sec-2-4"><h3>Population-Based CaseâControl Analyses</h3><div role="paragraph">No individual gene reached study-wide significance in any of the three gene-based collapsing models. Under the model of known pathogenic variants (i.e., those that have previously been implicated in causing disease<sup><a href="#core-r40" role="doc-biblioref" data-xml-rid="r40 r41" id="body-ref-r41" href-manipulated="true">40,41</a></sup>), <i>PTPN11</i> was the top-ranked gene, and <i>HNF1B</i> and <i>RYR2</i> were among the genes in the top 20 (Table S8). Owing to the genetic heterogeneity of the stillbirth phenotype, larger sample sizes will probably be needed to determine significant associations between variation in individual genes and death in utero.</div><div role="paragraph">Separately, we explored enrichment of loss-of-function variation at every possible LOEUF threshold for intolerance in our data set and assessed the significance of the observed minimum P value by permutation (number of permutations, 10,000). The 8081 genes harboring a loss-of-function variant in 241 case samples and 7239 control samples reflected 1825 unique LOEUF divisions among all the genes that were sequenced. The minimum observed P value was at position 61 of 10,000 permutations, which reflects a significant enrichment of loss-of-function variants at a threshold for high intolerance (LOEUF, â¤0.239) (<a href="#f2">Figure 2</a>). In this gene grouping, there was at least one loss-of-function variant in 35 of 241 cases (14.5%) and in 531 of 7239 controls (7.3%) (Table S9). In accounting for the control carrier frequency, we estimated that loss-of-function variants in intolerant genes contributed to the risk of stillbirth in approximately 7.2% of cases. We found no significant difference in the burden of loss-of-function variants between cases with a probable INCODE cause of death and those with an unexplained cause in this gene grouping (Table S10).</div><div class="figure-wrap" data-presentation="diminished-in-flow"><header><div class="label">Figure 2</div><nav><a href="#f2" title="Open in viewer" class="open-in-viewer" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f2.jpg"><i aria-hidden="true" class="icon-expand"></i></a></nav></header><figure id="f2" class="graphic"><a aria-label="Open figure in viewer" class="open-in-viewer" data-index="0" href="#f2" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f2.jpg"><img src="/cms/10.1056/NEJMoa1908753/asset/df562d50-26f1-4f15-9edf-50bc1559b2c3/assets/images/large/nejmoa1908753_f2.jpg" height="1866" width="1656" aria-labelledby="f2" loading="lazy"></a><figcaption><div class="caption">Enrichment of Loss-of-Function Variants in Stillbirth Cases, as Compared with Controls.</div><div class="notes"><div role="doc-footnote">To explore which genetic variants may be associated with stillbirth, we assigned an LOEUF (loss-of-function observed-to-expected upper boundary fraction) value to all 8081 genes with a loss-of-function variant in 241 case samples and 7239 control samples. The process resulted in 1825 unique LOEUF values. The LOEUF value is the upper boundary of a Poisson-derived confidence interval of the observed-to-expected ratio. A low LOEUF score indicates a depletion of loss-of-function variation (also referred to as âintoleranceâ to loss-of-function variation). The enrichment of loss-of-function variants was then assessed by comparing the count of variants in cases and controls in each of these 1825-gene groupings. Enrichment was evaluated with the use of a logistic-regression model that evaluated caseâcontrol status regarding the presence or absence of a loss-of-function variant in each gene grouping. Enrichment of presumed neutral synonymous variants at each intolerance threshold was used as a covariate to control for differences between case and control genomes that are unrelated to disease risk. The negative log of the enrichment statistic (unadjusted P value) at each LOEUF threshold is shown on the y axis. The x axis reflects the LOEUF threshold for each gene grouping. The minimum observed P value, which indicates the most significant enrichment across all LOEUF thresholds, occurs at a LOEUF threshold of 0.239 or less. The significance of the observed minimum P value was then assessed by permutation (number of permutations, 10,000).</div></div></figcaption></figure></div><div role="paragraph">We then partitioned the loss-of-function risk signal on the basis of an association with OMIM disease. We found that enrichment of loss-of-function variants in intolerant genes in the stillbirth cohort was concentrated in genes without an existing OMIM disease association (odds ratio, 2.22; 95% CI, 1.41 to 3.34) (<a href="#f3">Figure 3</a>). This enrichment of loss-of-function variants in genes that are not currently associated with any known diseases (non-OMIM genes) is similar to enrichment in genes already known to cause a human disease before or after birth (OMIM genes) (odds ratio, 2.02; 95% CI, 1.05 to 3.52). Among the intolerant genes (LOEUF, â¤0.239), at least one loss-of-function variant in a non-OMIM gene was found in 25 of 241 cases (10.4%) and in 358 of 7239 controls (4.9%) (<a href="#i3">interactive Table 3</a>). Accordingly, we estimate that 5.5% of stillbirth cases are potentially explained by loss-of-function variants in these candidate novel disease genes. Of these candidate genes, 44% have an orthologue in the mouse that causes a lethal phenotype when the gene is knocked out, according to the Mouse Genome Informatics database. The corresponding percentage for genes carrying a loss-of-function mutation in controls was 25.5%. The candidate disease genes were enriched for several biologic processes listed in the Gene Ontology database (Table S11).</div><div class="figure-wrap" data-presentation="diminished-in-flow"><header><div class="label">Figure 3</div><nav><a href="#f3" title="Open in viewer" class="open-in-viewer" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f3.jpg"><i aria-hidden="true" class="icon-expand"></i></a></nav></header><figure id="f3" class="graphic"><a aria-label="Open figure in viewer" class="open-in-viewer" data-index="0" href="#f3" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f3.jpg"><img src="/cms/10.1056/NEJMoa1908753/asset/db99f9fb-56d6-444f-9808-967ca1cae228/assets/images/large/nejmoa1908753_f3.jpg" height="1226" width="2640" aria-labelledby="f3" loading="lazy"></a><figcaption><div class="caption">Loss-of-Function Signal in Intolerant Genes in Three Clinical Cohorts.</div><div class="notes"><div role="doc-footnote">In this study, the LOEUF threshold that had the strongest enrichment of loss-of-function variants (â¤0.239) in the 241 cases in the stillbirth cohort was used to make point comparisons with the loss-of-function signal in a group of 251 live-birth cases with fetal anomalies and in 589 cases with postnatal disease. The odds ratio indicates the enrichment of loss-of-function variants in the gene set in cases as compared with a large group of controls. The gene sets are shown in three categories: all the genes below the LOEUF threshold (â¤0.239), those that had an existing disease association in the Online Mendelian Inheritance in Man (OMIM) database, and those that did not have such an association (non-OMIM).</div></div></figcaption></figure></div><div role="paragraph">We also found an enrichment of loss-of-function variants at the same intolerance threshold as in the stillbirth cohort in a postnatal disease cohort of 589 cases (odds ratio, 2.15; 95% CI, 1.68 to 2.72) and in a live-birth fetal-anomaly cohort of 251 cases (odds ratio, 1.50; 95% CI, 0.98 to 2.21). Unlike in the stillbirth cohort, however, loss-of-function signal was driven predominantly by known disease genes in the postnatal disease cohort (odds ratio, 2.80; 95% CI, 1.96 to 3.91) and in the live-birth fetal-anomaly cohort (odds ratio, 2.46; 95% CI, 1.37 to 4.10) (<a href="#f3">Figure 3</a>).</div></section></section><section id="sec-3" data-type="discussion"><h2>Discussion</h2><div role="paragraph">On the basis of exome sequencing analysis of singleton stillbirths, we found that 8.5% of those with a normal chromosomal microarray and without probable maternal or obstetrical causes were probably attributable to mendelian disorders. When we combined these data with the previously published results of a cytogenetic analysis performed in this cohort, in which 6.9% of stillbirths were aneuploid and 2.6% harbored a pathogenic copy number, we determined that a total of 18% of stillbirths had a known genetic cause.<sup><a href="#core-r40" role="doc-biblioref" data-xml-rid="r40" id="body-ref-r40-2" href-manipulated="true" aria-label="Reference 40">40</a></sup></div><div role="paragraph">Our yield from clinical exome sequencing was lower than that for cohorts with severe early-onset diseases.<sup><a href="#core-r11" role="doc-biblioref" data-xml-rid="r11 r13 r14" id="body-ref-r14-2" href-manipulated="true">11,13,14</a></sup> The yield was probably reduced by the absence of genotypic data from the parents, since we could not identify compound heterozygous variants that are potentially identifiable with knowledge of parental genotypes. Of the 21 possibly compound heterozygous variants that we identified but could not confirm through read-based phasing, 4 were in genes known to cause stillbirth (<i>ACE</i> and <i>INVS</i> in 1 case each and <i>NEB</i> in 2 cases). The interpretation of novel missense variants is also compromised by the absence of inheritance information. For this reason, such variants were excluded in our diagnostic pipeline. To evaluate the effect of these exclusions, we estimated the potential added contribution of de novo novel missense variants by pulling all ultra-rare damaging novel missense variants in our cohort. We identified an additional 7 ultra-rare variants in 3 genes known to cause stillbirth (<i>COL2A1</i> in 3 cases and <i>PEIZO1</i> and <i>HNF1B</i> in 2 cases each). With parental genotype information, a fraction of the identified novel missense variants would probably be confirmed as de novo and possibly compound heterozygous variants confirmed in trans (i.e., on separate alleles). We can therefore estimate that the diagnostic yield in known stillbirth genes lies between 3.7 and 8.1% (in 20 of 246 cases). This percentage is consistent with values in other studies that have reported a doubling of the diagnostic yield in family trio sequencing as compared with singleton sequencing and suggests that parental samples should be included in the analysis of stillbirths in clinical care.<sup><a href="#core-r12" role="doc-biblioref" data-xml-rid="r12 r14 r42 r43" id="body-ref-r43-1" href-manipulated="true">12,14,42,43</a></sup></div><div role="paragraph">In assessing how intolerant genes may contribute to stillbirth, we took an approach that makes no prior assumption regarding which human genes may have enrichment of loss-of-function variants. Because loss-of-function variants in the genes with less variation than expected are known to have a high likelihood of causing disease on the basis of research carried out over the past decade,<sup><a href="#core-r29" role="doc-biblioref" data-xml-rid="r29 r43" id="body-ref-r43-2" href-manipulated="true">29,43</a></sup> our unbiased approach was highly conservative. Our approach did not assume in advance that any particular set of genes carried contributing variants. Rather, we used a statistical framework that led to the discovery of the highest concentration of causal variants and corrected appropriately for all the comparisons that were evaluated. We think that this approach constitutes a more robust statistical method for thresholding in enrichment analyses of this kind.</div><div role="paragraph">Previous sequencing studies that have used a similar approach to prioritizing variants in postnatal disease cohorts have reported a preponderance of variants in genes that are associated with established diseases.<sup><a href="#core-r43" role="doc-biblioref" data-xml-rid="r43" id="body-ref-r43-3" href-manipulated="true" aria-label="Reference 43">43</a></sup> In this study, we observed in stillbirths an enrichment of loss-of-function variants in genes that are not currently known to cause human disease, whereas in two postnatal clinical populations (live-born infants with fetal anomalies and patients with undiagnosed postnatal disease) enrichment of loss-of-function variants was concentrated in known disease genes. To our knowledge, stillbirth may be the only condition that has been studied in which a novel risk signal (in non-OMIM genes) is of the same order of magnitude as that in known OMIM disease genes.</div><div role="paragraph">Almost half the candidate genes for an association with stillbirth are essential for life in the mouse, which supports the relevance of these genes to survival of the human fetus. Moreover, we find some consistency between the mouse and human phenotypes. For instance, we found a novel <a href="https://illustrated-glossary.nejm.org/index.html?term=stop-gain_variant" target="_blank">stopâgain variant</a> in <i>CASZ1</i> (encoding zinc-finger transcription factor) in a stillborn infant with hydrops and intrauterine growth restriction. Similarly, in one study, <i>Casz1<sup>â/â</sup></i> mice were found to have edema and morphologic anomalies of the heart before embryonic death at day 17.5.<sup><a href="#core-r44" role="doc-biblioref" data-xml-rid="r44" id="body-ref-r44" href-manipulated="true" aria-label="Reference 44">44</a></sup> A limited autopsy was performed in the stillborn infant, and additional abnormalities in the heart may have gone undetected. We also identified a frameshift variant in <i>NUP98</i>, encoding part of the nuclear pore complex, in a stillborn infant with severe intrauterine growth restriction (5th percentile for birth weight at 25 weeks of gestation). <i>Nup98<sup>â/â</sup></i> mice die early in utero with severe growth delay,<sup><a href="#core-r45" role="doc-biblioref" data-xml-rid="r45" id="body-ref-r45" href-manipulated="true" aria-label="Reference 45">45</a></sup> and deletions in <i>NUP98</i> have been identified in biopsy samples of placental tissue from women with recurrent pregnancy loss.<sup><a href="#core-r46" role="doc-biblioref" data-xml-rid="r46" id="body-ref-r46" href-manipulated="true" aria-label="Reference 46">46</a></sup></div><div role="paragraph">In the majority of stillbirth cases in our study, structural anomalies were not revealed on ultrasonography or autopsy (Table S3). However, ion channelopathies and cardiomyopathies could still be responsible for a proportion of stillbirths without structural anomalies. Previous studies have shown pathogenic mutations in cardiac-associated genes in cases of stillbirth without structural anomalies.<sup><a href="#core-r10" role="doc-biblioref" data-xml-rid="r10 r16 r47 r48" id="body-ref-r48" href-manipulated="true">10,16,47,48</a></sup> In particular, several forms of cardiomyopathy (including hypertrophic, dilated, and arrhythmogenic) may be accompanied by very minor or even no structural abnormalities. The presence of such disorders has been suggested to underlie a portion of sudden infant deaths and stillbirths.<sup><a href="#core-r49" role="doc-biblioref" data-xml-rid="r49" id="body-ref-r49" href-manipulated="true" aria-label="Reference 49">49</a></sup></div><div role="paragraph">Our findings suggest that a portion of stillbirth cases are caused by pathogenic variants in genes known to cause disorders in infants and adults, whereas a similar number of cases are caused by loss-of-function variants in genes that are critical to in utero survival but that are not currently known to be associated with stillbirth or postnatal disease. Exome sequencing analysis of larger, independent stillbirth cohorts would test the validity of our results and almost certainly uncover additional genes that are associated with stillbirth. We anticipate that the identification of specific genes and variants will further improve our understanding of the pathways leading to stillbirth and improve the counseling of affected families. Currently, no sequencing data from a sufficiently large cohort exist to perform this analysis. Our results show that exome sequencing in stillbirth can identify variants of genes that are incompatible with survival in utero.</div></section>



        
        
</div></section><section id="backmatter" data-extent="backmatter" data-location="articleTab_article"><div class="core-container"><section id="backnotes" data-location="notes_article"><h2>Notes</h2><div data-type="disclaimer" role="paragraph">The views expressed in this article are those of the authors and do not necessarily reflect the views of the National Institutes of Health or the Eunice Kennedy Shriver National Institute of Child Health and Development (NICHD).</div><div data-type="published/updated" role="paragraph">This article was published on August 12, 2020, at NEJM.org.</div><div role="paragraph">Supported by the <span class="named-content" data-type="funder">Institute for Genomic Medicine</span> with grants (HD45925, HD45944, HD45952, HD45953, HD45954, and HD45955) from the NICHD.</div><div role="paragraph"><a href="#ap2">Disclosure forms</a> provided by the authors are available with the full text of this article at NEJM.org.</div><div role="paragraph">We thank the patients, physicians, study coordinators, and research nurses who participated in the Stillbirth Collaborative Research Network.</div></section><section id="supplementary-materials" class="core-supplementary-materials"><h2>Supplementary Material</h2><div role="list"><div id="ap1" class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">Supplementary Appendix</span> <span class="core-filename">(nejmoa1908753_appendix.pdf)</span></div><div class="core-link"><ul><li><a href="/doi/suppl/10.1056/NEJMoa1908753/suppl_file/nejmoa1908753_appendix.pdf" download="nejmoa1908753_appendix.pdf" data-interactiontype="multimedia_download" data-multimedia-type="Supplementary Material" data-behavior="trackDownloadEvent" data-multimedia-filename="nejmoa1908753_appendix.pdf" data-doi="10.1056/NEJMoa1908753">Download</a></li><li>3.08 MB</li></ul></div></div><div id="ap2" class="core-supplementary-material" role="listitem"><div class="core-description"><span class="core-label">Disclosure Forms</span> <span class="core-filename">(nejmoa1908753_disclosures.pdf)</span></div><div class="core-link"><ul><li><a href="/doi/suppl/10.1056/NEJMoa1908753/suppl_file/nejmoa1908753_disclosures.pdf" download="nejmoa1908753_disclosures.pdf" data-interactiontype="multimedia_download" data-multimedia-type="Supplementary Material" data-behavior="trackDownloadEvent" data-multimedia-filename="nejmoa1908753_disclosures.pdf" data-doi="10.1056/NEJMoa1908753">Download</a></li><li>405.95 KB</li></ul></div></div></div></section><section id="bibliography" class="core-reference-list" role="doc-bibliography" data-location="references_article"><h2>References</h2><div role="list" data-method="clamp" id="collapsible-text"><div role="listitem" data-has="label"><div class="label">1.</div><div id="r1" class="citations"><div class="citation"><div class="citation-content">Wapner RJ, Lewis D. Genetics and metabolic causes of stillbirth. <em>Semin Perinatol</em> 2002;26:70-74.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r3-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1053/sper.2002.29853" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11876569/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000174004700010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genetics+and+metabolic+causes+of+stillbirth.&amp;publication_year=2002&amp;journal=Semin+Perinatol&amp;pages=70-74&amp;doi=10.1053%2Fsper.2002.29853&amp;pmid=11876569" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">2.</div><div id="r2" class="citations"><div class="citation"><div class="citation-content">Stillbirth Collaborative Research Network Writing Group. Causes of death among stillbirths. <em>JAMA</em> 2011;306:2459-2468.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r2" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2011.1823" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22166605/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000297983300019" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Causes+of+death+among+stillbirths.&amp;publication_year=2011&amp;journal=JAMA&amp;pages=2459-2468&amp;doi=10.1001%2Fjama.2011.1823&amp;pmid=22166605" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r2" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r3-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] and is unexplained in 25 to 60% of cases. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r2-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] from March 2006 through September 2008. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r3-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] which has been described previously, </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r2-4" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>d [...] 390 fetuses (76.2%; 95% CI, 72.2 to 79.8). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r2-5" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>e [...] than one cause identified in some cases. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">3.</div><div id="r3" class="citations"><div class="citation"><div class="citation-content">Reddy UM, Page GP, Saade GR, et al. Karyotype versus microarray testing for genetic abnormalities after stillbirth. <em>N Engl J Med</em> 2012;367:2185-2193.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r3" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_4_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa1908753&amp;key=10.1056%2FNEJMoa1201569&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23215556/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000311890600005" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Karyotype+versus+microarray+testing+for+genetic+abnormalities+after+stillbirth.&amp;publication_year=2012&amp;journal=N+Engl+J+Med&amp;pages=2185-2193&amp;doi=10.1056%2FNEJMoa1201569&amp;pmid=23215556" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r3" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r3-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] and is unexplained in 25 to 60% of cases. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r3-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] which has been described previously, </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r4-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] pathological karyotype or microarray). </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">4.</div><div id="r4" class="citations"><div class="citation"><div class="citation-content">Lee H, Deignan JL, Dorrani N, et al. Clinical exome sequencing for genetic identification of rare Mendelian disorders. <em>JAMA</em> 2014;312:1880-1887.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r4" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2014.14604" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25326637/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000344532600017" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Clinical+exome+sequencing+for+genetic+identification+of+rare+Mendelian+disorders.&amp;publication_year=2014&amp;journal=JAMA&amp;pages=1880-1887&amp;doi=10.1001%2Fjama.2014.14604&amp;pmid=25326637" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r4" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r12-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] to determine the cause of disease. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r4-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] pathological karyotype or microarray). </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">5.</div><div id="r5" class="citations"><div class="citation"><div class="citation-content">Yang Y, Muzny DM, Reid JG, et al. Clinical whole-exome sequencing for the diagnosis of mendelian disorders. <em>N Engl J Med</em> 2013;369:1502-1511.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_6_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa1908753&amp;key=10.1056%2FNEJMoa1306555&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24088041/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000328101700007" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Clinical+whole-exome+sequencing+for+the+diagnosis+of+mendelian+disorders.&amp;publication_year=2013&amp;journal=N+Engl+J+Med&amp;pages=1502-1511&amp;doi=10.1056%2FNEJMoa1306555&amp;pmid=24088041" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">6.</div><div id="r6" class="citations"><div class="citation"><div class="citation-content">Groopman EE, Marasa M, Cameron-Christie S, et al. Diagnostic utility of exome sequencing for kidney disease. <em>N Engl J Med</em> 2019;380:142-151.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_7_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa1908753&amp;key=10.1056%2FNEJMoa1806891&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30586318/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000455310300008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Diagnostic+utility+of+exome+sequencing+for+kidney+disease.&amp;publication_year=2019&amp;journal=N+Engl+J+Med&amp;pages=142-151&amp;doi=10.1056%2FNEJMoa1806891&amp;pmid=30586318" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">7.</div><div id="r7" class="citations"><div class="citation"><div class="citation-content">Splinter K, Adams DR, Bacino CA, et al. Effect of genetic diagnosis on patients with previously undiagnosed disease. <em>N Engl J Med</em> 2018;379:2131-2139.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_8_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa1908753&amp;key=10.1056%2FNEJMoa1714458&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30304647/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000451402500008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Effect+of+genetic+diagnosis+on+patients+with+previously+undiagnosed+disease.&amp;publication_year=2018&amp;journal=N+Engl+J+Med&amp;pages=2131-2139&amp;doi=10.1056%2FNEJMoa1714458&amp;pmid=30304647" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">8.</div><div id="r8" class="citations"><div class="citation"><div class="citation-content">Bagnall RD, Crompton DE, Petrovski S, et al. Exome-based analysis of cardiac arrhythmia, respiratory control, and epilepsy genes in sudden unexpected death in epilepsy. <em>Ann Neurol</em> 2016;79:522-534.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ana.24596" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26704558/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000374006100003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Exome-based+analysis+of+cardiac+arrhythmia%2C+respiratory+control%2C+and+epilepsy+genes+in+sudden+unexpected+death+in+epilepsy.&amp;publication_year=2016&amp;journal=Ann+Neurol&amp;pages=522-534&amp;doi=10.1002%2Fana.24596&amp;pmid=26704558" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">9.</div><div id="r9" class="citations"><div class="citation"><div class="citation-content">Yang Y, Muzny DM, Xia F, et al. Molecular findings among patients referred for clinical whole-exome sequencing. <em>JAMA</em> 2014;312:1870-1879.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2014.14601" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25326635/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000344532600016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Molecular+findings+among+patients+referred+for+clinical+whole-exome+sequencing.&amp;publication_year=2014&amp;journal=JAMA&amp;pages=1870-1879&amp;doi=10.1001%2Fjama.2014.14601&amp;pmid=25326635" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">10.</div><div id="r10" class="citations"><div class="citation"><div class="citation-content">Neubauer J, Lecca MR, Russo G, et al. Post-mortem whole-exome analysis in a large sudden infant death syndrome cohort with a focus on cardiovascular and metabolic genetic diseases. <em>Eur J Hum Genet</em> 2017;25:404-409.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r10" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ejhg.2016.199" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28074886/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000395996200006" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Post-mortem+whole-exome+analysis+in+a+large+sudden+infant+death+syndrome+cohort+with+a+focus+on+cardiovascular+and+metabolic+genetic+diseases.&amp;publication_year=2017&amp;journal=Eur+J+Hum+Genet&amp;pages=404-409&amp;doi=10.1038%2Fejhg.2016.199&amp;pmid=28074886" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r10" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r12-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] to determine the cause of disease. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r48" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] of stillbirth without structural anomalies. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">11.</div><div id="r11" class="citations"><div class="citation"><div class="citation-content">Lord J, McMullan DJ, Eberhardt RY, et al. Prenatal exome sequencing analysis in fetal structural anomalies detected by ultrasonography (PAGE): a cohort study. <em>Lancet</em> 2019;393:747-757.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r11" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0140-6736(18)31940-8" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30712880/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000459945500025" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Prenatal+exome+sequencing+analysis+in+fetal+structural+anomalies+detected+by+ultrasonography+%28PAGE%29%3A+a+cohort+study.&amp;publication_year=2019&amp;journal=Lancet&amp;pages=747-757&amp;doi=10.1016%2FS0140-6736%2818%2931940-8&amp;pmid=30712880" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r11" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r12-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] to determine the cause of disease. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r14-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] fetal structural anomalies (in 10 to 20%). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r14-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] cohorts with severe early-onset diseases. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">12.</div><div id="r12" class="citations"><div class="citation"><div class="citation-content">Meng L, Pammi M, Saronwala A, et al. Use of exome sequencing for infants in intensive care units: ascertainment of severe single-gene disorders and effect on medical management. <em>JAMA Pediatr</em> 2017;171(12):e173438-e173438.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r12" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jamapediatrics.2017.3438" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28973083/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000416971000003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Use+of+exome+sequencing+for+infants+in+intensive+care+units%3A+ascertainment+of+severe+single-gene+disorders+and+effect+on+medical+management.&amp;publication_year=2017&amp;journal=JAMA+Pediatr&amp;pages=e173438-e173438&amp;doi=10.1001%2Fjamapediatrics.2017.3438&amp;pmid=28973083" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r12" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r12-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] to determine the cause of disease. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] analysis of stillbirths in clinical care. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">13.</div><div id="r13" class="citations"><div class="citation"><div class="citation-content">Petrovski S, Aggarwal V, Giordano JL, et al. Whole-exome sequencing in the evaluation of fetal structural anomalies: a prospective cohort study. <em>Lancet</em> 2019;393:758-767.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r13" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0140-6736(18)32042-7" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30712878/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000459945500026" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Whole-exome+sequencing+in+the+evaluation+of+fetal+structural+anomalies%3A+a+prospective+cohort+study.&amp;publication_year=2019&amp;journal=Lancet&amp;pages=758-767&amp;doi=10.1016%2FS0140-6736%2818%2932042-7&amp;pmid=30712878" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r13" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r14-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] fetal structural anomalies (in 10 to 20%). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r14-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] cohorts with severe early-onset diseases. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">14.</div><div id="r14" class="citations"><div class="citation"><div class="citation-content">Wright CF, Fitzgerald TW, Jones WD, et al. Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data. <em>Lancet</em> 2015;385:1305-1314.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r14" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0140-6736(14)61705-0" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25529582/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000352272400036" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genetic+diagnosis+of+developmental+disorders+in+the+DDD+study%3A+a+scalable+analysis+of+genome-wide+research+data.&amp;publication_year=2015&amp;journal=Lancet&amp;pages=1305-1314&amp;doi=10.1016%2FS0140-6736%2814%2961705-0&amp;pmid=25529582" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r14" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r14-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] fetal structural anomalies (in 10 to 20%). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r14-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] cohorts with severe early-onset diseases. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] analysis of stillbirths in clinical care. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">15.</div><div id="r15" class="citations"><div class="citation"><div class="citation-content">Shamseldin HE, Kurdi W, Almusafri F, et al. Molecular autopsy in maternal-fetal medicine. <em>Genet Med</em> 2018;20:420-427.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r16-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2017.111" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28749478/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000429912900007" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Molecular+autopsy+in+maternal-fetal+medicine.&amp;publication_year=2018&amp;journal=Genet+Med&amp;pages=420-427&amp;doi=10.1038%2Fgim.2017.111&amp;pmid=28749478" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">16.</div><div id="r16" class="citations"><div class="citation"><div class="citation-content">Sahlin E, GrÃ©en A, Gustavsson P, et al. Identification of putative pathogenic single nucleotide variants (SNVs) in genes associated with heart disease in 290 cases of stillbirth. <em>PLoS One</em> 2019;14(1):e0210017-e0210017.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r16" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0210017" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30615648/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000455045900034" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+putative+pathogenic+single+nucleotide+variants+%28SNVs%29+in+genes+associated+with+heart+disease+in+290+cases+of+stillbirth.&amp;publication_year=2019&amp;journal=PLoS+One&amp;pages=e0210017-e0210017&amp;doi=10.1371%2Fjournal.pone.0210017&amp;pmid=30615648" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r16" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r16-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] cases or cases with structural anomalies. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r48" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] of stillbirth without structural anomalies. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">17.</div><div id="r17" class="citations"><div class="citation"><div class="citation-content">Dudley DJ, Goldenberg R, Conway D, et al. A new system for determining the causes of stillbirth. <em>Obstet Gynecol</em> 2010;116:254-260.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r17"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1097/AOG.0b013e3181e7d975" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20664383/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000280186300003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+new+system+for+determining+the+causes+of+stillbirth.&amp;publication_year=2010&amp;journal=Obstet+Gynecol&amp;pages=254-260&amp;doi=10.1097%2FAOG.0b013e3181e7d975&amp;pmid=20664383" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">18.</div><div id="r18" class="citations"><div class="citation"><div class="citation-content"><em>Analysis tool for annotated variants â a comprehensive platform for population-scale genomic analyses</em>. 2016 (<a href="https://www.biorxiv.org/content/10.1101/2020.06.08.136507v1.full">https://www.biorxiv.org/content/10.1101/2020.06.08.136507v1.full</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r18"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Analysis+tool+for+annotated+variants+%E2%80%94+a+comprehensive+platform+for+population-scale+genomic+analyses&amp;publication_year=2016" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">19.</div><div id="r19" class="citations"><div class="citation"><div class="citation-content">Epi4K Consortium, Epilepsy Phenome/Genome Project. Ultra-rare genetic variation in common epilepsies: a case-control sequencing study. <em>Lancet Neurol</em> 2017;16:135-143.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r19" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S1474-4422(16)30359-3" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28102150/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000391910800012" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Ultra-rare+genetic+variation+in+common+epilepsies%3A+a+case-control+sequencing+study.&amp;publication_year=2017&amp;journal=Lancet+Neurol&amp;pages=135-143&amp;doi=10.1016%2FS1474-4422%2816%2930359-3&amp;pmid=28102150" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r19" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r19-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] described site-based pruning strategy. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r19-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] differed by more than 10 percentage points. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r24-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] stratification (Figs. S3 and S4). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r19-4" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>d [...] neutral ultra-rare synonymous variants. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">20.</div><div id="r20" class="citations"><div class="citation"><div class="citation-content">Richards S, Aziz N, Bale S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. <em>Genet Med</em> 2015;17:405-424.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r20"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2015.30" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25741868/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000354096900013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Standards+and+guidelines+for+the+interpretation+of+sequence+variants%3A+a+joint+consensus+recommendation+of+the+American+College+of+Medical+Genetics+and+Genomics+and+the+Association+for+Molecular+Pathology.&amp;publication_year=2015&amp;journal=Genet+Med&amp;pages=405-424&amp;doi=10.1038%2Fgim.2015.30&amp;pmid=25741868" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">21.</div><div id="r21" class="citations"><div class="citation"><div class="citation-content">Hamosh A, Scott AF, Amberger J, Valle D, McKusick VA. Online Mendelian Inheritance in Man (OMIM). <em>Hum Mutat</em> 2000;15:57-61.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r21"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1098-1004(200001)15:1%3C57::AID-HUMU12%3E3.0.CO;2-G" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10612823/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000084613900011" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Online+Mendelian+Inheritance+in+Man+%28OMIM%29.&amp;publication_year=2000&amp;journal=Hum+Mutat&amp;pages=57-61&amp;doi=10.1002%2F%28SICI%291098-1004%28200001%2915%3A1%253C57%3A%3AAID-HUMU12%253E3.0.CO%3B2-G&amp;pmid=10612823" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">22.</div><div id="r22" class="citations"><div class="citation"><div class="citation-content">ThorvaldsdÃ³ttir H, Robinson JT, Mesirov JP. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. <em>Brief Bioinform</em> 2013;14:178-192.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r22"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bib/bbs017" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22517427/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000316694700006" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Integrative+Genomics+Viewer+%28IGV%29%3A+high-performance+genomics+data+visualization+and+exploration.&amp;publication_year=2013&amp;journal=Brief+Bioinform&amp;pages=178-192&amp;doi=10.1093%2Fbib%2Fbbs017&amp;pmid=22517427" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">23.</div><div id="r23" class="citations"><div class="citation"><div class="citation-content">Petrovski S, Todd JL, Durheim MT, et al. An exome sequencing study to assess the role of rare genetic variation in pulmonary fibrosis. <em>Am J Respir Crit Care Med</em> 2017;196:82-93.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r23" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1164/rccm.201610-2088OC" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28099038/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000404473300016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=An+exome+sequencing+study+to+assess+the+role+of+rare+genetic+variation+in+pulmonary+fibrosis.&amp;publication_year=2017&amp;journal=Am+J+Respir+Crit+Care+Med&amp;pages=82-93&amp;doi=10.1164%2Frccm.201610-2088OC&amp;pmid=28099038" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r23" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r24-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] stratification (Figs. S3 and S4). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r33-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] in 337 cases, and the quality was adequate </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">24.</div><div id="r24" class="citations"><div class="citation"><div class="citation-content">Zhu X, Padmanabhan R, Copeland B, et al. A case-control collapsing analysis identifies epilepsy genes implicated in trio sequencing studies focused on de novo mutations. <em>PLoS Genet</em> 2017;13(11):e1007104-e1007104.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r24" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1007104" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29186148/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000416836900026" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+case-control+collapsing+analysis+identifies+epilepsy+genes+implicated+in+trio+sequencing+studies+focused+on+de+novo+mutations.&amp;publication_year=2017&amp;journal=PLoS+Genet&amp;pages=e1007104-e1007104&amp;doi=10.1371%2Fjournal.pgen.1007104&amp;pmid=29186148" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r24" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r24-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] stratification (Figs. S3 and S4). </span></a></li><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r24-2"><i aria-hidden="true" class="icon-return"></i><span>b [...] pathogenic variants are found (Panel B). </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">25.</div><div id="r25" class="citations"><div class="citation"><div class="citation-content">Davydov EV, Goode DL, Sirota M, Cooper GM, Sidow A, Batzoglou S. Identifying a high fraction of the human genome to be under selective constraint using GERP++. <em>PLoS Comput Biol</em> 2010;6(12):e1001025-e1001025.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r25"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1001025" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21152010/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000285574600013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Identifying+a+high+fraction+of+the+human+genome+to+be+under+selective+constraint+using+GERP%2B%2B.&amp;publication_year=2010&amp;journal=PLoS+Comput+Biol&amp;pages=e1001025-e1001025&amp;doi=10.1371%2Fjournal.pcbi.1001025&amp;pmid=21152010" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">26.</div><div id="r26" class="citations"><div class="citation"><div class="citation-content">Petrovski S, Wang Q, Heinzen EL, Allen AS, Goldstein DB. Genic intolerance to functional variation and the interpretation of personal genomes. <em>PLoS Genet</em> 2013;9(8):e1003709-e1003709.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r26"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003709" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23990802/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000323830300059" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genic+intolerance+to+functional+variation+and+the+interpretation+of+personal+genomes.&amp;publication_year=2013&amp;journal=PLoS+Genet&amp;pages=e1003709-e1003709&amp;doi=10.1371%2Fjournal.pgen.1003709&amp;pmid=23990802" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">27.</div><div id="r27" class="citations"><div class="citation"><div class="citation-content">Samocha KE, Robinson EB, Sanders SJ, et al. A framework for the interpretation of de novo mutation in human disease. <em>Nat Genet</em> 2014;46:944-950.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r30"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3050" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25086666/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000341579400007" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+framework+for+the+interpretation+of+de+novo+mutation+in+human+disease.&amp;publication_year=2014&amp;journal=Nat+Genet&amp;pages=944-950&amp;doi=10.1038%2Fng.3050&amp;pmid=25086666" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">28.</div><div id="r28" class="citations"><div class="citation"><div class="citation-content">Fuller ZL, Berg JJ, Mostafavi H, Sella G, Przeworski M. Measuring intolerance to mutation in human genetics. <em>Nat Genet</em> 2019;51:772-776.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r30"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0383-1" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30962618/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000466842000004" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Measuring+intolerance+to+mutation+in+human+genetics.&amp;publication_year=2019&amp;journal=Nat+Genet&amp;pages=772-776&amp;doi=10.1038%2Fs41588-019-0383-1&amp;pmid=30962618" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">29.</div><div id="r29" class="citations"><div class="citation"><div class="citation-content">Karczewski KJ, Francioli LC, Tiao G, et al. Variation across 141,456 human exomes and genomes reveals the spectrum of loss-of-function intolerance across human protein-coding genes. January 30, 2019 (<a href="https://www.biorxiv.org/content/10.1101/531210v2">https://www.biorxiv.org/content/10.1101/531210v2</a>). preprint.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r29" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Variation+across+141%2C456+human+exomes+and+genomes+reveals+the+spectrum+of+loss-of-function+intolerance+across+human+protein-coding+genes&amp;publication_year=2019" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r29" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r29-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] Genome Aggregation Database, version 2.1.1. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r29-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] Effect Estimator (LOFTEE) annotations, </span></a></li><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r29-3"><i aria-hidden="true" class="icon-return"></i><span>c [...] types of variants, and regulatory regions. </span></a></li><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r29-4"><i aria-hidden="true" class="icon-return"></i><span>d [...] pathogenic variants are found (Panel B). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>e [...] research carried out over the past decade, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">30.</div><div id="r30" class="citations"><div class="citation"><div class="citation-content">Lek M, Karczewski KJ, Minikel EV, et al. Analysis of protein-coding genetic variation in 60,706 humans. <em>Nature</em> 2016;536:285-291.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r30"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19057" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000381804900026" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Analysis+of+protein-coding+genetic+variation+in+60%2C706+humans.&amp;publication_year=2016&amp;journal=Nature&amp;pages=285-291&amp;doi=10.1038%2Fnature19057&amp;pmid=27535533" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">31.</div><div id="r31" class="citations"><div class="citation"><div class="citation-content">Eilbeck K, Quinlan A, Yandell M. Settling the score: variant prioritization and Mendelian disease. <em>Nat Rev Genet</em> 2017;18:599-612.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r31" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg.2017.52" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28804138/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000410808600002" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Settling+the+score%3A+variant+prioritization+and+Mendelian+disease.&amp;publication_year=2017&amp;journal=Nat+Rev+Genet&amp;pages=599-612&amp;doi=10.1038%2Fnrg.2017.52&amp;pmid=28804138" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r31" style="display: none;"><ul role="none"><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r31-1"><i aria-hidden="true" class="icon-return"></i><span>a [...] pathogenic variants are found (Panel B). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r38" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] a mild form of Cornelia de Lange syndrome, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">32.</div><div id="r32" class="citations"><div class="citation"><div class="citation-content">Bartha I, di Iulio J, Venter JC, Telenti A. Human gene essentiality. <em>Nat Rev Genet</em> 2018;19:51-62.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg.2017.75" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29082913/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000419846000011" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Human+gene+essentiality.&amp;publication_year=2018&amp;journal=Nat+Rev+Genet&amp;pages=51-62&amp;doi=10.1038%2Fnrg.2017.75&amp;pmid=29082913" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">33.</div><div id="r33" class="citations"><div class="citation"><div class="citation-content">Gelfman S, Dugger S, de Araujo Martins Moreno C, et al. A new approach for rare variation collapsing on functional protein domains implicates specific genic regions in ALS. <em>Genome Res</em> 2019;29:809-818.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r33" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.243592.118" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30940688/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000466371300009" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+new+approach+for+rare+variation+collapsing+on+functional+protein+domains+implicates+specific+genic+regions+in+ALS.&amp;publication_year=2019&amp;journal=Genome+Res&amp;pages=809-818&amp;doi=10.1101%2Fgr.243592.118&amp;pmid=30940688" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r33" style="display: none;"><ul role="none"><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r33-1"><i aria-hidden="true" class="icon-return"></i><span>a [...] pathogenic variants are found (Panel B). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r33-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] in 337 cases, and the quality was adequate </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">34.</div><div id="r34" class="citations"><div class="citation"><div class="citation-content">Kosmicki JA, Samocha KE, Howrigan DP, et al. Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples. <em>Nat Genet</em> 2017;49:504-510.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000397603700013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Refining+the+role+of+de+novo+protein-truncating+variants+in+neurodevelopmental+disorders+by+using+population+reference+samples.&amp;publication_year=2017&amp;journal=Nat+Genet&amp;pages=504-510&amp;doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">35.</div><div id="r35" class="citations"><div class="citation"><div class="citation-content">Li H, Bloom JM, Farjoun Y, et al. A synthetic-diploid benchmark for accurate variant-calling evaluation. <em>Nat Methods</em> 2018;15:595-597.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r35"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0054-7" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30013044/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000440334000018" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+synthetic-diploid+benchmark+for+accurate+variant-calling+evaluation.&amp;publication_year=2018&amp;journal=Nat+Methods&amp;pages=595-597&amp;doi=10.1038%2Fs41592-018-0054-7&amp;pmid=30013044" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">36.</div><div id="r36" class="citations"><div class="citation"><div class="citation-content">Tartaglia M, Kalidas K, Shaw A, et al. PTPN11 mutations in Noonan syndrome: molecular spectrum, genotype-phenotype correlation, and phenotypic heterogeneity. <em>Am J Hum Genet</em> 2002;70:1555-1563.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r36"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1086/340847" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11992261/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000175445700016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=PTPN11+mutations+in+Noonan+syndrome%3A+molecular+spectrum%2C+genotype-phenotype+correlation%2C+and+phenotypic+heterogeneity.&amp;publication_year=2002&amp;journal=Am+J+Hum+Genet&amp;pages=1555-1563&amp;doi=10.1086%2F340847&amp;pmid=11992261" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">37.</div><div id="r37" class="citations"><div class="citation"><div class="citation-content">Mason-Suares H, Toledo D, Gekas J, et al. Juvenile myelomonocytic leukemia-associated variants are associated with neo-natal lethal Noonan syndrome. <em>Eur J Hum Genet</em> 2017;25:509-511.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r37"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ejhg.2016.202" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28098151/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000395996200020" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Juvenile+myelomonocytic+leukemia-associated+variants+are+associated+with+neo-natal+lethal+Noonan+syndrome.&amp;publication_year=2017&amp;journal=Eur+J+Hum+Genet&amp;pages=509-511&amp;doi=10.1038%2Fejhg.2016.202&amp;pmid=28098151" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">38.</div><div id="r38" class="citations"><div class="citation"><div class="citation-content">Gil-RodrÃ­guez MC, Deardorff MA, Ansari M, et al. De novo heterozygous mutations in SMC3 cause a range of Cornelia de Lange syndrome-overlapping phenotypes. <em>Hum Mutat</em> 2015;36:454-462.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r38"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/humu.22761" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25655089/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000352304200010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=De+novo+heterozygous+mutations+in+SMC3+cause+a+range+of+Cornelia+de+Lange+syndrome-overlapping+phenotypes.&amp;publication_year=2015&amp;journal=Hum+Mutat&amp;pages=454-462&amp;doi=10.1002%2Fhumu.22761&amp;pmid=25655089" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">39.</div><div id="r39" class="citations"><div class="citation"><div class="citation-content">Park ES, Putnam EA, Chitayat D, Child A, Milewicz DM. Clustering of FBN2 mutations in patients with congenital contractural arachnodactyly indicates an important role of the domains encoded by exons 24 through 34 during human development. <em>Am J Med Genet</em> 1998;78:350-355.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r39"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1096-8628(19980724)78:4%3C350::AID-AJMG9%3E3.0.CO;2-P" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9714438/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000075199200009" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Clustering+of+FBN2+mutations+in+patients+with+congenital+contractural+arachnodactyly+indicates+an+important+role+of+the+domains+encoded+by+exons+24+through+34+during+human+development.&amp;publication_year=1998&amp;journal=Am+J+Med+Genet&amp;pages=350-355&amp;doi=10.1002%2F%28SICI%291096-8628%2819980724%2978%3A4%253C350%3A%3AAID-AJMG9%253E3.0.CO%3B2-P&amp;pmid=9714438" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">40.</div><div id="r40" class="citations"><div class="citation"><div class="citation-content">Landrum MJ, Lee JM, Riley GR, et al. ClinVar: public archive of relationships among sequence variation and human phenotype. <em>Nucleic Acids Res</em> 2014;42:D980-D985.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r40" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1113" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24234437/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000331139800144" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=ClinVar%3A+public+archive+of+relationships+among+sequence+variation+and+human+phenotype.&amp;publication_year=2014&amp;journal=Nucleic+Acids+Res&amp;pages=D980-D985&amp;doi=10.1093%2Fnar%2Fgkt1113&amp;pmid=24234437" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r40" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r41" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] been implicated in causing disease </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r40-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] of stillbirths had a known genetic cause. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">41.</div><div id="r41" class="citations"><div class="citation"><div class="citation-content">Stenson PD, Mort M, Ball EV, et al. The Human Gene Mutation Database: towards a comprehensive repository of inherited mutation data for medical research, genetic diagnosis and next-generation sequencing studies. <em>Hum Genet</em> 2017;136:665-677.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r41"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-017-1779-6" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28349240/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000401341500001" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=The+Human+Gene+Mutation+Database%3A+towards+a+comprehensive+repository+of+inherited+mutation+data+for+medical+research%2C+genetic+diagnosis+and+next-generation+sequencing+studies.&amp;publication_year=2017&amp;journal=Hum+Genet&amp;pages=665-677&amp;doi=10.1007%2Fs00439-017-1779-6&amp;pmid=28349240" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">42.</div><div id="r42" class="citations"><div class="citation"><div class="citation-content">Yates CL, Monaghan KG, Copenheaver D, et al. Whole-exome sequencing on deceased fetuses with ultrasound anomalies: expanding our knowledge of genetic disease during fetal development. <em>Genet Med</em> 2017;19:1171-1178.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r43-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2017.31" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28425981/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000412303300013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Whole-exome+sequencing+on+deceased+fetuses+with+ultrasound+anomalies%3A+expanding+our+knowledge+of+genetic+disease+during+fetal+development.&amp;publication_year=2017&amp;journal=Genet+Med&amp;pages=1171-1178&amp;doi=10.1038%2Fgim.2017.31&amp;pmid=28425981" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">43.</div><div id="r43" class="citations"><div class="citation"><div class="citation-content">Zhu X, Petrovski S, Xie P, et al. Whole-exome sequencing in undiagnosed genetic diseases: interpreting 119 trios. <em>Genet Med</em> 2015;17:774-781.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r43" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2014.191" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25590979/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000362441900005" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Whole-exome+sequencing+in+undiagnosed+genetic+diseases%3A+interpreting+119+trios.&amp;publication_year=2015&amp;journal=Genet+Med&amp;pages=774-781&amp;doi=10.1038%2Fgim.2014.191&amp;pmid=25590979" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r43" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r43-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] analysis of stillbirths in clinical care. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] research carried out over the past decade, </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-3" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] are associated with established diseases. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">44.</div><div id="r44" class="citations"><div class="citation"><div class="citation-content">Liu Z, Li W, Ma X, et al. Essential role of the zinc finger transcription factor Casz1 for mammalian cardiac morphogenesis and development. <em>J Biol Chem</em> 2014;289:29801-29816.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r44"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M114.570416" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25190801/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000344370800026" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Essential+role+of+the+zinc+finger+transcription+factor+Casz1+for+mammalian+cardiac+morphogenesis+and+development.&amp;publication_year=2014&amp;journal=J+Biol+Chem&amp;pages=29801-29816&amp;doi=10.1074%2Fjbc.M114.570416&amp;pmid=25190801" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">45.</div><div id="r45" class="citations"><div class="citation"><div class="citation-content">Wu X, Kasper LH, Mantcheva RT, Mantchev GT, Springett MJ, van Deursen JM. Disruption of the FG nucleoporin NUP98 causes selective changes in nuclear pore complex stoichiometry and function. <em>Proc Natl Acad Sci U S A</em> 2001;98:3191-3196.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r45"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.051631598" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11248054/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000167521300048" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Disruption+of+the+FG+nucleoporin+NUP98+causes+selective+changes+in+nuclear+pore+complex+stoichiometry+and+function.&amp;publication_year=2001&amp;journal=Proc+Natl+Acad+Sci+U+S+A&amp;pages=3191-3196&amp;doi=10.1073%2Fpnas.051631598&amp;pmid=11248054" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">46.</div><div id="r46" class="citations"><div class="citation"><div class="citation-content">Kasak L, Rull K, SÃµber S, Laan M. Copy number variation profile in the placental and parental genomes of recurrent pregnancy loss families. <em>Sci Rep</em> 2017;7:45327-45327.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r46"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep45327" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28345611/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000397398600001" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Copy+number+variation+profile+in+the+placental+and+parental+genomes+of+recurrent+pregnancy+loss+families.&amp;publication_year=2017&amp;journal=Sci+Rep&amp;pages=45327-45327&amp;doi=10.1038%2Fsrep45327&amp;pmid=28345611" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">47.</div><div id="r47" class="citations"><div class="citation"><div class="citation-content">Brownstein CA, Poduri A, Goldstein RD, Holm IA. The genetics of sudden infant death syndrome. In: Duncan JR, Byard RW, eds. <em>SIDS sudden infant and early childhood death: the past, the present and the future.</em> Adelaide, SA, Australia: University of Adelaide Press, 2018.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r48"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.20851/sids-31" target="_blank">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=The+genetics+of+sudden+infant+death+syndrome&amp;publication_year=2018&amp;doi=10.20851%2Fsids-31" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">48.</div><div id="r48" class="citations"><div class="citation"><div class="citation-content">Crotti L, Tester DJ, White WM, et al. Long QT syndrome-associated mutations in intrauterine fetal death. <em>JAMA</em> 2013;309:1473-1482.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r48"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2013.3219" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23571586/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000317271700025" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Long+QT+syndrome-associated+mutations+in+intrauterine+fetal+death.&amp;publication_year=2013&amp;journal=JAMA&amp;pages=1473-1482&amp;doi=10.1001%2Fjama.2013.3219&amp;pmid=23571586" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">49.</div><div id="r49" class="citations"><div class="citation"><div class="citation-content">Narula N, Tester DJ, Paulmichl A, Maleszewski JJ, Ackerman MJ. Post-mortem whole exome sequencing with gene-specific analysis for autopsy-negative sudden unexplained death in the young: a case series. <em>Pediatr Cardiol</em> 2015;36:768-778.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r49"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00246-014-1082-4" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25500949/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000351553700010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Post-mortem+whole+exome+sequencing+with+gene-specific+analysis+for+autopsy-negative+sudden+unexplained+death+in+the+young%3A+a+case+series.&amp;publication_year=2015&amp;journal=Pediatr+Cardiol&amp;pages=768-778&amp;doi=10.1007%2Fs00246-014-1082-4&amp;pmid=25500949" target="_blank">Google Scholar</a></div></div></div></div></div></div><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--secondary" aria-expanded="false" aria-controls="collapsible-text"><span>Show all references</span></button></div></section></div></section></div><div class="core-collateral"><div id="core-collateral-info" role="tabpanel" data-core-tabs="core-collateral-info" tabindex="-1" aria-labelledby="pane-core-collateral-info"><header><h2 id="pane-core-collateral-info"><i class="icon-info" aria-hidden="true"></i>Information &amp; Authors</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><div role="tablist"><button aria-controls="tab-information" role="tab" id="tab-information-label" aria-selected="true" class="active">Information</button><button aria-controls="tab-contributors" role="tab" id="tab-contributors-label" tabindex="-1" aria-selected="false">Authors</button></div><section id="tab-information" aria-labelledby="tab-information-label" role="tabpanel" tabindex="0"><h3>Information</h3><section class="core-self-citation"><h4>Published In</h4><div class="core-journal-presentation"><div class="core-journal-description"><div property="isPartOf" typeof="Periodical"><span property="name">New England Journal of Medicine</span></div><div class="core-enumeration"><a href="/toc/nejm/383/12"><span property="isPartOf" typeof="PublicationVolume">Volume <span property="volumeNumber">383</span></span> â¢ <span property="isPartOf" typeof="PublicationIssue">Number <span property="issueNumber">12</span></span> â¢ <span property="datePublished">September 17, 2020</span></a></div><div class="core-pagination"><span class="heading">Pages</span>: <span class="content"><span property="pageStart">1107</span>-<span property="pageEnd">1116</span></span></div></div></div></section><section class="core-copyright"><h4>Copyright</h4><div role="paragraph">Copyright Â© 2020 Massachusetts Medical Society. All rights reserved.</div><div>For personal use only. Any commercial reuse of NEJM Group content requires <a href="mailto:permissions@nejm.org">permission</a>.</div></section><section data-translation="YXQYoa1908753" class="core-component-translation"><h4>Translation</h4><div role="paragraph"><a target="_blank" href="https://nejmqianyan.cn/article/YXQYoa1908753?sg=AbW1N">Chinese Translation ä¸­æç¿»è¯</a></div></section><section class="core-history"><h4>History</h4><div><b class="core-label">Published online</b>: August 12, 2020</div><div><b class="core-label">Published in issue</b>: September 17, 2020</div></section><section data-location="recirc_topics_article" class="core-classifications"><h4>Topics</h4><div class="keywords"><ol><li><a href="/browse/topic/obstetrics-gynecology-general" alt="View article keyword Obstetrics/Gynecology General" data-interactiontype="article_recirculation_click">Obstetrics/Gynecology General</a></li><li role="presentation" aria-hidden="true" class="list-line-break"></li><li><a href="/browse/topic/reproductive-medicine" alt="View article keyword Reproductive Medicine" data-interactiontype="article_recirculation_click">Reproductive Medicine</a></li><li role="presentation" aria-hidden="true" class="list-line-break"></li></ol></div></section></section><section id="tab-contributors" aria-labelledby="tab-contributors-label" role="tabpanel" tabindex="-1"><h3>Authors</h3><section class="core-authors"><h4>Authors</h4><div role="paragraph"><span property="author" typeof="Person"><span property="givenName">Kate E.</span> <span property="familyName">Stanley</span>, <span property="honorificSuffix">B.A.</span></span>, <span property="author" typeof="Person"><span property="givenName">Jessica</span> <span property="familyName">Giordano</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span>, <span property="author" typeof="Person"><span property="givenName">Vanessa</span> <span property="familyName">Thorsten</span>, <span property="honorificSuffix">M.P.H.</span></span>, <span property="author" typeof="Person"><span property="givenName">Christie</span> <span property="familyName">Buchovecky</span>, <span property="honorificSuffix">Ph.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Amanda</span> <span property="familyName">Thomas</span>, <span property="honorificSuffix">Ph.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Mythily</span> <span property="familyName">Ganapathi</span>, <span property="honorificSuffix">Ph.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Jun</span> <span property="familyName">Liao</span>, <span property="honorificSuffix">Ph.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Avinash V.</span> <span property="familyName">Dharmadhikari</span>, <span property="honorificSuffix">Ph.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Anya</span> <span property="familyName">Revah-Politi</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span>, <span property="author" typeof="Person"><span property="givenName">Michelle</span> <span property="familyName">Ernst</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span>, <span property="author" typeof="Person"><span property="givenName">Natalie</span> <span property="familyName">Lippa</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span>, <span property="author" typeof="Person"><span property="givenName">Halie</span> <span property="familyName">Holmes</span>, <span property="honorificSuffix">M.S., C.G.C.</span></span>, <span property="author" typeof="Person"><span property="givenName">Gundula</span> <span property="familyName">Povysil</span>, <span property="honorificSuffix">M.D., Ph.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Joseph</span> <span property="familyName">Hostyk</span>, <span property="honorificSuffix">B.S.</span></span>, <span property="author" typeof="Person"><span property="givenName">Corette B.</span> <span property="familyName">Parker</span>, <span property="honorificSuffix">Dr.P.H.</span></span>, <span property="author" typeof="Person"><span property="givenName">Robert</span> <span property="familyName">Goldenberg</span>, <span property="honorificSuffix">M.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">George R.</span> <span property="familyName">Saade</span>, <span property="honorificSuffix">M.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Donald J.</span> <span property="familyName">Dudley</span>, <span property="honorificSuffix">M.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Halit</span> <span property="familyName">Pinar</span>, <span property="honorificSuffix">M.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Carol</span> <span property="familyName">Hogue</span>, <span property="honorificSuffix">Ph.D., M.P.H.</span></span>, <span property="author" typeof="Person"><span property="givenName">Uma M.</span> <span property="familyName">Reddy</span>, <span property="honorificSuffix">M.D., M.P.H.</span></span>, <span property="author" typeof="Person"><span property="givenName">Robert M.</span> <span property="familyName">Silver</span>, <span property="honorificSuffix">M.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Vimla</span> <span property="familyName">Aggarwal</span>, <span property="honorificSuffix">M.B., B.S.</span></span>, <span property="author" typeof="Person"><span property="givenName">Andrew S.</span> <span property="familyName">Allen</span>, <span property="honorificSuffix">Ph.D.</span></span>, <span property="author" typeof="Person"><span property="givenName">Ronald J.</span> <span property="familyName">Wapner</span>, <span property="honorificSuffix">M.D.</span></span>, and <span property="author" typeof="Person"><span property="givenName">David B.</span> <span property="familyName">Goldstein</span>, <span property="honorificSuffix">Ph.D.</span></span></div></section><section class="core-affiliations"><h4>Affiliations</h4><div property="affiliation" typeof="Organization"><span property="name">From the Institute for Genomic Medicine at Vagelos College of Physicians and Surgeons, Columbia University Irving Medical Center (K.E.S., J.G., A.R.-P., M.E., N.L., H.H., G.P., J.H., V.A., R.J.W., D.B.G.), and the Departments of Obstetrics and Gynecology (J.G., R.G., R.J.W.) and Pathology and Cell Biology (C.B., A.T., M.G., J.L., A.V.D., V.A.), Columbia University Medical Center, New York; RTI International, Research Triangle Park (V.T., C.B.P.), and the Department of Biostatistics and Bioinformatics, Duke University, Durham (A.S.A.) â both in North Carolina; the Departments of Obstetrics and Gynecology and Cell Biology, University of Texas Medical Branch, Galveston (G.R.S.); the Division of MaternalâFetal Medicine, Department of Obstetrics and Gynecology, University of Virginia School of Medicine, Charlottesville (D.J.D.); the Division of Perinatal and Pediatric Pathology, Women and Infants Hospital, Warren Alpert School of Medicine of Brown University, Providence, RI (H.P.); Rollins School of Public Health, Emory University, Atlanta (C.H.); Eunice Kennedy Shriver National Institute of Child Health and Human Development, National Institutes of Health, Pregnancy and Perinatology Branch, Bethesda, MD (U.M.R.); and the University of Utah and Intermountain Healthcare, Salt Lake City (R.M.S.).</span></div></section><section class="core-authors-notes"><h4>Notes</h4><div role="doc-footnote">Address reprint requests to Dr. Goldstein at the Institute for Genomic Medicine, Hammer Health Sciences, Columbia University Irving Medical Center, 701 W. 168th St., New York, NY 10032, or at <a href="mailto:dbgoldstein@columbia.edu">dbgoldstein@columbia.edu</a>.</div><div role="doc-footnote"><div id="fn1" role="paragraph">Ms. Stanley and Ms. Giordano and Drs. Wapner and Goldstein contributed equally to this article.</div></div></section></section></div><div id="core-collateral-metrics" role="tabpanel" data-core-tabs="core-collateral-metrics" tabindex="-1" aria-labelledby="pane-core-collateral-metrics"><header><h2 id="pane-core-collateral-metrics"><i class="icon-timeline" aria-hidden="true"></i>Metrics &amp; Citations</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><div role="tablist"><button aria-controls="tab-metrics-inner" role="tab" id="tab-metrics-inner-label" aria-selected="true" class="active" data-interactiontype="article_tab" data-tabname="metricsWindow">Metrics</button><button aria-controls="tab-citations" role="tab" id="tab-citations-label" tabindex="-1" aria-selected="false" data-interactiontype="article_tab" data-tabname="metricsWindow">Citations<span class="citations-count">84</span></button></div><section id="tab-metrics-inner" aria-labelledby="tab-metrics-inner-label" role="tabpanel" tabindex="0"><h3>Metrics</h3><section>









    
    
        <div data-widget-def="UX3HTMLWidget" data-widget-id="95ea9f58-ec0e-4db2-b2ab-280fc876af53" data-location="article_metrics_article">
        



        
            <h4 class="">
                Altmetrics
            </h4>
        
        <div data-badge-details="right" data-badge-type="large-donut" data-doi="10.1056/NEJMoa1908753" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed" data-uuid="587305ea-62f9-09aa-694d-5d53609deacb"><div style="overflow:hidden;">
    <div class="altmetric-normal-legend">
        <a target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=88153839" style="display:inline-block;">
                <img alt="Article has an altmetric score of 165" src="https://badges.altmetric.com/?size=320&amp;score=165&amp;types=mbtttttf" width="180" height="180" style="border:0; margin:0; max-width: none;">
        </a>
        <p class="altmetric-see-more-details" style="padding-top: 10px; text-align: center;"><a target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=88153839">See more details</a></p>
    </div>
    <div id="_altmetric_popover_el_587305ea-62f9-09aa-694d-5d53609deacb" class="altmetric-embed right" style="margin:0; padding:0; display:inline-block; float:left; position:relative;">
        <div class="altmetric_container">
            <div class="altmetric-embed altmetric-popover-inner right">
                <div style="padding:0; margin: 0;" class="altmetric-embed altmetric-popover-content">
                        <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #FF0000;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=88153839&amp;tab=news">
          Picked up by <b>15</b> news outlets
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=88153839&amp;tab=blogs">
          Blogged by <b>4</b>
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=88153839&amp;tab=twitter">
          Posted by <b>49</b> X users
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #2445bd;">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.nejm.org&amp;citation_id=88153839&amp;tab=facebook">
          On <b>7</b> Facebook pages
      </a>
    </div>

    <div class="altmetric-embed readers" style="margin-top: 10px;">
          <div class="altmetric-embed tip_mendeley" style="padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;">
              <b>152</b> readers on Mendeley
          </div>
    </div>

                </div>
            </div>
        </div>
    </div>
</div></div>
<script async="" type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js" nonce="94d64844d6a5f20e-SJC"></script>

        </div>
    

</section></section><section id="tab-citations" aria-labelledby="tab-citations-label" role="tabpanel" tabindex="-1"><h3>Citations</h3><section>



        
            <h4 class="">
                Export citation
            </h4>
        
        <div class="citation-download cd-sec">
<p class="citation-download_msg">Select the format you want to export the citation of this publication.</p>
<form action="/action/downloadCitation" name="frmCitmgr" method="post" target="_self" class="citation-download_form"><input type="hidden" name="doi" value="10.1056/NEJMoa1908753"> <input type="hidden" name="downloadFileName" value="csp_383_"> <input type="hidden" name="include" value="abs">
<div class="dropdown dropdown-selectable dropdown-selectable--form-control"><input type="hidden" name="format" value=""> <label class="form-label">Format*</label> <button id="slct_format" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="btn dropdown-toggle d-flex align-items-center justify-content-between">Please Select </button>
<div aria-labelledby="slct_format" role="listbox" class="dropdown-menu"><a href="#" data-value="ris" class="dropdown-item">RIS (ProCite, Reference Manager)</a><a href="#" data-value="endnote" class="dropdown-item">EndNote</a><a href="#" data-value="bibtex" class="dropdown-item">BibTex</a><a href="#" data-value="medlars" class="dropdown-item">Medlars</a><a href="#" data-value="refworks" class="dropdown-item">RefWorks</a></div>
<div class="invalid-feedback">Please select an item in the list</div>
</div>
<div class="citation-download_checkbox"><label class="checkbox--primary"><input type="checkbox" name="direct" value="true"><span class="checkbox--primary-icons"><span class="checkbox--primary-unchecked"><span class="icon-component "><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#toggles-checboxUnchecked"></use></svg></span></span><span class="checkbox--primary-checked"><span class="icon-component "><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#toggles-checboxChecked"></use></svg></span></span></span><span class="checkbox--primary-label"><span class="checkbox--primary-label-text">Direct Import</span></span></label></div>
<div class="citation-download_btn"><button type="submit" name="submit" value="Download" class="ng-btn_default ng-btn_iconLeft" data-interactiontype="article_tools_citation_download" data-multimedia-type="Citation"><span class="icon-component "><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#citations"></use></svg></span><span class="ng-btn_text">Export citation</span></button></div>
</form></div>
</section><section>









    
    
        <div data-widget-def="UX3CitedByWidget" data-widget-id="efb4df9a-8e1a-47f9-bfb7-271959277a9f" data-location="citedby_article">
        



        
            <h4 class="">
                Cited by
            </h4>
        
        <section id="cited-by" class="cited-by"><div class="cited-by__wrapper"><div id="cited-by__content" class="cited-by__content"><ol data-source="/pb/widgets/citedBy?page=1&amp;widgetId=efb4df9a-8e1a-47f9-bfb7-271959277a9f&amp;pbContext=%3BrequestedJournal%3Ajournal%3Anejm%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3Adoi%5C%3A10.1056%2Fnejm_2020.383.issue-12%3Barticle%3Aarticle%3Adoi%5C%3A10.1056%2FNEJMoa1908753%3Bwgroup%3Astring%3AMMS+NextGen+Website+Group%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bwebsite%3Awebsite%3Amms-site%3Bjournal%3Ajournal%3Anejms" data-lazyload="false" data-total="84" class="cited-by__list list-unstyled"><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Mounika Endrakanti, </li><li class="list-inline-item cited-by__entry__author">Sarath R. S, </li><li class="list-inline-item cited-by__entry__author">Neerja Gupta, </li><li class="list-inline-item cited-by__entry__author">Madhulika Kabra, </li></ul><span class="cited-by__entry__title">Clinical Utility and Yield of Carrier Exome Sequencing in HighâRisk Indian Couples, </span><span class="cited-by__entry__series-title">Prenatal Diagnosis, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1002/pd.6830" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1002/pd.6830</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1002/pd.6830" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Roni Zemet, </li><li class="list-inline-item cited-by__entry__author">Christian M. Parobek, </li><li class="list-inline-item cited-by__entry__author">April D. Adams, </li><li class="list-inline-item cited-by__entry__author">Mohamad Ali Maktabi, </li><li class="list-inline-item cited-by__entry__author">Lena Shay, </li><li class="list-inline-item cited-by__entry__author">Linyan Meng, </li><li class="list-inline-item cited-by__entry__author">Pengfei Liu, </li><li class="list-inline-item cited-by__entry__author">Hongzheng Dai, </li><li class="list-inline-item cited-by__entry__author">Fan Xia, </li><li class="list-inline-item cited-by__entry__author">Christine Eng, </li><li class="list-inline-item cited-by__entry__author">Ignatia B. Van den Veyver, </li><li class="list-inline-item cited-by__entry__author">Liesbeth Vossaert, </li></ul><span class="cited-by__entry__title">Diagnostic Yield of Exome Sequencing for Pregnancies With and Without Fetal Anomalies and for Stillbirth, </span><span class="cited-by__entry__series-title">Prenatal Diagnosis, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1002/pd.6817" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1002/pd.6817</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1002/pd.6817" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Christina G. Tise, </li><li class="list-inline-item cited-by__entry__author">Kajal Verma, </li><li class="list-inline-item cited-by__entry__author">Ruth B. Lathi, </li></ul><span class="cited-by__entry__title">Pregnancy Loss: From Cytogenetics to Genomics, </span><span class="cited-by__entry__series-title">Early Pregnancy, </span><span class="cited-by__entry__page-range">(299-316), </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1017/9781009532549.028" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1017/9781009532549.028</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1017/9781009532549.028" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Ulrik Stoltze, </li><li class="list-inline-item cited-by__entry__author">Stefanie V. Junk, </li><li class="list-inline-item cited-by__entry__author">Anna Byrjalsen, </li><li class="list-inline-item cited-by__entry__author">HÃ©lÃ¨ne CavÃ©, </li><li class="list-inline-item cited-by__entry__author">Giovanni Cazzaniga, </li><li class="list-inline-item cited-by__entry__author">Sarah Elitzur, </li><li class="list-inline-item cited-by__entry__author">Eva Fronkova, </li><li class="list-inline-item cited-by__entry__author">Lisa Lyngsie Hjalgrim, </li><li class="list-inline-item cited-by__entry__author">Roland P. Kuiper, </li><li class="list-inline-item cited-by__entry__author">Louise Lundgren, </li><li class="list-inline-item cited-by__entry__author">Melina Mescher, </li><li class="list-inline-item cited-by__entry__author">Theis Mikkelsen, </li><li class="list-inline-item cited-by__entry__author">Agata Pastorczak, </li><li class="list-inline-item cited-by__entry__author">Marion Strullu, </li><li class="list-inline-item cited-by__entry__author">Jan Trka, </li><li class="list-inline-item cited-by__entry__author">Karin Wadt, </li><li class="list-inline-item cited-by__entry__author">Shai Izraeli, </li><li class="list-inline-item cited-by__entry__author">Arndt Borkhardt, </li><li class="list-inline-item cited-by__entry__author">Kjeld Schmiegelow, </li></ul><span class="cited-by__entry__title">Overt and covert genetic causes of pediatric acute lymphoblastic leukemia, </span><span class="cited-by__entry__series-title">Leukemia, </span><span class="cited-by__entry__volume"><strong>39</strong>, </span><span class="cited-by__entry__issue">5, </span><span class="cited-by__entry__page-range">(1031-1045), </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1038/s41375-025-02535-4" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1038/s41375-025-02535-4</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1038/s41375-025-02535-4" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Ida Vogel, </li><li class="list-inline-item cited-by__entry__author">Lotte Andreasen, </li><li class="list-inline-item cited-by__entry__author">Marie BalslevâHarder, </li><li class="list-inline-item cited-by__entry__author">Naja Becher, </li><li class="list-inline-item cited-by__entry__author">Anja Ernst, </li><li class="list-inline-item cited-by__entry__author">Kasper GadsbÃ¸ll, </li><li class="list-inline-item cited-by__entry__author">Tina Duelund HjortshÃ¸j, </li><li class="list-inline-item cited-by__entry__author">Marie Skov Hvidbjerg, </li><li class="list-inline-item cited-by__entry__author">Martin Larsen, </li><li class="list-inline-item cited-by__entry__author">Stina Lou, </li><li class="list-inline-item cited-by__entry__author">Ida Charlotte Bay Lund, </li><li class="list-inline-item cited-by__entry__author">Lars Henning Pedersen, </li><li class="list-inline-item cited-by__entry__author">Laura Kirstine SÃ¸nderberg Roos, </li><li class="list-inline-item cited-by__entry__author">Lene Sperling, </li><li class="list-inline-item cited-by__entry__author">Lone Sunde, </li><li class="list-inline-item cited-by__entry__author">Pernille Mathiesen TÃ¸rring, </li><li class="list-inline-item cited-by__entry__author">Cathrine Vedel, </li><li class="list-inline-item cited-by__entry__author">Olav BjÃ¸rn Petersen, </li></ul><span class="cited-by__entry__title">Whole Genome Sequencing in Prenatal Diagnostics: The Danish Approach to Guideline Formation and Implementation Within Public Healthcare, </span><span class="cited-by__entry__series-title">Prenatal Diagnosis, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1002/pd.6780" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1002/pd.6780</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1002/pd.6780" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Kosuke Taniguchi, </li><li class="list-inline-item cited-by__entry__author">Fuyuki Hasegawa, </li><li class="list-inline-item cited-by__entry__author">Yuka Okazaki, </li><li class="list-inline-item cited-by__entry__author">Asuka Hori, </li><li class="list-inline-item cited-by__entry__author">Hiroko OgataâKawata, </li><li class="list-inline-item cited-by__entry__author">Saki Aoto, </li><li class="list-inline-item cited-by__entry__author">Ohsuke Migita, </li><li class="list-inline-item cited-by__entry__author">Tomoko Kawai, </li><li class="list-inline-item cited-by__entry__author">Kazuhiko Nakabayashi, </li><li class="list-inline-item cited-by__entry__author">Kohji Okamura, </li><li class="list-inline-item cited-by__entry__author">Kana Fukui, </li><li class="list-inline-item cited-by__entry__author">Seiji Wada, </li><li class="list-inline-item cited-by__entry__author">Katsusuke Ozawa, </li><li class="list-inline-item cited-by__entry__author">Yushi Ito, </li><li class="list-inline-item cited-by__entry__author">Haruhiko Sago, </li><li class="list-inline-item cited-by__entry__author">Kenichiro Hata, </li></ul><span class="cited-by__entry__title">Approaches to Evaluate Whole Exome Sequencing Data That Incorporate Genetic Intolerance Scores for Congenital Anomalies, Including Intronic Regions Adjacent to Exons, </span><span class="cited-by__entry__series-title">Molecular Genetics &amp; Genomic Medicine, </span><span class="cited-by__entry__volume"><strong>13</strong>, </span><span class="cited-by__entry__issue">3, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1002/mgg3.70092" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1002/mgg3.70092</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1002/mgg3.70092" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">J. Ciaran Hutchinson, </li><li class="list-inline-item cited-by__entry__author">Lorraine Potocki, </li><li class="list-inline-item cited-by__entry__author">Ignatia B. Van den Veyver, </li></ul><span class="cited-by__entry__title">Current Controversies in Prenatal Diagnosis 2: Conventional Postmortem Examination Remains the Gold Standard for the Anatomical Examination of Fetal Loss, </span><span class="cited-by__entry__series-title">Prenatal Diagnosis, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1002/pd.6754" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1002/pd.6754</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1002/pd.6754" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Shunsuke Yamamoto, </li><li class="list-inline-item cited-by__entry__author">Masafumi Sanefuji, </li><li class="list-inline-item cited-by__entry__author">Hirosuke Inoue, </li><li class="list-inline-item cited-by__entry__author">Masataka Inoue, </li><li class="list-inline-item cited-by__entry__author">Yu Shimo, </li><li class="list-inline-item cited-by__entry__author">Shunichiro Toya, </li><li class="list-inline-item cited-by__entry__author">Maya Suzuki, </li><li class="list-inline-item cited-by__entry__author">Nozomi Abe, </li><li class="list-inline-item cited-by__entry__author">Norio Hamada, </li><li class="list-inline-item cited-by__entry__author">Utako Oba, </li><li class="list-inline-item cited-by__entry__author">Kentaro Nakashima, </li><li class="list-inline-item cited-by__entry__author">Masayuki Ochiai, </li><li class="list-inline-item cited-by__entry__author">Reiko Suga, </li><li class="list-inline-item cited-by__entry__author">Yuhki Koga, </li><li class="list-inline-item cited-by__entry__author">Mayumi Tsuji, </li><li class="list-inline-item cited-by__entry__author">Kiyoko Kato, </li><li class="list-inline-item cited-by__entry__author">Shouichi Ohga, </li></ul><span class="cited-by__entry__title">Parental occupational exposure to anticancer drugs and radiation: Risk of fetal loss and physical abnormalities in The Japan Environment and Children's Study, </span><span class="cited-by__entry__series-title">Early Human Development, </span><span class="cited-by__entry__volume"><strong>201</strong>, </span><span class="cited-by__entry__page-range">(106195), </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.1016/j.earlhumdev.2025.106195" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1016/j.earlhumdev.2025.106195</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1016/j.earlhumdev.2025.106195" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">S. Suganathan Soundararajan, </li></ul><span class="cited-by__entry__title">Addressing Stillbirth: Exploring Economic Status, Government Strategies, and Gaps, </span><span class="cited-by__entry__series-title">Cureus, </span><span class="cited-by__entry__pub-date">(2025).</span><a href="https://doi.org/10.7759/cureus.76923" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.7759/cureus.76923</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.7759/cureus.76923" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry border-bottom pb-16 mb-24"><ul class="cited-by__entry__authors list-inline"><li class="list-inline-item cited-by__entry__author">Luke A. Gatta, </li><li class="list-inline-item cited-by__entry__author">Allison M. McCarthy, </li><li class="list-inline-item cited-by__entry__author">Sarah S. Osmundson, </li></ul><span class="cited-by__entry__title">Contemporary uses of âlethalâ or âlife limitingâ terminology in perinatal research, </span><span class="cited-by__entry__series-title">Current Opinion in Obstetrics &amp; Gynecology, </span><span class="cited-by__entry__volume"><strong>37</strong>, </span><span class="cited-by__entry__issue">2, </span><span class="cited-by__entry__page-range">(49-54), </span><span class="cited-by__entry__pub-date">(2024).</span><a href="https://doi.org/10.1097/GCO.0000000000001010" title="Crossref" target="_blank" class="cited-by__entry__doi">https://doi.org/10.1097/GCO.0000000000001010</a><div class="cited-by__entry__extra-links"><a href="https://doi.org/10.1097/GCO.0000000000001010" target="_blank" class="cited-by__entry__visitable">Crossref</a></div></li><li class="cited-by__entry--see-more"><a href="javascript:void(0)" data-source="/pb/widgets/citedBy?page=1&amp;widgetId=efb4df9a-8e1a-47f9-bfb7-271959277a9f&amp;pbContext=%3BrequestedJournal%3Ajournal%3Anejm%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3Adoi%5C%3A10.1056%2Fnejm_2020.383.issue-12%3Barticle%3Aarticle%3Adoi%5C%3A10.1056%2FNEJMoa1908753%3Bwgroup%3Astring%3AMMS+NextGen+Website+Group%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bwebsite%3Awebsite%3Amms-site%3Bjournal%3Ajournal%3Anejms" class="cited-by__see-more">See more</a></li></ol></div><div class="cited-by__spinner justify-content-center align-items-center d-none"><div role="status" class="spinner-border text-gray"><span class="sr-only">Loading...</span></div></div></div></section>

        </div>
    

</section></section></div><div id="core-collateral-fulltext-options" role="tabpanel" data-core-tabs="core-collateral-fulltext-options" tabindex="-1" aria-labelledby="pane-core-collateral-fulltext-options"><header><h2 id="pane-core-collateral-fulltext-options"><i class="icon-eye" aria-hidden="true"></i>View Options</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><div class="section--wrapper"><h3>View options</h3><section class="format--pdf"><h4> <abbr title="Portable Document Format">PDF</abbr></h4><a href="https://www.nejm.org/doi/pdf/10.1056/NEJMoa1908753" class="btn btn--pdf" aria-label="View PDF" data-behavior="trackDownloadEvent" data-doi="10.1056/NEJMoa1908753" data-interactiontype="article_tools_download_pdf" data-multimedia-type="Article PDF" data-multimedia-filename="nejmoa1908753.pdf"><i aria-hidden="true" class="icon-pdf"></i><span>View PDF</span></a></section></div><!-- Its needed to duplicate this for PB check Collateral.js as well--></div><div id="core-collateral-media" role="tabpanel" data-core-tabs="core-collateral-media" tabindex="-1" aria-labelledby="pane-core-collateral-media"><header><h2 id="pane-core-collateral-media"><i class="icon-photo" aria-hidden="true"></i>Media</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><div role="tablist"><button aria-controls="tab-figures" role="tab" id="tab-figures-label" aria-selected="true" class="active">Figures</button><button aria-controls="tab-other" role="tab" id="tab-other-label" tabindex="-1" aria-selected="false" class="inactive" aria-disabled="true">Other</button></div><section id="tab-figures" aria-labelledby="tab-figures-label" role="tabpanel" tabindex="0"><h3>Figures</h3><div class="text-right"><a class="btn btn--inverse  open-in-viewer" href="#figures"><span>Open all in viewer</span><i class="icon-open_in_new" aria-hidden="true"></i></a></div><div id="core-f1"><figure class="graphic"><a class="open-in-viewer" href="#f1" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f1.jpg"><img src="/cms/10.1056/NEJMoa1908753/asset/fc1d9fab-8597-4cb8-b3b1-b9c2a5e17ea8/assets/images/large/nejmoa1908753_f1.jpg" height="2347" width="2640" aria-labelledby="f1" loading="lazy"></a><figcaption><div class="caption">Intolerance to Genetic Variation.</div><div class="notes"><div role="doc-footnote">Regions of the human genome that are under the strongest natural selection are the most likely to cause disease when they are variant. Until 2013, the primary approach that was used to identify such regions relied on the genetic similarities of different species.<sup><a href="#core-r25" role="doc-biblioref" data-xml-rid="r25" id="core-body-ref-r25" href-manipulated="true" aria-label="Reference 25" data-to-manipulate="false">25</a></sup> Genomic regions under selection show fewer DNA changes (e.g., nucleotide substitutions, deletions, or insertions) across species and are likely to be functionally important (Panel A). Pathogenic variants that cause human diseases have long been shown to fall preferentially within these âconstrainedâ regions. Although this approach is useful, it cannot identify genomic regions of particular importance in humans, as might happen because of the evolution of a novel function. In 2013, a new framework was developed to address this limitation: variation, solely within the human population, was used to identify genes with less functional variation than expected according to genomewide averages. Genes with a depletion of human variation are termed âintolerantâ<sup><a href="#core-r26" role="doc-biblioref" data-xml-rid="r26" id="core-body-ref-r26" href-manipulated="true" aria-label="Reference 26" data-to-manipulate="false">26</a></sup> and reflect parts of the genome under strong selection specifically in humans. Since the introduction of intolerance scoring, there have been a number of important elaborations focused on regions of genes, specific types of variants, and regulatory regions.<sup><a href="#core-r27" role="doc-biblioref" data-xml-rid="r27 r28 r29 r30" id="core-body-ref-r30" href-manipulated="true" data-to-manipulate="false">27-30</a></sup> Intolerance scores have now been shown to provide independent information about where in the human genome pathogenic variants are found (Panel B).<sup><a href="#core-r24" role="doc-biblioref" data-xml-rid="r24 r29 r31 r32 r33 r34" id="core-body-ref-r34" href-manipulated="true" data-to-manipulate="false">24,29,31â34</a></sup> For example, the gene encoding olfactory receptor 51E1 (<i>OR51E1</i>) is tolerant to variation and does not cause disease, whereas <i>MYBPC3</i> and <i>HNF1B</i> are intolerant and are known to cause cardiac and kidney disease, respectively. <i>CASZ1</i> is highly intolerant to variation in the healthy population, but no pathogenic variants have been reported in the literature regarding postnatal disease, which indicates that it may result in lethality when variant in utero.</div></div></figcaption></figure><div class="external-links"><a class="pill__to-original" href="#f1"><i class="icon-return" aria-hidden="true"></i><span>Go to Figure</span></a><a class="open-in-viewer" href="#f1" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f1.jpg"><span>Open in Viewer</span><i class="icon-expand" aria-hidden="true"></i></a></div></div><div id="core-f2"><figure class="graphic"><a class="open-in-viewer" href="#f2" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f2.jpg"><img src="/cms/10.1056/NEJMoa1908753/asset/df562d50-26f1-4f15-9edf-50bc1559b2c3/assets/images/large/nejmoa1908753_f2.jpg" height="1866" width="1656" aria-labelledby="f2" loading="lazy"></a><figcaption><div class="caption">Enrichment of Loss-of-Function Variants in Stillbirth Cases, as Compared with Controls.</div><div class="notes"><div role="doc-footnote">To explore which genetic variants may be associated with stillbirth, we assigned an LOEUF (loss-of-function observed-to-expected upper boundary fraction) value to all 8081 genes with a loss-of-function variant in 241 case samples and 7239 control samples. The process resulted in 1825 unique LOEUF values. The LOEUF value is the upper boundary of a Poisson-derived confidence interval of the observed-to-expected ratio. A low LOEUF score indicates a depletion of loss-of-function variation (also referred to as âintoleranceâ to loss-of-function variation). The enrichment of loss-of-function variants was then assessed by comparing the count of variants in cases and controls in each of these 1825-gene groupings. Enrichment was evaluated with the use of a logistic-regression model that evaluated caseâcontrol status regarding the presence or absence of a loss-of-function variant in each gene grouping. Enrichment of presumed neutral synonymous variants at each intolerance threshold was used as a covariate to control for differences between case and control genomes that are unrelated to disease risk. The negative log of the enrichment statistic (unadjusted P value) at each LOEUF threshold is shown on the y axis. The x axis reflects the LOEUF threshold for each gene grouping. The minimum observed P value, which indicates the most significant enrichment across all LOEUF thresholds, occurs at a LOEUF threshold of 0.239 or less. The significance of the observed minimum P value was then assessed by permutation (number of permutations, 10,000).</div></div></figcaption></figure><div class="external-links"><a class="pill__to-original" href="#f2"><i class="icon-return" aria-hidden="true"></i><span>Go to Figure</span></a><a class="open-in-viewer" href="#f2" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f2.jpg"><span>Open in Viewer</span><i class="icon-expand" aria-hidden="true"></i></a></div></div><div id="core-f3"><figure class="graphic"><a class="open-in-viewer" href="#f3" data-index="0" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f3.jpg"><img src="/cms/10.1056/NEJMoa1908753/asset/db99f9fb-56d6-444f-9808-967ca1cae228/assets/images/large/nejmoa1908753_f3.jpg" height="1226" width="2640" aria-labelledby="f3" loading="lazy"></a><figcaption><div class="caption">Loss-of-Function Signal in Intolerant Genes in Three Clinical Cohorts.</div><div class="notes"><div role="doc-footnote">In this study, the LOEUF threshold that had the strongest enrichment of loss-of-function variants (â¤0.239) in the 241 cases in the stillbirth cohort was used to make point comparisons with the loss-of-function signal in a group of 251 live-birth cases with fetal anomalies and in 589 cases with postnatal disease. The odds ratio indicates the enrichment of loss-of-function variants in the gene set in cases as compared with a large group of controls. The gene sets are shown in three categories: all the genes below the LOEUF threshold (â¤0.239), those that had an existing disease association in the Online Mendelian Inheritance in Man (OMIM) database, and those that did not have such an association (non-OMIM).</div></div></figcaption></figure><div class="external-links"><a class="pill__to-original" href="#f3"><i class="icon-return" aria-hidden="true"></i><span>Go to Figure</span></a><a class="open-in-viewer" href="#f3" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f3.jpg"><span>Open in Viewer</span><i class="icon-expand" aria-hidden="true"></i></a></div></div></section><section id="tab-other" aria-labelledby="tab-other-label" role="tabpanel" tabindex="-1"><h3>Other</h3></section></div><div id="core-collateral-tables" role="tabpanel" data-core-tabs="core-collateral-tables" tabindex="-1" aria-labelledby="pane-core-collateral-tables"><header><h2 id="pane-core-collateral-tables"><i class="icon-tables" aria-hidden="true"></i>Tables</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><!-- There is no content. --></div><div id="core-collateral-share" role="tabpanel" data-core-tabs="core-collateral-share" tabindex="-1" aria-labelledby="pane-core-collateral-share"><header><h2 id="pane-core-collateral-share"><i class="icon-share" aria-hidden="true"></i>Share</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><h3>Share</h3><div class="section--wrapper"><section><h4>CONTENT LINK</h4><div id="share-self"><p data-id="article-share-self-link" class="share-self__source">https://www.nejm.org/doi/full/10.1056/NEJMoa1908753</p><button data-id="article-share-access" aria-label="CONTENT LINK" class="share-self__action btn btn--inverse"><i aria-hidden="true" class="icon-copy-o"></i><span>Copy Link</span></button><div aria-live="polite" class="share-self__status"><p class="share-self__success"><i aria-hidden="true" class="icon-check_circle"></i><span>Copied!</span></p><p class="share-self__failed"><i aria-hidden="true" class="icon-x_btnclose"></i><span>Copying failed.</span></p></div></div></section><section data-location="share_tools_article"><h4>Share</h4><div class="share-buttons a2a a2a_kit" style="line-height: 16px;"><a href="/#facebook" aria-label="Share on Facebook" data-id="article-share-facebook" rel="nofollow noopener" role="link" target="_blank" title="Share on Facebook" class="btn btn--facebook a2a_button_facebook" data-interactiontype="social"><i aria-hidden="true" class="icon-facebook"></i><span>Facebook</span></a><a href="/#twitter" aria-label="Share on X (formerly Twitter)" data-id="article-share-twitter" rel="nofollow noopener" role="link" target="_blank" title="Share on X (formerly Twitter)" class="btn btn--twitter a2a_button_twitter" data-interactiontype="social"><i aria-hidden="true" class="icon-twitter"></i><span>X (formerly Twitter)</span></a><a href="/#linkedin" aria-label="Share on LinkedIn" data-id="article-share-linkedin" rel="nofollow noopener" role="link" target="_blank" title="Share on LinkedIn" class="btn btn--linkedin a2a_button_linkedin" data-interactiontype="social"><i aria-hidden="true" class="icon-linkedin"></i><span>LinkedIn</span></a><a href="/#email" aria-label="Share on email" data-id="article-share-email" rel="nofollow noopener" role="link" target="_blank" title="Share on email" class="btn btn--email a2a_button_email" data-interactiontype="content_email_click"><i aria-hidden="true" class="icon-mail"></i><span>email</span></a><a href="/#bluesky" aria-label="Share on Bluesky" data-id="article-share-bluesky" rel="nofollow noopener" role="link" target="_blank" title="Share on Bluesky" class="btn btn--bluesky a2a_button_bluesky" data-interactiontype="social"><i aria-hidden="true" class="icon-bluesky"></i><span>Bluesky</span></a></div></section></div></div><div data-location="references_article" id="core-collateral-references" role="tabpanel" aria-labelledby="pane-core-collateral-references" data-core-tabs="core-collateral-references" tabindex="-1"><header><h2 id="pane-core-collateral-references"><i aria-hidden="true" class="icon-references"></i>References</h2><button data-close="collateral" aria-label="Close"><i class="icon-close"></i></button></header><h3>References</h3><div role="list" class="section--wrapper"><div role="listitem" data-has="label"><div class="label">1.</div><div id="core-r1" class="citations"><div class="citation"><div class="citation-content">Wapner RJ, Lewis D. Genetics and metabolic causes of stillbirth. <em>Semin Perinatol</em> 2002;26:70-74.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r3-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1053/sper.2002.29853" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11876569/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000174004700010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genetics+and+metabolic+causes+of+stillbirth.&amp;publication_year=2002&amp;journal=Semin+Perinatol&amp;pages=70-74&amp;doi=10.1053%2Fsper.2002.29853&amp;pmid=11876569" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">2.</div><div id="core-r2" class="citations"><div class="citation"><div class="citation-content">Stillbirth Collaborative Research Network Writing Group. Causes of death among stillbirths. <em>JAMA</em> 2011;306:2459-2468.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r2-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2011.1823" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22166605/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000297983300019" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Causes+of+death+among+stillbirths.&amp;publication_year=2011&amp;journal=JAMA&amp;pages=2459-2468&amp;doi=10.1001%2Fjama.2011.1823&amp;pmid=22166605" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r2-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r3-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] and is unexplained in 25 to 60% of cases. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r2-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] from March 2006 through September 2008. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r3-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] which has been described previously, </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r2-4" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>d [...] 390 fetuses (76.2%; 95% CI, 72.2 to 79.8). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r2-5" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>e [...] than one cause identified in some cases. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">3.</div><div id="core-r3" class="citations"><div class="citation"><div class="citation-content">Reddy UM, Page GP, Saade GR, et al. Karyotype versus microarray testing for genetic abnormalities after stillbirth. <em>N Engl J Med</em> 2012;367:2185-2193.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r3-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_4_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa1908753&amp;key=10.1056%2FNEJMoa1201569&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23215556/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000311890600005" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Karyotype+versus+microarray+testing+for+genetic+abnormalities+after+stillbirth.&amp;publication_year=2012&amp;journal=N+Engl+J+Med&amp;pages=2185-2193&amp;doi=10.1056%2FNEJMoa1201569&amp;pmid=23215556" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r3-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r3-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] and is unexplained in 25 to 60% of cases. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r3-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] which has been described previously, </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r4-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] pathological karyotype or microarray). </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">4.</div><div id="core-r4" class="citations"><div class="citation"><div class="citation-content">Lee H, Deignan JL, Dorrani N, et al. Clinical exome sequencing for genetic identification of rare Mendelian disorders. <em>JAMA</em> 2014;312:1880-1887.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r4-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2014.14604" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25326637/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000344532600017" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Clinical+exome+sequencing+for+genetic+identification+of+rare+Mendelian+disorders.&amp;publication_year=2014&amp;journal=JAMA&amp;pages=1880-1887&amp;doi=10.1001%2Fjama.2014.14604&amp;pmid=25326637" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r4-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r12-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] to determine the cause of disease. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r4-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] pathological karyotype or microarray). </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">5.</div><div id="core-r5" class="citations"><div class="citation"><div class="citation-content">Yang Y, Muzny DM, Reid JG, et al. Clinical whole-exome sequencing for the diagnosis of mendelian disorders. <em>N Engl J Med</em> 2013;369:1502-1511.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_6_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa1908753&amp;key=10.1056%2FNEJMoa1306555&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24088041/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000328101700007" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Clinical+whole-exome+sequencing+for+the+diagnosis+of+mendelian+disorders.&amp;publication_year=2013&amp;journal=N+Engl+J+Med&amp;pages=1502-1511&amp;doi=10.1056%2FNEJMoa1306555&amp;pmid=24088041" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">6.</div><div id="core-r6" class="citations"><div class="citation"><div class="citation-content">Groopman EE, Marasa M, Cameron-Christie S, et al. Diagnostic utility of exome sequencing for kidney disease. <em>N Engl J Med</em> 2019;380:142-151.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_7_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa1908753&amp;key=10.1056%2FNEJMoa1806891&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30586318/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000455310300008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Diagnostic+utility+of+exome+sequencing+for+kidney+disease.&amp;publication_year=2019&amp;journal=N+Engl+J+Med&amp;pages=142-151&amp;doi=10.1056%2FNEJMoa1806891&amp;pmid=30586318" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">7.</div><div id="core-r7" class="citations"><div class="citation"><div class="citation-content">Splinter K, Adams DR, Bacino CA, et al. Effect of genetic diagnosis on patients with previously undiagnosed disease. <em>N Engl J Med</em> 2018;379:2131-2139.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_3_5_8_2&amp;dbid=4&amp;doi=10.1056%2FNEJMoa1908753&amp;key=10.1056%2FNEJMoa1714458&amp;site=mms-site" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30304647/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000451402500008" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Effect+of+genetic+diagnosis+on+patients+with+previously+undiagnosed+disease.&amp;publication_year=2018&amp;journal=N+Engl+J+Med&amp;pages=2131-2139&amp;doi=10.1056%2FNEJMoa1714458&amp;pmid=30304647" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">8.</div><div id="core-r8" class="citations"><div class="citation"><div class="citation-content">Bagnall RD, Crompton DE, Petrovski S, et al. Exome-based analysis of cardiac arrhythmia, respiratory control, and epilepsy genes in sudden unexpected death in epilepsy. <em>Ann Neurol</em> 2016;79:522-534.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ana.24596" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26704558/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000374006100003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Exome-based+analysis+of+cardiac+arrhythmia%2C+respiratory+control%2C+and+epilepsy+genes+in+sudden+unexpected+death+in+epilepsy.&amp;publication_year=2016&amp;journal=Ann+Neurol&amp;pages=522-534&amp;doi=10.1002%2Fana.24596&amp;pmid=26704558" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">9.</div><div id="core-r9" class="citations"><div class="citation"><div class="citation-content">Yang Y, Muzny DM, Xia F, et al. Molecular findings among patients referred for clinical whole-exome sequencing. <em>JAMA</em> 2014;312:1870-1879.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r12-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2014.14601" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25326635/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000344532600016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Molecular+findings+among+patients+referred+for+clinical+whole-exome+sequencing.&amp;publication_year=2014&amp;journal=JAMA&amp;pages=1870-1879&amp;doi=10.1001%2Fjama.2014.14601&amp;pmid=25326635" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">10.</div><div id="core-r10" class="citations"><div class="citation"><div class="citation-content">Neubauer J, Lecca MR, Russo G, et al. Post-mortem whole-exome analysis in a large sudden infant death syndrome cohort with a focus on cardiovascular and metabolic genetic diseases. <em>Eur J Hum Genet</em> 2017;25:404-409.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r10-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ejhg.2016.199" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28074886/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000395996200006" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Post-mortem+whole-exome+analysis+in+a+large+sudden+infant+death+syndrome+cohort+with+a+focus+on+cardiovascular+and+metabolic+genetic+diseases.&amp;publication_year=2017&amp;journal=Eur+J+Hum+Genet&amp;pages=404-409&amp;doi=10.1038%2Fejhg.2016.199&amp;pmid=28074886" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r10-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r12-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] to determine the cause of disease. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r48" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] of stillbirth without structural anomalies. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">11.</div><div id="core-r11" class="citations"><div class="citation"><div class="citation-content">Lord J, McMullan DJ, Eberhardt RY, et al. Prenatal exome sequencing analysis in fetal structural anomalies detected by ultrasonography (PAGE): a cohort study. <em>Lancet</em> 2019;393:747-757.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r11-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0140-6736(18)31940-8" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30712880/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000459945500025" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Prenatal+exome+sequencing+analysis+in+fetal+structural+anomalies+detected+by+ultrasonography+%28PAGE%29%3A+a+cohort+study.&amp;publication_year=2019&amp;journal=Lancet&amp;pages=747-757&amp;doi=10.1016%2FS0140-6736%2818%2931940-8&amp;pmid=30712880" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r11-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r12-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] to determine the cause of disease. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r14-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] fetal structural anomalies (in 10 to 20%). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r14-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] cohorts with severe early-onset diseases. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">12.</div><div id="core-r12" class="citations"><div class="citation"><div class="citation-content">Meng L, Pammi M, Saronwala A, et al. Use of exome sequencing for infants in intensive care units: ascertainment of severe single-gene disorders and effect on medical management. <em>JAMA Pediatr</em> 2017;171(12):e173438-e173438.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r12-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jamapediatrics.2017.3438" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28973083/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000416971000003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Use+of+exome+sequencing+for+infants+in+intensive+care+units%3A+ascertainment+of+severe+single-gene+disorders+and+effect+on+medical+management.&amp;publication_year=2017&amp;journal=JAMA+Pediatr&amp;pages=e173438-e173438&amp;doi=10.1001%2Fjamapediatrics.2017.3438&amp;pmid=28973083" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r12-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r12-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] to determine the cause of disease. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] analysis of stillbirths in clinical care. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">13.</div><div id="core-r13" class="citations"><div class="citation"><div class="citation-content">Petrovski S, Aggarwal V, Giordano JL, et al. Whole-exome sequencing in the evaluation of fetal structural anomalies: a prospective cohort study. <em>Lancet</em> 2019;393:758-767.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r13-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0140-6736(18)32042-7" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30712878/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000459945500026" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Whole-exome+sequencing+in+the+evaluation+of+fetal+structural+anomalies%3A+a+prospective+cohort+study.&amp;publication_year=2019&amp;journal=Lancet&amp;pages=758-767&amp;doi=10.1016%2FS0140-6736%2818%2932042-7&amp;pmid=30712878" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r13-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r14-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] fetal structural anomalies (in 10 to 20%). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r14-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] cohorts with severe early-onset diseases. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">14.</div><div id="core-r14" class="citations"><div class="citation"><div class="citation-content">Wright CF, Fitzgerald TW, Jones WD, et al. Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data. <em>Lancet</em> 2015;385:1305-1314.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r14-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0140-6736(14)61705-0" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25529582/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000352272400036" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genetic+diagnosis+of+developmental+disorders+in+the+DDD+study%3A+a+scalable+analysis+of+genome-wide+research+data.&amp;publication_year=2015&amp;journal=Lancet&amp;pages=1305-1314&amp;doi=10.1016%2FS0140-6736%2814%2961705-0&amp;pmid=25529582" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r14-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r14-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] fetal structural anomalies (in 10 to 20%). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r14-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] cohorts with severe early-onset diseases. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] analysis of stillbirths in clinical care. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">15.</div><div id="core-r15" class="citations"><div class="citation"><div class="citation-content">Shamseldin HE, Kurdi W, Almusafri F, et al. Molecular autopsy in maternal-fetal medicine. <em>Genet Med</em> 2018;20:420-427.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r16-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2017.111" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28749478/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000429912900007" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Molecular+autopsy+in+maternal-fetal+medicine.&amp;publication_year=2018&amp;journal=Genet+Med&amp;pages=420-427&amp;doi=10.1038%2Fgim.2017.111&amp;pmid=28749478" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">16.</div><div id="core-r16" class="citations"><div class="citation"><div class="citation-content">Sahlin E, GrÃ©en A, Gustavsson P, et al. Identification of putative pathogenic single nucleotide variants (SNVs) in genes associated with heart disease in 290 cases of stillbirth. <em>PLoS One</em> 2019;14(1):e0210017-e0210017.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r16-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0210017" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30615648/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000455045900034" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Identification+of+putative+pathogenic+single+nucleotide+variants+%28SNVs%29+in+genes+associated+with+heart+disease+in+290+cases+of+stillbirth.&amp;publication_year=2019&amp;journal=PLoS+One&amp;pages=e0210017-e0210017&amp;doi=10.1371%2Fjournal.pone.0210017&amp;pmid=30615648" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r16-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r16-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] cases or cases with structural anomalies. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r48" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] of stillbirth without structural anomalies. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">17.</div><div id="core-r17" class="citations"><div class="citation"><div class="citation-content">Dudley DJ, Goldenberg R, Conway D, et al. A new system for determining the causes of stillbirth. <em>Obstet Gynecol</em> 2010;116:254-260.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r17"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1097/AOG.0b013e3181e7d975" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20664383/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000280186300003" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+new+system+for+determining+the+causes+of+stillbirth.&amp;publication_year=2010&amp;journal=Obstet+Gynecol&amp;pages=254-260&amp;doi=10.1097%2FAOG.0b013e3181e7d975&amp;pmid=20664383" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">18.</div><div id="core-r18" class="citations"><div class="citation"><div class="citation-content"><em>Analysis tool for annotated variants â a comprehensive platform for population-scale genomic analyses</em>. 2016 (<a href="https://www.biorxiv.org/content/10.1101/2020.06.08.136507v1.full">https://www.biorxiv.org/content/10.1101/2020.06.08.136507v1.full</a>).</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r18"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Analysis+tool+for+annotated+variants+%E2%80%94+a+comprehensive+platform+for+population-scale+genomic+analyses&amp;publication_year=2016" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">19.</div><div id="core-r19" class="citations"><div class="citation"><div class="citation-content">Epi4K Consortium, Epilepsy Phenome/Genome Project. Ultra-rare genetic variation in common epilepsies: a case-control sequencing study. <em>Lancet Neurol</em> 2017;16:135-143.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r19-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S1474-4422(16)30359-3" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28102150/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000391910800012" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Ultra-rare+genetic+variation+in+common+epilepsies%3A+a+case-control+sequencing+study.&amp;publication_year=2017&amp;journal=Lancet+Neurol&amp;pages=135-143&amp;doi=10.1016%2FS1474-4422%2816%2930359-3&amp;pmid=28102150" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r19-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r19-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] described site-based pruning strategy. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r19-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] differed by more than 10 percentage points. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r24-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] stratification (Figs. S3 and S4). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r19-4" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>d [...] neutral ultra-rare synonymous variants. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">20.</div><div id="core-r20" class="citations"><div class="citation"><div class="citation-content">Richards S, Aziz N, Bale S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. <em>Genet Med</em> 2015;17:405-424.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r20"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2015.30" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25741868/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000354096900013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Standards+and+guidelines+for+the+interpretation+of+sequence+variants%3A+a+joint+consensus+recommendation+of+the+American+College+of+Medical+Genetics+and+Genomics+and+the+Association+for+Molecular+Pathology.&amp;publication_year=2015&amp;journal=Genet+Med&amp;pages=405-424&amp;doi=10.1038%2Fgim.2015.30&amp;pmid=25741868" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">21.</div><div id="core-r21" class="citations"><div class="citation"><div class="citation-content">Hamosh A, Scott AF, Amberger J, Valle D, McKusick VA. Online Mendelian Inheritance in Man (OMIM). <em>Hum Mutat</em> 2000;15:57-61.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r21"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1098-1004(200001)15:1%3C57::AID-HUMU12%3E3.0.CO;2-G" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10612823/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000084613900011" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Online+Mendelian+Inheritance+in+Man+%28OMIM%29.&amp;publication_year=2000&amp;journal=Hum+Mutat&amp;pages=57-61&amp;doi=10.1002%2F%28SICI%291098-1004%28200001%2915%3A1%253C57%3A%3AAID-HUMU12%253E3.0.CO%3B2-G&amp;pmid=10612823" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">22.</div><div id="core-r22" class="citations"><div class="citation"><div class="citation-content">ThorvaldsdÃ³ttir H, Robinson JT, Mesirov JP. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. <em>Brief Bioinform</em> 2013;14:178-192.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r22"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bib/bbs017" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22517427/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000316694700006" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Integrative+Genomics+Viewer+%28IGV%29%3A+high-performance+genomics+data+visualization+and+exploration.&amp;publication_year=2013&amp;journal=Brief+Bioinform&amp;pages=178-192&amp;doi=10.1093%2Fbib%2Fbbs017&amp;pmid=22517427" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">23.</div><div id="core-r23" class="citations"><div class="citation"><div class="citation-content">Petrovski S, Todd JL, Durheim MT, et al. An exome sequencing study to assess the role of rare genetic variation in pulmonary fibrosis. <em>Am J Respir Crit Care Med</em> 2017;196:82-93.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r23-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1164/rccm.201610-2088OC" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28099038/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000404473300016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=An+exome+sequencing+study+to+assess+the+role+of+rare+genetic+variation+in+pulmonary+fibrosis.&amp;publication_year=2017&amp;journal=Am+J+Respir+Crit+Care+Med&amp;pages=82-93&amp;doi=10.1164%2Frccm.201610-2088OC&amp;pmid=28099038" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r23-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r24-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] stratification (Figs. S3 and S4). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r33-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] in 337 cases, and the quality was adequate </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">24.</div><div id="core-r24" class="citations"><div class="citation"><div class="citation-content">Zhu X, Padmanabhan R, Copeland B, et al. A case-control collapsing analysis identifies epilepsy genes implicated in trio sequencing studies focused on de novo mutations. <em>PLoS Genet</em> 2017;13(11):e1007104-e1007104.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r24-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1007104" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29186148/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000416836900026" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+case-control+collapsing+analysis+identifies+epilepsy+genes+implicated+in+trio+sequencing+studies+focused+on+de+novo+mutations.&amp;publication_year=2017&amp;journal=PLoS+Genet&amp;pages=e1007104-e1007104&amp;doi=10.1371%2Fjournal.pgen.1007104&amp;pmid=29186148" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r24-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r24-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] stratification (Figs. S3 and S4). </span></a></li><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r24-2"><i aria-hidden="true" class="icon-return"></i><span>b [...] pathogenic variants are found (Panel B). </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">25.</div><div id="core-r25" class="citations"><div class="citation"><div class="citation-content">Davydov EV, Goode DL, Sirota M, Cooper GM, Sidow A, Batzoglou S. Identifying a high fraction of the human genome to be under selective constraint using GERP++. <em>PLoS Comput Biol</em> 2010;6(12):e1001025-e1001025.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r25"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1001025" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21152010/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000285574600013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Identifying+a+high+fraction+of+the+human+genome+to+be+under+selective+constraint+using+GERP%2B%2B.&amp;publication_year=2010&amp;journal=PLoS+Comput+Biol&amp;pages=e1001025-e1001025&amp;doi=10.1371%2Fjournal.pcbi.1001025&amp;pmid=21152010" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">26.</div><div id="core-r26" class="citations"><div class="citation"><div class="citation-content">Petrovski S, Wang Q, Heinzen EL, Allen AS, Goldstein DB. Genic intolerance to functional variation and the interpretation of personal genomes. <em>PLoS Genet</em> 2013;9(8):e1003709-e1003709.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r26"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003709" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23990802/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000323830300059" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Genic+intolerance+to+functional+variation+and+the+interpretation+of+personal+genomes.&amp;publication_year=2013&amp;journal=PLoS+Genet&amp;pages=e1003709-e1003709&amp;doi=10.1371%2Fjournal.pgen.1003709&amp;pmid=23990802" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">27.</div><div id="core-r27" class="citations"><div class="citation"><div class="citation-content">Samocha KE, Robinson EB, Sanders SJ, et al. A framework for the interpretation of de novo mutation in human disease. <em>Nat Genet</em> 2014;46:944-950.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r30"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3050" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25086666/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000341579400007" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+framework+for+the+interpretation+of+de+novo+mutation+in+human+disease.&amp;publication_year=2014&amp;journal=Nat+Genet&amp;pages=944-950&amp;doi=10.1038%2Fng.3050&amp;pmid=25086666" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">28.</div><div id="core-r28" class="citations"><div class="citation"><div class="citation-content">Fuller ZL, Berg JJ, Mostafavi H, Sella G, Przeworski M. Measuring intolerance to mutation in human genetics. <em>Nat Genet</em> 2019;51:772-776.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r30"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0383-1" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30962618/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000466842000004" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Measuring+intolerance+to+mutation+in+human+genetics.&amp;publication_year=2019&amp;journal=Nat+Genet&amp;pages=772-776&amp;doi=10.1038%2Fs41588-019-0383-1&amp;pmid=30962618" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">29.</div><div id="core-r29" class="citations"><div class="citation"><div class="citation-content">Karczewski KJ, Francioli LC, Tiao G, et al. Variation across 141,456 human exomes and genomes reveals the spectrum of loss-of-function intolerance across human protein-coding genes. January 30, 2019 (<a href="https://www.biorxiv.org/content/10.1101/531210v2">https://www.biorxiv.org/content/10.1101/531210v2</a>). preprint.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r29-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Variation+across+141%2C456+human+exomes+and+genomes+reveals+the+spectrum+of+loss-of-function+intolerance+across+human+protein-coding+genes&amp;publication_year=2019" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r29-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r29-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] Genome Aggregation Database, version 2.1.1. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r29-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] Effect Estimator (LOFTEE) annotations, </span></a></li><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r29-3"><i aria-hidden="true" class="icon-return"></i><span>c [...] types of variants, and regulatory regions. </span></a></li><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r29-4"><i aria-hidden="true" class="icon-return"></i><span>d [...] pathogenic variants are found (Panel B). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>e [...] research carried out over the past decade, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">30.</div><div id="core-r30" class="citations"><div class="citation"><div class="citation-content">Lek M, Karczewski KJ, Minikel EV, et al. Analysis of protein-coding genetic variation in 60,706 humans. <em>Nature</em> 2016;536:285-291.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r30"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19057" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000381804900026" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Analysis+of+protein-coding+genetic+variation+in+60%2C706+humans.&amp;publication_year=2016&amp;journal=Nature&amp;pages=285-291&amp;doi=10.1038%2Fnature19057&amp;pmid=27535533" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">31.</div><div id="core-r31" class="citations"><div class="citation"><div class="citation-content">Eilbeck K, Quinlan A, Yandell M. Settling the score: variant prioritization and Mendelian disease. <em>Nat Rev Genet</em> 2017;18:599-612.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r31-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg.2017.52" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28804138/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000410808600002" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Settling+the+score%3A+variant+prioritization+and+Mendelian+disease.&amp;publication_year=2017&amp;journal=Nat+Rev+Genet&amp;pages=599-612&amp;doi=10.1038%2Fnrg.2017.52&amp;pmid=28804138" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r31-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r31-1"><i aria-hidden="true" class="icon-return"></i><span>a [...] pathogenic variants are found (Panel B). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r38" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] a mild form of Cornelia de Lange syndrome, </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">32.</div><div id="core-r32" class="citations"><div class="citation"><div class="citation-content">Bartha I, di Iulio J, Venter JC, Telenti A. Human gene essentiality. <em>Nat Rev Genet</em> 2018;19:51-62.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg.2017.75" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29082913/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000419846000011" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Human+gene+essentiality.&amp;publication_year=2018&amp;journal=Nat+Rev+Genet&amp;pages=51-62&amp;doi=10.1038%2Fnrg.2017.75&amp;pmid=29082913" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">33.</div><div id="core-r33" class="citations"><div class="citation"><div class="citation-content">Gelfman S, Dugger S, de Araujo Martins Moreno C, et al. A new approach for rare variation collapsing on functional protein domains implicates specific genic regions in ALS. <em>Genome Res</em> 2019;29:809-818.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r33-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.243592.118" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30940688/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000466371300009" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+new+approach+for+rare+variation+collapsing+on+functional+protein+domains+implicates+specific+genic+regions+in+ALS.&amp;publication_year=2019&amp;journal=Genome+Res&amp;pages=809-818&amp;doi=10.1101%2Fgr.243592.118&amp;pmid=30940688" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r33-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation open-in-viewer" href="#f1" role="menuitem" data-target="#fv-body-ref-r33-1"><i aria-hidden="true" class="icon-return"></i><span>a [...] pathogenic variants are found (Panel B). </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r33-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] in 337 cases, and the quality was adequate </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">34.</div><div id="core-r34" class="citations"><div class="citation"><div class="citation-content">Kosmicki JA, Samocha KE, Howrigan DP, et al. Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples. <em>Nat Genet</em> 2017;49:504-510.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r34"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000397603700013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Refining+the+role+of+de+novo+protein-truncating+variants+in+neurodevelopmental+disorders+by+using+population+reference+samples.&amp;publication_year=2017&amp;journal=Nat+Genet&amp;pages=504-510&amp;doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">35.</div><div id="core-r35" class="citations"><div class="citation"><div class="citation-content">Li H, Bloom JM, Farjoun Y, et al. A synthetic-diploid benchmark for accurate variant-calling evaluation. <em>Nat Methods</em> 2018;15:595-597.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r35"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0054-7" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30013044/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000440334000018" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=A+synthetic-diploid+benchmark+for+accurate+variant-calling+evaluation.&amp;publication_year=2018&amp;journal=Nat+Methods&amp;pages=595-597&amp;doi=10.1038%2Fs41592-018-0054-7&amp;pmid=30013044" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">36.</div><div id="core-r36" class="citations"><div class="citation"><div class="citation-content">Tartaglia M, Kalidas K, Shaw A, et al. PTPN11 mutations in Noonan syndrome: molecular spectrum, genotype-phenotype correlation, and phenotypic heterogeneity. <em>Am J Hum Genet</em> 2002;70:1555-1563.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r36"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1086/340847" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11992261/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000175445700016" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=PTPN11+mutations+in+Noonan+syndrome%3A+molecular+spectrum%2C+genotype-phenotype+correlation%2C+and+phenotypic+heterogeneity.&amp;publication_year=2002&amp;journal=Am+J+Hum+Genet&amp;pages=1555-1563&amp;doi=10.1086%2F340847&amp;pmid=11992261" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">37.</div><div id="core-r37" class="citations"><div class="citation"><div class="citation-content">Mason-Suares H, Toledo D, Gekas J, et al. Juvenile myelomonocytic leukemia-associated variants are associated with neo-natal lethal Noonan syndrome. <em>Eur J Hum Genet</em> 2017;25:509-511.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r37"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ejhg.2016.202" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28098151/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000395996200020" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Juvenile+myelomonocytic+leukemia-associated+variants+are+associated+with+neo-natal+lethal+Noonan+syndrome.&amp;publication_year=2017&amp;journal=Eur+J+Hum+Genet&amp;pages=509-511&amp;doi=10.1038%2Fejhg.2016.202&amp;pmid=28098151" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">38.</div><div id="core-r38" class="citations"><div class="citation"><div class="citation-content">Gil-RodrÃ­guez MC, Deardorff MA, Ansari M, et al. De novo heterozygous mutations in SMC3 cause a range of Cornelia de Lange syndrome-overlapping phenotypes. <em>Hum Mutat</em> 2015;36:454-462.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r38"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/humu.22761" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25655089/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000352304200010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=De+novo+heterozygous+mutations+in+SMC3+cause+a+range+of+Cornelia+de+Lange+syndrome-overlapping+phenotypes.&amp;publication_year=2015&amp;journal=Hum+Mutat&amp;pages=454-462&amp;doi=10.1002%2Fhumu.22761&amp;pmid=25655089" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">39.</div><div id="core-r39" class="citations"><div class="citation"><div class="citation-content">Park ES, Putnam EA, Chitayat D, Child A, Milewicz DM. Clustering of FBN2 mutations in patients with congenital contractural arachnodactyly indicates an important role of the domains encoded by exons 24 through 34 during human development. <em>Am J Med Genet</em> 1998;78:350-355.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r39"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1096-8628(19980724)78:4%3C350::AID-AJMG9%3E3.0.CO;2-P" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9714438/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000075199200009" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Clustering+of+FBN2+mutations+in+patients+with+congenital+contractural+arachnodactyly+indicates+an+important+role+of+the+domains+encoded+by+exons+24+through+34+during+human+development.&amp;publication_year=1998&amp;journal=Am+J+Med+Genet&amp;pages=350-355&amp;doi=10.1002%2F%28SICI%291096-8628%2819980724%2978%3A4%253C350%3A%3AAID-AJMG9%253E3.0.CO%3B2-P&amp;pmid=9714438" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">40.</div><div id="core-r40" class="citations"><div class="citation"><div class="citation-content">Landrum MJ, Lee JM, Riley GR, et al. ClinVar: public archive of relationships among sequence variation and human phenotype. <em>Nucleic Acids Res</em> 2014;42:D980-D985.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r40-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1113" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24234437/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000331139800144" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=ClinVar%3A+public+archive+of+relationships+among+sequence+variation+and+human+phenotype.&amp;publication_year=2014&amp;journal=Nucleic+Acids+Res&amp;pages=D980-D985&amp;doi=10.1093%2Fnar%2Fgkt1113&amp;pmid=24234437" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r40-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r41" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] been implicated in causing disease </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r40-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] of stillbirths had a known genetic cause. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">41.</div><div id="core-r41" class="citations"><div class="citation"><div class="citation-content">Stenson PD, Mort M, Ball EV, et al. The Human Gene Mutation Database: towards a comprehensive repository of inherited mutation data for medical research, genetic diagnosis and next-generation sequencing studies. <em>Hum Genet</em> 2017;136:665-677.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r41"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-017-1779-6" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28349240/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000401341500001" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=The+Human+Gene+Mutation+Database%3A+towards+a+comprehensive+repository+of+inherited+mutation+data+for+medical+research%2C+genetic+diagnosis+and+next-generation+sequencing+studies.&amp;publication_year=2017&amp;journal=Hum+Genet&amp;pages=665-677&amp;doi=10.1007%2Fs00439-017-1779-6&amp;pmid=28349240" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">42.</div><div id="core-r42" class="citations"><div class="citation"><div class="citation-content">Yates CL, Monaghan KG, Copenheaver D, et al. Whole-exome sequencing on deceased fetuses with ultrasound anomalies: expanding our knowledge of genetic disease during fetal development. <em>Genet Med</em> 2017;19:1171-1178.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r43-1"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2017.31" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28425981/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000412303300013" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Whole-exome+sequencing+on+deceased+fetuses+with+ultrasound+anomalies%3A+expanding+our+knowledge+of+genetic+disease+during+fetal+development.&amp;publication_year=2017&amp;journal=Genet+Med&amp;pages=1171-1178&amp;doi=10.1038%2Fgim.2017.31&amp;pmid=28425981" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">43.</div><div id="core-r43" class="citations"><div class="citation"><div class="citation-content">Zhu X, Petrovski S, Xie P, et al. Whole-exome sequencing in undiagnosed genetic diseases: interpreting 119 trios. <em>Genet Med</em> 2015;17:774-781.</div><div class="external-links"><div class="to-citation__wrapper"><button aria-controls="to-citation__accordion-r43-core" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="Hide Citations" data-collapsed-title="Show Citations" class="accordion__toggle__title"></span></button></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/gim.2014.191" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25590979/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000362441900005" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Whole-exome+sequencing+in+undiagnosed+genetic+diseases%3A+interpreting+119+trios.&amp;publication_year=2015&amp;journal=Genet+Med&amp;pages=774-781&amp;doi=10.1038%2Fgim.2014.191&amp;pmid=25590979" target="_blank">Google Scholar</a></div><div role="menu" aria-label="links" class="to-citation__accordion no-separator" id="to-citation__accordion-r43-core" style="display: none;"><ul role="none"><li role="none"><a class="to-citation" href="#body-ref-r43-1" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>a [...] analysis of stillbirths in clinical care. </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-2" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>b [...] research carried out over the past decade, </span></a></li><li role="none"><a class="to-citation" href="#body-ref-r43-3" role="menuitem"><i aria-hidden="true" class="icon-return"></i><span>c [...] are associated with established diseases. </span></a></li></ul></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">44.</div><div id="core-r44" class="citations"><div class="citation"><div class="citation-content">Liu Z, Li W, Ma X, et al. Essential role of the zinc finger transcription factor Casz1 for mammalian cardiac morphogenesis and development. <em>J Biol Chem</em> 2014;289:29801-29816.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r44"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M114.570416" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25190801/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000344370800026" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Essential+role+of+the+zinc+finger+transcription+factor+Casz1+for+mammalian+cardiac+morphogenesis+and+development.&amp;publication_year=2014&amp;journal=J+Biol+Chem&amp;pages=29801-29816&amp;doi=10.1074%2Fjbc.M114.570416&amp;pmid=25190801" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">45.</div><div id="core-r45" class="citations"><div class="citation"><div class="citation-content">Wu X, Kasper LH, Mantcheva RT, Mantchev GT, Springett MJ, van Deursen JM. Disruption of the FG nucleoporin NUP98 causes selective changes in nuclear pore complex stoichiometry and function. <em>Proc Natl Acad Sci U S A</em> 2001;98:3191-3196.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r45"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.051631598" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11248054/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000167521300048" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Disruption+of+the+FG+nucleoporin+NUP98+causes+selective+changes+in+nuclear+pore+complex+stoichiometry+and+function.&amp;publication_year=2001&amp;journal=Proc+Natl+Acad+Sci+U+S+A&amp;pages=3191-3196&amp;doi=10.1073%2Fpnas.051631598&amp;pmid=11248054" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">46.</div><div id="core-r46" class="citations"><div class="citation"><div class="citation-content">Kasak L, Rull K, SÃµber S, Laan M. Copy number variation profile in the placental and parental genomes of recurrent pregnancy loss families. <em>Sci Rep</em> 2017;7:45327-45327.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r46"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep45327" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28345611/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000397398600001" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Copy+number+variation+profile+in+the+placental+and+parental+genomes+of+recurrent+pregnancy+loss+families.&amp;publication_year=2017&amp;journal=Sci+Rep&amp;pages=45327-45327&amp;doi=10.1038%2Fsrep45327&amp;pmid=28345611" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">47.</div><div id="core-r47" class="citations"><div class="citation"><div class="citation-content">Brownstein CA, Poduri A, Goldstein RD, Holm IA. The genetics of sudden infant death syndrome. In: Duncan JR, Byard RW, eds. <em>SIDS sudden infant and early childhood death: the past, the present and the future.</em> Adelaide, SA, Australia: University of Adelaide Press, 2018.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r48"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.20851/sids-31" target="_blank">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=The+genetics+of+sudden+infant+death+syndrome&amp;publication_year=2018&amp;doi=10.20851%2Fsids-31" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">48.</div><div id="core-r48" class="citations"><div class="citation"><div class="citation-content">Crotti L, Tester DJ, White WM, et al. Long QT syndrome-associated mutations in intrauterine fetal death. <em>JAMA</em> 2013;309:1473-1482.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r48"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jama.2013.3219" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23571586/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000317271700025" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Long+QT+syndrome-associated+mutations+in+intrauterine+fetal+death.&amp;publication_year=2013&amp;journal=JAMA&amp;pages=1473-1482&amp;doi=10.1001%2Fjama.2013.3219&amp;pmid=23571586" target="_blank">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div class="label">49.</div><div id="core-r49" class="citations"><div class="citation"><div class="citation-content">Narula N, Tester DJ, Paulmichl A, Maleszewski JJ, Ackerman MJ. Post-mortem whole exome sequencing with gene-specific analysis for autopsy-negative sudden unexplained death in the young: a case series. <em>Pediatr Cardiol</em> 2015;36:768-778.</div><div class="external-links"><div class="to-citation__wrapper"><a class="to-citation" href="#body-ref-r49"><i aria-hidden="true" class="icon-return"></i><span>Go to Citation</span></a></div><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00246-014-1082-4" target="_blank">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25500949/" target="_blank">PubMed</a></div><div class="core-xlink-isi"><a href="https://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;DestApp=WOS_CPL&amp;UsrCustomerID=5e3815c904498985e796fc91436abd9a&amp;SrcAuth=atyponcel&amp;SrcApp=literatum&amp;DestLinkType=FullRecord&amp;KeyUT=000351553700010" target="_blank">Web of Science</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?title=Post-mortem+whole+exome+sequencing+with+gene-specific+analysis+for+autopsy-negative+sudden+unexplained+death+in+the+young%3A+a+case+series.&amp;publication_year=2015&amp;journal=Pediatr+Cardiol&amp;pages=768-778&amp;doi=10.1007%2Fs00246-014-1082-4&amp;pmid=25500949" target="_blank">Google Scholar</a></div></div></div></div></div></div></div></div><aside data-core-aside="right-rail">



        
        <div id="DTM_Position_MedRectangle" class="ad">
 <!--emptycomment-->
</div>




        
        <div class="ng-page_layout-sidebarSpacing"></div>










    
    
        <div data-widget-def="relatedArticlesWidget" data-widget-id="8206d5cd-cb8c-4ff9-8c46-3d22e0dee215" data-location="recirc_curatedRelated_article">
        



        
        <div class="ng-related-articles"><h3 class="ng-subsection-title"><span class="ng-subsection-title_text">Related articles</span></h3><ul class="ng-related-articles_list"><li class="ng-related-articles_item"><div class="issue-item issue-item_sm"><div class="issue-item_meta"><ul class="issue-item_meta-list clearfix"><li class="issue-item_meta-item"><a href="/browse/nejm-article-type/correspondence" class="issue-item_type">Correspondence</a></li><li class="issue-item_meta-item"><span class="issue-item_date">Dec 30, 2020</span></li></ul></div><div class="issue-item_row"><div class="issue-item_left"><div class="issue-item_content clearfix"><h4 class="issue-item_title"><a href="/doi/full/10.1056/NEJMc2032136" data-interactiontype="article_recirculation_click" class="issue-item_title-link animation-underline">Causal Genetic Variants in Stillbirth</a></h4><div class="issue-item_authors-and-text"></div></div></div></div></div></li><li class="ng-related-articles_item"><div class="issue-item issue-item_sm"><div class="issue-item_meta"><ul class="issue-item_meta-list clearfix"><li class="issue-item_meta-item"><a href="/browse/nejm-article-type/editorial" class="issue-item_type">Editorial</a></li><li class="issue-item_meta-item"><span class="issue-item_date">Aug 12, 2020</span></li></ul></div><div class="issue-item_row"><div class="issue-item_left"><div class="issue-item_content clearfix"><h4 class="issue-item_title"><a href="/doi/full/10.1056/NEJMe2016410" data-interactiontype="article_recirculation_click" class="issue-item_title-link animation-underline">Genomic Insights into Stillbirth</a></h4><div class="issue-item_authors-and-text"><span class="issue-item_authors">M.H. Wojcik</span></div></div></div></div></div></li></ul></div>

        </div>
    





        
        <div class="ng-page_layout-sidebarSpacing"></div>




        
        <div id="nejm_jobs" class="nejm_jobs"> <!-- top red rectangle --><div id="nejm-widget" class="nejm-widget" style="display: block;"><div class="nejm-widget_logo">	<a target="nejm-win" href="https://www.nejmcareercenter.org/?query=fjw">		<img alt="CareerCenter" src="https://apps1.nejmcareercenter.org/images/nejmCCID-JobsWidget.gif" width="140" height="36">	</a></div><div class="nejm-widget_cont"> <!-- content --><div class="nejm-widget_title">PHYSICIAN JOBS</div><div class="nejm-widget_date">June 9, 2025</div><div class="nejm-widget_item"><div><span> Detroit, Michigan</span></div><div><span>Infectious Disease</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/886133/system-associate-med-director-of-epidemiology-and-infection-prevention-and-antimicrobial-stewardship/?query=fjwp&amp;rid=4557">System Associate Med Director of Epidemiology and Infection Prevention and Antimicrobial Stewardship</a></div></div><div class="nejm-widget_item"><div><span> Massachusetts</span></div><div><span>Psychiatry</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/890575/psychiatry/?query=fjwp&amp;rid=980">Psychiatry</a></div></div><div class="nejm-widget_item"><div><span> Valhalla, New York</span></div><div><span>Critical Care Medicine</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/883540/critical-care-nephrologist-westchester-medical-center/?query=fjwf&amp;rid=22466">Critical Care Nephrologist - Westchester Medical Center</a></div></div><div class="nejm-widget_item"><div><span> Spokane, Washington State</span></div><div><span>Neurology</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/882793/general-neurology-physician/?query=fjwf&amp;rid=329661">General Neurology Physician</a></div></div><div class="nejm-widget_item"><div><span> Arizona</span></div><div><span>Obstetrics &amp; Gynecology</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/888974/ob-gyn-3-5-day-work-week-in-phoenix-az/?query=fjwf&amp;rid=5127">OB/GYN - 3.5 Day Work Week in Phoenix, AZ</a></div></div><div class="nejm-widget_item"><div><span> Honolulu, Hawaii</span></div><div><span>Radiology</span></div><div><a target="nejm-job-win" onmouseover="jlho(this);" onmouseout="jlou(this);" href="https://www.nejmcareercenter.org/job/878339/thoracic-radiologist-840k-year-full-benefits-remote-options-available-/?query=fjwf&amp;rid=402813">Thoracic Radiologist : $840K/year + Full Benefits (Remote options available)</a></div></div></div> <!-- content --> <!-- footer --></div> <!-- nejm-widget --></div><script src="https://apps1.nejmcareercenter.org/?width=300&amp;sp=all"></script>




        
        <div class="ng-page_layout-sidebarSpacing"></div>










    
    
        <div data-widget-def="literatumAd" data-widget-id="001e4aa7-b6fe-4b21-9d5b-5bde81227c67" id="ad-article-right-rail-300x250-1">
        



        
        



    
        <div class="pb-house-message adplaceholder exists">
            <a href="/action/clickThrough?id=128743&amp;url=https%3A%2F%2Fstore.nejm.org%2Fsignup%2Fnejm%2Fregister%2Fnotablearticles2024%3Fpromo%3DONFQNAR4%26query%3Dcm-ra%26utm_source%3Dnejm%26utm_medium%3Dcm%26utm_campaign%3Dna24&amp;loc=%2Fdoi%2Ffull%2F10.1056%2FNEJMoa1908753&amp;pubId=41289191&amp;placeholderId=101337&amp;productId=1007"><img src="/sda/128743/rightRail--notable24_001.jpg" width="300" height="250"></a>
            
        </div>
    

    

    



        </div>
    





        
        <div class="ng-page_layout-sidebarSpacing"></div>




        
        <script src="https://widgets.nejm.org/onepub-widgets/loader/webcomponents-loader.js" async="" nonce="94d64844d6a5f20e-SJC">
<script src="https://widgets.nejm.org/onepub-widgets/loader/polyfill-support.js"  nonce="94d64844d6a5f20e-SJC"></script>
<script src="https://widgets.nejm.org/onepub-widgets/more-like-this.js" type="module" nonce="94d64844d6a5f20e-SJC"></script>

<more-like-this class="onepub-widget" dataid="onepub-more-like-this" collections="catalyst-article, clinician-article, evidence-article, nejm-ai-article, nejm-article" headertitle="More Like This" maxitems="5"><!----><div class="more-like-this onepub-more-like-this_"><!--?lit$401702055$--><onepub-fonts><wc-fetch></wc-fetch><!--?lit$401702055$--><div class="mlt-wrapper"><div class="mlt-header-container"><div class="mlt-header-content-left"><div class="mlt-header-title-container"><span class="mlt-header-title">More Like This</span></div><!--?lit$401702055$--></div></div><div class="mlt-body"><!--?lit$401702055$--><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm" href="https://www.nejm.org/browse/nejm-article-type/original-article" target="_self"><!--?lit$401702055$-->Original Article</a> <span class="mlt-article-site-label"><!--?lit$401702055$-->NEJM</span> <span class="mlt-article-pubdate"><!--?lit$401702055$-->Mar 13, 2025</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://www.nejm.org/doi/full/10.1056/NEJMoa2406295?query=recirc_Semantic" target="_self">Outcomes of a Program to Reduce Birth-Related Mortality in Tanzania</a></div><div class="mlt-article-authors"><!--?lit$401702055$-->B.A. Kamala and Others</div></div><!----><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm-evidence" href="https://evidence.nejm.org/browse/nejm-article-type/editorial?query=recirc_Semantic" target="_self"><!--?lit$401702055$-->Editorial</a> <span class="mlt-article-site-label"><!--?lit$401702055$-->NEJM Evidence</span> <span class="mlt-article-pubdate"><!--?lit$401702055$-->Apr 26, 2022</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://evidence.nejm.org/doi/full/10.1056/EVIDe2200074?query=recirc_Semantic" target="_self">Leveraging Artificial Intelligence to Improve Pregnancy Dating in Low-Resource Settings</a></div><div class="mlt-article-authors"><!--?lit$401702055$-->B.J. Wylie and A.C. Lee</div></div><!----><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm" href="https://www.nejm.org/browse/nejm-article-type/perspective" target="_self"><!--?lit$401702055$-->Perspective</a> <span class="mlt-article-site-label"><!--?lit$401702055$-->NEJM</span> <span class="mlt-article-pubdate"><!--?lit$401702055$-->Nov 21, 2024</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://www.nejm.org/doi/full/10.1056/NEJMp2402783?query=recirc_Semantic" target="_self">Saint Didacus, Fetal Death, and the Problem of Dual Loyalty</a></div><div class="mlt-article-authors"><!--?lit$401702055$-->N. Posever</div></div><!----><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm" href="https://www.nejm.org/browse/nejm-article-type/original-article" target="_self"><!--?lit$401702055$-->Original Article</a> <span class="mlt-article-site-label"><!--?lit$401702055$-->NEJM</span> <span class="mlt-article-pubdate"><!--?lit$401702055$-->Mar 30, 2023</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://www.nejm.org/doi/full/10.1056/NEJMoa2212111?query=recirc_Semantic" target="_self">Azithromycin to Prevent Sepsis or Death in Women Planning a Vaginal Birth</a></div><div class="mlt-article-authors"><!--?lit$401702055$-->A.T.N. Tita and Others</div></div><!----><!----><div class="mlt-article-container"><div class="mlt-article-container-header"><a data-location="recirc_Semantic" class="mlt-article-type nejm" href="https://www.nejm.org/browse/nejm-article-type/correspondence" target="_self"><!--?lit$401702055$-->Correspondence</a> <span class="mlt-article-site-label"><!--?lit$401702055$-->NEJM</span> <span class="mlt-article-pubdate"><!--?lit$401702055$-->Nov 28, 2024</span></div><div class="mlt-article-title"><a data-location="recirc_Semantic" href="https://www.nejm.org/doi/full/10.1056/NEJMc2412812?query=recirc_Semantic" target="_self">A Case of Vertical Transmission of Oropouche Virus in Brazil</a></div><div class="mlt-article-authors"><!--?lit$401702055$--></div></div><!----></div><div class="mlt-footer"></div></div></onepub-fonts><style>:host{display:block}@keyframes fadeIn{0%{opacity:0}100%{opacity:1}}.more-like-this{max-width:25rem}.more-like-this *{font-variant-numeric:lining-nums}.mlt-wrapper{animation:fadeIn 1s}.mlt-header-container{border-bottom:#4d4d4d solid .0625rem;border-top:#4d4d4d solid .125rem}.mlt-header-title{color:#1a1a1a;display:block;font-family:var(--sans-serif);font-size:1rem;font-weight:700;letter-spacing:.02rem;line-height:1.5rem;padding-bottom:.94rem;padding-top:.94rem;text-transform:uppercase}.mlt-header-description{color:#333;display:block;font-family:ff-scala-sans-pro;font-size:.75rem;letter-spacing:.05rem;margin-top:-.5rem;padding-bottom:1rem;text-transform:uppercase}.mlt-article-container{border-bottom:#e5e5e5 solid .0625rem;margin-top:1.38rem}.mlt-article-container-header{font-family:var(--sans-serif);font-size:.6875rem;letter-spacing:.04em;line-height:1.125rem;margin:0 0 1rem;text-transform:uppercase}.mlt-article-type{font-weight:700;line-height:1.25rem;letter-spacing:.04rem;text-decoration:none}.mlt-article-type:hover{text-decoration:underline}.nejm{color:#f30}.nejm-ai{color:#006197}.nejm-catalyst{color:#5708a6}.nejm-evidence{color:#69a323}.mlt-article-site-label::after,.mlt-article-type::after{border-right:1px solid #e5e5e5;content:'';display:inline;padding:.1875rem .375rem .125rem 0;margin:0 .375rem 0 0}.mlt-article-pubdate{color:#666;font-style:normal;font-weight:700;line-height:1.25rem;letter-spacing:.04em}.mlt-article-authors{color:#666;font-family:var(--sans-serif);font-size:1rem;font-style:normal;font-weight:400;letter-spacing:normal;line-height:1.5rem;margin-bottom:1.38rem}.mlt-article-site-label{color:#666;font-style:normal;font-weight:700;letter-spacing:.04em}.mlt-article-title{margin-bottom:.5rem}.mlt-article-title>a{background-image:linear-gradient(#f30,#f30);background-position:0 90%;background-repeat:no-repeat;background-size:0 0;color:#1a1a1a;font-family:var(--quadraat-web);font-size:1.25rem;font-style:normal;font-weight:700;letter-spacing:normal;line-height:1.8rem;margin:1.5rem 0 .35rem 0;text-decoration:none;transition:background .25s ease-in-out}.mlt-article-title>a:hover{background-position:0 98%;background-size:100% .125rem;text-decoration:none}.skeleton-card{box-shadow:0 0 4px 1px rgba(0,0,0,.1);margin:1rem 0;padding:1rem}</style></div></more-like-this>




        
        <div><div class="pb-dropzone" data-pb-dropzone="col-0" title="col-0"></div></div>










    
    
        <div data-widget-def="UX3HTMLWidget" data-widget-id="ad06db99-f228-4c92-9ad7-c4ae0fdbdbf1" id="bc-web-rec">
        



        
        <!-- BlueConic TEST empty HTML widget with DIV wrapper with id=bc-web-rec -->

        </div>
    

</aside></article><div data-extent="article-wrapper"><div class="core-container"></div><div class="after-credits"><div class="core-container"><a href="/doi/pdf/10.1056/NEJMoa1908753?download=true" id="downloadPdfUrl" data-doi="10.1056/NEJMoa1908753" data-behavior="trackDownloadEvent" data-interactiontype="article_tools_download_pdf" data-multimedia-type="Article PDF" data-multimedia-filename="nejmoa1908753.pdf"><span>Download PDF</span></a></div></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1056/NEJMoa1908753"}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><div class="articles-nav"></div><template id="figure_nav_template"><nav><a href="#" title="Open in viewer" class="open-in-viewer"><i aria-hidden="true" class="icon-expand"></i></a></nav></template><template id="fv_panel_template" data-sharesocial="Share"><div class="fv__panel js--hidden"><div class="fv__panel__text"><div class="fv__panel__contentTitle"></div><div class="fv__panel__contentText"></div></div></div></template>
<template id="fv_directory_template" data-overlay="View figure"><div class="fv__directory"><div class="fv__header"><nav class="tab__nav"></nav><a href="#" title="Close figure viewer" class="fv__close"><i aria-hidden="true" class="icon-close"></i></a></div><div class="fv__content tab__content"></div></div></template><template id="fv_directory_tabItem_template" data-figures="Figures" data-others="Others" data-tables="Tables"><button role="tab" data-toggle="tab" class="tab__nav__item"></button></template>
<template id="fv_lightbox_template" data-sr-back="Go to figure location within the article" data-sr-closealt="Close" data-sr-download="Toggle download panel" data-sr-download-pptx="Download PPT" data-sr-info="Toggle information panel" data-sr-panel="Close panel" data-sr-share="Toggle share panel" data-sr-zoom="Zoom" data-text-close="Back to article" data-text-nav-figures="All figures" data-text-nav-others="All others" data-text-nav-tables="All tables" data-text-viewall="View all material" data-title-back="Back to article" data-title-closealt="Close" data-title-download="Download" data-title-download-pptx="Download PPT" data-title-info="Info" data-title-share="Share" data-title-zoom="Zoom"></template>
<template id="fv_toolbar_template"><div class="fv__toolbar__info"><strong class="fv__toolbar__contentTitle uppercase"></strong><p class="fv__toolbar__contentText"></p></div></template><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-return"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-return"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-return"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--secondary"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-references="References" data-figure="Go to Figure" data-media="Go to Media" data-more="More" data-original="Go to original" data-table="Go to Table" data-inviewer="Open in Viewer" data-allinviewer="Open all in viewer"><svg data-tags="copyright" viewBox="0 0 24 24"><path d="M12 20.016q3.281 0 5.648-2.368T20.016 12t-2.368-5.648T12 3.984 6.352 6.352 3.984 12t2.368 5.648T12 20.016zm0-18q4.125 0 7.055 2.93T21.985 12t-2.93 7.055T12 21.985t-7.055-2.93T2.015 12t2.93-7.055T12 2.015zm-.14 7.125q-1.876 0-1.876 2.718v.282q0 2.718 1.875 2.718.704 0 1.172-.398t.469-1.008h1.781q0 1.172-1.031 2.063-.984.843-2.39.843-1.876 0-2.86-1.125t-.984-3.093v-.282q0-1.921.937-3 1.125-1.265 2.906-1.265 1.547 0 2.438.89.984.985.984 2.297H13.5q0-.328-.14-.61-.235-.468-.329-.562-.469-.468-1.172-.468z"></path></svg><span>Request permissions</span><i aria-hidden="true" class="icon-open_in_new"></i></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template>

        <div role="navigation" aria-label="Sticky Navigation" class="st-header"><div class="st-header__content"><div class="st-header__item st-header__menu"></div><div class="st-header__current st-header__item"><div class="st-header__label">Now Reading:</div><div class="st-header__title">Causal Genetic Variants in Stillbirth</div></div><div class="st-header__share st-header__item"><div class="share-dropblock dropdown"><a id="sticky-header-dropBlock" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="dropdown-toggle"><span title="" data-toggle="tooltip" data-trigger="hover" data-original-title="Share options"><span class="sr-only">Share</span><i aria-hidden="true" class="icon-share"></i></span></a><div aria-labelledby="sticky-header-dropBlock" data-location="share_tools_article" class="dropdown-menu"><ul class="rlist--inline a2a a2a_kit" style="line-height: 16px;"><li><a href="/#facebook" aria-label="Share on Facebook" data-id="article-share-facebook" rel="nofollow noopener" role="link" target="_blank" title="Share on Facebook" class="btn btn--facebook a2a_button_facebook" data-interactiontype="social"><i aria-hidden="true" class="icon-facebook"></i></a></li><li><a href="/#twitter" aria-label="Share on X (formerly Twitter)" data-id="article-share-twitter" rel="nofollow noopener" role="link" target="_blank" title="Share on X (formerly Twitter)" class="btn btn--twitter a2a_button_twitter" data-interactiontype="social"><i aria-hidden="true" class="icon-twitter"></i></a></li><li><a href="/#linkedin" aria-label="Share on LinkedIn" data-id="article-share-linkedin" rel="nofollow noopener" role="link" target="_blank" title="Share on LinkedIn" class="btn btn--linkedin a2a_button_linkedin" data-interactiontype="social"><i aria-hidden="true" class="icon-linkedin"></i></a></li><li><a href="/#email" aria-label="Share on email" data-id="article-share-email" rel="nofollow noopener" role="link" target="_blank" title="Share on email" class="btn btn--email a2a_button_email" data-interactiontype="content_email_click"><i aria-hidden="true" class="icon-mail"></i></a></li><li><a href="/#bluesky" aria-label="Share on Bluesky" data-id="article-share-bluesky" rel="nofollow noopener" role="link" target="_blank" title="Share on Bluesky" class="btn btn--bluesky a2a_button_bluesky" data-interactiontype="social"><i aria-hidden="true" class="icon-bluesky"></i></a></li></ul></div></div></div><div class="st-header__nav st-header__item"><div class="content-navigation"><a href="/doi/full/10.1056/NEJMp2023616" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><div aria-hidden="true" class="content-navigation__hint"><div class="content-navigation__hint__content"><h6>PREVIOUS ARTICLE</h6><div>Structural Racism, Social Risk Factors, and Covid-19 â A Dangerous Convergence for Black Americans</div></div></div><i aria-hidden="true" class="icon-arrow-left"></i><span>Previous</span></a><a href="/doi/full/10.1056/NEJMoa2001180" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><div aria-hidden="true" class="content-navigation__hint"><div class="content-navigation__hint__content"><h6>NEXT ARTICLE</h6><div>Two-Year, Randomized, Controlled Trial of Belimumab in Lupus Nephritis</div></div></div><span>Next</span><i aria-hidden="true" class="icon-arrow-right"></i></a></div></div></div><div class="st-header__track"><div id="progress-tracker" class="st-header__tracker" style="width: 0px;"></div></div></div><div class="fv figureViewer js--has-focus-mode"><div class="fv__directory" data-core-tabs="default" style="display: flex;"><div class="fv__header"><nav class="tab__nav" role="tablist"><button role="tab" data-toggle="tab" class="tab__nav__item active" aria-controls="#tab-pane-figures" aria-selected="true">Figures</button><button role="tab" data-toggle="tab" class="tab__nav__item inactive" aria-controls="#tab-pane-tables" tabindex="-1" aria-selected="false" aria-disabled="true">Tables</button></nav><a href="#" title="Close figure viewer" class="fv__close"><i aria-hidden="true" class="icon-close"></i></a></div><div class="fv__content tab__content"><div class="tab__pane fv__list fv__list--figures" id="tab-pane-figures" aria-label="All figures" tabindex="0" role="tabpanel"><div class="fv__item"><a class="fv__open-item fv__item__view fv__fit" href="#fv-f1" data-type="figures" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f1.jpg"><img src="/cms/10.1056/NEJMoa1908753/asset/fc1d9fab-8597-4cb8-b3b1-b9c2a5e17ea8/assets/images/large/nejmoa1908753_f1.jpg" height="2347" width="2640" aria-labelledby="f1" loading="lazy"><span class="fv__fit--overlay"><span>View figure</span></span></a><div class="fv__item__title">Figure 1</div><div class="fv__item__description"><figcaption><div class="caption">Intolerance to Genetic Variation.</div><div class="notes"><div role="doc-footnote">Regions of the human genome that are under the strongest natural selection are the most likely to cause disease when they are variant. Until 2013, the primary approach that was used to identify such regions relied on the genetic similarities of different species.<sup><a href="#core-r25" role="doc-biblioref" data-xml-rid="r25" id="fv-body-ref-r25" href-manipulated="true" aria-label="Reference 25" data-to-manipulate="false">25</a></sup> Genomic regions under selection show fewer DNA changes (e.g., nucleotide substitutions, deletions, or insertions) across species and are likely to be functionally important (Panel A). Pathogenic variants that cause human diseases have long been shown to fall preferentially within these âconstrainedâ regions. Although this approach is useful, it cannot identify genomic regions of particular importance in humans, as might happen because of the evolution of a novel function. In 2013, a new framework was developed to address this limitation: variation, solely within the human population, was used to identify genes with less functional variation than expected according to genomewide averages. Genes with a depletion of human variation are termed âintolerantâ<sup><a href="#core-r26" role="doc-biblioref" data-xml-rid="r26" id="fv-body-ref-r26" href-manipulated="true" aria-label="Reference 26" data-to-manipulate="false">26</a></sup> and reflect parts of the genome under strong selection specifically in humans. Since the introduction of intolerance scoring, there have been a number of important elaborations focused on regions of genes, specific types of variants, and regulatory regions.<sup><a href="#core-r27" role="doc-biblioref" data-xml-rid="r27 r28 r29 r30" id="fv-body-ref-r30" href-manipulated="true" data-to-manipulate="false">27-30</a></sup> Intolerance scores have now been shown to provide independent information about where in the human genome pathogenic variants are found (Panel B).<sup><a href="#core-r24" role="doc-biblioref" data-xml-rid="r24 r29 r31 r32 r33 r34" id="fv-body-ref-r34" href-manipulated="true" data-to-manipulate="false">24,29,31â34</a></sup> For example, the gene encoding olfactory receptor 51E1 (<i>OR51E1</i>) is tolerant to variation and does not cause disease, whereas <i>MYBPC3</i> and <i>HNF1B</i> are intolerant and are known to cause cardiac and kidney disease, respectively. <i>CASZ1</i> is highly intolerant to variation in the healthy population, but no pathogenic variants have been reported in the literature regarding postnatal disease, which indicates that it may result in lethality when variant in utero.</div></div></figcaption></div></div><div class="fv__item"><a class="fv__open-item fv__item__view fv__fit" href="#fv-f2" data-type="figures" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f2.jpg"><img src="/cms/10.1056/NEJMoa1908753/asset/df562d50-26f1-4f15-9edf-50bc1559b2c3/assets/images/large/nejmoa1908753_f2.jpg" height="1866" width="1656" aria-labelledby="f2" loading="lazy"><span class="fv__fit--overlay"><span>View figure</span></span></a><div class="fv__item__title">Figure 2</div><div class="fv__item__description"><figcaption><div class="caption">Enrichment of Loss-of-Function Variants in Stillbirth Cases, as Compared with Controls.</div><div class="notes"><div role="doc-footnote">To explore which genetic variants may be associated with stillbirth, we assigned an LOEUF (loss-of-function observed-to-expected upper boundary fraction) value to all 8081 genes with a loss-of-function variant in 241 case samples and 7239 control samples. The process resulted in 1825 unique LOEUF values. The LOEUF value is the upper boundary of a Poisson-derived confidence interval of the observed-to-expected ratio. A low LOEUF score indicates a depletion of loss-of-function variation (also referred to as âintoleranceâ to loss-of-function variation). The enrichment of loss-of-function variants was then assessed by comparing the count of variants in cases and controls in each of these 1825-gene groupings. Enrichment was evaluated with the use of a logistic-regression model that evaluated caseâcontrol status regarding the presence or absence of a loss-of-function variant in each gene grouping. Enrichment of presumed neutral synonymous variants at each intolerance threshold was used as a covariate to control for differences between case and control genomes that are unrelated to disease risk. The negative log of the enrichment statistic (unadjusted P value) at each LOEUF threshold is shown on the y axis. The x axis reflects the LOEUF threshold for each gene grouping. The minimum observed P value, which indicates the most significant enrichment across all LOEUF thresholds, occurs at a LOEUF threshold of 0.239 or less. The significance of the observed minimum P value was then assessed by permutation (number of permutations, 10,000).</div></div></figcaption></div></div><div class="fv__item"><a class="fv__open-item fv__item__view fv__fit" href="#fv-f3" data-type="figures" data-interactiontype="multimedia_click" data-multimedia-type="Figure" data-multimedia-filename="nejmoa1908753_f3.jpg"><img src="/cms/10.1056/NEJMoa1908753/asset/db99f9fb-56d6-444f-9808-967ca1cae228/assets/images/large/nejmoa1908753_f3.jpg" height="1226" width="2640" aria-labelledby="f3" loading="lazy"><span class="fv__fit--overlay"><span>View figure</span></span></a><div class="fv__item__title">Figure 3</div><div class="fv__item__description"><figcaption><div class="caption">Loss-of-Function Signal in Intolerant Genes in Three Clinical Cohorts.</div><div class="notes"><div role="doc-footnote">In this study, the LOEUF threshold that had the strongest enrichment of loss-of-function variants (â¤0.239) in the 241 cases in the stillbirth cohort was used to make point comparisons with the loss-of-function signal in a group of 251 live-birth cases with fetal anomalies and in 589 cases with postnatal disease. The odds ratio indicates the enrichment of loss-of-function variants in the gene set in cases as compared with a large group of controls. The gene sets are shown in three categories: all the genes below the LOEUF threshold (â¤0.239), those that had an existing disease association in the Online Mendelian Inheritance in Man (OMIM) database, and those that did not have such an association (non-OMIM).</div></div></figcaption></div></div></div><div class="tab__pane fv__list fv__list--tables" id="tab-pane-tables" aria-label="All tables" tabindex="-1" role="tabpanel"></div></div></div></div><div class="references-pop-up fade" aria-hidden="true"><div class="references-pop-up__heading text-dark d-flex align-items-center"><div class="references-pop-up__title flex-grow-1">Reference <span class="references-pop-up__heading-number">1</span></div><a class="references-pop-up__close text-dark" href="#" title="close pop-up" tabindex="-1" aria-hidden="true"><i class="icon-close" aria-hidden="true"></i></a></div><div class="references-pop-up__items citations to-citation__accordion external-links"></div></div></main>
    

</div>

                    
                        



        
        <div class="ng-page_layout ng-page_layout-fullWidth"><div class="container-fluid"><div class="row"><div class="col-12">



        
        




        
        <div class="ng-page_layout-contentSpacing"></div>










    
    
        <div data-widget-def="oneSearchMultiSearch" data-widget-id="778b3ad8-1a8d-43d8-8196-63b03df4a138" data-location="recirc_inIssue_bottom_article">
        



        
        <div class="os-ms os-ms-more-from-issue"><h3 class="ng-subsection-title"><a href="/toc/nejm/383/12" class="ng-subsection-title_link animation-icon-shift"><span class="ng-subsection-title_text">More from Vol. 383 No. 12</span><span class="ng-subsection-title_icon"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-longArrowRight"></use></svg></span></span></a></h3><ul class="os-ms-more-from-issue_list"><li class="os-ms-more-from-issue_item"><div class="issue-item issue-item_sm"><div class="issue-item_meta"><ul class="issue-item_meta-list clearfix"><li class="issue-item_meta-item"><a href="/browse/nejm-article-type/original-article" class="issue-item_type">Original Article</a></li><li class="issue-item_meta-item"><span class="issue-item_date">Sep 17, 2020</span></li><li class="issue-item_meta-item"><span class="issue-item_free">FREE</span></li></ul></div><div class="issue-item_row"><div class="issue-item_left"><div class="issue-item_content clearfix"><h4 class="issue-item_title"><a href="/doi/full/10.1056/NEJMoa2023325" data-interactiontype="article_recirculation_click" class="issue-item_title-link animation-underline">Brief Report: Targeting CD38 with Daratumumab in Refractory Systemic Lupus Erythematosus</a></h4><div class="issue-item_authors-and-text"><span class="issue-item_authors">L. Ostendorf and Others</span></div></div></div></div></div></li><li class="os-ms-more-from-issue_item"><div class="issue-item issue-item_sm"><div class="issue-item_meta"><ul class="issue-item_meta-list clearfix"><li class="issue-item_meta-item"><a href="/browse/nejm-article-type/original-article" class="issue-item_type">Original Article</a></li><li class="issue-item_meta-item"><span class="issue-item_date">Sep 17, 2020</span></li><li class="issue-item_meta-item"><span class="issue-item_free">FREE</span></li></ul></div><div class="issue-item_row"><div class="issue-item_left"><div class="issue-item_content clearfix"><h4 class="issue-item_title"><a href="/doi/full/10.1056/NEJMoa2001180" data-interactiontype="article_recirculation_click" class="issue-item_title-link animation-underline">Two-Year, Randomized, Controlled Trial of Belimumab in Lupus Nephritis</a></h4><div class="issue-item_authors-and-text"><span class="issue-item_authors">R. Furie and Others</span></div></div></div></div></div></li><li class="os-ms-more-from-issue_item"><div class="issue-item issue-item_sm"><div class="issue-item_meta"><ul class="issue-item_meta-list clearfix"><li class="issue-item_meta-item"><a href="/browse/nejm-article-type/original-article" class="issue-item_type">Original Article</a></li><li class="issue-item_meta-item"><span class="issue-item_date">Sep 17, 2020</span></li><li data-toggle="tooltip" data-original-title="Video" class="issue-item_meta-item"><span role="img" aria-label="Video" tabindex="0" class="issue-item_video"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#tooltip-video"></use></svg></span></span></li><li class="issue-item_meta-item"><span class="issue-item_free">FREE</span></li></ul></div><div class="issue-item_row"><div class="issue-item_left"><div class="issue-item_content clearfix"><h4 class="issue-item_title"><a href="/doi/full/10.1056/NEJMoa2013219" data-interactiontype="article_recirculation_click" class="issue-item_title-link animation-underline">Randomized Trial of Nocturnal Oxygen in Chronic Obstructive Pulmonary Disease</a></h4><div class="issue-item_authors-and-text"><span class="issue-item_authors">Y. Lacasse and Others</span></div></div></div></div></div></li></ul></div>

        </div>
    

</div></div></div></div>

                    
                        



        
        <div class="ng-page_layout-bottomAd"><div class="container-fluid"><div class="row"><div class="col-12">









    
    
        <div data-widget-def="literatumAd" data-widget-id="7e022d0d-4a48-44a7-af66-033cf5c13eb8" id="ad-article-bottom-FULLx320-1">
        



        
        



    
        <div class="pb-house-message adplaceholder exists">
            <!-- START: Circulation Footer 3 -->
<aside class="g-cta-subscribe">
	<style>
.bads_footer-banner24-ONJQNRF1{background:url(/pb-assets/contextual_messaging/footer-yel-background-2x-1513629627453.jpg) center center no-repeat;padding:20px;display:block;margin:0;min-height:inherit;background-size:cover;text-decoration:none!important;color:#1a1a1a;text-align:left}.bads_footer-banner24-ONJQNRF1 .g-cta-subscribe__title{font-size:42px;line-height:44px;letter-spacing:.2px;margin:0 0 20px;font-family:ff-quadraat-web-pro,sans-serif;font-weight:600}.bads_footer-banner24-ONJQNRF1-inner{max-width:1300px;margin:0 auto}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__actions .a-btn{width:130px;padding-left:12px}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__login{position:static;color:#000;font-size:18px;line-height:27px;border:0}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__actions{margin:20px 0 20px 0;list-style-type:none;padding:0}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__login a{color:#0a68ad}.bads_clearfix:after{visibility:hidden;display:block;font-size:0;content:" ";clear:both;height:0}.bads_clearfix{display:block}.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{float:left;width:50%;padding:30px 0}.bads_footer-banner24-ONJQNRF1-inner .bads_device-img{float:left;padding-left:30px;width:50%}.bads_footer-banner24-ONJQNRF1 .bads_device-img>img{display:block;max-width:100%;height:auto}.bads_footer-banner24-ONJQNRF1 .bads_h5{font-size:18px;text-transform:uppercase;line-height:24px;font-weight:600;margin:-15px 0 0;letter-spacing:.2px}@media only screen and (max-width:480px){.bads_footer-banner24-ONJQNRF1-inner .bads_device-img,.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{width:100%;float:none}.bads_footer-banner24-ONJQNRF1-inner .bads_device-img{padding-left:0}.bads_footer-banner24-ONJQNRF1 .g-cta-subscribe__title{font-size:20px;line-height:27px;margin-bottom:10px}.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{padding-top:0;padding-bottom:0}.bads_footer-banner24-ONJQNRF1{height:auto}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__actions{margin:0 0 12px;list-style-type:none}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__login{line-height:25px;font-size:15px;margin-bottom:10px}.bads_footer-banner24-ONJQNRF1 .bads_device-img{width:280px!important;margin:0 auto}.bads_footer-banner24-ONJQNRF1 .bads_h5{margin:0 0 3px;font-size:14px}}@media only screen and (min-width:480px) and (max-width:1024px){.bads_footer-banner24-ONJQNRF1 .g-cta-subscribe__title{font-size:20px;line-height:30px;margin-bottom:15px}.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{padding:0}.bads_footer-banner24-ONJQNRF1-inner .g-cta-subscribe__login{font-size:14px}.bads_footer-banner24-ONJQNRF1 .bads_h5{font-size:13px;margin-top:0}}@media only screen and (min-width:1025px) and (max-width:1200px){.bads_footer-banner24-ONJQNRF1-inner .bads_subs-btn{padding:10px 0}.bads_footer-banner24-ONJQNRF1 .bads_h5{margin:0 0 5px}}@media only screen and (min-width:1401px){.bads_footer-banner24-ONJQNRF1 .bads_device-img>img{float:right}}
	</style>
	<div class="bads_footer-banner24-ONJQNRF1">
	  <div class="bads_footer-banner24-ONJQNRF1-inner bads_clearfix">
		 <div class="bads_subs-btn">
			<p class="g-cta-subscribe__title">Tap into groundbreaking research and clinically relevant insights</p>
			<ul class="g-cta-subscribe__actions">
				<li class="g-cta-subscribe__actions-primary">
				  <a href="/action/clickThrough?id=127716&amp;url=%2Faction%2FstoreProxy%3Faction%3Dsubscribe%26url%3Dhttps%253A%252F%252Fwww.nejm.org%252Fdoi%252Ffull%252F10.1056%252FNEJMoa1908753%26promo%3DONFQNRF1&amp;loc=%2Fdoi%2Ffull%2F10.1056%2FNEJMoa1908753&amp;pubId=41289191&amp;placeholderId=101336&amp;productId=1035" class="ng-btn_primary ng-btn_iconRight"><span class="ng-btn_text">Subscribe </span><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#arrows-longArrowRight"></use></svg></span></a>
				</li>
			</ul>
			<p class="g-cta-subscribe__login">Already a subscriber? <a href="/action/clickThrough?id=127716&amp;url=%2Faction%2FstoreProxy%3Faction%3Dactivate%26url%3Dhttps%253A%252F%252Fwww.nejm.org%252Fdoi%252Ffull%252F10.1056%252FNEJMoa1908753%26promo%3DONFQANR4&amp;loc=%2Fdoi%2Ffull%2F10.1056%2FNEJMoa1908753&amp;pubId=41289191&amp;placeholderId=101336&amp;productId=1035">Activate</a> your online access.</p>
		 </div>

		<div class="bads_device-img">
		   <img src="/sda/127716/footer-devices-2x.png" alt="Devices">
		</div>
	   </div>
	</div>
</aside>
<!--/END: Circulation Footer 3 -->
            
        </div>
    

    

    



        </div>
    

</div></div></div></div>

                    
                        



        
        <footer class="ng-footer footer__top" style="position: relative;"><div data-location="footer" class="ng-footer_columns"><div class="container-fluid"><div class="row"><div class="col-lg-4 col-md-6 col-sm-12 ng-footer_column"><div class="row"><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title">ARTICLE CATEGORIES</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item">
        <a href="/browse/nejm-article-category/research" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Research</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/review" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Reviews</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/clinical-cases" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Clinical Cases</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/perspective" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Perspective</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/commentary" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Commentary</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/browse/nejm-article-category/other" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Other</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/medical-article-index" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Browse all Articles</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/toc/nejm/current" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Current Issue</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/loi/nejm" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Issue Index</span></a></li>
</ul>
</div><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title">RESOURCES</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item"><a href="/author-center/home" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Authors &amp; Reviewers</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/author-center/home" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Submit a Manuscript</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/products-and-services" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Subscribers</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://tools.ovid.com/ovidtools/nejm.html" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Institutional Administrators</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/media" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Media</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/how-to-advertise" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Advertisers</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/information-for-subscription-agents" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Agents</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/permissions" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Permissions &amp; Licensing</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/reprints" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Reprints</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://www.nejmcareercenter.org" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">NEJM CareerCenter</span></a>
    </li>
</ul>
</div></div></div><div class="col-lg-4 col-md-6 col-sm-12 ng-footer_column"><div class="row"><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title" aria-label="About us">ABOUT US</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item"><a href="/about-nejm/about-nejm" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">About NEJM</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://www.nejmgroup.org/" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">NEJM Group</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/products-and-services" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Products &amp; Services</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/editors-and-publishers" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Editors &amp; Publishers</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/advertising-policies" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Advertising Policies</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/contact-nejm" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text" aria-label="Contact us">Contact Us</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/frequently-asked-questions?#Accessibility" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Accessibility</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/frequently-asked-questions" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">FAQs</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/help" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Help</span></a></li>
    <li class="ng-footer_column-list-item"><a href="mailto:sitefeedback@nejm.org" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Site Feedback</span></a></li>
</ul>
</div><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title">SUBSCRIPTIONS</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=subscribe&amp;promo=ONF4NRS3" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Subscribe</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=renew&amp;product=nejm&amp;promo=ONFLNRR3&amp;prc=ONFARN52" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Renew</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=activate&amp;promo=ONFLNAA1" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Activate Subscription</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC4" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Create Account</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://myaccount.nejm.org" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Manage Account</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=paybill&amp;product=nejm&amp;promo=BNFLPBA1" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Pay Bill</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://www.wolterskluwer.com/en/solutions/ovid/nejm-complete-collection" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Institutional Sales</span></a></li>    
    <li class="ng-footer_column-list-item"><a href="https://myaccount.nejm.org/special-content" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Special Content</span></a></li>
</ul>
</div></div></div><div class="col-lg-4 col-md-12 ng-footer_column"><div class="row"><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title">STAY CONNECTED</h4>
<ul class="ng-footer_column-list">
    <li class="ng-footer_column-list-item"><a href="https://myaccount.nejm.org/emails?section=NEJM" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Email Alerts</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/action/storeProxy?action=register&amp;promo=ONFLNRC3" class="ng-footer_column-link animation-underline ro-mode_disabled"><span class="ng-footer_column-link-text">Create Account</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/products-and-services" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Apps</span></a></li>
    <li class="ng-footer_column-list-item"><a href="https://www.nejmcareercenter.org" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">NEJM CareerCenter</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/rss-feed/" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Podcasts</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/rss-feed/" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">RSS Feed</span></a></li>
    <li class="ng-footer_column-list-item"><a href="/about-nejm/institutional-access" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-text">Remote Access</span></a></li>
</ul>
</div><div class="col-6 ng-footer_column-nested">



        
        <h4 class="ng-footer_column-title" aria-label="Follow us">FOLLOW US</h4>
<ul class="ng-footer_column-list">
<li class="ng-footer_column-list-item"><a href="https://www.facebook.com/TheNewEnglandJournalofMedicine" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-facebook"></use></svg></span></span><span class="ng-footer_column-link-text">Facebook</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://twitter.com/nejm" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-twitter"></use></svg></span></span><span class="ng-footer_column-link-text">X (formerly Twitter)</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://www.instagram.com/nejm/" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-instagram"></use></svg></span></span><span class="ng-footer_column-link-text">Instagram</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://www.youtube.com/user/NEJMvideo" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-youtube"></use></svg></span></span><span class="ng-footer_column-link-text">Youtube</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://www.linkedin.com/company/nejm-group" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-linkedin"></use></svg></span></span><span class="ng-footer_column-link-text">LinkedIn</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://bsky.app/profile/nejm.org" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-bluesky"></use></svg></span></span><span class="ng-footer_column-link-text">Bluesky</span></a></li>
<li class="ng-footer_column-list-item"><a href="https://www.tiktok.com/@nejm.org" target="_blank" rel="noopener" class="ng-footer_column-link animation-underline"><span class="ng-footer_column-link-icon"><span class="icon-component"><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#social-tiktok"></use></svg></span></span><span class="ng-footer_column-link-text">TikTok</span></a></li>
</ul>
</div></div></div></div></div></div><div data-location="footer" class="ng-footer_journals"><div class="container-fluid"><div class="row"><div class="col-12">



        
        <h4 class="ng-footer_journals-label">JOURNALS</h4>
<ul class="ng-footer_journals-list">
<li class="ng-footer_journals-item"><a href="https://www.nejm.org/" target="_blank" rel="noopener" class="ng-footer_journals-link">The New England Journal of Medicine</a></li>
<li class="ng-footer_journals-item"><a href="https://catalyst.nejm.org/" target="_blank" rel="noopener" class="ng-footer_journals-link">NEJM Catalyst Innovations in Care Delivery</a></li>
<li class="ng-footer_journals-item"><a href="https://evidence.nejm.org/" target="_blank" rel="noopener" class="ng-footer_journals-link">NEJM Evidence</a></li>
<li class="ng-footer_journals-item"><a href="https://ai.nejm.org/" target="_blank" rel="noopener" class="ng-footer_journals-link">NEJM AI</a></li>
</ul>
</div></div></div></div><div data-location="legal" class="ng-footer_bottom"><div class="container-fluid"><div class="row"><div class="col-lg-7 col-md-6 ng-footer_bottom-left">



        
        <p>Copyright Â© 2025 <a href="https://www.massmed.org/" target="_blank" rel="noopener" class="ng-footer_bottom-link"><b>Massachusetts Medical Society</b></a>. All rights reserved, including those for text and data mining, AI training, and similar technologies. Electronic ISSN 1533-4406. Print ISSN 0028-4793.The content of this site is intended for health care professionals.</p>
</div><div class="col-lg-5 col-md-6 ng-footer_bottom-right">



        
        <style>
  #teconsent a {
    color: #666;
    text-decoration: none;
  }
  #teconsent a:hover {
    color: #666;
    text-decoration: underline;
  }
</style>
<ul class="ng-footer_bottom-list">
<li class="ng-footer_bottom-list-item"><a href="https://www.nejmgroup.org/legal/copyright-information.htm" target="_blank" rel="noopener" class="ng-footer_bottom-link">Copyright</a></li>
<li class="ng-footer_bottom-list-item"><a href="https://www.nejmgroup.org/legal/terms-of-use.htm" target="_blank" rel="noopener" class="ng-footer_bottom-link">Terms</a></li>
<li class="ng-footer_bottom-list-item"><a href="https://www.nejmgroup.org/legal/privacy-policy.htm" target="_blank" rel="noopener" class="ng-footer_bottom-link">Privacy Policy</a></li>
<li class="ng-footer_bottom-list-item"><span id="teconsent" consent="undefined" aria-label="Open Cookie Preferences Modal" role="complementary"><a role="link" id="icon-id039857000459121084" tabindex="0" lang="en" aria-haspopup="dialog" aria-label="Cookie Preferences, opens a dedicated popup modal window" class="truste_cursor_pointer">Cookie Preferences</a></span></li>
</ul>
<a href="https://www.nejmgroup.org/" title="NEJM Group" target="_blank" rel="noopener" class="ng-footer_logo-link"><img src="/specs/products/mms-nextgen/mms/releasedAssets/images/other-images/logo_group.svg" alt="NEJM Group logo" class="ng-footer_logo" loading="lazy" width="103" height="40"></a>
</div></div></div></div><div data-show-after="400" class="back-to-top back-to-top--align-right  "><button data-snap="footer__top" class="back-to-top__action position-relative p-0 m-0 border-0 d-flex flex-column align-items-center justify-content-center"><svg viewBox="0 0 36 36" class="position-relative"><path d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831" class="back-to-top__action__ring-bg"></path><path d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831" stroke-dasharray="0, 100" class="back-to-top__action__ring"></path></svg><span class="back-to-top__action-icon position-absolute d-flex"><span class="icon-component "><svg aria-hidden="true" style=""><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons-3acb348e68a3edd0c676698e8d598c68.spritemap.svg#arrows-up"></use></svg></span></span><span class="sr-only">Back to top</span></button></div></footer>

                    
                        



        
        <!-- TrustArc: Banner -->
<div id="consent_blackbar"><style>  #consent_blackbar{    position:relative;    width:100%;    z-index: 999999;  }    #trustarc-banner-overlay {  background-color: rgb(0, 0, 0);   opacity: 0.3;   position: fixed;   z-index: 1000000;   width: 100%;  height: 100%;   top: 0px;   left: 0px;   overflow: hidden;}.truste-title {  /*font-family: "Source Sans Pro", sans-serif;  font-size: 23px;  font-style: normal;  font-stretch: normal;*/  color: #222;  margin-bottom: 20px;  text-align: center;/*  font-weight:bold;*/}.truste-footer {  margin: 0 auto;      }.truste-left {  margin: 0px 0px 15px 15px;/*font-family: "Source Sans Pro", sans-serif;  font-size: 14px;    */}.truste-banner { margin: 0 auto; }.truste-messageColumn {/*font-family: "Source Sans Pro", sans-serif;  font-size: 14px;*/  color: #666666;  margin: 0px;}#truste-consent-button, #truste-consent-required {    background-color: #FF1E00;    color: #F2F1F1;    padding: 5px 10px;    border: 1px solid #FF1E00;    border-radius: 0px;    cursor: pointer;    text-transform:uppercase;/*    font-family: "Source Sans Pro",sans-serif;    font-size: 15px;*/  }      #truste-show-consent {    background-color: #FF1E00;    color: #F2F1F1;    border: 1px solid #FF1E00;    padding: 5px 10px;    border-radius: 0px;    cursor: pointer;    text-transform:uppercase;/*    font-family: "Source Sans Pro",sans-serif;    font-size: 15px;*/  }div.truste-consent-track-class {  position: fixed;  z-index:9999999;   background: #666;   border: 1px solid #333;    /*padding: 10px 60px;*/    bottom:0px;    right:0px;    left:0px;    margin: 10px;}.truste-left {  margin: 0;}div.text-content {  display: flex;  flex-direction: column;  align-self: baseline;}#truste-consent-text {    /*font-family: "Source Sans Pro", sans-serif;    font-size: 16px;*/    color: #fff;    margin: 15px 0px;}.truste-footer {  padding-top: 15px;}#truste-consent-content {  margin: 0 auto;    padding: 0 10px;    max-width: 1366px;}.truste-consent-footer {  display: none;}.flex-container {  display: block;  align-items: center;  justify-content: space-between;}#truste-consent-buttons {  float: right;  margin: 10px 5px 10px 0px;}#truste-show-consent { background: #F30 !important;}#truste-consent-button, #truste-consent-required { background: #F30 !important; } /* MEDIA QUERIES */  @media screen and (max-width: 500px) {  #truste-consent-track {    padding: 5px 20px;  }  }@media screen and (max-width: 375px) {  .truste-buttonsColumn {    float: none;    margin: 0;  }  #truste-consent-button, #truste-consent-required, #truste-show-consent {    width: 100%;    margin-bottom: 5px;  }          #truste-consent-buttons {  float: none;  margin: 10px 0px 10px 0px;	}}</style><div id="truste-consent-track" style="display: block; opacity: 1;">  <div id="trustarc-banner-overlay"></div>  <div class="truste-consent-track-class">    <div id="truste-consent-content" class="truste-banner">        <div class="flex-container">          <div class="text-content">            <div id="truste-consent-text" class="truste-messageColumn">This site uses cookies and related technologies, as described in our privacy policy, for purposes that may include site operation, analytics, enhanced user experience, or advertising. You may choose to consent to our use of these technologies, or manage your own preferences.                       </div>          </div>          <div id="truste-consent-buttons" class="truste-buttonsColumn">            <button id="truste-show-consent" aria-haspopup="dialog">Manage Settings</button>      <button id="truste-consent-button">Accept</button>      <!-- <button id="truste-consent-required">Decline All</button> -->          </div>        </div>                <div style="clear:both;"></div>    </div>  </div></div><!--emptycomment--></div>
<!-- /TrustArc: Banner -->

                    
                        



        
        <link rel="stylesheet" href="https://cssjs.nejm.org/mmsWidgets.css"><script nonce="94d64844d6a5f20e-SJC" type="text/javascript" src="https://cssjs.nejm.org/mmsWidgets.js"></script><script nonce="94d64844d6a5f20e-SJC" type="text/javascript">var litSSO = {
    "ucid": null,
    "accessToken": null,
    "authState": null
};

var loginUri = '/action/doSsoLogin';
var logoutUri = '/action/doLogout';

displayAuthenticatedMessage();

mmsWidgets.init({
    clientId: 'qsra7g4d6jwdsgn3zbe6udp6r6278mr9',
    origin: 'NEJM',
    xdReceiver: 'http://www.nejm.org/pb-assets/sso/xd.html',
    isFullPageSignIn: isSignInPage(),
    debug: true,

    onReady: function (e) {
        console.log('<<<<<<<<<< client: received onReady event.');
        $("A.litSsoLogin").click(function (e) {
            if (!$(e.currentTarget).hasClass("noMd") || $(e.currentTarget).parents(".header_sm-login").length === 0) {
                mmsWidgets.signIn(e);
                return false;
            }
        });
        $("A.litSsoLogout").click(function (e) {
            mmsWidgets.signOut(e);
            return false;
        });
        $("A.litSsoCreate").click(function (e) {
            mmsWidgets.showRegModal(e);
            return false;
        });
        $("A.litSsoSubscribe").click(function (e) {
            if (!$(e.currentTarget).hasClass("noMd") || $(e.currentTarget).parents(".header_sm-login").length === 0)
                location.assign(e.currentTarget.href);
            return false;
        });
        if (window.doSSOLogin) {
            window.setTimeout(function () {
                mmsWidgets.signIn();
            }, 500);
        }
        ;
    },
    onLoginSuccess: function (e) {
        console.log('<<<<<<<<<< client: received onLoginSuccess event.' + JSON.stringify(e));

        const redirectUrl = getRedirectUri(window.location.search, location.href.replace(window.location.origin, ""));
        document.cookie = "skipSso=; expires=; path=/";
        if (!litSSO.ucid || e.ucid != litSSO.ucid || !litSSO.accessToken || e.accessToken != litSSO.accessToken) {
            location.assign(loginUri + "?ucid=" + e.profile.ucid
                + "&accessToken=" + e.accessToken
                + "&authState=" + e.authState
                + "&email=" + encodeURIComponent(e.profile.email)
                + "&givenNames=" + e.profile.givenName
                + "&surname=" + e.profile.familyName
                + "&redirectUri=" + redirectUrl
                + "&uccLastUpdatedDate=" + encodeURIComponent(e.profile.uccLastUpdatedDate)
                + "&rememberMe=" + e.rememberMe);
        }
    },
    onAuthStateReady: function (e) {
        console.log('<<<<<<<<<< client: received onAuthStateReady event.' + JSON.stringify(e))

        var isMmsSsoAuthenticated = (e.authState && e.authState !== "anonymous" && e.authState !== "lead" && e.ucid && e.accessToken) ? true : false;
        var isLitSsoAuthenticated = (litSSO.ucid && litSSO.accessToken && litSSO.authState) ? true : false;
        console.log('           isMmsSsoAuthenticated=' + isMmsSsoAuthenticated + ', isLitSsoAuthenticated=' + isLitSsoAuthenticated);

        if (isMmsSsoAuthenticated) {
            if (!isLitSsoAuthenticated || litSSO.ucid != e.ucid) {
                if (window.skipSso)
                    document.cookie = "skipSso=" + encodeURIComponent(e.ucid) + "; expires=; path=/";

                if (!window.skipSso && document.cookie.indexOf("skipSso=" + encodeURIComponent(e.ucid)) == -1) {
                    location.assign(loginUri + "?ucid=" + e.profile.ucid
                        + "&accessToken=" + e.accessToken
                        + "&authState=" + e.authState
                        + "&email=" + encodeURIComponent(e.profile.email)
                        + "&givenNames=" + e.profile.givenName
                        + "&surname=" + e.profile.familyName
                        + "&redirectUri=" + location.href.replace(window.location.origin, "")
                        + "&uccLastUpdatedDate=" + encodeURIComponent(e.profile.uccLastUpdatedDate)
                        + "&rememberMe=" + e.rememberMe);
                } else
                    console.log('**** client: Skip SSO ****');
            } else if (e.accessToken !== litSSO.accessToken)
                updateLitSession(e.ucid, e.accessToken);
        } else if (isLitSsoAuthenticated)
            location.assign(logoutUri + "?redirectUri=" + encodeURIComponent(location.href.replace(window.location.origin, "")));

        if (document.getElementById("ssoerr-ucid"))
            document.getElementById("ssoerr-ucid").innerHTML = e.ucid;

    },
    onRegistrationSuccess: function (e) {
        console.log('<<<<<<<<<< client: received onRegistrationSuccess event.' + JSON.stringify(e))
        document.cookie = "skipSso=; expires=; path=/";
        var uccLastUpdatedDate = '1812-01-01 00:00:00.1 +0000';
        location.assign(loginUri + "?ucid=" + e.profile.ucid
            + "&accessToken=" + e.accessToken
            + "&authState=" + e.authState
            + "&email=" + encodeURIComponent(e.profile.email)
            + "&givenNames=" + e.profile.givenName
            + "&surname=" + e.profile.familyName
            + "&redirectUri=" + location.href.replace(window.location.origin, "")
            + "&uccLastUpdatedDate=" + encodeURIComponent(uccLastUpdatedDate)
            + "&rememberMe=" + e.rememberMe);
    },
    onLogoutSuccess: function (e) {
        console.log('<<<<<<<<<< client: received onLogoutSuccess event.' + JSON.stringify(e))

        document.cookie = "skipSso=; expires=; path=/";
        location.assign(logoutUri);
    },
});

function updateLitSession(ucid, accessToken) {
    $.ajax({
        method: "GET",
        url: "/action/doSsoSessionUpdate",
        data: {ucid: ucid, accessToken: accessToken},
        success: function (callData, textStatus) {
            console.log("Session accessToken updated")
            litSSO.accessToken = accessToken
        },
        error: function (jqXHR, textStatus, errorThrown) {
            alert("Your website session has expired and will now be reset.")
            console.log("Session update failed: " + jqXHR.responseText)
            location.assign(logoutUri + "?redirectUri=" + encodeURIComponent(location.href.replace(window.location.origin, "")));
        }
    });
}

function getRedirectUri(location, fallback) {
    const urlParams = new URLSearchParams(location);
    const uriUrlParam = urlParams.get('uri');
    if (uriUrlParam)
        return encodeURIComponent(uriUrlParam);
    else
        return encodeURIComponent(fallback);
}

function isSignInPage() {
    return document.querySelector('meta[name="pbContext"]').content.indexOf("customPage:string:/sign-in") > -1;
}

function displayAuthenticatedMessage() {
    if (isSignInPage()) {
        var div = document.getElementById('authMsg');
        if ("" === "PersonUser") {
            div.innerHTML += "You are already signed in to " + window.location.origin.replace("https://", "").replace("www.", "") + ". If you would like to sign in with a different email address, please use the form below.";
        } else {
            div.innerHTML = '';
        }
    }
}</script>

                    
                
            </div>
        </div><iframe name="trustarc_notice" id="trustarcNoticeFrame" title="Trustarc Cross-Domain Consent Frame" src="https://consent.trustarc.com/get?name=crossdomain.html&amp;domain=nejm.com" style="display: none;"></iframe><script src="https://consent.trustarc.com/asset/notice.js/v/v1.7-1576" async="async" crossorigin="" importance="high" nonce="94d64844d6a5f20e-SJC"></script>
        


        
        

            <script src="/products/mms-nextgen/mms/releasedAssets/js/build.lazyload.bundle-c8c4e7d20884f81951fb.js" nonce="94d64844d6a5f20e-SJC"></script>

        <script src="/products/mms-nextgen/mms/releasedAssets/js/main.bundle-3819a02ec7bc2aca01ec.js" nonce="94d64844d6a5f20e-SJC"></script>
        


<div id="adModal" tabindex="-1" aria-labelledby="exampleModalCenterTitle" class="ng-modal modal" style="display: none;" aria-hidden="true">
    <div role="document" class="ng-modal_dialog modal-dialog modal-dialog-centered ad-promoLayer">
        <div class="ng-modal_content modal-content p-0">
            <div class="ng-modal_body modal-body m-0">
                <div class="ng-modal_header modal-header">
                    <button type="button" data-dismiss="modal" aria-label="Close" class="ng-modal_close close"><span aria-hidden="true"><span class="icon-component "><svg aria-hidden="true"><use xlink:href="/specs/products/mms-nextgen/mms/releasedAssets/icons/icons.spritemap.svg#close"></use></svg></span></span>
                    </button>
                </div>

            </div>
        </div>
    </div>
</div>
<script type="text/javascript" src="/wro/1125813~product.js" nonce="94d64844d6a5f20e-SJC"></script>








<input id="showLayers" type="hidden" value="true">

    <!-- placeholder id=null, description=ad-layer-config -->
    <!-- placeholder id=null, description=ad-lauchLayer-1 -->
    <!-- placeholder id=null, description=ad-layer-promo-1 -->

    <input id="firstAd" type="hidden" value="">
    <input id="secondAd" type="hidden" value="">

    
    
    

    
    
    <input id="IsInstitution" type="hidden" value="false">
    <input id="IsSubscriber" type="hidden" value="true">
    <input id="CustomerId" type="hidden" value="">


<script type="text/javascript" nonce="94d64844d6a5f20e-SJC">
  jQuery.extend(jQuery.mmsLayers.config, {
    rules: [{
      "name": "Launch Layer Ad",
      "frequency": 30,
      "showOnPage": 5,
      "enabled": false,
      "cookieName": "MarketingLaunchLayer",
      "styleClass": "launchLayer"
    },
    {
      "name": "Promo Layer Ad",
      "frequency": 14,
      "enabled": true,
      "cookieName": "PromoLayer",
      "styleClass": "promoLayer"
    },
    {
      "name": "Interstitial Ad",
      "frequency": 7,
      "enabled": false,
      "cookieName": "InterstitialAd"
    },
    {
      "name": "Iperceptions - Ion Layer Ad",
      "showOnPage": 1,
      "enabled": false,
      "test": false,
      "cookieName":"Iperceptions"
    }]
 });
</script>

















    <script type="text/javascript" nonce="94d64844d6a5f20e-SJC">
        $(document).ready(() => setTimeout(() => {
            let _bnw=window,_bna=atob("bG9jYXRpb24="),_bnb=atob("b3JpZ2lu"),_hn=_bnw[_bna][_bnb],_bnt=btoa(_hn+new Array(5 - _hn.length % 4).join(" "));
            $.get("/resource/lodash?t="+_bnt);
        },4000));
    </script>












    

    
    

    
    
    
    
        
            
                <script type="text/javascript" src="/wro/1125813~article-metrics.js" nonce="94d64844d6a5f20e-SJC"></script>
            
            
            
        
    




<script type="text/javascript" src="/wro/1125813~full-text-analytics.js" nonce="94d64844d6a5f20e-SJC"></script>


    


            <script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94d64844cce7f20e',t:'MTc0OTUzMTM4Ny4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94d64844cce7f20e&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;f55e150c7a5f4a4ab9ef82f4a1c9c43c&quot;}" crossorigin="anonymous"></script><div style="display:none;" id="signInEmbedded" class="ucc-fullpage-signin ucc-widgets-nejm-next">
  <div class="capture_header">
    <h1>Sign In</h1>
  </div>    
  <div class="capture_signin">
    {* #signInForm *}
    <div class="ucc-form-inner-wrapper">
      {* signInEmailAddress *}
      {* currentPassword *}
      <div style="padding-top:10px;">
          <div class="capture_form_item capture_form_rememberme">
          <label class="ucc-form-checkbox">                
            <input checked="checked" id="ucc-remember-me-cb-embedded" name="ucc-remember-me-cb-embedded" type="checkbox" value="true">
            <span>Remember Me</span>
          </label>                           
          </div>
          <div class="capture_form_item capture_form_forgotpwd">
            <a role="button" class="ucc-anchor-button" data-capturescreen="forgotPassword">Forgot your password?</a>
          </div>
      </div>
      <div class="capture_form_item">      
        <button id="ucc-sign-in-submit-embedded" class="capture_primary capture-full-width-btn" type="submit">Sign In</button>
      </div>
      <div class="capture_form_item capture_form_footer">
        <div class="ucc-no-acct-msg">Don't have an account?</div>
        <div class="ucc-no-acct-msg-2">
          <a role="button" class="ucc-anchor-button" onclick="mmsWidgets.closeSignIn(event); mmsWidgets.showRegAuto(event);">Create Account</a>
          <span></span><a href="https://store.nejm.org/nejm?promo=ONF4NRS1">Subscribe</a>
        </div>
      </div>
      <div class="capture_form_item capture_form_submit">
        
      </div>      
    </div>
    {* /signInForm *}
  </div>
</div><div style="display:none;" id="forgotPassword" class="ucc-widgets-nejm-next">
  <div class="capture_header">
    <h1>Forgot Password</h1>
  </div>   
  <div class="ucc-form-inner-wrapper">
  <p style="color:black">Enter the email address associated with your account then click Continue. We will email you a link to a page where you can easily create a new password.</p>
  </div> 
  <div class="capture_forgotpassword">
    {* #forgotPasswordForm *}      
      <div class="ucc-form-inner-wrapper">
        {* forgotPasswordEmailAddress *}
        {* captcha *}
        <div class="capture_form_item" style="padding-top: 20px;">        
          <input id="ucc-widget-forgotpwd-submit" value="Continue" type="submit" class="capture_btn capture_primary capture-full-width-btn">
        </div>
      </div>
    {* /forgotPasswordForm *}
  </div>
  </div><div style="display:none;" id="forgotPasswordSuccess" class="ucc-widgets-nejm-next">
  <div class="capture_header">
    <h1>Create New Password</h1>
  </div>    
  <div class="capture_forgotpassword">
      <div class="ucc-form-inner-wrapper">
        <p style="padding-top: 30px; padding-bottom: 30px;">We've sent an email with instructions to create a new password. Your existing password has not been changed.</p>        
        <div class="capture_form_item">      
          <button class="capture_btn capture_primary capture-full-width-btn" onclick="janrain.capture.ui.modal.close()">CLOSE</button>
        </div>
      </div>
  </div>
  </div><div style="display:none;" id="resetPassword" class="ucc-widgets-nejm-next">
      <div class="row">      
        <div class="col-md-8 col-md-offset-2 reset-pwd-info"><p>To reset your password, enter a new password twice and click the 'Reset Password' button.</p></div>
      </div>
      <div class="col-md-4 col-md-offset-4">
        {* #changePasswordFormNoAuth *}
          {* newPassword *}
          {* newPasswordConfirm *}     
          <div class="capture_form_item">
            <input value="RESET PASSWORD" type="submit" class="capture_btn capture_primary capture-full-width-btn">
          </div>
          {* /changePasswordFormNoAuth *}
      </div>
    </div><div style="display:none;" id="resetPasswordSuccess">
        <div class="reset-pwd-info">
          <div class="reset-pwd-success-welcome"></div>
          <p class="reset-pwd-success-msg">Your password has been reset. You will need to <a href="/signin" onclick="mmsWidgets.signIn(event)">sign in</a> again using your new password to access site content and features.</p>
          <p class="reset-pwd-success-nav"></p>
        </div>      
    </div><div style="display:none;" id="resetPasswordRequestCode" class="ucc-widgets-nejm-next">
        {* #resetPasswordForm *}
        <div class="capture_form_error">The link that you followed to reset your password has expired.</div>
          <div class="col-md-8 col-md-offset-2">
            <p class="reset-pwd-info">Enter the email address associated with your account then click Continue. We will email you a link to a page where you can easily create a new password.</p>
          </div>
          <div class="col-md-4 col-md-offset-4">
            {* signInEmailAddress *}
            <div class="capture_form_item">
              <input value="continue" type="submit" class="capture_btn capture_primary capture-full-width-btn">
            </div>
          </div>
        {* /resetPasswordForm *}
    </div><div style="display:none;" id="resetPasswordRequestCodeSuccess">
      <div class="reset-pwd-info">
        <p>If the address matches an existing account, you will receive an email with instructions to reset your password.</p>
        <p class="reset-pwd-success-nav"></p>
      </div>
  </div>

        
        
    

<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">â</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">Moreâ¦</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><wam-jdvfxldzsaij></wam-jdvfxldzsaij><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="janrainEngageEmbed" style="display: none;"><div class="janrainContent" style="width: 380px !important; height: 156px !important; padding-left: 5px !important; padding-right: 5px !important; box-sizing: content-box !important; background-color: rgb(255, 255, 255) !important; border: 1px solid rgb(192, 192, 192) !important; border-radius: 10px !important; overflow: hidden !important; position: relative !important;"><div id="janrainView"><div class="janrainHeader" style="background-color: rgb(255, 255, 255) !important; font-size: 14px !important; color: rgb(102, 102, 102) !important; position: relative !important; height: 20px !important; left: -5px !important; padding: 5px 5px 0px !important; white-space: nowrap !important; width: 380px !important; z-index: 100 !important; border-radius: 10px 10px 0px 0px !important; text-align: left !important;"><div style="padding-left: 1px !important; font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important;">Sign in using your account with</div></div><div id="janrainProviderPages" style="padding-top: 5px; left: 5px; position: absolute;"><div class="janrainPage" pageindex="0"><ul class="providers" id="janrainProviders_0" style="float: left !important; list-style-type: none !important; margin: 0px !important; padding: 0px !important;"><li id="janrain-aol" role="button" style="list-style: none !important; height: 30px !important; width: 185px !important; margin-top: 0px !important; margin-right: 5px !important; margin-bottom: 5px !important; position: relative !important; border: 1px solid rgb(204, 204, 204) !important; border-radius: 5px !important; cursor: pointer !important; white-space: nowrap !important; overflow: hidden !important; background-image: -webkit-linear-gradient(bottom, rgb(238, 238, 238), rgb(255, 255, 255)) !important;"><a tabindex="1" href="javascript:void(0);" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; font-size: 12px !important; line-height: 14px !important; margin-left: auto !important; margin-right: auto !important; text-decoration: none !important; display: block !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; width: auto !important;"><span class="janrain-provider-icon-24 janrain-provider-icon-aol" style="margin-top: 3px !important; background-color: transparent !important;"></span><span class="janrain-provider-text-color-aol" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; cursor: pointer !important; margin-left: 7px !important; text-align: left !important; margin-top: 9px !important; vertical-align: top !important; display: inline-block !important;">AOL</span></a></li><li id="janrain-openid" role="button" style="list-style: none !important; height: 30px !important; width: 185px !important; margin-top: 0px !important; margin-right: 5px !important; margin-bottom: 5px !important; position: relative !important; border: 1px solid rgb(204, 204, 204) !important; border-radius: 5px !important; cursor: pointer !important; white-space: nowrap !important; overflow: hidden !important; background-image: -webkit-linear-gradient(bottom, rgb(238, 238, 238), rgb(255, 255, 255)) !important;"><a tabindex="3" href="javascript:void(0);" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; font-size: 12px !important; line-height: 14px !important; margin-left: auto !important; margin-right: auto !important; text-decoration: none !important; display: block !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; width: auto !important;"><span class="janrain-provider-icon-24 janrain-provider-icon-openid" style="margin-top: 3px !important; background-color: transparent !important;"></span><span class="janrain-provider-text-color-openid" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; cursor: pointer !important; margin-left: 7px !important; text-align: left !important; margin-top: 9px !important; vertical-align: top !important; display: inline-block !important;">OpenID</span></a></li></ul><ul class="providers" id="janrainProviders_1" style="float: left !important; list-style-type: none !important; margin: 0px !important; padding: 0px !important;"><li id="janrain-yahoo" role="button" style="list-style: none !important; height: 30px !important; width: 185px !important; margin-top: 0px !important; margin-right: 5px !important; margin-bottom: 5px !important; position: relative !important; border: 1px solid rgb(204, 204, 204) !important; border-radius: 5px !important; cursor: pointer !important; white-space: nowrap !important; overflow: hidden !important; background-image: -webkit-linear-gradient(bottom, rgb(238, 238, 238), rgb(255, 255, 255)) !important;"><a tabindex="2" href="javascript:void(0);" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; font-size: 12px !important; line-height: 14px !important; margin-left: auto !important; margin-right: auto !important; text-decoration: none !important; display: block !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; width: auto !important;"><span class="janrain-provider-icon-24 janrain-provider-icon-yahoo" style="margin-top: 3px !important; background-color: transparent !important;"></span><span class="janrain-provider-text-color-yahoo" style="font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; cursor: pointer !important; margin-left: 7px !important; text-align: left !important; margin-top: 9px !important; vertical-align: top !important; display: inline-block !important;">Yahoo!</span></a></li></ul></div></div><div style="background-color: rgb(255, 255, 255) !important; width: 380px !important; height: 10px !important; position: absolute !important; left: 0px !important; padding-left: 5px !important; padding-right: 5px !important; padding-bottom: 5px !important; bottom: 0px !important; font-size: 10px !important; text-align: left !important; color: rgb(102, 102, 102) !important; font-family: Helvetica, &quot;lucida grande&quot;, Verdana, sans-serif !important; border-radius: 0px 0px 10px 10px !important;"><div style="padding-left: 1px !important;"><a href="http://janrain.com/products/engage/social-login?utm_source=mms.rpxnow.com&amp;utm_medium=Partner&amp;utm_campaign=attribution" target="_blank">Social Login by Janrain</a></div></div></div></div></div><script type="text/javascript" src="https://api.altmetric.com/v1/internal-556fdf0f/doi/10.1056/nejmoa1908753?callback=_altmetric.embed_callback&amp;domain=www.nejm.org&amp;cache_until=1-10"></script><iframe name="captureIFrame_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-transactionid="en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" id="captureIFrame_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" src="about:blank" class="" data-captureiframeloadeventbound="yes" style="display: none;"></iframe><div id="janrainModalOverlay" style="inset: 0px; position: fixed; background-color: rgb(0, 0, 0); opacity: 0.4; display: none; z-index: 1000;"></div><div id="janrainModal" style="width: 380px; height: 131px; position: absolute; z-index: 1000; display: none;"><a href="#" class="janrain_modal_closebutton" style="position: absolute; cursor: pointer; z-index: 1000;">X</a><div style="display: block;" id="signIn" class="ucc-widgets-nejm-next janrain-capture-ui capture-ui-content capture_screen_container" data-capturescreenname="signIn" data-captureventadded="true">
    <div class="capture_header">
      <h1>Sign In</h1>
    </div>    
    <div class="capture_signin">
      <form id="capture_signIn_signInForm" name="signInForm" data-capturefield="signInForm" action="https://mms.us.janraincapture.com/widget/traditional_signin.jsonp" class="capture_form capture_signInForm" method="POST" novalidate="novalidate" data-transactionid="en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" target="captureIFrame_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" accept-charset="UTF-8" next="{&quot;noop&quot;:&quot;&quot;}"><div id="capture_signIn_signInForm_defaultSavedProfileMessage"></div><div id="capture_signIn_signInForm_errorMessages"></div><input id="capture_signIn_utf8" data-capturefield="undefined" value="â" type="hidden" class="capture_utf8" name="utf8"><input id="capture_signIn_screen_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="signIn" type="hidden" class="capture_screen_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="capture_screen"><input id="capture_signIn_js_version_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="d445bf4" type="hidden" class="capture_js_version_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="js_version"><input id="capture_signIn_transactionId_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" type="hidden" class="capture_transactionId_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="capture_transactionId"><input id="capture_signIn_form_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="signInForm" type="hidden" class="capture_form_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="form"><input id="capture_signIn_flow_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="standard" type="hidden" class="capture_flow_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="flow"><input id="capture_signIn_client_id_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="qsra7g4d6jwdsgn3zbe6udp6r6278mr9" type="hidden" class="capture_client_id_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="client_id"><input id="capture_signIn_redirect_uri_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="https://www.nejm.org" type="hidden" class="capture_redirect_uri_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="redirect_uri"><input id="capture_signIn_response_type_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="code" type="hidden" class="capture_response_type_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="response_type"><input id="capture_signIn_flow_version_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="20240514181256512029" type="hidden" class="capture_flow_version_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="flow_version"><input id="capture_signIn_settings_version_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="" type="hidden" class="capture_settings_version_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="settings_version"><input id="capture_signIn_locale_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="en-US" type="hidden" class="capture_locale_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="locale"><input id="capture_signIn_recaptcha_version_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" data-capturefield="undefined" value="2" type="hidden" class="capture_recaptcha_version_en2rrm1ft8b1cu8to7p38f0por99uqsbxjoo8ukw" name="recaptchaVersion">
      <div class="ucc-form-inner-wrapper">
        <div id="capture_signIn_form_item_signInEmailAddress" class="capture_form_item capture_email capture_form_item_signInEmailAddress" data-capturefield="undefined"><label for="capture_signIn_signInEmailAddress">Email Address</label><input id="capture_signIn_signInEmailAddress" data-capturefield="signInEmailAddress" value="" type="email" class="capture_signInEmailAddress capture_required capture_text_input" placeholder="Email Address" name="signInEmailAddress"><div class="capture_tip" style="display:none;"></div><div class="capture_tip_validating" data-elementname="signInEmailAddress">Validating</div><div class="capture_tip_error" data-elementname="signInEmailAddress"></div></div>
        <div id="capture_signIn_form_item_currentPassword" class="capture_form_item capture_password capture_form_item_currentPassword" data-capturefield="undefined"><label for="capture_signIn_currentPassword">Password</label><input id="capture_signIn_currentPassword" data-capturefield="currentPassword" value="" type="password" class="capture_currentPassword capture_required capture_text_input" placeholder="Password" name="currentPassword"><div class="capture_tip" style="display:none;"></div><div class="capture_tip_validating" data-elementname="currentPassword">Validating</div><div class="capture_tip_error" data-elementname="currentPassword"></div></div>
        <div style="padding-top:10px;">
            <div class="capture_form_item capture_form_rememberme">
                <label class="ucc-form-checkbox">                
                    <input checked="checked" id="ucc-remember-me-cb" name="ucc-remember-me-cb" type="checkbox" value="true">
                    <span>Remember Me</span>
                </label>          
            </div>
            <div class="capture_form_item capture_form_forgotpwd">
              <a role="button" class="ucc-anchor-button" data-capturescreen="forgotPassword">Forgot your password?</a>
            </div>
        </div>
        <div class="capture_form_item">
          <button id="ucc-sign-in-submit" class="capture_primary capture-full-width-btn" type="submit">Sign In</button>
        </div>
       
        <div class="capture_form_item">
          <div class="ucc-no-acct-msg">Don't have an account?</div>
          <div class="ucc-no-acct-msg-2" id="signInModalCreateAccountLink">
          <a role="button" class="ucc-anchor-button" onclick="mmsWidgets.closeSignIn(event); mmsWidgets.showRegAuto(event);">Create Account</a>
            <span></span><a href="https://store.nejm.org/nejm?promo=ONF4NRS1">Subscribe</a>
          </div>

          <div class="ucc-no-acct-msg-2" id="signInModalCloseLink" style="display:none">
          <a role="button" class="ucc-anchor-button" onclick="mmsWidgets.closeSignIn(event);">Create Account</a>
            <span></span><a href="https://store.nejm.org/nejm?promo=ONF4NRS1">Subscribe</a>
          </div>

        </div>
     
        <div class="capture_form_item capture_form_submit">
          
        </div>      
      </div>
      </form>
    </div>
  </div></div></body></html>